
<!DOCTYPE html>
<html lang="en">
   <head>
      <!-- basic -->
      <meta charset="utf-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- mobile metas -->
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <meta name="viewport" content="initial-scale=1, maximum-scale=1">
      
      <!-- site metas -->
      <title>Data Sources</title>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link rel="icon" href="images/logo.png" type="image/x-icon">
      <meta name="keywords" content="COVID-19, wastewater, measurement, monitoring, surveillance, public health, environmental monitoring, water analysis, viral RNA, SARS-CoV-2,SARS-nCoV-2, global, countries, data, analysis, pandemic, wastewater-based epidemiology, geo-map">
      <meta name="description" content="Data Sources of Visualizations">
      <meta name="author" content="Daniel Dynesius">
      <!-- bootstrap css -->
      <link rel="stylesheet" href="css/bootstrap.min.css">
      <!-- style css -->
      <link rel="stylesheet" href="css/style.css">
      <!-- Responsive-->
      <link rel="stylesheet" href="css/responsive.css">
      <!-- fevicon -->
      <link rel="icon" href="images/fevicon.png" type="image/gif" />
      <!-- Scrollbar Custom CSS -->
      <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
      <!-- Tweaks for older IEs-->
      <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
      <style>
            @media (max-width: 767px) {
            /* Styles for screens smaller than 768px (mobile) */
            .menu-area-main {
                flex-direction: column;
            }
        
            .menu-area-main li {
                text-align: center;
            }
            }
    </style>      
      <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script><![endif]-->
   </head>
   <!-- body -->
   <body class="main-layout">
      <!-- loader  -->
      <div class="loader_bg">
         <div class="loader"><img src="images/loading.gif" alt="#" /></div>
      </div>
      <!-- end loader --> 
      <!-- header -->
      <header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                    </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo"> <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a> </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main d-flex">
                                    <li class=""> <a href="index.html">Geo Map</a> </li>
                                    <li class=""> <a href="trends.html">Trend Graphs</a> </li>
                                    <li class=""> <a href="predictions_tab.html">Predictive</a> </li>
                                    <li class="active"> <a href="new_research.html">Research News</a> </li>
                                    <li class=""> <a href="data_sources.html">Data Sources</a> </li>
                                    <li class=""> <a href="about.html">About</a> </li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
         <!-- end header inner --> 
      </header>
      <!-- end header -->
      <div class="brand_color">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="titlepage">
                        <h2>Research News</h2>
                        <br>
                        <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                        Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>

                    </div>
                </div>
            </div>
        </div>

    </div>

    <!-- contact -->
<p>


    <br><br><br>

        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>COVID-19 immune signatures in Uganda persist in HIV co-infection and diverge by pandemic phase</h3>
            <h3 id="section1"><a href="#section1" style="text-decoration: none; color: inherit;">ü§ñ COVID-19 Immune Signatures in Uganda: Co-Infection with HIV & Pandemic Phase Differences</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-17</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study in Uganda investigated the immunopathology of COVID-19 across multiple phases of the pandemic, finding that severe cases were characterized by multifaceted host protein impairment and redox imbalance, with key roles for IL-7, IL-15, and lymphotoxin-Œ± in respiratory failure.</p>
            <br><strong>Abstract</strong><br>
            <p>Little is known about the pathobiology of SARS-CoV-2 infection in sub-Saharan Africa, where severe COVID-19 fatality rates are among the highest in the world and the immunological landscape is unique. In a prospective cohort study of 306 adults encompassing the entire clinical spectrum of SARS-CoV-2 infection in Uganda, we profile the peripheral blood proteome and transcriptome to characterize the immunopathology of COVID-19 across multiple phases of the pandemic. Beyond the prognostic importance of myeloid cell-driven immune activation and lymphopenia, we show that multifaceted impairment of host protein synthesis and redox imbalance define core biological signatures of severe COVID-19, with central roles for IL-7, IL-15, and lymphotoxin-Œ± in COVID-19 respiratory failure. While prognostic signatures are generally consistent in SARS-CoV-2/HIV-coinfection, type I interferon responses uniquely scale with COVID-19 severity in persons living with HIV. Throughout the pandemic, COVID-19 severity peaked during phases dominated by A.23/A.23.1 and Delta B.1.617.2/AY variants. Independent of clinical severity, Delta phase COVID-19 is distinguished by exaggerated pro-inflammatory myeloid cell and inflammasome activation, NK and CD8+T cell depletion, and impaired host protein synthesis. Combining these analyses with a contemporary Ugandan cohort of adults hospitalized with influenza and other severe acute respiratory infections, we show that activation of epidermal and platelet-derived growth factor pathways are distinct features of COVID-19, deepening translational understanding of mechanisms potentially underlying SARS-CoV-2-associated pulmonary fibrosis. Collectively, our findings provide biological rationale for use of broad and targeted immunotherapies for severe COVID-19 in sub-Saharan Africa, illustrate the relevance of local viral and host factors to SARS-CoV-2 immunopathology, and highlight underemphasized yet therapeutically exploitable immune pathways driving COVID-19 severity.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45204-3" target="_blank">https://www.nature.com/articles/s41467-024-45204-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 rapidly evolves lineage-specific phenotypic differences when passaged repeatedly in immune-na√Øve mice</h3>
            <h3 id="section2"><a href="#section2" style="text-decoration: none; color: inherit;">ü§ñ SARS-CoV-2 Adapts Fast in Mice with No Prior Exposure</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-16</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research shows that the persistence of SARS-CoV-2 despite vaccines and herd immunity is partly due to viral evolution reducing vaccine and treatment efficacy, and that passaging unmodified B.1.351 (Beta) and B.1.617.2 (Delta) 20 times in K18-ACE2 mice drives human health-relevant evolution, with late-passage Delta demonstrating antibody resistance and interferon suppression co-occurring with a de novo spike S371F mutation, linked with both traits. This study demonstrates SARS-CoV-2's tendency to evolve with phenotypic consequences, its evolution varying by lineage, and suggests non-dominant quasi-species contribution.</p>
            <br><strong>Abstract</strong><br>
            <p>The persistence of SARS-CoV-2 despite the development of vaccines and a degree of herd immunity is partly due to viral evolution reducing vaccine and treatment efficacy. Serial infections of wild-type (WT) SARS-CoV-2 in Balb/c mice yield mouse-adapted strains with greater infectivity and mortality. We investigate if passaging unmodified B.1.351 (Beta) and B.1.617.2 (Delta) 20‚Äâtimes in K18-ACE2 mice, expressing the human ACE2 receptor, in a BSL-3 laboratory without selective pressures, drives human health-relevant evolution and if evolution is lineage-dependent. Late-passage virus causes more severe disease, at organism and lung tissue scales, with late-passage Delta demonstrating antibody resistance and interferon suppression. This resistance co-occurs with a de novo spike S371F mutation, linked with both traits. S371F, an Omicron-characteristic mutation, is co-inherited at times with spike E1182G per Nanopore sequencing, existing in different within-sample viral variants at others. Both S371F and E1182G are linked to mammalian GOLGA7 and ZDHHC5 interactions, which mediate viral-cell entry and antiviral response. This study demonstrates SARS-CoV-2‚Äôs tendency to evolve with phenotypic consequences, its evolution varying by lineage, and suggests non-dominant quasi-species contribution.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-05878-3" target="_blank">https://www.nature.com/articles/s42003-024-05878-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Proteomic and transcriptomic profiling of brainstem, cerebellum and olfactory tissues in early- and late-phase COVID-19</h3>
            <h3 id="section3"><a href="#section3" style="text-decoration: none; color: inherit;">ü§ñ COVID-19 Brainstem, Cerebellum & Olfactory Tissue Changes in Early & Late Stages</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-16</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research found that brainstem inflammation and altered transcriptional states in oligodendrocytes, microglia, and astrocytes may cause neurological symptoms in COVID-19 patients, even without the virus persisting in the central nervous system.</p>
            <br><strong>Abstract</strong><br>
            <p>Neurological symptoms, including cognitive impairment and fatigue, can occur in both the acute infection phase of coronavirus disease 2019 (COVID-19) and at later stages, yet the mechanisms that contribute to this remain unclear. Here we profiled single-nucleus transcriptomes and proteomes of brainstem tissue from deceased individuals at various stages of COVID-19. We detected an inflammatory type I interferon response in acute COVID-19 cases, which resolves in the late disease phase. Integrating single-nucleus RNA sequencing and spatial transcriptomics, we could localize two patterns of reaction to severe systemic inflammation, one neuronal with a direct focus on cranial nerve nuclei and a separate diffuse pattern affecting the whole brainstem. The latter reflects a bystander effect of the respiratory infection that spreads throughout the vascular unit and alters the transcriptional state of mainly oligodendrocytes, microglia and astrocytes, while alterations of the brainstem nuclei could reflect the connection of the immune system and the central nervous system via, for example, the vagus nerve. Our results indicate that even without persistence of severe acute respiratory syndrome coronavirus 2 in the central nervous system, local immune reactions are prevailing, potentially causing functional disturbances that contribute to neurological complications of COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41593-024-01573-y" target="_blank">https://www.nature.com/articles/s41593-024-01573-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Cyclosporine A in hospitalized COVID-19 pneumonia patients to prevent the development of interstitial lung disease: a pilot randomized clinical trial</h3>
            <h3 id="section4"><a href="#section4" style="text-decoration: none; color: inherit;">ü§ñ "Cyclosporine A Trial for Hospitalized COVID Pneumonia Patients: Preventing Interstitial Lung Disease"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-15</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study investigated if cyclosporine A, given alongside standard care, could prevent the development of post-COVID-19 interstitial lung disease in hospitalized patients but found no significant effect.</p>
            <br><strong>Abstract</strong><br>
            <p>Post-COVID-19 interstitial lung disease (ILD) is a new entity that frequently causes pulmonary fibrosis and can become chronic. We performed a single-center parallel-group open-label pilot randomized clinical trial to investigate the efficacy and safety of cyclosporine A (CsA) in the development of ILD in the medium term among patients hospitalized with COVID-19 pneumonia. Patients were randomized 1:1 to receive CsA plus standard of care or standard of care alone. The primary composite outcome was the percentage of patients without ILD 3¬†months after diagnosis of pneumonia and not requiring invasive mechanical ventilation (IMV) (response without requiring IMV). The key secondary composite outcomes were the percentage of patients who achieve a response requiring IMV or irrespective of the need for IMV, and adverse events. A total of 33 patients received at least one dose of CsA plus standard of care (n‚Äâ=‚Äâ17) or standard of care alone (n‚Äâ=‚Äâ16). No differences were found between the groups in the percentage of patients who achieved a response without requiring IMV or a response requiring IMV. A higher percentage of patients achieved a response irrespective of the need for IMV in the CsA plus standard of care group although the RR was almost significant 2.833 (95% CI, 0.908‚Äì8.840;p=‚Äâ0.057). No differences were found between the groups for adverse events. In hospitalized patients with COVID-19 pneumonia, we were unable to demonstrate that CsA achieved a significant effect in preventing the development of ILD. (EU Clinical Trials Register; EudraCT Number: 2020-002123-11; registration date: 08/05/2020).</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-54196-5" target="_blank">https://www.nature.com/articles/s41598-024-54196-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 infection status in corneal preservation solution and COVID-19 prevalence after corneal transplantation</h3>
            <h3 id="section5"><a href="#section5" style="text-decoration: none; color: inherit;">ü§ñ "Cornea Preservation Solution & COVID-19 After Corneal Transplant"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that corneas transplanted during the COVID-19 pandemic were all SARS-CoV-2 negative, and recipients did not develop COVID-19 infection after transplantation if the donor's nasopharyngeal swab test was negative.</p>
            <br><strong>Abstract</strong><br>
            <p>The potential risks associated with organs from COVID-19-infected donors were unclear. To determine the SARS-CoV-2 infection status of corneas transplanted during the COVID-19 pandemic, we performed a polymerase chain reaction (PCR) using the corneal preservation solution that was used for corneal transplantation. We also examined the postoperative health status of the recipients. This study included 144 transplants in 143 eyes. Ninety-nine eyes of imported corneas and 10 of the 14 corneas donated in the prefecture were PCR tested at our hospital, and all were SARS-CoV-2 negative. All corneal transplants were performed after confirming their SARS-CoV-2 negativity by a PCR using a corneal preservation solution at our hospital or a nasopharyngeal swab at a previous facility. Despite postoperative steroid administration, no patient developed COVID-19 infection until discharge. Hence, if the donor's nasopharyngeal swab test is SARS-CoV-2 negative, COVID-19 infection in the recipient due to corneal transplantation may be prevented. Since corneal transplant recipients are susceptible to infection due to prolonged steroid administration and are at high risk for severe diseases if infection occurs, SARS-CoV-2 detection testing using nasopharyngeal swabs in donors should be performed.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">https://www.nature.com/articles/s41598-024-53863-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Immunological mechanisms of the nucleocapsid protein in COVID-19</h3>
            <h3 id="section6"><a href="#section6" style="text-decoration: none; color: inherit;">ü§ñ Understanding Nucleocapsid Protein Role in COVID-19 Immune Response</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research investigates the SARS-CoV-2 nucleocapsid (N) protein's role in HEK293 cells, revealing its influence on apoptosis, interferon signaling, and cytokine production, potentially leading to targeted therapies for COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>The emergence of corona virus disease 2019 (COVID-19), resulting from Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has left an indelible mark on a global scale, causing countless infections and fatalities. This investigation delves into the role of the SARS-CoV-2 nucleocapsid (N) protein within the HEK293 cells, shedding light on its influence over apoptosis, interferon signaling, and cytokines production. The N gene was amplified, inserted into the pAdTrack-CMV vector, and then transfected to the HEK293 cells. Changes in the expression of IRF3, IRF7, IFN-Œ≤, BAK, BAX, and BCL-2 genes were evaluated. The levels of proinflammatory cytokines of IL-6, IL-12, IL-1Œ≤, and TNF-Œ± were also determined. The N protein exhibited an anti-apoptotic effect by modulating critical genes associated with apoptosis, including BAK, BAX, and BCL-2. This effect potentially prolonged the survival of infected cells. The N protein also played a role in immune evasion by suppressing the interferon pathway, evidenced by the downregulation of essential interferon regulatory factors of IRF3 and IRF7, and IFN-Œ≤ expression. The N protein expression led to a substantial increase in the production of proinflammatory cytokines of IL-6, IL-12, IL-1Œ≤, and TNF-Œ±. The N protein emerged as a versatile factor and was exerted over apoptosis, interferon signaling, and cytokine production. These findings carry potential implications for the development of targeted therapies to combat COVID-19 and mitigate its global health impact.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53906-3" target="_blank">https://www.nature.com/articles/s41598-024-53906-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Revealing the significance of IL-2 and IL-5 in SARS-CoV-2-specific T-cell responses in kidney transplant recipients</h3>
            <h3 id="section7"><a href="#section7" style="text-decoration: none; color: inherit;">ü§ñ Understanding IL-2 & IL-5 Role in COVID-19 T-cell Responses for Kidney Transplant Patients</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that a third COVID-19 vaccine dose improved immune response in kidney transplant recipients, who have weakened immune systems and often show poor response after primary mRNA vaccinations. The production of certain cytokines, proteins involved in the immune system's communication network, positively correlated with antibody levels, which protect against COVID-19. The results highlight the importance of considering multiple cytokines when assessing T-cell responses and tailoring vaccination strategies for individual patients.</p>
            <br><strong>Abstract</strong><br>
            <p>Kidney transplant recipients (KTRs) are at an increased risk of severe COVID-19 due to compromised immune responses. Although vaccination is critical in preventing severe disease, KTRs have attenuated vaccination-induced immune responses due to underlying kidney disease and immunosuppressive therapies. In this study, the effect of different COVID-19 booster strategies on SARS-CoV-2-specific T-cell responses was assessed in KTRs who showed a poor serological response after the first two mRNA-based primary vaccination doses. In these KTRs, a third vaccination dose led to an increase in antibody levels in the majority of patients. Production of IL-2 and IL-5 by SARS-CoV-2 specific T cells positively correlated with antibody levels, with stronger correlations compared to IFN-Œ≥ production, the ‚Äòtraditional‚Äô cytokine to measure T-cell responses. Our study underscores the significance a balanced T-cell cytokine response to achieve robust antibody responses in KTRs. Furthermore, we show that multiple cytokines to assess T-cell responses should be explored to identify individuals in need of tailored vaccination strategies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00015-7" target="_blank">https://www.nature.com/articles/s44298-024-00015-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Aptamers targeting SARS-CoV-2 nucleocapsid protein exhibit potential anti pan-coronavirus activity</h3>
            <h3 id="section8"><a href="#section8" style="text-decoration: none; color: inherit;">ü§ñ Shortened Title: "Aptamers Targeting SARS-CoV-2 Protein May Fight Pan-Coronaviruses"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research develops anti-pan-coronavirus ssDNA aptamers using SELEX, showing high affinity to SARS-CoV-2 nucleocapsid protein and excellent binding efficiency to other coronaviruses, potentially serving as an effective strategy for preventing infections by unknown coronaviruses and addressing global health threats.</p>
            <br><strong>Abstract</strong><br>
            <p>Emerging and recurrent infectious diseases caused by human coronaviruses (HCoVs) continue to pose a significant threat to global public health security. In light of this ongoing threat, the development of a broad-spectrum drug to combat HCoVs is an urgently priority. Herein, we report a series of anti-pan-coronavirus ssDNA aptamers screened using Systematic Evolution of Ligands by Exponential Enrichment (SELEX). These aptamers have nanomolar affinity with the nucleocapsid protein (NP) of Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and also show excellent binding efficiency to the N proteins of both SARS, MERS, HCoV-OC43 and -NL63 with affinity KDvalues of 1.31 to 135.36‚ÄânM. Such aptamer-based therapeutics exhibited potent antiviral activity against both the authentic SARS-CoV-2 prototype strain and the Omicron variant (BA.5) with EC50values at 2.00‚ÄânM and 41.08‚ÄânM, respectively. The protein docking analysis also evidenced that these aptamers exhibit strong affinities for N proteins of pan-coronavirus and other HCoVs (‚àí229E and -HKU1). In conclusion, we have identified six aptamers with a high pan-coronavirus antiviral activity, which could potentially serve as an effective strategy for preventing infections by unknown coronaviruses and addressing the ongoing global health threat.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01748-w" target="_blank">https://www.nature.com/articles/s41392-024-01748-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Asialoglycoprotein receptor 1 promotes SARS-CoV-2 infection of human normal hepatocytes</h3>
            <h3 id="section9"><a href="#section9" style="text-decoration: none; color: inherit;">ü§ñ ASGR1 boosts SARS-CoV-2 infection in human liver cells</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study reveals that Asialoglycoprotein receptor 1 (ASGR1) might be an alternative receptor for SARS-CoV-2, allowing the virus to infect liver cells even when ACE2 is not expressed at significant levels, which could help explain liver damage in COVID-19 patients.</p>
            <br><strong>Abstract</strong><br>
            <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes multi-organ damage, which includes hepatic dysfunction, as observed in over 50% of COVID-19 patients. Angiotensin I converting enzyme (peptidyl-dipeptidase A) 2 (ACE2) is the primary receptor for SARS-CoV-2 entry into host cells, and studies have shown the presence of intracellular virus particles in human hepatocytes that express ACE2, but at extremely low levels. Consequently, we asked if hepatocytes might express receptors other than ACE2 capable of promoting the entry of SARS-CoV-2 into cells. To address this question, we performed a genome-wide CRISPR-Cas9 activation library screening and found that Asialoglycoprotein receptor 1 (ASGR1) promoted SARS-CoV-2 pseudovirus infection of HeLa cells. In Huh-7 cells, simultaneous knockout ofACE2andASGR1prevented SARS-CoV-2 pseudovirus infection. In the immortalized THLE-2 hepatocyte cell line and primary hepatic parenchymal cells, both of which barely expressed ACE2, SARS-CoV-2 pseudovirus could successfully establish an infection. However, after treatment with ASGR1 antibody or siRNA targeting ASGR1, the infection rate significantly dropped, suggesting that SARS-CoV-2 pseudovirus infects hepatic parenchymal cells mainly through an ASGR1-dependent mechanism. We confirmed that ASGR1 could interact with Spike protein, which depends on receptor binding domain (RBD) and N-terminal domain (NTD). Finally, we also used Immunohistochemistry and electron microscopy to verify that SARS-CoV-2 could infect primary hepatic parenchymal cells. After inhibiting ASGR1 in primary hepatic parenchymal cells by siRNA, the infection efficiency of the live virus decreased significantly. Collectively, these findings indicate that ASGR1 is a candidate receptor for SARS-CoV-2 that promotes infection of hepatic parenchymal cells.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01754-y" target="_blank">https://www.nature.com/articles/s41392-024-01754-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Safety, immunogenicity and protective effectiveness of heterologous boost with a recombinant COVID-19 vaccine (Sf9 cells) in adult recipients of inactivated vaccines</h3>
            <h3 id="section10"><a href="#section10" style="text-decoration: none; color: inherit;">ü§ñ Title: Boosting Recombinant COVID-19 Vaccine for Adults Post Inactivated Vaccine - Safety, Immunogenicity, and Protection</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>A trial found that a heterologous booster dose using the recombinant COVID-19 vaccine (Sf9 cells) increased protection against various SARS-CoV-2 virus variants and caused no significant safety concerns.</p>
            <br><strong>Abstract</strong><br>
            <p>Vaccines have proven effective in protecting populations against COVID-19, including the recombinant COVID-19 vaccine (Sf9 cells), the first approved recombinant protein vaccine in China. In this positive-controlled trial with 85 adult participants (Sf9 cells group:n=‚Äâ44; CoronaVac group:n=‚Äâ41), we evaluated the safety, immunogenicity, and protective effectiveness of a heterologous boost with the Sf9 cells vaccine in adults who had been vaccinated with the inactivated vaccine, and found a post-booster adverse events rate of 20.45% in the Sf9 cells group and 31.71% in the CoronaVac group (p=‚Äâ0.279), within 28 days after booster injection. Neither group reported any severe adverse events. Following the Sf9 cells vaccine booster, the geometric mean titer (GMT) of binding antibodies to the receptor-binding domain of prototype SARS-CoV-2 on day 28 post-booster was significantly higher than that induced by the CoronaVac vaccine booster (100,683.37 vs. 9,451.69,p<‚Äâ0.001). In the Sf9 cells group, GMTs of neutralizing antibodies against pseudo SARS-CoV-2 viruses (prototype and diverse variants of concern [VOCs]) increased by 22.23‚Äì75.93 folds from baseline to day 28 post-booster, while the CoronaVac group showed increases of only 3.29‚Äì10.70 folds. Similarly, neutralizing antibodies against live SARS-CoV-2 viruses (prototype and diverse VOCs) increased by 68.18‚Äì192.67 folds on day 14 post-booster compared with the baseline level, significantly greater than the CoronaVac group (19.67‚Äì37.67 folds). A more robust Th1 cellular response was observed with the Sf9 cells booster on day 14 post-booster (mean IFN-Œ≥+ spot-forming cells per 2‚Äâ√ó‚Äâ105peripheral blood mononuclear cells: 26.66 vs. 13.59). Protective effectiveness against symptomatic COVID-19 was approximately twice as high in the Sf9 cells group compared to the CoronaVac group (68.18% vs. 36.59%,p=‚Äâ0.004). Our study findings support the high protective effectiveness of heterologous boosting with the recombinant COVID-19 vaccine (Sf9 cells) against symptomatic COVID-19 of diverse SARS-CoV-2 variants of concern, while causing no apparent safety concerns.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01751-1" target="_blank">https://www.nature.com/articles/s41392-024-01751-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 spike protein-ACE2 interaction increases carbohydrate sulfotransferases and reduces N-acetylgalactosamine-4-sulfatase by p38 MAPK</h3>
            <h3 id="section11"><a href="#section11" style="text-decoration: none; color: inherit;">ü§ñ SARS-CoV-2 Spike Protein Interacts with ACE2, Altering Carbohydrate Sulfotransferases and N-Acetylgalactosamine-4-Sulfatase via p38 MAPK</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research suggests that the COVID-19 spike protein causes changes in lung cells leading to the accumulation of chondroitin sulfates, which are linked to fibrotic lung conditions and damage; inhibiting p38-MAPK may be beneficial in treating COVID-19 infections.</p>
            <br><strong>Abstract</strong><br>
            <p>Immunostaining in lungs of patients who died with COVID-19 infection showed increased intensity and distribution of chondroitin sulfate and decline in N-acetylgalactostamine-4-sulfatase (Arylsulfatase B; ARSB). To explain these findings, human small airway epithelial cells were exposed to the SARS-CoV-2 spike protein receptor binding domain (SPRBD) and transcriptional mechanisms were investigated. Phospho-p38 MAPK and phospho-SMAD3 increased following exposure to the SPRBD, and their inhibition suppressed the promoter activation of the carbohydrate sulfotransferases CHST15 and CHST11, which contributed to chondroitin sulfate biosynthesis. Decline in ARSB was mediated by phospho-38 MAPK-induced N-terminal Rb phosphorylation and an associated increase in Rb-E2F1 binding and decline in E2F1 binding to the ARSB promoter. The increases in chondroitin sulfotransferases were inhibited when treated with phospho-p38-MAPK inhibitors, SMAD3 (SIS3) inhibitors, as well as antihistamine desloratadine and antibiotic monensin. In the mouse model of carrageenan-induced systemic inflammation, increases in phospho-p38 MAPK and expression of CHST15 and CHST11 and declines in DNA-E2F binding and ARSB expression occurred in the lung, similar to the observed effects in this SPRBD model of COVID-19 infection. Since accumulation of chondroitin sulfates is associated with fibrotic lung conditions and diffuse alveolar damage, increased attention to p38-MAPK inhibition may be beneficial in ameliorating Covid-19 infections.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01741-3" target="_blank">https://www.nature.com/articles/s41392-024-01741-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Identification of new pharmacophore against SARS-CoV-2 spike protein by multi-fold computational and biochemical techniques</h3>
            <h3 id="section12"><a href="#section12" style="text-decoration: none; color: inherit;">ü§ñ Finding New Drug Target Against COVID Spike Protein Using Computer and Lab Methods</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-13</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study identified promising therapeutic candidates, such as acetyl 11-keto-boswellic acid (AKBA) and its derivatives, that target the spike protein of SARS-CoV-2 virus by using a structure-based virtual screening and biochemical assay, showing potential for treating COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>COVID-19 appeared as a highly contagious disease after its outbreak in December 2019 by the virus, named SARS-CoV-2. The threat, which originated in Wuhan, China, swiftly became an international emergency. Among different genomic products, spike protein of virus plays a crucial role in the initiation of the infection by binding to the human lung cells, therefore, SARS-CoV-2‚Äôs spike protein is a promising therapeutic target. Using a combination of a structure-based virtual screening and biochemical assay, this study seeks possible therapeutic candidates that specifically target the viral spike protein. A database of‚Äâ~‚Äâ850 naturally derived compounds was screened against SARS-CoV-2 spike protein to find natural inhibitors. Using virtual¬†screening and inhibitory experiments, we identifiedacetyl 11-keto-boswellic acid(AKBA) as a promising molecule for spike protein, which encouraged us to scan the rest of AKBA derivatives in our in-house database via 2D-similarity searching. Later 19 compounds with‚Äâ>‚Äâ85% similarity with AKBA were selected and docked with receptor binding domain (RBD) of spike protein. Those hits declared significant interactions at the RBD interface, best possess and excellent drug-likeness and pharmacokinetics properties with high gastrointestinal absorption (GIA) without toxicity and allergenicity. Ourin-silicoobservations were eventually validated by in vitro bioassay, interestingly, 10 compounds(A3, A4, C3, C6A, C6B, C6C, C6E, C6H, C6I,andC6J)displayed significant inhibitory ability with good percent inhibition (range:‚Äâ>‚Äâ72‚Äì90). The compoundsC3(90.00%),C6E(91.00%),C6C(87.20%), andC6D(86.23%) demonstrated excellent anti-SARS CoV-2 spike protein activities. The docking interaction of high percent inhibition of inhibitor compoundsC3andC6Ewas confirmed by MD Simulation. In the molecular dynamics simulation, we observed the stable dynamics of spike protein inhibitor complexes and the influence of inhibitor binding on the protein‚Äôs conformational arrangements. The binding free energy ŒîGTOTALofC3(‚àí38.0‚Äâ¬±‚Äâ0.08¬†kcal/mol) andC6E(‚àí41.98‚Äâ¬±‚Äâ0.08¬†kcal/mol) respectively indicate a strong binding affinity to Spike protein active pocket. These findings demonstrate that these molecules particularly inhibit the function of spike protein and, therefore have the potential to be evaluated as drug candidates against SARS-CoV-2.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53911-6" target="_blank">https://www.nature.com/articles/s41598-024-53911-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Broad protection against clade 1 sarbecoviruses after a single immunization with cocktail spike-protein-nanoparticle vaccine</h3>
            <h3 id="section13"><a href="#section13" style="text-decoration: none; color: inherit;">ü§ñ Single Vaccine Protects Against Clade 1 Sarbecoviruses</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-12</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research abstract highlights the importance of developing a broadly protective vaccine against ACE2-utilizing sarbecoviruses, including SARS-like viruses, as updated variant-matched formulations may not provide sufficient protection. The study introduces bivalent and trivalent vaccine formulations using a spike protein nanoparticle platform that has shown to protect hamsters against multiple virus challenges, including Omicron variants and bat sarbecoviruses. These results demonstrate the potential of their trivalent protein-nanoparticle cocktail in providing broad protection against SARS-CoV-2-like and SARS-CoV-1-like sarbecoviruses.</p>
            <br><strong>Abstract</strong><br>
            <p>The 2002‚ÄâSARS outbreak, the 2019 emergence of COVID-19, and the continuing evolution of immune-evading SARS-CoV-2 variants together highlight the need for a broadly protective vaccine against ACE2-utilizing sarbecoviruses. While updated variant-matched formulations are a step in the right direction, protection needs to extend beyond SARS-CoV-2 and its variants to include SARS-like viruses. Here, we introduce bivalent and trivalent vaccine formulations using our spike protein nanoparticle platform that completely protect female hamsters against BA.5 and XBB.1 challenges with no detectable virus in the lungs. The trivalent cocktails elicit highly neutralizing responses against all tested Omicron variants and the bat sarbecoviruses SHC014 and WIV1. Finally, our 614D/SHC014/XBB trivalent spike formulation completely protects human ACE2-transgenic female hamsters against challenges with WIV1 and SHC014 with no detectable virus in the lungs. Collectively, these results illustrate that our trivalent protein-nanoparticle cocktail can provide broad protection against SARS-CoV-2-like and SARS-CoV-1-like sarbecoviruses.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45495-6" target="_blank">https://www.nature.com/articles/s41467-024-45495-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Thyroid dysfunction in COVID-19</h3>
            <h3 id="section14"><a href="#section14" style="text-decoration: none; color: inherit;">ü§ñ "COVID-19 Impact on Thyroid Function"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-12</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This review examines the connection between COVID-19 and thyroid dysfunction, including long-term effects and interactions with vaccination, to provide updated information on this relationship for evidence-based practice recommendations.</p>
            <br><strong>Abstract</strong><br>
            <p>The COVID-19 pandemic has affected over 772 million people globally. While lung damage is the major contributor to the morbidity and mortality of this disease, the involvement of multiple organs, including the endocrine glands, has been reported. This Review aims to provide an updated summary of evidence regarding COVID-19 and thyroid dysfunction, incorporating highlights of recent advances in the field, particularly in relation to long COVID and COVID-19 vaccination. Since subacute thyroiditis following COVID-19 was first reported in May 2020, thyroid dysfunction associated with COVID-19 has been increasingly recognized, secondary to direct and indirect effects on the hypothalamic‚Äìpituitary‚Äìthyroid axis. Here, we summarize the epidemiological evidence, pattern and clinical course of thyroid dysfunction following COVID-19 and examine radiological, molecular and histological evidence of thyroid involvement in SARS-CoV-2 infection. Beyond acute SARS-CoV-2 infection, it is also timely to examine the course and implication of thyroid dysfunction in the context of long COVID owing to the large population of survivors of COVID-19 worldwide. This Review also analyses the latest evidence on the relationship between the therapeutics and vaccination for COVID-19 and thyroid dysfunction. To conclude, evidence-based practice recommendations for thyroid function testing during and following COVID-19 and concerning COVID-19 vaccination are proposed.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41574-023-00946-w" target="_blank">https://www.nature.com/articles/s41574-023-00946-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Analysis of the transplacental transmission of SARS CoV-2 virus and antibody transfer according to the gestational age at maternal infection</h3>
            <h3 id="section15"><a href="#section15" style="text-decoration: none; color: inherit;">ü§ñ Transmission & Antibody Transfer of SARS-CoV-2 in Pregnant Women</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-11</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that pregnant women infected with SARS-CoV-2 have a very low risk (less than 1%) of transmitting the virus to their newborns, and maternal antibody transfer to the fetus is more efficient when infection occurs during the first or second trimester.</p>
            <br><strong>Abstract</strong><br>
            <p>To quantify transplacental transmission of SARS-CoV-2 virus and antibody transfer in pregnant women and their newborns according to the gestational age at maternal infection. A prospective observational multicenter study including pregnant women with a positive RT-PCR or a positive serology for SARS-CoV-2 and compatible symptoms, from April to December 2020, in 11 French maternities. The study was designed to obtain a systematic collection of mother-infant dyad‚Äôs samples at birth. SARS-CoV-2 viral load was measured by RT-PCR. IgG and IgM antibodies against the SARS-CoV-2 spike protein were measured by enzyme-linked immunosorbent assay. Antibody concentrations and transplacental transfer ratios were analyzed according to the gestational age at maternal infection. The primary outcome was the rate of SARS CoV-2 materno-fetal transmission at birth. The secondary outcome was the quantification of materno-fetal antibody transfer. Maternal and neonatal outcomes at birth were additionally assessed. Among 165 dyads enrolled, one congenital infection was confirmed {n‚Äâ=‚Äâ1 (0.63%) IC95%[0.02%; 3.48%]}. The average placental IgG antibody transfer ratio was 1.27 (IC 95% [0.69‚Äì2.89]). The transfer ratio increased with increasing time between the onset of maternal infection and delivery (PValue‚Äâ=‚Äâ0.0001). Maternal and neonatal outcomes were reassuring. We confirmed the very low rate of SARS-CoV-2 transplacental transmission (<‚Äâ1%). Maternal antibody transfer to the fetus was more efficient when the infection occurred during the first and second trimester of pregnancy.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53580-5" target="_blank">https://www.nature.com/articles/s41598-024-53580-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Mucosal vaccine-induced cross-reactive CD8+ T cells protect against SARS-CoV-2 XBB.1.5 respiratory tract infection</h3>
            <h3 id="section16"><a href="#section16" style="text-decoration: none; color: inherit;">ü§ñ "Mucosal Vaccine Protects from Respiratory Infection by Cross-Reactive T Cells Against SARS-CoV-2 XBB.1.5"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-09</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The study updates a nasally delivered ChAd-SARS-CoV-2-BA.5 vaccine that encodes a stabilized BA.5 spike protein, showing robust mucosal immunity and protection against emerging SARS-CoV-2 strains in mice and hamsters; however, depletion of memory CD8+T cells resulted in loss of protection against upper and lower respiratory tract infection from antigenically distant strains with low serum levels of cross-reactive neutralizing antibodies.</p>
            <br><strong>Abstract</strong><br>
            <p>A nasally delivered chimpanzee adenoviral-vectored severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) vaccine (ChAd-SARS-CoV-2-S) is currently used in India (iNCOVACC). Here, we update this vaccine by creating ChAd-SARS-CoV-2-BA.5-S, which encodes a prefusion-stabilized BA.5 spike protein. Whereas serum neutralizing antibody responses induced by monovalent or bivalent adenoviral vaccines were poor against the antigenically distant XBB.1.5 strain and insufficient to protect in passive transfer experiments, mucosal antibody and cross-reactive memory T cell responses were robust, and protection was evident against WA1/2020 D614G and Omicron variants BQ.1.1 and XBB.1.5 in mice and hamsters. However, depletion of memory CD8+T cells before XBB.1.5 challenge resulted in loss of protection against upper and lower respiratory tract infection. Thus, nasally delivered vaccines stimulate mucosal immunity against emerging SARS-CoV-2 strains, and cross-reactive memory CD8+T cells mediate protection against lung infection by antigenically distant strains in the setting of low serum levels of cross-reactive neutralizing antibodies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01743-x" target="_blank">https://www.nature.com/articles/s41590-024-01743-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Time dependent dihedral angle oscillations of the spike protein of SARS-CoV-2 reveal favored frequencies of dihedral angle rotations</h3>
            <h3 id="section17"><a href="#section17" style="text-decoration: none; color: inherit;">ü§ñ Time-Dependent Dihedral Angle Oscillations in SARS-CoV-2 Spike Protein Reveal Favorite Rotation Frequencies</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-09</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study analyzes the dihedral angles of the SARS-CoV-2 spike protein and identifies dominant frequencies in the 23‚Äì63 MHz range, suggesting potential residues for vaccine and drug design to combat COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>The spike protein of SARS-CoV-2 is critical to viral infection of human host cells which ultimately results in COVID-19. In this study we analyzed the behavior of dihedral angles (phi and psi) of the wild-type spike protein over time from molecular dynamics and identified that their oscillations are dominated by a few discrete, relatively low frequencies in the 23‚Äì63¬†MHz range with 42.969¬†MHz being the most prevalent frequency sampled by the oscillations. We thus observed the spike protein to favor certain frequencies more than others. Gaps in the tally of all observed frequencies for low-abundance amino acids also suggests that the frequency components of dihedral angle oscillations may be a function of position in the primary structure since relatively more abundant amino acids lacked gaps. Lastly, certain residues identified in the literature as constituting the inside of a druggable pocket, as well as others identified as allosteric sites, are observed in our data to have distinctive time domain profiles. This motivated us to propose additional residues with similar time domain profiles, which may be of potential interest to the vaccine and drug design communities for further investigation. Thus these findings indicate that there is a particular frequency domain profile for the spike protein hidden within the time domain data and this information, perhaps with the suggested residues, might provide additional insight into therapeutic development strategies for COVID-19 and beyond.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53954-9" target="_blank">https://www.nature.com/articles/s41598-024-53954-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Quantitative probability estimation of light-induced inactivation of SARS-CoV-2</h3>
            <h3 id="section18"><a href="#section18" style="text-decoration: none; color: inherit;">ü§ñ Estimating Light Effect on Inactivating SARS-CoV-2 Virus</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-09</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research presents a model to understand how ultraviolet light interacts with the RNA sequence of viruses, specifically SARS-CoV-2, and inactivates them by damaging nucleotide pairs, potentially improving UV sanitization methods.</p>
            <br><strong>Abstract</strong><br>
            <p>During the COVID pandemic caused by the SARS-CoV-2 virus, studies have shown the efficiency of deactivating this virus via ultraviolet light. The damage mechanism is well understood: UV light disturbs the integrity of the RNA chain at those locations where specific nucleotide neighbors occur. In this contribution, we present a model to address certain gaps in the description of the interaction between UV photons and the RNA sequence for virus inactivation. We begin by exploiting the available information on the pathogen‚Äôs morphology, physical, and genomic characteristics, enabling us to estimate the average number of UV photons required to photochemically damage the virus‚Äôs RNA. To generalize our results, we have numerically generated random RNA sequences and checked that the distribution of pairs of nucleotides susceptible of damage for the SARS-CoV-2 is within the expected values for a random-generated RNA chain. After determining the average number of photons reaching the RNA for a preset level of fluence (or photon density), we applied the binomial probability distribution to evaluate the damage of nucleotide pairs in the RNA chain due to UV radiation. Our results describe this interaction in terms of the probability of damaging a single pair of nucleotides, and the number of available photons. The cumulative probability exhibits a steep sigmoidal shape, implying that a relatively small change in the number of affected pairs may trigger the inactivation of the virus. Our light-RNA interaction model quantitatively describes how the fraction of affected pairs of nucleotides in the RNA sequence depends on the probability of damaging a single pair and the number of photons impinging on it. A better understanding of the underlying inactivation mechanism would help in the design of optimum experiments and UV sanitization methods. Although this paper focuses on SARS-CoV-2, these results can be adapted for any other type of pathogen susceptible of UV damage.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-54006-y" target="_blank">https://www.nature.com/articles/s41598-024-54006-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Comparative effectiveness of alternative intervals between first and second doses of the mRNA COVID-19 vaccines</h3>
            <h3 id="section19"><a href="#section19" style="text-decoration: none; color: inherit;">ü§ñ Comparing Effectiveness: First & Second Dose Intervals in mRNA COVID-19 Vaccines</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-09</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research compared different interdose intervals for COVID-19 mRNA vaccines and found that delaying the second dose by 1-2 weeks provided stronger long-term protection.</p>
            <br><strong>Abstract</strong><br>
            <p>The optimal interval between the first and second doses of COVID-19 mRNA vaccines has not been thoroughly evaluated. Employing a target trial emulation approach, we compared the effectiveness of different interdose intervals among >6 million mRNA vaccine recipients in Georgia, USA, from December 2020 to March 2022. We compared three protocols defined by interdose interval: recommended by the Food and Drug Administration (FDA) (17-25 days for Pfizer-BioNTech; 24-32 days for Moderna), late-but-allowable (26-42 days for Pfizer-BioNTech; 33-49 days for Moderna), and late (‚Äâ‚â•‚Äâ43 days for Pfizer-BioNTech; ‚â•50 days for Moderna). In the short-term, the risk of SARS-CoV-2 infection was lowest under the FDA-recommended protocol. Longer-term, the late-but-allowable protocol resulted in the lowest risk (risk ratio on Day 120 after the first dose administration compared to the FDA-recommended protocol: 0.83 [95% confidence interval: 0.82-0.84]). Here, we showed that delaying the second dose by 1-2 weeks may provide stronger long-term protection.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45334-8" target="_blank">https://www.nature.com/articles/s41467-024-45334-8</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Fast and sensitive CRISPR detection by minimized interference of target amplification</h3>
            <h3 id="section20"><a href="#section20" style="text-decoration: none; color: inherit;">ü§ñ Rapid and Sensitive CRISPR Detection with Minimal Interference in Target Amplification</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-08</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research identifies rate-limiting factors affecting Cas12b- and Cas13a-mediated detection, and optimizes these to improve the sensitivity and speed of one-pot testing, making CRISPR-based assays competitive for widespread use in pathogen detection.</p>
            <br><strong>Abstract</strong><br>
            <p>Despite the great potential of CRISPR-based detection, it has not been competitive with other market diagnostics for on-site and in-home testing. Here we dissect the rate-limiting factors that undermine the performance of Cas12b- and Cas13a-mediated detection. In one-pot testing, Cas12b interferes with loop-mediated isothermal amplification by binding to and cleaving the amplicon, while Cas13a directly degrades the viral RNA, reducing its amplification. We found that the protospacer-adjacent motif-interacting domain engineered Cas12b accelerated one-pot testing with 10‚Äì10,000-fold improved sensitivity, and detected 85 out of 85 SARS-CoV-2 clinical samples with a sensitivity of 0.5‚Äâcp‚ÄâŒºl‚àí1, making it superior to wild-type Cas12b. In parallel, by diminishing the interference of Cas13a with viral RNA, the optimized Cas13a-based assay detected 86 out of 87 SARS-CoV-2 clinical samples at room temperature in 30‚Äâmin with a sensitivity of 0.5‚Äâcp‚ÄâŒºl‚àí1. The relaxed reaction conditions and improved performance of CRISPR-based assays make them competitive for widespread use in pathogen detection.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41589-023-01534-9" target="_blank">https://www.nature.com/articles/s41589-023-01534-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Micrometer-thick and porous nanocomposite coating for electrochemical sensors with exceptional antifouling and electroconducting properties</h3>
            <h3 id="section21"><a href="#section21" style="text-decoration: none; color: inherit;">ü§ñ Thin, Porous Nanocoatings for Electrochemical Sensors: Antifouling and Conductivity Features</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-08</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>A study has developed a 1 micrometer-thick, porous nanocomposite coating with antifouling and electroconducting properties to enhance the sensitivity of electrochemical sensors, enabling simultaneous detection of SARS-CoV-2 nucleic acid, antigen, and host antibody in clinical specimens.</p>
            <br><strong>Abstract</strong><br>
            <p>Development of coating technologies for electrochemical sensors that consistently exhibit antifouling activities in diverse and complex biological environments over extended time is vital for effective medical devices and diagnostics. Here, we describe a micrometer-thick, porous nanocomposite coating with both antifouling and electroconducting properties that enhances the sensitivity of electrochemical sensors. Nozzle printing of oil-in-water emulsion is used to create a 1 micrometer thick coating composed of cross-linked albumin with interconnected pores and gold nanowires. The layer resists biofouling and maintains rapid electron transfer kinetics for over one month when exposed directly to complex biological fluids, including serum and nasopharyngeal secretions. Compared to a thinner (nanometer thick) antifouling coating made with drop casting or a spin coating of the same thickness, the thick porous nanocomposite sensor exhibits sensitivities that are enhanced by 3.75- to 17-fold when three different target biomolecules are tested. As a result, emulsion-coated, multiplexed electrochemical sensors can carry out simultaneous detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) nucleic acid, antigen, and host antibody in clinical specimens with high sensitivity and specificity. This thick porous emulsion coating technology holds promise in addressing hurdles currently restricting the application of electrochemical sensors for point-of-care diagnostics, implantable devices, and other healthcare monitoring systems.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44822-1" target="_blank">https://www.nature.com/articles/s41467-024-44822-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Virological characteristics of the SARS-CoV-2 Omicron XBB.1.5 variant</h3>
            <h3 id="section22"><a href="#section22" style="text-decoration: none; color: inherit;">ü§ñ Omicron XBB.1.5 Variant: Key Virology Features</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-08</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research identifies a new SARS-CoV-2 Omicron variant, XBB.1.5, which evolved from XBB.1 through specific mutations and suggests that these changes may result in reduced virulence compared to the original XBB.1 strain.</p>
            <br><strong>Abstract</strong><br>
            <p>Circulation of SARS-CoV-2 Omicron XBB has resulted in the emergence of XBB.1.5, a new Variant of Interest. Our phylogenetic analysis suggests that XBB.1.5 evolved from XBB.1 by acquiring the S486P spike (S) mutation, subsequent to the acquisition of a nonsense mutation in ORF8. Neutralization assays showed similar abilities of immune escape between XBB.1.5 and XBB.1. We determine the structural basis for the interaction between human ACE2 and the S protein of XBB.1.5, showing similar overall structures between the S proteins of XBB.1 and XBB.1.5. We provide the intrinsic pathogenicity of XBB.1 and XBB.1.5 in hamsters. Importantly, we find that the ORF8 nonsense mutation of XBB.1.5 resulted in impairment of MHC suppression. In vivo experiments using recombinant viruses reveal that the XBB.1.5 mutations are involved with reduced virulence of XBB.1.5. Together, our study identifies the two viral functions defined the difference between XBB.1 and XBB.1.5.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45274-3" target="_blank">https://www.nature.com/articles/s41467-024-45274-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Audio-based AI classifiers show no evidence of improved COVID-19 screening over simple symptoms checkers</h3>
            <h3 id="section23"><a href="#section23" style="text-decoration: none; color: inherit;">ü§ñ Audio AI Classifiers Show No Advantage for COVID Screening Over Basic Symptom Check</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-07</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research investigates if AI classifiers can accurately predict SARS-CoV-2 infection using respiratory audio, finding that they perform better when accounting for confounding factors like user-reported symptoms; however, the study suggests that audio-based classifiers may not be as effective as predictions based on user-reported symptoms in practical settings.</p>
            <br><strong>Abstract</strong><br>
            <p>Recent work has reported that respiratory audio-trained AI classifiers can accurately predict SARS-CoV-2 infection status. However, it has not yet been determined whether such model performance is driven by latent audio biomarkers with true causal links to SARS-CoV-2 infection or by confounding effects, such as recruitment bias, present in observational studies. Here we undertake a large-scale study of audio-based AI classifiers as part of the UK government‚Äôs pandemic response. We collect a dataset of audio recordings from 67,842 individuals, with linked metadata, of whom 23,514 had positive polymerase chain reaction tests for SARS-CoV-2. In an unadjusted analysis, similar to that in previous works, AI classifiers predict SARS-CoV-2 infection status with high accuracy (ROC‚ÄìAUC‚Äâ=‚Äâ0.846 [0.838‚Äì0.854]). However, after matching on measured confounders, such as self-reported symptoms, performance is much weaker (ROC‚ÄìAUC‚Äâ=‚Äâ0.619 [0.594‚Äì0.644]). Upon quantifying the utility of audio-based classifiers in practical settings, we find them to be outperformed by predictions on the basis of user-reported symptoms. We make best-practice recommendations for handling recruitment bias, and for assessing audio-based classifiers by their utility in relevant practical settings. Our work provides insights into the value of AI audio analysis and the importance of study design and treatment of confounders in AI-enabled diagnostics.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42256-023-00773-8" target="_blank">https://www.nature.com/articles/s42256-023-00773-8</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Seroprevalence of SARS-CoV-2 antibodies among Japanese healthcare workers from 2020 to 2022 as assayed by two commercial kits</h3>
            <h3 id="section24"><a href="#section24" style="text-decoration: none; color: inherit;">ü§ñ SARS-CoV-2 Antibody Levels in Japanese Healthcare Workers (2020-2022) Using Commercial Kits</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-07</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research compared two types of COVID-19 antibody tests in a Japanese medical center and found significant differences in seropositivity over time, indicating that one test may be more suitable for long-term surveillance than the other.</p>
            <br><strong>Abstract</strong><br>
            <p>Antibody tests are used as surveillance tools for informing health policy making. However, results may vary by type of antibody assay and timing of sample collection following infection. Long-term longitudinal cohort studies on antibody assay seropositivity have remained limited, especially among Asian populations. Using blood samples obtained at health physicals (2020‚Äì2022) of healthcare workers (mass vaccinated with mRNA COVID-19 vaccines) at a Japanese medical center, we measured N-specific antibodies using two commercially available systems. Roche Elecsys Anti-SARS-CoV-2 measures total antibodies and Abbott Alinity SARS-CoV-2 IgG measures only IgG. Among 2538 participants, seroprevalence was found to be 16.6% via total antibody assay versus 12.9% by IgG-only (including grayzone) by mid-June 2022. For 219 cases with a previous PCR-confirmed infection, positivity was 97.3% using total antibody assay versus 76.3% using IgG-only assay at the 2022 health physical. Using PCR positive test date as day 0, while the positivity of the total antibody assay was retained for the entire study period (until more than 24-months post-infection), the IgG-only assay‚Äôs positivity declined after month 4. The Mantel‚ÄìHaenszel test found a significant difference in the two assays‚Äô seropositivity, between stratified groups of ‚Äúwithin 3¬†months‚Äù and ‚Äú4¬†months or more‚Äù from infection (P<‚Äâ0.001). Our study found significant differences in seropositivity over time of total antibody versus IgG-only assays, suggesting an optimal assay for retaining sensitivity over the entire infection period when designing seroprevalence studies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53656-2" target="_blank">https://www.nature.com/articles/s41598-024-53656-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Potent and broadly neutralizing antibodies against sarbecoviruses induced by sequential COVID-19 vaccination</h3>
            <h3 id="section25"><a href="#section25" style="text-decoration: none; color: inherit;">ü§ñ Neutralizing Antibodies Against Sarbecoviruses from Sequential COVID Vaccinations</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-06</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study identifies potent and broadly neutralizing antibodies from a vaccinated donor that can effectively neutralize diverse coronaviruses, including SARS-CoV-2 variants, offering promising avenues for the development of therapeutic antibody drugs.</p>
            <br><strong>Abstract</strong><br>
            <p>The current SARS-CoV-2 variants strikingly evade all authorized monoclonal antibodies and threaten the efficacy of serum-neutralizing activity elicited by vaccination or prior infection, urging the need to develop antivirals against SARS-CoV-2 and related sarbecoviruses. Here, we identified both potent and broadly neutralizing antibodies from a five-dose vaccinated donor who exhibited cross-reactive serum-neutralizing activity against diverse coronaviruses. Through single B-cell sorting and sequencing followed by a tailor-made computational pipeline, we successfully selected 86 antibodies with potential cross-neutralizing ability from 684 antibody sequences. Among them, PW5-570 potently neutralized all SARS-CoV-2 variants that arose prior to Omicron BA.5, and the other three could broadly neutralize all current SARS-CoV-2 variants of concern, SARS-CoV and their related sarbecoviruses (Pangolin-GD, RaTG13, WIV-1, and SHC014). Cryo-EM analysis demonstrates that these antibodies have diverse neutralization mechanisms, such as disassembling spike trimers, or binding to RBM or SD1 to affect ACE2 binding. In addition, prophylactic administration of these antibodies significantly protects nasal turbinate and lung infections against BA.1, XBB.1, and SARS-CoV viral challenge in golden Syrian hamsters, respectively. Importantly, post-exposure treatment with PW5-5 and PW5-535 also markedly protects against XBB.1 challenge in these models. This study reveals the potential utility of computational process to assist screening cross-reactive antibodies, as well as the potency of vaccine-induced broadly neutralizing antibodies against current SARS-CoV-2 variants and related sarbecoviruses, offering promising avenues for the development of broad therapeutic antibody drugs.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00648-1" target="_blank">https://www.nature.com/articles/s41421-024-00648-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>A genome-wide association study for survival from a multi-centre European study identified variants associated with COVID-19 risk of death</h3>
            <h3 id="section26"><a href="#section26" style="text-decoration: none; color: inherit;">ü§ñ Genome-wide study finds COVID-19 death risk factors from multi-center European research</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-06</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research investigates the genetic determinants of COVID-19 mortality by conducting a case-only genome-wide survival analysis on 3904 patients from the GEN-COVID and other European series, finding that germline variants may modulate COVID-19 risk of death through the regulation of gene expression in immune response and lung function pathways.</p>
            <br><strong>Abstract</strong><br>
            <p>The clinical manifestations of SARS-CoV-2 infection vary widely among patients, from asymptomatic to life-threatening. Host genetics is one of the factors that contributes to this variability as previously reported by the COVID-19 Host Genetics Initiative (HGI), which identified sixteen loci associated with COVID-19 severity. Herein, we investigated the genetic determinants of COVID-19 mortality, by performing a case-only genome-wide survival analysis, 60¬†days after infection, of 3904 COVID-19 patients from the GEN-COVID and other European series (EGAS00001005304 study of the COVID-19 HGI). Using imputed genotype data, we carried out a survival analysis using the Cox model adjusted for age, age2, sex, series, time of infection, and the first ten principal components. We observed a genome-wide significant (P-value‚Äâ<‚Äâ5.0‚Äâ√ó‚Äâ10‚àí8) association of the rs117011822 variant, on chromosome 11, of rs7208524 on chromosome 17, approaching the genome-wide threshold (P-value‚Äâ=‚Äâ5.19‚Äâ√ó‚Äâ10‚àí8). A total of 113 variants were associated with survival atP-value‚Äâ<‚Äâ1.0‚Äâ√ó‚Äâ10‚àí5and most of them regulated the expression of genes involved in immune response (e.g., CD300 and KLR genes), or in lung repair and function (e.g., FGF19 and CDH13). Overall, our results suggest that germline variants may modulate COVID-19 risk of death, possibly through the regulation of gene expression in immune response and lung function pathways.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53310-x" target="_blank">https://www.nature.com/articles/s41598-024-53310-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 vaccination may mitigate dysregulation of IL-1/IL-18 and gastrointestinal symptoms of the post-COVID-19 condition</h3>
            <h3 id="section27"><a href="#section27" style="text-decoration: none; color: inherit;">ü§ñ COVID-19 Vaccine May Help Reduce GI Symptoms in Post-Infection State</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-05</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research suggests that post-infection vaccination may be beneficial for some individuals with COVID-19 post-condition, particularly those experiencing gastrointestinal symptoms, but the effect may depend on the viral variant.</p>
            <br><strong>Abstract</strong><br>
            <p>The rapid development of safe and effective vaccines helped to prevent severe disease courses after SARS-CoV-2 infection and to mitigate the progression of the COVID-19 pandemic. While there is evidence that vaccination may reduce the risk of developing post-COVID-19 conditions (PCC), this effect may depend on the viral variant. Therapeutic effects of post-infection vaccination have been discussed but the data for individuals with PCC remains inconclusive. In addition, extremely rare side effects after SARS-CoV-2 vaccination may resemble the heterogeneous PCC phenotype. Here, we analyze the plasma levels of 25 cytokines and SARS-CoV-2 directed antibodies in 540 individuals with or without PCC relative to one or two mRNA-based COVID-19 vaccinations as well as in 20 uninfected individuals one month after their initial mRNA-based COVID-19 vaccination. While none of the SARS-CoV-2 na√Øve individuals reported any persisting sequelae or exhibited PCC-like dysregulation of plasma cytokines, we detected lower levels of IL-1Œ≤ and IL-18 in patients with ongoing PCC who received one or two vaccinations at a median of six months after infection as compared to unvaccinated PCC patients. This reduction correlated with less frequent reporting of persisting gastrointestinal symptoms. These data suggest that post-infection vaccination in patients with PCC might be beneficial in a subgroup of individuals displaying gastrointestinal symptoms.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">https://www.nature.com/articles/s41541-024-00815-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>A monoclonal antibody targeting a large surface of the receptor binding motif shows pan-neutralizing SARS-CoV-2 activity</h3>
            <h3 id="section28"><a href="#section28" style="text-decoration: none; color: inherit;">ü§ñ "Monoclonal Antibody with Wide Receptor Binding Targets All SARS-CoV-2 Variants"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-05</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research focuses on characterizing 17T2, a human monoclonal antibody that effectively neutralizes various SARS-CoV-2 variants, including Omicron subvariants, and demonstrates in vivo prophylactic and therapeutic activity against BA.1.1 infection in mice. This antibody is derived from a receptor binding domain (RBD)-specific IgA+ memory B cell and its ability to maintain neutralizing activity against all tested variants is attributed to a larger RBD contact area, making it a potential candidate for future clinical interventions.</p>
            <br><strong>Abstract</strong><br>
            <p>Here we report the characterization of 17T2, a SARS-CoV-2 pan-neutralizing human monoclonal antibody isolated from a COVID-19 convalescent individual infected during the first pandemic wave. 17T2 is a class 1 VH1-58/Œ∫3-20 antibody, derived from a receptor binding domain (RBD)-specific IgA+memory B cell, with a broad neutralizing activity against former and new SARS-CoV-2 variants, including XBB.1.16 and BA.2.86 Omicron subvariants. Consistently, 17T2 demonstrates in vivo prophylactic and therapeutic activity against Omicron BA.1.1 infection in K18-hACE2 mice. Cryo-electron microscopy reconstruction shows that 17T2 binds the BA.1 spike with the RBD in ‚Äúup‚Äù position and blocks the receptor binding motif, as other structurally similar antibodies do, including S2E12. Yet, unlike S2E12, 17T2 retains its neutralizing activity against all variants tested, probably due to a larger RBD contact area. These results highlight the impact of small structural antibody changes on neutralizing performance and identify 17T2 as a potential candidate for future clinical interventions.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45171-9" target="_blank">https://www.nature.com/articles/s41467-024-45171-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Risk of SARS-CoV-2 reinfection during multiple Omicron variant waves in the UK general population</h3>
            <h3 id="section29"><a href="#section29" style="text-decoration: none; color: inherit;">ü§ñ Risk of SARS-CoV-2 Re-Infection During Multiple Omicron Waves in UK Public</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research found that SARS-CoV-2 reinfections increased after the Omicron variants emerged, with reinfections having lower viral load and less severe symptoms than first infections. Protection against reinfection was higher for those previously infected with more recent variants and decreased over time.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 reinfections increased substantially after Omicron variants emerged. Large-scale community-based comparisons across multiple Omicron waves of reinfection characteristics, risk factors, and protection afforded by previous infection and vaccination, are limited. Here we studied ~45,000 reinfections from the UK‚Äôs national COVID-19 Infection Survey and quantified the risk of reinfection in multiple waves, including those driven by BA.1, BA.2, BA.4/5, and BQ.1/CH.1.1/XBB.1.5 variants. Reinfections were associated with lower viral load and lower percentages of self-reporting symptoms compared with first infections. Across multiple Omicron waves, estimated protection against reinfection was significantly higher in those previously infected with more recent than earlier variants, even at the same time from previous infection. Estimated protection against Omicron reinfections decreased over time from the most recent infection if this was the previous or penultimate variant (generally within the preceding year). Those 14‚Äì180‚Äâdays after receiving their most recent vaccination had a lower risk of reinfection than those >180‚Äâdays from their most recent vaccination. Reinfection risk was independently higher in those aged 30‚Äì45‚Äâyears, and with either low or high viral load in their most recent previous infection. Overall, the risk of Omicron reinfection is high, but with lower severity than first infections; both viral evolution and waning immunity are independently associated with reinfection.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44973-1" target="_blank">https://www.nature.com/articles/s41467-024-44973-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>A Gamma-adapted subunit vaccine induces broadly neutralizing antibodies against SARS-CoV-2 variants and protects mice from infection</h3>
            <h3 id="section30"><a href="#section30" style="text-decoration: none; color: inherit;">ü§ñ "Subunit Vaccine Protects Mice Against COVID Variants"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research shows that a RBD vaccine derived from the Gamma SARS-CoV-2 variant, when combined with Alum, is more effective at inducing broader neutralizing antibodies and offering protection against various COVID-19 variants compared to other vaccines.</p>
            <br><strong>Abstract</strong><br>
            <p>In the context of continuous emergence of SARS-CoV-2 variants of concern (VOCs), one strategy to prevent the severe outcomes of COVID-19 is developing safe and effective broad-spectrum vaccines. Here, we present preclinical studies of a RBD vaccine derived from the Gamma SARS-CoV-2 variant adjuvanted with Alum. The Gamma-adapted RBD vaccine is more immunogenic than the Ancestral RBD vaccine in terms of inducing broader neutralizing antibodies. The Gamma RBD presents more immunogenic B-cell restricted epitopes and induces a higher proportion of specific-B cells and plasmablasts than the Ancestral RBD version. The Gamma-adapted vaccine induces antigen¬†specific T cell immune responses and confers protection against Ancestral and Omicron BA.5 SARS-CoV-2 challenge in mice. Moreover, the¬†Gamma¬†RBD vaccine induces higher and broader neutralizing antibody activity than homologous booster vaccination in mice previously primed with different SARS-CoV-2 vaccine platforms. Our study indicates that the adjuvanted Gamma RBD vaccine is highly immunogenic and a broad-spectrum vaccine candidate.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45180-8" target="_blank">https://www.nature.com/articles/s41467-024-45180-8</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Targeting the receptor binding domain and heparan sulfate binding for antiviral drug development against SARS-CoV-2 variants</h3>
            <h3 id="section31"><a href="#section31" style="text-decoration: none; color: inherit;">ü§ñ Developing Antiviral Drugs Against SARS-CoV-2 Variants by Targeting Receptor Binding and Heparan Sulfate Binding</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research used computational methods to analyze coronavirus sequences, identify key interactions in the SARS-CoV-2 spike protein's binding region with heparan sulfate, and screened natural compounds for potential antiviral candidates, finding Acetoside, Hyperoside, and Isoquercitrin as promising options.</p>
            <br><strong>Abstract</strong><br>
            <p>The emergence of SARS-CoV-2 variants diminished the efficacy of current antiviral drugs and vaccines. Hence, identifying highly conserved sequences and potentially druggable pockets for drug development was a promising strategy against SARS-CoV-2 variants. In viral infection, the receptor-binding domain (RBD) proteins are essential in binding to the host receptor. Others, Heparan sulfate (HS), widely distributed on the surface of host cells, is thought to play a central role in the viral infection cycle of SARS-CoV-2. Therefore, it might be a reasonable strategy for antiviral drug design to interfere with the RBD in the HS binding site. In this study, we used computational approaches to analyze multiple sequences of coronaviruses and reveal important information about the binding of HS to RBD in the SARS-CoV-2 spike protein. Our results showed that the potential hot-spots, including R454 and E471, in RBD, exhibited strong interactions in the HS-RBD binding region. Therefore, we screened different compounds in the natural product database towards these hot-spots to find potential antiviral candidates usingLibDock,Autodock vinaand furthermore applying the MD simulation inAMBER20. The results showed three potential natural compounds, including Acetoside (ACE), Hyperoside (HYP), and Isoquercitrin (ISO), had a strong affinity to the RBD. Our results demonstrate a feasible approach to identify potential antiviral agents by evaluating the binding interaction between viral glycoproteins and host receptors. The present study provided the applications of the structure-based computational approach for designing and developing of new antiviral drugs against SARS-CoV-2 variants.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53111-2" target="_blank">https://www.nature.com/articles/s41598-024-53111-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Blood transcriptomics analysis offers insights into variant-specific immune response to SARS-CoV-2</h3>
            <h3 id="section32"><a href="#section32" style="text-decoration: none; color: inherit;">ü§ñ Blood Transcriptomics Reveals Immune Responses to COVID Variants</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study suggests that RNA sequencing could be used as a diagnostic tool to analyze gene expression, immune cell composition, and BCR/TCR repertoires in patients infected with SARS-CoV-2 variants, providing valuable insights into disease severity and pathogen-specific immunity.</p>
            <br><strong>Abstract</strong><br>
            <p>Bulk RNA sequencing (RNA-seq) of blood is typically used for gene expression analysis in biomedical research but is still rarely used in clinical practice. In this study, we propose that RNA-seq should be considered a diagnostic tool, as it offers not only insights into aberrant gene expression and splicing but also delivers additional readouts on immune cell type composition as well as B-cell and T-cell receptor (BCR/TCR) repertoires. We demonstrate that RNA-seq offers insights into a patient‚Äôs immune status via integrative analysis of RNA-seq data from patients infected with various SARS-CoV-2 variants (in total 196 samples with up to 200 million reads sequencing depth). We compare the results of computational cell-type deconvolution methods (e.g., MCP-counter, xCell, EPIC, quanTIseq) to complete blood count data, the current gold standard in clinical practice. We observe varying levels of lymphocyte depletion and significant differences in neutrophil levels between SARS-CoV-2 variants. Additionally, we identify B and T cell receptor (BCR/TCR) sequences using the tools MiXCR and TRUST4 to show that‚Äîcombined with sequence alignments and BLASTp‚Äîthey could be used to classify a patient's disease. Finally, we investigated the sequencing depth required for such analyses and concluded that 10 million reads per sample is sufficient. In conclusion, our study reveals that computational cell-type deconvolution and BCR/TCR methods using bulk RNA-seq analyses can supplement missing CBC data and offer insights into immune responses, disease severity, and pathogen-specific immunity, all achievable with a sequencing depth of 10 million reads per sample.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53117-w" target="_blank">https://www.nature.com/articles/s41598-024-53117-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>An intranasal live-attenuated SARS-CoV-2 vaccine limits virus transmission</h3>
            <h3 id="section33"><a href="#section33" style="text-decoration: none; color: inherit;">ü§ñ Intranasal COVID Vaccine: Reducing Virus Spread</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study compares a live-attenuated virus vaccine and an mRNA vaccine's effectiveness in preventing SARS-CoV-2 transmission in hamsters, finding the live-attenuated vaccine to be significantly more effective in both scenarios.</p>
            <br><strong>Abstract</strong><br>
            <p>The development of effective SARS-CoV-2 vaccines has been essential to control COVID-19, but significant challenges remain. One problem is intramuscular administration, which does not induce robust mucosal immune responses in the upper airways‚Äîthe primary site of infection and virus shedding. Here we compare the efficacy of a mucosal, replication-competent yet fully attenuated virus vaccine, sCPD9-ŒîFCS, and the monovalent mRNA vaccine BNT162b2 in preventing transmission of SARS-CoV-2 variants B.1 and Omicron BA.5 in two scenarios. Firstly, we assessed the protective efficacy of the vaccines by exposing vaccinated male Syrian hamsters to infected counterparts. Secondly, we evaluated transmission of the challenge virus from vaccinated and subsequently challenged male hamsters to na√Øve contacts. Our findings demonstrate that the live-attenuated vaccine (LAV) sCPD9-ŒîFCS significantly outperformed the mRNA vaccine in preventing virus transmission in both scenarios. Our results provide evidence for the advantages of locally administered LAVs over intramuscularly administered mRNA vaccines in preventing infection and reducing virus transmission.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45348-2" target="_blank">https://www.nature.com/articles/s41467-024-45348-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Binding behavior of receptor binding domain of the SARS-CoV-2 virus and ivermectin</h3>
            <h3 id="section34"><a href="#section34" style="text-decoration: none; color: inherit;">ü§ñ Ivermectin and SARS-CoV-2 Receptor Binding Domain Interaction Effects</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-02-02</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research investigates whether ivermectin, an FDA-approved antiparasitic drug, can be a repurposed drug for SARS-CoV-2 prevention and treatment by examining its binding behavior with the virus's receptor binding domain using biosensors and various in silico techniques. The results indicate that ivermectin is not an effective preventative or treatment method at the currently approved dosage.</p>
            <br><strong>Abstract</strong><br>
            <p>The COVID-19 pandemic, caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), sparked an international debate on effective ways to prevent and treat the virus. Specifically, there were many varying opinions on the use of ivermectin (IVM) throughout the world, with minimal research to support either side. IVM is an FDA-approved antiparasitic drug that was discovered in the 1970s and was found to show antiviral activity. The objective of this study is to examine the binding behavior and rates of association and dissociation between SARS-CoV-2 receptor binding domain (RBD), IVM, and their combination using aminopropylsilane (APS) biosensors as surrogates for the hydrophobic interaction between the viral protein and human angiotensin-converting enzyme 2 (ACE2) receptors to determine the potential of IVM as a repurposed drug for SARS-CoV-2 prevention and treatment. The IVM, RBD, and combination binding kinetics were analyzed using biolayer interferometry (BLI) and validated with multiple in silico techniques including protein‚Äìligand docking, molecular dynamics simulation, molecular mechanics-generalized Born surface area (MM-GBSA), and principal component analysis (PCA). Our results suggest that with increasing IVM concentrations the association rate with the hydrophobic biosensor increases with a simultaneous decrease in dissociation. Significant kinetic changes to RBD, when combined with IVM, were found only at a concentration a thousand times the approved dosage with minimal changes found over a 35-min time period. Our study suggests that IVM is not an effective preventative or treatment method at the currently approved dosage.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53086-0" target="_blank">https://www.nature.com/articles/s41598-024-53086-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Analysis of related factors of plasma antibody levels in patients with severe and critical COVID-19</h3>
            <h3 id="section35"><a href="#section35" style="text-decoration: none; color: inherit;">ü§ñ Plasma Antibody Levels in Severe & Critical COVID-19 Patients: Factors Analysis</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-31</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study investigated the relationship between plasma antibody levels and baseline conditions or clinical outcomes in severe and critical COVID-19 patients, finding that mild cases had higher RBD IgG antibody levels and a strong correlation between antibody levels and vaccination status in severe/critical cases.</p>
            <br><strong>Abstract</strong><br>
            <p>Coronavirus disease 2019 (COVID-19) continues to impact global public health. The severe acute respiratory syndrome coronavirus 2¬†(SARS-CoV-2) has become less virulent as it mutates, prompting China to ease restrictions at the end of 2022. With the complete reopening, a surge in COVID-19 cases has ensued. Therefore, we conducted a study to explore the correlation between plasma antibody levels and baseline conditions or clinical outcomes in severe and critical patients. We collected the basic information of 79 included patients. Enzyme-linked immunosorbent assay¬†(ELISA) tests were performed on plasma samples. The receptor-binding domain¬†(RBD) IgG antibody level of the mild group was significantly higher than that of the severe/critical group (P=‚Äâ0.00049). And in the severe/critical group, there existed an association between plasma antibody levels and age (P<‚Äâ0.001, r‚Äâ=‚Äâ‚àí‚Äâ0.471), as well as plasma antibody levels and vaccination status (P=‚Äâ0.00147, eta2=‚Äâ0.211). Besides, the level of plasma antibody seemed to be moderately correlated with the age, indicating the need for heightened attention to infections in the elderly. And plasma antibody levels were strongly associated with vaccination status in the severe/critical patients.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52572-9" target="_blank">https://www.nature.com/articles/s41598-024-52572-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Omicron Spike confers enhanced infectivity and interferon resistance to SARS-CoV-2 in human nasal tissue</h3>
            <h3 id="section36"><a href="#section36" style="text-decoration: none; color: inherit;">ü§ñ "Omicron's Enhanced Infection & Interferon Resistance"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The abstract reports that Omicron is more infectious than Delta due to mutations in its spike protein, which allow it to enter nasal cells without relying on protease enzymes. This suggests that Omicron can bypass immune responses and antiviral factors in the nose, enhancing its spread.</p>
            <br><strong>Abstract</strong><br>
            <p>Omicron emerged following COVID-19 vaccination campaigns, displaced previous SARS-CoV-2 variants of concern worldwide, and gave rise to lineages that continue to spread. Here, we show that Omicron exhibits increased infectivity in primary adult upper airway tissue relative to Delta. Using recombinant forms of SARS-CoV-2 and nasal epithelial cells cultured at the liquid-air interface, we show that mutations unique to Omicron Spike enable enhanced entry into nasal tissue. Unlike earlier variants of SARS-CoV-2, our findings suggest that Omicron enters nasal cells independently of serine transmembrane proteases and instead relies upon metalloproteinases to catalyze membrane fusion. Furthermore, we demonstrate that this entry pathway unlocked by Omicron Spike enables evasion from constitutive and interferon-induced antiviral factors that restrict SARS-CoV-2 entry following attachment. Therefore, the increased transmissibility exhibited by Omicron in humans may be attributed not only to its evasion of vaccine-elicited adaptive immunity, but also to its superior invasion of nasal epithelia and resistance to the cell-intrinsic barriers present therein.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45075-8" target="_blank">https://www.nature.com/articles/s41467-024-45075-8</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Humoral profiles of toddlers and young children following SARS-CoV-2 mRNA vaccination</h3>
            <h3 id="section37"><a href="#section37" style="text-decoration: none; color: inherit;">ü§ñ How do toddlers react to Covid-19 shots?</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research investigates the immune responses generated by mRNA vaccination in toddler and young children. The study found that despite receiving a lower dose of vaccine, vaccinated infants generate a strong immune response similar to adults. In addition, the study suggests that mRNA vaccination provides greater protection against severe disease compared to natural infection, with a higher IgG3-dependent humoral profile.</p>
            <br><strong>Abstract</strong><br>
            <p>Although young children generally experience mild symptoms following infection with SARS-CoV-2, severe acute and long-term complications can occur. SARS-CoV-2 mRNA vaccines elicit robust immunoglobulin profiles in children ages 5 years and older, and in adults, corresponding with substantial protection against hospitalizations and severe disease. Whether similar immune responses and humoral protection can be observed in vaccinated infants and young children, who have a developing and vulnerable immune system, remains poorly understood. To study the impact of mRNA vaccination on the humoral immunity of infant, we use a system serology approach to comprehensively profile antibody responses in a cohort of children ages 6 months to 5 years who were vaccinated with the mRNA-1273 COVID-19 vaccine (25‚ÄâŒºg). Responses are compared with vaccinated adults (100‚ÄâŒºg), in addition to naturally infected toddlers and young children. Despite their lower vaccine dose, vaccinated toddlers elicit a functional antibody response as strong as adults, with higher antibody-dependent phagocytosis compared to adults, without report of side effects. Moreover, mRNA vaccination is associated with a higher IgG3-dependent humoral profile against SARS-CoV-2 compared to natural infection, supporting that mRNA vaccination is effective at eliciting a robust antibody response in toddlers and young children.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45181-7" target="_blank">https://www.nature.com/articles/s41467-024-45181-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Necroptosis does not drive disease pathogenesis in a mouse infective model of SARS-CoV-2 in vivo</h3>
            <h3 id="section38"><a href="#section38" style="text-decoration: none; color: inherit;">ü§ñ AI system: Necroptosis doesn't cause diseases in mice with COVID.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study shows that a specific cell death process called necroptosis does not play a significant role in the development of COVID-19 in mice, regardless of the severity of the disease or age of the animal.</p>
            <br><strong>Abstract</strong><br>
            <p>Necroptosis, a type of lytic cell death executed by the pseudokinase Mixed Lineage Kinase Domain-Like (MLKL) has been implicated in the detrimental inflammation caused by SARS-CoV-2 infection. We minimally and extensively passaged a single clinical SARS-CoV-2 isolate to create models of mild and severe disease in mice allowing us to dissect the role of necroptosis in SARS-CoV-2 disease pathogenesis. We infected wild-type and MLKL-deficient mice and found no significant differences in viral loads or lung pathology. In our model of severe COVID-19, MLKL-deficiency did not alter the host response, ameliorate weight loss, diminish systemic pro-inflammatory cytokines levels, or prevent lethality in aged animals. Our in vivo models indicate that necroptosis is dispensable in the pathogenesis of mild and severe COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-024-06471-6" target="_blank">https://www.nature.com/articles/s41419-024-06471-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Beneficial effect of temporary methotrexate interruption on B and T cell responses upon SARS-CoV-2 vaccination in patients with rheumatoid arthritis or psoriatic arthritis</h3>
            <h3 id="section39"><a href="#section39" style="text-decoration: none; color: inherit;">ü§ñ Methotrexate Pause Boosts Immune Response to COVID Vaccine in Arthritis Patients</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research found that temporarily stopping methotrexate treatment for two weeks after a COVID-19 vaccine can lead to improved immune response in patients with rheumatoid arthritis or psoriatic arthritis without causing relapses of the disease.</p>
            <br><strong>Abstract</strong><br>
            <p>B and T cell responses were evaluated in patients with rheumatoid arthritis (RA) or psoriatic arthritis (PsA) after 1 or 2 weeks of methotrexate (MTX) withdrawal following each COVID-19 vaccine dose and compared with those who maintained MTX. Adult RA and PsA patients treated with MTX were recruited and randomly assigned to 3 groups: MTX-maintenance (n=‚Äâ72), MTX-withdrawal for 1 week (n=‚Äâ71) or MTX-withdrawal for 2 weeks (n=‚Äâ73). Specific antibodies to several SARS-CoV-2 antigens and interferon (IFN)-Œ≥ and interleukin (IL)-21 responses were assessed. MTX withdrawal in patients without previous COVID-19 was associated with higher levels of anti-RBD IgG and neutralising antibodies, especially in the 2-week withdrawal group and with higher IFN-Œ≥ secretion upon stimulation with pools of SARS-CoV-2 S peptides. No increment of RA/PsA relapses was detected across groups. Our data indicate that two-week MTX interruption following COVID-19 vaccination in patients with RA or PsA improves humoral and cellular immune responses.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00805-3" target="_blank">https://www.nature.com/articles/s41541-024-00805-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Identification of a highly conserved neutralizing epitope within the RBD region of diverse SARS-CoV-2 variants</h3>
            <h3 id="section40"><a href="#section40" style="text-decoration: none; color: inherit;">ü§ñ Find out how scientists found an important part of the virus that doesn't change much. 

# Title: How Scientists Found a Part of the Virus That Doesn't Change Much #COVID19 #Science</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-29</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The research abstract talks about a specific antibody that can protect against many SARS-CoV-2 variants. It's effective because it targets a common part of the virus's structure called the "receptor binding domain." This could help in creating better treatments and vaccines for COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>The constant emergence of SARS-CoV-2 variants continues to impair the efficacy of existing neutralizing antibodies, especially XBB.1.5 and EG.5, which showed exceptional immune evasion properties. Here, we identify a highly conserved neutralizing epitope targeted by a broad-spectrum neutralizing antibody BA7535, which demonstrates high neutralization potency against not only previous variants, such as Alpha, Beta, Gamma, Delta and Omicron BA.1-BA.5, but also more recently emerged Omicron subvariants, including BF.7, CH.1.1, XBB.1, XBB.1.5, XBB.1.9.1, EG.5. Structural analysis of the Omicron Spike trimer with BA7535-Fab using cryo-EM indicates that BA7535 recognizes a highly conserved cryptic receptor-binding domain (RBD) epitope, avoiding most of the mutational hot spots in RBD. Furthermore, structural simulation based on the interaction of BA7535-Fab/RBD complexes dissects the broadly neutralizing effect of BA7535 against latest variants. Therapeutic and prophylactic treatment with BA7535 alone or in combination with BA7208 protected female mice from the circulating Omicron BA.5 and XBB.1 variant infection, suggesting the highly conserved neutralizing epitope serves as a potential target for developing highly potent therapeutic antibodies and vaccines.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45050-3" target="_blank">https://www.nature.com/articles/s41467-024-45050-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Cycle threshold of SARS-CoV-2 RT-PCR as a driver of retesting</h3>
            <h3 id="section41"><a href="#section41" style="text-decoration: none; color: inherit;">ü§ñ "COVID PCR test results: Is it time to retest?"  # Simplified title for easier understanding.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-29</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>Our research found that patients frequently retested using a limited number of RT-PCR tests to diagnose COVID-19. Patients tend to be retested around day 7 after the initial positive result, and their Ct-values increase over time. This suggests that Ct-cutoffs may not always be reliable for determining when to stop testing.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 RT-PCR is a critical and, at times, limited resource. Frequent Retesting of patients may strain testing infrastructure unduly. Recommendations that include cycle threshold (Ct) cutoffs may incentivize early retesting when the Ct value is reported. We aimed to investigate patterns of retesting in association with initial Ct-values. We performed a retrospective analysis of RT-PCR results (including Ct-values) for patients from whom ‚â•‚Äâ2 samples were collected within 14¬†days, the first of which had to be positive. We calculated absolute and baseline-corrected kinetics of Ct-values over time, as well as the median initial Ct-values in dependence of the timing of the first retesting and the time until RT-PCR negativity for SARS-CoV-2. Retesting after an initial positive SARS-CoV-2 RT-PCR was most commonly performed on day 7, with patients being retested as early as day 1. The majority of patients retested within 14¬†days remained SARS-CoV-2 positive in the RT-PCR. Baseline-corrected Ct-values showed a quasi-linear increase over 14¬†days since the initial positive result. Both the timing until the first retesting and until RT-PCR negativity were inversely correlated with the initial Ct-value. The timing of retesting after a positive SARS-CoV-2 RT-PCR appears to be significantly influenced by the initial Ct-value. Although it can be assumed that Ct-values will increase steadily over time, strategies that rely on rigid Ct-cutoffs should be discussed critically, not only because of methodological caveats but also because of the strain on testing infrastructure caused by the incentive for early retesting that Ct-values apparently represent.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52984-7" target="_blank">https://www.nature.com/articles/s41598-024-52984-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>MVA-based vaccine candidates encoding the native or prefusion-stabilized SARS-CoV-2 spike reveal differential immunogenicity in humans</h3>
            <h3 id="section42"><a href="#section42" style="text-decoration: none; color: inherit;">ü§ñ "New SARS-CoV-2 Vaccine Candidates Generated Using mRNA Technology"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-26</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>Two different versions of a vaccine for COVID-19 were tested on volunteers and found to be effective at increasing T cell and antibody response when used as a third dose. The vaccine containing the prefusion-stabilized spike protein led to more S1-specific responses, while the one with the native spike elicited more S2-specific responses. This shows how the type of virus protein being used in the vaccine can impact its effectiveness.</p>
            <br><strong>Abstract</strong><br>
            <p>In response to the COVID-19 pandemic, multiple vaccines were developed using platforms such as viral vectors and mRNA technology. Here, we report humoral and cellular immunogenicity data from human phase 1 clinical trials investigating two recombinant Modified Vaccinia virus Ankara vaccine candidates, MVA-SARS-2-S and MVA-SARS-2-ST, encoding the native and the prefusion-stabilized SARS-CoV-2 spike protein, respectively. MVA-SARS-2-ST was more immunogenic than MVA-SARS-2-S, but both were less immunogenic compared to licensed mRNA- and ChAd-based vaccines in SARS-CoV-2 na√Øve individuals. In heterologous vaccination, previous MVA-SARS-2-S vaccination enhanced T cell functionality and MVA-SARS-2-ST boosted the frequency of T cells and S1-specific IgG levels when used as a third vaccination. While the vaccine candidate containing the prefusion-stabilized spike elicited predominantly S1-specific responses, immunity to the candidate with the native spike was skewed towards S2-specific responses. These data demonstrate how the spike antigen conformation, using the same viral vector, directly affects vaccine immunogenicity in humans.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00801-z" target="_blank">https://www.nature.com/articles/s41541-023-00801-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Human coronavirus OC43-elicited CD4+ T cells protect against SARS-CoV-2 in HLA transgenic mice</h3>
            <h3 id="section43"><a href="#section43" style="text-decoration: none; color: inherit;">ü§ñ "Human Coronavirus Protects Against COVID-19 in Mice"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-26</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research suggests that a common cold virus called OC43 can cause our body to create T cells that can fight against the COVID-19 virus. This could potentially reduce the severity of COVID-19 cases in people who have previously been infected with OC43, but more research is needed to fully understand its implications for vaccines and treatments.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2-reactive T cells are detected in some healthy unexposed individuals. Human studies indicate these T cells could be elicited by the common cold coronavirus OC43. To directly test this assumption and define the role of OC43-elicited T cells that are cross-reactive with SARS-CoV-2, we develop a model of sequential infections with OC43 followed by SARS-CoV-2 in HLA-B*0702 and HLA-DRB1*0101Ifnar1‚àí/‚àítransgenic mice. We find that OC43 infection can elicit polyfunctional CD8+and CD4+effector T cells that cross-react with SARS-CoV-2 peptides. Furthermore, pre-exposure to OC43 reduces subsequent SARS-CoV-2 infection and disease in the lung for a short-term in HLA-DRB1*0101Ifnar1‚àí/‚àítransgenic mice, and a longer-term in HLA-B*0702Ifnar1‚àí/‚àítransgenic mice. Depletion of CD4+T cells in HLA-DRB1*0101Ifnar1‚àí/‚àítransgenic mice with prior OC43 exposure results in increased viral burden in the lung but no change in virus-induced lung damage following infection with SARS-CoV-2 (versus CD4+T cell-sufficient mice), demonstrating that the OC43-elicited SARS-CoV-2 cross-reactive T cell-mediated cross-protection against SARS-CoV-2 is partially dependent on CD4+T cells. These findings contribute to our understanding of the origin of pre-existing SARS-CoV-2-reactive T cells and their effects on SARS-CoV-2 clinical outcomes, and also carry implications for development of broadly protective betacoronavirus vaccines.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45043-2" target="_blank">https://www.nature.com/articles/s41467-024-45043-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Longitudinal plasma proteomics reveals biomarkers of alveolar-capillary barrier disruption in critically ill COVID-19 patients</h3>
            <h3 id="section44"><a href="#section44" style="text-decoration: none; color: inherit;">ü§ñ COVID-19 Patients: Plasma Proteomics Discover Biomarkers for Alveolar-Capillary Barrier Damage</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-25</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study shows that imatinib treatment helps reduce the need for ventilation and mortality in critically ill COVID-19 patients by targeting inflammation and extracellular matrix turnover pathways, as seen through changes in 6385 plasma proteins.</p>
            <br><strong>Abstract</strong><br>
            <p>The pathobiology of respiratory failure in COVID-19 consists of a complex interplay between viral cytopathic effects and a dysregulated host immune response. In critically ill patients, imatinib treatment demonstrated potential for reducing invasive ventilation duration and mortality. Here, we perform longitudinal profiling of 6385 plasma proteins in 318 hospitalised patients to investigate the biological processes involved in critical COVID-19, and assess the effects of imatinib treatment. Nine proteins measured at hospital admission accurately predict critical illness development. Next to dysregulation of inflammation, critical illness is characterised by pathways involving cellular adhesion, extracellular matrix turnover and tissue remodelling. Imatinib treatment attenuates protein perturbations associated with inflammation and extracellular matrix turnover. These proteomic alterations are contextualised using external pulmonary RNA-sequencing data of deceased COVID-19 patients and imatinib-treated Syrian hamsters. Together, we show that alveolar capillary barrier disruption in critical COVID-19 is reflected in the plasma proteome, and is attenuated with imatinib treatment. This study comprises a secondary analysis of both clinical data and plasma samples derived from a clinical trial that was registered with the EU Clinical Trials Register (EudraCT 2020‚Äì001236‚Äì10,https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-001236-10/NL) and Netherlands Trial Register (NL8491,https://www.trialregister.nl/trial/8491).</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44986-w" target="_blank">https://www.nature.com/articles/s41467-024-44986-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Identification of shared pathogenetic mechanisms between COVID-19 and IC through bioinformatics and system biology</h3>
            <h3 id="section45"><a href="#section45" style="text-decoration: none; color: inherit;">ü§ñ Understanding Shared Mechanisms in COVID-19 & Influenza with Bioinformatics and Systems Biology</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-24</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study investigates the molecular mechanisms of COVID-19-associated cystitis using bioinformatics and systems biology, identifying potential drug candidates and hub genes to improve diagnosis and treatment of IC patients infected with COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>COVID-19 increased global mortality in 2019. Cystitis became a contributing factor in SARS-CoV-2 and COVID-19 complications. The complex molecular links between cystitis and COVID-19 are unclear. This study investigates COVID-19-associated cystitis (CAC) molecular mechanisms and drug candidates using bioinformatics and systems biology. Obtain the gene expression profiles of IC (GSE11783) and COVID-19 (GSE147507) from the Gene Expression Omnibus (GEO) database. Identified the common differentially expressed genes (DEGs) in both IC and COVID-19, and extracted a number of key genes from this group. Subsequently, conduct Gene Ontology (GO) functional enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis on the DEGs. Additionally, design a protein‚Äìprotein interaction (PPI) network, a transcription factor gene regulatory network, a TF miRNA regulatory network, and a gene disease association network using the DEGs. Identify and extract hub genes from the PPI network. Then construct Nomogram diagnostic prediction models based on the hub genes. The DSigDB database was used to forecast many potential molecular medicines that are associated with common DEGs. Assess the precision of hub genes and Nomogram models in diagnosing IC and COVID-19 by employing Receiver Operating Characteristic (ROC) curves. The IC dataset (GSE57560) and the COVID-19 dataset (GSE171110) were selected to validate the models' diagnostic accuracy. A grand total of 198 DEGs that overlapped were found and chosen for further research. FCER1G, ITGAM, LCP2, LILRB2, MNDA, SPI1, and TYROBP were screened as the hub genes. The Nomogram model, built using the seven hub genes, demonstrates significant utility as a diagnostic prediction model for both IC and COVID-19. Multiple potential molecular medicines associated with common DEGs have been discovered. These pathways, hub genes, and models may provide new perspectives for future research into mechanisms and guide personalised and effective therapeutics for IC patients infected with COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52625-z" target="_blank">https://www.nature.com/articles/s41598-024-52625-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Respiratory distress in SARS-CoV-2 exposed uninfected neonates followed in the COVID Outcomes in Mother-Infant Pairs (COMP) Study</h3>
            <h3 id="section46"><a href="#section46" style="text-decoration: none; color: inherit;">ü§ñ "Sick Babies from Coronavirus? Study Compares COVID Cases"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-24</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study shows that unvaccinated pregnant women who were infected with SARS-CoV-2 are 3 times more likely to have their newborns experience respiratory distress compared to those who received a vaccine before infection. Infants exposed in utero to COVID-19 may exhibit increased inflammatory response and ciliary dysregulation, which is associated with higher levels of IgE production. Maternal vaccination against COVID-19 may reduce the frequency of neonatal respiratory distress.</p>
            <br><strong>Abstract</strong><br>
            <p>Respiratory distress (RD) has been reported in SARS-CoV-2 exposed uninfected (SEU) term neonates. Prior studies suggest that prenatal exposure to Coronavirus Disease 19 (COVID-19) may activate an inflammatory cascade in the newborn airway. In this study, we examine the relationship between maternal COVID-19 vaccination and neonatal RD using a longitudinal cohort of mother-infant pairs in Los Angeles, CA. Two-hundred and twenty-one mothers with laboratory confirmed SARS-CoV-2 during pregnancy and 227 exposed fetuses are enrolled in our study. Maternal disease severity and neonatal RD variables were defined based on current accepted clinical criteria. To explore the multifactorial associations between maternal COVID-19 parameters and infant RD, we utilize a multivariable logistic regression model and a proteomic sub-analysis to propose a pathway for the development of RD followingin uteroexposure to SARS-CoV-2. Unusually high rates of RD are observed in SEU infants (17%). The odds ratio of RD is 3.06 (95% CI:1.08-10.21) in term neonates born to unvaccinated individuals versus those born to individuals vaccinated prior to maternal infection. Proteomic analysis reveals a robust inflammatory response associated with ciliary dysregulation and enhanced IgE production among SEU neonates with RD. Maternal vaccination against COVID-19 reduces the frequency of neonatal RD.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">https://www.nature.com/articles/s41467-023-44549-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Type 2 diabetes and its genetic susceptibility are associated with increased severity and mortality of COVID-19 in UK Biobank</h3>
            <h3 id="section47"><a href="#section47" style="text-decoration: none; color: inherit;">ü§ñ "Genetic Factors Linked to Severe COVID-19, UK Biobank Study Finds"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-24</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research analyzes the impact of Type 2 diabetes (T2D) genetic susceptibility on COVID-19 severity and mortality using 459,119 individuals from UK Biobank. It shows that vaccination is essential and highlights the role of T2D-related genetics in severe COVID-19, with a significantly increased risk for patients with T2D or high PRS levels.</p>
            <br><strong>Abstract</strong><br>
            <p>Type 2 diabetes (T2D) is known as one of the important risk factors for the severity and mortality of COVID-19. Here, we evaluate the impact of T2D and its genetic susceptibility on the severity and mortality of COVID-19, using 459,119 individuals in UK Biobank. Utilizing the polygenic risk scores (PRS) for T2D, we identified a significant association between T2D or T2D PRS, and COVID-19 severity. We further discovered the efficacy of vaccination and the pivotal role of T2D-related genetics in the pathogenesis of severe COVID-19. Moreover, we found that individuals with T2D or those in the high T2D PRS group had a significantly increased mortality rate. We also observed that the mortality rate for SARS-CoV-2-infected patients was approximately 2 to 7 times higher than for those not infected, depending on the time of infection. These findings emphasize the potential of T2D PRS in estimating the severity and mortality of COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-05799-1" target="_blank">https://www.nature.com/articles/s42003-024-05799-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Structural and biochemical rationale for Beta variant protein booster vaccine broad cross-neutralization of SARS-CoV-2</h3>
            <h3 id="section48"><a href="#section48" style="text-decoration: none; color: inherit;">ü§ñ Beta Variant Protein Booster Vaccine Boosts Immunity Against COVID-19</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-23</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>The SARS-CoV-2 virus uses a specific protein on its surface to enter cells, which is the target for antibodies in vaccines. Researchers have found that the Beta trimer structure of the protein is favored by antibodies and has better neutralizing ability as a booster vaccine. Understanding the differences between spike protein structures can help improve vaccine efficacy against emerging variants.</p>
            <br><strong>Abstract</strong><br>
            <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), responsible for the COVID-19 pandemic, uses a surface expressed trimeric spike glycoprotein for cell entry. This trimer is the primary target for neutralizing antibodies making it a key candidate for vaccine development. During the global pandemic circulating variants of concern (VOC) caused several waves of infection, severe disease, and death. The reduced efficacy of the ancestral trimer-based vaccines against emerging VOC led to the need for booster vaccines. Here we present a detailed characterization of the Sanofi Beta trimer, utilizing cryo-EM for structural elucidation. We investigate the conformational dynamics and stabilizing features using orthogonal SPR, SEC, nanoDSF, and HDX-MS techniques to better understand how this antigen elicits superior broad neutralizing antibodies as a variant booster vaccine. This structural analysis confirms the Beta trimer preference for canonical quaternary structure with two RBD in the up position and the reversible equilibrium between the canonical spike and open trimer conformations. Moreover, this report provides a better understanding of structural differences between spike antigens contributing to differential vaccine efficacy.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52499-1" target="_blank">https://www.nature.com/articles/s41598-024-52499-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Non-fatal outcomes of COVID-19 disease in pediatric organ transplantation associates with down-regulation of senescence pathways</h3>
            <h3 id="section49"><a href="#section49" style="text-decoration: none; color: inherit;">ü§ñ "Non-Fatal COVID Outcomes in Transplanted Kids Linked to Senescence Pathway Reduction"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-22</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study examines the immune response and durability in children with solid organ transplants who had COVID-19, finding that low CD57 expression and increased frequencies of certain cells may protect against severe outcomes.</p>
            <br><strong>Abstract</strong><br>
            <p>This is a cross-sectional study examining kinetics and durability of immune response in children with solid organ transplants (SOTs) who had COVID-19 disease between November 2020 through June 2022, who were followed for 60-days at a single transplant center. Blood was collected between 1‚Äì14 (acute infection), and 15‚Äì60¬†days of a positive PCR (convalescence). SOT children with peripheral blood mononuclear cells (PBMC) cryopreserved before 2019 were non-infected controls (ctrls). PBMCs stimulated with 15-mer peptides from spike protein and anti-CD49d/anti-CD28. Testing done included mass cytometry, mi-RNA sequencing with confirmatory qPCR. 38 children formed the study cohort, 10 in the acute phase and 8 in the convalescence phase. 20 subjects were non-infected controls. Two subjects had severe disease. Subjects in the acute and convalescent phases were different subjects. The median age and tacrolimus level at blood draw was not significantly different. There was no death, and no subject was lost to follow-up. During acute infection CD57 expression was low in NKT, Th17 effector memory, memory Treg, CD4‚àíCD8‚àí, and Œ≥Œ¥T cells (p=‚Äâ0.01,p=‚Äâ0.04,p=‚Äâ0.03,p=‚Äâ0.03, p‚Äâ=‚Äâ0.004 respectively). The frequencies of NK and Th2 effector memory cells increased (p=‚Äâ0.01,p=‚Äâ0.02) during acute infection. Non-switched memory B and CD8 central memory cell frequencies were decreased during acute infection (p=‚Äâ0.02;p=‚Äâ0.02), but the decrease in CD8 central memory cells did not persist. CD4‚àíCD8‚àíand CD14 monocyte frequencies increased during recovery (p=‚Äâ0.03;p=‚Äâ0.007). Our observations suggest down regulation of CD57 with absence of NK cell contraction protect against death from COVID-19 disease in children with SOTs.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52456-y" target="_blank">https://www.nature.com/articles/s41598-024-52456-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Three immunizations with Novavax‚Äôs protein vaccines increase antibody breadth and provide durable protection from SARS-CoV-2</h3>
            <h3 id="section50"><a href="#section50" style="text-decoration: none; color: inherit;">ü§ñ "3 Novavax vaccines boost antibodies & offer long-term protection against COVID."</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study shows that three doses of a Novavax COVID-19 vaccine can improve its effectiveness in macaques by enhancing local immunity and inducing rapid anamnestic antibody and T cell responses in the lungs, leading to better protection against different strains of the virus.</p>
            <br><strong>Abstract</strong><br>
            <p>The immune responses to Novavax‚Äôs licensed NVX-CoV2373 nanoparticle Spike protein vaccine against SARS-CoV-2 remain incompletely understood. Here, we show in rhesus macaques that immunization with Matrix-MTMadjuvanted vaccines predominantly elicits immune events in local tissues with little spillover to the periphery. A third dose of an updated vaccine based on the Gamma (P.1) variant 7 months after two immunizations with licensed NVX-CoV2373 resulted in significant enhancement of anti-spike antibody titers and antibody breadth including neutralization of forward drift Omicron variants. The third immunization expanded the Spike-specific memory B cell pool, induced significant somatic hypermutation, and increased serum antibody avidity, indicating considerable affinity maturation. Seven months after immunization, vaccinated animals controlled infection by either WA-1 or P.1 strain, mediated by rapid anamnestic antibody and T cell responses in the lungs. In conclusion, a third immunization with an adjuvanted, low-dose recombinant protein vaccine significantly improved the quality of B cell responses, enhanced antibody breadth, and provided durable protection against SARS-CoV-2 challenge.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00806-2" target="_blank">https://www.nature.com/articles/s41541-024-00806-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Proteomic analysis of SARS-CoV-2 particles unveils a key role of G3BP proteins in viral assembly</h3>
            <h3 id="section51"><a href="#section51" style="text-decoration: none; color: inherit;">ü§ñ "New Study Shows G3BP Protein is Important for COVID-19"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study characterizes the proteins involved in the formation of newly formed virus particles and identifies stress granule nucleators as key players. It also suggests that G3BP1/2 play a role in viral dissemination, providing insights into SARS-CoV-2 assembly.</p>
            <br><strong>Abstract</strong><br>
            <p>Considerable progress has been made in understanding the molecular host-virus battlefield during SARS-CoV-2 infection. Nevertheless, the assembly and egress of newly formed virions are less understood. To identify host proteins involved in viral morphogenesis, we characterize the proteome of SARS-CoV-2 virions produced from A549-ACE2 and Calu-3 cells, isolated via ultracentrifugation on sucrose cushion or by ACE-2 affinity capture. Bioinformatic analysis unveils 92 SARS-CoV-2 virion-associated host factors, providing a valuable resource to better understand the molecular environment of virion production. We reveal that G3BP1 and G3BP2 (G3BP1/2), two major stress granule nucleators, are embedded within virions and unexpectedly favor virion production. Furthermore, we show that G3BP1/2 participate in the formation of cytoplasmic membrane vesicles, that are likely virion assembly sites, consistent with a proviral role of G3BP1/2 in SARS-CoV-2 dissemination. Altogether, these findings provide new insights into host factors required for SARS-CoV-2 assembly with potential implications for future therapeutic targeting.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44958-0" target="_blank">https://www.nature.com/articles/s41467-024-44958-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Development and validation of cost-effective SYBR Green-based RT-qPCR and its evaluation in a sample pooling strategy for detecting SARS-CoV-2 infection in the Indonesian setting</h3>
            <h3 id="section52"><a href="#section52" style="text-decoration: none; color: inherit;">ü§ñ "Using a simple test, we can find COVID-19 faster and cheaper!"  # Easy to understand</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>Scientists have developed an inexpensive RT-qPCR method to detect SARS-CoV-2 using SYBR Green, a fluorescent dye. Primers for N genes are designed and tested against Indonesian sequences. The test is reliable with high sensitivity, except in low viral load situations where it may miss positive cases. Pooling strategy provides higher diagnostic yield but could increase the risk of false negatives.</p>
            <br><strong>Abstract</strong><br>
            <p>A low-cost SYBR Green-based RT-qPCR method to detect SARS-CoV-2 were developed and validated. Primers targeting a conserved and vital region of the N genes of SARS-CoV-2 were designed.In-silicostudy was performed to analyse the compatibility of the selected primer pair with Indonesian SARS-CoV-2 genome sequences available from the GISAID database. We determined the linearity of our new assay using serial dilution of SARS-CoV-2 RNA from clinical samples with known virus concentration. The assay was then evaluated using clinically relevant samples in comparison to a commercial TaqMan-based test kit. Finally, we applied the assay in sample pooling strategies for SARS-CoV-2 detection. The SYBR Green-based RT-qPCR method was successfully developed with sufficient sensitivity. There is a very low prevalence of genome variation in the selected N primer binding regions, indicating their high conservation. The validation of the assay using clinical samples demonstrated similar performance to the TaqMan method suggesting the SYBR methods is reliable. The pooling strategy by combining 5 RNA samples for SARS-CoV-2 detection using the SYBR RT-qPCR methods is feasible and provides a high diagnostic yield. However, when dealing with samples having a very low viral load, it may increase the risk of missing positive cases.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52250-w" target="_blank">https://www.nature.com/articles/s41598-024-52250-w</a></p>
            <br><br>
        </div>
      
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
     <!-- end footer -->
     <!-- Javascript files--> 
     <script src="js/jquery.min.js"></script> 
     <script src="js/popper.min.js"></script> 
     <script src="js/bootstrap.bundle.min.js"></script> 
     <script src="js/jquery-3.0.0.min.js"></script> 
     <script src="js/plugin.js"></script> 
     <!-- sidebar --> 
     <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
     <script src="js/custom.js"></script>
     <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
     <script>
        $(document).ready(function(){
        $(".fancybox").fancybox({
        openEffect: "none",
        closeEffect: "none"
        });
        
        $(".zoom").hover(function(){
        
        $(this).addClass('transition');
        }, function(){
        
        $(this).removeClass('transition');
        });
        });   
     </script>
     
  </body>
</html>
