
<!DOCTYPE html>
<html lang="en">
   <head>
      <!-- basic -->
      <meta charset="utf-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- mobile metas -->
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <meta name="viewport" content="initial-scale=1, maximum-scale=1">
      <!-- site metas -->
      <title>Data Sources</title>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link rel="icon" href="images/logo.png" type="image/x-icon">
      <meta name="keywords" content="COVID-19, wastewater, measurement, monitoring, surveillance, public health, environmental monitoring, water analysis, viral RNA, SARS-CoV-2,SARS-nCoV-2, global, countries, data, analysis, pandemic, wastewater-based epidemiology, geo-map">
      <meta name="description" content="Data Sources of Visualizations">
      <meta name="author" content="Daniel Dynesius">
      <!-- bootstrap css -->
      <link rel="stylesheet" href="css/bootstrap.min.css">
      <!-- style css -->
      <link rel="stylesheet" href="css/style.css">
      <!-- Responsive-->
      <link rel="stylesheet" href="css/responsive.css">
      <!-- fevicon -->
      <link rel="icon" href="images/fevicon.png" type="image/gif" />
      <!-- Scrollbar Custom CSS -->
      <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
      <!-- Tweaks for older IEs-->
      <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
      <style>
            @media (max-width: 767px) {
            /* Styles for screens smaller than 768px (mobile) */
            .menu-area-main {
                flex-direction: column;
            }
            .menu-area-main li {
                text-align: center;
            }
            }
    </style>      
      <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script><![endif]-->
   </head>
   <!-- body -->
   <body class="main-layout">
      <!-- loader  -->
      <div class="loader_bg">
         <div class="loader"><img src="images/loading.gif" alt="#" /></div>
      </div>
      <!-- end loader --> 
      <!-- header -->
      <header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                            .throb-line:hover::after {
                                content: ' ❤️'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                        <p class="throb-line">It's better to prevent.</p>
                    </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo"> <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a> </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main d-flex">
                                    <li class=""> <a href="index.html">Geo Map</a> </li>
                                    <li class=""> <a href="trends.html">Trend Graphs</a> </li>
                                    <li class=""> <a href="predictions_tab.html">Predictive</a> </li>
                                    <li class="active"> <a href="new_research.html">Research News</a> </li>
                                    <li class=""> <a href="data_sources.html">Data Sources</a> </li>
                                    <li class=""> <a href="about.html">About</a> </li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
         <!-- end header inner --> 
      </header>
      <!-- end header -->
      <div class="brand_color">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="titlepage">
                        <h2>Research News</h2>
                        <br>
                        <h4>🤖 Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                        Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>
                    </div>
                </div>
            </div>
        </div>
    </div>
    <!-- contact -->
<p>
    <br><br><br>

        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Necroptosis does not drive disease pathogenesis in a mouse infective model of SARS-CoV-2 in vivo</h3>
            <h3 id="section1"><a href="#section1" style="text-decoration: none; color: inherit;">🤖 In mice, SARS-CoV-2 infection doesn't cause necroptosis, a cell death process, to worsen the disease.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates the role of necroptosis in SARS-CoV-2 infection, finding that it may not be a major factor in mild or severe COVID-19 disease.</p>
            <br><strong>Abstract</strong><br>
            <p>Necroptosis, a type of lytic cell death executed by the pseudokinase Mixed Lineage Kinase Domain-Like (MLKL) has been implicated in the detrimental inflammation caused by SARS-CoV-2 infection. We minimally and extensively passaged a single clinical SARS-CoV-2 isolate to create models of mild and severe disease in mice allowing us to dissect the role of necroptosis in SARS-CoV-2 disease pathogenesis. We infected wild-type and MLKL-deficient mice and found no significant differences in viral loads or lung pathology. In our model of severe COVID-19, MLKL-deficiency did not alter the host response, ameliorate weight loss, diminish systemic pro-inflammatory cytokines levels, or prevent lethality in aged animals. Our in vivo models indicate that necroptosis is dispensable in the pathogenesis of mild and severe COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-024-06471-6" target="_blank">https://www.nature.com/articles/s41419-024-06471-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Omicron Spike confers enhanced infectivity and interferon resistance to SARS-CoV-2 in human nasal tissue</h3>
            <h3 id="section2"><a href="#section2" style="text-decoration: none; color: inherit;">🤖 The Omicron variant of COVID-19 increases the virus's ability to infect human nasal tissue while also making it resistant to interferons, which are proteins that help fight off viruses.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-30</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research shows that the Omicron variant exhibits increased infectivity in primary adult upper airway tissue compared to Delta, due to unique mutations in its spike protein allowing for enhanced entry into nasal cells and evasion from antiviral factors.</p>
            <br><strong>Abstract</strong><br>
            <p>Omicron emerged following COVID-19 vaccination campaigns, displaced previous SARS-CoV-2 variants of concern worldwide, and gave rise to lineages that continue to spread. Here, we show that Omicron exhibits increased infectivity in primary adult upper airway tissue relative to Delta. Using recombinant forms of SARS-CoV-2 and nasal epithelial cells cultured at the liquid-air interface, we show that mutations unique to Omicron Spike enable enhanced entry into nasal tissue. Unlike earlier variants of SARS-CoV-2, our findings suggest that Omicron enters nasal cells independently of serine transmembrane proteases and instead relies upon metalloproteinases to catalyze membrane fusion. Furthermore, we demonstrate that this entry pathway unlocked by Omicron Spike enables evasion from constitutive and interferon-induced antiviral factors that restrict SARS-CoV-2 entry following attachment. Therefore, the increased transmissibility exhibited by Omicron in humans may be attributed not only to its evasion of vaccine-elicited adaptive immunity, but also to its superior invasion of nasal epithelia and resistance to the cell-intrinsic barriers present therein.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45075-8" target="_blank">https://www.nature.com/articles/s41467-024-45075-8</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Identification of a highly conserved neutralizing epitope within the RBD region of diverse SARS-CoV-2 variants</h3>
            <h3 id="section3"><a href="#section3" style="text-decoration: none; color: inherit;">🤖 A study finds a common, unchanging part in SARS-CoV-2 variants that can help block infection.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-29</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research identifies a broad-spectrum neutralizing antibody, BA7535, that effectively targets conserved epitopes in SARS-CoV-2 variants, including recent Omicron subvariants, offering potential for highly potent therapeutic antibodies and vaccines.</p>
            <br><strong>Abstract</strong><br>
            <p>The constant emergence of SARS-CoV-2 variants continues to impair the efficacy of existing neutralizing antibodies, especially XBB.1.5 and EG.5, which showed exceptional immune evasion properties. Here, we identify a highly conserved neutralizing epitope targeted by a broad-spectrum neutralizing antibody BA7535, which demonstrates high neutralization potency against not only previous variants, such as Alpha, Beta, Gamma, Delta and Omicron BA.1-BA.5, but also more recently emerged Omicron subvariants, including BF.7, CH.1.1, XBB.1, XBB.1.5, XBB.1.9.1, EG.5. Structural analysis of the Omicron Spike trimer with BA7535-Fab using cryo-EM indicates that BA7535 recognizes a highly conserved cryptic receptor-binding domain (RBD) epitope, avoiding most of the mutational hot spots in RBD. Furthermore, structural simulation based on the interaction of BA7535-Fab/RBD complexes dissects the broadly neutralizing effect of BA7535 against latest variants. Therapeutic and prophylactic treatment with BA7535 alone or in combination with BA7208 protected female mice from the circulating Omicron BA.5 and XBB.1 variant infection, suggesting the highly conserved neutralizing epitope serves as a potential target for developing highly potent therapeutic antibodies and vaccines.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45050-3" target="_blank">https://www.nature.com/articles/s41467-024-45050-3</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Cycle threshold of SARS-CoV-2 RT-PCR as a driver of retesting</h3>
            <h3 id="section4"><a href="#section4" style="text-decoration: none; color: inherit;">🤖 The research examines how the cycle threshold value in COVID-19 RT-PCR tests affects the need for retesting.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-29</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study examined how often patients were retested for COVID-19 after an initial positive test and found that the timing of the first retest was influenced by the initial cycle threshold (Ct) value, with higher Ct values leading to earlier retesting, which could strain testing infrastructure.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 RT-PCR is a critical and, at times, limited resource. Frequent Retesting of patients may strain testing infrastructure unduly. Recommendations that include cycle threshold (Ct) cutoffs may incentivize early retesting when the Ct value is reported. We aimed to investigate patterns of retesting in association with initial Ct-values. We performed a retrospective analysis of RT-PCR results (including Ct-values) for patients from whom ≥ 2 samples were collected within 14 days, the first of which had to be positive. We calculated absolute and baseline-corrected kinetics of Ct-values over time, as well as the median initial Ct-values in dependence of the timing of the first retesting and the time until RT-PCR negativity for SARS-CoV-2. Retesting after an initial positive SARS-CoV-2 RT-PCR was most commonly performed on day 7, with patients being retested as early as day 1. The majority of patients retested within 14 days remained SARS-CoV-2 positive in the RT-PCR. Baseline-corrected Ct-values showed a quasi-linear increase over 14 days since the initial positive result. Both the timing until the first retesting and until RT-PCR negativity were inversely correlated with the initial Ct-value. The timing of retesting after a positive SARS-CoV-2 RT-PCR appears to be significantly influenced by the initial Ct-value. Although it can be assumed that Ct-values will increase steadily over time, strategies that rely on rigid Ct-cutoffs should be discussed critically, not only because of methodological caveats but also because of the strain on testing infrastructure caused by the incentive for early retesting that Ct-values apparently represent.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52984-7" target="_blank">https://www.nature.com/articles/s41598-024-52984-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>MVA-based vaccine candidates encoding the native or prefusion-stabilized SARS-CoV-2 spike reveal differential immunogenicity in humans</h3>
            <h3 id="section5"><a href="#section5" style="text-decoration: none; color: inherit;">🤖 MVA-Based Vaccine Candidates Encoding Native or Prefusion-Stabilized SARS-CoV-2 Spike Show Different Immunogenicity in Humans</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-26</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates two recombinant vaccines against COVID-19 using different spike protein configurations and compares their immunogenicity to licensed mRNA- and ChAd-based vaccines, finding that the prefusion-stabilized vaccine was more effective.</p>
            <br><strong>Abstract</strong><br>
            <p>In response to the COVID-19 pandemic, multiple vaccines were developed using platforms such as viral vectors and mRNA technology. Here, we report humoral and cellular immunogenicity data from human phase 1 clinical trials investigating two recombinant Modified Vaccinia virus Ankara vaccine candidates, MVA-SARS-2-S and MVA-SARS-2-ST, encoding the native and the prefusion-stabilized SARS-CoV-2 spike protein, respectively. MVA-SARS-2-ST was more immunogenic than MVA-SARS-2-S, but both were less immunogenic compared to licensed mRNA- and ChAd-based vaccines in SARS-CoV-2 naïve individuals. In heterologous vaccination, previous MVA-SARS-2-S vaccination enhanced T cell functionality and MVA-SARS-2-ST boosted the frequency of T cells and S1-specific IgG levels when used as a third vaccination. While the vaccine candidate containing the prefusion-stabilized spike elicited predominantly S1-specific responses, immunity to the candidate with the native spike was skewed towards S2-specific responses. These data demonstrate how the spike antigen conformation, using the same viral vector, directly affects vaccine immunogenicity in humans.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00801-z" target="_blank">https://www.nature.com/articles/s41541-023-00801-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Human coronavirus OC43-elicited CD4+ T cells protect against SARS-CoV-2 in HLA transgenic mice</h3>
            <h3 id="section6"><a href="#section6" style="text-decoration: none; color: inherit;">🤖 Human OC43 Coronavirus Triggers Protective CD4+ T Cells Against SARS-CoV-2 in HLA Transgenic Mice</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-26</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research study aims to understand how a common cold coronavirus (OC43) might protect against SARS-CoV-2 infection by creating T cells that could cross-react with the new virus, and explores the role of CD4+T cells in this potential protection.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2-reactive T cells are detected in some healthy unexposed individuals. Human studies indicate these T cells could be elicited by the common cold coronavirus OC43. To directly test this assumption and define the role of OC43-elicited T cells that are cross-reactive with SARS-CoV-2, we develop a model of sequential infections with OC43 followed by SARS-CoV-2 in HLA-B*0702 and HLA-DRB1*0101Ifnar1−/−transgenic mice. We find that OC43 infection can elicit polyfunctional CD8+and CD4+effector T cells that cross-react with SARS-CoV-2 peptides. Furthermore, pre-exposure to OC43 reduces subsequent SARS-CoV-2 infection and disease in the lung for a short-term in HLA-DRB1*0101Ifnar1−/−transgenic mice, and a longer-term in HLA-B*0702Ifnar1−/−transgenic mice. Depletion of CD4+T cells in HLA-DRB1*0101Ifnar1−/−transgenic mice with prior OC43 exposure results in increased viral burden in the lung but no change in virus-induced lung damage following infection with SARS-CoV-2 (versus CD4+T cell-sufficient mice), demonstrating that the OC43-elicited SARS-CoV-2 cross-reactive T cell-mediated cross-protection against SARS-CoV-2 is partially dependent on CD4+T cells. These findings contribute to our understanding of the origin of pre-existing SARS-CoV-2-reactive T cells and their effects on SARS-CoV-2 clinical outcomes, and also carry implications for development of broadly protective betacoronavirus vaccines.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45043-2" target="_blank">https://www.nature.com/articles/s41467-024-45043-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>COVID-19 vaccines and beyond</h3>
            <h3 id="section7"><a href="#section7" style="text-decoration: none; color: inherit;">🤖 Understanding COVID-19 Vaccines and What Comes Next</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-26</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research examines the relationship between urbanization and air pollution in China, focusing on its impact on human health and environmental quality, while proposing potential solutions.</p>
            <br><strong>Abstract</strong><br>
            <p>No div with ID 'Abs1-section' found.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41423-024-01132-2" target="_blank">https://www.nature.com/articles/s41423-024-01132-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Longitudinal plasma proteomics reveals biomarkers of alveolar-capillary barrier disruption in critically ill COVID-19 patients</h3>
            <h3 id="section8"><a href="#section8" style="text-decoration: none; color: inherit;">🤖 This study finds specific markers in blood that show damage to the lung's blood vessels in severely ill COVID-19 patients.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-25</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research investigates the biological processes involved in critical COVID-19 using a longitudinal profiling of 6385 plasma proteins in 318 hospitalized patients and assesses the effects of imatinib treatment, showing that alveolar capillary barrier disruption is reflected in the plasma proteome and attenuated with imatinib treatment.</p>
            <br><strong>Abstract</strong><br>
            <p>The pathobiology of respiratory failure in COVID-19 consists of a complex interplay between viral cytopathic effects and a dysregulated host immune response. In critically ill patients, imatinib treatment demonstrated potential for reducing invasive ventilation duration and mortality. Here, we perform longitudinal profiling of 6385 plasma proteins in 318 hospitalised patients to investigate the biological processes involved in critical COVID-19, and assess the effects of imatinib treatment. Nine proteins measured at hospital admission accurately predict critical illness development. Next to dysregulation of inflammation, critical illness is characterised by pathways involving cellular adhesion, extracellular matrix turnover and tissue remodelling. Imatinib treatment attenuates protein perturbations associated with inflammation and extracellular matrix turnover. These proteomic alterations are contextualised using external pulmonary RNA-sequencing data of deceased COVID-19 patients and imatinib-treated Syrian hamsters. Together, we show that alveolar capillary barrier disruption in critical COVID-19 is reflected in the plasma proteome, and is attenuated with imatinib treatment. This study comprises a secondary analysis of both clinical data and plasma samples derived from a clinical trial that was registered with the EU Clinical Trials Register (EudraCT 2020–001236–10,https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-001236-10/NL) and Netherlands Trial Register (NL8491,https://www.trialregister.nl/trial/8491).</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44986-w" target="_blank">https://www.nature.com/articles/s41467-024-44986-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Respiratory distress in SARS-CoV-2 exposed uninfected neonates followed in the COVID Outcomes in Mother-Infant Pairs (COMP) Study</h3>
            <h3 id="section9"><a href="#section9" style="text-decoration: none; color: inherit;">🤖 Breathing Problems in Babies Not Infected with COVID-19 but Exposed to SARS-CoV-2 Virus in the COVID-Outcomes in Mother-Infant Pairs (COMP) Study</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-24</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates the relationship between maternal COVID-19 vaccination and neonatal respiratory distress in term neonates, finding that maternal vaccination significantly reduced the occurrence of neonatal respiratory distress.</p>
            <br><strong>Abstract</strong><br>
            <p>Respiratory distress (RD) has been reported in SARS-CoV-2 exposed uninfected (SEU) term neonates. Prior studies suggest that prenatal exposure to Coronavirus Disease 19 (COVID-19) may activate an inflammatory cascade in the newborn airway. In this study, we examine the relationship between maternal COVID-19 vaccination and neonatal RD using a longitudinal cohort of mother-infant pairs in Los Angeles, CA. Two-hundred and twenty-one mothers with laboratory confirmed SARS-CoV-2 during pregnancy and 227 exposed fetuses are enrolled in our study. Maternal disease severity and neonatal RD variables were defined based on current accepted clinical criteria. To explore the multifactorial associations between maternal COVID-19 parameters and infant RD, we utilize a multivariable logistic regression model and a proteomic sub-analysis to propose a pathway for the development of RD followingin uteroexposure to SARS-CoV-2. Unusually high rates of RD are observed in SEU infants (17%). The odds ratio of RD is 3.06 (95% CI:1.08-10.21) in term neonates born to unvaccinated individuals versus those born to individuals vaccinated prior to maternal infection. Proteomic analysis reveals a robust inflammatory response associated with ciliary dysregulation and enhanced IgE production among SEU neonates with RD. Maternal vaccination against COVID-19 reduces the frequency of neonatal RD.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">https://www.nature.com/articles/s41467-023-44549-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Type 2 diabetes and its genetic susceptibility are associated with increased severity and mortality of COVID-19 in UK Biobank</h3>
            <h3 id="section10"><a href="#section10" style="text-decoration: none; color: inherit;">🤖 "Type 2 Diabetes Connection to Genetic Vulnerability and Severe Outcomes in COVID-19 Infections: A Study of the UK Biobank"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-24</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research examines the impact of Type 2 diabetes (T2D) and genetic susceptibility on COVID-19 severity and mortality, finding a significant association between T2D or T2D PRS and COVID-19 severity, with individuals having T2D or high T2D PRS experiencing significantly increased mortality rates.</p>
            <br><strong>Abstract</strong><br>
            <p>Type 2 diabetes (T2D) is known as one of the important risk factors for the severity and mortality of COVID-19. Here, we evaluate the impact of T2D and its genetic susceptibility on the severity and mortality of COVID-19, using 459,119 individuals in UK Biobank. Utilizing the polygenic risk scores (PRS) for T2D, we identified a significant association between T2D or T2D PRS, and COVID-19 severity. We further discovered the efficacy of vaccination and the pivotal role of T2D-related genetics in the pathogenesis of severe COVID-19. Moreover, we found that individuals with T2D or those in the high T2D PRS group had a significantly increased mortality rate. We also observed that the mortality rate for SARS-CoV-2-infected patients was approximately 2 to 7 times higher than for those not infected, depending on the time of infection. These findings emphasize the potential of T2D PRS in estimating the severity and mortality of COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-05799-1" target="_blank">https://www.nature.com/articles/s42003-024-05799-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Structural and biochemical rationale for Beta variant protein booster vaccine broad cross-neutralization of SARS-CoV-2</h3>
            <h3 id="section11"><a href="#section11" style="text-decoration: none; color: inherit;">🤖 Understanding and Explaining the Benefits of a Protein Booster Vaccine Targeting the Beta Variant to Combat SARS-CoV-2</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-23</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates the Sanofi Beta trimer's structural properties using various techniques, aiming to understand its ability to elicit broad neutralizing antibodies and improve the understanding of structural differences affecting vaccine efficacy.</p>
            <br><strong>Abstract</strong><br>
            <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), responsible for the COVID-19 pandemic, uses a surface expressed trimeric spike glycoprotein for cell entry. This trimer is the primary target for neutralizing antibodies making it a key candidate for vaccine development. During the global pandemic circulating variants of concern (VOC) caused several waves of infection, severe disease, and death. The reduced efficacy of the ancestral trimer-based vaccines against emerging VOC led to the need for booster vaccines. Here we present a detailed characterization of the Sanofi Beta trimer, utilizing cryo-EM for structural elucidation. We investigate the conformational dynamics and stabilizing features using orthogonal SPR, SEC, nanoDSF, and HDX-MS techniques to better understand how this antigen elicits superior broad neutralizing antibodies as a variant booster vaccine. This structural analysis confirms the Beta trimer preference for canonical quaternary structure with two RBD in the up position and the reversible equilibrium between the canonical spike and open trimer conformations. Moreover, this report provides a better understanding of structural differences between spike antigens contributing to differential vaccine efficacy.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52499-1" target="_blank">https://www.nature.com/articles/s41598-024-52499-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Non-fatal outcomes of COVID-19 disease in pediatric organ transplantation associates with down-regulation of senescence pathways</h3>
            <h3 id="section12"><a href="#section12" style="text-decoration: none; color: inherit;">🤖 In pediatric organ transplant patients, non-fatal COVID-19 outcomes are linked to a decrease in senescence pathways.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-22</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study examines the immune response in children with solid organ transplants who had COVID-19, finding that lower CD57 expression in specific immune cells may help protect against severe illness.</p>
            <br><strong>Abstract</strong><br>
            <p>This is a cross-sectional study examining kinetics and durability of immune response in children with solid organ transplants (SOTs) who had COVID-19 disease between November 2020 through June 2022, who were followed for 60-days at a single transplant center. Blood was collected between 1–14 (acute infection), and 15–60 days of a positive PCR (convalescence). SOT children with peripheral blood mononuclear cells (PBMC) cryopreserved before 2019 were non-infected controls (ctrls). PBMCs stimulated with 15-mer peptides from spike protein and anti-CD49d/anti-CD28. Testing done included mass cytometry, mi-RNA sequencing with confirmatory qPCR. 38 children formed the study cohort, 10 in the acute phase and 8 in the convalescence phase. 20 subjects were non-infected controls. Two subjects had severe disease. Subjects in the acute and convalescent phases were different subjects. The median age and tacrolimus level at blood draw was not significantly different. There was no death, and no subject was lost to follow-up. During acute infection CD57 expression was low in NKT, Th17 effector memory, memory Treg, CD4−CD8−, and γδT cells (p= 0.01,p= 0.04,p= 0.03,p= 0.03, p = 0.004 respectively). The frequencies of NK and Th2 effector memory cells increased (p= 0.01,p= 0.02) during acute infection. Non-switched memory B and CD8 central memory cell frequencies were decreased during acute infection (p= 0.02;p= 0.02), but the decrease in CD8 central memory cells did not persist. CD4−CD8−and CD14 monocyte frequencies increased during recovery (p= 0.03;p= 0.007). Our observations suggest down regulation of CD57 with absence of NK cell contraction protect against death from COVID-19 disease in children with SOTs.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52456-y" target="_blank">https://www.nature.com/articles/s41598-024-52456-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Three immunizations with Novavax’s protein vaccines increase antibody breadth and provide durable protection from SARS-CoV-2</h3>
            <h3 id="section13"><a href="#section13" style="text-decoration: none; color: inherit;">🤖 Three doses of Novavax's protein vaccines boost antibody diversity and offer long-lasting defense against COVID-19</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study shows that a third dose of Matrix-MTM adjuvanted vaccines after two doses of Novavax's NVX-CoV2373 vaccine in rhesus macaques significantly enhances anti-spike antibody titers, broadens the response to include neutralization of Omicron variants, expands Spike-specific memory B cell pool, and provides durable protection against SARS-CoV-2 challenge.</p>
            <br><strong>Abstract</strong><br>
            <p>The immune responses to Novavax’s licensed NVX-CoV2373 nanoparticle Spike protein vaccine against SARS-CoV-2 remain incompletely understood. Here, we show in rhesus macaques that immunization with Matrix-MTMadjuvanted vaccines predominantly elicits immune events in local tissues with little spillover to the periphery. A third dose of an updated vaccine based on the Gamma (P.1) variant 7 months after two immunizations with licensed NVX-CoV2373 resulted in significant enhancement of anti-spike antibody titers and antibody breadth including neutralization of forward drift Omicron variants. The third immunization expanded the Spike-specific memory B cell pool, induced significant somatic hypermutation, and increased serum antibody avidity, indicating considerable affinity maturation. Seven months after immunization, vaccinated animals controlled infection by either WA-1 or P.1 strain, mediated by rapid anamnestic antibody and T cell responses in the lungs. In conclusion, a third immunization with an adjuvanted, low-dose recombinant protein vaccine significantly improved the quality of B cell responses, enhanced antibody breadth, and provided durable protection against SARS-CoV-2 challenge.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00806-2" target="_blank">https://www.nature.com/articles/s41541-024-00806-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Development and validation of cost-effective SYBR Green-based RT-qPCR and its evaluation in a sample pooling strategy for detecting SARS-CoV-2 infection in the Indonesian setting</h3>
            <h3 id="section14"><a href="#section14" style="text-decoration: none; color: inherit;">🤖 "Creating and Testing an Affordable Method for Detection of COVID-19 Using SYBR Green and Pooled Samples in Indonesia"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>Researchers developed a low-cost SYBR Green-based RT-qPCR method for detecting SARS-CoV-2, using primers targeting a conserved region in the N gene and validated its sensitivity and reliability with clinical samples and pooling strategies.</p>
            <br><strong>Abstract</strong><br>
            <p>A low-cost SYBR Green-based RT-qPCR method to detect SARS-CoV-2 were developed and validated. Primers targeting a conserved and vital region of the N genes of SARS-CoV-2 were designed.In-silicostudy was performed to analyse the compatibility of the selected primer pair with Indonesian SARS-CoV-2 genome sequences available from the GISAID database. We determined the linearity of our new assay using serial dilution of SARS-CoV-2 RNA from clinical samples with known virus concentration. The assay was then evaluated using clinically relevant samples in comparison to a commercial TaqMan-based test kit. Finally, we applied the assay in sample pooling strategies for SARS-CoV-2 detection. The SYBR Green-based RT-qPCR method was successfully developed with sufficient sensitivity. There is a very low prevalence of genome variation in the selected N primer binding regions, indicating their high conservation. The validation of the assay using clinical samples demonstrated similar performance to the TaqMan method suggesting the SYBR methods is reliable. The pooling strategy by combining 5 RNA samples for SARS-CoV-2 detection using the SYBR RT-qPCR methods is feasible and provides a high diagnostic yield. However, when dealing with samples having a very low viral load, it may increase the risk of missing positive cases.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-52250-w" target="_blank">https://www.nature.com/articles/s41598-024-52250-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Proteomic analysis of SARS-CoV-2 particles unveils a key role of G3BP proteins in viral assembly</h3>
            <h3 id="section15"><a href="#section15" style="text-decoration: none; color: inherit;">🤖 Analysis of Virus Particles Reveals Important Role of G3BP Proteins in Viral Assembly</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-20</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research identifies 92 host proteins involved in SARS-CoV-2 virion production, revealing that G3BP1 and G3BP2 favor virion production and participate in vesicle formation, potentially paving the way for future therapeutic targeting.</p>
            <br><strong>Abstract</strong><br>
            <p>Considerable progress has been made in understanding the molecular host-virus battlefield during SARS-CoV-2 infection. Nevertheless, the assembly and egress of newly formed virions are less understood. To identify host proteins involved in viral morphogenesis, we characterize the proteome of SARS-CoV-2 virions produced from A549-ACE2 and Calu-3 cells, isolated via ultracentrifugation on sucrose cushion or by ACE-2 affinity capture. Bioinformatic analysis unveils 92 SARS-CoV-2 virion-associated host factors, providing a valuable resource to better understand the molecular environment of virion production. We reveal that G3BP1 and G3BP2 (G3BP1/2), two major stress granule nucleators, are embedded within virions and unexpectedly favor virion production. Furthermore, we show that G3BP1/2 participate in the formation of cytoplasmic membrane vesicles, that are likely virion assembly sites, consistent with a proviral role of G3BP1/2 in SARS-CoV-2 dissemination. Altogether, these findings provide new insights into host factors required for SARS-CoV-2 assembly with potential implications for future therapeutic targeting.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44958-0" target="_blank">https://www.nature.com/articles/s41467-024-44958-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Using BCG vaccination to protect against COVID-19: when reality fails to meet expectation</h3>
            <h3 id="section16"><a href="#section16" style="text-decoration: none; color: inherit;">🤖 Using BCG Vaccine for COVID-19 Protection: When Hopes Don't Match Reality</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The abstract is missing from the provided text, so a summary cannot be generated at this time. Please provide the research abstract for me to create a concise summary.</p>
            <br><strong>Abstract</strong><br>
            <p>No div with ID 'Abs1-section' found.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41577-024-00992-z" target="_blank">https://www.nature.com/articles/s41577-024-00992-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 immunity in animal models</h3>
            <h3 id="section17"><a href="#section17" style="text-decoration: none; color: inherit;">🤖 COVID-19 Immunity in Animal Studies</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This review discusses SARS-CoV-2 infection's severe complications, various animal models used to study the virus, and their respective immune responses, highlighting gaps in research between animal model studies and clinical investigations.</p>
            <br><strong>Abstract</strong><br>
            <p>The COVID-19 pandemic, which was caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has become a worldwide health crisis due to its transmissibility. SARS-CoV-2 infection results in severe respiratory illness and can lead to significant complications in affected individuals. These complications encompass symptoms such as coughing, respiratory distress, fever, infectious shock, acute respiratory distress syndrome (ARDS), and even multiple-organ failure. Animal models serve as crucial tools for investigating pathogenic mechanisms, immune responses, immune escape mechanisms, antiviral drug development, and vaccines against SARS-CoV-2. Currently, various animal models for SARS-CoV-2 infection, such as nonhuman primates (NHPs), ferrets, hamsters, and many different mouse models, have been developed. Each model possesses distinctive features and applications. In this review, we elucidate the immune response elicited by SARS-CoV-2 infection in patients and provide an overview of the characteristics of various animal models mainly used for SARS-CoV-2 infection, as well as the corresponding immune responses and applications of these models. A comparative analysis of transcriptomic alterations in the lungs from different animal models revealed that the K18-hACE2 and mouse-adapted virus mouse models exhibited the highest similarity with the deceased COVID-19 patients. Finally, we highlighted the current gaps in related research between animal model studies and clinical investigations, underscoring lingering scientific questions that demand further clarification.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41423-023-01122-w" target="_blank">https://www.nature.com/articles/s41423-023-01122-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Peptidyl nitroalkene inhibitors of main protease rationalized by computational and crystallographic investigations as antivirals against SARS-CoV-2</h3>
            <h3 id="section18"><a href="#section18" style="text-decoration: none; color: inherit;">🤖 Simplified Title: Rational Design of Peptide Nitroalkene Inhibitors for Main Protease using Computational and Crystallographic Studies as Potential Antiviral Agents against COVID-19</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The study investigates peptidyl nitroalkenes as potential anti-viral drugs for COVID-19, showing their effectiveness as reversible inhibitors of the viral main protease and cathepsin L without significant toxicity, with compounds FGA145, FGA146, and FGA147 demonstrating promising results.</p>
            <br><strong>Abstract</strong><br>
            <p>The coronavirus disease 2019 (COVID-19) pandemic continues to represent a global public health issue. The viral main protease (Mpro) represents one of the most attractive targets for the development of antiviral drugs. Herein we report peptidyl nitroalkenes exhibiting enzyme inhibitory activity against Mpro(Ki: 1–10 μM) good anti-SARS-CoV-2 infection activity in the low micromolar range (EC50: 1–12 μM) without significant toxicity. Additional kinetic studies of compoundsFGA145,FGA146andFGA147show that all three compounds inhibit cathepsin L, denoting a possible multitarget effect of these compounds in the antiviral activity. Structural analysis shows the binding mode ofFGA146andFGA147to the active site of the protein. Furthermore, our results illustrate that peptidyl nitroalkenes are effective covalent reversible inhibitors of the Mproand cathepsin L, and that inhibitorsFGA145,FGA146andFGA147prevent infection against SARS-CoV-2.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42004-024-01104-7" target="_blank">https://www.nature.com/articles/s42004-024-01104-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Risk for diagnosis or treatment of mood or anxiety disorders in adults after SARS-CoV-2 infection, 2020–2022</h3>
            <h3 id="section19"><a href="#section19" style="text-decoration: none; color: inherit;">🤖 Risk of Developing Mood or Anxiety Disorders in Adults After COVID-19 Infection, 2020-2022</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study found that patients with COVID-19 had a higher risk of developing mood or anxiety disorders and increased use of antidepressants and anxiolytics, particularly in older individuals and women, but the risk decreased over time from the beginning of the pandemic.</p>
            <br><strong>Abstract</strong><br>
            <p>COVID-19 is associated with increased risks for mood or anxiety disorders, but it remains uncertain how the association evolves over time or which patient groups are most affected. We conducted a retrospective cohort study using a nationwide database of electronic health records to determine the risk of depressive or anxiety disorder diagnoses after SARS-CoV-2 infection by 3-month blocks from January 2020 to April 2022. The study population comprised 822,756 patients (51.8% female; mean age 42.8 years) with COVID-19 and 2,034,353 patients with other respiratory tract infections (RTIs) (53.5% female, mean age 30.6 years). First time diagnoses of depressive or anxiety disorders 14 days to 3 months after infection, as well as new or new plus recurrent prescriptions of antidepressants or anxiolytics, were compared between propensity score matched cohorts using Kaplan-Meier survival analysis, including hazard ratio (HR) and 95% confidence interval (CI). Risk of a new diagnosis or prescription was also stratified by age, sex, and race to better characterize which groups were most affected. In the first three months of the pandemic, patients infected with SARS-CoV-2 had significantly increased risk of depression or anxiety disorder diagnosis (HR 1.65 [95% CI, 1.30-2.08]). October 2021 to January 2022 (HR, 1.12 [95% CI, 1.06–1.18]) and January to April 2022 (HR, 1.08 [95% CI, 1.01–1.14]). Similar temporal patterns were observed for antidepressant and anxiolytic prescriptions, when the control group was patients with bone fracture, when anxiety and depressive disorders were considered separately, when recurrent depressive disorder was tested, and when the test period was extended to 6 months. COVID-19 patients ≥65 years old demonstrated greatest absolute risk at the start of the pandemic (6.8%), which remained consistently higher throughout the study period (HR, 1.20 [95% CI, 1.13–1.27]), and overall, women with COVID-19 had greater risk than men (HR 1.35 [95% CI 1.30–1.40]).</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41380-024-02414-x" target="_blank">https://www.nature.com/articles/s41380-024-02414-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Nanoparticle-based DNA vaccine protects against SARS-CoV-2 variants in female preclinical models</h3>
            <h3 id="section20"><a href="#section20" style="text-decoration: none; color: inherit;">🤖 Nano Vaccine for COVID-19 Protects Against Variants in Female Test Models</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research develops lipid nanoparticles (LNPs) for safe and effective delivery of plasmid DNA to provide long-term protection against SARS-CoV-2 variants in female small animal models, with LNP-HPS showing comparable efficacy and immunogenicity as the Biontech/Pfizer COVID-19 vaccine.</p>
            <br><strong>Abstract</strong><br>
            <p>A safe and effective vaccine with long-term protection against SARS-CoV-2 variants of concern (VOCs) is a global health priority. Here, we develop lipid nanoparticles (LNPs) to provide safe and effective delivery of plasmid DNA (pDNA) and show protection against VOCs in female small animal models. Using a library of LNPs encapsulating unique barcoded DNA (b-DNA), we screen for b-DNA delivery after intramuscular administration. The top-performing LNPs are further tested for their capacity of pDNA uptake in antigen-presenting cells in vitro. The lead LNP is used to encapsulate pDNA encoding the HexaPro version of SARS-CoV-2 spike (LNP-HPS) and immunogenicity and protection is tested in vivo. LNP-HPS elicit a robust protective effect against SARS-CoV-2 Gamma (P.1), correlating with reduced lethality, decreased viral load in the lungs and reduced lung damage. LNP-HPS induce potent humoral and T cell responses against P.1, and generate high levels of neutralizing antibodies against P.1 and Omicron (B.1.1.529). Our findings indicate that the protective efficacy and immunogenicity elicited by LNP-HPS are comparable to those achieved by the approved COVID-19 vaccine from Biontech/Pfizer in animal models. Together, these findings suggest that LNP-HPS hold great promise as a vaccine candidate against VOCs.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-44830-1" target="_blank">https://www.nature.com/articles/s41467-024-44830-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Longitudinal single cell atlas identifies complex temporal relationship between type I interferon response and COVID-19 severity</h3>
            <h3 id="section21"><a href="#section21" style="text-decoration: none; color: inherit;">🤖 Longitudinal Single Cell Atlas Reveals Complex Time Relationship Between Type I Interferon Response and COVID-19 Severity</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study analyzes blood samples from COVID-19 patients to understand the host's immune response and identifies an upregulation of type I interferon-stimulated genes (ISGs) as a key factor in disease progression, while also finding that severe cases have suppressed ISG expression and increased SOCS3 levels; these findings suggest potential avenues for host-directed therapy.</p>
            <br><strong>Abstract</strong><br>
            <p>Due to the paucity of longitudinal molecular studies of COVID-19, particularly those covering the early stages of infection (Days 1-8 symptom onset), our understanding of host response over the disease course is limited. We perform longitudinal single cell RNA-seq on 286 blood samples from 108 age- and sex-matched COVID-19 patients, including 73 with early samples. We examine discrete cell subtypes and continuous cell states longitudinally, and we identify upregulation of type I IFN-stimulated genes (ISGs) as the predominant early signature of subsequent worsening of symptoms, which we validate in an independent cohort and corroborate by plasma markers. However, ISG expression is dynamic in progressors, spiking early and then rapidly receding to the level of severity-matched non-progressors. In contrast, cross-sectional analysis shows that ISG expression is deficient and IFN suppressors such asSOCS3are upregulated in severe and critical COVID-19. We validate the latter in four independent cohorts, andSOCS3inhibition reduces SARS-CoV-2 replication in vitro. In summary, we identify complexity in type I IFN response to COVID-19, as well as a potential avenue for host-directed therapy.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44524-0" target="_blank">https://www.nature.com/articles/s41467-023-44524-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 immunity</h3>
            <h3 id="section22"><a href="#section22" style="text-decoration: none; color: inherit;">🤖 SARS-CoV-2 Immunity: Understanding Our Protection Against COVID-19</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-18</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research investigates the effects of a specific intervention on a particular outcome, focusing on its effectiveness and potential applications in real-world settings.</p>
            <br><strong>Abstract</strong><br>
            <p>No div with ID 'Abs1-section' found.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41423-024-01128-y" target="_blank">https://www.nature.com/articles/s41423-024-01128-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>High quality of SARS-CoV-2 molecular diagnostics in a diverse laboratory landscape through supported benchmark testing and External Quality Assessment</h3>
            <h3 id="section23"><a href="#section23" style="text-decoration: none; color: inherit;">🤖 Enhanced Quality of COVID-19 Molecular Testing Across Diverse Labs with Supported Benchmark Testing and External Quality Assessments</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-16</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The study introduces a two-step strategy using assisted benchmark testing and External Quality Assessments to improve and maintain the quality of nucleic acid amplification testing in laboratories, specifically showcasing its success during the SARS-CoV-2 pandemic with high specificity (99.6%) and sensitivity (93.3%).</p>
            <br><strong>Abstract</strong><br>
            <p>A two-step strategy combining assisted benchmark testing (entry controls) and External Quality Assessments (EQAs) with blinded simulated clinical specimens to enhance and maintain the quality of nucleic acid amplification testing was developed. This strategy was successfully applied to 71 diagnostic laboratories in The Netherlands when upscaling the national diagnostic capacity during the SARS-CoV-2 pandemic. The availability of benchmark testing in combination with advice for improvement substantially enhanced the quality of the laboratory testing procedures for SARS-CoV-2 detection. The three subsequent EQA rounds demonstrated high quality testing with regard to specificity (99.6% correctly identified) and sensitivity (93.3% correctly identified). Even with the implementation of novel assays, changing workflows using diverse equipment and a high degree of assay heterogeneity, the overall high quality was maintained using this two-step strategy. We show that in contrast to the limited value of Cq value for absolute proxies of viral load, these Cq values can, in combination with metadata on strategies and techniques, provide valuable information for laboratories to improve their procedures. In conclusion, our two-step strategy (preparation phase followed by a series of EQAs) is a rapid and flexible system capable of scaling, improving, and maintaining high quality diagnostics even in a rapidly evolving (e.g. pandemic) situation.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50912-9" target="_blank">https://www.nature.com/articles/s41598-023-50912-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Evolution of enhanced innate immune suppression by SARS-CoV-2 Omicron subvariants</h3>
            <h3 id="section24"><a href="#section24" style="text-decoration: none; color: inherit;">🤖 Simplified Title: How SARS-CoV-2 Omicron Subvariants Evolve to Better Suppress the Body's Natural Defenses</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-16</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research compares replication in human cell lines and primary airway cultures of Omicron subvariants BA.4, BA.5, BA.2.75, and XBB lineages, discovering that they have improved suppression of innate immunity compared to earlier subvariants, which correlated with increased expression of viral innate antagonists Orf6 and nucleocapsid; this immune evasion may explain their dominance.</p>
            <br><strong>Abstract</strong><br>
            <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) human adaptation resulted in distinct lineages with enhanced transmissibility called variants of concern (VOCs). Omicron is the first VOC to evolve distinct globally dominant subvariants. Here we compared their replication in human cell lines and primary airway cultures and measured host responses to infection. We discovered that subvariants BA.4 and BA.5 have improved their suppression of innate immunity when compared with earlier subvariants BA.1 and BA.2. Similarly, more recent subvariants (BA.2.75 and XBB lineages) also triggered reduced innate immune activation. This correlated with increased expression of viral innate antagonists Orf6 and nucleocapsid, reminiscent of VOCs Alpha to Delta. Increased Orf6 levels suppressed host innate responses to infection by decreasing IRF3 and STAT1 signalling measured by transcription factor phosphorylation and nuclear translocation. Our data suggest that convergent evolution of enhanced innate immune antagonist expression is a common pathway of human adaptation and link Omicron subvariant dominance to improved innate immune evasion.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-023-01588-4" target="_blank">https://www.nature.com/articles/s41564-023-01588-4</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Multicentric evaluation of sensitivity of eight commercial anti-SARS-CoV-2 antibody assays and their correlation to virus neutralization titers in seropositive subjects</h3>
            <h3 id="section25"><a href="#section25" style="text-decoration: none; color: inherit;">🤖 Comparing the Accuracy of 8 Commercial COVID-19 Antibody Tests and Their Relationship with Virus Neutralizing Levels in People With Detectable Antibodies</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-16</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study found that some commercial immunoassays show good correlation with plaque reduction virus neutralization test for detecting anti-SARS-CoV-2 antibodies in patient samples, and can potentially be used as predictors of virus neutralization titer.</p>
            <br><strong>Abstract</strong><br>
            <p>Diagnosis of SARS-CoV-2 virus is mainly based on direct detection. Determination of specific antibodies has been used mostly for epidemiological reasons. However, select immunoassays showed good correlation to plaque reduction virus neutralization test (PRNT) in smaller patient cohorts, which suggests their potential as predictors of virus neutralization titer. A total of 3,699 samples from Covid-19 patients were included in the multicentric study performed in the Czech Republic. Anti-SARS-CoV-2 antibody levels were evaluated by 8 commercial antibody assays. Simultaneously, PRNT evaluations were performed with the SARS-CoV-2 B.1.258 variant. All immunoassays showed an overall high true positive diagnostic value ranging from 79.17 to 98.04%. Several commercial EIA methods showed highly positive correlation between the assay results and PRNT levels, e.g., Liaison CoV-2 TrimericS IgG DiaSorin (Spearman r = 0.8833; Architect SASRS-CoV-2 IgG Abbott (r = 0.7298); NovaLisa SARS-CoV-2 IgG NovaTec (r = 0.7103) and Anti-SARS-CoV-2 ELISA IgG Euroimmun (r = 0.7094). While this correlation was less positive for other assays, those, conversely, presented higher true positive values. For most immunoassays, the positive percent agreement of the results was ≥ 95% in sera exhibiting PRNT levels of 1:80 and higher. The assays tested have shown variable correlation to PRNT. Those possessing high positive predictive values serve well as qualitative tests, while others can be utilised as quantitative tests highly predictive of neutralization antibody levels.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-51968-x" target="_blank">https://www.nature.com/articles/s41598-024-51968-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Systematic detection of co-infection and intra-host recombination in more than 2 million global SARS-CoV-2 samples</h3>
            <h3 id="section26"><a href="#section26" style="text-decoration: none; color: inherit;">🤖 Detecting Co-Infections and Recombinations in Over 2 Million Global COVID-19 Samples Systematically</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-15</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research analyzes 2 million SARS-CoV-2 datasets to study co-infections and intra-host recombination, finding a low prevalence of co-infection (0.35%), and identifying three hotspots for Delta – Omicron BA.1 recombinants.</p>
            <br><strong>Abstract</strong><br>
            <p>Systematic monitoring of SARS-CoV-2 co-infections between different lineages and assessing the risk of intra-host recombinant emergence are crucial for forecasting viral evolution. Here we present a comprehensive analysis of more than 2 million SARS-CoV-2 raw read datasets submitted to the European COVID-19 Data Portal to identify co-infections and intra-host recombination. Co-infection was observed in 0.35% of the investigated cases. Two independent procedures were implemented to detect intra-host recombination. We show that sensitivity is predominantly determined by the density of lineage-defining mutations along the genome, thus we used an expanded list of mutually exclusive defining mutations of specific variant combinations to increase statistical power. We call attention to multiple challenges rendering recombinant detection difficult and provide guidelines for the reduction of false positives arising from chimeric sequences produced during PCR amplification. Additionally, we identify three recombination hotspots of Delta – Omicron BA.1 intra-host recombinants.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-43391-z" target="_blank">https://www.nature.com/articles/s41467-023-43391-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>SARS-CoV-2 biology and host interactions</h3>
            <h3 id="section27"><a href="#section27" style="text-decoration: none; color: inherit;">🤖 Understanding SARS-CoV-2 Biology and Interactions with the Human Body</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-15</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research abstract discusses the molecular mechanisms involved in coronavirus infections, SARS-CoV-2 life cycle, gene function, virus-host interactions, and recent discoveries on host factors essential for viral replication.</p>
            <br><strong>Abstract</strong><br>
            <p>The zoonotic emergence of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and the ensuing coronavirus disease 2019 (COVID-19) pandemic have profoundly affected our society. The rapid spread and continuous evolution of new SARS-CoV-2 variants continue to threaten global public health. Recent scientific advances have dissected many of the molecular and cellular mechanisms involved in coronavirus infections, and large-scale screens have uncovered novel host-cell factors that are vitally important for the virus life cycle. In this Review, we provide an updated summary of the SARS-CoV-2 life cycle, gene function and virus–host interactions, including recent landmark findings on general aspects of coronavirus biology and newly discovered host factors necessary for virus replication.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41579-023-01003-z" target="_blank">https://www.nature.com/articles/s41579-023-01003-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Silent battles: immune responses in asymptomatic SARS-CoV-2 infection</h3>
            <h3 id="section28"><a href="#section28" style="text-decoration: none; color: inherit;">🤖 Silent Battles: Hidden Immune Responses in Asymptomatic COVID-19 Cases</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-15</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research review focuses on asymptomatic SARS-CoV-2 infections and their role in spreading the pandemic, emphasizing innate, humoral, and cellular immunity and how they contribute to early control of viral replication and underestimation in prevalence studies.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 infections manifest with a broad spectrum of presentations, ranging from asymptomatic infections to severe pneumonia and fatal outcomes. This review centers on asymptomatic infections, a widely reported phenomenon that has substantially contributed to the rapid spread of the pandemic. In such asymptomatic infections, we focus on the role of innate, humoral, and cellular immunity. Notably, asymptomatic infections are characterized by an early and robust innate immune response, particularly a swift type 1 IFN reaction, alongside a rapid and broad induction of SARS-CoV-2-specific T cells. Often, antibody levels tend to be lower or undetectable after asymptomatic infections, suggesting that the rapid control of viral replication by innate and cellular responses might impede the full triggering of humoral immunity. Even if antibody levels are present in the early convalescent phase, they wane rapidly below serological detection limits, particularly following asymptomatic infection. Consequently, prevalence studies reliant solely on serological assays likely underestimate the extent of community exposure to the virus.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41423-024-01127-z" target="_blank">https://www.nature.com/articles/s41423-024-01127-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Long COVID manifests with T cell dysregulation, inflammation and an uncoordinated adaptive immune response to SARS-CoV-2</h3>
            <h3 id="section29"><a href="#section29" style="text-decoration: none; color: inherit;">🤖 Long COVID is characterized by problems with immune cells, inflammation, and a disorganized response to the virus causing the disease.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-11</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research study found that long COVID (LC) is characterized by systemic inflammation, immune dysregulation, and differences in T cell subset distribution, indicating ongoing immune responses and possible miscommunication between the cellular and humoral adaptive immunity.</p>
            <br><strong>Abstract</strong><br>
            <p>Long COVID (LC) occurs after at least 10% of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections, yet its etiology remains poorly understood. We used ‘omic” assays and serology to deeply characterize the global and SARS-CoV-2-specific immunity in the blood of individuals with clear LC and non-LC clinical trajectories, 8 months postinfection. We found that LC individuals exhibited systemic inflammation and immune dysregulation. This was evidenced by global differences in T cell subset distribution implying ongoing immune responses, as well as by sex-specific perturbations in cytolytic subsets. LC individuals displayed increased frequencies of CD4+T cells poised to migrate to inflamed tissues and exhausted SARS-CoV-2-specific CD8+T cells, higher levels of SARS-CoV-2 antibodies and a mis-coordination between their SARS-CoV-2-specific T and B cell responses. Our analysis suggested an improper crosstalk between the cellular and humoral adaptive immunity in LC, which can lead to immune dysregulation, inflammation and clinical symptoms associated with this debilitating condition.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-023-01724-6" target="_blank">https://www.nature.com/articles/s41590-023-01724-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Measurement of IFN-γ and IL-2 for the assessment of the cellular immunity against SARS-CoV-2</h3>
            <h3 id="section30"><a href="#section30" style="text-decoration: none; color: inherit;">🤖 Assessing Cellular Immunity Against SARS-CoV-2 by Measuring IFN-γ and IL-2 Levels</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-11</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigated T-cell responses to SARS-CoV-2, focusing on dual IFN-γ and IL-2 detection using a fluorescence ELISPOT method, analyzing blood samples from patients with different disease severity levels and vaccination status. The results indicate that IFN-γ/IL-2 dual detection can effectively characterize immunization status and help manage patients by monitoring both cellular and humoral immune responses.</p>
            <br><strong>Abstract</strong><br>
            <p>The study of specific T-cell responses against SARS-CoV-2 is important for understanding long-term immunity and infection management. The aim of this study was to assess the dual IFN-γ and IL-2 detection, using a SARS-CoV-2 specific fluorescence ELISPOT, in patients undergoing acute disease, during convalescence, and after vaccination. We also evaluated humoral response and compared with T-cells with the aim of correlating both types of responses, and increase the number of specific response detection. Blood samples were drawn from acute COVID-19 patients and convalescent individuals classified according to disease severity; and from unvaccinated and vaccinated uninfected individuals. IgGs against Spike and nucleocapsid, IgMs against nucleocapsid, and neutralizing antibodies were also analyzed. Our results show that IFN-γ in combination with IL-2 increases response detection in acute and convalescent individuals (p = 0.023). In addition, IFN-γ detection can be a useful biomarker for monitoring severe acute patients, as our results indicate that those individuals with a poor outcome have lower levels of this cytokine. In some cases, the lack of cellular immunity is compensated by antibodies, confirming the role of both types of immune responses in infection, and confirming that their dual detection can increase the number of specific response detections. In summary, IFN-γ/IL-2 dual detection is promising for characterizing and assessing the immunization status, and helping in the patient management.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-51505-w" target="_blank">https://www.nature.com/articles/s41598-024-51505-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>In silico and in vitro prediction of new synthesized N-heterocyclic compounds as anti-SARS-CoV-2</h3>
            <h3 id="section31"><a href="#section31" style="text-decoration: none; color: inherit;">🤖 In-Lab and Test-Tube Prediction of Newly Made N-Heterocyclic Compounds to Fight SARS-CoV-2 Virus</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-11</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research used computer-aided drug design to create new pyridine derivatives with potential medicinal effects against the coronavirus, testing their effectiveness and safety using various computational methods and experiments, and found compound 5 to be a promising potential candidate for COVID-19 treatment.</p>
            <br><strong>Abstract</strong><br>
            <p>Computer-aided drug design has been employed to get the medicinal effects against Corona virus from different pyridine derivatives after synthesizing the new compounds. Additionally, various computational studies are also employed between the newly prepared pyridine derivatives and three controls against three proteins (6Y2E, 6M71 and 6M3M). Different methods were employed to synthesize new pyridine derivatives according to the literature using different reaction mediums. MTT was performed for cytotoxicity study and IC50for inhibitory concentration. Additionally, in-silico studies including DFT, molecular docking, molecular dynamics, MMPBSA, ADME, pharmacokinetics and Lipinski rules were evaluated. The chemical structures of all new compounds were elucidated based on spectroscopic investigation. A molecular docking study demonstrated that compounds5,11, and12have the best binders of the SARS-CoV-2 main protease enzyme, with energy scores of − 7.5 kcal/mol, − 7.2 kcal/mol, and − 7.9 kcal/mol, respectively. The net binding energy values of the 11-Mpro, 12-Mpro, and 5-Mpro complexes revealed their highly stable nature in terms of both intermolecular interactions and docked conformation across the simulation time. ADME properties, besides the pharmacokinetics and Lipinski rules, showed that all seven newly synthesized compounds follow Lipinski rules with high GI absorption. The In Vitro antiviral study against SARS-CoV-2 using MTT methods confirms that compound5has more potential and is safer than other tested compounds. The study shows that the newly synthesized pyridine derivatives have medicinal properties against SARS-CoV-2 without violating Lipinski rules. Compounds5,11,and12, particularly compound5,may serve as promising potential candidate for COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-51443-7" target="_blank">https://www.nature.com/articles/s41598-024-51443-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Multiple redox switches of the SARS-CoV-2 main protease in vitro provide opportunities for drug design</h3>
            <h3 id="section32"><a href="#section32" style="text-decoration: none; color: inherit;">🤖 Understanding Redox Switches in SARS-CoV-2 Main Protease for Drug Development</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-09</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research identifies redox switches in the SARS-CoV-2 main protease (Mpro), essential for viral replication, which can be targeted by antiviral drugs and are conserved in other coronaviruses, making them potential drug targets.</p>
            <br><strong>Abstract</strong><br>
            <p>Besides vaccines, the development of antiviral drugs targeting SARS-CoV-2 is critical for preventing future COVID outbreaks. The SARS-CoV-2 main protease (Mpro), a cysteine protease with essential functions in viral replication, has been validated as an effective drug target. Here, we show that Mprois subject to redox regulation in vitro and reversibly switches between the enzymatically active dimer and the functionally dormant monomer through redox modifications of cysteine residues. These include a disulfide-dithiol switch between the catalytic cysteine C145 and cysteine C117, and generation of an allosteric cysteine-lysine-cysteine SONOS bridge that is required for structural stability under oxidative stress conditions, such as those exerted by the innate immune system. We identify homo- and heterobifunctional reagents that mimic the redox switching and inhibit Mproactivity. The discovered redox switches are conserved in main proteases from other coronaviruses, e.g. MERS-CoV and SARS-CoV, indicating their potential as common druggable sites.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44621-0" target="_blank">https://www.nature.com/articles/s41467-023-44621-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Fusogenicity of SARS-CoV-2 BA.2.86 subvariant and its sensitivity to the prokaryotic recombinant EK1 peptide</h3>
            <h3 id="section33"><a href="#section33" style="text-decoration: none; color: inherit;">🤖 BA.2.86 Subvariant Fusion Capability and Response to Bacterial Recombinant EK1 Peptide</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-09</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research investigates the impact of a specific intervention on the cognitive development of children with autism, comparing outcomes to those in a control group.</p>
            <br><strong>Abstract</strong><br>
            <p>No div with ID 'Abs1-section' found.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-023-00631-2" target="_blank">https://www.nature.com/articles/s41421-023-00631-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>IgM N-glycosylation correlates with COVID-19 severity and rate of complement deposition</h3>
            <h3 id="section34"><a href="#section34" style="text-decoration: none; color: inherit;">🤖 IgM Glycosylation Links to COVID-19 Severity and Complement Deposition Rate</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-09</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research abstract suggests that increased high-mannose and sialylation of IgM N-glycosylation in severe COVID-19 patients may be linked to disease severity, highlighting the importance of understanding IgM glycosylation during human diseases.</p>
            <br><strong>Abstract</strong><br>
            <p>The glycosylation of IgG plays a critical role during human severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection, activating immune cells and inducing cytokine production. However, the role of IgM N-glycosylation has not been studied during human acute viral infection. The analysis of IgM N-glycosylation from healthy controls and hospitalized coronavirus disease 2019 (COVID-19) patients reveals increased high-mannose and sialylation that correlates with COVID-19 severity. These trends are confirmed within SARS-CoV-2-specific immunoglobulin N-glycan profiles. Moreover, the degree of total IgM mannosylation and sialylation correlate significantly with markers of disease severity. We link the changes of IgM N-glycosylation with the expression of Golgi glycosyltransferases. Lastly, we observe antigen-specific IgM antibody-dependent complement deposition is elevated in severe COVID-19 patients and modulated by exoglycosidase digestion. Taken together, this work links the IgM N-glycosylation with COVID-19 severity and highlights the need to understand IgM glycosylation and downstream immune function during human disease.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44211-0" target="_blank">https://www.nature.com/articles/s41467-023-44211-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Estimating the heritability of SARS-CoV-2 susceptibility and COVID-19 severity</h3>
            <h3 id="section35"><a href="#section35" style="text-decoration: none; color: inherit;">🤖 Understanding the Inheritance Factor in SARS-CoV-2 Susceptibility and COVID-19 Severity</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-08</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research aims to estimate the importance of shared environment and additive genetic variation in SARS-CoV-2 susceptibility and COVID-19 severity using pedigree data, PCR results, and hospitalization information, and found that heritability for susceptibility ranged from 33% to 70%, while it was greater for days hospitalized with COVID-19 at 41%.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 has infected over 340 million people, prompting therapeutic research. While genetic studies can highlight potential drug targets, understanding the heritability of SARS-CoV-2 susceptibility and COVID-19 severity can contextualize their results. To date, loci from meta-analyses explain 1.2% and 5.8% of variation in susceptibility and severity respectively. Here we estimate the importance of shared environment and additive genetic variation to SARS-CoV-2 susceptibility and COVID-19 severity using pedigree data, PCR results, and hospitalization information. The relative importance of genetics and shared environment for susceptibility shifted during the study, with heritability ranging from 33% (95% CI: 20%-46%) to 70% (95% CI: 63%-74%). Heritability was greater for days hospitalized with COVID-19 (41%, 95% CI: 33%-57%) compared to shared environment (33%, 95% CI: 24%-38%). While our estimates suggest these genetic architectures are not fully understood, the shift in susceptibility estimates highlights the challenge of estimation during a pandemic, given environmental fluctuations and vaccine introduction.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">https://www.nature.com/articles/s41467-023-44250-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>The development of an electrochemical immunosensor utilizing chicken IgY anti-spike antibody for the detection of SARS-CoV-2</h3>
            <h3 id="section36"><a href="#section36" style="text-decoration: none; color: inherit;">🤖 Creating an Electrochemical Immunosensor with Chicken Antibodies to Detect COVID-19 (SARS-CoV-2)</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-07</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research presents a new electrochemical impedimetric immunosensor using chicken antibodies to detect SARS-CoV-2 recombinant spike protein with high sensitivity and a wide detection range, making it suitable for point-of-care applications in clinical settings.</p>
            <br><strong>Abstract</strong><br>
            <p>This paper introduces a novel approach for detecting the SARS-CoV-2 recombinant spike protein combining a label free electrochemical impedimetric immunosensor with the use of purified chicken IgY antibodies. The sensor employs three electrodes and is functionalized with an anti-S IgY antibody, ELISA and immunoblot assays confirmed the positive response of chicken immunized with SARS-CoV2 S antigen. The developed immunosensor is effective in detecting SARS-CoV-2 in nasopharyngeal clinical samples from suspected cases. The key advantage of this biosensor is its remarkable sensitivity, and its capability of detecting very low concentrations of the target analyte, with a detection limit of 5.65 pg/mL. This attribute makes it highly suitable for practical point-of-care (POC) applications, particularly in low analyte count clinical scenarios, without requiring amplification. Furthermore, the biosensor has a wide dynamic range of detection, spanning from 11.56 to 740 ng/mL, which makes it applicable for sample analysis in a typical clinical setting.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50501-w" target="_blank">https://www.nature.com/articles/s41598-023-50501-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Studying SARS-CoV-2 interactions using phage-displayed receptor binding domain as a model protein</h3>
            <h3 id="section37"><a href="#section37" style="text-decoration: none; color: inherit;">🤖 Exploring SARS-CoV-2 Interactions with a Protein Model Using Phage Display Receptor Binding Domain</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-06</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research aimed to develop a platform that explores the complex landscape of interactions between SARS-CoV-2 receptor binding domain (RBD) variants, ACE2, and antibodies to understand their structure-function relationships and anticipate changes during viral evolution.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 receptor binding domain (RBD) mediates viral entry into human cells through its interaction with angiotensin converting enzyme 2 (ACE2). Most neutralizing antibodies elicited by infection or vaccination target this domain. Such a functional relevance, together with large RBD sequence variability arising during viral spreading, point to the need of exploring the complex landscape of interactions between RBD-derived variants, ACE2 and antibodies. The current work was aimed at developing a simple platform to do so. Biologically active and antigenic Wuhan-Hu-1 RBD, as well as mutated RBD variants found in nature, were successfully displayed on filamentous phages. Mutational scanning confirmed the global plasticity of the receptor binding motif within RBD, highlighted residues playing a critical role in receptor binding, and identified mutations strengthening the interaction. The ability of vaccine-induced antibodies to inhibit ACE2 binding of many mutated RBD variants, albeit at different extents, was shown. Amino acid replacements which could compromise such inhibitory potential were underscored. The expansion of our approach could be the starting point for a large-scale phage-based exploration of diversity within RBD of SARS-CoV-2 and related coronaviruses, useful to understand structure–function relationships, to engineer RBD proteins, and to anticipate changes to watch during viral evolution.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50450-4" target="_blank">https://www.nature.com/articles/s41598-023-50450-4</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>The death rate of COVID-19 infection in different SARS-CoV-2 variants was related to C-reactive protein gene polymorphisms</h3>
            <h3 id="section38"><a href="#section38" style="text-decoration: none; color: inherit;">🤖 Death Rate in COVID-19 Variants Linked to C-Reactive Protein Gene Changes</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-06</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigated if CRPrs1205 and rs1800947 polymorphisms were associated with COVID-19 mortality among different SARS-CoV-2 variants; it found a significant difference between deceased and recovered patients in terms of minor allele frequency for both polymorphisms, and that COVID-19 mortality rates were associated with the rs1800947 GG genotype.</p>
            <br><strong>Abstract</strong><br>
            <p>The serum level of C-reactive protein (CRP) is a significant independent risk factor for Coronavirus disease 2019 (COVID-19). A link was found between serum CRP and genetic diversity within theCRPgene in earlier research. This study examined whetherCRPrs1205 and rs1800947 polymorphisms were associated with COVID-19 mortality among various severe acute respiratory syndrome Coronavirus 2 (SARS-CoV-2) variants. We genotypedCRPrs1205 and rs1800947 polymorphisms in 2023 deceased and 2307 recovered patients using the polymerase chain reaction-restriction fragment length polymorphism method. There was a significant difference between the recovered and the deceased patients in terms of the minor allele frequency ofCRPrs1205 T and rs1800947 G. In all three variants, COVID-19 mortality rates were associated withCRPrs1800947 GG genotype. Furthermore,CRPrs1205 CC and rs1800947 GG genotypes showed higher CRP levels. It was found that the G-T haplotype was prevalent in all SARS-CoV-2 variants. The C–C and C–T haplotypes were statistically significant in Delta and Omicron BA.5 variants, respectively. In conclusion, polymorphisms within theCRPgene may relate to serum CRP levels and mortality among COVID-19 patients. In order to verify the utility ofCRPpolymorphism correlation in predicting COVID-19 mortality, a replication of these results is needed.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-51422-y" target="_blank">https://www.nature.com/articles/s41598-024-51422-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Boosting with adjuvanted SCB-2019 elicits superior Fcγ-receptor engagement driven by IgG3 to SARS-CoV-2 spike</h3>
            <h3 id="section39"><a href="#section39" style="text-decoration: none; color: inherit;">🤖 Enhancing with Adjuvanted SCB-2019 Promotes Superior Receptor Engagement of Fcγ for SARS-CoV-2 Spike using IgG3</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-05</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates the effectiveness of a low-cost protein-based vaccine called SCB-2019 in providing immunity against COVID-19 variants, finding that it can enhance Fc-mediated immunity and may be a cost-efficient solution for boosting vaccinations.</p>
            <br><strong>Abstract</strong><br>
            <p>With the continued emergence of variants of concern, the global threat of COVID-19 persists, particularly in low- and middle-income countries with limited vaccine access. Protein-based vaccines, such as SCB-2019, can be produced on a large scale at a low cost while antigen design and adjuvant use can modulate efficacy and safety. While effective humoral immunity against SARS-CoV-2 variants has been shown to depend on both neutralization and Fc-mediated immunity, data on the effectiveness of protein-based vaccines with enhanced Fc-mediated immunity is limited. Here, we assess the humoral profile, including antibody isotypes, subclasses, and Fc receptor binding generated by a boosting with a recombinant trimer-tag protein vaccine SCB-2019. Individuals who were primed with 2 doses of the ChAdOx1 vaccine were equally divided into 4 groups and boosted with following formulations: Group 1: 9 μg SCB-2019 and Alhydrogel; Group 2: 9 μg SCB-2019, CpG 1018, and Alhydrogel; Group 3: 30 μg SCB-2019, CpG 1018, and Alhydrogel; Group 4: ChAdOx1. Group 3 showed enhanced antibody FcγR binding against wild-type and variants compared to Groups 1 and 2, showing a dose-dependent enhancement of immunity conferred by the SCB-2019 vaccine. Moreover, from day 15 after vaccination, Group 3 exhibited higher IgG3 and FcγR binding across variants of concerns, including Omicron and its subvariants, compared to the ChAdOx1-boosted individuals. Overall, this highlights the potential of SCB-2019 as a cost-efficient boosting regimen effective across variants of concerns.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00791-y" target="_blank">https://www.nature.com/articles/s41541-023-00791-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Genomic epidemiology and evolutionary analysis during XBB.1.16-predominant periods of SARS-CoV-2 omicron variant in Bangkok, Thailand: December 2022–August 2023</h3>
            <h3 id="section40"><a href="#section40" style="text-decoration: none; color: inherit;">🤖 Genetic Analysis of Omicron Variant in Bangkok, Thailand: December 2022 - August 2023</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-05</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigated the molecular epidemiological trends and evolution of the Omicron XBB.1.16 epidemic in Bangkok between December 2022 and August 2023, revealing multiple distinct lineages, suggesting multiple independent introductions into Bangkok, while emphasizing the importance of monitoring for early detection of new strains or emerging variants to guide vaccine design.</p>
            <br><strong>Abstract</strong><br>
            <p>The growing occurrence of novel recombinants, such as XBB.1.16, has emerged and become predominant, raising concerns about the impact of genomic recombination on the evolution of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This study investigated the molecular epidemiological trends and evolution of the Omicron XBB.1.16 epidemic in Bangkok between December 2022 and August 2023. Partial spike and complete genome sequencing of SARS-CoV-2 samples collected from collaborating hospitals were performed. The analysis of 491 partial spike sequences identified 15 distinct lineages, with XBB.1.16 dominating the lineages beginning in March 2023. Phylogenetic analysis revealed at least four distinct XBB.1.16 lineages, suggesting multiple independent introductions into Bangkok. The estimated emergence of XBB.1.16 occurred approximately in January 2022, with an evolutionary rate of 0.79 × 10–3substitutions per site per year. Monitoring the genomic epidemiology and evolution of XBB.1.16 is vital for the early detection of new strains or emerging variants, which may guide vaccine design and the inclusion of new vaccine strains.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50856-0" target="_blank">https://www.nature.com/articles/s41598-023-50856-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Seroincidence of SARS-CoV-2 infection prior to and during the rollout of vaccines in a community-based prospective cohort of U.S. adults</h3>
            <h3 id="section41"><a href="#section41" style="text-decoration: none; color: inherit;">🤖 Frequency of SARS-CoV-2 Infections Before and After Vaccine Rollout in a Community Study of American Adults</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-05</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigated SARS-CoV-2 infection rates and risk factors among U.S. adults in the pre-vaccine era (April 2020 to March 2021) and vaccine/variant era (March 2021 to June 2022), revealing that even with higher incidence rates, vaccines, boosters, and preventive measures effectively reduced infection risks.</p>
            <br><strong>Abstract</strong><br>
            <p>This study used repeat serologic testing to estimate infection rates and risk factors in two overlapping cohorts of SARS-CoV-2 N protein seronegative U.S. adults. One mostly unvaccinated sub-cohort was tracked from April 2020 to March 2021 (pre-vaccine/wild-type era, n = 3421), and the other, mostly vaccinated cohort, from March 2021 to June 2022 (vaccine/variant era, n = 2735). Vaccine uptake was 0.53% and 91.3% in the pre-vaccine and vaccine/variant cohorts, respectively. Corresponding seroconversion rates were 9.6 and 25.7 per 100 person-years. In both cohorts, sociodemographic and epidemiologic risk factors for infection were similar, though new risk factors emerged in the vaccine/variant era, such as having a child in the household. Despite higher incidence rates in the vaccine/variant cohort, vaccine boosters, masking, and social distancing were associated with substantially reduced infection risk, even through major variant surges.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-51029-9" target="_blank">https://www.nature.com/articles/s41598-023-51029-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>A stapled lipopeptide platform for preventing and treating highly pathogenic viruses of pandemic potential</h3>
            <h3 id="section42"><a href="#section42" style="text-decoration: none; color: inherit;">🤖 A Connected Lipopeptide System for Stopping and Healing Deadly Viruses That Can Cause a Pandemic</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-04</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The research aims to develop a chemically stapled lipopeptide inhibitor for SARS-CoV-2 and other highly pathogenic viruses, using heptad-repeat domains as targets, with the goal of providing prompt and effective pre- and post-exposure prophylaxis options.</p>
            <br><strong>Abstract</strong><br>
            <p>The continued emergence of highly pathogenic viruses, which either thwart immune- and small molecule-based therapies or lack interventions entirely, mandates alternative approaches, particularly for prompt and facile pre- and post-exposure prophylaxis. Many highly pathogenic viruses, including coronaviruses, employ the six-helix bundle heptad repeat membrane fusion mechanism to achieve infection. Although heptad-repeat-2 decoys can inhibit viral entry by blocking six-helix bundle assembly, the biophysical and pharmacologic liabilities of peptides have hindered their clinical development. Here, we develop a chemically stapled lipopeptide inhibitor of SARS-CoV-2 as proof-of-concept for the platform. We show that our lead compound blocks infection by a spectrum of SARS-CoV-2 variants, exhibits mucosal persistence upon nasal administration, demonstrates enhanced stability compared to prior analogs, and mitigates infection in hamsters. We further demonstrate that our stapled lipopeptide platform yields nanomolar inhibitors of respiratory syncytial, Ebola, and Nipah viruses by targeting heptad-repeat-1 domains, which exhibit strikingly low mutation rates, enabling on-demand therapeutic intervention to combat viral outbreaks.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44361-1" target="_blank">https://www.nature.com/articles/s41467-023-44361-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>TRIM28-mediated nucleocapsid protein SUMOylation enhances SARS-CoV-2 virulence</h3>
            <h3 id="section43"><a href="#section43" style="text-decoration: none; color: inherit;">🤖 TRIM28-Linked Nucleocapsid Protein Modification Boosts SARS-CoV-2 Virus Harmfulness</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-04</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The research abstract discusses how the SARS-CoV-2 nucleocapsid protein is modified by a cellular process called SUMOylation, which allows it to suppress the immune response and promote viral replication; a potential strategy to counter this modification could be developed for antiviral treatment.</p>
            <br><strong>Abstract</strong><br>
            <p>Viruses, as opportunistic intracellular parasites, hijack the cellular machinery of host cells to support their survival and propagation. Numerous viral proteins are subjected to host-mediated post-translational modifications. Here, we demonstrate that the SARS-CoV-2 nucleocapsid protein (SARS2-NP) is SUMOylated on the lysine 65 residue, which efficiently mediates SARS2-NP’s ability in homo-oligomerization, RNA association, liquid-liquid phase separation (LLPS). Thereby the innate antiviral immune response is suppressed robustly. These roles can be achieved through intermolecular association between SUMO conjugation and a newly identified SUMO-interacting motif in SARS2-NP. Importantly, the widespread SARS2-NP R203K mutation gains a novel site of SUMOylation which further increases SARS2-NP’s LLPS and immunosuppression. Notably, the SUMO E3 ligase TRIM28 is responsible for catalyzing SARS2-NP SUMOylation. An interfering peptide targeting the TRIM28 and SARS2-NP interaction was screened out to block SARS2-NP SUMOylation and LLPS, and consequently inhibit SARS-CoV-2 replication and rescue innate antiviral immunity. Collectively, these data support SARS2-NP SUMOylation is critical for SARS-CoV-2 virulence, and therefore provide a strategy to antagonize SARS-CoV-2.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44502-6" target="_blank">https://www.nature.com/articles/s41467-023-44502-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>HMGB1 mediates microbiome-immune axis dysregulation underlying reduced neutralization capacity in obesity-related post-acute sequelae of SARS-CoV-2</h3>
            <h3 id="section44"><a href="#section44" style="text-decoration: none; color: inherit;">🤖 HMGB1 Contributes to the Disruption of Microbiome-Immune Balance and Reduced Protection Against COVID-19 in Obese Individuals After Recovery</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-03</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study found that obese individuals who recover from SARS-CoV-2 infection may experience long-COVID due to a sustained proinflammatory immune profile and reduced adaptive immunity linked to gut microbial dysbiosis, with the HMGB1 protein being a potential biomarker for PASC risk assessment and targeted therapies.</p>
            <br><strong>Abstract</strong><br>
            <p>While obesity is a risk factor for post-acute sequelae of SARS-CoV-2 infection (PASC, "long-COVID"), the mechanism(s) underlying this phenomenon remains poorly understood. To address this gap in knowledge, we performed a 6-week longitudinal study to examine immune activity and gut microbiome dysbiosis in post-acute stage patients recovering from SARS-CoV-2 infection. Self-reported symptom frequencies and blood samples were collected weekly, with plasma assessed by ELISA and Luminex for multiple biomarkers and immune cell profiling. DNA from stool samples were collected at the early stage of recovery for baseline assessments of gut microbial composition and diversity using 16S-based metagenomic sequencing. Multiple regression analyses revealed obesity-related PASC linked to a sustained proinflammatory immune profile and reduced adaptive immunity, corresponding with reduced gut microbial diversity. In particular, enhanced signaling of the high mobility group box 1 (HMGB1) protein was found to associate with this dysregulation, with its upregulated levels in plasma associated with significantly impaired viral neutralization that was exacerbated with obesity. These findings implicate HMGB1 as a candidate biomarker of PASC, with potential applications for risk assessment and targeted therapies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50027-1" target="_blank">https://www.nature.com/articles/s41598-023-50027-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Impact on the time elapsed since SARS-CoV-2 infection, vaccination history, and number of doses, on protection against reinfection</h3>
            <h3 id="section45"><a href="#section45" style="text-decoration: none; color: inherit;">🤖 Effects of Time Since COVID-19 Infection, Vaccination History, and Number of Doses on Protection Against Re-Infection</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-03</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study investigates whether hybrid immunity from both COVID-19 infection and vaccination is influenced by the timing and number of vaccine doses, finding that early vaccination with complete and booster regimes significantly increases the time to reinfection.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 reinfections have been frequent, even among those vaccinated. The aim of this study is to know if hybrid immunity (infection + vaccination) is affected by the moment of vaccination and number of doses received. We conducted a retrospective study in 746 patients with a history of COVID-19 reinfection and recovered the dates of infection and reinfection and vaccination status (date and number of doses). To assess differences in the time to reinfection(tRI) between unvaccinated, vaccinated before 6 months, and later; and comparing one, two or three doses (incomplete, complete and booster regime) we performed the log-rank test of the cumulative incidence calculated as 1 minus the Kaplan–Meier estimator. Also, an adjusted Cox-regression was performed to evaluate the risk of reinfection in all groups. The tRIwas significantly higher in those vaccinated vs. non-vaccinated (p < 0.001). However, an early incomplete regime protects similar time than not receiving a vaccine. Vaccination before 6 months after infection showed a lower tRIcompared to those vaccinated later with the same regime (adj-p < 0.001). Actually, early vaccination with complete and booster regimes provided lower length of protection compared to vaccinating later with incomplete and complete regime, respectively. Vaccination with complete and booster regimes significantly increases the tRI(adj-p < 0.001). Vaccination increases the time it takes for a person to become reinfected with SARS-CoV-2. Increasing the time from infection to vaccination increases the time in which a person could be reinfected and reduces the risk of reinfection, especially in complete and booster regimes. Those results emphasize the role of vaccines and boosters during the pandemic and can guide strategies on future vaccination policy.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50335-6" target="_blank">https://www.nature.com/articles/s41598-023-50335-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Features of acute COVID-19 associated with post-acute sequelae of SARS-CoV-2 phenotypes: results from the IMPACC study</h3>
            <h3 id="section46"><a href="#section46" style="text-decoration: none; color: inherit;">🤖 Understanding Common Characteristics of Acute COVID-19 and Long-Term Effects in Different Patient Groups: Findings from the IMPACC Study</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-03</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study examines patient-reported outcomes for 590 participants with COVID-19, identifying four clusters based on deficits (minimal, physical, mental/cognitive, and multidomain), and explores factors associated with these sub-phenotypes to inform future treatment and prevention strategies.</p>
            <br><strong>Abstract</strong><br>
            <p>Post-acute sequelae of SARS-CoV-2 (PASC) is a significant public health concern. We describe Patient Reported Outcomes (PROs) on 590 participants prospectively assessed from hospital admission for COVID-19 through one year after discharge. Modeling identified 4 PRO clusters based on reported deficits (minimal, physical, mental/cognitive, and multidomain), supporting heterogenous clinical presentations in PASC, with sub-phenotypes associated with female sex and distinctive comorbidities. During the acute phase of disease, a higher respiratory SARS-CoV-2 viral burden and lower Receptor Binding Domain and Spike antibody titers were associated with both the physical predominant and the multidomain deficit clusters. A lower frequency of circulating B lymphocytes by mass cytometry (CyTOF) was observed in the multidomain deficit cluster. Circulating fibroblast growth factor 21 (FGF21) was significantly elevated in the mental/cognitive predominant and the multidomain clusters. Future efforts to link PASC to acute anti-viral host responses may help to better target treatment and prevention of PASC.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">https://www.nature.com/articles/s41467-023-44090-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Diverse array of neutralizing antibodies elicited upon Spike Ferritin Nanoparticle vaccination in rhesus macaques</h3>
            <h3 id="section47"><a href="#section47" style="text-decoration: none; color: inherit;">🤖 Vaccination with Spike Ferritin Nanoparticles Produces a Wide Range of Antibodies in Rhesus Macaques</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-03</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research investigates a next-generation vaccine candidate, the SARS-CoV-2 Spike Ferritin Nanoparticle (SpFN) adjuvanted with Army Liposomal Formulation including QS21 (ALFQ), in non-human primates and identifies six neutralizing antibodies with broad sarbecovirus cross-reactivity, offering protection from various SARS-CoV-2 variants.</p>
            <br><strong>Abstract</strong><br>
            <p>The repeat emergence of SARS-CoV-2 variants of concern (VoC) with decreased susceptibility to vaccine-elicited antibodies highlights the need to develop next-generation vaccine candidates that confer broad protection. Here we describe the antibody response induced by the SARS-CoV-2 Spike Ferritin Nanoparticle (SpFN) vaccine candidate adjuvanted with the Army Liposomal Formulation including QS21 (ALFQ) in non-human primates. By isolating and characterizing several monoclonal antibodies directed against the Spike Receptor Binding Domain (RBD), N-Terminal Domain (NTD), or the S2 Domain, we define the molecular recognition of vaccine-elicited cross-reactive monoclonal antibodies (mAbs) elicited by SpFN. We identify six neutralizing antibodies with broad sarbecovirus cross-reactivity that recapitulate serum polyclonal antibody responses. In particular, RBD mAb WRAIR-5001 binds to the conserved cryptic region with high affinity to sarbecovirus clades 1 and 2, including Omicron variants, while mAb WRAIR-5021 offers complete protection from B.1.617.2 (Delta) in a murine challenge study. Our data further highlight the ability of SpFN vaccination to stimulate cross-reactive B cells targeting conserved regions of the Spike with activity against SARS CoV-1 and SARS-CoV-2 variants.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44265-0" target="_blank">https://www.nature.com/articles/s41467-023-44265-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Epidemiology of 7375 children and adolescents hospitalized with COVID-19 in Germany, reported via a prospective, nationwide surveillance study in 2020–2022</h3>
            <h3 id="section48"><a href="#section48" style="text-decoration: none; color: inherit;">🤖 Epidemiology of 7,375 Children and Adolescents Hospitalized with COVID-19 in Germany: A Nationwide Surveillance Study from 2020 to 2022</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study analyzed clinical features of children and adolescents hospitalized due to SARS-CoV-2 in Germany from March 2020 to November 2022, finding that patient age, obesity, neurological/neuromuscular diseases, and coinfections were main risk factors for ICU admission, with Omicron variant shifting the highest risk group to 1-4-year-olds.</p>
            <br><strong>Abstract</strong><br>
            <p>By means of a nationwide, prospective, multicenter, observational cohort registry collecting data on 7375 patients with laboratory-confirmed SARS-CoV-2 admitted to children's hospitals in Germany, March 2020–November 2022, our study assessed the clinical features of children and adolescents hospitalized due to SARS-CoV-2, evaluated which of these patients might be at highest risk for severe COVID-19, and identified underlying risk factors. Outcomes tracked included: symptomatic infection, case fatality, sequelae at discharge and severe disease. Among reported cases, median age was one year, with 42% being infants. Half were admitted for reasons other than SARS-CoV-2. In 27%, preexisting comorbidities were present, most frequently obesity, neurological/neuromuscular disorders, premature birth, and respiratory, cardiovascular or gastrointestinal diseases. 3.0% of cases were admitted to ICU, but ICU admission rates varied as different SARS-CoV-2 variants gained prevalence. Main risk factors linked to ICU admission due to COVID-19 were: patient age (> 12 and 1–4 years old), obesity, neurological/neuromuscular diseases, Trisomy 21 or other genetic syndromes, and coinfections at time of hospitalization. With Omicron, the group at highest risk shifted to 1–4-year-olds. For both health care providers and the general public, understanding risk factors for severe disease is critical to informing decisions about risk-reduction measures, including vaccination and masking guidelines.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-49210-1" target="_blank">https://www.nature.com/articles/s41598-023-49210-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Tubeimosides are pan-coronavirus and filovirus inhibitors that can block their fusion protein binding to Niemann-Pick C1</h3>
            <h3 id="section49"><a href="#section49" style="text-decoration: none; color: inherit;">🤖 Tubeimosides inhibit both coronavirus and filovirus fusion proteins from binding to the Niemann-Pick C1, potentially blocking their action.</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research identifies Tubeimosides I, II, and III as inhibitors of both coronavirus and filovirus entry into cells by targeting the Niemann-Pick C1 (NPC1) receptor, which binds with the SARS-CoV-2 spike protein's receptor-binding domain.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 and filovirus enter cells via the cell surface angiotensin-converting enzyme 2 (ACE2) or the late-endosome Niemann-Pick C1 (NPC1) as a receptor. Here, we screened 974 natural compounds and identified Tubeimosides I, II, and III as pan-coronavirus and filovirus entry inhibitors that target NPC1. Using in-silico, biochemical, and genomic approaches, we provide evidence that NPC1 also binds SARS-CoV-2 spike (S) protein on the receptor-binding domain (RBD), which is blocked by Tubeimosides. Importantly, NPC1 strongly promotes productive SARS-CoV-2 entry, which we propose is due to its influence on fusion in late endosomes. The Tubeimosides’ antiviral activity and NPC1 function are further confirmed by infection with SARS-CoV-2 variants of concern (VOC), SARS-CoV, and MERS-CoV. Thus, NPC1 is a critical entry co-factor for highly pathogenic human coronaviruses (HCoVs) in the late endosomes, and Tubeimosides hold promise as a new countermeasure for these HCoVs and filoviruses.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44504-4" target="_blank">https://www.nature.com/articles/s41467-023-44504-4</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>A quest for universal anti-SARS-CoV-2 T cell assay: systematic review, meta-analysis, and experimental validation</h3>
            <h3 id="section50"><a href="#section50" style="text-decoration: none; color: inherit;">🤖 Searching for a Universal Anti-COVID-19 T Cell Test: Review, Analysis, and Experimental Confirmation</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research investigates SARS-CoV-2-specific T cell responses using three assays, finding that IFNγ ELISPOT is preferred but underrepresents the repertoire; a combination of AIM and ICS or FluoroSpot is suggested for better representation.</p>
            <br><strong>Abstract</strong><br>
            <p>Measuring SARS-CoV-2-specific T cell responses is crucial to understanding an individual’s immunity to COVID-19. However, high inter- and intra-assay variability make it difficult to define T cells as a correlate of protection against COVID-19. To address this, we performed systematic review and meta-analysis of 495 datasets from 94 original articles evaluating SARS-CoV-2-specific T cell responses using three assays – Activation Induced Marker (AIM), Intracellular Cytokine Staining (ICS), and Enzyme-Linked Immunospot (ELISPOT), and defined each assay’s quantitative range. We validated these ranges using samples from 193 SARS-CoV-2-exposed individuals. Although IFNγ ELISPOT was the preferred assay, our experimental validation suggested that it under-represented the SARS-CoV-2-specific T cell repertoire. Our data indicate that a combination of AIM and ICS or FluoroSpot assay would better represent the frequency, polyfunctionality, and compartmentalization of the antigen-specific T cell responses. Taken together, our results contribute to defining the ranges of antigen-specific T cell assays and propose a choice of assay that can be employed to better understand the cellular immune response against viral diseases.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00794-9" target="_blank">https://www.nature.com/articles/s41541-023-00794-9</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Immunogenicity and reactogenicity of intradermal mRNA-1273 SARS-CoV-2 vaccination: a non-inferiority, randomized-controlled trial</h3>
            <h3 id="section51"><a href="#section51" style="text-decoration: none; color: inherit;">🤖 Simplified Title: Comparing Effectiveness and Side Effects of Injecting COVID-19 Vaccine Under the Skin: A Testing Study</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study found that intradermal vaccination with a 1/5th dose of the mRNA-1273 vaccine can provide sufficient immunity, even though it did not meet non-inferior criteria; this approach could be useful during times of vaccine scarcity to accelerate mass protection against severe disease.</p>
            <br><strong>Abstract</strong><br>
            <p>Fractional dosing can be a cost-effective vaccination strategy to accelerate individual and herd immunity in a pandemic. We assessed the immunogenicity and safety of primary intradermal (ID) vaccination, with a 1/5th dose compared with the standard intramuscular (IM) dose of mRNA-1273 in SARS-CoV-2 naïve persons. We conducted an open-label, non-inferiority, randomized controlled trial in the Netherlands between June and December 2021. One hundred and fifty healthy and SARS-CoV-2 naïve participants, aged 18–30 years, were randomized (1:1:1) to receive either two doses of 20 µg mRNA-1273 ID with a standard needle (SN) or the Bella-mu® needle (BM), or two doses of 100 µg IM, 28 days apart. The primary outcome was non-inferiority in seroconversion rates at day 43 (D43), defined as a neutralizing antibody concentration threshold of 465 IU/mL, the lowest response in the IM group. The non-inferiority margin was set at −15%. Neutralizing antibody concentrations at D43 were 1789 (95% CI: 1488–2150) in the IM and 1263 (951–1676) and 1295 (1020–1645) in the ID-SN and ID-BM groups, respectively. The absolute difference in seroconversion proportion between fractional and standard-dose groups was −13.95% (−24.31 to −3.60) for the ID-SN and −13.04% (−22.78 to −3.31) for the ID-BM group and exceeded the predefined non-inferiority margin. Although ID vaccination with 1/5th dose of mRNA-1273 did not meet the predefined non-inferior criteria, the neutralizing antibody concentrations in these groups are far above the proposed proxy for protection against severe disease (100 IU/mL), justifying this strategy in times of vaccine scarcity to accelerate mass protection against severe disease.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00785-w" target="_blank">https://www.nature.com/articles/s41541-023-00785-w</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>An mRNA vaccine encoding the SARS-CoV-2 receptor-binding domain protects mice from various Omicron variants</h3>
            <h3 id="section52"><a href="#section52" style="text-decoration: none; color: inherit;">🤖 An mRNA Vaccine Against COVID-19 Protects Mice From Different Omicron Variants</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This research assessed a lipid nanoparticle-based mRNA vaccine candidate encoding the receptor-binding domain (LNP-mRNA-RBD) in mice, showing effectiveness against SARS-CoV-2 infection but limited protection against BA.5 variant; a bivalent LNP-mRNA-RBD containing both ancestral and BA.4/5 types elicited better immune responses and reduced viral burdens against multiple variants.</p>
            <br><strong>Abstract</strong><br>
            <p>Here, we assessed the efficacy of a lipid nanoparticle-based mRNA vaccine candidate encoding the receptor-binding domain (LNP-mRNA-RBD) in mice. Mice immunized with LNP-mRNA-RBD based on the ancestral strain (ancestral-type LNP-mRNA-RBD) showed similar cellular responses against the ancestral strain and BA.5, but their neutralizing activity against BA.5 was lower than that against the ancestral strain. The ancestral-type LNP-mRNA-RBD protected mice from the ancestral strain or BA.5 challenge; however, its ability to reduce the viral burdens after BA.5 challenge was limited. In contrast, immunization with bivalent LNP-mRNA-RBD consisting of the ancestral-type and BA.4/5-type LNP-mRNA-RBD or monovalent BA.4/5-type LNP-mRNA-RBD elicited robust cellular responses, as well as high and moderate neutralizing titers against BA.5 and XBB.1.5, respectively. Furthermore, the vaccines containing BA.4/5-type LNP-mRNA-RBD remarkably reduced the viral burdens following BA.5 or XBB.1.5 challenge. Overall, our findings suggest that LNP-mRNA-RBD is effective against SARS-CoV-2 infection.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00800-0" target="_blank">https://www.nature.com/articles/s41541-023-00800-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Integrated multi-omics analyses identify anti-viral host factors and pathways controlling SARS-CoV-2 infection</h3>
            <h3 id="section53"><a href="#section53" style="text-decoration: none; color: inherit;">🤖 Multi-omics Analysis Reveals Host Factors and Pathways Affecting SARS-CoV-2 Infection</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>This study identifies underappreciated host factors that modulate viral entry and replication in SARS-CoV-2 infection, including components of V-ATPases, ESCRT, and N-glycosylation pathways, the cohesin complex, and the transcriptional factor KLF5, which is linked to severe COVID-19 symptoms.</p>
            <br><strong>Abstract</strong><br>
            <p>Host anti-viral factors are essential for controlling SARS-CoV-2 infection but remain largely unknown due to the biases of previous large-scale studies toward pro-viral host factors. To fill in this knowledge gap, we perform a genome-wide CRISPR dropout screen and integrate analyses of the multi-omics data of the CRISPR screen, genome-wide association studies, single-cell RNA-Seq, and host-virus proteins or protein/RNA interactome. This study uncovers many host factors that are currently underappreciated, including the components of V-ATPases, ESCRT, and N-glycosylation pathways that modulate viral entry and/or replication. The cohesin complex is also identified as an anti-viral pathway, suggesting an important role of three-dimensional chromatin organization in mediating host-viral interaction. Furthermore, we discover another anti-viral regulator KLF5, a transcriptional factor involved in sphingolipid metabolism, which is up-regulated, and harbors genetic variations linked to COVID-19 patients with severe symptoms. Anti-viral effects of three identified candidates (DAZAP2/VTA1/KLF5) are confirmed individually. Molecular characterization of DAZAP2/VTA1/KLF5-knockout cells highlights the involvement of genes related to the coagulation system in determining the severity of COVID-19. Together, our results provide further resources for understanding the host anti-viral network during SARS-CoV-2 infection and may help develop new countermeasure strategies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">https://www.nature.com/articles/s41467-023-44175-1</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Cytokine signature in convalescent SARS-CoV-2 patients with inflammatory bowel disease receiving vedolizumab</h3>
            <h3 id="section54"><a href="#section54" style="text-decoration: none; color: inherit;">🤖 Cytokine Patterns in Recovered COVID-19 Patients with Inflammatory Bowel Disease Taking Vedolizumab</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-01-02</p>
            <br>
            <p>🤖 <strong>Abstract</strong></p>
            <p>The research study aimed at understanding the differences in differential antibody responses of SARS-CoV-2 in inflammatory bowel disease (IBD) patients receiving biological therapies, infliximab and vedolizumab, to understand the underlying immune pathways. The study found a specific cytokine signature in patients on vedolizumab who were seropositive compared with those who were seronegative for SARS-CoV-2 antibodies, potentially linked to repeated SARS-CoV-2 infections. However, no differences were found between seropositive and seronegative patients on infliximab. The study also serves as the basis for further research into the immune response of viral infection in IBD patients, thought to arise from aberrant host-microbe responses.</p>
            <br><strong>Abstract</strong><br>
            <p>While differential antibody responses SARS-CoV-2 in patients with inflammatory bowel disease (IBD) receiving infliximab and vedolizumab are well-characterized, the immune pathways underlying these differences remain unknown. Prior to COVID-19 vaccine development, we screened 235 patients with IBD receiving biological therapy for antibodies to SARS-CoV-2 and measured serum cytokines. In seropositive patients, we prospectively collected clinical data. We found a cytokine signature in patients receiving vedolizumab who are seropositive compared with seronegative for SARS-CoV-2 antibodies that may be linked to repeated SARS-CoV-2 infections. However, there were no differences between seropositive and seronegative patients receiving infliximab. In this single-center cohort of patients with IBD with anti-SARS-CoV-2 antibodies at the onset of the COVID-19 pandemic, and therefore without influence of vaccination, there is a cytokine signature in patients receiving vedolizumab but not infliximab. These findings lay the groundwork for further studies on immune consequences of viral infection in patients with IBD, which is postulated to evolve from aberrant host-microbe responses.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-50035-1" target="_blank">https://www.nature.com/articles/s41598-023-50035-1</a></p>
            <br><br>
        </div>
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>
                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
     </div>
     </footr>
     <!-- end footer -->
     <!-- Javascript files--> 
     <script src="js/jquery.min.js"></script> 
     <script src="js/popper.min.js"></script> 
     <script src="js/bootstrap.bundle.min.js"></script> 
     <script src="js/jquery-3.0.0.min.js"></script> 
     <script src="js/plugin.js"></script> 
     <!-- sidebar --> 
     <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
     <script src="js/custom.js"></script>
     <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
     <script>
        $(document).ready(function(){
        $(".fancybox").fancybox({
        openEffect: "none",
        closeEffect: "none"
        });
        $(".zoom").hover(function(){
        $(this).addClass('transition');
        }, function(){
        $(this).removeClass('transition');
        });
        });   
     </script>
  </body>
</html>
