
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Data Sources</title>
    <link rel="icon" href="images/logo.png" type="image/x-icon">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/responsive.css">
    <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
    <style>
        @media (max-width: 767px) {
            .menu-area-main {
                flex-direction: column;
            }
            .menu-area-main li {
                text-align: center;
            }
        }
        .toggle-content { display: none; }
        .toggle-button {
            background-color: #007bff;
            color: white;
            border: none;
            padding: 4px 12px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 14px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
        }
    </style>
</head>
<body class="main-layout">
<header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo">
                                <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a>
                            </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main d-flex">
                                    <li><a href="index.html">Geo Map</a></li>
                                    <li><a href="trends.html">Trend Graphs</a></li>
                                    <li><a href="predictions_tab.html">Predictive</a></li>
                                    <li class="active"><a href="new_research.html">Research News</a></li>
                                    <li><a href="data_sources.html">Data Sources</a></li>
                                    <li><a href="about.html">About</a></li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>
<div class="brand_color">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="titlepage">
                    <h2>Research News</h2>
                    <br>
                    <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                    Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>
                </div>
            </div>
        </div>
    </div>
</div>
<!-- contact -->
<p>
<br><br><br>

        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-04</strong> <a href="https://www.nature.com/articles/s41467-024-50760-9" target="_blank">
            ü§ñ Vaccine Comparison in Adults: A Phase IV Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-0', this)">
            More</button></h4>
            <div id="toggle-0" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multiple vaccine comparison in the same adults reveals vaccine-specific and age-related humoral response patterns: an open phase IV trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simpler version of the abstract:

As people get older, their bodies often don't respond as well to vaccines. This makes it harder to create effective vaccination strategies for older adults who are at higher risk of getting infectious diseases. Researchers studied over 300 people from different age groups - young adults (25-49), middle-aged adults (50-64), and older adults (65+). They were given a flu vaccine, followed by a pneumococcal conjugate vaccine, and then two SARS-CoV-2 mRNA vaccines. The study found that older adults had lower antibody levels after receiving the pneumococcal conjugate and SARS-CoV-2 mRNA vaccines compared to younger adults. It also showed that some people, mostly older men, didn't respond well to multiple vaccines. This research can help identify groups at risk of low vaccine responsiveness and inform targeted vaccination strategies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccine responsiveness is often reduced in older adults. Yet, our lack of understanding of low vaccine responsiveness hampers the development of effective vaccination strategies to reduce the impact of infectious diseases in the ageing population. Young-adult (25‚Äì49‚Äây), middle-aged (50-64‚Äây) and older-adult (‚Äâ‚â•‚Äâ65‚Äây) participants of the VITAL clinical trials (n‚Äâ=‚Äâ315, age-range: 28-98‚Äây), were vaccinated with an annual (2019‚Äì2020) quadrivalent influenza (QIV) booster vaccine, followed by a primary 13-valent pneumococcal-conjugate (PCV13) vaccine (summer/autumn 2020) and a primary series of two SARS-CoV-2 mRNA-1273 vaccines (spring 2021). This unique setup allowed investigation of humoral responsiveness towards multiple vaccines within the same individuals over the adult age-range. Booster QIV vaccination induced comparable H3N2 hemagglutination inhibition (HI) titers in all age groups, whereas primary PCV13 and mRNA-1273 vaccination induced lower antibody concentrations in older as compared to younger adults (primary endpoint). The persistence of humoral responses, towards the 6 months timepoint, was shorter in older adults for all vaccines (secondary endpoint). Interestingly, highly variable vaccine responder profiles overarching multiple vaccines were observed. Yet, approximately 10% of participants, mainly comprising of older male adults, were classified as low responders to multiple vaccines. This study aids the identification of risk groups for low vaccine responsiveness and hence supports targeted vaccination strategies. Trial number: NL69701.041.19, EudraCT: 2019-000836-24.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50760-9" target="_blank">https://www.nature.com/articles/s41467-024-50760-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-03</strong> <a href="https://www.nature.com/articles/s41541-024-00932-x" target="_blank">
            ü§ñ Boosting Immunity to Bird Flu: How Self-Amplifying mRNA Vaccines Work in the Body</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-1', this)">
            More</button></h4>
            <div id="toggle-1" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity and biodistribution of lipid nanoparticle formulated self-amplifying mRNA vaccines against H5 avian influenza</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Immunogenicity and Biodistribution of mRNA Vaccines Against H5 Hemagglutinin in Mice

Scientists created four special vaccines that use self-amplifying (sa) mRNA to teach mice's immune systems how to fight a specific virus. These vaccines contained different parts of the virus's hemagglutinin protein, which is like a key that helps the virus attach to and enter host cells.

The scientists tested these vaccines in mice and found that all four versions worked by triggering an adaptive immune response. However, the version with the full-length hemagglutinin protein was more effective than the others.

They also discovered that the strength of the immune response correlated with the dose of the vaccine. Moreover, the vaccines triggered not only antibody responses but also cellular immune responses and the production of a specific type of antibody called IgA in the lungs.

Interestingly, when the scientists injected one of these special vaccines into mice's muscles, they found that it spread not only to the injection site but also to other parts of the body, including the draining lymph nodes, spleen, lung, kidney, liver, and heart.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study reports on the immunogenicity and biodistribution of H5 hemagglutinin (HA)-based self-amplifying (sa) mRNA vaccines in mice. Four sa-mRNA vaccines encoding either a secreted full-length HA, a secreted HA head domain, a secreted HA stalk domain, or a full-length membrane-anchored HA were investigated. All vaccines elicited an adaptive immune response. However, the full-length HA sa-RNA vaccines demonstrated superior performance compared to head and stalk domain vaccines. The antibody titers positively correlated with the vaccine dose. Cellular immune responses and antigen-specific IgA antibodies in the lungs were also observed. The comparison of the sa-mRNA vaccines encoding the secreted and membrane-anchored full-length HA revealed that anchoring of the HA to the membrane significantly enhanced the antibody and cellular responses. In addition to the injection site, the intramuscularly injected sa-mRNA-LNPs were also detected in the draining lymph nodes, spleen, and to a lesser extent, in the lung, kidney, liver, and heart.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00932-x" target="_blank">https://www.nature.com/articles/s41541-024-00932-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s44298-024-00047-z" target="_blank">
            ü§ñ Uncovering Hidden Patterns in Enterovirus D68 Outbreaks</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-2', this)">
            More</button></h4>
            <div id="toggle-2" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inferring enterovirus D68 transmission dynamics from the genomic data of two 2022 North American outbreaks</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person could understand:

A type of virus called Enterovirus D68 (EV-D68) has been causing a lot of respiratory problems in kids all over the world. In 2014, there was a big outbreak of EV-D68 in North America. Recently, there was another outbreak in Ontario, Canada from August to October 2022. Our scientists did some genetic testing and found that the virus that spread in Ontario was very similar to one that was spreading in Maryland at the same time. We used computer models to figure out how quickly the virus was spreading (this is called the "reproduction number") and found that it was around 2.7 in Ontario and 2.1 in Maryland. We also found out that people were usually sick with EV-D68 for about 8 days on average. Interestingly, we discovered that the virus started spreading again when social distancing rules were relaxed after COVID-19. This suggests that when we ease up on precautions to control one disease, another might take advantage of it and spread more easily. Our study shows how using genetic information can help us track and understand outbreaks, which is important for keeping people safe and healthy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Enterovirus D68 (EV-D68) has emerged as a significant cause of acute respiratory illness in children globally, notably following its extensive outbreak in North America in 2014. A recent outbreak of EV-D68 was observed in Ontario, Canada, from August to October 2022. Our phylogenetic analysis revealed a notable genetic similarity between the Ontario outbreak and a concurrent outbreak in Maryland, USA. Utilizing Bayesian phylodynamic modeling on whole genome sequences (WGS) from both outbreaks, we determined the median peak time-varying reproduction number (Rt) to be 2.70, 95% HPD (1.76, 4.08) in Ontario and 2.10, 95% HPD (1.41, 3.17) in Maryland. The Rttrends in Ontario closely matched those derived via EpiEstim using reported case numbers. Our study also provides new insights into the median infection duration of EV-D68, estimated at 7.94 days, 95% HPD (4.55, 12.8) in Ontario and 10.8 days, 95% HPD (5.85, 18.6) in Maryland, addressing the gap in the existing literature surrounding EV-D68‚Äôs infection period. We observed that the estimated Time since the Most Recent Common Ancestor (TMRCA) and the epidemic‚Äôs origin coincided with the easing of COVID-19 related social contact restrictions in both areas. This suggests that the relaxation of non-pharmaceutical interventions, initially implemented to control COVID-19, may have inadvertently facilitated the spread of EV-D68. These findings underscore the effectiveness of phylodynamic methods in public health, demonstrating their broad application from local to global scales and underscoring the critical role of pathogen genomic data in enhancing public health surveillance and outbreak characterization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00047-z" target="_blank">https://www.nature.com/articles/s44298-024-00047-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s41467-024-50875-z" target="_blank">
            ü§ñ Unpacking Proteomic Secrets at a Systems Level</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-3', this)">
            More</button></h4>
            <div id="toggle-3" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>OmicScope unravels systems-level insights from quantitative proteomics data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine you have a big puzzle with many pieces that need to be analyzed together. This is what happens in shotgun proteomics, where scientists try to understand all the proteins in a sample. But this process can be very complicated and requires using different tools to get meaningful results. OmicScope is a new tool that helps solve these problems by taking in various types of data, preparing it for analysis (like joining pieces together, fixing missing information, and making sure everything is on the same scale), and then performing detailed analyses to see what's different between groups or over time. It also has a feature that combines results from multiple studies to give a broader understanding. Plus, OmicScope comes with tools to visualize the data and can be used in both computer programs (like Python) and through a web application, making it accessible to researchers and helping them get accurate and efficient results more easily.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Shotgun proteomics analysis presents multifaceted challenges, demanding diverse tool integration for insights. Addressing this complexity, OmicScope emerges as an innovative solution for quantitative proteomics data analysis. Engineered to handle various data formats, it performs data pre-processing ‚Äì including joining replicates, normalization, data imputation ‚Äì and conducts differential proteomics analysis for both static and longitudinal experimental¬†designs. Empowered by Enrichr with over 224 databases, OmicScope performs Over Representation Analysis (ORA) and Gene Set Enrichment Analysis (GSEA). Additionally, its Nebula module facilitates meta-analysis from independent datasets, providing a systems biology approach for enriched insights. Complete with a data visualization toolkit and accessible as Python package and a web application, OmicScope democratizes proteomics analysis, offering an efficient and high-quality pipeline for researchers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50875-z" target="_blank">https://www.nature.com/articles/s41467-024-50875-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s44325-024-00011-z" target="_blank">
            ü§ñ Long COVID Heart Risks: A Study in Hong Kong</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-4', this)">
            More</button></h4>
            <div id="toggle-4" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk factors for long-term cardiovascular post-acute sequelae of COVID-19 infection: A nested case-control study in Hong Kong</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who got very sick with COVID-19 can experience problems even after they recover. These problems can affect their heart and blood vessels. Researchers are trying to figure out what makes some people more likely to get these heart-related problems.

In this study, scientists looked at the medical records of over 3,400 people in Hong Kong who had heart-related problems after having COVID-19. They compared these patients with a group of people who didn't have any heart-related problems after having COVID-19.

The scientists found that two things made it more likely for someone to get heart-related problems: if they had been very sick with COVID-19 and were hospitalized, or if they already had some other blood vessel disease. On the other hand, getting more doses of the COVID-19 vaccine seemed to protect against heart-related problems, and people who visited their doctor more often were also less likely to get these problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>People with COVID-19 can experience post-acute sequelae of SARS-CoV-2 (PASC). Studies on risk factors of PASC outcomes are ongoing, especially for endocrine system-related diseases that may impact the cardiovascular system. Cardiac-related PASC is one of the burdens after COVID-19 infection. This study aimed to examine the risk factors of cardiac-related PASC. In this nested case-control study, we obtained electronic health records (EHRs) database from the Hong Kong Hospital Authority. We defined cases as patients with at least one cardiac-related PASC and controls as patients without any cardiac-related PASC. We applied the incidence density sampling and matched controls to cases on age and sex at a 1:10 ratio. Multivariable conditional logistic regression was used to determine the associations between risk factors and cardiac-related PASC. A total of 455 individuals with cardiac-related PASC and matched 3,423 controls were obtained in the underlying cohort. COVID-19-associated hospitalisation (aOR: 1.41, 95% CI: 1.03‚Äì1.93) and peripheral vascular disease (aOR: 2.98, 95% CI: 1.31‚Äì6.79) were associated with an increased likelihood of cardiac-related PASC. Higher doses of the COVID-19 vaccine (2 doses: 0.68 [0.52‚Äì0.89]; ‚â•3 doses: 0.56 [0.40‚Äì0.78]) and more frequent healthcare utilization visits (aOR: 0.95, 95% CI: 0.92‚Äì0.97) were associated with a lower likelihood of cardiac-related PASC. This is the first study to examine risk factors of cardiac-related PASC among the Chinese population. We identified peripheral vascular disease and COVID-19-associated hospitalisation as the risk factors for cardiac-related PASC. COVID-19 vaccination was protective against cardiac-related PASC, which should be prioritized for high-risk patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44325-024-00011-z" target="_blank">https://www.nature.com/articles/s44325-024-00011-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s41598-024-68100-8" target="_blank">
            ü§ñ Stimulating Muscle Strength After COVID-19 Infection Recovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-5', this)">
            More</button></h4>
            <div id="toggle-5" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Gastrocnemius electrical stimulation increases ankle dorsiflexion strength in patients with post-acute sequelae of SARS-COV-2 (PASC): a double-blind randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine you've been very sick with COVID-19 for a long time, and your muscles have weakened significantly. This is called Post-Acute Sequelae of SARS-CoV-2 (PASC). We wanted to see if using electrical stimulation on the back of your legs could help improve muscle strength. To do this, we asked 18 people with PASC to either use a real device that sends electrical signals to their muscles or a fake device for one hour each day over four weeks. We then measured how well they could lift their feet and how strong their leg muscles were after four weeks. We found out that the people who used the real device had significantly stronger legs compared to those who used the fake device. This improvement was even more noticeable when we looked at how well their muscles responded to the electrical stimulation. However, using this method did not improve other daily activities such as bathing or walking for them. Using electrical stimulation on the back of your legs may help strengthen your leg muscles if you have PASC, but more people and longer treatments would be needed to confirm these results.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-Acute sequelae of SARS-CoV-2 (PASC) is a multisystem disorder causing persistent musculoskeletal deconditioning and reduced lower extremity strength. Electrical stimulation (E-Stim) to the gastrocnemius muscle can enhance strength outcomes by increasing the frequency of muscle fiber activation. We investigated its effect on individuals with PASC. Participants were randomized into intervention (IG) or control (CG) groups. The IG self-administered daily one-hour E-Stim to both their gastrocnemius muscles using a functional device over 4-week, while the CG used a sham device. Primary outcomes were ankle dorsiflexion strength assessed via dynamometry during maximum voluntary contractions, and gastrocnemius voluntary activation (GVA) via surface electromyography. The secondary outcome assessed activities of daily living (ADL), instrumental ADL, and mobility queries. Percentage improvement was calculated. Eighteen patients were analyzed (IG‚Äâ=‚Äâ10; CG‚Äâ=‚Äâ8). After 4¬†week, the IG showed a significantly higher improvement in ankle dorsiflexion strength (222.64%) compared to the CG (51.27%,p=‚Äâ0.002). Additionally, the IG‚Äôs ankle dorsiflexion strength improvement significantly correlated with GVA improvement (rho‚Äâ=‚Äâ0.782) at 4¬†week. The secondary outcomes did not reveal significant changes in neither group. Self-administered gastrocnemius E-Stim improves ankle dorsiflexion strength in individuals with PASC. However, larger sample sizes and longer interventions are needed to validate these findings.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68100-8" target="_blank">https://www.nature.com/articles/s41598-024-68100-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41467-024-50614-4" target="_blank">
            ü§ñ Understanding Vaccine Hesitancy: A Web Search Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-6', this)">
            More</button></h4>
            <div id="toggle-6" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Measuring vaccination coverage and concerns of vaccine holdouts from web search logs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To create good plans for vaccines, people in charge need accurate information about who has gotten vaccinated, who is still deciding, and why they are making those decisions. However, the data available in the US is not very detailed: sometimes vaccination rates are reported late or not broken down enough, and surveys about vaccine hesitation are limited because they ask high-level questions and rely on people's own reports.

We used search engine logs and machine learning to help fill these gaps. We looked at anonymized Bing data from February to August 2021. First, we developed a tool that can accurately tell when someone is searching for the COVID-19 vaccine on Bing. Our tool agrees with the CDC's vaccination rates most of the time, but reports them a week or two before the CDC does. It also estimates more detailed vaccination rates at the ZIP code level, which shows how different areas have varying levels of vaccine-seeking.

To study why people are hesitant to get vaccinated, we used our tool to identify two groups: those who got vaccinated early and those who held out. We found that holdouts were 67% more likely to click on websites that aren't trusted, and they were much more concerned about things like vaccine requirements, how vaccines are developed, and myths about vaccines. Even within the group of holdouts, there were clusters with different concerns and levels of openness to getting vaccinated.

Finally, we looked at how people's concerns and vaccine-seeking behaviors changed over time, and found that certain indicators could predict when someone would go from being hesitant to seeking the vaccine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To design effective vaccine policies, policymakers need detailed data about who has been vaccinated, who is holding out, and why. However, existing data in the US are insufficient: reported vaccination rates are often delayed or not granular enough, and surveys of vaccine hesitancy are limited by high-level questions and self-report biases. Here we show how search engine logs and machine learning can help to fill these gaps, using anonymized Bing data from February to August 2021. First, we develop avaccine intent classifierthat accurately detects when a user is seeking the COVID-19 vaccine on Bing. Our classifier demonstrates strong agreement with CDC vaccination rates, while preceding CDC reporting by 1‚Äì2‚Äâweeks, and estimates more granular ZIP-level rates, revealing local heterogeneity in vaccine seeking. To study vaccine hesitancy, we use our classifier to identify two groups,vaccine early adoptersandvaccine holdouts. We find that holdouts, compared to early adopters matched on covariates, are 67% likelier to click on untrusted news sites, and are much more concerned about vaccine requirements, development, and vaccine myths. Even within holdouts, clusters emerge with different concerns and openness to the vaccine. Finally, we explore the temporal dynamics of vaccine concerns and vaccine seeking, and find that key indicators predict when individuals convert from holding out to seeking the vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50614-4" target="_blank">https://www.nature.com/articles/s41467-024-50614-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41467-024-50931-8" target="_blank">
            ü§ñ Stopping the Spread: A Small Molecule Inhibitor Blocks COVID-19 Replication and Transmission</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-7', this)">
            More</button></h4>
            <div id="toggle-7" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The small molecule inhibitor of SARS-CoV-2 3CLpro EDP-235 prevents viral replication and transmission in vivo</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has caused many deaths and economic problems worldwide. While some medicines have helped fight the virus, they haven't been effective for everyone and are not always available. A new medicine called EDP-235 is being developed to stop a specific part of the SARS-CoV-2 virus that causes COVID-19 from working. It's shown to be very effective against all known versions of the virus and also works against other related viruses. This medicine has an advantage over others because it still works even if the virus has developed some resistance. Additionally, EDP-235 is very safe for humans with a large safety margin. In experiments on hamsters and ferrets, EDP-235 was able to stop the virus from spreading and causing harm. It also prevented the virus from being transmitted between animals that were in close contact. Overall, these results suggest that EDP-235 is a powerful medicine that can stop many types of viruses and might be useful for treating COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has led to the deaths of millions of people and severe global economic impacts. Small molecule therapeutics have played an important role in the fight against SARS-CoV-2, the virus responsible for COVID-19, but their efficacy has been limited in scope and availability, with many people unable to access their benefits, and better options are needed. EDP-235 is specifically designed to inhibit the SARS-CoV-2 3CLpro, with potent nanomolar activity against all SARS-CoV-2 variants to date, as well as clinically relevant human and zoonotic coronaviruses. EDP-235 maintains potency against variants bearing mutations associated with nirmatrelvir resistance. Additionally, EDP-235 demonstrates a‚Äâ‚â•‚Äâ500-fold selectivity index against multiple host proteases. In a male Syrian hamster model of COVID-19, EDP-235 suppresses SARS-CoV-2 replication and viral-induced hamster lung pathology. In a female ferret model, EDP-235 inhibits production of SARS-CoV-2 infectious virus and RNA at multiple anatomical sites. Furthermore, SARS-CoV-2 contact transmission does not occur when na√Øve ferrets are co-housed with infected, EDP-235-treated ferrets. Collectively, these results demonstrate that EDP-235 is a broad-spectrum coronavirus inhibitor with efficacy in animal models of primary infection and transmission.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50931-8" target="_blank">https://www.nature.com/articles/s41467-024-50931-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41522-024-00538-0" target="_blank">
            ü§ñ COVID-19 Linked to Changes in Gut Bacteria and Inflammation Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-8', this)">
            More</button></h4>
            <div id="toggle-8" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Gut microbiota dysbiosis is associated with altered tryptophan metabolism and dysregulated inflammatory response in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The way COVID-19 develops in the body can be unpredictable. We looked at samples from people who were sick with COVID-19 and compared them to people who weren't infected. We found that people who got very sick had lower levels of good bacteria in their intestines, but this wasn't as clear for people who got antibiotics. The levels of certain chemicals in the blood also changed - some increased while others decreased. This was linked to changes in the types and levels of good bacteria, and also with an increase in chemicals that cause inflammation. Overall, our study found connections between changes in the microbiome and metabolism that might contribute to severe COVID-19 symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The clinical course of COVID-19 is variable and often unpredictable. To test the hypothesis that disease progression and inflammatory responses associate with alterations in the microbiome and metabolome, we analyzed metagenome, metabolome, cytokine, and transcriptome profiles of repeated samples from hospitalized COVID-19 patients and uninfected controls, and leveraged clinical information and post-hoc confounder analysis. Severe COVID-19 was associated with a depletion of beneficial intestinal microbes, whereas oropharyngeal microbiota disturbance was mainly linked to antibiotic use. COVID-19 severity was also associated with enhanced plasma concentrations of kynurenine and reduced levels of several other tryptophan metabolites, lysophosphatidylcholines, and secondary bile acids. Moreover, reduced concentrations of various tryptophan metabolites were associated with depletion ofFaecalibacterium, and tryptophan decrease and kynurenine increase were linked to enhanced production of inflammatory cytokines. Collectively, our study identifies correlated microbiome and metabolome alterations as a potential contributor to inflammatory dysregulation in severe COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41522-024-00538-0" target="_blank">https://www.nature.com/articles/s41522-024-00538-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41598-024-68592-4" target="_blank">
            ü§ñ Helping Sicker COVID Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-9', this)">
            More</button></h4>
            <div id="toggle-9" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of additional hemoperfusion in hospitalized patients with severe to critical COVID-19 disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The use of a special machine to remove bad stuff from the blood of very sick COVID-19 patients has not been proven to help them live longer. But it did seem to make their stay in the hospital shorter and helped prevent some serious complications like lung problems, sepsis, and bleeding. However, this treatment did not improve the number of patients who survived.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The evidence supporting additional hemoperfusion (HP) with cytokine adsorbents for improving clinical outcomes in severe to critical coronavirus disease 2019 (COVID-19) patients remains limited. We compared severe to critical COVID-19 patients who received additional HP with a cytokine adsorbent to matched cases receiving standard medical treatment (SMT). The primary outcome was hospital mortality. In our study, we matched 45 patients who received additional HP 1:1 with the SMT group based on key clinical parameters. The hospital mortality rates did not differ between the groups (33% vs 38%,p=‚Äâ0.83). The HP group had a significantly shorter ICU stay (22 vs 32¬†days; p‚Äâ=‚Äâ0.017) and reduced mechanical ventilation duration (15 vs 35¬†days;p<‚Äâ0.001). Additionally, the incidence of pulmonary complications (20% vs 42%; p‚Äâ=‚Äâ0.04), sepsis (38% vs 64%;p=‚Äâ0.02), and disseminated intravascular coagulopathy (DIC) (13% vs 33%;p=‚Äâ0.046) were significantly lower in the HP group. In conclusion,¬†among severe to critical COVID-19 patients, additional HP with a cytokine adsorbent did not improve hospital mortality. However, it reduced ICU length of stay, mechanical ventilator days, and incidences of lung complications, sepsis, and DIC.Trial registration: TCTR20231002006. Registered 02 October 2023 (retrospectively registered).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68592-4" target="_blank">https://www.nature.com/articles/s41598-024-68592-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41598-024-68617-y" target="_blank">
            ü§ñ Uncovering the Impact of 1mŒ® on Positive-Stranded RNA Virus-Based Replicons</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-10', this)">
            More</button></h4>
            <div id="toggle-10" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>1mŒ® influences the performance of various positive-stranded RNA virus-based replicons</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT 
                        Self-replicating RNAs (saRNAs) are special tools that use a type of enzyme from viruses to make more copies of themselves once inside the target cell. This helps spread the information about an antigen, which is essentially a part of what triggers the immune system's response against a specific disease. Recently, there have been several clinical trials using saRNA vaccines as they potentially offer lower vaccination doses compared to traditional mRNA-based approaches. A modification called N1- methyl-pseudouridine (1mŒ®) was also included in these newer mRNA vaccines. It enhances RNA stability and reduces the immune response triggered by RNAs. In this study, we looked at how this modified nucleoside affects different saRNA platforms based on various viruses. The results showed that it impacted different stages of replication depending on the virus. For example, an insect virus called TNCL had its replication impaired because the viral RNA was not recognized well by the RdRp enzyme. On the other hand, the translation step was severely affected in coxsackievirus B3 (CVB3), which belongs to the Picornaviridae family. Lastly, the effects of 1mŒ® on Semliki forest virus (SFV) were not harmful in vitro studies but did not offer any benefits when immunogenicity was tested in vivo.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Self-amplifying RNAs (saRNAs) are versatile vaccine platforms that take advantage of a viral RNA-dependent RNA polymerase (RdRp) to amplify the messenger RNA (mRNA) of an antigen of interest encoded within the backbone of the viral genome once inside the target cell. In recent years, more saRNA vaccines have been clinically tested with the hope of reducing the vaccination dose compared to the conventional mRNA approach. The use of N1-methyl-pseudouridine (1mŒ®), which enhances RNA stability and reduces the innate immune response triggered by RNAs, is among the improvements included in the current mRNA vaccines. In the present study, we evaluated the effects of this modified nucleoside on various saRNA platforms based on different viruses. The results showed that different stages of the replication process were affected depending on the backbone virus. For TNCL, an insect virus of theAlphanodavirusgenus, replication was impaired by poor recognition of viral RNA by RdRp. In contrast, the translation step was severely abrogated in coxsackievirus B3 (CVB3), a member of thePicornaviridaefamily. Finally, the effects of 1mŒ® on Semliki forest virus (SFV), were not detrimental in in vitro studies, but no advantages were observed when immunogenicity was tested in vivo.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68617-y" target="_blank">https://www.nature.com/articles/s41598-024-68617-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41467-024-49634-x" target="_blank">
            ü§ñ COVID-19 Vaccination Safety: A Study of 46 Million Adults in England</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-11', this)">
            More</button></h4>
            <div id="toggle-11" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cohort study of cardiovascular safety of different COVID-19 vaccination doses among 46 million adults in England</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The first dose of some COVID-19 vaccines helped lower heart problems in a big group of adults in England. But we didn't know if getting second or booster doses would have the same effect. Our study looked at health records from 45.7 million adults in England and compared heart problem rates before, during, and after their first, second, and booster doses. We found that getting any dose of COVID-19 vaccine helped lower common heart problems like heart attacks and strokes. However, we also noticed a small increase in rare heart issues like blood clots after the first dose of some vaccines and heart inflammation after mRNA vaccine doses. These results suggest that it's still safe to get vaccinated against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The first dose of COVID-19 vaccines led to an overall reduction in cardiovascular events, and in rare cases, cardiovascular complications. There is less information about the effect of second and booster doses on cardiovascular diseases. Using longitudinal health records from 45.7 million adults in England between December 2020 and January 2022, our study compared the incidence of thrombotic and cardiovascular complications up to 26 weeks after first, second and booster doses of brands and combinations of COVID-19 vaccines used during the UK vaccination program with the incidence before or without the corresponding vaccination. The incidence of common arterial thrombotic events (mainly acute myocardial infarction and ischaemic stroke) was generally lower after each vaccine dose, brand and combination. Similarly, the incidence of common venous thrombotic events, (mainly pulmonary embolism and lower limb deep venous thrombosis) was lower after vaccination. There was a higher incidence of previously reported rare harms after vaccination: vaccine-induced thrombotic thrombocytopenia after first ChAdOx1 vaccination, and myocarditis and pericarditis after first, second and transiently after booster mRNA vaccination (BNT-162b2 and mRNA-1273). These findings support the wide uptake of future COVID-19 vaccination programs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49634-x" target="_blank">https://www.nature.com/articles/s41467-024-49634-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-30</strong> <a href="https://www.nature.com/articles/s41598-024-68678-z" target="_blank">
            ü§ñ Tracking a New COVID Variant in India: What We Know</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-12', this)">
            More</button></h4>
            <div id="toggle-12" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Emergence of SARS-CoV-2 omicron variant JN.1 in Tamil Nadu, India - Clinical characteristics and novel mutations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that's easy to understand:

In December 2023, a new version of the COVID-19 virus called omicron became the most common variant in many parts of the world. We studied this new version, which we called JN.1, and found out what it looks like on a genetic level. We also looked at how its unique changes might affect the way it interacts with human cells. Our analysis showed that some of these changes could make it easier for the virus to attach to and enter human cells. These findings highlight the importance of keeping a close eye on the COVID-19 virus as it evolves, so we can stay ahead of any new threats and develop better ways to prevent and treat infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In December 2023, we observed a notable shift in the COVID-19 landscape, when JN.1 omicron emerged as the predominant SARS-CoV-2 variant with a 95% incidence. We characterized the clinical profile, and genetic changes in JN.1, an emerging SARS-CoV-2 variant of interest. Whole genome sequencing was performed on SARS-CoV-2 positive clinical specimens, followed by sequence analysis. Mutations within the spike protein sequences were analysed and compared with the previously reported lineages and sub-lineages, to identify the potential impact of the unique mutations on protein structure and possible alterations in the functionality. Several unique and dynamic mutations were identified herein. Molecular docking analysis showed changes in the binding affinity, and key interacting residues of wild-type and mutated structures with key host cell receptors of SARS-CoV-2 entry viz., ACE2, CD147, CD209L and AXL. Our data provides key insights on the emergence of newer variants and highlights the necessity for robust and sustained global genomic surveillance of SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68678-z" target="_blank">https://www.nature.com/articles/s41598-024-68678-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41598-024-68264-3" target="_blank">
            ü§ñ Detecting Aflatoxin Poisoning with a New High-Tech Sensor</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-13', this)">
            More</button></h4>
            <div id="toggle-13" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>On-site sensing for aflatoxicosis poisoning via ultraviolet excitable aptasensor based on fluorinated ethylene propylene strip: a promising forensic tool</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The contamination of food in Southeast Asia and Africa with a toxic substance called Aflatoxin B1 is a big problem. This toxin can cause serious liver damage and even death if people eat contaminated food. We need a simple way to detect this toxin quickly and easily, especially when there are no special facilities available. Our research team has developed a new tool that uses a special film strip to detect Aflatoxin B1 using ultraviolet light. This method is easy to make, can be used on-site, and lasts for 19 days. It's also reusable four times! We tested this tool with real-world samples like brown sugar, peanuts, and rice, and the results were very close to those from a more complicated laboratory test called High-Performance Liquid Chromatography. This discovery is exciting because it uses a unique UV light pattern and a special film strip that could lead to new and innovative ways to detect environmental pollutants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The environmental contamination by extremophileAspergillusspecies, i.e., Aflatoxin B1, is hardly controllable in Southeast Asia and Sub-Saharan Africa, which lack handling resources and controlled storage facilities. Acute aflatoxicosis poisoning from aflatoxin-prone dietary staples could cause acute hepatic necrosis, acute liver failure, and death. Here, as the cheaper, more straightforward, and facile on-site diagnostic kit is needed, we report an ultraviolet-excitable optical aptasensor based on a fluorinated ethylene propylene film strip. Molecular dynamics on the aptamer.AFB1 complex revealed that the AFB1 to the aptamer increases the overall structural stability, suggesting that the aptamer design is suitable for the intended application. Under various influencing factors, the proposed label-free strategy offers a fast 20-min on-site fabrication simplicity and 19-day shelf-life. The one-pot incubation provides an alternative to catalytic detection and exhibited 4 times reusability. The recovery of crude brown sugar, processed peanuts, and long-grain rice were 102.74‚Äâ¬±‚Äâ0.41 (n‚Äâ=‚Äâ3), 86.90‚Äâ¬±‚Äâ3.38 (n‚Äâ=‚Äâ3), and 98.50‚Äâ¬±‚Äâ0.42 (n‚Äâ=‚Äâ3), comparable to High-Performance Liquid Chromatography-Photodiode Array Detector results. This study is novel owing to the peculiar UV-active spectrum fingerprint and the convenient use of hydrophobic film strips that could promote breakthrough innovations and new frontiers for on-site/forensic detection of environmental pollutants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68264-3" target="_blank">https://www.nature.com/articles/s41598-024-68264-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-49891-w" target="_blank">
            ü§ñ COVID-19 in Wild Animals: A Growing Concern</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-14', this)">
            More</button></h4>
            <div id="toggle-14" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Widespread exposure to SARS-CoV-2 in wildlife communities</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract:

Wild animals are getting infected with the COVID-19 virus, which is spread by humans. While it's known that many different species can get sick from COVID, scientists didn't know how widespread this problem was or what factors made some animals more likely to catch the virus. To find out, researchers tested 23 different wild animal species for COVID and looked at how urbanization and human activity affected their risk of getting infected. The study found that six types of wildlife - including deer mice, opossums, raccoons, groundhogs, cottontails, and red bats - had COVID-19 antibodies in them, indicating they had been exposed to the virus. Furthermore, areas with high human activity had three times more COVID-19 cases than areas with low human use. The researchers also found that the virus sequences from nine wild animals matched those circulating among humans at the time, suggesting at least seven recent human-to-animal transmission events. Overall, these findings indicate that many wildlife communities have been exposed to COVID-19 and that areas with high human activity may be hotspots for cross-species transmission.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pervasive SARS-CoV-2 infections in humans have led to multiple transmission events to animals. While SARS-CoV-2 has a potential broad wildlife host range, most documented infections have been in captive animals and a single wildlife species, the white-tailed deer. The full extent of SARS-CoV-2 exposure among wildlife communities and the factors that influence wildlife transmission risk remain unknown. We sampled 23 species of wildlife for SARS-CoV-2 and examined the effects of urbanization and human use on seropositivity. Here, we document positive detections of SARS-CoV-2 RNA in six species, including the deer mouse, Virginia opossum, raccoon, groundhog, Eastern cottontail, and Eastern red bat between May 2022‚ÄìSeptember 2023 across Virginia and Washington, D.C., USA. In addition, we found that sites with high human activity had three times higher seroprevalence than low human-use areas. We obtained SARS-CoV-2 genomic sequences from nine individuals of six species which were assigned to seven Pango lineages of the Omicron variant. The close match to variants circulating in humans at the time suggests at least seven recent human-to-animal transmission events. Our data support that exposure to SARS-CoV-2 has been widespread in wildlife communities and suggests that areas with high human activity may serve as points of contact for cross-species transmission.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49891-w" target="_blank">https://www.nature.com/articles/s41467-024-49891-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">
            ü§ñ Understanding How Our Lifestyle Choices Affect COVID-19 Recovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-15', this)">
            More</button></h4>
            <div id="toggle-15" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modifiable lifestyle factors and the risk of post-COVID-19 multisystem sequelae, hospitalization, and death</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Effective ways to prevent long-term problems after having COVID are important for patients, doctors, and policy makers to consider. This study looked at how different lifestyle choices (like not smoking, eating well, exercising regularly, getting enough sleep) affected the risk of severe COVID symptoms, death, and hospitalization in a large group of people from the UK Biobank cohort (n=‚Äâ68,896). The results showed that people who made healthy lifestyle choices had a 36% lower risk of long-term COVID complications compared to those who didn't make healthy choices. This benefit was seen across all body systems, including heart, blood clotting, metabolism, gut health, kidneys, mental health, muscles, breathing, and fatigue. The good effects were mostly due to the direct impact of lifestyle choices on the body, rather than being influenced by pre-existing medical conditions (71% for any sequelae). Healthy lifestyles also reduced the risk of death and hospitalization after COVID. These associations remained true even when looking at different stages of infection, whether or not people had been hospitalized, vaccinated, or infected with a specific variant of SARS-CoV-2. These findings highlight the importance of maintaining a healthy lifestyle to mitigate long-term COVID effects and prepare for future pandemics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Effective prevention strategies for post-COVID complications are crucial for patients, clinicians, and policy makers to mitigate their cumulative burden. This study evaluated the association of modifiable lifestyle factors (smoking, alcohol intake, BMI, physical activity, sedentary time, sleep duration, and dietary habits) with COVID-19 multisystem sequelae, death, and hospitalization in the UK Biobank cohort (n=‚Äâ68,896). A favorable lifestyle (6-10 healthy factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae (HR, 0.64; 95% CI, 0.58-0.69; ARR at 210 days, 7.08%; 95% CI, 5.98-8.09) compared to an unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions spanned all 10 organ systems, including cardiovascular, coagulation, metabolic, gastrointestinal, kidney, mental health, musculoskeletal, respiratory disorders, and fatigue. This beneficial effect was largely attributable to direct lifestyle impacts independent of corresponding pre-infection comorbidities (71% for any sequelae). A favorable lifestyle was also related to the risk of post-COVID death (HR 0.59, 0.52-0.66) and hospitalization (HR 0.78, 0.73-0.84). These associations persisted across acute and post-acute infection phases, irrespective of hospitalization status, vaccination, or SARS-CoV-2 variant. These findings underscore the clinical and public health importance of adhering to a healthy lifestyle in mitigating long-term COVID-19 adverse impacts and enhancing future pandemic preparedness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">https://www.nature.com/articles/s41467-024-50495-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-27</strong> <a href="https://www.nature.com/articles/s41467-024-50286-0" target="_blank">
            ü§ñ Unraveling the Complexity of Immune Response: A Study on Neutralizing Antibody Lineages in Macaques</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-16', this)">
            More</button></h4>
            <div id="toggle-16" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-compartmental diversification of neutralizing antibody lineages dissected in SARS-CoV-2 spike-immunized macaques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus is still changing, which means we need to understand how our bodies are responding to it. In this study, scientists used a combination of techniques to see what happens when monkeys are vaccinated with the part of the virus that causes COVID-19 (called the spike protein). They found that the immune cells in the monkeys' bodies were able to evolve and adapt to recognize not just the original form of the spike protein, but also new forms that might appear later. This shows that our bodies have a built-in ability to learn and improve their response to the virus over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continued evolution of SARS-CoV-2 underscores the need to understand qualitative aspects of the humoral immune response elicited by spike immunization. Here, we combine monoclonal antibody (mAb) isolation with deep B cell receptor (BCR) repertoire sequencing of rhesus macaques immunized with prefusion-stabilized spike glycoprotein. Longitudinal tracing of spike-sorted B cell lineages in multiple immune compartments demonstrates increasing somatic hypermutation and broad dissemination of vaccine-elicited B cells in draining and non-draining lymphoid compartments, including the bone marrow, spleen and, most notably, periaortic lymph nodes. Phylogenetic analysis of spike-specific monoclonal antibody lineages identified through deep repertoire sequencing delineates extensive intra-clonal diversification that shaped neutralizing activity. Structural analysis of the spike in complex with a broadly neutralizing mAb provides a molecular basis for the observed differences in neutralization breadth between clonally related antibodies. Our findings highlight that immunization leads to extensive intra-clonal B cell evolution where members of the same lineage can both retain the original epitope specificity and evolve to recognize additional spike variants not previously encountered.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50286-0" target="_blank">https://www.nature.com/articles/s41467-024-50286-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-27</strong> <a href="https://www.nature.com/articles/s41598-024-68184-2" target="_blank">
            ü§ñ COVID-19's Lasting Impact on Pregnant Women and Babies in China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-17', this)">
            More</button></h4>
            <div id="toggle-17" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The effect of long-term COVID-19 infection on maternal and fetal complications: a retrospective cohort study conducted at a single center in China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The effects of long-term COVID-19 on pregnant women and their babies were studied over a year at a hospital in China. Researchers compared 623 pregnant women who had COVID-19 during pregnancy, those who didn't have COVID-19, and some who still had COVID-19 after they gave birth. They found that the women who had long-term COVID-19 were more likely to get high blood pressure during pregnancy (called gestational hypertension) and to develop diabetes during pregnancy (called gestational diabetes mellitus). They also found that their babies might not grow as much inside the womb as other babies did, a condition called fetal intrauterine growth restriction. However, the study didn't find any significant changes in blood clotting or other complications like premature birth or baby problems after birth.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Investigate the effect of long-term COVID-19 on maternal and fetal complications. A retrospective cohort study was conducted. A total of 623 pregnant women who delivered in Kunming First People's Hospital from November 1, 2022 to July 31, 2023 were selected. By employing statistical methods, we compared the associations between maternal and fetal complications in pregnant women with acute COVID-19 during pregnancy, long-term COVID-19, and non-COVID-19 pregnant women. In the final 623 samples, there were 209 pregnant women with acute COVID-19, 72 pregnant women with long-term COVID-19, and 342 pregnant women without COVID-19. The epidemiological and clinical characteristics of all subjects were similar. Pregnant individuals who developed long-term COVID-19 during their pregnancy had an increased risk of experiencing gestational hypertension (OR 3.344, 95% CI 1.544‚Äì7.243), gestational diabetes mellitus (OR 2.301, 95% CI 1.290‚Äì4.102), and fetal intrauterine growth restriction (OR 2.817, 95% CI 1.385‚Äì5.952). Multivariate binary logistic regression analysis showed that this association remained consistent even after adjusting for confounders and performing subgroup analyses. Other maternal and fetal complications, such as premature rupture of membranes, preterm delivery, neonatal asphyxia, and transfer of neonates to NICU, did not exhibit statistically significant associations. After linear regression analysis, the platelet count (Œ≤:‚Äâ‚àí‚Äâ0.127, 95% CI‚Äâ‚àí‚Äâ0.001‚Äì0.000) of pregnant women with long-term COVID-19 was slightly lower than that of non-COVID-19 pregnant women, and the other coagulation parameters were not statistically significant. The incidence of gestational hypertension, gestational diabetes mellitus and fetal intrauterine growth restriction in pregnant women with long-term COVID-19 is significantly increased, but it does not further increase the coagulation status.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68184-2" target="_blank">https://www.nature.com/articles/s41598-024-68184-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41467-024-50601-9" target="_blank">
            ü§ñ Forecasting Influenza Hospitalizations: A Two-Year Evaluation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-18', this)">
            More</button></h4>
            <div id="toggle-18" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Title evaluation of FluSight influenza forecasting in the 2021‚Äì22 and 2022‚Äì23 seasons with a new target laboratory-confirmed influenza hospitalizations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Accurate predictions can help us prepare for and respond to flu outbreaks more effectively. In 2021 and 2022, a group of scientists made weekly forecasts about how many people would be hospitalized with the flu one to four weeks in advance. They compared their predictions to actual numbers and found that some models were better than others at making accurate forecasts. In fact, an average of all these models (called the FluSight ensemble) was one of the most accurate, but it started to struggle a bit when looking further ahead into the future. This shows that even with multiple models working together, there can be challenges in predicting flu outbreaks, especially during times of rapid change.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Accurate forecasts can enable more effective public health responses during seasonal influenza epidemics. For the 2021‚Äì22 and 2022‚Äì23 influenza seasons, 26 forecasting teams provided national and jurisdiction-specific probabilistic predictions of weekly confirmed influenza hospital admissions for one-to-four weeks ahead. Forecast skill is evaluated using the Weighted Interval Score (WIS), relative WIS, and coverage. Six out of 23 models outperform the baseline model across forecast weeks and locations in 2021‚Äì22 and 12 out of 18 models in 2022‚Äì23. Averaging across all forecast targets, the FluSight ensemble is the 2ndmost accurate model measured by WIS in 2021‚Äì22 and the 5thmost accurate in the 2022‚Äì23 season. Forecast skill and 95% coverage for the FluSight ensemble and most component models degrade over longer forecast horizons. In this work we demonstrate that while the FluSight ensemble was a robust predictor, even ensembles face challenges during periods of rapid change.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50601-9" target="_blank">https://www.nature.com/articles/s41467-024-50601-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41467-024-50619-z" target="_blank">
            ü§ñ Unlocking mRNA Delivery with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-19', this)">
            More</button></h4>
            <div id="toggle-19" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>AGILE platform: a deep learning powered approach to accelerate LNP development for mRNA delivery</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Deep learning and chemistry are being used together to create special nanoparticles that can deliver medicine exactly where it's needed in the body. This is especially helpful for medicines made from a type of genetic material called mRNA. Right now, there's only one kind of nanoparticle that's widely used for mRNA delivery, which is mainly helping with COVID-19 vaccines. But scientists want to develop more types of nanoparticles that can target different cell types in the body. They've created a system called AGILE, which uses computers and chemistry to design and test these special nanoparticles quickly and efficiently. This allows them to find the best nanoparticle for each type of cell, making mRNA therapies even more effective and helping to broaden their use in medicine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Ionizable lipid nanoparticles (LNPs) are seeing widespread use in mRNA delivery, notably in SARS-CoV-2 mRNA vaccines. However, the expansion of mRNA therapies beyond COVID-19 is impeded by the absence of LNPs tailored for diverse cell types. In this study, we present the AI-Guided Ionizable Lipid Engineering (AGILE) platform, a synergistic combination of deep learning and combinatorial chemistry. AGILE streamlines ionizable lipid development with efficient library design, in silico lipid screening via deep neural networks, and adaptability to diverse cell lines. Using AGILE, we rapidly design, synthesize, and evaluate ionizable lipids for mRNA delivery, selecting from a vast library. Intriguingly, AGILE reveals cell-specific preferences for ionizable lipids, indicating tailoring for optimal delivery to varying cell types. These highlight AGILE‚Äôs potential in expediting the development of customized LNPs, addressing the complex needs of mRNA delivery in clinical practice, thereby broadening the scope and efficacy of mRNA therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50619-z" target="_blank">https://www.nature.com/articles/s41467-024-50619-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41598-024-66439-6" target="_blank">
            ü§ñ Progesterone Helps Reduce Inflammation in Airways</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-20', this)">
            More</button></h4>
            <div id="toggle-20" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Progesterone modulates the immune microenvironment to suppress ovalbumin-induced airway inflammation by inhibiting NETosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Asthma is a serious breathing problem that affects many people. Before puberty, girls tend to have fewer and less severe symptoms than boys. Scientists wanted to explore if the female sex hormone progesterone (P4) can help reduce inflammation in the lungs of mice with asthma-like symptoms caused by an allergic reaction. They studied two groups of female mice: one group had their ovaries removed to see how it would affect the results.

The scientists collected samples from the mice's blood, lung fluid, and tissues, and analyzed them. They found that P4 treatment helped reduce inflammation in the lungs, decreased mucus production, and reduced the number of immune cells in the lung fluid. It also increased the levels of certain beneficial cells while decreasing those that contribute to inflammation.

P4 was shown to inhibit a process called NETosis, which is linked to inflammation and airway damage. Additionally, it suppressed the activity of an enzyme involved in this process, leading to reduced production of reactive oxygen species. Furthermore, P4 treatment hindered the release of double-stranded DNA during NETosis.

These findings suggest that P4 may be beneficial in reducing inflammation associated with allergic asthma by modulating the immune microenvironment. The research indicates the potential of P4 as a therapeutic agent for ameliorating inflammation in OVA-induced allergic asthma mice.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Studies have demonstrated that prior to puberty, girls have a lower incidence and severity of asthma symptoms compared to boys. This study aimed to explore the role of progesterone (P4), a sex hormone, in reducing inflammation and altering the immune microenvironment in a mouse model of allergic asthma induced by OVA. Female BALB/c mice with or without ovariectomy to remove the influence of sex hormones were used for the investigations. Serum, bronchoalveolar lavage fluid (BALF), and lung tissue samples were collected for analysis. The results indicated that P4 treatment was effective in decreasing inflammation and mucus secretion in the lungs of OVA-induced allergic asthma mice. P4 treatment also reduced the influx of inflammatory cells into the BALF and increased the levels of Th1 and Th17 cytokines while decreasing the levels of Th2 and Treg cytokines in both BALF and lung microenvironment CD45+T cells. Furthermore, P4 inhibited the infiltration of inflammatory cells into the lungs, suppressed NETosis, and reduced the number of pulmonary CD4+T cells while increasing the number of regulatory T cells. The neutrophil elastase inhibitor GW311616A also suppressed airway inflammation and mucus production and modified the secretion of immune Th1, Th2, Th17, and Treg cytokines in lung CD45+immune cells. These changes led to an alteration of the immunological milieu with increased Th1 and Th17 cells, accompanied by decreased Th2, Treg, and CD44+T cells, similar to the effects of P4 treatment. Treatment with P4 inhibited NETosis by suppressing the p38 pathway activation, leading to reduced reactive oxygen species production. Moreover, P4 treatment hindered the release of double-stranded DNA during NETosis, thereby influencing the immune microenvironment in the lungs. These findings suggest that P4 treatment may be beneficial in reducing inflammation associated with allergic asthma by modulating the immune microenvironment. In conclusion, this research indicates the potential of P4 as a therapeutic agent for ameliorating inflammation in OVA-induced allergic asthma mice.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66439-6" target="_blank">https://www.nature.com/articles/s41598-024-66439-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41598-024-68055-w" target="_blank">
            ü§ñ Safety of High-Dose Nitric Oxide in Treating COVID-19 Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-21', this)">
            More</button></h4>
            <div id="toggle-21" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A safety evaluation of intermittent high-dose inhaled nitric oxide in viral pneumonia due to COVID-19: a randomised clinical study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>High doses of nitric oxide inhaled into the lungs have been found to help with infections, improve blood flow, and reduce inflammation. This study looked at whether giving high doses of nitric oxide, four times a day for seven days, could help people who were in hospital with viral pneumonia. The researchers compared this treatment to just standard care for 40 adults with viral pneumonia. They found that the treatment group had fewer problems related to breathing and were able to get off oxygen support sooner. Overall, the study suggests that giving high doses of nitric oxide is a safe and helpful way to treat people in hospital with viral pneumonia.</p><br>
                <p><strong>Abstract</strong></p>
                <p>High-dose inhaled Nitric Oxide (iNO) has been shown to have anti-inflammatory, vasodilator, and antimicrobial properties, resulting in improved arterial oxygenation as well as a beneficial therapeutic effect on lower respiratory tract infections. This study evaluated the safety and efficacy of 150-ppm intermittent iNO administered with a novel iNO-generator, for treating adults hospitalised for viral pneumonia. In this prospective, open-label, multicenter study, subjects aged 18‚Äì80, diagnosed with viral pneumonia received either standard supportive treatment alone (Control-Group) or combined with iNO for 40¬†min, 4 times per day up to 7¬†days (Treatment-Group). Out of 40 recruited subjects, 35 were included in the intention-to-treat population (34 with COVID-19). Adverse Events rate was similar between the groups (56.3% vs. 42.1%; respectively). No treatment-related adverse events were reported, while 2 serious adverse events were accounted for by underlying pre-existing conditions. Among the Treatment-Group, oxygen support duration was reduced by 2.7¬†days (Hazard Ratio‚Äâ=‚Äâ2.8; p‚Äâ=‚Äâ0.0339), a greater number of subjects reached oxygen saturation‚Äâ‚â•‚Äâ93% within hospitalisation period (Hazard Ratio‚Äâ=‚Äâ5.4; p‚Äâ=‚Äâ0.049), and a trend for earlier discharge was demonstrated. Intermittent 150-ppm iNO-treatment is well-tolerated, safe, and beneficial compared to usual care for spontaneously breathing hospitalised adults diagnosed with COVID-19 viral pneumonia.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68055-w" target="_blank">https://www.nature.com/articles/s41598-024-68055-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-25</strong> <a href="https://www.nature.com/articles/s41541-024-00922-z" target="_blank">
            ü§ñ Protecting Against Multiple COVID Strains with a Single Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-22', this)">
            More</button></h4>
            <div id="toggle-22" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Universal subunit vaccine protects against multiple SARS-CoV-2 variants and SARS-CoV</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract of the article that a young person could understand:

Scientists are trying to find a way to make a special medicine, called a vaccine, that can protect us against many different types of a virus called SARS-CoV-2. This virus causes COVID-19 and has been making people sick all over the world. The scientists were able to create a few different versions of this vaccine by taking parts from older versions of the virus and putting them together with a new part that they created. They tested these vaccines in mice and found out that one version, called S-6P-Delta-RBD, was very effective at protecting against many different types of the virus, including some that were quite similar to the original virus and others that had changed a lot over time. This vaccine worked so well that it even protected mice who hadn't been vaccinated before when they were challenged with the virus. The scientists think that this vaccine could be used to protect people against all sorts of SARS-CoV-2 viruses, including ones that might come in the future and cause new outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although Omicron RBD of SARS-CoV-2 accumulates many mutations, the backbone region (truncated RBD) of spike protein is highly conserved. Here, we designed several subunit vaccines by keeping the conserved spike backbone region of SARS-CoV-2 Omicron BA.1 subvariant (S-6P-no-RBD), or inserting the RBD of Delta variant (S-6P-Delta-RBD), Omicron (BA.5) variant (S-6P-BA5-RBD), or ancestral SARS-CoV-2 (S-6P-WT-RBD) to the above backbone construct, and evaluated their ability to induce immune responses and cross-protective efficacy against various SARS-CoV-2 variants and SARS-CoV. Among the four subunit vaccines, S-6P-Delta-RBD protein elicited broad and potent neutralizing antibodies against all SARS-CoV-2 variants tested, including Alpha, Beta, Gamma, and Delta variants, the BA.1, BA.2, BA.2.75, BA.4.6, and BA.5 Omicron subvariants, and the ancestral strain of SARS-CoV-2. This vaccine prevented infection and replication of SARS-CoV-2 Omicron, and completely protected immunized mice against lethal challenge with the SARS-CoV-2 Delta variant and SARS-CoV. Sera from S-6P-Delta-RBD-immunized mice protected naive mice against challenge with the Delta variant, with significantly reduced viral titers and without pathological effects. Protection correlated positively with the serum neutralizing antibody titer. Overall, the designed vaccine has potential for development as a universal COVID-19 vaccine and/or a pan-sarbecovirus subunit vaccine that will prevent current and future outbreaks caused by SARS-CoV-2 variants and SARS-related CoVs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00922-z" target="_blank">https://www.nature.com/articles/s41541-024-00922-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41598-024-67815-y" target="_blank">
            ü§ñ Unexplained Fatigue and Hormonal Imbalances: A Common Thread in Long COVID Cases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-23', this)">
            More</button></h4>
            <div id="toggle-23" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical and endocrine features of orthostatic intolerance detected in patients with long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had COVID-19 for a long time often get dizzy or faint when they stand up, but we don't know why this happens. To learn more, we studied 688 people who came to our clinic with long COVID and were suspected of having this problem. We tested them while they stood still for 10 minutes to see how their bodies reacted. In 33 out of the 86 people we tested, their body reacted strongly when they stood up - they got nausea (feeling sick to their stomach) and their heart beat faster than usual. When they stood up, their blood pressure also went up more in these 33 people compared to those who didn't react strongly. We found that older people with this problem had higher levels of a hormone called cortisol, while younger people had lower levels of another hormone called growth hormone. These changes might help us diagnose the cause of dizziness and fainting in people with long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Orthostatic intolerance (OI) is a key symptom of long COVID; however, the pathophysiology remains unknown. Among 688 long COVID patients who visited our clinic during the period from February 2021 to April 2023, 86 patients who were suspected of having OI and who underwent an active standing test (ST) were investigated to elucidate the clinical characteristics of OI in patients with long COVID. Of the 86 patients, 33 patients (38%) were ST-positive. Nausea and tachycardia in daily life were frequent complaints in the ST-positive group. The increase in heart rate (HR) during the ST was significantly greater during a 10-min period after standing in the ST-positive group (+‚Äâ30¬†bpm) than in the ST-negative group (+‚Äâ16¬†bpm). The initial increase in diastolic blood pressure (DBP) just after standing was significantly greater in the ST-positive group (+‚Äâ14¬†mmHg) than in the ST-negative group (+‚Äâ9¬†mmHg). Serum cortisol levels in the ST-positive patients aged over 20¬†years were higher and growth hormone levels in the patients under 20¬†years of age were lower than those in the ST-negative group. Autonomous nervous symptoms, transient DBP rise with increasing HR after standing, and endocrine dysfunctions are helpful for detecting OI related to long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67815-y" target="_blank">https://www.nature.com/articles/s41598-024-67815-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41598-024-67570-0" target="_blank">
            ü§ñ Protecting Against Multiple COVID Variants with an Inactivated Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-24', this)">
            More</button></h4>
            <div id="toggle-24" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An inactivated SARS-CoV-2 vaccine based on a Vero cell culture-adapted high-titer virus confers cross-protection in small animals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic is still ongoing, and many people are getting less protected against severe illness as time goes on. To keep everyone safe, it's essential to have access to affordable vaccines globally. Researchers created a new version of the virus that causes COVID-19, which they adapted to grow in a special cell culture. This new virus grew much faster and produced 50 times more vaccine material than the original version. Despite this change, the new virus still allowed our immune system to recognize and fight it off just like the original one. Studies showed that vaccines made from this new virus were effective against different COVID-19 variants and even protected animals from getting sick when they were exposed to these viruses. This suggests that adapting the virus to grow in cell culture could be a useful way to produce more vaccine, making it easier for people around the world to get protected against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Rapidly waning immunity against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) requires continued global access to affordable vaccines. Globally, inactivated SARS-CoV-2 vaccines have been widely used during the SARS-CoV-2 pandemic. In this proof-of-concept study we adapted an original-D614G SARS-CoV-2 virus to Vero cell culture as a strategy to enhance inactivated vaccine manufacturing productivity. A passage 60 (P60) virus showed enhanced fitness and 50-fold increased virus yield in a bioreactor compared to the original-D614G virus. It further remained susceptible to neutralization by plasma from SARS-CoV-2 vaccinated and convalescent individuals, suggesting exposure of relevant epitopes. Monovalent inactivated P60 and bivalent inactivated P60/omicron BA.1 vaccines induced neutralizing responses against original-D614G and BA.1 viruses in mice and hamsters, demonstrating that the P60 virus is a suitable vaccine antigen. Antibodies further cross-neutralized delta and BA.5 viruses. Importantly, the inactivated P60 vaccine protected hamsters against disease upon challenge with original-D614G or BA.1 virus, with minimal lung pathology and lower virus loads in the upper and lower airways. Antigenicity of the P60 virus was thus retained compared to the original virus despite the acquisition of cell culture adaptive mutations. Consequently, cell culture adaptation may be a useful approach to increase yields in inactivated vaccine antigen production.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67570-0" target="_blank">https://www.nature.com/articles/s41598-024-67570-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">
            ü§ñ COVID-19 Vaccine Tested in Phase 3 Trial Shows Promise</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-25', this)">
            More</button></h4>
            <div id="toggle-25" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of the tetravalent protein COVID-19 vaccine, SCTV01E: a phase 3 double-blind, randomized, placebo-controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Evolution of COVID-19 virus mutations highlights the need for vaccines that can protect against multiple strains at once. A new vaccine, SCTV01E, has been developed to target four different versions of the SARS-CoV-2 virus: Alpha, Beta, Delta, and Omicron BA.1. In a large-scale study, 9,223 participants were randomly divided into two groups: one received SCTV01E, while the other got a placebo. The results showed that SCTV01E was effective in preventing COVID-19, with a success rate of 69.4%. It also provided strong protection against all SARS-CoV-2 infections (79.7% and 82.4% at 14 days post-vaccination). Additionally, SCTV01E elicited a significantly higher immune response against the Omicron BA.5 variant compared to the placebo group. The vaccine was generally well-tolerated, with no severe adverse events or deaths reported. These findings suggest that SCTV01E could be an effective vaccine alternative against current and future SARS-CoV-2 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Evolution of SARS-CoV-2 variants emphasizes the need for multivalent vaccines capable of simultaneously targeting multiple strains. SCTV01E is a tetravalent COVID-19 vaccine derived from the spike protein of SARS-CoV-2 variants Alpha, Beta, Delta, and Omicron BA.1. In this double-blinded placebo-controlled pivotal efficacy trial (NCT05308576), the primary endpoint was vaccine efficacy (VE) against COVID-19 seven days post-vaccination in individuals without recent infection. Other endpoints included evaluating safety, immunogenicity, and the VE against all SARS-CoV-2 infections in individuals meeting the study criteria. Between December 26, 2022, and January 15, 2023, 9,223 individuals were randomized at a 1:1 ratio to receive SCTV01E or a placebo. SCTV01E showed a VE of 69.4% (95% CI: 50.6, 81.0) 7 days post-vaccination, with 75 cases in the placebo group and 23 in the SCTV01E group for the primary endpoint. VEs were 79.7% (95% CI: 51.0, 91.6) and 82.4% (95% CI: 57.9, 92.6), respectively, for preventing symptomatic infection and all SARS-CoV-2 infections 14 days post-vaccination. SCTV01E elicited a 25.0-fold higher neutralizing antibody response against Omicron BA.5 28 days post-vaccination compared to placebo. Reactogenicity was generally mild and transient, with no reported vaccine-related SAE, adverse events of special interest (AESI), or deaths. The trial aligned with the shift from dominant variants BA.5 and BF.7 to XBB, suggesting SCTV01E as a potential vaccine alternative effective against present and future variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">https://www.nature.com/articles/s41467-024-49832-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">
            ü§ñ COVID-19's Airway Problem: Spike Protein Blocks Important Cell Signal</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-26', this)">
            More</button></h4>
            <div id="toggle-26" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inflammation in the COVID-19 airway is due to inhibition of CFTR signaling by the SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people with COVID-19, the virus causes an overactive immune response that can lead to sickness and death. This happens because the virus activates certain cells in the airways that normally help fight off infections. Researchers have also found that a similar overactive immune response is seen in people with cystic fibrosis, a genetic disease that affects the airways.

The researchers wanted to know if the COVID-19 virus might be causing inflammation in the airways of people with COVID-19 by blocking a protein called CFTR. They used an artificial model of normal airway cells and found that exposing these cells to the COVID-19 spike protein led to the loss of CFTR function.

The researchers also showed that blocking the interaction between the COVID-19 spike protein and another protein called ACE2 prevented the loss of CFTR function. Furthermore, they found that the COVID-19 virus itself caused a loss of CFTR protein in airway cells, but this was rescued by blocking the interaction between the spike protein and ACE2.

Overall, these findings suggest that inflammation in the airways of people with COVID-19 may be due to the inhibition of CFTR signaling by the COVID-19 spike protein, leading to a cystic fibrosis-like clinical phenotype.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2-contributes to sickness and death in COVID-19 patients partly by inducing a hyper-proinflammatory immune response in the host airway. This hyper-proinflammatory state involves activation of signaling by NFŒ∫B, and unexpectedly, ENaC, the epithelial sodium channel. Post-infection inflammation may also contribute to "Long COVID"/PASC. Enhanced signaling by NFŒ∫B and ENaC also marks the airway of patients suffering from cystic fibrosis, a life-limiting proinflammatory genetic disease due to inactivating mutations in the CFTR gene. We therefore hypothesized that inflammation in the COVID-19 airway might similarly be due to inhibition of CFTR signaling by SARS-CoV-2 spike protein, and therefore activation of both NFŒ∫B and ENaC signaling. We used western blot and electrophysiological techniques, and an organoid model of normal airway epithelia, differentiated on an air‚Äìliquid-interface (ALI). We found that CFTR protein expression and CFTR cAMP-activated chloride channel activity were lost when the model epithelium was exposed to SARS-CoV-2 spike proteins. As hypothesized, the absence of CFTR led to activation of both TNFŒ±/NFŒ∫B signaling and Œ± and Œ≥ ENaC. We had previously shown that the cardiac glycoside drugs digoxin, digitoxin and ouabain blocked interaction of spike protein and ACE2. Consistently, addition of 30¬†nM concentrations of the cardiac glycoside drugs, prevented loss of both CFTR protein and CFTR channel activity. ACE2 and CFTR were found to co-immunoprecipitate in both basal cells and differentiated epithelia. Thus spike-dependent CFTR loss might involve ACE2 as a bridge between Spike and CFTR. In addition, spike exposure to the epithelia resulted in failure of endosomal recycling to return CFTR to the plasma membrane. Thus, failure of CFTR recovery from endosomal recycling might be a mechanism for spike-dependent loss of CFTR. Finally, we found that authentic SARS-CoV-2 virus infection induced loss of CFTR protein, which was rescued by the cardiac glycoside drugs digitoxin and ouabain. Based on experiments with this organoid model of small airway epithelia, and comparisons with 16HBE14o- and other cell types expressing normal CFTR, we predict that inflammation in the COVID-19 airway may be mediated by inhibition of CFTR signaling by the SARS-CoV-2 spike protein, thus inducing a cystic fibrosis-like clinical phenotype. To our knowledge this is the first time COVID-19 airway inflammation has been experimentally traced in normal subjects to a contribution from SARS-CoV-2 spike-dependent inhibition of CFTR signaling.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">https://www.nature.com/articles/s41598-024-66473-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-67028-3" target="_blank">
            ü§ñ Testing Plasma Exchange for Severe COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-27', this)">
            More</button></h4>
            <div id="toggle-27" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A randomised controlled trial of plasma exchange compared to standard of care in the treatment of severe COVID-19 infection (COVIPLEX)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 is a serious illness that can cause the blood to become very "sticky" and prone to clotting. Our goal was to see if removing these "stickier" parts from the blood could help reduce this problem. We also wanted to know if it would make things better or worse for patients with severe COVID-19 who were having trouble breathing. We looked at 22 people who needed extra oxygen or a machine to help them breathe, and randomly assigned them either to get the special treatment (called plasma exchange) or to just get their usual care. After a few days, we found that the plasma exchange did reduce some of the bad stuff in the blood that can cause clotting, but it didn't seem to make things better for people who were having trouble breathing or getting more clots.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 disease is associated with a hyperinflammatory, pro-thrombotic state and a high mortality. Our primary objective was to assess the change in inflammatory and thrombotic markers associated with PEX, and secondary objectives were to assess the effects of PEX on progression of respiratory failure and incidence of acute thrombotic events. We conducted a prospective, phase II, non-blinded randomised control trial of plasma exchange compared to standard of care in critically ill adults with severe COVID-19 associated¬†respiratory failure, requiring supplemental oxygen or ventilatory support and elevated thrombo-inflammatory markers (LDH, CRP, ferritin, and D-Dimer). Patients randomised to receive PEX were treated with a daily single volume plasma exchange for a minimum of five days. Twenty-two patients were randomised of who 11 received PEX. Demographic and clinical characteristics were similar between groups at presentation. PEX was associated with a significant reduction in pro-thrombotic markers FVIII, VWF and VWF Ag: ADAMTS 13 ratio (p<‚Äâ0.001). There were no differences in the reduction of inflammatory markers, severity of respiratory failure (p=‚Äâ0.7), thrombotic events (p=‚Äâ0.67), or mortality (p>‚Äâ0.99) at 28¬†days. PEX successfully reduced pro-thrombotic markers, although was not associated with reduction in inflammatory markers, respiratory failure, or thrombotic events.Trial registration: (NCT04623255); first posted on 10/11/2020.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67028-3" target="_blank">https://www.nature.com/articles/s41598-024-67028-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41467-024-50566-9" target="_blank">
            ü§ñ Uncovering the Secrets of SARS-CoV-2's Papain-Like Protease: What Makes It Tick and How to Outsmart It</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-28', this)">
            More</button></h4>
            <div id="toggle-28" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mutational profiling of SARS-CoV-2 papain-like protease reveals requirements for function, structure, and drug escape</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine a special protein called Papain-like protease (PLpro) that helps the SARS-CoV-2 virus multiply inside cells. This protein is crucial for the virus's survival, and scientists are interested in it as a potential target for new medicines.

To better understand PLpro, researchers used a powerful tool to scan millions of tiny changes in its genetic code and see how these changes affect the protein's ability to function properly within human cells. They discovered that some parts of the protein are very sensitive to even small changes, while others can tolerate more significant modifications without losing their job.

The study also found that specific areas around the protein's active site (where it does its main work) and a flexible loop (that helps it move and interact with other molecules) play crucial roles in how well PLpro works. Furthermore, the researchers noticed that changes to one part of the protein can indirectly affect its overall activity by influencing how it interacts with other molecules.

Using this knowledge, scientists were able to identify new versions of PLpro that might be resistant to a common inhibitor (a type of drug that stops or slows down the protein's function). This discovery is important because it suggests that when designing drugs against PLpro, scientists should consider not only the active site but also other flexible parts of the protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Papain-like protease (PLpro) is an attractive drug target for SARS-CoV-2 because it is essential for viral replication, cleaving viral poly-proteins pp1a and pp1ab, and has de-ubiquitylation and de-ISGylation activities, affecting innate immune responses. We employ Deep Mutational Scanning to evaluate the mutational effects on PLpro enzymatic activity and protein stability in mammalian cells. We confirm features of the active site and identify mutations in neighboring residues that alter activity. We characterize residues responsible for substrate binding and demonstrate that although residues in the blocking loop are remarkably tolerant to mutation, blocking loop flexibility is important for function. We additionally find a connected network of mutations affecting activity that extends far from the active site. We leverage our library to identify drug-escape variants to a common PLpro inhibitor scaffold and predict that plasticity in both the S4 pocket and blocking loop sequence should be considered during the drug design process.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50566-9" target="_blank">https://www.nature.com/articles/s41467-024-50566-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41467-024-50133-2" target="_blank">
            ü§ñ Protecting Against Deadly Viruses: A New Vaccine Shows Promise</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-29', this)">
            More</button></h4>
            <div id="toggle-29" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-antigen intranasal vaccine protects against challenge with sarbecoviruses and prevents transmission in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Immunization programs that protect people from getting sick with SARS-CoV-2 have been shown to be effective, but they are not always good at preventing people from getting infected. Mucosal (surface-related) vaccines may provide better protection against the spread of the virus and could potentially work well even if new variants or related viruses emerge. Here, we tested a multi-component intranasal vaccine called NanoSTING-SN that targets different parts of SARS-CoV-2 and found it to be very effective in preventing the virus from replicating in both the lungs and nasal passages when challenged with the Delta variant. We also showed that this vaccine prevents transmission of another SARS-CoV-2 variant (Omicron) to unvaccinated animals. Furthermore, we demonstrated that NanoSTING-SN protects against SARS-CoV, a different virus related to SARS-CoV-2, by showing that immunized mice did not lose weight and all survived the infection. In non-human primates, the vaccine elicited long-lasting (up to 6 months) serum IgG responses and nasal wash IgA responses that could protect against other viruses like XBB1.5 (another SARS-CoV-2 variant), SARS-CoV, and MERS-CoV. This has two implications: (1) mucosal multi-component vaccines may help reduce the spread of respiratory viruses, and (2) eliciting immunity to multiple antigens could be beneficial in developing pan-sarbecovirus vaccines that protect against a wide range of related viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Immunization programs against SARS-CoV-2 with commercial intramuscular vaccines prevent disease but are less efficient in preventing infections. Mucosal vaccines can provide improved protection against transmission, ideally for different variants of concern (VOCs) and related sarbecoviruses. Here, we report a multi-antigen, intranasal vaccine, NanoSTING-SN (NanoSTING-Spike-Nucleocapsid), eliminates virus replication in both the lungs and the nostrils upon challenge with the pathogenic SARS-CoV-2 Delta VOC. We further demonstrate that NanoSTING-SN prevents transmission of the SARS-CoV-2¬†Omicron VOC (BA.5) to vaccine-na√Øve hamsters. To evaluate protection against other sarbecoviruses, we immunized mice with NanoSTING-SN. We showed that immunization affords protection against SARS-CoV, leading to protection from weight loss and 100% survival in mice. In non-human primates, animals immunized with NanoSTING-SN show durable serum IgG responses (6 months) and nasal wash IgA responses cross-reactive to SARS-CoV-2 (XBB1.5), SARS-CoV and MERS-CoV antigens. These observations have two implications: (1) mucosal multi-antigen vaccines present a pathway to reducing transmission of respiratory viruses, and (2) eliciting immunity against multiple antigens can be advantageous in engineering pan-sarbecovirus vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50133-2" target="_blank">https://www.nature.com/articles/s41467-024-50133-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67828-7" target="_blank">
            ü§ñ Tracking COVID-19 in Brazil: Understanding Vaccine Response to New Variant</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-30', this)">
            More</button></h4>
            <div id="toggle-30" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genomic surveillance and vaccine response to the dominant SARS-CoV-2 XBB lineage in Rio Grande do Sul</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has been challenging to control due to new types of viruses emerging, making it hard for vaccines and treatments to work effectively. One of these variants, called XBB, has shown that it can spread quickly without causing severe illness, but still poses a threat to people's health. In the state of Rio Grande do Sul in Brazil, researchers looked at 746 samples from COVID-19 patients between 2022 and 2023. They found that by March 2023, XBB was the most common variant, making up about 83% of all cases. Most people who got infected with XBB were able to recover quickly, but a small percentage (about 2%) died from the infection. Interestingly, even those who had received three vaccine doses and had previously been infected with COVID-19 could get reinfected with XBB, with about 59% of these cases occurring in people whose last vaccine dose was over a year ago. This might be because their immune system's protection had worn off. The researchers also identified 90 mutations within the XBB variant that were spread across its genome and included changes to the part of the virus that makes it hard for the body to fight back. This study provides important insights into how the XBB variant took over in this region, and continued monitoring of the virus's evolution is crucial for effective public health management.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has been marked by novel viral variants, posing challenges to global public health. Recombination, a viral evolution mechanism, is implicated in SARS-CoV-2's ongoing evolution. The XBB recombinant lineage, known for evading antibody-mediated immunity, exhibits higher transmissibility without increased disease severity. We investigated the prevalence and genomic features of XBB in SARS-CoV-2-positive cases in Rio Grande do Sul (RS), Brazil. We sequenced 357 samples from epidemiological weeks (EW) 47/2022 to 17/2023, and included 389 publicly available sequences. Clinical and epidemiological data were obtained from DATASUS, e-SUS, and SIVEP GRIPE (data recording systems of the Brazilian Ministry of Health). Of these, 143 were classified as XBB and 586 were other Omicron lineages. In March 2023 (EW 10), XBB became dominant, accounting for 83.3% of cases. 97.7% of XBB-infected patients successfully recovered from the infection, with a low mortality rate (2.3%). Even after receiving three vaccine doses and having been previously infected, 59.5% of the patients experienced reinfection with XBB. However, for 54% of the individuals, the interval between their XBB infection and the last vaccine dose exceeded one year, potentially leading to a decline in antibody levels. In addition, we identified 90 mutations in RS circulating XBB, spread throughout the genome, notably in the Spike protein region associated with immune resistance. This study provides insights into the dynamics and impact of a recombinant variant becoming predominant for the first time in the state. Continued surveillance of SARS-CoV-2 genomic evolution is crucial for effective public health management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67828-7" target="_blank">https://www.nature.com/articles/s41598-024-67828-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41392-024-01906-0" target="_blank">
            ü§ñ Vaccine for the Nose: Boosting Immunity with a Shot to the Sinuses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-31', this)">
            More</button></h4>
            <div id="toggle-31" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intranasal adenovirus-vectored Omicron vaccine induced nasal immunoglobulin A has superior neutralizing potency than serum antibodies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The upper part of our respiratory system is where COVID-19 starts to take hold in the body. A special type of antibody called secretory IgA (sIgA) has been shown to protect against breakthrough infections from new COVID-19 variants, like Omicron. We found that a vaccine given directly into the nose using a harmless virus as a delivery system can stimulate the production of robust sIgA in the nasal passages. This sIgA was mainly made up of two or more identical units and accounted for about 40% of all proteins in the mucus lining the nasal passage. We also detected a small amount of another type of antibody, IgG, but not IgM, IgD, or IgE. After rinsing the nose with a saline solution, sIgA levels in the nasal passage were quickly replenished within just a few hours. Comparing sIgA in the nasal passages to antibodies in the blood, we found that sIgA was up to 3-logs more effective at binding to and neutralizing new COVID-19 variants. In contrast, blood antibodies failed to neutralize two newly emerged variants, XBB and BA.2.86, but sIgA still retained its ability to neutralize these variants. Furthermore, we discovered that sIgA was better than IgG or IgA at blocking the spread of COVID-19 between cells and protecting mice from a challenge with the XBB variant. By estimating the percentage of spike-specific sIgA in nasal mucus collected one month after intranasal vaccination, we found it to be around 2.6-3.9%. Our study provides insights for developing nose-sprayed vaccines that can stimulate the production of sIgA and build an effective first-line defense against emerging COVID-19 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The upper respiratory tract is the initial site of SARS-CoV-2 infection. Nasal spike-specific secretory immunoglobulin A (sIgA) correlates with protection against Omicron breakthrough infection. We report that intranasal vaccination using human adenovirus serotype 5 (Ad5) vectored Omicron spike in people who previously vaccinated with ancestral vaccine could induce robust neutralizing sIgA in the nasal passage. Nasal sIgA was predominantly present in dimeric and multimeric forms and accounted for nearly 40% of total proteins in nasal mucosal lining fluids (NMLFs). A low-level IgG could also be detected in NMLFs but not IgM, IgD, and IgE. After a complete nasal wash, sIgA in the nasal passage could be replenished rapidly within a few hours. A comparison of purified paired serum IgA, serum IgG, and nasal sIgA from the same individuals showed that sIgA was up to 3-logs more potent than serum antibodies in binding to spikes and in neutralizing Omicron subvariants. Serum IgG and IgA failed to neutralize XBB and BA.2.86, while nasal sIgA retained potent neutralization against these newly emerged variants. Further analysis showed that sIgA was more effective than IgG or IgA in blocking spike-mediated cell-to-cell transmission and protecting hACE2 mice from XBB challenge. Using a sIgA monoclonal antibody as a reference, we estimated that the total nasal sIgA contains about 2.6‚Äì3.9% spike-specific sIgA in NMLFs collected approximately one month after intranasal vaccination. Our study provided insights for developing intranasal vaccines that can induce sIgA to build an effective and mutation-resistant first-line immune barrier against constantly emerging variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01906-0" target="_blank">https://www.nature.com/articles/s41392-024-01906-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67806-z" target="_blank">
            ü§ñ Hope in a Field: Reducing COVID-19 Lung Damage with Magnetism</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-32', this)">
            More</button></h4>
            <div id="toggle-32" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Using a static magnetic field to attenuate the severity in COVID-19-invaded lungs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Two key things that can affect how quickly the coronavirus makes people sick are: 1) how well the virus's spike protein fits onto a special receptor on human cells called ACE2, and 2) how flexible the outer membrane of those cells is. This study wanted to see if a strong magnetic field could change either of these two things. They tested it on a type of lung cell in a lab dish (called Calu-3 cells) and also on mice. The results showed that a strong magnetic field can make the spike protein bind less well to ACE2 receptors, and it can also make the outer membrane of lung cells more flexible. This might be helpful for finding new ways to treat people with COVID-19 without having to touch or insert anything into their lungs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Two important factors affecting the progress of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) are the S-protein binding function of ACE2 receptors and the membrane fluidity of host cells. This study aimed to evaluate the effect of static magnetic field (SMF) on S-protein/ACE2 binding and cellular membrane fluidity of lung cells, and was performed in vitro using a Calu-3 cell model and in vivo using an animal model. The ability of ACE2 receptors to bind to SARS-CoV-2 spike protein on host cell surfaces under SMF stimulation was evaluated using fluorescence images. Host lung cell membrane fluidity was tested using fluorescence polarization to determine the effects of SMF. Our results indicate that 0.4¬†T SMF can affect binding between S-protein and ACE2 receptors and increase Calu-3 cell membrane fluidity, and that SMF exposure attenuates LPS-induced alveolar wall thickening in mice. These results may be of value for developing future non-contact, non-invasive, and low side-effect treatments to reduce disease severity in COVID-19-invaded lungs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67806-z" target="_blank">https://www.nature.com/articles/s41598-024-67806-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67967-x" target="_blank">
            ü§ñ COVID-19 Vaccine vs No Vaccine: How It Affects Eye Nerve Health</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-33', this)">
            More</button></h4>
            <div id="toggle-33" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Corneal nerve fiber morphology following COVID-19 infection in vaccinated and non-vaccinated population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people who got vaccinated and then got infected with COVID-19, we found out that their nerves in the front part of their eye (called the cornea) were healthier than those in people who didn't get vaccinated but still got infected. We compared these two groups to a group of people who weren't sick with COVID-19. The good news is that the vaccinated group had stronger and longer nerves, which could help protect them from complications related to COVID-19. However, we also found out that people who didn't get vaccinated had more immune cells (called dendritic cells) in their eyes than healthy individuals. This might be a sign of how the body responds differently when it's not protected by vaccines. Our study suggests that getting vaccinated might help keep the nerves in your eye safe from damage during a COVID-19 infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To examine corneal subbasal nerve changes in patients who received vaccination against SARS-CoV-2 virus and underwent COVID-19 infection compared to infected non-vaccinated patients and healthy controls. Twenty-nine eyes of 29 vaccinated patients (mean age: 36.66‚Äâ¬±‚Äâ12.25¬†years) within six months after PCR or Ag test proven COVID-19 infection and twenty-eight eyes of 28 age-matched infected, non-vaccinated patients (mean age: 42.14‚Äâ¬±‚Äâ14.17¬†years) were enrolled. Twenty-five age-matched healthy individuals (mean age: 47.52‚Äâ¬±‚Äâ18.45¬†years) served as controls. In vivo confocal microscopy (Heidelberg Retina Tomograph II Rostock Cornea Module, Germany) was performed in each group. Corneal subbasal nerve plexus morphology and corneal dendritic cells (DC) were evaluated. Significantly higher corneal nerve fiber density (P‚Äâ<‚Äâ0.001), nerve branch density (P‚Äâ<‚Äâ0.001), nerve fiber length (P‚Äâ<‚Äâ0.001), total branch density (P‚Äâ=‚Äâ0.007), nerve fiber area (P‚Äâ=‚Äâ0.001) and fractal dimension (P‚Äâ<‚Äâ0.001) values were observed in vaccinated patients after COVID-19 infection compared to the non-vaccinated group. Significantly higher DC density was observed in the non-vaccinated group compared to the control group (P‚Äâ=‚Äâ0.05). There was a statistically significant difference in the size of mature DCs (P‚Äâ<‚Äâ0.0001) but the size of immature DCs did not differ significantly among the 3 groups (P‚Äâ=‚Äâ0.132). Our results suggest that SARS-CoV-2 vaccination may have a protective effect against the complications of COVID-19 disease on the corneal subbasal nerve fibers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67967-x" target="_blank">https://www.nature.com/articles/s41598-024-67967-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-19</strong> <a href="https://www.nature.com/articles/s41467-024-50487-7" target="_blank">
            ü§ñ Uncovering Hidden COVID Risks in Everyday Interactions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-34', this)">
            More</button></h4>
            <div id="toggle-34" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Estimating the contribution of setting-specific contacts to SARS-CoV-2 transmission using digital contact tracing data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Understanding how COVID-19 spreads in crowded areas**

Scientists studied how COVID-19 spread during an outbreak in Beijing, China in 2022. They looked at data from over 22,000 people who got sick and more than 220,000 of their contacts. They found that when people were in close contact with others, like in a workplace or home, they were more likely to get COVID-19 from each other.

However, the researchers also created a computer model to see what would happen if there weren't any control measures in place. They discovered that even when people were in close contact, only about 11% of new cases came from these interactions. This was the lowest percentage among all types of settings they studied.

These findings suggest that public health officials should focus on finding a balance between using digital contact tracing to track COVID-19 spread and the challenges of tracking infections within crowded areas like homes or workplaces.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While many countries employed digital contact tracing to contain the spread of SARS-CoV-2, the contribution of cospace-time interaction (i.e., individuals who shared the same space and time) to transmission and to super-spreading in the real world has seldom been systematically studied due to the lack of systematic sampling and testing of contacts. To address this issue, we utilized data from 2230 cases and 220,878 contacts with detailed epidemiological information during the Omicron outbreak in Beijing in 2022. We observed that contact number per day of tracing for individuals in dwelling, workplace, cospace-time interactions, and community settings could be described by gamma distribution with distinct parameters. Our findings revealed that 38% of traced transmissions occurred through cospace-time interactions whilst control measures were in place. However, using a mathematical model to incorporate contacts in different locations, we found that without control measures, cospace-time interactions contributed to only 11% (95%CI: 10%‚Äì12%) of transmissions and the super-spreading risk for this setting was 4% (95%CI: 3%‚Äì5%), both the lowest among all settings studied. These results suggest that public health measures should be optimized to achieve a balance between the benefits of digital contact tracing for cospace-time interactions and the challenges posed by contact tracing within the same setting.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50487-7" target="_blank">https://www.nature.com/articles/s41467-024-50487-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-19</strong> <a href="https://www.nature.com/articles/s41392-024-01898-x" target="_blank">
            ü§ñ Hold and Rebuild: Hybrid Immunity Against Omicron After Breakthrough Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-35', this)">
            More</button></h4>
            <div id="toggle-35" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Heterogeneous hybrid immunity against Omicron variant JN.1 at 11 months following breakthrough infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The impact of climate change on polar bears is a major concern. Their main source of food is seals, which they hunt from ice floes. As the Arctic sea ice melts due to warmer temperatures, the hunting grounds of polar bears are shrinking. This could lead to them starving and having fewer babies. However, some polar bears have adapted by eating more plants and berries when there's no seal meat available. But this doesn't provide enough energy for them to reproduce. A solution might be to create artificial sea ice in areas where it naturally occurs, allowing polar bears to hunt seals and maintain their populations. This could also help protect other Arctic animals that rely on the sea ice as well.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Dear Editor,</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01898-x" target="_blank">https://www.nature.com/articles/s41392-024-01898-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-18</strong> <a href="https://www.nature.com/articles/s41467-024-50234-y" target="_blank">
            ü§ñ Protecting Against Respiratory Viruses with a New Type of Nanoparticle Treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-36', this)">
            More</button></h4>
            <div id="toggle-36" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An intranasal nanoparticle STING agonist protects against respiratory viruses in animal models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Respiratory viral infections are very bad for people all around the world. Even though vaccines have been really successful, there's a problem: viruses can change quickly and become hard to fight off with just vaccines. That's why scientists want to find something that can help protect us from these infections right away. In this study, researchers created a special treatment called NanoSTING. It's like a tiny particle that helps our immune system fight off bad germs. They tested it in mice and rats and found out it was safe for them. Then, they gave NanoSTING to hamsters who were infected with different strains of the SARS-CoV-2 virus, and it helped protect them. Even when they shared a space with other hamsters who had the virus, the treated hamsters didn't get sick as easily. They also tested NanoSTING against the flu in mice and found that it worked there too. By looking at how NanoSTING works, scientists think it could be used to help protect people from many different types of respiratory viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory viral infections cause morbidity and mortality worldwide. Despite the success of vaccines, vaccination efficacy is weakened by the rapid emergence of viral variants with immunoevasive properties. The development of an off-the-shelf, effective, and safe therapy against respiratory viral infections is thus desirable. Here, we develop NanoSTING, a nanoparticle formulation of the endogenous STING agonist, 2‚Ä≤‚àí3‚Ä≤ cGAMP, to function as an immune activator and demonstrate its safety in mice and rats. A single intranasal dose of NanoSTING protects against pathogenic strains of SARS-CoV-2 (alpha and delta VOC) in hamsters. In transmission experiments, NanoSTING reduces the transmission of SARS-CoV-2 Omicron VOC to na√Øve hamsters. NanoSTING also protects against oseltamivir-sensitive and oseltamivir-resistant strains of influenza in mice. Mechanistically, NanoSTING upregulates locoregional interferon-dependent and interferon-independent pathways in mice, hamsters, as well as non-human primates. Our results thus implicate NanoSTING as a broad-spectrum immune activator for controlling respiratory virus infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50234-y" target="_blank">https://www.nature.com/articles/s41467-024-50234-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41541-024-00919-8" target="_blank">
            ü§ñ Boosting Immunity to COVID-19: A New Way to Fight the Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-37', this)">
            More</button></h4>
            <div id="toggle-37" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Homologous but not heterologous COVID-19 vaccine booster elicits IgG4+ B-cells and enhanced Omicron subvariant binding</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person could understand:

Imagine you've gotten vaccinated against COVID-19, but now there are new versions of the virus that your original vaccination might not protect you as well against. To stay safe, scientists have been testing different types of boosters to see if they can improve protection. In this study, researchers looked at how different types of booster vaccinations (ones made with a special kind of virus and ones made with mRNA) affected the immune system's response in people who had already received their initial vaccinations. They found that both types of boosters were effective in increasing the level of antibodies and "memory B cells" that can help fight off future infections. However, they also discovered some differences between the two types of boosters, which might be important for future vaccine development.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Booster vaccinations are recommended to improve protection against severe disease from SARS-CoV-2 infection. With primary vaccinations involving various adenoviral vector and mRNA-based formulations, it remains unclear if these differentially affect the immune response to booster doses. We examined the effects of homologous (mRNA/mRNA) and heterologous (adenoviral vector/mRNA) vaccination on antibody and memory B cell (Bmem) responses against ancestral and Omicron subvariants. Healthy adults who received primary BNT162b2 (mRNA) or ChAdOx1 (vector) vaccination were sampled 1-month and 6-months after their 2nd and 3rd dose (homologous or heterologous) vaccination. Recombinant spike receptor-binding domain (RBD) proteins from ancestral, Omicron BA.2 and BA.5 variants were produced for ELISA-based serology, and tetramerized for immunophenotyping of RBD-specific Bmem. Dose 3 boosters significantly increased ancestral RBD-specific plasma IgG and Bmem in both cohorts. Up to 80% of ancestral RBD-specific Bmem expressed IgG1+. IgG4+Bmem were detectable after primary mRNA vaccination, and expanded significantly to 5‚Äì20% after dose 3, whereas heterologous boosting did not elicit IgG4+Bmem. Recognition of Omicron BA.2 and BA.5 by ancestral RBD-specific plasma IgG increased from 20% to 60% after the 3rd dose in both cohorts. Reactivity of ancestral RBD-specific Bmem to Omicron BA.2 and BA.5 increased following a homologous booster from 40% to 60%, but not after a heterologous booster. A 3rd mRNA dose generates similarly robust serological and Bmem responses in homologous and heterologous vaccination groups. The expansion of IgG4+Bmem after mRNA priming might result from the unique vaccine formulation or dosing schedule affecting the Bmem response duration and antibody maturation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00919-8" target="_blank">https://www.nature.com/articles/s41541-024-00919-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41598-024-67314-0" target="_blank">
            ü§ñ Recovering From Covid: How Pre-Existing Respiratory Conditions Affect Lung Function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-38', this)">
            More</button></h4>
            <div id="toggle-38" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pulmonary function trajectories in COVID-19 survivors with and without pre-existing respiratory disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many people who have had COVID-19 still feel like they're not breathing as well as they should be, even 3 or 12 months after they've recovered from the illness. This study looked at how two groups of patients fared: those who already had a respiratory disease (like asthma or COPD) when they got COVID-19, and those who didn't have any underlying lung problems. The researchers did tests to see if these patients could breathe well, and what they found was that about 49% of the people without pre-existing respiratory disease still had trouble breathing 3 months after discharge, compared to about 64% of those with pre-existing respiratory disease. However, over time, the people without underlying lung problems started to breathe better (in fact, their results improved significantly), but the people who already had respiratory issues didn't get much better. Even 12 months later, a significant number of both groups still struggled to breathe as well as they should have been able to.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A significant proportion of COVID-19 survivors still experience a reduced diffusion capacity three and twelve months after discharge. We aimed to compare pulmonary function trajectories between hospitalized COVID-19 patients with pre-existing respiratory disease (PRD) and patients without pre-existing respiratory disease (Non-PRD) at three and twelve months after hospital discharge. This single-centre retrospective cohort study included COVID-19 patients admitted to the VieCuri Medical Centre (Venlo, the Netherlands) between February and December 2020 that were invited to the outpatient clinic at three and twelve¬†months after discharge. During this visit, pulmonary function tests were performed and impairments were based on lower limit of normal. Data of 239 patients were analysed (65% male, 66‚Äâ¬±‚Äâ10¬†years, and 26% with a history of respiratory disease). Three months after discharge, 49% and 64% of the Non-PRD patients (n‚Äâ=‚Äâ177) and PRD patients (n‚Äâ=‚Äâ62) had a low diffusion capacity, respectively. This improved over time in Non-PRD patients (p=‚Äâ0.003), but not in PRD patients (p=‚Äâ0.250). A low diffusion capacity was still observed in 34% and 57% of the Non-PRD and PRD group, respectively, twelve months after discharge. Pulmonary function impairments, mainly a reduced diffusion capacity, are observed among hospitalized COVID-19 patients with PRD and Non-PRD, at three and twelve months follow-up. Although diffusion capacity impairments restore over time in Non-PRD patients, poor recovery was observed among PRD patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67314-0" target="_blank">https://www.nature.com/articles/s41598-024-67314-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41598-024-66197-5" target="_blank">
            ü§ñ Uncovering the Link Between COVID-19 and Parkinson's Disease</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-39', this)">
            More</button></h4>
            <div id="toggle-39" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 and Parkinson‚Äôs disease: a single-center study and Mendelian randomization study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the rewritten abstract:

Imagine you have a neurological condition called Parkinson's disease (PD). Scientists wanted to find out if having COVID-19 makes your symptoms worse or affects your health differently than people without PD. They asked questions to over 130 people with PD at a single hospital between December 2022 and March 2023. The results showed that about two-thirds of the participants had COVID-19, and about one-third experienced worsening motor symptoms (such as tremors or stiffness) after getting infected.

The scientists also used a statistical method called Mendelian randomization to see if having PD increases the risk of getting COVID-19 in the first place. They found that people with PD were more likely to get COVID-19, but it didn't make their symptoms worse or increase their chances of being hospitalized. Overall, the study suggests that people with PD may experience worsened motor symptoms after contracting COVID-19, and longer disease duration and less frequent vaccination are risk factors for infection in these patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the association between COVID-19 and Parkinson‚Äôs disease (PD) via a single-center study and a Mendelian randomization (MR) study. A questionnaire-based survey was conducted among PD patients at a single center from December 7, 2022, to March 10, 2023. Logistic regression analysis was performed to identify the infection-related risk factors. Subsequently, bidirectional two-sample Mendelian randomization was employed to explore the association between COVID-19 and PD. In the cross-sectional analysis, it was found that the prevalence of COVID-19 infection in PD patients was 65.7%. Forty-eight (35.3%) PD patients experienced exacerbation of motor symptoms following COVID-19 infection. Long PD disease duration (‚â• 10 years) (OR: 3.327,P=‚Äâ0.045) and long time since last vaccination (>‚Äâ12¬†m) (OR: 4.916,P=‚Äâ0.035) were identified as significant risk factors related to infection. The MR analysis results supported that PD increases the COVID-19 susceptibility (Œ≤‚Äâ=‚Äâ0.081, OR‚Äâ=‚Äâ1.084,P=‚Äâ0.006). However, the MR analysis showed that PD did not increases the COVID-19 severity and hospitalization, and no significant association of COVID-19 on PD was observed. The findings from this cross-sectional study suggest that individuals with PD may experience worsened motor symptoms following COVID-19 infection. Long disease duration (‚â•10 years) and long time since last vaccination (>‚Äâ12¬†m) are identified as important risk factors for infection in these patients. Furthermore, our MR study provides evidence supporting an association between PD and COVID-19 susceptibility.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66197-5" target="_blank">https://www.nature.com/articles/s41598-024-66197-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41598-024-63212-7" target="_blank">
            ü§ñ Predicting Death and Pulmonary Embolisms for COVID Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-40', this)">
            More</button></h4>
            <div id="toggle-40" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>At-admission prediction of mortality and pulmonary embolism in an international cohort of hospitalised patients with COVID-19 using statistical and machine learning methods</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>By September 2022, more than 600 million people had gotten sick with COVID-19 worldwide, and over 6.5 million had died. However, the tools used to predict who might die from COVID-19 were often made with small groups of people that weren't very representative of the larger population. This made it harder to develop accurate predictive models. 

One of the most serious complications of COVID-19 is a condition called pulmonary embolism (PE), which can be life-threatening if not treated promptly. By using a large dataset of over 800,000 COVID-19 patients from around the world, we created a machine learning model that can predict whether someone has PE or will die from COVID-19 at admission to the hospital. 

Our model had good results when tested on a separate group of people, with an accuracy rate of about 75% for predicting PE and about 72% for predicting death due to any cause. We also looked at patients in the UK and Spain separately and got similar results.

The most important factors that our model used to predict PE were age, whether the patient was a man or woman, where they were admitted, and if they had certain health conditions like heart or lung disease, dementia, diabetes, high blood pressure, cancer, obesity, or smoked. 

Our analysis can help hospitals prioritize who needs immediate attention based on their risk level, and it can also inform long-term research and decision-making about how to care for COVID-19 patients globally.</p><br>
                <p><strong>Abstract</strong></p>
                <p>By September 2022, more than 600 million cases of SARS-CoV-2 infection have been reported globally, resulting in over 6.5 million deaths. COVID-19 mortality risk estimators are often, however, developed with small unrepresentative samples and with methodological limitations. It is highly important to develop predictive tools for pulmonary embolism (PE) in COVID-19 patients as one of the most severe preventable complications of COVID-19. Early recognition can help provide life-saving targeted anti-coagulation therapy right at admission. Using a dataset of more than 800,000 COVID-19 patients from an international cohort, we propose a cost-sensitive gradient-boosted machine learning model that predicts occurrence of PE and death at admission. Logistic regression, Cox proportional hazards models, and Shapley values were used to identify key predictors for PE and death. Our prediction model had a test AUROC of 75.9% and 74.2%, and sensitivities of 67.5% and 72.7% for PE and all-cause mortality respectively on a highly diverse and held-out test set. The PE prediction model was also evaluated on patients in UK and Spain separately with test results of 74.5% AUROC, 63.5% sensitivity and 78.9% AUROC, 95.7% sensitivity. Age, sex, region of admission, comorbidities (chronic cardiac and pulmonary disease, dementia, diabetes, hypertension, cancer, obesity, smoking), and symptoms (any, confusion, chest pain, fatigue, headache, fever, muscle or joint pain, shortness of breath) were the most important clinical predictors at admission. Age, overall presence of symptoms, shortness of breath, and hypertension were found to be key predictors for PE using our extreme gradient boosted model. This analysis based on the, until now, largest global dataset for this set of problems can inform hospital prioritisation policy and guide long term clinical research and decision-making for COVID-19 patients globally. Our machine learning model developed from an international cohort can serve to better regulate hospital risk prioritisation of at-risk patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-63212-7" target="_blank">https://www.nature.com/articles/s41598-024-63212-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s44298-024-00037-1" target="_blank">
            ü§ñ Jamaican Fruit Bats Resistant to COVID-19 Virus Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-41', this)">
            More</button></h4>
            <div id="toggle-41" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Jamaican fruit bat (Artibeus jamaicensis) insusceptibility to mucosal inoculation with SARS-CoV-2 Delta variant is not caused by receptor compatibility</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The original ancestor of the SARS-CoV-2 virus is thought to have come from bats. However, we know very little about how these viruses behave in bats themselves, or whether they can infect other animals besides humans. Some tests have shown that SARS-CoV-2 can infect rodents, monkeys, and a type of bat called Egyptian fruit bats. In this study, scientists tried using another type of bat, the Jamaican fruit bat, to see if it could be used as a model to study how SARS-CoV-2 behaves in its natural reservoir (bat population). They found that SARS-CoV-2 was unable to efficiently replicate in these bats and did not cause any significant immune response or shedding. This suggests that the Jamaican fruit bat is not a suitable model for studying SARS-CoV-2 infection in its natural reservoir.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ancestral sarbecovirus giving rise to SARS-CoV-2 is posited to have originated in bats. While SARS-CoV-2 causes asymptomatic to severe respiratory disease in humans, little is known about the biology, virus tropism, and immunity of SARS-CoV-2-like sarbecoviruses in bats. SARS-CoV-2 has been shown to infect multiple mammalian species, including various rodent species, non-human primates, and Egyptian fruit bats. We show that SARS-CoV-2 can utilize Jamaican fruit bat (Artibeus jamaicensis)ACE2 spike for entry in vitro. Therefore, we investigate the Jamaican fruit bat as a possible in vivo model to study reservoir responses. We find that SARS-CoV-2 Delta does not efficiently replicate in Jamaican fruit bats in vivo. We observe infectious viruses in the lungs of only one animal on day 1 post-inoculation and find no evidence of shedding or seroconversion. This is possibly due to host factors restricting virus egress after aborted replication. Furthermore, we observe no significant immune gene expression changes in the respiratory tract but do observe changes in the intestinal metabolome after inoculation. This suggests that, despite its broad host range, SARS-CoV-2 is unable to infect all bat species, and Jamaican fruit bats are not an appropriate model to study SARS-CoV-2 reservoir infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00037-1" target="_blank">https://www.nature.com/articles/s44298-024-00037-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41598-024-67536-2" target="_blank">
            ü§ñ COVID-19 Lung Function After Hospital Discharge</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-42', this)">
            More</button></h4>
            <div id="toggle-42" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-discharge spirometry evaluation in patients recovering from moderate-to-critical COVID-19: a cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who had severe cases of COVID-19 were studied to see how many of them still had lung problems after they got better. The study found that out of 408 people, about half (46.8%) still had abnormal lung function. Some had trouble breathing in and out, while others had lungs that were working fine but were not able to get enough air.

The researchers also looked for things that might be connected to these lung problems. They found that people who had fluid in their lungs when they were sick with COVID-19 (shown on a chest X-ray), or those who already had heart problems, or those who got tired easily during a simple walking test were more likely to have abnormal lung function.

This study shows that people who had severe cases of COVID-19 might still be having trouble with their lungs after they recover. The researchers suggest that these people should get regular tests to check how well their lungs are working.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the prevalence of abnormal lung function and its associated factors among patients recovering from COVID-19 is crucial for enhancing post-COVID care strategies. This study primarily aimed to determine the prevalence and types of spirometry abnormalities among post-COVID-19 patients in Malaysia, with a secondary objective of identifying its associated factors. Conducted at the COVID-19 Research Clinic, Faculty of Medicine, University Technology MARA, from March 2021 to December 2022, this study included patients at least three months post-discharge from hospitals following moderate-to-critical COVID-19. Of 408 patients studied, abnormal spirometry was found in 46.8%, with 28.4% exhibiting a restrictive pattern, 17.4% showing preserved ratio impaired spirometry (PRISm), and 1.0% displaying an obstructive pattern. Factors independently associated with abnormal spirometry included consolidation on chest X-ray (OR 8.1, 95% CI 1.75‚Äì37.42,p=‚Äâ0.008), underlying cardiovascular disease (OR 3.5, 95% CI 1.19‚Äì10.47,p=‚Äâ0.023), ground-glass opacity on chest X-ray (OR 2.6, 95% CI 1.52‚Äì4.30,p<‚Äâ0.001), and oxygen desaturation during the 6-min walk test (OR 1.9, 95% CI 1.20‚Äì3.06,p=‚Äâ0.007). This study highlights that patients recovering from moderate-to-critical COVID-19 often exhibit abnormal spirometry, notably a restrictive pattern and PRISm. Routine spirometry screening for high-risk patients is recommended.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67536-2" target="_blank">https://www.nature.com/articles/s41598-024-67536-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s42003-024-06529-3" target="_blank">
            ü§ñ Targeting COVID-19 with Precision</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-43', this)">
            More</button></h4>
            <div id="toggle-43" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Influence of AAV vector tropism on long-term expression and Fc-Œ≥ receptor binding of an antibody targeting SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Imagine a special vaccine that can protect people with weak immune systems from getting sick. Scientists created a new way to make this vaccine using a tiny virus called an AAV vector. They added a part of a powerful antibody that can fight COVID-19 to the virus, so it can help protect people who are vulnerable to the disease.

When they tested this vaccine in mice, they found that it worked very well and kept the levels of the protective antibody high for over a year. The best part is that it didn't cause any bad reactions or side effects in the mice.

The scientists also discovered that the type of virus used in the vaccine affected how well it worked and how long it lasted in the body. They found that using a specific type of AAV vector led to higher levels of the protective antibody and better protection against COVID-19 in mice.

Overall, this study suggests that using AAV vectors to deliver powerful antibodies like the one they used could be a safe and effective way to protect people with weak immune systems from getting sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-acting passive immunization strategies are needed to protect immunosuppressed vulnerable groups from infectious diseases. To further explore this concept for COVID-19, we constructed Adeno-associated viral (AAV) vectors encoding the human variable regions of the SARS-CoV-2 neutralizing antibody, TRES6, fused to murine constant regions. An optimized vector construct was packaged in hepatotropic (AAV8) or myotropic (AAVMYO) AAV capsids and injected intravenously into syngeneic TRIANNI-mice. The highest TRES6 serum concentrations (511‚Äâ¬µg/ml) were detected 24 weeks after injection of the myotropic vector particles and mean TRES6 serum concentrations remained above 100‚Äâ¬µg/ml for at least one year. Anti-drug antibodies or TRES6-specific T cells were not detectable. After injection of the AAV8 particles, vector mRNA was detected in the liver, while the AAVMYO particles led to high vector mRNA levels in the heart and skeletal muscle. The analysis of the Fc-glycosylation pattern of the TRES6 serum antibodies revealed critical differences between the capsids that coincided with different binding activities to murine Fc-Œ≥-receptors. Concomitantly, the vector-based immune prophylaxis led to protection against SARS-CoV-2 infection in K18-hACE2 mice. High and long-lasting expression levels, absence of anti-drug antibodies and favourable Fc-Œ≥-receptor binding activities warrant further exploration of myotropic AAV vector-based delivery of antibodies and other biologicals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06529-3" target="_blank">https://www.nature.com/articles/s42003-024-06529-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41541-024-00921-0" target="_blank">
            ü§ñ COVID-19 Vaccine Testing: A Standardized Approach</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-44', this)">
            More</button></h4>
            <div id="toggle-44" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Standardized immunological assays for assessing COVID-19 vaccines by the CEPI-Centralized Laboratory Network</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Imagine you're working in a big team of scientists who are all testing different things to help people with a serious virus called SARS-CoV-2. To make sure everyone is doing their tests correctly, we created a special system called the CEPI Centralized Laboratory Network. This system helped us share and control our tests across all the labs, making sure they were done in the same way everywhere.

This way, we can trust the results we get from each lab and compare them easily. It's like having a secret recipe that everyone follows to make sure the cake turns out the same every time! We're going to explain how this system works in more detail below.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The CEPI Centralized Laboratory Network implemented key steps in the transfer and monitoring of the developed immunological SARS-CoV-2 assays to ensure standardization across all the facilities of the network. This comprehensive evaluation reinforces the reliability of the generated data and establishes a solid foundation for a standardized approach, enabling precise inter-laboratory comparisons and contributing to the overall integrity of our network‚Äôs clinical results. Herein, we will provide a brief elaboration on the specific measures and procedures implemented to standardize the transfer of assays across our network.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00921-0" target="_blank">https://www.nature.com/articles/s41541-024-00921-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-12</strong> <a href="https://www.nature.com/articles/s41598-024-64094-5" target="_blank">
            ü§ñ COVID-19 Survivors with Mental Health Issues Have Higher Platelet Activation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-45', this)">
            More</button></h4>
            <div id="toggle-45" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preferential and sustained platelet activation in COVID-19 survivors with mental disorders</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person can understand:

People with mental health conditions like depression or anxiety are more likely to get very sick if they catch COVID-19. One reason for this might be that their blood vessels are already damaged, which makes it harder for them to fight off the virus.

Researchers wanted to see if people who had mental health conditions were also more likely to have problems with platelets in their blood after recovering from COVID-19. Platelets help blood to clot and can become overactive when someone is sick. The researchers looked at 141 people who had recovered from COVID-19 and found that those who had a history of mental health conditions were more likely to have platelet problems.

This suggests that people with mental health conditions might be more prone to severe COVID-19 symptoms because their blood is more likely to get overactive and cause inflammation. This could be an important clue for doctors trying to understand why some people are more vulnerable to COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pre-existing mental disorders are considered a risk factor for severe COVID-19 outcomes, possibly because of higher vascular burden. Moreover, an unconventional platelet activation characterizes COVID-19 and contributes to inflammatory and thrombotic manifestations. In the light of the inflammation theory of mental disorders, we hypothesized that patients with mental disorders could be sensitive to the SARS-CoV-2 elicited platelet activation. We investigated platelet activation in 141 COVID-19 survivors at one month after clearance of the virus, comparing subjects with or without an established pre-existing diagnosis of mental disorder according to the DSM-5. We found that platelets from patients with a positive history of psychiatric disorder underwent unconventional activation more frequently than conventional activation or no activation at all. Such preferential activation was not detected when platelets from patients without a previous psychiatric diagnosis were studied. When testing the effects of age, sex, and psychiatric history on the platelet activation, GLZM multivariate analysis confirmed the significant effect of diagnosis only. These findings suggest a preferential platelet activation during acute COVID-19 in patients with a pre-existing psychiatric disorder, mediated by mechanisms associated with thromboinflammation. This event could have contributed to the higher risk of severe outcome in the psychiatric population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64094-5" target="_blank">https://www.nature.com/articles/s41598-024-64094-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-12</strong> <a href="https://www.nature.com/articles/s41564-024-01755-1" target="_blank">
            ü§ñ Developing a Safe and Effective Live-Attenuated COVID Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-46', this)">
            More</button></h4>
            <div id="toggle-46" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A safe, effective and adaptable live-attenuated SARS-CoV-2 vaccine to reduce disease and transmission using one-to-stop genome modifications</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Modified live vaccines that are genetically similar to the COVID-19 virus but less harmful have been developed to provide broader protection against new variants of the virus. These modified vaccines, called OTS-206 and OTS-228, were designed to boost the body's immune system by removing certain parts of the original SARS-CoV-2 genome that could interfere with its effectiveness. They are also genetically stable, can be easily administered through the nose, and have an adjustable level of severity. In animal studies, a single dose of OTS-228 was found to provide complete protection against COVID-19 infection and also helped prevent transmission of the virus, including new variants such as Omicron. This approach could potentially be applied to other emerging viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Approved vaccines are effective against severe COVID-19, but broader immunity is needed against new variants and transmission. Therefore, we developed genome-modified live-attenuated vaccines (LAV) by recoding the SARS-CoV-2 genome, including ‚Äòone-to-stop‚Äô (OTS) codons, disabling Nsp1 translational repression and removing ORF6, 7ab and 8 to boost host immune responses, as well as the spike polybasic cleavage site to optimize the safety profile. The resulting OTS-modified SARS-CoV-2 LAVs, designated as OTS-206 and OTS-228, are genetically stable and can be intranasally administered, while being adjustable and sustainable regarding the level of attenuation. OTS-228 exhibits an optimal safety profile in preclinical animal models, with no side effects or detectable transmission. A single-dose vaccination induces a sterilizing immunity in vivo against homologous WT SARS-CoV-2 challenge infection and a broad protection against Omicron BA.2, BA.5 and XBB.1.5, with reduced transmission. Finally, this promising LAV approach could be applicable to other emerging viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-024-01755-1" target="_blank">https://www.nature.com/articles/s41564-024-01755-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00549-0" target="_blank">
            ü§ñ Uncovering Hidden Risks: Predicting Long COVID Outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-47', this)">
            More</button></h4>
            <div id="toggle-47" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identification of risk factors of Long COVID and predictive modeling in the RECOVER EHR cohorts</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background
People who got sick with SARS-CoV-2 might develop new health problems later on. These problems, called post-acute sequelae of SARS-CoV-2 infection (PASC) or Long COVID, can affect many different parts of the body. Not much research has looked into why some people get Long COVID and how to predict it.

Methods
This study used electronic medical records from two big healthcare networks in New York and Florida to look for factors that might increase the risk of getting Long COVID. The study included both people who had SARS-CoV-2 and those who didn't. It looked at data from March 2020 to November 2021.

Results
The study found complex relationships between certain health factors and different Long COVID conditions. It suggested that people who had severe SARS-CoV-2, were underweight, or had pre-existing health conditions (like cancer or liver disease) might be more likely to get Long COVID. Some Long COVID conditions, such as dementia, malnutrition, and heart failure, could be predicted fairly well. Others, like fatigue, anxiety, sleep disorders, and depression, were harder to predict.

Conclusions
This study found that the relationships between health factors and Long COVID are complicated. However, it suggests that using machine learning-based models might help identify people who are at risk of getting different Long COVID conditions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSARS-CoV-2-infected patients may develop new conditions in the period after the acute infection. These conditions, the post-acute sequelae of SARS-CoV-2 infection (PASC, or Long COVID), involve a diverse set of organ systems. Limited studies have investigated the predictability of Long COVID development and its associated risk factors.MethodsIn this retrospective cohort study, we used electronic healthcare records from two large-scale PCORnet clinical research networks, INSIGHT (~1.4 million patients from New York) and OneFlorida+ (~0.7 million patients from Florida), to identify factors associated with having Long COVID, and to develop machine learning-based models for predicting Long COVID development. Both SARS-CoV-2-infected and non-infected adults were analysed during the period of March 2020 to November 2021. Factors associated with Long COVID risk were identified by removing background associations and correcting for multiple tests.ResultsWe observed complex association patterns between baseline factors and a variety of Long COVID conditions, and we highlight that severe acute SARS-CoV-2 infection, being underweight, and having baseline comorbidities (e.g., cancer and cirrhosis) are likely associated with increased risk of developing Long COVID. Several Long COVID conditions, e.g., dementia, malnutrition, chronic obstructive pulmonary disease, heart failure, PASC diagnosis U099, and acute kidney failure are well predicted (C-index‚Äâ>‚Äâ0.8). Moderately predictable conditions include atelectasis, pulmonary embolism, diabetes, pulmonary fibrosis, and thromboembolic disease (C-index 0.7‚Äì0.8). Less predictable conditions include fatigue, anxiety, sleep disorders, and depression (C-index around 0.6).ConclusionsThis observational study suggests that association patterns between investigated factors and Long COVID are complex, and the predictability of different Long COVID conditions varies. However, machine learning-based predictive models can help in identifying patients who are at risk of developing a variety of Long COVID conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00549-0" target="_blank">https://www.nature.com/articles/s43856-024-00549-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s41598-024-66901-5" target="_blank">
            ü§ñ COVID-19 and Cesarean Section Complications</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-48', this)">
            More</button></h4>
            <div id="toggle-48" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of perioperative COVID-19 infection on postoperative complication in cesarean section using Korean National Health insurance data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The impact of COVID-19 on pregnancy has raised concerns about how it might affect the care that women receive during childbirth, specifically when they have a C-section. To investigate this, researchers analyzed data from South Korea's national health insurance system. They looked at two groups of patients who had C-sections: those who tested positive for COVID-19 and those who didn't. The study found that women with COVID-19 were more likely to experience complications like pneumonia, breathing problems, blood clots, and infections after their surgery. These women also stayed in the hospital longer and had higher medical bills than those without COVID-19. The researchers' analysis suggests that having COVID-19 during pregnancy can lead to more serious problems for women undergoing C-sections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The vulnerability during pregnancy has raised concerns about the potential impact of COVID-19 on obstetric anesthesia, an essential aspect of maternal care during cesarean section procedures. To evaluate the influence of COVID-19 infection on obstetric anesthesia during cesarean section, we analyzed the data from Korean National Health Insurance System (NHIS). This retrospective study utilized data from Korean NHIS. We included patients admitted with operation codes specific to cesarean section between January 1, 2020, and December 31, 2021. We classified patients into a COVID (+) group with a diagnosis code (U071) 30¬†days around surgery and a COVID (‚àí) group without the code in the same period. The primary outcome was 30-day mortality that was defined as death within 30¬†days of admission due to any causes. Secondary outcomes were pulmonary complications (pneumonia, acute respiratory distress syndrome [ARDS], pulmonary thromboembolism [PTE], or unexpected postoperative mechanical ventilation), ICU admission, cardiac arrest, myocardial infarction [MI], other thromboembolic events, surgical site infection, sepsis, acute renal failure [ARF], and hepatic failure. Among 75,268 patients who underwent cesarean section, 107 had a COVID-19 diagnosis code, while 75,161 did not. After 1:4 propensity score matching (PSM), 535 patients were included in each group. 30-day mortality showed no significant differences between the two groups both before and after PSM. The COVID (+) group demonstrated significantly elevated rates of pneumonia, ARDS, PTE, and surgical site infection both before and after PSM. Hospital length of stay and admission costs were also significantly longer and higher, respectively, in the COVID (+) group before and after PSM. In subgroup analysis, no differences were observed in mortality and postoperative complications based on the anesthesia method after matching. COVID-19 infection is associated with increased rates of postoperative complications, including pneumonia, ARDS, PTE, surgical site infection, longer hospital stays, and increased admission costs, in patients who underwent cesarean section.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66901-5" target="_blank">https://www.nature.com/articles/s41598-024-66901-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">
            ü§ñ Uncovering Patterns of Re-infection and Prolonged Illness in People with COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-49', this)">
            More</button></h4>
            <div id="toggle-49" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Insights from an N3C RECOVER EHR-based cohort study characterizing SARS-CoV-2 reinfections and Long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Many people have gotten COVID-19 twice, but scientists don't know much about what happens when this happens. We want to learn more about getting COVID-19 again and how it might affect some people who got it once before.

We looked at the health records of over 3 million people to understand this better. Our findings show that most people who got COVID-19 twice did so during a certain time period, and we found that some people got it multiple times.

Interestingly, our study shows that some people experienced more severe symptoms after their first infection than after getting it again. We also noticed that older people and those with severe initial infections were at higher risk of dying.

Overall, our research suggests that the severity of a second COVID-19 infection might be related to how severe the first one was, and that long-term health effects (known as Long COVID) seem more common after the initial infection than after getting it again. Further studies can build on these findings to help us understand COVID-19 reinfections better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundAlthough the COVID-19 pandemic has persisted for over 3 years, reinfections with SARS-CoV-2 are not well understood. We aim to characterize reinfection, understand development of Long COVID after reinfection, and compare severity of reinfection with initial infection.MethodsWe use an electronic health record study cohort of over 3 million patients from the National COVID Cohort Collaborative as part of the NIH Researching COVID to Enhance Recovery Initiative. We calculate summary statistics, effect sizes, and Kaplan‚ÄìMeier curves to better understand COVID-19 reinfections.ResultsHere we validate previous findings of reinfection incidence (6.9%), the occurrence of most reinfections during the Omicron epoch, and evidence of multiple reinfections. We present findings that the proportion of Long COVID diagnoses is higher following initial infection than reinfection for infections in the same epoch. We report lower albumin levels leading up to reinfection and a statistically significant association of severity between initial infection and reinfection (chi-squared value: 25,697,p-value: <0.0001) with a medium effect size (Cramer‚ÄôsV: 0.20, DoF‚Äâ=‚Äâ3). Individuals who experienced severe initial and first reinfection were older in age and at a higher mortality risk than those who had mild initial infection and reinfection.ConclusionsIn a large patient cohort, we find that the severity of reinfection appears to be associated with the severity of initial infection and that Long COVID diagnoses appear to occur more often following initial infection than reinfection in the same epoch. Future research may build on these findings to better understand COVID-19 reinfections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">https://www.nature.com/articles/s43856-024-00539-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-10</strong> <a href="https://www.nature.com/articles/s42003-024-06500-2" target="_blank">
            ü§ñ High-Throughput Testing for Broad Immune Response Against Many Antigens</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-50', this)">
            More</button></h4>
            <div id="toggle-50" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Two-dimensional high-throughput on-cell screening of immunoglobulins against broad antigen repertoires</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###
 
Imagine you want to find special helpers in your blood that can fight a specific bad guy. But, there are so few of these helpers around that it's hard to find them. To make things worse, we don't know what the bad guy looks like, which makes it harder to create good fighters against it. A team of scientists has come up with a clever way to find these special helpers and also figure out how they work. They use a system where they take tiny pieces of the bad guy (made by a bacteriophage - a virus that helps us) and attach them to tiny balls. Then, they mix this with blood from people who have recovered from the sickness or have been vaccinated. The special helpers in their blood will stick to the right parts of the bad guy on the ball, and then those helpers can be isolated and studied for potential use as medicines. This process is very efficient and allows many helpers to be tested quickly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Identifying high-affinity antibodies in human serum is challenging due to extremely low number of circulating B cells specific to the desired antigens. Delays caused by a lack of information on the immunogenic proteins of viral origin hamper the development of therapeutic antibodies. We propose an efficient approach allowing for enrichment of high-affinity antibodies against pathogen proteins with simultaneous epitope mapping, even in the absence of structural information about the pathogenic immunogens. To screen therapeutic antibodies from blood of recovered donors, only pathogen transcriptome is required to design an antigen polypeptide library, representing pathogen proteins, exposed on the bacteriophage surface. We developed a two-dimensional screening approach enriching lentiviral immunoglobulin libraries from the convalescent or vaccinated donors against bacteriophage library expressing the overlapping set of polypeptides covering the spike protein of SARS-CoV-2. This platform is suitable for pathogen-specific immunoglobulin enrichment and allows high-throughput selection of therapeutic human antibodies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06500-2" target="_blank">https://www.nature.com/articles/s42003-024-06500-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-10</strong> <a href="https://www.nature.com/articles/s41586-024-07671-y" target="_blank">
            ü§ñ Dendritic Cells Regulate Blood Platelet Production</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-51', this)">
            More</button></h4>
            <div id="toggle-51" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Plasmacytoid dendritic cells control homeostasis of megakaryopoiesis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that a young person could understand:

 Platelets are tiny cells in our blood that help us heal from cuts and fights off infections. Our body needs to make new platelets all the time, but we didn't know how it works. Scientists used special cameras to watch what happens inside our bone marrow, where platelets are made. They found that a type of cell called a plasmacytoid dendritic cell (pDC) acts like a sensor, detecting when platelet-making cells are dying and sending a signal to make more new platelets. This system helps keep the right number of platelets in our blood. But when pDCs get activated by viruses like COVID-19, they can't do their job properly and we end up with too many platelets. This study helps us understand how our body makes platelets and what happens when it gets out of balance.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Platelet homeostasis is essential for vascular integrity and immune defence1,2. Although the process of platelet formation by fragmenting megakaryocytes (MKs; thrombopoiesis) has been extensively studied, the cellular and molecular mechanisms required to constantly replenish the pool of MKs by their progenitor cells (megakaryopoiesis) remains unclear3,4. Here we use intravital imaging to track the cellular dynamics of megakaryopoiesis over days. We identify plasmacytoid dendritic cells (pDCs) as homeostatic sensors that monitor the bone marrow for apoptotic MKs and deliver IFNŒ± to the MK niche triggering local on-demand proliferation and maturation of MK progenitors. This pDC-dependent feedback loop is crucial for MK and platelet homeostasis at steady state and under stress. pDCs are best known for their ability to function as vigilant detectors of viral infection5. We show that virus-induced activation of pDCs interferes with their function as homeostatic sensors of megakaryopoiesis. Consequently, activation of pDCs by SARS-CoV-2 leads to excessive megakaryopoiesis. Together, we identify a pDC-dependent homeostatic circuit that involves innate immune sensing and demand-adapted release of inflammatory mediators to maintain homeostasis of the megakaryocytic lineage.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07671-y" target="_blank">https://www.nature.com/articles/s41586-024-07671-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">
            ü§ñ Developing Antibodies Against SARS-CoV-2: Identifying Targeting Molecules in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-52', this)">
            More</button></h4>
            <div id="toggle-52" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Selection and characterization of human scFvs targeting the SARS-CoV-2 nucleocapsid protein isolated from antibody libraries of COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

A new type of coronavirus called SARS-CoV-2 started spreading in China in 2019, causing an illness named COVID-19. At first, most scientists focused on studying the spike part of the virus, but they soon realized that another important part, called the nucleocapsid (N) protein, plays a key role in making the virus copy itself and hide from our immune system. The N protein also triggers a strong response from our immune system, making it an attractive target for developing antibodies to fight the virus. This study created a library of special antibody fragments from COVID-19 patients' cells to create a large collection of antibodies that could be used in research and possibly as tests or treatments. These fragments were then divided into two groups: those that strongly stuck to the N protein when produced inside yeast cells or mammalian cells, and others that were purified as separate proteins with strong affinity for the N protein. This set of antibodies against the N protein offers a new tool for scientists and could potentially be used in medical tests or treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In 2019, the novel SARS-CoV-2 coronavirus emerged in China, causing the pneumonia named COVID-19. At the beginning, all research efforts were focused on the spike (S) glycoprotein. However, it became evident that the nucleocapsid (N) protein is pivotal in viral replication, genome packaging and evasion of the immune system, is highly immunogenic, which makes it another compelling target for antibody development alongside the spike protein. This study focused on the construction of single chain fragments variable (scFvs) libraries from SARS-CoV-2-infected patients to establish a valuable, immortalized and extensive antibodies source. We used the Intracellular Antibody Capture Technology to select a panel of scFvs against the SARS-CoV-2 N protein. The whole panel of scFv was expressed and characterized both as intrabodies and recombinant proteins. ScFvs were then divided into 2 subgroups: those that exhibited high binding activity to N protein when expressed in yeast or in mammalian cells as intrabodies, and those purified as recombinant proteins, displaying affinity for recombinant N protein in the nanomolar range. This panel of scFvs against the N protein represents a novel platform for research and potential diagnostic applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">https://www.nature.com/articles/s41598-024-66558-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66410-5" target="_blank">
            ü§ñ Red Blood Cells Have a New Trick: They Can Detect Viruses Too</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-53', this)">
            More</button></h4>
            <div id="toggle-53" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Human red blood cells express the RNA sensor TLR7</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Red blood cells in our bodies have a special tool called TLR9 that helps them detect certain patterns in DNA. But can they also detect other types of genetic material? We discovered that human red blood cells actually have another tool called TLR7 that helps them recognize RNA, which is similar to DNA but carries instructions from viruses and other tiny life forms. This TLR7 is attached to a protein on the surface of red blood cells and is more active in people who are very sick with COVID-19 compared to healthy individuals. In experiments, we found that red blood cells can bind to synthetic RNA and even to RNA from certain viruses. This means that red blood cells might be taking up excess RNA from our bodies, performing another function beyond just carrying oxygen.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Red blood cells (RBCs) express the nucleic acid-binding toll-like receptor 9 (TLR9) and bind CpG-containing DNA. However, whether human RBCs express other nucleic acid-binding TLRs is unknown. Here we show that human RBCs express the RNA sensor TLR7. TLR7 is present on the red cell membrane and is associated with the RBC membrane protein Band 3. In patients with SARS-CoV2-associated sepsis, TLR7-Band 3 interactions in the RBC membrane are increased when compared with healthy controls. In vitro, RBCs bind synthetic ssRNA and RNA from ssRNA viruses. Thus, RBCs may serve as a previously unrecognized sink for exogenous RNA, expanding the repertoire of non-gas exchanging functions performed by RBCs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66410-5" target="_blank">https://www.nature.com/articles/s41598-024-66410-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s44298-024-00036-2" target="_blank">
            ü§ñ Uncovering a Key to COVID's Success: How the SARS-CoV-2 Virus Adapted to Humans</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-54', this)">
            More</button></h4>
            <div id="toggle-54" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evolution at Spike protein position 519 in SARS-CoV-2 facilitated adaptation to humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As COVID-19 continues to spread, scientists are trying to understand how the virus first emerged in humans. They're also working to develop new treatments that can fight the virus more effectively. To do this, they need to know which parts of the virus are most important for it to survive and multiply in humans. Recently, researchers have been searching for specific changes in the virus's genetic code that might have helped it adapt to human hosts. They found a particular change at position 519 in the Spike protein on the surface of the virus. This change, from an asparagine (N) to a histidine (H), is important because it helps the virus bind to a receptor called ACE2 in human cells. The researchers created a mutant version of the virus with the ancestral N519 and found that it didn't replicate well in human lung cells. They also discovered that this change made it harder for the virus to infect cells that express the human ACE2 receptor. This suggests that the evolutionary transition at position 519 was crucial for SARS-CoV-2's ability to survive and multiply in humans. Furthermore, this change might be a target for developing new treatments, such as small molecule inhibitors, that can specifically block the Spike protein's interaction with ACE2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As the COVID-19 pandemic enters its fourth year, the pursuit of identifying a progenitor virus to SARS-CoV-2 and understanding the mechanism of its emergence persists, albeit against the backdrop of intensified efforts to monitor the ongoing evolution of the virus and the influx of new mutations. Surprisingly, few residues hypothesized to be essential for SARS-CoV-2 emergence and adaptation to humans have been validated experimentally, despite the importance that these mutations could contribute to the development of effective antivirals. To remedy this, we searched for genomic regions in the SARS-CoV-2 genome that show evidence of past selection around residues unique to SARS-CoV-2 compared with closely related coronaviruses. In doing so, we identified a residue at position 519 in Spike within the receptor binding domain that holds a static histidine in human-derived SARS-CoV-2 sequences but an asparagine in SARS-related coronaviruses from bats and pangolins. In experimental validation, the SARS-CoV-2 Spike protein mutant carrying the putatively ancestral H519N substitution showed reduced replication in human lung cells, suggesting that the histidine residue contributes to viral fitness in the human host. Structural analyses revealed a potential role of Spike residue 519 in mediating conformational transitions necessary for Spike prior to binding with ACE2. Pseudotyped viruses bearing the putatively ancestral N519 also demonstrated significantly reduced infectivity in cells expressing the human ACE2 receptor compared to H519. ELISA data corroborated that H519 enhances Spike binding affinity to the human ACE2 receptor compared to the putatively ancestral N519. Collectively, these findings suggest that the evolutionary transition at position 519 of the Spike protein played a critical role in SARS-CoV-2 emergence and adaptation to the human host. Additionally, this residue presents as a potential drug target for designing small molecule inhibitors tailored to this site.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00036-2" target="_blank">https://www.nature.com/articles/s44298-024-00036-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66891-4" target="_blank">
            ü§ñ Warning Signs of a Rare COVID-19 Complication in Children</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-55', this)">
            More</button></h4>
            <div id="toggle-55" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Severity predictors for multisystemic inflammatory syndrome in children after SARS-CoV-2 infection in Vietnam</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

A serious condition called Multisystemic Inflammatory Syndrome in Children (MIS-C) can cause different levels of illness in kids, ranging from mild symptoms to life-threatening problems. However, scientists don't fully understand what makes some cases more severe than others. This study aimed to find out which factors are linked to more serious outcomes by using a special statistical method called multivariate regression. The researchers looked at data from 391 children who were hospitalized in Vietnam between January and June 2023. They found that certain markers, such as high levels of a protein called CRP (meaning the child is likely to be very sick), low levels of albumin (which could indicate organ problems), low absolute lymphocyte count (a sign of weakened immune system), high ferritin levels (indicating inflammation), and low left ventricular ejection fraction (LVEF) (meaning heart problems), were strongly connected to severe cases. The study also showed that these factors increased the risk of shock, a life-threatening condition. In conclusion, this research suggests that monitoring absolute lymphocyte count, serum albumin, CRP levels, and LVEF can help identify which children with MIS-C are at higher risk for severe complications. More studies are needed to confirm if using these markers can improve treatment outcomes in kids with MIS-C.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multisystemic inflammatory syndrome in children (MIS-C) might manifest in a broad spectrum of clinical scenarios, ranging from mild features to multi-organ dysfunction and mortality. However, this novel entity has a heterogenicity of data regarding prognostic factors associated with severe outcomes. The present study aimed to identify independent predictors for severity by using multivariate regression models. A total of 391 patients (255 boys and 136 girls) were admitted to Vietnam National Children‚Äôs Hospital from January 2022 to June 2023. The median age was 85 (range: 2‚Äì188) months, and only 12 (3.1%) patients had comorbidities. 161 (41.2%) patients required PICU admission, and the median PICU LOS was 4 (2‚Äì7) days.¬†We observed independent factors related to PICU¬†admission, including CRP\(\ge \)50 (mg/L) (OR 2.52, 95% CI 1.39‚Äì4.56, p‚Äâ=‚Äâ0.002),¬†albumin\(\le \)30 (g/L) (OR 3.18, 95% CI 1.63‚Äì6.02, p‚Äâ=‚Äâ0.001), absolute lymphocyte count\(\le \)2 (√ó‚Äâ109/L) (OR 2.18, 95% CI 1.29‚Äì3.71, p‚Äâ=‚Äâ0.004), ferritin‚Äâ‚â•‚Äâ300 (ng/mL) (OR 2.35, 95% CI 1.38‚Äì4.01), p‚Äâ=‚Äâ0.002), and LVEF‚Äâ<‚Äâ60 (%) (OR 2.48, 95% CI 1.28‚Äì4.78, p‚Äâ=‚Äâ0.007). Shock developed in 140 (35.8%) patients, especially for those decreased absolute lymphocyte\(\le \)2 (√ó‚Äâ109/L) (OR 2.48, 95% CI 1.10‚Äì5.61, p‚Äâ=‚Äâ0.029), albumin\(\le \)30 (g/L) (OR 2.53, 95% CI 1.22‚Äì5.24, p‚Äâ=‚Äâ0.013), or LVEF‚Äâ<‚Äâ60 (%) (OR 2.24, 95% CI 1.12‚Äì4.51, p‚Äâ=‚Äâ0.022). In conclusion, our study emphasized that absolute lymphocyte count, serum albumin, CRP, and LVEF were independent predictors for MIS-C severity. Further well-designed investigations are required to validate their efficacy in predicting MIS-C severe cases, especially compared to other parameters. As MIS-C is a new entity and severe courses may progress aggressively, identifying high-risk patients optimizes clinicians' follow-up and management to improve disease outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66891-4" target="_blank">https://www.nature.com/articles/s41598-024-66891-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-08</strong> <a href="https://www.nature.com/articles/s43856-024-00506-x" target="_blank">
            ü§ñ Multiple Health Conditions Linked to Serious and Long-Term COVID Effects</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-56', this)">
            More</button></h4>
            <div id="toggle-56" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Complex patterns of multimorbidity associated with severe COVID-19 and long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

When people get very sick with COVID-19, it's often those who already have other health problems that are at highest risk. To make sure we had enough hospital beds and vaccines during the pandemic, we looked into studies about this. But there aren't many detailed investigations on what kinds of medical conditions could lead to severe COVID-19 in people who were thought to be less likely to get very sick.

### METHODS ###

We took a large database from over 500,000 UK Biobank participants and grouped their health records into almost 15,000 categories. This was done to identify any diseases that might make someone more prone to severe COVID-19 or its lingering effects, Long COVID.

### RESULTS ###

We found about 679 diseases that are linked to a higher risk of getting very sick with COVID-19 (or experiencing Long COVID). These conditions cover almost all areas of medical specialties and are particularly common among cardio-respiratory and endocrine-renal diseases. For 57 of these diseases, we confirmed their connection to severe COVID-19 based on how likely people were to survive and genetic factors that might predispose them. Notably, symptoms like feeling unwell and fatigue could also be risk factors for getting very sick with COVID-19. Interestingly, some diseases related by cause, such as post-inflammatory lung scarring or rheumatoid arthritis, showed differing risks for severe COVID-19, possibly indicating separate mechanisms of how these conditions affect the body.

### CONCLUSIONS ###

Our findings provide a unique reference that shows how having multiple health issues can increase the risk of getting very sick with COVID-19 and how considering all types of medical diagnoses can help understand genetic risk factors.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundEarly evidence that patients with (multiple) pre-existing diseases are at highest risk for severe COVID-19 has been instrumental in the pandemic to allocate critical care resources and later vaccination schemes. However, systematic studies exploring the breadth of medical diagnoses are scarce but may help to understand severe COVID-19 among patients at supposedly low risk.MethodsWe systematically harmonized >12 million primary care and hospitalisation health records from ~500,000 UK Biobank participants into 1448 collated disease terms to systematically identify diseases predisposing to severe COVID-19 (requiring hospitalisation or death) and its post-acute sequalae, Long COVID.ResultsHere we identify 679 diseases associated with an increased risk for severe COVID-19 (n=‚Äâ672) and/or Long COVID (n=‚Äâ72) that span almost all clinical specialties and are strongly enriched in clusters of cardio-respiratory and endocrine-renal diseases. For 57 diseases, we establish consistent evidence to predispose to severe COVID-19 based on survival and genetic susceptibility analyses. This includes a possible role of symptoms of malaise and fatigue as a so far largely overlooked risk factor for severe COVID-19. We finally observe partially opposing risk estimates at known risk loci for severe COVID-19 for etiologically related diseases, such as post-inflammatory pulmonary fibrosis or rheumatoid arthritis, possibly indicating a segregation of disease mechanisms.ConclusionsOur results provide a unique reference that demonstrates how 1) complex co-occurrence of multiple ‚Äì including non-fatal ‚Äì conditions predispose to increased COVID-19 severity and 2) how incorporating the whole breadth of medical diagnosis can guide the interpretation of genetic risk loci.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00506-x" target="_blank">https://www.nature.com/articles/s43856-024-00506-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-66079-w" target="_blank">
            ü§ñ Long COVID: How Metal Perception Affects the Tongue</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-57', this)">
            More</button></h4>
            <div id="toggle-57" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impaired metal perception and regulation of associated human foliate papillae tongue transcriptome in long-COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People with COVID-19 often have trouble tasting or smelling things. We wondered if this is because of changes in the tiny bumps on their tongues called papillae. We asked hospital workers who had either already had COVID-19 and still had antibodies (IgG+) or hadn't (IgG-) to fill out a questionnaire about how they were sensing things like flavors and smells. Some of them also did a test where they had to choose which flavor they thought was being given to them, and others gave us tiny pieces of their tongues with the papillae for testing. We found that people who still had COVID-19 antibodies and couldn't taste metal flavors had even higher levels of those antibodies than people who could taste them. When we looked at the genes in the papillae from people who couldn't smell or taste well, we saw that they had trouble with a lot of genes related to smelling things. This makes us wonder if some of these genes might be helping us sense metal flavors and if they might be affected by COVID-19. We need more research to figure this out.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Chemosensory impairment is an outstanding symptom of SARS-CoV-2 infections. We hypothesized that measured sensory impairments are accompanied by transcriptomic changes in the foliate papillae area of the tongue. Hospital personnel with known SARS-CoV-2 immunoglobulin G (IgG) status completed questionnaires on sensory perception (n=‚Äâ158). A subcohort ofn=‚Äâ141 participated in forced choice taste tests, andn=‚Äâ43 participants consented to donate tongue swabs of the foliate papillae area for whole transcriptome analysis. The study included four groups of participants differing in IgG levels (‚â•‚Äâ10 AU/mL‚Äâ=‚ÄâIgG+;‚Äâ<‚Äâ10 AU/mL‚Äâ=‚ÄâIgG-) and self-reported sensory impairment (SSI¬±). IgG+subjects not detecting metallic taste had higher IgG+levels than IgG+participants detecting iron gluconate (p=‚Äâ0.03). Smell perception was the most impaired biological process in the transcriptome data from IgG+/SSI+participants subjected to gene ontology enrichment. IgG+/SSI+subjects demonstrated lower expression levels of 166 olfactory receptors (OR) and 9 taste associated receptors (TAS) of which OR1A2, OR2J2, OR1A1, OR5K1 and OR1G1, as well as TAS2R7 are linked to metallic perception. The question raised by this study is whether odorant receptors on the tongue (i) might play a role in metal sensation, and (ii) are potential targets for virus-initiated sensory impairments, which needs to be investigated in future functional studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66079-w" target="_blank">https://www.nature.com/articles/s41598-024-66079-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-65038-9" target="_blank">
            ü§ñ Hope in a Vial: Biomolecules in Hymenopteran Venom as COVID-19 Therapies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-58', this)">
            More</button></h4>
            <div id="toggle-58" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Potential therapeutic biomolecules of hymenopteran venom against SARS-CoV-2 from Egyptian patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The bioactive molecules found in the venom of certain insects might be able to help fight COVID-19. Researchers studied the venom from honeybees and Asian wasps, and found that it has potential therapeutic effects against SARS-CoV-2. The wasp venom performed better than the bee venom, reducing the virus by 90% at a concentration of 0.184 milligrams per milliliter. However, further studies are needed to confirm these results and explore their potential as antiviral treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The therapeutic potential of insect-derived bioactive molecules as anti-SARS-CoV-2 agents has shown promising results. Hymenopteran venoms, notably fromApis mellifera(honeybee) andVespa orientalis(oriental wasp), were examined for the first time in an in vitro setting for their potential anti-COVID-19 activity. This assessment utilized an immunodiagnostic system to detect the SARS-CoV-2 nucleocapsid antigen titer reduction. Further analyses, including cytotoxicity assays, plaque reduction assays, and in silico docking-based screening, were performed to evaluate the efficacy of the most potent venom. Results indicated that bee and wasp venoms contain bioactive molecules with potential therapeutic effects against SARS-CoV-2.Nevertheless, the wasp venom exhibited superior efficacy compared to bee venom, achieving a 90% maximal (EC90) concentration effect of antigen depletion at 0.184¬†mg/mL, in contrast to 2.23¬†mg/mL for bee venom. The cytotoxicity of the wasp venom was assessed on Vero E6 cells 48¬†h post-treatment using the MTT assay. The CC50of the cell growth was 0.16617¬†mg/mL for Vero E6 cells. The plaque reduction assay of wasp venom revealed 50% inhibition (IC50) at a 0.208¬†mg/mL concentration. The viral count at 50% inhibition was 2.5‚Äâ√ó‚Äâ104PFU/mL compared to the initial viral count of 5‚Äâ√ó‚Äâ104PFU/mL. In silico data for the wasp venom revealed a strong attraction to binding sites on the ACE2 protein, indicating ideal interactions. This substantiates the potential of wasp venom as a promising viral inhibitor against SARS-CoV-2, suggesting its consideration as a prospective natural preventive and curative antiviral drug. In conclusion, hymenopteran venoms, particularly wasp venom, hold promise as a source of potential therapeutic biomolecules against SARS-CoV-2. More research and clinical trials are needed to evaluate these results and investigate their potential for translation into innovative antiviral therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65038-9" target="_blank">https://www.nature.com/articles/s41598-024-65038-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-66003-2" target="_blank">
            ü§ñ A Protein That Fights COVID-19 Infection in Human Airway Tissues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-59', this)">
            More</button></h4>
            <div id="toggle-59" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>NF-Œ∫B inhibitor alpha controls SARS-CoV-2 infection in ACE2-overexpressing human airway organoids</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As COVID-19 continues to spread, scientists need ways to study how it interacts with human airway cells. We've developed a 3D model of the airway epithelium using adult stem cells that can replicate the virus and maintain its natural structure and cell diversity. This model was infected with different COVID-19 variants and subjected to single-cell RNA sequencing, which showed that when cells were only slightly infected, they produced high levels of interferon-lambda, a protein that helps fight infections. In more heavily infected cells, a gene called IkBa was consistently upregulated, which helped regulate the infection but also allowed a protein called NF-kB to move into the cell's nucleus. When scientists overexpressed a mutant form of IkBa that couldn't be broken down, it reduced NF-kB translocation and increased viral infection, demonstrating the importance of this feedback loop in controlling viral replication.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As SARS-CoV-2 continues to spread worldwide, tractable primary airway cell models that recapitulate the cell-intrinsic response to arising viral variants are needed. Here we describe an adult stem cell-derived human airway organoid model overexpressing the ACE2 receptor (ACE2-OE) that supports robust viral replication while maintaining 3D architecture and cellular diversity of the airway epithelium. ACE2-OE organoids were infected with SARS-CoV-2 variants and subjected to single-cell RNA-sequencing. Interferon-lambda was¬†upregulated in cells with low-level infection while the NF-kB inhibitor alpha gene (encoding IkBa) was consistently upregulated in infected cells, and its expression positively correlated with infection levels. Confocal microscopy showed more IkBa expression in infected than bystander cells, but found concurrent nuclear translocation of NF-kB that IkBa usually prevents. Overexpressing a nondegradable IkBa mutant reduced NF-kB translocation and increased viral infection. These data demonstrate the functionality of ACE2-OE organoids in SARS-CoV-2 research and underscore that the strength of the NF-kB feedback loop in infected cells controls viral replication.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66003-2" target="_blank">https://www.nature.com/articles/s41598-024-66003-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-03</strong> <a href="https://www.nature.com/articles/s41467-024-49415-6" target="_blank">
            ü§ñ Mapping Viral Pathways: A New Way to Understand How Viruses Infect Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-60', this)">
            More</button></h4>
            <div id="toggle-60" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Recreating the biological steps of viral infection on a cell-free bioelectronic platform to profile viral variants of concern</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus is constantly changing and some of these changes can make it harder for us to detect and track the virus. To stay one step ahead, scientists need tools that can not only identify the presence of the virus but also predict how well it can infect cells. Our new electronic sensing platform simulates how the virus enters host cells and reports its progress as electrical signals. We focused on two key steps in this process: the virus attaching to the cell membrane and then fusing with it. These two steps can be distinguished using our platform, which allows us to identify specific patterns in the virus's behavior. Using this tool, we were able to see that even small changes in the virus could affect its ability to infect cells, and we hope that our platform will help scientists track and understand other viruses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Viral mutations frequently outpace technologies used to detect harmful variants. Given the continual emergence of SARS-CoV-2 variants, platforms that can identify the presence of a virus and its propensity for infection are needed. Our electronic biomembrane sensing platform recreates distinct SARS-CoV-2 host cell entry pathways and reports the progression of entry as electrical signals. We focus on two necessary entry processes mediated by the viral Spike protein: virus binding and membrane fusion, which can be distinguished electrically. We find that closely related variants of concern exhibit distinct fusion signatures that correlate with trends in cell-based infectivity assays, allowing us to report quantitative differences in their fusion characteristics and hence their infectivity potentials. We use SARS-CoV-2 as our prototype, but we anticipate that this platform can extend to other enveloped viruses and cell lines to quantifiably assess virus entry.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49415-6" target="_blank">https://www.nature.com/articles/s41467-024-49415-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-03</strong> <a href="https://www.nature.com/articles/s41467-024-49443-2" target="_blank">
            ü§ñ Vaccines May Protect Against Multiple COVID Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-61', this)">
            More</button></h4>
            <div id="toggle-61" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Three SARS-CoV-2 spike protein variants delivered intranasally by measles and mumps vaccines are broadly protective</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As new versions of the COVID-19 virus keep emerging, scientists are racing to create vaccines that can protect us from all these different variants. Here, researchers have developed a new type of vaccine that can be given through the nose and is effective against many different versions of the virus at once. They created this vaccine by combining three parts: one from the measles vaccine, one from the mumps vaccine, and a new part that specifically targets COVID-19. When tested in mice and hamsters, this new vaccine generated strong immune responses and protected them from infection with different strains of the virus. The good news is that people who already have immunity from other viruses like measles and mumps don't need to worry - it won't interfere with the effectiveness of this new COVID-19 vaccine. This new vaccine platform has a lot of potential as a next-generation COVID-19 vaccine candidate.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As the new SARS-CoV-2 Omicron variants and subvariants emerge, there is an urgency to develop intranasal, broadly protective vaccines. Here, we developed highly efficacious, intranasal trivalent SARS-CoV-2 vaccine candidates (TVC) based on three components of the MMR vaccine: measles virus (MeV), mumps virus (MuV) Jeryl Lynn (JL1) strain, and MuV JL2 strain. Specifically, MeV, MuV-JL1, and MuV-JL2 vaccine strains, each expressing prefusion spike (preS-6P) from a different variant of concern (VoC), were combined to generate TVCs. Intranasal immunization of IFNAR1‚àí/‚àímice and female hamsters with TVCs generated high levels of S-specific serum IgG antibodies, broad neutralizing antibodies, and mucosal IgA antibodies as well as tissue-resident memory T cells in the lungs. The immunized female hamsters were protected from challenge with SARS-CoV-2 original WA1, B.1.617.2, and B.1.1.529 strains. The preexisting MeV and MuV immunity does not significantly interfere with the efficacy of TVC. Thus, the trivalent platform is a promising next-generation SARS-CoV-2 vaccine candidate.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49443-2" target="_blank">https://www.nature.com/articles/s41467-024-49443-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-01</strong> <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">
            ü§ñ Contraceptive Access During COVID-19 Pandemic in Brazil</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-62', this)">
            More</button></h4>
            <div id="toggle-62" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A particular epidemiological profile: disparities in access to contraceptive methods in Brazil during the SARS-CoV-2 (COVID-19) pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, some people in Brazil had trouble getting access to birth control. We did a study to see who was affected most. We asked 1,018 people questions about their background and how they got birth control. Most of them were women between 20 and 39 years old and were white. Some people changed the way they got birth control during this time, and others wanted to try new methods. However, many had trouble finding a doctor who could help them with these changes or with continuing their current birth control method. The biggest problem was getting an appointment with a healthcare provider. Our study found that some groups of people, specifically younger black and biracial women with lower education and income, faced greater difficulties in accessing birth control and family planning services during this time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our objective was to study disparities in access to contraception during the COVID-19 pandemic. We performed a cross-sectional study at the University of Campinas, Brazil using a Google questionnaire applied from December 2021 until February 2022, disseminated via snowball technique. The survey asked about sociodemographic characteristics and contraceptive use, as well as the demand for new methods and difficulties in continuing to use contraceptives during the COVID-19 pandemic. We analyzed 1018 completed questionnaires; in total, 742 (72.9%) were women aged between 20 and 39¬†years, 746 (73.3%) were White and 602 (59.2%) used contraceptives. During the COVID-19 pandemic, about 23% of respondents changed their method and approximately 20% of respondents looked for new methods. Among the latter, 31.3% reported some difficulty with obtaining guidance on new methods while only 5.3% of the respondents reported some difficulty with continuing their contraceptive. The main difficulty in both cases was the difficulty with getting a healthcare provider appointment. Our results point to a particular epidemiological population, of younger black and biracial women, with lower education and lower income, which suffered health disparities during the COVID-19 pandemic and found difficulties with using contraceptives and accessing family planning services.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">https://www.nature.com/articles/s41598-024-65946-w</a></p>
                
                <br><br>
            </div>
        </div>
        
</p>
</div>
<br><br><br><br><br><br><br><br>
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
      <!-- end footer -->
      <!-- Javascript files--> 
      <script src="js/jquery.min.js"></script> 
      <script src="js/popper.min.js"></script> 
      <script src="js/bootstrap.bundle.min.js"></script> 
      <script src="js/jquery-3.0.0.min.js"></script> 
      <script src="js/plugin.js"></script> 
      <!-- sidebar --> 
      <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
      <script src="js/custom.js"></script>
      <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
      <script>
         $(document).ready(function(){
         $(".fancybox").fancybox({
         openEffect: "none",
         closeEffect: "none"
         });
         
         $(".zoom").hover(function(){
         
         $(this).addClass('transition');
         }, function(){
         
         $(this).removeClass('transition');
         });
         });
         
      </script> 
   </body>
</html>
<!-- end footer -->
<script>
function toggleAbstract(id, button) {
    const content = document.getElementById(id);
    if (content.style.display === "none") {
        content.style.display = "block";
        button.textContent = "Less";
    } else {
        content.style.display = "none";
        button.textContent = "More";
    }
}
document.addEventListener("DOMContentLoaded", function() {
    const buttons = document.querySelectorAll(".toggle-button");
    buttons.forEach(button => {
        const contentId = button.getAttribute("onclick").match(/'([^']+)'/)[1];
        const content = document.getElementById(contentId);
        if (content) {
            content.style.display = "none";
        }
    });
});
</script>
</body>
</html>
