
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Research News</title>
    <link rel="icon" href="images/logo.png" type="image/x-icon">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/responsive.css">
    <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
    <style>
        @media (max-width: 767px) {
            .menu-area-main {
                flex-direction: column;
            }
            .menu-area-main li {
                text-align: center;
            }
        }
        .toggle-content { display: none; }
        .toggle-button {
            background-color: #007bff;
            color: white;
            border: none;
            padding: 4px 12px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 14px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
        }
    </style>
</head>
<body class="main-layout">
<header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo">
                                <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a>
                            </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main d-flex">
                                    <li><a href="index.html">Geo Map</a></li>
                                    <li><a href="trends.html">Trend Graphs</a></li>
                                    <li><a href="predictions_tab.html">Predictive</a></li>
                                    <li class="active"><a href="new_research.html">Research News</a></li>
                                    <li><a href="data_sources.html">Data Sources</a></li>
                                    <li><a href="about.html">About</a></li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>
<div class="brand_color">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="titlepage">
                    <h2>Research News</h2>
                    <br>
                    <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                    Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>
                </div>
            </div>
        </div>
    </div>
</div>
<!-- contact -->
<p>
<br><br><br>

        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-31</strong> <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">
            ü§ñ Scientists Discover New Way COVID Virus Evades Immune System, Leading to Emergence of New Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-0', this)">
            More</button></h4>
            <div id="toggle-0" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibodies utilizing VL6-57 light chains target a convergent cryptic epitope on SARS-CoV-2 spike protein and potentially drive the genesis of Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have been studying how the virus that causes COVID-19, called SARS-CoV-2, changes over time and how our immune system responds to these changes. They found a group of antibodies, which are like tiny soldiers that fight off the virus, that are able to recognize and attack the virus in many people who haven't been infected before. These antibodies work together with the immune system's "memory" to help protect us from getting sick again. However, they also discovered that some new variants of SARS-CoV-2 have developed a way to evade these antibodies by making small changes to their structure. This suggests that our immune system is constantly adapting and putting pressure on the virus to change, which in turn drives the evolution of new variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Continued evolution of SARS-CoV-2 generates variants to challenge antibody immunity established by infection and vaccination. A connection between population immunity and genesis of virus variants has long been suggested but its molecular basis remains poorly understood. Here, we identify a class of SARS-CoV-2 neutralizing public antibodies defined by their shared usage of VL6-57 light chains. Although heavy chains of diverse genotypes are utilized, convergent HCDR3 rearrangements have been observed among these public antibodies to cooperate with germline VL6-57 LCDRs to target a convergent epitope defined by RBD residues S371-S373-S375. Antibody repertoire analysis identifies that this class of VL6-57 antibodies is present in SARS-CoV-2-naive individuals and is clonally expanded in most COVID-19 patients. We confirm that Omicron-specific substitutions at S371, S373 and S375 mediate escape of antibodies of the VL6-57 class. These findings support that this class of public antibodies constitutes a potential immune pressure promoting the introduction of S371L/F-S373P-S375F in Omicron variants. The results provide further molecular evidence to support that antigenic evolution of SARS-CoV-2 is driven by antibody mediated population immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">https://www.nature.com/articles/s41467-024-51770-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">
            ü§ñ Identifying Signs of Long-Term COVID-19 Health Effects Early On</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-1', this)">
            More</button></h4>
            <div id="toggle-1" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Early biological markers of post-acute sequelae of SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an explanation of how our bodies respond to SARS-CoV-2 infection that can lead to long-term health problems.

Scientists studied 136 people who just found out they had COVID-19 and were interested in knowing why some people got sick a second time, even after their initial symptoms went away. These participants collected nasal swab samples from themselves over the first month after their symptoms started, gave blood samples at different times, and answered questions about how they felt.

The researchers looked for signs that could show if someone was more likely to get PASC (a long-term health problem related to COVID-19). They compared two groups: people who fully recovered from COVID-19 with those who didn't. The results showed that people who got sick a second time had higher levels of the virus in their bodies and took longer to develop antibodies against the virus, but they didn't have any differences in their immune system markers.

These findings suggest that early viral activity and how our bodies respond to it might be important for understanding why some people get PASC.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To understand the roles of acute-phase viral dynamics and host immune responses in post-acute sequelae of SARS-CoV-2 infection (PASC), we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR test. Participants self-collected up to 21 nasal specimens within the first 28 days post-symptom onset; interviewer-administered questionnaires and blood samples were collected at enrollment, days 9, 14, 21, 28, and month 4 and 8 post-symptom onset. Defining PASC as the presence of any COVID-associated symptom at their 4-month visit, we compared viral markers (quantity and duration of nasal¬†viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-Œ±, IFN-Œ±, IFN-Œ≥, MCP, IP-10, and Spike IgG) over the acute period. Compared to those who fully recovered, those reporting PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA and N-antigen, burden of RNA and infectious viral shedding, and lower Spike-specific IgG levels within 9 days post-illness onset. No significant differences were identified among a panel of host immune markers. Our results suggest early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC, highlighting the importance of understanding early biological markers in the natural history of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">https://www.nature.com/articles/s41467-024-51893-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">
            ü§ñ How Safe is the mRNA-1273 Vaccine for Adults Over Time?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-2', this)">
            More</button></h4>
            <div id="toggle-2" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term safety and effectiveness of mRNA-1273 vaccine in adults: COVE trial open-label and booster phases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Primary vaccination with a new type of vaccine called mRNA-1273 was shown to be safe and very effective in preventing the spread of COVID-19 among adults across 99 sites in the US. This study also looked at how well an additional dose, or booster shot, worked after several months. The results showed that getting a booster shot significantly reduced the risk of catching COVID-19, especially during outbreaks caused by new variants like Omicron. The longer people waited between their initial vaccination and the booster shot, the lower their chance was of getting COVID-19. This study also found that the booster shot helped create a stronger immune response against COVID-19 than the initial vaccine, which is good news for protecting against future outbreaks. Overall, the mRNA-1273 vaccine showed promise in keeping people safe from COVID-19 and its variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary vaccination with mRNA-1273 (100-¬µg) was safe and efficacious at preventing coronavirus disease 2019 (COVID-19) in the previously reported, blinded Part A of the phase 3 Coronavirus Efficacy (COVE; NCT04470427) trial in adults (‚â•18 years) across 99 U.S. sites. The open-label (Parts B and C) primary objectives were evaluation of long-term safety and effectiveness of primary vaccination plus a 50-¬µg booster dose; immunogenicity was a secondary objective. Of 29,035 open-label participants, 19,609 received boosters (mRNA-1273 [n=‚Äâ9647]; placebo-mRNA-1273 [n=‚Äâ9952]; placebo [n=‚Äâ10] groups). Booster safety was consistent with that reported for primary vaccination. Incidences of COVID-19 and severe COVID-19 were higher during the Omicron BA.1 than Delta variant waves and boosting versus non-boosting was associated with a significant, 47.0% (95% CI : 39.0-53.9%) reduction of Omicron BA.1 incidence (24.6 [23.4‚Äâ‚àí‚Äâ25.8] vs 46.4 [40.6‚Äâ‚àí‚Äâ52.7]/1000 person-months). In an exploratory Cox regression model adjusted for time-varying covariates, a longer median interval between primary vaccination and boosting (mRNA-1273 [13 months] vs placebo-mRNA-1273 [8 months]) was associated with significantly lower, COVID-19 risk (24.0% [16.0%‚Äâ‚àí‚Äâ32.0%]) during Omicron BA.1 predominance. Boosting elicited greater immune responses against SARS-CoV-2 than primary vaccination, irrespective of prior SARS-CoV-2 infection. Primary vaccination and boosting with mRNA-1273 demonstrated acceptable safety, effectiveness and immunogenicity against COVID-19, including emergent variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">https://www.nature.com/articles/s41467-024-50376-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-28</strong> <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">
            ü§ñ Clotting Protein Plays Key Role in COVID-19 Symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-3', this)">
            More</button></h4>
            <div id="toggle-3" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Fibrin drives thromboinflammation and neuropathology in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had severe COVID-19 often experience life-threatening blood clots and neurological symptoms that can persist even after they recover. Despite this, we still don't fully understand how these complications occur or what treatment options are available. In people who have had severe COVID-19, a protein called fibrinogen is found in high levels in the lungs and brain, which correlates with the severity of their condition and can predict long-term cognitive problems.

Our research shows that fibrin - a component of blood clots - binds to the SARS-CoV-2 virus, forming pro-inflammatory blood clots that drive systemic inflammation and neurological damage. We found that fibrin promotes oxidative stress, activates macrophages in the lungs, suppresses natural killer cells, and contributes to neuroinflammation and neuronal loss after infection.

We also discovered that a monoclonal antibody targeting the inflammatory domain of fibrin can protect against microglial activation, neuronal injury, and thromboinflammation in the lung after infection. This suggests that fibrin-targeting immunotherapy may be a potential therapeutic approach for patients with acute COVID-19 and long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Life-threatening thrombotic events and neurological symptoms are prevalent in COVID-19 and are¬†persistent in patients with long COVID experiencing post-acute sequelae of SARS-CoV-2 infection1,2,3,4. Despite the clinical evidence1,5,6,7, the underlying mechanisms of coagulopathy in COVID-19 and its consequences in inflammation and neuropathology remain poorly understood and treatment options are insufficient. Fibrinogen, the central structural component of blood clots, is abundantly deposited in the lungs and brains of patients with COVID-19, correlates with disease severity and is a predictive biomarker for post-COVID-19 cognitive deficits1,5,8,9,10. Here we show that fibrin binds to the SARS-CoV-2 spike protein, forming proinflammatory blood clots that drive systemic thromboinflammation and neuropathology in COVID-19. Fibrin, acting through its inflammatory domain, is required for oxidative stress and macrophage activation in the lungs, whereas it¬†suppresses natural killer cells, after SARS-CoV-2 infection. Fibrin promotes neuroinflammation and neuronal loss after infection, as well as innate immune activation in the brain and lungs independently of active infection. A¬†monoclonal antibody targeting the inflammatory fibrin domain provides¬†protection from microglial activation and neuronal injury, as well as from thromboinflammation in the lung after infection. Thus, fibrin drives inflammation and neuropathology in SARS-CoV-2 infection, and fibrin-targeting immunotherapy may represent a therapeutic intervention for patients with acute COVID-19 and long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">https://www.nature.com/articles/s41586-024-07873-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">
            ü§ñ Understanding How Our Immune System Responds to Different COVID-19 Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-4', this)">
            More</button></h4>
            <div id="toggle-4" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibody Fc receptor binding and T cell responses to homologous and heterologous immunization with inactivated or mRNA vaccines against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the abstract rewritten for a young person:

The two COVID-19 vaccines being used, CoronaVac (C) and BNT162b2 (B), are good at protecting against severe illness but have different strengths when it comes to fighting off the virus. While they're both effective in preventing serious cases of COVID-19, one vaccine is better than the other at making antibodies that can neutralize the virus.

To find out more about how these vaccines work and which ones might be more effective for different people, researchers conducted a study with over 3,000 participants who received either the same vaccine (homologous) or a different type of vaccine (heterologous) as their booster shot. The results showed that:

* Participants who got the BNT162b2 booster shot had higher levels of antibodies against the virus and its spike protein compared to those who got the CoronaVac booster.
* Those who received the CoronaVac vaccine as their first dose followed by a homologous (same type) booster had higher levels of antibodies against a specific part of the virus called nucleocapsid.
* Participants who got the BNT162b2 vaccine initially and then boosted with either vaccine saw an increase in their T cell response, which is important for fighting off infections.

The study found that these differences might be due to how the vaccines stimulate the immune system to produce antibodies and T cells. This could explain why some people who received CoronaVac had lower levels of neutralizing antibodies but were still protected against severe illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Whole virion inactivated vaccine CoronaVac (C) and Spike (S) mRNA BNT162b2 (B) vaccines differ greatly in their ability to elicit neutralizing antibodies but have somewhat comparable effectiveness in protecting from severe COVID-19. We conducted further analyses for a randomized trial (Cobovax study, NCT05057169) of third dose homologous and heterologous booster vaccination, i.e. four interventions CC-C, CC-B, BB-C and BB-B. Here, we assess vaccine immunogenicity beyond neutralizing function, including S and non-S antibodies with Fc receptor (FcR) binding, antibody avidity and T cell specificity to 6 months post-vaccination. Ancestral and Omicron S-specific IgG and FcR binding are significantly higher by BNT162b2 booster than CoronaVac, regardless of first doses. Nucleocapsid (N) antibodies are only increased in homologous boosted CoronaVac participants (CC-C). CoronaVac primed participants have lower baseline S-specific CD4+IFNŒ≥+cells, but are significantly increased by either CoronaVac or BNT162b2 boosters. Priming vaccine content defined T cell peptide specificity preference, with S-specific T cells dominating B primed groups and non-S structural peptides contributing more in C primed groups, regardless of booster type. S-specific CD4+T cell responses, N-specific antibodies, and antibody effector functions via Fc receptor binding may contribute to protection and compensate for less potent neutralizing responses in CoronaVac recipients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">https://www.nature.com/articles/s41467-024-51427-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-26</strong> <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">
            ü§ñ Protection from COVID-19 with a Single Spray</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-5', this)">
            More</button></h4>
            <div id="toggle-5" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A single-dose intranasal live-attenuated codon deoptimized vaccine provides broad protection against SARS-CoV-2 and its variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's how severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus that causes COVID-19, continues to affect people and economies worldwide. While vaccines that target a specific part of the virus called the spike protein have been very effective in preventing serious illness, they don't always protect against new variants or prevent mild infections from spreading. We created an experimental vaccine, CDO-7N-1, by making small changes to the genetic code of the virus. When tested on mice, hamsters, and monkeys, this vaccine showed promise: it didn't cause much harm in the lungs, but still triggered a strong immune response that could fight off the virus. Most importantly, animals who received this vaccine were protected from getting sick with COVID-19, even when exposed to different variants of the virus. This is good news because it means that immunity provided by CDO-7N-1 may be more effective against future SARS-CoV-2 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, COVID-19) continues its significant health and economic impact globally. Despite the success of spike-protein vaccines in preventing severe disease, long-lasting protection against emerging variants and the prevention of breakthrough infections and transmission remain elusive. We generate an intranasal live-attenuated SARS-CoV-2 vaccine, CDO-7N-1, using codon deoptimization. CDO-7N-1 shows highly attenuated replication and minimal or no lung pathology in vivo over multiple passages. It induces robust mucosal and systemic neutralizing antibody and T-cell subset responses, in mice (female K18-hACE2 and male HFH4-hACE2 mice), hamsters, and macaques triggered by a single immunization. Mice and hamsters vaccinated with CDO-7N-1 are protected from challenge with wild-type (WT) SARS-CoV-2 and other variants of concern. Serum from vaccinated animals neutralizes WT SARS-CoV-2, variants of concern (beta and delta), variants of interest (omicron XBB.1.5) and SARS-CoV-1. Antibody responses are sustained and enhanced by repeated immunization or infection with WT SARS-CoV-2. Immunity against all SARS-CoV-2 proteins by CDO-7N-1 should improve efficacy against future SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">https://www.nature.com/articles/s41467-024-51535-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-22</strong> <a href="https://www.nature.com/articles/s41598-024-70172-5" target="_blank">
            ü§ñ Uncovering Key Cannabinoids in COVID-19 Infection and Their Impact on Inflammation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-6', this)">
            More</button></h4>
            <div id="toggle-6" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Circulating endocannabinoid levels in SARS-CoV-2 infection and their potential role in the inflammatory response</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The levels of certain helpful chemicals called endocannabinoids are different in people who have COVID-19, especially when they're first getting sick and after they've recovered. Scientists looked at the blood of 64 people to see how these chemicals were behaving. They found that one type of endocannabinoid was higher than usual in people with COVID-19, which might help explain some of the symptoms they experience. In contrast, two other types of endocannabinoids were lower in people who had recently recovered from COVID-19. This study suggests that these chemicals could play a role in how our bodies respond to and recover from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Plasma levels of endocannabinoids (eCBs) are very dynamic and variable in different circumstances and pathologies. The aim of the study was to determine the levels of the main eCBs andN-acylethanolamines (NAEs) in COVID-19 patients during the acute and post-acute phase of SARS-CoV-2 infection. Samples collected before December 31, 2020 were used for the determination of circulating eCB levels by liquid chromatography-tandem mass spectrometry (LC‚ÄìMS/MS). The association between plasma eCB measurements and biochemical and hematological parameters, as well as serum IL-6 levels, was evaluated.¬†Samples of 64 individuals were analysed, n‚Äâ=‚Äâ18 healthy donors, n‚Äâ=‚Äâ30 acute, and n‚Äâ=‚Äâ16 post-acute patients.¬†Plasma levels of 2-arachidonoylglycerol (2-AG), were significantly elevated in COVID-19 patients when compared to healthy individuals. PlasmaN-palmitoylethanolamide (PEA) andN-arachidonoylethanolamide (AEA) levels were found to be decreased in post-acute patient samples. These results suggest that 2-AG plays an important role in the inflammatory cascade in COVID-19 disease; in addition, eCBs might be involved in the post-acute pathogenesis of COVID-19. This study provides evidence of altered levels of circulating eCBs as a consequence of SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70172-5" target="_blank">https://www.nature.com/articles/s41598-024-70172-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-20</strong> <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">
            ü§ñ Long COVID in Monkeys: Unlocking the Immune Secrets</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-7', this)">
            More</button></h4>
            <div id="toggle-7" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Non-human primate model of long-COVID identifies immune associates of hyperglycemia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Hyperglycemia, or high blood sugar, is a problem that some people experience after having COVID-19. It's not well understood because there haven't been many animal studies on it. In this study, scientists used African green monkeys to create a model of what happens in humans after COVID-19. They found that these monkeys had a mix of chemical signals in their blood that was related to high blood sugar during and after the infection. The monkeys also had low levels of sugar stored in their livers, which contributed to the high blood sugar. However, the scientists didn't find any long-term viral activity in the monkeys' livers or pancreas. Interestingly, giving the monkeys a COVID-19 vaccine made with mRNA helped regulate their blood sugar levels. Overall, this study suggests that African green monkeys can be a useful model for studying and developing treatments for metabolic problems related to COVID-19 in humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hyperglycemia, and exacerbation of pre-existing deficits in glucose metabolism, are manifestations of the post-acute sequelae of SARS-CoV-2. Our understanding of metabolic decline after acute COVID-19 remains unclear due to the lack of animal models. Here, we report a non-human primate model of metabolic post-acute sequelae of SARS-CoV-2 using SARS-CoV-2 infected African green monkeys. Using this model, we identify a dysregulated blood chemokine signature during acute COVID-19 that correlates with elevated and persistent hyperglycemia four months post-infection. Hyperglycemia also correlates with liver glycogen levels, but there is no evidence of substantial long-term SARS-CoV-2 replication in the liver and pancreas. Finally, we report a favorable glycemic effect of the SARS-CoV-2 mRNA vaccine, administered on day 4 post-infection. Together, these data suggest that the African green monkey model exhibits important similarities to humans and can be utilized to assess therapeutic candidates to combat COVID-related metabolic defects.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">https://www.nature.com/articles/s41467-024-50339-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-14</strong> <a href="https://www.nature.com/articles/s41587-024-02346-5" target="_blank">
            ü§ñ Fast Discovery of Targeted Antibodies with a Microfluidic Tool</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-8', this)">
            More</button></h4>
            <div id="toggle-8" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rapid discovery of monoclonal antibodies by microfluidics-enabled FACS of single pathogen-specific antibody-secreting cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that young people can understand:

Scientists are using special proteins called monoclonal antibodies to fight viruses and save lives during pandemics. These antibodies come from a type of cell in our body that makes them, and it's hard to study these cells because the current methods are slow, expensive, or tricky. We've developed a new way to quickly find good antibodies from these special cells. It uses tiny devices (microfluidics) to sort through millions of cells, and a machine (flow cytometry) to find the best ones. With this method, we were able to find strong antibodies against the COVID-19 virus in just two weeks! This is important because it allows us to study how our bodies make these protective proteins and how we can use them to fight other viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monoclonal antibodies are increasingly used to prevent and treat viral infections and are pivotal in pandemic response efforts. Antibody-secreting cells (ASCs; plasma cells and plasmablasts) are an excellent source of high-affinity antibodies with therapeutic potential. Current methods to study antigen-specific ASCs either have low throughput, require expensive and labor-intensive screening or are technically demanding and therefore not widely accessible. Here we present a straightforward technology for the rapid discovery of monoclonal antibodies from ASCs. Our approach combines microfluidic encapsulation of single cells into an antibody capture hydrogel with antigen bait sorting by conventional flow cytometry. With our technology, we screened millions of mouse and human ASCs and obtained monoclonal antibodies against severe acute respiratory syndrome coronavirus 2 with high affinity (<1‚ÄâpM) and neutralizing capacity (<100‚Äâng‚Äâml‚àí1) in 2‚Äâweeks with a high hit rate (>85% of characterized antibodies bound the target). By facilitating access to the underexplored ASC compartment, the approach enables efficient antibody discovery and immunological studies into the generation of protective antibodies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41587-024-02346-5" target="_blank">https://www.nature.com/articles/s41587-024-02346-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-14</strong> <a href="https://www.nature.com/articles/s41586-024-07820-3" target="_blank">
            ü§ñ Unraveling the Secrets of Neutrophil Extracellular Traps</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-9', this)">
            More</button></h4>
            <div id="toggle-9" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Recognition and control of neutrophil extracellular trap formation by MICL</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people's bodies, tiny things called neutrophils help fight off infections. When they're working well, everything is fine. But sometimes, these neutrophils can get out of control and create web-like structures made of DNA and proteins called neutrophil extracellular traps (NETs). If NETs aren't managed properly, they can make inflammation worse and hurt the body.

Our study found that a special receptor in neutrophils called MICL is very important for regulating these NETs. When MICL recognizes the DNA in NETs, it helps prevent uncontrolled growth of NETs through a process involving a molecule called ROS-PAD4. However, if MICL doesn't work properly or gets blocked by autoantibodies (like when the body mistakenly attacks itself), it can lead to severe inflammation and worsen conditions like rheumatoid arthritis, lupus, and COVID-19.

Interestingly, our research also shows that having a dysfunctional MICL receptor might even help during infections with certain types of fungi, as it allows for the release of more NETs to fight off these pathogens. Overall, our findings highlight the crucial role of MICL in regulating neutrophil activity and NET formation, and how its malfunction can contribute to various diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Regulation of neutrophil activation is critical for disease control. Neutrophil extracellular traps (NETs), which are web-like structures composed of DNA and neutrophil-derived proteins, are formed following pro-inflammatory signals; however, if this process is uncontrolled, NETs contribute to disease pathogenesis, exacerbating inflammation and host tissue damage1,2. Here we show that myeloid inhibitory C-type lectin-like (MICL), an inhibitory C-type lectin receptor, directly recognizes DNA in NETs; this interaction is vital to regulate neutrophil activation. Loss or inhibition of MICL functionality leads to uncontrolled NET formation through the ROS‚ÄìPAD4 pathway and the development of an auto-inflammatory feedback loop. We show that in the context of rheumatoid arthritis, such dysregulation leads to exacerbated pathology in both mouse models and in human patients, where autoantibodies to MICL inhibit key functions of this receptor. Of note, we also detect similarly inhibitory anti-MICL autoantibodies in patients with other diseases linked to aberrant NET formation, including lupus and severe COVID-19. By contrast, dysregulation of NET release is protective during systemic infection with the fungal pathogenAspergillus fumigatus. Together, we show that the recognition of NETs by MICL represents a fundamental autoregulatory pathway that controls neutrophil activity and NET formation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07820-3" target="_blank">https://www.nature.com/articles/s41586-024-07820-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-14</strong> <a href="https://www.nature.com/articles/s41598-024-69781-x" target="_blank">
            ü§ñ Boosting Chicken Immunity to Fight Bacterial Infections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-10', this)">
            More</button></h4>
            <div id="toggle-10" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Induction of trained immunity in broiler chickens following delivery of oligodeoxynucleotide containing CpG motifs to protect againstEscherichia colisepticemia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Imagine a special way to help chickens fight off germs and stay healthy. Scientists discovered that giving chickens tiny bits of DNA called CpG-ODN can boost their immune system and even teach it to remember past infections, making them better at fighting new ones. This is called "trained immunity." The scientists wanted to see if they could do the same in chickens. They gave some chickens a special DNA treatment twice when they were just one and four days old. Then, they challenged these chickens with a bad germ called E. coli when they were 27 days old. The chickens that got the CpG-ODN treatment had stronger immune systems and were able to fight off the E. coli better than those who didn't get treated. This shows that giving young chickens this special DNA treatment can help protect them from getting sick later on.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Oligodeoxynucleotides containing CpG motifs (CpG-ODN) can promote antimicrobial immunity in chickens by enriching immune compartments and activating immune cells. Innate memory, or trained immunity, has been demonstrated in humans and mice, featuring the absence of specificity to the initial stimulus and subsequently cross-protection against pathogens. We hypothesize that CpG-ODN can induce trained immunity in chickens. We delivered single or multiple administrations of CpG-ODN to birds and mitochondrial oxidative phosphorylation (OXPHOS) and glycolysis of peripheral blood mononuclear cells were quantified using Seahorse XFp. Next, chickens were administered with CpG-ODN twice at 1 and 4¬†day of age and challenged withEscherichia coliat 27¬†days of age. The CpG-ODN administered groups had significantly higher mitochondrial OXPHOS until 21¬†days of age while cellular glycolysis gradually declined by 14¬†days of age. The group administered with CpG-ODN twice at 1 and 4¬†days of age had significantly higher survival, lower clinical score and bacterial load following challenge withE. coliat 27 d of age. This study demonstrated the induction of trained immunity in broiler chickens following administration of CpG-ODN twice during the first 4¬†days of age to protect birds againstE. colisepticemia at 27¬†days of age.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-69781-x" target="_blank">https://www.nature.com/articles/s41598-024-69781-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-13</strong> <a href="https://www.nature.com/articles/s41598-024-69825-2" target="_blank">
            ü§ñ Testing a Measles Vaccine in Mice</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-11', this)">
            More</button></h4>
            <div id="toggle-11" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity of a peptide-based vaccine for measles: a pilot evaluation in a mouse model</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Measles is a serious illness that can cause problems for people who get infected. To fight it, our bodies need both antibodies (which are like special fighters) and T cells (which help those fighters do their job). We previously found 13 tiny pieces of the measles virus that could attach to special helpers in our cells called HLA molecules. In this study, we tested these pieces in a special kind of mouse that can help us understand how the human body works. Our tests showed that some of these pieces were good at making the immune system respond and helping the body fight off the measles virus. When we used four of the best pieces to make a vaccine and tested it on mice, it worked really well - the mice who got the vaccine did better than those who didn't after they were infected with the virus. This shows that using T cell helpers might be a good way to create vaccines against measles.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although neutralizing antibody is an established correlate of protection for measles, T cell-mediated responses play at least two critical roles in immunity to measles: first, through provision of ‚Äòhelp‚Äô enabling robust humoral immune responses; and second, through clearance of measles virus-infected cells. Previously, we identified 13 measles-derived peptides that bound to human leukocyte antigen (HLA) molecules in Priess cells infected with measles virus. In this study, we evaluated the immunogenicity of these peptides in a transgenic mouse model. Our results demonstrated that these peptides induced Th1-biased immune responses at varying levels. Of the 13 peptides, the top four immunogenic peptides were further selected for a viral challenge study in mice. A vaccine based on a combination of these four peptides reduced morbidity and weight loss after viral challenge compared to placebo. Our results emphasize the potential of T cell-mediated, peptide-based vaccines against measles.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-69825-2" target="_blank">https://www.nature.com/articles/s41598-024-69825-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-13</strong> <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">
            ü§ñ Uncovering the Mechanisms Behind COVID-19 Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-12', this)">
            More</button></h4>
            <div id="toggle-12" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 N protein-induced Dicer, XPO5, SRSF3, and hnRNPA3 downregulation causes pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people with COVID-19, some essential molecules that regulate the virus's replication are not working properly. This is because a protein from the virus (N protein) is breaking down these molecules (Dicer and XPO5). As a result, the body can't produce a type of genetic material called miRNA, which helps prevent damage to the cells. The N protein also causes stress in the cells and triggers pneumonia.

When scientists knocked down these essential molecules in mice, it made their pneumonia worse. However, when they overexpressed them (made more), it helped reduce the severity of the pneumonia. Older mice had lower levels of these molecules in their lungs and developed more severe pneumonia than younger mice.

Scientists also found that using certain medications (PJ34 or anastrozole) can help restore the function of these essential molecules, which in turn can alleviate the symptoms of COVID-19-related pneumonia. These findings could lead to improved treatments for people with SARS-CoV-2-associated pneumonia.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Though RNAi and RNA-splicing machineries are involved in regulating severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication, their precise roles in coronavirus disease 2019 (COVID-19) pathogenesis remain unclear. Herein, we show that decreased RNAi component (Dicer and XPO5) and splicing factor (SRSF3 and hnRNPA3) expression correlate with increased COVID-19 severity. SARS-CoV-2 N protein induces the autophagic degradation of Dicer, XPO5, SRSF3, and hnRNPA3, inhibiting miRNA biogenesis and RNA splicing and triggering DNA damage, proteotoxic stress, and pneumonia. Dicer, XPO5, SRSF3, and hnRNPA3 knockdown increases, while their overexpression decreases, N protein-induced pneumonia‚Äôs severity. Older mice show lower expression of Dicer, XPO5, SRSF3, and hnRNPA3 in their lung tissues and exhibit more severe N protein-induced pneumonia than younger mice. PJ34, a poly(ADP-ribose) polymerase inhibitor, or anastrozole, an aromatase inhibitor, ameliorates N protein- or SARS-CoV-2-induced pneumonia by restoring Dicer, XPO5, SRSF3, and hnRNPA3 expression. These findings will aid in developing improved treatments for SARS-CoV-2-associated pneumonia.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">https://www.nature.com/articles/s41467-024-51192-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-12</strong> <a href="https://www.nature.com/articles/s41598-024-68929-z" target="_blank">
            ü§ñ Quick Lockdowns Stop COVID Spread in Danish Cities</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-13', this)">
            More</button></h4>
            <div id="toggle-13" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Short, stringent lockdowns halted SARS-CoV-2 transmissions in Danish municipalities</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In late 2020, scientists were worried about a new version of the COVID-19 virus that was spreading among minks in Denmark. This could have made vaccines less effective. To stop this, some areas in Denmark were locked down for a short time to see if it would help reduce infections. By comparing these areas with others that weren't locked down, researchers found that the lockdowns worked really well. In fact, they reduced infections by about 31%. This study shows how strict lockdowns can be effective in breaking chains of infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In late 2020, the focus of the global effort against the COVID-19 pandemic centered around the development of a vaccine, when reports of a mutated SARS-CoV-2 virus variant in a population of 17 million farmed mink came from Denmark, threatening to jeopardize this effort. Spillover infections of the new variant between mink and humans were feared to threaten the efficacy of upcoming vaccines. In this study the ensuing short-lived yet stringent lockdowns imposed in 7 of the countries 98 municipalities are analysed for their effectiveness to reduce SARS-CoV-2 infections. Synthetic counterfactuals are created for each of these municipalities using a weighted average combination of the remaining municipalities not targeted by the stringent measures. This allows for a clear overview regarding the development of test-positivity rates, citizen mobility behaviours and lastly daily infection numbers in response to the restrictions. The findings show that these targeted, short-term lockdowns significantly curtailed further infections, demonstrating a marked decrease, first in citizens mobility and then in daily cases when compared to their synthetic counterfactuals. Overall, the estimates indicate average reductions to infection numbers to be around 31%. This study underscores the potential of strict, yet severe lockdowns in breaking ongoing infection dynamics, by utilising a rare quasi-experimental design case that avoids bias introduced through treatment selection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68929-z" target="_blank">https://www.nature.com/articles/s41598-024-68929-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-12</strong> <a href="https://www.nature.com/articles/s41467-024-51046-w" target="_blank">
            ü§ñ Blocking a Key Protein Helps Stop COVID-19 Spread</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-14', this)">
            More</button></h4>
            <div id="toggle-14" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Variant-proof high affinity ACE2 antagonist limits SARS-CoV-2 replication in upper and lower airways</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 is a type of coronavirus that can change in ways that make vaccines and treatments less effective. Scientists want to develop a new treatment that can protect people from getting very sick, even if the virus changes over time. In this study, researchers tested a special protein called RBD-62 on monkeys that were infected with a particularly bad version of SARS-CoV-2. The results showed that RBD-62 was able to stop the virus from spreading in both the upper and lower airways, which is different from how current vaccines work. Also, the treatment didn't interfere with the monkey's immune system or create antibodies against itself, which means it could be a useful tool for preventing severe disease caused by SARS-CoV-2 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 has the capacity to evolve mutations that escape vaccine- and infection-acquired immunity and antiviral drugs. A variant-agnostic therapeutic agent that protects against severe disease without putting selective pressure on the virus would thus be a valuable biomedical tool that would maintain its efficacy despite the ongoing emergence of new variants. Here, we challenge male rhesus macaques with SARS-CoV-2 Delta‚Äîthe most pathogenic variant in a highly susceptible animal model. At the time of challenge, we also treat the macaques with aerosolized RBD-62, a protein developed through multiple rounds of in vitro evolution of SARS-CoV-2 RBD to acquire 1000-fold enhanced ACE2 binding affinity. RBD-62 treatment equivalently suppresses virus replication in both upper and lower airways, a phenomenon not previously observed with clinically approved vaccines. Importantly, RBD-62 does not block the development of virus-specific T- and B-cell responses and does not elicit anti-drug immunity. These data provide proof-of-concept that RBD-62 can prevent severe disease from a highly virulent variant.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51046-w" target="_blank">https://www.nature.com/articles/s41467-024-51046-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-10</strong> <a href="https://www.nature.com/articles/s41598-024-69343-1" target="_blank">
            ü§ñ Feline Stem Cells and Viruses: A Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-15', this)">
            More</button></h4>
            <div id="toggle-15" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evaluation of feline mesenchymal stem cell susceptibility to feline viruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Feline mesenchymal stem cells are a type of cell that can become different types of cells and are often used to study immune-related diseases in cats. However, it's unclear if these cells are susceptible to certain viruses that affect cats. This study looked at how well three common feline viruses - FCoV, FHV-1, and FPV - can infect two types of mesenchymal stem cells: those from fat tissue (fAD-MSCs) and umbilical cord tissue (fUC-MSCs). The results showed that FCoV and FHV-1 were able to infect both types of cells, but FPV did not affect fUC-MSCs. Additionally, all three viruses were successfully isolated from fAD-MSCs. These findings suggest that certain viruses can replicate in mesenchymal stem cells, which could limit their use in treating viral diseases caused by these specific viruses. This study has important implications for veterinarians when managing viral diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Feline mesenchymal stem cells (fMSCs) are well known for their robust differentiation capabilities and are commonly used in studying immune-related diseases in cats. Despite their importance, the susceptibility of fMSCs to viral infections remains uncertain. This study aimed to assess the susceptibility of feline adipose-derived mesenchymal stem cells (fAD-MSCs) and feline umbilical cord-derived mesenchymal stem cells (fUC-MSCs) to common feline viruses, including feline coronavirus (FCoV), feline herpesvirus type 1 (FHV-1), and feline panleukopenia virus (FPV). The results demonstrated that both FCoV and FHV-1 were able to infect both types of cells, while FPV did not exhibit cytopathic effects on fUC-MSCs. Furthermore, all three viruses were successfully isolated from fAD-MSCs. These findings suggest that certain feline viruses can replicate in fMSCs, indicating potential limitations in using fMSCs for treating viral diseases caused by these specific viruses. This study has important clinical implications for veterinarians, particularly in the management of viral diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-69343-1" target="_blank">https://www.nature.com/articles/s41598-024-69343-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">
            ü§ñ Most Immune Cells After Vaccination or Infection Are Unusual</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-16', this)">
            More</button></h4>
            <div id="toggle-16" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Atypical and non-classical CD45RBlomemory B cells are the majority of circulating SARS-CoV-2 specific B cells following mRNA vaccination or COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Resting memory B cells are like a group of people who remember past infections or vaccinations. Scientists used a special tool called mass cytometry to analyze these cells in patients with COVID-19, bacterial sepsis (a type of serious infection), or those who received an mRNA vaccine. They found that resting memory B cells can be divided into two main groups: classical and atypical.

The scientists were able to track which group a cell belonged to by looking at a protein called CD45RB on the surface of the cell. They discovered that after activation, this protein level remained stable, allowing them to follow the path of activated B cells and plasmablasts (a type of immune cell) from either classical or atypical memory B cells.

In patients with COVID-19 and those who received an mRNA vaccine, scientists found that most SARS-CoV-2 specific memory B cells belonged to the atypical group. This group was also linked to the production of antibodies against the virus. On the other hand, classical memory B cells were activated by bacterial sepsis.

These findings suggest that scientists can use CD45RB levels to track resting memory B cells and their activated progeny. They also indicate that atypical and non-classical memory B cells play a role in fighting SARS-CoV-2 infections and responding to mRNA vaccinations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Resting memory B cells can be divided into classical or atypical groups, but the heterogenous marker expression on activated memory B cells makes similar classification difficult. Here, by longitudinal analysis of mass cytometry and CITE-seq data from cohorts with COVID-19, bacterial sepsis, or BNT162b2 mRNA vaccine, we observe that resting B cell memory consist of classical CD45RB+memory and CD45RBlomemory, of which the latter contains of two distinct groups of CD11c+atypical and CD23+non-classical memory cells. CD45RB levels remain stable in these cells after activation, thereby enabling the tracking of activated B cells and plasmablasts derived from either CD45RB+or CD45RBlomemory B cells. Moreover, in both COVID-19 patients and mRNA vaccination, CD45RBloB cells formed the majority of SARS-CoV2 specific memory B cells and correlated with serum antibodies, while CD45RB+memory are activated by bacterial sepsis. Our results thus identify that stably expressed CD45RB levels can be exploited to trace resting memory B cells and their activated progeny, and suggest that atypical and non-classical CD45RBlomemory B cells contribute to SARS-CoV-2 infection and vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">https://www.nature.com/articles/s41467-024-50997-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s43856-024-00582-z" target="_blank">
            ü§ñ Tracking New COVID Strains with a Tiny Diagnostic Test</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-17', this)">
            More</button></h4>
            <div id="toggle-17" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A micro-disc-based multiplex method for monitoring emerging SARS-CoV-2 variants using the molecular diagnostic tool Intelli-OVI</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that's easier for young people to understand:

Scientists have developed a new way to quickly and easily identify changes (mutations) in the COVID-19 virus, which can spread very easily and change often. This method is called Intelli-OVI and it uses tiny discs with special patterns to detect different versions of the virus. By using this method and computer algorithms, scientists were able to identify more than 20 different variations of the COVID-19 virus at the same time. This approach is like having a super-efficient filter that can catch many different types of viruses, including new ones that haven't been seen before. It's also flexible and can be updated as needed to keep up with changing viruses, making it potentially useful for other viruses beyond COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundHighly transmissible viruses including SARS-CoV-2 frequently accumulate novel mutations that are detected via high-throughput sequencing. However, there is a need to develop an alternative rapid and non-expensive approach. Here we developed a novel multiplex DNA detection method Intelli-OVI for analysing existing and novel mutations of SARS-CoV-2.MethodsWe have developed Intelli-OVI that includes the micro-disc-based method IntelliPlex and computational algorithms of objective variant identification (OVI). More than 250 SARS-CoV-2 positive samples including wastewater ones were analysed to verify the efficiency of the method.ResultsIntelliPlex uses micro-discs printed with a unique pictorial pattern as a labelling conjugate for DNA probes, and OVI allows simultaneous identification of several variants using multidimensional data obtained by the IntelliPlex method. Importantly, de novo mutations can be identified by decreased signals, which indicates that there is an emergence of de novo variant virus as well as prompts the need to design additional primers and probes. We have upgraded probe panel according to the emergence of new variants and demonstrated that Intelli-OVI efficiently identified more than 20 different SARS-CoV-2 variants by using 35 different probes simultaneously.ConclusionsIntelli-OVI can be upgraded to keep up with rapidly evolving viruses as we showed in this study using SARS-CoV-2 as an example and may be suitable for other viruses but would need to be validated.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00582-z" target="_blank">https://www.nature.com/articles/s43856-024-00582-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s41467-024-50612-6" target="_blank">
            ü§ñ Unlocking Human Genomics: Advanced Modeling for Large Cohort Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-18', this)">
            More</button></h4>
            <div id="toggle-18" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>MOCHA‚Äôs advanced statistical modeling of scATAC-seq data enables functional genomic inference in large human cohorts</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) is a way to study how genes are regulated in individual cells. However, there's been a problem with analyzing this type of data to understand gene regulation in complex diseases like cancer or COVID-19. A new tool called MOCHA has been developed to address these issues and provide a more accurate picture of how genes are turned on or off in different cells. MOCHA can identify which parts of the genome are open, account for technical errors that can make it look like certain areas are open when they're not, reduce false positives that can occur when analyzing single cell data, find alternative starting points for gene transcription, and even build a timeline of how genes are regulated over time. These advancements make MOCHA a powerful tool for studying gene regulation in human diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) is being increasingly used to study gene regulation. However, major analytical gaps limit its utility in studying gene regulatory programs in complex diseases. In response, MOCHA (Model-based single cell Open CHromatin Analysis) presents major advances over existing analysis tools, including: 1) improving identification of sample-specific open chromatin, 2) statistical modeling of technical drop-out with zero-inflated methods, 3) mitigation of false positives in single cell analysis, 4) identification of alternative transcription-starting-site regulation, and 5) modules for inferring temporal gene regulatory networks from longitudinal data. These advances, in addition to open chromatin analyses, provide a robust framework after quality control and cell labeling to study gene regulatory programs in human disease. We benchmark MOCHA with four state-of-the-art tools to demonstrate its advances. We also construct cross-sectional and longitudinal gene regulatory networks, identifying potential mechanisms of COVID-19 response. MOCHA provides researchers with a robust analytical tool for functional genomic inference from scATAC-seq data.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50612-6" target="_blank">https://www.nature.com/articles/s41467-024-50612-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s41598-024-68898-3" target="_blank">
            ü§ñ Uncovering Hidden Interactions: A Scientific Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-19', this)">
            More</button></h4>
            <div id="toggle-19" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-omics analysis of antiviral interactions ofElizabethkingia anophelisand Zika virus</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

The tiny living things inside mosquitoes' stomachs help decide what happens when a mosquito gets infected with a virus. There's a special kind of bacteria that lives inside the mosquito and can fight off many different kinds of viruses. But scientists don't fully understand how this bacteria works its magic. This study tries to figure out why this bacteria is so good at stopping viruses. By looking at what's in the mosquito's waste and what the bacteria itself does, scientists found clues that might help them develop new ways to stop mosquitoes from spreading viruses. They discovered that two important things that viruses need to grow are less abundant when the special bacteria is around. This discovery could lead to new tools to prevent outbreaks of mosquito-borne diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The microbial communities residing in the mosquito midgut play a key role in determining the outcome of mosquito pathogen infection.Elizabethkingia anophelis, originally isolated from the midgut ofAnopheles gambiaepossess a broad-spectrum antiviral phenotype, yet a gap in knowledge regarding the mechanistic basis of its interaction with viruses exists. The current study aims to identify pathways and genetic factors linked toE. anophelisantiviral activity. The understanding ofE. anophelisantiviral mechanism could lead to novel transmission barrier tools to prevent arboviral outbreaks. We utilized a non-targeted multi-omics approach, analyzing extracellular lipids, proteins, metabolites of culture supernatants coinfected with ZIKV andE. anophelis. We observed a significant decrease in arginine and phenylalanine levels, metabolites that are essential for viral replication and progression of viral infection. This study provides insights into the molecular basis ofE. anophelisantiviral phenotype. The findings lay a foundation for in-depth mechanistic studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68898-3" target="_blank">https://www.nature.com/articles/s41598-024-68898-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-08</strong> <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">
            ü§ñ Uncovering the Complexity of Monkeypox Virus Infections Through Advanced Analysis Techniques</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-20', this)">
            More</button></h4>
            <div id="toggle-20" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-omics characterization of the monkeypox virus infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Multiple omics analyzes of a certain type of virus called Vaccinia have helped scientists understand how it works. However, there is still much to be learned about the monkeypox virus (MPXV), which causes similar but slightly different symptoms in humans despite being very similar to Vaccinia. To better understand this, we studied the genes, proteins, and phosphoproteins of MPXV-infected human cells. Our analysis revealed changes not only in immune-related pathways but also in other important cellular processes. We found that certain host and viral proteins are dynamically modified through phosphorylation, suggesting that a specific group of enzymes called MAPKs play a crucial role in this process. Moreover, we discovered that the H5 protein's ability to bind to double-stranded DNA is influenced by its dynamic modification. Our comprehensive dataset provides insights into the signaling events and hotspots affected by MPXV, expanding our knowledge on poxviruses. By integrating pathway analysis and drug-target prediction approaches, we identified potential targets for treating MPXV and Vaccinia infections. This includes inhibitors of MTOR, CHUK/IKBKB, and splicing factor kinases, which showed significant antiviral efficacy against both viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multiple omics analyzes of Vaccinia virus (VACV) infection have defined molecular characteristics of poxvirus biology. However, little is known about the monkeypox (mpox) virus (MPXV) in humans, which has a different disease manifestation despite its high sequence similarity to VACV. Here, we perform an in-depth multi-omics analysis of the transcriptome, proteome, and phosphoproteome signatures of MPXV-infected primary human fibroblasts to gain insights into the virus-host interplay. In addition to expected perturbations of immune-related pathways, we uncover regulation of the HIPPO and TGF-Œ≤ pathways. We identify dynamic phosphorylation of both host and viral proteins, which suggests that MAPKs are key regulators of differential phosphorylation in MPXV-infected cells. Among the viral proteins, we find dynamic phosphorylation of H5 that influenced the binding of H5 to dsDNA. Our extensive dataset highlights signaling events and hotspots perturbed by MPXV, extending the current knowledge on poxviruses. We use integrated pathway analysis and drug-target prediction approaches to identify potential drug targets that affect virus growth. Functionally, we exemplify the utility of this approach by identifying inhibitors of MTOR, CHUK/IKBKB, and splicing factor kinases with potent antiviral efficacy against MPXV and VACV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">https://www.nature.com/articles/s41467-024-51074-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-07</strong> <a href="https://www.nature.com/articles/s41586-024-07722-4" target="_blank">
            ü§ñ Unlocking the Secrets of MIS-C: Understanding Molecular Mimicry</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-21', this)">
            More</button></h4>
            <div id="toggle-21" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Molecular mimicry in multisystem inflammatory syndrome in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Multisystem inflammatory syndrome in children (MIS-C) is a serious condition that happens to some kids after they get infected with COVID-19. But scientists don't know exactly how this connection works.

Researchers looked at blood samples from kids who had MIS-C and found special proteins in their bodies that were being attacked by their own immune system. They also noticed that these kids' immune systems were reacting strongly to a specific part of the SARS-CoV-2 virus, called the nucleocapsid protein.

Interestingly, this part of the virus is very similar in sequence to a human protein called SNX8, which helps regulate an antiviral pathway associated with MIS-C pathogenesis. As a result, many kids with anti-SNX8 autoantibodies also have immune cells that can react to both the SARS-CoV-2 nucleocapsid protein and the human SNX8 protein.

These findings suggest that kids who get MIS-C develop an immune response to the SARS-CoV-2 nucleocapsid protein that is linked to cross-reactivity with the self-protein SNX8. This discovery sheds light on the mechanism connecting COVID-19 infection to the inflammatory syndrome, and may have implications for understanding other post-infectious autoinflammatory diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multisystem inflammatory syndrome in children (MIS-C) is a severe, post-infectious sequela of SARS-CoV-2 infection1,2, yet the pathophysiological mechanism connecting the infection to the broad inflammatory syndrome remains unknown. Here we leveraged a large set of samples from patients with MIS-C to identify a distinct set of host proteins targeted by patient autoantibodies including a particular autoreactive epitope within SNX8, a protein involved in regulating an antiviral pathway associated with MIS-C pathogenesis. In parallel, we also probed antibody responses from patients with MIS-C to the complete SARS-CoV-2 proteome and found enriched reactivity against a distinct domain of the SARS-CoV-2 nucleocapsid protein. The immunogenic regions of the viral nucleocapsid and host SNX8 proteins bear remarkable sequence similarity. Consequently, we found that many children with anti-SNX8 autoantibodies also have cross-reactive T cells engaging both the SNX8 and the SARS-CoV-2 nucleocapsid protein epitopes. Together, these findings suggest that patients with MIS-C develop a characteristic immune response to the SARS-CoV-2 nucleocapsid protein that is associated with cross-reactivity to the self-protein SNX8, demonstrating a mechanistic link between the infection and the inflammatory syndrome, with implications for better understanding a range of post-infectious autoinflammatory diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07722-4" target="_blank">https://www.nature.com/articles/s41586-024-07722-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-07</strong> <a href="https://www.nature.com/articles/s41598-024-68983-7" target="_blank">
            ü§ñ Caring for Each Other Amid Chaos</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-22', this)">
            More</button></h4>
            <div id="toggle-22" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparison of psychopathology, purpose in life and moral courage between nursing home and hospital healthcare workers during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic had a big impact on people who work in healthcare, but it may have affected them differently depending on where they worked. Some people thought that people who work in hospitals would be more affected than those who work in nursing homes, but this study found the opposite to be true. People who worked in nursing homes were more anxious, depressed, and experienced post-traumatic stress disorder than those who worked in hospitals.

The study also looked at how having a sense of purpose and being courageous can help people cope with the stress of working in healthcare during a pandemic. They found that people who have a strong sense of purpose are less likely to experience anxiety, depression, and burnout. People who are courageous are also less likely to experience depression and acute stress.

The study also found that when family or friends get sick, it can make things harder for healthcare workers. This knowledge could be useful in preparing for future pandemics and helping people who work in healthcare to cope with the stress of working during a pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic deeply affected healthcare workers, although the impact may have differed according to different workplace contexts. The aim of this current research was to compare the psychopathology presented by hospital versus nursing home healthcare workers during the COVID-19 pandemic and to analyse the predictive role of purpose in life and moral courage in the appearance of psychopathology. This was an observational, cross-sectional study carried out on a sample of 108 healthcare workers, 54 each from a hospital or nursing homes, who were recruited during the 5 and 6th waves of the COVID-19 pandemic in Spain. Various self-reported scales were used to assess anxiety, depression, acute/post-traumatic stress disorder, drug and alcohol abuse, burnout, purpose in life, and moral courage. Compared to the hospital healthcare workers, nursing home healthcare workers had higher scores and a higher prevalence of anxiety (74.1% vs. 42%), depression (40.7% vs. 14.8%), and post-traumatic stress disorder (55.6% vs. 25.9). In the overall sample, purpose in life was a protective factor against psychopathology (OR‚Äâ=‚Äâ0.54) and burnout (OR‚Äâ=‚Äâ0.48); moral courage was a protective factor against depression (OR‚Äâ=‚Äâ0.47) and acute stress (OR‚Äâ=‚Äâ0.45); and exposure of family/friends to SARS-CoV-2 was a risk factor for acute stress (OR‚Äâ=‚Äâ2.24), post-traumatic stress disorder (OR‚Äâ=‚Äâ1.33), and higher burnout depersonalisation subscale scores (OR‚Äâ=‚Äâ1.84). In conclusion, the increased presence of psychopathology in nursing home healthcare workers may be influenced by workplace and occupational contexts, personal factors such as exposure of family/friends to SARS-CoV-2, or internal dimensions such as purpose in life and moral courage. This knowledge could be useful for understanding how a future epidemic or pandemic might affect the mental health of healthcare workers in different labour contexts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68983-7" target="_blank">https://www.nature.com/articles/s41598-024-68983-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-06</strong> <a href="https://www.nature.com/articles/s41598-024-67977-9" target="_blank">
            ü§ñ Detecting COVID-19 Antibodies in a Small Lab Dish</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-23', this)">
            More</button></h4>
            <div id="toggle-23" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Performance evaluation of microfluidic microplate-based fluorescent ELISA for qualitative detection of SARS-CoV-2‚Äìspecific IgG and IgM</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We tested a new way of doing blood tests (ELISAs) that can quickly detect antibodies against the virus that causes COVID-19. This test is done in small tubes called microplates and uses a special light to make the results clearer. We compared this new method with traditional ELISAs and found it worked just as well, if not better, in most cases. The new test was also able to detect antibodies earlier on after someone gets sick with COVID-19. This new technology has the potential to make blood tests for COVID-19 faster and easier to do, which could be helpful in a pandemic situation.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We evaluated the diagnostic performance of newly developed microfluidic microplate-based fluorescent ELISA for anti-SARS-CoV-2 antibody detection: the Veri-Q opti COVID-19 IgG and IgM ELISAs (hereafter, ‚ÄúOpti IgG/M‚Äù; MiCo BioMed, Gyeonggi-do, Republic of Korea), in comparison with conventional ELISAs. A total of 270 serum samples were analyzed, among which 90 samples were serially obtained from 25 COVID-19 patients. Another 180 samples were collected from 180 SARS-CoV-2‚Äìnegative individuals. As comparative assays, we used SCoV-2 Detect IgG/M ELISA (hereafter, ‚ÄúInBios IgG/M‚Äù; InBios, Seattle, WA, USA) and Veri-Q COVID-19 IgG/IgM ELISA (hereafter, ‚ÄúVeri-Q IgG/M‚Äù; MiCo BioMed). Compared with conventional ELISAs, the Opti IgG yielded 97.1‚Äì100.0% positive percent agreement, 95.2‚Äì98.0% negative percent agreement, 96.3‚Äì97.8% total percent agreement, and kappa values of 0.90‚Äì0.94. Between the Opti IgM and the InBios IgM, the values were 93.7%, 96.6%, 95.9%, and 0.89, respectively. For the Opti IgG, sensitivities for the samples collected from 0‚Äì7, 8‚Äì14, 15‚Äì21, and‚Äâ‚â•‚Äâ22¬†days after symptom onset were 40.0, 58.3, 94.1, and 100.0%, respectively. The values for the Opti IgM were 30.0, 54.2, 88.2, and 80%, respectively. The diagnostic specificities of the Opti IgG and IgM were 99.4 and 97.2%, respectively. The microfluidic microplate-based fluorescent ELISAs showed comparable diagnostic performance to conventional ELISAs for detecting anti-SARS-CoV-2 antibodies. With the combination of high throughput, a simplified workflow, and the ability to analyze reduced volumes, this new technology has great potential for improving SARS-CoV-2 serologic testing.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67977-9" target="_blank">https://www.nature.com/articles/s41598-024-67977-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-06</strong> <a href="https://www.nature.com/articles/s41598-024-68162-8" target="_blank">
            ü§ñ COVID-19 Infections That Don't Show Symptoms Disappear in China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-24', this)">
            More</button></h4>
            <div id="toggle-24" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Asymptomatic infection and disappearance of clinical symptoms of COVID-19 infectors in China 2022‚Äì2023: a cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To understand how COVID-19 affects people differently across China, we analyzed data from 64,515 patients who were diagnosed with the virus using a test that looks for the virus's genetic material or a quick test that can detect its protein. These patients shared their experiences and medical information online between December 24, 2022, and January 16, 2023. Among them, 59,669 reported not having symptoms at all. The most common symptoms were related to the upper respiratory tract, such as dry or itchy throats (64.16%), sore throats (59.95%), hoarseness (57.90%), and nasal congestion (53.39%). Our analysis showed that men, people without pre-existing conditions, those who had been infected before, and those who were vaccinated all had a higher chance of not showing symptoms. When looking at how quickly symptoms went away for everyone, we found that being younger than 60, male, without pre-existing conditions, vaccinated, or having received treatment all contributed to a faster recovery. The most common symptoms were related to the upper respiratory tract, which is consistent with what's known about the Omicron variant of COVID-19. Our findings suggest that factors like age, gender, and previous infections can affect how COVID-19 affects people and their chances of recovering quickly. Importantly, being vaccinated has a positive impact on preventing and treating COVID-19. Additionally, using traditional Chinese medicine may also be beneficial, but more guidance is needed to ensure it's used correctly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To explore the clinical characteristics of patients infected with SARS-CoV-2 nationwide, especially the effect factors of asymptomatic infection and disappearance of clinical symptoms. A total of 66,448 COVID-19 patients in China who have been diagnosed by nucleic acid test or rapid antigen test were surveyed online (December 24, 2022 to January 16, 2023). Our cross-sectional study used descriptive analyses and binary Logistics regression model to assess the correlation between the clinical characteristics and relative factors, including age, gender, pre-existing conditions, reinfection, vaccination and treatment. A total of 64,515 valid questionnaires were collected. Among included participants, 5969 of which were asymptomatic. The symptoms were mainly upper respiratory symptoms, including dry and itchy throat (64.16%), sore throat (59.95%), hoarseness (57.90%), nasal congestion (53.39%). In binary Logistics regression model, we found that male, no pre-existing conditions, reinfection and vaccination have positive correlations with the appearance of asymptomatic COVID-19 patients. In Cox proportional-hazards regression model, considering all clinical symptoms disappeared in 14¬†days as outcome, we found that‚Äâ‚â§‚Äâ60¬†years old, male, no pre-existing conditions, vaccination and adopted treatment have positive correlations with rapid amelioration of clinical symptoms in COVID-19 patients. The clinical symptoms of the participants were mainly upper respiratory symptoms which were according with the infection of Omicron variant. Factors including age, gender, pre-existing conditions and reinfection could influence the clinical characteristics and prognosis of COVID-19 patients. Importantly, vaccination has positive significance for the prevention and treatment of COVID-19. Lastly, the use of Chinese medicine maybe beneficial to COVID-19 patients, however, reasonable guidance is necessary.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68162-8" target="_blank">https://www.nature.com/articles/s41598-024-68162-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-05</strong> <a href="https://www.nature.com/articles/s41588-024-01844-1" target="_blank">
            ü§ñ Understanding COVID-19 and Influenza Risks</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-25', this)">
            More</button></h4>
            <div id="toggle-25" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genetic risk factors for COVID-19 and influenza are largely distinct</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's what we found out about two common respiratory illnesses: COVID-19 and the flu.

These two diseases have some similar symptoms and things that make you more likely to get them, but nobody knew if they were caused by similar genetic problems. We studied a big group of people who had the flu (18,334) and compared them to a much bigger group of people who didn't have the flu (276,295). What we found was that the genetic problems that make you more likely to get COVID-19 aren't the same as the ones that make you more likely to get the flu.

We also discovered two new genetic problems that can increase your chances of getting the flu. And when we tested these problems in a lab, it turned out that one of them was very important for helping the flu virus to infect cells. By blocking this problem, we were able to reduce how easily the flu virus could infect cells by 57%.

This is similar to what scientists have found before about COVID-19: if you have genetic problems that make you less likely to get COVID-19, it's often because they affect a protein on your cell surface that the virus uses to get in. This means that these findings might point to new ways to prevent or treat both COVID-19 and the flu by targeting these proteins.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) and influenza are respiratory illnesses caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and influenza viruses, respectively. Both diseases share symptoms and clinical risk factors1, but the extent to which these conditions have a common genetic etiology is unknown. This is partly because host genetic risk factors are well characterized for COVID-19 but not for influenza, with the largest published genome-wide association studies for these conditions including >2 million individuals2and about 1,000 individuals3,4,5,6, respectively. Shared genetic risk factors could point to targets to prevent or treat both infections. Through a genetic study of 18,334 cases with a positive test for influenza and 276,295 controls, we show that published COVID-19 risk variants are not associated with influenza. Furthermore, we discovered and replicated an association between influenza infection and noncoding variants inB3GALT5andST6GAL1, neither of which was associated with COVID-19. In vitro small interfering RNA knockdown of ST6GAL1‚Äîan enzyme that adds sialic acid to the cell surface, which is used for viral entry‚Äîreduced influenza infectivity by 57%. These results mirror the observation that variants that downregulateACE2, the SARS-CoV-2 receptor, protect against COVID-19 (ref.7). Collectively, these findings highlight downregulation of key cell surface receptors used for viral entry as treatment opportunities to prevent COVID-19 and influenza.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41588-024-01844-1" target="_blank">https://www.nature.com/articles/s41588-024-01844-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-05</strong> <a href="https://www.nature.com/articles/s41598-024-68816-7" target="_blank">
            ü§ñ Detecting and Differentiating Low and High Risk Java Virus Infections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-26', this)">
            More</button></h4>
            <div id="toggle-26" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Detection and differentiation of low virulence and virulentOrthoavulavirus javaenseusing a molecular beacon with RT-LAMP</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Newcastle disease is a serious illness that affects chickens and other birds. To diagnose it, scientists use a special test called PCR (polymerase chain reaction). However, this test can take some time to get results.

Researchers have developed a new way to detect Newcastle disease using a technique called MB-RT-LAMP. This method combines two tests in one: it finds the virus and tells whether it's mild or severe at the same time.

Studies show that this new method is very sensitive, meaning it can detect small amounts of the virus. It's also very specific, meaning it only identifies Newcastle disease and not other viruses. The test is even able to tell if the virus has changed into a more severe form.

What's great about this new method is that it doesn't require any special equipment or extra steps like isolating RNA. This makes it perfect for use in veterinary laboratories with limited resources. By using MB-RT-LAMP, scientists can quickly identify Newcastle disease and its severity, making it easier to control the spread of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Newcastle disease (ND), an economically important disease in poultry, is caused by virulent strains of the genetically diverseOrthoavulavirus javaense(OAVJ). Laboratories rely on quantitative real-time reverse transcription PCR (qRT-PCR) to detect OAVJ and differentiate between OAVJ pathotypes. This study demonstrates that a fusion cleavage site based molecular beacon with reverse transcription loop mediated isothermal amplification (MB-RT-LAMP) assay can detect and differentiate OAVJ pathotypes in a single assay. Data show that the assay can rapidly identify diverse OAVJ genotypes with sensitivity only one log-fold lower than the current fusion qRT-PCR assay (104copies), exhibits a high degree of specificity for OAVJ, and the molecular beacon can differentiate mesogenic/velogenic sequences from lentogenic sequences. Further, data show that a two-minute rapid lysis protocol preceding MB-RT-LAMP can detect and differentiate OAVJ RNA from both spiked samples and oropharyngeal swabs without the need for RNA isolation. As the MB-RT-LAMP assay can rapidly detect and discriminate between lentogenic and mesogenic/velogenic sequences of OAVJ within one assay, without the need for RNA isolation, and is adaptable to existing veterinary diagnostic laboratory workflow without additional equipment, this assay could be a rapid primary screening tool before qRT-PCR based validation in resource limited settings.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68816-7" target="_blank">https://www.nature.com/articles/s41598-024-68816-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-04</strong> <a href="https://www.nature.com/articles/s41467-024-50760-9" target="_blank">
            ü§ñ Vaccine Comparison in Adults: A Phase IV Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-27', this)">
            More</button></h4>
            <div id="toggle-27" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multiple vaccine comparison in the same adults reveals vaccine-specific and age-related humoral response patterns: an open phase IV trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simpler version of the abstract:

As people get older, their bodies often don't respond as well to vaccines. This makes it harder to create effective vaccination strategies for older adults who are at higher risk of getting infectious diseases. Researchers studied over 300 people from different age groups - young adults (25-49), middle-aged adults (50-64), and older adults (65+). They were given a flu vaccine, followed by a pneumococcal conjugate vaccine, and then two SARS-CoV-2 mRNA vaccines. The study found that older adults had lower antibody levels after receiving the pneumococcal conjugate and SARS-CoV-2 mRNA vaccines compared to younger adults. It also showed that some people, mostly older men, didn't respond well to multiple vaccines. This research can help identify groups at risk of low vaccine responsiveness and inform targeted vaccination strategies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccine responsiveness is often reduced in older adults. Yet, our lack of understanding of low vaccine responsiveness hampers the development of effective vaccination strategies to reduce the impact of infectious diseases in the ageing population. Young-adult (25‚Äì49‚Äây), middle-aged (50-64‚Äây) and older-adult (‚Äâ‚â•‚Äâ65‚Äây) participants of the VITAL clinical trials (n‚Äâ=‚Äâ315, age-range: 28-98‚Äây), were vaccinated with an annual (2019‚Äì2020) quadrivalent influenza (QIV) booster vaccine, followed by a primary 13-valent pneumococcal-conjugate (PCV13) vaccine (summer/autumn 2020) and a primary series of two SARS-CoV-2 mRNA-1273 vaccines (spring 2021). This unique setup allowed investigation of humoral responsiveness towards multiple vaccines within the same individuals over the adult age-range. Booster QIV vaccination induced comparable H3N2 hemagglutination inhibition (HI) titers in all age groups, whereas primary PCV13 and mRNA-1273 vaccination induced lower antibody concentrations in older as compared to younger adults (primary endpoint). The persistence of humoral responses, towards the 6 months timepoint, was shorter in older adults for all vaccines (secondary endpoint). Interestingly, highly variable vaccine responder profiles overarching multiple vaccines were observed. Yet, approximately 10% of participants, mainly comprising of older male adults, were classified as low responders to multiple vaccines. This study aids the identification of risk groups for low vaccine responsiveness and hence supports targeted vaccination strategies. Trial number: NL69701.041.19, EudraCT: 2019-000836-24.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50760-9" target="_blank">https://www.nature.com/articles/s41467-024-50760-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-03</strong> <a href="https://www.nature.com/articles/s41541-024-00932-x" target="_blank">
            ü§ñ Boosting Immunity to Bird Flu: How Self-Amplifying mRNA Vaccines Work in the Body</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-28', this)">
            More</button></h4>
            <div id="toggle-28" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity and biodistribution of lipid nanoparticle formulated self-amplifying mRNA vaccines against H5 avian influenza</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Immunogenicity and Biodistribution of mRNA Vaccines Against H5 Hemagglutinin in Mice

Scientists created four special vaccines that use self-amplifying (sa) mRNA to teach mice's immune systems how to fight a specific virus. These vaccines contained different parts of the virus's hemagglutinin protein, which is like a key that helps the virus attach to and enter host cells.

The scientists tested these vaccines in mice and found that all four versions worked by triggering an adaptive immune response. However, the version with the full-length hemagglutinin protein was more effective than the others.

They also discovered that the strength of the immune response correlated with the dose of the vaccine. Moreover, the vaccines triggered not only antibody responses but also cellular immune responses and the production of a specific type of antibody called IgA in the lungs.

Interestingly, when the scientists injected one of these special vaccines into mice's muscles, they found that it spread not only to the injection site but also to other parts of the body, including the draining lymph nodes, spleen, lung, kidney, liver, and heart.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study reports on the immunogenicity and biodistribution of H5 hemagglutinin (HA)-based self-amplifying (sa) mRNA vaccines in mice. Four sa-mRNA vaccines encoding either a secreted full-length HA, a secreted HA head domain, a secreted HA stalk domain, or a full-length membrane-anchored HA were investigated. All vaccines elicited an adaptive immune response. However, the full-length HA sa-RNA vaccines demonstrated superior performance compared to head and stalk domain vaccines. The antibody titers positively correlated with the vaccine dose. Cellular immune responses and antigen-specific IgA antibodies in the lungs were also observed. The comparison of the sa-mRNA vaccines encoding the secreted and membrane-anchored full-length HA revealed that anchoring of the HA to the membrane significantly enhanced the antibody and cellular responses. In addition to the injection site, the intramuscularly injected sa-mRNA-LNPs were also detected in the draining lymph nodes, spleen, and to a lesser extent, in the lung, kidney, liver, and heart.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00932-x" target="_blank">https://www.nature.com/articles/s41541-024-00932-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s41467-024-50875-z" target="_blank">
            ü§ñ Unpacking Proteomic Secrets at a Systems Level</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-29', this)">
            More</button></h4>
            <div id="toggle-29" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>OmicScope unravels systems-level insights from quantitative proteomics data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine you have a big puzzle with many pieces that need to be analyzed together. This is what happens in shotgun proteomics, where scientists try to understand all the proteins in a sample. But this process can be very complicated and requires using different tools to get meaningful results. OmicScope is a new tool that helps solve these problems by taking in various types of data, preparing it for analysis (like joining pieces together, fixing missing information, and making sure everything is on the same scale), and then performing detailed analyses to see what's different between groups or over time. It also has a feature that combines results from multiple studies to give a broader understanding. Plus, OmicScope comes with tools to visualize the data and can be used in both computer programs (like Python) and through a web application, making it accessible to researchers and helping them get accurate and efficient results more easily.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Shotgun proteomics analysis presents multifaceted challenges, demanding diverse tool integration for insights. Addressing this complexity, OmicScope emerges as an innovative solution for quantitative proteomics data analysis. Engineered to handle various data formats, it performs data pre-processing ‚Äì including joining replicates, normalization, data imputation ‚Äì and conducts differential proteomics analysis for both static and longitudinal experimental¬†designs. Empowered by Enrichr with over 224 databases, OmicScope performs Over Representation Analysis (ORA) and Gene Set Enrichment Analysis (GSEA). Additionally, its Nebula module facilitates meta-analysis from independent datasets, providing a systems biology approach for enriched insights. Complete with a data visualization toolkit and accessible as Python package and a web application, OmicScope democratizes proteomics analysis, offering an efficient and high-quality pipeline for researchers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50875-z" target="_blank">https://www.nature.com/articles/s41467-024-50875-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s44325-024-00011-z" target="_blank">
            ü§ñ Long COVID Heart Risks: A Study in Hong Kong</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-30', this)">
            More</button></h4>
            <div id="toggle-30" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk factors for long-term cardiovascular post-acute sequelae of COVID-19 infection: A nested case-control study in Hong Kong</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who got very sick with COVID-19 can experience problems even after they recover. These problems can affect their heart and blood vessels. Researchers are trying to figure out what makes some people more likely to get these heart-related problems.

In this study, scientists looked at the medical records of over 3,400 people in Hong Kong who had heart-related problems after having COVID-19. They compared these patients with a group of people who didn't have any heart-related problems after having COVID-19.

The scientists found that two things made it more likely for someone to get heart-related problems: if they had been very sick with COVID-19 and were hospitalized, or if they already had some other blood vessel disease. On the other hand, getting more doses of the COVID-19 vaccine seemed to protect against heart-related problems, and people who visited their doctor more often were also less likely to get these problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>People with COVID-19 can experience post-acute sequelae of SARS-CoV-2 (PASC). Studies on risk factors of PASC outcomes are ongoing, especially for endocrine system-related diseases that may impact the cardiovascular system. Cardiac-related PASC is one of the burdens after COVID-19 infection. This study aimed to examine the risk factors of cardiac-related PASC. In this nested case-control study, we obtained electronic health records (EHRs) database from the Hong Kong Hospital Authority. We defined cases as patients with at least one cardiac-related PASC and controls as patients without any cardiac-related PASC. We applied the incidence density sampling and matched controls to cases on age and sex at a 1:10 ratio. Multivariable conditional logistic regression was used to determine the associations between risk factors and cardiac-related PASC. A total of 455 individuals with cardiac-related PASC and matched 3,423 controls were obtained in the underlying cohort. COVID-19-associated hospitalisation (aOR: 1.41, 95% CI: 1.03‚Äì1.93) and peripheral vascular disease (aOR: 2.98, 95% CI: 1.31‚Äì6.79) were associated with an increased likelihood of cardiac-related PASC. Higher doses of the COVID-19 vaccine (2 doses: 0.68 [0.52‚Äì0.89]; ‚â•3 doses: 0.56 [0.40‚Äì0.78]) and more frequent healthcare utilization visits (aOR: 0.95, 95% CI: 0.92‚Äì0.97) were associated with a lower likelihood of cardiac-related PASC. This is the first study to examine risk factors of cardiac-related PASC among the Chinese population. We identified peripheral vascular disease and COVID-19-associated hospitalisation as the risk factors for cardiac-related PASC. COVID-19 vaccination was protective against cardiac-related PASC, which should be prioritized for high-risk patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44325-024-00011-z" target="_blank">https://www.nature.com/articles/s44325-024-00011-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s44298-024-00047-z" target="_blank">
            ü§ñ Uncovering Hidden Patterns in Enterovirus D68 Outbreaks</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-31', this)">
            More</button></h4>
            <div id="toggle-31" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inferring enterovirus D68 transmission dynamics from the genomic data of two 2022 North American outbreaks</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person could understand:

A type of virus called Enterovirus D68 (EV-D68) has been causing a lot of respiratory problems in kids all over the world. In 2014, there was a big outbreak of EV-D68 in North America. Recently, there was another outbreak in Ontario, Canada from August to October 2022. Our scientists did some genetic testing and found that the virus that spread in Ontario was very similar to one that was spreading in Maryland at the same time. We used computer models to figure out how quickly the virus was spreading (this is called the "reproduction number") and found that it was around 2.7 in Ontario and 2.1 in Maryland. We also found out that people were usually sick with EV-D68 for about 8 days on average. Interestingly, we discovered that the virus started spreading again when social distancing rules were relaxed after COVID-19. This suggests that when we ease up on precautions to control one disease, another might take advantage of it and spread more easily. Our study shows how using genetic information can help us track and understand outbreaks, which is important for keeping people safe and healthy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Enterovirus D68 (EV-D68) has emerged as a significant cause of acute respiratory illness in children globally, notably following its extensive outbreak in North America in 2014. A recent outbreak of EV-D68 was observed in Ontario, Canada, from August to October 2022. Our phylogenetic analysis revealed a notable genetic similarity between the Ontario outbreak and a concurrent outbreak in Maryland, USA. Utilizing Bayesian phylodynamic modeling on whole genome sequences (WGS) from both outbreaks, we determined the median peak time-varying reproduction number (Rt) to be 2.70, 95% HPD (1.76, 4.08) in Ontario and 2.10, 95% HPD (1.41, 3.17) in Maryland. The Rttrends in Ontario closely matched those derived via EpiEstim using reported case numbers. Our study also provides new insights into the median infection duration of EV-D68, estimated at 7.94 days, 95% HPD (4.55, 12.8) in Ontario and 10.8 days, 95% HPD (5.85, 18.6) in Maryland, addressing the gap in the existing literature surrounding EV-D68‚Äôs infection period. We observed that the estimated Time since the Most Recent Common Ancestor (TMRCA) and the epidemic‚Äôs origin coincided with the easing of COVID-19 related social contact restrictions in both areas. This suggests that the relaxation of non-pharmaceutical interventions, initially implemented to control COVID-19, may have inadvertently facilitated the spread of EV-D68. These findings underscore the effectiveness of phylodynamic methods in public health, demonstrating their broad application from local to global scales and underscoring the critical role of pathogen genomic data in enhancing public health surveillance and outbreak characterization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00047-z" target="_blank">https://www.nature.com/articles/s44298-024-00047-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-02</strong> <a href="https://www.nature.com/articles/s41598-024-68100-8" target="_blank">
            ü§ñ Stimulating Muscle Strength After COVID-19 Infection Recovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-32', this)">
            More</button></h4>
            <div id="toggle-32" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Gastrocnemius electrical stimulation increases ankle dorsiflexion strength in patients with post-acute sequelae of SARS-COV-2 (PASC): a double-blind randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine you've been very sick with COVID-19 for a long time, and your muscles have weakened significantly. This is called Post-Acute Sequelae of SARS-CoV-2 (PASC). We wanted to see if using electrical stimulation on the back of your legs could help improve muscle strength. To do this, we asked 18 people with PASC to either use a real device that sends electrical signals to their muscles or a fake device for one hour each day over four weeks. We then measured how well they could lift their feet and how strong their leg muscles were after four weeks. We found out that the people who used the real device had significantly stronger legs compared to those who used the fake device. This improvement was even more noticeable when we looked at how well their muscles responded to the electrical stimulation. However, using this method did not improve other daily activities such as bathing or walking for them. Using electrical stimulation on the back of your legs may help strengthen your leg muscles if you have PASC, but more people and longer treatments would be needed to confirm these results.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-Acute sequelae of SARS-CoV-2 (PASC) is a multisystem disorder causing persistent musculoskeletal deconditioning and reduced lower extremity strength. Electrical stimulation (E-Stim) to the gastrocnemius muscle can enhance strength outcomes by increasing the frequency of muscle fiber activation. We investigated its effect on individuals with PASC. Participants were randomized into intervention (IG) or control (CG) groups. The IG self-administered daily one-hour E-Stim to both their gastrocnemius muscles using a functional device over 4-week, while the CG used a sham device. Primary outcomes were ankle dorsiflexion strength assessed via dynamometry during maximum voluntary contractions, and gastrocnemius voluntary activation (GVA) via surface electromyography. The secondary outcome assessed activities of daily living (ADL), instrumental ADL, and mobility queries. Percentage improvement was calculated. Eighteen patients were analyzed (IG‚Äâ=‚Äâ10; CG‚Äâ=‚Äâ8). After 4¬†week, the IG showed a significantly higher improvement in ankle dorsiflexion strength (222.64%) compared to the CG (51.27%,p=‚Äâ0.002). Additionally, the IG‚Äôs ankle dorsiflexion strength improvement significantly correlated with GVA improvement (rho‚Äâ=‚Äâ0.782) at 4¬†week. The secondary outcomes did not reveal significant changes in neither group. Self-administered gastrocnemius E-Stim improves ankle dorsiflexion strength in individuals with PASC. However, larger sample sizes and longer interventions are needed to validate these findings.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68100-8" target="_blank">https://www.nature.com/articles/s41598-024-68100-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41467-024-50614-4" target="_blank">
            ü§ñ Understanding Vaccine Hesitancy: A Web Search Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-33', this)">
            More</button></h4>
            <div id="toggle-33" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Measuring vaccination coverage and concerns of vaccine holdouts from web search logs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To create good plans for vaccines, people in charge need accurate information about who has gotten vaccinated, who is still deciding, and why they are making those decisions. However, the data available in the US is not very detailed: sometimes vaccination rates are reported late or not broken down enough, and surveys about vaccine hesitation are limited because they ask high-level questions and rely on people's own reports.

We used search engine logs and machine learning to help fill these gaps. We looked at anonymized Bing data from February to August 2021. First, we developed a tool that can accurately tell when someone is searching for the COVID-19 vaccine on Bing. Our tool agrees with the CDC's vaccination rates most of the time, but reports them a week or two before the CDC does. It also estimates more detailed vaccination rates at the ZIP code level, which shows how different areas have varying levels of vaccine-seeking.

To study why people are hesitant to get vaccinated, we used our tool to identify two groups: those who got vaccinated early and those who held out. We found that holdouts were 67% more likely to click on websites that aren't trusted, and they were much more concerned about things like vaccine requirements, how vaccines are developed, and myths about vaccines. Even within the group of holdouts, there were clusters with different concerns and levels of openness to getting vaccinated.

Finally, we looked at how people's concerns and vaccine-seeking behaviors changed over time, and found that certain indicators could predict when someone would go from being hesitant to seeking the vaccine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To design effective vaccine policies, policymakers need detailed data about who has been vaccinated, who is holding out, and why. However, existing data in the US are insufficient: reported vaccination rates are often delayed or not granular enough, and surveys of vaccine hesitancy are limited by high-level questions and self-report biases. Here we show how search engine logs and machine learning can help to fill these gaps, using anonymized Bing data from February to August 2021. First, we develop avaccine intent classifierthat accurately detects when a user is seeking the COVID-19 vaccine on Bing. Our classifier demonstrates strong agreement with CDC vaccination rates, while preceding CDC reporting by 1‚Äì2‚Äâweeks, and estimates more granular ZIP-level rates, revealing local heterogeneity in vaccine seeking. To study vaccine hesitancy, we use our classifier to identify two groups,vaccine early adoptersandvaccine holdouts. We find that holdouts, compared to early adopters matched on covariates, are 67% likelier to click on untrusted news sites, and are much more concerned about vaccine requirements, development, and vaccine myths. Even within holdouts, clusters emerge with different concerns and openness to the vaccine. Finally, we explore the temporal dynamics of vaccine concerns and vaccine seeking, and find that key indicators predict when individuals convert from holding out to seeking the vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50614-4" target="_blank">https://www.nature.com/articles/s41467-024-50614-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41522-024-00538-0" target="_blank">
            ü§ñ COVID-19 Linked to Changes in Gut Bacteria and Inflammation Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-34', this)">
            More</button></h4>
            <div id="toggle-34" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Gut microbiota dysbiosis is associated with altered tryptophan metabolism and dysregulated inflammatory response in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The way COVID-19 develops in the body can be unpredictable. We looked at samples from people who were sick with COVID-19 and compared them to people who weren't infected. We found that people who got very sick had lower levels of good bacteria in their intestines, but this wasn't as clear for people who got antibiotics. The levels of certain chemicals in the blood also changed - some increased while others decreased. This was linked to changes in the types and levels of good bacteria, and also with an increase in chemicals that cause inflammation. Overall, our study found connections between changes in the microbiome and metabolism that might contribute to severe COVID-19 symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The clinical course of COVID-19 is variable and often unpredictable. To test the hypothesis that disease progression and inflammatory responses associate with alterations in the microbiome and metabolome, we analyzed metagenome, metabolome, cytokine, and transcriptome profiles of repeated samples from hospitalized COVID-19 patients and uninfected controls, and leveraged clinical information and post-hoc confounder analysis. Severe COVID-19 was associated with a depletion of beneficial intestinal microbes, whereas oropharyngeal microbiota disturbance was mainly linked to antibiotic use. COVID-19 severity was also associated with enhanced plasma concentrations of kynurenine and reduced levels of several other tryptophan metabolites, lysophosphatidylcholines, and secondary bile acids. Moreover, reduced concentrations of various tryptophan metabolites were associated with depletion ofFaecalibacterium, and tryptophan decrease and kynurenine increase were linked to enhanced production of inflammatory cytokines. Collectively, our study identifies correlated microbiome and metabolome alterations as a potential contributor to inflammatory dysregulation in severe COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41522-024-00538-0" target="_blank">https://www.nature.com/articles/s41522-024-00538-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-01</strong> <a href="https://www.nature.com/articles/s41467-024-50931-8" target="_blank">
            ü§ñ Stopping the Spread: A Small Molecule Inhibitor Blocks COVID-19 Replication and Transmission</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-35', this)">
            More</button></h4>
            <div id="toggle-35" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The small molecule inhibitor of SARS-CoV-2 3CLpro EDP-235 prevents viral replication and transmission in vivo</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has caused many deaths and economic problems worldwide. While some medicines have helped fight the virus, they haven't been effective for everyone and are not always available. A new medicine called EDP-235 is being developed to stop a specific part of the SARS-CoV-2 virus that causes COVID-19 from working. It's shown to be very effective against all known versions of the virus and also works against other related viruses. This medicine has an advantage over others because it still works even if the virus has developed some resistance. Additionally, EDP-235 is very safe for humans with a large safety margin. In experiments on hamsters and ferrets, EDP-235 was able to stop the virus from spreading and causing harm. It also prevented the virus from being transmitted between animals that were in close contact. Overall, these results suggest that EDP-235 is a powerful medicine that can stop many types of viruses and might be useful for treating COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has led to the deaths of millions of people and severe global economic impacts. Small molecule therapeutics have played an important role in the fight against SARS-CoV-2, the virus responsible for COVID-19, but their efficacy has been limited in scope and availability, with many people unable to access their benefits, and better options are needed. EDP-235 is specifically designed to inhibit the SARS-CoV-2 3CLpro, with potent nanomolar activity against all SARS-CoV-2 variants to date, as well as clinically relevant human and zoonotic coronaviruses. EDP-235 maintains potency against variants bearing mutations associated with nirmatrelvir resistance. Additionally, EDP-235 demonstrates a‚Äâ‚â•‚Äâ500-fold selectivity index against multiple host proteases. In a male Syrian hamster model of COVID-19, EDP-235 suppresses SARS-CoV-2 replication and viral-induced hamster lung pathology. In a female ferret model, EDP-235 inhibits production of SARS-CoV-2 infectious virus and RNA at multiple anatomical sites. Furthermore, SARS-CoV-2 contact transmission does not occur when na√Øve ferrets are co-housed with infected, EDP-235-treated ferrets. Collectively, these results demonstrate that EDP-235 is a broad-spectrum coronavirus inhibitor with efficacy in animal models of primary infection and transmission.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50931-8" target="_blank">https://www.nature.com/articles/s41467-024-50931-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41467-024-49634-x" target="_blank">
            ü§ñ COVID-19 Vaccination Safety: A Study of 46 Million Adults in England</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-36', this)">
            More</button></h4>
            <div id="toggle-36" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cohort study of cardiovascular safety of different COVID-19 vaccination doses among 46 million adults in England</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The first dose of some COVID-19 vaccines helped lower heart problems in a big group of adults in England. But we didn't know if getting second or booster doses would have the same effect. Our study looked at health records from 45.7 million adults in England and compared heart problem rates before, during, and after their first, second, and booster doses. We found that getting any dose of COVID-19 vaccine helped lower common heart problems like heart attacks and strokes. However, we also noticed a small increase in rare heart issues like blood clots after the first dose of some vaccines and heart inflammation after mRNA vaccine doses. These results suggest that it's still safe to get vaccinated against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The first dose of COVID-19 vaccines led to an overall reduction in cardiovascular events, and in rare cases, cardiovascular complications. There is less information about the effect of second and booster doses on cardiovascular diseases. Using longitudinal health records from 45.7 million adults in England between December 2020 and January 2022, our study compared the incidence of thrombotic and cardiovascular complications up to 26 weeks after first, second and booster doses of brands and combinations of COVID-19 vaccines used during the UK vaccination program with the incidence before or without the corresponding vaccination. The incidence of common arterial thrombotic events (mainly acute myocardial infarction and ischaemic stroke) was generally lower after each vaccine dose, brand and combination. Similarly, the incidence of common venous thrombotic events, (mainly pulmonary embolism and lower limb deep venous thrombosis) was lower after vaccination. There was a higher incidence of previously reported rare harms after vaccination: vaccine-induced thrombotic thrombocytopenia after first ChAdOx1 vaccination, and myocarditis and pericarditis after first, second and transiently after booster mRNA vaccination (BNT-162b2 and mRNA-1273). These findings support the wide uptake of future COVID-19 vaccination programs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49634-x" target="_blank">https://www.nature.com/articles/s41467-024-49634-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41598-024-68592-4" target="_blank">
            ü§ñ Helping Sicker COVID Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-37', this)">
            More</button></h4>
            <div id="toggle-37" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of additional hemoperfusion in hospitalized patients with severe to critical COVID-19 disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The use of a special machine to remove bad stuff from the blood of very sick COVID-19 patients has not been proven to help them live longer. But it did seem to make their stay in the hospital shorter and helped prevent some serious complications like lung problems, sepsis, and bleeding. However, this treatment did not improve the number of patients who survived.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The evidence supporting additional hemoperfusion (HP) with cytokine adsorbents for improving clinical outcomes in severe to critical coronavirus disease 2019 (COVID-19) patients remains limited. We compared severe to critical COVID-19 patients who received additional HP with a cytokine adsorbent to matched cases receiving standard medical treatment (SMT). The primary outcome was hospital mortality. In our study, we matched 45 patients who received additional HP 1:1 with the SMT group based on key clinical parameters. The hospital mortality rates did not differ between the groups (33% vs 38%,p=‚Äâ0.83). The HP group had a significantly shorter ICU stay (22 vs 32¬†days; p‚Äâ=‚Äâ0.017) and reduced mechanical ventilation duration (15 vs 35¬†days;p<‚Äâ0.001). Additionally, the incidence of pulmonary complications (20% vs 42%; p‚Äâ=‚Äâ0.04), sepsis (38% vs 64%;p=‚Äâ0.02), and disseminated intravascular coagulopathy (DIC) (13% vs 33%;p=‚Äâ0.046) were significantly lower in the HP group. In conclusion,¬†among severe to critical COVID-19 patients, additional HP with a cytokine adsorbent did not improve hospital mortality. However, it reduced ICU length of stay, mechanical ventilator days, and incidences of lung complications, sepsis, and DIC.Trial registration: TCTR20231002006. Registered 02 October 2023 (retrospectively registered).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68592-4" target="_blank">https://www.nature.com/articles/s41598-024-68592-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-31</strong> <a href="https://www.nature.com/articles/s41598-024-68617-y" target="_blank">
            ü§ñ Uncovering the Impact of 1mŒ® on Positive-Stranded RNA Virus-Based Replicons</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-38', this)">
            More</button></h4>
            <div id="toggle-38" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>1mŒ® influences the performance of various positive-stranded RNA virus-based replicons</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT 
                        Self-replicating RNAs (saRNAs) are special tools that use a type of enzyme from viruses to make more copies of themselves once inside the target cell. This helps spread the information about an antigen, which is essentially a part of what triggers the immune system's response against a specific disease. Recently, there have been several clinical trials using saRNA vaccines as they potentially offer lower vaccination doses compared to traditional mRNA-based approaches. A modification called N1- methyl-pseudouridine (1mŒ®) was also included in these newer mRNA vaccines. It enhances RNA stability and reduces the immune response triggered by RNAs. In this study, we looked at how this modified nucleoside affects different saRNA platforms based on various viruses. The results showed that it impacted different stages of replication depending on the virus. For example, an insect virus called TNCL had its replication impaired because the viral RNA was not recognized well by the RdRp enzyme. On the other hand, the translation step was severely affected in coxsackievirus B3 (CVB3), which belongs to the Picornaviridae family. Lastly, the effects of 1mŒ® on Semliki forest virus (SFV) were not harmful in vitro studies but did not offer any benefits when immunogenicity was tested in vivo.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Self-amplifying RNAs (saRNAs) are versatile vaccine platforms that take advantage of a viral RNA-dependent RNA polymerase (RdRp) to amplify the messenger RNA (mRNA) of an antigen of interest encoded within the backbone of the viral genome once inside the target cell. In recent years, more saRNA vaccines have been clinically tested with the hope of reducing the vaccination dose compared to the conventional mRNA approach. The use of N1-methyl-pseudouridine (1mŒ®), which enhances RNA stability and reduces the innate immune response triggered by RNAs, is among the improvements included in the current mRNA vaccines. In the present study, we evaluated the effects of this modified nucleoside on various saRNA platforms based on different viruses. The results showed that different stages of the replication process were affected depending on the backbone virus. For TNCL, an insect virus of theAlphanodavirusgenus, replication was impaired by poor recognition of viral RNA by RdRp. In contrast, the translation step was severely abrogated in coxsackievirus B3 (CVB3), a member of thePicornaviridaefamily. Finally, the effects of 1mŒ® on Semliki forest virus (SFV), were not detrimental in in vitro studies, but no advantages were observed when immunogenicity was tested in vivo.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68617-y" target="_blank">https://www.nature.com/articles/s41598-024-68617-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-30</strong> <a href="https://www.nature.com/articles/s41598-024-68836-3" target="_blank">
            ü§ñ Predicting Monkeypox Outbreaks with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-39', this)">
            More</button></h4>
            <div id="toggle-39" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Human monkeypox disease prediction using novel modified restricted Boltzmann machine-based equilibrium optimizer</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
 
A new virus called monkeypox has started to spread in many countries, causing concern about a possible global pandemic. The symptoms of monkeypox are similar to those of chickenpox and measles, but with some differences like different types of skin blisters. Deep learning techniques have been successfully used in the past to predict diseases from images. This paper uses deep learning methods to predict if someone has monkeypox based on images. We first collected data from a standard dataset called Monkeypox Skin Lesion Dataset and pre-processed it by resizing, normalizing, and augmenting the images. Then, we used a technique called Convolutional Block Attention Module (CBAM) to extract features from the images. Finally, we used a modified version of Restricted Boltzmann Machine (RBM) to make predictions, with parameters tuned using an optimization algorithm inspired by nature called Equilibrium Optimizer (EO). Our results show that our model performs better than other models at predicting monkeypox. Specifically, it has 75.68%, 70%, 60.87%, and 43.75% lower Root Mean Squared Error (RMSE) compared to PSO-SVM, Xception-CBAM-Dense, ShuffleNet, and RBM respectively. Similarly, our model has 9.22%, 7.75%, 3.77%, and 10.90% higher accuracy compared to PSO-SVM, Xception-CBAM-Dense, ShuffleNet, and RBM respectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While the globe continues to struggle to recover from the devastation brought on by the COVID-19 virus's extensive distribution, the recent worrying rise in human monkeypox outbreaks in several nations raises the possibility of a novel worldwide pandemic. The symptoms of human monkeypox resemble those of chickenpox and traditional measles, with a few subtle variations like the various kinds of skin blisters. A range of deep learning techniques have demonstrated encouraging results in image-oriented tumor cell, Covid-19 diagnosis, and skin disease prediction tasks. Hence, it becomes necessary to perform the prediction of the new monkeypox disease using deep learning techniques. In this paper, an image-oriented human monkeypox disease prediction is performed with the help of novel deep learning methodology. Initially, the data is gathered from the standard benchmark dataset called Monkeypox Skin Lesion Dataset. From the collected data, the pre-processing is accomplished using image resizing and image normalization as well as data augmentation techniques. These pre-processed images undergo the feature extraction that is performed by the Convolutional Block Attention Module (CBAM) approach. The extracted features undergo the final prediction phase using the Modified Restricted Boltzmann Machine (MRBM), where the parameter tuning in RBM is accomplished by the nature inspired optimization algorithm referred to as Equilibrium Optimizer (EO), with the consideration of error minimization as the major objective function. Simulation findings demonstrate that the proposed model performed better than the remaining models at monkeypox prediction. The proposed MRBM-EO for the suggested human monkeypox disease prediction model in terms of RMSE is 75.68%, 70%, 60.87%, and 43.75% better than PSO-SVM, Xception-CBAM-Dense, ShuffleNet, and RBM respectively. Similarly, the proposed MRBM-EO for the suggested human monkeypox disease prediction model with respect to accuracy is 9.22%, 7.75%, 3.77%, and 10.90% better than PSO-SVM, Xception-CBAM-Dense, ShuffleNet, and RBM respectively.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68836-3" target="_blank">https://www.nature.com/articles/s41598-024-68836-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-30</strong> <a href="https://www.nature.com/articles/s41598-024-68678-z" target="_blank">
            ü§ñ Tracking a New COVID Variant in India: What We Know</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-40', this)">
            More</button></h4>
            <div id="toggle-40" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Emergence of SARS-CoV-2 omicron variant JN.1 in Tamil Nadu, India - Clinical characteristics and novel mutations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that's easy to understand:

In December 2023, a new version of the COVID-19 virus called omicron became the most common variant in many parts of the world. We studied this new version, which we called JN.1, and found out what it looks like on a genetic level. We also looked at how its unique changes might affect the way it interacts with human cells. Our analysis showed that some of these changes could make it easier for the virus to attach to and enter human cells. These findings highlight the importance of keeping a close eye on the COVID-19 virus as it evolves, so we can stay ahead of any new threats and develop better ways to prevent and treat infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In December 2023, we observed a notable shift in the COVID-19 landscape, when JN.1 omicron emerged as the predominant SARS-CoV-2 variant with a 95% incidence. We characterized the clinical profile, and genetic changes in JN.1, an emerging SARS-CoV-2 variant of interest. Whole genome sequencing was performed on SARS-CoV-2 positive clinical specimens, followed by sequence analysis. Mutations within the spike protein sequences were analysed and compared with the previously reported lineages and sub-lineages, to identify the potential impact of the unique mutations on protein structure and possible alterations in the functionality. Several unique and dynamic mutations were identified herein. Molecular docking analysis showed changes in the binding affinity, and key interacting residues of wild-type and mutated structures with key host cell receptors of SARS-CoV-2 entry viz., ACE2, CD147, CD209L and AXL. Our data provides key insights on the emergence of newer variants and highlights the necessity for robust and sustained global genomic surveillance of SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68678-z" target="_blank">https://www.nature.com/articles/s41598-024-68678-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-49891-w" target="_blank">
            ü§ñ COVID-19 in Wild Animals: A Growing Concern</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-41', this)">
            More</button></h4>
            <div id="toggle-41" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Widespread exposure to SARS-CoV-2 in wildlife communities</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract:

Wild animals are getting infected with the COVID-19 virus, which is spread by humans. While it's known that many different species can get sick from COVID, scientists didn't know how widespread this problem was or what factors made some animals more likely to catch the virus. To find out, researchers tested 23 different wild animal species for COVID and looked at how urbanization and human activity affected their risk of getting infected. The study found that six types of wildlife - including deer mice, opossums, raccoons, groundhogs, cottontails, and red bats - had COVID-19 antibodies in them, indicating they had been exposed to the virus. Furthermore, areas with high human activity had three times more COVID-19 cases than areas with low human use. The researchers also found that the virus sequences from nine wild animals matched those circulating among humans at the time, suggesting at least seven recent human-to-animal transmission events. Overall, these findings indicate that many wildlife communities have been exposed to COVID-19 and that areas with high human activity may be hotspots for cross-species transmission.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pervasive SARS-CoV-2 infections in humans have led to multiple transmission events to animals. While SARS-CoV-2 has a potential broad wildlife host range, most documented infections have been in captive animals and a single wildlife species, the white-tailed deer. The full extent of SARS-CoV-2 exposure among wildlife communities and the factors that influence wildlife transmission risk remain unknown. We sampled 23 species of wildlife for SARS-CoV-2 and examined the effects of urbanization and human use on seropositivity. Here, we document positive detections of SARS-CoV-2 RNA in six species, including the deer mouse, Virginia opossum, raccoon, groundhog, Eastern cottontail, and Eastern red bat between May 2022‚ÄìSeptember 2023 across Virginia and Washington, D.C., USA. In addition, we found that sites with high human activity had three times higher seroprevalence than low human-use areas. We obtained SARS-CoV-2 genomic sequences from nine individuals of six species which were assigned to seven Pango lineages of the Omicron variant. The close match to variants circulating in humans at the time suggests at least seven recent human-to-animal transmission events. Our data support that exposure to SARS-CoV-2 has been widespread in wildlife communities and suggests that areas with high human activity may serve as points of contact for cross-species transmission.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49891-w" target="_blank">https://www.nature.com/articles/s41467-024-49891-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">
            ü§ñ Understanding How Our Lifestyle Choices Affect COVID-19 Recovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-42', this)">
            More</button></h4>
            <div id="toggle-42" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modifiable lifestyle factors and the risk of post-COVID-19 multisystem sequelae, hospitalization, and death</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Effective ways to prevent long-term problems after having COVID are important for patients, doctors, and policy makers to consider. This study looked at how different lifestyle choices (like not smoking, eating well, exercising regularly, getting enough sleep) affected the risk of severe COVID symptoms, death, and hospitalization in a large group of people from the UK Biobank cohort (n=‚Äâ68,896). The results showed that people who made healthy lifestyle choices had a 36% lower risk of long-term COVID complications compared to those who didn't make healthy choices. This benefit was seen across all body systems, including heart, blood clotting, metabolism, gut health, kidneys, mental health, muscles, breathing, and fatigue. The good effects were mostly due to the direct impact of lifestyle choices on the body, rather than being influenced by pre-existing medical conditions (71% for any sequelae). Healthy lifestyles also reduced the risk of death and hospitalization after COVID. These associations remained true even when looking at different stages of infection, whether or not people had been hospitalized, vaccinated, or infected with a specific variant of SARS-CoV-2. These findings highlight the importance of maintaining a healthy lifestyle to mitigate long-term COVID effects and prepare for future pandemics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Effective prevention strategies for post-COVID complications are crucial for patients, clinicians, and policy makers to mitigate their cumulative burden. This study evaluated the association of modifiable lifestyle factors (smoking, alcohol intake, BMI, physical activity, sedentary time, sleep duration, and dietary habits) with COVID-19 multisystem sequelae, death, and hospitalization in the UK Biobank cohort (n=‚Äâ68,896). A favorable lifestyle (6-10 healthy factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae (HR, 0.64; 95% CI, 0.58-0.69; ARR at 210 days, 7.08%; 95% CI, 5.98-8.09) compared to an unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions spanned all 10 organ systems, including cardiovascular, coagulation, metabolic, gastrointestinal, kidney, mental health, musculoskeletal, respiratory disorders, and fatigue. This beneficial effect was largely attributable to direct lifestyle impacts independent of corresponding pre-infection comorbidities (71% for any sequelae). A favorable lifestyle was also related to the risk of post-COVID death (HR 0.59, 0.52-0.66) and hospitalization (HR 0.78, 0.73-0.84). These associations persisted across acute and post-acute infection phases, irrespective of hospitalization status, vaccination, or SARS-CoV-2 variant. These findings underscore the clinical and public health importance of adhering to a healthy lifestyle in mitigating long-term COVID-19 adverse impacts and enhancing future pandemic preparedness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">https://www.nature.com/articles/s41467-024-50495-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41598-024-68264-3" target="_blank">
            ü§ñ Detecting Aflatoxin Poisoning with a New High-Tech Sensor</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-43', this)">
            More</button></h4>
            <div id="toggle-43" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>On-site sensing for aflatoxicosis poisoning via ultraviolet excitable aptasensor based on fluorinated ethylene propylene strip: a promising forensic tool</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The contamination of food in Southeast Asia and Africa with a toxic substance called Aflatoxin B1 is a big problem. This toxin can cause serious liver damage and even death if people eat contaminated food. We need a simple way to detect this toxin quickly and easily, especially when there are no special facilities available. Our research team has developed a new tool that uses a special film strip to detect Aflatoxin B1 using ultraviolet light. This method is easy to make, can be used on-site, and lasts for 19 days. It's also reusable four times! We tested this tool with real-world samples like brown sugar, peanuts, and rice, and the results were very close to those from a more complicated laboratory test called High-Performance Liquid Chromatography. This discovery is exciting because it uses a unique UV light pattern and a special film strip that could lead to new and innovative ways to detect environmental pollutants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The environmental contamination by extremophileAspergillusspecies, i.e., Aflatoxin B1, is hardly controllable in Southeast Asia and Sub-Saharan Africa, which lack handling resources and controlled storage facilities. Acute aflatoxicosis poisoning from aflatoxin-prone dietary staples could cause acute hepatic necrosis, acute liver failure, and death. Here, as the cheaper, more straightforward, and facile on-site diagnostic kit is needed, we report an ultraviolet-excitable optical aptasensor based on a fluorinated ethylene propylene film strip. Molecular dynamics on the aptamer.AFB1 complex revealed that the AFB1 to the aptamer increases the overall structural stability, suggesting that the aptamer design is suitable for the intended application. Under various influencing factors, the proposed label-free strategy offers a fast 20-min on-site fabrication simplicity and 19-day shelf-life. The one-pot incubation provides an alternative to catalytic detection and exhibited 4 times reusability. The recovery of crude brown sugar, processed peanuts, and long-grain rice were 102.74‚Äâ¬±‚Äâ0.41 (n‚Äâ=‚Äâ3), 86.90‚Äâ¬±‚Äâ3.38 (n‚Äâ=‚Äâ3), and 98.50‚Äâ¬±‚Äâ0.42 (n‚Äâ=‚Äâ3), comparable to High-Performance Liquid Chromatography-Photodiode Array Detector results. This study is novel owing to the peculiar UV-active spectrum fingerprint and the convenient use of hydrophobic film strips that could promote breakthrough innovations and new frontiers for on-site/forensic detection of environmental pollutants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68264-3" target="_blank">https://www.nature.com/articles/s41598-024-68264-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-27</strong> <a href="https://www.nature.com/articles/s41598-024-68184-2" target="_blank">
            ü§ñ COVID-19's Lasting Impact on Pregnant Women and Babies in China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-44', this)">
            More</button></h4>
            <div id="toggle-44" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The effect of long-term COVID-19 infection on maternal and fetal complications: a retrospective cohort study conducted at a single center in China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The effects of long-term COVID-19 on pregnant women and their babies were studied over a year at a hospital in China. Researchers compared 623 pregnant women who had COVID-19 during pregnancy, those who didn't have COVID-19, and some who still had COVID-19 after they gave birth. They found that the women who had long-term COVID-19 were more likely to get high blood pressure during pregnancy (called gestational hypertension) and to develop diabetes during pregnancy (called gestational diabetes mellitus). They also found that their babies might not grow as much inside the womb as other babies did, a condition called fetal intrauterine growth restriction. However, the study didn't find any significant changes in blood clotting or other complications like premature birth or baby problems after birth.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Investigate the effect of long-term COVID-19 on maternal and fetal complications. A retrospective cohort study was conducted. A total of 623 pregnant women who delivered in Kunming First People's Hospital from November 1, 2022 to July 31, 2023 were selected. By employing statistical methods, we compared the associations between maternal and fetal complications in pregnant women with acute COVID-19 during pregnancy, long-term COVID-19, and non-COVID-19 pregnant women. In the final 623 samples, there were 209 pregnant women with acute COVID-19, 72 pregnant women with long-term COVID-19, and 342 pregnant women without COVID-19. The epidemiological and clinical characteristics of all subjects were similar. Pregnant individuals who developed long-term COVID-19 during their pregnancy had an increased risk of experiencing gestational hypertension (OR 3.344, 95% CI 1.544‚Äì7.243), gestational diabetes mellitus (OR 2.301, 95% CI 1.290‚Äì4.102), and fetal intrauterine growth restriction (OR 2.817, 95% CI 1.385‚Äì5.952). Multivariate binary logistic regression analysis showed that this association remained consistent even after adjusting for confounders and performing subgroup analyses. Other maternal and fetal complications, such as premature rupture of membranes, preterm delivery, neonatal asphyxia, and transfer of neonates to NICU, did not exhibit statistically significant associations. After linear regression analysis, the platelet count (Œ≤:‚Äâ‚àí‚Äâ0.127, 95% CI‚Äâ‚àí‚Äâ0.001‚Äì0.000) of pregnant women with long-term COVID-19 was slightly lower than that of non-COVID-19 pregnant women, and the other coagulation parameters were not statistically significant. The incidence of gestational hypertension, gestational diabetes mellitus and fetal intrauterine growth restriction in pregnant women with long-term COVID-19 is significantly increased, but it does not further increase the coagulation status.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68184-2" target="_blank">https://www.nature.com/articles/s41598-024-68184-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-27</strong> <a href="https://www.nature.com/articles/s41467-024-50286-0" target="_blank">
            ü§ñ Unraveling the Complexity of Immune Response: A Study on Neutralizing Antibody Lineages in Macaques</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-45', this)">
            More</button></h4>
            <div id="toggle-45" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-compartmental diversification of neutralizing antibody lineages dissected in SARS-CoV-2 spike-immunized macaques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus is still changing, which means we need to understand how our bodies are responding to it. In this study, scientists used a combination of techniques to see what happens when monkeys are vaccinated with the part of the virus that causes COVID-19 (called the spike protein). They found that the immune cells in the monkeys' bodies were able to evolve and adapt to recognize not just the original form of the spike protein, but also new forms that might appear later. This shows that our bodies have a built-in ability to learn and improve their response to the virus over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continued evolution of SARS-CoV-2 underscores the need to understand qualitative aspects of the humoral immune response elicited by spike immunization. Here, we combine monoclonal antibody (mAb) isolation with deep B cell receptor (BCR) repertoire sequencing of rhesus macaques immunized with prefusion-stabilized spike glycoprotein. Longitudinal tracing of spike-sorted B cell lineages in multiple immune compartments demonstrates increasing somatic hypermutation and broad dissemination of vaccine-elicited B cells in draining and non-draining lymphoid compartments, including the bone marrow, spleen and, most notably, periaortic lymph nodes. Phylogenetic analysis of spike-specific monoclonal antibody lineages identified through deep repertoire sequencing delineates extensive intra-clonal diversification that shaped neutralizing activity. Structural analysis of the spike in complex with a broadly neutralizing mAb provides a molecular basis for the observed differences in neutralization breadth between clonally related antibodies. Our findings highlight that immunization leads to extensive intra-clonal B cell evolution where members of the same lineage can both retain the original epitope specificity and evolve to recognize additional spike variants not previously encountered.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50286-0" target="_blank">https://www.nature.com/articles/s41467-024-50286-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41598-024-66439-6" target="_blank">
            ü§ñ Progesterone Helps Reduce Inflammation in Airways</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-46', this)">
            More</button></h4>
            <div id="toggle-46" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Progesterone modulates the immune microenvironment to suppress ovalbumin-induced airway inflammation by inhibiting NETosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Asthma is a serious breathing problem that affects many people. Before puberty, girls tend to have fewer and less severe symptoms than boys. Scientists wanted to explore if the female sex hormone progesterone (P4) can help reduce inflammation in the lungs of mice with asthma-like symptoms caused by an allergic reaction. They studied two groups of female mice: one group had their ovaries removed to see how it would affect the results.

The scientists collected samples from the mice's blood, lung fluid, and tissues, and analyzed them. They found that P4 treatment helped reduce inflammation in the lungs, decreased mucus production, and reduced the number of immune cells in the lung fluid. It also increased the levels of certain beneficial cells while decreasing those that contribute to inflammation.

P4 was shown to inhibit a process called NETosis, which is linked to inflammation and airway damage. Additionally, it suppressed the activity of an enzyme involved in this process, leading to reduced production of reactive oxygen species. Furthermore, P4 treatment hindered the release of double-stranded DNA during NETosis.

These findings suggest that P4 may be beneficial in reducing inflammation associated with allergic asthma by modulating the immune microenvironment. The research indicates the potential of P4 as a therapeutic agent for ameliorating inflammation in OVA-induced allergic asthma mice.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Studies have demonstrated that prior to puberty, girls have a lower incidence and severity of asthma symptoms compared to boys. This study aimed to explore the role of progesterone (P4), a sex hormone, in reducing inflammation and altering the immune microenvironment in a mouse model of allergic asthma induced by OVA. Female BALB/c mice with or without ovariectomy to remove the influence of sex hormones were used for the investigations. Serum, bronchoalveolar lavage fluid (BALF), and lung tissue samples were collected for analysis. The results indicated that P4 treatment was effective in decreasing inflammation and mucus secretion in the lungs of OVA-induced allergic asthma mice. P4 treatment also reduced the influx of inflammatory cells into the BALF and increased the levels of Th1 and Th17 cytokines while decreasing the levels of Th2 and Treg cytokines in both BALF and lung microenvironment CD45+T cells. Furthermore, P4 inhibited the infiltration of inflammatory cells into the lungs, suppressed NETosis, and reduced the number of pulmonary CD4+T cells while increasing the number of regulatory T cells. The neutrophil elastase inhibitor GW311616A also suppressed airway inflammation and mucus production and modified the secretion of immune Th1, Th2, Th17, and Treg cytokines in lung CD45+immune cells. These changes led to an alteration of the immunological milieu with increased Th1 and Th17 cells, accompanied by decreased Th2, Treg, and CD44+T cells, similar to the effects of P4 treatment. Treatment with P4 inhibited NETosis by suppressing the p38 pathway activation, leading to reduced reactive oxygen species production. Moreover, P4 treatment hindered the release of double-stranded DNA during NETosis, thereby influencing the immune microenvironment in the lungs. These findings suggest that P4 treatment may be beneficial in reducing inflammation associated with allergic asthma by modulating the immune microenvironment. In conclusion, this research indicates the potential of P4 as a therapeutic agent for ameliorating inflammation in OVA-induced allergic asthma mice.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66439-6" target="_blank">https://www.nature.com/articles/s41598-024-66439-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41598-024-68055-w" target="_blank">
            ü§ñ Safety of High-Dose Nitric Oxide in Treating COVID-19 Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-47', this)">
            More</button></h4>
            <div id="toggle-47" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A safety evaluation of intermittent high-dose inhaled nitric oxide in viral pneumonia due to COVID-19: a randomised clinical study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>High doses of nitric oxide inhaled into the lungs have been found to help with infections, improve blood flow, and reduce inflammation. This study looked at whether giving high doses of nitric oxide, four times a day for seven days, could help people who were in hospital with viral pneumonia. The researchers compared this treatment to just standard care for 40 adults with viral pneumonia. They found that the treatment group had fewer problems related to breathing and were able to get off oxygen support sooner. Overall, the study suggests that giving high doses of nitric oxide is a safe and helpful way to treat people in hospital with viral pneumonia.</p><br>
                <p><strong>Abstract</strong></p>
                <p>High-dose inhaled Nitric Oxide (iNO) has been shown to have anti-inflammatory, vasodilator, and antimicrobial properties, resulting in improved arterial oxygenation as well as a beneficial therapeutic effect on lower respiratory tract infections. This study evaluated the safety and efficacy of 150-ppm intermittent iNO administered with a novel iNO-generator, for treating adults hospitalised for viral pneumonia. In this prospective, open-label, multicenter study, subjects aged 18‚Äì80, diagnosed with viral pneumonia received either standard supportive treatment alone (Control-Group) or combined with iNO for 40¬†min, 4 times per day up to 7¬†days (Treatment-Group). Out of 40 recruited subjects, 35 were included in the intention-to-treat population (34 with COVID-19). Adverse Events rate was similar between the groups (56.3% vs. 42.1%; respectively). No treatment-related adverse events were reported, while 2 serious adverse events were accounted for by underlying pre-existing conditions. Among the Treatment-Group, oxygen support duration was reduced by 2.7¬†days (Hazard Ratio‚Äâ=‚Äâ2.8; p‚Äâ=‚Äâ0.0339), a greater number of subjects reached oxygen saturation‚Äâ‚â•‚Äâ93% within hospitalisation period (Hazard Ratio‚Äâ=‚Äâ5.4; p‚Äâ=‚Äâ0.049), and a trend for earlier discharge was demonstrated. Intermittent 150-ppm iNO-treatment is well-tolerated, safe, and beneficial compared to usual care for spontaneously breathing hospitalised adults diagnosed with COVID-19 viral pneumonia.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-68055-w" target="_blank">https://www.nature.com/articles/s41598-024-68055-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41467-024-50601-9" target="_blank">
            ü§ñ Forecasting Influenza Hospitalizations: A Two-Year Evaluation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-48', this)">
            More</button></h4>
            <div id="toggle-48" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Title evaluation of FluSight influenza forecasting in the 2021‚Äì22 and 2022‚Äì23 seasons with a new target laboratory-confirmed influenza hospitalizations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Accurate predictions can help us prepare for and respond to flu outbreaks more effectively. In 2021 and 2022, a group of scientists made weekly forecasts about how many people would be hospitalized with the flu one to four weeks in advance. They compared their predictions to actual numbers and found that some models were better than others at making accurate forecasts. In fact, an average of all these models (called the FluSight ensemble) was one of the most accurate, but it started to struggle a bit when looking further ahead into the future. This shows that even with multiple models working together, there can be challenges in predicting flu outbreaks, especially during times of rapid change.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Accurate forecasts can enable more effective public health responses during seasonal influenza epidemics. For the 2021‚Äì22 and 2022‚Äì23 influenza seasons, 26 forecasting teams provided national and jurisdiction-specific probabilistic predictions of weekly confirmed influenza hospital admissions for one-to-four weeks ahead. Forecast skill is evaluated using the Weighted Interval Score (WIS), relative WIS, and coverage. Six out of 23 models outperform the baseline model across forecast weeks and locations in 2021‚Äì22 and 12 out of 18 models in 2022‚Äì23. Averaging across all forecast targets, the FluSight ensemble is the 2ndmost accurate model measured by WIS in 2021‚Äì22 and the 5thmost accurate in the 2022‚Äì23 season. Forecast skill and 95% coverage for the FluSight ensemble and most component models degrade over longer forecast horizons. In this work we demonstrate that while the FluSight ensemble was a robust predictor, even ensembles face challenges during periods of rapid change.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50601-9" target="_blank">https://www.nature.com/articles/s41467-024-50601-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-26</strong> <a href="https://www.nature.com/articles/s41467-024-50619-z" target="_blank">
            ü§ñ Unlocking mRNA Delivery with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-49', this)">
            More</button></h4>
            <div id="toggle-49" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>AGILE platform: a deep learning powered approach to accelerate LNP development for mRNA delivery</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Deep learning and chemistry are being used together to create special nanoparticles that can deliver medicine exactly where it's needed in the body. This is especially helpful for medicines made from a type of genetic material called mRNA. Right now, there's only one kind of nanoparticle that's widely used for mRNA delivery, which is mainly helping with COVID-19 vaccines. But scientists want to develop more types of nanoparticles that can target different cell types in the body. They've created a system called AGILE, which uses computers and chemistry to design and test these special nanoparticles quickly and efficiently. This allows them to find the best nanoparticle for each type of cell, making mRNA therapies even more effective and helping to broaden their use in medicine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Ionizable lipid nanoparticles (LNPs) are seeing widespread use in mRNA delivery, notably in SARS-CoV-2 mRNA vaccines. However, the expansion of mRNA therapies beyond COVID-19 is impeded by the absence of LNPs tailored for diverse cell types. In this study, we present the AI-Guided Ionizable Lipid Engineering (AGILE) platform, a synergistic combination of deep learning and combinatorial chemistry. AGILE streamlines ionizable lipid development with efficient library design, in silico lipid screening via deep neural networks, and adaptability to diverse cell lines. Using AGILE, we rapidly design, synthesize, and evaluate ionizable lipids for mRNA delivery, selecting from a vast library. Intriguingly, AGILE reveals cell-specific preferences for ionizable lipids, indicating tailoring for optimal delivery to varying cell types. These highlight AGILE‚Äôs potential in expediting the development of customized LNPs, addressing the complex needs of mRNA delivery in clinical practice, thereby broadening the scope and efficacy of mRNA therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50619-z" target="_blank">https://www.nature.com/articles/s41467-024-50619-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-25</strong> <a href="https://www.nature.com/articles/s41541-024-00922-z" target="_blank">
            ü§ñ Protecting Against Multiple COVID Strains with a Single Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-50', this)">
            More</button></h4>
            <div id="toggle-50" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Universal subunit vaccine protects against multiple SARS-CoV-2 variants and SARS-CoV</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract of the article that a young person could understand:

Scientists are trying to find a way to make a special medicine, called a vaccine, that can protect us against many different types of a virus called SARS-CoV-2. This virus causes COVID-19 and has been making people sick all over the world. The scientists were able to create a few different versions of this vaccine by taking parts from older versions of the virus and putting them together with a new part that they created. They tested these vaccines in mice and found out that one version, called S-6P-Delta-RBD, was very effective at protecting against many different types of the virus, including some that were quite similar to the original virus and others that had changed a lot over time. This vaccine worked so well that it even protected mice who hadn't been vaccinated before when they were challenged with the virus. The scientists think that this vaccine could be used to protect people against all sorts of SARS-CoV-2 viruses, including ones that might come in the future and cause new outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although Omicron RBD of SARS-CoV-2 accumulates many mutations, the backbone region (truncated RBD) of spike protein is highly conserved. Here, we designed several subunit vaccines by keeping the conserved spike backbone region of SARS-CoV-2 Omicron BA.1 subvariant (S-6P-no-RBD), or inserting the RBD of Delta variant (S-6P-Delta-RBD), Omicron (BA.5) variant (S-6P-BA5-RBD), or ancestral SARS-CoV-2 (S-6P-WT-RBD) to the above backbone construct, and evaluated their ability to induce immune responses and cross-protective efficacy against various SARS-CoV-2 variants and SARS-CoV. Among the four subunit vaccines, S-6P-Delta-RBD protein elicited broad and potent neutralizing antibodies against all SARS-CoV-2 variants tested, including Alpha, Beta, Gamma, and Delta variants, the BA.1, BA.2, BA.2.75, BA.4.6, and BA.5 Omicron subvariants, and the ancestral strain of SARS-CoV-2. This vaccine prevented infection and replication of SARS-CoV-2 Omicron, and completely protected immunized mice against lethal challenge with the SARS-CoV-2 Delta variant and SARS-CoV. Sera from S-6P-Delta-RBD-immunized mice protected naive mice against challenge with the Delta variant, with significantly reduced viral titers and without pathological effects. Protection correlated positively with the serum neutralizing antibody titer. Overall, the designed vaccine has potential for development as a universal COVID-19 vaccine and/or a pan-sarbecovirus subunit vaccine that will prevent current and future outbreaks caused by SARS-CoV-2 variants and SARS-related CoVs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00922-z" target="_blank">https://www.nature.com/articles/s41541-024-00922-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">
            ü§ñ COVID-19 Vaccine Tested in Phase 3 Trial Shows Promise</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-51', this)">
            More</button></h4>
            <div id="toggle-51" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of the tetravalent protein COVID-19 vaccine, SCTV01E: a phase 3 double-blind, randomized, placebo-controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Evolution of COVID-19 virus mutations highlights the need for vaccines that can protect against multiple strains at once. A new vaccine, SCTV01E, has been developed to target four different versions of the SARS-CoV-2 virus: Alpha, Beta, Delta, and Omicron BA.1. In a large-scale study, 9,223 participants were randomly divided into two groups: one received SCTV01E, while the other got a placebo. The results showed that SCTV01E was effective in preventing COVID-19, with a success rate of 69.4%. It also provided strong protection against all SARS-CoV-2 infections (79.7% and 82.4% at 14 days post-vaccination). Additionally, SCTV01E elicited a significantly higher immune response against the Omicron BA.5 variant compared to the placebo group. The vaccine was generally well-tolerated, with no severe adverse events or deaths reported. These findings suggest that SCTV01E could be an effective vaccine alternative against current and future SARS-CoV-2 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Evolution of SARS-CoV-2 variants emphasizes the need for multivalent vaccines capable of simultaneously targeting multiple strains. SCTV01E is a tetravalent COVID-19 vaccine derived from the spike protein of SARS-CoV-2 variants Alpha, Beta, Delta, and Omicron BA.1. In this double-blinded placebo-controlled pivotal efficacy trial (NCT05308576), the primary endpoint was vaccine efficacy (VE) against COVID-19 seven days post-vaccination in individuals without recent infection. Other endpoints included evaluating safety, immunogenicity, and the VE against all SARS-CoV-2 infections in individuals meeting the study criteria. Between December 26, 2022, and January 15, 2023, 9,223 individuals were randomized at a 1:1 ratio to receive SCTV01E or a placebo. SCTV01E showed a VE of 69.4% (95% CI: 50.6, 81.0) 7 days post-vaccination, with 75 cases in the placebo group and 23 in the SCTV01E group for the primary endpoint. VEs were 79.7% (95% CI: 51.0, 91.6) and 82.4% (95% CI: 57.9, 92.6), respectively, for preventing symptomatic infection and all SARS-CoV-2 infections 14 days post-vaccination. SCTV01E elicited a 25.0-fold higher neutralizing antibody response against Omicron BA.5 28 days post-vaccination compared to placebo. Reactogenicity was generally mild and transient, with no reported vaccine-related SAE, adverse events of special interest (AESI), or deaths. The trial aligned with the shift from dominant variants BA.5 and BF.7 to XBB, suggesting SCTV01E as a potential vaccine alternative effective against present and future variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">https://www.nature.com/articles/s41467-024-49832-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41598-024-67570-0" target="_blank">
            ü§ñ Protecting Against Multiple COVID Variants with an Inactivated Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-52', this)">
            More</button></h4>
            <div id="toggle-52" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An inactivated SARS-CoV-2 vaccine based on a Vero cell culture-adapted high-titer virus confers cross-protection in small animals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic is still ongoing, and many people are getting less protected against severe illness as time goes on. To keep everyone safe, it's essential to have access to affordable vaccines globally. Researchers created a new version of the virus that causes COVID-19, which they adapted to grow in a special cell culture. This new virus grew much faster and produced 50 times more vaccine material than the original version. Despite this change, the new virus still allowed our immune system to recognize and fight it off just like the original one. Studies showed that vaccines made from this new virus were effective against different COVID-19 variants and even protected animals from getting sick when they were exposed to these viruses. This suggests that adapting the virus to grow in cell culture could be a useful way to produce more vaccine, making it easier for people around the world to get protected against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Rapidly waning immunity against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) requires continued global access to affordable vaccines. Globally, inactivated SARS-CoV-2 vaccines have been widely used during the SARS-CoV-2 pandemic. In this proof-of-concept study we adapted an original-D614G SARS-CoV-2 virus to Vero cell culture as a strategy to enhance inactivated vaccine manufacturing productivity. A passage 60 (P60) virus showed enhanced fitness and 50-fold increased virus yield in a bioreactor compared to the original-D614G virus. It further remained susceptible to neutralization by plasma from SARS-CoV-2 vaccinated and convalescent individuals, suggesting exposure of relevant epitopes. Monovalent inactivated P60 and bivalent inactivated P60/omicron BA.1 vaccines induced neutralizing responses against original-D614G and BA.1 viruses in mice and hamsters, demonstrating that the P60 virus is a suitable vaccine antigen. Antibodies further cross-neutralized delta and BA.5 viruses. Importantly, the inactivated P60 vaccine protected hamsters against disease upon challenge with original-D614G or BA.1 virus, with minimal lung pathology and lower virus loads in the upper and lower airways. Antigenicity of the P60 virus was thus retained compared to the original virus despite the acquisition of cell culture adaptive mutations. Consequently, cell culture adaptation may be a useful approach to increase yields in inactivated vaccine antigen production.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67570-0" target="_blank">https://www.nature.com/articles/s41598-024-67570-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41598-024-67815-y" target="_blank">
            ü§ñ Unexplained Fatigue and Hormonal Imbalances: A Common Thread in Long COVID Cases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-53', this)">
            More</button></h4>
            <div id="toggle-53" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical and endocrine features of orthostatic intolerance detected in patients with long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had COVID-19 for a long time often get dizzy or faint when they stand up, but we don't know why this happens. To learn more, we studied 688 people who came to our clinic with long COVID and were suspected of having this problem. We tested them while they stood still for 10 minutes to see how their bodies reacted. In 33 out of the 86 people we tested, their body reacted strongly when they stood up - they got nausea (feeling sick to their stomach) and their heart beat faster than usual. When they stood up, their blood pressure also went up more in these 33 people compared to those who didn't react strongly. We found that older people with this problem had higher levels of a hormone called cortisol, while younger people had lower levels of another hormone called growth hormone. These changes might help us diagnose the cause of dizziness and fainting in people with long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Orthostatic intolerance (OI) is a key symptom of long COVID; however, the pathophysiology remains unknown. Among 688 long COVID patients who visited our clinic during the period from February 2021 to April 2023, 86 patients who were suspected of having OI and who underwent an active standing test (ST) were investigated to elucidate the clinical characteristics of OI in patients with long COVID. Of the 86 patients, 33 patients (38%) were ST-positive. Nausea and tachycardia in daily life were frequent complaints in the ST-positive group. The increase in heart rate (HR) during the ST was significantly greater during a 10-min period after standing in the ST-positive group (+‚Äâ30¬†bpm) than in the ST-negative group (+‚Äâ16¬†bpm). The initial increase in diastolic blood pressure (DBP) just after standing was significantly greater in the ST-positive group (+‚Äâ14¬†mmHg) than in the ST-negative group (+‚Äâ9¬†mmHg). Serum cortisol levels in the ST-positive patients aged over 20¬†years were higher and growth hormone levels in the patients under 20¬†years of age were lower than those in the ST-negative group. Autonomous nervous symptoms, transient DBP rise with increasing HR after standing, and endocrine dysfunctions are helpful for detecting OI related to long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67815-y" target="_blank">https://www.nature.com/articles/s41598-024-67815-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">
            ü§ñ COVID-19's Airway Problem: Spike Protein Blocks Important Cell Signal</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-54', this)">
            More</button></h4>
            <div id="toggle-54" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inflammation in the COVID-19 airway is due to inhibition of CFTR signaling by the SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people with COVID-19, the virus causes an overactive immune response that can lead to sickness and death. This happens because the virus activates certain cells in the airways that normally help fight off infections. Researchers have also found that a similar overactive immune response is seen in people with cystic fibrosis, a genetic disease that affects the airways.

The researchers wanted to know if the COVID-19 virus might be causing inflammation in the airways of people with COVID-19 by blocking a protein called CFTR. They used an artificial model of normal airway cells and found that exposing these cells to the COVID-19 spike protein led to the loss of CFTR function.

The researchers also showed that blocking the interaction between the COVID-19 spike protein and another protein called ACE2 prevented the loss of CFTR function. Furthermore, they found that the COVID-19 virus itself caused a loss of CFTR protein in airway cells, but this was rescued by blocking the interaction between the spike protein and ACE2.

Overall, these findings suggest that inflammation in the airways of people with COVID-19 may be due to the inhibition of CFTR signaling by the COVID-19 spike protein, leading to a cystic fibrosis-like clinical phenotype.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2-contributes to sickness and death in COVID-19 patients partly by inducing a hyper-proinflammatory immune response in the host airway. This hyper-proinflammatory state involves activation of signaling by NFŒ∫B, and unexpectedly, ENaC, the epithelial sodium channel. Post-infection inflammation may also contribute to "Long COVID"/PASC. Enhanced signaling by NFŒ∫B and ENaC also marks the airway of patients suffering from cystic fibrosis, a life-limiting proinflammatory genetic disease due to inactivating mutations in the CFTR gene. We therefore hypothesized that inflammation in the COVID-19 airway might similarly be due to inhibition of CFTR signaling by SARS-CoV-2 spike protein, and therefore activation of both NFŒ∫B and ENaC signaling. We used western blot and electrophysiological techniques, and an organoid model of normal airway epithelia, differentiated on an air‚Äìliquid-interface (ALI). We found that CFTR protein expression and CFTR cAMP-activated chloride channel activity were lost when the model epithelium was exposed to SARS-CoV-2 spike proteins. As hypothesized, the absence of CFTR led to activation of both TNFŒ±/NFŒ∫B signaling and Œ± and Œ≥ ENaC. We had previously shown that the cardiac glycoside drugs digoxin, digitoxin and ouabain blocked interaction of spike protein and ACE2. Consistently, addition of 30¬†nM concentrations of the cardiac glycoside drugs, prevented loss of both CFTR protein and CFTR channel activity. ACE2 and CFTR were found to co-immunoprecipitate in both basal cells and differentiated epithelia. Thus spike-dependent CFTR loss might involve ACE2 as a bridge between Spike and CFTR. In addition, spike exposure to the epithelia resulted in failure of endosomal recycling to return CFTR to the plasma membrane. Thus, failure of CFTR recovery from endosomal recycling might be a mechanism for spike-dependent loss of CFTR. Finally, we found that authentic SARS-CoV-2 virus infection induced loss of CFTR protein, which was rescued by the cardiac glycoside drugs digitoxin and ouabain. Based on experiments with this organoid model of small airway epithelia, and comparisons with 16HBE14o- and other cell types expressing normal CFTR, we predict that inflammation in the COVID-19 airway may be mediated by inhibition of CFTR signaling by the SARS-CoV-2 spike protein, thus inducing a cystic fibrosis-like clinical phenotype. To our knowledge this is the first time COVID-19 airway inflammation has been experimentally traced in normal subjects to a contribution from SARS-CoV-2 spike-dependent inhibition of CFTR signaling.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">https://www.nature.com/articles/s41598-024-66473-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41467-024-50133-2" target="_blank">
            ü§ñ Protecting Against Deadly Viruses: A New Vaccine Shows Promise</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-55', this)">
            More</button></h4>
            <div id="toggle-55" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-antigen intranasal vaccine protects against challenge with sarbecoviruses and prevents transmission in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Immunization programs that protect people from getting sick with SARS-CoV-2 have been shown to be effective, but they are not always good at preventing people from getting infected. Mucosal (surface-related) vaccines may provide better protection against the spread of the virus and could potentially work well even if new variants or related viruses emerge. Here, we tested a multi-component intranasal vaccine called NanoSTING-SN that targets different parts of SARS-CoV-2 and found it to be very effective in preventing the virus from replicating in both the lungs and nasal passages when challenged with the Delta variant. We also showed that this vaccine prevents transmission of another SARS-CoV-2 variant (Omicron) to unvaccinated animals. Furthermore, we demonstrated that NanoSTING-SN protects against SARS-CoV, a different virus related to SARS-CoV-2, by showing that immunized mice did not lose weight and all survived the infection. In non-human primates, the vaccine elicited long-lasting (up to 6 months) serum IgG responses and nasal wash IgA responses that could protect against other viruses like XBB1.5 (another SARS-CoV-2 variant), SARS-CoV, and MERS-CoV. This has two implications: (1) mucosal multi-component vaccines may help reduce the spread of respiratory viruses, and (2) eliciting immunity to multiple antigens could be beneficial in developing pan-sarbecovirus vaccines that protect against a wide range of related viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Immunization programs against SARS-CoV-2 with commercial intramuscular vaccines prevent disease but are less efficient in preventing infections. Mucosal vaccines can provide improved protection against transmission, ideally for different variants of concern (VOCs) and related sarbecoviruses. Here, we report a multi-antigen, intranasal vaccine, NanoSTING-SN (NanoSTING-Spike-Nucleocapsid), eliminates virus replication in both the lungs and the nostrils upon challenge with the pathogenic SARS-CoV-2 Delta VOC. We further demonstrate that NanoSTING-SN prevents transmission of the SARS-CoV-2¬†Omicron VOC (BA.5) to vaccine-na√Øve hamsters. To evaluate protection against other sarbecoviruses, we immunized mice with NanoSTING-SN. We showed that immunization affords protection against SARS-CoV, leading to protection from weight loss and 100% survival in mice. In non-human primates, animals immunized with NanoSTING-SN show durable serum IgG responses (6 months) and nasal wash IgA responses cross-reactive to SARS-CoV-2 (XBB1.5), SARS-CoV and MERS-CoV antigens. These observations have two implications: (1) mucosal multi-antigen vaccines present a pathway to reducing transmission of respiratory viruses, and (2) eliciting immunity against multiple antigens can be advantageous in engineering pan-sarbecovirus vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50133-2" target="_blank">https://www.nature.com/articles/s41467-024-50133-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41467-024-50566-9" target="_blank">
            ü§ñ Uncovering the Secrets of SARS-CoV-2's Papain-Like Protease: What Makes It Tick and How to Outsmart It</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-56', this)">
            More</button></h4>
            <div id="toggle-56" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mutational profiling of SARS-CoV-2 papain-like protease reveals requirements for function, structure, and drug escape</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Imagine a special protein called Papain-like protease (PLpro) that helps the SARS-CoV-2 virus multiply inside cells. This protein is crucial for the virus's survival, and scientists are interested in it as a potential target for new medicines.

To better understand PLpro, researchers used a powerful tool to scan millions of tiny changes in its genetic code and see how these changes affect the protein's ability to function properly within human cells. They discovered that some parts of the protein are very sensitive to even small changes, while others can tolerate more significant modifications without losing their job.

The study also found that specific areas around the protein's active site (where it does its main work) and a flexible loop (that helps it move and interact with other molecules) play crucial roles in how well PLpro works. Furthermore, the researchers noticed that changes to one part of the protein can indirectly affect its overall activity by influencing how it interacts with other molecules.

Using this knowledge, scientists were able to identify new versions of PLpro that might be resistant to a common inhibitor (a type of drug that stops or slows down the protein's function). This discovery is important because it suggests that when designing drugs against PLpro, scientists should consider not only the active site but also other flexible parts of the protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Papain-like protease (PLpro) is an attractive drug target for SARS-CoV-2 because it is essential for viral replication, cleaving viral poly-proteins pp1a and pp1ab, and has de-ubiquitylation and de-ISGylation activities, affecting innate immune responses. We employ Deep Mutational Scanning to evaluate the mutational effects on PLpro enzymatic activity and protein stability in mammalian cells. We confirm features of the active site and identify mutations in neighboring residues that alter activity. We characterize residues responsible for substrate binding and demonstrate that although residues in the blocking loop are remarkably tolerant to mutation, blocking loop flexibility is important for function. We additionally find a connected network of mutations affecting activity that extends far from the active site. We leverage our library to identify drug-escape variants to a common PLpro inhibitor scaffold and predict that plasticity in both the S4 pocket and blocking loop sequence should be considered during the drug design process.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50566-9" target="_blank">https://www.nature.com/articles/s41467-024-50566-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-67028-3" target="_blank">
            ü§ñ Testing Plasma Exchange for Severe COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-57', this)">
            More</button></h4>
            <div id="toggle-57" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A randomised controlled trial of plasma exchange compared to standard of care in the treatment of severe COVID-19 infection (COVIPLEX)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 is a serious illness that can cause the blood to become very "sticky" and prone to clotting. Our goal was to see if removing these "stickier" parts from the blood could help reduce this problem. We also wanted to know if it would make things better or worse for patients with severe COVID-19 who were having trouble breathing. We looked at 22 people who needed extra oxygen or a machine to help them breathe, and randomly assigned them either to get the special treatment (called plasma exchange) or to just get their usual care. After a few days, we found that the plasma exchange did reduce some of the bad stuff in the blood that can cause clotting, but it didn't seem to make things better for people who were having trouble breathing or getting more clots.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 disease is associated with a hyperinflammatory, pro-thrombotic state and a high mortality. Our primary objective was to assess the change in inflammatory and thrombotic markers associated with PEX, and secondary objectives were to assess the effects of PEX on progression of respiratory failure and incidence of acute thrombotic events. We conducted a prospective, phase II, non-blinded randomised control trial of plasma exchange compared to standard of care in critically ill adults with severe COVID-19 associated¬†respiratory failure, requiring supplemental oxygen or ventilatory support and elevated thrombo-inflammatory markers (LDH, CRP, ferritin, and D-Dimer). Patients randomised to receive PEX were treated with a daily single volume plasma exchange for a minimum of five days. Twenty-two patients were randomised of who 11 received PEX. Demographic and clinical characteristics were similar between groups at presentation. PEX was associated with a significant reduction in pro-thrombotic markers FVIII, VWF and VWF Ag: ADAMTS 13 ratio (p<‚Äâ0.001). There were no differences in the reduction of inflammatory markers, severity of respiratory failure (p=‚Äâ0.7), thrombotic events (p=‚Äâ0.67), or mortality (p>‚Äâ0.99) at 28¬†days. PEX successfully reduced pro-thrombotic markers, although was not associated with reduction in inflammatory markers, respiratory failure, or thrombotic events.Trial registration: (NCT04623255); first posted on 10/11/2020.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67028-3" target="_blank">https://www.nature.com/articles/s41598-024-67028-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67828-7" target="_blank">
            ü§ñ Tracking COVID-19 in Brazil: Understanding Vaccine Response to New Variant</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-58', this)">
            More</button></h4>
            <div id="toggle-58" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genomic surveillance and vaccine response to the dominant SARS-CoV-2 XBB lineage in Rio Grande do Sul</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has been challenging to control due to new types of viruses emerging, making it hard for vaccines and treatments to work effectively. One of these variants, called XBB, has shown that it can spread quickly without causing severe illness, but still poses a threat to people's health. In the state of Rio Grande do Sul in Brazil, researchers looked at 746 samples from COVID-19 patients between 2022 and 2023. They found that by March 2023, XBB was the most common variant, making up about 83% of all cases. Most people who got infected with XBB were able to recover quickly, but a small percentage (about 2%) died from the infection. Interestingly, even those who had received three vaccine doses and had previously been infected with COVID-19 could get reinfected with XBB, with about 59% of these cases occurring in people whose last vaccine dose was over a year ago. This might be because their immune system's protection had worn off. The researchers also identified 90 mutations within the XBB variant that were spread across its genome and included changes to the part of the virus that makes it hard for the body to fight back. This study provides important insights into how the XBB variant took over in this region, and continued monitoring of the virus's evolution is crucial for effective public health management.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has been marked by novel viral variants, posing challenges to global public health. Recombination, a viral evolution mechanism, is implicated in SARS-CoV-2's ongoing evolution. The XBB recombinant lineage, known for evading antibody-mediated immunity, exhibits higher transmissibility without increased disease severity. We investigated the prevalence and genomic features of XBB in SARS-CoV-2-positive cases in Rio Grande do Sul (RS), Brazil. We sequenced 357 samples from epidemiological weeks (EW) 47/2022 to 17/2023, and included 389 publicly available sequences. Clinical and epidemiological data were obtained from DATASUS, e-SUS, and SIVEP GRIPE (data recording systems of the Brazilian Ministry of Health). Of these, 143 were classified as XBB and 586 were other Omicron lineages. In March 2023 (EW 10), XBB became dominant, accounting for 83.3% of cases. 97.7% of XBB-infected patients successfully recovered from the infection, with a low mortality rate (2.3%). Even after receiving three vaccine doses and having been previously infected, 59.5% of the patients experienced reinfection with XBB. However, for 54% of the individuals, the interval between their XBB infection and the last vaccine dose exceeded one year, potentially leading to a decline in antibody levels. In addition, we identified 90 mutations in RS circulating XBB, spread throughout the genome, notably in the Spike protein region associated with immune resistance. This study provides insights into the dynamics and impact of a recombinant variant becoming predominant for the first time in the state. Continued surveillance of SARS-CoV-2 genomic evolution is crucial for effective public health management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67828-7" target="_blank">https://www.nature.com/articles/s41598-024-67828-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41392-024-01906-0" target="_blank">
            ü§ñ Vaccine for the Nose: Boosting Immunity with a Shot to the Sinuses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-59', this)">
            More</button></h4>
            <div id="toggle-59" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intranasal adenovirus-vectored Omicron vaccine induced nasal immunoglobulin A has superior neutralizing potency than serum antibodies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The upper part of our respiratory system is where COVID-19 starts to take hold in the body. A special type of antibody called secretory IgA (sIgA) has been shown to protect against breakthrough infections from new COVID-19 variants, like Omicron. We found that a vaccine given directly into the nose using a harmless virus as a delivery system can stimulate the production of robust sIgA in the nasal passages. This sIgA was mainly made up of two or more identical units and accounted for about 40% of all proteins in the mucus lining the nasal passage. We also detected a small amount of another type of antibody, IgG, but not IgM, IgD, or IgE. After rinsing the nose with a saline solution, sIgA levels in the nasal passage were quickly replenished within just a few hours. Comparing sIgA in the nasal passages to antibodies in the blood, we found that sIgA was up to 3-logs more effective at binding to and neutralizing new COVID-19 variants. In contrast, blood antibodies failed to neutralize two newly emerged variants, XBB and BA.2.86, but sIgA still retained its ability to neutralize these variants. Furthermore, we discovered that sIgA was better than IgG or IgA at blocking the spread of COVID-19 between cells and protecting mice from a challenge with the XBB variant. By estimating the percentage of spike-specific sIgA in nasal mucus collected one month after intranasal vaccination, we found it to be around 2.6-3.9%. Our study provides insights for developing nose-sprayed vaccines that can stimulate the production of sIgA and build an effective first-line defense against emerging COVID-19 variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The upper respiratory tract is the initial site of SARS-CoV-2 infection. Nasal spike-specific secretory immunoglobulin A (sIgA) correlates with protection against Omicron breakthrough infection. We report that intranasal vaccination using human adenovirus serotype 5 (Ad5) vectored Omicron spike in people who previously vaccinated with ancestral vaccine could induce robust neutralizing sIgA in the nasal passage. Nasal sIgA was predominantly present in dimeric and multimeric forms and accounted for nearly 40% of total proteins in nasal mucosal lining fluids (NMLFs). A low-level IgG could also be detected in NMLFs but not IgM, IgD, and IgE. After a complete nasal wash, sIgA in the nasal passage could be replenished rapidly within a few hours. A comparison of purified paired serum IgA, serum IgG, and nasal sIgA from the same individuals showed that sIgA was up to 3-logs more potent than serum antibodies in binding to spikes and in neutralizing Omicron subvariants. Serum IgG and IgA failed to neutralize XBB and BA.2.86, while nasal sIgA retained potent neutralization against these newly emerged variants. Further analysis showed that sIgA was more effective than IgG or IgA in blocking spike-mediated cell-to-cell transmission and protecting hACE2 mice from XBB challenge. Using a sIgA monoclonal antibody as a reference, we estimated that the total nasal sIgA contains about 2.6‚Äì3.9% spike-specific sIgA in NMLFs collected approximately one month after intranasal vaccination. Our study provided insights for developing intranasal vaccines that can induce sIgA to build an effective and mutation-resistant first-line immune barrier against constantly emerging variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01906-0" target="_blank">https://www.nature.com/articles/s41392-024-01906-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67806-z" target="_blank">
            ü§ñ Hope in a Field: Reducing COVID-19 Lung Damage with Magnetism</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-60', this)">
            More</button></h4>
            <div id="toggle-60" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Using a static magnetic field to attenuate the severity in COVID-19-invaded lungs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Two key things that can affect how quickly the coronavirus makes people sick are: 1) how well the virus's spike protein fits onto a special receptor on human cells called ACE2, and 2) how flexible the outer membrane of those cells is. This study wanted to see if a strong magnetic field could change either of these two things. They tested it on a type of lung cell in a lab dish (called Calu-3 cells) and also on mice. The results showed that a strong magnetic field can make the spike protein bind less well to ACE2 receptors, and it can also make the outer membrane of lung cells more flexible. This might be helpful for finding new ways to treat people with COVID-19 without having to touch or insert anything into their lungs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Two important factors affecting the progress of coronavirus disease 2019 (COVID-19) caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) are the S-protein binding function of ACE2 receptors and the membrane fluidity of host cells. This study aimed to evaluate the effect of static magnetic field (SMF) on S-protein/ACE2 binding and cellular membrane fluidity of lung cells, and was performed in vitro using a Calu-3 cell model and in vivo using an animal model. The ability of ACE2 receptors to bind to SARS-CoV-2 spike protein on host cell surfaces under SMF stimulation was evaluated using fluorescence images. Host lung cell membrane fluidity was tested using fluorescence polarization to determine the effects of SMF. Our results indicate that 0.4¬†T SMF can affect binding between S-protein and ACE2 receptors and increase Calu-3 cell membrane fluidity, and that SMF exposure attenuates LPS-induced alveolar wall thickening in mice. These results may be of value for developing future non-contact, non-invasive, and low side-effect treatments to reduce disease severity in COVID-19-invaded lungs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67806-z" target="_blank">https://www.nature.com/articles/s41598-024-67806-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-22</strong> <a href="https://www.nature.com/articles/s41598-024-67967-x" target="_blank">
            ü§ñ COVID-19 Vaccine vs No Vaccine: How It Affects Eye Nerve Health</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-61', this)">
            More</button></h4>
            <div id="toggle-61" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Corneal nerve fiber morphology following COVID-19 infection in vaccinated and non-vaccinated population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In people who got vaccinated and then got infected with COVID-19, we found out that their nerves in the front part of their eye (called the cornea) were healthier than those in people who didn't get vaccinated but still got infected. We compared these two groups to a group of people who weren't sick with COVID-19. The good news is that the vaccinated group had stronger and longer nerves, which could help protect them from complications related to COVID-19. However, we also found out that people who didn't get vaccinated had more immune cells (called dendritic cells) in their eyes than healthy individuals. This might be a sign of how the body responds differently when it's not protected by vaccines. Our study suggests that getting vaccinated might help keep the nerves in your eye safe from damage during a COVID-19 infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To examine corneal subbasal nerve changes in patients who received vaccination against SARS-CoV-2 virus and underwent COVID-19 infection compared to infected non-vaccinated patients and healthy controls. Twenty-nine eyes of 29 vaccinated patients (mean age: 36.66‚Äâ¬±‚Äâ12.25¬†years) within six months after PCR or Ag test proven COVID-19 infection and twenty-eight eyes of 28 age-matched infected, non-vaccinated patients (mean age: 42.14‚Äâ¬±‚Äâ14.17¬†years) were enrolled. Twenty-five age-matched healthy individuals (mean age: 47.52‚Äâ¬±‚Äâ18.45¬†years) served as controls. In vivo confocal microscopy (Heidelberg Retina Tomograph II Rostock Cornea Module, Germany) was performed in each group. Corneal subbasal nerve plexus morphology and corneal dendritic cells (DC) were evaluated. Significantly higher corneal nerve fiber density (P‚Äâ<‚Äâ0.001), nerve branch density (P‚Äâ<‚Äâ0.001), nerve fiber length (P‚Äâ<‚Äâ0.001), total branch density (P‚Äâ=‚Äâ0.007), nerve fiber area (P‚Äâ=‚Äâ0.001) and fractal dimension (P‚Äâ<‚Äâ0.001) values were observed in vaccinated patients after COVID-19 infection compared to the non-vaccinated group. Significantly higher DC density was observed in the non-vaccinated group compared to the control group (P‚Äâ=‚Äâ0.05). There was a statistically significant difference in the size of mature DCs (P‚Äâ<‚Äâ0.0001) but the size of immature DCs did not differ significantly among the 3 groups (P‚Äâ=‚Äâ0.132). Our results suggest that SARS-CoV-2 vaccination may have a protective effect against the complications of COVID-19 disease on the corneal subbasal nerve fibers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67967-x" target="_blank">https://www.nature.com/articles/s41598-024-67967-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-19</strong> <a href="https://www.nature.com/articles/s41467-024-50487-7" target="_blank">
            ü§ñ Uncovering Hidden COVID Risks in Everyday Interactions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-62', this)">
            More</button></h4>
            <div id="toggle-62" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Estimating the contribution of setting-specific contacts to SARS-CoV-2 transmission using digital contact tracing data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Understanding how COVID-19 spreads in crowded areas**

Scientists studied how COVID-19 spread during an outbreak in Beijing, China in 2022. They looked at data from over 22,000 people who got sick and more than 220,000 of their contacts. They found that when people were in close contact with others, like in a workplace or home, they were more likely to get COVID-19 from each other.

However, the researchers also created a computer model to see what would happen if there weren't any control measures in place. They discovered that even when people were in close contact, only about 11% of new cases came from these interactions. This was the lowest percentage among all types of settings they studied.

These findings suggest that public health officials should focus on finding a balance between using digital contact tracing to track COVID-19 spread and the challenges of tracking infections within crowded areas like homes or workplaces.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While many countries employed digital contact tracing to contain the spread of SARS-CoV-2, the contribution of cospace-time interaction (i.e., individuals who shared the same space and time) to transmission and to super-spreading in the real world has seldom been systematically studied due to the lack of systematic sampling and testing of contacts. To address this issue, we utilized data from 2230 cases and 220,878 contacts with detailed epidemiological information during the Omicron outbreak in Beijing in 2022. We observed that contact number per day of tracing for individuals in dwelling, workplace, cospace-time interactions, and community settings could be described by gamma distribution with distinct parameters. Our findings revealed that 38% of traced transmissions occurred through cospace-time interactions whilst control measures were in place. However, using a mathematical model to incorporate contacts in different locations, we found that without control measures, cospace-time interactions contributed to only 11% (95%CI: 10%‚Äì12%) of transmissions and the super-spreading risk for this setting was 4% (95%CI: 3%‚Äì5%), both the lowest among all settings studied. These results suggest that public health measures should be optimized to achieve a balance between the benefits of digital contact tracing for cospace-time interactions and the challenges posed by contact tracing within the same setting.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50487-7" target="_blank">https://www.nature.com/articles/s41467-024-50487-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-19</strong> <a href="https://www.nature.com/articles/s41392-024-01898-x" target="_blank">
            ü§ñ Hold and Rebuild: Hybrid Immunity Against Omicron After Breakthrough Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-63', this)">
            More</button></h4>
            <div id="toggle-63" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Heterogeneous hybrid immunity against Omicron variant JN.1 at 11 months following breakthrough infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The impact of climate change on polar bears is a major concern. Their main source of food is seals, which they hunt from ice floes. As the Arctic sea ice melts due to warmer temperatures, the hunting grounds of polar bears are shrinking. This could lead to them starving and having fewer babies. However, some polar bears have adapted by eating more plants and berries when there's no seal meat available. But this doesn't provide enough energy for them to reproduce. A solution might be to create artificial sea ice in areas where it naturally occurs, allowing polar bears to hunt seals and maintain their populations. This could also help protect other Arctic animals that rely on the sea ice as well.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Dear Editor,</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01898-x" target="_blank">https://www.nature.com/articles/s41392-024-01898-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-18</strong> <a href="https://www.nature.com/articles/s41467-024-50234-y" target="_blank">
            ü§ñ Protecting Against Respiratory Viruses with a New Type of Nanoparticle Treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-64', this)">
            More</button></h4>
            <div id="toggle-64" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An intranasal nanoparticle STING agonist protects against respiratory viruses in animal models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Respiratory viral infections are very bad for people all around the world. Even though vaccines have been really successful, there's a problem: viruses can change quickly and become hard to fight off with just vaccines. That's why scientists want to find something that can help protect us from these infections right away. In this study, researchers created a special treatment called NanoSTING. It's like a tiny particle that helps our immune system fight off bad germs. They tested it in mice and rats and found out it was safe for them. Then, they gave NanoSTING to hamsters who were infected with different strains of the SARS-CoV-2 virus, and it helped protect them. Even when they shared a space with other hamsters who had the virus, the treated hamsters didn't get sick as easily. They also tested NanoSTING against the flu in mice and found that it worked there too. By looking at how NanoSTING works, scientists think it could be used to help protect people from many different types of respiratory viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory viral infections cause morbidity and mortality worldwide. Despite the success of vaccines, vaccination efficacy is weakened by the rapid emergence of viral variants with immunoevasive properties. The development of an off-the-shelf, effective, and safe therapy against respiratory viral infections is thus desirable. Here, we develop NanoSTING, a nanoparticle formulation of the endogenous STING agonist, 2‚Ä≤‚àí3‚Ä≤ cGAMP, to function as an immune activator and demonstrate its safety in mice and rats. A single intranasal dose of NanoSTING protects against pathogenic strains of SARS-CoV-2 (alpha and delta VOC) in hamsters. In transmission experiments, NanoSTING reduces the transmission of SARS-CoV-2 Omicron VOC to na√Øve hamsters. NanoSTING also protects against oseltamivir-sensitive and oseltamivir-resistant strains of influenza in mice. Mechanistically, NanoSTING upregulates locoregional interferon-dependent and interferon-independent pathways in mice, hamsters, as well as non-human primates. Our results thus implicate NanoSTING as a broad-spectrum immune activator for controlling respiratory virus infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50234-y" target="_blank">https://www.nature.com/articles/s41467-024-50234-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41541-024-00919-8" target="_blank">
            ü§ñ Boosting Immunity to COVID-19: A New Way to Fight the Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-65', this)">
            More</button></h4>
            <div id="toggle-65" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Homologous but not heterologous COVID-19 vaccine booster elicits IgG4+ B-cells and enhanced Omicron subvariant binding</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person could understand:

Imagine you've gotten vaccinated against COVID-19, but now there are new versions of the virus that your original vaccination might not protect you as well against. To stay safe, scientists have been testing different types of boosters to see if they can improve protection. In this study, researchers looked at how different types of booster vaccinations (ones made with a special kind of virus and ones made with mRNA) affected the immune system's response in people who had already received their initial vaccinations. They found that both types of boosters were effective in increasing the level of antibodies and "memory B cells" that can help fight off future infections. However, they also discovered some differences between the two types of boosters, which might be important for future vaccine development.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Booster vaccinations are recommended to improve protection against severe disease from SARS-CoV-2 infection. With primary vaccinations involving various adenoviral vector and mRNA-based formulations, it remains unclear if these differentially affect the immune response to booster doses. We examined the effects of homologous (mRNA/mRNA) and heterologous (adenoviral vector/mRNA) vaccination on antibody and memory B cell (Bmem) responses against ancestral and Omicron subvariants. Healthy adults who received primary BNT162b2 (mRNA) or ChAdOx1 (vector) vaccination were sampled 1-month and 6-months after their 2nd and 3rd dose (homologous or heterologous) vaccination. Recombinant spike receptor-binding domain (RBD) proteins from ancestral, Omicron BA.2 and BA.5 variants were produced for ELISA-based serology, and tetramerized for immunophenotyping of RBD-specific Bmem. Dose 3 boosters significantly increased ancestral RBD-specific plasma IgG and Bmem in both cohorts. Up to 80% of ancestral RBD-specific Bmem expressed IgG1+. IgG4+Bmem were detectable after primary mRNA vaccination, and expanded significantly to 5‚Äì20% after dose 3, whereas heterologous boosting did not elicit IgG4+Bmem. Recognition of Omicron BA.2 and BA.5 by ancestral RBD-specific plasma IgG increased from 20% to 60% after the 3rd dose in both cohorts. Reactivity of ancestral RBD-specific Bmem to Omicron BA.2 and BA.5 increased following a homologous booster from 40% to 60%, but not after a heterologous booster. A 3rd mRNA dose generates similarly robust serological and Bmem responses in homologous and heterologous vaccination groups. The expansion of IgG4+Bmem after mRNA priming might result from the unique vaccine formulation or dosing schedule affecting the Bmem response duration and antibody maturation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00919-8" target="_blank">https://www.nature.com/articles/s41541-024-00919-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41598-024-67314-0" target="_blank">
            ü§ñ Recovering From Covid: How Pre-Existing Respiratory Conditions Affect Lung Function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-66', this)">
            More</button></h4>
            <div id="toggle-66" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pulmonary function trajectories in COVID-19 survivors with and without pre-existing respiratory disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many people who have had COVID-19 still feel like they're not breathing as well as they should be, even 3 or 12 months after they've recovered from the illness. This study looked at how two groups of patients fared: those who already had a respiratory disease (like asthma or COPD) when they got COVID-19, and those who didn't have any underlying lung problems. The researchers did tests to see if these patients could breathe well, and what they found was that about 49% of the people without pre-existing respiratory disease still had trouble breathing 3 months after discharge, compared to about 64% of those with pre-existing respiratory disease. However, over time, the people without underlying lung problems started to breathe better (in fact, their results improved significantly), but the people who already had respiratory issues didn't get much better. Even 12 months later, a significant number of both groups still struggled to breathe as well as they should have been able to.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A significant proportion of COVID-19 survivors still experience a reduced diffusion capacity three and twelve months after discharge. We aimed to compare pulmonary function trajectories between hospitalized COVID-19 patients with pre-existing respiratory disease (PRD) and patients without pre-existing respiratory disease (Non-PRD) at three and twelve months after hospital discharge. This single-centre retrospective cohort study included COVID-19 patients admitted to the VieCuri Medical Centre (Venlo, the Netherlands) between February and December 2020 that were invited to the outpatient clinic at three and twelve¬†months after discharge. During this visit, pulmonary function tests were performed and impairments were based on lower limit of normal. Data of 239 patients were analysed (65% male, 66‚Äâ¬±‚Äâ10¬†years, and 26% with a history of respiratory disease). Three months after discharge, 49% and 64% of the Non-PRD patients (n‚Äâ=‚Äâ177) and PRD patients (n‚Äâ=‚Äâ62) had a low diffusion capacity, respectively. This improved over time in Non-PRD patients (p=‚Äâ0.003), but not in PRD patients (p=‚Äâ0.250). A low diffusion capacity was still observed in 34% and 57% of the Non-PRD and PRD group, respectively, twelve months after discharge. Pulmonary function impairments, mainly a reduced diffusion capacity, are observed among hospitalized COVID-19 patients with PRD and Non-PRD, at three and twelve months follow-up. Although diffusion capacity impairments restore over time in Non-PRD patients, poor recovery was observed among PRD patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67314-0" target="_blank">https://www.nature.com/articles/s41598-024-67314-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-17</strong> <a href="https://www.nature.com/articles/s41598-024-66197-5" target="_blank">
            ü§ñ Uncovering the Link Between COVID-19 and Parkinson's Disease</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-67', this)">
            More</button></h4>
            <div id="toggle-67" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 and Parkinson‚Äôs disease: a single-center study and Mendelian randomization study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the rewritten abstract:

Imagine you have a neurological condition called Parkinson's disease (PD). Scientists wanted to find out if having COVID-19 makes your symptoms worse or affects your health differently than people without PD. They asked questions to over 130 people with PD at a single hospital between December 2022 and March 2023. The results showed that about two-thirds of the participants had COVID-19, and about one-third experienced worsening motor symptoms (such as tremors or stiffness) after getting infected.

The scientists also used a statistical method called Mendelian randomization to see if having PD increases the risk of getting COVID-19 in the first place. They found that people with PD were more likely to get COVID-19, but it didn't make their symptoms worse or increase their chances of being hospitalized. Overall, the study suggests that people with PD may experience worsened motor symptoms after contracting COVID-19, and longer disease duration and less frequent vaccination are risk factors for infection in these patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the association between COVID-19 and Parkinson‚Äôs disease (PD) via a single-center study and a Mendelian randomization (MR) study. A questionnaire-based survey was conducted among PD patients at a single center from December 7, 2022, to March 10, 2023. Logistic regression analysis was performed to identify the infection-related risk factors. Subsequently, bidirectional two-sample Mendelian randomization was employed to explore the association between COVID-19 and PD. In the cross-sectional analysis, it was found that the prevalence of COVID-19 infection in PD patients was 65.7%. Forty-eight (35.3%) PD patients experienced exacerbation of motor symptoms following COVID-19 infection. Long PD disease duration (‚â• 10 years) (OR: 3.327,P=‚Äâ0.045) and long time since last vaccination (>‚Äâ12¬†m) (OR: 4.916,P=‚Äâ0.035) were identified as significant risk factors related to infection. The MR analysis results supported that PD increases the COVID-19 susceptibility (Œ≤‚Äâ=‚Äâ0.081, OR‚Äâ=‚Äâ1.084,P=‚Äâ0.006). However, the MR analysis showed that PD did not increases the COVID-19 severity and hospitalization, and no significant association of COVID-19 on PD was observed. The findings from this cross-sectional study suggest that individuals with PD may experience worsened motor symptoms following COVID-19 infection. Long disease duration (‚â•10 years) and long time since last vaccination (>‚Äâ12¬†m) are identified as important risk factors for infection in these patients. Furthermore, our MR study provides evidence supporting an association between PD and COVID-19 susceptibility.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66197-5" target="_blank">https://www.nature.com/articles/s41598-024-66197-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s42003-024-06529-3" target="_blank">
            ü§ñ Targeting COVID-19 with Precision</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-68', this)">
            More</button></h4>
            <div id="toggle-68" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Influence of AAV vector tropism on long-term expression and Fc-Œ≥ receptor binding of an antibody targeting SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Imagine a special vaccine that can protect people with weak immune systems from getting sick. Scientists created a new way to make this vaccine using a tiny virus called an AAV vector. They added a part of a powerful antibody that can fight COVID-19 to the virus, so it can help protect people who are vulnerable to the disease.

When they tested this vaccine in mice, they found that it worked very well and kept the levels of the protective antibody high for over a year. The best part is that it didn't cause any bad reactions or side effects in the mice.

The scientists also discovered that the type of virus used in the vaccine affected how well it worked and how long it lasted in the body. They found that using a specific type of AAV vector led to higher levels of the protective antibody and better protection against COVID-19 in mice.

Overall, this study suggests that using AAV vectors to deliver powerful antibodies like the one they used could be a safe and effective way to protect people with weak immune systems from getting sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-acting passive immunization strategies are needed to protect immunosuppressed vulnerable groups from infectious diseases. To further explore this concept for COVID-19, we constructed Adeno-associated viral (AAV) vectors encoding the human variable regions of the SARS-CoV-2 neutralizing antibody, TRES6, fused to murine constant regions. An optimized vector construct was packaged in hepatotropic (AAV8) or myotropic (AAVMYO) AAV capsids and injected intravenously into syngeneic TRIANNI-mice. The highest TRES6 serum concentrations (511‚Äâ¬µg/ml) were detected 24 weeks after injection of the myotropic vector particles and mean TRES6 serum concentrations remained above 100‚Äâ¬µg/ml for at least one year. Anti-drug antibodies or TRES6-specific T cells were not detectable. After injection of the AAV8 particles, vector mRNA was detected in the liver, while the AAVMYO particles led to high vector mRNA levels in the heart and skeletal muscle. The analysis of the Fc-glycosylation pattern of the TRES6 serum antibodies revealed critical differences between the capsids that coincided with different binding activities to murine Fc-Œ≥-receptors. Concomitantly, the vector-based immune prophylaxis led to protection against SARS-CoV-2 infection in K18-hACE2 mice. High and long-lasting expression levels, absence of anti-drug antibodies and favourable Fc-Œ≥-receptor binding activities warrant further exploration of myotropic AAV vector-based delivery of antibodies and other biologicals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06529-3" target="_blank">https://www.nature.com/articles/s42003-024-06529-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41598-024-67536-2" target="_blank">
            ü§ñ COVID-19 Lung Function After Hospital Discharge</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-69', this)">
            More</button></h4>
            <div id="toggle-69" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-discharge spirometry evaluation in patients recovering from moderate-to-critical COVID-19: a cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who had severe cases of COVID-19 were studied to see how many of them still had lung problems after they got better. The study found that out of 408 people, about half (46.8%) still had abnormal lung function. Some had trouble breathing in and out, while others had lungs that were working fine but were not able to get enough air.

The researchers also looked for things that might be connected to these lung problems. They found that people who had fluid in their lungs when they were sick with COVID-19 (shown on a chest X-ray), or those who already had heart problems, or those who got tired easily during a simple walking test were more likely to have abnormal lung function.

This study shows that people who had severe cases of COVID-19 might still be having trouble with their lungs after they recover. The researchers suggest that these people should get regular tests to check how well their lungs are working.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the prevalence of abnormal lung function and its associated factors among patients recovering from COVID-19 is crucial for enhancing post-COVID care strategies. This study primarily aimed to determine the prevalence and types of spirometry abnormalities among post-COVID-19 patients in Malaysia, with a secondary objective of identifying its associated factors. Conducted at the COVID-19 Research Clinic, Faculty of Medicine, University Technology MARA, from March 2021 to December 2022, this study included patients at least three months post-discharge from hospitals following moderate-to-critical COVID-19. Of 408 patients studied, abnormal spirometry was found in 46.8%, with 28.4% exhibiting a restrictive pattern, 17.4% showing preserved ratio impaired spirometry (PRISm), and 1.0% displaying an obstructive pattern. Factors independently associated with abnormal spirometry included consolidation on chest X-ray (OR 8.1, 95% CI 1.75‚Äì37.42,p=‚Äâ0.008), underlying cardiovascular disease (OR 3.5, 95% CI 1.19‚Äì10.47,p=‚Äâ0.023), ground-glass opacity on chest X-ray (OR 2.6, 95% CI 1.52‚Äì4.30,p<‚Äâ0.001), and oxygen desaturation during the 6-min walk test (OR 1.9, 95% CI 1.20‚Äì3.06,p=‚Äâ0.007). This study highlights that patients recovering from moderate-to-critical COVID-19 often exhibit abnormal spirometry, notably a restrictive pattern and PRISm. Routine spirometry screening for high-risk patients is recommended.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67536-2" target="_blank">https://www.nature.com/articles/s41598-024-67536-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41541-024-00921-0" target="_blank">
            ü§ñ COVID-19 Vaccine Testing: A Standardized Approach</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-70', this)">
            More</button></h4>
            <div id="toggle-70" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Standardized immunological assays for assessing COVID-19 vaccines by the CEPI-Centralized Laboratory Network</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Imagine you're working in a big team of scientists who are all testing different things to help people with a serious virus called SARS-CoV-2. To make sure everyone is doing their tests correctly, we created a special system called the CEPI Centralized Laboratory Network. This system helped us share and control our tests across all the labs, making sure they were done in the same way everywhere.

This way, we can trust the results we get from each lab and compare them easily. It's like having a secret recipe that everyone follows to make sure the cake turns out the same every time! We're going to explain how this system works in more detail below.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The CEPI Centralized Laboratory Network implemented key steps in the transfer and monitoring of the developed immunological SARS-CoV-2 assays to ensure standardization across all the facilities of the network. This comprehensive evaluation reinforces the reliability of the generated data and establishes a solid foundation for a standardized approach, enabling precise inter-laboratory comparisons and contributing to the overall integrity of our network‚Äôs clinical results. Herein, we will provide a brief elaboration on the specific measures and procedures implemented to standardize the transfer of assays across our network.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00921-0" target="_blank">https://www.nature.com/articles/s41541-024-00921-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s41598-024-63212-7" target="_blank">
            ü§ñ Predicting Death and Pulmonary Embolisms for COVID Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-71', this)">
            More</button></h4>
            <div id="toggle-71" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>At-admission prediction of mortality and pulmonary embolism in an international cohort of hospitalised patients with COVID-19 using statistical and machine learning methods</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>By September 2022, more than 600 million people had gotten sick with COVID-19 worldwide, and over 6.5 million had died. However, the tools used to predict who might die from COVID-19 were often made with small groups of people that weren't very representative of the larger population. This made it harder to develop accurate predictive models. 

One of the most serious complications of COVID-19 is a condition called pulmonary embolism (PE), which can be life-threatening if not treated promptly. By using a large dataset of over 800,000 COVID-19 patients from around the world, we created a machine learning model that can predict whether someone has PE or will die from COVID-19 at admission to the hospital. 

Our model had good results when tested on a separate group of people, with an accuracy rate of about 75% for predicting PE and about 72% for predicting death due to any cause. We also looked at patients in the UK and Spain separately and got similar results.

The most important factors that our model used to predict PE were age, whether the patient was a man or woman, where they were admitted, and if they had certain health conditions like heart or lung disease, dementia, diabetes, high blood pressure, cancer, obesity, or smoked. 

Our analysis can help hospitals prioritize who needs immediate attention based on their risk level, and it can also inform long-term research and decision-making about how to care for COVID-19 patients globally.</p><br>
                <p><strong>Abstract</strong></p>
                <p>By September 2022, more than 600 million cases of SARS-CoV-2 infection have been reported globally, resulting in over 6.5 million deaths. COVID-19 mortality risk estimators are often, however, developed with small unrepresentative samples and with methodological limitations. It is highly important to develop predictive tools for pulmonary embolism (PE) in COVID-19 patients as one of the most severe preventable complications of COVID-19. Early recognition can help provide life-saving targeted anti-coagulation therapy right at admission. Using a dataset of more than 800,000 COVID-19 patients from an international cohort, we propose a cost-sensitive gradient-boosted machine learning model that predicts occurrence of PE and death at admission. Logistic regression, Cox proportional hazards models, and Shapley values were used to identify key predictors for PE and death. Our prediction model had a test AUROC of 75.9% and 74.2%, and sensitivities of 67.5% and 72.7% for PE and all-cause mortality respectively on a highly diverse and held-out test set. The PE prediction model was also evaluated on patients in UK and Spain separately with test results of 74.5% AUROC, 63.5% sensitivity and 78.9% AUROC, 95.7% sensitivity. Age, sex, region of admission, comorbidities (chronic cardiac and pulmonary disease, dementia, diabetes, hypertension, cancer, obesity, smoking), and symptoms (any, confusion, chest pain, fatigue, headache, fever, muscle or joint pain, shortness of breath) were the most important clinical predictors at admission. Age, overall presence of symptoms, shortness of breath, and hypertension were found to be key predictors for PE using our extreme gradient boosted model. This analysis based on the, until now, largest global dataset for this set of problems can inform hospital prioritisation policy and guide long term clinical research and decision-making for COVID-19 patients globally. Our machine learning model developed from an international cohort can serve to better regulate hospital risk prioritisation of at-risk patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-63212-7" target="_blank">https://www.nature.com/articles/s41598-024-63212-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-16</strong> <a href="https://www.nature.com/articles/s44298-024-00037-1" target="_blank">
            ü§ñ Jamaican Fruit Bats Resistant to COVID-19 Virus Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-72', this)">
            More</button></h4>
            <div id="toggle-72" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Jamaican fruit bat (Artibeus jamaicensis) insusceptibility to mucosal inoculation with SARS-CoV-2 Delta variant is not caused by receptor compatibility</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The original ancestor of the SARS-CoV-2 virus is thought to have come from bats. However, we know very little about how these viruses behave in bats themselves, or whether they can infect other animals besides humans. Some tests have shown that SARS-CoV-2 can infect rodents, monkeys, and a type of bat called Egyptian fruit bats. In this study, scientists tried using another type of bat, the Jamaican fruit bat, to see if it could be used as a model to study how SARS-CoV-2 behaves in its natural reservoir (bat population). They found that SARS-CoV-2 was unable to efficiently replicate in these bats and did not cause any significant immune response or shedding. This suggests that the Jamaican fruit bat is not a suitable model for studying SARS-CoV-2 infection in its natural reservoir.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ancestral sarbecovirus giving rise to SARS-CoV-2 is posited to have originated in bats. While SARS-CoV-2 causes asymptomatic to severe respiratory disease in humans, little is known about the biology, virus tropism, and immunity of SARS-CoV-2-like sarbecoviruses in bats. SARS-CoV-2 has been shown to infect multiple mammalian species, including various rodent species, non-human primates, and Egyptian fruit bats. We show that SARS-CoV-2 can utilize Jamaican fruit bat (Artibeus jamaicensis)ACE2 spike for entry in vitro. Therefore, we investigate the Jamaican fruit bat as a possible in vivo model to study reservoir responses. We find that SARS-CoV-2 Delta does not efficiently replicate in Jamaican fruit bats in vivo. We observe infectious viruses in the lungs of only one animal on day 1 post-inoculation and find no evidence of shedding or seroconversion. This is possibly due to host factors restricting virus egress after aborted replication. Furthermore, we observe no significant immune gene expression changes in the respiratory tract but do observe changes in the intestinal metabolome after inoculation. This suggests that, despite its broad host range, SARS-CoV-2 is unable to infect all bat species, and Jamaican fruit bats are not an appropriate model to study SARS-CoV-2 reservoir infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00037-1" target="_blank">https://www.nature.com/articles/s44298-024-00037-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-12</strong> <a href="https://www.nature.com/articles/s41564-024-01755-1" target="_blank">
            ü§ñ Developing a Safe and Effective Live-Attenuated COVID Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-73', this)">
            More</button></h4>
            <div id="toggle-73" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A safe, effective and adaptable live-attenuated SARS-CoV-2 vaccine to reduce disease and transmission using one-to-stop genome modifications</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Modified live vaccines that are genetically similar to the COVID-19 virus but less harmful have been developed to provide broader protection against new variants of the virus. These modified vaccines, called OTS-206 and OTS-228, were designed to boost the body's immune system by removing certain parts of the original SARS-CoV-2 genome that could interfere with its effectiveness. They are also genetically stable, can be easily administered through the nose, and have an adjustable level of severity. In animal studies, a single dose of OTS-228 was found to provide complete protection against COVID-19 infection and also helped prevent transmission of the virus, including new variants such as Omicron. This approach could potentially be applied to other emerging viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Approved vaccines are effective against severe COVID-19, but broader immunity is needed against new variants and transmission. Therefore, we developed genome-modified live-attenuated vaccines (LAV) by recoding the SARS-CoV-2 genome, including ‚Äòone-to-stop‚Äô (OTS) codons, disabling Nsp1 translational repression and removing ORF6, 7ab and 8 to boost host immune responses, as well as the spike polybasic cleavage site to optimize the safety profile. The resulting OTS-modified SARS-CoV-2 LAVs, designated as OTS-206 and OTS-228, are genetically stable and can be intranasally administered, while being adjustable and sustainable regarding the level of attenuation. OTS-228 exhibits an optimal safety profile in preclinical animal models, with no side effects or detectable transmission. A single-dose vaccination induces a sterilizing immunity in vivo against homologous WT SARS-CoV-2 challenge infection and a broad protection against Omicron BA.2, BA.5 and XBB.1.5, with reduced transmission. Finally, this promising LAV approach could be applicable to other emerging viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-024-01755-1" target="_blank">https://www.nature.com/articles/s41564-024-01755-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-12</strong> <a href="https://www.nature.com/articles/s41598-024-64094-5" target="_blank">
            ü§ñ COVID-19 Survivors with Mental Health Issues Have Higher Platelet Activation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-74', this)">
            More</button></h4>
            <div id="toggle-74" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preferential and sustained platelet activation in COVID-19 survivors with mental disorders</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person can understand:

People with mental health conditions like depression or anxiety are more likely to get very sick if they catch COVID-19. One reason for this might be that their blood vessels are already damaged, which makes it harder for them to fight off the virus.

Researchers wanted to see if people who had mental health conditions were also more likely to have problems with platelets in their blood after recovering from COVID-19. Platelets help blood to clot and can become overactive when someone is sick. The researchers looked at 141 people who had recovered from COVID-19 and found that those who had a history of mental health conditions were more likely to have platelet problems.

This suggests that people with mental health conditions might be more prone to severe COVID-19 symptoms because their blood is more likely to get overactive and cause inflammation. This could be an important clue for doctors trying to understand why some people are more vulnerable to COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pre-existing mental disorders are considered a risk factor for severe COVID-19 outcomes, possibly because of higher vascular burden. Moreover, an unconventional platelet activation characterizes COVID-19 and contributes to inflammatory and thrombotic manifestations. In the light of the inflammation theory of mental disorders, we hypothesized that patients with mental disorders could be sensitive to the SARS-CoV-2 elicited platelet activation. We investigated platelet activation in 141 COVID-19 survivors at one month after clearance of the virus, comparing subjects with or without an established pre-existing diagnosis of mental disorder according to the DSM-5. We found that platelets from patients with a positive history of psychiatric disorder underwent unconventional activation more frequently than conventional activation or no activation at all. Such preferential activation was not detected when platelets from patients without a previous psychiatric diagnosis were studied. When testing the effects of age, sex, and psychiatric history on the platelet activation, GLZM multivariate analysis confirmed the significant effect of diagnosis only. These findings suggest a preferential platelet activation during acute COVID-19 in patients with a pre-existing psychiatric disorder, mediated by mechanisms associated with thromboinflammation. This event could have contributed to the higher risk of severe outcome in the psychiatric population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64094-5" target="_blank">https://www.nature.com/articles/s41598-024-64094-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">
            ü§ñ Uncovering Patterns of Re-infection and Prolonged Illness in People with COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-75', this)">
            More</button></h4>
            <div id="toggle-75" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Insights from an N3C RECOVER EHR-based cohort study characterizing SARS-CoV-2 reinfections and Long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Many people have gotten COVID-19 twice, but scientists don't know much about what happens when this happens. We want to learn more about getting COVID-19 again and how it might affect some people who got it once before.

We looked at the health records of over 3 million people to understand this better. Our findings show that most people who got COVID-19 twice did so during a certain time period, and we found that some people got it multiple times.

Interestingly, our study shows that some people experienced more severe symptoms after their first infection than after getting it again. We also noticed that older people and those with severe initial infections were at higher risk of dying.

Overall, our research suggests that the severity of a second COVID-19 infection might be related to how severe the first one was, and that long-term health effects (known as Long COVID) seem more common after the initial infection than after getting it again. Further studies can build on these findings to help us understand COVID-19 reinfections better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundAlthough the COVID-19 pandemic has persisted for over 3 years, reinfections with SARS-CoV-2 are not well understood. We aim to characterize reinfection, understand development of Long COVID after reinfection, and compare severity of reinfection with initial infection.MethodsWe use an electronic health record study cohort of over 3 million patients from the National COVID Cohort Collaborative as part of the NIH Researching COVID to Enhance Recovery Initiative. We calculate summary statistics, effect sizes, and Kaplan‚ÄìMeier curves to better understand COVID-19 reinfections.ResultsHere we validate previous findings of reinfection incidence (6.9%), the occurrence of most reinfections during the Omicron epoch, and evidence of multiple reinfections. We present findings that the proportion of Long COVID diagnoses is higher following initial infection than reinfection for infections in the same epoch. We report lower albumin levels leading up to reinfection and a statistically significant association of severity between initial infection and reinfection (chi-squared value: 25,697,p-value: <0.0001) with a medium effect size (Cramer‚ÄôsV: 0.20, DoF‚Äâ=‚Äâ3). Individuals who experienced severe initial and first reinfection were older in age and at a higher mortality risk than those who had mild initial infection and reinfection.ConclusionsIn a large patient cohort, we find that the severity of reinfection appears to be associated with the severity of initial infection and that Long COVID diagnoses appear to occur more often following initial infection than reinfection in the same epoch. Future research may build on these findings to better understand COVID-19 reinfections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">https://www.nature.com/articles/s43856-024-00539-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00549-0" target="_blank">
            ü§ñ Uncovering Hidden Risks: Predicting Long COVID Outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-76', this)">
            More</button></h4>
            <div id="toggle-76" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identification of risk factors of Long COVID and predictive modeling in the RECOVER EHR cohorts</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background
People who got sick with SARS-CoV-2 might develop new health problems later on. These problems, called post-acute sequelae of SARS-CoV-2 infection (PASC) or Long COVID, can affect many different parts of the body. Not much research has looked into why some people get Long COVID and how to predict it.

Methods
This study used electronic medical records from two big healthcare networks in New York and Florida to look for factors that might increase the risk of getting Long COVID. The study included both people who had SARS-CoV-2 and those who didn't. It looked at data from March 2020 to November 2021.

Results
The study found complex relationships between certain health factors and different Long COVID conditions. It suggested that people who had severe SARS-CoV-2, were underweight, or had pre-existing health conditions (like cancer or liver disease) might be more likely to get Long COVID. Some Long COVID conditions, such as dementia, malnutrition, and heart failure, could be predicted fairly well. Others, like fatigue, anxiety, sleep disorders, and depression, were harder to predict.

Conclusions
This study found that the relationships between health factors and Long COVID are complicated. However, it suggests that using machine learning-based models might help identify people who are at risk of getting different Long COVID conditions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSARS-CoV-2-infected patients may develop new conditions in the period after the acute infection. These conditions, the post-acute sequelae of SARS-CoV-2 infection (PASC, or Long COVID), involve a diverse set of organ systems. Limited studies have investigated the predictability of Long COVID development and its associated risk factors.MethodsIn this retrospective cohort study, we used electronic healthcare records from two large-scale PCORnet clinical research networks, INSIGHT (~1.4 million patients from New York) and OneFlorida+ (~0.7 million patients from Florida), to identify factors associated with having Long COVID, and to develop machine learning-based models for predicting Long COVID development. Both SARS-CoV-2-infected and non-infected adults were analysed during the period of March 2020 to November 2021. Factors associated with Long COVID risk were identified by removing background associations and correcting for multiple tests.ResultsWe observed complex association patterns between baseline factors and a variety of Long COVID conditions, and we highlight that severe acute SARS-CoV-2 infection, being underweight, and having baseline comorbidities (e.g., cancer and cirrhosis) are likely associated with increased risk of developing Long COVID. Several Long COVID conditions, e.g., dementia, malnutrition, chronic obstructive pulmonary disease, heart failure, PASC diagnosis U099, and acute kidney failure are well predicted (C-index‚Äâ>‚Äâ0.8). Moderately predictable conditions include atelectasis, pulmonary embolism, diabetes, pulmonary fibrosis, and thromboembolic disease (C-index 0.7‚Äì0.8). Less predictable conditions include fatigue, anxiety, sleep disorders, and depression (C-index around 0.6).ConclusionsThis observational study suggests that association patterns between investigated factors and Long COVID are complex, and the predictability of different Long COVID conditions varies. However, machine learning-based predictive models can help in identifying patients who are at risk of developing a variety of Long COVID conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00549-0" target="_blank">https://www.nature.com/articles/s43856-024-00549-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s41598-024-66901-5" target="_blank">
            ü§ñ COVID-19 and Cesarean Section Complications</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-77', this)">
            More</button></h4>
            <div id="toggle-77" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of perioperative COVID-19 infection on postoperative complication in cesarean section using Korean National Health insurance data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The impact of COVID-19 on pregnancy has raised concerns about how it might affect the care that women receive during childbirth, specifically when they have a C-section. To investigate this, researchers analyzed data from South Korea's national health insurance system. They looked at two groups of patients who had C-sections: those who tested positive for COVID-19 and those who didn't. The study found that women with COVID-19 were more likely to experience complications like pneumonia, breathing problems, blood clots, and infections after their surgery. These women also stayed in the hospital longer and had higher medical bills than those without COVID-19. The researchers' analysis suggests that having COVID-19 during pregnancy can lead to more serious problems for women undergoing C-sections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The vulnerability during pregnancy has raised concerns about the potential impact of COVID-19 on obstetric anesthesia, an essential aspect of maternal care during cesarean section procedures. To evaluate the influence of COVID-19 infection on obstetric anesthesia during cesarean section, we analyzed the data from Korean National Health Insurance System (NHIS). This retrospective study utilized data from Korean NHIS. We included patients admitted with operation codes specific to cesarean section between January 1, 2020, and December 31, 2021. We classified patients into a COVID (+) group with a diagnosis code (U071) 30¬†days around surgery and a COVID (‚àí) group without the code in the same period. The primary outcome was 30-day mortality that was defined as death within 30¬†days of admission due to any causes. Secondary outcomes were pulmonary complications (pneumonia, acute respiratory distress syndrome [ARDS], pulmonary thromboembolism [PTE], or unexpected postoperative mechanical ventilation), ICU admission, cardiac arrest, myocardial infarction [MI], other thromboembolic events, surgical site infection, sepsis, acute renal failure [ARF], and hepatic failure. Among 75,268 patients who underwent cesarean section, 107 had a COVID-19 diagnosis code, while 75,161 did not. After 1:4 propensity score matching (PSM), 535 patients were included in each group. 30-day mortality showed no significant differences between the two groups both before and after PSM. The COVID (+) group demonstrated significantly elevated rates of pneumonia, ARDS, PTE, and surgical site infection both before and after PSM. Hospital length of stay and admission costs were also significantly longer and higher, respectively, in the COVID (+) group before and after PSM. In subgroup analysis, no differences were observed in mortality and postoperative complications based on the anesthesia method after matching. COVID-19 infection is associated with increased rates of postoperative complications, including pneumonia, ARDS, PTE, surgical site infection, longer hospital stays, and increased admission costs, in patients who underwent cesarean section.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66901-5" target="_blank">https://www.nature.com/articles/s41598-024-66901-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-10</strong> <a href="https://www.nature.com/articles/s42003-024-06500-2" target="_blank">
            ü§ñ High-Throughput Testing for Broad Immune Response Against Many Antigens</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-78', this)">
            More</button></h4>
            <div id="toggle-78" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Two-dimensional high-throughput on-cell screening of immunoglobulins against broad antigen repertoires</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###
 
Imagine you want to find special helpers in your blood that can fight a specific bad guy. But, there are so few of these helpers around that it's hard to find them. To make things worse, we don't know what the bad guy looks like, which makes it harder to create good fighters against it. A team of scientists has come up with a clever way to find these special helpers and also figure out how they work. They use a system where they take tiny pieces of the bad guy (made by a bacteriophage - a virus that helps us) and attach them to tiny balls. Then, they mix this with blood from people who have recovered from the sickness or have been vaccinated. The special helpers in their blood will stick to the right parts of the bad guy on the ball, and then those helpers can be isolated and studied for potential use as medicines. This process is very efficient and allows many helpers to be tested quickly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Identifying high-affinity antibodies in human serum is challenging due to extremely low number of circulating B cells specific to the desired antigens. Delays caused by a lack of information on the immunogenic proteins of viral origin hamper the development of therapeutic antibodies. We propose an efficient approach allowing for enrichment of high-affinity antibodies against pathogen proteins with simultaneous epitope mapping, even in the absence of structural information about the pathogenic immunogens. To screen therapeutic antibodies from blood of recovered donors, only pathogen transcriptome is required to design an antigen polypeptide library, representing pathogen proteins, exposed on the bacteriophage surface. We developed a two-dimensional screening approach enriching lentiviral immunoglobulin libraries from the convalescent or vaccinated donors against bacteriophage library expressing the overlapping set of polypeptides covering the spike protein of SARS-CoV-2. This platform is suitable for pathogen-specific immunoglobulin enrichment and allows high-throughput selection of therapeutic human antibodies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06500-2" target="_blank">https://www.nature.com/articles/s42003-024-06500-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-10</strong> <a href="https://www.nature.com/articles/s41586-024-07671-y" target="_blank">
            ü§ñ Dendritic Cells Regulate Blood Platelet Production</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-79', this)">
            More</button></h4>
            <div id="toggle-79" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Plasmacytoid dendritic cells control homeostasis of megakaryopoiesis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that a young person could understand:

 Platelets are tiny cells in our blood that help us heal from cuts and fights off infections. Our body needs to make new platelets all the time, but we didn't know how it works. Scientists used special cameras to watch what happens inside our bone marrow, where platelets are made. They found that a type of cell called a plasmacytoid dendritic cell (pDC) acts like a sensor, detecting when platelet-making cells are dying and sending a signal to make more new platelets. This system helps keep the right number of platelets in our blood. But when pDCs get activated by viruses like COVID-19, they can't do their job properly and we end up with too many platelets. This study helps us understand how our body makes platelets and what happens when it gets out of balance.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Platelet homeostasis is essential for vascular integrity and immune defence1,2. Although the process of platelet formation by fragmenting megakaryocytes (MKs; thrombopoiesis) has been extensively studied, the cellular and molecular mechanisms required to constantly replenish the pool of MKs by their progenitor cells (megakaryopoiesis) remains unclear3,4. Here we use intravital imaging to track the cellular dynamics of megakaryopoiesis over days. We identify plasmacytoid dendritic cells (pDCs) as homeostatic sensors that monitor the bone marrow for apoptotic MKs and deliver IFNŒ± to the MK niche triggering local on-demand proliferation and maturation of MK progenitors. This pDC-dependent feedback loop is crucial for MK and platelet homeostasis at steady state and under stress. pDCs are best known for their ability to function as vigilant detectors of viral infection5. We show that virus-induced activation of pDCs interferes with their function as homeostatic sensors of megakaryopoiesis. Consequently, activation of pDCs by SARS-CoV-2 leads to excessive megakaryopoiesis. Together, we identify a pDC-dependent homeostatic circuit that involves innate immune sensing and demand-adapted release of inflammatory mediators to maintain homeostasis of the megakaryocytic lineage.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07671-y" target="_blank">https://www.nature.com/articles/s41586-024-07671-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">
            ü§ñ Developing Antibodies Against SARS-CoV-2: Identifying Targeting Molecules in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-80', this)">
            More</button></h4>
            <div id="toggle-80" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Selection and characterization of human scFvs targeting the SARS-CoV-2 nucleocapsid protein isolated from antibody libraries of COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

A new type of coronavirus called SARS-CoV-2 started spreading in China in 2019, causing an illness named COVID-19. At first, most scientists focused on studying the spike part of the virus, but they soon realized that another important part, called the nucleocapsid (N) protein, plays a key role in making the virus copy itself and hide from our immune system. The N protein also triggers a strong response from our immune system, making it an attractive target for developing antibodies to fight the virus. This study created a library of special antibody fragments from COVID-19 patients' cells to create a large collection of antibodies that could be used in research and possibly as tests or treatments. These fragments were then divided into two groups: those that strongly stuck to the N protein when produced inside yeast cells or mammalian cells, and others that were purified as separate proteins with strong affinity for the N protein. This set of antibodies against the N protein offers a new tool for scientists and could potentially be used in medical tests or treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In 2019, the novel SARS-CoV-2 coronavirus emerged in China, causing the pneumonia named COVID-19. At the beginning, all research efforts were focused on the spike (S) glycoprotein. However, it became evident that the nucleocapsid (N) protein is pivotal in viral replication, genome packaging and evasion of the immune system, is highly immunogenic, which makes it another compelling target for antibody development alongside the spike protein. This study focused on the construction of single chain fragments variable (scFvs) libraries from SARS-CoV-2-infected patients to establish a valuable, immortalized and extensive antibodies source. We used the Intracellular Antibody Capture Technology to select a panel of scFvs against the SARS-CoV-2 N protein. The whole panel of scFv was expressed and characterized both as intrabodies and recombinant proteins. ScFvs were then divided into 2 subgroups: those that exhibited high binding activity to N protein when expressed in yeast or in mammalian cells as intrabodies, and those purified as recombinant proteins, displaying affinity for recombinant N protein in the nanomolar range. This panel of scFvs against the N protein represents a novel platform for research and potential diagnostic applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">https://www.nature.com/articles/s41598-024-66558-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66410-5" target="_blank">
            ü§ñ Red Blood Cells Have a New Trick: They Can Detect Viruses Too</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-81', this)">
            More</button></h4>
            <div id="toggle-81" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Human red blood cells express the RNA sensor TLR7</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Red blood cells in our bodies have a special tool called TLR9 that helps them detect certain patterns in DNA. But can they also detect other types of genetic material? We discovered that human red blood cells actually have another tool called TLR7 that helps them recognize RNA, which is similar to DNA but carries instructions from viruses and other tiny life forms. This TLR7 is attached to a protein on the surface of red blood cells and is more active in people who are very sick with COVID-19 compared to healthy individuals. In experiments, we found that red blood cells can bind to synthetic RNA and even to RNA from certain viruses. This means that red blood cells might be taking up excess RNA from our bodies, performing another function beyond just carrying oxygen.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Red blood cells (RBCs) express the nucleic acid-binding toll-like receptor 9 (TLR9) and bind CpG-containing DNA. However, whether human RBCs express other nucleic acid-binding TLRs is unknown. Here we show that human RBCs express the RNA sensor TLR7. TLR7 is present on the red cell membrane and is associated with the RBC membrane protein Band 3. In patients with SARS-CoV2-associated sepsis, TLR7-Band 3 interactions in the RBC membrane are increased when compared with healthy controls. In vitro, RBCs bind synthetic ssRNA and RNA from ssRNA viruses. Thus, RBCs may serve as a previously unrecognized sink for exogenous RNA, expanding the repertoire of non-gas exchanging functions performed by RBCs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66410-5" target="_blank">https://www.nature.com/articles/s41598-024-66410-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s44298-024-00036-2" target="_blank">
            ü§ñ Uncovering a Key to COVID's Success: How the SARS-CoV-2 Virus Adapted to Humans</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-82', this)">
            More</button></h4>
            <div id="toggle-82" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evolution at Spike protein position 519 in SARS-CoV-2 facilitated adaptation to humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As COVID-19 continues to spread, scientists are trying to understand how the virus first emerged in humans. They're also working to develop new treatments that can fight the virus more effectively. To do this, they need to know which parts of the virus are most important for it to survive and multiply in humans. Recently, researchers have been searching for specific changes in the virus's genetic code that might have helped it adapt to human hosts. They found a particular change at position 519 in the Spike protein on the surface of the virus. This change, from an asparagine (N) to a histidine (H), is important because it helps the virus bind to a receptor called ACE2 in human cells. The researchers created a mutant version of the virus with the ancestral N519 and found that it didn't replicate well in human lung cells. They also discovered that this change made it harder for the virus to infect cells that express the human ACE2 receptor. This suggests that the evolutionary transition at position 519 was crucial for SARS-CoV-2's ability to survive and multiply in humans. Furthermore, this change might be a target for developing new treatments, such as small molecule inhibitors, that can specifically block the Spike protein's interaction with ACE2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As the COVID-19 pandemic enters its fourth year, the pursuit of identifying a progenitor virus to SARS-CoV-2 and understanding the mechanism of its emergence persists, albeit against the backdrop of intensified efforts to monitor the ongoing evolution of the virus and the influx of new mutations. Surprisingly, few residues hypothesized to be essential for SARS-CoV-2 emergence and adaptation to humans have been validated experimentally, despite the importance that these mutations could contribute to the development of effective antivirals. To remedy this, we searched for genomic regions in the SARS-CoV-2 genome that show evidence of past selection around residues unique to SARS-CoV-2 compared with closely related coronaviruses. In doing so, we identified a residue at position 519 in Spike within the receptor binding domain that holds a static histidine in human-derived SARS-CoV-2 sequences but an asparagine in SARS-related coronaviruses from bats and pangolins. In experimental validation, the SARS-CoV-2 Spike protein mutant carrying the putatively ancestral H519N substitution showed reduced replication in human lung cells, suggesting that the histidine residue contributes to viral fitness in the human host. Structural analyses revealed a potential role of Spike residue 519 in mediating conformational transitions necessary for Spike prior to binding with ACE2. Pseudotyped viruses bearing the putatively ancestral N519 also demonstrated significantly reduced infectivity in cells expressing the human ACE2 receptor compared to H519. ELISA data corroborated that H519 enhances Spike binding affinity to the human ACE2 receptor compared to the putatively ancestral N519. Collectively, these findings suggest that the evolutionary transition at position 519 of the Spike protein played a critical role in SARS-CoV-2 emergence and adaptation to the human host. Additionally, this residue presents as a potential drug target for designing small molecule inhibitors tailored to this site.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00036-2" target="_blank">https://www.nature.com/articles/s44298-024-00036-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66891-4" target="_blank">
            ü§ñ Warning Signs of a Rare COVID-19 Complication in Children</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-83', this)">
            More</button></h4>
            <div id="toggle-83" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Severity predictors for multisystemic inflammatory syndrome in children after SARS-CoV-2 infection in Vietnam</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

A serious condition called Multisystemic Inflammatory Syndrome in Children (MIS-C) can cause different levels of illness in kids, ranging from mild symptoms to life-threatening problems. However, scientists don't fully understand what makes some cases more severe than others. This study aimed to find out which factors are linked to more serious outcomes by using a special statistical method called multivariate regression. The researchers looked at data from 391 children who were hospitalized in Vietnam between January and June 2023. They found that certain markers, such as high levels of a protein called CRP (meaning the child is likely to be very sick), low levels of albumin (which could indicate organ problems), low absolute lymphocyte count (a sign of weakened immune system), high ferritin levels (indicating inflammation), and low left ventricular ejection fraction (LVEF) (meaning heart problems), were strongly connected to severe cases. The study also showed that these factors increased the risk of shock, a life-threatening condition. In conclusion, this research suggests that monitoring absolute lymphocyte count, serum albumin, CRP levels, and LVEF can help identify which children with MIS-C are at higher risk for severe complications. More studies are needed to confirm if using these markers can improve treatment outcomes in kids with MIS-C.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multisystemic inflammatory syndrome in children (MIS-C) might manifest in a broad spectrum of clinical scenarios, ranging from mild features to multi-organ dysfunction and mortality. However, this novel entity has a heterogenicity of data regarding prognostic factors associated with severe outcomes. The present study aimed to identify independent predictors for severity by using multivariate regression models. A total of 391 patients (255 boys and 136 girls) were admitted to Vietnam National Children‚Äôs Hospital from January 2022 to June 2023. The median age was 85 (range: 2‚Äì188) months, and only 12 (3.1%) patients had comorbidities. 161 (41.2%) patients required PICU admission, and the median PICU LOS was 4 (2‚Äì7) days.¬†We observed independent factors related to PICU¬†admission, including CRP\(\ge \)50 (mg/L) (OR 2.52, 95% CI 1.39‚Äì4.56, p‚Äâ=‚Äâ0.002),¬†albumin\(\le \)30 (g/L) (OR 3.18, 95% CI 1.63‚Äì6.02, p‚Äâ=‚Äâ0.001), absolute lymphocyte count\(\le \)2 (√ó‚Äâ109/L) (OR 2.18, 95% CI 1.29‚Äì3.71, p‚Äâ=‚Äâ0.004), ferritin‚Äâ‚â•‚Äâ300 (ng/mL) (OR 2.35, 95% CI 1.38‚Äì4.01), p‚Äâ=‚Äâ0.002), and LVEF‚Äâ<‚Äâ60 (%) (OR 2.48, 95% CI 1.28‚Äì4.78, p‚Äâ=‚Äâ0.007). Shock developed in 140 (35.8%) patients, especially for those decreased absolute lymphocyte\(\le \)2 (√ó‚Äâ109/L) (OR 2.48, 95% CI 1.10‚Äì5.61, p‚Äâ=‚Äâ0.029), albumin\(\le \)30 (g/L) (OR 2.53, 95% CI 1.22‚Äì5.24, p‚Äâ=‚Äâ0.013), or LVEF‚Äâ<‚Äâ60 (%) (OR 2.24, 95% CI 1.12‚Äì4.51, p‚Äâ=‚Äâ0.022). In conclusion, our study emphasized that absolute lymphocyte count, serum albumin, CRP, and LVEF were independent predictors for MIS-C severity. Further well-designed investigations are required to validate their efficacy in predicting MIS-C severe cases, especially compared to other parameters. As MIS-C is a new entity and severe courses may progress aggressively, identifying high-risk patients optimizes clinicians' follow-up and management to improve disease outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66891-4" target="_blank">https://www.nature.com/articles/s41598-024-66891-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-08</strong> <a href="https://www.nature.com/articles/s43856-024-00506-x" target="_blank">
            ü§ñ Multiple Health Conditions Linked to Serious and Long-Term COVID Effects</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-84', this)">
            More</button></h4>
            <div id="toggle-84" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Complex patterns of multimorbidity associated with severe COVID-19 and long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

When people get very sick with COVID-19, it's often those who already have other health problems that are at highest risk. To make sure we had enough hospital beds and vaccines during the pandemic, we looked into studies about this. But there aren't many detailed investigations on what kinds of medical conditions could lead to severe COVID-19 in people who were thought to be less likely to get very sick.

### METHODS ###

We took a large database from over 500,000 UK Biobank participants and grouped their health records into almost 15,000 categories. This was done to identify any diseases that might make someone more prone to severe COVID-19 or its lingering effects, Long COVID.

### RESULTS ###

We found about 679 diseases that are linked to a higher risk of getting very sick with COVID-19 (or experiencing Long COVID). These conditions cover almost all areas of medical specialties and are particularly common among cardio-respiratory and endocrine-renal diseases. For 57 of these diseases, we confirmed their connection to severe COVID-19 based on how likely people were to survive and genetic factors that might predispose them. Notably, symptoms like feeling unwell and fatigue could also be risk factors for getting very sick with COVID-19. Interestingly, some diseases related by cause, such as post-inflammatory lung scarring or rheumatoid arthritis, showed differing risks for severe COVID-19, possibly indicating separate mechanisms of how these conditions affect the body.

### CONCLUSIONS ###

Our findings provide a unique reference that shows how having multiple health issues can increase the risk of getting very sick with COVID-19 and how considering all types of medical diagnoses can help understand genetic risk factors.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundEarly evidence that patients with (multiple) pre-existing diseases are at highest risk for severe COVID-19 has been instrumental in the pandemic to allocate critical care resources and later vaccination schemes. However, systematic studies exploring the breadth of medical diagnoses are scarce but may help to understand severe COVID-19 among patients at supposedly low risk.MethodsWe systematically harmonized >12 million primary care and hospitalisation health records from ~500,000 UK Biobank participants into 1448 collated disease terms to systematically identify diseases predisposing to severe COVID-19 (requiring hospitalisation or death) and its post-acute sequalae, Long COVID.ResultsHere we identify 679 diseases associated with an increased risk for severe COVID-19 (n=‚Äâ672) and/or Long COVID (n=‚Äâ72) that span almost all clinical specialties and are strongly enriched in clusters of cardio-respiratory and endocrine-renal diseases. For 57 diseases, we establish consistent evidence to predispose to severe COVID-19 based on survival and genetic susceptibility analyses. This includes a possible role of symptoms of malaise and fatigue as a so far largely overlooked risk factor for severe COVID-19. We finally observe partially opposing risk estimates at known risk loci for severe COVID-19 for etiologically related diseases, such as post-inflammatory pulmonary fibrosis or rheumatoid arthritis, possibly indicating a segregation of disease mechanisms.ConclusionsOur results provide a unique reference that demonstrates how 1) complex co-occurrence of multiple ‚Äì including non-fatal ‚Äì conditions predispose to increased COVID-19 severity and 2) how incorporating the whole breadth of medical diagnosis can guide the interpretation of genetic risk loci.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00506-x" target="_blank">https://www.nature.com/articles/s43856-024-00506-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-66079-w" target="_blank">
            ü§ñ Long COVID: How Metal Perception Affects the Tongue</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-85', this)">
            More</button></h4>
            <div id="toggle-85" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impaired metal perception and regulation of associated human foliate papillae tongue transcriptome in long-COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People with COVID-19 often have trouble tasting or smelling things. We wondered if this is because of changes in the tiny bumps on their tongues called papillae. We asked hospital workers who had either already had COVID-19 and still had antibodies (IgG+) or hadn't (IgG-) to fill out a questionnaire about how they were sensing things like flavors and smells. Some of them also did a test where they had to choose which flavor they thought was being given to them, and others gave us tiny pieces of their tongues with the papillae for testing. We found that people who still had COVID-19 antibodies and couldn't taste metal flavors had even higher levels of those antibodies than people who could taste them. When we looked at the genes in the papillae from people who couldn't smell or taste well, we saw that they had trouble with a lot of genes related to smelling things. This makes us wonder if some of these genes might be helping us sense metal flavors and if they might be affected by COVID-19. We need more research to figure this out.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Chemosensory impairment is an outstanding symptom of SARS-CoV-2 infections. We hypothesized that measured sensory impairments are accompanied by transcriptomic changes in the foliate papillae area of the tongue. Hospital personnel with known SARS-CoV-2 immunoglobulin G (IgG) status completed questionnaires on sensory perception (n=‚Äâ158). A subcohort ofn=‚Äâ141 participated in forced choice taste tests, andn=‚Äâ43 participants consented to donate tongue swabs of the foliate papillae area for whole transcriptome analysis. The study included four groups of participants differing in IgG levels (‚â•‚Äâ10 AU/mL‚Äâ=‚ÄâIgG+;‚Äâ<‚Äâ10 AU/mL‚Äâ=‚ÄâIgG-) and self-reported sensory impairment (SSI¬±). IgG+subjects not detecting metallic taste had higher IgG+levels than IgG+participants detecting iron gluconate (p=‚Äâ0.03). Smell perception was the most impaired biological process in the transcriptome data from IgG+/SSI+participants subjected to gene ontology enrichment. IgG+/SSI+subjects demonstrated lower expression levels of 166 olfactory receptors (OR) and 9 taste associated receptors (TAS) of which OR1A2, OR2J2, OR1A1, OR5K1 and OR1G1, as well as TAS2R7 are linked to metallic perception. The question raised by this study is whether odorant receptors on the tongue (i) might play a role in metal sensation, and (ii) are potential targets for virus-initiated sensory impairments, which needs to be investigated in future functional studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66079-w" target="_blank">https://www.nature.com/articles/s41598-024-66079-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-66003-2" target="_blank">
            ü§ñ A Protein That Fights COVID-19 Infection in Human Airway Tissues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-86', this)">
            More</button></h4>
            <div id="toggle-86" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>NF-Œ∫B inhibitor alpha controls SARS-CoV-2 infection in ACE2-overexpressing human airway organoids</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As COVID-19 continues to spread, scientists need ways to study how it interacts with human airway cells. We've developed a 3D model of the airway epithelium using adult stem cells that can replicate the virus and maintain its natural structure and cell diversity. This model was infected with different COVID-19 variants and subjected to single-cell RNA sequencing, which showed that when cells were only slightly infected, they produced high levels of interferon-lambda, a protein that helps fight infections. In more heavily infected cells, a gene called IkBa was consistently upregulated, which helped regulate the infection but also allowed a protein called NF-kB to move into the cell's nucleus. When scientists overexpressed a mutant form of IkBa that couldn't be broken down, it reduced NF-kB translocation and increased viral infection, demonstrating the importance of this feedback loop in controlling viral replication.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As SARS-CoV-2 continues to spread worldwide, tractable primary airway cell models that recapitulate the cell-intrinsic response to arising viral variants are needed. Here we describe an adult stem cell-derived human airway organoid model overexpressing the ACE2 receptor (ACE2-OE) that supports robust viral replication while maintaining 3D architecture and cellular diversity of the airway epithelium. ACE2-OE organoids were infected with SARS-CoV-2 variants and subjected to single-cell RNA-sequencing. Interferon-lambda was¬†upregulated in cells with low-level infection while the NF-kB inhibitor alpha gene (encoding IkBa) was consistently upregulated in infected cells, and its expression positively correlated with infection levels. Confocal microscopy showed more IkBa expression in infected than bystander cells, but found concurrent nuclear translocation of NF-kB that IkBa usually prevents. Overexpressing a nondegradable IkBa mutant reduced NF-kB translocation and increased viral infection. These data demonstrate the functionality of ACE2-OE organoids in SARS-CoV-2 research and underscore that the strength of the NF-kB feedback loop in infected cells controls viral replication.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66003-2" target="_blank">https://www.nature.com/articles/s41598-024-66003-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-04</strong> <a href="https://www.nature.com/articles/s41598-024-65038-9" target="_blank">
            ü§ñ Hope in a Vial: Biomolecules in Hymenopteran Venom as COVID-19 Therapies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-87', this)">
            More</button></h4>
            <div id="toggle-87" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Potential therapeutic biomolecules of hymenopteran venom against SARS-CoV-2 from Egyptian patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The bioactive molecules found in the venom of certain insects might be able to help fight COVID-19. Researchers studied the venom from honeybees and Asian wasps, and found that it has potential therapeutic effects against SARS-CoV-2. The wasp venom performed better than the bee venom, reducing the virus by 90% at a concentration of 0.184 milligrams per milliliter. However, further studies are needed to confirm these results and explore their potential as antiviral treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The therapeutic potential of insect-derived bioactive molecules as anti-SARS-CoV-2 agents has shown promising results. Hymenopteran venoms, notably fromApis mellifera(honeybee) andVespa orientalis(oriental wasp), were examined for the first time in an in vitro setting for their potential anti-COVID-19 activity. This assessment utilized an immunodiagnostic system to detect the SARS-CoV-2 nucleocapsid antigen titer reduction. Further analyses, including cytotoxicity assays, plaque reduction assays, and in silico docking-based screening, were performed to evaluate the efficacy of the most potent venom. Results indicated that bee and wasp venoms contain bioactive molecules with potential therapeutic effects against SARS-CoV-2.Nevertheless, the wasp venom exhibited superior efficacy compared to bee venom, achieving a 90% maximal (EC90) concentration effect of antigen depletion at 0.184¬†mg/mL, in contrast to 2.23¬†mg/mL for bee venom. The cytotoxicity of the wasp venom was assessed on Vero E6 cells 48¬†h post-treatment using the MTT assay. The CC50of the cell growth was 0.16617¬†mg/mL for Vero E6 cells. The plaque reduction assay of wasp venom revealed 50% inhibition (IC50) at a 0.208¬†mg/mL concentration. The viral count at 50% inhibition was 2.5‚Äâ√ó‚Äâ104PFU/mL compared to the initial viral count of 5‚Äâ√ó‚Äâ104PFU/mL. In silico data for the wasp venom revealed a strong attraction to binding sites on the ACE2 protein, indicating ideal interactions. This substantiates the potential of wasp venom as a promising viral inhibitor against SARS-CoV-2, suggesting its consideration as a prospective natural preventive and curative antiviral drug. In conclusion, hymenopteran venoms, particularly wasp venom, hold promise as a source of potential therapeutic biomolecules against SARS-CoV-2. More research and clinical trials are needed to evaluate these results and investigate their potential for translation into innovative antiviral therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65038-9" target="_blank">https://www.nature.com/articles/s41598-024-65038-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-03</strong> <a href="https://www.nature.com/articles/s41467-024-49415-6" target="_blank">
            ü§ñ Mapping Viral Pathways: A New Way to Understand How Viruses Infect Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-88', this)">
            More</button></h4>
            <div id="toggle-88" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Recreating the biological steps of viral infection on a cell-free bioelectronic platform to profile viral variants of concern</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus is constantly changing and some of these changes can make it harder for us to detect and track the virus. To stay one step ahead, scientists need tools that can not only identify the presence of the virus but also predict how well it can infect cells. Our new electronic sensing platform simulates how the virus enters host cells and reports its progress as electrical signals. We focused on two key steps in this process: the virus attaching to the cell membrane and then fusing with it. These two steps can be distinguished using our platform, which allows us to identify specific patterns in the virus's behavior. Using this tool, we were able to see that even small changes in the virus could affect its ability to infect cells, and we hope that our platform will help scientists track and understand other viruses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Viral mutations frequently outpace technologies used to detect harmful variants. Given the continual emergence of SARS-CoV-2 variants, platforms that can identify the presence of a virus and its propensity for infection are needed. Our electronic biomembrane sensing platform recreates distinct SARS-CoV-2 host cell entry pathways and reports the progression of entry as electrical signals. We focus on two necessary entry processes mediated by the viral Spike protein: virus binding and membrane fusion, which can be distinguished electrically. We find that closely related variants of concern exhibit distinct fusion signatures that correlate with trends in cell-based infectivity assays, allowing us to report quantitative differences in their fusion characteristics and hence their infectivity potentials. We use SARS-CoV-2 as our prototype, but we anticipate that this platform can extend to other enveloped viruses and cell lines to quantifiably assess virus entry.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49415-6" target="_blank">https://www.nature.com/articles/s41467-024-49415-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-03</strong> <a href="https://www.nature.com/articles/s41467-024-49443-2" target="_blank">
            ü§ñ Vaccines May Protect Against Multiple COVID Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-89', this)">
            More</button></h4>
            <div id="toggle-89" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Three SARS-CoV-2 spike protein variants delivered intranasally by measles and mumps vaccines are broadly protective</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As new versions of the COVID-19 virus keep emerging, scientists are racing to create vaccines that can protect us from all these different variants. Here, researchers have developed a new type of vaccine that can be given through the nose and is effective against many different versions of the virus at once. They created this vaccine by combining three parts: one from the measles vaccine, one from the mumps vaccine, and a new part that specifically targets COVID-19. When tested in mice and hamsters, this new vaccine generated strong immune responses and protected them from infection with different strains of the virus. The good news is that people who already have immunity from other viruses like measles and mumps don't need to worry - it won't interfere with the effectiveness of this new COVID-19 vaccine. This new vaccine platform has a lot of potential as a next-generation COVID-19 vaccine candidate.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As the new SARS-CoV-2 Omicron variants and subvariants emerge, there is an urgency to develop intranasal, broadly protective vaccines. Here, we developed highly efficacious, intranasal trivalent SARS-CoV-2 vaccine candidates (TVC) based on three components of the MMR vaccine: measles virus (MeV), mumps virus (MuV) Jeryl Lynn (JL1) strain, and MuV JL2 strain. Specifically, MeV, MuV-JL1, and MuV-JL2 vaccine strains, each expressing prefusion spike (preS-6P) from a different variant of concern (VoC), were combined to generate TVCs. Intranasal immunization of IFNAR1‚àí/‚àímice and female hamsters with TVCs generated high levels of S-specific serum IgG antibodies, broad neutralizing antibodies, and mucosal IgA antibodies as well as tissue-resident memory T cells in the lungs. The immunized female hamsters were protected from challenge with SARS-CoV-2 original WA1, B.1.617.2, and B.1.1.529 strains. The preexisting MeV and MuV immunity does not significantly interfere with the efficacy of TVC. Thus, the trivalent platform is a promising next-generation SARS-CoV-2 vaccine candidate.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49443-2" target="_blank">https://www.nature.com/articles/s41467-024-49443-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-01</strong> <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">
            ü§ñ Contraceptive Access During COVID-19 Pandemic in Brazil</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-90', this)">
            More</button></h4>
            <div id="toggle-90" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A particular epidemiological profile: disparities in access to contraceptive methods in Brazil during the SARS-CoV-2 (COVID-19) pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, some people in Brazil had trouble getting access to birth control. We did a study to see who was affected most. We asked 1,018 people questions about their background and how they got birth control. Most of them were women between 20 and 39 years old and were white. Some people changed the way they got birth control during this time, and others wanted to try new methods. However, many had trouble finding a doctor who could help them with these changes or with continuing their current birth control method. The biggest problem was getting an appointment with a healthcare provider. Our study found that some groups of people, specifically younger black and biracial women with lower education and income, faced greater difficulties in accessing birth control and family planning services during this time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our objective was to study disparities in access to contraception during the COVID-19 pandemic. We performed a cross-sectional study at the University of Campinas, Brazil using a Google questionnaire applied from December 2021 until February 2022, disseminated via snowball technique. The survey asked about sociodemographic characteristics and contraceptive use, as well as the demand for new methods and difficulties in continuing to use contraceptives during the COVID-19 pandemic. We analyzed 1018 completed questionnaires; in total, 742 (72.9%) were women aged between 20 and 39¬†years, 746 (73.3%) were White and 602 (59.2%) used contraceptives. During the COVID-19 pandemic, about 23% of respondents changed their method and approximately 20% of respondents looked for new methods. Among the latter, 31.3% reported some difficulty with obtaining guidance on new methods while only 5.3% of the respondents reported some difficulty with continuing their contraceptive. The main difficulty in both cases was the difficulty with getting a healthcare provider appointment. Our results point to a particular epidemiological population, of younger black and biracial women, with lower education and lower income, which suffered health disparities during the COVID-19 pandemic and found difficulties with using contraceptives and accessing family planning services.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">https://www.nature.com/articles/s41598-024-65946-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-28</strong> <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">
            ü§ñ Tracking COVID-19 through the Body</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-91', this)">
            More</button></h4>
            <div id="toggle-91" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 antigenemia and RNAemia in association with disease severity in patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), also known as COVID-19, can cause symptoms ranging from mild cold-like symptoms to severe lung infections. Researchers studied the levels of viral proteins and genetic material in the blood of 117 patients with confirmed COVID-19 cases. They found that high levels of these viral components were associated with more severe illness and higher mortality rates. In particular, they discovered that patients who eventually died had extremely high levels of viral protein in their blood within the first week of hospitalization. This suggests that early detection of high viral protein levels could serve as an indicator of increased risk of death from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus responsible for COVID-19, causes a spectrum of symptoms ranging from mild upper to severe lower respiratory tract infections.However, the dynamics of nucleocapsid (N) protein antigenemia and RNAemia are not fully understood. We conducted a cohort study involving 117 patients with clinically confirmed COVID-19, focusing on the kinetics of antigenemia and RNAemia and their association with various clinical characteristics. The patients had a median age of 66.0¬†years (52.0‚Äì79.0¬†years), with a gender distribution of 46.2% male and 53.8% female. Antigenemia reached 100% in fatal cases during the first week after admission. The sensitivity/specificity of antigenemia for diagnosis were 64.7%/73.0% at admission, 69.1%/100% in Week 1, and 66.3%/100% in Week 2. Additionally, the rates of antigenemia in asymptomatic patients were 27.3% upon admission and 22.0% in Week 1, respectively; however, no antigenemia was in samples collected in Week 2. Viral RNAemia was not detected in asymptomatic patients, but RNAemia viral loads were elevated in fatal cases. Kaplan‚ÄìMeier survival curves demonstrated a higher mortality rate when antigenemia concentrations were elevated in the follow-up samples (P=‚Äâ0.005). Our study provides a comprehensive analysis of the kinetics of viral N-protein antigenemia and RNAemia according to disease severity and clinical classification. Our findings suggest that highest concentrations of antigenemia in fatal cases occur in the first week after admission, indicating that early elevated antigenemia may serve as a marker of mortality risk.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">https://www.nature.com/articles/s41598-024-65489-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-28</strong> <a href="https://www.nature.com/articles/s41598-024-66080-3" target="_blank">
            ü§ñ Revitalizing Huangshi's Urban Emergency Care</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-92', this)">
            More</button></h4>
            <div id="toggle-92" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improving urban emergency medical service systems through brownfield transformation in Huangshi, China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A city's emergency medical service (EMS) system is very important for saving lives during disasters. However, a recent study found that the current EMS system in Huangshi was not covering enough people, with only about 61% of the population having access to it. This was because the existing system had an inefficient layout and wasn't using some areas effectively.

To fix this problem, researchers proposed transforming old and unused urban spaces called brownfields into new EMS centers. They identified the best brownfields based on their size and how easily people could get to them from different parts of the city. By creating these new EMS parks, the coverage rate increased to about 90% of the population.

The researchers also came up with a new way for patients to be treated. Existing EMS facilities would serve as first-checkup centers where doctors could quickly assess patients and decide if they need more serious care. The urban brownfield EMS parks would then take in patients who need further treatment and isolation, making it easier for them to get the help they need.

Using computer maps and network analysis, the researchers were able to show that this new system would work well, with all patients being able to get to their next level of care within 30 minutes. The study's method - which includes assessing demand, optimizing resources, and checking feasibility - not only improved Huangshi's EMS system but also found ways to reuse urban brownfields. This approach could be helpful for other cities looking to improve their emergency medical services.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A comprehensive emergency medical service (EMS) system significantly enhances a city‚Äôs capacity to prevent and mitigate disasters. Using Huangshi as a case study, this research evaluated the service radium coverage rate of the current EMS system by examining its transport capacity, population density, and prevalence rate, finding it to be only 61.49% with an inefficient spatial layout. To address this, we proposed transforming urban brownfields into EMS parks. By selecting the most suitable brownfields based on capacity and service radius, we increased the coverage rate to 90.21%. We introduced a new "consultation-referral" model, where existing EMS facilities serve as pre-diagnosis and triage centers, and the urban brownfield EMS parks function as isolation and centralized treatment centers. GIS network analysis confirmed the feasibility, showing all transit times to be under 30¬†min. The methodology outlined in this study‚Äîcomprising "demand assessment, supply optimization, and feasibility verification"‚Äînot only strengthens the city‚Äôs EMS system but also facilitates the renewal of urban brownfields. This approach can serve as a valuable reference for enhancing EMS systems in other cities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66080-3" target="_blank">https://www.nature.com/articles/s41598-024-66080-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-25</strong> <a href="https://www.nature.com/articles/s41598-024-65471-w" target="_blank">
            ü§ñ Fractional Waves in Nonlinear Systems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-93', this)">
            More</button></h4>
            <div id="toggle-93" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Solitary wave solutions of the time fractional Benjamin Bona Mahony Burger equation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The time fractional Benjamin Bona Mahony Burger equation is important for understanding how water waves and certain types of sound waves behave. It's also used to describe how some kinds of plasma waves move in crystals. In this study, we used a new method to solve this equation, which involves different ways of describing how things change over time. We tested this method with some sample numbers and compared the results to what we know is true, as well as to other methods that have been used before. What we found was that using this new method can help us understand how certain types of waves move in a more accurate way, especially when they are shaped like bells and influenced by specific mathematical functions. Our results showed that this approach works well and can be useful for solving similar problems in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The present study examines the approximate solutions of the time fractional Benjamin Bona Mahony Burger equation. This equation is critical for characterizing the dynamics of water waves and fluid acoustic gravity waves, as well as explaining the unidirectional propagation of long waves in nonlinear dispersive systems. This equation also describes cold plasma for hydromagnetic and audio waves in harmonic crystals. The natural transform decomposition method is used to obtain the analytical solution to the time fractional Benjamin Bona Mahony Burger equation. The proposed method uses the Caputo, Caputo Fabrizio, and Atangana Baleanu Caputo derivatives to describe the fractional derivative. We utilize a numerical example with appropriate initial conditions to assess the correctness of our findings. The results of the proposed method are compared to those of the exact solution and various existing techniques, such as the fractional homotopy analysis transform method and the homotopy perturbation transform technique. As a result, bell shaped solitons are discovered under the influence of hyperbolic functions. By comparing the outcomes with tables and graphs, the findings demonstrate the efficacy and effectiveness of the suggested approach.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65471-w" target="_blank">https://www.nature.com/articles/s41598-024-65471-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-21</strong> <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">
            ü§ñ COVID-19 Wastewater Tracking in Ontario</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-94', this)">
            More</button></h4>
            <div id="toggle-94" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 viral titer measurements in Ontario, Canada wastewaters throughout the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, a program was started to test wastewater from across Ontario, Canada, to see if there were any signs of the virus. This helped predict when cases might rise or fall, which was very useful. In this study, we have a big set of data that shows how SARS-CoV-2 and another type of virus called PMMoV were present in wastewater from 107 different locations across Ontario over three years. We also included information about where each location is, how many people it serves, and what the water looks like. This will help future researchers and public health experts understand and prepare for similar situations in the future. The data has been especially helpful in tracking COVID-19 cases locally, particularly when there were fewer clinical tests being done, which highlights the importance of wastewater surveillance in monitoring disease spread.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, the Province of Ontario, Canada, launched a wastewater surveillance program to monitor SARS-CoV-2, inspired by the early work and successful forecasts of COVID-19 waves in the city of Ottawa, Ontario. This manuscript presents a dataset from January 1, 2021, to March 31, 2023, with RT-qPCR results for SARS-CoV-2 genes and PMMoV from 107 sites across all 34 public health units in Ontario, covering 72% of the province‚Äôs and 26.2% of Canada‚Äôs population. Sampling occurred 2‚Äì7 times weekly, including geographical coordinates, serviced populations, physico-chemical water characteristics, and flowrates. In doing so, this manuscript ensures data availability and metadata preservation to support future research and epidemic preparedness through detailed analyses and modeling. The dataset has been crucial for public health in tracking disease locally, especially with the rise of the Omicron variant and the decline in clinical testing, highlighting wastewater-based surveillance‚Äôs role in estimating disease incidence in Ontario.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">https://www.nature.com/articles/s41597-024-03414-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-12</strong> <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">
            ü§ñ Most Older COVID Immunity Fights Familiar Strains, Not New Ones</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-95', this)">
            More</button></h4>
            <div id="toggle-95" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nonconserved epitopes dominate reverse preexisting T cell immunity in COVID-19 convalescents</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The body is getting better at fighting the new coronavirus as it spreads around the world. But scientists wondered if people who had already recovered from COVID-19 might also have some protection against other, similar viruses. They looked at how T cells (a type of immune cell) responded to peptides (short pieces of protein) from both the new and old coronaviruses. They found that in some people who had recovered from COVID-19, their T cells were also able to recognize and fight the old virus. They identified specific parts of the new coronavirus that could trigger this cross-protection, and showed that these peptides are more stable when they bind to a special protein on immune cells called HLA. This discovery could help create vaccines that protect against multiple coronaviruses at once.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The herd immunity against SARS-CoV-2 is continuously consolidated across the world during the ongoing pandemic. However, the potential function of the nonconserved epitopes in the reverse preexisting cross-reactivity induced by SARS-CoV-2 to other human coronaviruses is not well explored. In our research, we assessed T cell responses to both conserved and nonconserved peptides shared by SARS-CoV-2 and SARS-CoV, identifying cross-reactive CD8+T cell epitopes using enzyme-linked immunospot and intracellular cytokine staining assays. Then, in vitro refolding and circular dichroism were performed to evaluate the thermal stability of the HLA/peptide complexes. Lastly, single-cell T cell receptor reservoir was analyzed based on tetramer staining. Here, we discovered that cross-reactive T cells targeting SARS-CoV were present in individuals who had recovered from COVID-19, and identified SARS-CoV-2 CD8+T cell epitopes spanning the major structural antigens. T cell responses induced by the nonconserved peptides between SARS-CoV-2 and SARS-CoV were higher and played a dominant role in the cross-reactivity in COVID-19 convalescents. Cross-T cell reactivity was also observed within the identified series of CD8+T cell epitopes. For representative immunodominant peptide pairs, although the HLA binding capacities for peptides from SARS-CoV-2 and SARS-CoV were similar, the TCR repertoires recognizing these peptides were distinct. Our results could provide beneficial information for the development of peptide-based universal vaccines against coronaviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">https://www.nature.com/articles/s41392-024-01876-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-11</strong> <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">
            ü§ñ Following Antibodies After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-96', this)">
            More</button></h4>
            <div id="toggle-96" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Tracking the evolution of anti-SARS-CoV-2 antibodies and long-term humoral immunity within 2¬†years after COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that a young person could understand:

The COVID-19 virus has caused big problems around the world. To understand how to make people immune to it, scientists need to know how our bodies respond to the infection. One way our bodies fight the virus is by producing special proteins called antibodies. These antibodies can help us recover from the illness and even give us protection against getting sick again in the future.

Scientists looked at blood samples from over 16,000 people who had recovered from COVID-19. They wanted to see how long these antibodies lasted and what made them stronger or weaker. They found that one type of antibody, called anti-SARS-CoV-2 (N), peaked at four months after infection but stayed strong for a long time. Another type, called anti-SARS-CoV-2 (S), kept getting stronger over the 15-month period they studied. This suggests that our bodies can make these antibodies in different ways and that they may provide lasting protection against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that gave rise to COVID-19 infection produced a worldwide health crisis. The virus can cause a serious or even fatal disease. Comprehending the complex immunological responses triggered by SARS-CoV-2 infection is essential for identifying pivotal elements that shape the course of the disease and its enduring effects on immunity. The span and potency of antibody responses provide valuable perspicuity into the resilience of post-infection immunity. The analysis of existing literature reveals a diverse controversy, confining varying data about the persistence of particular antibodies as well as the multifaceted factors that impact their development and titer, Within this study we aimed to understand the dynamics of anti-SARS-CoV-2 antibodies against nucleocapsid (anti-SARS-CoV-2 (N)) and spike (anti-SARS-CoV-2 (N)) proteins in long-term immunity in convalescent patients, as well as the factors influencing the production and kinetics of those antibodies. We collected 6115 serum samples from 1611 convalescent patients at different post-infection intervals up to 21¬†months Study showed that in the fourth month, the anti-SARS-CoV-2 (N) exhibited their peak mean value, demonstrating a 79% increase compared to the initial month. Over the subsequent eight months, the peak value experienced a modest decline, maintaining a relatively elevated level by the end of study. Conversely, anti-SARS-CoV-2 (S) exhibited a consistent increase at each three-month interval over the 15-month period, culminating in a statistically significant peak mean value at the study‚Äôs conclusion. Our findings demonstrate evidence of sustained seropositivity rates for both anti-SARS-CoV-2 (N) and (S), as well as distinct dynamics in the long-term antibody responses, with anti-SARS-CoV-2 (N) levels displaying remarkable persistence and anti-SARS-CoV-2 (S) antibodies exhibiting a progressive incline.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">https://www.nature.com/articles/s41598-024-64414-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-11</strong> <a href="https://www.nature.com/articles/s41598-024-64367-z" target="_blank">
            ü§ñ Lost Sense of Smell Linked to Shrinkage in Brain Area After Mild COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-97', this)">
            More</button></h4>
            <div id="toggle-97" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reduced olfactory bulb volume accompanies olfactory dysfunction after mild SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The ability to smell is a common problem after someone gets COVID-19, but scientists don't fully understand why this happens. In this study, researchers used MRI scans to look at the size of the olfactory bulb (a part of the brain that helps us smell) in people who had recovered from mild to moderate cases of COVID-19. They also asked these people about their sense of smell and did tests to see how well they could smell. The results showed that people who said they still had trouble smelling after recovering from COVID-19 had smaller olfactory bulbs on their MRI scans. The size of the olfactory bulb at the beginning of the study also predicted how well people could smell later on. This study suggests that the size of the olfactory bulb might be a helpful indicator for diagnosing and treating problems with sense of smell after COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite its high prevalence, the determinants of smelling impairment in COVID-19 remain not fully understood. In this work, we aimed to examine the association between olfactory bulb volume and the clinical trajectory of COVID-19-related smelling impairment in a large-scale magnetic resonance imaging (MRI) analysis. Data of non-vaccinated COVID-19 convalescents recruited within the framework of the prospective Hamburg City Health Study COVID Program between March and December 2020 were analyzed. At baseline, 233 participants underwent MRI and neuropsychological testing as well as a structured questionnaire for olfactory function. Between March and April 2022, olfactory function was assessed at follow-up including quantitative olfactometric testing with Sniffin‚Äô Sticks. This study included 233 individuals recovered from mainly mild to moderate SARS-CoV-2 infections. Longitudinal assessment demonstrated a declining prevalence of self-reported olfactory dysfunction from 67.1% at acute infection, 21.0% at baseline examination and 17.5% at follow-up. Participants with post-acute self-reported olfactory dysfunction had a significantly lower olfactory bulb volume at baseline than normally smelling individuals. Olfactory bulb volume at baseline predicted olfactometric scores at follow-up. Performance in neuropsychological testing was not significantly associated with the olfactory bulb volume. Our work demonstrates an association of long-term self-reported smelling dysfunction and olfactory bulb integrity in a sample of individuals recovered from mainly mild to moderate COVID-19. Collectively, our results highlight olfactory bulb volume as a surrogate marker that may inform diagnosis and guide rehabilitation strategies in COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64367-z" target="_blank">https://www.nature.com/articles/s41598-024-64367-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-07</strong> <a href="https://www.nature.com/articles/s41366-024-01556-w" target="_blank">
            ü§ñ Sleep, Obesity, and Long COVID: A Study of Vaccinated Individuals</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-98', this)">
            More</button></h4>
            <div id="toggle-98" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Associations between obesity, a composite risk score for probable long COVID, and sleep problems in SARS-CoV-2 vaccinated individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract:

Imagine you got vaccinated against a bad virus called SARS-CoV-2. But even after getting two shots, some people might still feel unwell for a longer time than others. Scientists wanted to know if being overweight or obese could make this worse. They looked at data from 59,000 adults who had gotten vaccinated and found that people with obesity were more likely to have symptoms of long COVID, like fatigue and breathing problems. They also saw that both obesity and long COVID were linked to sleep disturbances, such as insomnia and sleep apnea. However, the researchers couldn't confirm if being overweight or obese directly caused these problems. More studies are needed to understand this connection better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundPreliminary data suggests that obesity might hasten the decline in mRNA vaccine-induced immunity against SARS-CoV-2. However, whether this renders individuals with obesity more susceptible to long COVID symptoms post-vaccination remains uncertain. Given sleep‚Äôs critical role in immunity, exploring the associations between obesity, probable long COVID symptoms, and sleep disturbances is essential.MethodsWe analyzed data from a survey of 5919 adults aged 18 to 89, all of whom received two SARS-CoV-2 mRNA vaccinations. Participants were categorized into normal weight, overweight, and obesity groups based on ethnicity-specific BMI cutoffs. The probability of long COVID was evaluated using the Post-Acute Sequelae of SARS-CoV-2 (PASC) score, as our survey did not permit confirmation of acute SARS-CoV-2 infection through methods such as antibody testing. Additionally, sleep patterns were assessed through questionnaires.ResultsParticipants with obesity exhibited a significantly higher adjusted odds ratio (OR) of having a PASC score of 12 or higher, indicative of probable long COVID in our study, compared to those with normal weight (OR: 1.55, 95% CI: 1.05, 2.28). No significant difference was observed for overweight individuals (OR: 0.92 [95% CI: 0.63, 1.33]). Both obesity and probable long COVID were associated with increased odds of experiencing a heightened sleep burden, such as the presence of obstructive sleep apnea or insomnia (P<‚Äâ0.001). However, no significant interaction between BMI and probable long COVID status was found.ConclusionsEven post-vaccination, individuals with obesity may encounter a heightened risk of experiencing prolonged COVID-19 symptoms. However, confirming our observations necessitates comprehensive studies incorporating rigorous COVID infection testing, such as antibody assays - unavailable in our anonymous survey. Additionally, it is noteworthy that the correlation between probable long COVID and sleep disturbances appears to be independent of BMI.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41366-024-01556-w" target="_blank">https://www.nature.com/articles/s41366-024-01556-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-03</strong> <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">
            ü§ñ Coronavirus Antibodies Linked to Severity of Mild Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-99', this)">
            More</button></h4>
            <div id="toggle-99" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Titers of IgG and IgA against SARS-CoV-2 proteins and their association with symptoms in mild COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract for a young person:

When people get COVID-19, their bodies produce special proteins called antibodies to fight the virus. These antibodies can target different parts of the virus, like the spike or nucleocapsid. If someone has mild COVID-19, we don't know as much about their antibody levels. In this study, scientists looked at the antibody levels in 200 unvaccinated hospital workers who had mild COVID-19 and took two blood samples from them after they got infected. They found that people with certain symptoms or characteristics had different levels of antibodies against specific parts of the virus. For example, people who had a fever when they first got sick had higher antibody levels than those who didn't get as sick. Over time, some antibody levels went down, but others stayed the same. This study helped us understand how our bodies respond to COVID-19 and what might happen if we get symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humoral immunity in COVID-19 includes antibodies (Abs) targeting spike (S) and nucleocapsid (N) SARS-CoV-2 proteins. Antibody levels are known to correlate with disease severity, but titers are poorly reported in mild or asymptomatic cases. Here, we analyzed the titers of IgA and IgG against SARS-CoV-2 proteins in samples from 200 unvaccinated Hospital Workers (HWs) with mild COVID-19 at two time points after infection. We analyzed the relationship between Ab titers and patient characteristics, clinical features, and evolution over time. Significant differences in IgG and IgA titers against N, S1 and S2 proteins were found when samples were segregated according to time T1 after infection, seroprevalence at T1, sex and age of HWs and symptoms at infection. We found that IgM‚Äâ+‚Äâsamples had higher titers of IgG against N antigen and IgA against S1 and S2 antigens than IgM‚Äâ‚àí‚Äâsamples. There were significant correlations between anti-S1 and S2 Abs. Interestingly, IgM‚Äâ+‚Äâpatients with dyspnea had lower titers of IgG and IgA against N, S1 and S2 than those without dyspnea. Comparing T1 and T2, we found that IgA against N, S1 and S2 but only IgG against certain Ag decreased significantly. In conclusion, an association was established between Ab titers and the development of infection symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">https://www.nature.com/articles/s41598-024-59634-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-30</strong> <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">
            ü§ñ Long-Term Recovery from Post-COVID Health Issues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-100', this)">
            More</button></h4>
            <div id="toggle-100" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Three-year outcomes of post-acute sequelae of COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Severe coronavirus infections can cause long-term health problems for many people. These problems, called post-acute sequelae of COVID-19 (PASC), can affect different parts of the body. Most studies have only looked at these issues for up to 2 years after infection, but we wanted to know what happens beyond that. We studied a group of over 135,000 people who had been infected with coronavirus and compared them to a similar number of people who didn't get infected. We followed both groups for 3 years to see how many of the people who got infected died or developed PASC during that time. For people who weren't hospitalized when they got infected, we found that the risk of death went away after the first year, and the risk of new health problems due to PASC decreased over time. However, even in the third year, these people still experienced 9-10 extra health issues for every 1,000 people. On the other hand, people who were hospitalized when they got infected had a lower but still increased risk of death in the third year after infection (about 29% more likely). They also continued to have significant new health problems due to PASC during that time, resulting in about 90 extra health issues for every 1,000 people. Overall, our study shows that while risks decrease over time, there is still a significant burden of death and health loss among people who were hospitalized when they got infected with coronavirus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection causes post-acute sequelae of coronavirus disease 2019 (COVID-19) (PASC) in many organ systems. Risks of these sequelae have been characterized up to 2‚Äâyears after infection, but longer-term follow-up is limited. Here we built a cohort of 135,161 people with SARS-CoV-2 infection and 5,206,835 controls from the US Department of Veterans Affairs who were followed for 3‚Äâyears to estimate risks of death and PASC. Among non-hospitalized individuals, the increased risk of death was no longer present after the first year of infection, and risk of incident PASC declined over the 3‚Äâyears but still contributed 9.6 (95% confidence interval (CI): 0.4‚Äì18.7) disability-adjusted life years (DALYs) per 1,000 persons in the third year. Among hospitalized individuals, risk of death declined but remained significantly elevated in the third year after infection (incidence rate ratio: 1.29 (95% CI: 1.19‚Äì1.40)). Risk of incident PASC declined over the 3‚Äâyears, but substantial residual risk remained in the third year, leading to 90.0 (95% CI: 55.2‚Äì124.8) DALYs per 1,000 persons. Altogether, our findings show reduction of risks over time, but the burden of mortality and health loss remains in the third year among hospitalized individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">https://www.nature.com/articles/s41591-024-02987-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-29</strong> <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">
            ü§ñ Long-Term Protection Against COVID-19 Re-Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-101', this)">
            More</button></h4>
            <div id="toggle-101" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Initial COVID-19 severity influenced by SARS-CoV-2-specific T cells imprints T-cell memory and inversely affects reinfection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The body's immune response to COVID-19 can affect how severe the disease is and whether someone gets infected again in the long run. We studied 165 people who had COVID-19 and found that their immune system responded differently depending on how sick they were when they first got the virus. People who recovered quickly from mild cases of COVID-19 had a strong immune response, which helped them get better faster. On the other hand, people who got very sick with COVID-19 took longer to recover and had a weaker immune response. This is important because it can affect how likely someone is to get infected again in the future. Our study helps us understand how the body's immune response affects COVID-19 and can guide doctors on how to manage future outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The immunoprotective components control COVID-19 disease severity, as well as long-term adaptive immunity maintenance and subsequent reinfection risk discrepancies across initial COVID-19 severity, remain unclarified. Here, we longitudinally analyzed SARS-CoV-2-specific immune effectors during the acute infection and convalescent phases of 165 patients with COVID-19 categorized by severity. We found that early and robust SARS-CoV-2-specific CD4+and CD8+T cell responses ameliorate disease progression and shortened hospital stay, while delayed and attenuated virus-specific CD8+T cell responses are prominent severe COVID-19 features. Delayed antiviral antibody generation rather than titer level associates with severe outcomes. Conversely, initial COVID-19 severity imprints the long-term maintenance of SARS-CoV-2-specific adaptive immunity, demonstrating that severe convalescents exhibited more sustained virus-specific antibodies and memory T cell responses compared to mild/moderate counterparts. Moreover, initial COVID-19 severity inversely correlates with SARS-CoV-2 reinfection risk. Overall, our study unravels the complicated interaction between temporal characteristics of virus-specific T cell responses and COVID-19 severity to guide future SARS-CoV-2 wave management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">https://www.nature.com/articles/s41392-024-01867-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-27</strong> <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">
            ü§ñ COVID-19 Respiratory Complications After SARS Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-102', this)">
            More</button></h4>
            <div id="toggle-102" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Acute and post-acute respiratory complications of SARS-CoV-2 infection: population-based cohort study in South Korea and Japan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who had COVID-19 are more likely to have breathing problems or long-term lung issues than the general population. A study looked at two large groups of people in Korea (2.3 million) and Japan (3.1 million) to see if this was true. They found that 17% of Koreans who were part of the study had COVID-19, and those people were more likely to have breathing problems or long-term lung issues than others. The risk of having these problems increased with how severe their COVID-19 symptoms were. However, getting vaccinated against COVID-19 seemed to help reduce this risk. The study also found that even six months after recovering from COVID-19, some people still had a higher risk of breathing problems or long-term lung issues than others. This study helps us understand the potential respiratory complications that can occur after COVID-19 infection and how factors like vaccination status and time since infection can affect this risk.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Considering the significant burden of post-acute COVID-19 conditions among patients infected with SARS-CoV-2, we aimed to identify the risk of acute respiratory complications or post-acute respiratory sequelae. A binational population-based cohort study was conducted to analyze the risk of acute respiratory complications or post-acute respiratory sequelae after SARS-CoV-2 infection. We used a Korean nationwide claim-based cohort (K-COV-N;n=‚Äâ2,312,748; main cohort) and a Japanese claim-based cohort (JMDC;n=‚Äâ3,115,606; replication cohort) after multi-to-one propensity score matching. Among 2,312,748 Korean participants (mean age, 47.2‚Äâyears [SD, 15.6]; 1,109,708 [48.0%] female), 17.1% (394,598/2,312,748) were infected with SARS-CoV-2. The risk of acute respiratory complications or post-acute respiratory sequelae is significantly increased in people with SARS-CoV-2 infection compared to the general population (acute respiratory complications: HR, 8.06 [95% CI, 6.92-9.38]; post-acute respiratory sequelae: 1.68 [1.62-1.75]), and the risk increased with increasing COVID-19 severity. We identified COVID-19 vaccination as an attenuating factor, showing a protective association against acute or post-acute respiratory conditions. Furthermore, while the excess post-acute risk diminished with time following SARS-CoV-2 infection, it persisted beyond 6‚Äâmonths post-infection. The replication cohort showed a similar pattern in the association. Our study comprehensively evaluates respiratory complications in post-COVID-19 conditions, considering attenuating factors such as vaccination status, post-infection duration, COVID-19 severity, and specific respiratory conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">https://www.nature.com/articles/s41467-024-48825-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-22</strong> <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">
            ü§ñ COVID After Vaccination: What Happens to People Who Get Sick After Getting Vaccinated?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-103', this)">
            More</button></h4>
            <div id="toggle-103" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID conditions following COVID-19 vaccination: a retrospective matched cohort study of patients with SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person could understand:

We wanted to see if people who got vaccinated against COVID-19 were less likely to get new health problems after catching the virus. We looked at data from eight big healthcare systems in the US and found that people who got vaccinated had a lower risk of many new health problems, like skin issues or blood problems, compared to those who didn't get vaccinated. However, people who got vaccinated were slightly more likely to have mental health problems. Overall, getting vaccinated before catching COVID-19 might help reduce the long-term effects of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 vaccinations protect against severe illness and death, but associations with post-COVID conditions (PCC) are less clear. We aimed to evaluate the association between prior COVID-19 vaccination and new-onset PCC among individuals with SARS-CoV-2 infection across eight large healthcare systems in the United States. This retrospective matched cohort study used electronic health records (EHR) from patients with SARS-CoV-2 positive tests during March 2021-February 2022. Vaccinated and unvaccinated COVID-19 cases were matched on location, test date, severity of acute infection, age, and sex. Vaccination status was ascertained using EHR and integrated data on externally administered vaccines. Adjusted relative risks (RRs) were obtained from Poisson regression. PCC was defined as a new diagnosis in one of 13 PCC categories 30 days to 6 months following a positive SARS-CoV-2 test. The study included 161,531 vaccinated COVID-19 cases and 161,531 matched unvaccinated cases. Compared to unvaccinated cases, vaccinated cases had a similar or lower risk of all PCC categories except mental health disorders (RR: 1.06, 95% CI: 1.02‚Äì1.10). Vaccination was associated with ‚â•10% lower risk of sensory (RR: 0.90, 0.86‚Äì0.95), circulatory (RR: 0.88, 0.83‚Äì0.94), blood and hematologic (RR: 0.79, 0.71‚Äì0.89), skin and subcutaneous (RR: 0.69, 0.66‚Äì0.72), and non-specific COVID-19 related disorders (RR: 0.53, 0.51‚Äì0.56). In general, associations were stronger at younger ages but mostly persisted regardless of SARS-CoV-2 variant period, receipt of ‚â•3 vs. 1‚Äì2 vaccine doses, or time since vaccination. Pre-infection vaccination was associated with reduced risk of several PCC outcomes and hence may decrease the long-term consequences of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">https://www.nature.com/articles/s41467-024-48022-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-16</strong> <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">
            ü§ñ Viral Traffic: How People Moving Around Spread Respiratory Viruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-104', this)">
            More</button></h4>
            <div id="toggle-104" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impacts of human mobility on the citywide transmission dynamics of 18 respiratory viruses in pre- and post-COVID-19 pandemic years</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many scientists have used data from people's phones to understand how diseases like COVID-19 spread. However, they know less about how other common viruses that cause respiratory infections move through a population. We looked at the relationship between people moving around in Seattle over four years and the spread of 17 different viruses that are always present in the community, as well as COVID-19. Before 2020, when kids went back to school and people started mixing more within the city, or when tourists came into town, it often coincided with an increase in these viruses. But after the COVID-19 lockdown in March 2020, the number of these viruses circulating in the community dropped significantly. During this time, how much people were moving around was a good indicator of whether these viruses would start spreading again. However, when restrictions on movement were lifted, it took some time for these viruses to come back and their relationships with mobility patterns became weaker. The return of these viruses happened at different times, but they had stronger and longer-lasting connections with how much people were moving around than COVID-19 did. Overall, knowing where people are going and what they're doing is most helpful in predicting the spread of respiratory viruses during times of big changes in behavior or when new outbreaks start.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Many studies have used mobile device location data to model SARS-CoV-2 dynamics, yet relationships between mobility behavior and endemic respiratory pathogens are less understood. We studied the effects of population mobility on the transmission of 17 endemic viruses and SARS-CoV-2 in Seattle over a 4-year period, 2018-2022. Before 2020, visits to schools and daycares, within-city mixing, and visitor inflow preceded or coincided with seasonal outbreaks of endemic viruses. Pathogen circulation dropped substantially after the initiation of COVID-19 stay-at-home orders in March 2020. During this period, mobility was a positive, leading indicator of transmission of all endemic viruses and lagging and negatively correlated with SARS-CoV-2 activity. Mobility was briefly predictive of SARS-CoV-2 transmission when restrictions relaxed but associations weakened in subsequent waves. The rebound of endemic viruses was heterogeneously timed but exhibited stronger, longer-lasting relationships with mobility than SARS-CoV-2. Overall, mobility is most predictive of respiratory virus transmission during periods of dramatic behavioral change and at the beginning of epidemic waves.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">https://www.nature.com/articles/s41467-024-48528-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-30</strong> <a href="https://www.nature.com/articles/s41598-024-60372-4" target="_blank">
            ü§ñ Life After Covid-19: How Children's Bodies Recover 1 Year Later</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-105', this)">
            More</button></h4>
            <div id="toggle-105" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-Covid-19 condition (Long Covid) in children and young people 12 months after infection or reinfection with the Omicron variant: a prospective observational study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Children and young people who got infected with the Omicron variant of COVID-19 were studied one year after their initial diagnosis. The researchers wanted to see if there were any differences in how they felt or what symptoms they had, depending on whether it was their first time getting infected or if they had been reinfected. They found that many kids and young people still experienced fatigue and sleep problems a year later, but the severity of these symptoms didn't differ much between those who got infected for the first time and those who were reinfected. Overall, about 18% of the children and young people in both groups had symptoms that lasted longer than expected, but there wasn't a significant difference between them. This suggests that clinicians might not need to consider how many times someone has been infected with Omicron or what variant they have when developing treatment plans. However, more studies are needed to confirm these findings and understand the cause of the lingering symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our previous study in children and young people (CYP) at 3- and 6-months post-infection showed that 12‚Äì16% of those infected with the Omicron (B.1.1.529) variant of SARS-CoV-2 met the research definition of Long Covid, with no differences between first-positive and reinfected CYP. The primary objective of the current study is to explore the impact of the Omicron variant of SARS-CoV-2 infection on young people 12 months post infection. 345 CYP aged 11‚Äì17¬†years with a first laboratory-confirmed infection with the Omicron variant and 360 CYP reinfected with the Omicron variant completed an online questionnaire assessing demographics, symptoms, and their impact shortly after testing and again at 3-, 6-and 12-months post-testing. Vaccination status was determined from information held at UKHSA. Comparisons between groups were made using chi-squared, Mann‚ÄìWhitney U, and Kruskal‚ÄìWallis tests. The most common symptoms in first-positive and reinfected CYP 12-months post-testing were tiredness (35.7 and 33.6% respectively) and sleeping difficulties (27.5 and 28.3% respectively). Symptom profiles, severity and impact were similar in the two infection status groups. Overall, by 12-months, 17.4% of first-positives and 21.9% of reinfected CYP fulfilled the research consensus Long Covid definition (p=‚Äâ0.13). 12-months post Omicron infection, there is little difference between first-positive and reinfected CYP with respect to symptom profiles and impact. Clinicians may not therefore need to consider number of infections and type of variant when developing treatment plans. Further studies are needed to assess causality of reported symptoms up to 12-months after SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-60372-4" target="_blank">https://www.nature.com/articles/s41598-024-60372-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-29</strong> <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">
            ü§ñ Uncovering the Changing Nature of COVID-19 Virus Mutations in a Transplant Patient</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-106', this)">
            More</button></h4>
            <div id="toggle-106" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamic diversity of SARS-CoV-2 genetic mutations in a lung transplantation patient with persistent COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract for a young person:

Imagine you have a weakened immune system, like someone who has had a transplant or is taking medicines that make their body more vulnerable to infections. Even with treatments like Remdesivir, which helps fight COVID-19 by blocking the virus's ability to multiply, these patients can sometimes develop persistent viral infections that don't respond to treatment.

In one such case, a patient developed COVID-19 caused by a variant of the Omicron strain and continued to have the virus in their system even after taking Remdesivir. Our analysis showed that the virus underwent many changes as it tried to evade the treatment, eventually developing mutations that made it resistant to Remdesivir.

The specific mutation we found was in a protein called nsp12, which helps the virus multiply by allowing it to use RNA-dependent RNA polymerase. This mutation prevented the drug from working effectively and allowed the virus to continue spreading in the patient's system. Our findings highlight the challenges of treating COVID-19 in immunosuppressed patients and the importance of monitoring for emerging mutations that could threaten their health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Numerous SARS-CoV-2 variant strains with altered characteristics have emerged since the onset of the COVID-19 pandemic. Remdesivir (RDV), a ribonucleotide analogue inhibitor of viral RNA polymerase, has become a valuable therapeutic agent. However, immunosuppressed hosts may respond inadequately to RDV and develop chronic persistent infections. A patient with respiratory failure caused by interstitial pneumonia, who had undergone transplantation of the left lung, developed COVID-19 caused by Omicron BA.5 strain with persistent chronic viral shedding, showing viral fusogenicity. Genome-wide sequencing analyses revealed the occurrence of several viral mutations after RDV treatment, followed by dynamic changes in the viral populations. The C799F mutation in nsp12 was found to play a pivotal role in conferring RDV resistance, preventing RDV-triphosphate from entering the active site of RNA-dependent RNA polymerase. The occurrence of diverse mutations is a characteristic of SARS-CoV-2, which mutates frequently. Herein, we describe the clinical case of an immunosuppressed host in whom inadequate treatment resulted in highly diverse SARS-CoV-2 mutations that threatened the patient‚Äôs health due to the development of drug-resistant variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">https://www.nature.com/articles/s41467-024-47941-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-17</strong> <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">
            ü§ñ Boosting Immunity After Long COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-107', this)">
            More</button></h4>
            <div id="toggle-107" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improvement of immune dysregulation in individuals with long COVID at 24-months following SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT

This study looked at how people's bodies react to a long time after they've had COVID-19. It compared those who got mild to moderate long COVID (LC) with others who already recovered from COVID-19 (MC). The researchers found that LC patients' immune systems were more active for up to 8 months, but eventually returned to normal like the MC group's after 24 months.

The study also showed that people with LC had higher levels of antibodies against COVID-19 at certain points in time. However, their immune cells, known as T and B cells, behaved similarly to those who had already recovered from COVID-19 over time.

Most importantly, about 62% of the people in the LC group reported feeling better and having improved health-related quality of life after 24 months. Some factors like PTX3 levels (a marker of inflammation), CRP levels (another inflammation marker), and platelet count were linked to these improvements.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigates the humoral and cellular immune responses and health-related quality of life measures in individuals with mild to moderate long COVID (LC) compared to age and gender matched recovered COVID-19 controls (MC) over 24 months. LC participants show elevated nucleocapsid IgG levels at 3 months, and higher neutralizing capacity up to 8 months post-infection. Increased spike-specific and nucleocapsid-specific CD4+T cells, PD-1, and TIM-3 expression on CD4+and CD8+T cells were observed at 3 and 8 months, but these differences do not persist at 24 months. Some LC participants had detectable IFN-Œ≥ and IFN-Œ≤, that was attributed to reinfection and antigen re-exposure. Single-cell RNA sequencing at the 24 month timepoint shows similar immune cell proportions and reconstitution of na√Øve T and B cell subsets in LC and MC. No significant differences in exhaustion scores or antigen-specific T cell clones are observed. These findings suggest resolution of immune activation in LC and return to comparable immune responses between LC and MC over time. Improvement in self-reported health-related quality of life at 24 months was also evident in the majority of LC (62%). PTX3, CRP levels and platelet count are associated with improvements in health-related quality of life.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">https://www.nature.com/articles/s41467-024-47720-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-09</strong> <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">
            ü§ñ COVID-19 Vaccination: Immunity After Two Types of Shots</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-108', this)">
            More</button></h4>
            <div id="toggle-108" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2-specific cellular and humoral immunity after bivalent BA.4/5 COVID-19-vaccination in previously infected and non-infected individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

We wanted to see how people's bodies would react to a special vaccine called bivalent BA.4/5-adapted mRNA-vaccine after they had previously been infected with SARS-CoV-2. We also looked at whether the vaccine could tell us if someone might get infected again in the future.

Our study found that people who already had COVID-19 before getting vaccinated made higher levels of antibodies and were better at stopping the virus from spreading. However, compared to people who never had COVID-19, they didn't have as much extra protection.

We also saw that the vaccine helped both groups make more antibodies against the original strain of COVID-19 than against newer versions like BA.1, BA.2, or BA.5. When it came to T cells and how their bodies responded to these different strains, though, we found no difference between people who had previously been infected and those who hadn't.

Notably, people who never had COVID-19 before getting vaccinated were more likely to get re-infected with SARS-CoV-2. This was because they had lower levels of antibodies that can stop the virus and fewer T cells that help fight it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Knowledge is limited as to how prior SARS-CoV-2 infection influences cellular and humoral immunity after booster-vaccination with bivalent BA.4/5-adapted mRNA-vaccines, and whether vaccine-induced immunity may indicate subsequent infection. In this observational study, individuals with prior infection (n=‚Äâ64) showed higher vaccine-induced anti-spike IgG-antibodies and neutralizing titers, but the relative increase was significantly higher in non-infected individuals (n=‚Äâ63). In general, both groups showed higher neutralizing activity towards the parental strain than towards Omicron-subvariants BA.1, BA.2 and BA.5. In contrast, CD4 or CD8 T cell levels towards spike from the parental strain and the Omicron-subvariants, and cytokine expression profiles were similar irrespective of prior infection. Breakthrough infections occurred more frequently among previously non-infected individuals, who had significantly lower vaccine-induced spike-specific neutralizing activity and CD4 T cell levels. In summary, we show that immunogenicity after BA.4/5-bivalent vaccination differs between individuals with and without prior infection. Moreover, our results may help to improve prediction of breakthrough infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">https://www.nature.com/articles/s41467-024-47429-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-02</strong> <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">
            ü§ñ Unmasking Hidden Allergies After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-109', this)">
            More</button></h4>
            <div id="toggle-109" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Incident allergic diseases in post-COVID-19 condition: multinational cohort studies from South Korea, Japan and the UK</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that a young person could understand:

We studied how people who had COVID-19 were more likely to develop other health problems, like asthma or allergies, afterwards. We looked at a big group of people in South Korea, Japan, and the UK over time and found that those who had COVID-19 were 20% more likely to get these other health problems than those who didn't have COVID-19. This risk was even higher for people with severe COVID-19. However, if they got vaccinated against COVID-19, it helped protect them from developing these other health problems. Our findings are similar in different countries and populations, which makes us more confident that they're true.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As mounting evidence suggests a higher incidence of adverse consequences, such as disruption of the immune system, among patients with a history of COVID-19, we aimed to investigate post-COVID-19 conditions on a comprehensive set of allergic diseases including asthma, allergic rhinitis, atopic dermatitis, and food allergy. We used nationwide claims-based cohorts in South Korea (K-CoV-N;n=‚Äâ836,164; main cohort) and Japan (JMDC;n=‚Äâ2,541,021; replication cohort A) and the UK Biobank cohort (UKB;n=‚Äâ325,843; replication cohort B) after 1:5 propensity score matching. Among the 836,164 individuals in the main cohort (mean age, 50.25 years [SD, 13.86]; 372,914 [44.6%] women), 147,824 were infected with SARS-CoV-2 during the follow-up period (2020‚àí2021). The risk of developing allergic diseases, beyond the first 30 days of diagnosis of COVID-19, significantly increased (HR, 1.20; 95% CI, 1.13‚àí1.27), notably in asthma (HR, 2.25; 95% CI, 1.80‚àí2.83) and allergic rhinitis (HR, 1.23; 95% CI, 1.15‚àí1.32). This risk gradually decreased over time, but it persisted throughout the follow-up period (‚â•6 months). In addition, the risk increased with increasing severity of COVID-19. Notably, COVID-19 vaccination of at least two doses had a protective effect against subsequent allergic diseases (HR, 0.81; 95% CI, 0.68‚àí0.96). Similar findings were reported in the replication cohorts A and B. Although the potential for misclassification of pre-existing allergic conditions as incident diseases remains a limitation, ethnic diversity for evidence of incident allergic diseases in post-COVID-19 condition has been validated by utilizing multinational and independent population-based cohorts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">https://www.nature.com/articles/s41467-024-47176-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-30</strong> <a href="https://www.nature.com/articles/s41541-024-00859-3" target="_blank">
            ü§ñ Breakthrough Vaccine Candidate for Cytomegalovirus Developed Using Innovative Technology</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-110', this)">
            More</button></h4>
            <div id="toggle-110" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Highly stable and immunogenic CMV T cell vaccine candidate developed using a synthetic MVA platform</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Human cytomegalovirus (CMV) is a common virus that can cause problems after organ transplants. A vaccine to prevent CMV infections in people who have received transplants has not been approved yet. Researchers have created a new version of this vaccine, called T10-F10, which is more stable and easier to produce on a large scale. They also tested this new vaccine in mice and found that it can stimulate the immune system just like the original vaccine. This means that T10-F10 could be a better option for transplant patients to prevent CMV infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Human cytomegalovirus (CMV) is the most common infectious cause of complications post-transplantation, while a CMV vaccine for transplant recipients has yet to be licensed. Triplex, a multiantigen Modified Vaccinia Ankara (MVA)-vectored CMV vaccine candidate based on the immunodominant antigens phosphoprotein 65 (pp65) and immediate-early 1 and 2 (IE1/2), is in an advanced stage of clinical development. However, its limited genetic and expression stability restricts its potential for large-scale production. Using a recently developed fully synthetic MVA (sMVA) platform, we developed a new generation Triplex vaccine candidate, T10-F10, with different sequence modifications for enhanced vaccine stability. T10-F10 demonstrated genetic and expression stability during extensive virus passaging. In addition, we show that T10-F10 confers comparable immunogenicity to the original Triplex vaccine to elicit antigen-specific T cell responses in HLA-transgenic mice. These results demonstrate improvements in translational vaccine properties of an sMVA-based CMV vaccine candidate designed as a therapeutic treatment for transplant recipients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00859-3" target="_blank">https://www.nature.com/articles/s41541-024-00859-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-28</strong> <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">
            ü§ñ Understanding Mpox Outbreaks in Nigeria: A Scientific Review</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-111', this)">
            More</button></h4>
            <div id="toggle-111" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamics of Mpox infection in Nigeria: a systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>TheMpox outbreak continues in many parts of West and Central Africa, with Nigeria having the most reported cases last year. To understand how to stop this disease from spreading, scientists looked at all the available information on Mpox in Nigeria over the past 50 years. They used a special method called PRISMA to analyze 92 articles that were relevant to their search. Out of these, they chose 23 articles for further study. Interestingly, most of theMpox cases in Nigeria came from the southern part of the country. The scientists found that Mpox was spreading gradually from south to north. They discovered three important reasons why Mpox is transmitted in Nigeria: poverty, lack of basic healthcare, and people getting infected through unsafe sex practices. These findings remind us that it's crucial to strengthen current efforts and work together across different sectors to understand howMpox spreads in Nigeria and find ways to stop it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The seasonal outbreaks of Mpox continue in most parts of West and Central Africa. In the past year, Nigeria had the highest number of reported cases. Here, we used the PRISMA guidelines to carry out a systematic review and meta-analysis of available evidence on Mpox in Nigeria to assess the prevalence, transmission pattern, diagnostic approach, and other associated factors useful for mitigating the transmission of the disease. All relevant observational studies in PubMed/MEDLINE, Embase, AJOL, Web of Science, Scopus and Google Scholar on Mpox in Nigeria were assessed within the last fifty years (1972 to 2022). In all, 92 relevant articles were retrieved, out of which 23 were included in the final qualitative analysis. Notably, most of the cases of Mpox in Nigeria were from the southern part of the country. Our findings showed a progressive spread from the southern to the northern region of the country. We identified the following factors as important in the transmission of Mpox in Nigeria; poverty, lack of basic healthcare facilities, and risk of exposure through unsafe sexual practices. Our findings reiterate the need to strengthen and expand existing efforts as well as establish robust multi-sectoral collaboration to understand the dynamics of Mpox Nigeria.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">https://www.nature.com/articles/s41598-024-58147-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">
            ü§ñ Double-Edged Shield: Fast-Tracking an mRNA Antibody Cocktail Against Orthopoxviruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-112', this)">
            More</button></h4>
            <div id="toggle-112" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rapid development of double-hit mRNA antibody cocktail against orthopoxviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The Orthopoxvirus genus, which includes viruses like smallpox and monkeypox, is still a major health threat around the world. Making therapeutic antibodies against these viruses is difficult and expensive because of the way they are made and engineered. A new technology that uses mRNA (a type of genetic material) to create antibodies quickly and easily has emerged as a powerful tool for fighting these viruses. In this study, researchers used a existing method to make four different combinations of mRNA that can produce antibodies against multiple Orthopoxviruses. They tested these in mice and found that they produced protective antibodies quickly after injection. Moreover, when the mice were challenged with a lethal smallpox virus, the antibody treatments showed significant protection from weight loss and death. One combination, called Mix2a, even protected against two different forms of the virus: one inside the cells and another outside them. This study demonstrates that mRNA-encoded antibodies can be used to combat Orthopoxviruses and may also work against other emerging viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>TheOrthopoxvirusgenus, especially variola virus (VARV), monkeypox virus (MPXV), remains a significant public health threat worldwide. The development of therapeutic antibodies against orthopoxviruses is largely hampered by the high cost of antibody engineering and manufacturing processes. mRNA-encoded antibodies have emerged as a powerful and universal platform for rapid antibody production. Herein, by using the established lipid nanoparticle (LNP)-encapsulated mRNA platform, we constructed four mRNA combinations that encode monoclonal antibodies with broad neutralization activities against orthopoxviruses. In vivo characterization demonstrated that a single intravenous injection of each LNP-encapsulated mRNA antibody in mice resulted in the rapid production of neutralizing antibodies. More importantly, mRNA antibody treatments showed significant protection from weight loss and mortality in the vaccinia virus (VACV) lethal challenge mouse model, and a unique mRNA antibody cocktail, Mix2a, exhibited superior in vivo protection by targeting both intracellular mature virus (IMV)-form and extracellular enveloped virus (EEV)-form viruses. In summary, our results demonstrate the proof-of-concept production of orthopoxvirus antibodies via the LNP-mRNA platform, highlighting the great potential of tailored mRNA antibody combinations as a universal strategy to combat orthopoxvirus as well as other emerging viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">https://www.nature.com/articles/s41392-024-01766-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">
            ü§ñ Understanding How Fever and Time Between Vaccines Affect First Four Doses of COVID-19 Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-113', this)">
            More</button></h4>
            <div id="toggle-113" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Factors impacting antibody kinetics, including fever and vaccination intervals, in SARS-CoV-2-na√Øve adults receiving the first four mRNA COVID-19 vaccine doses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Antibodies are like superheroes that help fight off viruses. This study looked at how well a specific vaccine worked in healthy adults who had never been infected with COVID-19. The researchers collected blood samples from participants after each dose of the vaccine and measured the levels of antibodies they produced. They found that several factors made a difference in how well the vaccine worked, including which type of vaccine was used, whether the person was male or female, their age, if they smoked or not, if they took certain medications, and how long it had been since their last dose. The results can help doctors figure out who might need extra doses to get good protection against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the antibody response following the initial four doses of mRNA vaccines (BNT162b2 or mRNA-1273) in SARS-CoV-2-na√Øve healthy adults and investigate factors influencing antibody titer increases, this prospective cohort study was conducted in Japan from March 2021. The study included participants who received either the 1st and 2nd doses (n‚Äâ=‚Äâ467), 3rd dose (n‚Äâ=‚Äâ157), or 4th dose (n‚Äâ=‚Äâ89). Blood samples were collected before and up to 6¬†months after each dose, and anti-receptor-binding domain antibody levels were measured. Multivariate analysis (usin multiple linear regression or linear mixed models) revealed several factors significantly associated with higher post-vaccination antibody levels, including mRNA-1273 vaccine (after the 1st and 2nd dose), male gender (after the 3rd and 4th doses), younger age (after the 1st and 2nd dose), non-smoking status (after the 2nd dose), non-use of immunosuppressive agents (after the 1st dose), higher pre-vaccination antibody titers (after the 2nd, 3rd, and 4th doses), and higher post-vaccination fever (after the 2nd and 4th doses). Furthermore, longer intervals since the last dose were significantly associated with higher antibody levels after the 3rd and 4th doses. These findings provide valuable insights for optimizing vaccination strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">https://www.nature.com/articles/s41598-024-57931-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-21</strong> <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">
            ü§ñ Long COVID Mental Health Risks</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-114', this)">
            More</button></h4>
            <div id="toggle-114" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term risk of psychiatric disorder and psychotropic prescription after SARS-CoV-2 infection among UK general population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had COVID-19 are more likely to develop mental health issues like anxiety, depression, and sleep problems compared to those who haven't had the virus. Researchers looked at data from over 700,000 people in the UK and found that one year after infection, those who had had COVID-19 were more likely to be prescribed medication for these types of issues. The risk was even higher for people who had been hospitalized due to COVID-19. However, the study also found that people who were fully vaccinated were less likely to develop mental health issues compared to those who weren't vaccinated or were only partially vaccinated. This suggests that vaccination may help protect against some long-term effects of COVID-19 on mental health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite evidence indicating increased risk of psychiatric issues among COVID-19 survivors, questions persist about long-term mental health outcomes and the protective effect of vaccination. Using UK Biobank data, three cohorts were constructed: SARS-CoV-2 infection (n=‚Äâ26,101), contemporary control with no evidence of infection (n=‚Äâ380,337) and historical control predating the pandemic (n=‚Äâ390,621). Compared with contemporary controls, infected participants had higher subsequent risks of incident mental health at 1‚Äâyear (hazard ratio (HR): 1.54, 95% CI 1.42‚Äì1.67;P=‚Äâ1.70‚Äâ√ó‚Äâ10‚àí24; difference in incidence rate: 27.36, 95% CI 21.16‚Äì34.10 per 1,000 person-years), including psychotic, mood, anxiety, alcohol use and sleep disorders, and prescriptions for antipsychotics, antidepressants, benzodiazepines, mood stabilizers and opioids. Risks were higher for hospitalized individuals (2.17, 1.70‚Äì2.78;P=‚Äâ5.80‚Äâ√ó‚Äâ10‚àí10) than those not hospitalized (1.41, 1.30‚Äì1.53;P=‚Äâ1.46‚Äâ√ó‚Äâ10‚àí16), and were reduced in fully vaccinated people (0.97, 0.80‚Äì1.19;P=‚Äâ0.799) compared with non-vaccinated or partially vaccinated individuals (1.64, 1.49‚Äì1.79;P=‚Äâ4.95‚Äâ√ó‚Äâ10‚àí26). Breakthrough infections showed similar risk of psychiatric diagnosis (0.91, 0.78‚Äì1.07;P=‚Äâ0.278) but increased prescription risk (1.42, 1.00‚Äì2.02;P=‚Äâ0.053) compared with uninfected controls. Early identification and treatment of psychiatric disorders in COVID-19 survivors, especially those severely affected or unvaccinated, should be a priority in the management of long COVID. With the accumulation of breakthrough infections in the post-pandemic era, the findings highlight the need for continued optimization of strategies to foster resilience and prevent escalation of subclinical mental health symptoms to severe disorders.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">https://www.nature.com/articles/s41562-024-01853-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-18</strong> <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">
            ü§ñ COVID-19 Brain Biomarkers Revealed</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-115', this)">
            More</button></h4>
            <div id="toggle-115" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neurofilament light chain and glial fibrillary acid protein levels are elevated in post-mild COVID-19 or asymptomatic SARS-CoV-2 cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
The COVID-19 pandemic had a big impact on many people's lives. Now that the pandemic is over for some individuals, scientists want to see how it affected their brains. To do this, they used two special proteins (sNfL and sGFAP) as markers of brain damage. These proteins are like flags that indicate when there might be problems with brain cells.

In a study involving 147 adults who had mild COVID-19 or were asymptomatic but still tested positive for the virus, scientists found higher levels of these proteins in their blood compared to people who never had COVID-19 (controls). Some of the adults with a history of COVID-19 also reported cognitive problems, which was associated with even higher levels of these proteins.

After 10 months, the levels of sNfL and sGFAP decreased but were still higher than those in the control group. This suggests that some brain damage might persist for an extended period after recovery from mild COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Given the huge impact of the COVID-19 pandemic, it appears of paramount importance to assess the cognitive effects on the population returning to work after COVID-19 resolution. Serum levels of neurofilament light chain (sNfL) and glial fibrillary acidic protein (sGFAP) represent promising biomarkers of neuro-axonal damage and astrocytic activation. In this cohort study, we explored the association between sNfL and sGFAP concentrations and cognitive performance in a group of 147 adult workers with a previous asymptomatic SARS-CoV-2 infection or mild COVID-19, one week and, in 49 of them, ten months after SARS-Cov2 negativization and compared them to a group of 82 age and BMI-matched healthy controls (HCs). sNfL and sGFAP concentrations were assessed using SimoaTM assay Neurology 2-Plex B Kit. COVID-19 patients were interviewed one-on-one by trained physicians and had to complete a list of questionnaires, including the Cognitive Failure Questionnaire (CFQ). At the first assessment (T0), sNfL and sGFAP levels were significantly higher in COVID-19 patients than in HCs (p‚Äâ<‚Äâ0.001 for both). The eleven COVID-19 patients with cognitive impairment had significantly higher levels of sNfL and sGFAP than the others (p‚Äâ=‚Äâ0.005 for both). At the subsequent follow-up (T1), sNfL and sGFAP levels showed a significant decrease (median sNfL 18.3¬†pg/mL; median sGFAP 77.2¬†pg/mL), although they were still higher than HCs (median sNfL 7.2¬†pg/mL, median sGFAP 63.5¬†pg/mL). Our results suggest an ongoing damage involving neurons and astrocytes after SARS-Cov2 negativization, which reduce after ten months even if still evident compared to HCs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">https://www.nature.com/articles/s41598-024-57093-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-05</strong> <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">
            ü§ñ Tracking SARS-CoV-2 in Air</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-116', this)">
            More</button></h4>
            <div id="toggle-116" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Frequency, kinetics and determinants of viable SARS-CoV-2 in bioaerosols from ambulatory COVID-19 patients infected with the Beta, Delta or Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that a young person can understand:

Imagine you're sitting in a crowded room with someone who has COVID-19. You might wonder if it's possible to get infected just by breathing the same air as them. Scientists have been trying to figure out if the tiny particles they breathe out (called aerosols) can carry live virus that can make others sick. They studied people with COVID-19 and found that a significant number of them were releasing small aerosol particles containing live virus for up to 9 days after getting symptoms. The scientists also discovered that these aerosol particles could be predicted using a simple blood test, which can help identify who is most likely to spread the virus. This study provides important information for developing strategies to reduce the spread of COVID-19 in crowded areas and suggests improving ventilation as a key measure to prevent transmission.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Airborne transmission of SARS-CoV-2 aerosol remains contentious. Importantly, whether cough or breath-generated bioaerosols can harbor viable and replicating virus remains largely unclarified. We performed size-fractionated aerosol sampling (Andersen cascade impactor) and evaluated viral culturability in human cell lines (infectiousness), viral genetics, and host immunity in ambulatory participants with COVID-19. Sixty-one percent (27/44) and 50% (22/44) of participants emitted variant-specific culture-positive aerosols <10Œºm and <5Œºm, respectively, for up to 9 days after symptom onset. Aerosol culturability is significantly associated with lower neutralizing antibody titers, and suppression of transcriptomic pathways related to innate immunity and the humoral response. A nasopharyngeal Ct <17 rules-in ~40% of aerosol culture-positives and identifies those who are probably highly infectious. A parsimonious three transcript blood-based biosignature is highly predictive of infectious aerosol generation (PPV‚Äâ>‚Äâ95%). There is considerable heterogeneity in potential infectiousness i.e., only 29% of participants were probably highly infectious (produced culture-positive aerosols <5Œºm at ~6 days after symptom onset). These data, which comprehensively confirm variant-specific culturable SARS-CoV-2 in aerosol, inform the targeting of transmission-related interventions and public health containment strategies emphasizing improved ventilation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">https://www.nature.com/articles/s41467-024-45400-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-27</strong> <a href="https://www.nature.com/articles/s41541-024-00842-y" target="_blank">
            ü§ñ MVA Vaccines Protect Against Deadly Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-117', this)">
            More</button></h4>
            <div id="toggle-117" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>MVA-based vaccines are protective against lethal eastern equine encephalitis virus aerosol challenge in cynomolgus macaques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Macaque monkeys were given two shots of a vaccine to protect them against a serious virus called eastern equine encephalitis (EEEV). The vaccine worked very well, and the monkeys developed strong antibodies that kept them safe until they were challenged with a small dose of the virus 84 days later. In fact, most of the monkeys who got the vaccine survived, while only a few of those who didn't get the vaccine made it through. Even more importantly, the vaccinated monkeys didn't show any signs of brain damage or other serious problems caused by the virus. These results suggest that this vaccine is effective in protecting against EEEV-induced disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>MVA-based monovalent eastern equine encephalitis virus (MVA-BN-EEEV) and multivalent western, eastern, and Venezuelan equine encephalitis virus (MVA-BN-WEV) vaccines were evaluated in the cynomolgus macaque aerosol model of EEEV infection. Macaques vaccinated with two doses of 5 √ó 108infectious units of the MVA-BN-EEEV or MVA-BN-WEV vaccine by the intramuscular route rapidly developed robust levels of neutralizing antibodies to EEEV that persisted at high levels until challenge at day 84 via small particle aerosol delivery with a target inhaled dose of 107PFU of EEEV FL93-939. Robust protection was observed, with 7/8 animals receiving MVA-BN-EEEV and 100% (8/8) animals receiving MVA-BN-WEV surviving while only 2/8 mock vaccinated controls survived lethal challenge. Complete protection from viremia was afforded by both vaccines, with near complete protection from vRNA loads in tissues and any pathologic evidence of central nervous system damage. Overall, the results indicate both vaccines are effective in eliciting an immune response that is consistent with protection from aerosolized EEEV-induced disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00842-y" target="_blank">https://www.nature.com/articles/s41541-024-00842-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-26</strong> <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">
            ü§ñ COVID-19 Vaccination and Long-Term Health Risks After Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-118', this)">
            More</button></h4>
            <div id="toggle-118" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Persistence in risk and effect of COVID-19 vaccination on long-term health consequences after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The long-term risks of getting very sick from COVID-19 are still not well understood, and it's unclear how much protection vaccination provides against these risks. In this study, researchers looked at a large group of people in Hong Kong who had gotten COVID-19 and compared them to a similar-sized group of people who hadn't gotten the virus. They found that over time, the risk of dying from any cause decreased among people who had gotten COVID-19 compared to those who hadn't. People who were fully vaccinated or had received booster shots seemed to be at lower risk for serious health problems, such as heart disease, and death 30-90 days after getting infected. However, even up to a year later, people who weren't fully vaccinated or had only gotten partial vaccination continued to have a higher risk of developing complications from COVID-19. This study provided important real-world evidence that supports the effectiveness of COVID-19 vaccines in reducing long-term risks associated with getting infected.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The persisting risk of long-term health consequences of SARS-CoV-2 infection and the protection against such risk conferred by COVID-19 vaccination remains unclear. Here we conducted a retrospective territory-wide cohort study on 1,175,277 patients with SARS-CoV-2 infection stratified by their vaccination status and non-infected controls to evaluate the risk of clinical sequelae, cardiovascular and all-cause mortality using a territory-wide public healthcare database with population-based vaccination records in Hong Kong. A progressive reduction in risk of all-cause mortality was observed over one year between patients with SARS-CoV-2 infection and controls. Patients with complete vaccination or have received booster dose incurred a lower risk of health consequences including major cardiovascular diseases, and all-cause mortality than unvaccinated or patients with incomplete vaccination 30-90 days after infection. Completely vaccinated and patients with booster dose of vaccines did not incur significant higher risk of health consequences from 271 and 91 days of infection onwards, respectively, whilst un-vaccinated and incompletely vaccinated patients continued to incur a greater risk of clinical sequelae for up to a year following SARS-CoV-2 infection. This study provided real-world evidence supporting the effectiveness of COVID-19 vaccines in reducing the risk of long-term health consequences of SARS-CoV-2 infection and its persistence following infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">https://www.nature.com/articles/s41467-024-45953-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-20</strong> <a href="https://www.nature.com/articles/s41598-024-54492-0" target="_blank">
            ü§ñ Uncovering Hidden Patterns in Time-Fractional Nonlinear Epidemic Models</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-119', this)">
            More</button></h4>
            <div id="toggle-119" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel technique to study the solutions of time fractional nonlinear smoking epidemic model</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The main goal of this research was to use a new mathematical technique called natural transform decomposition to solve a special kind of epidemic model that takes into account fractional derivatives in different senses. This model was designed to understand how smoking habits change among adults and its impact on public health in a community. A sensitivity analysis was done to compare two states: when the disease (in this case, smoking) persists in a population or dies out. The model consisted of five groups representing different types of smokers. We used our new technique to solve the model and compared the results with other existing numerical methods. Our findings show that our method is efficient and effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The primary goal of the current work is to use a novel technique known as the natural transform decomposition method to approximate an analytical solution for the fractional smoking epidemic model. In the proposed method, fractional derivatives are considered in the Caputo, Caputo‚ÄìFabrizio, and Atangana‚ÄìBaleanu‚ÄìCaputo senses. An epidemic model is proposed to explain the dynamics of drug use among adults. Smoking is a serious issue everywhere in the world. Notwithstanding the overwhelming evidence against smoking, it is nonetheless a harmful habit that is widespread and accepted in society. The considered nonlinear mathematical model has been successfully used to explain how smoking has changed among people and its effects on public health in a community. The two states of being endemic and disease-free, which are when the disease dies out or persists in a population, have been compared using sensitivity analysis. The proposed technique has been used to solve the model, which consists of five compartmental agents representing various smokers identified, such as potential smokersV, occasional smokersG, smokersT, temporarily quittersO, and permanently quittersW. The results of the suggested method are contrasted with those of existing numerical methods. Finally, some numerical findings that illustrate the tables and figures are shown. The outcomes show that the proposed method is efficient and effective.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-54492-0" target="_blank">https://www.nature.com/articles/s41598-024-54492-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-14</strong> <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">
            ü§ñ Coronavirus on Transplanted Eyes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-120', this)">
            More</button></h4>
            <div id="toggle-120" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection status in corneal preservation solution and COVID-19 prevalence after corneal transplantation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic caused many people to worry about getting infected from donated organs used in transplants. To figure out if this was a real concern, researchers tested corneas that were donated during the pandemic and found that none of them had the virus. They also looked at the health of patients who received these transplanted corneas and did not find any cases of COVID-19. This suggests that as long as the donor has tested negative for the virus, it's unlikely to be transmitted through a corneal transplant. Therefore, it's essential to test donors for SARS-CoV-2 before donating organs, especially for high-risk patients like those who receive transplanted corneas and are on prolonged steroid treatment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The potential risks associated with organs from COVID-19-infected donors were unclear. To determine the SARS-CoV-2 infection status of corneas transplanted during the COVID-19 pandemic, we performed a polymerase chain reaction (PCR) using the corneal preservation solution that was used for corneal transplantation. We also examined the postoperative health status of the recipients. This study included 144 transplants in 143 eyes. Ninety-nine eyes of imported corneas and 10 of the 14 corneas donated in the prefecture were PCR tested at our hospital, and all were SARS-CoV-2 negative. All corneal transplants were performed after confirming their SARS-CoV-2 negativity by a PCR using a corneal preservation solution at our hospital or a nasopharyngeal swab at a previous facility. Despite postoperative steroid administration, no patient developed COVID-19 infection until discharge. Hence, if the donor's nasopharyngeal swab test is SARS-CoV-2 negative, COVID-19 infection in the recipient due to corneal transplantation may be prevented. Since corneal transplant recipients are susceptible to infection due to prolonged steroid administration and are at high risk for severe diseases if infection occurs, SARS-CoV-2 detection testing using nasopharyngeal swabs in donors should be performed.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">https://www.nature.com/articles/s41598-024-53863-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-05</strong> <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">
            ü§ñ Protecting Against Post-COVID Gut Issues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-121', this)">
            More</button></h4>
            <div id="toggle-121" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 vaccination may mitigate dysregulation of IL-1/IL-18 and gastrointestinal symptoms of the post-COVID-19 condition</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic was slowed down by the creation of safe and effective vaccines. While these vaccines also seem to help prevent long-term problems after recovering from COVID-19, it's still unclear how well they work for everyone. Some people who got vaccinated after having COVID-19 reported fewer symptoms, but more research is needed to confirm this. In a study, scientists looked at the levels of 25 different cytokines (molecules that help fight infections) and SARS-CoV-2 antibodies in 540 people with or without long-term COVID-19 problems. They also checked these levels in 20 people who had just gotten vaccinated but didn't have any prior COVID-19 infection. The results showed that people with long-term COVID-19 problems who got vaccinated had lower levels of IL-1Œ≤ and IL-18, which correlated with fewer gastrointestinal symptoms. This suggests that vaccination might be beneficial for a subgroup of people with long-term COVID-19 problems who experience gastrointestinal symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid development of safe and effective vaccines helped to prevent severe disease courses after SARS-CoV-2 infection and to mitigate the progression of the COVID-19 pandemic. While there is evidence that vaccination may reduce the risk of developing post-COVID-19 conditions (PCC), this effect may depend on the viral variant. Therapeutic effects of post-infection vaccination have been discussed but the data for individuals with PCC remains inconclusive. In addition, extremely rare side effects after SARS-CoV-2 vaccination may resemble the heterogeneous PCC phenotype. Here, we analyze the plasma levels of 25 cytokines and SARS-CoV-2 directed antibodies in 540 individuals with or without PCC relative to one or two mRNA-based COVID-19 vaccinations as well as in 20 uninfected individuals one month after their initial mRNA-based COVID-19 vaccination. While none of the SARS-CoV-2 na√Øve individuals reported any persisting sequelae or exhibited PCC-like dysregulation of plasma cytokines, we detected lower levels of IL-1Œ≤ and IL-18 in patients with ongoing PCC who received one or two vaccinations at a median of six months after infection as compared to unvaccinated PCC patients. This reduction correlated with less frequent reporting of persisting gastrointestinal symptoms. These data suggest that post-infection vaccination in patients with PCC might be beneficial in a subgroup of individuals displaying gastrointestinal symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">https://www.nature.com/articles/s41541-024-00815-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-24</strong> <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">
            ü§ñ Respiratory Distress in Unvaccinated Newborns Exposed to COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-122', this)">
            More</button></h4>
            <div id="toggle-122" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Respiratory distress in SARS-CoV-2 exposed uninfected neonates followed in the COVID Outcomes in Mother-Infant Pairs (COMP) Study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###

Some newborn babies who were exposed to SARS-CoV-2 in the womb have had trouble breathing, even though they weren't infected themselves. Research suggests that when a mom has COVID-19 during pregnancy, it can cause an inflammatory response in her baby's airways. This study looked at how often newborns who were exposed to SARS-CoV-2 in the womb developed respiratory distress, and whether their moms got vaccinated against COVID-19 before getting infected. The researchers found that a lot of babies (17%) had trouble breathing after being exposed to SARS-CoV-2 in the womb. But when their moms got vaccinated against COVID-19 before getting infected, the risk of these babies having trouble breathing went down. The study also looked at what might be happening in the bodies of these babies and found that they were having a strong inflammatory response that was affecting their cilia (small hair-like structures) and causing them to produce more IgE, which is an antibody that can trigger allergic reactions. Overall, the research suggests that getting vaccinated against COVID-19 during pregnancy can help protect newborns from respiratory distress.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory distress (RD) has been reported in SARS-CoV-2 exposed uninfected (SEU) term neonates. Prior studies suggest that prenatal exposure to Coronavirus Disease 19 (COVID-19) may activate an inflammatory cascade in the newborn airway. In this study, we examine the relationship between maternal COVID-19 vaccination and neonatal RD using a longitudinal cohort of mother-infant pairs in Los Angeles, CA. Two-hundred and twenty-one mothers with laboratory confirmed SARS-CoV-2 during pregnancy and 227 exposed fetuses are enrolled in our study. Maternal disease severity and neonatal RD variables were defined based on current accepted clinical criteria. To explore the multifactorial associations between maternal COVID-19 parameters and infant RD, we utilize a multivariable logistic regression model and a proteomic sub-analysis to propose a pathway for the development of RD followingin uteroexposure to SARS-CoV-2. Unusually high rates of RD are observed in SEU infants (17%). The odds ratio of RD is 3.06 (95% CI:1.08-10.21) in term neonates born to unvaccinated individuals versus those born to individuals vaccinated prior to maternal infection. Proteomic analysis reveals a robust inflammatory response associated with ciliary dysregulation and enhanced IgE production among SEU neonates with RD. Maternal vaccination against COVID-19 reduces the frequency of neonatal RD.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">https://www.nature.com/articles/s41467-023-44549-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-11</strong> <a href="https://www.nature.com/articles/s41590-023-01724-6" target="_blank">
            ü§ñ COVID Symptoms Last Longer Than Expected Due to Immune System Disorder</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-123', this)">
            More</button></h4>
            <div id="toggle-123" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID manifests with T cell dysregulation, inflammation and an uncoordinated adaptive immune response to SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Long COVID happens when someone has had a severe case of coronavirus for at least 8 months, but scientists still don't fully understand why it occurs. To learn more about Long COVID, researchers studied the blood of people who had clear Long COVID and those who didn't have any symptoms 8 months after being infected. They found that people with Long COVID had inflammation and problems with their immune system. This was shown by changes in how different types of T cells were distributed, which meant their immune systems were still responding to the virus. People with Long COVID also had more CD4+T cells that were ready to go to inflamed areas and exhausted SARS-CoV-2-specific CD8+T cells. They had higher levels of antibodies against the coronavirus and a miscommunication between their T and B cell responses. This mix-up might cause inflammation, symptoms, and problems with the immune system in people with Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long COVID (LC) occurs after at least 10% of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infections, yet its etiology remains poorly understood. We used ‚Äòomic‚Äù assays and serology to deeply characterize the global and SARS-CoV-2-specific immunity in the blood of individuals with clear LC and non-LC clinical trajectories, 8‚Äâmonths postinfection. We found that LC individuals exhibited systemic inflammation and immune dysregulation. This was evidenced by global differences in T cell subset distribution implying ongoing immune responses, as well as by sex-specific perturbations in cytolytic subsets. LC individuals displayed increased frequencies of CD4+T cells poised to migrate to inflamed tissues and exhausted SARS-CoV-2-specific CD8+T cells, higher levels of SARS-CoV-2 antibodies and a mis-coordination between their SARS-CoV-2-specific T and B cell responses. Our analysis suggested an improper crosstalk between the cellular and humoral adaptive immunity in LC, which can lead to immune dysregulation, inflammation and clinical symptoms associated with this debilitating condition.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-023-01724-6" target="_blank">https://www.nature.com/articles/s41590-023-01724-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-08</strong> <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">
            ü§ñ Unpacking the Genetic Risk Factors for SARS-CoV-2 Infection and Severe COVID-19 Symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-124', this)">
            More</button></h4>
            <div id="toggle-124" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Estimating the heritability of SARS-CoV-2 susceptibility and COVID-19 severity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten abstract that's easier for young people to understand:

The SARS-CoV-2 virus has spread to over 340 million people worldwide, making it a priority to find effective treatments. While scientists can look at the virus's genetic code to identify potential targets for new medicines, they also want to know why some people are more likely to get sick or have severe symptoms. So far, studies have shown that genetics and shared environments (like where you live and who you're around) explain only a small part of how susceptible someone is to getting the virus. We used data from families and hospital records to estimate how much genetics and shared environments contribute to SARS-CoV-2 susceptibility and COVID-19 severity. Our results suggest that genetics play a bigger role in making people more or less susceptible to the virus, but this can change over time due to factors like changes in the environment and the introduction of vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 has infected over 340 million people, prompting therapeutic research. While genetic studies can highlight potential drug targets, understanding the heritability of SARS-CoV-2 susceptibility and COVID-19 severity can contextualize their results. To date, loci from meta-analyses explain 1.2% and 5.8%¬†of variation in susceptibility and severity¬†respectively. Here we estimate the importance of shared environment and additive genetic variation to SARS-CoV-2 susceptibility and COVID-19 severity using pedigree data, PCR results, and hospitalization information. The relative importance of genetics and shared environment for susceptibility shifted during the study, with heritability ranging from 33% (95% CI: 20%-46%) to 70% (95% CI: 63%-74%). Heritability was greater for days hospitalized with COVID-19 (41%, 95% CI: 33%-57%) compared to shared environment (33%, 95% CI: 24%-38%). While our estimates suggest these genetic architectures are not fully understood, the shift in susceptibility estimates highlights the challenge of estimation during a pandemic, given environmental fluctuations and vaccine introduction.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">https://www.nature.com/articles/s41467-023-44250-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-03</strong> <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">
            ü§ñ COVID-19 Symptoms That Linger</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-125', this)">
            More</button></h4>
            <div id="toggle-125" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Features of acute COVID-19 associated with post-acute sequelae of SARS-CoV-2 phenotypes: results from the IMPACC study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID-19 symptoms are a major public health concern. We studied 590 people who had been hospitalized for COVID-19 and followed them for one year after they were discharged. Our analysis showed that there are different types of post-COVID-19 symptoms, including minimal, physical, mental/cognitive, and a combination of all three. Women were more likely to experience these symptoms, especially if they had certain underlying health conditions. We also found that people who experienced severe respiratory problems during their initial COVID-19 illness were more likely to have physical or combined symptoms later on. Additionally, we discovered that people with mental/cognitive predominant symptoms had higher levels of a protein called FGF21 in their blood. Understanding the connection between post-COVID-19 symptoms and the body's initial response to the virus could help us develop better treatments and prevention strategies for these long-term effects.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sequelae of SARS-CoV-2 (PASC) is a significant public health concern. We describe Patient Reported Outcomes (PROs) on 590 participants prospectively assessed from hospital admission for COVID-19 through one year after discharge. Modeling identified 4 PRO clusters based on reported deficits (minimal, physical, mental/cognitive, and multidomain), supporting heterogenous clinical presentations in PASC, with sub-phenotypes associated with female sex and distinctive comorbidities. During the acute phase of disease, a higher respiratory SARS-CoV-2 viral burden and lower Receptor Binding Domain and Spike antibody titers were associated with both the physical predominant and the multidomain deficit clusters. A lower frequency of circulating B lymphocytes by mass cytometry (CyTOF) was observed in the multidomain deficit cluster. Circulating fibroblast growth factor 21 (FGF21) was significantly elevated in the mental/cognitive predominant and the multidomain clusters. Future efforts to link PASC to acute anti-viral host responses may help to better target treatment and prevention of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">https://www.nature.com/articles/s41467-023-44090-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-02</strong> <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">
            ü§ñ Understanding Viral Infection at the Molecular Level</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-126', this)">
            More</button></h4>
            <div id="toggle-126" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Integrated multi-omics analyses identify anti-viral host factors and pathways controlling SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many people have gotten sick from a virus called SARS-CoV-2, but scientists still don't know as much as they would like about how to fight it off. Most past studies have focused on things that can make the virus stronger, rather than weaker. To help fill this knowledge gap, researchers used a powerful tool called CRISPR to "turn off" different parts of human cells and see which ones were important for keeping the virus at bay. They also analyzed lots of other data from multiple sources, including genetic information from people who have been sick with COVID-19.

What they found was surprising: many things that were thought to be unimportant are actually key players in fighting off the virus! This includes some complex systems that help cells move things around and make new proteins. They also discovered a new player called KLF5, which is involved in making special fats that cells need. Some people who have gotten really sick with COVID-19 have had genetic variations that affect how well their bodies can fight off the virus.

The researchers were able to confirm that three of these newly-discovered players (DAZAP2/VTA1/KLF5) are indeed important for fighting off the virus, and that they work together in some way. They also found that when these players are not working properly, it's more likely that people will get very sick with COVID-19. Overall, this study helps us better understand how to fight off SARS-CoV-2 and might even lead to new treatments or therapies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Host anti-viral factors are essential for controlling SARS-CoV-2 infection but remain largely unknown due to the biases of previous large-scale studies toward pro-viral host factors. To fill in this knowledge gap, we perform a genome-wide CRISPR dropout screen and integrate analyses of the multi-omics data of the CRISPR screen, genome-wide association studies, single-cell RNA-Seq, and host-virus proteins or protein/RNA interactome. This study uncovers many host factors that are currently underappreciated, including the components of V-ATPases, ESCRT, and N-glycosylation pathways that modulate viral entry and/or replication. The cohesin complex is also identified as an anti-viral pathway, suggesting an important role of three-dimensional chromatin organization in mediating host-viral interaction. Furthermore, we discover another anti-viral regulator KLF5, a transcriptional factor involved in sphingolipid metabolism, which is up-regulated, and harbors genetic variations linked to COVID-19 patients with severe symptoms. Anti-viral effects of three identified candidates (DAZAP2/VTA1/KLF5) are confirmed individually. Molecular characterization of DAZAP2/VTA1/KLF5-knockout cells highlights the involvement of genes related to the coagulation system in determining the severity of COVID-19. Together, our results provide further resources for understanding the host anti-viral network during SARS-CoV-2 infection and may help develop new countermeasure strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">https://www.nature.com/articles/s41467-023-44175-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">
            ü§ñ SARS-CoV-2 Variants in Kids: What You Need to Know</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-127', this)">
            More</button></h4>
            <div id="toggle-127" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralization of SARS-CoV-2 Omicron BQ.1, BQ.1.1 and XBB.1 variants following SARS-CoV-2 infection or vaccination in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Emergence of highly transmissible Omicron subvariants led to increased COVID-19 infections and illnesses in children. However, not much is known about how well vaccinated or previously infected children can fight these new subvariants. Here, we studied the ability of 151 children of different ages who were hospitalized with severe COVID-19 or a rare multisystem inflammatory syndrome (MIS-C) to neutralize SARS-CoV-2 variants compared to 62 vaccinated children. We found that older children had better neutralizing antibodies after vaccination or infection than younger children. Vaccinated adolescents showed higher levels of neutralizing antibodies against one specific variant, but unfortunately, their antibodies didn't work well against newer subvariants like BQ.1, BQ.1.1 and XBB.1. In contrast, vaccinated young children under 5 years old had more cross-protective antibodies against these new variants than older children. Our study shows that in children, neutralizing activity is low against these new variants whether they've been infected or vaccinated. This suggests that we need better SARS-CoV-2 vaccines to induce longer-lasting and more cross-reactive neutralizing antibodies to effectively protect children from emerging variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Emergence of highly transmissible Omicron subvariants led to increased SARS-CoV-2 infection and disease in children. However, minimal knowledge exists regarding the neutralization capacity against circulating Omicron BA.4/BA.5, BA.2.75, BQ.1, BQ.1.1 and XBB.1 subvariants following SARS-CoV-2 vaccination in children versus during acute or convalescent COVID-19, or versus multisystem inflammatory syndrome (MIS-C). Here, we evaluate virus-neutralizing capacity against SARS-CoV-2 variants in 151 age-stratified children (‚Äâ<5, 5‚Äì11, 12‚Äì21 years old) hospitalized with acute severe COVID-19 or MIS-C or convalescent mild (outpatient) infection compared with 62 age-stratified vaccinated children. An age-associated effect on neutralizing antibodies is observed against SARS-CoV-2 following acute COVID-19 or vaccination. The primary series BNT162b2 mRNA vaccinated adolescents show higher vaccine-homologous WA-1 neutralizing titers compared with <12 years vaccinated children. Post-infection antibodies did not neutralize BQ.1, BQ.1.1 and XBB.1 subvariants. In contrast, monovalent mRNA vaccination induces more cross-neutralizing antibodies in young children <5 years against BQ.1, BQ.1.1 and XBB.1 variants compared with ‚â•5 years old children. Our study demonstrates that in children, infection and monovalent vaccination-induced neutralization activity is low against BQ.1, BQ.1.1 and XBB.1 variants. These findings suggest a need for improved SARS-CoV-2 vaccines to induce durable, more cross-reactive neutralizing antibodies to provide effective protection against emerging variants in children.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">https://www.nature.com/articles/s41467-023-43152-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">
            ü§ñ SARS-CoV-2 Can Spread Through the Air in Minutes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-128', this)">
            More</button></h4>
            <div id="toggle-128" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Infectivity of exhaled SARS-CoV-2 aerosols is sufficient to transmit covid-19 within minutes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Exhaled viruses from people with COVID-19 were spreading the disease very quickly and widely. However, scientists didn't have enough information about how infectious these viruses are when they're released in the air. We did some experiments to find out how many viruses are being emitted by people who just started showing symptoms of COVID-19. We collected air samples from six different individuals and were able to grow five of them in a lab dish, which allowed us to count the number of viruses. We found that when these people sang, they released a lot of viruses into the air - in one case, up to 127 virus particles per second! If someone with this amount of virus entered a room, we calculated that another person could breathe in enough virus to get infected within just a few minutes, even if the room has normal ventilation. This shows that when one person with COVID-19 sings or breathes heavily, they can easily infect others around them in just a short time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Exhaled SARS-CoV-2-containing aerosols contributed significantly to the rapid and vast spread of covid-19. However, quantitative experimental data on the infectivity of such aerosols is missing. Here, we quantified emission rates of infectious viruses in exhaled aerosol from individuals within their first days after symptom onset from covid-19. Six aerosol samples from three individuals were culturable, of which five were successfully quantified using TCID50. The source strength of the three individuals was highest during singing, when they exhaled 4, 36, or 127 TCID50/s, respectively. Calculations with an indoor air transmission model showed that if an infected individual with this emission rate entered a room, a susceptible person would inhale an infectious dose within 6 to 37¬†min in a room with normal ventilation. Thus, our data show that exhaled aerosols from a single person can transmit covid-19 to others within minutes at normal indoor conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">https://www.nature.com/articles/s41598-023-47829-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-11-17</strong> <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">
            ü§ñ Living with COVID Long-Term: A Global Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-129', this)">
            More</button></h4>
            <div id="toggle-129" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The burden of post-acute COVID-19 symptoms in a multinational network cohort analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that explains the article in simpler terms:

We looked at data from over 3 million people who got tested for COVID-19 in Spain and the UK. We wanted to see how many people had symptoms after they were no longer sick with COVID-19, and which symptoms were most common. Our study found that fewer people were having lingering symptoms as time went on, but some symptoms like trouble smelling or tasting things were still more likely if you'd had COVID-19 than if you hadn't been infected at all. We also saw that when people got re-infected with COVID-19, they were more likely to have ongoing symptoms than after their first infection. This suggests that long COVID is a real issue and we need to learn more about it so we can find ways to make it less of a problem.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Persistent symptoms following the acute phase of COVID-19 present a major burden to both the affected and the wider community. We conducted a cohort study including over 856,840 first COVID-19 cases, 72,422 re-infections and more than 3.1 million first negative-test controls from primary care electronic health records from Spain and the UK (Sept 2020 to Jan 2022 (UK)/March 2022 (Spain)). We characterised post-acute COVID-19 symptoms and identified key symptoms associated with persistent disease. We estimated incidence rates of persisting symptoms in the general population and among COVID-19 patients over time. Subsequently, we investigated which WHO-listed symptoms were particularly differential by comparing their frequency in COVID-19 cases vs. matched test-negative controls. Lastly, we compared persistent symptoms after first infections vs. reinfections.Our study shows that the proportion of COVID-19 cases affected by persistent post-acute COVID-19 symptoms declined over the study period. Risk for altered smell/taste was consistently higher in patients with COVID-19 vs test-negative controls. Persistent symptoms were more common after reinfection than following a first infection. More research is needed into the definition of long COVID, and the effect of interventions to minimise the risk and impact of persistent symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">https://www.nature.com/articles/s41467-023-42726-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-11-14</strong> <a href="https://www.nature.com/articles/s41598-023-47218-1" target="_blank">
            ü§ñ Living with COVID: Understanding Health Impacts</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-130', this)">
            More</button></h4>
            <div id="toggle-130" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Functionality, physical activity, fatigue and quality of life in patients with acute COVID-19 and Long COVID infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A lot of people who got very sick from COVID-19 still feel unwell for a long time after they get better. They might have trouble doing daily things, feel tired all the time, and that makes their life not as good as it could be. This study looked at how people with this long-term sickness compare to others who got over COVID-19 quickly. It found that people with the long-term sickness had trouble with daily activities, didn't exercise much, were really tired, and that made their life worse. The study also showed that if you can do everyday things easily, it helps make your life better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A prominent feature of COVID-19, both in the short and long term, is the reduction in quality of life (QoL) due to low functionality scores and the presence of fatigue, which can hinder daily activities. The main objective of this study is to compare the functional status, level of physical activity, fatigue, and QoL of patients with Long COVID to other COVID-19 patients who did not develop persistent illness, and to determine whether there is a relationship between these variables and QoL. A cross-sectional study was conducted with 170 participants who had been infected with COVID-19 or had developed Long COVID. The main variables studied were functionality, physical activity, QoL and fatigue, measured using the PostCOVID-19 Functional Status Scale (PCFS), International Physical Activity Questionnaire (IPAQ), Short Form 12 (SF-12), and Fatigue Severity Scale (FSS). The main findings show a significant relationship (p<‚Äâ0.001) between reduced functionality, lower physical activity levels, increased fatigue severity, and poorer QoL in Long COVID patients. Furthermore, these variables are also related to worse QoL, but only functional status predicts it. In conclusion, our results have shown highly significant correlations between the group with COVID-19 and Long COVID regarding functional status, level of physical activity, QoL, and fatigue.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-47218-1" target="_blank">https://www.nature.com/articles/s41598-023-47218-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-07</strong> <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">
            ü§ñ A Study on COVID-19 and Post-Acute Sick Leave in Denmark</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-131', this)">
            More</button></h4>
            <div id="toggle-131" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A hybrid register and questionnaire study of Covid-19 and post-acute sick leave in Denmark</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT 
People who got sick with COVID-19 often took a long time to recover and needed to stay home from work or school. We wanted to understand how common this was, so we looked at data from over 88,000 people who either had COVID-19 (37,482) or were tested for it but didn't have the virus (51,336). Our study showed that about 33 out of every 1,000 people took more than a month to get back on their feet after getting sick. This was especially true for women, and older adults between 50-65 years old. Also, if someone already had health conditions like obesity, lung diseases, or fibromyalgia, they were more likely to take a long time recovering. These findings could help doctors develop better treatments and tests for people who still aren't feeling well after having COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sick leave is an underexplored indicator of the societal burden of SARS-CoV-2. Here, ¬†we report findings about self-reported sick leave and risk factors thereof from a hybrid survey and register study, which include 37,482 RT-PCR confirmed SARS-CoV-2 cases and 51,336 test-negative controls who were tested during the index- and alpha-dominant waves. We observe that an additional 33 individuals per 1000 took substantial sick leave following acute infection compared to persons with no known history of infection, where substantial sick leave is defined as >1 month of sick leave within the period 1‚Äì9 months after the RT-PCR test date. Being female, 50‚Äì65 years, or having certain pre-existing health conditions such as obesity, chronic lung diseases, and fibromyalgia each increase risk for taking substantial sick leave. Altogether, these results may help motivate improved diagnostic and treatment options for persons living with post-Covid conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">https://www.nature.com/articles/s41467-023-42048-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-06</strong> <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">
            ü§ñ Eye Health at Risk: Understanding Blindness from Blood Vessels in the Eye During and After a Viral Outbreak</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-132', this)">
            More</button></h4>
            <div id="toggle-132" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Retinal vascular occlusion risks during the COVID-19 pandemic and after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The new coronavirus, also known as COVID-19, has been linked to problems with blood vessels in the body, including those in the eye. But scientists haven't yet understood how often people get blockages in these blood vessels due to COVID-19. This study looked at whether there were more or less cases of these blockages before and during the COVID-19 pandemic. They compared the number of cases from 2018-2019 to 2020-2021, as well as in people who tested positive for COVID-19 between 2020 and 2021. They also looked at how many cases there were in people with COVID-19 before and after their diagnosis. The results showed that the number of blockages in blood vessels in the eye didn't change significantly during the pandemic. In fact, one type of blockage decreased slightly, while another increased. The study found that having COVID-19 did not make it more likely to get these blockages.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) has been reported to affect vascular networks including the eye. However, evidence on the causal relationship between COVID-19 infection and retinal vascular occlusions remains limited. This study aimed to determine the change in retinal vascular occlusion incidence during COVID-19 era and whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection induces retinal vascular occlusion. Retinal vein occlusion (RVO) and retinal artery occlusion (RAO) incidences during 2018‚Äì2019 and 2020‚ÄìJuly 2021 were compared, those in confirmed and suspected COVID-19 patients diagnosed from 2020 to January 2021 were calculated, and those in COVID-19 patients during 180¬†days prior and 180¬†days after diagnosis were assessed. Additionally, the standardized incidence ratio of RVOs in COVID-19 patients was analyzed. Incidence rates per 100,000 people/year of RVO during 2018‚Äì2019 and 2020‚Äì2021 was 102.0 and 98.8, respectively. RAO incidence rates during 2018‚Äì2019 and 2020‚Äì2021 were 11.7 and 12.0, respectively. In both confirmed and suspected COVID-19 patients, the incidence of RVO and RAO did not change significantly from 180¬†days before to after diagnosis in the adjusted model. RVO incidence slightly decreased while RAO incidence increased during the COVID-19 pandemic. SARS-CoV-2 infection did not significantly increase RVO or RAO incidence.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">https://www.nature.com/articles/s41598-023-44199-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-02</strong> <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">
            ü§ñ Testing for Monkeypox in High-Risk Groups in Spain</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-133', this)">
            More</button></h4>
            <div id="toggle-133" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Self-sampling monkeypox virus testing in high-risk populations, asymptomatic or with unrecognized Mpox, in Spain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new virus called monkeypox has been spreading around the world, mainly affecting men who have sex with other men. We wanted to see how common this virus is in people who are at higher risk of getting it, like gay and bisexual men, trans women, and non-binary people who may not even know they have it or aren't showing symptoms. We used a special method where participants took their own swabs from their anal and pharyngeal areas (throat) to test for the virus. Out of 113 people in our study, 7 had tested positive for monkeypox, which is about 6% of the total. Five of these people didn't even show symptoms that would make us think they had the virus. Our results suggest that some people with monkeypox might be able to spread it without showing any symptoms at all. This means that if we only test people who say they have symptoms, we might miss cases where the virus is still present and could potentially cause more outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recent monkeypox virus (MPXV) outbreak was of global concern and has mainly affected gay, bisexual and other men who have sex with men (GBMSM). Here we assess prevalence of MPXV in high-risk populations of GBMSM, trans women (TW) and non-binary people without symptoms or with unrecognized monkeypox (Mpox) symptoms, using a self-sampling strategy. Anal and pharyngeal swabs are tested by MPXV real-time PCR and positive samples are tested for cytopathic effect (CPE) in cell culture. 113 individuals participated in the study, 89 (78.76%) were cis men, 17 (15.04%) were TW. The median age was 35.0 years (IQR: 30.0‚Äì43.0), 96 (85.02%) individuals were gay or bisexual and 72 (63.72%) were migrants. Seven participants were MPXV positive (6.19% (95% CI: 1.75%‚Äì10.64%)). Five tested positive in pharyngeal swabs, one in anal swab and one in both. Six did not present symptoms recognized as MPXV infection. Three samples were positive for CPE, and showed anti-vaccinia pAb staining by FACS and confocal microscopy. This suggests that unrecognized Mpox cases can shed infectious virus. Restricting testing to individuals reporting Mpox symptoms may not be sufficient to contain outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">https://www.nature.com/articles/s41467-023-40490-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-29</strong> <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">
            ü§ñ COVID-19 Vaccines and Long-Haul Symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-134', this)">
            More</button></h4>
            <div id="toggle-134" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of monovalent COVID-19 vaccines on viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT
Scientists studied if some viruses, besides COVID-19, might be making people feel sick for a long time after they had COVID. They also looked at how getting vaccinated against COVID-19 might affect these other viruses and the symptoms of long-COVID syndrome. The researchers collected data from 252 patients who had been infected with COVID-19 and were experiencing long-COVID symptoms. Some of these patients had received a vaccine against COVID-19, while others hadn't. They compared the levels of antibodies (special proteins in the blood) for some viruses between vaccinated and non-vaccinated patients, as well as with people who had never been infected with COVID-19. The results showed that those who had gotten vaccinated had fewer symptoms like fatigue and multiorgan problems. They also had lower levels of antibodies for certain viruses, suggesting that the vaccine might have helped stop these viruses from making them feel sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Epstein‚ÄìBarr virus (EBV) reactivation may be involved in long-COVID symptoms, but reactivation of other viruses as a factor has received less attention. Here we evaluated the reactivation of parvovirus-B19 and several members of the Herpesviridae family (DNA viruses) in patients with long-COVID syndrome. We hypothesized that monovalent COVID-19 vaccines inhibit viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome, thereby reducing clinical symptoms. Clinical and laboratory data for 252 consecutive patients with PCR-verified past SARS-CoV-2 infection and long-COVID syndrome (155 vaccinated and 97 non-vaccinated) were recorded during April 2021‚ÄìMay 2022 (median 243 days post-COVID-19 infection). DNA virus‚Äìrelated IgG and IgM titers were compared between vaccinated and non-vaccinated long-COVID patients and with age- and sex-matched non-infected, unvaccinated (pan-negative for spike-antibody) controls. Vaccination with monovalent COVID-19 vaccines was associated with significantly less frequent fatigue and multiorgan symptoms (p<‚Äâ0.001), significantly less cumulative DNA virus‚Äìrelated IgM positivity, significantly lower levels of plasma IgG subfractions 2 and 4, and significantly lower quantitative cytomegalovirus IgG and IgM and EBV IgM titers. These results indicate that anti-SARS-CoV-2 vaccination may interrupt viral cross-talk in patients with long-COVID syndrome (ClinicalTrials.gov Identifier: NCT05398952).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">https://www.nature.com/articles/s41541-023-00739-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-23</strong> <a href="https://www.nature.com/articles/s41467-023-41587-x" target="_blank">
            ü§ñ Viral Immunity: A New Standard?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-135', this)">
            More</button></h4>
            <div id="toggle-135" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Monkeypox virus-infected individuals mount comparable humoral immune responses as Smallpox-vaccinated individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In 2022, a strange virus called monkeypox was found in people who had not traveled to areas where it is common. As the number of cases grew, public health agencies started offering a special vaccine to men who are at high risk of getting infected. We developed tests that can detect if someone has been infected with the monkeypox virus or if they have had the vaccine. Our research shows that having just one dose of the vaccine doesn't give people strong protection against the virus, but more doses can help. When people get infected with the virus, their bodies respond in a similar way to when they have the vaccine. We also found that some parts of the monkeypox virus can be used as markers to tell if someone has been infected or vaccinated. This knowledge could help us develop new vaccines and treatments in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In early 2022, a cluster of monkeypox virus (MPXV) infection (mpox) cases were identified within the UK with no prior travel history to MPXV-endemic regions. Subsequently, case numbers exceeding 80,000 were reported worldwide, primarily affecting gay, bisexual, and other men who have sex with men (GBMSM). Public health agencies worldwide have offered the IMVANEX Smallpox vaccination to these individuals at high-risk to provide protection and limit the spread of MPXV. We have developed a comprehensive array of ELISAs to study poxvirus-induced antibodies, utilising 24 MPXV and 3 Vaccinia virus (VACV) recombinant antigens. Panels of serum samples from individuals with differing Smallpox-vaccine doses and those with prior MPXV infection were tested on these assays, where we observed that one dose of Smallpox vaccination induces a low number of antibodies to a limited number of MPXV antigens but increasing with further vaccination doses. MPXV infection induced similar antibody responses to diverse poxvirus antigens observed in Smallpox-vaccinated individuals. We identify MPXV A27 as a serological marker of MPXV-infection, whilst MPXV M1 (VACV L1) is likely IMVANEX-specific. Here, we demonstrate analogous humoral antigen recognition between both MPXV-infected or Smallpox-vaccinated individuals, with binding to diverse yet core set of poxvirus antigens, providing opportunities for future vaccine (e.g., mRNA) and therapeutic (e.g., mAbs) design.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-41587-x" target="_blank">https://www.nature.com/articles/s41467-023-41587-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-23</strong> <a href="https://www.nature.com/articles/s41598-023-43236-1" target="_blank">
            ü§ñ AI-Powered Monkeypox Detection: A Deep Learning Approach</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-136', this)">
            More</button></h4>
            <div id="toggle-136" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Hyper-parameter tuned deep learning approach for effective human monkeypox disease detection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Human monkeypox is a rare virus that can spread quickly and cause problems. To treat it, doctors need to diagnose it early. Recently, using computer models to detect the virus has become more popular. Monkeypox is a special kind of illness that can jump from animals to people. There's been a big outbreak in over 40 countries. Because of this, scientists are looking for new ways to test for monkeypox that don't rely on PCR tests, which can be hard to get in some places. In this research, we created a computer model that uses pictures of skin lesions to accurately identify monkeypox. We combined two types of neural networks and used a technique called transfer learning to help the model learn from existing images. We also fine-tuned the model using three different methods to make it even better. When tested on pictures of monkeypox lesions, our model got 98.18% accurate. This suggests that our model could be useful in real-world settings for diagnosing human monkeypox.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Human monkeypox is a very unusual virus that can devastate society. Early identification and diagnosis are essential to treat and manage an illness effectively. Human monkeypox disease detection using deep learning models has attracted increasing attention recently. The virus that causes monkeypox may be passed to people, making it a zoonotic illness. The latest monkeypox epidemic has hit more than 40 nations. Computer-assisted approaches using Deep Learning techniques for automatically identifying skin lesions have shown to be a viable alternative in light of the fast proliferation and ever-growing problems of supplying PCR (Polymerase Chain Reaction) Testing in places with limited availability. In this research, we introduce a deep learning model for detecting human monkeypoxes that is accurate and resilient by tuning its hyper-parameters. We employed a mixture of convolutional neural networks and transfer learning strategies to extract characteristics from medical photos and properly identify them. We also used hyperparameter optimization strategies to fine-tune the Model and get the best possible results. This paper proposes a Yolov5 model-based method for differentiating between chickenpox and Monkeypox lesions on skin pictures. The Roboflow skin lesion picture dataset was subjected to three different hyperparameter tuning strategies: the SDG optimizer, the Bayesian optimizer, and Learning without Forgetting. The proposed Model had the highest classification accuracy (98.18%) when applied to photos of monkeypox skin lesions. Our findings show that the suggested Model surpasses the current best-in-class models and may be used in clinical settings for actual Human Monkeypox disease detection and diagnosis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-43236-1" target="_blank">https://www.nature.com/articles/s41598-023-43236-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-22</strong> <a href="https://www.nature.com/articles/s41467-023-41628-5" target="_blank">
            ü§ñ Protecting Against a Related Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-137', this)">
            More</button></h4>
            <div id="toggle-137" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>mRNA vaccines encoding fusion proteins of monkeypox virus antigens protect mice fromvaccinia viruschallenge</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The recent outbreaks of monkeypox have raised concerns about finding effective vaccines. However, the current approved vaccines have either been associated with safety concerns or are in short supply. Scientists have developed three new vaccines that use a promising technology called mRNA. These vaccines target specific proteins from the monkeypox virus and can be given as a single dose to provide protection against the virus. In experiments using mice, all three vaccines were able to protect them from a lethal viral challenge within 7 days of vaccination. The results suggest that these new vaccines could be a viable alternative to current vaccines and may offer better protection against monkeypox.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recent outbreaks of mpox have raised concerns over the need for effective vaccines. However, the current approved vaccines have either been associated with safety concerns or are in limited supply. mRNA vaccines, which have shown high efficacy and safety againstSARS-CoV-2infection, are a promising alternative. In this study, three mRNA vaccines are developed that encodemonkeypox virus(MPXV) proteins A35R and M1R, including A35R extracellular domain -M1R fusions (VGPox 1 and VGPox 2) and a mixture of encapsulated full-length mRNAs for A35R and M1R (VGPox 3). All three vaccines induce early anti-A35R antibodies in femaleBalb/cmice, but only VGPox 1 and 2 generate detectable levels of anti-M1R antibodies at day 7 after vaccination. However, all three mRNA vaccine groups completely protect mice from a lethal dose ofvaccinia virus(VACV) challenge. A single dose of VGPox 1, 2, and 3 provide protection against the lethal viral challenge within 7 days post-vaccination. Long-term immunity and protection were also observed in all three candidates. Additionally, VGPox 2 provided better passive protection. These results suggest that the VGPox series vaccines enhance immunogenicity and can be a viable alternative to current whole-virus vaccines to defend against mpox.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-41628-5" target="_blank">https://www.nature.com/articles/s41467-023-41628-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-20</strong> <a href="https://www.nature.com/articles/s41467-023-41541-x" target="_blank">
            ü§ñ After a Mild COVID-19 Infection: Who's at Risk?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-138', this)">
            More</button></h4>
            <div id="toggle-138" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predictors of the post-COVID condition following mild SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who had mental health issues, breathing problems, or general health concerns before they got COVID-19 were more likely to experience long-term symptoms after their initial illness. Researchers used data from over 214,000 people in Norway and found that these groups were at higher risk of developing post-COVID complaints. They also used computer models to see if they could predict which people would develop post-COVID symptoms. The results showed that the model was able to accurately predict the occurrence of long-term symptoms in about 79% of cases, using information such as pre-infection health characteristics and whether or not the person had received a COVID-19 vaccine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Whereas the nature of the post-COVID condition following mild acute COVID-19 is increasingly well described in the literature, knowledge of its risk factors, and whether it can be predicted, remains limited. This study, conducted in Norway, uses individual-level register data from 214,667 SARS-CoV-2 infected individuals covering a range of demographic, socioeconomic factors, as well as cause-specific healthcare utilization in the years prior to infection to assess the risk of post-COVID complaints ‚â•3 months after testing positive. We find that the risk of post-COVID was higher among individuals who prior to infection had been diagnosed with psychological (OR‚Äâ=‚Äâ2.12, 95% CI 1.84‚Äì2.44), respiratory (OR‚Äâ=‚Äâ2.03, 95% CI 1.78‚Äì2.32), or general and unspecified health problems (OR‚Äâ=‚Äâ1.78, 95% CI 1.52‚Äì2.09). To assess the predictability of post-COVID after mild initial disease, we use machine learning methods and find that pre-infection characteristics, combined with information on the SARS-CoV-2 virus type and vaccine status, to a considerable extent (AUC‚Äâ=‚Äâ0.79, 95% CI 0.75‚Äì0.81) could predict the occurrence of post-COVID complaints in our sample.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-41541-x" target="_blank">https://www.nature.com/articles/s41467-023-41541-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-19</strong> <a href="https://www.nature.com/articles/s41598-023-42421-6" target="_blank">
            ü§ñ Breastmilk Offers Hope in Fight Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-139', this)">
            More</button></h4>
            <div id="toggle-139" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Breastmilk from COVID-19 negative lactating mothers shows neutralizing activity against SARS-COV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT ###
Breast milk helps protect newborn babies from getting sick with infections, and scientists have found that it also has special powers to fight the virus that causes COVID-19. This study looked at breast milk from two groups of women: those who had COVID-19 when they gave birth, and those who didn't have the virus. They tested this milk for its ability to stop the virus and found some surprising results. The women who had COVID-19 when they gave birth had special antibodies in their milk that helped fight the virus, but these disappeared after a few days. On the other hand, the milk from women who didn't have COVID-19 was still able to fight the virus, even though it didn't contain these special antibodies. This suggests that there might be other, non-specific compounds in breast milk that can help protect babies from getting sick with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Breastmilk protects newborns from infections through specific and nonspecific compounds. This study investigated the neutralizing activity against SARS-CoV-2 of breastmilk from SARS-CoV-2 negative, unvaccinated mothers, and compared it to that from infected nursing mothers. We enrolled women after COVID-19 swab testing results upon maternity admission, and divided them into two groups: group A, COVID-19-positive mothers, and group B, negative mothers. Breastmilk was randomly sampled at 2, 7, and 20¬†days postpartum. We collected 19 samples for Group A and 41 for Group B. A microneutralization assay was used to determine the 50% neutralization (NT50) titre. The presence of neutralizing antibodies was also determined. Group A had 100% neutralizing samples at 2¬†days postpartum (T0), declining 7¬†days postpartum (T1) and 20¬†days postpartum (T2). Group B samples exhibited neutralizing activity mostly at 7¬†days postpartum (T1) (90%). Negative mothers' samples showed no correlation between NT50titres and antibodies' presence, suggesting that non-specific breastmilk components may exert antiviral action against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-42421-6" target="_blank">https://www.nature.com/articles/s41598-023-42421-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-15</strong> <a href="https://www.nature.com/articles/s41598-023-42321-9" target="_blank">
            ü§ñ Long COVID Symptoms Lasting Months After Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-140', this)">
            More</button></h4>
            <div id="toggle-140" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk of long COVID main symptoms after SARS-CoV-2 infection: a systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the abstract rewritten for a young person:

This review looked at how often people who got sick with COVID-19 still have symptoms four weeks or more after getting infected. It also compared how their quality of life changed after being sick. The researchers searched lots of databases to find studies that reported on these topics. They found 33 studies and included 20 of them in the analysis.

The results showed that people who got sick with COVID-19 were more likely to have fatigue (feeling tired all the time), shortness of breath, memory difficulties, and concentration problems than people who didn't get infected. The quality of life after being hospitalized with severe COVID-19 was also lower compared to those who weren't as sick.

These risks might change over time as vaccines and new variants of the virus become more common.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This review aimed to summarise the relative risk (RR) of the main symptoms of long COVID in people infected with SARS-CoV-2 compared to uninfected controls, as well as the difference in health-related quality of life (HRQoL) after infection. MEDLINE, EMBASE, PubMed, NLM-LitCovid, WHO-COVID-19, arXiv and Europe-PMC were searched up to 23rd March 2022. Studies reporting risk (four or more weeks after infection) of fatigue, shortness of breath, and cognitive dysfunction, as well as comparative HRQoL outcomes, were included. Pairwise random-effects meta-analyses were performed to pool risks of individual symptoms. Thirty-three studies were identified; twenty studies reporting symptom risks were included in the meta-analyses. Overall, infection with SARS-CoV-2 carried significantly higher risk of fatigue (RR 1.72, 95% confidence intervals [CIs] 1.41, 2.10), shortness of breath (RR 2.60, 95% CIs 1.96, 3.44), memory difficulties (RR 2.53, 95% CIs 1.30, 4.93), and concentration difficulties (RR 2.14, 95% CIs 1.25, 3.67). Quality of life findings were varied and comparisons between studies were challenging due to different HRQoL instruments used and study heterogeneity, although studies indicated that severe hospitalised COVID is associated with a significantly poorer HRQoL after infection. These risks are likely to constantly change as vaccines, reinfections, and new variants alter global immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-42321-9" target="_blank">https://www.nature.com/articles/s41598-023-42321-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-14</strong> <a href="https://www.nature.com/articles/s41392-023-01615-0" target="_blank">
            ü§ñ Antibodies From Early COVID Survivors Protect Against Virus Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-141', this)">
            More</button></h4>
            <div id="toggle-141" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broadly neutralizing antibodies derived from the earliest COVID-19 convalescents protect mice from SARS-CoV-2 variants challenge</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simpler version of the abstract:

Three years ago, a new virus called COVID-19 was first reported in Wuhan, China. Scientists developed special medicines called vaccines to fight this virus. In this study, researchers created six special antibodies from two people who had recovered from COVID-19 and tested how well they could stop different strains of the virus from infecting cells. Two of these antibodies were very good at stopping many types of viruses and even protected mice from deadly infections when given after exposure to the virus. When scientists looked closely at how these antibodies worked, they found that they blocked a specific part of the virus (called RBD) that it uses to attach to human cells. This study shows that people who have recovered from COVID-19 or been vaccinated may still be protected against new and changing strains of the virus due to the presence of such broadly neutralizing antibodies in their bodies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) was first reported three years ago, when a group of individuals were infected with the original SARS-CoV-2 strain, based on which vaccines were developed. Here, we develop six human monoclonal antibodies (mAbs) from two elite convalescents in Wuhan and show that these mAbs recognize diverse epitopes on the receptor binding domain (RBD) and can inhibit the infection of SARS-CoV-2 original strain and variants of concern (VOCs) to varying degrees, including Omicron strains XBB and XBB.1.5. Of these mAbs, the two most broadly and potently neutralizing mAbs (7B3 and 14B1) exhibit prophylactic activity against SARS-CoV-2 WT infection and therapeutic effects against SARS-CoV-2 Delta variant challenge in K18-hACE2 KI mice. Furthermore, post-exposure treatment with 7B3 protects mice from lethal Omicron variants infection. Cryo-EM analysis of the spike trimer complexed with 14B1 or 7B3 reveals that these two mAbs bind partially overlapped epitopes onto the RBD of the spike, and sterically disrupt the binding of human angiotensin-converting enzyme 2 (hACE2) to RBD. Our results suggest that mAbs with broadly neutralizing activity against different SARS-CoV-2 variants are present in COVID-19 convalescents infected by the ancestral SARS-CoV-2 strain, indicating that people can benefit from former infections or vaccines despite the extensive immune escape of SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-023-01615-0" target="_blank">https://www.nature.com/articles/s41392-023-01615-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-03</strong> <a href="https://www.nature.com/articles/s41598-023-41545-z" target="_blank">
            ü§ñ Classifying Monkeypox Skin Lesions with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-142', this)">
            More</button></h4>
            <div id="toggle-142" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Utilizing convolutional neural networks to classify monkeypox skin lesions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Monkeypox is a rare illness that can cause bad skin symptoms in humans. It's hard to tell if someone has it just by looking at their skin, especially in places where medical tests are not available. Computers can be trained to look at pictures and make good decisions. This study used one of these computer methods called Convolutional Neural Networks (CNNs) to help identify monkeypox symptoms. To make the CNNs work better, we used another method called Grey Wolf Optimizer (GWO). It helped a lot! The new model was really accurate - 95.3% correct in identifying if someone had monkeypox or not. This could be very helpful for doctors to quickly and correctly diagnose monkeypox, which would lead to better treatment and outcomes for patients. It also has important implications for public health, as it could help control and prevent outbreaks of the disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox is a rare¬†viral disease¬†that can cause severe illness in humans, presenting with skin lesions and rashes. However, accurately diagnosing monkeypox based on visual inspection of the lesions can be challenging and time-consuming, especially in resource-limited settings where laboratory tests may not be available. In recent years,¬†deep learning methods, particularly¬†Convolutional Neural Networks¬†(CNNs), have shown great potential in image recognition and classification tasks. To this end, this study proposes an approach using CNNs to classify¬†monkeypox skin¬†lesions. Additionally, the study optimized the¬†CNN model¬†using the¬†Grey Wolf Optimizer¬†(GWO) algorithm, resulting in a significant improvement in accuracy, precision, recall, F1-score, and AUC compared to the non-optimized model. The¬†GWO optimization¬†strategy can enhance the performance of CNN models on similar tasks. The optimized model achieved an impressive accuracy of 95.3%, indicating that the¬†GWO optimizer¬†has improved the model's ability to discriminate between positive and negative classes. The proposed approach has several potential benefits for improving the accuracy and efficiency of¬†monkeypox diagnosis¬†and surveillance. It could enable faster and more accurate diagnosis of¬†monkeypox skin lesions, leading to earlier detection and better patient outcomes. Furthermore, the approach could have crucial¬†public health implications¬†for controlling and preventing¬†monkeypox outbreaks. Overall, this study offers a novel and highly effective approach for diagnosing monkeypox, which could have significant real-world applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-41545-z" target="_blank">https://www.nature.com/articles/s41598-023-41545-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-08-31</strong> <a href="https://www.nature.com/articles/s41591-023-02517-y" target="_blank">
            ü§ñ COVID-19 Vaccines Tested in Humans Show Promise</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-143', this)">
            More</button></h4>
            <div id="toggle-143" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Original SARS-CoV-2 monovalent and Omicron BA.4/BA.5 bivalent COVID-19 mRNA vaccines: phase 2/3 trial interim results</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new type of COVID-19 vaccine, called mRNA-12.73.222, was tested in a study to see how well it worked compared to an older version of the vaccine, called mRNA-12.73. The researchers wanted to know if the new vaccine would be safe and if it could protect against two types of COVID-19: Omicron BA.4/BA.5 and an earlier version of the virus that was circulating in Wuhan, China. 

In the study, 511 adults who had previously received mRNA-12.73 as their first round of vaccinations and booster shots were given a second booster shot of either mRNA-12.73 or mRNA-12.73.222. The researchers found that the new vaccine, mRNA-12.73.222, was better at producing antibodies against Omicron BA.4/BA.5 compared to mRNA-12.73. They also found that it was just as good at producing antibodies against the earlier version of the virus. 

The study showed that mRNA-12.73.222 was safe and did not cause any new problems. However, the researchers noted that more monitoring is needed as new variants of COVID-19 emerge.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This ongoing, open-label, phase 2/3 trial compared the safety and immunogenicity of the Omicron BA.4/BA.5-containing bivalent mRNA-1273.222 vaccine with the ancestral Wuhan-Hu-1 mRNA-1273 as booster doses. Two groups of adults who previously received mRNA-1273 as primary vaccination series and booster doses were enrolled in a sequential, nonrandomized manner and received single-second boosters of mRNA-1273 (n=‚Äâ376) or bivalent mRNA-1273.222 (n=‚Äâ511). Primary objectives were safety and the noninferiority or superiority of neutralizing antibody (nAb) responses against Omicron BA.4/BA.5 and ancestral SARS-CoV-2 with the D614G mutation (ancestral SARS-CoV-2 (D614G)), 28‚Äâdays post boost. Superiority and noninferiority were based on prespecified success criteria (lower bounds of 95% CI‚Äâ>‚Äâ1 and <‚Äâ0.677, respectively) of the mRNA-1273.222:mRNA-1273 geometric mean ratios. Bivalent Omicron BA.4/BA.5-containing mRNA-1273.222 elicited superior nAb responses against BA.4/BA.5 versus mRNA-1273 and noninferior responses against ancestral SARS-CoV-2 (D614G) at day 29 post boost in participants without detectable prior SARS-CoV-2 infection. Day 29 seroresponses against Omicron BA.4/BA.5 were higher for mRNA-1273.222 than for mRNA-1273 and similar against ancestral SARS-CoV-2 (D614G), both meeting noninferiority criterion. The safety profile of mRNA-1273.222 was similar to that previously reported for mRNA-1273 with no new safety concerns identified. Continued monitoring of neutralization and real-world vaccine effectiveness are needed as additional divergent-virus variants emerge. ClinicalTrials.gov registration: NCT04927065.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-023-02517-y" target="_blank">https://www.nature.com/articles/s41591-023-02517-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-08-26</strong> <a href="https://www.nature.com/articles/s41598-023-40586-8" target="_blank">
            ü§ñ Post-Covid Syndrome in Brazil Lasts 2 Years, Study Finds</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-144', this)">
            More</button></h4>
            <div id="toggle-144" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-Covid condition and clinic characteristics associated with SARS-CoV-2 infection: a 2-year follow-up to Brazilian cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Until January 2023, Brazil had one of the highest numbers of people who got COVID-19 in the world - over 37 million cases! Despite the good work being done to vaccinate people against COVID-19, many are still experiencing symptoms long after they've recovered from the initial illness. This is known as Post-Covid Condition (PCC). As a result, Brazil needs to prepare for an increasing number of patients who will have chronic health problems due to COVID-19. To better understand these long-term effects and how they affect people's quality of life, it's crucial to study patients over extended periods. We looked at a group of people in Brazil who got COVID-19 during the first year of the pandemic and followed them up two years later to see what health impacts they experienced. Our findings showed that many people had reported symptoms like muscle pain and cognitive difficulties after two years, with women and those who were previously hospitalized being more likely to experience these issues. This study is important for helping public health authorities develop strategies to control the spread of COVID-19 and mitigate its effects on society.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Until January 2023, Brazil recorded 37 million COVID-19 cases despite the decrease in mortality due to mass vaccination efforts against COVID-19. The infection continues to challenge researchers and health professionals with the persistent symptoms and onset manifestations after the acute phase of the disease, namely Post-Covid Condition (PCC). Being one of the countries with the highest infection rate, Brazil must prepare for a growing number of patients with chronic health consequences of COVID-19. Longitudinal studies that follow patients over extended periods are crucial in understanding the long-term impacts of COVID-19, including potential health consequences and the effects on quality of life. We describe the clinical profile of a cohort of COVID-19 patients infected during the first year of the pandemic in Brazil and a follow-up after two years to investigate the health impacts of SARS-CoV-2 infection. The first wave of SARS-CoV-2 infection in Brazil featured extensive drug misuse, notably the ineffective COVID kit comprised of ivermectin, antimalarials and azithromycin, and elevated in-hospital mortality. In the second phase of the study, Post-Covid Condition was reported by symptomatic COVID-19 subjects across different severity levels two years after infection. Long haulers are more likely to be women, previously hospitalized, and reported a range of symptoms from muscle pain to cognitive deficit. Our longitudinal study is essential to inform public health authorities to develop strategies and policies to control the spread of the virus and mitigate its impacts on society.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-40586-8" target="_blank">https://www.nature.com/articles/s41598-023-40586-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-08-23</strong> <a href="https://www.nature.com/articles/s41467-023-40460-1" target="_blank">
            ü§ñ COVID-19 Long-Haulers: Immunity Not to Blame</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-145', this)">
            More</button></h4>
            <div id="toggle-145" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Persistent symptoms after COVID-19 are not associated with differential SARS-CoV-2 antibody or T cell immunity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that a young person could understand:

Scientists wanted to figure out why some people who had mild COVID-19 symptoms still felt sick long after they recovered. They thought maybe their immune system was doing something weird when they first got infected, but not in a good way. To test this idea, they looked at the blood of 86 healthcare workers who had mild or no symptoms when they were first infected with COVID-19. They checked to see if these workers' bodies were making antibodies (like superpower proteins) and if their immune cells were working properly. What they found was that people who still felt sick long after recovery and those who didn't feel sick at all had similar levels of antibodies and functioning immune cells. This suggests that the way our immune system responds to COVID-19 might not be the reason why some people get Long Covid.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Among the unknowns in decoding the pathogenesis of SARS-CoV-2 persistent symptoms in Long Covid is whether there is a contributory role of abnormal immunity during acute infection. It has been proposed that Long Covid is a consequence of either an excessive or inadequate initial immune response. Here, we analyze SARS-CoV-2 humoral and cellular immunity in 86 healthcare workers with laboratory confirmed mild or asymptomatic SARS-CoV-2 infection during the first wave. Symptom questionnaires allow stratification into those with persistent symptoms and those without for comparison. During the period up to 18-weeks post-infection, we observe no difference in antibody responses to spike RBD or nucleoprotein, virus neutralization, or T cell responses. Also, there is no difference in the profile of antibody waning. Analysis at 1-year, after two vaccine doses, comparing those with persistent symptoms to those without, again shows similar SARS-CoV-2 immunity. Thus, quantitative differences in these measured parameters of SARS-CoV-2 adaptive immunity following mild or asymptomatic acute infection are unlikely to have contributed to Long Covid causality. ClinicalTrials.gov (NCT04318314).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-40460-1" target="_blank">https://www.nature.com/articles/s41467-023-40460-1</a></p>
                
                <br><br>
            </div>
        </div>
        
</p>
</div>
<br><br><br><br><br><br><br><br>
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
      <!-- end footer -->
      <!-- Javascript files--> 
      <script src="js/jquery.min.js"></script> 
      <script src="js/popper.min.js"></script> 
      <script src="js/bootstrap.bundle.min.js"></script> 
      <script src="js/jquery-3.0.0.min.js"></script> 
      <script src="js/plugin.js"></script> 
      <!-- sidebar --> 
      <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
      <script src="js/custom.js"></script>
      <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
      <script>
         $(document).ready(function(){
         $(".fancybox").fancybox({
         openEffect: "none",
         closeEffect: "none"
         });
         
         $(".zoom").hover(function(){
         
         $(this).addClass('transition');
         }, function(){
         
         $(this).removeClass('transition');
         });
         });
         
      </script> 
   </body>
</html>
<!-- end footer -->
<script>
function toggleAbstract(id, button) {
    const content = document.getElementById(id);
    if (content.style.display === "none") {
        content.style.display = "block";
        button.textContent = "Less";
    } else {
        content.style.display = "none";
        button.textContent = "More";
    }
}
document.addEventListener("DOMContentLoaded", function() {
    const buttons = document.querySelectorAll(".toggle-button");
    buttons.forEach(button => {
        const contentId = button.getAttribute("onclick").match(/'([^']+)'/)[1];
        const content = document.getElementById(contentId);
        if (content) {
            content.style.display = "none";
        }
    });
});
</script>
</body>
</html>
