
<!DOCTYPE html>
<html lang="en">
   <head>
      <!-- basic -->
      <meta charset="utf-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <!-- mobile metas -->
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <meta name="viewport" content="initial-scale=1, maximum-scale=1">
      
      <!-- site metas -->
      <title>Data Sources</title>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <link rel="icon" href="images/logo.png" type="image/x-icon">
      <meta name="keywords" content="COVID-19, wastewater, measurement, monitoring, surveillance, public health, environmental monitoring, water analysis, viral RNA, SARS-CoV-2,SARS-nCoV-2, global, countries, data, analysis, pandemic, wastewater-based epidemiology, geo-map">
      <meta name="description" content="Data Sources of Visualizations">
      <meta name="author" content="Daniel Dynesius">
      <!-- bootstrap css -->
      <link rel="stylesheet" href="css/bootstrap.min.css">
      <!-- style css -->
      <link rel="stylesheet" href="css/style.css">
      <!-- Responsive-->
      <link rel="stylesheet" href="css/responsive.css">
      <!-- fevicon -->
      <link rel="icon" href="images/fevicon.png" type="image/gif" />
      <!-- Scrollbar Custom CSS -->
      <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
      <!-- Tweaks for older IEs-->
      <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
      <style>
            @media (max-width: 767px) {
            /* Styles for screens smaller than 768px (mobile) */
            .menu-area-main {
                flex-direction: column;
            }
        
            .menu-area-main li {
                text-align: center;
            }
            }
    </style>      
      <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script><![endif]-->
   </head>
   <!-- body -->
   <body class="main-layout">
      <!-- loader  -->
      <div class="loader_bg">
         <div class="loader"><img src="images/loading.gif" alt="#" /></div>
      </div>
      <!-- end loader --> 
      <!-- header -->
      <header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                    </div>
                  </div>
               </div>
            </div>
         </div>
         <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo"> <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a> </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main d-flex">
                                    <li class=""> <a href="index.html">Geo Map</a> </li>
                                    <li class=""> <a href="trends.html">Trend Graphs</a> </li>
                                    <li class=""> <a href="predictions_tab.html">Predictive</a> </li>
                                    <li class="active"> <a href="new_research.html">Research News</a> </li>
                                    <li class=""> <a href="data_sources.html">Data Sources</a> </li>
                                    <li class=""> <a href="about.html">About</a> </li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
         <!-- end header inner --> 
      </header>
      <!-- end header -->
      <div class="brand_color">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <div class="titlepage">
                        <h2>Research News</h2>
                        <br>
                        <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                        Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>

                    </div>
                </div>
            </div>
        </div>

    </div>

    <!-- contact -->
<p>


    <br><br><br>

        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Analysis and comparative study of a deterministic mathematical model of SARS-COV-2 with fractal-fractional operators: a case study</h3>
            <h3 id="section1"><a href="#section1" style="text-decoration: none; color: inherit;">ü§ñ Comparing Fractional Fractal Models in SARS-COV-2 Spread</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-18</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research investigates a fractal-fractional-order mathematical model that considers hospitalized patients and vaccinations, using various derivative methods and stability analysis, and fits the model with real data from Pakistan.</p>
            <br><strong>Abstract</strong><br>
            <p>In this paper, we investigate a fractal-fractional-order mathematical model with the influence of hospitalized patients and the impact of vaccination with fractal-fractional operators. The respective derivatives are considered in the Caputo, Caputo Fabrizio, and Atangana‚ÄìBaleanu senses of fractional order\( \alpha \)and fractal dimension\( \tau \). For the proposed problem, some results regarding basic reproduction number and stability are given. Using the next-generation matrix approach, we have investigated the global and local stability of several types of equilibrium points. We provide a detailed analysis of the existence and uniqueness of the solution. Moreover, we fit the model with the real data of Pakistan from June 01, 2020, till March 24, 2021. Then, we use the fractal-fractional derivative to find a numerical solution for the model. MATLAB software is used for numerical illustration. Graphical presentations corresponding to different parameteric values are given as well.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-56557-6" target="_blank">https://www.nature.com/articles/s41598-024-56557-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Neurofilament light chain and glial fibrillary acid protein levels are elevated in post-mild COVID-19 or asymptomatic SARS-CoV-2 cases</h3>
            <h3 id="section2"><a href="#section2" style="text-decoration: none; color: inherit;">ü§ñ "Elevated Neurofilament Light Chain & Glial Fibrillary Acid Protein Levels in Mild COVID-19 or Asymptomatic Cases"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-18</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that adults who previously had COVID-19 had higher levels of sNfL and sGFAP biomarkers, associated with cognitive impairment, even 10 months after the infection; however, these levels decreased over time but remained elevated compared to healthy controls.</p>
            <br><strong>Abstract</strong><br>
            <p>Given the huge impact of the COVID-19 pandemic, it appears of paramount importance to assess the cognitive effects on the population returning to work after COVID-19 resolution. Serum levels of neurofilament light chain (sNfL) and glial fibrillary acidic protein (sGFAP) represent promising biomarkers of neuro-axonal damage and astrocytic activation. In this cohort study, we explored the association between sNfL and sGFAP concentrations and cognitive performance in a group of 147 adult workers with a previous asymptomatic SARS-CoV-2 infection or mild COVID-19, one week and, in 49 of them, ten months after SARS-Cov2 negativization and compared them to a group of 82 age and BMI-matched healthy controls (HCs). sNfL and sGFAP concentrations were assessed using SimoaTM assay Neurology 2-Plex B Kit. COVID-19 patients were interviewed one-on-one by trained physicians and had to complete a list of questionnaires, including the Cognitive Failure Questionnaire (CFQ). At the first assessment (T0), sNfL and sGFAP levels were significantly higher in COVID-19 patients than in HCs (p‚Äâ<‚Äâ0.001 for both). The eleven COVID-19 patients with cognitive impairment had significantly higher levels of sNfL and sGFAP than the others (p‚Äâ=‚Äâ0.005 for both). At the subsequent follow-up (T1), sNfL and sGFAP levels showed a significant decrease (median sNfL 18.3¬†pg/mL; median sGFAP 77.2¬†pg/mL), although they were still higher than HCs (median sNfL 7.2¬†pg/mL, median sGFAP 63.5¬†pg/mL). Our results suggest an ongoing damage involving neurons and astrocytes after SARS-Cov2 negativization, which reduce after ten months even if still evident compared to HCs.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">https://www.nature.com/articles/s41598-024-57093-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Clustering analysis for the evolutionary relationships of SARS-CoV-2 strains</h3>
            <h3 id="section3"><a href="#section3" style="text-decoration: none; color: inherit;">ü§ñ SARS-CoV-2 Strain Relationships: Clustering Analysis and Evolution</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-18</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research uses hierarchical clustering analysis to investigate SARS-CoV-2 strain relationships, finding that specific strains, like BF.28, BE.1.1.1, BA.5.3, and BA.5.6.4, may be the majority sources for future evolution, while others show increased immune evasion, transmissibility, and pathogenicity, highlighting the need to closely monitor these trends for public health and society.</p>
            <br><strong>Abstract</strong><br>
            <p>To explore the differences and relationships between the available SARS-CoV-2 strains and predict the potential evolutionary direction of these strains, we employ the hierarchical clustering analysis to investigate the evolutionary relationships between the SARS-CoV-2 strains utilizing the genomic sequences collected in China till January 7, 2023. We encode the sequences of the existing SARS-CoV-2 strains into numerical data throughk-mer algorithm, then propose four methods to select the representative sample from each type of strains to comprise the dataset for clustering analysis. Three hierarchical clustering algorithms named Ward-Euclidean, Ward-Jaccard, and Average-Euclidean are introduced through combing the Euclidean and Jaccard distance with the Ward and Average linkage clustering algorithms embedded in the OriginPro software. Experimental results reveal that BF.28, BE.1.1.1, BA.5.3, and BA.5.6.4 strains exhibit distinct characteristics which are not observed in other types of SARS-CoV-2 strains, suggesting their being the majority potential sources which the future SARS-CoV-2 strains‚Äô evolution from. Moreover, BA.2.75, CH.1.1, BA.2, BA.5.1.3, BF.7, and B.1.1.214 strains demonstrate enhanced abilities in terms of immune evasion, transmissibility, and pathogenicity. Hence, closely monitoring the evolutionary trends of these strains is crucial to mitigate their impact on public health and society as far as possible.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57001-5" target="_blank">https://www.nature.com/articles/s41598-024-57001-5</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Aesculus hippocastanum extract and the main bioactive constituent Œ≤-escin as antivirals agents against coronaviruses, including SARS-CoV-2</h3>
            <h3 id="section4"><a href="#section4" style="text-decoration: none; color: inherit;">ü§ñ Title: Aesculus Hippocastanum Extract and Œ≤-Escin for Anti-Coronavirus Action</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-17</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study reveals that Œ≤-escin and Aesculus hippocastanum L. seed extract have broad-spectrum antiviral properties against coronaviruses, including SARS-CoV-2, making them potential treatments for respiratory infections and future pandemics.</p>
            <br><strong>Abstract</strong><br>
            <p>Respiratory viruses can cause life-threatening illnesses. The focus of treatment is on supportive therapies and direct antivirals. However, antivirals may cause resistance by exerting selective pressure. Modulating the host response has emerged as a viable therapeutic approach for treating respiratory infections. Additionally, considering the probable future respiratory virus outbreaks emphasizes the need for broad-spectrum therapies to be prepared for the next pandemics. One of the principal bioactive constituents found in the seed extract ofAesculus hippocastanumL. (AH) is Œ≤-escin. The clinical therapeutic role of Œ≤-escin and AH has been associated with their anti-inflammatory effects. Regarding their mechanism of action, we and others have shown that Œ≤-escin and AH affect NF-Œ∫B signaling. Furthermore, we have reported the virucidal and broad-spectrum antiviral properties of Œ≤-escin and AH against enveloped viruses such as RSV, in vitro and in vivo. In this study, we demonstrate that Œ≤-escin and AH have antiviral and virucidal activities against SARS-CoV-2 and CCoV, revealing broad-spectrum antiviral activity against coronaviruses. Likewise, they exhibited NF-Œ∫B and cytokine modulating activities in epithelial and macrophage cell lines infected with coronaviruses in vitro. Hence, Œ≤-escin and AH are promising broad-spectrum antiviral, immunomodulatory, and virucidal drugs against coronaviruses and respiratory viruses, including SARS-CoV-2.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-56759-y" target="_blank">https://www.nature.com/articles/s41598-024-56759-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Clearance of persistent SARS-CoV-2 associates with increased neutralizing antibodies in advanced HIV disease post-ART initiation</h3>
            <h3 id="section5"><a href="#section5" style="text-decoration: none; color: inherit;">ü§ñ "SARS-CoV-2 Clearance and Neutralizing Antibodies in Advanced HIV after ART Start"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-15</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research found that neutralizing antibodies play a crucial role in clearing SARS-CoV-2 infection, even in individuals with immunosuppression due to HIV, while suppressive antiretroviral therapy helps prevent the evolution of escape mutants.</p>
            <br><strong>Abstract</strong><br>
            <p>SARS-CoV-2 clearance requires adaptive immunity but the contribution of neutralizing antibodies and T cells in different immune states is unclear. Here we ask which adaptive immune responses associate with clearance of long-term SARS-CoV-2 infection in HIV-mediated immunosuppression after suppressive antiretroviral therapy (ART) initiation. We assembled a cohort of SARS-CoV-2 infected people in South Africa (n=‚Äâ994) including participants with advanced HIV disease characterized by immunosuppression due to T cell depletion. Fifty-four percent of participants with advanced HIV disease had prolonged SARS-CoV-2 infection (>1 month). In the five vaccinated participants with advanced HIV disease tested, SARS-CoV-2 clearance associates with emergence of neutralizing antibodies but not SARS-CoV-2 specific CD8 T cells, while CD4 T cell responses were not determined due to low cell numbers. Further, complete HIV suppression is not required for clearance, although it is necessary for an effective vaccine response. Persistent SARS-CoV-2 infection led to SARS-CoV-2 evolution, including virus with extensive neutralization escape in a Delta variant infected participant. The results provide evidence that neutralizing antibodies are required for SARS-CoV-2 clearance in HIV-mediated immunosuppression recovery, and that suppressive ART is necessary to curtail evolution of co-infecting pathogens to reduce individual health consequences as well as public health risk linked with generation of escape mutants.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-46673-2" target="_blank">https://www.nature.com/articles/s41467-024-46673-2</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Prevalence, outcomes and associated factors of SARS-CoV-2 infection in psoriasis patients of Southwest China: a cross-sectional survey</h3>
            <h3 id="section6"><a href="#section6" style="text-decoration: none; color: inherit;">ü§ñ SARS-CoV-2 Infection in Psoriasis Patients in Southwest China: Prevalence, Outcomes & Factors - A Cross-Sectional Study</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-15</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study examined the prevalence of SARS-CoV-2 infection in psoriasis patients and its effects on their condition, finding that 84.2% were infected and risk factors included demographics, occupation, protection against COVID-19, vaccination status, nail impairment, and psoriasis severity; among those infected, 5.8% required hospitalization and 23.6% experienced exacerbation of their psoriasis, while booster dose vaccinations reduced the likelihood of COVID-19 complications.</p>
            <br><strong>Abstract</strong><br>
            <p>In this study we aimed to investigate the prevalence of SARS-CoV-2 infection in psoriasis patients, and outcomes of SARS-CoV-2 infection and associated risk factors. A cross-sectional survey was conducted from February 2023 to March 2023. Information was obtained with online questionnaire about psoriasis patients on demographic characteristics, clinical characteristics, SARS-CoV-2 infection and outcomes, vaccination, and routine protection against COVID-19. Logistic regression analysis was used to explore risk factors with SARS-CoV-2 infection and exacerbation of psoriasis. A total of 613 participants were recruited. 516 (84.2%) were infected, and associated factors were sex, working status, routine protection against COVID-19, COVID-19 vaccination, impaired nail, infection exacerbate psoriasis, and severity of psoriasis. Among the patients infected with SARS-CoV-2, 30 (5.8%) required hospitalization, 122 (23.6%) had psoriasis exacerbation due to SARS-CoV-2 infection, and associated factors were subtype of psoriasis, discontinuation of psoriasis treatment during SARS-CoV-2 infection, response following COVID-19 vaccination, and severity of psoriasis. Booster dose vaccination contributed a low probability of COVID-19 sequelae. COVID-19 vaccine‚Äôs effectiveness was unsatisfactory, while booster dose vaccination reduced the occurrence of COVID-19 sequelae in psoriasis patients of Southwest China. Patients treated with psoriasis shown to be safe, without a higher incidence of SARS-CoV-2 infection or COVID-19hospitalization compared to untreated patients. Stopping treatment during SARS-CoV-2 infection led to psoriasis exacerbation, so psoriasis treatment could be continued except severe adverse reaction.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-54424-y" target="_blank">https://www.nature.com/articles/s41598-024-54424-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Decay pattern of SARS-CoV-2 RNA surface contamination in real residences</h3>
            <h3 id="section7"><a href="#section7" style="text-decoration: none; color: inherit;">ü§ñ Surface RNA Contamination Pattern of SARS-CoV-2 in Homes</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research studied the decay pattern of COVID-19 surface contamination in Shanghai homes and found that time was the most significant factor affecting SARS-CoV-2 RNA levels, with a 90% reduction in three days, indicating fomite transmission played a limited role in spread.</p>
            <br><strong>Abstract</strong><br>
            <p>The COVID-19 pandemic has provided valuable lessons that deserve deep thought to prepare for the future. The decay pattern of surface contamination by SARS-CoV-2 RNA in the residences of COVID-19 patients is important but still unknown. We collected 2,233 surface samples from 21 categories of objects in 141 residences of COVID-19 patients in Shanghai when attacked by the omicron variant in spring 2022. Several characteristics of the patients and their residences were investigated to identify relevant associations. The decay of contamination was explored to determine the persistence. Approximately 8.7% of the surface samples were tested positive for SARS-CoV-2 RNA. The basin, water tap, and sewer inlet had the highest positive rates, all exceeding 20%. Only time was significantly associated with the level of surface contamination with SARS-CoV-2, showing a negative association. The decrease fit a first-order decay model with a decay rate of 0.77‚Äâ¬±‚Äâ0.07¬†day‚àí1, suggesting a 90% reduction in three days. Positive associations between the cumulative number of newly diagnosed patients in the same building and the positive rate of SARS-CoV-2 RNA in the public corridor were significant during the three days. Our results, in conjunction with the likely lower infectivity or viability, demonstrate that fomite transmission played a limited role in COVID-19 spread. The time determined SARS-CoV-2 RNA contamination, which was reduced by three days. This study is the first to show the decay patterns of SARS-CoV-2 contamination in real residential environments, providing insight into the patterns of transmission, as well as community-based prevention and control of similar threats.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-54445-7" target="_blank">https://www.nature.com/articles/s41598-024-54445-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Filamentous fungus-produced human monoclonal antibody provides protection against SARS-CoV-2 in hamster and non-human primate models</h3>
            <h3 id="section8"><a href="#section8" style="text-decoration: none; color: inherit;">ü§ñ Fungus-made Human Antibody Protects Hamsters and Primates from COVID-19</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research shows that using a genetically modified fungus can efficiently produce human monoclonal antibodies that effectively neutralize various SARS-CoV-2 virus variants, providing potential protection against different forms of the virus in both hamsters and non-human primates.</p>
            <br><strong>Abstract</strong><br>
            <p>Monoclonal antibodies are an increasingly important tool for prophylaxis and treatment of acute virus infections like SARS-CoV-2 infection. However, their use is often restricted due to the time required for development, variable yields and high production costs, as well as the need for adaptation to newly emerging virus variants. Here we use the genetically modified filamentous fungus expression systemThermothelomyces heterothallica(C1), which has a naturally high biosynthesis capacity for secretory enzymes and other proteins, to produce a human monoclonal IgG1 antibody (HuMab 87G7) that neutralises the SARS-CoV-2 variants of concern (VOCs) Alpha, Beta, Gamma, Delta, and Omicron. Both the mammalian cell and C1 produced HuMab 87G7 broadly neutralise SARS-CoV-2 VOCs in vitro and also provide protection against VOC Omicron in hamsters. The C1 produced HuMab 87G7 is also able to protect against the Delta VOC in non-human primates. In summary, these findings show that the C1 expression system is a promising technology platform for the development of HuMabs in preventive and therapeutic medicine.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-46443-0" target="_blank">https://www.nature.com/articles/s41467-024-46443-0</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Prebiotic inulin ameliorates SARS-CoV-2 infection in hamsters by modulating the gut microbiome</h3>
            <h3 id="section9"><a href="#section9" style="text-decoration: none; color: inherit;">ü§ñ Inulin improves SARS-CoV-2 outcome in hamsters by adjusting gut bacteria</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that dietary supplementation with the prebiotic inulin improved survival rates and reduced weight loss in hamsters infected with SARS-CoV-2, suggesting potential benefits for COVID-19 prevention through modifying gut microbiome and increasing beneficial short-chain fatty acids.</p>
            <br><strong>Abstract</strong><br>
            <p>Current treatment options for COVID-19 are limited, with many antivirals and immunomodulators restricted to the most severe cases and preventative care limited to vaccination. As the SARS-CoV-2 virus and its increasing variants threaten to become a permanent fixture of our lives, this new reality necessitates the development of cost-effective and accessible treatment options for COVID-19. Studies have shown that there are correlations between the gut microbiome and severity of COVID-19, especially with regards to production of physiologically beneficial short-chain fatty acids (SCFAs) by gut microbes. In this study, we used a Syrian hamster model to study how dietary consumption of the prebiotic inulin affected morbidity and mortality resulting from SARS-CoV-2 infection. After two weeks of observation, we discovered that inulin supplementation attenuated morbid weight loss and increased survival rate in hamster subjects. An analysis of microbiome community structure showed significant alterations in 15 genera. Notably, there were also small increases in fecal DCA and a significant increase in serum DCA, perhaps highlighting a role for this secondary bile acid in conferring protection against SARS-CoV-2. In light of these results, inulin and other prebiotics are promising targets for future investigation as preventative treatment options for COVID-19.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41538-024-00248-z" target="_blank">https://www.nature.com/articles/s41538-024-00248-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Antibody-independent protection against heterologous SARS-CoV-2 challenge conferred by prior infection or vaccination</h3>
            <h3 id="section10"><a href="#section10" style="text-decoration: none; color: inherit;">ü§ñ Prior Infection or Vaccine Provides Protection Against Different COVID-19 Strain Challenge</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-14</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study shows that immunity against SARS-CoV-2 can protect against virus challenge even without antibodies, emphasizing the importance of T cells and interferon-Œ≥ in combating severe infections and managing milder cases, which should be considered in vaccine development.</p>
            <br><strong>Abstract</strong><br>
            <p>Vaccines have reduced severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) morbidity and mortality, yet emerging variants challenge their effectiveness. The prevailing approach to updating vaccines targets the antibody response, operating under the presumption that it is the primary defense mechanism following vaccination or infection. This perspective, however, can overlook the role of T cells, particularly when antibody levels are low or absent. Here we show, through studies in mouse models lacking antibodies but maintaining functional B cells and lymphoid organs, that immunity conferred by prior infection or mRNA vaccination can protect against SARS-CoV-2 challenge independently of antibodies. Our findings, using three distinct models inclusive of a novel human/mouse ACE2 hybrid, highlight that CD8+T cells are essential for combating severe infections, whereas CD4+T cells contribute to managing milder cases, with interferon-Œ≥ having an important function in this antibody-independent defense. These findings highlight the importance of T cell responses in vaccine development, urging a broader perspective on protective immunity beyond just antibodies.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01787-z" target="_blank">https://www.nature.com/articles/s41590-024-01787-z</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Factors associated with the SARS-CoV-2 immunoglobulin-G titer levels in convalescent whole-blood donors: a Chinese cross-sectional study</h3>
            <h3 id="section11"><a href="#section11" style="text-decoration: none; color: inherit;">ü§ñ "Chinese Study on Blood Donor SARS-CoV-2 Antibody Levels"</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-13</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study investigates factors associated with SARS-CoV-2 antibody levels in Chinese convalescent blood donors and suggests that BMI, smoking habits, and exercise frequency may be predictive factors for IgG levels in male donors, potentially aiding in optimizing the screening process for potential blood donors to treat immunocompromised or severely ill COVID-19 patients.</p>
            <br><strong>Abstract</strong><br>
            <p>Blood transfusions from convalescent Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) infected patients could be used to treat patients with severe infections or immunocompromised patients. However, it is necessary to select the optimal donors to maximize the utilization of resources. In this study, we investigated the associations among body mass index (BMI), tobacco smoking, exercise frequency and duration, and alcohol consumption with the SARS-CoV-2 immunoglobulin-G (IgG) antibody titer levels with in the Chinese convalescent blood donor population. Here we show that BMI, smoking habits, and exercise frequency appear to be predictive factors for IgG levels in convalescent male blood donors. However, these variables were not observed as predictive of IgG levels in female convalescent blood donors. The findings could be used to optimize the screening for potential blood donors to treat immunocompromised or severely ill COVID-19 patients.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-56462-y" target="_blank">https://www.nature.com/articles/s41598-024-56462-y</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Exploring cell-free assays for COVID-19 serosurvey</h3>
            <h3 id="section12"><a href="#section12" style="text-decoration: none; color: inherit;">ü§ñ Investigating Cell-Free Tests for COVID-19 Blood Analysis</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-13</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research tested three cell-free methods as potential alternatives to assess SARS-CoV-2 seroprevalence in human populations, finding that they performed equally to the current standard methods and could be used without specialized facilities or personnel.</p>
            <br><strong>Abstract</strong><br>
            <p>Serosurveys to monitor immunity toward COVID-19 in the population are primarily performed using an ELISA to screen samples for SARS-CoV-2 antibodies, followed by confirmation by a virus neutralization test, which is considered the Gold Standard. However, virus neutralization test may not be feasible for some laboratories because of the requirement for specific facilities and trained personnel. In an attempt to address this limitation, we evaluated three cell-free methods as potential alternatives for assessing SARS-CoV-2 seroprevalence in human population from plasma. We report the establishment of two inhibition ELISAs designed to detect anti-Spike RBD IgG antibodies and a microsphere quantitative suspension array technology assay, based on the Luminex xMAP platform, to measure the presence of antibodies against various SARS-CoV-2 antigens, including anti-RBD. These methods were also compared to a commercial chemiluminescent immunoassay designed for anti-RBD antibodies detection and to the combined ELISA‚Äâ+‚Äâvirus neutralization test strategy. These cell-free assays performed equally to estimate the percentage of positive and negative samples and could be used to determine the prevalence of SARS-CoV-2 antibodies in human population, at least in cohort with high-expected prevalence, without the use of seroneutralization assay.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-55852-6" target="_blank">https://www.nature.com/articles/s41598-024-55852-6</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Distinct evolution of SARS-CoV-2 Omicron XBB and BA.2.86/JN.1 lineages combining increased fitness and antibody evasion</h3>
            <h3 id="section13"><a href="#section13" style="text-decoration: none; color: inherit;">ü§ñ Simplified Title: Evolution of Omicron XBB and BA.2.86/JN.1: Fitness and Antibody Evasion in SARS-CoV-2 Variants</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-13</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>In January 2024, several SARS-CoV-2 subvariants, including XBB, BA.2.86.1, and JN.1, account for over 80% of circulating variants; while these variants show different fitness levels and immune evasion properties, existing antivirals remain effective, and vaccines still offer some protection against infections.</p>
            <br><strong>Abstract</strong><br>
            <p>The unceasing circulation of SARS-CoV-2 leads to the continuous emergence of novel viral sublineages. Here, we isolate and characterize XBB.1, XBB.1.5, XBB.1.9.1, XBB.1.16.1, EG.5.1.1, EG.5.1.3, XBF, BA.2.86.1 and JN.1 variants, representing >80% of circulating variants in January 2024. The XBB subvariants carry few but recurrent mutations in the spike, whereas BA.2.86.1 and JN.1 harbor >30 additional changes. These variants replicate in IGROV-1 but no longer in Vero E6 and are not markedly fusogenic. They potently infect nasal epithelial cells, with EG.5.1.3 exhibiting the highest fitness. Antivirals remain active. Neutralizing antibody (NAb) responses from vaccinees and BA.1/BA.2-infected individuals are markedly lower compared to BA.1, without major differences between variants. An XBB breakthrough infection enhances NAb responses against both XBB and BA.2.86 variants. JN.1 displays lower affinity to ACE2 and higher immune evasion properties compared to BA.2.86.1. Thus, while distinct, the evolutionary trajectory of these variants combines increased fitness and antibody evasion.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-46490-7" target="_blank">https://www.nature.com/articles/s41467-024-46490-7</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Inhalation of ACE2-expressing lung exosomes provides prophylactic protection against SARS-CoV-2</h3>
            <h3 id="section14"><a href="#section14" style="text-decoration: none; color: inherit;">ü§ñ Exposure to lung exosomes with ACE2 offers protection from SARS-CoV-2</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-12</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This research explores using exosomes derived from human lung spheroid cells as a potential prophylactic agent to block SARS-CoV-2 infection by neutralizing the virus in both vitro and vivo, demonstrating its effectiveness against various strains and potentially providing broad-spectrum protection against COVID-19.</p>
            <br><strong>Abstract</strong><br>
            <p>Continued emergence of SARS-CoV-2 variants of concern that are capable of escaping vaccine-induced immunity highlights the urgency of developing new COVID-19 therapeutics. An essential mechanism for SARS-CoV-2 infection begins with the viral spike protein binding to the human ACE2. Consequently, inhibiting this interaction becomes a highly promising therapeutic strategy against COVID-19. Herein, we demonstrate that ACE2-expressing human lung spheroid cells (LSC)-derived exosomes (LSC-Exo) could function as a prophylactic agent to bind and neutralize SARS-CoV-2, protecting the host against SARS-CoV-2 infection. Inhalation of LSC-Exo facilitates its deposition and biodistribution throughout the whole lung in a female mouse model. We show that LSC-Exo blocks the interaction of SARS-CoV-2 with host cells in vitro and in vivo by neutralizing the virus. LSC-Exo treatment protects hamsters from SARS-CoV-2-induced disease and reduced viral loads. Furthermore, LSC-Exo intercepts the entry of multiple SARS-CoV-2 variant pseudoviruses in female mice and shows comparable or equal potency against the wild-type strain, demonstrating that LSC-Exo may act as a broad-spectrum protectant against existing and emerging virus variants.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45628-x" target="_blank">https://www.nature.com/articles/s41467-024-45628-x</a></p>
            <br><br>
        </div>
        <div style="margin: 20px; max-width: 1000px; margin-left: auto; margin-right: auto;">
            <h3>Administrative data ICD-10 diagnostic codes identifies most lab-confirmed SARS-CoV-2 admissions but misses many discharged from the Emergency Department</h3>
            <h3 id="section15"><a href="#section15" style="text-decoration: none; color: inherit;">ü§ñ Lab-Confirmed COVID Admissions Identified by ICD-10 Codes, Misses Some Discharged from ED</a></h3><br>
            <p><strong style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); background-clip: text; -webkit-background-clip: text; color: transparent;">Publication Date</strong>: 2024-03-12</p>
            <br>
            <p>ü§ñ <strong>Abstract</strong></p>
            <p>This study found that the ICD-10 code U07.1 effectively identified most lab-confirmed SARS-CoV-2 cases in hospitalized patients but missed a significant number of cases discharged from emergency departments, which should be considered when using administrative data for research and public health planning.</p>
            <br><strong>Abstract</strong><br>
            <p>We estimated the operating characteristics of ICD-10 code U07.1, introduced by the World Health Organization in 2020, to identify lab-confirmed SARS-CoV-2. CCEDRRN is a national research registry of adults (March 2020‚ÄìAugust 2021) with suspected/confirmed SARS-CoV-2 identified in Canadian emergency departments (EDs) using chart review (symptoms, clinical information, and lab test results including SARS-CoV-2 polymerase chain reaction, PCR results). CCEDRRN data were linked to administrative hospitalization discharge and ED ICD-10 diagnostic codes (accessed centrally via the Canadian Institute for Health Information). We identified ICD-10 diagnostic codes in CCEDRRN participants. We defined lab-confirmed SARS-CoV-2 based on at least one positive PCR in the 0‚Äì14¬†days before the ED presentation and/or during hospitalization (in those admitted from ED). We performed separate analyses for CCEDRRN participants discharged from ED and those hospitalized from the ED. Additional analyses were stratified by province, sex, age, and (for hospitalized patients) timing of the first PCR test. The sensitivity of ICD-10 code U07.1 for a positive SARS-CoV-2 test was 93.6% (95% CI 93.0‚Äì94.1%) in those hospitalized from ED and 83.0% (95% CI 82.1‚Äì83.9%) in those discharged from the ED. Sensitivity was similar across provinces and demographics, but in each stratified analysis, values were higher in those hospitalized versus those discharged from ED. The ICD-10 diagnostic code for U07.1 within administrative data identified most lab-confirmed SARS-CoV-2 within persons hospitalized from ED, although a significant number of cases discharged from ED were missed. This should be considered when using administrative data for research and public health planning.</p>
            <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-49501-7" target="_blank">https://www.nature.com/articles/s41598-023-49501-7</a></p>
            <br><br>
        </div>
      
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
     <!-- end footer -->
     <!-- Javascript files--> 
     <script src="js/jquery.min.js"></script> 
     <script src="js/popper.min.js"></script> 
     <script src="js/bootstrap.bundle.min.js"></script> 
     <script src="js/jquery-3.0.0.min.js"></script> 
     <script src="js/plugin.js"></script> 
     <!-- sidebar --> 
     <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
     <script src="js/custom.js"></script>
     <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
     <script>
        $(document).ready(function(){
        $(".fancybox").fancybox({
        openEffect: "none",
        closeEffect: "none"
        });
        
        $(".zoom").hover(function(){
        
        $(this).addClass('transition');
        }, function(){
        
        $(this).removeClass('transition');
        });
        });   
     </script>
     
  </body>
</html>
