
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Research News</title>
    <link rel="icon" href="images/logo.png" type="image/x-icon">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/responsive.css">
    <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
    <style>
        @media (max-width: 767px) {
            .menu-area-main {
                flex-direction: column;
            }
            .menu-area-main li {
                text-align: center;
            }
        }
        .toggle-content { display: none; }
        .toggle-button {
            background-color: #007bff;
            color: white;
            border: none;
            padding: 4px 12px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 14px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
        }
    </style>
</head>
<body class="main-layout">
<header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo">
                                <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a>
                            </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main">
                                    <li><a href="index.html">Geo Map</a></li>
                                    <li class=""> <a href="search_geo.html">Geo Map Search</a> </li>
                                    <li><a href="trends.html">Trend Graphs</a></li>
                                    <li><a href="predictions_tab.html">Predictive</a></li>
                                    <li class="active"><a href="new_research.html">Research News</a></li>
                                    <li><a href="data_sources.html">Data Sources</gita></li>
                                    <li><a href="about.html">About</a></li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>
<div class="brand_color">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="titlepage">
                    <h2>Research News</h2>
                    <br>
                    <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                    Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>
                </div>
            </div>
        </div>
    </div>
</div>
<!-- contact -->
<p>
<br><br><br>

        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-15</strong> <a href="https://www.nature.com/articles/s41541-025-01125-w" target="_blank">
            ü§ñ Outpatient Experimental Infection to Develop Affordable RSV Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-0', this)">
            More</button></h4>
            <div id="toggle-0" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk analysis for outpatient experimental infection as a pathway for affordable RSV vaccine development</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists are working on new ways to test vaccines quickly, using controlled infections that mimic real-life cases. They want to do these tests in doctor's offices instead of hospitals to save money and make things easier for participants.

They studied how well this approach worked with a virus called RSV (which causes respiratory problems) and found that it was safe and only had one transmission event. The study showed that doing vaccines in doctor's offices can be just as effective as doing them in hospitals, with some added benefits like being more convenient for participants.

The researchers propose creating guidelines and safety measures to make this approach work better and faster for vaccine development.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Controlled human infection models (CHIMs) are an important tool for accelerating clinical development of vaccines. CHIM costs are driven by quarantine facilities but may be reduced by performing CHIM in the outpatient setting. Furthermore, outpatient CHIMs offer benefits beyond costs, such as a participant-friendly approach and increased real-world aspect. We analyze safety, logistic and ethical risks of respiratory syncytial virus (RSV) CHIM in the outpatient setting. A review of the literature identified outpatient CHIMs involving respiratory pathogens. RSV transmission risk was assessed using data from our inpatient and outpatient RSV CHIMs (EudraCT 020-004137-21). Fifty-nine outpatient CHIMs using RSV,Streptococcus pneumoniae, rhinovirus, and an ongoingBordetella Pertussisoutpatient CHIM were included. One transmission event was recorded. In an inpatient RSV CHIM, standard droplet and isolation measures were sufficient to limit RSV transmission and no symptomatic third-party transmission was measured in the first outpatient RSV CHIM. Logistic and ethical advantages support outpatient CHIM adoption. We propose a framework for outpatient RSV CHIM with risk mitigation strategies to enhance affordable vaccine development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01125-w" target="_blank">https://www.nature.com/articles/s41541-025-01125-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-15</strong> <a href="https://www.nature.com/articles/s41522-025-00691-0" target="_blank">
            ü§ñ How bacteria use a protein to fight off a type of attack from our immune system.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-1', this)">
            More</button></h4>
            <div id="toggle-1" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bacterial biofilm-derived H-NS protein acts as a defense against Neutrophil Extracellular Traps (NETs)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Bacteria make a special kind of DNA that lives outside their cells, called extracellular DNA (eDNA). As bacteria form colonies, this eDNA helps hold them together and makes it harder for other organisms to attack. Neutrophils are a type of white blood cell that can trap and kill bacteria with the help of eDNA. However, some protein called H-NS can stop neutrophils from doing their job by disabling their ability to form traps and killing free-living bacteria. This makes H-NS a potential target for new treatments to control diseases caused by NETs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Extracellular DNA (eDNA) is crucial for the structural integrity of bacterial biofilms as they undergo transformation from B-DNA to Z-DNA as the biofilm matures. This transition to Z-DNA increases biofilm rigidity and prevents binding by canonical B-DNA-binding proteins, including nucleases. One of the primary defenses against bacterial infections are Neutrophil Extracellular Traps (NETs), wherein neutrophils release their own eDNA to trap and kill bacteria. Here we show that H-NS, a bacterial nucleoid associated protein (NAP) that is also released during biofilm development, is able to incapacitate NETs. Indeed, when exposed to human derived neutrophils, H-NS prevented the formation of NETs and lead to NET eDNA retraction in previously formed NETs. NETs that were exposed to H-NS also lost their ability to kill free-living bacteria which made H-NS an attractive therapeutic candidate for the control of NET-related human diseases. A model of H-NS release from biofilms and NET incapacitation is discussed.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41522-025-00691-0" target="_blank">https://www.nature.com/articles/s41522-025-00691-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-15</strong> <a href="https://www.nature.com/articles/s41598-025-96662-8" target="_blank">
            ü§ñ The impact of health issues on people's interactions with others in Belgium from 2022-2023.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-2', this)">
            More</button></h4>
            <div id="toggle-2" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The role of frailty in shaping social contact patterns in Belgium, 2022‚Äì2023</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Infectious diseases spread when people with respiratory problems come into contact with others. But sometimes, older people or those who are very frail might need special attention when it comes to preventing the spread of disease. This study looked at how people in Flanders, Belgium interacted with each other over a year. We asked 6,000 people about their contacts and found that some were more frail than others. The study used this information to create a model that shows how frailty affects the spread of disease. By understanding this better, we might be able to make non-medical interventions more effective in controlling outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Social contact data are essential for understanding the spread of respiratory infectious diseases and designing effective prevention strategies. However, many studies often overlook the heterogeneity in mixing patterns among older age groups and individual frailty levels, assuming homogeneity across these sub-populations. This shortcoming may undermine non-pharmaceutical interventions by not targeting specific contact behaviours, potentially reducing their effectiveness in controlling disease. To address this gap, we conducted a contact survey in Flanders, Belgium (June 2022‚ÄìJune 2023). We collected data from 5995 participants (overall response rates of 19.34%) who recorded 31,375 contacts with distinct individuals. Within this cohort, 14.50% were classified as frail, and 46.85% were classified as non-frail. On average, participants report 5.48 contacts daily, with a median of 4 contacts (IQR: 2‚Äì7). These contacts vary based on participants‚Äô age and frailty levels, influenced by the locations of their interactions. Using the collected data, we reconstructed frailty-dependent contact matrices and developed a contact-based mathematical model that integrates participants‚Äô and contactees‚Äô frailty levels to investigate how frailty levels affect transmission dynamics. Incorporating frailty levels into the mathematical model substantially alters the shape of epidemic curves and peak incidences. Such insights might provide useful insights for informing non-pharmaceutical interventions, indicating the potential benefit of similar data collection in different countries.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96662-8" target="_blank">https://www.nature.com/articles/s41598-025-96662-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-15</strong> <a href="https://www.nature.com/articles/s41598-025-97704-x" target="_blank">
            ü§ñ Air Pollution Effects on Mortality in Portugal</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-3', this)">
            More</button></h4>
            <div id="toggle-3" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessment of air pollution and mortality in Portugal using AirQ+ and the effects of COVID-19 on their relationship</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

This study looked at how air pollution affects people in Portugal from 2010 to 2021, especially during the COVID-19 pandemic. The researchers used special computer model to compare long-term air quality data with health effects. They found that because of fewer cars on the road and less pollution, there was a big decrease in bad air particles like nitrogen dioxide and tiny particles. However, people were still exposed to some good air pollutants called ozone, which made respiratory problems worse.

The study also estimated how many deaths can be prevented by reducing air pollution. The results showed that with fewer people dying from air pollution-related diseases, the pandemic actually helped reduce deaths from these causes. But the researchers say more work is needed to fully understand the impact of air pollution and to make sure we're doing everything we can to keep our air clean.

This study shows how important it is to use computer models like AirQ+ to help plan public health strategies and track progress towards making the world a healthier place.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study uses the World Health Organization‚Äôs AirQ+ model to assess the relationship between air pollution and mortality in Portugal from 2010 to 2021, focusing on the impact of the COVID-19 pandemic. By integrating AirQ+ with Linear Mixed Models, we analyzed long-term air pollution data and its health effects. Results indicate a significant decrease in\({\text{NO}}_{2}\)and\({\text{PM}}_{2.5}\)concentrations in 2020 and 2021 due to COVID-19 restrictions and reduced transportation emissions. Conversely,\({\text{O}}_{3}\)exposure slightly increased. The model estimates over 5000 annual deaths from\({\text{NO}}_{2}\)and\({\text{PM}}_{2.5}\)exposure and over 139 annual deaths from\({\text{O}}_{3}\)-related respiratory diseases for 2010‚Äì2021. Despite limitations like the need for better assessment of pollutant mixtures and climatic variables, the study shows a decrease in\({\text{NO}}_{2}\)-related disease burden during the pandemic. These trends reflect anomalies in mortality and pollution data rather than policy improvements. The study underscores the utility of AirQ+ in guiding public health strategies and tracking progress toward the 2030 Agenda, offering insights into reducing mortality and morbidity through decreased air pollutant exposure and highlighting the need for sustained, multidimensional pollution reduction efforts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-97704-x" target="_blank">https://www.nature.com/articles/s41598-025-97704-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-14</strong> <a href="https://www.nature.com/articles/s41392-025-02208-9" target="_blank">
            ü§ñ COVID-19 Treatment Trial: New Medicine Tested</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-4', this)">
            More</button></h4>
            <div id="toggle-4" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Meplazumab, a CD147 antibody, for severe COVID-19: a double-blind, randomized, placebo-controlled, phase 3 clinical trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists tested a new medicine called meplazumab to see if it can help people who are very sick with COVID-19. The study showed that meplazumab can help reduce death and improve symptoms in these patients. It worked best for people who smoke, and the results were also positive when looked at in more detail. Meplazumab was safe and well-tolerated, meaning it didn't cause many side effects. This is a promising discovery that could lead to new treatments for severe COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Meplazumab, a humanized CD147 antibody, showed favorable safety and clinical benefits in phase 1 and phase 2/3 seamless clinical studies. Further evaluation of its therapeutic efficacy in patients with severe COVID-19 is needed. In this phase 3 add-on study, we randomized patients with severe COVID-19 in a 1:1 ratio to receive 0.2‚Äâmg/kg meplazumab or placebo via intravenous injection, and evaluated efficacy and safety within 56 days. Between February 2023 and November 2023, 108 patients with severe COVID-19 were randomized to two groups, with their baseline characteristics generally balanced. The primary endpoint, 28-day all-cause mortality was 1.96% in the meplazumab group vs 7.69% in the placebo group (P=‚Äâ0.1703). Supplementary analysis using composite strategy indicated a significant reduction of 28-day all-cause mortality in meplazumab compared to placebo (3.92% vs 15.38%,P=‚Äâ0.044). Meplazumab also significantly reduced the mortality in smoking subjects on day 28 (P=‚Äâ0.047) compared to placebo in supplementary analysis. The secondary endpoint, 56-day all-cause mortality, was 1.96% in the meplazumab group and 11.54% in the placebo group (P=‚Äâ0.048), which was 3.92% and 15.38%, respectively (P=‚Äâ0.044) by supplementary analysis. Additional secondary endpoints showed potential benefits, including increased hospital discharge rates, improved clinical outcomes, and improved viral nucleotide conversion rate. Meplazumab demonstrated good safety and tolerability, with no grade‚Äâ‚â•‚Äâ3 TEAEs observed. These promising results indicate that meplazumab reduces mortality and enhances clinical benefits in severe COVID-19 patients with a good safety profile, providing effective and specific therapeutics for severe COVID-19 (the trial was registered at ClinicalTrials.gov (NCT05679479)).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-025-02208-9" target="_blank">https://www.nature.com/articles/s41392-025-02208-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-14</strong> <a href="https://www.nature.com/articles/s41598-025-96279-x" target="_blank">
            ü§ñ SARS-CoV-2 infection reduces certain genes involved in immune response, which improves as the body recovers.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-5', this)">
            More</button></h4>
            <div id="toggle-5" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>ACE2 and TMPRSS2 gene expression is reduced acutely in SARS-CoV-2 patients but returns to normal with recovery</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler language:

During the SARS-CoV-2 pandemic, researchers found that a protein called ACE2 and an enzyme called TMPRSS2 play important roles in helping the virus enter cells. But previous studies have shown mixed results about whether having high levels of these proteins makes you more or less likely to get infected. To learn more, scientists looked at swabs from people who were infected with SARS-CoV-2 and compared them to their family members who weren't infected. They found that the levels of ACE2 and TMPRSS2 decreased when people were first getting sick, but went back up after they recovered. This suggests that there are other factors at play that might make some people more or less likely to get infected with SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the SARS-CoV-2 pandemic, angiotensin-converting enzyme 2(ACE2)was identified as the major entry receptor for the virus and transmembrane serine protease2 (TMPRSS2)as an important SARS-CoV-2 entry factor. Previous studies investigating the impact ofACE2andTMPRSS2gene expression on SARS-CoV-2 susceptibility in adults have yielded inconsistent results, thereby underscoring the need for further research in this domain. We obtained nasopharyngeal swabs from infected adults during the acute and late convalescent phase of SARS-CoV-2 infection and compared the expression of both genes with non-infected household member contacts. We found thatACE2andTMPRSS2gene expression is temporarily reduced during the acute phase of SARS-CoV-2 infection presumably due to viral disruption of transcription. Post-recovery, however, the expression ofACE2andTMPRSS2was comparable to non-infected household contacts. The lack of significant differences in ACE2 and TMPRSS2 gene expression between SARS-CoV-2-positive adults and uninfected household controls suggests that factors influencing susceptibility to SARS-CoV-2 infection in adults may extend beyond ACE2 and TMPRSS2 expression. However, the findings should be interpreted with caution due to the study‚Äôs limited sample size and the heterogeneity of COVID-19 cases. Other non-physiological factors, such as enhanced hygiene practices following the infection of a household member, may also contribute to absence of infection among healthy controls.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96279-x" target="_blank">https://www.nature.com/articles/s41598-025-96279-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-14</strong> <a href="https://www.nature.com/articles/s41598-025-96161-w" target="_blank">
            ü§ñ COVID-19 Disease Severity Predicted by Hormone and Immune System Imbalance in South African People</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-6', this)">
            More</button></h4>
            <div id="toggle-6" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Integrated renin angiotensin system dysregulation and immune profiles predict COVID-19 disease severity in a South African cohort</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The renin-angiotensin system plays a key role in how our bodies respond to coronavirus infections. A protein called angiotensin-converting enzyme 2 (ACE2) helps the virus enter our cells, and its activity is linked to the severity of the disease. When people get more severe symptoms, their ACE1 activity actually goes down, while ACE2 activity increases.

Researchers studied a group of over 170 people who had coronavirus in Cape Town. They found that lower ACE1 activity was associated with higher mortality rates, while higher ACE2 activity was linked to worse severity. These findings suggest that measuring ACE1 and ACE2 activities could help predict the severity of the disease and identify potential treatments.

The researchers also discovered other biomarkers, such as HGF and EN-RAGE, which are linked to the severity of the disease. Adding these markers to ACE1 activity improved the accuracy of predicting whether someone would be ambulatory or hospitalized.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Renin-angiotensin system (RAS) dysregulation is an important component of the complex pathophysiology of SARS-CoV-2 and other coronavirus infections. Thus, angiotensin-converting enzyme 2 (ACE2), the entry receptor and key to the alternative RAS, was proposed as a severity/prognostic biomarker for risk-stratification. However, experimental RAS data from diverse cohorts are limited, particularly analyses integrating RAS with immune biomarkers. Participants (n=‚Äâ172) in Cape Town were sampled longitudinally (including a recovery timepoint [>‚Äâ3-month]), across WHO asymptomatic to critical severity. Using fluorometric assays and LC-MS/MS RAS Fingerprinting¬Æ, results show serum ACE1 activity significantly decreases with increasing COVID-19 severity (P<‚Äâ0.01) and mortality (P<‚Äâ0.05), while increased ACE2 activity is associated with worse severity (P<‚Äâ0.01). Neither enzyme activity correlates with viral load proxy or nasal ACE mRNA levels. ACE1 and ACE2 activities were the most effective severity biomarkers compared to 96 established immune markers obtained via proximity extension assay, as demonstrated by principal component analysis. A multivariate variable selection model using random forest classification identified biomarkers discriminating COVID-19 severity (AUC‚Äâ=‚Äâ0.82), the strongest being HGF, EN-RAGE, cathepsin L. Adding ACE1 activity and anti-SARS-CoV-2 antibody titres improved differentiation between ambulatory and hospitalised participants. Notably, RAS dysregulation has unique severity associations in coronavirus infections with implications for treatment and pathophysiological mechanisms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96161-w" target="_blank">https://www.nature.com/articles/s41598-025-96161-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-14</strong> <a href="https://www.nature.com/articles/s41541-025-01103-2" target="_blank">
            ü§ñ Understanding how the COVID-19 virus triggers an immune response that protects against infection.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-7', this)">
            More</button></h4>
            <div id="toggle-7" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A systems biology approach to define SARS-CoV-2 correlates of protection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young person can understand this study by thinking of it like a super powerful computer program that helps doctors figure out why some people get sick from COVID-19 and others don't.

Researchers used a special computer tool to look at data from 90 monkeys who got infected with COVID-19. They wanted to know what makes someone immune to the virus, so they looked for clues in the monkeys' bodies.

They found two main things: first, that having antibodies against the virus's spike protein can help protect people from getting sick; second, that having a certain type of immune response can also help. 

But here's the important part: it doesn't mean that having more antibodies will always keep you safe. Sometimes, even if you have lots of antibodies, you can still get sick.

This study is helping doctors understand how to make better vaccines and treatments for COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Correlates of protection (CoPs) for SARS-CoV-2 have yet to be sufficiently defined. This study uses the machine learning platform, SIMON, to accurately predict the immunological parameters that reduced clinical pathology or viral load following SARS-CoV-2 challenge in a cohort of 90 non-human primates. We found that anti-SARS-CoV-2 spike antibody and neutralising antibody titres were the best predictors of clinical protection and low viral load in the lung. Since antibodies to SARS-CoV-2 spike showed the greatest association with clinical protection and reduced viral load, we next used SIMON to investigate the immunological features that predict high antibody titres. It was found that a pre-immunisation response to seasonal beta-HCoVs and a high frequency of peripheral intermediate and non-classical monocytes predicted low SARS-CoV-2 spike IgG titres. In contrast, an elevated T cell response as measured by IFNŒ≥ ELISpot predicted high IgG titres. Additional predictors of clinical protection and low SARS-CoV-2 burden included a high abundance of peripheral T cells. In contrast, increased numbers of intermediate monocytes predicted clinical pathology and high viral burden in the throat. We also conclude that an immunisation strategy that minimises pathology post-challenge did not necessarily mediate viral control. This would be an important finding to take forward into the development of future vaccines aimed at limiting the transmission of SARS-CoV-2. These results contribute to SARS-CoV-2 CoP definition and shed light on the factors influencing the success of SARS-CoV-2 vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01103-2" target="_blank">https://www.nature.com/articles/s41541-025-01103-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-13</strong> <a href="https://www.nature.com/articles/s43856-025-00840-8" target="_blank">
            ü§ñ Here's a simplified title:

Developing a Personalized Measure of Vaccine Side Effects Using Wearable Technology.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-8', this)">
            More</button></h4>
            <div id="toggle-8" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Development of a personalized digital biomarker of vaccine-associated reactogenicity using wearable sensors and digital twin technology</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

When people get vaccinated, their body reacts with some symptoms like headaches or fatigue. But it's hard to measure these symptoms because they can vary from person to person. To solve this problem, researchers created a special tool that uses sensors on the skin to track how a person's body changes after getting vaccinated.

They wore special wristbands that tracked tiny changes in their body and compared them to what was normal for each person before they got vaccinated. This helped them identify what made people react differently to vaccines.

The team found that this new method can predict how much someone will react to a vaccine better than any other way, and it even helps doctors understand why some vaccines work better on certain people.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundEffective response to vaccination requires activation of the innate immune system, triggering the synthesis of inflammatory cytokines. The degree of subjective symptoms related to this, referred to as reactogenicity, may predict their eventual immune response. However, the subjective nature of these symptoms is influenced by the nocebo effect, making it difficult to accurately quantify a person‚Äôs physiologic response. The use of wearable sensors allows for the identification of objective evidence of physiologic changes a person experiences following vaccination, but as these changes are subtle, they can only be detected when an individual‚Äôs pre-vaccination normal variability is considered.MethodsWe use a wearable torso sensor patch and a machine learning method of similarity-based modeling (SBM) to create a physiologic digital twin for 88 people receiving 104 COVID vaccine doses. By using each individual‚Äôs pre-vaccine digital twin, we are able to effectively control for expected physiologic variations unique to that individual, leaving only vaccine-induced differences. We use these individualized differences between the pre- and post-vaccine period to develop a multivariate digital biomarker for objectively measuring the degree and duration of physiologic changes each individual experiences following vaccination.ResultsHere we show that the multivariate digital biomarker better predicted systemic reactogenicity than any one physiologic data type and correlated with vaccine-induced changes in humoral and cellular immunity in a 20-person subset.ConclusionsA digital biomarker is capable of objectively identifying an individual‚Äôs unique response to vaccination and could play a future role in personalizing vaccine regimens.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00840-8" target="_blank">https://www.nature.com/articles/s43856-025-00840-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-13</strong> <a href="https://www.nature.com/articles/s43856-025-00846-2" target="_blank">
            ü§ñ Long-term health effects of COVID-19 in France</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-9', this)">
            More</button></h4>
            <div id="toggle-9" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A multidimensional network of factors associated with long COVID in the French population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who get long COVID have many different causes. To understand why some people get it and how to prevent it, scientists looked at what happens to adults in France after they got sick with Covid-19.

They asked a group of 1813 people about their symptoms and things that might make them more or less likely to get long COVID. They found out that certain things can increase the risk, such as being female, having health problems before, or getting vaccinated against Covid-19.

This means that long COVID is not just caused by the virus itself, but also by other factors like your age, health, and how well you understand what's happening with long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundVarious factors associated with long COVID have been evidenced, but the heterogeneity of definitions and epidemiological investigations has often hidden risk pathways relevant for understanding and preventing this condition.MethodsThis nationwide random sampling survey conducted in France after the Omicron waves in autumn 2022 assessed eight sets of factors potentially associated with long COVID in a structured epidemiological investigation based on a conceptual model accounting for the relationships between these sets of factors. A representative sample of 1813 adults of whom 55% were infected with SARS-CoV-2 was assessed for infection dates and context, post-COVID symptoms and these factors. Four definitions of long COVID, including the World Health Organisation‚Äôs, were used.ResultsFemale sex, household size (‚â•2), low financial security, negative impact of COVID-19 pandemic on occupation and work conditions, number of comorbidities (‚â•2), presence of respiratory disease, mental and sensory disorders, number of SARS-CoV-2 infections (‚â•2) and initial symptoms (‚â•6), perceived high severity of COVID-19 are positively and consistently associated with long COVID. Age‚Äâ‚â•‚Äâ75 years, retirement, SARS-CoV-2 vaccination (‚â•2 doses) and good perceived information regarding long Covid are negatively associated with the condition.ConclusionsThe broad spectrum of factors confirmed here strongly suggests that long COVID should be regarded not only as a direct complication of SARS-CoV-2 infection but also as driven by a broader network of contextual, medical, psychological and social factors. These factors should be better considered in strategies aimed at limiting the long COVID burden in the general population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00846-2" target="_blank">https://www.nature.com/articles/s43856-025-00846-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-12</strong> <a href="https://www.nature.com/articles/s41598-025-95735-y" target="_blank">
            ü§ñ Niacin treatment may help with some infections in animals, but more research is needed to confirm this.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-10', this)">
            More</button></h4>
            <div id="toggle-10" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Meta-analysis of niacin and NAD metabolite treatment in infectious disease animal studies suggests benefit but requires confirmation in clinically relevant models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

When our bodies get sick, they need help fighting off the infection. Some studies have shown that giving people or animals something called niacin (a form of vitamin B3) or a related substance can help make their bodies better at defending against infections and reduce damage to organs.

Researchers looked at 56 animal studies that tested whether niacin or these substances could be used to treat infections in animals. They found that in some cases, the animals that received niacin or the substances showed less organ damage and more antioxidants in their blood and tissues.

However, these results don't necessarily mean that niacin or these substances can be safely used in humans to treat infections. More research is needed to see if they work as well in people as they do in animals, and to make sure the studies are done properly so we know for sure what we're looking at.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Disruption of nicotinamide adenine dinucleotide (NAD) biosynthesis and function during infection may impair host defenses and aggravate inflammatory and oxidative organ injury. Increasingly, studies are investigating whether niacin or NAD metabolite treatment is beneficial in infection and sepsis animal models. We examined whether this preclinical experience supports clinical trials. A systematic review of three data bases was conducted through 2/29/2024 and a meta-analysis was performed comparing niacin or NAD metabolite treatment to control in adult animal models employing microbial challenges. Fifty-six studies met inclusion criteria, with 24 published after 2019. Most studies employed mouse (n‚Äâ=‚Äâ40 studies) or rat (n‚Äâ=‚Äâ12) models and administered either a bacterial toxin (n‚Äâ=‚Äâ28) or bacterial (n‚Äâ=‚Äâ19) challenge. Four and three studies employed viral or fungal challenges respectively. Studies investigated an NAD metabolite alone (n‚Äâ=‚Äâ44), niacin alone (n‚Äâ=‚Äâ9), or both (n‚Äâ=‚Äâ3), usually administered before or within 24h after challenge (n‚Äâ=‚Äâ50). Only three and four studies included standard antimicrobial support or started treatment‚Äâ>‚Äâ24h after challenge respectively. In similar patterns with differing animal types (p‚Äâ‚â•‚Äâ0.06), compared to control across those studies investigating the parameter, niacin or NAD treatment decreased the odds ratio of mortality [95% confidence interval (CI)] [0.28 (0.17, 0.49)] and in blood or tissue increased antioxidant levels [standardized mean differences (95%CI)] (SMD) [3.61 (2.20,5.02)] and decreased levels of microbes [‚àí¬†2.44 (‚àí¬†3.34, ‚àí¬†1.55)], histologic and permeability organ injury scoring [‚àí¬†1.62 (‚àí¬†2.27, ‚àí¬†0.98) and ‚àí¬†1.31(‚àí¬†1.77, ‚àí¬†0.86) respectively], levels of TNFŒ±, IL-6 and IL-1Œ≤ [‚àí¬†2.47 (‚àí¬†3.30, ‚àí¬†1.64), ‚àí¬†3.17 (‚àí¬†4.74, ‚àí¬†1.60) and ‚àí¬†8.44 (‚àí¬†12.4, ‚àí¬†4.5) respectively] and myeloperoxidase (MPO) [‚àí¬†1.60 (‚àí¬†2.06, ‚àí¬†1.14)], although with significant, primarily quantitative heterogeneity for each (I2‚â•‚Äâ53%, p‚Äâ<‚Äâ0.01) except MPO. Treatment increased blood or tissue NAD+levels and decreased chemical organ injury measures and oxidation markers but differently comparing species (p‚Äâ‚â§‚Äâ0.05). Only 2 and 9 survival studies described power analyses or animal randomization respectively and no study described treatment or non-histologic outcome measure blinding. Among survival studies, Egger‚Äôs analysis (p‚Äâ=‚Äâ0.002) suggested publication bias. While suggestive, published animal studies do not yet support clinical trials testing niacin and NAD metabolite treatment for infection and sepsis. Animal studies simulating clinical conditions and with randomized, blinded designs are needed to investigate this potentially promising therapeutic approach.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95735-y" target="_blank">https://www.nature.com/articles/s41598-025-95735-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-12</strong> <a href="https://www.nature.com/articles/s41598-025-97198-7" target="_blank">
            ü§ñ Association Between Emergency ICU Overcrowding and Organ Donation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-11', this)">
            More</button></h4>
            <div id="toggle-11" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Association of emergency intensive care unit occupancy due to brain-dead organ donors with ambulance diversion</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We studied how having brain-dead patients in the intensive care unit (ICU) affects emergency ambulance visits. From 2021 to 2023, we looked at days when brain-dead patients were in the ICU and compared them to other days. We found that when brain-dead patients were in the ICU, it stayed occupied for longer periods, which led to more ambulance visits. This means that managing brain-dead donors in the ICU can cause more problems for emergency services. Our study shows that finding ways to balance taking care of these patients with keeping the ICU available for other emergencies is very important.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our study aims to explore how intensive care unit (ICU) occupancy by brain-dead organ donors affects emergency ambulance diversions. In this retrospective, single-center study at an emergency ICU (EICU), brain-dead organ donors were managed until organ procurement. We classified each day between August 1, 2021, and July 31, 2023, as either an exposure day (any day with a brain-dead organ donor in the EICU from admission to organ procurement) or a control day (all other days). The study compared these days and used multiple logistic regression analysis to assess the impact of EICU occupancy by brain-dead organ donors on ambulance diversions. Over two years, 6,058 emergency patients were transported by ambulance, with 1327 admitted to the EICU, including 13 brain-dead organ donors. Brain-dead donors had longer EICU stays (17 vs. 2¬†days,P<‚Äâ0.001). With 168 exposure and 562 control days, EICU occupancy was higher on exposure days (75% vs. 67%,P=‚Äâ0.003), leading to more ambulance diversions. Logistic regression showed exposure days significantly increased ambulance diversions, with an odds ratio of 1.79 (95% CIs 1.10‚Äì2.88). This study shows that managing brain-dead organ donors in the EICU leads to longer stays and higher occupancy, resulting in more frequent ambulance diversions. These findings highlight the critical need for policies that optimize ICU resource allocation while maintaining the infrastructure necessary to support organ donation programs and ensuring continued care for brain-dead donors, who play an essential role in addressing the organ shortage crisis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-97198-7" target="_blank">https://www.nature.com/articles/s41598-025-97198-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-12</strong> <a href="https://www.nature.com/articles/s41598-025-92475-x" target="_blank">
            ü§ñ Coronavirus found in wild animals in Cambodia may be re-infected into humans.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-12', this)">
            More</button></h4>
            <div id="toggle-12" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Study of coronavirus diversity in wildlife in Northern Cambodia suggests continuous circulation of SARS-CoV-2-related viruses in bats</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the rewritten abstract:

Since COVID-19 started, scientists have been studying coronaviruses that live in bats in Cambodia. This study looked at how many different types of coronaviruses were found in bats and other animals in two provinces where wildlife trade happens often. We collected samples from 2020 to 2022 using special tools to catch the viruses without hurting the animals. We tested over 2600 samples and found that about 2% had coronaviruses inside them. The most common types of coronaviruses were found in bats, but also in some rodents. Some bat viruses are related to COVID-19. This study shows that bats can carry many different types of coronaviruses and that they might be a source of new diseases. We need to keep watching for these viruses to stop them from spreading to humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Since SARS-CoV-2‚Äôs emergence, studies in Southeast Asia, including Cambodia, have identified related coronaviruses (CoVs) in rhinolophid bats. This pilot study investigates the prevalence and diversity of CoVs in wildlife from two Cambodian provinces known for wildlife trade and environmental changes, factors favoring zoonotic spillover risk. Samples were collected from 2020 to 2022 using active (capture and swabbing of bats and rodents) and non-invasive (collection of feces from bat caves and wildlife habitats) methods. RNA was screened for CoVs using conventional pan-CoVs and real-time Sarbecovirus-specific PCR systems. Positive samples were sequenced and phylogenetic analysis was performed on the partial RdRp gene. A total of 2608 samples were collected: 867 rectal swabs from bats, 159 from rodents, 41 from other wild animals, and 1541 fecal samples. The overall prevalence of CoVs was 2.0%, with a 3.3% positive rate in bats, 2.5% in rodents, and no CoVs detected in other wildlife species. Alpha-CoVs were exclusive to bats, while Beta-CoVs were found in both bats and rodents. Seven SARS-CoV-2-related viruses were identified inRhinolophus shamelibats sampled in August 2020, March 2021, and December 2021. Our results highlight diverse CoVs in Cambodian bats and rodents and emphasize bats as significant reservoirs. They also suggest continuous circulation of bat SARS-CoV-2-related viruses may occur in a region where ecological and human factors could favor virus emergence. Continuous surveillance and integrated approaches are crucial to managing and mitigating emerging zoonotic diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92475-x" target="_blank">https://www.nature.com/articles/s41598-025-92475-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-11</strong> <a href="https://www.nature.com/articles/s41598-025-96452-2" target="_blank">
            ü§ñ Impact of Early COVID-19 Pandemic on Mental Health in Serbia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-13', this)">
            More</button></h4>
            <div id="toggle-13" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Emotional distress during the first year of the COVID-19 pandemic-large sample serbian study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

The COVID-19 pandemic has caused big problems with people's feelings and mental well-being. This study looked at how it affected people in Serbia, especially doctors who took care of others. It found that some groups were more likely to feel stressed or anxious, such as women, doctors who weren't psychiatrists, and those who didn't have a good life outside of work. Doctors who worked hard and under tough conditions also had a higher risk of mental health problems. This means we need to help these people the most, especially doctors, so they can stay healthy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has significantly impacted mental health, with healthcare professionals being particularly vulnerable. This study explores the emotional distress experienced in Serbia, focusing on factors that contribute to symptoms of depression, anxiety, and stress. A total of 3328 participants from the general and healthcare populations were surveyed using the Depression Anxiety Stress Scales‚ÄîShort Form (DASS-21). The analysis revealed that certain groups, such as women, non-psychiatrist doctors, individuals with a history of mental disorders, and those dissatisfied with their financial, familial, and living circumstances, were more likely to experience heightened levels of emotional stress. Moreover, healthcare professionals, especially those working in challenging conditions, may be at increased risk for mental health issues. These findings underscore the need for targeted interventions and ongoing support for those most affected, particularly within the healthcare sector, to mitigate the long-term mental health impacts of the pandemic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96452-2" target="_blank">https://www.nature.com/articles/s41598-025-96452-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-11</strong> <a href="https://www.nature.com/articles/s41467-025-56284-0" target="_blank">
            ü§ñ SARS-CoV-2 Long-Term Health Effects in Children and Teens</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-14', this)">
            More</button></h4>
            <div id="toggle-14" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cardiovascular post-acute sequelae of SARS-CoV-2 in children and adolescents: cohort study using electronic health records</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

Children and teenagers who had COVID-19 are more likely to get heart problems later on. A big study looked at how many people with COVID-19 got heart issues compared to those who never got COVID-19. They found that kids and teens with pre-existing heart problems were even more likely to develop new heart issues after having COVID-19. This means doctors need to keep a close eye on these patients for long-term heart health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The risk of cardiovascular outcomes following SARS-CoV-2 infection has been reported in adults, but evidence in children and adolescents is limited. This paper assessed the risk of a multitude of cardiac signs, symptoms, and conditions 28-179 days after infection, with outcomes stratified by the presence of congenital heart defects (CHDs), using electronic health records (EHR) data from 19 children‚Äôs hospitals and health institutions from the United States within the RECOVER consortium between March 2020 and September 2023. The cohort included 297,920 SARS-CoV-2-positive individuals and 915,402 SARS-CoV-2-negative controls. Every individual had at least a six-month follow-up after cohort entry. Here we show that children and adolescents with prior SARS-CoV-2 infection are at a statistically significant increased risk of various cardiovascular outcomes, including hypertension, ventricular arrhythmias, myocarditis, heart failure, cardiomyopathy, cardiac arrest, thromboembolism, chest pain, and palpitations, compared to uninfected controls. These findings were consistent among patients with and without CHDs. Awareness of the heightened risk of cardiovascular disorders after SARS-CoV-2 infection can lead to timely referrals, diagnostic evaluations, and management to mitigate long-term cardiovascular complications in children and adolescents.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56284-0" target="_blank">https://www.nature.com/articles/s41467-025-56284-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-11</strong> <a href="https://www.nature.com/articles/s43856-025-00830-w" target="_blank">
            ü§ñ Affecting the way the immune system responds to COVID vaccine changes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-15', this)">
            More</button></h4>
            <div id="toggle-15" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modifying the glycosylation profile of SARS-CoV-2 spike-based subunit vaccines alters focusing of the humoral immune response in a mouse model</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Vaccines made from proteins are very effective at preventing diseases. But did you know that tiny sugar molecules attached to these proteins can affect how well the vaccine works? Researchers tested different types of sugar molecules on a specific virus (SARS-CoV-2) and found that some types of sugar make the vaccine much stronger, while others don't work as well.

The team used a special way to attach sugar molecules to the virus protein, called glycosylation. They then gave the vaccines to mice and tested how well they worked. The results showed that:

* Some vaccines with certain types of sugar molecules made the mice's immune systems produce lots of antibodies.
* Vaccines with other types of sugar molecules didn't work as well.

The researchers think that understanding how sugar molecules affect vaccine effectiveness could help make better vaccines in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundProtein subunit vaccines have a strong track record of efficacy and safety and have been widely applied for prevention of a variety of infectious diseases. The impacts of post-translational modifications of vaccine antigens are often overlooked, despite the fact that they can vary significantly depending on the expression hosts (e.g., bacteria, yeast, plant, insect or mammalian cells) and the culture conditions used for their manufacturing.MethodsUsing SARS-CoV-2 spike trimers as model antigens, we sought to evaluate the immunological impact of modulating their state of glycosylation. Spike proteins rich in complex-type (CT), high-mannose (HM) or paucimannose (PM) N-linked glycans were produced using Chinese Hamster Ovary (CHO) cells (cultured with or without the mannosidase inhibitor kifunensine) or insect cells.ResultsHere we show that when these antigens are adjuvanted with liposomes composed of sulfated lactosyl archaeol (SLA), all glycoforms are highly immunogenic and induce abundant spike-specific serum IgG and IFN-Œ≥ producing T-cells within female C57BL/6 mice. The spike antigen with CT glycans induces a significantly more potent neutralizing immune response, which directly correlates to more abundant receptor binding domain (RBD)-specific IgG when comparing to the antigen with HM glycans. This observation remains true whether the spike is resistin- or T4 foldon-trimerized, indicating that the glycosylation effect is not trimerization domain-specific. Spike with PM glycans induces remarkably low titers of neutralizing antibodies and RBD-specific IgG.ConclusionsThe results highlight the significant impacts of a vaccine‚Äôs antigen glycosylation profile in directing the immune response, which should be an important consideration for designing efficient protein-based vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00830-w" target="_blank">https://www.nature.com/articles/s43856-025-00830-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-11</strong> <a href="https://www.nature.com/articles/s42003-025-07999-9" target="_blank">
            ü§ñ Dengue Fever: How Climate and Other Factors Affect Its Spread in Thailand</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-16', this)">
            More</button></h4>
            <div id="toggle-16" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Climate, inter-serotype competition and arboviral interactions shape dengue dynamics in Thailand</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Dengue fever is spreading rapidly around the world, with Thailand being a particularly important place to understand how it works.

Normally, when dengue outbreaks happen, they are all about one type of the virus, but in Thailand, all four types of the virus can coexist and affect people at the same time. This happens because the climate and mosquitoes there make it an ideal place for all four types of the virus to survive and multiply.

The virus changes over time due to a natural process called "inter-serotype competition", where the different types of the virus compete with each other, leading to some becoming stronger and more contagious while others weaken. This creates new versions of the virus that can spread more easily.

The good news is that if we understand how the dengue virus works in Thailand, we can also use this knowledge to stop it from spreading to other countries and to prevent similar outbreaks from happening elsewhere.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The incidence and global spread of dengue are reaching alarming levels. Thailand represents a critical disease epicenter and demands an understanding of the environmental and evolutionary pressures that sustain DENV transmission. Unlike most affected countries experiencing recurrent outbreaks of the same serotype or replacement of one serotype for another, Thailand is an ecological niche for all four serotypes. Favorable climate and mosquito vector availability maintain a landscape defined by stable, endemic circulation of genotypes, with minimal genetic variation attributed to sporadic, external introductions. This equilibrium is achieved through inter-serotype competition, characterized by reproductive fitness levels that maintain infections (Re>1) and elevated evolutionary rates (‚Äâ~‚Äâ10-4), which steadily increase the genetic diversity of each serotype. This conclusion is reinforced by the identification of numerous positively selected mutations, skewed in the direction of non-structural proteins conferring replication and transmission advantages versus those present in structural proteins evading neutralizing antibodies. Precipitous drops in DENV cases following outbreaks of Chikungunya suggest that interactions with other arboviruses also impact DENV dynamics through vector competition, replication inhibition or partial cross-protection. Thailand is a major exporter of DENV cases and novel emergent lineages gaining fitness here are likely to spread internationally. Surveillance is therefore paramount to monitor diversification trends and take measures to avoid the establishment of similar sustained, local transmission in other countries.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07999-9" target="_blank">https://www.nature.com/articles/s42003-025-07999-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-11</strong> <a href="https://www.nature.com/articles/s43856-025-00827-5" target="_blank">
            ü§ñ Comparing How Chronic Illnesses are Recorded in Electronic Health Records</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-17', this)">
            More</button></h4>
            <div id="toggle-17" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identifying commonalities and differences between EHR representations of PASC and ME/CFS in the RECOVER EHR cohort</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Many people who got COVID-19 still feel very sick afterwards (PASC) and some people also have a condition called myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS). They might share similar symptoms like tiredness, breathing problems, and brain fog. To understand how these two conditions are related, researchers looked at data from 6.5 million adults who got COVID-19 or were diagnosed with ME/CFS. They found that some common symptoms, but not all, were shared by both groups. Some people had more breathing problems and fatigue after COVID-19, while others had pain, sleep troubles, and brain fog. This means that treating these illnesses might involve some similar approaches.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundShared symptoms and biological abnormalities between post-acute sequelae of SARS-CoV-2 infection (PASC) and myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) could suggest common pathophysiological bases and would support coordinated treatment efforts. Empirical studies comparing these syndromes are needed to better understand their commonalities and differences.MethodsWe analyzed electronic health record data from 6.5 million adult patients from the National COVID Cohort Collaborative. PASC and ME/CFS diagnostic groups were defined based on recorded diagnoses, and other recorded conditions within the two groups were used to train separate machine learning-driven computable phenotypes (CPs). The most predictive conditions for each CP were examined and compared, and the overlap of patients labeled by each CP was examined. Condition records from the diagnostic groups were also used to statistically derive condition clusters. Rates of subphenotypes based on these clusters were compared between PASC and ME/CFS groups.ResultsApproximately half of patients labeled by one CP are also labeled by the other. Dyspnea, fatigue, and cognitive impairment are the most-predictive conditions shared by both CPs, whereas other most-predictive conditions are specific to one CP. Recorded conditions separate into cardiopulmonary, neurological, and comorbidity clusters, with the cardiopulmonary cluster showing partial specificity for the PASC groups.ConclusionsData-driven approaches indicate substantial overlap in the condition records associated with PASC and ME/CFS diagnoses. Nevertheless, cardiopulmonary conditions are somewhat more commonly associated with PASC diagnosis, whereas other conditions, such as pain and sleep disturbances, are more associated with ME/CFS diagnosis. These findings suggest that symptom management approaches to these illnesses could overlap.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00827-5" target="_blank">https://www.nature.com/articles/s43856-025-00827-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-09</strong> <a href="https://www.nature.com/articles/s41591-025-03650-6" target="_blank">
            ü§ñ Tumor Treatment for Certain Cancer Patients Using Their Own Immune Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-18', this)">
            More</button></h4>
            <div id="toggle-18" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Autologous T cell therapy for PRAME+advanced solid tumors in HLA-A*02+patients: a phase 1 trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists are testing a new way to treat cancer by changing the immune system to recognize and attack cancer cells. This method involves using a special part of the immune system called T cells to target tumor-associated proteins inside the cancer cells.

So far, many trials have shown limited success in treating solid tumors with this approach. However, researchers want to improve it, so they're conducting a new trial.

In this trial, 40 patients with certain types of cancer are getting a treatment that changes their immune system's T cells to recognize and attack tumor-associated proteins. The goal is to evaluate the safety and effectiveness of this treatment and find the right dose for future use.

The results show that the treatment was safe, but it didn't quite work as well as expected in terms of reducing tumor size. However, many patients showed some signs of improvement, such as long-term survival or improved symptoms. The researchers also found that higher doses of the treatment seemed to be more effective.

Overall, this new approach is showing promise for treating certain types of solid tumors, including melanoma.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In contrast to chimeric antigen receptor T cells, T cell receptor (TCR)-engineered T cells can target intracellular tumor-associated antigens crucial for treating solid tumors. However, most trials published so far show limited clinical activity. Here we report interim data from a first-in-human, multicenter, open-label, 3‚Äâ+‚Äâ3 dose-escalation/de-escalation phase 1 trial studying IMA203, an autologous preferentially expressed antigen in melanoma (PRAME)-directed TCR T cell therapy in HLA-A*02+patients with PRAME+recurrent and/or refractory solid tumors, including melanoma and sarcoma. Primary objectives include the evaluation of safety and tolerability and the determination of the maximum tolerated dose (MTD) and/or recommended dose for extension. Secondary objectives include the evaluation of IMA203 TCR-engineered T cell persistence in peripheral blood, tumor response as well as duration of response. A total of 27 patients were enrolled in the phase 1a dose escalation and 13 patients in the phase 1b dose extension. IMA203 T cells were safe, and the MTD was not reached. Of the 41 patients receiving treatment (that is, who started lymphodepletion), severe cytokine release syndrome was observed in 4.9% (2/41), and severe neurotoxicity did not occur. In the 40 patients treated with IMA203, an overall response rate consisting of patients with unconfirmed or confirmed response (u/cORR) of 52.5% (21/40) and a cORR of 28.9% (11/38) was observed with a median duration of response of 4.4‚Äâmonths (range, 2.4‚Äì23.0, 95% confidence interval: 2.6‚Äìnot reached) across multiple indications. Rapid T cell engraftment and long-term persistence of IMA203 T cells were observed. IMA203 T cells trafficked to all organs, and confirmed responses were more frequent in patients with higher dose. T cell exhaustion was not observed in the periphery; deep responses were enriched at higher PRAME expression; and higher T cell infiltration resulted in longer progression-free survival. Overall, IMA203 showed promising anti-tumor activity in multiple solid tumors, including refractory melanoma. ClinicalTrials.gov identifier:NCT03686124.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-025-03650-6" target="_blank">https://www.nature.com/articles/s41591-025-03650-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-09</strong> <a href="https://www.nature.com/articles/s41598-025-96417-5" target="_blank">
            ü§ñ Potential of Melissa Officinalis to Fight Flu</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-19', this)">
            More</button></h4>
            <div id="toggle-19" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antiviral potential ofMelissa officinalisextracts against influenza and emerging coronaviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This plant has been used for centuries to fight off illnesses like the flu and other respiratory problems. Scientists tested different parts of the plant to see if they could stop certain viruses from working properly. They found that one part of the plant worked really well against some viruses, including SARS-CoV-2 and MERS-CoV. Other parts of the plant showed promise too. When scientists looked at how the active ingredients in the plant interact with the viruses, they found some promising compounds. This suggests that extracts from Melissa officinalis could be used to make new medicines to fight off respiratory infections like COVID-19 and the flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Melissa officinalisis a perennial medicinal plant traditionally used for its diverse biological activities, including antiviral properties. This study investigates the antiviral efficacy of various extracts, including water, acetone, alkaloid, non-alkaloid, ethanol, and methanol extracts, against influenza A (H1N1), SARS-CoV-2, and MERS-CoV. The water extract demonstrated significant inhibitory effects on SARS-CoV-2 (IC50= 421.9¬†¬µg/mL) and MERS-CoV (IC50= 222.1¬†¬µg/mL) in Vero E6 cells (an African green monkey kidney cell line), with a CC50of 4221¬†¬µg/mL, indicating a favorable selectivity index. Additionally, it exhibited strong activity against H1N1 in Madin-Darby canine kidney cell line (MDCK cells) (IC50= 57.30¬†¬µg/mL, CC50= 3073¬†¬µg/mL). Among all the extracts, the methanol extract showed the highest antiviral activity. It has IC50=‚Äâ2.549¬†¬µg/ml and selectivity index (SI)‚Äâ=‚Äâ230 against H1N1.While it showed IC50=‚Äâ10.83¬†¬µg/ml against SARS-CoV-2 and 9.82¬†¬µg/ml against MERS-CoV with SI values of 54.2 and 59.77, respectively. Molecular docking studies revealed that 5-Methyl-5¬†H-naphtho[2,3-c]carbazole,7¬†H-Dibenzo[b, g]carbazole, 7-methyl, hesperidin, luteolin-7-glucoside-3‚Ä≤-glucuronide, Melitric acid A, and other compounds exhibited high binding affinities to the receptor-binding domains (RBDs) of SARS-CoV-2 and MERS-CoV spike glycoproteins, suggesting their potential to interfere with viral entry. Furthermore, GC-MS-identified bioactive compounds, including docosahexaenoic acid (DHA), eicosapentaenoic acid (EPA), paromomycin, and phenolic acids, demonstrated additional antiviral potential. These results underscore the potential ofM. officinalisextracts as natural antiviral agents, offering a foundation for further in vitro and in vivo validation and potential therapeutic applications against respiratory viral infections, including coronaviruses and influenza viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96417-5" target="_blank">https://www.nature.com/articles/s41598-025-96417-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-08</strong> <a href="https://www.nature.com/articles/s43856-025-00759-0" target="_blank">
            ü§ñ Nationwide Study of Long COVID's Causes and Effects on Americans</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-20', this)">
            More</button></h4>
            <div id="toggle-20" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A nationwide study of risk factors for long COVID and its economic and mental health consequences in the United States</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

In the US, many people are worried about how long COVID will affect their health and economy. But not enough studies have looked at which groups are most at risk.

This study used data from over 375,000 adults to find out which factors put people at risk for long COVID. The results show that lower household income, being Hispanic or female, and having a certain identity (such as LGBTQ+) can increase the risk of long COVID.

Having long COVID also leads to higher rates of unemployment, financial problems, and mental health issues. The study estimates that 24 million working adults have long COVID and are at risk of these problems, which could cost the economy $211 billion in 2022 and $218 billion in 2023.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundIn the United States, concerns have been increasingly raised over the future public health and economic burden of long COVID including disability and declines in labor force participation. However, only a handful of U.S. studies have explored sociodemographic or socioeconomic characteristics that put people at risk of long COVID or have investigated its economic and mental health sequelae.MethodsUsing repeated cross-sectional data on over 375,000 adults including nearly 50,000 adults with long COVID pooled from U.S. nationally-representative Household Pulse Survey data collected between September and November 2022 and between August and October 2023, I fit age- and gender-adjusted and multivariable modified Poisson regression models to examine multiple sociodemographic and socioeconomic factors as predictors of long COVID. I further estimate the risks of unemployment, financial hardship, and anxiety and depression among working-aged adults and adults with current long COVID symptoms, and estimate the economic burden of lost wages due to long COVID.ResultsNearly one in seven adults (~35 million) and working-aged adults (~30 million) reported having a history of long COVID by late 2022 and late 2023. In age- and gender-adjusted models and fully-adjusted multivariable models, I find several factors predict long COVID including lower household income, and being Hispanic, female, gay/lesbian or bisexual. I also find having long COVID is linked to higher risks of recent unemployment, financial hardship, and anxiety and depressive symptomatology, with evidence of dose-response relationships.ConclusionsOverall, an estimated 24 million working-aged adults with long COVID had been or may still be at risk of adverse socioeconomic and mental health outcomes. The lost earnings due to long COVID among working-aged adults are estimated to total $211 billion in 2022 and $218 billion in 2023. These findings highlight the substantial public health and economic implications of long COVID among Americans.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00759-0" target="_blank">https://www.nature.com/articles/s43856-025-00759-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-07</strong> <a href="https://www.nature.com/articles/s41598-025-96642-y" target="_blank">
            ü§ñ Vaccines Developed Faster with New Protein Testing Method</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-21', this)">
            More</button></h4>
            <div id="toggle-21" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Accelerated vaccine process development by orthogonal protein characterization</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic sped up how vaccines are made. Scientists had to create new tests quickly to ensure the vaccines were safe and worked properly. As new versions of the virus emerged, scientists had to update their tests to detect changes. They developed a special way to test vaccine ingredients using two different methods: one that looks at the ingredient's shape (Simple Wes) and another that looks at its tiny parts (liquid-chromatography tandem mass spectrometry). This helped them make decisions fast, so they could start testing the vaccines in just under 2 weeks after a small study was done.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic altered the vaccine development paradigm with accelerated timelines from concept through clinical safety and efficacy. Characterization and release assays for vaccine programs were developed under similar time constraints to support bioprocess development, scaleup and formulation. During the development of these vaccines, SARS-CoV-2 variants of concern (VOCs) emerged requiring integration of additional antigens into the target product profile. Biochemical testing to support the addition of new antigen variants (identity, quantity, antigenicity/potency) needed substantial re-development. Here we present a reversed-phase high-performance liquid-chromatography method for antigen purity with orthogonal identification characterization comprising of Simple Wes and liquid-chromatography tandem mass spectrometry (LC/MS/MS) to support accelerated process development for recombinant protein vaccines. This suite of assays was deployed to support rapid, scientific decision-making enabling the transition from completion of a placebo-controlled dose-ranging Phase 2 study to the start of the global Phase 3 safety and efficacy trial in less than 2¬†weeks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96642-y" target="_blank">https://www.nature.com/articles/s41598-025-96642-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-07</strong> <a href="https://www.nature.com/articles/s41598-025-95644-0" target="_blank">
            ü§ñ Genetic Tool Discovers New Regulator for Important Protein</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-22', this)">
            More</button></h4>
            <div id="toggle-22" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>CRISPR-Cas9 genetic screens reveal regulation of TMPRSS2 by the Elongin BC-VHL complex</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A certain enzyme on the outside of cells helps viruses like COVID-19 get inside healthy cells. We wanted to know how this happens and what makes it happen in a way that's not too helpful for our bodies. To do this, we looked at which things turn out the enzymes, using special tools that helped us find answers. It turns out that certain proteins are in charge of keeping the enzyme on or off. We also found that even if we block these proteins, it doesn't stop some versions of COVID-19 from getting inside cells.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The TMPRSS2 cell surface protease is used by a broad range of respiratory viruses to facilitate entry into target cells. Together with ACE2, TMPRSS2 represents a key factor for SARS-CoV-2 infection, as TMPRSS2 mediates cleavage of viral spike protein, enabling direct fusion of the viral envelope with the host cell membrane. Since the start of the COVID-19 pandemic, TMPRSS2 has gained attention as a therapeutic target for protease inhibitors which would inhibit SARS-CoV-2 infection, but little is known about TMPRSS2 regulation, particularly in cell types physiologically relevant for SARS-CoV-2 infection. Here, we performed an unbiased genome-wide CRISPR-Cas9 library screen, together with a library targeted at epigenetic modifiers and transcriptional regulators, to identify cellular factors that modulate cell surface expression of TMPRSS2 in human colon epithelial cells. We find that endogenous TMPRSS2 is regulated by the Elongin BC-VHL complex and HIF transcription factors. Depletion of Elongin B or treatment of cells with PHD inhibitors resulted in downregulation of TMPRSS2 and inhibition of SARS-CoV-2 infection. We show that TMPRSS2 is still utilised by SARS-CoV-2 Omicron variants for entry into colonic epithelial cells. Our study enhances our understanding of the regulation of endogenous surface TMPRSS2 in cells physiologically relevant to SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95644-0" target="_blank">https://www.nature.com/articles/s41598-025-95644-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-07</strong> <a href="https://www.nature.com/articles/s41598-025-92242-y" target="_blank">
            ü§ñ Effectiveness of a combination medicine for fighting the new Covid strain</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-23', this)">
            More</button></h4>
            <div id="toggle-23" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antiviral effect of Bromelain combined with acetylcysteine against SARS-CoV-2 Omicron variant</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

The coronavirus that causes Covid-19 is still a big threat to public health. Right now, there's no cheap and effective treatment for very severe cases of Covid-19. Researchers found a new medicine called BromAc that has special powers against the Omicron variant of the virus. They tested it in labs and in patient samples, and it killed the virus and stopped it from spreading. This is good news because it might help us treat future waves of the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recent pandemic represented one of the biggest challenges of modern civilization. SARS-CoV-2 remains an imminent public health threat and currently, there is no effective and greatly affordable treatment for severe COVID-19. Although standard management with dexamethasone, and physical management including physiotherapy, prone positioning and mechanical ventilation are used, severe disease patients may still succumb to infection. In this regard, BromAc¬Æis a combination therapy of a refined protein derived from Bromelain and acetylcysteine, that shows significant mucolytic and anti-inflammatory properties. In the present study, we performed in vitro, and ex vivo analyses to assess the effect of BromAc¬Æin inhibiting Omicron variant of SARS-CoV-2 at different levels. Here, we provide evidence of the in vitro virucidal activity of BromAc¬Æin Vero-ACE2/TMPRSS2 cell line infected with the Omicron variant. BromAc¬Æcan also abrogate SARS-CoV-2 RNA genomic copies in tracheal aspirate (TA) samples from critically ill COVID-19 patients after long term exposure. These results were confirmed by lower spike expression observed in EpCAM+PanCKnegepithelial cells from tracheal aspirate samples after BromAc¬Ætreatment. Furthermore, atomized BromAc¬Æpromoted cleavage of the S1 Spike subunit in TA samples, demonstrating the mechanism of the antiviral activity displayed by BromAc¬Æin human samples. These results bring novel evidence of antiviral activity in cell lines in vitro as well as in tracheal aspirate samples from critically ill COVID-19 patients, which support its potential use as an adjunct to COVID-19 management in future waves of Omicron subvariants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92242-y" target="_blank">https://www.nature.com/articles/s41598-025-92242-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-07</strong> <a href="https://www.nature.com/articles/s43856-025-00826-6" target="_blank">
            ü§ñ Spread of COVID-19 in New York City</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-24', this)">
            More</button></h4>
            <div id="toggle-24" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 dynamics in New York City during March 2020‚ÄìAugust 2023</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The new coronavirus that causes COVID-19 has been spreading since 2020 and will likely keep causing outbreaks. To understand how it spreads, researchers used data from New York City to create a model that shows how many people got infected, died, or were sick with COVID-19. They found out that the virus spread quickly and caused more than twice as many infections as people who actually got sick. The virus also became less deadly over time.

The model showed that certain factors, like winter weather, made people more likely to get infected. This information can help us better understand how to stop the spread of COVID-19 and protect public health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundThe severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been widespread since 2020 and will likely continue to cause substantial recurring epidemics. However, understanding the underlying infection burden and dynamics, particularly since late 2021 when the Omicron variant emerged, is challenging. Here, we leverage extensive surveillance data available in New York City (NYC) and a comprehensive model-inference system to reconstruct SARS-CoV-2 dynamics therein through August 2023.MethodsWe fit a metapopulation network SEIRSV (Susceptible-Exposed-Infectious-(re)Susceptible-Vaccination) model to age- and neighborhood-specific data of COVID-19 cases, emergency department visits, and deaths in NYC from the pandemic onset in March 2020 to August 2023. We further validate the model-inference estimates using independent SARS-CoV-2 wastewater viral load data.ResultsThe validated model-inference estimates indicate a very high infection burden‚Äîthe number of infections (i.e., including undetected asymptomatic/mild infections) totaled twice the population size (‚Äâ>‚Äâ5 times documented case count) during the first 3.5 years. Estimated virus transmissibility increased around 3-fold, whereas estimated infection-fatality risk (IFR) decreased by >10-fold during this period. The detailed estimates also reveal highly complex variant dynamics and immune landscape, and higher infection risk during winter in NYC over the study period.ConclusionsThis study provides highly detailed epidemiological estimates and identifies key transmission dynamics and drivers of SARS-CoV-2 during its first 3.5 years of circulation in a large urban center (i.e., NYC). These transmission dynamics and drivers may be relevant to other populations and inform future planning to help mitigate the public health burden of SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00826-6" target="_blank">https://www.nature.com/articles/s43856-025-00826-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-05</strong> <a href="https://www.nature.com/articles/s41598-025-96055-x" target="_blank">
            ü§ñ How COVID-19 affects daily life more than other respiratory illnesses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-25', this)">
            More</button></h4>
            <div id="toggle-25" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The implications of somatic symptom disorder on the impairment of daily life are greater in post-COVID syndrome than in asthma or COPD - results of a cross-sectional study in a rehabilitation clinic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

A study looked at how three conditions affect each other. There were 371 people with different breathing problems, including COVID-19 syndrome, asthma, and chronic lung disease. The researchers wanted to see if having anxiety or depression made it harder for them to do daily activities.

They found that people with post-COVID syndrome had more trouble with their daily lives because of somatic symptom disorder (feeling sick all the time). This was stronger in people with COVID-19 than those with asthma or chronic lung disease. The study also showed that anxiety and depression were big predictors of how hard it is to do things for people with asthma and chronic lung disease.

The researchers think that doctors should pay extra attention to somatic symptom disorder when treating patients with these conditions, especially during rehabilitation.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aim was to compare the relationship between somatic symptom disorder (SSD), anxiety, depression, clinical symptoms, and daily life impairment (DLI) in post-COVID syndrome (PCS), asthma and chronic obstructive pulmonary disease (COPD). In a cross-sectional study, 371 patients (161 PCS, 121 asthma, 89 COPD) of a pulmonary rehabilitation clinic received the questionnaires PHQ-15 (Patient Health Questionnaire-15) and SSD-12 (Somatic Symptom Disorder-12) to determine SSD, GAD-7 (Generalized Anxiety Disorder-7) to determine anxiety disorder, and PHQ-9 (Patient Health Questionnaire-9) to determine depression. Lung function was estimated using whole-body plethysmography. Predictors for DLI were assessed by regression models and ROC analyses. Association of SSD with DLI was stronger in PCS (odds ratio 13.8; 95% confidence interval 1.7-109.9) than in asthma (8.5; 2.4‚Äì30.1), and was not significant in COPD (1.9; 0.5‚Äì7.5). In asthma and COPD, strongest predictors were GAD-7 (15.0; 1.9-116.8) and PHQ-9 (8.9; 1.1‚Äì71.8), respectively. Diffusion capacity was predictive in COPD (0.947; 0.916‚Äì0.979) and asthma (0.967; 0.943‚Äì0.993), but not in PCS. To conclude, SSD appears to have greater impact on DLI in PCS than asthma or COPD patients. This should be recognized appropriately during rehabilitation. Furthermore, increased psychological comorbidity should also be considered and adequately treated in asthma and COPD if necessary.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96055-x" target="_blank">https://www.nature.com/articles/s41598-025-96055-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-05</strong> <a href="https://www.nature.com/articles/s41598-025-96191-4" target="_blank">
            ü§ñ Brain Study: How COVID-19 Affects Kids' Brains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-26', this)">
            More</button></h4>
            <div id="toggle-26" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multimodal MRI analysis of COVID-19 effects on pediatric brain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic has caused problems with people's brains, especially in kids. Scientists are still learning about how it affects their brains. This study looked at 19 kids who had mild COVID-19 and compared them to 22 healthy kids. The scientists took pictures of their brains using a special machine and found that some parts of the brain were bigger or thinner in the kids with COVID-19. They also found that these kids' brains were connected differently than usual. This means that COVID-19 might be affecting how kids' brains work, and scientists need to keep studying it to understand what's happening long-term.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has raised significant concerns regarding its impact on the central nervous system, including the brain. While the effects on adult populations are well documented, less is known about its implications for pediatric populations. This study investigates alterations in cortical metrics and structural covariance networks (SCNs) based on the Local Gyrification Index (LGI) in children with mild COVID-19, alongside changes in non-invasive MRI proxies related to glymphatic function. We enrolled 19 children with COVID-19 and 22 age-comparable healthy controls. High-resolution T1-weighted and diffusion-weighted MRI images were acquired. Cortical metrics, including thickness, surface area, volume, and LGI, were compared using vertex-wise general linear models. SCNs were analyzed for differences in global and nodal metrics, and MRI proxies, including diffusion tensor imaging along the perivascular space and choroid plexus (CP) volume, were also assessed. Our results showed increased cortical area, volume, and LGI in the left superior parietal cortex, as well as increased cortical thickness in the left lateral occipital cortex among children with COVID-19. SCN analysis revealed altered network topology and larger CP volumes in the COVID group, suggesting virus-induced neuroinflammation. These findings provide evidence of potential brain alterations in children following mild COVID-19, emphasizing the need for further investigation into long-term neurodevelopmental outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96191-4" target="_blank">https://www.nature.com/articles/s41598-025-96191-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-04</strong> <a href="https://www.nature.com/articles/s41598-025-95956-1" target="_blank">
            ü§ñ Study on Immunity Against New COVID Variants in Kids</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-27', this)">
            More</button></h4>
            <div id="toggle-27" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing antibodies to SARS-CoV-2 variants of concern:¬†a pediatric surveillance study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers wanted to understand how kids (5-12 years old) develop antibodies to fight SARS-CoV-2 when they get infected or vaccinated. They studied 619 kids who came to clinics or hospitals with symptoms related to COVID-19. The study found that most kids developed antibodies after getting vaccinated, but the antibodies didn't work as well against a new strain called Omicron. Kids who were already sick with COVID-19 before being vaccinated still had good antibody responses. This means we need to keep monitoring how well our vaccines work over time, especially against new variants like Omicron.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Knowledge regarding the pediatric immune response to SARS-CoV-2 infection and/or vaccination remains limited, particularly for the variants of concern (VOC). Our objective was to evaluate the neutralizing antibody response against SARS-CoV-2 VOC in the naturally infected and/or vaccinated pediatric population. Participants aged 5‚Äì12 years who presented to either an outpatient clinic or emergency room were eligible for participation in this study. Participants were divided into four groups based on infection and vaccination status. Plasma was tested using immunoassays targeting anti-SARS-CoV-2 IgG, spike protein, and nucleocapsid. A total of 619 participants met study inclusion. Natural infection was identified in 189/619 children (31%), 284/619 were vaccinated (46%) and 69/619 were both naturally infected and vaccinated (11%). Participants that were vaccinated had received one (n=‚Äâ169/619; 27%) or two (n=‚Äâ115/619; 19%) vaccine doses. The median time between the 1st and 2nd vaccine doses was 56 days, interquartile range 50‚Äì56. A general upward trend in antibody positivity was observed across all VOC as the study proceeded over a 5-month period. Omicron antibody responses were lower than those of other VOC, both in relation to the percentage of positive cases and over time. Neither asthma nor diabetes altered antibody responses, but antibody titres were reduced for a variety of VOC in those children receiving immunotherapy or with leukopenia. This study demonstrated decreased neutralizing antibody responses against the Omicron variant, regardless of past infection or vaccination status. These findings emphasize the need for continued neutralizing antibody surveillance.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95956-1" target="_blank">https://www.nature.com/articles/s41598-025-95956-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-04</strong> <a href="https://www.nature.com/articles/s41598-025-94317-2" target="_blank">
            ü§ñ Assessing immunity to COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-28', this)">
            More</button></h4>
            <div id="toggle-28" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A high-throughput, fully automated competition assay to evaluate SARS-CoV-2 neutralizing responses and epitope specificity in clinical samples</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

Coronavirus disease 2019 (COVID-19) is still a big health issue worldwide. We created a machine that can quickly test many people's bodies for antibodies against the new coronavirus. We looked at samples from people who got the virus and those who were vaccinated, and we found out that getting vaccinated makes your body make more antibodies to fight the virus than if you got sick naturally. The most important part of these antibodies is the one that targets a specific part of the virus called the RBD. Having this antibody is very helpful in fighting off the virus. We also discovered that having antibodies against other parts of the S1 protein, which is on the surface of the virus, can help fight off the virus too. This helps us understand how to make better tests and vaccines that can protect people from getting sick with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease-2019 (COVID-19) remains a critical global health concern. We developed a fully automated, high-throughput competition immunoassay to elucidate how epitope recognition on the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) spike receptor-binding domain (RBD) correlates with neutralizing activity. Analysis of clinical samples from both SARS-CoV-2-infected and vaccinated individuals revealed that vaccination elicits significantly higher antibody titers across multiple S1 subunit epitopes compared to natural infection. Notably, median antibody levels against the receptor-binding motif (RBM) exceeded 50% in both cohorts, highlighting the RBM as a key target for antibody induction irrespective of immune origin. Furthermore, the strongest correlation with neutralizing activity was observed for antibodies directed against the broader S1 subunit, indicating that epitopes outside the RBM also contribute to neutralization. These findings underscore the importance of both RBM- and non-RBM-directed antibodies in effective immune defense against SARS-CoV-2. Our assay enables large-scale, reliable quantification of neutralizing antibodies and provides critical insights for developing improved diagnostic antigens and vaccine strategies aimed at eliciting robust, multi-epitope immune responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94317-2" target="_blank">https://www.nature.com/articles/s41598-025-94317-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-04</strong> <a href="https://www.nature.com/articles/s41541-025-01115-y" target="_blank">
            ü§ñ How Vaccines Affect Flu Virus Spread in Chickens</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-29', this)">
            More</button></h4>
            <div id="toggle-29" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of inactivated vaccine on transmission and evolution of H9N2 avian influenza virus in chickens</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

The H9N2 avian flu virus is common in chickens around the world and can sometimes spread to humans. Even though people have been vaccinated against it, the virus still causes problems in chicken flocks. We tested how well these vaccines work by letting vaccinated chickens get infected with the virus again. We found that the virus grew stronger and became more contagious after going through vaccinated chickens. This could make it easier for the virus to spread to humans. But if we give people a vaccine that makes their noses and throats sick, it can stop the virus from spreading. This means we need new ways to protect against H9N2 avian flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>H9N2 avian influenza virus (AIV) is endemic in poultry worldwide and increasingly zoonotic. Despite the long-term widespread use of inactivated vaccines, H9N2 AIVs remain dominant in chicken flocks. We demonstrated that inactivated vaccines did not prevent the replication of H9N2 AIVs in the upper airway of vaccinated chickens. Viral transmission was enhanced during sequential passage in vaccinated chickens, which was attributed to the restricted production of defective interfering particles and the introduction of stable mutations (NP-N417D, M1-V219I, and NS1-R140W) which enhanced viral replication. Notably, the genetic diversity of H9N2 AIVs was greater and included more potential mammal/human-adapted mutations after passage through vaccinated chickens than through na√Øve chickens, which might facilitate the emergence of mammal-adapted strains. By contrast, vaccines inducing cellular/mucosal immunity in the upper respiratory tract effectively limit H9N2 AIV. These findings highlight the limitations of inactivated vaccines and the need for revised vaccination strategies to control H9N2 AIV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01115-y" target="_blank">https://www.nature.com/articles/s41541-025-01115-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-03</strong> <a href="https://www.nature.com/articles/s41467-025-57905-4" target="_blank">
            ü§ñ New medicine stops long COVID symptoms better than before</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-30', this)">
            More</button></h4>
            <div id="toggle-30" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel PLpro inhibitor improves outcomes in a pre-clinical model of long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the rewritten abstract:

The coronavirus pandemic showed us how easily diseases can spread around the world when people are connected. One medicine called Paxlovid helped people with COVID-19, but it had some problems. Some people couldn't take it, it didn't stop long-term symptoms, and it didn't work for everyone.

Researchers found a new target to help. They created a new medicine that targets a different part of the virus, and it works well in mice. The mice model shows how this medicine could prevent and treat long-term COVID-19 problems like lung issues and brain damage. This is an important step towards developing a treatment for people with long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic caused by the coronavirus SARS-CoV-2 has highlighted the vulnerability of a globally connected population to zoonotic viruses. The FDA-approved coronavirus antiviral Paxlovid targets the essential SARS-CoV-2 main protease, Mpro. Whilst effective in the acute phase of a COVID infection, Paxlovid cannot be used by all patients, can lead to viral recurrence, and does not protect against post-acute sequelae of COVID-19 (PASC), commonly known as long COVID, an emerging significant health burden that remains poorly understood and untreated. Alternative antivirals that are addressing broader patient needs are urgently required. We here report our drug discovery efforts to target PLpro, a further essential coronaviral protease, for which we report a novel chemical scaffold that targets SARS-CoV-2 PLpro with low nanomolar activity, and which exhibits activity against PLpro of other pathogenic coronaviruses. Our lead compound shows excellent in vivo efficacy in a mouse model of severe acute disease. Importantly, our mouse model recapitulates long-term pathologies matching closely those seen in PASC patients. Our lead compound offers protection against a range of PASC symptoms in this model, prevents lung pathology and reduces brain dysfunction. This provides proof-of-principle that PLpro inhibition may have clinical relevance for PASC prevention and treatment going forward.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57905-4" target="_blank">https://www.nature.com/articles/s41467-025-57905-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-02</strong> <a href="https://www.nature.com/articles/s41598-025-95494-w" target="_blank">
            ü§ñ Plant-made virus fighter helps prevent COVID.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-31', this)">
            More</button></h4>
            <div id="toggle-31" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An ACE2-Fc decoy produced in glycoengineered plants neutralizes ancestral and newly emerging SARS-CoV-2 variants and demonstrates therapeutic efficacy in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

New viruses that are causing COVID-19 are getting better at evading our immune system and making some medicines less effective. However, these new viruses still need to attach to the cells in our body to enter. We've created a special substance called ACE2-Fc that can block this attachment and prevent the virus from entering the cell. When we tested it on animals, it worked really well and even helped against some of the new, more resistant viruses. This gives us hope for new ways to fight COVID-19 using plants to make medicines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Newly emerging SARS-CoV-2 variants of concern (VOCs) continue to drive COVID-19 waves and are typically associated with immune escape and increased resistance to current therapeutics including monoclonal antibodies. By contrast, VOCs still display strong binding to the host cell receptor ACE2. Consistent with these properties, we have now found that a soluble ACE2-Fc decoy produced in glycoengineered plants effectively neutralizes different SARS-CoV-2 isolates and exhibits even increased potency against VOCs as compared to an ancestral virus strain. In a golden Syrian hamster model, therapeutic intranasal delivery of ACE2-Fc effectively reduced weight loss and SARS-CoV-2 replication in the lungs when administered 24¬†h post-inoculation. This protective effect was not observed upon treatment of the infected animals with a non-binding ACE2-Fc mutant, demonstrating that the plant-derived ACE2-Fc decoy interferes specifically with the attachment of the virus to host cells. The results obtained provide support for further development of decoy-based antiviral approaches by plant molecular pharming.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95494-w" target="_blank">https://www.nature.com/articles/s41598-025-95494-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-02</strong> <a href="https://www.nature.com/articles/s41598-025-96294-y" target="_blank">
            ü§ñ Medicine helps reduce inflammation and seriousness of COVID-19 in older people.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-32', this)">
            More</button></h4>
            <div id="toggle-32" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Metformin alleviates inflammatory response and severity rate of COVID-19 infection in elderly individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists looked at if taking a certain medicine called metformin before getting COVID-19 helped or hurt the chances of having a bad outcome. They looked at 413 people with type 2 diabetes who got COVID-19 and compared them to 4204 others.

They found that:

* People who took metformin were less likely to go to the ICU hospital
* Those taking metformin had pneumonia fewer times than those not taking it
* People over 60 years old on metformin had lower levels of inflammation and higher numbers of white blood cells compared to those not on metformin

Overall, taking metformin before getting COVID-19 might help make the illness less severe.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the relationship between metformin and the outcome of coronavirus disease 2019 (COVID-19) infection. The study included 413 patients with type 2 diabetes among the 5217 patients enrolled in a COVID-19 study, and analyzed whether receiving metformin therapy prior to infection was associated with risk of ICU admission, development of pneumonia and length of hospital stay. The study also examined the correlation between metformin treatment and levels of IL-6, CRP, serum ferritin (SF), lymphocyte, CD4 at admission, as well as the increase in open reading frame 1ab gene cycle threshold (ORF1abCT) after one week of hospitalization. There were no differences in age, sex, BMI, comorbidities, number of vaccine doses or eGFR between patients receiving and not receiving metformin therapy. In the ICU group, the proportion of patients not receiving metformin was 92.5%, significantly higher than the 69.2% of patients not admitted to ICU (p=‚Äâ0.010). In the pneumonia group, the proportion of patients not receiving metformin was 78.6%, significantly higher than the 67.2% in the non-pneumonia group (p=‚Äâ0.020). Compared with patients receiving no treatment, those receiving metformin had a shorter hospital stay (12.1‚Äâ¬±‚Äâ5.9 days vs. 14.5‚Äâ¬±‚Äâ8.2 days, p‚Äâ=‚Äâ0.001). In the patients‚Äâ‚â•‚Äâ60 years old, those receiving treatment had significantly lower levels of IL-6 (median, 12.3 pg/ml vs. 4.0 pg/ml, p‚Äâ=‚Äâ0.026) and significantly higher levels of Lymphocyte (median, 1.2‚Äâ√ó‚Äâ109/L vs. 1.4‚Äâ√ó‚Äâ109/L, p‚Äâ=‚Äâ0.015) compared with those not receiving treatment. However, for the patients under 60, there were no significant differences observed in IL-6 and Lymphocyte levels between those receiving treatment and those not. Metformin can reduce the severity of COVID-19 infection and attenuate the inflammatory response associated with COVID-19 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96294-y" target="_blank">https://www.nature.com/articles/s41598-025-96294-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-01</strong> <a href="https://www.nature.com/articles/s41598-025-95133-4" target="_blank">
            ü§ñ The impact of COVID-19 on people with Type 2 Diabetes and nerve damage (neuropathy)</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-33', this)">
            More</button></h4>
            <div id="toggle-33" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 and neuropathy in type 2 diabetes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

This study looked at how having COVID-19 affects people with type 2 diabetes and nerve damage called diabetic peripheral neuropathy.

We studied 76 people with type 2 diabetes who got COVID-19. We found that those who developed long-term COVID symptoms were more likely to be women, have high cholesterol or weight, and low blood pressure.

People with long-term COVID had changes in their cholesterol levels, weight, and nerve sensitivity, but these effects didn't affect the development or progression of diabetic peripheral neuropathy.

Having COVID-19 might affect how well someone's symptoms get worse over time, but it doesn't change the fact that they have diabetic peripheral neuropathy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigated the risk factors for COVID-19 and its impact on diabetic peripheral neuropathy (DPN) in patients with type 2 diabetes (T2D). Patients with T2D underwent assessments with the NICE post-COVID questionnaire, DN4 questionnaire, vibration perception threshold (VPT), and corneal confocal microscopy (CCM) before and 11.0‚Äâ¬±‚Äâ8.9 months after developing COVID-19. Of 76 participants with T2D, 35 (46.1%) developed COVID-19, of whom 8 (22.9%) developed severe COVID-19 and 9 (25.7%) developed long-COVID. The development of COVID-19 was associated with lower systolic blood pressure (P<‚Äâ0.05). The presence and severity of DPN were not associated with developing COVID-19, severe COVID-19, or long-COVID (P=‚Äâ0.42‚Äì0.94). Women were eight times more likely to develop long-COVID (P<‚Äâ0.05) and elevated body weight, LDL, and VPT were associated with the development of long-COVID (P<‚Äâ0.05‚Äâ‚àí‚Äâ0.01). The long-COVID group exhibited significant changes in triglycerides and LDL¬†(P<‚Äâ0.05¬†for both) and body weight (P<‚Äâ0.01) at follow-up. Their impact on clinical and neuropathy measures was comparable in patients with and without COVID-19 (P=‚Äâ0.08‚Äì0.99). There was a significant reduction in corneal nerve measures (P<‚Äâ0.05-0.0001) in patients with and without COVID-19. A low systolic blood pressure, altered lipids, body weight, higher VPT,¬†and gender may determine the impact of COVID-19 in patients with T2D, but there was no evidence of an impact of COVID-19 on the development or progression of DPN.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95133-4" target="_blank">https://www.nature.com/articles/s41598-025-95133-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-01</strong> <a href="https://www.nature.com/articles/s41467-025-58504-z" target="_blank">
            ü§ñ Vaccine Offers Protection Against Deadly Bird Flu</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-34', this)">
            More</button></h4>
            <div id="toggle-34" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intranasal influenza virus-vectored vaccine offers protection against clade 2.3.4.4b H5N1 infection in small animal models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Avian influenza virus has been present in birds since 1997 and can sometimes infect humans. A new strain of the virus has been found in cattle, causing concern that it could spread to other animals and people.

To prevent this, scientists have developed a vaccine for cattle using a different strain of the virus. They tested the vaccine in mice and hamsters and found that it provides strong protection against the virus. The vaccine also triggers an immune response that helps fight off the virus. While more work is needed, this vaccine has the potential to help protect humans if the virus were to spread.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The highly pathogenic avian influenza (HPAI) H5N1 virus has been endemic in aquatic birds since 1997, causing outbreaks in domestic poultry and occasional human infections worldwide. Recently, the cross-species transmission of a new reassortant variant from clade 2.3.4.4b of H5N1 to cattle in the US has heightened concerns regarding the expansion of host range and potential human infection. As eradicating the H5N1 virus from its reservoir is impossible, it is essential to prepare for a potential pandemic caused by an H5N1 derivative. Utilizing a deleted-NS1 live attenuated influenza viral vector vaccine system (DelNS1 LAIV), a system we have previously used in the development of a COVID-19 vaccine, we have rapidly developed an intranasal vaccine for cattle H5N1 and related clade 2.3.4.4b strains, based on publicly available sequences. Our research demonstrates that a single intranasal immunization can provide effective protection against lethal challenges from HPAI cattle or mink H5N1 variants, offering strong, sustained immunity after two months in female mouse and male hamster models. Immunogenicity analysis reveals that intranasal vaccination with DelNS1 LAIV induces robust neutralizing antibody, mucosal IgA and T cell responses in mice. It is crucial to further evaluate the DelNS1-H5N1 LAIV system to prepare for potential future H5N1 outbreaks in humans.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-58504-z" target="_blank">https://www.nature.com/articles/s41467-025-58504-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-04-01</strong> <a href="https://www.nature.com/articles/s41467-025-57849-9" target="_blank">
            ü§ñ Pregnant Women and COVID-19: What Happens After SARS-CoV-2 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-35', this)">
            More</button></h4>
            <div id="toggle-35" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID after SARS-CoV-2 during pregnancy in the United States</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Pregnancy can affect how our bodies fight off infections like COVID-19. But it's not clear if pregnancy makes us more or less likely to get Long COVID (a condition where symptoms last longer than usual after an initial infection).

This study looked at people who got infected with COVID-19 during pregnancy and compared them to women who got infected outside of pregnancy. It found that:

* Pregnant women are more likely to get Long COVID, but not as much as non-pregnant women.
* Some factors like being Black, older than 35, or having a weight problem can increase the risk of getting Long COVID in pregnant women.

Overall, this study suggests that pregnant women are at higher risk for Long COVID, but not as high as previously thought.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pregnancy alters immune responses and clinical manifestations of COVID-19, but its impact on Long COVID remains uncertain. This study investigated Long COVID risk in individuals with SARS-CoV-2 infection during pregnancy compared to reproductive-age females infected outside of pregnancy. A retrospective analysis of two U.S. databases, the National Patient-Centered Clinical Research Network (PCORnet) and the National COVID Cohort Collaborative (N3C), identified 29,975 pregnant individuals (aged 18‚Äì50) with SARS-CoV-2 infection in pregnancy from PCORnet and 42,176 from N3C between March 2020 and June 2023. At 180 days after infection, estimated Long COVID risks for those infected during pregnancy were 16.47 per 100 persons (95% CI, 16.00‚Äì16.95) in PCORnet using the PCORnet computational phenotype (CP) model and 4.37 per 100 persons (95% CI, 4.18‚Äì4.57) in N3C using the N3C CP model. Compared to matched non-pregnant individuals, the adjusted hazard ratios for Long COVID were 0.86 (95% CI, 0.83‚Äì0.90) in PCORnet and 0.70 (95% CI, 0.66‚Äì0.74) in N3C. The observed risk factors for Long COVID included Black race/ethnicity, advanced maternal age, first- and second-trimester infection, obesity, and comorbid conditions. While the findings suggest a high incidence of Long COVID among pregnant individuals, their risk was lower than that of matched non-pregnant females.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57849-9" target="_blank">https://www.nature.com/articles/s41467-025-57849-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-30</strong> <a href="https://www.nature.com/articles/s41541-025-01113-0" target="_blank">
            ü§ñ Protective immune response found in monkeys to COVID-19 virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-36', this)">
            More</button></h4>
            <div id="toggle-36" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>RBD-depleted SARS-CoV-2 spike generates protective immunity in cynomolgus macaques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists created a new way to make vaccines for COVID-19 by changing part of the virus's outer layer. This change helped the vaccine work better against different versions of the virus, including the omicron variant.

They made special particles that looked like the virus but didn't have the bad part that made people sick. When animals were given these new particles as a second dose, they had better protection against getting infected and making people sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 pandemic revealed the rapid evolution of circulating strains. This led to new variants carrying mostly mutations within the receptor binding domain, which is immunodominant upon immunization and infection. In order to steer the immune response away from RBD epitopes to more conserved domains, we generated S glycoprotein trimers without RBD and stabilized them by formaldehyde cross-linking. The cryoEM structure demonstrated that SŒîRBD folds into the native prefusion conformation, stabilized by one specific cross-link between S2 protomers. SŒîRBD was coated onto lipid vesicles, to produce synthetic virus-like particles, SŒîRBD-LV, which were utilized in a heterologous prime-boost strategy. Immunization of cynomolgus macaques either three times with the mRNA Comirnaty vaccine or two times followed by SŒîRBD-LV showed that the SŒîRBD-LV boost induced similar antibody titers and neutralization of different variants, including omicron. Upon challenge with omicron XBB.3, both the Comirnaty only and Comirnaty/SŒîRBD-LV vaccination schemes conferred similar overall protection from infection for both the Comirnaty only and Comirnaty/SŒîRBD-LV vaccination schemes. However, the SŒîRBD-LV boost indicated better protection against lung infection than the Comirnaty strategy alone. Together our findings indicate that SŒîRBD is highly immunogenic and provides improved protection compared to a third mRNA boost indicative of superior antibody-based protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01113-0" target="_blank">https://www.nature.com/articles/s41541-025-01113-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-29</strong> <a href="https://www.nature.com/articles/s41598-025-95503-y" target="_blank">
            ü§ñ Membrane Targeted Vaccine Using Exosomes to Fight Viruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-37', this)">
            More</button></h4>
            <div id="toggle-37" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Membrane-targeted immunogenic compositions using exosome mimetic approach for vaccine development against SARS-CoV-2 and other pathogens</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that explains the research in simple terms:

Scientists are looking for a better way to make vaccines quickly and safely. They found that using part of the COVID-19 virus protein can create a vaccine that is more effective and safer than other types of vaccines. This new method works by attaching antigens to a viral membrane, which helps the body build a stronger immune response. The researchers also discovered that they can use this method to make vaccines for other diseases, including bacteria and tumors.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has underscored the urgent need for a vaccine strategy that is safe, effective, rapid, cost-efficient, and scalable for large-scale deployment during widespread infectious outbreaks. Here, we present a new vaccination strategy that meets these critical requirements. The SARS-CoV-2¬†S protein consists of the S1 and S2 subunits. The S2 subunit acts as the viral cell membrane fusion protein, and mutations in its C-terminal region facilitate the transport of the entire S protein to the cell membrane. When we expressed the SARS-CoV-2¬†S protein with a deletion of 21 amino acids from its C-terminal region in various cell types, we observed a dense presence of the protein in the cell membrane, as determined by IHC, dot blot, and ELISA. In the cell membrane-SARS-CoV-2¬†S protein complex, the cell membrane functions as an exosome mimic, carrying protein antigens (S protein) in their most natural form, as no further protocols are used to attach antigens to the membrane. We demonstrate that using the membrane-S protein component as a vaccine yields a more robust and protective immune response, with enhanced safety compared to mRNA-based or inactivated virus-based vaccines against SARS-CoV-2. Additionally, we show that fusing the transmembrane domain of the Vesicular Stomatitis Virus (VSV) G protein with the SARS-CoV-2 S1 protein effectively transports the S1 protein to the cell membrane, similar to SARS-CoV-2¬†S Œî21. We propose that designing the S2 subunit of the SARS-CoV-2¬†S protein, or its analogues such as the VSV-G protein, as carriers for fusing bacterial, viral, or tumor proteins with antigenic properties‚Äîand transporting them to the cell membrane‚Äîcould result in a comprehensive vaccination protocol applicable to all bacteria, viruses, and even tumors.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95503-y" target="_blank">https://www.nature.com/articles/s41598-025-95503-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-29</strong> <a href="https://www.nature.com/articles/s41541-025-01081-5" target="_blank">
            ü§ñ Impact of Early Vaccination on Influenza Pandemics in the US</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-38', this)">
            More</button></h4>
            <div id="toggle-38" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modeling the impact of early vaccination in an influenza pandemic in the United States</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Imagine there's a big flu outbreak and you want to help stop it from spreading. We studied how giving just one dose of the flu vaccine at different times during an outbreak could save lives. We found that if people get vaccinated early, they're much more likely to stay healthy. If someone gets vaccinated too late, not as many will stay healthy. The good news is that even if vaccines are given a little late, it's still worth doing because it can help slow down the spread of the flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We modeled the impact of initiating one-dose influenza vaccination at 3 months vs 6 months after declaration of a pandemic over a 1-year timeframe in the US population. Three vaccine effectiveness (VE) and two pandemic severity levels were considered, using an epidemic curve based on typical seasonal influenza epidemics. Vaccination from 3 months with a high, moderate, or low effectiveness vaccine would prevent ~95%, 84%, or 38% deaths post-vaccination, respectively, compared with 21%, 18%, and 8%, respectively following vaccination at 6 months, irrespective of pandemic severity. While the pandemic curve would not be flattened from vaccination from 6 months, a moderate/high effectiveness vaccine could flatten the curve if administered from 3 months. Overall, speed of initiating a vaccination campaign is more important than VE in reducing the health impacts of an influenza pandemic. Preparedness strategies may be able to minimize future pandemic impacts by prioritizing rapid vaccine roll-out.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01081-5" target="_blank">https://www.nature.com/articles/s41541-025-01081-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-29</strong> <a href="https://www.nature.com/articles/s41598-025-94654-2" target="_blank">
            ü§ñ Improving a Computer Model to Better Diagnose COVID-19 from Chest Images</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-39', this)">
            More</button></h4>
            <div id="toggle-39" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A BiLSTM model enhanced with multi-objective arithmetic optimization for COVID-19 diagnosis from CT images</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The coronavirus is always changing, making it hard to diagnose COVID-19 using computer algorithms. This research creates a new way to improve these algorithms by combining different techniques. It tests this new method on publicly available medical data and shows that it works very well, achieving 95% accuracy and specificity.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In response to the relentless mutation of the coronavirus disease, current artificial intelligence algorithms for the automated diagnosis of COVID-19 via CT imaging exhibit suboptimal accuracy and efficiency. This manuscript proposes a multi-objective optimization algorithm (MOAOA) to enhance the BiLSTM model for COVID-19 automated diagnosis. The proposed approach involves configuring several hyperparameters for the bidirectional long short-term memory (BiLSTM), optimized using the MOAOA intelligent optimization algorithm, and subsequently validated on publicly accessible medical datasets. Remarkably, our model achieves an impressive 95.32% accuracy and 95.09% specificity. Comparative analysis with state-of-the-art techniques demonstrates that the proposed model significantly enhances accuracy, efficiency, and other performance metrics, yielding superior results.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94654-2" target="_blank">https://www.nature.com/articles/s41598-025-94654-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-29</strong> <a href="https://www.nature.com/articles/s41598-025-96032-4" target="_blank">
            ü§ñ COVID-19 and Chronic Kidney Disease</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-40', this)">
            More</button></h4>
            <div id="toggle-40" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Association between COVID-19 and the development of chronic kidney disease in patients without initial acute kidney injury</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young person can understand this:

COVID-19 makes some people more likely to get very bad kidney problems that are hard to fix. We looked at a big group of people who got COVID-19 and another big group of people who got the flu. We wanted to know if getting COVID-19 made their kidneys worse over time.

We found out that it did! People who got COVID-19 were more likely to have bad kidney problems, like having their kidneys not working properly, than people who got the flu. This is especially true for older people and those with diabetes or high blood pressure.

We need to keep an eye on people's kidneys if they had COVID-19 because it might make them more sick in the long run.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While the association between COVID-19 and acute kidney injury (AKI) is well documented, the impact of COVID-19 on the development of advanced chronic kidney disease (CKD) remains unclear, particularly in patients without initial AKI. Using the TriNetX healthcare database, we conducted a matched cohort study comparing 141,587 COVID-19 and 141,587 influenza patients. We excluded patients with AKI within one month of infection and matched groups on demographics, comorbidities, and baseline laboratory values. The primary outcome was the incidence of advanced CKD (stages 3‚Äì5) at the 12-month follow-up. COVID-19 patients showed higher 12-month risks of advanced CKD (hazard ratio [HR]:2.02, 95% confidence interval [CI]:1.69‚Äì2.42,p<‚Äâ0.0001), AKI (HR 3.04, 95%CI:2.61‚Äì3.55,p<‚Äâ0.0001), and estimated glomerular filtration rate‚Äâ<‚Äâ60 mL/min/1.73 m2(HR:3.01, 95%CI:2.74‚Äì3.30,p<‚Äâ0.0001) compared to influenza patients. Subgroup analyses showed consistently elevated risks across sexes and in patients over 45 years, while younger patients did not demonstrate an increased risk of advanced CKD at the 12-month follow-up. Diabetes mellitus and hypertension have emerged as the strongest predictors of advanced CKD development. In conclusion, COVID-19 is associated with an increased risk of long-term renal dysfunction compared with influenza, suggesting the need for extended monitoring of kidney function in high-risk populations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-96032-4" target="_blank">https://www.nature.com/articles/s41598-025-96032-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-28</strong> <a href="https://www.nature.com/articles/s41598-025-94765-w" target="_blank">
            ü§ñ Identifying risks for long-term effects of COVID-19 in Spain</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-41', this)">
            More</button></h4>
            <div id="toggle-41" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identifying risk factors and predicting long COVID in a Spanish cohort</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many people are still getting sick with COVID-19 long time after they first got infected. To understand why some people get Long COVID more than others, researchers looked at data from 675,126 patients in Spain who had COVID-19 and compared them to people without Long COVID.

They found that older adults and women were more likely to get Long COVID. Certain symptoms like chest pain and losing the sense of smell also increased the risk. But getting vaccinated before or after getting COVID-19 greatly reduced the risk of developing Long COVID.

The researchers created a model to predict who might get Long COVID, which uses data on age, sex, symptoms, and vaccination status. They were able to create an accurate online calculator that helps doctors and patients identify potential Long COVID risks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Many studies have investigated symptoms, comorbidities, demographic factors, and vaccine effects in relation to long COVID (LC-19) across global populations. However, a number of these studies have shortcomings, such as inadequate LC-19 categorisation, lack of sex disaggregation, or a narrow focus on certain risk factors like symptoms or comorbidities alone. We address these gaps by investigating the demographic factors, comorbidities, and symptoms present during the acute phase of primary COVID-19 infection among patients with LC-19 and comparing them to typical non-Long COVID-19 patients. Additionally, we assess the impact of COVID-19 vaccination on these patients. Drawing on data from the Regional Health System of the Region of Murcia in southeastern Spain, our analysis includes comprehensive information from clinical and hospitalisation records, symptoms, and vaccination details of over 675126 patients across 10 hospitals. We calculated age and sex-adjusted odds ratios (AOR) to identify protective and risk factors for LC-19. Our findings reveal distinct symptomatology, comorbidity patterns, and demographic characteristics among patients with LC-19 versus those with typical non-Long COVID-19. Factors such as age, female sex (AOR = 1.39, adjusted p < 0.001), and symptoms like chest pain (AOR > 1.55, adjusted p < 0.001) or hyposmia (AOR > 1.5, adjusted p < 0.001) significantly increase the risk of developing LC-19. However, vaccination demonstrates a strong protective effect, with vaccinated individuals having a markedly lower risk (AOR = 0.10, adjusted p < 0.001), highlighting the importance of vaccination in reducing LC-19 susceptibility. Interestingly, symptoms and comorbidities show no significant differences when disaggregated by type of LC-19 patient. Vaccination before infection is the most important factor and notably decreases the likelihood of long COVID. Particularly, mRNA vaccines offer more protection against developing LC-19 than viral vector-based vaccines (AOR = 0.48). Additionally, we have developed a model to predict LC-19 that incorporates all studied risk factors, achieving a balanced accuracy of 73% and ROC-AUC of 0.80. This model is available as a free online LC-19 calculator, accessible at (LC-19 Calculator).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94765-w" target="_blank">https://www.nature.com/articles/s41598-025-94765-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-28</strong> <a href="https://www.nature.com/articles/s41467-025-57706-9" target="_blank">
            ü§ñ PD-1 protein makes immune system less active in severe infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-42', this)">
            More</button></h4>
            <div id="toggle-42" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>PD-L1+plasma cells suppress T lymphocyte responses in patients with sepsis and mouse sepsis models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Sepsis is a serious illness that kills many people who are very sick. To find ways to help these patients, we need to understand how their bodies' immune system works when they're in sepsis. We looked at mice with sepsis and found a special group of cells called regulatory plasma cells that can help control the immune response. These cells work by talking to other cells through special signals, like a messenger, to keep the body from getting too overactive. When we tested this idea on people with sepsis, it worked - but only for those who were very sick and had a higher risk of dying. This discovery could lead to new treatments to help patients with sepsis by fixing their immune system's problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Sepsis, a leading cause of death in intensive care units, is associated with immune alterations that increase the patients‚Äô risk of secondary infections and mortality, so better understandings of the pathophysiology of sepsis-induced immunosuppression is essential for the development of therapeutic strategies. In a murine model of sepsis that recapitulates immune alterations observed in patients, here we demonstrate that PD-L1+CD44+B220LowCD138+IgM+regulatory plasma cells are induced in spleen and regulate ex vivo proliferation and IFN…£ secretion induced by stimulation of T splenocytes. This effect is mediated both by cell-cell contact through increased PD-L1 expression on plasma cells and by production of a soluble factor. These observations are recapitulated in three cohorts of critically ill patients with bacterial and viral sepsis in association with increased mortality. Our findings thus reveal the function of regulatory plasma cells in the pathophysiology of sepsis-induced immune alterations, and present a potential therapeutic target for improving immune cell function impaired by sepsis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57706-9" target="_blank">https://www.nature.com/articles/s41467-025-57706-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-28</strong> <a href="https://www.nature.com/articles/s41420-025-02412-0" target="_blank">
            ü§ñ How do certain immune cells trigger cell death in stem cells?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-43', this)">
            More</button></h4>
            <div id="toggle-43" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Proinflammatory cytokines sensitise mesenchymal stromal cells to apoptosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists studied how a type of immune cell called mesenchymal stromal cells (MSCs) can help with inflammation. They wanted to know if MSCs could be made better at fighting inflammation by getting ready for it first.

They found out that when MSCs are exposed to certain chemicals that make them angry, like from an infection, they become more sensitive to dying. This helps their body get rid of them faster so they can do their job.

The team also discovered that a special process inside the cells called apoptosis (programmed cell death) is what kills the MSCs. When this process happens, it breaks down the cell into tiny pieces that the immune system can see and remove.

By understanding how this works, scientists hope to make MSCs even better at fighting inflammation in people who need their help.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Mesenchymal stromal cells (MSCs) exert broad therapeutic effects across a range of inflammatory diseases. Their mechanism of action has largely been attributed to paracrine signalling, orchestrated by an array of factors produced by MSCs that are collectively termed the ‚Äúsecretome‚Äù. Strategies to enhance the release of these soluble factors by pre-exposure to inflammatory cytokines, a concept known as ‚Äúlicensing‚Äù, is thought to provide a means of enhancing MSC efficacy. Yet, recent evidence shows that intravenously infused MSCs entrapped within the lungs undergo apoptosis, and their subsequent clearance by host phagocytes is essential for their therapeutic efficacy. We therefore sought to clarify the mechanisms governing regulated cell death in MSCs and how exposure to inflammatory cytokines impacts this process. Our results show that MSCs are relatively resistant to cell death induced via the extrinsic pathway of apoptosis, as well as stimuli that induce necroptosis, a form of regulated inflammatory cell death. Instead, efficient killing of MSCs required triggering of the mitochondrial pathway of apoptosis, via inhibition of the pro-survival proteins MCL-1 and BCL-XL. Apoptotic bodies were readily released by MSCs during cell disassembly, a process that was inhibited in vitro and in vivo when the apoptotic effectors BAK and BAX were genetically deleted. Licensing of MSCs by pre-exposure to the inflammatory cytokines TNF and IFN-Œ≥ increased the sensitivity of MSCs to intrinsic apoptosis in vitro and accelerated their in vivo clearance by host cells within the lungs after intravenous infusion. Taken together, our study demonstrates that inflammatory ‚Äúlicensing‚Äù of MSCs facilitates cell death by increasing their sensitivity to triggers of the intrinsic pathway of apoptosis and accelerating the kinetics of apoptotic cell disassembly.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41420-025-02412-0" target="_blank">https://www.nature.com/articles/s41420-025-02412-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-27</strong> <a href="https://www.nature.com/articles/s41598-025-94940-z" target="_blank">
            ü§ñ Wearing a Surgical Mask During Chemotherapy Is Safe for Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-44', this)">
            More</button></h4>
            <div id="toggle-44" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Wearing a surgical mask during chemotherapy session is safe</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract rewritten for a young person:

Wearing surgical masks is now a common practice during the pandemic. But did you know that wearing one while getting chemotherapy might affect how well your body takes in oxygen? A study was done to see if this could happen and what it means for cancer patients. One hundred twenty-six people with cancer who got chemotherapy were studied. They wore surgical masks before and after treatment, and their vital signs and blood work were checked. The results showed that wearing a mask didn't have big effects on how well they took in oxygen or felt. However, some patients might get a little dehydrated or have lower oxygen levels slightly. This means that wearing surgical masks is generally safe for cancer patients getting chemotherapy, but they should be careful and check with their doctor if anything feels off.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Surgical masks(SM) have become essential to our daily lives with the COVID-19 pandemic. It is recommended as the cheapest, most effective preventive method. The effects of SM on patients receiving chemotherapy are unknown. Our study aimed to investigate the effects of SM on oxygenation and CO2 retention in cancer patients receiving chemotherapy and to examine its possible clinical consequences. Patients diagnosed with cancer and receiving chemotherapy were included in the study. Venous blood gas, SO2 by pulse oximeter, and vital signs were recorded before and after treatment. Acute toxicities encountered during treatment were recorded. One hundred twenty-six patients with a median age of 60 (33‚Äì85) were evaluated in the study. The comparison of pre-post treatment parameters showed statistically significant changes in Ph (7.37 vs. 7.35,p<‚Äâ0.01), pCO2 (44.2 vs. 45.8,p=‚Äâ0.049), HCO3 (25.7 vs. 25.3,p=‚Äâ0.003), SpO2 (97.0 vs. 96.0,p=‚Äâ0.08), fever (36.4 vs. 36.3,p=‚Äâ0.023). All the changes were clinically insignificant and in normal ranges. Chemotherapy-related acute toxicity was noted in 4 (3.2%) of the patients. Lung morbidity, cancer type, lung metastasis status, treatment applied, duration of therapy, and acute toxicity do not affect the current parameters. In our study, it was shown that constantly wearing a SM in patients receiving chemotherapy caused CO2 retention and a tendency to hypoxemia. However, the current changes were clinically insignificant and within the normal range. Surgical masks can be used safely in cancer patients receiving systemic treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94940-z" target="_blank">https://www.nature.com/articles/s41598-025-94940-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s41598-025-95037-3" target="_blank">
            ü§ñ How Covid-19 Affects Cancer Patients Differently and What May Cause It</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-45', this)">
            More</button></h4>
            <div id="toggle-45" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Differences in Covid-19 deaths amongst cancer patients and possible mediators for this relationship</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

People who are not white but Black get sick from COVID-19 more often and have a higher chance of dying from it than people who are white. Some cancers can increase this risk. A study looked at people with cancer who tested positive for COVID-19 in Louisiana between 2011 and 2021.

The study found that cancer patients who died from COVID-19 were 6 times more likely to die than those who didn't have cancer. It also showed that having certain health problems, like kidney disease, can make it harder to recover from COVID-19.

This means we need to find ways to help people with chronic health issues like kidney disease after they get sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Previous research demonstrated Non-Hispanic Black populations experience higher COVID-19 mortality rates than Non-Hispanic White individuals. Additionally, cancer status is a known risk factor for COVID-19 death. While prior studies investigated comorbidities as exploratory variables in differences in COVID-19 hospitalization, none have explored their role in COVID-19-related deaths. This study aimed to evaluate whether Charlson Comorbidity Index (CCI) and subsequently, individual diseases are potential explanatory variables for this relationship. The analysis focused on Non-Hispanic Black and Non-Hispanic White cancer patients aged 20 or older, diagnosed between 2011 and 2019, who tested positive for COVID-19 from the start of pandemic through June 30, 2021 from Louisiana Tumor Registry. Two separate mediation analyses were conducted. First checked whether overall comorbidity, measured by CCI, could explain the difference in COVID-19 mortality. If so, further checked which individual comorbidities contributed to this difference. The hazard rate for Non-Hispanic Black cancer patients dying from COVID-19 was 6.46 times than that of Non-Hispanic White patients. The CCI accounted for 12.7% of the differences observed in COVID-19 mortality, with renal disease as the top contributor, explaining 4.9%. These findings could help develop interventions to reduce COVID-19 mortality and address the disproportionate impact, especially by managing chronic conditions like renal disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95037-3" target="_blank">https://www.nature.com/articles/s41598-025-95037-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s41586-025-08651-6" target="_blank">
            ü§ñ Small Molecule Stops COVID-19 from Entering Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-46', this)">
            More</button></h4>
            <div id="toggle-46" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A small-molecule SARS-CoV-2 inhibitor targeting the membrane protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists found a way to stop coronaviruses from spreading by targeting their main protein. They created a special molecule called JNJ-9676 that can bind to this protein and stop it from working properly. When tested on mice, JNJ-9676 showed that it could greatly reduce the amount of virus in their bodies and even kill the virus completely. This is good news because it might help us create new treatments for coronavirus infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The membrane (M) protein of betacoronaviruses is well conserved and has a key role in viral assembly1,2. Here we describe the identification of JNJ-9676, a small-molecule inhibitor targeting the coronavirus M protein. JNJ-9676 demonstrates in vitro nanomolar antiviral activity against SARS-CoV-2, SARS-CoV and sarbecovirus strains from bat and pangolin zoonotic origin. Using cryogenic electron microscopy¬†(cryo-EM), we determined a binding pocket of JNJ-9676 formed by the transmembrane domains of the M protein dimer. Compound binding stabilized the M protein dimer in an altered conformational state between its long and short forms, preventing the release of infectious virus. In a pre-exposure Syrian golden hamster model, JNJ-9676 (25‚Äâmg per kg twice per day) showed excellent efficacy, illustrated by a significant reduction in viral load and infectious virus in the lung by 3.5 and 4 log10-transformed RNA copies and 50% tissue culture infective dose (TCID50) per mg lung, respectively. Histopathology scores at this dose were reduced to the baseline. In a post-exposure hamster model, JNJ-9676 was efficacious at 75‚Äâmg per kg twice per day even when added at 48‚Äâh after infection, when peak viral loads were observed. The M protein is an attractive antiviral target to block coronavirus replication, and JNJ-9676 represents an interesting chemical series towards identifying clinical candidates addressing the current and future coronavirus pandemics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-025-08651-6" target="_blank">https://www.nature.com/articles/s41586-025-08651-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s41586-025-08773-x" target="_blank">
            ü§ñ Coronavirus Treatment: Targeting the Virus's Outer Layer</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-47', this)">
            More</button></h4>
            <div id="toggle-47" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A coronavirus assembly inhibitor that targets the viral membrane protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract simplified for a young person:

We found a special medicine called CIM-834 that stops the coronavirus from making more copies of itself. This medicine works by targeting a protein on the surface of the virus that helps it assemble new particles. We tested this medicine on mice and hamsters infected with SARS-CoV-2, and it was able to stop the virus from spreading and even prevented transmission. Our research discovered a new way to target and treat coronaviruses, which could help in fighting against these diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus membrane protein (M) is the main organizer of coronavirus assembly1,2,3. Here, we report on an M-targeting molecule, CIM-834, that blocks the assembly of SARS-CoV-2. CIM-834 was obtained through high-throughput phenotypic antiviral screening followed by medicinal-chemistry efforts and target elucidation. CIM-834 inhibits the replication of SARS-CoV-2 (including a broad panel of variants) and SARS-CoV. In SCID mice and Syrian hamsters intranasally infected with SARS-CoV-2, oral treatment reduced lung viral titres to nearly undetectable levels, even (as shown in mice) when treatment was delayed until 24‚Äâh before the end point. Treatment of infected hamsters prevented transmission to untreated sentinels. Transmission electron microscopy studies show that virion assembly is completely absent in cells treated with CIM-834. Single-particle cryo-electron microscopy reveals that CIM-834 binds and stabilizes the M protein in its short form, thereby preventing the conformational switch to the long form, which is required for successful particle assembly. In conclusion, we have discovered a new druggable target in the replication cycle of coronaviruses and a small molecule that potently inhibits it.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-025-08773-x" target="_blank">https://www.nature.com/articles/s41586-025-08773-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s41467-025-58327-y" target="_blank">
            ü§ñ Effectiveness of an Additional COVID Vaccine Dose in Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-48', this)">
            More</button></h4>
            <div id="toggle-48" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effectiveness of a booster dose of aerosolized or intramuscular adenovirus type 5 vectored COVID-19 vaccine in adults: a multicenter, partially randomized, platform trial in China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler terms:

We did an experiment to see if giving people a booster shot of a special vaccine for COVID-19 made them less likely to get sick. We gave 4089 people one of two types of vaccine shots: one was breathed into their lungs, and the other was injected into their muscle. Some people who didn't want a second shot got the other kind instead. We followed everyone's progress for six months to see if they got COVID-19. The results showed that getting the lung-shots gave us 52% fewer cases of COVID-19, but the muscle-injections weren't as effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We conducted a multicenter, partially randomized, platform trial to assess the effectiveness of a booster dose of an aerosolized or intramuscular adenovirus type 5 vectored COVID-19 vaccine (Ad5-nCoV) in Chinese adults (NCT05855408). Between May 23, 2023, and August 28, 2023, 4089 eligible participants were equally randomized to receive either a booster dose of aerosolized Ad5-nCoV via oral inhalation at 0.1‚ÄâmL (IH Ad5-nCoV, n‚Äâ=‚Äâ2039) or an intramuscular injection of Ad5-nCoV at 0.5‚ÄâmL (IM Ad5-nCoV, n‚Äâ=‚Äâ2050). Additionally, 2008 participants who declined the booster but consented to participate in COVID-19 surveillance were enrolled in the control group. All participants were monitored for symptomatic COVID-19 over a six-month surveillance period for the primary outcome. From 14 days after the vaccination, 14 (15/1000 person-years), 19 (20/1000 person-years), and 34 (37/1000 person-years) COVID-19 cases were confirmed in the IH Ad5-nCoV group, the IM Ad5-nCoV group, and the control group, respectively, which resulted in an adjusted effectiveness of 52.3% (95% CI 10.4 to 74.6) for IH Ad5-nCoV and 37.2% (95% CI -11.2 to 64.5) for IM Ad5-nCoV. The IH Ad5-nCoV booster was associated with a lower incidence of symptomatic COVID-19, but there is no solid evidence that IH Ad5-nCoV was more effective than IM Ad5-nCoV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-58327-y" target="_blank">https://www.nature.com/articles/s41467-025-58327-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s42003-025-07933-z" target="_blank">
            ü§ñ SARS-CoV-2 Virus and Human Connection Map</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-49', this)">
            More</button></h4>
            <div id="toggle-49" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multimodal SARS-CoV-2 interactome sketches the virus-host spatial organization</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists created a 3D model of how proteins in a coronavirus (SARS-CoV-2) work together inside a cell. They used data from the way these proteins interact with each other to build this model. The model shows where viral proteins are in relation to cellular parts, which helps us understand how viruses take over cells.

The study also found important complexes between viral and human proteins, which can help develop new drugs. This 3D model is an important step towards creating realistic models of how proteins interact, which can be used to fight emerging infectious diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>An accurate spatial representation of protein-protein interaction networks is needed to achieve a realistic and biologically relevant representation of interactomes. Here, we leveraged the spatial information included in Proximity-Dependent Biotin Identification (BioID) interactomes of SARS-CoV-2 proteins to calculate weighted distances and model the organization of the SARS-CoV-2-human interactome in three dimensions (3D) within a cell-like volume. Cell regions with viral occupancy were highlighted, along with the coordination of viral proteins exploiting the cellular machinery. Profiling physical intra-virus and virus-host contacts enabled us to demonstrate both the accuracy and the predictive value of our 3D map for direct interactions, meaning that proteins in closer proximity tend to interact physically. Several functionally important virus-host complexes were detected, and robust structural models were obtained, opening the way to structure-directed drug discovery screens. This PPI discovery pipeline approach brings us closer to a realistic spatial representation of interactomes, which, when applied to viruses or other pathogens, can provide significant information for infection. Thus, it represents a promising tool for coping with emerging infectious diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07933-z" target="_blank">https://www.nature.com/articles/s42003-025-07933-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-26</strong> <a href="https://www.nature.com/articles/s41467-025-58328-x" target="_blank">
            ü§ñ Monkeypox Vaccine Works by Boosting Both Body's Defenses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-50', this)">
            More</button></h4>
            <div id="toggle-50" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An mRNA vaccine against monkeypox virus inhibits infection by co-activation of humoral and cellular immune responses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Monkeypox viruses are causing many outbreaks, so we need more vaccines. We studied which parts of the virus can trigger an immune response and found 12 important proteins that work well together to give protection. We also discovered that by combining these proteins in certain ways, like Mix-4, Mix-8, or Mix-12, we get better protection. Our study showed that a new type of vaccine that targets both the body's humoral and cellular immune systems can give complete protection against monkeypox.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The persistent monkeypox outbreaks intensify the demand for monkeypox vaccines. Based on the mRNA vaccine platform, we conduct a systematic screening of monkeypox virus (MPXV) surface proteins from two types of viral particles, extracellular enveloped viruses (EVs) and intracellular mature viruses (MVs). This screening unveils 12 important antigens with diverse levels of neutralizing immunogenicity. Further assessment reveals that the combinations of 4, 8, and 12 of these antigens, namely Mix-4, Mix-8, and Mix-12, induce varying degrees of immune protection, with Mix-12 being the most potent. This finding demonstrates the significance of not only the level but also the diversity of the neutralizing antibodies in providing potent immune protection. Additionally, we utilize a T cell-epitope enrichment strategy, analyzing the complete proteome sequence of the MPXV to predict antigenic epitope-rich regions. Integration of these epitope-rich regions into a cellular immune-targeting antigen, named MPX-EPs, showcases that a cellular immune-targeting mRNA vaccine can independently confer immune protection. Furthermore, co-immunization with Mix-12 and MPX-EPs achieves complete protection against MPXV challenge. Overall, these results suggest an effective approach to enhance the immune protection of mRNA vaccines through the specific coordination of humoral and cellular immune responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-58328-x" target="_blank">https://www.nature.com/articles/s41467-025-58328-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41467-025-58006-y" target="_blank">
            ü§ñ Vaccine that uses tiny particles to help the body fight off different kinds of viruses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-51', this)">
            More</button></h4>
            <div id="toggle-51" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Kinetically activating nanovaccine mimicking multidimensional immunomodulation of natural infection for broad protection against heterologous viruses in animal models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists are trying to make better vaccines that can protect us from different kinds of viruses. Right now, vaccines work well for some viruses but not others. They came up with a new idea called KE-VAC, which is like a super-powered vaccine. It helps our immune system fight off more types of viruses and stays strong for longer. This new vaccine works even better than other vaccines we have now, and it can protect us from many different strains of the flu virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Immunity by vaccination can protect human against heterologous viruses. However, protective abilities of artificial vaccines are still weaker than natural infections. Here we develop a kinetically engineered vaccine (KE-VAC) that mimics the multidimensional immunomodulation in natural infections via dynamic activation of antigen presenting cells with masked TLR7/8 agonist and sustained supplies of antigens and adjuvants to lymph nodes, leading to follicular helper T and germinal centre B cell activation in vaccinated mice. KE-VAC demonstrates superior efficacy than traditional alum and mRNA vaccines, achieving a 100% survival rate with increased neutralizing antibodies titers and polyfunctional CD8+T cells, recognizing heterologous SARS-CoV-2 variants, and inducing broad and long-term protection against multiple strains of influenza viruses. Prime/boost vaccination with KE-VAC also protect aged ferrets from severe fever with thrombocytopenia syndrome virus infection, with no virus detected in any organs at day 6 p.i. The efficacy of KE-VAC across various pathogens thus highlights its potential as an effective vaccine against emerging infectious risks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-58006-y" target="_blank">https://www.nature.com/articles/s41467-025-58006-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41533-025-00424-y" target="_blank">
            ü§ñ Hospital Patients Get Sick with Many Conditions After Being Admitted for COVID or Flu</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-52', this)">
            More</button></h4>
            <div id="toggle-52" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multimorbidity incidence following hospitalization for SARS-CoV-1 infection or influenza over two decades: a territory-wide retrospective cohort study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Imagine you get very sick with SARS or the flu. When you get better, some people worry that you might get other health problems later. A scientist studied what happens to people who were really sick with either SARS or the flu, and wanted to know if those who got SARS would have more trouble getting other illnesses.

The study looked at thousands of people in Hong Kong who had been hospitalized for one of these two illnesses. The scientists found that:

- People who got SARS didn't get as many new health problems as those who got the flu.
- Some new health issues did come up, but they happened to different groups over time.

The scientist thought that getting really sick with either illness was not making people more likely to have trouble with their health later on.</p><br>
                <p><strong>Abstract</strong></p>
                <p>An infection of SARS-CoV-1, the causative agent of Severe Acute Respiratory Syndrome (SARS), may be followed by long-term clinical sequala. We hypothesized a greater 20-year multimorbidity incidence in people hospitalized for SARS-CoV-1 infection than those for influenza during similar periods. We conducted a retrospective cohort study using a territory-wide public healthcare database in Hong Kong. All patients aged ‚â•15 hospitalized for SARS in 2003 or influenza in 2002 or 2004 with no more than one of 30 listed chronic disease were included. Demographics, clinical history, and medication use were adjusted for in the inverse-probability-of-treatment-weighted Poisson regression analyses. We identified 1255 hospitalizations for SARS-CoV-1 infection and 687 hospitalizations for influenza. Overall crude multimorbidity incident rates were 1.5 per 100 person-years among SARS patients and 5.6 among influenza patients. Adjusted multimorbidity incidence rate ratio (IRR) was estimated at 0.78 [95% confidence interval (CI), 0.70‚Äì0.86) for SARS patients compared with influenza patients. Analysis by follow-up period shows a potentially greater risk among SARS patients in the first year of follow-up (IRR 1.33, 95% CI 0.97‚Äì1.84), with the risk in influenza patients increasing in subsequent years. Subgroup analyses by age and sex showed consistent results with the main analysis that SARS-CoV-1 infection was not followed by a higher incidence of multimorbidity than influenza. Notable differences in the patterns of multimorbidity were identified between the two arms. To conclude, we found no evidence of a higher multimorbidity incidence after hospitalization for SARS than for influenza over the long-term.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41533-025-00424-y" target="_blank">https://www.nature.com/articles/s41533-025-00424-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41598-024-81864-3" target="_blank">
            ü§ñ Oral Bacteria Comparison in People with COVID-19 and Those Who Are Healthy</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-53', this)">
            More</button></h4>
            <div id="toggle-53" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative metagenomic analysis of the oral microbiome in COVID-19 patients and healthy individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

COVID-19 can affect how our mouths work, making it harder for good bacteria to live in balance with bad ones. To understand this, scientists looked at the mouth germs of people who had COVID-19. They found that the mix of good and bad germs changed a lot, leading to lower numbers of helpful microbes and more of opportunistic pathogens. This could be related to long-term health problems for people who have had COVID-19. By studying this, scientists can find new ways to help diagnose and treat infections. Further research is needed to figure out how these changes affect people in the long run.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19, caused by SARS-CoV-2, affects multiple body systems, including the oral cavity, where it may disrupt the oral microbiome in ways that contribute to disease pathology. Understanding the long-term interaction between SARS-CoV-2 and the oral microbiome is crucial, as it may reveal microbial markers valuable for diagnosing or monitoring persistent health issues in COVID-19 survivors. Metagenomic sequencing revealed significant microbial shifts in the oral microbiome of COVID-19 patients, showing reduced microbial diversity and increased prevalence of opportunistic pathogens compared to healthy individuals. Alpha diversity measures indicated lower microbial diversity and evenness, while beta diversity analyses demonstrated distinct microbial community compositions. Core microbiome analysis identified unique taxa in COVID-19 patients that may contribute to disease pathology, while differential abundance analysis highlighted specific taxa shifts, including an increase in potential pathogens. Our findings advance the understanding of microbial changes in the oral microbiome associated with COVID-19 and suggest potential targets for microbiome-based interventions. While these results indicate associations with possible health impacts, further research is needed to determine causative links and long-term implications for COVID-19 survivors. This foundational research highlights the potential for microbiome science to inform diagnostic tools, such as microbial markers for disease progression, and therapeutic approaches, including targeted probiotics, which could ultimately support better patient outcomes and public health strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81864-3" target="_blank">https://www.nature.com/articles/s41598-024-81864-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41598-025-95140-5" target="_blank">
            ü§ñ Comparing the Spread of Two COVID-19 Variants Over Time</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-54', this)">
            More</button></h4>
            <div id="toggle-54" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative spatial‚Äìtemporal analysis of SARS-CoV-2 lineages B.1.1.33 and BQ.1.1 Omicron variant across pandemic phases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

The COVID-19 pandemic has created new challenges as it spreads to different parts of the world. This study looks at how the virus spread in Brazil's state of Espirito Santo, which is an important hub for trade due to its busy ports. Even though the state has a relatively small population, it had more cases and deaths than other places in the country. The researchers found that the early days of the pandemic were dominated by one type of virus, but as time went on, another type became more common. This new type spread quickly around the world and showed different ways of spreading compared to the earlier days. The study suggests that public health officials need to adapt their strategies to understand how the virus spreads and to monitor its evolution.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The evolution of COVID-19 pandemic has been characterized by the rapid emergence of new SARS-CoV-2 variants, each of which poses unique challenges to public health. This study analyzes the dispersion profiles during the Pre-Omicron and Post-Omicron phases in different epidemiological contexts. The Brazilian state of Espirito Santo, despite its low population density, plays a critical role as a commercial hub due to its intense port activity, which may have contributed to COVID-19 cases and mortality rates being higher than the national average. The state recorded 34,000 confirmed cases and 377 deaths per 100,000 inhabitants. Genomic surveillance revealed that the Pre-Omicron phase was dominated by the B.1.1.33 lineage, characterized by localized intraregional circulation. In contrast, the Post-Omicron phase, dominated by the BQ.1.1 lineage, exhibited greater diversity in circulating lineages, increased international interactions, and rapid viral dissemination, highlighting distinct transmission dynamics between such periods. This study highlights the need for adaptive public health strategies that account for both viral behavior and regional socioeconomic factors, while highlighting the strategic importance of Espirito Santo in monitoring SARS-CoV-2 evolution.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-95140-5" target="_blank">https://www.nature.com/articles/s41598-025-95140-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41598-025-92389-8" target="_blank">
            ü§ñ Analysis of Antibodies for COVID-19 in Human Blood</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-55', this)">
            More</button></h4>
            <div id="toggle-55" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Flow cytometric analysis of the SARS coronavirus 2 antibodies in human plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

COVID-19 is an illness caused by a virus called SARS-CoV-2. Our bodies make special proteins called antibodies that help fight off this virus. These antibodies can be used to see who has had COVID-19, even if they're not showing symptoms.

We developed a test to detect these antibodies in people's blood. We tested 100 people who hadn't had COVID-19 before and 100 people who had. The results showed that the people with COVID-19 had different types of antibodies in their blood than the people who didn't have it.

The types of antibodies that were most common were anti-RBD IgG, which is a key protein on the virus. We also found that older patients, those with severe symptoms, and men tended to have more of these antibodies.

This test can help doctors see if someone has had COVID-19 in the past and how their body reacted to it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 is an infectious disease caused by the severe acute respiratory syndrome coronavirus (SARS-CoV-2). Anti-SARS-CoV-2 antibodies can provide information on patient immunity, identify asymptomatic patients, and track the spread of COVID-19. Efforts have been made to develop methods to detect anti-SARS-CoV-2 antibodies in humans. Here, we describe a flow cytometric assay for the simultaneous detection of anti-SARS-CoV-2 IgG and IgM in human plasma. To assess the antibody response against the different SARS-CoV-2 structural proteins, five viral recombinant proteins, including spike protein subunit 1 (S1), N-terminal domain of S1 (S1A), spike receptor-binding domain (RBD), spike protein subunit 2 (S2), and nucleocapsid protein (N), were generated. A comparison of the antibody profiles detected by the assay with plasma from 100 healthy blood donors collected prior to the COVID-19 pandemic and plasma from 100 virologically confirmed COVID-19 patients demonstrated a clear discrimination between the two groups. Among the COVID-19 patients, the antibody responses for the viral proteins, as determined by their prevalence, were anti-RBD IgG‚Äâ=‚Äâanti-N IgG‚Äâ>‚Äâanti-S1 IgG‚Äâ>‚Äâanti-S1A IgG‚Äâ>‚Äâanti-S2 IgG, and anti-RBD IgM‚Äâ>‚Äâanti-S1 IgM‚Äâ>‚Äâanti-N IgM‚Äâ>‚Äâanti-S2 IgM. The prevalence of anti-SARS-CoV-2 IgG and IgM was not associated with sex, age, race, days after the onset of symptoms, or severity of illness, except for a higher prevalence of anti-S2 IgG being observed in men than in women. The levels of anti-RBD IgG were higher in patients 65¬†years and older and in patients who had severe symptoms. Similarly, patients who had severe symptoms exhibited higher levels of anti-S1 and anti-S1A IgG than patients who had mild or moderate symptoms. The levels of anti-RBD IgM tended to be higher in men but did not differ among age, race, days after the onset of symptoms, or severity of illness. Our study indicates that the flow cytometric assay, especially using RBD as target antigen, can be used to detect simultaneously anti-SARS-CoV-2 IgG and IgM antibodies in human plasma.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92389-8" target="_blank">https://www.nature.com/articles/s41598-025-92389-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-25</strong> <a href="https://www.nature.com/articles/s41598-025-94616-8" target="_blank">
            ü§ñ Characterizing a new vaccine that uses many different antigens at once.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-56', this)">
            More</button></h4>
            <div id="toggle-56" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel in-vitro expression assay by LC/MS/MS enables multi-antigen mRNA vaccine characterization</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

New technology has been developed to study messenger RNA (mRNA) vaccines in more detail. This technology combines two existing methods, flow cytometry and liquid chromatography mass spectrometry, to create a new way to analyze the proteins created by mRNA. The new method is antibody-free, can test multiple samples at once, and is highly sensitive and selective. It's being used to study different types of flu vaccines and improve their performance. This technology helps us understand how mRNA vaccines work better and make them more effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The new era of messenger RNA (mRNA) vaccines has led to development of a novel, state-of-the-art characterization method for this class of molecules. Currently, flow cytometry-based assays with antigen-specific antibodies are utilized for monitoring in-vitro expression (IVE) of mRNA. Here we present development, optimization, and application of an in-vitro expression liquid chromatography tandem mass spectrometry (IVE-LC/MS/MS) assay as an orthogonal method to IVE-flow cytometry that can be used for in-depth characterization of the expressed protein antigens and monitoring their relative expression levels in the cell post-mRNA transfection. The IVE-LC/MS/MS assessment accomplished the detection of influenza hemagglutinin (HA) antigens of four distinct strains simultaneously. The workflow is presented here, highlighting the optimization of all necessary steps required for protein purification and mass spectrometry method setup. The IVE-LC/MS/MS assay is a robust and versatile technique that complements the IVE-flow cytometry method and offers several advantages, such as being antibody-free, capable of multiplexing, and highly sensitive and selective. The various studies in this work, including evaluating dose‚Äìresponse relationships, refining transfection protocols, and examining mRNA-LNP stability under various conditions showcase the significant benefits of applying IVE-LC/MS/MS across different experimental settings. IVE-LC/MS/MS is a powerful tool for understanding and improving the performance and quality of mRNA LNPs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94616-8" target="_blank">https://www.nature.com/articles/s41598-025-94616-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-24</strong> <a href="https://www.nature.com/articles/s41467-025-57647-3" target="_blank">
            ü§ñ Molecular Testing Kit for Quick Disease Detection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-57', this)">
            More</button></h4>
            <div id="toggle-57" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Portable molecular diagnostic platform for rapid point-of-care detection of mpox and other diseases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The World Health Organization has warned about a new disease called mpox. It's spreading quickly around the world and we need to find fast and reliable ways to test for it. A team developed a new device called Dragonfly that can do this in just minutes. It's small, portable, and works well even without electricity. The device was tested on many samples and worked very well, showing that it can detect mpox with high accuracy. This is important because the world needs to be able to test for this disease quickly, especially since many people have weakened immunity from smallpox vaccinations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The World Health Organization‚Äôs designation of mpox as a public health emergency of international concern in August 2024 underscores the urgent need for effective diagnostic solutions to combat this escalating threat. The rapid global spread of clade II mpox, coupled with the sustained human-to-human transmission of the more virulent clade I mpox in the Democratic Republic of Congo, highlights a critical gap in point-of-care diagnostics for this emergent disease. In response, we developed Dragonfly, a portable molecular diagnostic platform for point-of-care use that integrates power-free nucleic acid extraction (<5‚Äâminutes) with lyophilised colourimetric LAMP chemistry. The platform demonstrated an analytical limit-of-detection of 100 genome copies per reaction for monkeypox virus, effectively distinguishing it from other orthopoxviruses, herpes simplex virus, and varicella-zoster virus. Clinical validation on 164 samples, including 51 mpox-positive cases, yielded 96.1% sensitivity and 100% specificity for orthopoxviruses, and 94.1% sensitivity and 100% specificity for monkeypox virus. Here, we present a rapid, accessible, and robust point-of-care diagnostic solution for mpox, suitable for both low- and high-resource settings, addressing the global resurgence of orthopoxviruses in the context of declining smallpox immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57647-3" target="_blank">https://www.nature.com/articles/s41467-025-57647-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-23</strong> <a href="https://www.nature.com/articles/s41598-025-94801-9" target="_blank">
            ü§ñ How Azithromycin Treats COVID-19 in People with Lung Diseases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-58', this)">
            More</button></h4>
            <div id="toggle-58" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A bioinformatics approach combined with experimental validation analyzes the efficacy of azithromycin in treating SARS-CoV-2 infection in patients with IPF and COPD These authors contributed equally: Yining Xie, Guangshu Chen, and Weiling Wu</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People with lung diseases like COPD and IPF are getting sick with COVID-19. We want to know if Azithromycin, an antibiotic, can help them feel better. 

We looked at genes in the lungs that are affected by both lung disease and COVID-19. We also looked at how Azithromycin works on these genes. Our results show that Azithromycin can stop some of the bad genes from working properly. This means it could be a good medicine to help people with lung diseases feel better when they get sick with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The swift transmission rate and unfavorable prognosis associated with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) have prompted the pursuit of more effective therapeutic interventions. Azithromycin (AZM) has garnered significant attention for its distinctive pharmacological mechanisms in the treatment of SARS-CoV-2. This study aims to elucidate the biological rationale for employing AZM in patients with chronic obstructive pulmonary disease (COPD) and idiopathic pulmonary fibrosis (IPF) who are infected with SARS-CoV-2. Genetic data about COVID-19, COPD, and IPF were independently obtained from the GeneCards database. And 40 drug targets about AZM were retrieved from the STITCH database. The analysis revealed that 311 DEGs were common among COPD, IPF, and COVID-19, and we further found eight genes that interacted with AZM targets. We conducted an analysis of hub genes and their corresponding signaling pathways in these patient cohorts. Additionally, we explored the inhibitory effects of AZM on these hub genes. AZM demonstrated a significant inhibitory effect on eight key genes, except for AR and IL-17¬†A. These findings suggest that AZM may serve as a promising therapeutic agent for patients with COPD and IPF and SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94801-9" target="_blank">https://www.nature.com/articles/s41598-025-94801-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-22</strong> <a href="https://www.nature.com/articles/s42003-025-07827-0" target="_blank">
            ü§ñ How to identify and use antibodies to fight SARS-CoV-2 variants.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-59', this)">
            More</button></h4>
            <div id="toggle-59" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structural and virological identification of neutralizing antibody footprint provides insights into therapeutic antibody design against SARS-CoV-2 variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Scientists found that a special type of antibody, called NT-108, can greatly help people with COVID-19 by making their symptoms better. To understand how it works, they studied it in hamsters and took a picture of it using a super powerful microscope called cryo-EM. They discovered that when it binds to the virus, it stops it from working properly. This helps us learn how to create even better antibodies to fight the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Medical treatments using potent neutralizing SARS-CoV-2 antibodies have achieved remarkable improvements in clinical symptoms, changing the situation for the severity of COVID-19 patients. We previously reported an antibody, NT-108 with potent neutralizing activity. However, the structural and functional basis for the neutralizing activity of NT-108 has not yet been understood. Here, we demonstrated the therapeutic effects of NT-108 in a hamster model and its protective effects at low doses. Furthermore, we determined the cryo-EM structure of NT-108 in complex with SARS-CoV-2 spike. The single-chain Fv construction of NT-108 improved the cryo-EM maps because of the prevention of preferred orientations induced by Fab orientation. The footprints of NT-108 illuminated how escape mutations such as E484K evade from class 2 antibody recognition without ACE2 affinity attenuation. The functional and structural basis for the potent neutralizing activity of NT-108 provides insights into the rational design of therapeutic antibodies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07827-0" target="_blank">https://www.nature.com/articles/s42003-025-07827-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s41598-025-94092-0" target="_blank">
            ü§ñ Adverse Pregnancy Outcomes in Louisiana from 2017 to 2022</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-60', this)">
            More</button></h4>
            <div id="toggle-60" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Trends in adverse pregnancy outcomes in Louisiana, 2017 to 2022</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

During the COVID-19 pandemic, many people got pregnant or tried to get pregnant. This can affect their health during pregnancy and after having a baby. Researchers looked at birth records in Louisiana from 2017 to 2022 to see if there were changes in how pregnancies went. They found that some women who got pregnant during the early part of the pandemic were more likely to have high blood pressure, preeclampsia, and other health problems during pregnancy.

The study suggests that the stress of the pandemic may have affected some women's health during pregnancy, especially if they got pregnant very early in their period. The researchers think that supporting these women with extra care and attention can help prevent serious health problems for both them and their babies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Natural disasters can lead to more adverse pregnancy outcomes (APO). It is unclear if the extended COVID-19 pandemic has impacted APOs and pre-existing conditions among perinatal populations with increased risk of severe maternal morbidity and mortality.A retrospective chart review was conducted of hospital records and birth certificates in the largest birth hospital in Louisiana from 2017 to 2022. Amongst 27,877 births (50.9% White, 38.3% Black, 28.9‚Äâ¬±‚Äâ5.6¬†years), gestational diabetes (GDM) was lowest in pre-pandemic conceptions (11.0%, June 2017-May 2019) and rose to 16.4% early pandemic (October 2019-February 2020) but leveled off at 12.2% in peak (March 2020-February 2021) and late pandemic (March 2021-September 2021). Individuals who conceived in early and peak pandemic were 47% (95% CI 33, 63) and 11% (95% CI 2, 20) more likely to develop GDM respectively, compared to pre-pandemic conceptions. Individuals who delivered during early (aRR: 1.54, 95% CI 1.33‚Äì1.78), peak (aRR: 1.48, 95% CI 1.32‚Äì1.65), and late (aRR: 1.62, 95% CI 1.41, 1.85) pandemic were more likely to develop preeclampsia and HELLP syndrome compared to pre-pandemic conceptions. Individuals were also 17% (95% CI 5, 32) more likely to enter pregnancy with chronic hypertension in peak pandemic compared to pre-pandemic. In paired analysis (n=‚Äâ3390), individuals with a pandemic conception that occurred early pandemic had a higher risk of developing GDM compared to their pre-pandemic pregnancy (aOR 3.26, 95% CI 1.52, 6.97). Supporting birthing individuals amongst significant stressful events, especially in early gestation, is critical for preventing APOs and severe maternal morbidity and mortality.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94092-0" target="_blank">https://www.nature.com/articles/s41598-025-94092-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s42003-025-07844-z" target="_blank">
            ü§ñ High-tech Sensor to Study Protein Interactions in Real-Time</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-61', this)">
            More</button></h4>
            <div id="toggle-61" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multiplexed proteomic biosensor platform for label-free real-time simultaneous kinetic screening of thousands of protein interactions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists need to study how proteins interact with each other to develop new medicines and diagnostic tools. But making this happen is expensive and time-consuming. We created a machine that can make thousands of different proteins on tiny chips, allowing us to quickly measure how they interact. This technology, called SPOC, uses special sensors to detect the interactions in real-time. It's like a super-efficient lab where we can test many proteins at once, which helps us understand protein interactions better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Studying protein interactions in high throughput (HTP) is critical for advancing many aspects of drug discovery, biomarker identification, and diagnostic development. However, existing methods for producing functional protein libraries are costly, time-consuming, and lack real-time kinetic screening capabilities. To address these limitations, we developed an automated platform for HTP production and screening of a library of proteins on biosensor surfaces, facilitating large-scale measurement of interaction kinetics. This technology, Sensor-Integrated Proteome On Chip (SPOC¬Æ), involves cell-free protein expression in nanowells directly from customizable libraries of DNA, facilitating simultaneous capture-purification of up to 2400 unique full-length folded proteins or proteoforms onto a single gold biosensor chip. SPOC protein biosensors can then be screened by real-time label-free analysis, including surface plasmon resonance (SPR) to generate kinetic data. Fluorescent and SPR assays were used to demonstrate zero crosstalk between protein spots. The functionality of SPOC proteins was validated using a variety of assays. Monoclonal antibodies were found to selectively bind their SPOC protein targets and anti-RBD antibody was used to demonstrate discriminated binding kinetics to numerous SARS-CoV-2 RBD variants on-chip. With advantages of HTP, flexibility, low-cost, and real-time kinetic profiling, the SPOC platform addresses challenges of interrogating protein interactions at scale for research and clinical applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07844-z" target="_blank">https://www.nature.com/articles/s42003-025-07844-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s41598-025-91811-5" target="_blank">
            ü§ñ Forecasting Epidemic Spread with Real-Time Data</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-62', this)">
            More</button></h4>
            <div id="toggle-62" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel approach to forecasting reproduction numbers of spatiotemporal stochastic epidemic spread using a PDE-based model and real-time infection data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic showed that predicting how fast diseases spread is very hard. We need better ways to make predictions so we can stop the spread. A new computer model was created to help estimate how many people are infected and who is likely to get sick next. The model worked really well with real data from a county in Ohio, USA, and matched results from other methods used by epidemiologists. This means our new model could be useful for predicting the spread of future diseases, especially in areas like counties where it's hard to get accurate data quickly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic highlighted the need for improved epidemic spread forecasting, a critical precursor for developing optimal control measures for spread mitigation. Well-recognized shortcomings in computing basic and effective reproduction numbers (\(\mathscr {R}_0\),\(\mathscr {R}_e\))-fundamental metrics for forecasting-underscore the need for new methods for estimating them from available data. We present a novel computational framework for estimating reproduction numbers from empirical spread data. The framework is derived from a mechanistic, spatiotemporal, Partial Differential Equation (PDE) model of epidemic spread utilizing mathematical results from PDE epidemic models. Forecasts of spatiotemporal effective reproduction number\(\mathscr {R}_e\)using the framework are found to be in excellent agreement with COVID-19 spread trends for Hamilton County, Ohio, USA, for three distinct periods. Furthermore, the forecasts are shown to align with corresponding reproduction numbers computed independently using the Wallinga-Teunis and Cori retrospective methods used in epidemiology. In summary, the results establish the validity of the framework and indicate applicability to future epidemics-especially for regions such as counties and for timeframes extending in weeks-even during dynamic phases when obtainable real-time infection spread data will likely be sparse.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91811-5" target="_blank">https://www.nature.com/articles/s41598-025-91811-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s41598-025-94600-2" target="_blank">
            ü§ñ Study of Leprosy Cases in Brazil After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-63', this)">
            More</button></h4>
            <div id="toggle-63" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An interrupted time series study of the leprosy case detection in Brazil after the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

The COVID-19 pandemic affected how quickly people in Brazil were diagnosed with leprosy from 2017 to 2022. The study looked at data on leprosy cases in Brazil and found that the pandemic made it harder to detect new cases, especially among children under 15.

In the early days of the pandemic, new leprosy cases slowed down by 0.55 per month. But after a while, the number of new cases started to increase again. 

The study also found that some areas in Brazil were doing better at detecting leprosy than others when it came to children under 15.

Because this happened, we need to make sure that there are strategies in place to help catch new leprosy cases quickly and prevent long-term problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 pandemics affected several health systems processes, including leprosy care. This study aimed to estimate the impact of the COVID-19 pandemic on the leprosy case detection rate in Brazil from 2017 to 2022. Data was retrieved from Sinan, a Brazilian notification system, and monthly leprosy detection rate in the overall population and in individuals under 15¬†years of age were the main outcomes. The series was interrupted in February 2020, when the COVID-19 public health emergency was declared in Brazil. The data were analysed via Prais‚ÄìWinsten regression. Over the 72¬†months analysed, the COVID-19 pandemic led to an immediate 0.55 reduction (95% CI 0.48‚Äì0.62) in the overall leprosy detection rate, with a subsequent monthly increase of 1.01 (95% CI 1.00‚Äì1.02). For the population under 15¬†years of age, the pandemic caused an immediate 0.48 reduction (95% CI 0.40‚Äì0.57), followed by a monthly increase of 1.01 (95% CI 1.01‚Äì1.02) after the onset of the pandemic. Subnational analysis revealed that most federative units followed the trend for the overall detection rate, but high heterogeneity was observed regarding individuals under 15¬†years of age. Therefore, it is urgent to target strategies to minimize delayed diagnosis and long-term consequences of leprosy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94600-2" target="_blank">https://www.nature.com/articles/s41598-025-94600-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s41598-025-92304-1" target="_blank">
            ü§ñ COVID-19: Levels of Certain Growth Factors in Blood and Skin</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-64', this)">
            More</button></h4>
            <div id="toggle-64" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Determination of both the expression and serum levels of epidermal growth factor and transforming growth factor Œ≤1 genes in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People with COVID-19 were studied to see how different factors affect their bodies when they get worse sick. There were 45 people, mostly men and women, who were divided into four groups based on how bad their symptoms were.

The researchers took blood samples from the patients and looked at things like:

* Platelet count (how many tiny cells in the blood)
* C-reactive protein (a marker of inflammation)
* Ferritin (a measure of inflammation)
* D-dimer (a marker of blood clotting)
* LDH (an enzyme that can be a sign of tissue damage)

They also looked at how much epidermal growth factor (EGF) and transforming growth factor-beta 1 (TGF-Œ≤1) were produced in the body.

The results showed that as people got worse sick, some things went up (like inflammation markers) while others went down (like EGF and TGF-Œ≤1 levels). This could help doctors understand how to manage COVID-19 infection severity and lung damage.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed to evaluate the effects of both the expression and serum levels ofEpidermal growth factor(EGF) andTransforming growth factor-Œ≤1(TGF-Œ≤1) genes in patients with different degrees of cellular damage as mild, moderate, severe, and critical illness that can lead to fibrosis caused by SARS-CoV-2. Totally 45 individuals (male: 21(46.67%); female: 24(53.33%)) with COVID-19 infection were included in this study. Four groups were constituted as mild (n‚Äâ=‚Äâ16)], moderate (n‚Äâ=‚Äâ10), severe (n‚Äâ=‚Äâ10), and critical (n‚Äâ=‚Äâ9) according to the severity of the disease. Blood samples were drawn from the patients, and all of the hemograms,EGFandTGFŒ≤1gene expression, and serum levels were evaluated. The mean age of individuals was 57.311‚Äâ¬±‚Äâ18.383 (min: 28, max: 94). Significant differences were found among the groups for PLT (œá2=‚Äâ9.955;p=‚Äâ0.019), CRP (œá2=‚Äâ7.693;p=‚Äâ0.053), Ferritin (œá2=‚Äâ22.196;p<‚Äâ0.001), D-dimer (œá2=‚Äâ21.982;p=‚Äâ0.000), LDH (œá2=‚Äâ21.807;p<‚Äâ0.001) and all these parameters (exclude PLT in severe groups) was increased depending on the severity of the disease. Additionally, significant differences were detected forEGF(œá2=‚Äâ29.528;p<‚Äâ0.001),TGFB1(œá2=‚Äâ28.981;p<‚Äâ0.001) expression (that increased depending on the disease severity), andEGF(œá2=‚Äâ7.84;p=‚Äâ0.049),TGFB1(œá2=‚Äâ17.451;p=‚Äâ0.001) serum concentration levels (that decreased depending on the disease severity). This study found statistically significant differences for bothEGF2‚àíŒîŒîCt.TGFŒ≤12‚àíŒîŒîCtandEGF,TGFŒ≤1serum concentration values among all patient groups. As disease severity increased,EGF2‚àíŒîŒîCt.TGFŒ≤12‚àíŒîŒîCtlevels increased, whileEGFandTGFŒ≤1serum concentration levels decreased.Perhaps this study will be useful in managing COVID-19 infection severity and pulmonary fibrosis cases secondary to COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92304-1" target="_blank">https://www.nature.com/articles/s41598-025-92304-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-21</strong> <a href="https://www.nature.com/articles/s41467-025-58173-y" target="_blank">
            ü§ñ Visualizing how COVID-19 spreads throughout the body using a special medical test.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-65', this)">
            More</button></h4>
            <div id="toggle-65" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Whole-body visualization of SARS-CoV-2 biodistribution in vivo by immunoPET imaging in non-human primates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Millions of people worldwide have been infected with COVID-19. While treatments and vaccines are helping, the virus is still spreading and changing. Scientists don't fully understand how it affects people in the long term, so they need more research. To study the virus without harming patients, researchers used a special imaging technique to track its movement in the body. In an experiment with monkeys, they found that even after recovery, some viruses stayed in the lungs and brain for months. This new method can help us better understand how COVID-19 affects people long-term.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has caused at least 780 million cases globally. While available treatments and vaccines have reduced the mortality rate, spread and evolution of the virus are ongoing processes. Despite extensive research, the long-term impact of SARS-CoV-2 infection is still poorly understood and requires further investigation. Routine analysis provides limited access to the tissues of patients, necessitating alternative approaches to investigate viral dissemination in the organism. We address this issue by implementing a whole-body in vivo imaging strategy to longitudinally assess the biodistribution of SARS-CoV-2. We demonstrate in a COVID-19 non-human primate model that a single injection of radiolabeled [89Zr]COVA1-27-DFO human monoclonal antibody targeting a preserved epitope of the SARS-CoV-2 spike protein allows longitudinal tracking of the virus by positron emission tomography with computed tomography (PET/CT). Convalescent animals exhibit a persistent [89Zr]COVA1-27-DFO PET signal in the lungs, as well as in the brain, three months following infection. This imaging approach also allows viral detection in various organs, including the airways and kidneys, of exposed animals during the acute infection phase. Overall, the technology we developed offers a comprehensive assessment of SARS-CoV-2 distribution in vivo and provides a promising approach for the non-invasive study of long-COVID pathophysiology.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-58173-y" target="_blank">https://www.nature.com/articles/s41467-025-58173-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-20</strong> <a href="https://www.nature.com/articles/s41598-025-92956-z" target="_blank">
            ü§ñ New SARS-CoV-2 Vaccine Creates Strong Immunity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-66', this)">
            More</button></h4>
            <div id="toggle-66" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Computationally designed multi-epitope vaccine construct targeting the SARS-CoV-2 spike protein elicits robust immune responses in silico</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We need to create a new kind of vaccine to fight against COVID-19 viruses that keep changing. Our team is designing a special vaccine called MEVC (Multi-Epitope Vaccine Construct) using a part of the virus's protein. We've used computers to find the right parts of the protein that will trigger a strong immune response in people.

Our MEVC has four different parts that work together, and we think it can help protect 90% of the world's population. It's been designed to be safe and not cause any allergic reactions. We've also tested it on computers to see if it will work well with our bodies' immune systems.

Now we need to test it in a lab to make sure it really works and is safe for people to use.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our research is driven by the need to design an advanced multi-epitope vaccine construct (MEVC) using the S-protein of SARS-CoV-2 to combat the emergence of new variants. Through rigorous computational screening, we have identified linear and discontinuous B-cell epitopes, CD8‚Äâ+‚Äâand CD4‚Äâ+‚ÄâT-cell epitopes, ensuring extensive MEVC coverage across 90.03% of the global population. The MEVC, featuring four CD4‚Äâ+‚Äâand four CD8‚Äâ+‚ÄâT-cell epitopes connected linearly with two adjuvant proteins on both ends, has been carefully designed to elicit robust immune response. Our in-silico analysis has confirmed the construct‚Äôs antigenicity, non-allergenicity, and non-toxicity with optimized codon sequences for enhanced expression in E. coli K12. Furthermore, molecular docking and dynamics analyses have demonstrated its strong binding affinity with TLR-3 and TLR 4, and in-silico immune simulation yielded promising results on heightened B-cell and T-cell-mediated immunity. However, wet lab experiments are essential to validate computational findings to revolutionize the development of vaccines against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92956-z" target="_blank">https://www.nature.com/articles/s41598-025-92956-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-20</strong> <a href="https://www.nature.com/articles/s41598-025-94938-7" target="_blank">
            ü§ñ Measuring COVID-19 virus from Thai wastewater to track how many people may have been infected.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-67', this)">
            More</button></h4>
            <div id="toggle-67" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Measuring SARS-CoV-2 RNA in Bangkok wastewater treatment plants and estimating infected population after fully opening the country in 2023, Thailand</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Wastewater scientists are trying to use wastewater to figure out if there are many people in Bangkok who have COVID-19. They collected water from 10 treatment plants and tested it for COVID-19. In most cases, they found that 51% of the wastewater contained COVID-19. When they looked at the amount of COVID-19 in the water, it went up over time - sometimes even before new cases appeared. The scientists also found a connection between COVID-19 in the water and how many people were getting sick each week. They think this might help them predict when new cases will start appearing.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Wastewater-based epidemiology (WBE) has been employed for monitoring the presence of SARS-CoV-2 infected population. Herein, the study aims to apply the WBE for surveillance and monitoring SARS-CoV-2 in Bangkok, where the highest official covid-19 cases reported in Thailand, during the fully opening for international tourists in early 2023. A total of 200 wastewater samples (100 influent and 100 effluent samples) were collected from 10 wastewater treatment plants (WWTPs) during January‚ÄìMay 2023. SARS-CoV-2 RNA was detected by real time qRT-PCR with accounting for 51% (102/200). Of these, 88% (88/100) and 14% (14/100) were detected in influent and effluent samples, respectively. The SARS-CoV-2 RNA concentration was detected in ranged of 4.76‚Äâ√ó‚Äâ102‚Äì1.48‚Äâ√ó‚Äâ105copies/L. The amount of SARS-CoV-2 RNA has increased approximately 4 times from the lag phase (January‚ÄìMarch) to the log phase (April‚ÄìMay). Spearman‚Äôs correlation coefficient revealed that correlation between estimated infected population and weekly reported cases was statistically significant (p-value‚Äâ=‚Äâ0.017). SARS-CoV-2 RNA in influent had a statistically significant relationship with weekly reported cases (r=‚Äâ0.481,p-value‚Äâ<‚Äâ0.001). Lag time analysis revealed early warning 1‚Äì3 weeks before rising covid-19 cases observed. GIS was applied for spatial-temporal analysis at the province level, suggesting real time dashboard should be further developed.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94938-7" target="_blank">https://www.nature.com/articles/s41598-025-94938-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41598-025-94428-w" target="_blank">
            ü§ñ Hematological cancers common in Africa</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-68', this)">
            More</button></h4>
            <div id="toggle-68" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence of hematological malignancies in Africa: A systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Many types of blood cancer are common in Africa, but no one has measured how widespread they are on the continent as a whole. This study looked at 34 articles about these cancers to see what percentage of people in Africa have them. The study found that 27% of people in Africa have some kind of blood cancer. Leukemia was the most common type, followed by lymphoma. Different countries and age groups had different rates, but overall, it's clear that there is a big problem with blood cancers in Africa.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hematological malignancies are a class of neoplasms that include a variety of diverse diseases that all develop from and change into lymphatic and bone marrow cells. Hematological malignancies significantly contribute to illness and mortality in African nations. The prevalence of these malignancies has not been evaluated in this continent. The purpose of this systematic review and meta-analysis was to assess the pooled prevalence of hematological malignancies in Africa. From October to November 2023, the electronic databases PubMed, Google Scholar, Web of Science, Research Gate, Embase, and Scopus were extensively searched to identify pertinent research. The Newcastle‚ÄíOttawa Quality Scale for cross-sectional studies was used to assess the quality of the included studies. The analysis tool used was STATA-14. To calculate the pooled prevalence of hematological malignancies, a random effects model was used. Heterogeneity was measured by using the I2value. Subgroup analysis was conducted for country, age of study subjects, population type, study design, and publication year. We evaluated publication bias through the implementation of a funnel plot and Egger‚Äôs test and conducted a sensitivity analysis. A total of 34 published articles including 43,099 study participants were included. The pooled prevalence of hematological malignancies was 27.30%. There was high heterogeneity, with an I2value of 99.2%. Leukemia had the highest pooled prevalence (53.69%) among the hematological malignancy types, followed by lymphoma (38.36%). According to subgroup analysis conducted in African countries, Kenya had the highest pooled prevalence (44.69%). On the other hand, the lowest pooled prevalence reported in Nigeria (20.52%). Furthermore, the age-based subgroup analysis of the study participants revealed that children had a greater pooled prevalence of hematological malignancies than adults (60.92% vs. 17.02%), respectively. In African populations, the pooled prevalence of hematological malignancies was 27.30%. This suggests that there is a significant prevalence of hematological malignancy, necessitating regular monitoring and accurate diagnosis.Trial registrationPROSPERO CRD42023427152.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94428-w" target="_blank">https://www.nature.com/articles/s41598-025-94428-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41598-025-94025-x" target="_blank">
            ü§ñ Factors that Influence How Young Workers Protect Themselves from New Infectious Diseases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-69', this)">
            More</button></h4>
            <div id="toggle-69" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Factors affecting emerging infectious disease prevention behaviors among young workers based on ecological modeling</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young people play an important role in preventing illnesses at work. This study looked at why some young workers are better at taking care of themselves to avoid getting sick than others.

The researchers studied 260 young workers in South Korea and found that:

- Healthy women were more likely to take care of themselves.
- Women who live with family members who have a serious illness are less likely to follow prevention rules.
- People who think they're more likely to get sick and people who believe they can control their chances of getting sick are more likely to follow prevention rules.

The study suggests that companies should help workers develop good habits for staying healthy by creating clear guidelines and programs. This will help them feel more confident in preventing illnesses before an outbreak happens.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This cross-sectional, descriptive survey study aimed to identify the factors affecting emerging infectious disease prevention behaviors among young workers aged‚Äâ<‚Äâ40 years. The factors affecting infection prevention behaviors among young workers were investigated with respect to perception of personal, inter-personal, organizational, and community responses based on ecological modeling. A total of 260 young workers residing in Jeollabuk-do region in South Korea were selected via convenience sampling. Multiple regression analysis performed to assess the factors affecting emerging infectious disease prevention behaviors. Multiple regression analysis revealed that infection prevention behaviors were significantly higher among healthy female participants compared with that in not very healthy female participants. Furthermore, infection prevention behaviors were significantly lower among those residing with family members with a confirmed diagnosis compare with that in those who did not. The infection prevention behaviors were higher among participants with higher levels of perceived susceptibility and perceived self-efficacy showed higher infection prevention behaviors. Thus, health managers should actively promote the implementation of infection prevention behaviors in the workplace by developing executable infection prevention guidelines and programs to enhance the perceived susceptibility and self-efficacy of workers in preparation for outbreaks of emerging infectious diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94025-x" target="_blank">https://www.nature.com/articles/s41598-025-94025-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41598-025-85733-5" target="_blank">
            ü§ñ Predicting How Bad Coronavirus is for Each Person Using Artificial Intelligence</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-70', this)">
            More</button></h4>
            <div id="toggle-70" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting coronavirus disease 2019 severity using explainable artificial intelligence techniques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

Researchers created a tool to predict how severe COVID-19 will be for new patients. They looked at data from 3,301 people who had COVID-19 between 2020 and 2022. They used special computer algorithms to find which features (like age, blood test results, etc.) were most important in predicting severity. They tested the tool on two groups of patients: one with early cases (before October 2020) and another with later cases (after October 2020). The tool was very accurate, especially when looking at just four key features. This simple model can help doctors make better decisions about treatment for COVID-19 patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Predictive models for determining coronavirus disease 2019 (COVID-19) severity have been established; however, the complexity of the interactions among factors limits the use of conventional statistical methods. This study aimed to establish a simple and accurate predictive model for COVID-19 severity using an explainable machine learning approach. A total of 3,301 patients‚Äâ‚â•‚Äâ18 years diagnosed with COVID-19 between February 2020 and October 2022 were included. The discovery cohort comprised patients whose disease onset fell before October 1, 2020 (N=‚Äâ1,023), and the validation cohort comprised the remaining patients (N=‚Äâ2,278). Pointwise linear and logistic regression models were used to extract 41 features. Reinforcement learning was used to generate a simple model with high predictive accuracy. The primary evaluation was the area under the receiver operating characteristic curve (AUC). The predictive model achieved an AUC of ‚â•‚Äâ0.905 using four features: serum albumin levels, lactate dehydrogenase levels, age, and neutrophil count. The highest AUC value was 0.906 (sensitivity, 0.842; specificity, 0.811) in the discovery cohort and 0.861 (sensitivity, 0.804; specificity, 0.675) in the validation cohort. Simple and well-structured predictive models were established, which may aid in patient management and the selection of therapeutic interventions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85733-5" target="_blank">https://www.nature.com/articles/s41598-025-85733-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41586-025-08728-2" target="_blank">
            ü§ñ How Somatic Mutations Help Make Antibodies Better</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-71', this)">
            More</button></h4>
            <div id="toggle-71" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Regulated somatic hypermutation enhances antibody affinity maturation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Our bodies have special places called germinal centers where our immune system improves its ability to fight off infections by making better antibodies. These antibodies are made through a process called somatic hypermutation, which introduces small changes into the genetic code.

The problem is that these random changes can sometimes make the antibodies worse, not better. To fix this, we need to balance the number of good mutations with bad ones. If some B cells (the cells that make antibodies) are making really good antibodies and dividing quickly, they might actually get fewer good mutations because their bodies are trying to protect them from getting too many bad changes.

We tested an idea about how to control this process in mice who were given special vaccines or a model antigen. We found that when the B cells made high-affinity antibodies, their cell cycles slowed down and they got fewer bad mutations. This helps keep good B cells healthy and makes our immune system better at making good antibodies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Germinal centres are specialized microenvironments where B‚Äâcells undergo affinity maturation. B‚Äâcells expressing antibodies whose affinity is improved by somatic hypermutation are selected for expansion by limiting numbers of T‚Äâfollicular helper cells. Cell division is accompanied by mutation of the immunoglobulin genes, at what is believed to be a fixed rate of around 1‚Äâ√ó‚Äâ10‚àí3per base pair per cell division1. As mutagenesis is random, the probability of acquiring deleterious mutations outweighs the probability of acquiring affinity-enhancing mutations. This effect might be heightened, and even become counterproductive, in B‚Äâcells that express high-affinity antibodies and undergo the greatest number of cell divisions2. Here we experimentally examine a theoretical model that explains how affinity maturation could be optimized by varying the rate of somatic hypermutation such that cells that express higher-affinity antibodies divide more but mutate less per division. Data obtained from mice immunized with SARS-CoV-2 vaccines¬†or a model antigen align with the theoretical model and show that cells producing high-affinity antibodies shorten the G0/G1 phases of the cell cycle and reduce their mutation rates. We propose that these mechanisms safeguard high-affinity B‚Äâcell lineages and enhance the outcomes of antibody affinity maturation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-025-08728-2" target="_blank">https://www.nature.com/articles/s41586-025-08728-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41599-025-04727-w" target="_blank">
            ü§ñ Understanding and analyzing health messages in China during the COVID-19 pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-72', this)">
            More</button></h4>
            <div id="toggle-72" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Navigating health communication in China: a corpus-based critical discourse analysis of COVID-19 news from 2020 to 2023</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

This study looked at how news about COVID-19 was covered in Chinese newspapers from 2020 to 2023. It analyzed over 37,000 news articles to see what words and ideas were used to describe the pandemic. The study found six main ways that the pandemic was represented: as a deadly global health crisis, a major disaster, China's response as successful, a national effort to fight it together, China's economy recovering well, and some politicians being unfair about the pandemic.

The study also looked at how these representations were connected to the politics and history of China. The findings can help healthcare workers and others communicate more effectively and sensitively about COVID-19 in different cultures.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study delves into the communication of the COVID-19 pandemic in China by examining the representation of the pandemic in Chinese news coverage from January 2020 to March 2023. Analyzing 37,484 news reports from leading English newspapers in China, the study employs a corpus-based Discourse-Historical Approach to uncover how specific lexical and discursive strategies construct representations in the Chinese media. Overall, six major representations have been identified: COVID-19 as a lethal global health crisis, the pandemic as an extensive disaster, China‚Äôs response as effective, the pandemic fight as a unifying war, steady Chinese economic recovery, and politicization of the pandemic as unethical. Further analysis connects the representations with the sociopolitical and historical context in China. The findings of the study offer insights that can inform precise and culturally sensitive communication strategies for health professionals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-025-04727-w" target="_blank">https://www.nature.com/articles/s41599-025-04727-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41598-025-86637-0" target="_blank">
            ü§ñ Vitamin B12 as a Treatment for Long COVID Symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-73', this)">
            More</button></h4>
            <div id="toggle-73" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Vitamin B12 as an epidrug for regulating peripheral blood biomarkers in long COVID-associated visuoconstructive deficit</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>After having a mild case of COVID-19, some people start to have problems with understanding and processing visual information. This is called visuoconstructive deficit or VCD.

Researchers found out that taking vitamin B12 can help reduce inflammation in the body and might even treat this problem. They took blood samples from people with persistent VCD and added different amounts of vitamin B12 to them. The results showed that vitamin B12 helped decrease inflammation and increased a protein called HGF, which is good for brain health.

This means that vitamin B12 could be a new way to help people with long COVID who have trouble understanding and processing visual information.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Approximately four months after recovering from a mild COVID-19 infection, around 25% of individuals developed visuoconstructive deficit (VCD), which was found to be correlated with an increase in peripheral immune markers and alterations in structural and metabolic brain imaging. Recently, it has been demonstrated that supplemental vitamin B12 regulates hyperinflammation during moderate and severe COVID-19 through methyl-dependent epigenetic mechanisms. Herein, whole peripheral blood cultures were produced using samples obtained from patients with confirmed persistent VCD, and controls without impairment, between 10 and 16 months after mild COVID-19. This experimental model was used to assess the leukocyte expression patterns of 11 biomarkers previously associated with VCD in long COVID and explore the potential of pharmacological B12 in regulating these genes. The results showed that patients with persistent VCD displayed continued upregulation ofCCL11andLIFcompared to controls. It is worth noting that elevated serum levels of CCL11 have been previously linked to age-related neurodegenerative diseases. Notably, the addition of 1 nM of vitamin B12 to blood cultures from individuals with VCD normalized the mRNA levels ofCCL11, upregulated the neuroprotectiveHGF, and, to a lesser extent, downregulatedCSF2andCXCL10. There was an inverse correlation observed betweenCCL11mRNA levels and methylation levels of specific cytosines in its promoter region. These findings underscore the significance of systemic inflammation in persistent VCD associated with long COVID. Moreover, the study provides evidence suggesting that B12, acting as an epidrug, shows promise as a therapeutic approach for addressing this cognitive impairment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86637-0" target="_blank">https://www.nature.com/articles/s41598-025-86637-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41598-025-94476-2" target="_blank">
            ü§ñ Glucose medicine stops virus from working in lungs</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-74', this)">
            More</button></h4>
            <div id="toggle-74" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nebulized 2-deoxylated glucose analogues inhibit respiratory viral infection in advanced in vitro airway models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Our goal is to create a new type of medicine that can fight viral infections in the lungs, which are often very hard to treat.

Researchers wanted to find out if three special chemicals (2-DGA) could stop viruses from working properly and reduce inflammation in the airways.

They used special models that mimic how the body's airways work. They found that these 2-DGA chemicals were effective against many types of viral infections, including those that cause colds, flu, and other respiratory problems.

The scientists also discovered that they could be delivered through a nebulizer, which is a machine that turns liquid medicine into a mist that can be breathed in.

Their research showed that having more advanced and realistic models for testing new medicines is very important to ensure their safety and effectiveness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory viral infections, such as those caused by rhinoviruses (RVs) and human corona viruses (HCoV), result in a serious strain on healthcare systems and public health, underscoring an urgent need for inhaled broad-spectrum antiviral therapies. However, their development is challenging, as no standardized in vitro methodologies that can fully replicate the in vivo environment have been established. In this work, we aimed to investigate the antiviral and anti-inflammatory effect of three 2-deoxylated glucose analogues (2-DGA): 2-deoxy-D-glucose, 2-fluoro-2-deoxy-D-glucose and 2-fluoro-2-deoxy-D-mannose (2-FDM), by utilizing advanced in vitro air-liquid interface (ALI) airway models. We demonstrated that commonly used ALI models have variable susceptibility to RV, HCoV and influenza A virus (IAV) infection. Further, we showed that 2-DGA have an anti-inflammatory effect and suppress respiratory viral replication in models mimicking the upper and lower respiratory airways. Moreover, we confirmed that 2-DGA can be delivered via nebulization in vitro, highlighting their potential to be used as broad-spectrum inhaled antivirals. Finally, our results demonstrate the importance of incorporating complex in vitro methodologies, such as primary cell ALI cultures and aerosol exposure, at an early stage of drug development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-94476-2" target="_blank">https://www.nature.com/articles/s41598-025-94476-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-19</strong> <a href="https://www.nature.com/articles/s41746-025-01456-x" target="_blank">
            ü§ñ People who had COVID-19 can be identified by unique patterns in their wearables.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-75', this)">
            More</button></h4>
            <div id="toggle-75" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Wearable data reveals distinct characteristics of individuals with persistent symptoms after a SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who get sick from COVID-19 sometimes still feel tired and breathe harder than usual even after they're no longer sick. Researchers wanted to know what's going on with their bodies and how it affects their lives.

They looked at the data from people who wore special devices that tracked their movements and feelings, like heart rate and fatigue levels. They found out that people who were still feeling tired and short of breath even after getting over COVID-19 had higher resting heart rates and lower activity levels compared to those who didn't get sick.

The study also showed that these symptoms can affect a person's quality of life long after the illness is gone. This suggests that some people may have underlying health issues or fitness levels that contribute to their lingering symptoms. Wearable devices could be useful in tracking changes in health and providing insights into how infectious diseases like COVID-19 affect people over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the factors associated with persistent symptoms after SARS-CoV-2 infection is critical to improving long-term health outcomes. Using a wearable-derived behavioral and physiological dataset (n=‚Äâ20,815), we identified individuals characterized by self-reported persistent fatigue and shortness of breath after SARS-CoV-2 infection. Compared with symptom-free COVID-19 positive (n‚Äâ=‚Äâ150) and negative controls (n=‚Äâ150), these individuals (n=‚Äâ50) had higher resting heart rates (mean difference 2.37/1.49 bpm) and lower daily step counts (mean 3030/2909 steps fewer), even at least three weekspriorto SARS-CoV-2 infection. In addition, persistent fatigue and shortness of breath were associated with a significant reduction in mean quality of life (WHO-5, EQ-5D), evenbeforeinfection. Here we show that persistent symptoms after SARS-CoV-2 infection may be associated with pre-existing lower fitness levels or health conditions. These findings additionally highlight the potential of wearable devices to track health dynamics and provide valuable insights into long-term outcomes of infectious diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-025-01456-x" target="_blank">https://www.nature.com/articles/s41746-025-01456-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-18</strong> <a href="https://www.nature.com/articles/s41598-025-93664-4" target="_blank">
            ü§ñ SARS-CoV-2 can leave some people with lasting changes to their hearing.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-76', this)">
            More</button></h4>
            <div id="toggle-76" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Altered auditory brainstem responses are post-acute sequela of SARS-CoV-2 (PASC)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

People who had COVID-19 are still experiencing symptoms months later, like feeling tired and having brain fog. Some people with Long-COVID also have tinnitus (ringing in their ears) and hearing problems. Scientists wanted to see if there's a link between these auditory problems and how hard it is for them to think clearly. They studied 82 people: 37 who had Long-COVID and 45 healthy people. The study found that:

- People with Long-COVID were more likely to have tinnitus, hearing problems, and feeling tired.
- Their brains took longer to process sound when they heard fast or slow clicks.
- When their ears rang (tinnitus), it made them feel even more tired.
- Younger people with Long-COVID had a problem with how well their brain processed sound, which is similar to what happens as we get older.

This study helps scientists find a new way to measure how hard it is for people to think clearly, using the way their ears work.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Post-acute Sequela of SARS-CoV-2 (PASC) syndrome, also known as Long-COVID, often presents with subjective symptoms such as brain fog and cognitive fatigue. Increased tinnitus, and decreased hearing in noise ability also occur with PASC, yet whether auditory manifestations of PASC are linked with the cognitive symptoms is not known. Electrophysiology, specifically the Auditory Brainstem Response (ABR), provides objective measures of auditory processing. We hypothesized that ABR findings would be linked to PASC and with subjective feelings of cognitive fatigue. Eighty-two individuals, 37 with PASC (mean age: 47.5, Female: 83%) and 45 healthy controls (mean age: 38.5, Female: 76%), were studied with an auditory test battery that included audiometry and ABR measures. Peripheral hearing thresholds did not differ between groups. The PASC group had a higher prevalence of tinnitus, anxiety, depression, and hearing handicap in addition to increased subjective cognitive fatigue. ABR latency findings showed a significantly greater increase in the wave V latency for PASC subjects when a fast (61.1 clicks/sec) compared to a slow click (21.1 clicks/sec) was used. The increase in latency correlated with cognitive fatigue scores and predicted PASC status. The ABR V/I amplitude ratio was examined as a measure of central gain. Although these ratios were not significantly elevated in the full PASC group, to minimize the cofounding effect of age, the cohort was median split on age. Elevated V/I amplitude ratios were significant predictors of both predicted PASC group classification and cognitive fatigue scores in the younger PASC subjects compared to age-matched controls providing evidence of elevated central gain in younger individuals with PASC. More frequent tinnitus also significantly predicted higher subjective cognitive fatigue scores. Our findings suggest that PASC may alter the central auditory pathway and lead to slower conduction and elevated auditory neurophysiology responses at the midbrain, a pattern associated with the typical aging process. This study marks a significant stride toward establishing an objective measure of subjective cognitive fatigue through assessment of the central auditory system.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-93664-4" target="_blank">https://www.nature.com/articles/s41598-025-93664-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-18</strong> <a href="https://www.nature.com/articles/s41467-025-57947-8" target="_blank">
            ü§ñ Boosting Protection Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-77', this)">
            More</button></h4>
            <div id="toggle-77" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Complement-mediated enhancement of SARS-CoV-2 antibody neutralisation potency in vaccinated individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists are trying to understand how our bodies can fight off a new coronavirus. They want to know what makes someone immune to the virus and if it will still work after we've had some time without being vaccinated.

They found that adding back something called the complement system, which helps fight off infections, makes the vaccine more effective. But this doesn't work for everyone, and different people respond differently. 

It also seems that taking out a part of the virus that's used to enter our cells can make it easier to kill. This is good news because it might help protect us against new versions of the coronavirus that are coming up.

Overall, scientists found some important clues about how to make vaccines and treatments more effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>With the continued emergence of SARS-CoV-2 variants and concerns of waning immunity, there is a need for better defined correlates of protection to aid future vaccine and therapeutic developments. Whilst neutralising antibody titres are associated with protection, these are typically determined in the absence of the complement system, which has the potential to enhance neutralisation titres and strengthen correlates with protection in vivo. Here we show that replenishment of the complement system in neutralisation assays can significantly enhance neutralisation titres, with up to an ~83-fold increase in neutralisation of the BA.1.1.529 strain using cross-reactive sera from vaccination against the ancestral strain. The magnitude of enhancement significantly varies between individuals, viral strains (wild-type/VIC01 and Omicron/BA.1), and cell lines (Vero E6 and Calu-3), and is abrogated following heat-inactivation of the complement source. Utilising ACE2 competition assays, we show that the mechanism of action is partially mediated by reducing ACE2-spike interactions. Through the addition of compstatin (a C3 inhibitor) to live virus neutralisation assays, the complement protein C3 is shown to be required for maximum efficiency. These findings further our understanding of SARS-CoV-2 immunity and neutralisation, with implications for protection against emerging variants and assessing future vaccine and therapeutic developments.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57947-8" target="_blank">https://www.nature.com/articles/s41467-025-57947-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-17</strong> <a href="https://www.nature.com/articles/s41598-025-93487-3" target="_blank">
            ü§ñ Global Malaria Impact: Before and After Covid</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-78', this)">
            More</button></h4>
            <div id="toggle-78" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Global burden of malaria before and after the COVID-19 pandemic based on the global burden of disease study 2021</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Malaria affects many people worldwide, especially in poor countries. Even though progress was made, the Covid-19 pandemic slowed down efforts to make it disappear globally.

Researchers used data from 2021 to see how malaria spread before and after the pandemic. They found that:

* The number of people getting malaria went up a little bit in some areas
* The number of deaths from malaria also increased
* Children under 5 years old and poor countries were affected the most

By 2030, there will be more cases of malaria, especially if nothing changes.

The Covid-19 pandemic has made it harder to control malaria.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Malaria poses significant public health challenges and huge disease and economic burdens across the world, notably in low-income countries. Although great strides have been achieved, the COVID-19 pandemic hinders the progress towards global elimination of malaria. This study utilizes data from the global burden of disease study 2021 data sources to assess trends in incidence and mortality of, and disability-adjusted life years (DALYs) lost due to malaria before and after the global COVID-19 pandemic, and projects the incidence of malaria in 2030 at international, regional and national levels. The age-standardized incidence of malaria declined from 3789.28 per 100,000 populations in 2010 to 3332.96 per 100,000 population in 2019, followed by a rapid increase to 3485.27 per 100,000 populations in 2021. The estimated annual percentage change was 2.26% (95% confidence interval: 1.84‚Äì2.68%) from 2019 to 2021, suggesting a significant acceleration in the increasing rate compared to previous years. The numbers of DALYs lost and death due to malaria also increased in 2021. Children under 5 years of age and regions with low socio-demographic index were disproportionately affected with the highest burden of malaria associated with the COVID-19 pandemic. The pandemic is projected to lead to an additional 472.59 malaria cases per 100,000 populations by 2030. The global COVID-19 pandemic has posed substantial challenges to the global malaria elimination program as revealed by increasing incidence, death and DALYs lost across the world.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-93487-3" target="_blank">https://www.nature.com/articles/s41598-025-93487-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-17</strong> <a href="https://www.nature.com/articles/s41541-025-01096-y" target="_blank">
            ü§ñ Randomized vaccine trial for kids ages 3-17 to test a nasal COVID-19 vaccine.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-79', this)">
            More</button></h4>
            <div id="toggle-79" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A randomized phase I trial of intranasal SARS-CoV-2 vaccine dNS1-RBD in children aged 3‚Äì17‚Äâyears</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This vaccine is a new way to protect against COVID-19 made using an old flu virus. A study was done to see if it's safe for kids from 3 to 17 years old. They got two doses of the vaccine or a fake placebo, and were monitored for any side effects over several months. The results showed that the vaccine didn't cause many problems and is likely safe for this age group.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The intranasal SARS-CoV-2 vaccine dNS1-RBD (Pneucolin¬Æ), based on a live-attenuated influenza virus vector, has obtained Emergency Use Authorization in China for individuals aged 18‚Äâyears and older. Here, we conducted a single-center, double-blind, placebo-controlled, age de-escalation phase 1 clinical trial to evaluate the safety of the dNS1-RBD in children aged 3‚Äì17‚Äâyears (ChiCTR2300068044). Sixty-three participants received 2 intranasal doses of the vaccine or placebo at days 0 and 14. Safety assessments included adverse events/reactions within 30 days and serious adverse events (SAEs) over 12‚Äâmonths. Blood and nasal secretion samples were collected to further monitor blood indices and viral shedding. The vaccine group showed similar adverse reaction rates to the placebo group (39.0% vs 36.4%), with no SAEs related to vaccination. Data suggested that the dNS1-RBD vaccine is well-tolerated in children aged 3‚Äì17‚Äâyears, and warrants further studies on its safety, immunogenicity and efficacy in this population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01096-y" target="_blank">https://www.nature.com/articles/s41541-025-01096-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-17</strong> <a href="https://www.nature.com/articles/s41541-025-01100-5" target="_blank">
            ü§ñ How Effective Are Early COVID Vaccines Against Severe Illness?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-80', this)">
            More</button></h4>
            <div id="toggle-80" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Differential efficacy of first licensed western vaccines protecting without immunopathogenesis Wuhan-1-challenged hamsters from severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Four COVID-19 vaccines were made and tested. Three are special kinds (mRNA-based) and one uses a different helper called adenovirus.

We created a group of hamsters that get very sick when they have COVID-19. We vaccinated them with the new vaccines, Alum+S, or their special helpers, and then we challenged them to see if they got sick again.

The vaccinated hamsters all had good immune systems and didn't get very sick. But some of them still got a little bit sick in their lungs. Only one type of vaccine helped keep them from getting really sick.

We also found out that the special helper (Alum+S) made them feel worse, not better. This helper is used in two other vaccines, but it didn't seem to work well in those ones either.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Four COVID-19 vaccines were developed, tested, and authorized early in Europe and the US. Comirnaty and Spikevax are mRNA-based, whereas Jcovden and Vaxzevria utilize adenoviral vectors (AdV). We described a hamster model of COVID-19 utilizing Wuhan-1 strain SARS-CoV-2, in which vaccine-associated immunopathogenesis can be induced by Alum-adjuvanted Spike protein (Alum+S). Such animals were vaccinated with the authorized vaccines or Alum+S, challenged, and examined. All vaccinated hamsters produced antibodies targeting S. Neutralizing antibodies (nAb) were induced only by authorized vaccines. While nAbs were present after one vaccination with AdV-vaccines, mRNA vaccines needed a boost immunization. Upon challenge, all authorized vaccines protected from severe disease. Less tissue damage and no live virus (one exception) were detectable in the lungs. In contrast, Alum+S immunized hamsters developed VAERD. Our data reveal the absence of induction of VAERD by early commercial vaccines in hamsters, while animals¬¥ immune responses and protection seem to match the clinical vaccine efficacy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01100-5" target="_blank">https://www.nature.com/articles/s41541-025-01100-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-14</strong> <a href="https://www.nature.com/articles/s41541-025-01093-1" target="_blank">
            ü§ñ A very stable medicine that uses a special delivery system to help your body fight off the herpes zoster virus.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-81', this)">
            More</button></h4>
            <div id="toggle-81" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A highly stable lyophilized mRNA vaccine for Herpes Zoster provides potent cellular and humoral responses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Shingles is a painful rash that happens when a virus called varicella-zoster reactivates in older people or those with weak immune systems. There's already a vaccine for shingles, but it can cause strong reactions. Scientists want to make a new vaccine that works better and is easier on the body.

They made 10 different vaccines using special instructions (called mRNA) that target a key part of the virus. When tested in mice, some of these vaccines showed great promise by triggering both types of immune responses (cellular and humoral) and producing strong antibodies against the virus. The new vaccine form is also stable and works as well as current vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Herpes zoster (HZ) is a painful vesicular rash that occurs upon varicella-zoster virus (VZV) reactivation in older adults and immunocompromised individuals. Although there is currently an approved vaccine for the prevention of shingles, its administration is commonly associated with high reactogenicity. This highlights the need to develop new vaccine alternatives with long lasting immunity and improved tolerability upon administration. In the present study, 10 different vaccine candidate designs using two different codon optimizations targeting the VZV glycoprotein E (gE) were generated. A subset of mRNA constructs were formulated into lipid nanoparticles and assessed for their ability to induce specific cellular and humoral immune responses following vaccination in mice. Notably, the selected mRNA vaccine candidates induced high levels of antibodies and robust CD4+but also CD8+immune responses. Moreover, we showed that our alternate lyophilized vaccine provides comparable immunogenicity to current liquid frozen formulations and is stable under long-term storage conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01093-1" target="_blank">https://www.nature.com/articles/s41541-025-01093-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-13</strong> <a href="https://www.nature.com/articles/s44298-025-00101-4" target="_blank">
            ü§ñ New Nanobodies Help Fight SARS-CoV-2 Variant</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-82', this)">
            More</button></h4>
            <div id="toggle-82" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Potent bivalent nanobody constructs that protect against the SARS-CoV-2 XBB variant</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists created special antibodies to fight the new, more contagious coronavirus variant Omicron (XBB). They made these antibodies in a way that lets them target and kill the virus. The best ones were tested in mice and showed promise as a potential treatment for this new strain of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Most antibody-based therapeutics approved for SARS-CoV-2 treatment have shown greatly reduced neutralization activity against emerging Omicron variants. To target recent Omicron variants, we developed XBB-specific antibody-like therapeutics by screening a yeast surface-displayed single-domain antibody library against the receptor binding domain of the XBB spike protein. Three lead nanobodies, XNb 4.13, XNb 4.14, and XNb 4.15, were selected based on their binding affinity to the XBB spike protein and were fused to a mouse Fc domain. While all three constructs showed sub-nanomolar binding affinities to the XBB S protein, XNb 4.13-Fc and XNb 4.14-Fc also neutralized XBB in vitro. Intraperitoneal injection of XNb 4.13-Fc and XNb 4.14-Fc and intranasal delivery of XNb 4.13-Fc protected transgenic mice from a challenge with XBB virus with a significant reduction in viral lung titers post-infection. These antibody-like constructs identified using yeast surface display have potential as therapeutics that can protect against SARS-CoV-2 XBB variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-025-00101-4" target="_blank">https://www.nature.com/articles/s44298-025-00101-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-12</strong> <a href="https://www.nature.com/articles/s41598-025-92991-w" target="_blank">
            ü§ñ Graphene Sensor for Detecting COVID-19 Using Infrared Light</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-83', this)">
            More</button></h4>
            <div id="toggle-83" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Graphene metasurfaces biosensor for COVID-19 detection in the infra-red regime</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new device was created to detect COVID-19. It uses special materials like graphene and gold to help identify the virus. The device can see two different parts of the virus at the same time, which makes it very good at detecting the virus. It also works really well with computers to make sure it's always accurate. This means we have a new way to quickly and accurately detect COVID-19, which is important for stopping the spread of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study presents the design and analysis of a biosensor for COVID-19 detection, integrating graphene metasurfaces with gold, silver, and GST materials. The proposed sensor architecture combines a square ring resonator with a circular ring resonator, optimized through COMSOL Multiphysics simulations in the infrared regime. The sensor demonstrates exceptional performance characteristics, with absorption values exceeding 99.5% in the primary detection band (4.2‚Äì4.6¬†Œºm) and approximately 97.5% in the secondary band (5.0‚Äì5.5¬†Œºm). The device exhibits high sensitivity (4000¬†nm/RIU), a detection limit of 0.078, and a figure of merit of 16.000 RIU‚Åª¬π when utilizing crystalline GST as the substrate material. The sensor‚Äôs performance was further enhanced through machine learning optimization using XGBoost regression, achieving perfect correlation (R¬≤ = 100%) between predicted and experimental values across various operational parameters. The dual-band detection mechanism, combined with the integration of advanced materials and machine learning optimization, offers a promising platform for rapid, label-free, and highly sensitive COVID-19 detection. This research contributes to the development of next-generation biosensing technologies for viral detection and disease diagnosis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92991-w" target="_blank">https://www.nature.com/articles/s41598-025-92991-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-12</strong> <a href="https://www.nature.com/articles/s41598-025-92908-7" target="_blank">
            ü§ñ Long COVID affects a part of the throat called the epipharynx, and researchers found ways to identify which signals from the virus are causing it. They also suggest that rubbing this area may help alleviate symptoms.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-84', this)">
            More</button></h4>
            <div id="toggle-84" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Spatial transcriptomics of the epipharynx in long COVID identifies SARS-CoV-2 signalling pathways and the therapeutic potential of epipharyngeal abrasive therapy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who got coronavirus are still getting sick months or even years later. This study found that the part of the body called the epipharynx is affected by the virus and can make people feel worse.

The researchers wanted to see if a special treatment for the epipharynx could help people feel better. They looked at the gene expression (how cells are making proteins) in patients who got coronavirus and had long-term symptoms, compared to healthy people.

They found that the patients had leftover virus in their epipharynx and were having trouble with inflammation and immune response. The treatment helped clear out the virus, reduce inflammation, and calm down the immune system. This study shows that long COVID might be caused by a problem with the epipharynx and that this treatment could help alleviate symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In this study, the critical role of the epipharynx in managing long-term coronavirus disease 2019 (COVID-19), and in particular, how residual SARS-CoV-2 RNA affects signalling pathways in the epipharynx were investigated via spatial gene expression analysis (Visium HD). Moreover, we hypothesize that epipharyngeal abrasive therapy (EAT) targeting the epipharynx could improve long COVID symptoms by modulating local inflammation and gene expression. We conducted a comparative analysis of the gene expression profiles of three patients with long COVID and two control individuals without COVID-19. Residual SARS-CoV-2 RNA was detected in the epipharynx of patients with long COVID, along with the activation of signalling pathways in epithelial and immune cells. After EAT, the viral RNA was either completely cleared or significantly reduced. T-cell receptor signalling pathways were suppressed; the levels of proinflammatory cytokines, such as interleukin-6 and tumour necrosis factor-Œ±, were reduced; and excessive antibody production was mitigated. Histology showed that EAT effectively eliminated the inflamed, dysfunctional ciliated epithelium. This study clarifies that SARS-CoV-2 has long-term effects on the immune response in the epipharynx, emphasizing the need to focus on chronic epipharyngitis as a potential cause of long COVID. Furthermore, EAT may offer a promising approach to alleviating persistent long COVID symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92908-7" target="_blank">https://www.nature.com/articles/s41598-025-92908-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-12</strong> <a href="https://www.nature.com/articles/s41586-025-08697-6" target="_blank">
            ü§ñ Tumor Necrosis Factor Links Epstein-Barr Virus to Rare Illness in Children</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-85', this)">
            More</button></h4>
            <div id="toggle-85" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>TGFŒ≤ links EBV to multisystem inflammatory syndrome in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an explanation of the abstract that a young person can understand:

When some people get very sick with coronavirus (COVID-19), their bodies react in a bad way and get into a state of high inflammation. This can be especially true for kids.

Scientists discovered that this reaction happens because the virus that causes COVID-19 affects how the immune system, which is meant to fight off infections, works. The problem is that the immune system gets stuck and can't work properly again.

The research found that a special protein called TGFŒ≤ makes the immune system not work right. When this protein is blocked, the immune system starts working again.

The study also showed that some kids who got sick with COVID-19 have an increased risk of getting reinfected with another virus called EBV (which causes mono). The research suggests that this might be because the TGFŒ≤ protein makes the immune system not work right.

Overall, the study shows that what happens in people's bodies when they get very sick with COVID-19 can lead to some serious problems and that understanding how the immune system works is important for finding new treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In a subset of children and adolescents, SARS-CoV-2 infection induces a severe acute hyperinflammatory shock1termed multisystem inflammatory syndrome in children (MIS-C) at four to eight weeks after infection. MIS-C is characterized by a specific T cell expansion2and systemic hyperinflammation3. The pathogenesis of MIS-C remains largely unknown. Here we show that acute MIS-C is characterized by impaired reactivation of virus-reactive memory T cells, which depends on increased serum levels of the cytokine TGFŒ≤ resembling those that occur during severe COVID-19 (refs.4,5). This functional impairment in T cell reactivity is accompanied by the presence of TGFŒ≤-response signatures in T cells, B cells and monocytes along with reduced antigen-presentation capabilities of monocytes, and can be reversed by blocking TGFŒ≤. Furthermore, T cell receptor repertoires of patients with MIS-C exhibit expansion of T cells expressing TCRVŒ≤21.3, resembling Epstein‚ÄìBarr virus (EBV)-reactive T cell clones capable of eliminating EBV-infected B cells. Additionally, serum TGFŒ≤ in patients with MIS-C can trigger EBV reactivation, which is reversible with TGFŒ≤ blockade. Clinically, the TGFŒ≤-induced defect in T cell reactivity correlates with a higher EBV seroprevalence in patients with MIS-C compared with age-matched controls, along with the occurrence of EBV reactivation. Our findings establish a connection between SARS-CoV-2 infection and COVID-19 sequelae in children, in which impaired T cell cytotoxicity triggered by TGFŒ≤ overproduction leads to EBV reactivation and subsequent hyperinflammation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-025-08697-6" target="_blank">https://www.nature.com/articles/s41586-025-08697-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-12</strong> <a href="https://www.nature.com/articles/s41598-025-90260-4" target="_blank">
            ü§ñ How the SIRVB Model Helps Understand COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-86', this)">
            More</button></h4>
            <div id="toggle-86" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Application and significance of SIRVB model in analyzing COVID-19 dynamics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

In the summer of 2024, many countries had more COVID-19 cases than before. But thanks to vaccination efforts, it's no longer a deadly pandemic. In our lab, students learned about how viruses spread and used real data from the World Health Organization to understand this better. Now we're updating one of these models to include "breakthrough" cases - when someone gets sick despite being vaccinated. We think this is the simplest model that can help predict future outbreaks of infectious diseases, which is important for researchers and decision-makers.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In the summer of 2024, COVID-19 positive cases spiked in many countries, but it is no longer a deadly pandemic thanks to global herd immunity to the SARS-CoV-2 viruses. In our physical chemistry lab in spring 2024, students practice kinetic models, SIR (Susceptible, Infected, and Recovered) and SIRV (Susceptible, Infected, Recovered, Vaccinated) using COVID-19 positive cases and vaccination data from World Health Organization (WHO). In this report, we further introduce virus breakthrough to the existing model updating it the SIRVB (Susceptible, Infectious, Recovered, Vaccinated, Breakthrough) model. We believe this is the simplest model possible to explain the COVID-19 kinetics/dynamics in all countries in the past four years. Parameters obtained from such practice correlate with many indices of different countries. These models and parameters have significant value to researchers and policymakers in predicting the stages of future outbreaks of infectious diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90260-4" target="_blank">https://www.nature.com/articles/s41598-025-90260-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-11</strong> <a href="https://www.nature.com/articles/s41598-025-92534-3" target="_blank">
            ü§ñ Effects of Lockdown on Physical Fitness in Healthy Male Students During the COVID-19 Pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-87', this)">
            More</button></h4>
            <div id="toggle-87" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of lockdown during COVID-19 pandemic on physical endurance among healthy male university students in Wuhan: a retrospective cohort study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

This study looked at how the coronavirus disease 2019 (COVID-19) pandemic affected healthy college men in China who were running 1000 meters and doing lung capacity tests over four years. We wanted to see if they got weaker or had less energy. 

We found that:

* The runners who weren't overweight got worse at running as time went on.
* All the runners got weaker at doing their lung capacity test, but it was especially bad after the pandemic started and during its worst part.
* Some runners gained weight over time, which also made it harder for them to run well.

This study shows that people need to exercise more during a pandemic to stay healthy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed at investigating the impact of coronavirus disease 2019 (COVID-19) on the time required to complete a 1000-m run and vital capacity (VC) among healthy male students at three universities in Wuhan, China, followed for four years covering the prepandemic (2019), early/late pandemic (2020/2021), and post-pandemic (2022) periods. Demographic information including age, body height/weight, and body mass index (BMI) were also collected. After dividing the participants into four groups based on BMI (i.e., underweight: BMI‚Äâ<‚Äâ18.5; normal: 18.5‚Äâ‚â§‚ÄâBMI‚Äâ<‚Äâ25; overweight: 25‚Äâ‚â§‚ÄâBMI‚Äâ<‚Äâ30; obese: BMI‚Äâ‚â•‚Äâ30), the associations of 1000-m run performance with changes in BMI and VC between different groups and within each group in different periods were analyzed. A 4-year follow-up on 10,037 participants demonstrated a progressive deterioration of 1000-m performance (p<‚Äâ0.001) except the obese group. All participants showed a drop in VC in the post-pandemic period (allp<‚Äâ0.001) compatible with their impaired 1000-m run performance. The underweight/normal BMI groups also exhibited an annual increase in BMI (allp<‚Äâ0.001). Impairment in 1000-m performance was most conspicuous during the early pandemic period and as the pandemic subsided. Our findings showed deterioration in physical endurance and VC among university students during the pandemic, highlighting the need for cardiopulmonary exercise reinforcement during a pandemic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92534-3" target="_blank">https://www.nature.com/articles/s41598-025-92534-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-11</strong> <a href="https://www.nature.com/articles/s41401-025-01516-8" target="_blank">
            ü§ñ How SARS-CoV-2 Virus Affects Blood Vessels</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-88', this)">
            More</button></h4>
            <div id="toggle-88" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>CircSARS-CV2-N1368 from SARS-CoV-2 impairs endothelial cell function through the upregulation of ATF7 to activate TLR4/NF-Œ∫B/ROS signaling</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young person can help fight COVID-19 
by keeping blood vessels open.

Scientists looked at small pieces of genetic material called RNA that are found inside a coronavirus called SARS-CoV-2.
They found three different parts of the virus that might be bad for blood vessels.

When these parts were put into cells, they made it harder for them to work properly. They also caused the cells to get damaged and not work as well.

The scientists then found out what was making this happen. It was like a sponge that soaked up a special message from the virus.
This message told another protein to start producing bad chemicals in the cell, which hurt the blood vessels.

To stop this from happening, the scientists gave the cells something called a special shield. This helped keep the bad chemicals away and kept the blood vessels healthy.

This is important because it might help us understand how COVID-19 affects the body and find ways to prevent damage to our blood vessels.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 can encode circular RNAs (circRNAs); however, the potential effects of exogenous SARS-CoV-2 circRNAs on cardiovascular sequelae remain unknown. Three circRNAs derived from the nucleocapsid (N) gene of SARS-CoV-2, namely, circSARS-CV2-Ns, were identified for functional studies. In particular, circSARS-CV2-N1368 was shown to enhance platelet adhesiveness to endothelial cells (ECs) and inhibit EC-dependent vascular relaxation. Moreover, exogenous expression of circSARS-CV2-N1368 suppressed EC proliferation and migration and decreased angiogenesis and cardiac organoid beating. Mechanistically, we elucidated that circSARS-CV2-N1368 sponged the microRNA miR-103a-3p, which could reverse circSARS-CV2-N1368-induced EC damage. Additionally, activating transcription factor 7 (ATF7) was identified as a target gene of miR-103a-3p, and Toll-like receptor 4 (TLR4) was verified as a downstream gene of ATF7 that mediates circARS-CV2-N1368-induced activation of nuclear factor kappa B (NF-Œ∫B) signaling and ROS production in ECs. Importantly, the reactive oxygen species (ROS) scavenger NAC mitigated the circSARS-CV2-N1368-promoted EC impairment. Our findings reveal that the TLR4/NF-Œ∫B/ROS signal pathway is critical for mediating circSARS-CV2-N1368-promoted oxidative damage in ECs, providing insights into the endothelial impairment caused by circSARS-CV2-Ns.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41401-025-01516-8" target="_blank">https://www.nature.com/articles/s41401-025-01516-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-11</strong> <a href="https://www.nature.com/articles/s41598-025-93476-6" target="_blank">
            ü§ñ Geographic disparities affect health outcomes for people with diabetes in specific diseases.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-89', this)">
            More</button></h4>
            <div id="toggle-89" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Geographic inequalities and factors associated with unfavorable outcomes in diabetes-tuberculosis and diabetes-covid comorbidities in Brazil</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

COVID-19 spread quickly around the world, putting a lot of pressure on healthcare systems. We looked at two specific problems: people with tuberculosis (TB) also having diabetes, and people with diabetes also getting COVID-19. 

We analyzed data from Brazil between 2020 and 2022 to see how common these combinations were in different parts of the country. 

We found that TB and diabetes together affected about 3.2 people per 100,000, mostly in the Central-West region. Diabetes and COVID-19 together affected much fewer people (only 0.4 per 100,000), but still appeared in many places like S√£o Paulo, Rio de Janeiro, and Belo Horizonte.

These findings show that both problems need to be taken seriously by health authorities to protect the public's health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid spread of COVID-19 have overwhelmed health systems, especially in the care of chronic disease such as tuberculosis and diabetes. The objective of the study was to analyze the magnitude and relevance of tuberculosis-diabetes and diabetes-COVID-19 comorbidities in spatial risk areas and their factors associated with unfavorable outcomes in the Brazilian population between 2020 and 2022. An ecological study was carried out in Brazilian municipalities. The population was composed by cases of tuberculosis-diabetes and diabetes-COVID-19 comorbidities, registered in the Influenza Epidemiological Surveillance Information System (SIVEP-GRIPE) and in DATASUS from 2020 to 2022. The Scan Statistics technique was used to identify spatial risk clusters. Binary logistic regression was then employed to understand the relationship between outcomes and comorbidities, considering clinical and sociodemographic variables. A total of 24,750 cases of tuberculosis-diabetes comorbidity were identified, which consisted of an incidence of 3.2 cases per 100,000 inhabitants. Risk clusters were identified in the Central-West and North regions. 303,210 cases of diabetes- COVID-19 comorbidity were identified, resulting in an incidence of 0.4 cases per 100,000 inhabitants. S√£o Paulo-SP, Rio de Janeiro-RJ and Belo Horizonte-MG were the municipalities with the highest spatial risk of illness. The analysis of the spatial risk areas revealed distinct patterns in the geographic distribution of comorbidities. Based on the findings, it is concluded that comorbidities between tuberculosis and diabetes, as well as between COVID-19 and diabetes, represent significant challenges for public health in Brazil, deserving attention from health authorities and the scientific community.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-93476-6" target="_blank">https://www.nature.com/articles/s41598-025-93476-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-10</strong> <a href="https://www.nature.com/articles/s41467-025-56473-x" target="_blank">
            ü§ñ What cells are damaged by COVID-19?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-90', this)">
            More</button></h4>
            <div id="toggle-90" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Integrated histopathology, spatial and single cell transcriptomics resolve cellular drivers of early and late alveolar damage in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that explains it simply:

COVID-19 can cause severe lung damage that leads to respiratory failure. We studied how COVID-19 affects the lungs at a tiny cell level and found out what happens when the damage gets worse over time. Our study shows that different types of cells change in response to the damage, including some that help fight infection and others that promote scarring. This helps us understand how COVID-19 causes severe lung disease and could lead to new treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The most common cause of death due to COVID-19 remains respiratory failure. Yet, our understanding of the precise cellular and molecular changes underlying lung alveolar damage is limited. Here, we integrate single cell transcriptomic data of COVID-19¬†and donor lung¬†tissue with spatial transcriptomic data stratifying histopathological stages of diffuse alveolar damage. We identify changes in cellular composition across progressive damage, including waves of molecularly distinct macrophages and depletion of epithelial and endothelial populations. Predicted markers of pathological states identify immunoregulatory signatures, including IFN-alpha and metallothionein signatures in early damage, and fibrosis-related collagens in late damage. Furthermore, we predict a fibrinolytic shutdown via endothelial upregulation ofSERPINE1/PAI-1. Cell-cell interaction analysis revealed macrophage-derivedSPP1/osteopontin signalling as a key regulator during early steps of alveolar damage. These results provide a comprehensive, spatially resolved atlas of alveolar damage progression in COVID-19, highlighting the cellular mechanisms underlying pro-inflammatory and pro-fibrotic pathways in severe disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56473-x" target="_blank">https://www.nature.com/articles/s41467-025-56473-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-10</strong> <a href="https://www.nature.com/articles/s41598-025-92794-z" target="_blank">
            ü§ñ Effect of bacterial infection on antibodies in COVID-19 patients' noses and throat.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-91', this)">
            More</button></h4>
            <div id="toggle-91" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bacterial lysates in modifying sIgA levels in the upper respiratory tract in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied how people with COVID-19 did after taking special medicine called Immunovac VP4. They looked at whether it helped their bodies make more antibodies that fight off germs in their noses, throats, and mouths. The study found that patients who took Immunovac VP4 had better immune systems and recovered faster from the virus. Those who got the medicine made more special helpers called secretory immunoglobulin A (sIgA) in their nasal swabs and saliva. This can help prevent COVID-19 and reduce symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A great deal of evidence has accumulated suggesting an important role of mucosal immunity not only in preventing COVID-19 but also in the pathogenesis of this infection. The aim of the study was to evaluate the levels of secretory immunoglobulin A (sIgA) in different compartments of the upper respiratory tract in COVID-19 patients in relation to the severity of the disease and treatment with a bacteria-based immunomodulating agent (Immunovac VP4). The titers of sIgA were determined by ELISA in nasal epithelial swabs, pharyngeal swabs, and salivary gland secretions at baseline and on days 14 and 30 of treatment. The levels of nasal, pharyngeal and salivary sIgA were significantly lower in more severe patients (subgroup A) than in less severe patients (subgroup B),p<‚Äâ0.01. In subgroup A, the patients who received Immunovac VP4 had higher pharyngeal sIgA levels in convalescent period than those who did not receive the therapyp<‚Äâ0.05. In subgroup B patients, an increase in immunoglobulin levels was observed from baseline to day¬†14 of treatment whether they received the add-on therapy or not,p<‚Äâ0.01. On day¬†30 of treatment, the sIgA levels in the standard treatment group, however, decreased, while the patients receiving the immunomodulating agent maintained high sIgA levels,p<‚Äâ0.05. Oxygen saturation significantly increased by day¬†14 in both groups,p<‚Äâ0.001. However, it was higher in the Immunovac VP4 group than in the standard treatment group,p<‚Äâ0.01. Thus, addition of a bacterial lysate-based immunomodulating agent to the treatment regimen for moderate-to-severe COVID-19 induces the production of pharyngeal and salivary sIgA. SIgA production is inversely correlated to CRP levels and percentage of lung involvement on CT scan and is directly correlated to SpO2levels.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92794-z" target="_blank">https://www.nature.com/articles/s41598-025-92794-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-08</strong> <a href="https://www.nature.com/articles/s42003-025-07820-7" target="_blank">
            ü§ñ How SARS-CoV-2 Affects the Immune System of Mice</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-92', this)">
            More</button></h4>
            <div id="toggle-92" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effector CD8 T cell differentiation in primary and breakthrough SARS-CoV-2 infection in mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The immune system's response to COVID-19 is not fully understood. Scientists studied the responses of two types of immune cells, called T cells, in mice infected with COVID-19 and flu viruses. They found that:

* Both viruses trigger a strong attack from these T cells in the lungs.
* However, COVID-19 makes it harder for some memory T cells to work properly.
* These memory T cells can remember previous infections and respond quickly if you get infected again.
* Some people's immune systems are more effective at fighting off COVID-19 due to certain proteins on their T cells.

The study suggests that new treatments could help control the damage caused by a strong immune response and promote a healthy immune system.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The nature of the effector and memory T cell response in the lungs following acute SARS-CoV-2 infections remains largely unknown. To define the pulmonary T-cell response to COVID-19, we compared effector and memory T-cell responses to SARS-CoV-2 and influenza A virus (IAV) in mice. Both viruses elicited potent effector T cell responses in lungs, but memory T cells showed exaggerated contraction in SARS-CoV-2-infected mice. Specifically, unlike the T-bet/EOMES-driven effector transcription program in IAV lungs, SARS-CoV-2-specific CD8 T cells embarked on a STAT-3-centric transcriptional program, a defining characteristic of a pro-fibro-inflammatory program: limited cytotoxicity, diminished expression of tissue-protective inhibitory receptors (PD-1, LAG-3, and TIGIT), and augmented mucosal imprinting (CD103). Circulating CD45RO+HLA-DR+CD8 T cells in hospitalized COVID-19 patients expressed elevated levels of STAT-3 and low levels of TIGIT. IL-6 blockade experiments implicated IL-6 in STAT-3 induction and downregulation of PD-1 expression on SARS-CoV-2-specific primary effector CD8 T cells. Memory CD8 T cells specific to a single epitope, induced by mucosal vaccination, differentiated into cytotoxic effectors and expressed high levels of CD103, effectively reducing viral burden in lungs following a breakthrough SARS-CoV-2 infection. Our findings have implications for developing targeted immunotherapies to mitigate immunopathology and promote protective T cell immunity to SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07820-7" target="_blank">https://www.nature.com/articles/s42003-025-07820-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-07</strong> <a href="https://www.nature.com/articles/s41467-025-57632-w" target="_blank">
            ü§ñ Microglial cells slow down repair of nerve tissue due to aging</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-93', this)">
            More</button></h4>
            <div id="toggle-93" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Senescent-like microglia limit remyelination through the senescence associated secretory phenotype</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As we age, our bodies' ability to repair damaged areas in the brain, called myelin, gets weaker. This can lead to a disease called multiple sclerosis (MS), where the protective covering around nerve fibers is damaged.

Some scientists think that when our cells get old, they become "senescent" and stop working properly. These senescent cells can accumulate in areas of damage in the brain, including those caused by MS.

Research showed that using special treatments to remove these senescent cells (called senolytics) helped repair damaged myelin in younger and middle-aged mice. However, this treatment didn't work as well in older mice.

In older mice, scientists found that the senescent cells kept producing chemicals that blocked the growth of new brain tissue. These chemicals were found to be particularly effective at stopping the development of a type of cell called oligodendrocytes, which are responsible for making myelin.

This suggests that targeting these chemicals might help improve repair of damaged myelin in older people with MS.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The capacity to regenerate myelin in the central nervous system diminishes with age. This decline is particularly evident in multiple sclerosis (MS), a chronic demyelinating disease. Whether cellular senescence, a hallmark of aging, contributes to remyelination impairment remains unknown. Here, we show that senescent cells accumulate within demyelinated lesions after injury, and treatments with senolytics enhances remyelination in young and middle-aged mice but not aged mice. In young mice, we observe the upregulation of senescence-associated transcripts, primarily in microglia and macrophages, after demyelination, followed by a reduction during remyelination. However, in aged mice, senescence-associated factors persist within lesions, correlating with inefficient remyelination. Proteomic analysis of the senescence-associated secretory phenotype (SASP) reveals elevated levels of CCL11/Eotaxin-1 in lesions of aged mice, which is found to inhibit oligodendrocyte maturation. These results suggest therapeutic targeting of SASP components, such as CCL11, may improve remyelination in aging and MS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57632-w" target="_blank">https://www.nature.com/articles/s41467-025-57632-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-07</strong> <a href="https://www.nature.com/articles/s41746-025-01542-0" target="_blank">
            ü§ñ Using AI and Medicine to Better Understand AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-94', this)">
            More</button></h4>
            <div id="toggle-94" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Red teaming ChatGPT in medicine to yield real-world insights on model behavior</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We wanted to make sure that large language models used in healthcare are fair and accurate. To do this, we brought together a team of doctors, students, and experts to test these models with real patient cases. They looked at how the models responded and marked responses as safe or not safe. We tested three different models and found that some responses were not suitable for patients' sensitive information. Our goal is to help create better models that work safely and accurately in healthcare.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Red teaming, the practice of adversarially exposing unexpected or undesired model behaviors, is critical towards improving equity and accuracy of large language models, but non-model creator-affiliated red teaming is scant in healthcare. We convened teams of clinicians, medical and engineering students, and technical professionals (80 participants total) to stress-test models with real-world clinical cases and categorize inappropriate responses along axes of safety, privacy, hallucinations/accuracy, and bias. Six medically-trained reviewers re-analyzed prompt-response pairs and added qualitative annotations. Of 376 unique prompts (1504 responses), 20.1% were inappropriate (GPT-3.5: 25.8%; GPT-4.0: 16%; GPT-4.0 with Internet: 17.8%). Subsequently, we show the utility of our benchmark by testing GPT-4o, a model released after our event (20.4% inappropriate). 21.5% of responses appropriate with GPT-3.5 were inappropriate in updated models. We share insights for constructing red teaming prompts, and present our benchmark for iterative model assessments.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-025-01542-0" target="_blank">https://www.nature.com/articles/s41746-025-01542-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41467-025-57446-w" target="_blank">
            ü§ñ Monitoring how well our bodies take up and use the antigens from an mRNA vaccine.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-95', this)">
            More</button></h4>
            <div id="toggle-95" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Monitoring mRNA vaccine antigen expression in vivo using PET/CT</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists made a special tool to see where and when a new type of vaccine works in animals. They attached a small part of an enzyme called eDHFR to the vaccine and used a special radioactive dye to track it with a machine that takes pictures. The pictures showed that the vaccine works for a little while at the injection site, but doesn't cause any big immune response. This means they can use this tool to see how future vaccines work in animals before they are used in humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Noninvasive visualization of the distribution and persistence of mRNA vaccine antigen expression in mammalian systems has implications for the development and evaluation of future mRNA vaccines. Here, we genetically fuseE. colidihydrofolate reductase (eDHFR) to the delta furin diproline modified SARS-CoV-2 spike glycoprotein (S2P‚àÜf) mRNA vaccine and image its expression in female mice and male non-human primates using [18F]fluoropropyl-trimethoprim ([18F]FP-TMP). Whole body positron emission tomography (PET) imaging revealed transient expression of the vaccine antigen in the injection site and draining lymph nodes (dLNs). Fusion of eDHFR did not impact S2P immunogenicity and no humoral or cellular immune response was detected against eDHFR in either species. In this work, we show that eDHFR can be used as an mRNA-encoded PET reporter gene to monitor the spatiotemporal dynamics of mRNA vaccine antigen expression in vivo. This technique could be applied in clinical translation of future mRNA vaccines or therapeutics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57446-w" target="_blank">https://www.nature.com/articles/s41467-025-57446-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s42003-025-07769-7" target="_blank">
            ü§ñ Antibodies That Fight Many Variants of COVID-19 After One Initial Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-96', this)">
            More</button></h4>
            <div id="toggle-96" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Potent and broadly neutralizing antibodies against sarbecoviruses elicited by single ancestral SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists discovered two new types of antibodies that can fight many different versions of the COVID-19 virus, even ones that are changing rapidly. These antibodies were found in people who only got infected with an older version of the virus. One of these antibodies was very effective and helped protect hamsters from getting sick when they were infected with a newer version of the virus. This is good news because it shows that we can create powerful new tools to fight the virus, even if it's changing all the time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The emergence of various SARS-CoV-2 variants presents challenges for antibody therapeutics, emphasizing the need for more potent and broadly neutralizing antibodies. Here, we employed an unbiased screening approach and successfully isolated two antibodies from individuals with only exposure to ancestral SARS-CoV-2. One of these antibodies, CYFN1006-1, exhibited robust cross-neutralization against a spectrum of SARS-CoV-2 variants, including the latest KP.2, KP.3 and XEC, with consistent IC50values ranging from ~1 to 5‚Äâng/mL. It also displayed broad neutralization activity against SARS-CoV and related sarbecoviruses. Structural analysis revealed that these antibodies target shared hotspot but mutation-resistant epitopes, with their Fabs locking receptor binding domains (RBDs) in the ‚Äúdown‚Äù conformation through interactions with adjacent Fabs and RBDs, and cross-linking Spike trimers into di-trimers. In vivo studies conducted in a JN.1-infected hamster model validated the protective efficacy of CYFN1006-1. These findings suggest that antibodies with cross-neutralization activities can be identified from individuals with exclusively ancestral virus exposure.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07769-7" target="_blank">https://www.nature.com/articles/s42003-025-07769-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41598-025-92279-z" target="_blank">
            ü§ñ Impact of COVID-19 on People's Decisions to Visit Doctors for Eye Problems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-97', this)">
            More</button></h4>
            <div id="toggle-97" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of coronavirus disease 2019 pandemic on the trends of care-seeking behavior for ocular diseases: a systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the coronavirus pandemic, people visited eye doctors less often than before. We looked at data from 2020 to 2022 to see how many more or fewer people went to see an eye doctor compared to before the pandemic.

We found that visits decreased by about 58% during the pandemic and lockdown. There were also changes in who was visiting eye doctors - women visited less often, but adults visited more often than they did before the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed to assess the clinical and epidemiological impacts of the coronavirus disease 2019 pandemic on the number of ophthalmology outpatient department (oOPD) visits. PubMed and EMBASE were searched for literature published between January 1, 2020, and December 5, 2022. The extracted data were pooled using a random-effects model. The primary outcome was the number of oOPD visits. Of the 335 screened articles, 21 and 16 were included in the qualitative and quantitative syntheses, respectively. Among the 16 studies included in the meta-analysis, 7 involving 4,204,209 individuals reported the number of oOPD visits during the pandemic. Compared with the number of pre-pandemic visits, the numbers of oOPD visits declined to 58.1% (95% confidence interval [CI], 0.378‚Äì0.784) and 29.8% (95% CI 0.130‚Äì0.465) during the pandemic and lockdown, respectively. The proportions of female patient visits decreased from 50.9 to 47.8% and from 48.3 to 42.3% during the pandemic and lockdown, respectively. The proportions of adult visits increased from 86.3 to 89.6% and decreased from 90.6 to 80.1% during the pandemic and lockdown, respectively. The decrease in oOPD visits during the pandemic may have caused delays in diagnosis and treatment, potentially exacerbating the existing ocular diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92279-z" target="_blank">https://www.nature.com/articles/s41598-025-92279-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41467-025-57374-9" target="_blank">
            ü§ñ A simple way to title this would be:

Comparing Protein Energies for Structural and Evolutionary Insights</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-98', this)">
            More</button></h4>
            <div id="toggle-98" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A fast approach for structural and evolutionary analysis based on energetic profile protein comparison</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists want to figure out how proteins work together and change over time. A new way to do this uses special numbers (energy profiles) that describe how proteins are shaped. This method is faster and works better than other ways, which rely on matching parts of proteins (like atoms). The study showed that these energy profiles can tell us a lot about protein structure and even help find new medicine combinations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In structural bioinformatics, the efficiency of predicting protein similarity, function, and evolutionary relationships is crucial. Our approach proposed herein leverages protein energy profiles derived from a knowledge-based potential, deviating from traditional methods relying on structural alignment or atomic distances. This method assigns unique energy profiles to individual proteins, facilitating rapid comparative analysis for both structural similarities and evolutionary relationships across various hierarchical levels. Our study demonstrates that energy profiles contain substantial information about protein structure at class, fold, superfamily, and family levels. Notably, these profiles accurately distinguish proteins across species, illustrated by the classification of coronavirus spike glycoproteins and bacteriocin proteins. Introducing a separation measure based on energy profile similarity, our method shows significant correlation with a network-based approach, emphasizing the potential of energy profiles as efficient predictors for drug combinations with faster computational requirements. Our key insight is that the sequence-based energy profile strongly correlates with structure-derived energy, enabling rapid and efficient protein comparisons based solely on sequences.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57374-9" target="_blank">https://www.nature.com/articles/s41467-025-57374-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41593-025-01871-z" target="_blank">
            ü§ñ How does COVID-19 affect the brain?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-99', this)">
            More</button></h4>
            <div id="toggle-99" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Microglia dysfunction, neurovascular inflammation and focal neuropathologies are linked to IL-1- and IL-6-related systemic inflammation in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

COVID-19 can cause many problems with the brain and nervous system. We think that tiny immune cells called microglia play a big role in this. To study it, we looked at brain tissue from people who died from COVID-19.

We found that some parts of these microglial cells didn't work right, which made other brain cells sick too. When one part of the brain got sick, it affected many others. This might be why so many different things can go wrong with the nervous system after getting COVID-19.

The problems started when the virus was in the body and caused inflammation (swelling) in some parts of the brain. The microglial cells were trying to fight the virus but ended up making things worse instead.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 is associated with diverse neurological abnormalities, but the underlying mechanisms are unclear. We hypothesized that microglia, the resident immune cells of the brain, are centrally involved in this process. To study this, we developed an autopsy platform allowing the integration of molecular anatomy, protein and mRNA datasets in postmortem mirror blocks of brain and peripheral organ samples from cases of COVID-19. We observed focal loss of microglial P2Y12R, CX3CR1‚ÄìCX3CL1 axis deficits and metabolic failure at sites of virus-associated vascular inflammation in severely affected medullary autonomic nuclei and other brain areas. Microglial dysfunction is linked to mitochondrial injury at sites of excessive synapse and myelin phagocytosis and loss of glutamatergic terminals, in line with proteomic changes of synapse assembly, metabolism and neuronal injury. Furthermore, regionally heterogeneous microglial changes are associated with viral load and central and systemic inflammation related to interleukin (IL)-1 or IL-6 via virus-sensing pattern recognition receptors and inflammasomes. Thus, SARS-CoV-2-induced inflammation might lead to a primarily gliovascular failure in the brain, which could be a common contributor to diverse COVID-19-related neuropathologies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41593-025-01871-z" target="_blank">https://www.nature.com/articles/s41593-025-01871-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41598-025-86921-z" target="_blank">
            ü§ñ Cell Line for Cultured Meat Production</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-100', this)">
            More</button></h4>
            <div id="toggle-100" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Establishment & characterization of a non-adherent insect cell line for cultivated meat</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Insect cells are being studied as a way to make food from insects. This can be a good option for making meat because insect cells grow quickly and easily in many different conditions. To study this, scientists took cells from caterpillar eggs and made them grow on their own without animals. They were able to get very high numbers of cells in just one type of container. The team looked at what the cells used up as food and how much protein and other nutrients they had, which can help us make insect-based meat products that are healthy and nutritious.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study presents a blueprint for developing, scaling, and analyzing novel insect cell lines for food. The large-scale production of cultivated meat requires the development and analysis of cell lines that are simple to grow and easy to scale. Insect cells may be a favorable cell source due to their robust growth properties, adaptability to different culture conditions, and resiliency in culture. Cells were isolated from Tobacco hornworm (Manduca sexta)embryos and subsequently adapted to single-cell suspension culture in animal-free growth media. Cells were able to reach relatively high cell densities of over 20¬†million cells per mL in shake flasks. Cell growth data is presented in various culture vessels and spent media analysis was performed to better understand cell metabolic processes. Finally, a preliminary nutritional profile consisting of proximate, amino acid, mineral, and fatty acid analysis is reported.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86921-z" target="_blank">https://www.nature.com/articles/s41598-025-86921-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-06</strong> <a href="https://www.nature.com/articles/s41598-025-92254-8" target="_blank">
            ü§ñ Protecting Cancer Patients from New COVID Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-101', this)">
            More</button></h4>
            <div id="toggle-101" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cracking the code of a correlate of protection against SARS-CoV-2 breakthrough infection in cancer patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People with cancer who have been vaccinated against COVID-19 want to know how well their bodies can protect themselves from getting infected again if they get the virus that causes COVID-19.

They studied 760 people with cancer who were followed for six months after they got two booster shots. The researchers looked at how many of these people had antibodies in their blood that are meant to fight off the virus.

They found out that when people have more of these antibodies, they are less likely to get infected again. But it's like trying to find a magic number - if you reach 20 something level of antibodies, you're probably safe. However, new variants came along and now we need even more antibodies to be protected.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The level of protection against SARS-CoV-2 breakthrough infections conferred by the presence of anti-S1 SARS-CoV-2 antibodies (IgGs) in cancer patients is still understudied. This work examines the existence of an anti-S1 immunoglobulin G (IgG) -based correlate of protection (CoP) established by prospectively collected observational data about breakthrough infections with different SARS-CoV-2 variants in a large cohort study with vaccinated cancer patients. 760 cancer patients were longitudinally followed-up, starting before first vaccination until six months after second booster. Anti-S1 SARS-CoV-2 IgGs were quantified in serum samples (N=‚Äâ2958) and breakthrough infections were monitored using questionnaires, routine COVID-19 testing and medical chart review. A Generalized Estimating Equations approach was used to model the binary infection status as endpoint in relation to anti-S1 IgG titers. It is observed that higher anti-S1 IgG titers correspond to a lower probability of breakthrough infection. For the early pandemic phase, a protective anti-S1 IgG titer above 20.42 BAU/mL was observed. However, with the emergence of the Omicron variant, higher anti-S1 IgG titers are required to be protective, but no clear CoP could be identified.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-92254-8" target="_blank">https://www.nature.com/articles/s41598-025-92254-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-05</strong> <a href="https://www.nature.com/articles/s41590-025-02100-2" target="_blank">
            ü§ñ The importance of adding a chemical group to cytidine in mRNA for immune system function and fighting off viruses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-102', this)">
            More</button></h4>
            <div id="toggle-102" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A critical role ofN4-acetylation of cytidine in mRNA by NAT10 in T cell expansion and antiviral immunity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>T cells need to multiply fast after getting activated. But we didn't know how this happens at a molecular level. We found that one enzyme called NAT10 helps T cells make proteins faster by adding a special mark to their messenger RNA. This makes the protein-making process work more efficiently.

If we remove NAT10 from mouse T cells, they can't multiply properly and get stuck in a cycle of cell division. This also affects how well their immune response works when fighting off viruses.

Older people with lower levels of NAT10 may have trouble making new immune cells, which could explain why their bodies don't fight off infections as well.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Following activation, naive T cells exit quiescence and require global translation for rapid expansion, yet the underlying mechanisms remain unclear. Here, we show that during T cell activation, cells upregulate the expression ofN-acetyltransferase 10 (NAT10), an enzyme responsible forN4-acetylcytidine (ac4C) modification of mRNAs. ac4C-modifiedMycmRNAs show higher translation efficiency, enabling rapid synthesis of MYC protein and supporting robust T cell expansion. Conditional deletion ofNat10in mouse T cells causes severe cell cycle arrest and limitation of cell expansion due to MYC deficiency, ultimately exacerbating infection in an acute lymphocytic choriomeningitis virus model. Additionally, T cells from older individuals with lower NAT10 levels show proliferative defects, which may partially account for impaired antiviral responses in older individuals. This study reveals a mechanism governing T cell expansion, signal-dependent mRNA degradation induction and the potential in vivo biological significance of ac4C modification in T cell-mediated immune responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-025-02100-2" target="_blank">https://www.nature.com/articles/s41590-025-02100-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-05</strong> <a href="https://www.nature.com/articles/s43587-025-00819-z" target="_blank">
            ü§ñ How immune system changes with age affect individual responses to illness and vaccines.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-103', this)">
            More</button></h4>
            <div id="toggle-103" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Single-cell immune aging clocks reveal inter-individual heterogeneity during infection and vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Our bodies get weaker with age, making us more prone to diseases that affect the immune system. Scientists have created a way to measure how our immune cells are aging, using data from over 1,000 people aged 18-97 years.

They found that some parts of our immune system age faster than others, and this can be affected by things like getting vaccinated or having COVID-19. They also discovered that some people's immune systems get better with age if they're exposed to certain types of germs, like the BCG vaccine.

This new tool can help us understand how aging affects the immune system and maybe even find ways to keep our immune systems healthy as we get older.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Aging affects human immune system functionality, increasing susceptibility to immune-mediated diseases. While gene expression programs accurately reflect immune function, their relationship with biological immune aging and health status remains unclear. Here we developed robust, cell-type-specific aging clocks (sc-ImmuAging) for the myeloid and lymphoid immune cell populations in circulation within peripheral blood mononuclear cells, using single-cell RNA-sequencing data from 1,081 healthy individuals aged from 18 to 97‚Äâyears. Application of sc-ImmuAging to transcriptome data of patients with COVID-19 revealed notable age acceleration in monocytes, which decreased during recovery. Furthermore, inter-individual variations in immune aging induced by vaccination were identified, with individuals exhibiting elevated baseline interferon response genes showing age rejuvenation in CD8+T cells after BCG vaccination. sc-ImmuAging provides a powerful tool for decoding immune aging dynamics, offering insights into age-related immune alterations and potential interventions to promote healthy aging.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43587-025-00819-z" target="_blank">https://www.nature.com/articles/s43587-025-00819-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-05</strong> <a href="https://www.nature.com/articles/s41467-025-57087-z" target="_blank">
            ü§ñ How children were affected by COVID-19 in Belgium and what it tells us about its impact.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-104', this)">
            More</button></h4>
            <div id="toggle-104" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessing the role of children in the COVID-19 pandemic in Belgium using perturbation analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract:

As COVID-19 spread, it was important to understand how different age groups were passing the virus on to each other. We studied what happened in Belgium from November 2020 to February 2022. We looked at how often people in different age groups were infected and when. We used a special computer model that helped us see how easily the virus spread.

At first, older adults were spreading the virus most of all. But then, schools started opening again and younger kids began getting infected too. When we had to stay home more, it helped stop the spread. Later, as new viruses became common, older teenagers and some younger kids were the main ones passing it on again.

This study shows that our computer model worked well to figure out how different age groups are connected when it comes to spreading the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the evolving role of different age groups in virus transmission is essential for effective pandemic management. We investigated SARS-CoV-2 transmission in Belgium from November 2020 to February 2022, focusing on age-specific patterns. Using a next generation matrix approach integrating social contact data and simulating population susceptibility evolution, we performed a longitudinal perturbation analysis¬†of the effective reproduction number to unravel age-specific transmission dynamics. From November to December 2020, adults in the [18,¬†60) age group were the main transmission drivers, while children contributed marginally. This pattern shifted between January and March 2021, when in-person education resumed, and the Alpha variant emerged: children aged under 12 years old were crucial in transmission. Stringent social distancing measures in March 2021 helped diminish the noticeable contribution of the [18,¬†30) age group. By June 2021, as the Delta variant became the predominant strain, adults aged [18,¬†40) years emerged as main contributors to transmission, with a resurgence in children‚Äôs contribution during September-October 2021. This study highlights the effectiveness of our methodology in identifying age-specific transmission patterns.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57087-z" target="_blank">https://www.nature.com/articles/s41467-025-57087-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-04</strong> <a href="https://www.nature.com/articles/s42003-025-07763-z" target="_blank">
            ü§ñ Chitosan-based compounds kill many different viruses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-105', this)">
            More</button></h4>
            <div id="toggle-105" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Synthetic heparan sulfate mimics based on chitosan derivatives show broad-spectrum antiviral activity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Viruses can enter cells by attaching to special molecules on the cell's surface. Researchers thought that maybe some medicines could block this process to stop the virus from entering the cell. They tested some special compounds that are similar to these molecules and found ones that work against several different viruses, including the ones that cause COVID-19 and a common cold. These compounds were safe for animals and helped stop the virus after it had already entered the body. This is promising news because it means there might be new medicines to help fight respiratory viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Enveloped viruses enter cells by binding to receptors present on host cell membranes, which trigger internalization and membrane fusion. For many viruses, this either directly or indirectly involves interaction with membrane-anchored carbohydrates, such as heparan sulfate, providing a potential target for a broad-spectrum antiviral approach. Based on this hypothesis, we screened a library of functionalized chitosan sulfates that mimic heparan sulfate in cellular membranes for inhibition of SARS-CoV-2 and respiratory syncytial virus (RSV) entry. An array of compounds blocking SARS-CoV-2 and RSV were identified, with the lead compound displaying broad-spectrum activity against multiple viral strains and clinical isolates. Mechanism of action studies showed the drug to block viral entry irreversibly, likely via a virucidal mechanism. Importantly, the drug was non-toxic in vivo and showed potent post-exposure therapeutic activity against both SARS-CoV-2 and RSV. Together, these results highlight the potential of functionalized carbohydrates as broad-spectrum antivirals targeting respiratory viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07763-z" target="_blank">https://www.nature.com/articles/s42003-025-07763-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-04</strong> <a href="https://www.nature.com/articles/s41467-025-57149-2" target="_blank">
            ü§ñ Using COVID-19 vaccine technology to fight cancer.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-106', this)">
            More</button></h4>
            <div id="toggle-106" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>LNP-RNA-mediated antigen presentation leverages SARS-CoV-2-specific immunity for cancer treatment</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists created special tiny particles called lipid nanoparticles (LNPs) that can carry a message from our bodies to fight against COVID-19. They found out that this method is very effective and wants to use it to help treat cancer.

To do this, they designed new tiny particles with different shapes that can deliver genetic information into tumors, which are abnormal growths in the body. These particles were tested on mice and showed great promise.

The results were promising, with the new particles helping to stop tumor growth and even extending the lives of the mice. The scientists also tested the particles on human cancer cells outside the body and found that they worked well too.

This discovery could lead to a new way to treat cancer by harnessing our bodies' natural immune response against COVID-19 vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Lipid nanoparticle (LNP)-mRNA vaccines have demonstrated protective capability in combating SARS-CoV-2. Their extensive deployment across the global population leads to the broad presence of T-cell immunity against the SARS-CoV-2 spike protein, presenting an opportunity to harness this immunological response as a universal antigen target for cancer treatment. Herein, we design and synthesize a series of amino alcohol- or amino acid-derived ionizable lipids (AA lipids) and develop an LNP-RNA-based antigen presentation platform to redirect spike-specific T-cell immunity against cancer in mouse models. First, in a prime-boost regimen, AA2 LNP encapsulating spike mRNA elicit stronger T-cell immunity against the spike epitopes compared to FDA-approved LNPs (ALC-0315 and SM-102), highlighting the superior delivery efficiency of AA2 LNP. Next, AA15V LNP efficiently delivers self-amplifying RNAs (saRNAs) encoding spike epitope-loaded single-chain trimer (sSE-SCT) MHC I molecules into tumor tissues, thereby inducing the presentation of spike epitopes. Our results show that a single intratumoral (i.t.) treatment of AA15V LNP-sSE-SCTs suppresses tumor growth and extends the survival of B16F10 melanoma and A20 lymphoma tumor-bearing mice vaccinated with AA2 LNP-spike mRNA. Additionally, AA15V LNP-sSE-SCTs enable SE-SCT expression in ex vivo human glioblastoma and lung cancer samples, suggesting its potential in clinical translation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57149-2" target="_blank">https://www.nature.com/articles/s41467-025-57149-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-04</strong> <a href="https://www.nature.com/articles/s41467-025-56994-5" target="_blank">
            ü§ñ Cohort Study on Fatigue Before and After Covid Infection in the Netherlands</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-107', this)">
            More</button></h4>
            <div id="toggle-107" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prospective cohort study of fatigue before and after SARS-CoV-2 infection in the Netherlands</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Many people who get sick with COVID-19 feel tired and this doesn't go away quickly. We looked at how bad the fatigue feels in different groups, like age, sex, and health conditions. We found that people who got infected with COVID-19 felt more tired when they first got sick, but this didn't last as long for some types of the virus.

Some good news is that even after getting sick, most people feel better soon. But we also found out that some groups, like those with certain health conditions or older adults, might be more affected by fatigue.

Overall, COVID-19 did cause some people to feel tired, but it's not as big of a problem as we thought it would be.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Fatigue is one of the most common persistent symptoms of SARS-CoV-2 infection. We aimed to assess fatigue during and after a SARS-CoV-2 infection by age, sex, presence of a medical risk condition, SARS-CoV-2 variant and vaccination status, accounting for pre-infection fatigue and compared with uninfected individuals. We used data from an ongoing prospective cohort study in the Netherlands (VASCO). We included 22,705 first infections reported between 12 July 2021 and 9 March 2024. Mean fatigue scores increased during infection, declined rapidly in the first 90 days post-infection, but remained elevated until at least 270 days for Delta and 120 days for Omicron infections. Prevalence of severe fatigue was 18.5% before first infection. It increased to 24.4% and 22.5% during acute infection and decreased to 21.2% and 18.9% at 90 days after Delta and Omicron infection, respectively. The prevalence among uninfected participants was lower than among matched Delta-infected participants during the acute phase of the infection and 90 days post-infection. For matched Omicron-infected individuals this was only observed during the acute phase. We observed no differences in mean post- vs pre-infection fatigue scores at 90-270 days post-infection by vaccination status. The impact of SARS-CoV-2 infection on the prevalence of severe fatigue was modest at population level, especially for Omicron.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56994-5" target="_blank">https://www.nature.com/articles/s41467-025-56994-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-04</strong> <a href="https://www.nature.com/articles/s41467-025-57284-w" target="_blank">
            ü§ñ Vaccine Effectiveness in Early HIV Treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-108', this)">
            More</button></h4>
            <div id="toggle-108" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Safety, immunogenicity and effect on viral rebound of HTI vaccines combined with a TLR7 agonist in early-treated HIV-1 infection: a randomized, placebo-controlled phase 2a trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young person, let's talk about a new way to help people with HIV.

Scientists are trying to make a vaccine that will keep HIV from making people sick. They've already tested one part of the vaccine in a different study, so now they're combining it with another part and testing how safe it is and if it works.

In this new study, 50 men with HIV took part and got either the new vaccine or a fake version. The most common side effects were pain at the injection site, feeling sick like they had the flu, headaches, and feeling tired. But the good news is that the vaccine worked well and helped their bodies make strong immune responses.

The researchers also found out that people who made stronger immune responses were more likely to stay off HIV medicine for longer periods of time. This means the vaccine might be able to help people with HIV live healthier lives without always needing medication.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Building on results from the AELIX-002 trial with HIVACAT T-cell immunogen (HTI)-based vaccines, the AELIX-003 (NCT04364035) trial tested the safety of the combination of ChAdOx1.HTI (C) and MVA.HTI (M), with the TLR7 agonist vesatolimod (VES), in a double-blind, placebo-controlled, randomized clinical trial in 50 virally suppressed early-treated men with HIV-1 infection. Secondary objectives included immunogenicity and effects on viral rebound kinetics during a 24-week antiretroviral treatment interruption (ATI). The most common treatment-related adverse events were mild-to-moderate injection-site pain, influenza-like illness, headache, and fatigue. Strong, broad, and HTI-focused T-cell responses were induced by vaccination. All participants experienced viral rebound in ATI; 33.3% and 23.5% (P=‚Äâ0.4494) of CCMM‚Äâ+‚ÄâVES and placebo recipients, respectively, remained off antiretroviral therapy for 24 weeks. Post¬†hoc analysis confirmed a correlation between levels of HTI-specific T cells and prolonged time off antiretroviral therapy. The combination of HTI vaccines and VES was safe and elicited robust T-cell responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57284-w" target="_blank">https://www.nature.com/articles/s41467-025-57284-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-04</strong> <a href="https://www.nature.com/articles/s41421-024-00766-w" target="_blank">
            ü§ñ Endoplasmic Reticulum Helps Regulate Immune Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-109', this)">
            More</button></h4>
            <div id="toggle-109" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>TMEM41B is an endoplasmic reticulum Ca2+release channel maintaining naive T cell quiescence and responsiveness</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the simplified abstract:

In mammalian cells, there is a special part called endoplasmic reticulum that helps with things like recycling trash and making lipids. We found out that TMEM41B, a protein that lives in this ER, acts as a channel to release calcium ions. When we studied how it works, we saw that it releases more calcium when it gets too full or not enough when it's overfull.

When people don't have enough TMEM41B, they get too much calcium in their cells and it can affect how their immune system works. Without enough calcium, their T cells (which help fight infections) become more sensitive to things that normally wouldn't trigger them to work. This means that people without enough TMEM41B might be more likely to get sick or have a stronger response to certain infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In mammalian cells, endoplasmic reticulum (ER) passively releases Ca2+under steady state, but channels involved remain elusive. Here, we report that TMEM41B, an ER-resident membrane protein critical for autophagy, lipid metabolism, and viral infection, functions as an ER Ca2+release channel. Biochemically, purified recombinant TMEM41B forms a concentration-dependent Ca2+channel in single-channel electrophysiology assays. Cellularly, TMEM41B deficiency causes ER Ca2+overload, while overexpression of TMEM41B depletes ER Ca2+. Immunologically, ER Ca2+overload leads to upregulation of IL-2 and IL-7 receptors in naive T cells, which in turn increases basal signaling of JAK-STAT, AKT-mTOR, and MAPK pathways. This dysregulation drives TMEM41B-deficient naive T cells into a metabolically activated yet immunologically naive state. ER Ca2+overload also downregulates CD5, lowering the activation threshold of TMEM41B-deficient T cells and leading to heightened T cell responses during infections. In summary, we identify TMEM41B as a concentration-dependent ER Ca2+release channel, revealing an unexpected role of ER Ca2+in naive T cell quiescence and responsiveness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00766-w" target="_blank">https://www.nature.com/articles/s41421-024-00766-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-03</strong> <a href="https://www.nature.com/articles/s12276-025-01416-1" target="_blank">
            ü§ñ Impact of COVID virus on blood cell production and aging.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-110', this)">
            More</button></h4>
            <div id="toggle-110" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 pseudovirus dysregulates hematopoiesis and induces inflammaging of hematopoietic stem and progenitor cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

When you have COVID-19, it doesn't just affect your lungs - it can also hurt your blood cells. Some studies found that the virus can attack special blood cells called stem cells. These stem cells are important for making all types of blood cells.

Scientists studied how the Omicron variant of the coronavirus affects these stem cells. They found that it makes them get sick and not work properly. The virus also made them change their behavior, like becoming more inflammatory and older than they should be.

To understand what's happening, scientists looked at the genes in the infected stem cells. They saw that some genes were turned on, which can lead to inflammation and aging.

The scientists then tried a special medicine called nanographene oxide to see if it could help fix the problem. It did partially, but not completely.

This study helps us understand how COVID-19 affects our blood cells and immune system, and might give us ideas for new treatments in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection primarily affects the respiratory system but may induce hematological alterations such as anemia, lymphopenia and thrombocytopenia. Previous studies have reported that SARS-CoV-2 efficiently infects hematopoietic stem and progenitor cells (HSPCs); however, the subsequent effects on hematopoiesis and immune reconstitution have not yet been described. Here we evaluated the pathological effects of infection of umbilical-cord-blood-derived HSPCs with the SARS-CoV-2 Omicron variant pseudovirus (PsV). Transcriptomic analysis of Omicron PsV-infected HSPCs revealed the upregulation of genes involved in inflammation, aging and the NLRP3 inflammasome, suggesting a potential trigger of inflammaging. Omicron PsV-infected HSPCs presented decreased numbers of multipotential progenitors (granulocyte‚Äíerythrocyte‚Äímacrophage‚Äímegakaryocyte colony-forming units) ex vivo and repopulated primitive hematopoietic stem cells (Ki-67‚àíhCD34+cells) in an HSPC transplantation NOD-scid IL2rŒ≥nullmouse model (Omicron mouse). Furthermore, Omicron PsV infection induced myeloid-biased differentiation of HSPCs. Treatment with nanographene oxide, an antiviral agent, partially mitigated the myeloid bias and inflammaging phenotype both in vitro and in vivo. These findings provide insights into the abnormal hematopoietic and immune effects of SARS-CoV-2 infection and highlight potential therapeutic interventions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s12276-025-01416-1" target="_blank">https://www.nature.com/articles/s12276-025-01416-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-03</strong> <a href="https://www.nature.com/articles/s44328-025-00029-y" target="_blank">
            ü§ñ Sensor detects COVID-19 antibodies better than regular test.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-111', this)">
            More</button></h4>
            <div id="toggle-111" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Serologic LSPR-nanosensor against SARS-COV-2 antibodies and related variants outperforms ELISA in sensitivity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists created a tiny sensor using gold particles that can detect the presence of a specific protein from COVID-19. They coated these gold particles with a fake version of this protein to make them sensitive to its detection. The sensor was tested on different people's blood samples, including those who had recently been infected or weren't infected at all. It worked well and could detect very small amounts of the antibodies that the body produces in response to the virus. This means it can help find out if someone has COVID-19 early on, even before symptoms appear.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Localized surface plasmon resonance (LSPR) is an optical phenomenon derived from the dielectric properties of noble metals, resulting in a highly change-sensitive spectrum that can be used on a sensor platform. We have used gold nanorods to develop a diagnostic assay to detect antibodies against the nucleocapsid (N) protein of SARS-CoV-2. This approach is particularly valuable, considering the limited role of serology in recognizing acute infections. Gold nanoparticles were coated with the recombinant N protein and characterized by spectroscopy, fluorometry, and electron microscopy. Positive and negative COVID-19 sera samples were initially categorized through qRT-PCR and further validated using an anti-IgG ELISA. The nanosensor was accurate and able to detect very low levels of anti-SARS-CoV-2 antibodies derived from different virus variants early in infection (before 10 days post-infection). The nanoplatform exhibited high sensitivity and specificity, is suitable for mass production, and has easy implementation and automatic read-out.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44328-025-00029-y" target="_blank">https://www.nature.com/articles/s44328-025-00029-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-02</strong> <a href="https://www.nature.com/articles/s41541-025-01098-w" target="_blank">
            ü§ñ Boosting Rabies Vaccine Effectiveness</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-112', this)">
            More</button></h4>
            <div id="toggle-112" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Optimizing rabies mRNA vaccine efficacy via RABV-G structural domain screening and heterologous prime-boost immunization</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new way to make a vaccine is being tested to prevent rabies. Scientists looked at how well different parts of the virus work together to make a strong immune response. They found that making a full-length version of the virus works best and gives full protection against rabies. When they mixed two different vaccines, one that was made first (prime) and another that was made second (boost), it gave even better results than using just one vaccine. This new way of making vaccines has promise for preventing rabies in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>mRNA vaccine has become a promising technology platform for rabies prevention. This study explores the roles of different structural domains of rabies virus glycoprotein (RABV-G) and heterologous prime-boost strategies for enhanced immune responses and protection. The results suggested that mRNA vaccines encoding full-length RABV-G (RABV-Full) and RABV-R333Q induced strong immune responses and provided full protection against rabies, while mRNA vaccines encoding ectodomain/transmembrane domain (RABV-TE) and ectodomain (RABV-E) were less effective. Heterologous immunization results revealed that mRNA-primed strategies yielded higher long-lasting VNTs, but lower early VNTs than inactivated rabies virus (IRV)-primed strategies. 2√óRABV-Full and IRV‚Äâ>‚ÄâRABV-Full provided 100% protection, while that of RABV-Full>IRV was 90%. Transcriptome analysis showed that rabies mRNA vaccine induced both MHCI and MHCII antigen presentation, as well as B/T cell activation. In conclusion, full-length RABV-G mRNA vaccines, particularly with an ‚ÄòIRV prime and RABV-Full boost‚Äô strategy, hold great potential for rabies prevention.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01098-w" target="_blank">https://www.nature.com/articles/s41541-025-01098-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-03-01</strong> <a href="https://www.nature.com/articles/s41541-025-01076-2" target="_blank">
            ü§ñ Vaccine Booster Effectiveness Against Omicron Strain</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-113', this)">
            More</button></h4>
            <div id="toggle-113" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cellular immune breadth of an Omicron-specific, self-amplifying monovalent mRNA vaccine booster for COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists compared two different vaccines to see which one made the best immune system response when given as a booster shot. They looked at two types of vaccines: one that uses a special delivery method (samRNA) and one that uses an adenovector (ChAdOx1 nCoV-19). 

The samRNA vaccine, GEMCOVAC-OM, was more effective at making immune cells that remember Omicron viruses. It also triggered a stronger response against different variants of the virus, like XBB and BA.2.86.

This means the samRNA vaccine (GEMCOVAC-OM) is promising for future use because it creates a longer-lasting and broader immunity to help fight against new variations of SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Selecting a booster vaccine strategy that generates cellular immune breadth is crucial for effectively recalling cellular reservoirs upon infection with severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) variants. This post hoc analysis from a multicentre, randomized phase 3 study (CTRI/2022/10/046475) compared the cellular immune breadth induced by self-replicating mRNA (samRNA) vaccine GEMCOVAC-OM, encoding Omicron B.1.1.529 Spike protein, with the adenovector vaccine ChAdOx1 nCoV-19, encoding Wuhan variant Spike protein, when administered as a booster. GEMCOVAC-OM elicited significant expansion of memory B-cells (MBCs) specific to Omicron B.1.1.529, compared to ChAdOx1 nCoV-19. GEMCOVAC-OM also induced more B-cells reactive to Omicron XBB.1.5 and BA.2.86 Spike proteins. Additionally, GEMCOVAC-OM triggered higher frequencies of Omicron-Spike-specific T-cells, including stem cell, central, and effector memory subsets. In summary, while ChAdOx1 nCoV-19 showed some cross-reactivity, GEMCOVAC-OM induced a more targeted immune response. GEMCOVAC-OM offers a broader, longer-lasting immunity, making it a promising candidate for future vaccine development and global distribution.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01076-2" target="_blank">https://www.nature.com/articles/s41541-025-01076-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-28</strong> <a href="https://www.nature.com/articles/s41598-025-91566-z" target="_blank">
            ü§ñ Detection of COVID-19 in the air and on surfaces at hospitals in Kuala Lumpur.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-114', this)">
            More</button></h4>
            <div id="toggle-114" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Detection of SARS-CoV-2 in bioaerosols and surface samples from healthcare facilities in Klang Valley, Malaysia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

The COVID-19 pandemic made people all over the world sick, especially doctors and nurses who take care of patients in hospitals. But we didn't know if the virus was still present in the air after people left the hospital. This study looked at whether SARS-CoV-2, the virus that causes COVID-19, was still around in the air and on surfaces in hospitals in Malaysia from 2021 to 2022. They found that the virus was more likely to be on surfaces than in the air, and some of the variants of the virus were also present. This means that doctors and nurses need to take extra precautions to prevent the spread of COVID-19 in hospitals.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Coronavirus disease 2019 (COVID-19) pandemic has caused significant global threats, as Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) is primarily transmitted through airborne droplets and bioaerosols. Healthcare workers are particularly at high risk, yet there is limited research on the presence of SARS-CoV-2 in bioaerosols within healthcare facilities in Malaysia. This study aimed to determine the presence and viability of SARS-CoV-2 and its variants of concern in the air and ventilation systems of designated COVID-19 facilities from December 2021 to February 2022. Samples were collected from two hospitals and one quarantine centre (QC), including medical wards, intensive care units, emergency departments, and QC halls. Air samples were obtained using air samplers, while surface samples were taken from return air grilles. SARS-CoV-2 ribonucleic acid (RNA) and its variants were detected using reverse transcription droplet digital polymerase chain reaction (RT-ddPCR) and PCR-based genotyping, respectively. Results showed that Hospital A had a higher rate (24.6%) of positive samples than Hospital B (8.8%). Surface samples had a higher positivity rate (50.0%) compared to air samples (8.3%). The detected variants included delta (34.7%), a mixture of delta and omicron (8.7%), non-variant of concern (non-VOC) (8.7%), and omicron (4.3%). This study emphasizes the need for strict airborne infection control measures for healthcare workers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91566-z" target="_blank">https://www.nature.com/articles/s41598-025-91566-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-28</strong> <a href="https://www.nature.com/articles/s41418-025-01459-x" target="_blank">
            ü§ñ How does a specific protein cause severe symptoms from COVID-19 without involving certain cell death processes?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-115', this)">
            More</button></h4>
            <div id="toggle-115" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>IL-1Œ≤ drives SARS-CoV-2-induced disease independently of the inflammasome and pyroptosis signalling</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Severe COVID-19 causes big swelling in the body. When this happens, it can lead to cell death called pyroptosis. Researchers studied how this pyroptosis might help COVID-19 spread and cause sickness. They found that removing certain parts of the cells that make this pyroptosis didn't stop the virus from spreading or make people healthier. However, they did find that if IL-1Œ≤ (a type of protein) wasn't released by the cells, it helped make people feel better and live longer with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Excessive inflammation and cytokine release are hallmarks of severe COVID-19. Certain programmed cell death processes can drive inflammation, however, their role in the pathogenesis of severe COVID-19 is unclear. Pyroptosis is a pro-inflammatory form of regulated cell death initiated by inflammasomes and executed by the pore-forming protein gasdermin D (GSDMD). Using an established mouse adapted SARS-CoV-2 virus and a panel of gene-targeted mice we found that deletion of the inflammasome (NLRP1/3 and the adaptor ASC) and pore forming proteins involved in pyroptosis (GSDMA/C/D/E) only marginally reduced IL-1Œ≤ levels and did not impact disease outcome or viral loads. Furthermore, we found that SARS-CoV-2 infection did not trigger GSDMD activation in mouse lungs. Finally, we did not observe any difference between WT animals and mice with compound deficiencies in the pro-inflammatory initiator caspases (C1/11/12‚àí/‚àí). This indicates that the classical canonical and non-canonical pro-inflammatory caspases known to process and activate pro-IL-1Œ≤, pro-IL-18 and GSDMD do not substantially contribute to SARS-CoV-2 pathogenesis. However, the loss of IL-1Œ≤, but not the absence of IL-18, ameliorated disease and enhanced survival in SARS-CoV-2 infected animals compared to wildtype mice. Collectively, these findings demonstrate that IL-1Œ≤ is an important factor contributing to severe SARS-CoV-2 disease, but its release was largely independent of inflammasome and pyroptotic pathways.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41418-025-01459-x" target="_blank">https://www.nature.com/articles/s41418-025-01459-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-28</strong> <a href="https://www.nature.com/articles/s41467-025-57267-x" target="_blank">
            ü§ñ Aging Hamsters Get Shortness of Breath After Exercise Due to Lung Damage</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-116', this)">
            More</button></h4>
            <div id="toggle-116" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Respiratory long COVID in aged hamsters features impaired lung function post-exercise with bronchiolization and fibrosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Millions of people got very sick with a coronavirus called SARS-CoV-2, which can cause long-term problems with breathing. We studied how the lungs heal after being infected, using hamsters as models. 

We found that after the initial sickness, people's lungs still didn't work properly for a long time - up to 7 weeks after getting sick! But only if they exercised did their lungs start to show signs of improvement.

Our study showed that some parts of the lung kept getting scarred and damaged, which is similar to what happens in people with respiratory problems from COVID-19. However, our hamsters didn't develop other long-term symptoms like some people do.

We also noticed that certain cells in the lungs started to change and behave differently over time. These changes might be related to long-term health problems for people who got COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-term consequences of SARS-CoV-2 infection affect millions of people and strain public health systems. The underlying pathomechanisms remain unclear, necessitating further research in appropriate animal models. This study aimed to characterize the trajectory of lung regeneration over 112 days in the male hamster model by combining morphological, transcriptomic and functional readouts. We demonstrate that in the acute phase, SARS-CoV-2 Delta-infected, male, aged hamsters show a severe impairment of lung function at rest. In the chronic phase, similar impairments persisted up to 7 weeks post-infection but were only evident after exercise on a rodent treadmill. The male hamster model recapitulates chronic pulmonary fibrotic changes observed in many patients with respiratory long COVID, but lacks extra-pulmonary long-term lesions. We show that sub-pleural and interstitial pulmonary fibrosis as well as alveolar bronchiolization persist until 112 dpi. Interestingly, CK8+alveolar differentiation intermediate (ADI) cells are becoming less prominent in the alveolar proliferation areas from 28 dpi on. Instead, CK14+airway basal cells and SCGB1A1+club cells, expressing cell proliferation markers, mainly populate alveolar bronchiolization areas at later time-points. We postulate that pulmonary fibrosis and SCGB1A1+club cell-rich areas of alveolar bronchiolization represent potential risk factors for other diseases in long-COVID survivors.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57267-x" target="_blank">https://www.nature.com/articles/s41467-025-57267-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-27</strong> <a href="https://www.nature.com/articles/s41541-025-01095-z" target="_blank">
            ü§ñ How Does COVID Infection Help Build Resistance to Other Respiratory Viruses?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-117', this)">
            More</button></h4>
            <div id="toggle-117" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection primes cross-protective respiratory IgA in a MyD88- and MAVS-dependent manner</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that simplifies the original text while retaining its meaning:

Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) keeps changing, making it harder for our bodies to fight off. We've found that some antibodies in the nose and throat can help prevent infections better than others. Now we're trying to understand how the immune system works to create vaccines that can protect us against many different versions of SARS-CoV-2.

We tested this idea by giving hamsters a vaccine that had been made from an inactivated virus, and they were completely protected against many different strains of SARS-CoV-2. We also found that boosting the immune system with just the Spike protein (a part of the virus) could create strong antibodies in mice that helped protect them from infection.

Our findings suggest that developing vaccines that can work against many different versions of SARS-CoV-2 is possible, and we're one step closer to creating a better way to fight this virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is constantly evolving mutations in the Spike protein to evade humoral immunity. Respiratory tract antiviral IgA antibodies are superior to circulating IgG antibodies in preventing SARS-CoV-2 infection. However, the role of innate immune signals required for the induction of mucosal IgA against SARS-CoV-2 infection is unknown. Here we show that hamsters recovered from ancestral SARS-CoV-2 infection are cross-protected against heterologous SARS-CoV-2 alpha, gamma, delta, and omicron BA.1 variants. Intranasal vaccination with an inactivated whole virus vaccine completely protects hamsters against heterologous SARS-CoV-2 infection. In addition, we show that intranasal boost vaccination of mice recovered from SARS-CoV-2 infection with unadjuvanted Spike protein induces robust levels of respiratory anti-Spike IgA and protects the mice from a heterologous SARS-CoV-2 infection. Furthermore, our findings suggest that MyD88 and MAVS play a role in the induction of the memory IgA response following an intranasal booster with unadjuvanted Spike protein in mice recovered from the SARS-CoV-2 infection. These findings provide a useful basis for the development of cross-protective mucosal vaccines against heterologous SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01095-z" target="_blank">https://www.nature.com/articles/s41541-025-01095-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-91439-5" target="_blank">
            ü§ñ COVID-19 causes lung problems in people with kidney transplants and how their immune system reacts.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-118', this)">
            More</button></h4>
            <div id="toggle-118" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immune dysregulation in COVID-19 induced ARDS in kidney transplant recipients revealed by single-cell RNA sequencing</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who got a kidney transplant are more likely to get very sick from COVID-19 than people who didn't get a transplant. This study looked at how their bodies respond to COVID-19 and found some key differences.

Researchers took blood samples from 108,000 cells and used special machines to see what was happening inside those cells when someone with a kidney transplant got COVID-19. They found that certain immune cells were more active or less active in people with kidney transplants compared to those without transplants.

In general, the study found that:

* Immune cells in people with kidney transplants made less of an inflammatory response than people who didn't get a transplant.
* Some immune cells that help fight off infections were missing or not working properly in people with kidney transplants.
* This makes it harder for their bodies to recover from COVID-19.

The study's main goal was to find ways to improve treatment for people with kidney transplants who get COVID-19. By understanding how their immune systems work differently, researchers can develop new strategies to help them fight off the infection more effectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSince the emergence of COVID-19 at the end of 2019, the disease has led to widespread acute respiratory distress syndrome (ARDS), particularly among kidney transplant recipients (KTRs), who are at increased risk due to long-term immunosuppressive therapy. This study aims to explore the differences in immune responses between kidney transplant recipients and non-kidney transplant recipients in COVID-19-induced ARDS to identify potential therapeutic targets for improving outcomes. Single-cell RNA sequencing was performed on 108,320 cells derived from peripheral blood samples to construct a global single-cell map of COVID-19 induced ARDS in kidney transplant recipients(ARDSKT), COVID-19 induced ARDS in non transplant recipients(ARDSNKT), and healthy controls. Subsequently, using cellular clustering analysis, we obtained single-cell maps of different cell types. We employed enrichment analysis to determine the pathways involved in different subpopulations and focused on the role of key immune cells such as monocytes, megakaryocytes, B cells, and CD8+T cells in the pathogenesis of ARDS. Significant immune differences were observed between ARDSKTand ARDSNKT. In ARDSKT, the S100A9+MK subpopulation, which activates the NF-Œ∫B signaling pathway, was elevated, promoting inflammation. In contrast, the S100A12+monocyte subpopulation that activates the chemokine signaling pathway was more abundant in ARDSNKT, reflecting a stronger inflammatory response, while its abundance was reduced in ARDSKTdue to immunosuppression. The CXCR4+B subpopulation, crucial for adaptive immunity, was significantly reduced in ARDSKT. Additionally, the XAF1+Teff subpopulation, associated with apoptosis, was more abundant in ARDSKT, potentially impairing immune recovery. This study highlights the immune differences between ARDSKTand ARDSNKT, revealing the impact of immunosuppression on immune dysregulation. These findings suggest that targeting specific immune pathways can improve therapeutic strategies for ARDSKT.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91439-5" target="_blank">https://www.nature.com/articles/s41598-025-91439-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41392-025-02154-6" target="_blank">
            ü§ñ New vaccine protects against certain strains of COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-119', this)">
            More</button></h4>
            <div id="toggle-119" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A recombinant protein vaccine induces protective immunity against SARS-CoV-2 JN.1 and XBB-lineage subvariants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new type of COVID-19 vaccine has been made to protect against the latest variants of the virus. The main goal is to stop the immune system from getting used to older versions of the virus and not being able to fight off newer ones. This vaccine targets a part of the virus that changes often, so it can keep up with new variants. It's been shown to work well in tests and provides protection against some of the most common new variants. The good news is that even if you've already had COVID-19 or vaccinated previously, this vaccine can still boost your immunity and protect you from getting sick again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The emergence of XBB- and JN.1-lineages with remarkable immune evasion characteristics have led to rises in breakthrough infections within populations. In addition, the unfavorable impacts of immune imprinting, stemming from continuous exposure to antigens from circulated viruses, have been observed to incline immune response against earlier lineages, thereby declining the neutralization to newly emerged Omicron subvariants. In response to this, the advancement of next-generation vaccines against COVID-19 targeting components from new subvariants such as XBB-lineage is imperative. In the current study, a self-assembled trimeric recombinant protein (RBDXBB.1.5-HR) was generated by concatenating the sequences of the receptor binding domain (RBD) derived from XBB.1.5 with heptad-repeat 1 (HR1) and HR2 sequences from the spike S2 subunit. Adjuvanted-RBDXBB.1.5-HR induced robust humoral and cellular immune responses, characterized by elevated neutralization against JN.1-inculuded subvariants and a substantial population of antigen-specific T memory cells. Protective immunity conferred by RBDXBB.1.5-HR vaccine was preserved post-immunization, as evidenced by germinal center B (GC B) and T follicular helper (Tfh) responses, sustained neutralization potency, and an increase in memory B cells (MBCs) and long-lived plasma cells (LLPCs). The RBDXBB.1.5-HR vaccine showed a favorable boosting effect when administered heterologously after three doses of inactivated virus (IV) and mRNA vaccines. Significantly, it provided protection against live Omicron EG.5.1 viruses in vivo. The monovalent RBDXBB.1.5-HR vaccine showed favorable safety and immunogenicity, boosting neutralizing antibodies against JN.1- and XBB-lineage subvariants in individuals with prior COVID-19 vaccinations. These findings highlight its clinical potential in safeguarding against circulating Omicron subvariants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-025-02154-6" target="_blank">https://www.nature.com/articles/s41392-025-02154-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-90819-1" target="_blank">
            ü§ñ Post COVID-19 health problems in people who were hospitalised or treated at home.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-120', this)">
            More</button></h4>
            <div id="toggle-120" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID-19 condition in prospective inpatient and outpatient cohorts</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>After getting very sick from COVID-19, some people experience long-term symptoms that don't go away easily. This is called post-COVID condition.

To figure out what happens after a person gets COVID-19, researchers looked at 115 people who had the virus for more than two years. They checked if these people still had symptoms like coughing or feeling tired all the time.

The researchers found that some people were still sick even after almost two years. These people might have had a bad case of COVID-19 to start with, or they might be overweight or have sleep problems.

They also looked at how well their bodies worked and what was in their blood. They found out that some people's immune systems got mixed up after getting COVID-19, which made it harder for them to get better.

The researchers want to do more studies to understand why this happens to some people and how to help those who are still sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Viral persistence, immune dysregulation, hypocortisolism, and pulmonary tissue damage from acute infection are proposed as pathogenic mechanisms underlying post-COVID-19 condition (PCC). In this prospective observational study, we followed 62 COVID-19 inpatients and 53 COVID-19 outpatients for 24 months after the infection. During this period, we assessed prolonged symptoms, lung function, and a set of immunological markers and a proportion of the patient group was assessed with computed tomography three months post-infection. The prevalence of PCC, as assessed by four medical specialists, decreased from 51% at three months to 18% at 24 months. Risk factors included the severity of the acute infection and comorbidities of obstructive sleep apnea or obesity. Patients with PCC had higher serum levels of anti-SARS-CoV-2 S1 and N protein antibodies. In the whole group, spirometry results, orthostatic hypotension, or levels of soluble suppression of tumorigenicity 2, interleukin 6 (IL-6), high-sensitivity C-reactive protein (hs-CRP), or cortisol had no association with PCC. However, using symptom clusters, patients with cognitive problems had lower cortisol levels, while patients with ongoing respiratory or myalgic symptoms had higher levels of IL-6 and hs-CRP. However, more extensive studies with clustering are needed to validate these results.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90819-1" target="_blank">https://www.nature.com/articles/s41598-025-90819-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-91388-z" target="_blank">
            ü§ñ Factors that affect people's quality of life after having COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-121', this)">
            More</button></h4>
            <div id="toggle-121" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Factors associated with reduction in quality of life after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who had COVID-19 often experience long-term changes in how they feel about their health and well-being. This study looked at 3,475 people who had COVID-19 to see what factors were most related to their quality of life.

The researchers used a questionnaire to measure the participants' quality of life before and after their initial infection. They found that fatigue, muscle pain, stress, and age were important predictors of quality of life both at the beginning and after 9 months.

As time went on, they discovered that improvements in these symptoms were associated with better quality of life. However, it's impossible to predict whether someone's quality of life will improve or worsen over the next year based on their initial symptoms.

The study highlights how certain symptoms like fatigue and stress can impact people's ability to do daily activities.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-term changes in health-related quality of life (HrQoL) after SARS-CoV-2 infection are common, but their causes and consequences are poorly understood. This prospective, population-based study examined associations between HrQoL and 49 demographic and clinical variables. HrQoL was assessed using the European Quality-of-Life-5-Dimensions-5-Level-Version in 3,475 participants (56% female; aged 18‚Äì88 years) approximately 9 months (baseline) and 26 months (follow-up) after their initial infection. Results were compared with the demographic and clinical variables using recursive feature elimination and random forest regression analyses. A statistically significant improvement in HrQoL was observed during the observation period. At baseline, 39% of the variance in HrQoL was explained by fatigue, muscle pain, number of remaining symptoms (RS), perceived stress, and age. At follow-up, fatigue, RS, perceived stress, muscle and joint pain, and age explained 54% of the variance in HrQoL. Changes in HrQoL were associated with changes in fatigue, RS, and perceived stress, meaning that if these decreased from baseline to follow-up, then HrQoL was improved. However, it was not possible to predict whether an individual‚Äôs HrQoL would improve or worsen 1¬†year later based on baseline variable scores. The aforementioned symptoms are specifically associated with impairment in the population‚Äôs usual activities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91388-z" target="_blank">https://www.nature.com/articles/s41598-025-91388-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-90918-z" target="_blank">
            ü§ñ Effect of Oxidation on COVID-19 Spike Protein Structure</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-122', this)">
            More</button></h4>
            <div id="toggle-122" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The effect of cysteine oxidation on conformational changes of SARS-CoV-2 spike protein using atomistic simulations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus uses a special protein to get into human cells. This protein has a group that can get damaged by oxygen, which happens during infections. We used computer simulations to see how this damage affects the protein's shape and how it works.

We found that when this group gets damaged, it makes it easier for the protein to change its shape and move towards its target in human cells. This could make the virus more likely to get into cells.

We also looked at how this damage changes the way the protein interacts with other molecules on the surface of the cell. Our study shows that damaging this group can disrupt the normal functioning of the protein, which could be a target for new treatments aimed at reducing oxidative stress or modifying the protein's structure.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 Spike (S) protein plays a central role in viral entry into host cells, making it a key target for therapeutic interventions. Oxidative stress, often triggered during viral infections, can cause oxidation of cysteine in this protein. Here we investigate the impact of cysteine oxidation, specifically the formation of cysteic acid, on the conformational dynamics of the SARS-CoV-2 S protein using atomistic simulations. In particular, we examine how cysteine oxidation influences the transitions of the S protein‚Äôs receptor-binding domain (RBD) between ‚Äúdown‚Äù (inaccessible) and ‚Äúup‚Äù (accessible) states, which are critical for host cell receptor engagement. Using solvent-accessible surface area (SASA) analysis, we identify key cysteine residues susceptible to oxidation. The results of targeted molecular dynamics (TMD) and umbrella sampling (US) simulations reveal that oxidation reduces the energy barrier for RBD transitions by approximately 30¬†kJ mol‚àí1, facilitating conformational changes and potentially enhancing viral infectivity. Furthermore, we analyze the interactions between oxidized cysteine residues and glycans, as well as alterations in hydrogen bonds and salt bridges. Our results show that oxidation disrupts normal RBD dynamics, influencing the energy landscape of conformational transitions. Our work provides novel insights into the role of cysteine oxidation in modulating the structural dynamics of the SARS-CoV-2 S protein, highlighting potential targets for antiviral strategies aimed at reducing oxidative stress or modifying post-translational changes. These findings contribute to a deeper understanding of viral infectivity and pathogenesis under oxidative conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90918-z" target="_blank">https://www.nature.com/articles/s41598-025-90918-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-86631-6" target="_blank">
            ü§ñ Double-Blind Trial to See if Hyperoxygen Therapy Helps People with Persistent Symptoms After Brain Injury</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-123', this)">
            More</button></h4>
            <div id="toggle-123" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A double-blind randomized trial of hyperbaric oxygen for persistent symptoms after brain injury</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler terms:

We did a study to see if hyperbaric oxygen therapy (HBO) can help people with brain injuries who have persistent symptoms. We gave one group 40 sessions of HBO and another group 40 fake sessions over 12 weeks. Three months later, both groups got more real HBO sessions.

At different times during the study, we asked participants to fill out questionnaires about how they felt, including their sense of smell, anxiety, sleep problems, and balance issues. We also did some tests to measure their brain function and daily life.

The results showed that the people who got actual HBO had more improvements in these areas compared to the group that got fake sessions. Both groups improved on things like depression, headaches, and quality of life. Even after getting additional HBO sessions later, the original HBO group continued to improve.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In this double-blind randomized trial, adults with persistent symptoms following non-stroke brain injury received 40 hyperbaric oxygen (HBO2) sessions or 40 sham sessions over 12 weeks. Three months later, all were offered 40 unblinded HBO2sessions. Participants completed the Neurobehavioral Symptom Inventory (NSI) at baseline, 13 weeks (after 40 chamber sessions), 6 months, 9 months (after the second chamber series), and 12 months, with prime outcome at 13 weeks, and additional questionnaires, neuropsychological tests, and functional measures. We enrolled 49 participants and analyzed 47 due to drop-out/exclusion (26 males, 40 with traumatic brain injury). Baseline NSI was 35.9‚Äâ¬±‚Äâ15.8 in the HBO2group (n‚Äâ=26) and 30.7‚Äâ¬±‚Äâ16.9 in the sham group (n‚Äâ=21) (p=‚Äâ0.28). Mean 13-week change scores were 10.6‚Äâ¬±‚Äâ10.6 (HBO2group) and 3.6‚Äâ¬±‚Äâ5.9 (sham group) (mean difference 7.0, 95% CI 1.7‚Äì12.3,p=‚Äâ0.01). The HBO2group improved on measures of olfaction, anxiety, sleep difficulties, and vestibular complaints. Both groups reported improvements in depression, headaches, PTSD symptoms, physical quality of life, and degree to which difficulties interfere with daily life. With an additional 40 HBO2sessions, the original HBO2group reported additional improvements on NSI at 12 months. Only 15 original sham participants completed the second chamber series, limiting conclusions from that data.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86631-6" target="_blank">https://www.nature.com/articles/s41598-025-86631-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41599-025-04567-8" target="_blank">
            ü§ñ COVID-19 Pandemic's Impact on Dental Education in Peru</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-124', this)">
            More</button></h4>
            <div id="toggle-124" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Transversal competencies among Peruvian dental students in times of the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

During the COVID-19 pandemic, dental students in Peru were asked about their skills and abilities to work with others, manage tasks, and interact with people from different backgrounds. This study looked at what these skills are, how they relate to each other, and whether they differ between certain groups of students.

Researchers used a survey that included questions about the students' personal characteristics and their skills in three main areas: working with people, managing tasks, and understanding systems. They found that some of the skills were similar across all groups, but others showed different levels of ability.

They also discovered that students who worked well with others had lower skills in other areas, like task management. However, they didn't find any strong connections between these skills and things like age or income.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Transversal competencies are not related to any specific degree program but are crucial with respect to university students‚Äô academic and professional training and entry into the labor force. Therefore, this study aimed to identify the transversal competencies (including instrumental, interpersonal, and systemic competencies) exhibited by Peruvian dental students in times of the COVID-19 pandemic as well as the relationships between these competencies and various sociodemographic characteristics. A cross-sectional study second to 6th-year students in a Peruvian dental school was performed. A validated paper and pencil questionnaire that collected information concerning transversal competencies was used in this research. This questionnaire contained three sections; the first section solicited informed consent, the second section focused on the sociodemographic characteristics of participants (6 items) and the third section pertained to transversal competencies (185 items). The survey was conducted between May and August 2022. Descriptive statistics were calculated with respect to each of the competencies investigated in this research. The Friedman, Dunn‚ÄíBonferroni post-hoc, Mann‚ÄíWhitney U, and Kruskal‚ÄíWallis tests were used to compare the variables included in this study. The three categories of competencies exhibited similar median values (3), while four subcompetencies exhibited lower median values. Furthermore, interpersonal competencies exhibited significantly lower values (p=‚Äâ0.003) than did instrumental or systemic competencies. No statistically significant relationships were observed between each group of transversal competencies and the corresponding sociodemographic variables.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-025-04567-8" target="_blank">https://www.nature.com/articles/s41599-025-04567-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-26</strong> <a href="https://www.nature.com/articles/s41598-025-90262-2" target="_blank">
            ü§ñ Understanding Lactobacillus Strains in Egyptian Dairy Products</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-125', this)">
            More</button></h4>
            <div id="toggle-125" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Whole genome sequencing of promisingLactobacillus delbrueckiisubsp.bulgaricusstrains isolated from Egyptian dairy products for probiotic characteristics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Imagine taking tiny helpers called probiotics that can make you healthier. They're like superheroes for your body. Researchers in Egypt collected 22 samples from dairy products to see if they had special powers against bad bacteria, viruses, and other germs. They tested these superpowers in extreme conditions and found three promising ones that could survive in the human gut. Further analysis showed these microbes were safe and effective, with helpful genes that kept them strong and healthy for humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Probiotics are living microorganisms that, when given in an adequate dose, have a healthy impact on human well-being. With global interest in self-care, dietary supplements especially probiotics is expanding rapidly due to their supported health effects. In this study, a total of twenty-two samples were collected from dairy products in Egypt‚Äôs markets, firstly isolated then identified and screened for probiotic properties under stressful conditions as NaCl, acid and bile salt conditions. After evaluation of the antimicrobial effect against widespread gram negative and gram positive human infecting bacteria, besides the antiviral assessment against (SARS-CoV-2)¬†virus which has disturbed the world, the antibiotic susceptibility test was done. Only three promising isolates were subjected for whole genome sequence with high-output next generation technology and the obtained data was subjected to a full bioinformatics analysis. The results obtained stated the advance of our isolates to tolerate the stress factors that can struggle in the human gut as well as the antimicrobial effects. All these bright characteristics were confirmed and illustrated in detail through different and reliable genome analysis tools. Our isolates were founded to have stable genome through containing mobile genetic elements like phages and CRISPR clusters that confirm the safety and quality for human health.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90262-2" target="_blank">https://www.nature.com/articles/s41598-025-90262-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-25</strong> <a href="https://www.nature.com/articles/s41598-025-91010-2" target="_blank">
            ü§ñ Neuroleptic Medicines Used in Severe COVID Patients Who Have Difficulty Breathing After Leaving the Hospital</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-126', this)">
            More</button></h4>
            <div id="toggle-126" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neuroleptics used in critical COVID associated with moderate-severe dyspnea after hospital discharge</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Dyspnea, or shortness of breath, is a big problem for some people who got COVID-19 a long time ago. Doctors want to know why it happens and how they can help. This study looked at 100 patients who got COVID-19 and saw what happened to them afterwards.

The study found that:

* Many people had trouble breathing after being in the hospital (56.6%).
* Some things that the doctors gave these patients during their stay in the hospital did not seem to make a difference.
* The amount of time they spent in the hospital with certain medicines was a factor.
* People who already had other health problems were more likely to have trouble breathing after being discharged.
* When people had trouble breathing, it could last for a long time and make them feel sad, anxious, or weak.

The study says that patients who got severe COVID-19 are more likely to have trouble breathing in the long term. The doctors recommend that these patients get special care to help them feel better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Dyspnea is a prominent symptom in patients with long COVID due to its high prevalence and significant clinical impact. However, the influence of commonly used medications in critically ill patients on long-term dyspnea remains unclear. This study aimed to identify risk factors and assess the impacts associated with moderate to severe dyspnea in COVID-19 survivors. This study evaluated patients admitted to a university hospital between April 2020 and April 2021. Data were collected on clinical preconditions, hospital and ICU stays, and the use of corticosteroids, neuroleptics, neuromuscular blockers, midazolam, fentanyl, and noradrenaline. Post-discharge evaluations were conducted at 1 and 12 months, assessing dyspnea, frailty, quality of life, functional capacity, anxiety, and depression. Descriptive statistics, including frequencies and percentages, were used, and logistic regression analysis was performed to identify factors associated with moderate to severe dyspnea at 1 and 12 months post-discharge. Statistical significance was defined asP<‚Äâ0.05. A total of 100 patients were prospectively included in the study; all underwent the 1-month evaluation, and 63 completed the 12-month evaluation. Limiting dyspnea, defined as an mMRC score‚Äâ>‚Äâ1, was observed in 56.6% of patients at 1 month and 33.9% at 12 months post-discharge. Independent factors associated with limiting dyspnea at 1 month included the total dose of neuroleptics administered during hospitalization and the presence of pre-existing comorbidities. The use of corticosteroids, neuromuscular blockers, midazolam, fentanyl, and noradrenaline showed no significant association with limiting dyspnea. Dyspnea at 1 month post-discharge was an independent risk factor for the persistence of limiting dyspnea at 12 months. Patients with limiting dyspnea at 12 months exhibited higher levels of depression, anxiety, and frailty, alongside reduced quality of life and functionality. Patients with severe COVID-19 exhibit a high prevalence of limiting dyspnea in the long term. The total dose of neuroleptics administered during hospitalization and the presence of comorbidities were independently associated with limiting dyspnea after discharge. At 12 months post-discharge, individuals with persistent limiting dyspnea frequently demonstrated additional physical and mental health impairments, underscoring the need for comprehensive evaluation and management to mitigate the burden of long-term disabilities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91010-2" target="_blank">https://www.nature.com/articles/s41598-025-91010-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s44298-025-00092-2" target="_blank">
            ü§ñ How these new COVID variants spread in animals.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-127', this)">
            More</button></h4>
            <div id="toggle-127" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pathogenesis and transmission of SARS-CoV-2 D614G, Alpha, Gamma, Delta, and Omicron variants in golden hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

New versions of the coronavirus that cause COVID-19 have been spreading. We tested these new versions, called D614G, Alpha, Gamma, Delta, and Omicron, to see how well they work, how easily they spread, and how sick they make people.

We compared them in different ways:
- In a special laboratory test, some viruses replicated faster and were better at spreading than others.
- We also looked at how the viruses affected lung cells and whether they could be transmitted through the air.
- Some viruses, like Delta and Omicron, spread more easily and made people sicker. Others, like Gamma, caused more damage to the lungs.

These tests help us understand new variants of the coronavirus and prepare for future changes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Since the emergence of SARS-CoV-2 in humans, novel variants have evolved to become dominant circulating lineages. These include D614G (B.1 lineage), Alpha (B.1.1.7), Gamma (P.1), Delta (B.1.617.2), and Omicron BA.1 (B.1.1.529) and BA.2 (B.1.1.529.2) viruses. Here, we compared the viral replication, pathogenesis, and transmissibility of these variants. Replication kinetics and innate immune response against the viruses were tested in ex vivo human nasal epithelial cells (HNEC) and induced pluripotent stem cell-derived lung organoids (IPSC-LOs), and the golden hamster model was employed to test pathogenicity and potential for transmission by the respiratory route. Delta, BA.1, and BA.2 viruses replicated more efficiently, and outcompeted D614G, Alpha, and Gamma viruses in an HNEC competition assay. BA.1 and BA.2 viruses, however, replicated poorly in IPSC-LOs compared to other variants. Moreover, BA.2 virus infection significantly increased secretion of IFN-Œª1, IFN-Œª2, IFN-Œª3, IL-6, and IL-1RA in HNECs relative to D614G infection, but not in IPSC-LOs. The BA.1 and BA.2 viruses replicated less effectively in hamster lungs compared to the other variants; and while the Gamma virus reached titers comparable to D614G and Delta viruses, it caused greater lung pathology. Lastly, the Gamma and Delta variants transmitted more efficiently by the respiratory route compared to the other viruses, while BA.1 and BA.2 viruses transmitted less efficiently. These findings demonstrate the ongoing utility of experimental risk assessment as SARS-CoV-2 variants continue to evolve.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-025-00092-2" target="_blank">https://www.nature.com/articles/s44298-025-00092-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s41467-025-57198-7" target="_blank">
            ü§ñ COVID-19 Recovery Treatment with Blood Exchange Trial</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-128', this)">
            More</button></h4>
            <div id="toggle-128" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Plasma exchange therapy for the post COVID-19 condition: a phase II, double-blind, placebo-controlled, randomized trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Millions of people got sick with COVID-19 and still feel terrible afterwards. There's no good way to make them feel better yet. Researchers tried using a treatment called plasma exchange to help people with this feeling. They gave some people real treatment and some fake treatment, and then followed up with them for 3 months. The results showed that both groups felt safe while being treated, but the treatment didn't actually make anyone's symptoms better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The post-COVID-19 condition (PCC) is a highly debilitating and persistent postinfectious syndrome that affects millions of people worldwide and has no effective treatment. Therapeutic plasma exchange (TPE) has the potential to improve the PCC by clearing the peripheral soluble pro-inflammatory immune milieu derived from acute or persistent SARS-CoV-2 infection. In a phase II, double-blind, placebo-controlled, randomized trial, fifty subjects with PCC were randomly assigned (1:1) to receive six sessions of either TPE or a sham plasma exchange and were followed for 90 days (ClinicalTrials.gov registration: NCT05445674). The primary endpoint was safety; secondary endpoints included functional status, symptomology, quality of life, neurocognitive symptoms, and peripheral biochemistry, hematology, coagulation and inflammation parameters. Both study arms had a similarly favorable safety profile. There were no diferences between groups in any of the efficacy parameters evaluated. Whereas TPE is safe, it did not lead to any discernible improvement of the PCC in this clinical trial.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57198-7" target="_blank">https://www.nature.com/articles/s41467-025-57198-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s41598-025-90324-5" target="_blank">
            ü§ñ COVID-19 Immune Response in Hospitalized Patients in Sardinia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-129', this)">
            More</button></h4>
            <div id="toggle-129" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Viral and immune profiles during the first wave of SARS-CoV-2 infection in hospitalized patients in Sardinia, Italy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We studied people who had COVID-19 in Italy four months after they got sick. We took blood and nasal swab samples from them to see if their bodies still had antibodies that could fight the virus. We found that these antibodies not only worked well in the bloodstream but also helped protect the nose and throat from getting infected. We also tested how well the body's antibodies would work against different versions of the virus, like Omicron. What we found was surprising: people who got COVID-19 earlier had weaker protection against newer Omicron variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We performed a retrospective immunological analysis of the antibody response in serum and in nasopharyngeal swabs (NPS) obtained from 46 individuals infected with ancestral SARS-CoV-2 Wuhan-Hu-1 strain during the first COVID-19 wave in Cagliari (Sardinia, Italy), with a 4-month follow-up after the hospital admission. We implemented a comprehensive antibody response in serum and in mucosal samples using assays established in our laboratories. In NPS we evaluated the viral load by real time PCR, presence and kinetics of anti-Spike IgG and IgA by ELISA as well as their anti-Wuhan neutralization activity, showing induction and persistence of anti-viral immunity at the mucosal level. Neutralizing antibodies were measured in serum and NPS using a safe pseudovirus-based assay validated after comparison with a standard neutralization test using live SARS-CoV-2. We evaluated cross-neutralizing antibodies against all the major early variants of concerns (VoC) in sera. Of note, we detected a remarkable reduction of neutralizing activity against BA.1 compared to BA.2 and BA.5 Omicron subvariants, which was confirmed in sera from an analogous cohort of patients at the San Raffaele hospital in Milan, a geographically distant region of Italy, infected with the ancestral virus during the same period of time.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90324-5" target="_blank">https://www.nature.com/articles/s41598-025-90324-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s44161-025-00612-6" target="_blank">
            ü§ñ Effects of SARS-CoV-2 and COVID-19 vaccine on heart muscle cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-130', this)">
            More</button></h4>
            <div id="toggle-130" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The cellular and molecular cardiac tissue responses in human inflammatory cardiomyopathies after SARS-CoV-2 infection and COVID-19 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that simplifies the original text for a young person:

Myocarditis, a condition where the heart muscle becomes inflamed, can be caused by different things like the coronavirus or even a vaccine. We studied people who got myocarditis after getting vaccinated (Post-Vaccination) and those who had it after having COVID-19 (Post-COVID-19). Our goal was to understand what's happening in their bodies.

We found that even though some parts of their immune system are similar, the way the body reacts to each situation is different. For example, people who got vaccinated showed a certain pattern of inflammation in their blood, while those who had COVID-19 showed a different pattern. We also saw that some cells in the heart and blood vessels were acting differently depending on what happened.

This helps us understand how myocarditis happens in people with or without COVID-19 infection or vaccination, and might lead to new ways to treat it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Myocarditis, characterized by inflammatory cell infiltration, can have multiple etiologies, including severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection or, rarely, mRNA-based coronavirus disease 2019 (COVID-19) vaccination. The underlying cellular and molecular mechanisms remain poorly understood. In this study, we performed single-nucleus RNA sequencing on left ventricular endomyocardial biopsies from patients with myocarditis unrelated to COVID-19 (Non-COVID-19), after SARS-CoV-2 infection (Post-COVID-19) and after COVID-19 vaccination (Post-Vaccination). We identified distinct cytokine expression patterns, with interferon-Œ≥ playing a key role in Post-COVID-19, and upregulatedIL16andIL18expression serving as a hallmark of Post-Vaccination myocarditis. Although myeloid responses were similar across all groups, the Post-Vaccination group showed a higher proportion of CD4+T cells, and the Post-COVID-19 group exhibited an expansion of cytotoxic CD8+T and natural killer cells. Endothelial cells showed gene expression changes indicative of vascular barrier dysfunction in the Post-COVID-19 group and ongoing angiogenesis across all groups. These findings highlight shared and distinct mechanisms driving myocarditis in patients with and without a history of SARS-CoV-2 infection or vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44161-025-00612-6" target="_blank">https://www.nature.com/articles/s44161-025-00612-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s41598-025-90730-9" target="_blank">
            ü§ñ What antibody test after vaccination helps detect if the body can fight COVID-19?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-131', this)">
            More</button></h4>
            <div id="toggle-131" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 neutralizing antibody determination after vaccination using spectrophotometric measurement of lateral flow immunochromatography</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers wanted to see if a new test could detect whether people have immunity to COVID-19 after getting vaccinated. They compared this new test with an existing one and found it was very accurate, even in small amounts of antibodies. The new test showed high success rates (99% positive, 92% negative) when compared to the existing test. However, for some groups, like those who had only received one Pfizer vaccine, the accuracy was lower. This could be helpful because it would make it easier to measure vaccine responses without needing special equipment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Neutralizing antibody titers have been found to be strongly correlated with observed vaccine effectiveness against symptomatic and severe COVID-19. Few non-high complexity assays are currently available to detect the presence of neutralizing antibodies. This retrospective single-center cross-sectional study compared the performance of a lateral flow immunochromatography assay coupled with a spectrophotometric measurement system for detecting SARS-CoV-2 neutralizing antibodies against an enzyme-linked immunosorbent assay (ELISA) neutralization antibody assay in the context of post-vaccination responses. The limit of detection was similar to the ELISA with strong linearity throughout the measuring interval. Repeatability, interfering substances, and cross-reactivity studies were found to be robust. Results for 274 plasma samples on whom SARS-CoV-2 RNA test and vaccination status, including vaccination number and manufacturer, was known showed a positive predictive value (PPV) of 99.0% (CI 96.4‚Äì99.7%) and a negative predictive value (NPV) of 91.9% (CI 83.4‚Äì96.2%) compared to ELISA. The PPV for all vaccination number and manufacturer subgroups was‚Äâ>‚Äâ95% except for those individuals who had only 1 Pfizer vaccination (PPV of 80%). The NPV for those who were PCR positive with no vaccinations was 100% while only 88.1% for those without a previous positive test or vaccination. The NPV for those with Pfizer vaccinations was 80% in contrast to 100% for those with Moderna vaccinations. Alternative methodologies requiring less sophisticated laboratory support to measure neutralizing antibodies may be useful to measure vaccine responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90730-9" target="_blank">https://www.nature.com/articles/s41598-025-90730-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s41563-024-02115-4" target="_blank">
            ü§ñ Medical Records for Vaccine Treatments Using Tiny Needles Under the Skin</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-132', this)">
            More</button></h4>
            <div id="toggle-132" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>On-patient medical record and mRNA therapeutics using intradermal microneedles</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have created a way for doctors to store important medical information on patients' skin using tiny needles that are almost invisible to the eye. These needles carry special particles that hold the patient's medical history and can be read by special cameras. This technology uses artificial intelligence to keep the information safe and can help doctors make good decisions when paper records aren't available.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Medical interventions often require timed series of doses, thus necessitating accurate medical record-keeping. In many global settings, these records are unreliable or unavailable at the point of care, leading to less effective treatments or disease prevention. Here we present an invisible-to-the-naked-eye on-patient medical record-keeping technology that accurately stores medical information in the patient skin as part of microneedles that are used for intradermal therapeutics. We optimize the microneedle design for both a reliable delivery of messenger RNA (mRNA) therapeutics and the near-infrared fluorescent microparticles that encode the on-patient medical record-keeping. Deep learning-based image processing enables encoding and decoding of the information with excellent temporal and spatial robustness. Long-term studies in a swine model demonstrate the safety, efficacy and reliability of this approach for the co-delivery of on-patient medical record-keeping and the mRNA vaccine encoding severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This technology could help healthcare workers make informed decisions in circumstances where reliable record-keeping is unavailable, thus contributing to global healthcare equity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41563-024-02115-4" target="_blank">https://www.nature.com/articles/s41563-024-02115-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-24</strong> <a href="https://www.nature.com/articles/s41598-025-90426-0" target="_blank">
            ü§ñ Comparing how our bodies react to different severe conditions, like COVID-19 and sepsis during surgery.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-133', this)">
            More</button></h4>
            <div id="toggle-133" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative metabolomic analysis reveals shared and unique features of COVID-19 cytokine storm and surgical sepsis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

COVID-19 can cause severe inflammation in the body, similar to sepsis. Studying how our bodies process nutrients during these illnesses can help us understand what happens inside them. This study looked at the way COVID-19 patients and people who have surgery develop inflammation in their blood. It found that both groups had problems with certain parts of their metabolism, such as making amino acids and dealing with nitrogen. The biggest difference was seen in how they handled a specific pathway called kynurenine synthesis, which is involved in energy production.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The clinical manifestations of the cytokine storm (CS) associated with COVID-19 resemble the acute phase of sepsis. Metabolomics may contribute to understanding the specific pathobiology of these two syndromes. The aim of this study was to compare serum metabolomic profiles in CS associated with COVID-19 vs. septic surgery patients. In a retrospective cross-sectional study, serum samples from patients with CS associated with COVID-19, with and without comorbidity, as well as serum samples from patients with surgical sepsis were investigated. Targeted metabolomic analysis was performed on all samples using LC‚ÄìMS/MS. Analysis revealed that similar alterations in the serum metabolome of patients with COVID-19 and surgical septic patients were associated with amino acid metabolism, nitrogen metabolism, inflammatory status, methionine cycle and glycolysis. The most significant difference was found for serum levels of metabolites of kynurenine synthesis, tricarboxylic acid cycle, gamma-aminobutyric acid and niacinamide. The metabolic pathway of cysteine and methionine metabolism was significantly disturbed in COVID-19 and surgical septic patients. For the first time, the similarities and differences between the serum metabolomic profiles of patients with CS associated with COVID-19 and patients with surgical sepsis were investigated for patients from the Northwest of the Russian Federation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90426-0" target="_blank">https://www.nature.com/articles/s41598-025-90426-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-23</strong> <a href="https://www.nature.com/articles/s41434-025-00521-0" target="_blank">
            ü§ñ MRNA Vaccine Helps Mice Fight Off Flu</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-134', this)">
            More</button></h4>
            <div id="toggle-134" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A capless hairpin-protected mRNA vaccine encoding the full-length Influenza A hemagglutinin protects mice against a lethal Influenza A infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists created a new type of mRNA that can help make vaccines more effective. This new mRNA doesn't need to have a special start point (like a cap) to work, which makes it more stable and lasts longer. When used in a vaccine, this mRNA helped protect mice from the flu virus by creating antibodies against the flu's outer protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The success of mRNA vaccines in controlling the COVID 19 pandemic has confirmed the efficacy of synthetically synthesized mRNA in humans and has also provided a blueprint on how to design them in terms of molecular structure and cost. We describe a mRNA vector that, unlike linear mRNAs used in current vaccines/therapeutics, does not require a 5‚Ä≤ cap to function. The described mRNA vector initiates translation from an internal ribosomal entry site (IRES) and contains specially designed self-folding secondary structures (hairpins) to protect the 5‚Ä≤ end against degradation, dramatically improving its stability. The produced mRNA did not require any additional modifications for functionality. The 5‚Ä≤ hairpins completely inhibited cap-dependent translation, and all vectors containing them required an IRES to express protein. When this capless mRNA vector was constructed to express the full-length Influenza A membrane protein hemagglutinin (HA), complexed with pre-formed lipid-based nanoparticles, and then injected into mice as a vaccine, it generated high titers of anti-HA antibodies and protected mice against a lethal dose of Influenza A.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41434-025-00521-0" target="_blank">https://www.nature.com/articles/s41434-025-00521-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-23</strong> <a href="https://www.nature.com/articles/s41541-025-01073-5" target="_blank">
            ü§ñ How vaccines for one disease protect against another</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-135', this)">
            More</button></h4>
            <div id="toggle-135" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Molecular convergence of neutralizing antibodies in human revealed by repeated rabies vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People need to get shots against rabies many times to build strong protection. Scientists studied how the body makes immune cells that can fight the disease after multiple vaccinations.

They found that getting booster shots really helps make these immune cells stronger and more effective at fighting the disease, especially at a specific part of the virus called site III. Most of these new immune cells have similar features and work better because of their shape and chemical properties.

The scientists also discovered which parts of the antibody molecules are important for making them very good at fighting the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Rabies vaccines require repeated immunization to robustly elicit neutralizing antibodies that prevent fatal diseases. Here, we analyzed rabies glycoprotein antibody repertoires at both polyclonal and monoclonal levels following repeated vaccination. Booster vaccination dramatically elevated the neutralizing activity of recalled antibodies, primarily targeting an immunodominant site III epitope with hydrophilic and rugged structures. Strikingly, the majority of site III-directed antibodies in the recall response used a convergent VHgene (IGHV3-30), and they exhibited more hydrophilic and shorter paratopes than non-site III antibodies, providing physicochemical advantages for binding to site III. Additionally, several amino acids on heavy chain CDR3 were identified as key sites for acquiring an ultrapotent neutralizing activity through site III binding. Our in-depth analysis of antibody repertoires revealed the molecular signatures of neutralizing antibodies generated by repeated rabies vaccination, possibly as a result of adaptive convergence.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01073-5" target="_blank">https://www.nature.com/articles/s41541-025-01073-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41598-025-89791-7" target="_blank">
            ü§ñ COVID-19 Pandemic Affects Ocular Diseases in China.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-136', this)">
            More</button></h4>
            <div id="toggle-136" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Changes in the spectrum of ocular disease during the COVID-19 pandemic in late 2022 in the Hubei Province</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the coronavirus disease-2019 pandemic in 2022, people in Hubei Province experienced different eye problems. Researchers looked at how these problems changed over time.

They asked people who had COVID-19 if they were feeling sick and found that many were experiencing eye strain, pain, and dryness. They also looked at medical records from ophthalmology clinics during the pandemic and found that:

* Many patients with COVID-19 symptoms visited the clinic.
* The number of people with certain eye diseases like corneal infections, glaucoma, and retinal problems increased.
* There were fewer young people and older adults visiting the clinic compared to before and after the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed to investigate changes in the ocular disease spectrum during the coronavirus disease-2019 (COVID-19) pandemic in late 2022 in the Hubei Province. This retrospective observational study was conducted in two parts. The first part involved collecting COVID-19-related information from residents of Hubei Province through an online questionnaire survey. The second part involved extracting electronic medical records from ophthalmology outpatient departments at two hospitals in Hubei Province during the pandemic and epidemic prevention and control periods, analyzing changes in the spectrum of ocular diseases. In the first part, 31.65% of patients with systemic symptoms of COVID-19 experienced ocular discomfort. The most common ocular symptoms were eye fatigue, ocular pain and dry eye. In the second part, 76.5% of patients who visited the ophthalmic clinic had COVID-19-related systemic symptoms during pandemic period. The proportion of patients with cornea/keratitis, glaucoma/acute angle-closure glaucoma (AACG) and vitreoretinal disease/retinal vein obstruction (RVO)/acute macular neuroretinalpathy (AMN) increased markedly during pandemic period. Additionally, the number of patients under 18 years and over 60 years decreased significantly compared to the same age groups pre- & post-pandemic. The COVID-19 pandemic has led to certain changes in the spectrum of ocular diseases, which warrants the attention of ophthalmologists.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89791-7" target="_blank">https://www.nature.com/articles/s41598-025-89791-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41598-025-87363-3" target="_blank">
            ü§ñ How COVID-19 Spreads in Cities Without Knowing People's Behavior</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-137', this)">
            More</button></h4>
            <div id="toggle-137" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Model predicted human mobility explains COVID-19 transmission in urban space without behavioral data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus spreads easily in crowded areas when people are close together. It's hard to understand how it spreads because there are too many different ways people move around cities. To solve this problem, scientists created a new way of modeling the spread of the virus that doesn't need super-precise information about how people move. This new model works well for 30 big cities in the US, India and Brazil.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus is primarily transmitted through in-person interactions, and so its growth in urban space is a complex function of human mobility behaviors that cannot be adequately explained by standard epidemiological models. Recent studies leveraged fine-grained urban mobility data to accurately model the viral spread, but such data pose privacy concerns and are often difficult to collect, especially in low- and middle-income countries (LMICs). Here, we show that the metapopulation epidemiological model incorporated with a simple gravity mobility model can be sufficient to capture most of the complex epidemic dynamics in urban space, largely reducing the need for empirical mobility data. Extensive experiments on 30 cities in the United States, India and Brazil show that our model consistently reproduces complex, distinctive COVID-19 growth curves with high accuracy. It also provides a theoretical explanation of the emergence of urban ‚Äúsuperspreading‚Äù, where a few high-risk neighborhoods account for most subsequent infections. Furthermore, with the aid of the proposed framework, we can inform mobility-aware travel restrictions to achieve a better balance between social cost and disease prevention, which facilitates sustainable epidemic control and supports the gradual transition to a post-pandemic world.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87363-3" target="_blank">https://www.nature.com/articles/s41598-025-87363-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41598-025-88904-6" target="_blank">
            ü§ñ Long COVID affects sex for men in Japan.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-138', this)">
            More</button></h4>
            <div id="toggle-138" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence of erectile dysfunction as long-COVID symptom in hospitalized Japanese patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that simplifies the original meaning:

After having COVID-19, some men in Japan developed a condition called erectile dysfunction. Researchers studied 609 Japanese men to see if they had ED and what factors might be connected to it. They found that men with ED reported feeling less hopeful about recovering from their illness and felt more tired and short of breath than those without ED. The study also showed that these men experienced higher levels of sadness, pain, and anxiety before and after getting COVID-19. It's believed that mental support can help men who are experiencing erectile dysfunction as a result of long-term effects of COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease-2019 (COVID-19) is associated with a wide range of post-acute sequelae. The prevalence of erectile dysfunction (ED) that developed after COVID-19 and the associated underlying factors were analyzed based on a questionnaire survey, COVID-19 Recovery Study II in Japan. A case‚Äìcontrol study was conducted with those with or without ED one and two years hospitalized with COVID-19 between March and September 2021. Six hundred and nine Japanese men, with a median age of 48¬†years, were analyzed. During the study period, 116 subjects (19.0%) had erectile dysfunction. The patients with ED responded with less subjective awareness of recovery and high breathless and fatigue scores compared to those without ED. The patients with ED also showed higher Hospital Anxiety and Depression Scale-D (depression) and the EuroQol 5-dimensions 5-level scores for pain/discomfort and anxiety/depression scores compared before COVID-19 infection. Sleep disturbance was suggested to be associated with erectile dysfunction using an exploratory clustering analysis in the one-year survey. There were no associations of COVID-19 severity, reinfection, vaccination frequency, antiviral treatment for COVID-19 with the presence of erectile dysfunction. It was considered that mental support for the subject with erectile dysfunction as a long-COVID symptom is warranted.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-88904-6" target="_blank">https://www.nature.com/articles/s41598-025-88904-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41598-025-91042-8" target="_blank">
            ü§ñ Impact of COVID-19 on Children's Immune Systems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-139', this)">
            More</button></h4>
            <div id="toggle-139" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of COVID-19 infection on Kawasaki disease and immune status in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who get COVID-19 can sometimes develop a condition called Kawasaki disease. This study looked at 161 kids who got Kawasaki disease during the pandemic and found that:

* Kids who got COVID-19 were more likely to not respond well to treatment
* Kids who got COVID-19 had lower levels of a type of immune system cell called CD4
* Some proteins in their immune system were higher than normal

This suggests that getting COVID-19 might make it harder for the body's immune system to fight off Kawasaki disease, and could lead to more severe symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients infected with SARS-CoV-2 may experience acute and long-term immune disorders. Immunological factors are thought to play an important role in Kawasaki disease. To analyze the impact of COVID-19 infection on Kawasaki disease, this study retrospectively analyzed 161 children with Kawasaki disease onset during the COVID-19 pandemic. The proportion of IVIG-Resistant individuals and the rate of corticosteroid use in the 1‚Äì7 weeks from COVID-19 infection to Kawasaki disease onset were higher than that of the noninfected group, even after excluding suspected cases of multiple system inflammatory syndrome. Compared to the noninfected group, the level of CD4 was lower, and the levels of CD3‚àíCD16+CD56+, complement C4, TNF-Œ±, and IFN-Œ≥ were higher in the 1‚Äì7 weeks after COVID-19 infection. In conclusion, the risk of IVIG resistance was significantly increased in children with Kawasaki disease onset 1‚Äì7 weeks after COVID-19 infection, which may be related to the long-term impact of COVID-19 on immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-91042-8" target="_blank">https://www.nature.com/articles/s41598-025-91042-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41409-025-02525-1" target="_blank">
            ü§ñ COVID-19 affects people who had CAR T-cell therapy for cancer</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-140', this)">
            More</button></h4>
            <div id="toggle-140" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical characteristics and outcomes of BCMA-targeted CAR-T cell recipients with COVID-19 during the Omicron wave: a retrospective study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Patients with a type of blood cancer called multiple myeloma are more likely to get very sick if they catch coronavirus. Even though some patients respond well to a special treatment that uses immune cells, they can still have problems and be at risk for getting very sick from the virus.

We looked at two groups: those who caught coronavirus and those who did not. We found out that older people are more likely to get very sick, but if they get better after the treatment, they're less likely to get sick again. Men also seem to be more protected than women.

The good news is that some patients can still protect themselves from getting sick by doing well on their treatment and not having certain complications with their disease at the time of infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients with relapsed or refractory multiple myeloma (R/R-MM) are more susceptible to develop severe coronavirus disease 2019 (COVID-19) for their immunocompromised states. Despite good responses to B-cell maturation antigen (BCMA)-targeted chimeric antigen receptor (CAR)-T cell therapy, deficiencies in humoral immunity following CAR-T cell infusions can still cause life-threatening complications in these patients. We conducted a comparative study to delineate the clinical characteristics and outcomes between recipients of BCMA-targeted CAR-T cell therapy who contracted COVID-19 vs. unaffected counterparts. Advanced age (odds ratio [OR]‚Äâ=‚Äâ1.367, 95% confidence interval [CI]‚Äâ=‚Äâ1.017‚Äì1.838,P=‚Äâ0.038) was a risk factor for developing severe COVID-19, while complete remission (CR) achieved by CAR-T cell therapy (OR‚Äâ=‚Äâ0.012, 95% CI‚Äâ=‚Äâ0.000‚Äì0.674,P=‚Äâ0.032) was protective. Male sex (hazard ratio [HR]‚Äâ=‚Äâ5.274, 95% CI‚Äâ=‚Äâ1.584‚Äì17.562,P=‚Äâ0.007) and CR achieved by CAR-T cell therapy (HR‚Äâ=‚Äâ3.107, 95% CI‚Äâ=‚Äâ1.025‚Äì9.418,P=‚Äâ0.045) were protective factors associated with COVID-19 duration. CR achieved by CAR-T cell therapy (HR‚Äâ=‚Äâ0.064, 95% CI‚Äâ=‚Äâ0.007‚Äì0.589,P=‚Äâ0.015) was also a protective factor for OS, while progression disease at the time of COVID-19 diagnosis (HR‚Äâ=‚Äâ14.206, 95% CI‚Äâ=‚Äâ1.555‚Äì129.819,P=‚Äâ0.019) was regarded as a risk factor. Thus, older patients with R/R-MM and those who do not achieve CR after CAR-T cell therapy should be most protected from COVID-19 infection by the Omicron variant.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41409-025-02525-1" target="_blank">https://www.nature.com/articles/s41409-025-02525-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-21</strong> <a href="https://www.nature.com/articles/s41598-025-89736-0" target="_blank">
            ü§ñ How do plant extracts fight COVID-19 virus?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-141', this)">
            More</button></h4>
            <div id="toggle-141" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effects of organic salts of virucidal and antiviral compounds fromNelumbo nuciferaandKaempferia parvifloraagainst SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

This research looks at plants that might have medicine to fight viruses like COVID-19. We took extracts from the seeds of lotus flowers and Thai ginseng, and found out what chemicals were inside them. The chemicals we found can kill or stop certain viruses. Two chemicals, neferine and nuciferine, were especially good at fighting COVID-19. We also tested some other compounds from these plants and found that they had fewer side effects when used together with the first two chemicals.

We took a closer look at one of the plants and found nine special helpers called flavonoids. Only one of them was very good at killing or stopping the virus, but it didn't work as well when we tested it in bigger amounts.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The present work investigates virucidal and antiviral compounds in the extracts of seed embryos of a lotus,Nelumbo nucifera, and a Thai ginseng,Kaempferia parviflora. Separation of the extracts led to the identification of antiviral compounds against SARS-CoV-2. Neferine (1) and nuciferine (3) fromN. nucifera, as well as their respective HCl salts (2and4), exhibited virucidal and antiviral activities against SARS-CoV-2. Virucidal activity of neferine salt (2) (EC504.78 ¬µM) was 7.5 times better than its free-base, neferine (1) (EC5036.01 ¬µM), and the salt (2) also improved the selectivity index (SI), showing less cytotoxicity than1. This work demonstrates that organic salts have an impact on biological activities. A crude extract ofK. parviflorarhizomes displayed virucidal activity (EC5042.11¬†¬µg/mL) and antiviral activity (EC5039.28¬†¬µg/mL). Isolation of a crude extract ofK. parviflorarhizomes led to the identification of nine flavonoids (5‚Äì13). Among these flavonoids, only 5,7,4‚Äô-trimethoxyflavone (8) was found to show virucidal (EC50437.90 ¬µM) and antiviral (EC5050.97 ¬µM) activities against SARS-CoV-2. However, flavonoids (5‚Äì13) did not inhibit SARS-CoV-2 3CLproenzyme at the concentrations of 10 ¬µM and 100 ¬µM. In conclusion, our data underscores the therapeutic potential ofN. nuciferaandK. parvifloraderived bioactive compounds against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89736-0" target="_blank">https://www.nature.com/articles/s41598-025-89736-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-19</strong> <a href="https://www.nature.com/articles/s41598-025-90153-6" target="_blank">
            ü§ñ Effects of COVID-19 on Chronic Kidney Disease Patients in New York City.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-142', this)">
            More</button></h4>
            <div id="toggle-142" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long term outcomes of patients with chronic kidney disease after COVID-19 in an urban population in the Bronx</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We looked at how COVID-19 affected people with chronic kidney disease (CKD) over time. We found that patients who got hospitalized for COVID-19 had a higher chance of getting their kidneys even worse and having more heart problems than those who didn't get sick or were only mildly ill. This means that people with CKD are at risk if they get COVID-19, especially if they need to go to the hospital.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We investigated the long-term kidney and cardiovascular outcomes of patients with chronic kidney disease (CKD) after COVID-19. Our retrospective cohort consisted of 834 CKD patients with COVID-19 and 6,167 CKD patients without COVID-19 between 3/11/2020 to 7/1/2023. Multivariate competing risk regression models were used to estimate risk (as adjusted hazard ratios (aHR) with 95% confidence intervals (CI)) of CKD progression to a more advanced stage (Stage 4 or 5) and major adverse kidney events (MAKE), and risk of major adverse cardiovascular events (MACE) at 6-, 12-, and 24-month follow up. Hospitalized COVID-19 patients at 12 and 24¬†months (aHR 1.62 95% CI[1.24,2.13] and 1.76 [1.30, 2.40], respectively), but not non-hospitalized COVID-19 patients, were at higher risk of CKD progression compared to those without COVID-19. Both hospitalized and non-hospitalized COVID-19 patients were at higher risk of MAKE at 6-, 12- and 24-months compared to those without COVID-19. Hospitalized COVID-19 patients at 6-, 12- and 24-months (aHR 1.73 [1.21, 2.50], 1.77 [1.34, 2.33], and 1.31 [1.05, 1.64], respectively), but not non-hospitalized COVID-19 patients, were at higher risk of MACE compared to those without COVID-19. COVID-19 increases the risk of long-term CKD progression and cardiovascular events in patients with CKD. These findings highlight the need for close follow up care and therapies that slow CKD progression in this high-risk subgroup.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90153-6" target="_blank">https://www.nature.com/articles/s41598-025-90153-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-18</strong> <a href="https://www.nature.com/articles/s41392-025-02156-4" target="_blank">
            ü§ñ How SARS-CoV-2 Affects Fertilized Egg Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-143', this)">
            More</button></h4>
            <div id="toggle-143" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 compromises blastocyst quality by modifying the ovarian microenvironment</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A group of people are talking about how to make their community a better place for everyone, especially children and young adults. They share ideas on how to improve the environment, education, and opportunities for growth.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Dear Editor,</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-025-02156-4" target="_blank">https://www.nature.com/articles/s41392-025-02156-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-18</strong> <a href="https://www.nature.com/articles/s41467-025-56800-2" target="_blank">
            ü§ñ Monkeypox Vaccine Tests in Mice: Early Protection Linked to Mild Symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-144', this)">
            More</button></h4>
            <div id="toggle-144" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Strong and early monkeypox virus-specific immunity associated with mild disease after intradermal clade-IIb-infection in CAST/EiJ-mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Monkeypox is a virus that lives in Africa. It can spread to other parts of the world and get worse over time because it's changing. Scientists want to understand how this happens so they can make new treatments and preventions.

They've found an animal model, mice, where they can test how the monkeypox virus behaves. They used these mice to see how different types of the virus affect them. The results showed that one type of the virus makes the mice get sick less quickly than another type, which is good news because it means scientists might be able to make treatments for this type more easily.

This study helps researchers develop new ways to stop monkeypox from spreading and getting worse.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox virus (MPXV) is a zoonotic poxvirus long endemic in West and Central Africa. Outbreaks, first the global spread of clade II outside Africa in 2022, and since 2023 the accelerating spread of clade I in central Africa, point to MPXV adaptations that pose the risk of it becoming more transmissible in humans. Animal models mimicking the clinical disease outcome in humans are important to better understand pathogenesis, host tropism, and the contribution of genetic mutations. Here, we demonstrate that MPXV infection via tail scarification in CAST/EiJ mice is an appropriate animal model to mimic human mpox. In our study, disease outcome is milder in clade IIb than clade IIa-infected mice, which is associated with enhanced immunogenicity early during infection. This suggests that clade IIb more efficiently activates host immune responses, highlighting how this animal model could facilitate studying new MPXV variants to help develop efficient antivirals and preventive measures.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56800-2" target="_blank">https://www.nature.com/articles/s41467-025-56800-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-18</strong> <a href="https://www.nature.com/articles/s41598-025-89639-0" target="_blank">
            ü§ñ Web tool to identify viruses without labels or seeing damage they cause.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-145', this)">
            More</button></h4>
            <div id="toggle-145" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A web-based artificial intelligence system for label-free virus classification and detection of cytopathic effects</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists created a special computer program called AIRVIC to help identify viruses that make cells sick. The program looks at pictures of infected cells taken with microscopes and can tell if they're infected or not. It can even figure out which virus is causing the infection.

The program was tested on different types of viruses and worked very well, except for one type of virus in one type of cell line. But overall, it's a big improvement over traditional methods that require lots of time and effort.

The best part is that AIRVIC is easy to use online, so researchers all around the world can use it to help diagnose viral infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Identifying viral replication within cells demands labor-intensive isolation methods, requiring specialized personnel and additional confirmatory tests. To facilitate this process, we developed an AI-powered automated system called AI Recognition of Viral CPE (AIRVIC), specifically designed to detect and classify label-free cytopathic effects (CPEs) induced by SARS-CoV-2, BAdV-1, BPIV3, BoAHV-1, and two strains of BoGHV-4 in Vero and MDBK cell lines. AIRVIC utilizes convolutional neural networks, with ResNet50 as the primary architecture, trained on 40,369 microscopy images at various magnifications. AIRVIC demonstrated strong CPE detection, achieving 100% accuracy for the BoGHV-4 DN-599 strain in MDBK cells, the highest among tested strains. In contrast, the BoGHV-4 MOVAR 33/63 strain in Vero cells showed a lower accuracy of 87.99%, the lowest among all models tested. For virus classification, a multi-class accuracy of 87.61% was achieved for bovine viruses in MDBK cells; however, it dropped to 63.44% when the virus was identified without specifying the cell line. To the best of our knowledge, this is the first research article published in English to utilize AI for distinguishing animal virus infections in cell culture. AIRVIC‚Äôs hierarchical structure highlights its adaptability to virological diagnostics, providing unbiased infectivity scoring and facilitating viral isolation and antiviral efficacy testing. Additionally, AIRVIC is accessible as a web-based platform, allowing global researchers to leverage its capabilities in viral diagnostics and beyond.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89639-0" target="_blank">https://www.nature.com/articles/s41598-025-89639-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-17</strong> <a href="https://www.nature.com/articles/s41467-025-57094-0" target="_blank">
            ü§ñ How do a specific set of genes affect our ability to get sick from certain viruses?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-146', this)">
            More</button></h4>
            <div id="toggle-146" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The CXCL8/MAPK/hnRNP-K axis enables susceptibility to infection by EV-D68, rhinovirus, and influenza virus in vitro</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Some viruses can make us very sick by triggering an overactive immune response in our body. Scientists studied how one specific virus, EV-D68, interacts with our body and found that it uses a pathway involving a protein called CXCL8 to multiply. When we understand this interaction better, we might be able to develop new treatments for viral infections like the flu and common cold.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory viruses pose an ongoing threat to human health with excessive cytokine secretion contributing to severe illness and mortality. However, the relationship between cytokine secretion and viral infection remains poorly understood. Here we elucidate the role ofCXCL8as an early response gene to EV-D68 infection. Silencing CXCL8 or its receptors, CXCR1/2, impedes EV-D68 replication in vitro. Upon recognition of CXCL8 by CXCR1/2, the MAPK pathway is activated, facilitating the translocation of nuclear hnRNP-K to the cytoplasm. This translocation increases the recognition of viral RNA by hnRNP-K in the cytoplasm, promoting the function of the 5‚Ä≤ untranslated region in the viral genome. Moreover, our investigations also reveal the importance of the CXCL8 signaling pathway in the replication of both influenza virus and rhinovirus. In summary, our findings hint that these viruses exploit the CXCL8/MAPK/hnRNP-K axis to enhance viral replication in respiratory cells in vitro.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-57094-0" target="_blank">https://www.nature.com/articles/s41467-025-57094-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-17</strong> <a href="https://www.nature.com/articles/s41598-025-90081-5" target="_blank">
            ü§ñ Effect of COVID-19 on people with severe drinking problems being treated with a specific medicine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-147', this)">
            More</button></h4>
            <div id="toggle-147" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of Covid-19 pandemic on trajectories of patients with severe alcohol use disorder treated with disulfiram</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young people can be affected by the COVID-19 pandemic in different ways, including those who struggle with alcohol problems. The pandemic has changed many things, but it's also made some issues worse. 

We looked at 45 patients who were supposed to attend a special program for treating their addiction while following certain rules (like taking a pill called disulfiram). We wanted to see how the pandemic affected them.

The data shows that during the pandemic, more people had to cancel their appointments, but after it ended, things started going back to normal. This makes us think that some people with severe drinking problems need new and better ways to get help, maybe even from a computer or phone instead of in person.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The manifestations and progression of alcohol use disorder (AUD) are influenced by a number of contextual factors, with the current coronavirus pandemic being a significant example. This pandemic has profoundly impacted nearly all aspects of human life and has, therefore, strongly influenced patients suffering from AUD. In some cases, the pandemic has led to a reduction in severity, while in others, it has had the opposite effect. In our own work we have been investigating the negative impact of the pandemic on 45 patients with AUD who were undergoing outpatient treatment, including supervised use of disulfiram (Antabuse), in a close-knit program. A linear trend analysis demonstrated significant alterations in the retention rate over a 3-year period, encompassing the pre-pandemic, pandemic, and post-pandemic periods. During the pandemic the number of treatment cancellations virtually increased. Following the pandemic, a tendency towards the normalization of patient numbers was observed. Our data indicate a high level of vulnerability among patients with severe AUD and highlight a need for the development of alternative, possibly telemedical, treatment methods.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-90081-5" target="_blank">https://www.nature.com/articles/s41598-025-90081-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-15</strong> <a href="https://www.nature.com/articles/s41598-024-84843-w" target="_blank">
            ü§ñ The effects of depression in teenagers during a worldwide pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-148', this)">
            More</button></h4>
            <div id="toggle-148" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The Burden of adolescent depression and the impact of COVID-19 across 204 countries and regions from 1990 to 2021: results from the 2021 global burden of disease study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

This study looked at how much depression affects people aged 10 to 24 around the world from 1990 to 2021. We also wanted to see how the coronavirus pandemic affected this age group and if there are bigger disparities in mental health depending on where you live.

We found that even though some things about depression haven't changed over time, like its impact on younger teens, more serious problems happened as people got older. Girls tend to get it worse than boys. And before the pandemic, richer countries had better mental health outcomes.

The pandemic made depression way worse and widened gaps in mental health between rich and poor areas of the world.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aims to analyze the trends in the burden of depression among adolescents aged 10 to 24 years globally from 1990 to 2021, with a focus on the impact of COVID-19 on adolescent depression and health inequalities. Using data from the 2021 Global Burden of Disease Study, we examined age-standardized prevalence, incidence, and disability-adjusted life years (DALYs) for depression among adolescents aged 10‚Äì24 years. Estimated annual percentage change (EAPC) was used to assess temporal trends. Age-period-cohort (APC) analysis estimated age, period, and cohort effects. Bayesian APC (BAPC) analysis projected future trends. Decomposition analysis further explored drivers of changes in depression burden. Slope Index of Inequality (SII) and Concentration Index (CI) were calculated to assess health inequalities across regions and countries. From 1990 to 2021, the global incidence, prevalence and DALY rates of adolescent depression remained stable. Depression incidence and prevalence increased with age, with the 20‚Äì24 age group showing the highest rates. The burden of depression was higher in females than in males. The COVID-19 pandemic significantly impacted adolescent depression, with reported prevalence, incidence, and DALY rates in 2020 and 2021 far exceeding predicted values, and the burden of depression is expected to continue rising. Health inequalities between adolescents in high- and low-income regions have widened, particularly following the pandemic. The COVID-19 pandemic significantly exacerbated the burden of depression and intensified health inequalities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84843-w" target="_blank">https://www.nature.com/articles/s41598-024-84843-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-15</strong> <a href="https://www.nature.com/articles/s41598-025-89717-3" target="_blank">
            ü§ñ New Title: 
Aminiminocycline helps fight coronavirus.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-149', this)">
            More</button></h4>
            <div id="toggle-149" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>9-aminominocycline potentiates the efficacy of EIDD-1931 and PF-332 by targeting the papain like protease enzyme of SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an explanation of the abstract in simpler terms:

The researchers found that there are three important enzymes in the virus that makes COVID-19: 3CLpro, PLpro, and RdRp. Right now, some medicines target these viruses to help people get better. But none of those medicines work on the PLpro enzyme, which is needed for the virus to make more copies of itself and stop the body's immune system from fighting it.

The scientists discovered a new medicine called 9-aminominocycline (9-AMN) that can inhibit this PLpro enzyme. It works by stopping the enzyme from working properly and can even work on different versions of the virus like Delta and Omicron. When used together with other medicines, 9-AMN can be much more effective at stopping the virus.

This new medicine is promising for helping people treat COVID-19, especially those infected with the most difficult-to-treat variants of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The 3-chymotrypsin-like protease (3CLpro), papain-like protease (PLpro), and RNA-dependent RNA polymerase (RdRp) are key enzymes in SARS-CoV-2 replication and serve as critical targets for an antiviral drug. Currently, Paxlovid¬Æ and Lagevrio‚Ñ¢ specifically target 3CLpro and RdRp, respectively, for COVID-19 treatment. However, no antivirals target for the SARS-CoV-2 PLpro enzyme, essential for viral replication and suppression of the host antiviral immune response. This study identified 9-aminominocycline (9-AMN) as a potent inhibitor of SARS-CoV-2 PLpro. Unlike the parent compound minocycline, 9-AMN inhibits PLpro‚Äôs proteolytic and deubiquitinase activities by approximately 90%, with IC50values of 4.15¬†¬µM and 4.55¬†¬µM, respectively, while showing no effect on the enzymatic activity of 3CLpro or RdRp. Enzyme kinetics reveal that 9-AMN functions as a mixed PLpro inhibitor and binds to its active site, disrupting its function as predicted by computer modeling. Furthermore, 9-AMN demonstrates, efficacy against the Delta and Omicron variants, with EC50values of 1.04¬†¬µM and 2.35¬†¬µM, respectively. When combined with EIDD-1931 (an active form of molnupiravir) or nirmatrelvir (PF-332), 9-AMN exhibits synergistic effects, significantly reducing the doses required to inhibit the Omicron variant. In conclusion, 9-AMN inhibits SARS-CoV-2 replication, and PLpro activity, highlighting its potential as a promising candidate for COVID-19 treatment strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89717-3" target="_blank">https://www.nature.com/articles/s41598-025-89717-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-14</strong> <a href="https://www.nature.com/articles/s41598-025-89940-y" target="_blank">
            ü§ñ Shift in how people's immune systems change after a new coronavirus mutation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-150', this)">
            More</button></h4>
            <div id="toggle-150" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Population shift in antibody immunity following the emergence of a SARS-CoV-2 variant of concern</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

New and worse versions of the coronavirus started to spread in South Africa. The government did a survey to see who had antibodies against the virus, which is like having superpowers to fight it off. They looked at people's blood samples from 2020 and 2021.

They found that some people had special powers that only worked against an old version of the virus (D614G). But as time went on, more and more people started to have powers that worked against a new and worse version (Beta).

The good news is that many people's powers got stronger over time. The bad news is that this means the new and worse version of the virus is becoming more common. This could help us understand why some people are getting sick again, and how we can fix it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) variants of concern (VOCs) exhibit escape from pre-existing immunity and elicit variant-specific immune responses. In South Africa, the second wave of SARS-CoV-2 infections was driven by the Beta VOC, which coincided with the country-wide National COVID-19 Antibody Survey (NCAS). The NCAS was conducted between November 2020 and February 2021 to understand the burden of SARS-CoV-2 infection through seroprevalence. We evaluated 649 NCAS sera for spike binding and pseudovirus neutralizing antibodies. We classified individuals as ancestral or D614G neutralizers (114/649), Beta neutralizers (96/649), double neutralizers (375/649) or non-neutralizers (62/649). We observed a consistent decrease in preferential neutralization against the D614G variant from 68 to 18% of individuals over the four sampling months. Concurrently, samples with equivalent neutralization of both variants, or with enhanced neutralization of the Beta variant, increased from 32 to 82% of samples. Neutralization data showed that geometric mean titers (GMTs) against D614G dropped 2.4-fold, while GMTs against Beta increased 2-fold during this same period. A shift in population humoral immunity in favor of Beta-directed or cross-neutralizing antibody responses, paralleled the increase in genomic frequency of the Beta variant in South Africa. Understanding similar population immunity shifts could elucidate immunity gaps that drive SARS-CoV-2 evolution.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89940-y" target="_blank">https://www.nature.com/articles/s41598-025-89940-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-13</strong> <a href="https://www.nature.com/articles/s41541-025-01085-1" target="_blank">
            ü§ñ Combined vaccine gives good protection but also causes more lung problems in a special type of mouse.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-151', this)">
            More</button></h4>
            <div id="toggle-151" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Combined immunization with SARS-CoV-2 spike and SARS-CoV nucleocapsid protects K18-hACE2 mice but increases lung pathology</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists are working on new vaccines against COVID-19. The original vaccines focus on a protein called spike that the virus uses to infect people. However, some new versions of the virus have changed this protein, making it harder for the vaccine to work. Researchers found a different part of the virus, called nucleocapsid, that is similar across all versions. They created new vaccines using this part and combined it with the spike protein. These new vaccines showed promise in keeping people safe from COVID-19, but they also caused more immune response than expected. This means the vaccine may not be ideal for everyone, especially those who are already sick.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccines against SARS-CoV-2 have targeted the spike protein and have been successful at preventing disease. However, with the emergence of variants, spike-specific vaccines become less effective. The nucleocapsid protein is relatively conserved among variants of SARS-CoV-2 and is a candidate for addition to spike in next generation vaccines for the induction of T cell protection. Previous studies on SARS-CoV have suggested that the induction of an immune response to nucleocapsid could result in enhanced disease. Using the K18-hACE2 mouse model we investigated immunization with a variant nucleocapsid, from SARS CoV (N1) alone or in combination with spike from SARS-CoV-2 and compared this to nucleocapsid from SARS-CoV-2 (N2). The spike-nucleocapsid-based vaccines conferred protection against SARS-CoV-2 in lungs and brain and decreased lung pathology compared to control mice. However, higher T and B cell immune responses were observed in N1-immunized mice prior to challenge, whether delivered alone or with spike, and immunization with N1 resulted in increased lung pathology compared to immunization with spike or N2. These findings suggest that spike-nucleocapsid-based vaccines are safe and effective, even with variant nucleocapsid sequences, but that viral control in this mouse model may be associated with higher lung pathology, compared to spike immunization alone, due to the immunogenic qualities of the nucleocapsid antigen.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01085-1" target="_blank">https://www.nature.com/articles/s41541-025-01085-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-13</strong> <a href="https://www.nature.com/articles/s41467-025-56857-z" target="_blank">
            ü§ñ Systemic Lupus Erythematosus Caused by Genetic Mutation and Viral Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-152', this)">
            More</button></h4>
            <div id="toggle-152" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The systemic lupus erythematosus-associated NCF190Hallele synergizes with viral infection to cause mouse lupus but also limits virus spread</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists want to understand how some people get autoimmune diseases like lupus. They found a special genetic mutation called NCF190H that makes people more likely to develop lupus. When mice with this mutation got sick with a virus called norovirus, they developed symptoms similar to humans with lupus. The study shows that the virus and the genetic mutation together made the mice's immune system go haywire, causing them to produce antibodies and get inflammation in their kidneys and joints. This means that the NCF190H mutation and norovirus infection work together to create a model of human lupus in mice.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Studying how single nucleotide polymorphisms (SNPs) crosstalk with non-autologous factors to cause complex autoimmune diseases is challenging. An amino acid replacement in the neutrophil cytosolic factor 1 (NCF1-339/NCF1R90H) leading to lower reactive oxygen species induction has been reported as the major SNP for systemic lupus erythematosus (SLE). Here we show that infection with the murine norovirus (MNV) contributes to the induction of lupus inNcf190Hmice. Mutant NCF190Hupregulates the IFN-Œ±/JAK1/STAT1 pathway in macrophages and anti-MNV-antibody production. In parallel, the MNV infection of NCF190Hmice upregulates Toll-like receptor 7 in macrophages, plasmacytoid dendritic cells and B220+splenocytes, thereby promoting germinal center formation and lupus-associated autoantibodies production. These compounded effects lead to protection against MNV infection but also glomeruloneph	ritis with proteinuria and lupus arthritis in the absence of chemical inducers such as pristane. Our data thus suggest that this SLE-associated SNP, NCF190H, synergizes with MNV infection to induce the development of mouse lupus.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56857-z" target="_blank">https://www.nature.com/articles/s41467-025-56857-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-13</strong> <a href="https://www.nature.com/articles/s41598-025-89930-0" target="_blank">
            ü§ñ How an mRNA Covid Vaccine Can Help Fight Melanoma</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-153', this)">
            More</button></h4>
            <div id="toggle-153" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intratumoral administration of mRNA COVID-19 vaccine delays melanoma growth in mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Cancer treatment is not always effective because the immune system is not strong enough to fight it off. In some cases, the cancer makes the immune system weaker, making it harder for the body to fight back. Vaccines can help boost the immune system. Researchers tested a special vaccine made from COVID-19 mRNA in mice with melanoma (a type of skin cancer). They found that this vaccine helped stop the tumor from growing and even gave the mice more time to live. The vaccine also helped increase the number of good cells called T-cells, which fight cancer. When used with other medicines that help the immune system, the vaccine worked even better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Immunotherapies are effective for cancer treatment but are limited in ‚Äòcold‚Äô tumor microenvironments due to a lack of infiltrating CD8+T cells, key players in the anti-cancer immune response. The onset of the COVID-19 pandemic sparked the widespread use of mRNA-formulated vaccines and is well documented that vaccination induces a Th1-skewed immune response. Here, we evaluated the effects of an intratumoral injection of the mRNA COVID-19 vaccine in subcutaneous melanoma tumor mouse models. Tumor growth and survival studies following a single intratumoral injection of the COVID-19 vaccine showed significant tumor suppression and prolonged survival in established B16F10 subcutaneous tumor-bearing mice. mRNA vaccine treatment resulted in a significant increase in CD8+T cell infiltration into the tumor microenvironment, as observed using intravital imaging and flow cytometry. Further tumor growth suppression was achieved using additional mRNA vaccine treatments. Combination administration of mRNA vaccine with immune checkpoint therapies demonstrated enhanced effects, further delaying tumor growth and improving the survival time of tumor-bearing mice. This study demonstrates that mRNA vaccines may be used as adjuvants for immunotherapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89930-0" target="_blank">https://www.nature.com/articles/s41598-025-89930-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-13</strong> <a href="https://www.nature.com/articles/s41467-025-56902-x" target="_blank">
            ü§ñ Antiviral Drug Candidates Based on a Specific Molecule</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-154', this)">
            More</button></h4>
            <div id="toggle-154" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Design of quinoline SARS-CoV-2 papain-like protease inhibitors as oral antiviral drug candidates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler language:

New versions of the coronavirus are constantly changing, making it harder to find effective treatments. Scientists have created a new type of medicine that can stop the virus from replicating. They designed a special medicine that targets a part of the virus called papain-like protease (PLpro). The new medicine is based on a type of molecule called quinoline and has shown promising results in laboratory tests. In animal studies, the medicine was able to improve survival rates, reduce weight loss, and prevent damage to lung tissue. This gives scientists hope that these medicines could be used to treat coronavirus infections in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ever-evolving SARS-CoV-2 variants necessitate the development of additional oral antivirals. This study presents the systematic design of quinoline-containing SARS-CoV-2 papain-like protease (PLpro) inhibitors as potential oral antiviral drug candidates. By leveraging the recently discovered Val70Ubbinding site in PLpro, we designed a series of quinoline analogs demonstrating potent PLproinhibition and antiviral activity. Notably, the X-ray crystal structures of 6 lead compounds reveal that the 2-aryl substitution can occupy either the Val70Ubsite as expected or the BL2 groove in a flipped orientation. The in vivo leadJun13296exhibits favorable pharmacokinetic properties and potent inhibition against SARS-CoV-2 variants and nirmatrelvir-resistant mutants. In a mouse model of SARS-CoV-2 infection, oral treatment withJun13296significantly improves survival, reduces body weight loss and lung viral titers, and prevents lung tissue damage. These results underscore the potential of quinoline PLproinhibitors as promising oral SARS-CoV-2 antiviral candidates, instilling hope for the future of SARS-CoV-2 treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56902-x" target="_blank">https://www.nature.com/articles/s41467-025-56902-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-13</strong> <a href="https://www.nature.com/articles/s41467-025-56776-z" target="_blank">
            ü§ñ Common Respiratory Viruses Come Back After Covid-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-155', this)">
            More</button></h4>
            <div id="toggle-155" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterising the asynchronous resurgence of common respiratory viruses following the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic caused many common respiratory viruses to stop spreading for a while. When restrictions were lifted, these viruses started coming back again in different orders. We looked at data from all around the world and found that each virus came back at its own specific time.

Here's what we found:

1. The first virus to come back was rhinovirus.
2. Then came seasonal coronavirus, followed by other viruses like parainfluenza, respiratory syncytial virus, adenovirus, metapneumovirus, and influenza A.
3. Influenza B virus was the last to come back.

Even when restrictions were lifted again, the same pattern happened, but one virus came back earlier than expected - influenza A.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic and relevant non-pharmaceutical interventions (NPIs) interrupted the circulation of common respiratory viruses. These viruses demonstrated an unprecedented asynchronous resurgence as NPIs were relaxed. We compiled a global dataset from a systematic review, online surveillance reports and unpublished data from Respiratory Virus Global Epidemiology Network, encompassing 92 sites. We compared the resurgence timings of respiratory viruses within each site and synthesised differences in timings across sites, using a generalised linear mixed-effects model. We revealed a distinct sequential timing in the first post-pandemic resurgence: rhinovirus resurged the earliest, followed by seasonal coronavirus, parainfluenza virus, respiratory syncytial virus, adenovirus, metapneumovirus and influenza A virus, with influenza B virus exhibiting the latest resurgence. Similar sequential timing was observed in the second resurgence except influenza A virus caught up with metapneumovirus. The consistent asynchrony across geographical regions suggests that virus-specific characteristics, rather than location-specific factors, determining the relative timing of resurgence.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56776-z" target="_blank">https://www.nature.com/articles/s41467-025-56776-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-12</strong> <a href="https://www.nature.com/articles/s41598-024-59954-z" target="_blank">
            ü§ñ Vaccination Impact on Long COVID in Mexico</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-156', this)">
            More</button></h4>
            <div id="toggle-156" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Association between Mexican vaccination schemes and the duration of long COVID syndrome symptoms</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler language:

The COVID-19 pandemic had a big impact on the world, causing many people to get seriously sick and leading to long-lasting health problems known as Long COVID. We looked at how common Long COVID is in a specific area of Mexico (northeastern Mexico) and whether it's related to things like age, sex, vaccinations, or other health issues.

We asked 804 patients about their medical history, vaccinations, symptoms they experienced, and more. About 20% of the people we studied had Long COVID. We found that vaccinated people were less likely to have Long COVID than unvaccinated people, and men were also protected against Long COVID.

However, having a high body mass index (BMI) made it more likely for people to develop Long COVID. Even three years after the pandemic, many people still had Long COVID symptoms, and there's not enough information about how to treat or prevent this condition.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic had a profound global impact, characterized by a high fatality rate and the emergence of enduring consequences known as Long COVID. Our study sought to gauge the prevalence of Long COVID syndrome in northeastern Mexico, correlating it with patients' comorbidities and vaccination records. We carried out an observational cross-sectional approach, by administering a comprehensive questionnaire covering patients‚Äô medical history, demographics, vaccination status, COVID-related symptoms, their duration, and any treatment received. Our participant cohort included 804 patients, averaging 41.5 (SD 13.6) years in age, with 59.3% being women. Notably, 168 individuals (20.9%) met Long COVID criteria. Our analysis of COVID-19 long lasting compared vaccination schemes, unveiling a significant difference between vaccinated and unvaccinated groups (p=‚Äâ‚Äâ<‚Äâ0.001). Through linear regression model, we found male gender (Œ≤‚Äâ=‚Äâ‚àí‚Äâ0.588,p<‚Äâ0.001) and vaccination status (Œ≤‚Äâ=‚Äâ0.221,p=‚Äâ0.015) acted as protective factors against Long COVID symptom duration, while higher BMI was a risk factor (Œ≤‚Äâ=‚Äâ‚àí‚Äâ0.131,p=‚Äâ0.026). We saw that the duration of Long COVID was different within vaccinated patients, and we did not find any association of comorbidities with an increase in the presence of symptoms. Even three years after the pandemic, a significant prevalence of Long COVID persists, and there is still a lack of standardized information and any possible treatment regarding this condition.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-59954-z" target="_blank">https://www.nature.com/articles/s41598-024-59954-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-12</strong> <a href="https://www.nature.com/articles/s41598-025-89707-5" target="_blank">
            ü§ñ COVID-19's impact on healthcare in Almaty, Kazakhstan from 2021-2022.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-157', this)">
            More</button></h4>
            <div id="toggle-157" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The burden of COVID-19 in primary care of Almaty, Kazakhstan, 2021‚Äì2022</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

People in primary care helped stop and diagnose COVID-19 by providing good healthcare at home. A big study looked at over 174,000 people who had COVID-19 in Kazakhstan from 2021 to 2022. They found that most were between 30 and 39 years old and didn't have a severe case of the disease. Women were more likely to get it than men, especially older women. Pregnant women had a low risk of getting very sick. People with other health problems, like diabetes or lung disease, were more likely to get a bad case of COVID-19. The study shows how important primary care is in helping people recover from the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary healthcare played a crucial role during the COVID-19 pandemic by preventing, timely diagnosing, and referring severe cases to hospitals, as well as monitoring and counseling patients via telemedicine. We used a cross-sectional approach to analyze the severity outcomes of 174,540 COVID-19 cases treated in primary care in Almaty between 2021 and 2022, by age, sex, disease severity, and comorbidities. Outpatients with COVID-19 were mainly aged 30‚Äì39 (20.3%) with a mild course (88.9%). Among adults, females predominated (‚â•‚Äâ60‚Äì25.5% vs. 19.2%, <‚Äâ0.001), and among children (0‚Äì17), boys ‚àí‚Äâ21.2% vs. 12.1% (p<‚Äâ0.001). A higher risk for moderate to severe COVID-19 and adverse outcomes was assessed among older adults, particularly those aged 60 and older compared with younger groups (OR‚Äâ=‚Äâ9.01, 95% CI: 7.72‚Äì10.51). Pregnant women had a low risk of severe disease (OR‚Äâ=‚Äâ0.5, 95% CI: 0.38‚Äì0.65). Patients with concomitant disease were at higher likelihood of severe COVID-19 (p<‚Äâ0.001, OR‚Äâ=‚Äâ2.51, 95% CI: 1.9‚Äì3.15 for obesity,p<‚Äâ0.001, OR‚Äâ=‚Äâ1.43, 95% CI: 1.27‚Äì1.6 for diabetes mellitus, OR‚Äâ=‚Äâ1.16, 95% CI: 1.07‚Äì1.26 for arterial hypertension, andp<‚Äâ0.001, OR‚Äâ=‚Äâ2.5, 95% CI: 2.13‚Äì3.02 for chronic obstructive pulmonary disease). The study emphasizes an often-overlooked impact of COVID-19 on primary care, which is essential for improving outpatient care.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89707-5" target="_blank">https://www.nature.com/articles/s41598-025-89707-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-11</strong> <a href="https://www.nature.com/articles/s41598-025-89753-z" target="_blank">
            ü§ñ Health effects of high radon in homes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-158', this)">
            More</button></h4>
            <div id="toggle-158" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Serum biomarkers associated with health impacts of high residential radon exposure: a metabolomic pilot study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People living in places with lots of natural radiation, like high levels of radon, might get sick more easily or have diseases later in life. But we don't know as much about how this affects people's bodies. This study looked at what happens inside the body when people are exposed to high levels of radon for a long time.

They took blood samples from 30 healthy people who lived in areas with different amounts of radon and compared their bodies' chemicals to see if there were any differences. They found that some chemicals in the blood could be used to predict which groups were more likely to get certain diseases. These chemicals might be useful for doctors to find out if someone is at risk of getting sick from radon exposure.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-term epidemiological evidence suggests that populations exposed to high natural radiation levels for extended periods may have an increased risk of cancer and other diseases. However, research on health effects in high-radon areas, particularly regarding disease-related biomarkers, remains limited. This study aimed to investigate serum metabolic biomarkers associated with diseases in individuals from areas with high radon exposure. Metabolic profiling was performed using ultra-high-performance liquid chromatography coupled with high-resolution mass spectrometry on 30 healthy participants comprising 15 individuals from a low-residential radon exposure group and 15 from a high-residential radon exposure group. Multivariate analysis, receiver operating characteristic (ROC) curve analysis, and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis were applied. Partial least-squares discriminant analysis revealed significant differences (P< 0.05) between the two groups, identifying 92 metabolites. ROC analysis (AUC‚Äâ‚â•‚Äâ0.85) highlighted 12 key candidates associated with high radon exposure. KEGG pathway analysis linked D-sphingosine to lung cancer development and 3-methylhistidine to kidney disease, early preeclampsia, and Alzheimer‚Äôs disease. These findings suggest that D-sphingosine and 3-methylhistidine are promising serum biomarkers for identifying high-risk individuals with prolonged radon exposure and contribute to the identification of novel biomarkers in future studies on high-radon exposure areas.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89753-z" target="_blank">https://www.nature.com/articles/s41598-025-89753-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-10</strong> <a href="https://www.nature.com/articles/s41598-025-89696-5" target="_blank">
            ü§ñ Monkeypox Response in War-Torn Syria 2024</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-159', this)">
            More</button></h4>
            <div id="toggle-159" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Monkeypox awareness and public health challenges in conflict affected syria 2024</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new disease called Monkeypox is spreading around the world. It's mostly affecting people who don't usually get it. To stop it from spreading, we need to know what people know and do about it. This study looked at 2000 people living in Syria, where there has been a lot of fighting. They asked people if they knew things like how Monkeypox is spread, how to protect themselves, and what the symptoms are. 

Most people didn't know enough about vaccination and testing for the disease. A lot of people were worried about it, but city people thought more positively about it than rural people did. People who were better educated or had higher incomes knew more about Monkeypox too.

We need to teach people in these areas more about how to keep themselves safe from the disease so they can stay healthy and prevent it from spreading further.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox (MPOX) is a zoonotic viral disease that has re-emerged and spread to non-endemic regions, drawing global attention. Understanding the knowledge, attitudes, and practices (KAP) of vulnerable populations, particularly in conflict-affected areas, is crucial for effective public health responses. This community-based cross-sectional study evaluates the KAP of the general Syrian population regarding MPOX, focusing on areas affected by conflict. The study was conducted between October 25 and November 22, 2024, using a culturally adapted and validated structured online and paper-based questionnaire. A total of 2,035 responses were collected. The sample included predominantly female participants (63.8%), aged 21‚Äì30 years (41.9%), mostly married (73.4%), and urban residents (72.4%). Statistical analysis included confidence intervals (95%) and significance levels (p<‚Äâ0.05). The study revealed that while 80.3% of participants understood causative factors, 81.2% transmission, 89% prevention, and 78.2% symptoms, significant knowledge gaps were identified in vaccination (13.4%) and diagnostic tests (42.7%), with 58.9% of respondents demonstrating poor overall knowledge. Negative attitudes were reported in 65.8% of participants, although urban residents exhibited slightly more positive attitudes. Good practices were observed in 73.6% of respondents, with common behaviors including isolation and frequent handwashing (58.8%). However, practices did not significantly differ across genders or marital status. Higher knowledge scores were associated with males (OR‚Äâ=‚Äâ1.721, 95% CI= [1.42‚Äì2.07]), higher education levels (OR‚Äâ=‚Äâ1.324, 95% CI= [1.07‚Äì1.62]), and higher incomes (OR‚Äâ=‚Äâ2.550, 95% CI= [1.56‚Äì4.15]). The conflict-affected context likely exacerbated knowledge gaps and negative attitudes, underscoring the urgent need for culturally tailored educational campaigns and public health initiatives. These efforts are critical for improving MPOX preparedness in vulnerable populations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89696-5" target="_blank">https://www.nature.com/articles/s41598-025-89696-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-10</strong> <a href="https://www.nature.com/articles/s41467-025-56317-8" target="_blank">
            ü§ñ Targeting a specific enzyme to treat diseases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-160', this)">
            More</button></h4>
            <div id="toggle-160" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Macrocycle-based PROTACs selectively degrade cyclophilin A and inhibit HIV-1 and HCV</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have found a new way to fight viruses by targeting specific proteins in the body that help them multiply. They created special molecules called PROTEIN-CUTTING CHIMERAS that attach to these protein molecules and break them down. This breaks down the virus's ability to replicate, which can make people sick.

These special molecules are made from a naturally occurring substance found in plants and have a unique shape that makes them effective against certain viruses. They work best when they target a specific protein called cyclophilin A, which is used by many different types of viruses, including HIV and hepatitis C.

When the virus's cyclophilin A protein is broken down, it helps stop the virus from replicating. This can be an effective way to fight off infections and reduce the risk of developing resistance to certain treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Targeting host proteins that are crucial for viral replication offers a promising antiviral strategy. We have designed and characterised antiviral PROteolysis TArgeting Chimeras (PROTACs) targeting the human protein cyclophilin A (CypA), a host cofactor for unrelated viruses including human immunodeficiency virus (HIV) and hepatitis C virus (HCV). The PROTAC warheads are based on fully synthetic macrocycles derived from sanglifehrin A, which are structurally different from the classical Cyp inhibitor, cyclosporine A. Our Cyp-PROTACs decrease CypA levels in cell lines and primary human cells and have high specificity for CypA confirmed by proteomics experiments. Critically, CypA degradation facilitates improved antiviral activity against HIV-1 in primary human CD4+‚ÄâT cells compared to the non-PROTAC parental inhibitor, at limiting inhibitor concentrations. Similarly, we observe antiviral activity against HCV replicon in a hepatoma cell line. We propose that CypA-targeting PROTACs inhibit viral replication potently and anticipate reduced evolution of viral resistance and broad efficacy against unrelated viruses. Furthermore, they provide powerful tools for probing cyclophilin biology.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56317-8" target="_blank">https://www.nature.com/articles/s41467-025-56317-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-08</strong> <a href="https://www.nature.com/articles/s41598-025-89472-5" target="_blank">
            ü§ñ Healthcare Workers at Risk from COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-161', this)">
            More</button></h4>
            <div id="toggle-161" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection rates and associated risk factors in healthcare workers: systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, many healthcare workers got sick with the virus. Researchers wanted to know how common it was for healthcare workers to get infected and what made them more likely to catch it.

They looked at over 300,000 healthcare workers from around the world and found that about 11% of them got infected with SARS-CoV-2.

The people who were most likely to get infected were:

- Those who lived with family members who had COVID-19
- People who cleaned hospitals or worked in jobs where they came into contact with sick patients
- Workers who didn't have the best training on keeping themselves and others safe from germs
- Healthcare workers who wore bad protective gear

On the other hand, some healthcare workers were less likely to get infected if:

- They had been quarantined before getting COVID-19
- They cleaned high-touch areas regularly</p><br>
                <p><strong>Abstract</strong></p>
                <p>To protect healthcare workforce during the COVID-19 pandemic, rigorous efforts were made to reduce infection rates among healthcare workers (HCWs), especially prior to vaccine availability. This study aimed to investigate the prevalence of SARS-CoV-2 infections among HCWs and identify potential risk factors associated with transmission. We searched MEDLINE, Embase, and Google Scholar from 1 December 2019 to 5 February 2024. From 498 initial records, 190 articles were reviewed, and 63 studies were eligible. ROBINS-E tool revealed a lower risk of bias in several domains; however, some concerns related to confounding and exposure measurement were identified. Globally, 11% (95% confidence interval (CI) 9‚Äì13) of 283,932 HCWs were infected with SARS-CoV-2. Infection rates were associated with a constellation of risk factors and major circulating SARS-CoV-2 variants. Household exposure (odds ratio (OR) 7.07; 95% CI 3.93‚Äì12.73), working as a cleaner (OR 2.72; 95% CI 1.39‚Äì5.32), occupational exposure (OR 1.79; 95% CI 1.49‚Äì2.14), inadequate training on infection prevention and control (OR 1.46; 95% CI 1.14‚Äì1.87), insufficient use of personal protective equipment (OR 1.45; 95% CI 1.14‚Äì1.84), performing aerosol generating procedures (OR 1.36; 95% CI 1.21‚Äì1.52) and inadequate hand hygiene (OR 1.17; 95% CI 0.79‚Äì1.73) were associated with an increased SARS-CoV-2 infection. Conversely, history of quarantine (OR 0.23; 95% CI 0.08‚Äì0.60) and frequent decontamination of high touch areas (OR 0.52; 95% CI 0.42‚Äì0.64) were protective factors against SARS-CoV-2 infection. This study quantifies the substantial global burden of SARS-CoV-2 infection among HCWs. We underscore the urgent need for effective infection prevention and control measures, particularly addressing factors such as household exposure and occupational practices by HCWs, including cleaning staff.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89472-5" target="_blank">https://www.nature.com/articles/s41598-025-89472-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-08</strong> <a href="https://www.nature.com/articles/s41598-025-89272-x" target="_blank">
            ü§ñ Pharmacovigilance Study: How Medicines Cause Autoimmune Disease</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-162', this)">
            More</button></h4>
            <div id="toggle-162" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Real-world pharmacovigilance study of drug-induced autoimmune hepatitis from the FAERS database</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

This study looked at which medicines are most likely to cause autoimmune hepatitis (a disease where the body's immune system attacks itself). It used a big database of medicine reports from the US government to see which medicines were most linked to this condition. The study found that 22 medicines were strongly associated with an increased risk of autoimmune hepatitis, including antibiotics, antivirals, and some medicines for cancer and heart problems.

The study also looked at how quickly the problem started after taking these medicines. Some medicines caused autoimmune hepatitis within just a few days, while others took months or even years to cause the condition.

By understanding which medicines are most likely to cause this problem, doctors can be more careful when prescribing them and monitor patients for signs of autoimmune hepatitis. This study helps us know which medicines to use with caution and how to prevent autoimmune hepatitis.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aims to identify and evaluate the most common drugs associated with the risks of autoimmune hepatitis (AIH) using the FDA Adverse Event Reporting System (FAERS) database. Adverse drug events (ADEs) associated with drug-induced AIH (DI-AIH) were retrieved from the FAERS database (January 2004‚ÄìJune 2024). Disproportionality analysis was performed to identify drugs significantly linked to DI-AIH, and time-to-onset (TTO) analyses were conducted to evaluate the timing and risk profiles of DI-AIH adverse reactions. Our study identified 2,511 ADEs linked to autoimmune hepatitis. Disproportionality analysis identified 22 drugs significantly associated with AIH risk, including 4 antibiotics, 3 antivirals, 4 cardiovascular drugs, 5 antitumor agents, 2 immunomodulators, 2 nonsteroidal anti-inflammatory drugs, and 1 drug each from the respiratory and nervous system categories. The highest DI-AIH risks were observed with minocycline (ROR‚Äâ=‚Äâ53.97), nitrofurantoin (ROR‚Äâ=‚Äâ57.02), and doxycycline (ROR‚Äâ=‚Äâ16.12). Antitumor drugs had the shortest median TTO (77.00 days), whereas cardiovascular drugs exhibited the longest (668.30 days). Through a comprehensive analysis of the FAERS database, our study identified drugs strongly associated with AIH. Preventing DI-AIH requires careful drug selection and monitoring. This study provides evidence-based insights into implicated drugs, aiming to optimize clinical management and mitigate risks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89272-x" target="_blank">https://www.nature.com/articles/s41598-025-89272-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-08</strong> <a href="https://www.nature.com/articles/s41598-025-87700-6" target="_blank">
            ü§ñ COVID-19 pandemic pressures led to undue pressure on healthcare workers.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-163', this)">
            More</button></h4>
            <div id="toggle-163" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Perceived coercion amongst healthcare workers during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract rewritten for a young person:

Healthcare workers faced extreme stress during the COVID-19 pandemic due to the high risk of getting sick from patients. This stress can affect their mental health and well-being. Researchers created a tool to measure how much healthcare workers feel like they're being forced into doing certain jobs without any control over it. They found that many healthcare workers felt some level of pressure, but not as much as expected. However, those who felt more pressured reported higher levels of stress, anxiety, and burnout. This study shows that feeling trapped in a high-stress job can have serious consequences for our mental health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Direct and prolonged exposure to stress and uncertainty among healthcare workers (HCWs)¬†during the COVID-19 pandemic likely had a significant negative impact on their mental health and general wellbeing. Although the contributors to such distress remain to be fully understood, the construct ofperceived coercionappears to be relevant. Among HCWs, perceived coercion is conceptualised as appraisals about lack of control/‚Äòfreedom to choose‚Äô and pressure to perform patient-care duties in the context of unprecedented threat of contagion from patients. To improve our understanding of perceived coercion amongst HCWs, we developed a 10-item scale‚Äîthe Pandemic-specific Perceived Coercion Scale for Healthcare Workers (PPCS-HCW) scale‚Äîdesigned to be adaptable and applicable for use in future mass-contagion emergencies. A preliminary (exploratory) factor analysis (N=‚Äâ546) showed that relevant items coalesced around three factors‚Äî‚Äòinternal pressure‚Äô, ‚Äòexternal pressure‚Äô and ‚Äòperceived coercion‚Äô, that partly overlap with previous conceptualisations of perceived coercion. The exploratory conceptual and psychometric structure was confirmed in a separate sample of HCWs from the UK and Norway (N=‚Äâ483).On average, across the three PPCS-HCW scale factors, HCWs showed low levels of perceived coercion (M=‚Äâ0.22 (95% CI [0.11, 0.33] on a ‚àí‚Äâ3 to‚Äâ+‚Äâ3 scale). However, cluster analysis identified three groups: low (‚àí‚Äâ1.09 (95% CI [‚àí‚Äâ1.20, ‚àí‚Äâ0.99]), moderate (0.17 (95% CI [0.08, 0.25]) and high scoring (1.57 (95% CI [1.47, 1.67]) PPCS-HCW clusters. High scoring participants showed higher levels of psychological distress, avoidance coping and compassion fatigue. In summary, our findings suggest that perceived coercion is a relevant construct in understanding the adverse psychological impact of large-scale contagion emergencies on HCWs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87700-6" target="_blank">https://www.nature.com/articles/s41598-025-87700-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-07</strong> <a href="https://www.nature.com/articles/s41419-025-07400-x" target="_blank">
            ü§ñ New Research Reveals Key Role of Enzyme in Human Placenta</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-164', this)">
            More</button></h4>
            <div id="toggle-164" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genetically edited human placental organoids cast new light on the role of ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

During pregnancy, some things don't go as planned. In people who have these problems, a certain gene called ACE2 doesn't work properly. This study looked at what happens when ACE2 isn't working right. The researchers created special tiny organs that can grow and develop like real placentas. They studied how ACE2 affects these organs and found some surprising things.

Without ACE2, the tiny organs didn't grow as quickly and had more cell death. Without ACE2, they also didn't grow evenly or look normal. On the other hand, when people have a certain version of the ACE2 gene that's commonly found in people with pregnancy problems, their ACE2 works better than expected. However, even though it works better, it doesn't do its job as well - like helping to break down special chemicals.

This study helps us understand more about how ACE2 affects placental development during pregnancy and might give clues for future treatments for people with these issues.</p><br>
                <p><strong>Abstract</strong></p>
                <p>ACE2 expression is altered in pregnancy disorders andACE2gene variants are associated with several major pregnancy complications including small-for-gestational-age, fetal growth restriction and preeclampsia. This study utilised gene-editing to generate bothACE2knockout andACE2rs2074192 placental organoids, facilitating mechanistic studies into the role ofACE2in placental development, and the effect of fetal carriage ofACE2rs2074192 CC, CT and TT genotypes. Parameters of cell and organoid growth were measured, together with qPCR, Western Blotting, and ELISA assessments, in all groups from both organoid models. Here, we report thatACE2knockout results in delayed placental cell growth and increased cell death.ACE2knockout organoids had lower ACE protein expression, reduced organoid diameters and asymmetrical growth. Placental organoids with the ACE2 rs2074192 TT genotype had significantly higher expression ofACE2mRNA and ACE2 protein with elevated ACE2:ACE expression ratio and no change in ACE protein. Despite increased expression of ACE2 protein, ACE2 enzyme activity was significantly decreased in ACE2 rs2074192 TT placental organoids. TT organoids also had reduced diameters and asymmetrical growth. Our research provides a new molecular understanding of the role of ACE2 in placental development, with potential implications for pregnancy in the carriage of theACE2rs2074192 gene variant.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-025-07400-x" target="_blank">https://www.nature.com/articles/s41419-025-07400-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-07</strong> <a href="https://www.nature.com/articles/s41467-025-56603-5" target="_blank">
            ü§ñ Effects of Viral Infection on the Immune System.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-165', this)">
            More</button></h4>
            <div id="toggle-165" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Metabolic deficiencies underlie reduced plasmacytoid dendritic cell IFN-I production following viral infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

When our body fights off viruses, a type of protein called Interferon plays a crucial role. Special cells called plasmacytoid dendritic cells make this protein, but after they produce it for a while, they stop working and become more susceptible to getting sick themselves. We wanted to know why this happens.

We found out that when our body is fighting off a virus, these special cells lose their ability to use two types of energy: oxygen and sugar. This makes them less good at making Interferon. But if we help them keep using these energies, they can make more Interferon and fight the virus better. However, if we give them too much Interferon, it can actually cause problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Type I Interferons (IFN-I) are central to host protection against viral infections, with plasmacytoid dendritic cells (pDC) being the most significant source, yet pDCs lose their IFN-I production capacity following an initial burst of IFN-I, resulting in susceptibility to secondary infections. The underlying mechanisms of these dynamics are not well understood. Here we find that viral infection reduces the capacity of pDCs to engage both oxidative and glycolytic metabolism. Mechanistically, we identify lactate dehydrogenase B (LDHB) as a positive regulator of pDC IFN-I production in mice and humans; meanwhile, LDHB deficiency is associated with suppressed IFN-I production, pDC metabolic capacity, and viral control following infection. In addition, preservation of LDHB expression is sufficient to partially retain the function of otherwise exhausted pDCs, both in vitro and in vivo. Furthermore, restoring LDHB in vivo in pDCs from infected mice increases IFNAR-dependent, infection-associated pathology. Our work thus identifies a mechanism for balancing immunity and pathology during viral infections, while also providing insight into the highly preserved infection-driven pDC inhibition.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56603-5" target="_blank">https://www.nature.com/articles/s41467-025-56603-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-07</strong> <a href="https://www.nature.com/articles/s41467-025-56151-y" target="_blank">
            ü§ñ Fighting virus by altering how cells make proteins.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-166', this)">
            More</button></h4>
            <div id="toggle-166" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Targeting eEF1A reprograms translation and uncovers broad-spectrum antivirals against cap or m6A protein synthesis routes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Plitidepsin is a medicine that helps treat COVID-19 by stopping certain parts of the virus from replicating inside our bodies. It works by blocking a part of the cell where the virus attaches, which makes it harder for the virus to multiply.

When cells in our body are exposed to plitidepsin, they start making more proteins that help keep themselves healthy and stop making other proteins that would allow the virus to replicate.

However, some viruses have special ways of replicating that plitidepsin can't block. To find new medicines that can fight these types of viruses, scientists need to understand how cells respond to antiviral treatments like plitidepsin.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Plitidepsin is an antitumoral compound safe for treating COVID-19 that targets the translation elongation factor eEF1A. Here we detect that plitidepsin decreases de novo cap-dependent translation of SARS-CoV-2 and non-viral RNAs but affects less than 13% of the host proteome, thus preserving cellular viability. In response to plitidepsin, cells upregulate EIF2AK3 and proteins that reduce translation, but also proteins that support proteostasis via ribosome synthesis and cap-independent translation by eIF4G2 and IGF2BP2. While plitidepsin inhibits cap- or internal ribosome entry sites (IRES)-mediated translation, its impact on N6-methyladenosine (m6A) translation is limited. In agreement, plitidepsin blocks members ofCoronaviridae,Flaviviridae,PneumoviridaeandHerpesviridaefamilies. Yet, it fails to inhibit retroviruses that exploit m6A synthesis routes and are blocked by drugs targeting IGF2BP2 m6A reader. By deciphering the molecular fingerprint of cells treated with therapies targeting translation we identify a rational approach to select broad-spectrum antivirals with potential to counteract future pandemic viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56151-y" target="_blank">https://www.nature.com/articles/s41467-025-56151-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-07</strong> <a href="https://www.nature.com/articles/s41598-025-89170-2" target="_blank">
            ü§ñ Changes to Kidney Function and Covid-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-167', this)">
            More</button></h4>
            <div id="toggle-167" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Incidence and pathological features of IgA nephropathy before and during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Kidney disease called IgA nephropathy (IgAN) affects many people. We looked at kidney biopsies from two hospitals to see how IgAN changed between 2016 and 2023, during and after the COVID-19 pandemic. We found that IgAN happened more often during the pandemic than before it started. People with IgAN had different types of damage in their kidneys, which got worse during the pandemic. This means we need to be more careful about IgAN patients now.</p><br>
                <p><strong>Abstract</strong></p>
                <p>IgA nephropathy (IgAN) imposes a substantial burden of illness and death. However, a systematic evaluation of the impact of the COVID-19 pandemic on the incidence and pathology of IgAN has yet to be performed. In this study, we analyzed kidney biopsy results from two medical institutions, covering the timeframe from January 2016 to May 2023. We conducted statistical analyses on various glomerular diseases according to their corresponding pathological diagnoses. Our objective was to compare the incidence of different glomerular diseases before and during the COVID-19 pandemic. The study focused on variations in the incidence of IgAN, and collected clinical and pathological data to assess pathological changes using the Oxford Classification (MEST-C). The findings revealed a significant increase in the incidence of IgAN during the COVID-19 pandemic, from 39.9% prior to the pandemic to 46.3% during it, representing a net increase of 6.4% (P<‚Äâ0.001). Although clinical manifestations and laboratory indicators of disease activity in IgAN patients remained consistent over both periods, observable changes occurred in pathological features. Specifically, the proportions of M1 and E1 lesions according to the Oxford classification significantly increased during the pandemic, with odds ratios of 11.764 (95% CI 5.583‚Äì24.789,P<‚Äâ0.001) and 1.718 (95% CI 1.114‚Äì2.649,P=‚Äâ0.014), respectively. Our results indicate that the incidence of IgAN has risen during the COVID-19 pandemic, along with exacerbated renal damage and elevated proportions of M1 and E1 in the Oxford classification.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-89170-2" target="_blank">https://www.nature.com/articles/s41598-025-89170-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-06</strong> <a href="https://www.nature.com/articles/s41598-025-88518-y" target="_blank">
            ü§ñ Older people with severe COVID-19 may be more likely to die if they have certain types of bad bacteria in their mouths.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-168', this)">
            More</button></h4>
            <div id="toggle-168" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pathogenic and periodontal bacteria may contribute to the fatal outcome of critically ill elderly COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten abstract that explains the research in simple terms:

People who had COVID-19 either recovered or got very sick and died. We wanted to know if there was a difference in their bodies' microbiomes (the good bacteria inside them). 

We looked at 48 people: 24 who recovered from COVID-19 and 24 who didn't survive. We found that the people who died had higher levels of bad bacteria like Klebsiella pneumoniae and other infections.

The study suggests that having COVID-19 can trigger a stronger immune response against periodontal diseases, which are related to gum disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Some studies suggest that the respiratory microbiome of COVID-19 patients differs from that of healthy individuals, infected patients may have reduced diversity and increased levels of opportunistic bacteria, however, the role of the microbiome in fatal SARS-CoV-2 infection remains poorly understood. Our study aimed to determine whether there are differences in the respiratory microbiome between patients who recovered from COVID-19 and those who died, by characterizing the bacterial communities of both groups. A total of 24 patients who recovered from COVID-19 and 24 who died were included in the study, patient data were analyzed for signs, symptoms and clinical variables. Airway samples were collected and the 16¬†S rRNA variable regions V3-V4 were amplified and sequenced using the Illumina MiSeq platform. Elevated levels of blood urea nitrogen, creatinine and lactate dehydrogenase, and higher frequencies of cardiovascular disease, diabetes mellitus and renal disease were observed in patients with a fatal outcome. Compared to patients who recovered from COVID-19, patients who died exhibited a microbiome enriched in periodontal and pathogenic bacteria such asKlebsiella pneumoniae. Our results highlighted a dual relationship between SARS CoV-2 infection and an exacerbated periodontopathogen-induced immune response.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-88518-y" target="_blank">https://www.nature.com/articles/s41598-025-88518-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-06</strong> <a href="https://www.nature.com/articles/s41467-025-56659-3" target="_blank">
            ü§ñ How T cells remember past infections or threats.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-169', this)">
            More</button></h4>
            <div id="toggle-169" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>APMAT analysis reveals the association between CD8 T cell receptors, cognate antigen, and T cell phenotype and persistence</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists are trying to understand how our bodies respond to certain proteins from viruses like SARS-CoV-2. They want to know what makes each person's immune system react differently to these proteins. To solve this, they've created a new tool called APMAT that helps them study how T cells (a type of immune cell) interact with specific proteins and genetic sequences.

They used APMAT to analyze data from 62 people who had COVID-19. What they found was surprising: the way a protein is made and its shape, as well as the genetics of the person's immune system, can determine how their T cells respond to it. This suggests that our immune systems are not completely random in their reactions - there may be patterns and rules that govern how we fight off infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Elucidating the relationships between a class I peptide antigen, a CD8 T cell receptor (TCR) specific to that antigen, and the T cell phenotype that emerges following antigen stimulation, remains a mostly unsolved problem, largely due to the lack of large data sets that can be mined to resolve such relationships. Here, we describe Antigen-TCR Pairing and Multiomic Analysis of T-cells (APMAT), an integrated experimental-computational framework designed for the high-throughput capture and analysis of CD8 T cells, with paired antigen, TCR sequence, and single-cell transcriptome. Starting with 951 putative antigens representing a comprehensive survey of the SARS-CoV-2 viral proteome, we utilize APMAT for the capture and single cell analysis of CD8 T cells from 62 HLA A*02:01 COVID-19 participants. We leverage this comprehensive dataset to integrate with peptide antigen properties, TCR CDR3 sequences, and T cell phenotypes to show that distinct physicochemical features of the antigen-TCR pairs strongly associate with both T cell phenotype and T cell persistence. This analysis suggests that CD8 T cell phenotype following antigen stimulation is at least partially deterministic, rather than the result of stochastic biological properties.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56659-3" target="_blank">https://www.nature.com/articles/s41467-025-56659-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-05</strong> <a href="https://www.nature.com/articles/s41586-024-08511-9" target="_blank">
            ü§ñ Protection from getting sick with COVID-19 again after first infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-170', this)">
            More</button></h4>
            <div id="toggle-170" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Differential protection against SARS-CoV-2 reinfection pre- and post-Omicron</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The virus that causes COVID-19, SARS-CoV-2, has changed a lot over time. New versions like Omicron have shown up with more mutations, making it harder for our bodies to fight off. We studied how well people who got infected before and after Omicron could get infected again. Before Omicron, getting sick helped keep you from getting sick again. But after Omicron, that protection didn't last long. This means the virus is evolving in ways that are making it harder for our bodies to fight back. We need to update vaccines regularly to stay ahead of these changes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has rapidly evolved over short timescales, leading to the emergence of more transmissible variants such as Alpha and Delta1,2,3. The arrival of the Omicron variant marked a major shift, introducing numerous extra mutations in the spike gene compared with earlier variants1,2. These evolutionary changes have raised concerns regarding their potential impact on immune evasion, disease severity and the effectiveness of vaccines and treatments1,3. In this epidemiological study, we identified two distinct patterns in the protective effect of natural infection against reinfection in the Omicron versus pre-Omicron eras. Before Omicron, natural infection provided strong and durable protection against reinfection, with minimal waning over time. However, during the Omicron era, protection was robust only for those recently infected, declining rapidly over time and diminishing within a year. These results demonstrate that SARS-CoV-2 immune protection is shaped by a dynamic interaction between host immunity and viral evolution, leading to contrasting reinfection patterns before and after Omicron‚Äôs first wave. This shift in patterns suggests a change in evolutionary pressures, with intrinsic transmissibility driving adaptation pre-Omicron and immune escape becoming dominant post-Omicron, underscoring the need for periodic vaccine updates to sustain immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-08511-9" target="_blank">https://www.nature.com/articles/s41586-024-08511-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-05</strong> <a href="https://www.nature.com/articles/s41598-025-88914-4" target="_blank">
            ü§ñ The severity of COVID-19 illness can be predicted by looking at three specific health issues: muscle loss (sarcopenia), fat buildup in muscles (myosteatosis), and inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-171', this)">
            More</button></h4>
            <div id="toggle-171" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sarcopenia, myosteatosis and inflammation are independent prognostic factors of SARS-CoV-2 pneumonia patients admitted to the ICU</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We studied people who were very sick with COVID-19 when they came to the intensive care unit (ICU). We looked at how muscle loss and inflammation affected their chances of dying within 90 days after admission.

Muscle loss was shown to be a big risk factor for death, along with high levels of inflammatory chemicals in the body. We also found that people who had more muscle density in certain parts of their body were less likely to die.

Our study suggests that muscle problems and inflammation can be important predictors of how well someone will do after being critically ill with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aims of our study were to assess the correlations between sarcopenia and myosteatosis assessed by CT-scan at T4 and/or L3 levels and inflammation in critically ill COVID patients on ICU admission, and their respective prognostic value on day 90 death (D90-death). It is a retrospective monocentric study. Sarcopenia was defined by skeletal muscle cross sectional surface area (CSA) and myosteatosis by skeletal muscle density (SMD) at L3 and T4 levels. Inflammatory biomarkers were collected on ICU admission. Of the 239 patients, 74 died by D90; 66.6% get sarcopenia on ICU admission. CSA at T4 level was an independent risk factor for D90-death (1.66[1.03; 2.66];p=‚Äâ0.04), as were procalcitonin (2.03[1.2; 3.43];p=‚Äâ0.01) and IL-6 levels (1.56[0.96; 2.54];p=‚Äâ0.07). In addition, we found correlation factors of 0.79 (p<‚Äâ0.01) between SMD at T4 and L3 levels, and a correlation factor of 0.64 (p<‚Äâ0.01) between CSA at T4 and L3 levels.These results indicate a poorer prognosis following a decrease in muscle surface area, a decrease in density, and an increase in inflammatory biomarkers such as Il6. It also suggests that incorporating indices of sarcopenia with inflammatory biomarkers may improve prognostic accuracy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-88914-4" target="_blank">https://www.nature.com/articles/s41598-025-88914-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-04</strong> <a href="https://www.nature.com/articles/s41598-024-82387-7" target="_blank">
            ü§ñ Determining Certain Nasal Medications in Human Samples Using a Specialized Lab Technique.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-172', this)">
            More</button></h4>
            <div id="toggle-172" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Validated HPLC method for simultaneous determination of azelastine hydrochloride fluticasone propionate and oxymetazoline in nasal mucosa and nasopharyngeal swabs from real human samples</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that explains it simply:

Researchers have found that using three medicines together (azelastine hydrochloride, fluticasone propionate, and oxymetazoline) is better at treating seasonal allergies and COVID-19 than using one medicine alone. They developed a way to test for these medicines in nasal mucus and swabs from people. Their method worked well and accurately measured the amount of each medicine in the samples. The results showed that the medicines were stable, didn't change much over time, and could be used to monitor how much medicine is working in the nose.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A combination of three co-administrated drugs, such as azelastine hydrochloride (AZT), fluticasone propionate (FP), and oxymetazoline (OXY), is more effective than single therapy for the treatment of seasonal allergy and COVID-19. We established an efficient methodology for the determination of those analytes in spiked nasal mucosa and nasopharyngeal swabs from real human samples. A simple and quick protein precipitation method was used for sample extraction, using acetonitrile. RP-HPLC/DAD method was performed using an Exsil 100 ODS C18 (250‚Äâ√ó‚Äâ4.6¬†mm, 5¬†Œºm) column with an acetonitrile: water (70:30 v/v) solvent system at a flow rate of 0.7 mL/min. A photodiode array detector was applied at 240¬†nm. A good separation of the three proposed analytes with a short run time of 10¬†min was noted. Our method was validated according to FDA guidelines for bioanalytical validation methods. Calibration curves were linear in nasal mucosa samples at concentration ranges of 8‚Äì125, 10‚Äì100, and 10‚Äì125¬†¬µg/mL, with average recoveries‚Äâ¬±‚ÄâSD of 101.56%¬±0.39, 102.45%¬±0.86, and 104.61%¬±4.52 for AZT, FP, and OXY; respectively. The results of precision and accuracy are within acceptable limits. According to stability assays, the three analytes under investigation were stable throughout sample preparation, storage, and injection. Our method was applied to real nasopharyngeal swabs. It shows that the results of the swabs were not affected by gender or age. Good recoveries with low % RSD were observed: 99.03% ¬± 0.75, 100.02% ¬± 0.94, and 100.94% ¬± 1.98 for both genders, and 100.45% ¬± 0.96, 100.69% ¬± 1.08, and 100.32% ¬± 1.53 for different ages for AZT, FP, and OXY; respectively. Moreover, the amount of those drugs in the nasal mucosa was observed for seven hours, and a constant concentration with a low% RSD was noted for the first four hours. Therefore, this method can be applied to monitor the therapeutic dose in the nasal mucosa for the determination of those analytes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82387-7" target="_blank">https://www.nature.com/articles/s41598-024-82387-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-04</strong> <a href="https://www.nature.com/articles/s42003-024-07442-5" target="_blank">
            ü§ñ What is the SARS-CoV-2 virus doing to our cells?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-173', this)">
            More</button></h4>
            <div id="toggle-173" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 ORF3a accessory protein is a water-permeable channel that induces lysosome swelling</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract:

The SARS-CoV-2 virus has a special helper protein called ORF3a that helps it escape from host cells. This protein makes it harder for the cell's waste disposal system (lyosomes) to work properly, which is important for the virus to leave the cell. Scientists wanted to understand how ORF3a works, but there was confusion about its role.

To figure this out, researchers used different experiments and techniques like watching how the protein moves through the cell membrane, using super computers to simulate its behavior, and looking at it under a microscope. They found that ORF3a doesn't act as previously thought, but instead helps move water across the cell membrane.

This new discovery suggests that ORF3a plays a key role in helping the virus escape by making lysosomes work less efficiently, which is crucial for the virus to complete its life cycle.</p><br>
                <p><strong>Abstract</strong></p>
                <p>ORF3a, the most¬†abundantly expressed accessory protein of SARS-CoV-2, plays an essential role in virus egress by inactivating lysosomes through their deacidification. However, the mechanism underlying this¬†process remains unclear. While seminal studies suggested ORF3a being a cation-selective channel (i.e., viroporin), recent works disproved this conclusion. To unravel the potential function of ORF3a, here we employed a multidisciplinary approach including patch-clamp electrophysiology, videoimaging, molecular dynamics (MD)¬†simulations, and electron microscopy. Preliminary structural analyses and¬†patch-clamp recordings in HEK293 cells rule out ORF3a functioning¬†as either viroporin or proton (H+) channel. Conversely, videoimaging experiments demonstrate that ORF3a mediates the transmembrane transport of water. MD¬†simulations identify the tetrameric assembly¬†of ORF3a as the functional water transporter, with a putative selectivity filter for water permeation that includes two essential¬†asparagines, N82 and N119. Consistent with this, N82L and N82W mutations abolish ORF3a-mediated water permeation. Finally, ORF3a expression in HEK293 cells leads to lysosomal volume increase, mitochondrial damage, and accumulation of intracellular membranes, all alterations reverted by the N82W mutation. We propose a novel function for ORF3a as a lysosomal water-permeable channel, essential for lysosome deacidification and inactivation, key steps to promote virus egress.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07442-5" target="_blank">https://www.nature.com/articles/s42003-024-07442-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-03</strong> <a href="https://www.nature.com/articles/s41598-025-88621-0" target="_blank">
            ü§ñ Study of Growing Bat Tissues to Understand Infections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-174', this)">
            More</button></h4>
            <div id="toggle-174" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Establishment of a bat lung organoid culture model for studying bat-derived infectious diseases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Bats can carry many viruses that make people sick. To learn more about these viruses and how they work, scientists use special cells from bats. However, these cells don't perfectly show what's happening in real bats.

Researchers created a new way to grow lung tissue (called organoids) from bat cells. These tiny tissues are like miniature lungs that can mimic the behavior of real lungs. 

This breakthrough allowed them to study how viruses affect the lungs in bats and how they might be transmitted to people. The discovery also included finding out which part of the lung is most susceptible to certain viruses.

These new models could help scientists better understand bat-borne diseases and potentially develop new treatments or vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Bat is considered a natural reservoir of various important pathogens, including severe acute respiratory syndrome coronavirus (SARS-CoV), SARS-CoV-2, Ebola virus, and Nipah virus. To study these viruses‚Äô pathogenicity and proliferation efficacy and viral tolerance mechanisms in bats, bat-derived cell lines, and primary cultured cells are used. However, these do not adequately reflect the exact biology of bats, and establishing new bat-related research models is necessary. Organoid culture can recapitulate organ structure, functions, and diseases. The respiratory tract is one of the primary routes of viral infection, and the establishment of bat lung organoids (BLO) is necessary to study the viral susceptibility in bats. Therefore, we aimed to establish a culture method of BLO fromRousettus leschenaultiathat died of natural causes. The generated BLO successfully recapitulated the characteristics of pulmonary epithelial structure and morphology. BLO expressed the entry receptors for coronavirus, Angiotensin-converting enzyme 2 (ACE2), and Transmembrane Protease Serine 2 (TMPRSS2), and alveolar type 2 cells were successfully sorted from BLO, which has an important role for the development of viral infection in the respiratory system. Furthermore, we showed that BLO had no susceptibility to Pteropine orthoreovirus (PRV) compared to bat intestinal organoids. Collectively, our established bat organoid culture models including this BLO might become promising in vitro biomaterials to study the biology of bat-derived infectious diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-88621-0" target="_blank">https://www.nature.com/articles/s41598-025-88621-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-03</strong> <a href="https://www.nature.com/articles/s42003-025-07600-3" target="_blank">
            ü§ñ Hammerhead enzyme picks best shape to help break down virus RNA</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-175', this)">
            More</button></h4>
            <div id="toggle-175" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A hammerhead ribozyme selects mechanically stable conformations for catalysis against viral RNA</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

Ribozymes are special molecules that can target and break down other molecules called nucleic acids. They are found in all kinds of living things, from bacteria to humans. One type of ribozyme is called hammerhead, and it's really small but also very good at doing its job.

Scientists have been trying to understand how these ribozymes work for a long time. They're not sure if the way they move and bend actually helps them break down the nucleic acids. 

We used special tools to study one type of hammerhead ribozyme that can target a certain virus. Our results show that when we change the shape of this ribozyme, it starts to work better. We think that the tiny metal ions in the molecule help pick the right shape for it to work properly. This discovery could help us use ribozymes as medicine or tools in biology.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Ribozymes, widely found in prokaryotes and eukaryotes, target nucleic acids and can be engineered as biotechnical tools or for gene regulation or immune therapy. Among them, hammerhead is the smallest and best characterized ribozyme. However, the structure and biochemical data of ribozymes have been disagreed on, making the understanding of its catalysis mechanism a longstanding issue. Particularly, the role of conformational dynamics in ribozyme catalysis remains elusive. Here, we use single-molecule magnetic tweezers to reveal a concerted catalysis mechanism of mechanical conformational selection for a mini hammerhead ribozyme against a viral RNA sequence from the SARS-CoV-2. We identify a conformational set containing five mechanical conformers of the mini ribozyme, where magnesium ions select the active one. Our results are supported by molecular dynamics simulations. Our understanding of the RNA catalytic mechanism will be beneficial for ribozyme‚Äôs biotechnological applications and as potential therapeutics against RNA viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07600-3" target="_blank">https://www.nature.com/articles/s42003-025-07600-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-03</strong> <a href="https://www.nature.com/articles/s41392-025-02132-y" target="_blank">
            ü§ñ Effects of Vaccines on Older Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-176', this)">
            More</button></h4>
            <div id="toggle-176" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing antibody responses to three XBB protein vaccines in older adults</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the simplified version:

The COVID-19 pandemic has shown how important it is to have strong immune systems. Some vaccines are working well against new strains of the virus, but we don't know enough about how they work for older people over 65.

This study looked at three different vaccines that contain a specific type of protection (XBB) and compared them in 90 people over 65. The results show that all three vaccines helped the immune system make antibodies to fight against new strains of the virus. 

The best two vaccines were found to work better than the third one. Some strains of the virus are more resistant to the vaccine, but most older adults can still get a strong response.

This means that booster shots with XBB-containing vaccines can help keep older people safe from new strains of COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ongoing COVID-19 pandemic has underscored the importance of strong immune defenses against emerging SARS-CoV-2 variants. While COVID-19 vaccines containing XBB subvariants have proven effective in neutralizing new SARS-CoV-2 variants, a gap remains in knowledge regarding neutralizing antibody responses in older adults aged >65‚Äâyears against these newly emerged variants. This study was therefore undertaken to investigate and compare neutralizing antibody responses to three XBB-containing protein-based vaccines (trivalent XBB.1.5 vaccine, bivalent Omicron XBB vaccine, and tetravalent XBB.1 vaccine) head-to-head in 90 individuals aged >65‚Äâyears. The results showed that all three XBB-containing vaccines substantially enhanced the neutralizing antibody response, with 100% of vaccinees having detectable antibody titers against ancestral D614G and variants BA.5, XBB.1.5, JN.1, KP.2, and KP.3 after booster immunization. Subsequent analysis indicated that the trivalent XBB.1.5 and tetravalent XBB.1 vaccines elicited higher levels of neutralizing antibodies compared to the bivalent Omicron XBB vaccine. The KP.2 and KP.3 variants displayed antibody resistance comparable to the JN.1 variant. Older adults produce similar neutralizing antibody responses to the vaccines regardless of their underlying medical conditions. These findings indicate that booster vaccination with XBB-containing vaccines can effectively elicit strong neutralizing responses against a number of SARS-CoV-2 variants in older adults over 65‚Äâyears, which will help guide vaccine strategies in this elderly population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-025-02132-y" target="_blank">https://www.nature.com/articles/s41392-025-02132-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-03</strong> <a href="https://www.nature.com/articles/s44298-025-00096-y" target="_blank">
            ü§ñ New Vaccine Shows Partial Protection Against New Coronavirus Strains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-177', this)">
            More</button></h4>
            <div id="toggle-177" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An XBB.1.5-based inactivated SARS-CoV-2 vaccine partially protects against XBB.1.5 and JN.1 strains in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>New COVID variant is spreading fast worldwide. Vaccination is key to prevent it. Most vaccines are mRNA-based, but we need alternative options. We created an inactivated vaccine that protects against new and emerging strains like Omicron XBB.1.5 and JN.1. This vaccine works by making the body produce antibodies that fight off these viruses, reducing their spread in the body.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 Omicron BA.2.86 variant and its descendant lineages, including JN.1, are rapidly spreading and becoming dominant globally. Vaccination is an essential primary preventative measure. While mRNA vaccines have been widely used worldwide, it is essential that we continue to prepare alternative vaccine modalities. Consistent with WHO recommendations, we developed an inactivated Omicron XBB.1.5 vaccine and assessed its efficacy against XBB.1.5 and JN.1 strains. Immunization with the inactivated XBB.1.5 vaccine induced antigen-specific antibodies leading to protection from XBB.1.5 and antigenically distinct JN.1 strains in a hamster model. In addition, we found that immunization reduced viral replication in hamster respiratory organs, suggesting protection against XBB.1.5 and JN.1 variants. Our findings highlight the potential of inactivated vaccines against evolving SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-025-00096-y" target="_blank">https://www.nature.com/articles/s44298-025-00096-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-03</strong> <a href="https://www.nature.com/articles/s43856-025-00745-6" target="_blank">
            ü§ñ SARS-CoV-2 can infect the pancreas if a specific gene (PLAC8) is not present.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-178', this)">
            More</button></h4>
            <div id="toggle-178" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Host factor PLAC8 is required for pancreas infection by SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract written for young people:

COVID-19 made us worried about our lungs, but now scientists are learning that it can also affect other parts of the body, including the pancreas. The pancreas is a vital organ that helps regulate blood sugar levels and produces digestive enzymes.

Researchers wanted to know more about how COVID-19 affects the pancreas and whether there's a specific protein called PLAC8 that plays a role in this process. They studied patients with different levels of severity of COVID-19 and found some surprising connections between PLAC8, SARS-CoV-2 (the virus that causes COVID-19), and pancreatic damage.

Their findings suggest that the pancreas is indeed infected by SARS-CoV-2 and that PLAC8 might be a key player in this process. This could lead to new targets for treating people who develop complications from COVID-19, such as kidney problems or diabetes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundAlthough COVID-19 initially caused great concern about respiratory symptoms, mounting evidence shows that also the pancreas is productively infected by SARS-CoV-2. However, the severity of pancreatic SARS-CoV-2 infection and its pathophysiology is still under debate. Here, we investigate the consequences of SARS-CoV-2 pancreatic infection and the role of the host factor Placenta-associated protein (PLAC8).MethodsWe analyze plasma levels of pancreatic enzymes and inflammatory markers in a retrospective cohort study of 120 COVID-19 patients distributed in 3 severity-stratified groups. We study the expression of SARS-CoV-2 and PLAC8 in the pancreas of deceased COVID-19 patients as well as in non-infected donors. We perform pseudovirus infection experiments in PLAC8 knock-out PDAC and human beta cell-derived cell lines and validate results with SARS-CoV-2 virus.ResultsWe find that analysis of circulating pancreatic enzymes aid the stratification of patients according to COVID-19 severity and predicts outcomes. Interestingly, we find an association between PLAC8 expression and SARS-CoV-2 infection in postmortem analysis of COVID-19 patients both in the pancreas and in other bonafide SARS-CoV-2 target tissues. Functional experiments demonstrate the requirement of PLAC8 in SARS-CoV-2 pancreatic productive infection by pseudovirus and full SARS-CoV-2 infectious virus inoculum from Wuhan-1 and BA.1 strains. Finally, we observe an overlap between PLAC8 and SARS-CoV-2 immunoreactivities in the pancreas of deceased patients.ConclusionsOur data indicate the human pancreas as a SARS-CoV-2 target with plausible signs of injury and demonstrate that the host factor PLAC8 is required for SARS-CoV-2 pancreatic infection, thus defining new target opportunities for COVID-19-associated pancreatic pathogenesis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00745-6" target="_blank">https://www.nature.com/articles/s43856-025-00745-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-02-01</strong> <a href="https://www.nature.com/articles/s41467-025-56651-x" target="_blank">
            ü§ñ How does a medicine work if people keep developing resistance?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-179', this)">
            More</button></h4>
            <div id="toggle-179" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Distal protein-protein interactions contribute to nirmatrelvir resistance</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract:

The main protein of the COVID-19 virus helps make more proteins by cutting up large pieces of genetic material. Scientists are looking at this protein because it's a target for medicines to fight the virus. However, some versions of the protein have developed resistance to these medicines.

We created a special version of the protein that can't be treated with these medicines, but still makes many copies of itself like normal. By studying how the protein works and interacts with its helpers, we found out that certain parts of the protein are important for it to function correctly. This helps us understand why some proteins become resistant to medicines and how we can make better treatments in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 main protease, Mpro, is responsible for processing the viral polyproteins into individual proteins, including the protease itself. Mprois a key target of anti-COVID-19 therapeutics such as nirmatrelvir (the active component of Paxlovid). Resistance mutants identified clinically and in viral passage assays contain a combination of active site mutations (e.g., E166V, E166A, L167F), which reduce inhibitor binding and enzymatic activity, and non-active site mutations (e.g., P252L, T21I, L50F), which restore the fitness of viral replication. To probe the role of the non-active site mutations in fitness rescue, here we use an Mprotriple mutant (L50F/E166A/L167F) that confers nirmatrelvir drug resistance with a viral fitness level similar to the wild-type. By comparing peptide and full-length Mproprotein as substrates, we demonstrate that the binding of Mprosubstrate involves more than residues in the active site. Particularly, L50F and other non-active site mutations can enhance the Mprodimer-dimer interactions and help place the nsp5-6 substrate at the enzyme catalytic center. The structural and enzymatic activity data of MproL50F, L50F/E166A/L167F, and others underscore the importance of considering the whole substrate protein in studying Mproand substrate interactions, and offers important insights into Mprofunction, resistance development, and inhibitor design.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56651-x" target="_blank">https://www.nature.com/articles/s41467-025-56651-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-31</strong> <a href="https://www.nature.com/articles/s41590-024-02064-9" target="_blank">
            ü§ñ Human immune system's ability to fight viruses discovered</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-180', this)">
            More</button></h4>
            <div id="toggle-180" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Temporal profiling of human lymphoid tissues reveals coordinated defense against viral challenge</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

When we get a cold or flu, our body's immune system tries to fight it off. But scientists don't know exactly how it works in our nose. They wanted to find out what happens when our nose gets infected with a virus.

They took a closer look at the tiny tissues in our nose and found that certain cells worked together to defend against the infection. In the early stages, some immune cells formed a shield around the viruses, while others helped repair damaged tissue. The study also showed that some immune cells kept fighting off the virus even after we got better.

This discovery helps us understand how our body's immune system works when it's fighting off infections in our nose.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Adaptive immunity is generated in lymphoid organs, but how these structures defend themselves during infection in humans is unknown. The nasal epithelium is a major site of viral entry, with adenoid nasal-associated lymphoid tissue (NALT) generating early adaptive responses. In the present study, using a nasopharyngeal biopsy technique, we investigated longitudinal immune responses in NALT after a viral challenge, using severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection as a natural experimental model. In acute infection, infiltrating monocytes formed a subepithelial and perifollicular shield, recruiting neutrophil extracellular trap-forming neutrophils, whereas tissue macrophages expressed pro-repair molecules during convalescence to promote the restoration of tissue integrity. Germinal center B cells expressed antiviral transcripts that inversely correlated with fate-defining transcription factors. Among T cells, tissue-resident memory CD8 T cells alone showed clonal expansion and maintained cytotoxic transcriptional programs into convalescence. Together, our study provides unique insights into how human nasal adaptive immune responses are generated and sustained in the face of viral challenge.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-02064-9" target="_blank">https://www.nature.com/articles/s41590-024-02064-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-31</strong> <a href="https://www.nature.com/articles/s41598-024-75891-3" target="_blank">
            ü§ñ Safety and Efficacy Study of Cepharanthine for Mild COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-181', this)">
            More</button></h4>
            <div id="toggle-181" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Safety and efficacy of oral administrated cepharanthine in non-hospitalized, asymptomatic or mild COVID-19 patients: a Double-blind, randomized, placebo-controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers tested a natural remedy called Cepharanthine on people with mild COVID-19 to see if it could help them recover faster. They gave some participants 120mg or 60mg of Cepharanthine every day for 5 days, while others got a placebo (a dummy medicine). The results showed that neither the 120mg nor the 60mg dose of Cepharanthine helped people recover faster than those who got a placebo. However, when only people who took their medication correctly were looked at, the 60mg dose did seem to help them recover slightly faster. The researchers also found no serious side effects from taking the medicine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cepharanthine (CEP) is a natural remedy that potently inhibits SARS-CoV-2 activity both in vitro and in vivo. To evaluate the efficacy and safety of CEP compared with placebo in adults with asymptomatic or mild coronavirus disease 2019 (COVID-19), we conducted a proof-of-concept, double-blind, randomized, placebo-controlled trial. Patients were randomized to receive 120¬†mg/day of CEP, 60¬†mg/day CEP or placebo for 5 days. Main outcome was the time from randomization to negative nasopharyngeal swab and safety. Among 262 randomized participants, 188 completed the trial among group of 120¬†mg/day CEP (n=‚Äâ65), 60¬†mg/day CEP (n=‚Äâ68) and placebo (n=‚Äâ55). Neither 120¬†mg/day or 60¬†mg/day CEP shortened the time to negative significantly compared with placebo. However, 60¬†mg/day CEP showed a slight trend (difference=-0.77 days, hazard ratio (HR)‚Äâ=‚Äâ1.40, 95% CI 0.97‚Äì2.01,p=‚Äâ0.072). In analysis of participants with good medication compliance, 60¬†mg/day CEP significantly shortened the time to negative (difference=-0.87 days, HR‚Äâ=‚Äâ1.56, 95% CI 1.03‚Äì2.37,p=‚Äâ0.035). Adverse events were not different among the three groups, and no serious adverse events occurred. In conclusion, treatment of asymptomatic or mild Covid-19 with 120¬†mg/day or 60¬†mg/day did not shorten the time to negative significantly. However, 60¬†mg/day CEP showed a slight trend which needs future confirmatory trials to validate. (NCT05398705).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75891-3" target="_blank">https://www.nature.com/articles/s41598-024-75891-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-31</strong> <a href="https://www.nature.com/articles/s41598-025-88405-6" target="_blank">
            ü§ñ Impact of COVID-19 on Pulmonary Patients' Mental Health and Recovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-182', this)">
            More</button></h4>
            <div id="toggle-182" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Posttraumatic stress and growth in pulmonary patients recovered from COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Many people who got very sick from COVID-19 had problems with their mental health afterwards. Some people even developed a condition called Post-Traumatic Stress Disorder (PTSD). Researchers looked at 62 people who were recovering from COVID-19 and found that about 40% of them had PTSD symptoms. They also discovered that certain things, like how well they thought and how healthy they felt after COVID-19, affected how bad their PTSD was. Interestingly, some people with more severe PTSD even experienced positive changes, like becoming stronger or finding new meaning in life. This study shows that COVID-19 can have a big impact on mental health, especially for people who already had lung problems. It's an important reminder that doctors need to take care of both the physical and emotional health of patients during this time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has profoundly affected mental health, with many survivors experiencing psychological challenges, including Post-Traumatic Stress Disorder (PTSD). This study assessed PTSD symptoms and Post-Traumatic Growth (PTG) among 62 individuals recovering from COVID-19 infection, all of whom were under the care of the Department of Pneumonology, Oncology, and Allergology at the Medical University of Lublin. Results revealed that 40.32% of participants exhibited PTSD symptoms. Key predictors of PTSD severity included cognitive symptoms and post-COVID self-rated health, with cognitive symptoms positively associated and self-rated health negatively associated with PTSD severity. A positive correlation was also found between PTSD severity and PTG, suggesting that while individuals endure significant psychological distress, they may also experience personal growth, such as enhanced resilience and a redefined life perspective. These findings highlight the dual psychological impact of COVID-19 infection, particularly for individuals with preexisting pulmonary conditions. They underscore the importance of holistic, integrated care that addresses both the reduction of PTSD symptoms and the promotion of meaningful psychological growth in COVID-19 survivors.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-88405-6" target="_blank">https://www.nature.com/articles/s41598-025-88405-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-30</strong> <a href="https://www.nature.com/articles/s41598-025-87146-w" target="_blank">
            ü§ñ How Dengue Virus Infection Can Be Stopped with a Special Medicine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-183', this)">
            More</button></h4>
            <div id="toggle-183" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Domperidone inhibits dengue virus infection by targeting the viral envelope protein and nonstructural protein 1</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

Dengue is a disease spread by mosquitoes that can be serious and hard to treat. A new study looked at if a medicine called domperidone could help control dengue. Domperidone is usually used to stop people from getting sick when they eat too much. The researchers found out that domperidone can bind to the dengue virus in special ways, which makes it harder for the virus to multiply. When tested on cells infected with dengue, domperidone stopped the virus from making more copies and released less of its bad proteins. This means domperidone could be a new way to treat or prevent dengue.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Dengue is a mosquito-borne disease caused by dengue virus (DENV) infection, which remains a major public health concern worldwide owing to the lack of specific treatments or antiviral drugs available. This study investigated the potential repurposing of domperidone, an antiemetic and gastrokinetic agent, to control DENV infection. Domperidone was identified by pharmacophore-based virtual screening as a small molecule that can bind to both the viral envelope (E) and the nonstructural protein 1 (NS1) of DENV. Molecular dynamics (MD) simulations and surface plasmon resonance (SPR) analysis were subsequently performed to determine specific interactions of domperidone with the DENV E and NS1 proteins and their binding affinity. Treatment of immortalized human hepatocyte-like cells (imHC) with domperidone could inhibit DENV production and NS1 secretion in a dose-dependent manner following infection with DENV serotype 2. These inhibitory effects were mediated by reduction in viral RNA replication and viral E and NS1 protein expression, but not by interference with virus entry into cells or NS1 oligomerization. The suppression of DENV production and NS1 secretion by domperidone was observed across all four DENV serotypes to varying degrees between different virus strains. The findings from our study suggest viral target-based repurposing of domperidone for modulating DENV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87146-w" target="_blank">https://www.nature.com/articles/s41598-025-87146-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-30</strong> <a href="https://www.nature.com/articles/s41597-025-04510-1" target="_blank">
            ü§ñ Long COVID Herbal Treatment Effects in Blood Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-184', this)">
            More</button></h4>
            <div id="toggle-184" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A single-cell RNA sequencing dataset of peripheral blood cells in long COVID patients on herbal therapy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

After the COVID-19 pandemic, some people are still experiencing problems with breathing and thinking too much. To help them feel better, some people are using special plant medicines called herbal remedies. But we don't really know if these medicines work well enough or if they're safe to use. We wanted to study what happens when people take these herbal remedies for long COVID. We looked at tiny cells in blood samples from 181,000 people who took different herbal medicines and compared the results before and after treatment. This helps us understand more about how herbal remedies might help with long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Following the coronavirus disease 2019 (COVID-19) pandemic, the rise of long COVID, characterized by persistent respiratory and cognitive dysfunctions, has become a significant health concern. This leads to an increased role of complementary and alternative medicine in addressing this condition. However, our comprehension of the effectiveness and safety of herbal medicines for long COVID remains limited. Here, we present a single-cell RNA sequencing (scRNA-seq) dataset of peripheral whole blood cells derived from participants in a clinical study involving three commercially available herbal medicines, targeting fatigue and brain fog in long COVID. The dataset comprises 181,205 quality control (QC)-passed cells, along with clinical metadata, enabling a comparative analysis of immune cell populations before and after treatment. To ensure the technical validity of our dataset, we implemented rigorous quality checks throughout stages of the study, including sample preparation, sequencing, and bioinformatic data analysis levels. This transcriptomic data may serve as a resource to deepen our insights into the role of herbal medicines in management of long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41597-025-04510-1" target="_blank">https://www.nature.com/articles/s41597-025-04510-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-30</strong> <a href="https://www.nature.com/articles/s41541-024-01051-3" target="_blank">
            ü§ñ Identifying fake COVID-19 Vaccines with Special Tool</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-185', this)">
            More</button></h4>
            <div id="toggle-185" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identifying falsified COVID-19 vaccines by analysing vaccine vial label and excipient profiles using MALDI-ToF mass spectrometry</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 Vaccines: A New Way to Check for Fakes

Imagine getting a vaccine that's been tampered with or made fake. This is a big problem because it can put people's lives at risk. Scientists have found a way to detect fake COVID-19 vaccines using a special machine called mass spectrometry.

They tested the COVISHIELD vaccine, which was found in some countries as a fake. They discovered that by looking at the labels on the vials and the glue used to stick them together, they could identify whether it was real or not. This method is quick, easy, and doesn't require opening the vaccine.

The scientists also tested if this method worked when different people tried to make fake vaccines. It did! They found that even if a fake vaccine had the same ingredients as a real one, it didn't have the same special patterns in its labels and glue. This means they can tell the difference between a real and fake vaccine.

This new way of checking vaccines could help stop fake ones from getting into people's hands and keep everyone safe.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid development and worldwide distribution of COVID-19 vaccines is a remarkable achievement of biomedical research and logistical implementation. However, these developments are associated with the risk of a surge of substandard and falsified (SF) vaccines, as illustrated by the 184 incidents with SF and diverted COVID-19 vaccines which have been reported during the pandemic in 48 countries, with a paucity of methods for their detection in supply chains. In this context, matrix-assisted laser desorption ionisation-time of flight (MALDI-ToF) mass spectrometry (MS) is globally available for fast and accurate analysis of bacteria in patient samples, offering a potentially accessible solution to identify SF vaccines. We analysed the COVISHIELD‚Ñ¢ COVID-19 vaccine; falsified versions of which were found in India, Myanmar and Uganda. We demonstrate for the first time that analysis of spectra from the vaccine vial label and its adhesive could be used as a novel approach to detect falsified vaccines. Vials tested by this approach could be retained in the supply chain since it is non-invasive. We also assessed whether MALDI-ToF MS could be used to distinguish the COVISHIELD‚Ñ¢ vaccine from surrogates of falsified vaccines and the effect of temperature on vaccine stability. Both polysorbate 80 and L-histidine excipients of the genuine vaccine could be detected by the presence of a unique combination of MALDI-ToF MS peaks which allowed us to distinguish between the genuine vaccines and falsified vaccine surrogates. Furthermore, even if a falsified product contained polysorbate 80 at the same concentration as used in the genuine vaccine, the characteristic spectral profile of polysorbate 80 used in genuine products is a reliable internal marker for vaccine authenticity. Our findings demonstrate that MALDI-ToF MS analysis of extracts from vial labels and the vaccine excipients themselves can be used independently to detect falsified vaccines. This approach has the potential to be integrated into the national regulatory standards and WHO‚Äôs Prevent, Detect, and Respond strategy as a novel effective tool for detecting falsified vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01051-3" target="_blank">https://www.nature.com/articles/s41541-024-01051-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-30</strong> <a href="https://www.nature.com/articles/s41467-025-56516-3" target="_blank">
            ü§ñ How to use a special tool to detect genetic material quickly and accurately.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-186', this)">
            More</button></h4>
            <div id="toggle-186" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Tunable control of Cas12 activity promotes universal and fast one-pot nucleic acid detection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We created a new way to detect epidemic pathogens quickly and easily. Our method uses a special molecule called Cas12 and a chemical called heparin sodium to make it work. This method can detect many different types of viruses and bacteria, including those that cause monkeypox, flu, and COVID-19, in just 15-20 minutes. It's also very good at telling the difference between real and fake signals, with over 95% accuracy. The best part is that it uses a cheap chemical called heparin sodium, which costs only $0.01 to $0.04 per thousand uses. This makes it a promising tool for testing viruses and bacteria in clinics or even at home.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The CRISPR-based detection methods have been widely applied, yet they remain limited by the non-universal nature of one-pot diagnostic approaches. Here, we report a universal one-pot fluorescent method for the detection of epidemic pathogens, delivering results within 15-20‚Äâmin. This method uses heparin sodium to precisely tunes thecis-cleavage capability of Cas12 via interference with the Cas12a-crRNA binding process, thereby generating significant fluorescence due to the accumulation of isothermal amplification products. Additionally, this universal assay accommodates both classic and suboptimal PAMs, as well as various Cas12a subtypes such as LbCas12a, AsCas12a, and AapCas12b. Such a robust method demonstrates sensitivity and specificity exceeding 95% in the detection of monkeypox pseudovirus, influenza A virus, and SARS-CoV-2 from saliva or wastewater samples, when compared with qPCR or RT-qPCR. Moreover, the cost of heparin sodium per thousand uses is $0.01 to $0.04 only. Collectively, this universal and fast one-pot approach based on heparin sodium offers potential possibilities for point-of-care testing.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-56516-3" target="_blank">https://www.nature.com/articles/s41467-025-56516-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-30</strong> <a href="https://www.nature.com/articles/s41591-024-03426-4" target="_blank">
            ü§ñ SARS-CoV-2 Infection Linked to Higher Amyloid Protein in Blood</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-187', this)">
            More</button></h4>
            <div id="toggle-187" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Plasma proteomic evidence for increased Œ≤-amyloid pathology after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

People have found out that getting very sick with certain viruses might make you more likely to get dementia later in life. We want to know if this is true for COVID-19, which causes severe respiratory syndrome. To figure it out, we looked at what's happening inside people's bodies and brains after they had a COVID-19 infection. What we found was that some people who got sick with COVID-19 had more of a protein called beta-amyloid in their blood, which is linked to Alzheimer's disease. This protein change might be associated with worse brain health and more cognitive problems later on. Our study didn't prove that COVID-19 directly causes dementia, but it suggests that getting very sick with it might increase the risk.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Previous studies have suggested that systemic viral infections may increase risks of dementia. Whether this holds true for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus infections is unknown. Determining this is important for anticipating the potential future incidence of dementia. To begin to do this, we measured plasma biomarkers linked to Alzheimer‚Äôs disease pathology in the UK Biobank before and after serology-confirmed SARS-CoV-2 infections. SARS-CoV-2 infection was associated with biomarkers associated with Œ≤-amyloid pathology: reduced plasma AŒ≤42:AŒ≤40 ratio and, in more vulnerable participants, lower plasma AŒ≤42 and higher plasma pTau-181. The plasma biomarker changes were greater in participants who had been hospitalized with COVID-19 or had reported hypertension previously. We showed that the changes in biomarkers were linked to brain structural imaging patterns associated with Alzheimer‚Äôs disease, lower cognitive test scores and poorer overall health evaluations. Our data from this post hoc case‚Äìcontrol matched study thus provide observational biomarker evidence that SARS-CoV-2 infection can be associated with greater brain Œ≤-amyloid pathology in older adults. While these results do not establish causality, they suggest that SARS-CoV-2 (and possibly other systemic inflammatory diseases) may increase the risk of future Alzheimer‚Äôs disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-03426-4" target="_blank">https://www.nature.com/articles/s41591-024-03426-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-29</strong> <a href="https://www.nature.com/articles/s41586-024-08477-8" target="_blank">
            ü§ñ Evolution of COVID-19 virus on changing immune system</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-188', this)">
            More</button></h4>
            <div id="toggle-188" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 evolution on a dynamic immune landscape</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 virus has changed over time to make it harder for our bodies to fight off. It does this because our immune system creates antibodies that can attack the virus, but the virus is smart and changes its shape so we don't have enough good antibodies to stop it.

We created a model to understand how the virus evolves and which people are most likely to get infected by each type of virus. We used data from all over the world to make our predictions and they matched what actually happened in the past.

Our model shows that the virus can spread more easily because some people have immunity from previous infections or vaccines, and it helps us understand how to make better vaccines and predict which viruses are most likely to appear in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Since the onset of the pandemic, many SARS-CoV-2 variants have emerged, exhibiting substantial evolution in the virus‚Äô spike protein1, the main target of neutralizing antibodies2. A plausible hypothesis proposes that the virus evolves to evade antibody-mediated neutralization (vaccine- or infection-induced) to maximize its ability to infect an immunologically experienced population1,3. Because viral infection induces neutralizing antibodies, viral evolution may thus navigate on a dynamic immune landscape that is shaped by local infection history. Here we developed a comprehensive mechanistic model, incorporating deep mutational scanning data4,5, antibody pharmacokinetics and regional genomic surveillance data, to predict the variant-specific relative number of susceptible individuals over time. We show that this quantity precisely matched historical variant dynamics, predicted future variant dynamics and explained global differences in variant dynamics. Our work strongly suggests that the ongoing pandemic continues to shape variant-specific population immunity, which determines a variant‚Äôs ability to transmit, thus defining variant fitness. The model can be applied to any region by utilizing local genomic surveillance data, allows contextualizing risk assessment of variants and provides information for vaccine design.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-08477-8" target="_blank">https://www.nature.com/articles/s41586-024-08477-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-29</strong> <a href="https://www.nature.com/articles/s41586-024-08471-0" target="_blank">
            ü§ñ How do bats deal with viruses and stay healthy?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-189', this)">
            More</button></h4>
            <div id="toggle-189" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bat genomes illuminate adaptations to viral tolerance and disease resistance</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Animals can get diseases from each other, and sometimes those diseases come from animals. Bats are special because they might carry more diseases than any other animal. The Bat1K project looked at 10 types of bats to see how their bodies fight off diseases.

They found that some parts of the bats' immune system have extra adaptations that help them fight off viruses better than other animals. This is interesting because it helps us understand why bats are less likely to get sick from viruses.

The researchers also studied a specific gene called ISG15, which helps our bodies fight off viruses in humans. They found out that this gene works differently in bats, and actually has special powers against certain viruses like SARS-CoV-2. This is important because it could help us understand how to make vaccines or treatments for those viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Zoonoses are infectious diseases transmitted from animals to humans. Bats have been suggested to harbour more zoonotic viruses than any other mammalian order1. Infections in bats are largely asymptomatic2,3, indicating limited tissue-damaging inflammation and immunopathology. To investigate the genomic basis of disease resistance, the Bat1K project generated reference-quality genomes of ten bat species, including potential viral reservoirs. Here we describe a systematic analysis covering 115 mammalian genomes that revealed that signatures of selection in immune genes are more prevalent in bats than in other mammalian orders. We found an excess of immune gene adaptations in the ancestral chiropteran branch and in many descending bat lineages, highlighting viral entry and detection factors, and regulators of antiviral and inflammatory responses.ISG15, which is an antiviral gene contributing to hyperinflammation during COVID-19 (refs.4,5), exhibits key residue changes in rhinolophid and hipposiderid bats. Cellular infection experiments show species-specific antiviral differences and an essential role of protein conjugation in antiviral function of batISG15, separate from its role in secretion and inflammation in humans. Furthermore, in contrast to humans,ISG15in most rhinolophid and hipposiderid bats has strong anti-SARS-CoV-2 activity. Our work reveals molecular mechanisms that contribute to viral tolerance and disease resistance in bats.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-08471-0" target="_blank">https://www.nature.com/articles/s41586-024-08471-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-28</strong> <a href="https://www.nature.com/articles/s41590-024-02072-9" target="_blank">
            ü§ñ Mucosal immune cells fight COVID-19 better than expected</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-190', this)">
            More</button></h4>
            <div id="toggle-190" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Robust mucosal SARS-CoV-2-specific T cells effectively combat COVID-19 and establish polyfunctional resident memory in patient lungs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract that explains it simply:

When we get sick with a coronavirus like COVID-19, our immune system tries to fight the virus. But sometimes this can go wrong and make us sicker. Scientists wanted to know how our immune system is fighting the virus in the lungs.

They looked at cells in the lungs and blood of people who had COVID-19. They found that these special immune cells are good at keeping us safe from getting very sick. These cells work best when they're in the lungs, not just in the blood. After we get better, these cells stay in our lungs to keep an eye out for any future infections and help protect us.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Mucosal antigen-specific T cells are pivotal for pathogen clearance and immune modulation in respiratory infections. Dysregulated T cell responses exacerbate coronavirus disease 2019 severity, marked by cytokine storms and respiratory failure. Despite extensive description in peripheral blood, the characteristics of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)-specific T cells in the lungs remain elusive. Here we conducted integrated single-cell profiling of SARS-CoV-2-specific T cells in 122 bronchoalveolar lavage fluid (BALF) and 280 blood samples from 159 patients, including 27 paired BALF and blood samples from 24 patients. SARS-CoV-2-specific T cells were robustly elicited in BALF irrespective of prior vaccination, correlating with diminished viral loads, lessened systemic inflammation and improved respiratory function. SARS-CoV-2-specific T cells in BALF exhibited profound activation, along with proliferative and multi-cytokine-producing capabilities and a glycolysis-driven metabolic signature, which were distinct from those observed in peripheral blood mononuclear cells. After viral clearance, these specific T cells maintained a polyfunctional tissue-resident memory phenotype, highlighting their critical roles in infection control and long-term protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-02072-9" target="_blank">https://www.nature.com/articles/s41590-024-02072-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-28</strong> <a href="https://www.nature.com/articles/s41598-025-86242-1" target="_blank">
            ü§ñ How a coronavirus causes blood vessel damage</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-191', this)">
            More</button></h4>
            <div id="toggle-191" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 S-protein expression drives syncytia formation in endothelial cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract written in simple language:

A new virus called SARS-CoV-2 can infect two main types of cells: skin cells and blood vessel cells. When these cells get infected, they start to fuse together into bigger cells that don't work properly. This can lead to problems with the way the body's barrier works. Researchers studied how this happens in blood vessel cells and found that certain signals in the virus make it happen. They also tested different ways of stopping it from happening and found that some things, like stretching the cells or keeping them stiff, helped prevent it. The study shows that blood vessel cells are not just bystanders when it comes to SARS-CoV-2, but are actually an important part of how the virus causes problems in people who get infected with COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 is a viral infection, best studied in the context of epithelial cell infection. Epithelial cells, when infected with SARS-CoV-2 express the viral S-protein, which causes host cells to fuse together into large multi-nucleated cells known as syncytia. Because SARS-CoV-2 infections also frequently present with cardiovascular phenotypes, we sought to understand if S-protein expression would also result in syncytia formation in endothelial cells. S-protein expression in endothelial cells was sufficient to induce the formation of multi-nucleated cells, with an average of 10% of all cells forming syncytia with an average of 6 nuclei per syncytia after 72¬†h of S-protein expression. Formation of syncytia was associated with the formation of gaps between cells, suggesting the potential for syncytia formation to compromise barrier function. Inhibition of myosin light chain kinase (MLCK), but not Rho-associated protein kinase, inhibited the formation of syncytia, suggesting a role for MLCK in syncytia formation. Further supporting the role of cellular contractility in syncytia formation, we also observed a reduction in the occurrence of syncytia for endothelial cells grown on substrates with reduced stiffness. Because endothelial cells are exposed to physiological forces due to blood flow, we examined the effects of cyclic biaxial stretch and fluid shear stress. While biaxial stretch did not affect syncytia formation, endothelial cells exposed to fluid shear stress were more resistant to syncytia formation. Finally, we observed that endothelial cells are suitable host cells for SARS-CoV-2 viral infection and replication, and that viral infection also causes syncytia formation. Our studies indicate that endothelial cells, in addition to epithelial cells, should also be considered a target for SARS-CoV-2 infection and a driver of COVID-19-associated pathology.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86242-1" target="_blank">https://www.nature.com/articles/s41598-025-86242-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-28</strong> <a href="https://www.nature.com/articles/s41598-025-87748-4" target="_blank">
            ü§ñ Abnormal Heart Rate Linked to Brain Blood Flow in Rare Condition</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-192', this)">
            More</button></h4>
            <div id="toggle-192" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Novel brain SPECT imaging unravels abnormal cerebral perfusion in patients with postural orthostatic tachycardia syndrome and cognitive dysfunction</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

People with postural orthostatic tachycardia syndrome (POTS) often have problems with their brain function. They might not get enough blood flow to the brain when they stand up.

Researchers looked at 56 people with POTS who had brain scans to see if there was less blood flowing to the brain. They also asked these people how they felt and what symptoms they were experiencing.

Most of the people with POTS had big problems with their autonomic nervous system (which controls things like heart rate and digestion) and many had stomach problems too. The brain scans showed that most people had reduced blood flow to certain parts of the brain, including areas that control thinking and movement.

This suggests that the reduced blood flow is making these people feel worse and having a big impact on their quality of life.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cognitive dysfunction is frequently reported in individuals with postural orthostatic tachycardia syndrome (POTS), possibly resulting from reduced cerebral blood flow (CBF). We used brain SPECT, an accessible imaging modality that has not been systematically evaluated in this patient group. Retrospective review of participants from our registry was undertaken to identify those who had a brain SPECT performed for investigation of cognitive dysfunction. Abnormal CBF was taken as z-score‚Äâ>‚Äâ2 standard deviations of healthy control reference values. Patient reported outcome measures (PROMs) such as autonomic, gastric and quality of life symptom scores were analyzed. From a total of 56 participants (mean 34.8‚Äâ¬±‚Äâ10.7 years, 88% females), PROMs indicate: moderate to severe autonomic dysfunction in 75%; at least mild to moderate gastroparesis in 23%; low global health rating and utility scores. Abnormal CBF was seen in 61% but did not differ by POTS triggers. The regions with the lowest mean z-scores were the lateral prefrontal and sensorimotor cortices. Hierarchal regression analyses found number of brain regions with abnormal CBF, autonomic and gastric symptoms to account for 51% of variances in health utility. Cerebral hypoperfusion is prevalent in those with POTS and cognitive dysfunction even whilst supine, contributing to reduced quality of life.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87748-4" target="_blank">https://www.nature.com/articles/s41598-025-87748-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-27</strong> <a href="https://www.nature.com/articles/s41392-025-02139-5" target="_blank">
            ü§ñ Boosters Made from Genetically Engineered Virus Can Protect Against Specific COVID Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-193', this)">
            More</button></h4>
            <div id="toggle-193" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Recombinant XBB.1.5 boosters induce robust neutralization against KP.2- and KP.3-included JN.1 sublineages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>New Variants of COVID-19 Make Some Vaccines Less Effective

Scientists studied how different variants of a COVID-19 virus affect vaccines. They found that some new variants can evade the body's immune response, even if someone has been vaccinated.

Researchers looked at blood samples from people who had gotten infected with different types of COVID-19 and received different vaccinations. They tested these samples to see how well they could fight off new variants.

The study showed that:

* People who got booster shots did better against the new variants.
* However, some new variants were much harder to fight off than others.
* Boosters are still needed for protection against JN.1 and similar sublineages.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The newly emerged variants of severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) demonstrate resistance to present therapeutic antibodies as well as the capability to evade vaccination-elicited antibodies. JN.1 sublineages were demonstrated as one of the most immune-evasive variants, showing higher neutralization resistance compared to XBB.1.5. In this study, serum samples were collected from adult participants including those who had gone through the BA.5/BF.7, EG.5/HK.3 and XBB/JN.1 infection waves, characterized by different infection and vaccination histories. We evaluated the neutralization in these serum samples against pseudoviruses of Omicron lineages. We further investigated humoral immune response of recombinant XBB vaccines against Omicron variants and estimated the neutralization resistance of JN.1 sublineages, including KP.2 and KP.3. Our results showed that sera from previous circulating Omicron subvariant breakthrough infections exhibited low neutralization against pseudoviruses of Omicron lineages. The GMTs of 50% neutralization against all tested pseudoviruses were significantly elevated in sera from individuals who received WSK-V102C or WSK-V102D boosters. Importantly, the GMTs of 50% neutralization in serum samples from individuals 4 months after a WSK-V102D booster against XBB.1.5, JN.1, JN.1.13, KP.2 and KP.3 pseudoviruses were 3479, 1684, 1397, 1247 and 1298, with 9.86-, 9.79-, 8.73-, 8.66- and 8.16-fold increase compared to those without booster, respectively, indicating that boosting with XBB.1.5 subunit vaccines still induced strong antibody responses against JN.1 sublineages. However, JN.1 sublineages, including KP.2 and KP.3, revealed more than 2-fold decreases in neutralizing antibody titers compared to XBB.1.5, suggesting significantly enhanced neutralization evasion and the necessity of boosters based on JN.1, KP.2 or KP.3.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-025-02139-5" target="_blank">https://www.nature.com/articles/s41392-025-02139-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-27</strong> <a href="https://www.nature.com/articles/s41598-025-86685-6" target="_blank">
            ü§ñ Motivation of Healthcare Workers During COVID-19 Pandemic in Uganda</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-194', this)">
            More</button></h4>
            <div id="toggle-194" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Determinants maintaining healthcare personnel‚Äôs motivation during COVID-19 pandemic in Uganda</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

The COVID-19 pandemic was very hard on healthcare workers in Uganda. They faced many challenges like fear of getting sick, losing income, and feeling alone. But even with these tough times, most healthcare workers stayed motivated to do their jobs.

To figure out why some healthcare workers were more motivated than others, researchers looked at data from 120 healthcare workers at a hospital. They found that younger workers, those who got extra help from their bosses, and those who felt supported by mentors were more likely to stay motivated.

The study also showed that feeling lonely or isolated was bad for mental health during the pandemic. This means that having good support systems is very important for keeping healthcare workers happy and healthy.

Overall, the researchers think that hospitals should focus on giving healthcare workers technical help, like training and resources, as well as emotional support, like talking to their bosses and getting help from mentors. This can help keep healthcare workers motivated even in hard times.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundMaintaining the physical and psychological well-being of healthcare workers (HCWs) is crucial for health system resilience. In sub-Saharan Africa, particularly Uganda, HCWs faced significant challenges during the coronavirus disease 2019 (COVID-19) pandemic, compounded by pre-existing resource constraints. This study investigated challenges faced by HCWs at a designated COVID-19 hospital (‚Äòthe Hospital‚Äô) and explored determinants of maintaining healthcare personnel‚Äôs motivation during the COVID-19 pandemic in Uganda.MethodsA facility-based cross-sectional study was conducted at the Hospital from May to July 2023, with 120 HCWs categorised by profession. Data were collected using a structured questionnaire. Descriptive and logistic regression analyses were performed to identify motivation-related factors. The questionnaire assessed sociodemographic data, COVID-19 knowledge, medical history, support received, and pandemic-related challenges. Psychological impact was measured using a four-point Likert scale.ResultsAmong respondents, 61.7% worked during the first lockdown and 93.3% during the second. COVID-19 knowledge was high, with over 70% answering questions correctly, although vaccine effectiveness had a slightly lower accuracy rate. The study revealed that 32.5% of HCWs tested positive for COVID-19 and 25% displayed vaccine hesitancy. Key challenges included fear of infection, transportation disruptions, income reduction, and social isolation. Notably, 92.5% reported significant anxiety related to infection. Factors positively associated with remaining motivated were age (‚â§‚Äâ30 versus >‚Äâ30 years; adjusted odds ratio [AOR]: 10.34, 95% CI; 1.92‚Äì55.74), profession (co-medical/non-medical staff versus medical doctors/medical officers; AOR: 11.66, 95% CI: 1.03‚Äì132.22), receiving mentoring/tutoring (AOR: 18.87, 95% CI: 2.55‚Äì139.72), and information from supervisors/management (AOR: 12.5, 95% CI: 2.60‚Äì60.42). In contrast, psychological impact was negatively affected by isolation from family and friends (AOR: 0.61, 95% CI: 0.41‚Äì0.92).ConclusionsDespite challenges, most HCWs at the Hospital remained motivated. These findings emphasised the importance of technical and psychological support, such as mentoring and effective communication from upper-level supervisors, rather than financial incentives, in maintaining motivation among the HCWs. Comprehensive support systems are essential for sustaining HCWs‚Äô motivation during the pandemic, especially in resource-limited settings. Further research should address the long-term mental health effects and enhance HCWs‚Äô resilience in future health crises.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86685-6" target="_blank">https://www.nature.com/articles/s41598-025-86685-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-27</strong> <a href="https://www.nature.com/articles/s41598-024-80833-0" target="_blank">
            ü§ñ COVID-19 Affects Brain Health More Than Expected</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-195', this)">
            More</button></h4>
            <div id="toggle-195" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Patients with neurological or psychiatric complications of COVID-19 have worse long-term functional outcomes: COVID-CNS‚ÄîA multicentre case‚Äìcontrol study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Many people who get seriously sick from COVID-19 often have problems that make it hard for them to take care of themselves, go to work, or deal with their mental health. We looked at two groups of people: those who got very sick from COVID-19 and had some kind of problem in their brain or mind (like depression), and those who just got a regular cold-like illness. 

In the group that was really sick, more women, older people, and those with certain medical conditions were having trouble with everyday things like taking care of themselves or going to work. However, these same groups didn't seem to have as much trouble with feeling sad or worried.

Taking some special medicines called statins or ACE inhibitors actually helped some of the people who were really sick from COVID-19 not have as many problems with their daily lives.</p><br>
                <p><strong>Abstract</strong></p>
                <p>It is established that patients hospitalised with COVID-19 often have ongoing morbidity affecting activity of daily living (ADL), employment, and mental health. However, little is known about the relative outcomes in patients with COVID-19 neurological or psychiatric complications. We conducted a UK multicentre case‚Äìcontrol study of patients hospitalised with COVID-19 (controls) and those who developed COVID-19 associated acute neurological or psychiatric complications (cases). Among the 651 patients, [362 (55%) cases and 289 (45%) controls], a higher proportion of cases had impairment in ADLs (199 [68.9%] vs 101 [51.8%], OR 2.06, p‚Äâ<‚Äâ0.0002) and reported symptoms impacting employment (159 [58.2%] vs 69 [35.6%] OR 2.53, p‚Äâ<‚Äâ0.0001). There was no significant difference in the proportion with depression or anxiety between case and control groups overall. For cases, impairment of ADLs was associated with increased risk in female sex, age‚Äâ>‚Äâ50¬†years and hypertension (OR 5.43, p‚Äâ<‚Äâ0.003, 3.11, p‚Äâ=‚Äâ0.02, 3.66, p‚Äâ=‚Äâ0.04). Those receiving either statins or angiotensin converting enzyme¬†(ACE) inhibitors had a lower risk of impairment in ADLs (OR 0.09, p‚Äâ=‚Äâ0.0006, 0.17, p‚Äâ=‚Äâ0.03). Patients with neurological or psychiatric complications of COVID-19 had worse functional outcomes than those with respiratory COVID-19 alone in terms of ADLs and employment. Female sex, age‚Äâ>‚Äâ50¬†years, and hypertension were associated with worse outcomes, and statins or ACE inhibitors with better outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80833-0" target="_blank">https://www.nature.com/articles/s41598-024-80833-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-26</strong> <a href="https://www.nature.com/articles/s41598-025-87993-7" target="_blank">
            ü§ñ Detection System for Monkeypox from Medical Images</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-196', this)">
            More</button></h4>
            <div id="toggle-196" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Capsule network approach for monkeypox (CAPSMON) detection and subclassification in medical imaging system</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

We created a new computer model called ESACN to help doctors diagnose diseases from skin images. The model can look at pictures of different skin conditions, like monkeypox, chickenpox, measles, and normal skin, and tell them apart. Our model works better than other models because it can see patterns in the images that others can't. We tested the model with 659 images and it got better results than before. This means our model could help doctors make more accurate diagnoses and treat patients faster.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In response to the pressing need for the detection of Monkeypox caused by the Monkeypox virus (MPXV), this study introduces the Enhanced Spatial-Awareness Capsule Network (ESACN), a Capsule Network architecture designed for the precise multi-class classification of dermatological images. Addressing the shortcomings of traditional Machine Learning and Deep Learning models, our ESACN model utilizes the dynamic routing and spatial hierarchy capabilities of CapsNets to differentiate complex patterns such as those seen in monkeypox, chickenpox, measles, and normal skin presentations. CapsNets‚Äô inherent ability to recognize and process crucial spatial relationships within images outperforms conventional CNNs, particularly in tasks that require the distinction of visually similar classes. Our model‚Äôs superior performance, demonstrated through rigorous evaluation, exhibits significant improvements in accuracy, precision, recall, and F1 score, even with limited data. The results highlight the potential of ESACN as a reliable tool for enhancing diagnostic accuracy in medical settings. In our case study, the ESACN model was applied to a dataset comprising 659 images across four classes: 178 images of Monkeypox, 171 of Chickenpox, 80 of Measles, and 230 of Normal skin conditions. This case study underscores the model‚Äôs effectiveness in real-world applications, providing robust and accurate classification that could greatly aid in early diagnosis and treatment planning in clinical environments.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87993-7" target="_blank">https://www.nature.com/articles/s41598-025-87993-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-25</strong> <a href="https://www.nature.com/articles/s44298-024-00085-7" target="_blank">
            ü§ñ Spike protein modifications in coronaviruses matter for the virus's survival and transmission.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-197', this)">
            More</button></h4>
            <div id="toggle-197" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Conserved role of spike S2 domain N-glycosylation acrossbetacoronaviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The outer layer of the SARS-CoV-2 virus is made up of special sugar molecules called N-glycans. These sugar molecules help protect us from getting sick, but they also play a crucial role in how the virus works.

Scientists looked at how removing certain parts of these sugar molecules affected the way the virus behaves. They found that if you remove too much of the sugar molecules, the virus can't attach to cells or enter them properly.

This study shows that the sugar molecules are important for the virus's life cycle and helps us understand why some viruses can spread more easily than others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Besides acting as an immunological shield, the N-glycans of SARS-CoV-2 are also critical for viral life cycle. As the S2 subunit of spike is highly conserved across betacoronaviruses, we determined the functional significance of the five ‚Äòstem N-glycans‚Äô located in S2 between N1098-N1194. Studies were performed with 31 Asn-to-Gln mutants, betacoronavirus virus-like particles and single-cycle viral replicons. Deletions of stem N-glycans enhanced S1 shedding from trimeric spike, reduced ACE2 binding and abolished syncytia formation. When three or more N-glycans were deleted, spike expression on cell surface and incorporation into virions was both reduced. Viral entry function was progressively lost upon deleting the N1098 glycan in combination with additional glycosite modifications. In addition to SARS-CoV-2, deleting stem N-glycans in SARS-CoV and MERS-CoV spike also prevented viral entry into target cells. These data suggest multiple functional roles for the stem N-glycans, and evolutionarily conserved properties for these complex carbohydrates across human betacoronaviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00085-7" target="_blank">https://www.nature.com/articles/s44298-024-00085-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-24</strong> <a href="https://www.nature.com/articles/s41598-024-80538-4" target="_blank">
            ü§ñ Predicting Hospital Admissions for Children with COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-198', this)">
            More</button></h4>
            <div id="toggle-198" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Utilizing machine learning to predict hospital admissions for pediatric COVID-19 patients (PrepCOVID-Machine)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

During the COVID-19 pandemic, hospitals were overwhelmed with patients. To make sure only those who really need help are admitted, scientists wanted to create machines that can predict which kids might get very sick from COVID-19. They collected data from 2200 kids in Malaysia who had COVID-19 and looked at how many variables (like age, fever, and breathing problems) were important for predicting sickness. They used a machine learning tool called Recursive Feature Elimination to find the most important variables. The scientists found that 12 of these variables were very good at predicting which kids would get sick enough to need hospital care. One machine learning model, called Adaptive Boosting, was the best at doing this job. This model can help doctors make better decisions about who needs medical attention.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has burdened healthcare systems globally. To curb high hospital admission rates, only patients with genuine medical needs are admitted. However, machine learning (ML) models to predict COVID-19 hospitalization in Asian children are lacking. This study aimed to develop and validate ML models to predict pediatric COVID-19 hospitalization. We collected secondary data with 2200 patients and 65 variables from Malaysian aged 0 to 12 with COVID-19 between 1st February 2020 and 31st March 2022. The sample was partitioned into training, internal, and external validation groups. Recursive Feature Elimination (RFE) was employed for feature selection, and we trained seven supervised classifiers. Grid Search was used to optimize the hyperparameters of each algorithm. The study analyzed 1988 children and 30 study variables after data were processed. The RFE algorithm selected 12 highly predicted variables for COVID-19 hospitalization, including age, male sex, fever, cough, rhinorrhea, shortness of breath, vomiting, diarrhea, seizures, body temperature, chest indrawing, and abnormal breath sounds. With external validation, Adaptive Boosting was the highest-performing classifier (AUROC‚Äâ=‚Äâ0.95) to predict COVID-19 hospital admission in children. We validated AdaBoost as the best to predict COVID-19 hospitalization among children. This model may assist front-line clinicians in making medical disposition decisions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80538-4" target="_blank">https://www.nature.com/articles/s41598-024-80538-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-24</strong> <a href="https://www.nature.com/articles/s41598-024-84379-z" target="_blank">
            ü§ñ Impact of COVID-19 on Older Adults During a Pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-199', this)">
            More</button></h4>
            <div id="toggle-199" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Consequences of COVID-19 for geriatric patients during a pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study looked at 468 older adults (70+ years) who got COVID-19 in Germany between February 2020 and March 2021. The study found that half of these patients died, but if they received special care for older adults, they lived longer (74.3% vs 51.8%). The main problems were staying healthy, moving around, and doing daily activities. It seems that better care for older adults with COVID-19 can save lives, but might not improve their quality of life in the long run. More research is needed to understand what works best for this group.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the outcomes of geriatric COVID-19 patients in a German academic setting during the pandemic. This study included 468 consecutive geriatric patients (‚â•‚Äâ70 years) who tested positive for SARS-CoV-2 and were treated at the University of Duisburg-Essen from 2/2020 to 3/2021. 74 patients were transferred to a geriatric hospital and a 12-month follow-up (prospective study) was performed in 51 patients. Clinical assessments evaluated depression (GDS), apathy (AES), cognitive status (MMST), mobility (TUG), health status (EQ-5D-5¬†L), and daily living activities (Barthel Index). Demographic and clinical data were also analyzed. Results showed that the mortality in this vulnerable group was 52% (n=‚Äâ209). Long-term survival was higher in patients who received comprehensive geriatric treatment (74.3% vs. 51.8%). The duration of inpatient stay at the primary hospital was 13.3‚Äâ¬±‚Äâ3.6 days, with 28.8% (n=‚Äâ135) requiring intensive care. At the 12-month mark more patients with geriatric treatment lived in nursing homes. Barthel-Index/Timed-Up-and-Go-Test/MMST/AES/GDS, and EQ-5D-5¬†L indicated worse outcomes in the group who received geriatric treatment. Specialized geriatric care may improve survival in geriatric COVID-19 patients despite decreased long-term outcomes. Further research, including international studies like NAPKON, are encouraged to confirm these findings and explore potential interventions for improved outcomes in this vulnerable population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84379-z" target="_blank">https://www.nature.com/articles/s41598-024-84379-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-24</strong> <a href="https://www.nature.com/articles/s41598-025-86846-7" target="_blank">
            ü§ñ COVID-19 Prognosis Biomarker Found in Throat Mucus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-200', this)">
            More</button></h4>
            <div id="toggle-200" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>KLRB1 expression in nasopharyngeal mucosa as a prognostic biomarker in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the rewritten abstract:

Severe cases of COVID-19 are happening again, which means we need better ways to predict who might get very sick or die from it. This study looked at swabs taken from people's noses (nasopharyngeal swabs) to see if they could find any clues about how well someone would do. They tested 95 patients in 2020 and found that two things, called genes, were important for predicting who might get very sick or die - one of those genes is called KLRB1. When it comes down to it, having low levels of this gene or being older are the most important factors. This means we could use a special test that checks for both these things to figure out if someone is at risk before they even go to the hospital. It would make doctors' jobs easier and help them take better care of patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The resurgence of COVID-19 and the rise in severe outcomes emphasize the need for reliable prognostic markers to guide patient care and optimize ICU and hospital resources. This study investigates the potential of nasopharyngeal swabs to identify biomarkers that predict ICU admission or death in hospitalized COVID-19 patients. We analyzed nasopharyngeal exudates from 95 hospitalized patients in 2020 using high-plex RNA quantification on the NanoString¬ÆnCounter platform. Comparative analysis identified four genes, withKLRB1(Killer cell lectin like receptor B1) (Odds Ratio OR 0.5, 95% CI: 0.27‚Äì0.96), along with age (OR 3.3, 95% CI: 1.25‚Äì8.93) emerging as independent prognostic markers in multivariate analysis. These findings were validated using qRT-PCR in an independent cohort of 168 patients hospitalized in 2022. While univariate analysis identified a significant association betweenKLRB1expression and vaccination status (p<‚Äâ0.05), only lowKLRB1expression (OR 1.135, 95% CI: 1.0-1.280), and age (OR 1.033, 95% CI: 1.006‚Äì1.061) were confirmed as independent risk factors for ICU admission or death, regardless of other studied variables such as comorbidities, vaccination status, or smoking habits. Our findings suggest thatKLRB1expression could improve prognostic tools by identifying patients at higher risk upon admission. IncorporatingKLRB1into multiplex diagnostic kits alongside SARS-CoV-2 detection could streamline prognostic assessment, providing a more comprehensive and efficient approach to patient management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86846-7" target="_blank">https://www.nature.com/articles/s41598-025-86846-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-23</strong> <a href="https://www.nature.com/articles/s41598-024-84140-6" target="_blank">
            ü§ñ Effects of COVID-19 on Children Over Time</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-201', this)">
            More</button></h4>
            <div id="toggle-201" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term effects of SARS-CoV-2 infection and vaccination in a population-based pediatric cohort</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

After COVID-19 spread widely in kids and teens, this study looked at how getting sick with COVID-19 or getting vaccinated affected their health. We followed a group of 789 kids for almost four years to see if they had any lingering symptoms from COVID-19. Most of them got infected with COVID-19, but many also got vaccinated. The good news is that most kids' overall health was the same as before the pandemic, but some experienced fatigue and exhaustion. Getting vaccinated helped prevent them from getting other cold-like infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the omicron wave of the COVID-19 pandemic and with SARS-CoV-2 vaccines becoming available, seroprevalence rates rose in children and adolescents. This study investigated the impact of both SARS-CoV-2 infections and vaccinations on the incidence of acute and prolonged symptoms in real-world conditions during the transition from the pandemic to the endemic phase. Participants from a pediatric population based seroprevalence study (CorKID study)¬†were followed up at least two and for almost four years by survey of health status features and symptoms suggestive of post-COVID syndrome (PCS). In a subgroup (n‚Äâ=‚Äâ259) SARS-CoV-2 antibody serology was further investigated. 789 participants of the original CorKID study cohort (n‚Äâ=‚Äâ2.121; 37.2%) were included. 67.9% reported at least one SARS-CoV2 infection. 46.6% had received one or more SARS-CoV-2 vaccinations. In the vast majority of serologically tested participants antibodies again SARS-CoV-2 spike (98.9%) or nucleocapsid (93.3%) antigen were detected following infection and/or vaccination. At least 30% experienced one unrecognized SARS-CoV-2 infection. The overall health status was comparable between children, irrespective of SARS-CoV-2 infections and similar to pre-pandemic assessment. However, a subset of young adolescents exhibited a decline in physical performance compared to pre-pandemic conditions. After infection, PCS-like symptoms persisted in 7% of the respondents for more than three months and up to four years. SARS-CoV-2 vaccinated participants (47%) reported 12% less acute flu-like infections other than SARS-CoV-2. Nearly all participants developed SARS-CoV-2 antibodies in this longitudinal study through either vaccination or infection during the Omicron wave. About 7% of participants suffered from PCS symptoms, predominately fatigue and exhaustion. Furthermore, participants who received vaccinations against SARS-CoV-2 reported a lower frequency of acute infections during follow-up.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84140-6" target="_blank">https://www.nature.com/articles/s41598-024-84140-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-23</strong> <a href="https://www.nature.com/articles/s41598-024-80519-7" target="_blank">
            ü§ñ Identification of Viruses That Can Cause Skin Infections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-202', this)">
            More</button></h4>
            <div id="toggle-202" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identification of potential biomarkers for 2022 Mpox virus infection: a transcriptomic network analysis and machine learning approach</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The monkeypox virus (MPXV) is spreading rapidly around the world, especially in areas where it doesn't normally exist. Scientists wanted to understand how this new strain works and what genes are involved. They looked at DNA data from cells infected with different strains of MPXV and found 798 genes that were only expressed when skin cells were infected with the new strain. Further analysis showed that these genes were related to cell growth, immune response, and cancer. The researchers also identified six genes that could potentially be used as biomarkers to detect infections and develop new treatments for MPXV.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox virus (MPXV), a zoonotic pathogen, re-emerged in 2022 with the Clade IIb variant, raising global health concerns due to its unprecedented spread in non-endemic regions. Recent studies have shown that Clade IIb (2022 MPXV) is marked by unique genomic mutations and epidemiological behaviors, suggesting variations in host-virus interactions. This study aimed to identify the differentially expressed genes (DEGs) induced by the 2022 MPXV infection through comprehensive bioinformatics analyses of microarray and RNA-Seq datasets from post-infected cell types with different MPXV clades. Subsequently, gene expression network analyses pinpoint the key DEGs, followed by their candidate drug assessment using the Drug SIGnatures DataBase (DSigDB) and validation by multiple machine learning algorithms. Comparative differential gene expression (DGE) analysis revealed 798 DEGs exclusive to the 2022 MPXV invasion in the skin cell types¬†(keratinocytes). Intriguingly, 13 key DEGs were identified across hubs and clusters, highlighting their aberrant expressions in cell cycle regulation, immune responses, and cancer pathways. Biomarker screening via Random Forest (RF) model (selected with PyCaret from multiple models) and validation through t-distributed stochastic neighbor embedding (t-SNE) algorithm, principal component analysis (PCA), and ROC curve analysis employing Logistic Regression and Random Forest, identified 6 key DEGs (TXNRD1,CCNB1,BUB1,CDC20,BUB1B, andCCNA2) as promising biomarkers (AUC‚Äâ>‚Äâ0.7) for clade IIb infection. This study anticipates that further investigation and clinical trials will catalyze novel detection and therapeutic options to combat 2022 MPXV infection in humans.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80519-7" target="_blank">https://www.nature.com/articles/s41598-024-80519-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-22</strong> <a href="https://www.nature.com/articles/s41598-025-85351-1" target="_blank">
            ü§ñ Effects of Nucleosides on the Body</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-203', this)">
            More</button></h4>
            <div id="toggle-203" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Synthesis, H2S releasing properties, antiviral and antioxidant activities and acute cardiac effects of nucleoside 5‚Ä≤-dithioacetates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists created special molecules that can release hydrogen sulfide, a gas that has healing properties. They attached this gas to old molecules like those found in DNA and RNA. The new molecules worked well at releasing the gas and even showed some anti-viral effects against SARS-CoV-2. Some of these molecules also helped improve heart function and boost energy output when used on people.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hydrogen sulfide (H2S) is an endogenous gasotransmitter with cardioprotective and antiviral effects. In this work, new cysteine-selective nucleoside-H2S-donor hybrid molecules were prepared by conjugating nucleoside biomolecules with a thiol-activatable dithioacetyl group. 5‚Ä≤-Dithioacetate derivatives were synthesized from the canonical nucleosides (uridine, adenosine, cytidine, guanosine and thymidine), and the putative 5‚Ä≤-thio metabolites were also produced from uridine and adenosine. According to our measurements made with an H2S-specific sensor, nucleoside dithioacetates are moderately fast H2S donors, the guanosine derivative showed the fastest kinetics and the adenosine derivative the slowest. The antioxidant activity of 5‚Ä≤-thionucleosides is significantly higher than that of trolox, but lower than that of ascorbic acid, while intact dithioacetates have no remarkable antioxidant effect. In human Calu cells, the guanosine derivative showed a moderate anti-SARS-CoV-2 effect which was also confirmed by virus yield reduction assay. Dithioacetyl-adenosine and its metabolite showed similar acute cardiac effects as adenosine, however, it is noteworthy that both 5‚Ä≤-thio modified adenosines increased left ventricular ejection fraction or stroke volume, which was not observed with native adenosine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85351-1" target="_blank">https://www.nature.com/articles/s41598-025-85351-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-22</strong> <a href="https://www.nature.com/articles/s41598-025-87566-8" target="_blank">
            ü§ñ How effective are non-medicinal treatments around the world?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-204', this)">
            More</button></h4>
            <div id="toggle-204" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>From node to network: weaving a global perspective on efficacy and costs of non-pharmaceutical interventions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

During the COVID-19 pandemic, many countries used special rules to slow down the spread of the disease. We studied how these rules worked and when they should be used. Our research showed that certain rules are more effective for longer periods of time. If a country starts using stricter rules too early, it can help prevent the spread of the virus even better. But if it's not able to keep using those rules for as long as needed, trying to start them too late can actually make things worse. Using rules that are less strict but still important for a longer time can also be effective. As the virus becomes more common and less deadly, some special rules that help protect vulnerable people are better than others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Non-pharmaceutical intervention (NPI) policies, ranging from mild measures to total isolation, were implemented worldwide during the COVID-19 pandemic. We adopt a systematic approach to guide policymakers in deploying NPI policies to mitigate the pandemic‚Äôs effects while balancing their social and economic impacts. Our results show that each NPI has an optimal duration, beyond which its effectiveness plateaus. Stricter policies require longer durations, and when sustained for the optimal period, earlier implementation is more effective. However, when this duration is unattainable, timing becomes critical, as both early and late implementation reduce efficacy. Stringent policies with insufficient durations may perform worse than less restrictive measures applied over the same period, and an NPI policy aimed at minimizing overall healthcare burden under a fixed policy duration may significantly intensify peak-time strains. Finally, as the virus becomes more transmissible and less lethal, the effectiveness gap between stringent and less restrictive policies narrows, with targeted interventions for vulnerable groups outperforming universal strict measures.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87566-8" target="_blank">https://www.nature.com/articles/s41598-025-87566-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-22</strong> <a href="https://www.nature.com/articles/s41598-025-86961-5" target="_blank">
            ü§ñ Discovery of Powerful Drugs That Block a Specific Enzyme</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-205', this)">
            More</button></h4>
            <div id="toggle-205" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identification of potent TMPRSS4 inhibitors through structural modeling and molecular dynamics simulations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Some bad things happen when the TMPRSS4 enzyme is working too much. It helps viruses like COVID-19 spread inside our bodies and also makes some cancers grow. We want to find new medicines that can stop this enzyme from working so well. Since we don't know the exact shape of the enzyme, we made models of it using computers. Then, we tested different medicines on these models and found five good candidates: Ergotamine, S55746, NPC478048, Lifirafenib, and NPC77101. These medicines look like they could stop TMPRSS4 from working too well and might be useful to fight diseases caused by this enzyme.</p><br>
                <p><strong>Abstract</strong></p>
                <p>TMPRSS4, a transmembrane serine protease type II, is associated with various pathological illnesses. It has been found to activate SARS-CoV-2, enhance viral infection of human small-intestinal enterocytes and is overexpressed in different types of cancers. Therefore, this study aims to disover potential TMPRSS4 inhibitors that have better binding affinity than the approved inhibitors: 2-hydroxydiarylamide and tyroserleutide. Since no 3D-structure is known for TMPRSS4, structural models for the TMPRSS4 serine protease domain were developed. The modeled structures were validated and subjected to molecular dynamics simulations. FDA-approved, clinical/preclinical drugs and natural products were docked to the pocket of TMPRSS4. Moreover, through a systematic analysis, MD simulations and MM-GBSA binding free energy calculations revealed that the best candidates Ergotamine, S55746, NPC478048, Lifirafenib, and NPC77101 are highly stable drug candidates in complex with TMPRSS4, displaying low RMSD and RMSF values with strong binding stability. Among these compounds, Ergotamine showed the most favorable binding energy (-33.73¬†kcal/mol). Overall, our in silico results revealed that these compounds could act as potent TMPRSS4 inhibitors and need to be validated by future experimental studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86961-5" target="_blank">https://www.nature.com/articles/s41598-025-86961-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-22</strong> <a href="https://www.nature.com/articles/s41598-025-87354-4" target="_blank">
            ü§ñ COVID-19 and vaping do not seem to be linked for long-term health problems in American adults.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-206', this)">
            More</button></h4>
            <div id="toggle-206" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>E-cigarettes are not associated with post-acute COVID-19 syndrome among US adults</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had COVID-19 are often left feeling tired and having trouble breathing, which is called post-acute COVID-19 syndrome. E-cigarettes can increase the risk of getting COVID-19 because they affect a protein in our bodies that helps fight off viruses. Some people worry that e-cigarette use might make it harder to recover from COVID-19.

A study looked at 107,000 adults who had tested positive for COVID-19 and used e-cigarettes or not. The study found that being female, overweight, smoking, or having depression, asthma, or COPD made it more likely that people would have post-acute COVID-19 syndrome. However, using e-cigarettes was not found to be a risk factor for getting PCS.

This means that if you're an adult who had COVID-19 and are concerned about e-cigarette use affecting your recovery, there's no clear evidence to suggest that it will make things worse.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has resulted in many survivors experiencing post-acute COVID-19 syndrome (PCS) with symptoms including fatigue, breathlessness, and cognitive complaints. E-cigarette use has already been associated with increased susceptibility to COVID-19 because of its effects on ACE2 receptor expression and inflammation, raising concern that it might worsen the long-term outcomes of COVID-19, including PCS. While traditional smoking is associated with a higher risk of PCS, the role of e-cigarettes remains unclear due to conflicting evidence. Using 2022 Behavioral Risk Factor Surveillance System (BRFSS) data, this study investigated the association between e-cigarette use and PCS among US adults who tested positive for COVID-19. The final sample included 107,249 adults after the exclusion of respondents with missing information. It analyzed e-cigarette use (never, former, current) and controlled for key covariates such as age, gender, BMI, smoking, and chronic diseases. The results showed that female gender, obesity, current smoking, and a history of depression, asthma, and chronic obstructive pulmonary disease (COPD) were significantly associated with higher odds of PCS. Nevertheless, e-cigarette use was not related significantly to increased odds for PCS (current e-cigarette use: aOR‚Äâ=‚Äâ1.07, 95 CI: 0.96, 1.20; former e-cigarette use: aOR‚Äâ=‚Äâ1.03, 95 CI: 0.96, 1.12). The mediation analysis showed no indirect effect of the use of e-cigarettes on PCS via COPD. In conclusion our findings did not reveal an independent or indirect association between PCS with e-cigarette use.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-87354-4" target="_blank">https://www.nature.com/articles/s41598-025-87354-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-21</strong> <a href="https://www.nature.com/articles/s41440-025-02109-y" target="_blank">
            ü§ñ Effects of Medications on Heart Disease in Dogs</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-207', this)">
            More</button></h4>
            <div id="toggle-207" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical studies in Myxomatous Mitral Valve Disease dogs: most prescribed ACEI inhibits ACE2 enzyme activity and ARB increases AngII pool in plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

High blood pressure is affecting 1.3 billion people worldwide, mostly in low-income countries. Two common medicines, ACE inhibitors and ARBs, were originally meant to help lower blood pressure but are actually making it worse. Researchers found that giving dogs with a heart condition called Myxomatous Mitral Valve Disease (MMVD) a special mix of medicine and plant cells made their blood pressure better controlled. This new combination increased the activity of an enzyme that helps regulate blood pressure and had no negative side effects. The study suggests that biologics made from plants could be used to treat high blood pressure in humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The¬†hypertension patient population has doubled since 1990, affecting 1.3 billion globally and >75% live in low-and middle-income countries. Angiotensin Converting Enzyme Inhibitors (ACEI) and Angiotensin Receptor Blockers (ARB) are the¬†most prescribed drugs¬†(>160 million times in the US), but mortality increased >30% since 1990s globally. Clinical relevance of Myxomatous Mitral Valve Disease (MMVD) is directly linked to WHO group 2 pulmonary hypertension, with no disease specific therapies. Therefore, MMVD pet dogs with elevated systolic blood pressure treated with ACEI/ARB, were supplemented with oral ACE2 enzyme and Angiotensin1-7 (Ang1-7) bioencapsulated in plant cells. The oral ACE2/Ang1-7 was well tolerated by healthy and MMVD dogs with no adverse events and increased sACE2 activity by 670‚Äì755% with ARB (Telmisartan) than with ACEI (Enalapril) background therapy. In vitro rhACE2 activity was inhibited >90% by ACEIs enalapril/benazeprilat at higher doses but lisinopril inhibited at much lower doses. Membrane ACE2 activity evaluated in exosomes was 43-fold higher than the sACE2 and this was also inhibited 211% by ACEI, when compared to ARB. Background ACEI treatment reduced the Ang-II pool by 11-20-fold and proportionately decreased the abundance of Ang1-7‚Äâ+‚ÄâAng1-5 peptides. In contrast, ARB treatment increased Ang-II pool 11-20-fold and Ang1-7‚Äâ+‚ÄâAng1-5 by 160‚Äì260%. Systolic blood pressure was regulated by ARB better than ACEI, despite very high Ang-II levels. This first report on evaluation of metabolic pools in the RAS pathway identifies surprising interactions between ACEI/ARB/ACE2 and significant changes in key molecular dynamics. Affordable biologics developed in plant cells may offer potential new treatment options for hypertension.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41440-025-02109-y" target="_blank">https://www.nature.com/articles/s41440-025-02109-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-21</strong> <a href="https://www.nature.com/articles/s41467-024-55273-z" target="_blank">
            ü§ñ Health effects on children from COVID-19 vary by racial and ethnic group.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-208', this)">
            More</button></h4>
            <div id="toggle-208" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Racial/ethnic differences in post-acute sequelae of SARS-CoV-2 in children and adolescents in the United States</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

COVID-19 can cause different problems in kids depending on their skin color. We looked at 225,000 kids who got COVID-19 to see if this is true. The study found that some groups of color were more or less likely to get certain symptoms like fever, hair loss, or trouble with breathing. For example, Asian American kids were more likely to get fever and respiratory problems, while black kids were less likely to get skin symptoms. This means that kids from different racial backgrounds might react differently to COVID-19, even if they have the same kind of illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Racial/ethnic differences are associated with the symptoms and conditions of post-acute sequelae SARS-CoV-2 infection (PASC) in adults. These differences may exist among children and warrant further exploration. We conducted a retrospective cohort study with difference-in-differences analyzes to assess these differences in children and adolescents under the age of 21. The study utilized data from the RECOVER Initiative in the United States, which aims to learn about the long-term effects of COVID-19. The cohort included 225,723 patients with SARS-CoV-2 infection or COVID-19 diagnosis between March 2020 and October 2022. The study compared minority racial/ethnic groups to Non-Hispanic White (NHW) individuals, stratified by severity during the acute phase of COVID-19. Within the severe group, Asian American/Pacific Islanders (AAPI) had a higher prevalence of fever/chills and respiratory¬†signs and symptoms, Hispanic patients showed greater hair loss prevalence in severe COVID-19 cases, while Non-Hispanic Black (NHB) patients had fewer skin symptoms in comparison to NHW patients. Within the non-severe group, AAPI patients had increased POTS/dysautonomia and respiratory symptoms, and NHB patients showed more cognitive symptoms than NHW patients. In conclusion, racial/ethnic differences related to COVID-19 exist among PASC symptoms and conditions in pediatrics, and these differences are associated with the severity of illness during acute COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55273-z" target="_blank">https://www.nature.com/articles/s41467-024-55273-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-21</strong> <a href="https://www.nature.com/articles/s42003-024-07262-7" target="_blank">
            ü§ñ Predicting New COVID Virus Mutations with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-209', this)">
            More</button></h4>
            <div id="toggle-209" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Paying attention to the SARS-CoV-2 dialect : a deep neural network approach to predicting novel protein mutations</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract:

Scientists have been trying to figure out how new viruses change their genes, which affects life science research. Usually, they do this experimentally, but it's expensive and takes time. A new way using computer algorithms has shown great promise in understanding DNA sequences. We created a new method called Deep Novel Mutation Search that uses computers to predict how proteins might change. We used the SARS-CoV-2 virus as an example and built a model that looks at three things: how the protein's meaning changes, its grammar changes, and how its parts interact with each other. By combining these three views, we can better understand how new viruses evolve. This method could help us predict new mutations early on and warn people about potential future threats to public health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Predicting novel mutations has long-lasting impacts on life science research. Traditionally, this problem is addressed through wet-lab experiments, which are often expensive and time consuming. The recent advancement in neural language models has provided stunning results in modeling and deciphering sequences. In this paper, we propose a Deep Novel Mutation Search (DNMS) method, using deep neural networks, to model protein sequence for mutation prediction. We use SARS-CoV-2 spike protein as the target and use a protein language model to predict novel mutations. Different from existing research which is often limited to mutating the reference sequence for prediction, we propose a parent-child mutation prediction paradigm where a parent sequence is modeled for mutation prediction. Because mutations introduce changing context to the underlying sequence, DNMS models three aspects of the protein sequences: semantic changes, grammatical changes, and attention changes, each modeling protein sequence aspects from shifting of semantics, grammar coherence, and amino-acid interactions in latent space. A ranking approach is proposed to combine all three aspects to capture mutations demonstrating evolving traits, in accordance with real-world SARS-CoV-2 spike protein sequence evolution. DNMS can be adopted for an early warning variant detection system, creating public health awareness of future SARS-CoV-2 mutations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07262-7" target="_blank">https://www.nature.com/articles/s42003-024-07262-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-18</strong> <a href="https://www.nature.com/articles/s43856-025-00740-x" target="_blank">
            ü§ñ Chimpanzee virus used in medicine for long-term storage</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-210', this)">
            More</button></h4>
            <div id="toggle-210" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A lyophilised formulation of chimpanzee adenovirus vector for long-term stability outside the deep-freeze cold chain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists tested how well an adenovirus vaccine can stay good for five years without refrigeration. They made a special form of the vaccine that could be stored at a very low temperature and checked its quality over time. After two years, they found that the vaccine still worked as expected in mice and rabbits.

The study showed that the vaccine stayed stable for five years with only a small loss of effectiveness (about 20%). This is good news because it means the vaccine can be stored in places where refrigeration isn't available, like in some low-income countries.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundThe adenovirus-vaccine platform has come to prominence with the COVID-19 vaccination campaigns. The objective of this study was to validate a formulation that was suitable for lyophilisation and long-term storage at 5 (2‚Äì8) ¬∞C.MethodsVaccine stability was assessed up to five years at 5‚Äâ¬∞C using a lyophilised formulation of the chimpanzee-adenovirus vector ChAd155 encoding a respiratory syncytial virus (RSV) antigen. Vaccine potency was assessed by functional infectivity assay. Other assessments of vaccine stability included those for capsid integrity, particle content, and DNA release. Vaccine efficacy and safety were assessed after two years in a murine model of RSV challenge and a rabbit toxicology model, respectively.ResultsHere, we show that the potency loss from lyophilisation was 0.12 log10. The potency loss over five years at 5‚Äâ¬∞C was estimated at 0.21 log10(95%CI 0.10‚Äì0.30). This coincides with a 25% increase in the ratio of non-infectious particles/infectious particles. After two years of storage at 5‚Äâ¬∞C, (i) the loss of infectivity was 0.17 log10; (ii) the vaccine remained immunogenic and effective at clearing RSV from the lungs in a mouse-challenge model; and (iii) the vaccine was not associated with any adverse safety signal in a rabbit toxicology model.ConclusionsThe 5-year stability of the lyophilised adenovirus-vector vaccine is within our acceptable limit (‚Äâ<‚Äâ0.3 log10decrease). Its formulation process is amenable to manufacturing scale-up and should help in providing adenovirus-based vaccines where the cold chain is problematic, such as in low-income countries, and in pre-epidemic stockpiling.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-025-00740-x" target="_blank">https://www.nature.com/articles/s43856-025-00740-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-18</strong> <a href="https://www.nature.com/articles/s43856-024-00675-9" target="_blank">
            ü§ñ New Vaccine Shows Protection Against Omicron Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-211', this)">
            More</button></h4>
            <div id="toggle-211" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Beta-variant recombinant SARS CoV-2 vaccine induces durable cross-reactive antibodies against Omicron BA variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Young people can be protected better from new Covid-19 variants by a specific vaccine that targets the original strain of the virus. This vaccine was tested against another vaccine that is used to boost immunity, and it showed that it produces higher levels of antibodies that protect against new variants. When people get infected with Covid-19, both vaccines help their bodies produce more antibodies that fight the infection. However, the first vaccine is more effective at producing these protective antibodies over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundWe previously reported the safety and immunogenicity data from a randomized trial comparing the booster responses of vaccinees who received monovalent (MV) recombinant protein Beta-variant (MVB.1.351) and MV ancestral protein (MVD614) vaccines with AS03 adjuvant (Sanofi/GSK) to booster response of vaccinees who received mRNA MV ancestral strain BNT162b2 vaccine (Pfizer-BioNTech).MethodsFirst booster of the vaccines was administered in adult participants previously primed with 2 doses of MV ancestral strain BNT162b2. A subset of these participants with available blood samples collected at Day 0 (D0), at 28 days (D28), and 3 months (M3) post-booster were contacted for additional testing (195/208 participants). The persistence of cross-neutralizing antibodies, including against Omicron BA.1 and BA.4/5, up to 3 months after boosting was evaluated using a validated pseudovirus neutralization assay.ResultsAcross the whole population, MVB.1.351 vaccine induces highest NAbs titers against Omicron BA.1 and BA.4/5 variants at D28 and M3 post-booster. In participants with SARS-CoV-2 infection between D28 and M3, both MVB.1.351 and BNT162b2 vaccine groups show an increase in GMTs against Omicron BA.1 and Omicron BA.4/5 following infection. Among uninfected participants, the ratio of M3 to D28 GMTs was higher for the MVB.1.351 group than the BNT162b2 group against Omicron BA.1 (0.64 [0.53;0.77] versus 0.43 [0.35;0.53]), Omicron BA.4/5 (0.61 [0.50; 0.75] versus 0.44 [0.34; 0.56]), and D614 (0.68 [0.58,0.81] versus 0.46 [0.39,0.55]).ConclusionsThe MVB.1.351 vaccine induces higher and durable cross-neutralizing antibodies against Omicron subvariants up to 3 months after boosting compared to an MV ancestral and mRNA BNT162b2 booster vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00675-9" target="_blank">https://www.nature.com/articles/s43856-024-00675-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-18</strong> <a href="https://www.nature.com/articles/s41598-025-86426-9" target="_blank">
            ü§ñ Changes in how fast people with chronic lung disease worsen after getting COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-212', this)">
            More</button></h4>
            <div id="toggle-212" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Change in exacerbation rate of COPD patients before and after COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the abstract rewritten for a young person:

The COVID-19 pandemic has changed how doctors treat people with lung disease like COPD. This study looked at what happened when people with COPD got sick with COVID-19. They found that when people with COPD got COVID-19, they had more bad days and needed more hospital care. In fact, people who didn't have bad days before getting COVID-19 started having them afterwards. The pandemic also made it cost a lot more to take care of people with COPD.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has profoundly affected global health system, significantly altering not only the acute management of viral infection, but also management strategies for chronic diseases. This study aimed to investigate the impact of COVID-19 infection on exacerbation rates and the economic burden in patients with COPD. We conducted a retrospective cohort study using data from the national insurance reimbursement data of South Korea. Eligible participants included COPD patients diagnosed with COVID-19 between January and December 2020. We analyzed exacerbation rates, healthcare utilization, and medical costs pre- and post-COVID-19 infection. In 3,445 COPD patients who were infected by COVID-19, COVID-19 infection resulted in increased annual moderate-to-severe and severe exacerbations compared to pre-COVID-19 infection (IRR‚Äâ=‚Äâ1.062 [95%CI 1.027‚Äì1.099]; IRR‚Äâ=‚Äâ1.315 [95%CI 1.182‚Äì1.481], respectively). Among previously non-exacerbators, 11.2% of patients transitioned to exacerbator after COVID-19 infection. Older age, comorbidities and use of triple therapy were the factors associated with transitioners. Direct medical costs escalated significantly from approximately $6810 to $11,032, reflecting the increased intensity of care after COVID-19 infection. COVID-19 infection has significantly increased rate of exacerbations in patients with COPD and imposed a heavier economic burden on healthcare system. Among non-exacerbators, substantial number of patients transitioned to exacerbators after COVID-19 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86426-9" target="_blank">https://www.nature.com/articles/s41598-025-86426-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-18</strong> <a href="https://www.nature.com/articles/s41467-025-55871-5" target="_blank">
            ü§ñ What viruses in the SARS-CoV-2 family cause illness?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-213', this)">
            More</button></h4>
            <div id="toggle-213" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reverse mutational scanning of SARS-CoV-2 spike BA.2.86 identifies epitopes contributing to immune escape from polyclonal sera</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new type of coronavirus has been found with many changes in its protein structure that help it avoid being recognized by our immune system. Researchers created different versions of this virus by removing some of these changes to see which ones are most important for it to evade our body's defenses. They found that certain parts of the virus's surface protein are key to its ability to escape our immune system, and that getting vaccinated with a booster shot can help improve our ability to fight back against this type of virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recently detected Omicron BA.2.86 lineage contains more than 30 amino acid mutations relative to BA.2. BA.2.86 and its JN.1 derivative evade neutralization by serum antibodies of fully vaccinated individuals. In this study, we elucidate epitopes driving the immune escape of BA.2.86 and JN.1 via pseudovirus neutralization. Here we generate 33 BA.2.86 mutants, each reverting a single mutation back to BA.2. We use this library in an approach that we call reverse mutational scanning to define distinct neutralization titers against each epitope. Mutations within the receptor binding domain at K356T, V483Œî, and to a lesser extent N460K, A484K, and F486P enhance immune escape. Interestingly, 16insMPLF within the spike N-terminal domain and P621S within S1/S2 also significantly contribute to antibody escape of BA.2.86. Upon XBB.1.5 booster vaccination, neutralization titers against JN.1 and BA.2.86 improve considerably, and residual immune escape is driven by 16insMPLF, N460K, E554K, and to a lesser extent P621S, and A484K.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55871-5" target="_blank">https://www.nature.com/articles/s41467-025-55871-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-17</strong> <a href="https://www.nature.com/articles/s41467-024-54615-1" target="_blank">
            ü§ñ Pigs with a special gene can get very sick from the coronavirus that causes COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-214', this)">
            More</button></h4>
            <div id="toggle-214" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Human ACE2 transgenic pigs are susceptible to SARS-CoV-2 and develop COVID-19-like disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists are trying to create models of COVID-19 using animals because it helps them understand how the virus works and develop better vaccines and treatments. Pigs are good models because they look similar to humans, but they can't get infected with SARS-CoV-2 themselves. To fix this, scientists made pigs that have a human protein called ACE2, which makes them susceptible to COVID-19. When they got infected, the pigs showed symptoms like fever and coughing, just like people with COVID-19. The study also found that their lungs were damaged in a way similar to what happens in real people with fatal cases of COVID-19. This means that these pigs can be used as a model for studying COVID-19, which is good news for finding new treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Animal models that accurately reflect COVID-19 are vital for understanding mechanisms of disease and advancing development of improved vaccines and therapeutics. Pigs are increasingly recognized as valuable models for human disease due to their genetic, anatomical, physiological, and immunological similarities to humans, and they present a more ethically viable alternative to non-human primates. However, pigs are not susceptible to SARS-CoV-2 infection which limits their utility as a model. To address this, we have developed transgenic pigs expressing human ACE2 that are susceptible to SARS-CoV-2 infection. Following challenge, clinical signs consistent with COVID-19, including fever, coughing and respiratory distress were observed, with virus replication detected in the nasal turbinates, trachea and lungs up to the study endpoint, seven days post-infection. Notably, examination of tissues revealed immunopathology in the lungs consistent with histological changes observed in fatal human COVID-19 cases. This study establishes human ACE2 transgenic pigs as a large animal model that accurately reflects many aspects of COVID-19 disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54615-1" target="_blank">https://www.nature.com/articles/s41467-024-54615-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-17</strong> <a href="https://www.nature.com/articles/s41598-025-86155-z" target="_blank">
            ü§ñ Safest ways to stay healthy in school during pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-215', this)">
            More</button></h4>
            <div id="toggle-215" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sociodemographic factors associated with dental students knowledge and attitudes regarding disinfection as a control measure to reduce the spread of COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an explanation of the abstract in simpler terms:

A group of 503 dental students from Peru participated in a study about how they feel about disinfecting their workplaces to stop the spread of COVID-19. The researchers wanted to see if certain things, like where they're from and if someone in their family got sick with COVID-19, affected what they think about disinfecting.

The results showed that most dental students have positive attitudes towards disinfecting, but only a small percentage (14%) actually know enough about it. People who came from the city were 52% less likely to know enough about disinfecting. But being from the city wasn't as important for what they thought about disinfecting.

The researchers found that some things, like where someone is from and if their family member got sick with COVID-19, didn't seem to affect how much people think about disinfecting.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite maintaining a lower mortality rate and greater control of victims infected by COVID-19, the world‚Äôs population and science are still confronted with this coronavirus. Therefore, the aim was to assess the association between sociodemographic factors and the level of knowledge and attitudes of dental students regarding disinfection as a control measure to reduce the spread of COVID-19. This cross-sectional study evaluated 503 dental students from the capital city and one Peruvian province between February and June 2022. A validated 13-item questionnaire was used. A Poisson regression model with robust variance was used to evaluate the influence of the variables sex, age, year of study, marital status, place of origin, death of a family member due to COVID-19, and history of COVID-19, with the level of knowledge and attitudes of the dental students, considering a significance level ofp<‚Äâ0.05. Of the total participants, 14.3% showed sufficient knowledge, and 89.3% showed positive attitudes regarding disinfection as a control measure to reduce the spread of COVID-19. Furthermore, those from the capital city were 52% less likely to have sufficient knowledge regarding disinfection as a control measure to reduce the spread of COVID-19, compared to those from the province (APR‚Äâ=‚Äâ0.48; 95% CI: 0.31‚Äì0.75). Moreover, none of the variables considered in this study were significantly associated with attitudes toward this topic (p>‚Äâ0.05). A minority of dental students presented sufficient knowledge, while the majority presented positive attitudes regarding disinfection as a control measure to reduce the spread of COVID-19. In addition, being from the capital city was a limiting factor for sufficient knowledge. The variables sex, age, year of study, marital status, place of origin, death of a family member due to COVID-19, and history of COVID-19 were not influential factors for positive attitudes on this topic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86155-z" target="_blank">https://www.nature.com/articles/s41598-025-86155-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-17</strong> <a href="https://www.nature.com/articles/s41467-025-55931-w" target="_blank">
            ü§ñ Boosting immunity to COVID-19 with extra vaccine shots.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-216', this)">
            More</button></h4>
            <div id="toggle-216" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing antibody immune correlates in COVAIL trial recipients of an mRNA second COVID-19 vaccine boost</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied 985 people who got a booster shot to protect against COVID-19. They looked at how well their immune systems worked after getting the vaccine. The results showed that if someone had been infected with COVID-19 before and then got vaccinated, having high levels of antibodies in their blood made them less likely to get sick from the new Omicron variant. However, this wasn't true for people who hadn't gotten infected before - they still needed a good immune response to be protected. This discovery raises questions about how our immune systems work when we're exposed to different variants of COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Neutralizing antibody titer has been a surrogate endpoint for guiding COVID-19 vaccine approval and use, although the pandemic‚Äôs evolution and the introduction of variant-adapted vaccine boosters raise questions as to this surrogate‚Äôs contemporary performance. For 985 recipients of an mRNA second bivalent or monovalent booster containing various Spike inserts [Prototype (Ancestral), Beta, Delta, and/or Omicron BA.1 or BA.4/5] in the COVAIL trial (NCT05289037), titers against 5 strains were assessed as correlates of risk of symptomatic COVID-19 (‚ÄúCOVID-19‚Äù) and as correlates of relative (Pfizer-BioNTech Omicron vs. Prototype) booster protection against COVID-19 over 6 months of follow-up during the BA.2-BA.5 Omicron-dominant period. Consistently across the Moderna and Pfizer-BioNTech vaccine platforms and across all variant Spike inserts assessed, both peak and exposure-proximal (‚Äúpredicted-at-exposure‚Äù) titers correlated with lower Omicron COVID-19 risk in individuals previously infected with SARS-CoV-2, albeit significantly less so in na√Øve individuals [e.g., exposure-proximal hazard ratio per 10-fold increase in BA.1 titer 0.74 (95% CI 0.59, 0.94) for na√Øve vs. 0.41 (95% CI 0.23, 0.64) for non-na√Øve; interaction p‚Äâ=‚Äâ0.013]. Neutralizing antibody titer was a strong inverse correlate of Omicron COVID-19 in non-na√Øve individuals and a weaker correlate in na√Øve individuals, posing questions about how prior infection alters the neutralization correlate.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55931-w" target="_blank">https://www.nature.com/articles/s41467-025-55931-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-17</strong> <a href="https://www.nature.com/articles/s41598-025-86407-y" target="_blank">
            ü§ñ Ocular Surface Temperature After COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-217', this)">
            More</button></h4>
            <div id="toggle-217" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evaluation of ocular surface temperature in post-COVID-19 patients with different degrees of fever via infrared thermal imaging</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how hot or cold the surface of the eye is in people who had COVID-19 and are recovering. Researchers used a special tool to measure this temperature, called infrared thermal imaging.

They compared the temperatures of three groups: those with no fever, those with mild or moderate fever, and those with high fever. They found that:

* The eyes of people with high fever got significantly hotter than those with no fever.
* When people with COVID-19 were recovering for a short time (less than 5 days), their eyes seemed to get even hotter when they opened them up.
* This could be because the eyes are getting dry, which can cause a decrease in temperature.

The researchers think that this might help explain why some people with COVID-19 have symptoms like dry, itchy eyes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aimed to evaluate ocular surface temperature (OST) in post-COVID-19 patients with different degrees of fever via infrared thermal imaging. There were 16 participants (32 eyes) in the control group, 22 participants (44 eyes) in the moderate and low post-COVID-19 fever group (M & L fever group), and 18 participants (36 eyes) in the high post-COVID-19 fever group (H fever group). All participants underwent an ophthalmic slit lamp examination and ocular thermography. Among the control group, M & L fever group and H fever group, there were no significant differences in the upper eyelid temperature (UET), inner canthus temperature (ICT), outer canthus temperature (OCT), initial central corneal temperature (initial CCT), third-second central corneal temperature (3s-CCT), or sixth-second central corneal temperature (6s-CCT). However, the change in central corneal temperature measured within 1, 3, and 6¬†s (change in CCT within 1, 3, and 6¬†s) of the H fever group were significantly greater than those of the control group (0.15‚Äâ¬±‚Äâ0.12¬†¬∞C vs. 0.08‚Äâ¬±‚Äâ0.09¬†¬∞C,p=‚Äâ0.007; 0.30‚Äâ¬±‚Äâ0.22¬†¬∞C vs. 0.17‚Äâ¬±‚Äâ0.17¬†¬∞C,p=‚Äâ0.005; 0.45‚Äâ¬±‚Äâ0.30¬†¬∞C vs. 0.26‚Äâ¬±‚Äâ0.23¬†¬∞C,p=‚Äâ0.004, respectively) and M & L fever group (0.15‚Äâ¬±‚Äâ0.12¬†¬∞C vs. 0.08‚Äâ¬±‚Äâ0.08¬†¬∞C,p=‚Äâ0.008; 0.30‚Äâ¬±‚Äâ0.22¬†¬∞C vs. 0.16‚Äâ¬±‚Äâ0.14¬†¬∞C,p=‚Äâ0.001; 0.45‚Äâ¬±‚Äâ0.30¬†¬∞C vs. 0.23‚Äâ¬±‚Äâ0.20¬†¬∞C,p<‚Äâ0.001, respectively). To further investigate the relationship between OST and post-COVID-19 fever, we compared the OST of long recovery time (5 days‚Äâ<‚Äârecovery time‚Äâ<‚Äâ14 days; 9 patients, 18 eyes) and short recovery time (recovery time‚Äâ‚â§‚Äâ5 days; 9 patients, 18 eyes) in the H fever group. We found that the 6s-CCT in the short recovery time group was significantly lower than that in the long recovery time group (32.43‚Äâ¬±‚Äâ1.09¬†¬∞C vs.33.10‚Äâ¬±‚Äâ0.82¬†¬∞C,p=‚Äâ0.044). Additionally, the change in CCT within 1¬†s, 3¬†s, and 6¬†s in the short recovery time group were all significantly greater than those in the long recovery time group (0.19‚Äâ¬±‚Äâ0.13¬†¬∞C vs. 0.11‚Äâ¬±‚Äâ0.10¬†¬∞C,p=‚Äâ0.048; 0.38‚Äâ¬±‚Äâ0.24¬†¬∞C vs. 0.22‚Äâ¬±‚Äâ0.17¬†¬∞C,p=‚Äâ0.026; 0.58‚Äâ¬±‚Äâ0.31¬†¬∞C vs. 0.32‚Äâ¬±‚Äâ0.24¬†¬∞C,p=‚Äâ0.016, respectively). In conclusion, the central corneal temperature (CCT) of patients who have an insufficient recovery time from COVID-19 infection or who exhibit severe infection symptoms could decrease faster when the eyes open. This may be due to dry eye disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86407-y" target="_blank">https://www.nature.com/articles/s41598-025-86407-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-17</strong> <a href="https://www.nature.com/articles/s41541-024-01057-x" target="_blank">
            ü§ñ Annual flu shots no longer give people immunity after each season.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-218', this)">
            More</button></h4>
            <div id="toggle-218" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibody responses against influenza A decline with successive years of annual influenza vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

Researchers studied how well immunity to flu lasts after getting vaccinated. They looked at hundreds of hospital workers in Australia who got vaccinated during 2020-2021. The researchers took blood samples before and after vaccination to see how their bodies responded. They found that people who had been vaccinated most recently (not previously) had the best immune response, while those who were already up-to-date on their vaccinations didn't respond as well. This suggests that getting vaccinated too frequently can weaken the immune system's ability to fight flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Influenza vaccine effectiveness and immunogenicity can be compromised with repeated vaccination. We assessed immunological markers in a cohort of healthcare workers (HCW) from six public hospitals around Australia during 2020‚Äì2021. Sera were collected pre-vaccination and ~14 and ~180 days post-vaccination and assessed in haemagglutination inhibition assay against egg-grown vaccine and equivalent cell-grown viruses. Responses to vaccination were compared by the number of prior vaccinations. Baseline sera were available for 595 HCW in 2020 and 1031 in 2021. 5% had not been vaccinated during five years prior to enrolment and 55% had been vaccinated every year. Post-vaccination titres for all vaccine antigens were lowest among HCW vaccinated in all 5-prior years and highest among HCW with 0 or 1 prior vaccinations, even after adjustment. This was observed for both influenza A subtypes and was dependent on pre-vaccination titre. Expanded cohorts are needed to better understand how this translates to vaccine effectiveness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01057-x" target="_blank">https://www.nature.com/articles/s41541-024-01057-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-16</strong> <a href="https://www.nature.com/articles/s41598-025-85595-x" target="_blank">
            ü§ñ Impact of genetic changes on how the body breaks down certain COVID-19 medicines.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-219', this)">
            More</button></h4>
            <div id="toggle-219" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>In-silico screening and analysis of missense SNPs in human CYP3A4/5 affecting drug-enzyme interactions of FDA-approved COVID-19 antiviral drugs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The human body has tiny changes in its genes called single nucleotide polymorphisms (SNPs). These changes can affect how enzymes work, which is important because some enzymes help break down medicines to fight viruses like SARS-CoV-2. We used computers to look at how different versions of these enzymes interact with certain medicines and found that some SNPs can make the enzymes more or less effective at breaking down the medicine. We also identified 94 genes that could potentially affect how well these enzymes work, which is important for developing personalized medicine treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Single nucleotide polymorphisms (SNPs) represent the prevailing form of genetic variations observed in the human population. Such variations could alter the encoded enzymes‚Äô activities. CYP3A4/5 enzymes are involved in metabolizing drugs, notably antivirals against SARS-CoV-2. In this work, we computationally investigated antiviral-enzyme interactions of CYP3A4/5 genetic variants. We also examined the deleterious impact of 751 missense single nucleotide polymorphisms (SNPs) within the CYP3A4/5 genes. An ensemble of bioinformatics tools, [SIFT, PolyPhen-2, cadd, revel, metaLr, mutation assessor, Panther, SNP&GO, PhD-SNP, SNAP, Meta-SNP, FATHMM, I-Mutant, MuPro, INPS, CONSURF, GPS 5.0, MusiteDeep and NetPhos], identified a total of 94 variants (47 SNPs in CYP3A4, 47 SNPs in CYP3A5) to potentially impact the structural integrity as well as the activity of the CYP3A4/5 enzymes. Molecular docking was done to recognize the structural stability and binding properties of the CYP3A4/5 protein isoforms with 3 FDA-approved antiviral drugs. Our findings indicated that the CYP3A4 gene variants; R418T, I335T and R130P and the CYP3A5 gene variants; I335T, L133P and R130Q are considered the most deleterious missense SNPs. These mutants potentially affect drug-enzyme binding and hence may alter therapeutic response. Cataloguing deleterious SNPs is essential for personalized gene-based pharmacotherapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85595-x" target="_blank">https://www.nature.com/articles/s41598-025-85595-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-16</strong> <a href="https://www.nature.com/articles/s43856-024-00720-7" target="_blank">
            ü§ñ Use of Safety and Effectiveness of a Special Medicine to Prevent Severe COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-220', this)">
            More</button></h4>
            <div id="toggle-220" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Uptake and safety of Sotrovimab for prevention of severe COVID-19 in a cohort and self-controlled case series study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Sotrovimab is a medicine that helps people with very bad COVID-19 symptoms. There have been some concerns about its safety, but scientists want to make sure it's safe for everyone.

To find out, they looked at how many people from different groups took the medicine and if they got any bad side effects. They also checked if taking the medicine made anyone more likely to get hospitalized.

What they found is that:

* The age and sex of people who took Sotrovimab were similar to those who didn't take it.
* More people from some groups (like Indians, Asians, and Bangladeshis) took the medicine than others (like Black Caribbean and Black African people).
* There was no evidence that taking Sotrovimab caused any bad side effects or increased the risk of getting hospitalized.

So, while there might be some differences in how well Sotrovimab works for different groups of people, it doesn't seem to cause any major safety concerns.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSotrovimab is a neutralising monoclonal antibody (nMAB) currently available to treat extremely clinically vulnerable COVID-19 patients in England. Trials have shown it to have mild to moderate side effects, however, evidence regarding its safety in real-world settings remains insufficient.MethodsDescriptive and multivariable logistic regression analyses were conducted to evaluate uptake, and a self-controlled case series analysis performed to measure the risk of hospital admission (hospitalisation) associated with 49 pre-specified suspected adverse outcomes in the period 2‚Äì28 days post-Sotrovimab treatment among eligible patients treated between December 11, 2021 and May 24, 2022.ResultsHere we show that among treated and untreated eligible individuals, the mean ages (54.6 years, SD: 16.1 vs 54.1, SD: 18.3) and sex distribution (women: 60.9% vs 58.1%; men: 38.9% vs 41.1%) are similar. There are marked variations in uptake between ethnic groups, which is higher amongst individuals categorised ethnically as Indian (15.0%; 95%CI 13.8, 16.3), Other Asian (13.7%; 95%CI 11.9, 15.8), white (13.4%; 95%CI 13.3, 13.6), and Bangladeshi (11.4%; 95%CI 8.8, 14.6); and lower amongst Black Caribbean individuals (6.4%; 95%CI 5.4, 7.5) and Black Africans (4.7%; 95%CI 4.1, 5.4). We find no increased risk of any of the suspected adverse outcomes in the period 2‚Äì28 days post-treatment.ConclusionsWe find no safety signals of concern for possible adverse outcomes in the period 2-28 days post treatment with Sotrovimab. However, there is evidence of unequal uptake of Sotrovimab treatment across ethnic groups.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00720-7" target="_blank">https://www.nature.com/articles/s43856-024-00720-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-16</strong> <a href="https://www.nature.com/articles/s41598-024-84411-2" target="_blank">
            ü§ñ Simulation of Heat Effects and Particle Spread on Railway Coaches</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-221', this)">
            More</button></h4>
            <div id="toggle-221" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>3D modelling and simulation of thermal effects and dispersion of particles carrying infectious respiratory agents in a railway transport coach</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

When people breathe, cough, or sneeze, tiny particles can spread into the air. These particles can carry germs that make us sick. Scientists studied how these particles move in crowded places like trains and classrooms. They used special computer tools to model how the air moves and where the particles go.

In one experiment, they tested what happens when people wear masks or not. They found that even with a mask, some particles can still spread around. But they also showed that if the ventilation system is set up right, it can help clean the air.

This research helps us understand how to stay safe in crowded places and prevent the spread of germs. It's especially important for places like hospitals and schools where people are more likely to be together.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Even though the COVID-19 pandemic now belongs to the long history of infectious diseases that have struck humanity, pathogenic biological agents continue to pose a recurring threat in private places, but also and mainly in places where the public congregates. In our recent research published in this journal in 2022 and 2023, we considered the illustrative example of a commuter train coach in which a symptomatic or asymptomatic passenger, assumed to be infected with a respiratory disease, sits among other travellers. The passenger emits liquid particles containing, for example, COVID-19 virions or any other pathogen. The size spectrum of particles varies depending on whether they are produced during breathing, speaking, coughing or sneezing. More specifically, droplets associated with breathing are in the range of 1‚Äì10¬†¬µm in aerodynamic diameter, while at the other end of the spectrum, drops associated with coughing can reach 100‚Äì1000¬†¬µm. In the first part of our research, we used Computational Fluid Dynamics (CFD) to model and simulate in 3D the transport and dispersion of particles from 1¬†¬µm to 1¬†mm in the turbulent flow generated by the ventilation of the railway coach. We used both the Eulerian approach and the Lagrangian approach and showed that the results were strictly similar and illustrated the very distinct aerodynamics, on one hand, of the aerosol of droplets suspended in the air and, on the other hand, of the drops falling or behaving like projectiles depending on their initial speed. In the second part of our research, we developed a model of filtration through a typical surgical mask and possible leaks around the mask if it is poorly adjusted. We resumed the twin experiment of the railway coach and compared the distribution of droplets depending on whether the passengers (including the infected one) wear masks or not and whether the masks are perfectly fitted or worn loosely. Our method made it possible to quantify the particles suspended in the air of the railway coach depending on whether the infected passenger wore their mask more or less well. In this third article, we specifically explore how thermal effects due to the presence of passengers influence the spatio-temporal distribution in the railway coach of aerosols produced by the breathing infected person. We demonstrate that the influence of thermal effects on aerodynamics is very significant and can be very favourable for air decontamination if the ventilation system is judiciously configured. Beyond its application to a commuter train, our work confirms the value of validated CFD tools for describing the airflow and dispersion of particles in complex spaces that do not always allow experimentation. The models that we have developed are applicable to any other semi-confined, ventilated public place, such as a classroom, a hospital room or a performance hall, and they enable the objective assessment of whether the occupation of these spaces could be critical with regard to infectious contamination and of how to limit this ubiquitous risk.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84411-2" target="_blank">https://www.nature.com/articles/s41598-024-84411-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s41380-024-02886-x" target="_blank">
            ü§ñ Long COVID Affects Brain Volume</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-222', this)">
            More</button></h4>
            <div id="toggle-222" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Choroid plexus volume is enlarged in long COVID and associated with cognitive and brain changes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

After having COVID-19, some people experience long-term symptoms like trouble thinking. Research shows that these problems are related to changes in the brain. One part of the brain called the choroid plexus may be involved. The study looked at 129 people with these long-term symptoms after COVID-19 and compared them to healthy people. They found that the choroid plexus was bigger in those with symptoms, which made them think slower and had problems with brain function. This is related to changes in other parts of the brain and immune system. The study suggests that the choroid plexus plays a role in how these long-term symptoms affect the brain.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients with post-COVID condition (PCC) present with diverse symptoms which persist at long-term after SARS-CoV-2 infection. Among these symptoms, cognitive impairment is one of the most prevalent and has been related to brain structural and functional changes. The underlying mechanisms of these cognitive and brain alterations remain elusive but neuroinflammation and immune mechanisms have been majorly considered. In this sense, the choroid plexus (ChP) volume has been proposed as a marker of neuroinflammation in immune-mediated conditions and the ChP epithelium has been found particularly susceptible to the effects of SARS-CoV-2. The objective was to investigate the ChP in PCC and evaluate its relationships with cognition, brain, and immunological alterations. One-hundred and twenty-nine patients with PCC after a mean of 14.79‚Äâ¬±‚Äâ7.17 months of evolution since the infection and 36 healthy controls were recruited. Participants underwent a neuropsychological, and neuroimaging assessment and immunological markers evaluation. Results revealed ChP volume enlargement in PCC compared to healthy controls. The ChP enlargement was associated with cognitive dysfunction, grey matter volume reduction in frontal and subcortical areas, white matter integrity and diffusivity changes and functional connectivity changes. These ChP changes were also related to intermediate monocytes levels. Findings suggest that the ChP integrity may play a relevant role in the pathophysiology of cognitive deficits and the observed brain changes in PCC. The previously documented function of the ChP in maintaining brain homeostasis and regulating the entry of immune cells into the brain supports the presence of neuroinflammatory mechanisms in this disorder.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41380-024-02886-x" target="_blank">https://www.nature.com/articles/s41380-024-02886-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s41541-024-01050-4" target="_blank">
            ü§ñ Antibody Booster Peptide</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-223', this)">
            More</button></h4>
            <div id="toggle-223" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A synthetic cyclic peptide for promoting antigen presentation and immune activation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We created a new vaccine that can deliver different types of antigens to help our body fight infections better. Our vaccine is made from a special kind of protein called cyclic peptides, which are good at keeping the antigens stable. We tested this vaccine on people and found that it was very effective in triggering an immune response against certain infections, like strep throat and COVID-19. The vaccine even worked better than regular vaccines with commercial adjuvants (helps boost the immune system). What's special about our vaccine is that it uses a natural compound called lipoamino acid to help antigens be taken up by cells in the lymph nodes, which makes it more effective. This means we can develop new and promising vaccines using this technology.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cyclic peptides are often used as scaffolds for the multivalent presentation of drug molecules due to their structural stability and constrained conformation. We identified a cyclic deca-peptide incorporating lipoamino acids for delivering T helper and B cell epitopes against group AStreptococcus(GAS), eliciting robust humoral immune responses. In this study, we assessed the function-immunogenicity relationship of the multi-component vaccine candidate (referred to as VC-13) to elucidate a mechanism of action. We identified a potential universal delivery platform, not only capable of adjuvanting different peptide epitopes (e.g., NS1 and 88/30 from group AStreptococcus, gonadotropin hormone releasing hormone [GnRH]), but also protein antigens (e.g., bovine serum albumin [BSA], receptor binding domain (RBD) of the SARS-CoV-2 protein responsible for COVID-19 infection [SARS-CoV-2 RBD]) and small molecular haptens (e.g., cocaine). All vaccine candidates self-assembled into sub-500 nm nanoparticles and induced high antigen-specific systemic IgG titers and opsonic potential compared to the antigen co-administered with a commercial adjuvant, complete Freund‚Äôs adjuvant. Notably, presence of the cyclic decapeptide in this vaccine increased accumulation in the draining inguinal lymph nodes, facilitating cellular uptake of peptide antigens. Furthermore, the lipoamino acid promoted dendritic cell activation, acting as both toll-like receptors 2 and 4 -targeting moiety. Our study revealed the importance of the cyclic decapeptide and lipoamino acid presence in antigen presentation and immune response activation, leading onto the development of a fully synthetic, self-assembled, and promising platform for the delivery of subunit vaccines and anti-drug vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01050-4" target="_blank">https://www.nature.com/articles/s41541-024-01050-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s42003-024-07351-7" target="_blank">
            ü§ñ Small RNA stops coronavirus and flu viruses from reproducing.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-224', this)">
            More</button></h4>
            <div id="toggle-224" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>G-quadruplex-forming small RNA inhibits coronavirus and influenza A virus replication</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have discovered a new way to fight future pandemics by creating a special type of RNA that can stop certain viruses from working. When they added this RNA to cells, it quickly got into the cell and stopped the virus from spreading. It even helped animals get better when they were sick with the same virus. This new approach is different from other ways of fighting viruses, and it might be used to create a new type of medicine that can deliver the treatment without using special containers.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Future pandemic threats may be caused by novel coronaviruses and influenza A viruses. Here we show that when directly added to a cell culture, 12mer guanine RNA¬†(G12) and its phosphorothioate-linked derivatives (G12(S)), rapidly entered cytoplasm and suppressed the propagation of human coronaviruses and influenza A viruses to between 1/100 and nearly 1/1000 of normal virus infectivity without cellular toxicity and induction of innate immunity. Moreover, G12(S) alleviated the weight loss caused by coronavirus infection in mice. G12(S) might exhibit a stable G-tetrad with left-handed parallel-stranded G-quadruplex, and inhibit the replication process by impeding interaction between viral nucleoproteins and viral RNA in the cytoplasm. Unlike previous antiviral strategies that target the G-quadruplexes of the viral genome, we now show that excess exogenous G-quadruplex-forming small RNA displaces genomic RNA from ribonucleoprotein, effectively inhibiting viral replication. The approach has the potential to facilitate the creation of versatile middle-molecule antivirals featuring lipid nanoparticle-free delivery.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07351-7" target="_blank">https://www.nature.com/articles/s42003-024-07351-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s42003-025-07491-4" target="_blank">
            ü§ñ Lipid Peptides Block Coronavirus Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-225', this)">
            More</button></h4>
            <div id="toggle-225" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intranasally administrated fusion-inhibitory lipopeptides block SARS-CoV-2 infection in mice and enable long-term protective immunity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have discovered special helpers called lipopeptides that can fight against the coronavirus (COVID-19). They tested these helpers on mice and found that they:

- Stop the virus from entering cells
- Help keep the lungs healthy
- Prevent weight loss and reduce the amount of virus in the body
- Even save the lives of mice infected with different versions of the virus

These lipopeptides are safe to use repeatedly and work well even after repeated infections. They also help create strong immunity, which can protect against new variants of the virus. This discovery could lead to a new way to fight COVID-19 and other viruses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We have assessed antiviral activity and induction of protective immunity of fusion-inhibitory lipopeptides derived from the C-terminal heptad-repeat domain of SARS-CoV-2 spike glycoprotein in transgenic mice expressing human ACE2 (K18-hACE2). The lipopeptides block SARS-CoV-2 infection in cell lines and lung-derived organotypic cultures. Intranasal administration in mice allows the maintenance of homeostatic transcriptomic immune profile in lungs, prevents body-weight loss, decreases viral load and shedding, and protects mice from death caused by SARS-CoV-2 variants. Prolonged administration of high-dose lipopeptides has neither adverse effects nor impairs peptide efficacy in subsequent SARS-CoV-2 challenges. The peptide-protected mice develop cross-reactive neutralizing antibodies against both SARS-CoV-2 used for the initial infection and recently circulating variants, and are completely protected from a second lethal infection, suggesting that they developed SARS-CoV-2-specific immunity. This strategy provides an additional antiviral approach in the global effort against COVID-19 and may contribute to development of rapid responses against emerging pathogenic viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-025-07491-4" target="_blank">https://www.nature.com/articles/s42003-025-07491-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s41598-025-85440-1" target="_blank">
            ü§ñ Predicting COVID-19 Spread with a Special Kind of Computer Model</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-226', this)">
            More</button></h4>
            <div id="toggle-226" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Leveraging dynamics informed neural networks for predictive modeling of COVID-19 spread: a hybrid SEIRV-DNNs approach</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract in simpler language:

We created a new way to predict how infectious diseases spread over time using artificial intelligence and math. We combined two things: math that describes how diseases spread, and a type of computer program that can learn from data. We tested our method on real data from China and found it worked well. This new approach can be used for other diseases like the flu or dengue fever with some changes to the rules. It will help doctors and health officials make better predictions and come up with more effective ways to stop the spread of disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A dynamics informed neural networks (DINNs) incorporating the susceptible-exposed-infectious-recovered-vaccinated (SEIRV) model was developed to enhance the understanding of the temporal evolution dynamics of infectious diseases. This work integrates differential equations with deep neural networks to predict time-varying parameters in the SEIRV model. Experimental results based on reported data from China between January 1, and December 1, 2022, demonstrate that the proposed dynamics informed neural networks (DINNs) method can accurately learn the dynamics and predict future states. Our proposed hybrid SEIRV-DNNs model can also be applied to other infectious diseases such as influenza and dengue, with some modifications to the compartments and parameters in the model to accommodate the related control measures. This approach will facilitate improving predictive modeling and optimizing public health intervention strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85440-1" target="_blank">https://www.nature.com/articles/s41598-025-85440-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-15</strong> <a href="https://www.nature.com/articles/s41598-025-86260-z" target="_blank">
            ü§ñ Genetic clues to COVID-19 severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-227', this)">
            More</button></h4>
            <div id="toggle-227" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genetic proxies for clinical traits are associated with increased risk of severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People have discussed using genetic information in healthcare, but we don't know how well it works with traditional risk factors like age, sex, and lifestyle. We looked at severe COVID-19 to see if genetic data could help predict who is at higher risk.

We used genetic data from 450,000 people and compared the results to medical records of over 9,500 people diagnosed with severe COVID-19 before vaccines were available. We found that certain genetic scores were linked to a higher risk of severe COVID-19.

Some of these genetic scores are related to heart problems or dementia. Using all this information together helped us better understand what's happening in the body and how genes might be involved.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Routine use of genetic data in healthcare is much-discussed, yet little is known about its performance in epidemiological models including traditional risk factors. Using severe COVID-19 as an exemplar, we explore the integration of polygenic risk scores (PRS) into disease models alongside sociodemographic and clinical variables. PRS were optimized for 23 clinical variables and related traits previously-associated with severe COVID-19 in up to 450,449 UK Biobank participants, and tested in 9,560 individuals diagnosed in the pre-vaccination era. Associations were further adjusted for (i) sociodemographic and (ii) clinical variables. Pathway analyses of PRS were performed to improve biological understanding of disease. In univariate analyses, 17 PRS were associated with increased risk of severe COVID-19 and, of these, four remained associated with COVID-19 outcomes following adjustment for sociodemographic/clinical variables: hypertension PRS (OR‚Äâ=‚Äâ1.1, 95%CI 1.03‚Äì1.18), atrial fibrillation PRS (OR‚Äâ=‚Äâ1.12, 95%CI 1.03‚Äì1.22), peripheral vascular disease PRS (OR‚Äâ=‚Äâ0.9, 95%CI 0.82‚Äì0.99), and Alzheimer‚Äôs disease PRS (OR‚Äâ=‚Äâ1.14, 95%CI 1.05‚Äì1.25). Pathway analyses revealed enrichment of genetic variants in pathways for cardiac muscle contraction (genesN=‚Äâ5;beta[SE]=‚Äâ3.48[0.60]; adjusted-P=‚Äâ1.86‚Äâ√ó‚Äâ10‚àí5). These findings underscore the potential for integrating genetic data into epidemiological models and highlight the advantages of utilizing multiple trait PRS rather than a single PRS for a specific outcome of interest.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-86260-z" target="_blank">https://www.nature.com/articles/s41598-025-86260-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41392-024-02114-6" target="_blank">
            ü§ñ Antibody to fight COVID-19 variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-228', this)">
            More</button></h4>
            <div id="toggle-228" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A broadly neutralizing antibody against the SARS-CoV-2 Omicron sub-variants BA.1, BA.2, BA.2.12.1, BA.4, and BA.5</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new coronavirus called SARS-CoV-2 has spread all around the world, and its variations like Omicron are making it harder to fight. The virus is getting better at avoiding our immune system's defenses, so we need to find new ways to stop it.

We've discovered a new medicine that can help: a special antibody called CR9. It targets the part of the virus that makes people sick, and it works really well in both labs and in living animals. We used special tools to see how it works, and now we know how it can help treat Omicron infections.

This discovery is important because it shows us that we need to keep working on new medicines to stay ahead of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The global spread of Severe Acute Respiratory Syndrome Coronavirus 2. (SARS-CoV-2) and its variant strains, including Alpha, Beta, Gamma, Delta, and now Omicron, pose a significant challenge. With the constant evolution of the virus, Omicron and its subtypes BA.1, BA.2, BA.3, BA.4, and BA.5 have developed the capacity to evade neutralization induced by previous vaccination or infection. This evasion highlights the urgency in discovering new monoclonal antibodies (mAbs) with neutralizing activity, especially broadly neutralizing antibodies (bnAbs), to combat the virus.In the current study, we introduced a fully human neutralizing mAb, CR9, that targets Omicron variants. We demonstrated the mAb‚Äôs effectiveness in inhibiting Omicron replication both in vitro and in vivo. Structural analysis using cryo-electron microscopy (cryo-EM) revealed that CR9 binds to an epitope formed by RBD residues, providing a molecular understanding of its neutralization mechanism. Given its potency and specificity, CR9 holds promise as a potential adjunct therapy for treating Omicron infections. Our findings highlight the importance of continuous mAb discovery and characterization in addressing the evolving threat of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-02114-6" target="_blank">https://www.nature.com/articles/s41392-024-02114-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41467-024-55272-0" target="_blank">
            ü§ñ How neutrophils affect lung blood flow during COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-229', this)">
            More</button></h4>
            <div id="toggle-229" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutrophil adhesion to vessel walls impairs pulmonary circulation in COVID-19 pathology</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

COVID-19 can cause blood clots in the lungs, making it harder for oxygen to reach the body. Researchers studied how this happens using mice with severe pneumonia. They found that when neutrophils (a type of white blood cell) stick too tightly to the blood vessels in the lungs, it blocks the flow of oxygen and makes it harder for people to breathe. The researchers also looked at what was happening inside the bodies of people who were severely sick with COVID-19. They found that the same thing was happening - when neutrophils stuck too tightly to the blood vessels, it caused problems with oxygen delivery. This suggests that sticking of neutrophils is a key factor in making COVID-19 worse.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Microthrombus formation is associated with COVID-19 severity; however, the detailed mechanism remains unclear. In this study, we investigated mouse models with severe pneumonia caused by SARS-CoV-2 infection by using our in vivo two-photon imaging system. In the lungs of SARS-CoV-2-infected mice, increased expression of adhesion molecules in intravascular neutrophils prolonged adhesion time to the vessel wall, resulting in platelet aggregation and impaired lung perfusion. Re-analysis of scRNA-seq data from peripheral blood mononuclear cells from COVID-19 cases revealed increased expression levels ofCD44andSELLin neutrophils in severe COVID-19 cases compared to a healthy group, consistent with our observations in the mouse model. These findings suggest that pulmonary perfusion defects caused by neutrophil adhesion to pulmonary vessels contribute to COVID-19 severity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55272-0" target="_blank">https://www.nature.com/articles/s41467-024-55272-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41598-025-85806-5" target="_blank">
            ü§ñ Bacterial infections spread through nose and digestive system during Covid-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-230', this)">
            More</button></h4>
            <div id="toggle-230" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessment of upper respiratory and gut bacterial microbiomes during COVID-19 infection in adults: potential aerodigestive transmission</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus affects not just our lungs, but also our gut. Research has found that people with COVID-19 have different types of bacteria in their gut and lungs compared to healthy people. This can lead to problems breathing and digestive issues. In some cases, this disruption can be severe enough to require hospital care.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 is the viral pathogen responsible for COVID-19. Although morbidity and mortality frequently occur as a result of lung disease, the gastrointestinal (GI) tract is recognized as a primary location for SARS-CoV-2. Connections and interactions between the microbiome of the gut and respiratory system have been linked with viral infections via what has been referred to as the ‚Äògut-lung axis‚Äô with potential aerodigestive communication in health and disease. This research explored the relationship between the microbiomes of the upper respiratory and GI tracts in patients with COVID-19 and examined Extraesophageal reflux (EOR), a mechanism which could contribute to dysregulated communication between the GI and respiratory tract (as identified in COVID-19). 97 patients with a laboratory diagnosis of COVID-19 infection, and 50 age-matched controls were recruited and stool, saliva and sputum were obtained from each participant. ELISA Pepsin tests and Reflux Symptom Index scores (RSI) were conducted for EOR assessment. DNA sequencing of the V4 region of the 16¬†S rRNA gene was performed for microbiome analysis. No differences were observed between the fecal microbiome‚Äôs alpha and Shannon diversity indices; however, a distinct microbial composition was observed in COVID-19 patients (when compared to the controls). The respiratory microbiota from individuals with COVID-19 demonstrated a statistically significant reduction in Shannon diversity and bacterial richness alongside an overall reduction in the prevalence of organisms from a typical healthy respiratory microbiome. Furthermore, the bacterial richness of the stool and sputum samples was significantly lower among COVID-19 patients admitted to ICU. A significantly higher RSI score and salivary pepsin level were detected among those with COVID-19. The data indicates that COVID-19 is associated with a dysregulation of both the gut and lung microbiome with a more marked perturbation in the lung, particularly among COVID-19 patients who had been admitted to the ICU. The presence of increased RSI scores, combined with elevated levels of Pepsin, suggests that increased micro-aspiration may occur, which is consistent with of under-recognized interactions between the GI and lung microbiomes in COVID-19 patients and requires additional study. Such studies would benefit from the insights provided by biological samples which reflect the continuum of the aerodigestive tract.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85806-5" target="_blank">https://www.nature.com/articles/s41598-025-85806-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41541-025-01066-4" target="_blank">
            ü§ñ How mRNA Vaccines Work Against SARS-CoV-2 Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-231', this)">
            More</button></h4>
            <div id="toggle-231" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>mRNA vaccines with RBD mutations have broad-spectrum activity against SARS-CoV-2 variants in mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

New virus strains have made some vaccines less effective over time. To solve this problem, scientists created two new mRNA vaccines, BSCoV05 and BSCoV06. They added specific mutations to these vaccines that help them fight a wider range of viruses. When tested in mice, both vaccines worked well and produced antibodies that could protect against many different virus strains. These vaccines showed promise and should be further tested in monkeys to see if they can provide long-term protection against future virus variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The emergence of SARS-CoV-2 variants with defined mutations that enhance pathogenicity or facilitate immune evasion has resulted in a continual decline in the protective efficacy of existing vaccines. Therefore, there is a pressing need for a vaccine capable of combating future variants. In this study, we designed new mRNA vaccines, BSCoV05 and BSCoV06, and generated point mutations in the receptor-binding domain (RBD) of the original Wuhan strain to increase their broad-spectrum antiviral activity. Additionally, we used the BA.1 RBD as a control. Both vaccines elicited a robust immune response in BALB/c and K18-hACE2 mice, generating high levels of specific binding antibodies against the BA.2 RBD. Moreover, all three vaccines induced neutralizing antibodies against the prototype viral strain and relevant variants, including the Alpha and Beta strains and the Omicron variants BA.1, BA.2, BA.5, XBB.1.5, XBB.1.16, EG.5.1, and EG.5.1.1, with BSCoV06 demonstrating broader neutralizing antibody activity. Both BSCoV05 and BSCoV06 also elicited a cellular immune response. After the challenge, both BSCoV05 and BSCOV06 provided protection against the EG.5.1 strain in both mouse strains. Therefore, these two vaccines merit further evaluation in nonhuman primates, and this vaccine design strategy should be explored for its potential application in combating future SARS-CoV-2 variants, offering valuable insights into broad-spectrum vaccine development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01066-4" target="_blank">https://www.nature.com/articles/s41541-025-01066-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41540-025-00488-z" target="_blank">
            ü§ñ COVID-19 Symptoms Persist Due to Misguided Immune Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-232', this)">
            More</button></h4>
            <div id="toggle-232" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dysregulated autoantibodies targeting AGTR1 are associated with the accumulation of COVID-19 symptoms</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version:

COVID-19 can cause many different symptoms, but scientists still don't fully understand why. This study looked at people who had COVID-19 and checked for special proteins in their blood called autoantibodies. They found that some of these proteins were connected to specific symptoms like fever, muscle pain, and losing the sense of smell.

They also discovered a type of protein that was linked to serious symptoms and that increased when people got more sick. The study suggests that these autoantibodies might play a big role in how severe COVID-19 is and could be part of what makes some people sicker than others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) presents a wide spectrum of symptoms, the causes of which remain poorly understood. This study explored the associations between autoantibodies (AABs), particularly those targeting G protein-coupled receptors (GPCRs) and renin‚Äíangiotensin system (RAS) molecules, and the clinical manifestations of COVID-19. Using a cross-sectional analysis of 244 individuals, we applied multivariate analysis of variance, principal component analysis, and multinomial regression to examine the relationships between AAB levels and key symptoms. Significant correlations were identified between specific AABs and symptoms such as fever, muscle aches, anosmia, and dysgeusia. Notably, anti-AGTR1 antibodies, which contribute to endothelial glycocalyx (eGC) degradation, a process reversed by losartan, have emerged as strong predictors of core symptoms. AAB levels increased with symptom accumulation, peaking in patients exhibiting all four key symptoms. These findings highlight the role of AABs, particularly anti-AGTR1 antibodies, in determining symptom severity and suggest their involvement in the pathophysiology of COVID-19, including vascular complications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41540-025-00488-z" target="_blank">https://www.nature.com/articles/s41540-025-00488-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-13</strong> <a href="https://www.nature.com/articles/s41598-025-85852-z" target="_blank">
            ü§ñ Rise in Respiratory Viruses in Shanghai After COVID Restrictions Lifted</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-233', this)">
            More</button></h4>
            <div id="toggle-233" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Resurgence of common respiratory viruses and mycoplasma pneumoniae after ending the zero-COVID policy in Shanghai</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>China stopped its COVID-19 restrictions almost three years ago. Now, a study found that common cold viruses and other infections are back in schools and hospitals. Before the restrictions were lifted, some viruses were less common in children who got sick. But after they ended, more people got infected with different types of viruses, especially those that affect the lungs. The most common lung virus changed from one type to another. There were also more cases of kids getting two or more infections at once. This means China needs to plan for more respiratory illnesses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>China has adhered to policies of zero-COVID for almost three years since the outbreak of COVID-19, which has remarkably affected the circulation of respiratory pathogens. However, China has begun to end the zero-COVID policies in late 2022. Here, we reported a resurgence of common respiratory viruses andMycoplasma pneumoniaewith unique epidemiological characteristics among children after ending the zero-COVID policy in Shanghai, China, 2023. Children hospitalized with acute respiratory tract infections were enrolled from January 2022 to December 2023. Nine common respiratory viruses and 2 atypical bacteria were detected in respiratory specimens from the enrolled patients using a multiplex PCR-based assay. The data were analyzed and compared between the periods before (2022) and after (2023) ending the zero-COVID policies. A total of 8550 patients were enrolled, including 6170 patients in 2023 and 2380 patients in 2022. Rhinovirus (14.2%) was the dominant pathogen in 2022, however,Mycoplasma pneumoniae(38.8%) was the dominant pathogen in 2023. Compared with 2022, the detection rates of pathogens were significantly increased in 2023 (72.9% vs. 41.8%,p<‚Äâ0.001). An out‚Äêof-season epidemic of respiratory syncytial virus was observed during the spring and summer of 2023. The median age of children infected with respiratory viruses in 2023 was significantly greater than that in 2022. Besides, mixed infections were more frequent in 2023 (23.8% vs. 28.9%,p<‚Äâ0.001). China is now facing multiple respiratory pathogen epidemics with changing seasonality, altered age distribution, and increasing mixed infection rates among children in 2023. Our finding highlights the need for public health interventions to prepare for the respiratory pathogen outbreaks in the post-COVID-19 era.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85852-z" target="_blank">https://www.nature.com/articles/s41598-025-85852-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-11</strong> <a href="https://www.nature.com/articles/s41598-025-85919-x" target="_blank">
            ü§ñ Impact of COVID-19 on Brain Function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-234', this)">
            More</button></h4>
            <div id="toggle-234" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neurological post-COVID syndrome is associated with substantial impairment of verbal short-term and working memory</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a rewritten version of the abstract that is easier for young people to understand:

Many people who had COVID-19 are still feeling tired, having trouble remembering things, and getting mental fogginess months or even years after they got sick. We wanted to study how their brains are affected by this condition.

We looked at 60 people who were still experiencing symptoms of COVID-19 (called Post-COVID Syndrome), 15 people who had recovered from COVID-19 but weren't having any problems, and 15 healthy people as a comparison group. We asked them about their basic information and how they're feeling. Then we gave them some tests to see how well they were doing in different parts of their brain.

The results showed that the people with Post-COVID Syndrome were much more tired, depressed, and anxious than the other groups. They also did worse on many of the cognitive tests, like remembering things and paying attention. The researchers found that being tired was the main thing affecting how well they could focus and remember things.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A substantial proportion of patients suffer from Post-COVID Syndrome (PCS) with fatigue and impairment of memory and concentration being the most important symptoms. We here set out to perform in-depth neuropsychological assessment of PCS patients referred to the Neurologic PCS clinic compared to patients without sequelae after COVID-19 (non-PCS) and healthy controls (HC) to decipher the most prevalent cognitive deficits. We includedn=‚Äâ60 PCS patients with neurologic symptoms,n=‚Äâ15 non-PCS patients andn=‚Äâ15 healthy controls. Basic socioeconomic data and subjective complaints were recorded. This was followed by a detailed neuropsychological test battery, including assessments of general orientation, motor and cognitive fatigue, screening of depressive and anxiety symptoms, information processing speed, concentration, visuomotor processing speed, attention, verbal short-term and working memory, cognitive flexibility, semantic and phonematic word fluency, as well as verbal and visual memory functions. Neurologic PCS patients had more complaints with significantly higher fatigue scores as well as higher levels of depressive and anxiety symptoms compared to Non-PCS and HC. Deep neuropsychological assessment showed that neurologic PCS patients performed worse in a general screening of cognitive deficits compared to HC. Neurologic PCS patients showed impaired mental flexibility as an executive subfunction, verbal short-term memory, working memory and general reactivity (prolonged reaction time). Multiple regression showed fatigue affected processing speed; depression did not. Self-reported cognitive deficits of patients with neurologic PCS including fatigue, concentration, and memory deficits, are well mirrored in impaired performance of cognitive domains of concentration and working memory. The present results should be considered to optimize treatment algorithms for therapy and rehabilitation programs of PCS patients with neurologic symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85919-x" target="_blank">https://www.nature.com/articles/s41598-025-85919-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-10</strong> <a href="https://www.nature.com/articles/s41598-025-85771-z" target="_blank">
            ü§ñ Detecting Mpox and Non-Mpox with AI</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-235', this)">
            More</button></h4>
            <div id="toggle-235" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Robustly detecting mpox and non-mpox using a deep learning framework based on image inpainting</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's an abstract rewritten for a young person:

Scientists are working on a new way to diagnose mpox using artificial intelligence. Right now, it's hard to get accurate results because of noise and other issues in real-life images. A new method called "Mask, Inpainting, and Measure" (MIM) uses AI to fill in missing parts of an image and then checks if the filled-in part matches the original part. This helps detect mpox more accurately than existing methods. The researchers tested MIM on different datasets and found that it worked well, even with unknown categories and weird inputs. They also created a free app for smartphones to help people identify mpox easier.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Due to the lack of efficient mpox diagnostic technology, mpox cases continue to increase. Recently, the great potential of deep learning models in detecting mpox and non-mpox has been proven. However, existing methods are susceptible to interference from various noises in real-world settings, require diverse non-mpox images, and fail to detect abnormal input, which makes them unsuitable for practical deployment and application. To address these challenges, we proposed a novel strategy based on image inpainting called ‚ÄúMask, Inpainting, and Measure‚Äù (MIM). In MIM‚Äôs pipeline, a generative adversarial network learns feature representations of mpox images by inpainting the masked mpox images. On this basis, MIM measure the similarity between the inpainted image and the original image to detect mpox and non-mpox. Compared with multi-class classification models, MIM can handle unknown categories and abnormal inputs more effectively. We used the recognized mpox dataset (MSLD) and a dataset containing 18 categories of non-mpox skin diseases to verify the effectiveness and robustness of MIM. Experimental results show that the average AUROC of MIM achieves 0.8237. In addition, external clinical testing further demonstrates the robustness of MIM. Importantly, we developed a free smartphone app to help the public and healthcare professionals detect mpox more conveniently.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85771-z" target="_blank">https://www.nature.com/articles/s41598-025-85771-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-10</strong> <a href="https://www.nature.com/articles/s41467-025-55822-0" target="_blank">
            ü§ñ How certain genes affect the way our bodies fight off infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-236', this)">
            More</button></h4>
            <div id="toggle-236" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>lncRNA NORAD modulates STAT3/STAT1 balance and innate immune responses in human cells via interaction with STAT3</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

In our research, we discovered how two molecules interact in cells: a long non-coding RNA called NORAD and a protein called STAT3. We found that when NORAD is present, it helps STAT3 move to the nucleus where it can suppress the body's immune response to viruses. But if NORAD is not there, STAT3 stays outside and the immune system works better against the virus. Our study shows that this interaction happens in both embryonic cells and adult cells, and it may be specific to humans because of a genetic change that happened in our ancestors.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long non-coding RNAs (lncRNAs) are pivotal regulators of cellular processes. Here we reveal an interaction between the lncRNA NORAD, noted for its role in DNA stability, and the immune related transcription factor STAT3 in embryonic and differentiated human cells. Results from NORAD knockdown experiments implicate NORAD in facilitating STAT3 nuclear localization and suppressing antiviral gene activation. In NORAD-deficient cells, STAT3 remains cytoplasmic, allowing STAT1 to enhance antiviral activity. Analysis of RNA expression data from in vitro experiments and clinical samples demonstrates reduced NORAD upon viral infection. Additionally, evolutionary conservation analysis suggests that this regulatory function of NORAD is restricted to humans, potentially owing to the introduction of an Alu element in hominoids. Our findings thus suggest that NORAD functions as a modulator of STAT3-mediated immune suppression, adding to the understanding of lncRNAs in immune regulation and evolutionary adaptation in host defense mechanisms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55822-0" target="_blank">https://www.nature.com/articles/s41467-025-55822-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-10</strong> <a href="https://www.nature.com/articles/s41564-024-01903-7" target="_blank">
            ü§ñ Vaccines help fight a specific virus infection.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-237', this)">
            More</button></h4>
            <div id="toggle-237" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Vaccine-induced T cell responses controlOrthoflaviviruschallenge infection without neutralizing antibodies in humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study tested how well two vaccines, for yellow fever and Japanese encephalitis, work together to keep people safe from these diseases when they're exposed to them again.

The researchers gave 33 healthy adults either one or both of the vaccines, then infected some with the actual viruses. The results showed that people who got vaccinated had less viral infection and fewer symptoms than those who didn't get vaccinated. 

In particular, if you got vaccinated for yellow fever, it seemed to make you more resistant to Japanese encephalitis, even if your body didn't produce special antibodies that protect against it.

This study found that T cells in the immune system might be what's really helping people stay healthy after they're infected with these diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>T cells have been identified as correlates of protection in viral infections. However, the level of vaccine-induced T cells needed and the extent to which they alone can control acute viral infection in humans remain uncertain. Here we conducted a double-blind, randomized controlled trial involving vaccination and challenge in 33 adult human volunteers, using the live‚Äìattenuated yellow fever (YF17D) and chimeric Japanese encephalitis‚ÄìYF17D (JE/YF17D) vaccines. BothOrthoflavivirusvaccines share T cell epitopes but have different neutralizing antibody epitopes. The primary objective was to assess the extent to which vaccine-induced T cell responses, independent of neutralizing antibodies, were able to reduce post-challenge viral RNAaemia levels. Secondary objectives included an assessment of surrogate measures of viral control, including post-challenge antibody titres and symptomatic outcomes. YF17D vaccinees had reduced levels of JE/YF17D challenge viraemia, compared with those without previous YF17D vaccination (mean log10(area under the curve genome copies per ml): 2.23 versus 3.22;P=‚Äâ0.039). Concomitantly, YF17D vaccinees had lower post-JE/YF17D challenge antibody titres that reduced JE virus plaque number by 50%, or PRNT50(mean log10(PRNT50titre): 1.87 versus 2.5;P<‚Äâ0.0001) and symptomatic rates (6% (n=‚Äâ1/16) versus 53% (n=‚Äâ9/17),P=‚Äâ0.007). There were no unexpected safety events. Importantly, after challenge infection, several vaccinees had undetectable viraemia and no seroconversion, even in the absence of neutralizing antibodies. Indeed, high vaccine-induced T cell responses, specifically against the capsid protein, were associated with a level of viral control conventionally interpreted as sterilizing immunity. Our findings reveal the importance of T cells in controlling acute viral infection and suggests a potential correlate of protection against orthoflaviviral infections. ClinicalTrials.gov registration:NCT05568953.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-024-01903-7" target="_blank">https://www.nature.com/articles/s41564-024-01903-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-10</strong> <a href="https://www.nature.com/articles/s41541-025-01062-8" target="_blank">
            ü§ñ COVID-19 Vaccine Offers Broad Protection Against New Strains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-238', this)">
            More</button></h4>
            <div id="toggle-238" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A bivalent COVID-19 mRNA vaccine elicited broad immune responses and protection against Omicron subvariants infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler terms:

New versions of the coronavirus are emerging all the time and making it harder for vaccines to work properly. To address this, scientists created a new vaccine called M1.2 that includes parts of two different types of the virus: one from an old strain and another from a recent variant. They tested this vaccine in mice and hamsters and found that it made their bodies produce strong defenses against many different strains of the virus. The vaccine also helped to activate immune cells called T cells, which are important for fighting off infections. While the vaccine didn't completely protect against one specific strain of the virus, it did offer broad protection against many other variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Continuously emerging SARS-CoV-2 Omicron subvariants pose a threat thwarting the effectiveness of approved COVID-19 vaccines. Especially, the protection breadth and degree of these vaccines against antigenically distant Omicron subvariants is unclear. Here, we report the immunogenicity and efficacy of a bivalent mRNA vaccine, PTX-COVID19-M1.2 (M1.2), which encodes native spike proteins from Wuhan-Hu-1 (D614G) and Omicron BA.2.12.1, in mouse and hamster models. Both primary series and booster vaccination using M1.2 elicited potent and broad nAbs against Wuhan-Hu-1 (D614G) and some Omicron subvariants. Strong spike-specific T cell responses against Wuhan-Hu-1 and Omicron subvariants, including JN.1, were also induced. Vaccination with M1.2 protected animals from Wuhan-Hu-1 and multiple Omicron subvariants challenges. Interestingly, protection against XBB.1.5 lung infection did not correlate with nAb levels. These results indicate that M1.2 generated a broadly protective immune response against antigenically distant Omicron subvariants, and spike-specific T cells probably contributed to the breadth of the protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-025-01062-8" target="_blank">https://www.nature.com/articles/s41541-025-01062-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-10</strong> <a href="https://www.nature.com/articles/s41467-025-55823-z" target="_blank">
            ü§ñ Understanding COVID-19 in people who have received transplants and how it affects their bodies.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-239', this)">
            More</button></h4>
            <div id="toggle-239" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Host-microbe multiomic profiling identifies distinct COVID-19 immune dysregulation in solid organ transplant recipients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People who have had an organ transplant are more susceptible to the coronavirus. Scientists studied 86 people who had received transplants and compared them to 172 healthy individuals. They found that transplant patients:

- Had more virus in their noses
- Had a harder time getting rid of the virus
- Produced fewer special immune cells called plasmablasts
- Had an overactive immune response

However, transplant patients with severe illness didn't have as strong of an immune response to fight off the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) poses significant risks for solid organ transplant recipients, who have atypical but poorly characterized immune responses to infection. We aim to understand the host immunologic and microbial features of COVID-19 in transplant recipients by leveraging a prospective multicenter cohort of 86 transplant recipients age- and sex-matched with 172 non-transplant controls. We find that transplant recipients have higher nasal SARS-CoV-2 viral abundance and impaired viral clearance, and lower anti-spike IgG levels. In addition, transplant recipients exhibit decreased plasmablasts and transitional B cells, and increased senescent T cells. Blood and nasal transcriptional profiling demonstrate unexpected upregulation of innate immune signaling pathways and increased levels of several proinflammatory serum chemokines. Severe disease in transplant recipients, however, is characterized by a less robust induction of pro-inflammatory genes and chemokines. Together, our study reveals distinct immune features and altered viral dynamics in solid organ transplant recipients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55823-z" target="_blank">https://www.nature.com/articles/s41467-025-55823-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-09</strong> <a href="https://www.nature.com/articles/s41598-024-82886-7" target="_blank">
            ü§ñ The Role of Realism in Virtual Humans for Breathing Exercise Training</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-240', this)">
            More</button></h4>
            <div id="toggle-240" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Usability, acceptance, and the role of realism in virtual humans for breathing exercise training</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Breathing exercises can help with mental health and physical well-being. However, not many people can afford or have time to get professional help. A new technology called Virtual Reality (VR) offers a way to guide people through breathing exercises with a virtual coach. Our study tested this idea with 20 participants who used a VR system that looked like a real person guiding them through different types of breathing exercises. We found that the more realistic the coach looked, the more people trusted it and stuck with it. This technology could make professional help more accessible to those who need it, especially in places where it's hard to find affordable or available services.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Breathing exercises are used to address a multitude of issues ranging from mental health to physical well-being. While they are recommended for various health conditions, access to professional support is often limited due to time and cost constraints. Virtual Reality technology offers a promising solution for enhancing mindfulness and relaxation practices, including breathing exercises. In our study, we explored the effectiveness of a computer-generated ‚Äúvirtual breathing coach‚Äù. Our virtual coach has a human appearance, speaks in a human voice, and displays breathing physiology while guiding the participants through a set of breathing exercises of different cycle lengths. The agent exhibits breathing behaviour with the aim of augmenting the influence the agent exerts through the mechanisms of physiological entrainment known from human-to-human interaction. Our empirical, exploratory study with 20 participants, was conducted using non-immersive VR where the system ran on a web browser. On the one hand, we investigated the perception of the virtual human coach, and on the other hand, we evaluated the usability, acceptability, and adherence of the system. We also evaluated the role of realism in acceptance of and adherence to the virtual coach. Participants‚Äô responses were recorded using questionnaires. Participants were influenced by the virtual coach and adhered to it if they perceived the virtual coach as more realistic. The participants reported that the system ran well, and they would be willing to continue using the system in the future. Although the virtual human-to-human interaction experience provided by our system is not on par with the naturalistic human-to-human interaction, the results indicate that people are willing to replace a real human coach with a virtual coach. Systems like this are a step towards improving the affordability and on-demand access to another human, such as a coach in our system, especially where access and affordability are limited such as in healthcare. The qualities such as trust and likeability in an embodied interaction can make this interaction more efficient and effective.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82886-7" target="_blank">https://www.nature.com/articles/s41598-024-82886-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-09</strong> <a href="https://www.nature.com/articles/s41598-024-84392-2" target="_blank">
            ü§ñ COVID-19 vaccine effectiveness in Thailand</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-241', this)">
            More</button></h4>
            <div id="toggle-241" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Serological insights from SARS-CoV-2 heterologous prime and boost responses in Thailand</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

During the COVID-19 pandemic, some countries used different vaccines together to make sure everyone had enough vaccine doses. This was called "heterologous" vaccination. We studied 647 people who got three different vaccines: CoronaVac, ChAdOx1 nCoV-19, and BNT162b2. We looked at how well their bodies made antibodies to fight COVID-19.

We found that when people got two different vaccines together, they made stronger antibody responses than if they had only gotten one vaccine. This is good news because it means the vaccines are working better together. We also found that getting a booster shot from a different vaccine type can help make even more antibodies.

By analyzing how well people's bodies respond to different vaccines, scientists can create new tools to track COVID-19 cases and improve vaccine development.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, heterologous vaccination strategies were employed to alleviate the strain on vaccine supplies. The Thailand Ministry of Health adopted these strategies using vector, inactivated, and mRNA vaccines. However, this approach has introduced challenges for SARS-CoV-2 sero-epidemiology studies. Our study analysed 647 samples from healthcare workers who received CoronaVac, ChAdOx1 nCoV-19, and BNT162b2 vaccines. The serological profile encompassed responses to various SARS-CoV-2 variants and vectors, measuring IgG, IgM, and IgA isotypes, alongside IgG avidity assays. The results demonstrated that heterologous CoronaVac/ChAdOx1 nCoV-19 schedules elicited significantly stronger antibody responses compared to homologous schedules (IgG: 1.2-fold, IgM: 10.9-fold, IgA: 3.1-fold increase). Additionally, a heterologous BNT162b2 boost at 4-weeks post-initial vaccination showed greater antibody levels than a ChAdOx1 nCoV-19 boost (IgG: 1.1-fold, IgM: slight decrease, IgA: 1.5-fold increase). Using a combination of three analytes, IgG against wild-type Spike trimer, nucleoprotein and Omicron receptor binding domains, enabled the clustering of responses within a statistical Gaussian mixture model that successfully discriminates between breakthrough infections and vaccination types (F-score‚Äâ=‚Äâ0.82). The development of statistical models to predict breakthrough infections can improve serological surveillance. Overall, our study underscores the necessity for vaccine (re-)development and the creation of serological tools to monitor vaccine performance.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84392-2" target="_blank">https://www.nature.com/articles/s41598-024-84392-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-09</strong> <a href="https://www.nature.com/articles/s41598-024-84138-0" target="_blank">
            ü§ñ Understanding COVID-19 Infections After Multiple Vaccinations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-242', this)">
            More</button></h4>
            <div id="toggle-242" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralization of omicron subvariants and antigenic cartography following multiple COVID 19 vaccinations and repeated omicron non JN.1 or JN.1 infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract that simplifies it for a young person:

We've been studying how people's bodies respond to different types of COVID-19 vaccines. We looked at 206 people who had received vaccines, got infected with different types of COVID-19, or had both. Our goal was to see how well their immune systems could fight off new variants of the virus.

The results showed that the immunity from just one vaccine dose isn't enough to protect against all new variants. People who had been infected multiple times or had gotten vaccinated and then got infected again had slightly better protection. But even with repeated infections, some people still didn't have enough protection.

We also found out that newer variants of COVID-19 are becoming more resistant to the antibodies produced by vaccines. However, if someone gets infected with these new variants multiple times, their immune system can learn to fight them off better over time.

This research helps us understand how to make better COVID-19 vaccines in the future and how to protect people from new variants of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ongoing emergence of SARS-CoV-2 variants, combined with antigen exposures from different waves and vaccinations, poses challenges in updating COVID-19 vaccine antigens. We collected 206 sera from individuals with vaccination-only, hybrid immunity, and single or repeated omicron post-vaccination infections (PVIs), including non-JN.1 and JN.1, and evaluated neutralization against omicron BA.5, BA.2.75, BQ.1.1, XBB.1.16, XBB.1.5, and JN.1. Neutralizing antibodies exhibited a narrow breadth against BA.5 and BA.2.75 and failed to neutralize BQ.1.1 and XBB lineages after three to five doses of the ancestral monovalent vaccine. Hybrid immunity elicited higher neutralizing titers than vaccination alone, but titers remained relatively low. A single omicron PVI elicited lower neutralization titers to all variants compared to wild-type (WT), indicating immunological imprinting. Repeated omicron PVIs, particularly JN.1, slightly mitigated these effects by increasing broad neutralization responses to all variants, though not significantly. Antigenic mapping demonstrated that XBB lineages and JN.1 are antigenically distant from WT and also evaded antibodies induced by earlier omicron variants (BA.1‚Äì5) PVIs. However, repeated JN.1 PVIs shortened this antigenic distance, indicating broader neutralization across omicron variants. These findings highlight SARS-CoV-2 immunity following various antigen boosts and the impact of repeated omicron JN.1 exposure on broad immunity, informing future COVID-19 vaccination strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84138-0" target="_blank">https://www.nature.com/articles/s41598-024-84138-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-08</strong> <a href="https://www.nature.com/articles/s41467-025-55840-y" target="_blank">
            ü§ñ Seasonal Flu Changes in Southeast Asian Countries During Pandemics</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-243', this)">
            More</button></h4>
            <div id="toggle-243" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Disruption of seasonal influenza circulation and evolution during the 2009 H1N1 and COVID-19 pandemics in Southeastern Asia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract in simpler terms:

In Asia, especially in Southeastern countries, certain types of flu viruses (H3N2 and B/Victoria) spread easily from one year to another. However, we don't fully understand how these viruses move around during pandemics like COVID-19 or when there's a big outbreak (like the 2009 H1N1 pandemic).

We studied data on genetics, travel patterns, and airline flights between 2007 and 2023 to see where these flu viruses came from and where they went. We found that during the COVID-19 pandemic, the viruses didn't spread as usual in autumn and winter to countries outside Asia. But the H1N1 pandemic only disrupted some of the H3N2 virus's movement.

We also discovered that some flu viruses (like H3N2) are more likely to keep spreading in Asia during pandemics, while others (like B/Victoria) don't spread as much. Our research shows how different types of pandemics affect the way flu viruses move around, which can help us predict future outbreaks and come up with strategies to stop them.</p><br>
                <p><strong>Abstract</strong></p>
                <p>East, South, and Southeast Asia (together referred to as Southeastern Asia hereafter) have been recognized as critical areas fuelling the global circulation of seasonal influenza. However, the seasonal influenza migration network within Southeastern Asia remains unclear, including how pandemic-related disruptions altered this network. We leveraged genetic, epidemiological, and airline travel data between 2007-2023 to characterise the dispersal patterns of influenza A/H3N2 and B/Victoria viruses both out of and within Southeastern Asia, including during perturbations by the 2009 A/H1N1 and COVID-19 pandemics. During the COVID-19 pandemic, consistent autumn-winter movement waves from Southeastern Asia to temperate regions were interrupted for both subtype/lineages, however the A/H1N1 pandemic only disrupted A/H3N2 spread. We find a higher persistence of A/H3N2 than B/Victoria circulation in Southeastern Asia and identify distinct pandemic-related disruptions in A/H3N2 antigenic evolution between two pandemics, compared to interpandemic levels; similar patterns are observed in B/Victoria using genetic distance. The internal movement structure within Southeastern Asia markedly diverged during the COVID-19 pandemic season, and to a lesser extent, during the 2009 A/H1N1 pandemic season. Our findings provide insights into the heterogeneous impact of two distinct pandemic-related disruptions on influenza circulation, which can help anticipate the effects of future pandemics and potential mitigation strategies on influenza dynamics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55840-y" target="_blank">https://www.nature.com/articles/s41467-025-55840-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-07</strong> <a href="https://www.nature.com/articles/s43856-024-00729-y" target="_blank">
            ü§ñ Monitoring sewer health in cities with different income levels.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-244', this)">
            More</button></h4>
            <div id="toggle-244" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Longitudinal monitoring of sewershed resistomes in socioeconomically diverse urban neighborhoods</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We wanted to see if people from different backgrounds in Calgary, Canada have the same amount of bad bacteria that make them sick when they're exposed to antibiotics. We looked at the water used to treat sewage in eight neighborhoods with different levels of wealth. We also looked at sewage water from cities all around the world.

What we found is that people from all different neighborhoods in Calgary had almost the same bad bacteria, even though some areas are wealthier than others. This means that having a lot of money doesn't seem to help prevent you from getting sick when you take antibiotics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundUnderstanding factors associated with antimicrobial resistance (AMR) distribution across populations is a necessary step in planning mitigation measures. While associations between AMR and socioeconomic-status (SES), including employment and education have been increasingly recognized in low- and middle-income settings, connections are less clear in high-income countries where SES remains an important influence on other health outcomes.MethodsWe explored the relationship between SES and AMR in Calgary, Canada using spatially-resolved wastewater-based surveillance of resistomes detected by metagenomics across eight socio-economically diverse urban neighborhoods. Resistomes were established by shotgun-sequencing of wastewater pellets, and qPCR of targeted-AMR genes. SES status was established using 2021 Canadian census data. Conducting this comparison during the height of COVID-related international travel restrictions (Dec. 2020‚ÄìOct. 2021) allowed the hypotheses linking SES and AMR to be assessed with limited confounding. These were compared with sewage metagenomes from 244 cities around the world, linked with Human Development Index (HDI).ResultsWastewater metagenomes from Calgary‚Äôs socioeconomically diverse neighborhoods exhibit highly similar resistomes, with no quantitative differences (p>‚Äâ0.05), low Bray-Curtis dissimilarity, and no significant correlations with SES. By comparison, dissimilarity is observed between globally-sourced resistomes (p<‚Äâ0.05), underscoring the homogeneity of resistomes in Calgary‚Äôs sub-populations. The analysis of globally-sourced resistomes alongside Calgary‚Äôs resistome further reveals lower AMR burden in Calgary relative to other cities around the world. This is particularly pronounced for the most clinically-relevant AMR genes (e.g., beta-lactamases, macrolide-lincosamide-streptogramin).ConclusionsThis work showcases the effectiveness of inclusive and comprehensive wastewater-based surveillance for exploring the interplay between SES and AMR.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00729-y" target="_blank">https://www.nature.com/articles/s43856-024-00729-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-07</strong> <a href="https://www.nature.com/articles/s41467-025-55843-9" target="_blank">
            ü§ñ Vaccine Safety and Effectiveness Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-245', this)">
            More</button></h4>
            <div id="toggle-245" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Safety and immunogenicity of an optimized self-replicating RNA platform for low dose or single dose vaccine applications: a randomized, open label Phase I study in healthy volunteers</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists have created a new way to make vaccines that can protect people from diseases like rabies and the flu. This method uses special molecules called RNA that can do double duty - they help our bodies fight off infections AND make us immune to getting sick in the first place.

In tests, these new vaccines worked really well. They were able to give people protection at much lower doses than traditional vaccines. And when tested on humans, there were no serious side effects reported.

This is a big step forward because it could lead to easier and more effective vaccines that can help keep us safe from many different diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Self-replicating RNA (srRNA) technology, in comparison to mRNA vaccines, has shown dose-sparing by approximately 10-fold and more durable immune responses. However, no improvements are observed in the adverse events profile. Here, we develop an srRNA vaccine platform with optimized non-coding regions and demonstrate immunogenicity and safety in preclinical and clinical development. Optimized srRNA vaccines generate protective immunity (according to the WHO defined thresholds) at doses up to 1,000,000-fold lower than mRNA in female mouse models of influenza and rabies. Clinically, safety and immunogenicity of RBI-4000, an srRNA vector encoding the rabies glycoprotein, was evaluated in a Phase I study (NCT06048770). RBI-4000 was able to elicit de novo protective immunity in the majority of healthy participants when administered at a dose of 0.1, 1, or 10 microgram (71%, 94%, 100%, respectively) in a prime-boost schedule. Similarly, we observe immunity above the WHO benchmark of protection following a single administration in most participants at both 1 and 10 microgram doses. There are no serious adverse events reported across all cohorts. These data establish the high therapeutic index of optimized srRNA vectors, demonstrating feasibility of both low dose and single dose approaches for vaccine applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55843-9" target="_blank">https://www.nature.com/articles/s41467-025-55843-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-07</strong> <a href="https://www.nature.com/articles/s41467-025-55824-y" target="_blank">
            ü§ñ Female mice get protected from multiple types of coronavirus.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-246', this)">
            More</button></h4>
            <div id="toggle-246" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multivalent S2 subunit vaccines provide broad protection against Clade 1 sarbecoviruses in female mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Scientists are trying to make a vaccine that can protect people from many different types of viruses, including SARS-CoV-2, which caused the pandemic. They think targeting a special part of the virus called S2 could help create a vaccine that works against many different variants.

They made a new type of vaccine using nanoparticles that displays multiple copies of the S2 subunit of SARS-CoV-1 and another version of the S2 subunit from SARS-CoV-2. This vaccine protects mice from different types of sarbecoviruses, including some that could be a threat to humans.

The scientists found that the antibodies created by this vaccine help protect against these viruses. They believe their S2-based vaccine could provide protection against many different types of sarbecoviruses and offer clues about how it works.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continuing emergence of immune evasive SARS-CoV-2 variants and the previous SARS-CoV-1 outbreak collectively underscore the need for broadly protective sarbecovirus vaccines. Targeting the conserved S2 subunit of SARS-CoV-2 is a particularly promising approach to elicit broad protection. Here, we describe a nanoparticle vaccine displaying multiple copies of the SARS-CoV-1 S2 subunit. This vaccine alone, or as a cocktail with a SARS-CoV-2 S2 subunit vaccine, protects female transgenic K18-hACE2 mice from challenges with Omicron subvariant XBB as well as several sarbecoviruses identified as having pandemic potential including the bat sarbecovirus WIV1, BANAL-236, and a pangolin sarbecovirus. Challenge studies in female Fc-Œ≥ receptor knockout mice reveal that antibody-based cellular effector mechanisms play a role in protection elicited by these vaccines. These results demonstrate that our S2-based vaccines provide broad protection against clade 1 sarbecoviruses and offer insight into the mechanistic basis for protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-025-55824-y" target="_blank">https://www.nature.com/articles/s41467-025-55824-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-07</strong> <a href="https://www.nature.com/articles/s41598-025-85234-5" target="_blank">
            ü§ñ How mRNA changes when packaged in special tiny bubbles affects how liver cells respond.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-247', this)">
            More</button></h4>
            <div id="toggle-247" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of mRNA formulated with lipid nanoparticles on the transcriptomic and epigenetic profiles of F4/80+liver-associated macrophages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists tested how using tiny packages called lipid nanoparticles (LNPs) with a special message (mRNA) in mice affects their livers. They found that when they injected the mRNA/LNPs into one leg, it triggered a strong immune response and caused certain liver cells to become active. The liver cells showed changes in what genes were turned on or off, which made them behave like a type of immune cell called an M1-like cell. This suggests that using mRNA/LNPs can greatly boost the immune system's ability to fight infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Delivery of an mRNA formulated with lipid nanoparticles (LNPs) induces robust humoral and cell-mediated branches of the immune response. Depending on the LNP formula, mRNA encoding proteins can be detected in the liver upon intramuscular administration of mRNA/LNP in mice. This study investigated the impact of mRNA/LNP administration on liver-associated macrophages at the transcriptomic and epigenetic levels in a mouse model. An mRNA encoding ovalbumin (OVA) formulated with LNPs, was administered intramuscularly, and a robust OVA-specific antibody was detected in the serum on Day 7. F4/80+liver-associated macrophages were isolated and subjected to RNA sequencing, which identified 554 genes whose expression levels were altered compared with those in the PBS control group. The expression of genes involved in macrophage inflammatory functions, such asTnf,Il6andMarco, were upregulated. Gene ontology enrichment analysis revealed that IL-6/JAK/STAT3 and TNFŒ±/NF-Œ∫B hallmarks were significantly enriched, and mRNA/LNP-exposed liver-associated macrophages were characterized as M1-like cells based on the macrophage transcriptomic profiles. Enrichment of the active histone mark H3K4me3 showed that clusters of loci were highly increased in the mRNA/LNP group, indicating an impact of mRNA/LNPs on macrophage epigenetic profiles. Thecis-regulatory regions ofTnf,Il6andMarcoshowed enrichment of H3K4me3 marks, which correlated well with their increased transcription. Taken together, our data indicated that mRNA/LNP administration via the intramuscular route influences the gene expression and epigenetic profiles of liver-associated macrophages, reflecting its robust ability to induce an immune response.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85234-5" target="_blank">https://www.nature.com/articles/s41598-025-85234-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-06</strong> <a href="https://www.nature.com/articles/s41598-024-85068-7" target="_blank">
            ü§ñ Distribution of Respiratory Viruses by Age Group in Kyoto, Japan during the Winter Season 2023-24.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-248', this)">
            More</button></h4>
            <div id="toggle-248" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Epidemiology of respiratory viruses according to age group, 2023‚Äì24 winter season, Kyoto, Japan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>People get sick with colds and flu more often at different times of the year and in different parts of Japan. To understand how many people are getting sick with viral infections, scientists developed a program to track these illnesses. They collected over 2,900 samples from patients who thought they had a cold or flu in Kyoto City. The samples were tested for three main viruses: SARS-CoV-2, influenza, and RSV.

The study found that the most common virus for each age group was:
For kids under 6: Rhinovirus/enterovirus
For kids 6-17: RSV
For adults 18-64: SARS-CoV-2
For older adults 65+: Influenza

The viruses were detected in different amounts in each age group. This study shows that tracking illnesses by age can help us understand how they spread and stay healthy better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The seasonality and epidemiology of viral acute respiratory infections (ARIs) have changed since the coronavirus disease 2019 pandemic. However, molecular-based ARI surveillance has not been conducted in Japan. We developed a regional surveillance program to define the local epidemiology of ARIs. Between December 2023 and March 2024, 2,992 upper respiratory samples collected from patients suspected of having ARIs at five facilities in Kyoto City, Japan, were tested for SARS-CoV-2, influenza virus, and respiratory syncytial virus (RSV) using RT‚ÄíPCR. Samples negative for these viruses were randomly selected for testing with the FilmArray Respiratory Panel, and the detection rates of other viruses were estimated. SARS-CoV-2, influenza virus, and RSV were detected in 598 (20.3%), 165 (5.6%), and 40 (1.4%) of the 2,949 samples with valid RT‚ÄíPCR results, respectively. The most prevalent viruses in the‚Äâ<‚Äâ6, 6‚Äì17, 18‚Äì64, and‚Äâ‚â•‚Äâ65¬†year age groups were rhinovirus/enterovirus, RSV, and SARS-CoV-2; influenza virus, seasonal coronavirus, and rhinovirus/enterovirus; SARS-CoV-2, seasonal coronavirus, and influenza virus; and SARS-CoV-2, seasonal coronavirus, and influenza virus, respectively. Significant differences in the detection rates of these viruses were detected between the age groups. This study highlights the importance of age-stratified molecular-based surveillance for a comprehensive understanding of the epidemiology of ARIs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-85068-7" target="_blank">https://www.nature.com/articles/s41598-024-85068-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-06</strong> <a href="https://www.nature.com/articles/s43856-024-00668-8" target="_blank">
            ü§ñ Effects of long-term treatment on people with long COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-249', this)">
            More</button></h4>
            <div id="toggle-249" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of extended-course oral nirmatrelvir/ritonavir in established Long COVID: a case series</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a rewritten version of the abstract:

Imagine you have Long COVID, which means you're still feeling sick long after your initial illness has passed. Researchers looked at 13 people with Long COVID who took an extended course (more than 5 days) of a medicine called Paxlovid.

They found that some people felt better and had fewer symptoms, but not everyone benefited the same way. Some even got worse. Two people who took Paxlovid while they were already sick from COVID eventually felt back to normal.

The researchers want to learn more about how this medicine works for different people with Long COVID. They hope to figure out which patients benefit most and what dosage is best, so they can recommend the treatment as an option for those struggling with Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundPrior case series suggest that a 5-day course of oral Paxlovid (nirmatrelvir/ritonavir) benefits some people with Long COVID, within and/or outside of the context of an acute reinfection. To the best of our knowledge, there have been no prior case series of people with Long COVID who have attempted longer courses of nirmatrelvir/ritonavir.MethodsWe documented a case series of 13 individuals with Long COVID who initiated extended courses (>5 days; range: 7.5‚Äì30 days) of oral nirmatrelvir/ritonavir outside (n=‚Äâ11) of and within (n=‚Äâ2) the context of an acute SARS-CoV-2 infection. Participants reported on symptoms and health experiences before, during, and after their use of nirmatrelvir/ritonavir.ResultsAmong those who take an extended course of nirmatrelvir/ritonavir outside of the context of an acute infection, some experience a meaningful reduction in symptoms, although not all benefits persist. Others experience no effect on symptoms. One participant stopped early due to intense stomach pain. For the two participants who took an extended course of nirmatrelvir/ritonavir within the context of an acute reinfection, both report eventually returning to their pre-re-infection baseline.ConclusionsExtended courses of nirmatrelvir/ritonavir may have meaningful benefits for some people with Long COVID but not others. We encourage researchers to study how and why nirmatrelvir/ritonavir benefits some and what course length is most effective, with the goal of informing clinical recommendations for using nirmatrelvir/ritonavir and/or other antivirals as a potential treatment for Long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00668-8" target="_blank">https://www.nature.com/articles/s43856-024-00668-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-06</strong> <a href="https://www.nature.com/articles/s41422-024-01060-w" target="_blank">
            ü§ñ Biological process that reduces inflammation in response to a specific chemical.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-250', this)">
            More</button></h4>
            <div id="toggle-250" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nonenzymatic lysined-lactylation induced by glyoxalase II substrate SLG dampens inflammatory immune responses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Immunometabolism is crucial in regulating immunity and inflammation. A mechanism preventing aberrant activation-induced immunopathology remains unclear. We found that glyoxalase II (GLO2) in the glycolysis branching pathway is downregulated by NF-Œ∫B signaling during innate immune activation, leading to its accumulation in the cytosol and the induction of nonenzymatic lactylation of proteins. This process is facilitated by a nearby cysteine residue that reacts with GLO2 to form a reversible intermediate. We found that this lactylation affects 2255 proteins mostly in activated macrophages, which are involved in immune activation and inflammation. Overexpression or mutation of GLO2 blocks this feedback lactylation, promoting inflammation but inhibiting immune homeostasis. Our findings suggest that the regulation of immunometabolism by GLO2 is closely linked to human inflammatory disorders.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Immunometabolism is critical in the regulation of immunity and inflammation; however, the mechanism of preventing aberrant activation-induced immunopathology remains largely unclear. Here, we report that glyoxalase II (GLO2) in the glycolysis branching pathway is specifically downregulated by NF-Œ∫B signaling during innate immune activation via tristetraprolin (TTP)-mediated mRNA decay. As a result, its substrateS-D-lactoylglutathione (SLG) accumulates in the cytosol and directly inducesd-lactyllysine modification of proteins. This nonenzymatic lactylation by SLG is greatly facilitated by a nearby cysteine residue, as it initially reacts with SLG to form a reversibleS-lactylated thiol intermediate, followed bySN-transfer of the lactyl moiety to a proximal lysine. Lactylome profiling identifies 2255 lactylation sites mostly in cytosolic proteins of activated macrophages, and global protein structure analysis suggests that proximity to a cysteine residue determines the susceptibility of lysine to SLG-mediatedd-lactylation. Furthermore, lactylation is preferentially enriched in proteins involved in immune activation and inflammatory pathways, andd-lactylation at lysine 310 (K310) of RelA attenuates inflammatory signaling and NF-Œ∫B transcriptional activity to restore immune homeostasis. Accordingly, TTP-binding site mutation or overexpression of GLO2 in vivo blocks this feedback lactylation in innate immune cells and promotes inflammation, whereas genetic deficiency or pharmacological inhibition of GLO2 restricts immune activation and attenuates inflammatory immunopathology both in vitro and in vivo. Importantly, dysregulation of the GLO2/SLG/d-lactylation regulatory axis is closely associated with human inflammatory phenotypes. Overall, our findings uncover an immunometabolic feedback loop of SLG-induced nonenzymaticd-lactylation and implicate GLO2 as a promising target for combating clinical inflammatory disorders.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41422-024-01060-w" target="_blank">https://www.nature.com/articles/s41422-024-01060-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-05</strong> <a href="https://www.nature.com/articles/s41598-024-84745-x" target="_blank">
            ü§ñ Fenofibrates' Solubility Improver</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-251', this)">
            More</button></h4>
            <div id="toggle-251" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Significantly improving the solubility and anti-inflammatory activity of fenofibric acid with native and methyl-substituted beta-cyclodextrins via complexation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Researchers have been studying how anti-inflammatory medicines work to treat chronic diseases like arthritis. One common medicine, fenofibric acid (FFA), has shown potential in reducing inflammation and pain associated with arthritis. To better understand its effects, scientists have created compounds that release the active ingredient into a special form, called cyclodextrins. These compounds help improve FFA's stability and effectiveness at preventing inflammation. The study found that combining FFA with cyclodextrins (a type of molecule) enhanced its anti-inflammatory properties and allowed it to be released more effectively over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The solubility of commonly used anti-inflammatory drugs has become a significant concern in contemporary medicine. Furthermore, inflammatory arthritis stands out as the most prevalent chronic inflammatory disease globally. The disease‚Äôs pathology is characterized by heightened inflammation and oxidative stress, culminating in chronic pain and the loss of joint functionality. Fenofibric acid (FFA) exhibits notable anti-inflammatory potential. This research assesses the anti-inflammatory effects of FFA, both in its standalone form and as inclusion complexes (ICs) with Œ≤-cyclodextrin and its methyl derivatives. FFA is encapsulated within the cavities of cyclodextrins (CDs), a fact confirmed by spectral changes observed in FFA. Distinct rock and seed-like morphologies are apparent for FFA with CDs, indicating that the CDs have influenced the surface of the guest. The introduction of CDs significantly enhances the thermal stability of FFA. ICs demonstrate superior results in inflammation activity compared to FFA alone. The efficacy of FFA complexed with CDs in mitigating inflammation positions it as a promising new drug. Additionally, our findings reveal that incorporating FFA into the CD cavity as a drug release system enhances the pharmacological profile of this substance, FFA.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84745-x" target="_blank">https://www.nature.com/articles/s41598-024-84745-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-04</strong> <a href="https://www.nature.com/articles/s41467-024-55682-0" target="_blank">
            ü§ñ RNA Substrate Preferences

Spontaneous base flipping allows a nucleotide to move to the 3' end of an RNA strand, changing its preferences for being incorporated into new RNA molecules. This process occurs naturally as RNA polymerase unwinds and then re-anneals in double-stranded DNA.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-252', this)">
            More</button></h4>
            <div id="toggle-252" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Spontaneous base flipping helps drive Nsp15‚Äôs preferences in double stranded RNA substrates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Coronaviruses evade detection by their host immune system by using a protein called endoribonuclease Nsp15 to break down viral double-stranded RNA. This enzyme targets specific bases on RNA, and its effectiveness depends on whether these bases are paired or not.

Researchers have been trying to understand how Nsp15 works, but their previous studies didn't give a clear answer. To figure this out, the researchers created special RNA molecules that show the same behavior as the natural virus. They then tested different conditions under which they were more likely to be broken down by Nsp15, and found that it was actually unpaired bases (those not paired) in these special RAs that made them vulnerable to Nsp15.

The study suggests that while Nsp15 can break down any base on the RNA, it is more effective at doing so when there are certain bulges or "bumps" on the RNA. This helps the researchers understand how Nsp15 works and how it affects different parts of the virus as it infects a host.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronaviruses evade detection by the host immune system with the help of the endoribonuclease Nsp15, which regulates levels of viral double stranded RNA by cleaving 3‚Ä≤ of uridine (U). While prior structural data shows that to cleave double stranded RNA, Nsp15‚Äôs target U must be flipped out of the helix, it is not yet understood whether Nsp15 initiates flipping or captures spontaneously flipped bases. We address this gap by designing fluorinated double stranded RNA substrates that allow us to directly relate a U‚Äôs sequence context to both its tendency to spontaneously flip and its susceptibility to cleavage by Nsp15. Through a combination of nuclease assays,19F NMR spectroscopy, mass spectrometry, and single particle cryo-EM, we determine that Nsp15 acts most efficiently on unpaired Us, particularly those that are already flipped. Across sequence contexts, we find Nsp15‚Äôs cleavage efficiency to be directly related to that U‚Äôs tendency to spontaneously flip. Overall, our findings unify previous characterizations of Nsp15‚Äôs cleavage preferences, and suggest that activity of Nsp15 during infection is partially driven by bulged or otherwise relatively accessible Us that appear at strategic positions in the viral RNA.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55682-0" target="_blank">https://www.nature.com/articles/s41467-024-55682-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-04</strong> <a href="https://www.nature.com/articles/s41598-025-85210-z" target="_blank">
            ü§ñ Covid-19 Treatment Predictions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-253', this)">
            More</button></h4>
            <div id="toggle-253" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Serial biomarker measurements may be helpful to predict the successful application of high flow nasal cannula in COVID-19 pneumonia patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>High flow nasal cannula (HFNC) reduces the need for intubation in patients with severe COVID-19 pneumonia. To predict success, researchers studied 194 patients with severe COVID-19. They found that CRP and D-dimer levels can help predict who will benefit from HFNC.

Higher CRP levels at the start of treatment were associated with better outcomes, particularly on Day 1. Lower D-dimer levels also predicted success.</p><br>
                <p><strong>Abstract</strong></p>
                <p>High flow nasal cannula (HFNC) can reduce the need for intubation in patients with coronavirus disease-19 (COVID-19) pneumonia induced acute hypoxemic respiratory failure (AHRF), but predictors of HFNC success could be characterized better. C-reactive protein (CRP) and D-dimer are associated with COVID-19 severity and progression. However, no one has evaluated the use of serial CRP and D-dimer ratios to predict HFNC success. We retrospectively studied 194 HFNC-treated patients admitted between August 2020 and October 2022. CRP and D-dimer levels relative to baseline at HFNC initiation were calculated up to three days thereafter. Intubated and non-intubated patient comparisons were assessed by the Kruskal-Wallis rank sum test and t-test. Ninety-two patients were intubated and 102 were not. Median CRP ratios were lower in non-intubated versus intubated patients (0.69 v. 0.96,p=‚Äâ0.050 for Day 1; 0.49 v. 0.61,p=‚Äâ0.028 for Day 2; 0.33 v. 0.64,p=‚Äâ0.008 for Day 3). D-dimer ratios did not change. CRP ratio monitoring in patients with AHRF due to COVID-19 within the first three days of HFNC application can serve as an objective adjunctive clinical tool to identify individuals who can continue to be supported with HFNC without escalating to invasive mechanical ventilation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-025-85210-z" target="_blank">https://www.nature.com/articles/s41598-025-85210-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-04</strong> <a href="https://www.nature.com/articles/s41598-024-81711-5" target="_blank">
            ü§ñ Inhibitors of NSP3-macrodomain
Synthesis and In Silico Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-254', this)">
            More</button></h4>
            <div id="toggle-254" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exploring pyrazolines as potential inhibitors of NSP3-macrodomain of SARS-CoV-2: synthesis and in silico analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Synthesis of new and potential antiviral compounds.

A chemical method has been developed to create these compounds, which can be used to fight COVID-19. This method involves reacting Œ≤-amino ketones with phenyl hydrazine in a solvent and then filtering the resulting product. The goal is to create molecules that can target a specific protein on the surface of SARS-CoV-2, called NSP3-microdomain (Mac-1). 

Computer simulations suggest that certain compounds produced from this method may be effective at inhibiting Mac-1 in the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 has proved to be a global health crisis during the pandemic, and the emerging JN.1 variant is a potential threat. Therefore, finding alternative antivirals is of utmost priority. In the current report, we present the synthesis of new and potential anti-viral pyrazoline compounds. Here we report a chemical scheme where Œ≤-aryl Œ≤-anilino ketones react with phenyl hydrazine in potassium hydroxide to give the corresponding 3,5-diarylpyrazoline. The protocol is applicable to a variety of Œ≤-amino ketones and tolerates several functional groups. This method is efficient and proceeds regioselectivity since the Œ≤-Anilino group acts as a protecting group for alkenes of chalcones. We identified the NSP3-microdomain (Mac-1) of SARS-CoV-2 as a putative target for newly synthesized triaryl-2-pyrazoline compounds. The molecular dynamics simulation-based free energy estimation suggests compounds 7a, 7d, 7¬†g, 7i, 7k, and 7¬†L as promising Mac-1 inhibitors. The detailed structural inspection of MD simulation trajectories sheds light on the structural and functional dynamics involved in the SARS-CoV-2 Mac-1. The data presented here is expected to guide the design and development of better anti-SARS-CoV-2 therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81711-5" target="_blank">https://www.nature.com/articles/s41598-024-81711-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-03</strong> <a href="https://www.nature.com/articles/s41598-024-77854-0" target="_blank">
            ü§ñ Infections from Aspergillus tamarii, especially in combination with COVID-19, can lead to severe complications. Research is needed to better understand this co-infection.

### TITLE ###
Combating Infections from Aspergillus tamarii by Study on COVID-19 Using In Vitro and In Silico Studies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-255', this)">
            More</button></h4>
            <div id="toggle-255" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Combating COVID-19 and its co-infection byAspergillus tamariiSP73-EGY using in vitro and in silico Studies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

A new treatment called the Aspergillus tamariiSP73-EGY isolate extract showed strong antiviral effects against several types of viruses, including COVID-19 and herpes viruses. This extract had high selectivity (meaning it targeted specific viruses) and was more effective than other treatments at killing these viruses. The extract also showed effectiveness against certain bacteria, such as E. coli and P. aeruginosa. Further research confirmed that the extract's antiviral properties are due to a natural compound called Kojic acid. This study provides potential new options for treating viral infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has caused significant mortality and morbidity for millions of people. Severe Acute Respiratory Syndrome-2 (SARS-CoV-2) virus is capable of causing severe and fatal diseases. We evaluated the antiviral properties ofAspergillus tamariiSP73-EGY isolate extract against low pathogenic coronavirus (229E), Adeno-7- and Herpes-2 viruses. The extract showed a high selectivity index (SI‚Äâ=‚Äâ43.4) and a significant inhibition of 229E (IC50=‚Äâ8.205¬†Œºg/ml). It was stronger than the drug control, remdesivir (IC50=‚Äâ38.2¬†Œºg/ml, SI‚Äâ=‚Äâ7.29).¬†However, the extract showed minimal efficacy against Adeno-7- and Herpes-2-Viruses (IC50=‚Äâ22.52, 47.79¬†Œºg/ml, and SI‚Äâ=‚Äâ6.75, 5.08, respectively). It exhibited profound efficacy against the highly pathogenic SARS-CoV-2 (IC50=‚Äâ8.306¬†Œºg/ml, SI‚Äâ=‚Äâ42.2). Kojic acid, the primary component of the extract, showed substantial antiviral activity against SARS-CoV-2 (IC50=‚Äâ23.4¬†Œºg/ml, SI‚Äâ=‚Äâ5.6), Remdesivir (IC50=‚Äâ4.55¬†Œºg/ml, SI‚Äâ=‚Äâ61.45). Therefore, the extract demonstrated the most notable antiviral characteristics against coronavirus infection. Co-infecting microorganisms may contribute to immune system deterioration and airway injury caused by SARS-CoV-2. The extract showed significant efficacy againstE. coliandP. aeruginosa,with an inhibition range of 3.5‚Äì10¬†mm at a concentration of 200¬†mg/ml. A molecular docking study showed that hexadecanoic, Kojic, octanoic acids, and 4(4-Methylbenzylidene)cyclohexane-1,3-dione have stronger binding affinity to the SARS-CoV-2 Mprothan Remdesivir. Molecular dynamics simulations were employed to examine the structural stability and flexibility of these complexes. This confirmed the high binding affinities of Kojic acid and 4(4-Methylbenzylidene)cyclohexane-1,3-dione, thereby proving their potential as novel anti-SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77854-0" target="_blank">https://www.nature.com/articles/s41598-024-77854-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-03</strong> <a href="https://www.nature.com/articles/s41564-024-01878-5" target="_blank">
            ü§ñ Phenotype Evolution of Spike During COVID-19 Pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-256', this)">
            More</button></h4>
            <div id="toggle-256" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Phenotypic evolution of SARS-CoV-2 spike during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

SARS-CoV-2 variants are different versions of the virus that cause COVID-19. Scientists have been studying these variations to understand how they affect the virus and its ability to infect cells. To do this, they created a panel of viruses with different spike proteins from 27 different variants found during the pandemic. They then tested these viruses in both laboratory and animal models. The results showed that some spikes (the protein part of the virus) had changed over time and were not as effective at infecting certain cells. However, some spikes still showed signs of their pre-pandemic versions, suggesting that new variants may emerge with similar characteristics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 variants are mainly defined by mutations in their spike. It is therefore critical to understand how the evolutionary trajectories of spike affect virus phenotypes. So far, it has been challenging to comprehensively compare the many spikes that emerged during the pandemic in a single experimental platform. Here we generated a panel of recombinant viruses carrying different spike proteins from 27 variants circulating between 2020 and 2024 in the same genomic background. We then assessed several of their phenotypic traits both in vitro and in vivo. We found distinct phenotypic trajectories of spike among and between variants circulating before and after the emergence of Omicron variants. Spike of post-Omicron variants maintained enhanced tropism for the nasal epithelium and large airways but displayed, over time, several phenotypic traits typical of the pre-Omicron variants. Hence, spike with phenotypic features of both pre- and post-Omicron variants may continue to emerge in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-024-01878-5" target="_blank">https://www.nature.com/articles/s41564-024-01878-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41421-024-00743-3" target="_blank">
            ü§ñ Intravenous mRNA vaccine for cancer treatment.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-257', this)">
            More</button></h4>
            <div id="toggle-257" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The guided fire from within: intratumoral administration of mRNA-based vaccines to mobilize memory immunity and direct immune responses against pathogen to target solid tumors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We investigated a new approach to treat certain types of cancer by introducing pathogen antigens into tumors using mRNA-lipid nanoparticles. This method effectively suppresses tumor growth and extends the lifespan of mice with existing immunity to these pathogens. We found that pre-existing immunity can enhance the effectiveness of this treatment, which combines intratumoral injections with a COVID-19 vaccine encoding the spike protein of SARS-CoV-2.

The treatment works by:

* Killing cancer cells displaying the pathogen antigen (spike protein)
* Reprogramming the tumor microenvironment to attract immune cells
* Triggering extensive tumor antigen-specific T cell responses

Combining this approach with an anti-PD-L1 therapy further boosted its effectiveness. The method could be used in various pathogens, including Hepatitis B Virus and Common Human Coronaviruses, making it a promising treatment option for different types of cancer.

This new approach expands therapeutic opportunities by using multiple pathogen antigens, which may help overcome potential drug resistance.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We investigated a novel cancer immunotherapy strategy that effectively suppresses tumor growth in multiple solid tumor models and significantly extends the lifespan of tumor-bearing mice by introducing pathogen antigens into tumors via mRNA-lipid nanoparticles. The pre-existing immunity against the pathogen antigen can significantly enhance the efficacy of this approach. In mice previously immunized with BNT162b2, an mRNA-based COVID-19 vaccine encoding the spike protein of the SARS-CoV-2 virus, intratumoral injections of the same vaccine efficiently tagged the tumor cells with mRNA-expressed spike protein. This action rapidly mobilized the pre-existing memory immunity against SARS-CoV-2 to kill the cancer cells displaying the spike protein, while concurrently reprogramming the tumor microenvironment (TME) by attracting immune cells. The partial elimination of tumor cells in a normalized TME further triggered extensive tumor antigen-specific T cell responses through antigen spreading, eventually resulting in potent and systemic tumor-targeting immune responses. Moreover, combining BNT162b2 treatment with anti-PD-L1 therapy yielded a more substantial therapeutic impact, even in ‚Äúcold tumor‚Äù types that are typically less responsive to treatment. Given that the majority of the global population has acquired memory immunity against various pathogens through infection or vaccination, we believe that, in addition to utilizing the widely held immune memory against SARS-CoV-2 via COVID-19 vaccine, mRNA vaccines against other pathogens, such as Hepatitis B Virus (HBV), Common Human Coronaviruses (HCoVs), and the influenza virus, could be rapidly transitioned into clinical use and holds great promise in treating different types of cancer. The extensive selection of pathogen antigens expands therapeutic opportunities and may also overcome potential drug resistance.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00743-3" target="_blank">https://www.nature.com/articles/s41421-024-00743-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41598-024-84519-5" target="_blank">
            ü§ñ Long COVID in transgender and gender nonbinary people in the United States

Long COVID affects many people who have experienced chronic illnesses that were severe enough to require hospitalization, but long-term effects are becoming more apparent. For some individuals, this can be a serious health issue.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-258', this)">
            More</button></h4>
            <div id="toggle-258" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID in transgender and gender nonbinary people in the United States</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research on Long COVID has not fully considered the experiences of transgender and nonbinary (TNB) individuals in the US.

A study of 1,234 adults with COVID-19 found that long COVID is a significant issue for this group, affecting 10% or more of those who experienced symptoms. The prevalence varies by gender identity:

* Transmasculine and nonbinary people assigned female sex at birth tend to have longer symptoms (4.8-12.9 months)
* Gender-affirming individuals are less likely to experience long COVID
* Hormone therapy may not be a factor in the development of long COVID for TNB individuals</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite recommendations in the US National Research Action Plan on Long COVID, gender identity is rarely reported in research and surveillance used to guide public health programming and clinical care. We analyzed data from a cross-sectional study of COVID-19 in a nationwide sample of transgender and nonbinary (TNB) people (N‚Äâ=‚Äâ2,134). Participants were surveyed between June 14, 2021 and May 1, 2022. Data were restricted to 817 participants who reported confirmed or suspected COVID-19 to estimate the prevalence of long COVID, defined as symptoms persisting for‚Äâ‚â•‚Äâ3¬†months. Ten percent of participants with a history of COVID-19 reported symptom duration consistent with long COVID, ranging from 4.8% to 12.9% across gender identities. Long COVID was most common in transmasculine and nonbinary people assigned female sex at birth. There was no evidence of an association with reported hormone therapy, supporting current recommendations that prioritize gender-affirming care during treatment for long COVID. As a condition which profoundly impacts health and productivity, long COVID is likely to exacerbate existing disparities. Principles of equity demand that we reduce barriers to prevention, diagnosis, and care for long COVID, and ensure that research and surveillance are inclusive of TNB people and disaggregate findings by gender identity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84519-5" target="_blank">https://www.nature.com/articles/s41598-024-84519-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41467-024-55421-5" target="_blank">
            ü§ñ ### Small Molecule Inhibitors of Coronavirus 3CL Protease from Screening Library

### INSTRUCTIONS FOR DEVELOPMENT OF SMALL MOLECULE NON-COVALENT CORONAVIRUS 3CL PROTEASE INHIBITORS FROM DNA-ENCODED CHEMICAL LIBRARY SCREENING</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-259', this)">
            More</button></h4>
            <div id="toggle-259" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Development of small molecule non-covalent coronavirus 3CL protease inhibitors from DNA-encoded chemical library screening</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new type of virus has emerged worldwide and is causing hundreds of deaths each week. Scientists are looking for new medicines that can stop this virus from multiplying in people's bodies. To find a new way to treat the virus, researchers looked at over 2 billion possible chemicals and found tiny molecules that can interfere with the virus's ability to multiply. These compounds have been shown to be effective against many types of viruses, including COVID-19. By studying how these compounds interact with a specific part of the virus called the 3CL protease, scientists were able to identify several new medicines that can help stop the virus from multiplying and reduce the risk of serious illness or death. These compounds may be used in combination with other medications like Paxlovid (nirmatrelvir), which has already been approved for use against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Variants of SARS-CoV-2 have continued to emerge across the world and cause hundreds of deaths each week. Due to the limited efficacy of vaccines against SARS-CoV-2 and resistance to current therapies, additional anti-viral therapeutics with pan-coronavirus activity are of high interest. Here, we screen 2.8 billion compounds from a DNA-encoded chemical library and identify small molecules that are non-covalent inhibitors targeting the conserved 3CL protease of SARS-CoV-2 and other coronaviruses. We perform structure-based optimization, leading to the creation of a series of potent, non-covalent SARS-CoV-2 3CL protease inhibitors, for coronavirus infections. To characterize their binding mechanism to the 3CL protease, we determine 16 co-crystal structures and find that optimized inhibitors specifically interact with both protomers of the native homodimer of 3CL protease. Since 3CL protease is catalytically competent only in the dimeric state, these data provide insight into the design of drug-like inhibitors targeting the native homodimer state. With a binding mode different from the covalent 3CL inhibitor nirmatrelvir, the protease inhibitor in the COVID drug Paxlovid, these compounds may overcome resistance reported for nirmatrelvir and complement its clinical utility.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55421-5" target="_blank">https://www.nature.com/articles/s41467-024-55421-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41467-024-55358-9" target="_blank">
            ü§ñ ## Binding of SARS-CoV-2 to the Host Cell Membrane

## Using Atomic Force Microscopy (AFM) to Study Single-Molecule Dynamics at the Molecular Level.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-260', this)">
            More</button></h4>
            <div id="toggle-260" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Probing SARS-CoV-2 membrane binding peptide via single-molecule AFM-based force spectroscopy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The spike protein on the COVID-19 virus (SARS-CoV-2) has a special job: it helps bind to the membranes of host cells, allowing the virus to enter and infect them. Researchers investigated how this binding works by studying two different versions of a protein found in the viral spike protein. They found that when these proteins are present together with cholesterol from host cell membranes, they can attach more easily and become more efficient at entering host cells. However, removing cholesterol from the membrane makes it harder for the virus to infect. Additionally, this part of the protein appears to play a role in helping the virus stay attached to the host cell membrane. These findings suggest that targeting a specific part of the viral spike protein could be an effective way to prevent infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 spike protein‚Äôs membrane-binding domain bridges the viral and host cell membrane, a critical step in triggering membrane fusion. Here, we investigate how the SARS-CoV-2 spike protein interacts with host cell membranes, focusing on a membrane-binding peptide (MBP) located near the TMPRSS2 cleavage site. Through in vitro and computational studies, we examine both primed (TMPRSS2-cleaved) and unprimed versions of the MBP, as well as the influence of its conserved disulfide bridge on membrane binding. Our results show that the MBP preferentially associates with cholesterol-rich membranes, and we find that cholesterol depletion significantly reduces viral infectivity. Furthermore, we observe that the disulfide bridge stabilizes the MBP‚Äôs interaction with the membrane, suggesting a structural role in viral entry. Together, these findings highlight the importance of membrane composition and peptide structure in SARS-CoV-2 infectivity and suggest that targeting the disulfide bridge could provide a therapeutic strategy against infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55358-9" target="_blank">https://www.nature.com/articles/s41467-024-55358-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41598-024-84598-4" target="_blank">
            ü§ñ Long-term COVID-19 cases in Hainan Province, China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-261', this)">
            More</button></h4>
            <div id="toggle-261" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence and risk factors of long COVID-19 persisting for 2 years in Hainan Province: a population-based prospective study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new strain of coronavirus has led to a lingering illness called persistent symptoms and complications that can last for weeks or months after recovery. Researchers studied 960 people who had been infected with COVID-2 in China to understand more about long-term effects. They found that about 12.5% of participants experienced prolonged health problems, including fatigue, hair loss, and dizziness. Certain factors increased the risk of experiencing these complications, such as being over 65 years old or having chronic diseases, irregular eating habits, not getting enough sleep, feeling anxious, or not getting vaccinated regularly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) can lead to persistent symptoms, sequelae, and other medical complications that may last for weeks or months after recovery. The aim of the study is to assess the prevalence and risk factors of long COVID-19 persisting for 2 years in Hainan Province, China, to aid in its recognition, prevention, and treatment. Between July and August 2022, 960 individuals with confirmed SARS-CoV-2 infection in Hainan, China, were recruited. An epidemiological questionnaire was conducted via phone interviews to assess participants‚Äô recovery status after 2 years. Among the participants, 120 patients (12.5%) experienced at least one long COVID-19 complication. The most common symptoms were cough (33.3%, 40/120), followed by fatigue (25.9%, 31/120), hair loss (23.3%, 28/120), and dizziness (20.8%, 25/120). Independent risk factors included age over 65, moderate to severe infection, chronic diseases, irregular diet, late sleeping, anxiety, and fewer than 2 vaccinations (p<‚Äâ0.05). While most individuals infected with COVID-19 fully recover, approximately 12.5% experience intermediate or long-term effects. This study is the first to identify the incidence and associated risk factors of long COVID-19 with the longest follow-up time, providing valuable insights for the timely restoration of pre-COVID-19 health.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84598-4" target="_blank">https://www.nature.com/articles/s41598-024-84598-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41598-024-84100-0" target="_blank">
            ü§ñ Researching the impact of vaccines on COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-262', this)">
            More</button></h4>
            <div id="toggle-262" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Healthcare workers safety: a cohort study using healthcare utilisation databases on vaccination and vaccine timeliness impact against SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Healthcare workers (HCWs) are at high risk of contracting COVID-19, potentially spreading it to patients. A study looked at how vaccination affects SARS-CoV-2 infections among HCWs in Italy's Marche region. 

The study involved 21,118 HCWs aged 18-70 from five health authorities, who were fully vaccinated or not. It analyzed factors such as full vaccination rates and its association with infection risk.

High levels of full vaccination (81%) reduced the risk of SARS-CoV-2 infections by 77%. Factors associated with higher vaccination rates included age, role in healthcare, prior exposure to COVID-19, and health status. The study also found that vaccination timeliness is crucial for reducing infection risk among HCWs.

Overall, vaccination is essential to protect against COVID-19 spread, regardless of an individual's characteristics or job roles.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Healthcare Workers (HCWs) are at ongoing risk of SARS-CoV-2 infection, potentially contributing to its transmission. This study assessed full vaccination and vaccination timeliness impact on SARS-CoV-2 infections among HCWs in Italy‚Äôs Marche Region, using Healthcare Utilization Databases. We evaluated vaccination coverage and its associated factors. The cohort comprised 21,118 HCWs aged 18‚Äì70 from the region‚Äôs five Local Health Authorities (LHA), enrolled between February 2020¬†-¬†May 2021. Factors associated with full vaccination were assessed using multiple logistic regression. The impact of vaccination status, time to vaccination, occupational role, age, gender, and health status on infection risk was analysed with a multiple Cox regression model, adjusting for vaccination coverage velocity, swabbing probability, and monthly intensive care unit admissions rate in each LHA. Of the cohort, 81.2% were fully vaccinated. Factors associated with full vaccination included age, role, LHA, prior infection, and health status. Vaccination reduced infection risk by 77% (95%¬†CI: 70‚Äì82). Infection risk was higher among healthcare assistants, nurses/physiotherapists/technicians compared to physicians, among male HCWs, and it decreased as vaccination timeliness increased. Vaccination timeliness is crucial for reducing SARS-CoV-2 infection risk among HCWs, regardless of their characteristics. This underscores the importance of efficiently organizing vaccination administration across different territories and for all HCW categories.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84100-0" target="_blank">https://www.nature.com/articles/s41598-024-84100-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2025-01-02</strong> <a href="https://www.nature.com/articles/s41598-024-84586-8" target="_blank">
            ü§ñ The COVID-19 pandemic has significantly impacted global health trends. In this region, we can identify one-year patterns of respiratory pathogens using epidemiology.

### ONE-YEAR EPIDEMIOLOGICAL PATTERNS OF RESPIRATORY PATHOGENS AROUND AGE, GENDER, AND SEASONS IN CHENGDU DURING THE POST-COVID ERA</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-263', this)">
            More</button></h4>
            <div id="toggle-263" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>One-year epidemiological patterns of respiratory pathogens across age, gender, and seasons in Chengdu during the post-COVID era</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study analyzed the most common causes of respiratory infections in a hospital in Sichuan, China. It found that many people got infected with one or more of six types of viruses: Chlamydia pneumoniae, Mycoplasma pneumoniae, Adenovirus, Influenza A virus, Influenza B virus, and Respiratory Syncytial Virus. The study involved over 22,000 patients and found that most people got infected with one or more of these viruses. Each type of virus had its own highest rate of infection, but Mycoplasma pneumoniae had the highest overall rate. Some people were also found to have co-infections with other viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory tract infections caused by various pathogens remain a significant public health concern due to their high prevalence and potential for severe complications. This study systematically analyzed the epidemiological characteristics of six common respiratory pathogens‚ÄîChlamydia pneumoniae(CP),Mycoplasma pneumoniae(MP), Adenovirus (AdV), Influenza A virus (FluA), Influenza B virus (FluB), and Respiratory Syncytial Virus (RSV)‚Äîin patients from Sichuan Jinxin Xinan Women and Children‚Äôs Hospital between April 2023 and March 2024. Throat swab samples were collected from a total of 22,717 individuals. Each sample was processed using the AUTOMOLEC 3000 analyzer and the PCR-fluorescent probe method. The results showed that 10,171 (44.8%) individuals tested positive for at least one pathogen. MP had the highest overall positive rate (21.83%), followed by FluA (17.50%) and FluB (14.84%). MP showed the highest mean monthly (average) positive rate (16.84% ¬± 8.41). Significant differences were found between MP and AdV, CP and RSV in average positive rate (p<‚Äâ0.05). Co-infection analysis revealed frequent associations between MP and AdV, MP and CP, and FluB with MP. Seasonal analysis indicated distinct peaks: FluA and FluB in winter, RSV in spring, and MP in summer, autumn and winter. Age-stratified analysis showed higher positivity rates of RSV in children aged 0‚Äì6 years, MP and CP in the 7‚Äì17 years group. Gender-based differences were only observed in RSV positive samples. These findings provide crucial insights into the prevalence and seasonal distribution of respiratory pathogens in Chengdu, offering valuable data to inform public health strategies in the post-COVID era.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84586-8" target="_blank">https://www.nature.com/articles/s41598-024-84586-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-31</strong> <a href="https://www.nature.com/articles/s41598-024-81990-y" target="_blank">
            ü§ñ An experimental method to study the activity of the SARS-CoV-2 3CL-protease enzyme.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-264', this)">
            More</button></h4>
            <div id="toggle-264" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An isothermal calorimetry assay for determining steady state kinetic and Ensitrelvir inhibition parameters for SARS-CoV-2 3CL-protease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study used Isothermal Titration Calorimetry to characterize the kinetics of the SARS-CoV-2 protease, 3CLpro. The ITC method allowed researchers to measure how much energy is required to react with a substance (in this case, Ensitrelvir), and it helped them understand how fast or slowly the enzyme works. This information can be used to find the rate at which the virus multiplies in cells and to develop new drugs that target the virus more effectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This manuscript details the application of Isothermal Titration Calorimetry (ITC) to characterize the kinetics of 3CLpro, the main protease from the Severe Acute Respiratory Syndrome CoronaVirus-2 (SARS-CoV-2), and its inhibition by Ensitrelvir, a known non-covalent inhibitor. 3CLprois essential for producing the proteins necessary for viral infection, which led to the COVID-19 pandemic. The ITC-based assay provided rapid and reliable measurements of 3CLproactivity, allowing for the direct derivation of the kinetic enzymatic constantsKMandkcatby monitoring the thermal power required to maintain a constant temperature as the substrate is consumed. The manuscript highlights several advantages of the proposed ITC-based assay over traditional methods used to study 3CLpro, such as F√∂rster Resonance Energy Transfer (FRET) and Liquid Chromatography-Mass Spectrometry (LC‚ÄìMS) and overcomes the need for non-biological substrates or discontinuous post-reaction steps. The ease of application of the ITC method allowed for the determination of the temperature dependence of the catalytic constants, enabling the estimation of the reaction activation energy. Additionally, the assay was used to determine the inhibition mode and kinetic parameters for 3CLproinhibition by Ensitrelvir. This molecule was revealed to act as a slow- and tight-binding inhibitor that forms an initial E‚Ä¢I complex (KI=‚Äâ9.9‚Äâ¬±‚Äâ0.7¬†nM) quickly transitioning to a tighter E‚Ä¢I* assembly (KI*‚Äâ=‚Äâ1.1‚Äâ¬±‚Äâ0.2¬†nM). This versatile calorimetric method is proposed for general use in the discovery and development of drugs targeting 3CLpro.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81990-y" target="_blank">https://www.nature.com/articles/s41598-024-81990-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-31</strong> <a href="https://www.nature.com/articles/s41598-024-84276-5" target="_blank">
            ü§ñ Evolution of Vaccine Components in COVID-19 Treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-265', this)">
            More</button></h4>
            <div id="toggle-265" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evolution of SARS-CoV-2 spike trimers towards optimized heparan sulfate cross-linking and inter-chain mobility</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract ###
The outer layer of COVID-19 virus particles helps bind to receptors on cells that cause disease. However, this binding site is made up of a type of protein called heparan sulfate (HS). Research suggests that some versions of the virus may change their shape so they can easily bind to the HS layer. But how do these changes affect the way the virus moves inside host cells? Scientists have found that Omicron, one of the variants of COVID-19, has developed a special binding site on its outer proteins that helps it move around more easily through the complex outer layer of infected cells. This property is achieved by allowing multiple HS molecules to bind simultaneously, enabling the virus to switch between different chains. By targeting these interactions with pharmaceutical substances, scientists believe they can interfere with the disease-causing process and potentially protect against future infections caused by similar variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The heparan sulfate (HS)-rich extracellular matrix (ECM) serves as an initial interaction site for the homotrimeric spike (S) protein of SARS-CoV-2 to facilitate subsequent docking to angiotensin-converting enzyme 2 (ACE2) receptors and cellular infection. More recent variants, notably Omicron, have evolved by swapping several amino acids to positively charged residues to enhance the interaction of the S-protein trimer with the negatively charged HS. However, these enhanced interactions may reduce Omicron‚Äôs ability to move through the HS-rich ECM to effectively find ACE2 receptors and infect cells, raising the question of how to mechanistically explain HS-associated viral movement. In this work, we show that Omicron S proteins have evolved to balance HS interaction stability and dynamics, resulting in enhanced mobility on an HS-functionalized artificial matrix. This property is achieved by the ability of Omicron S-proteins to cross-link at least two HS chains, allowing direct S-protein switching between chains as a prerequisite for cell surface mobility. Optimized HS interactions can be targeted pharmaceutically, as an HS mimetic significantly suppressed surface binding and cellular infection specifically of the Omicron variant. These findings suggest a robust way to interfere with SARS-CoV-2 Omicron infection and potentially future variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-84276-5" target="_blank">https://www.nature.com/articles/s41598-024-84276-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41598-024-82660-9" target="_blank">
            ü§ñ Analysis of Respiratory Pathogen Detection in Hospitalized Children After Zero COVID Policy Ending</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-266', this)">
            More</button></h4>
            <div id="toggle-266" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Analysis of respiratory pathogen detection in hospitalized children with acute respiratory tract infections after ending the zero COVID policy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study analyzed the infection patterns of certain respiratory viruses in children before and after a three-year "zero COVID" policy in China was ended, which limited contact with other countries where these viruses are common. The researchers collected nasal aspirates from 42,379 hospitalized children with acute respiratory tract infections over two periods: before the policy and six months after it ended. They found that certain viruses such as human rhinovirus, RSV, and adenovirus were more prevalent in children during this period.</p><br>
                <p><strong>Abstract</strong></p>
                <p>After ending the three-year zero COVID policy in China, the epidemiology of other respiratory pathogens has been affected. This study aimed to characterize of common respiratory pathogen infections in pediatric patients hospitalized for acute respiratory tract infections (ARTIs) in Suzhou before and after ending the zero COVID policy. Nasopharyngeal aspirates (NPAs) were obtained from children with ARTIs (aged‚Äâ‚â§‚Äâ16 years) at the Children‚Äôs Hospital of Soochow University for the detection of respiratory syncytial virus (RSV), influenza A (FluA), FluB, human parainfluenza virus (HPIV), adenovirus (ADV), human rhinovirus (HRV), bocavirus (BoV), human metapneumovirus (HMPV), and mycoplasma pneumoniae (MP). The data were compared between two periods: January 2020 to December 2022 (before ending the zero COVID policy) and January 2023 to May 2024 (after ending the zero COVID policy). Patients were divided into four groups: 0‚Äì2, ‚â•‚Äâ3‚Äì5, ‚â•‚Äâ6‚Äì10, and ‚â•‚Äâ11‚Äì16 years. A total of 42,379 patients were enrolled and the top four pathogens identified were MP, HRV, RSV and HPIV with positive rates of 20.2%, 19.5%, 15.1%, and 6.9%, respectively. A total of 28,352 positive cases were detected, with positive rates of 54.0% (n=‚Äâ11,850/21,941) and 80.7% (n=‚Äâ16,502/20,438) before and after ending the zero COVID policy, respectively. Total RSV, HRV, HPIV, and MP positivity increased by 27.8%, 39.0%, 12.3%, and 322.7%, respectively, after ending the zero COVID policy compared to positivity before the policy. After ending the zero COVID policy, the positive rates of RSV, HRV, and HPIV increased most in children aged 0‚Äì2 years, with increases by 88.8% (OR: 2.3, 95% CI: 2.2‚Äì2.5), 50.0% (OR: 1.6, 95% CI: 1.5‚Äì1.7), and 69.6% (OR: 1.8, 95% CI: 1.6-2.0), respectively. The greatest increase in MP positivity was 316.9% in the 3‚Äì5 years (OR: 5.5, 95% CI: 4.9‚Äì6.1). After ending the zero COVID policy, the RSV-positive rate increased most in summer, while HRV was predominantly circulated in spring and the MP-positive rate peaked in autumn. Ending the zero COVID policy facilitated the transmission of common respiratory pathogens in children. Post-pandemic surveillance and control of respiratory pathogens must be strengthened to reduce health risks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82660-9" target="_blank">https://www.nature.com/articles/s41598-024-82660-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41598-024-83232-7" target="_blank">
            ü§ñ Mild stomach issues that can develop after a COVID-19 infection, especially among young adults and those living in urban areas like the Bronx.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-267', this)">
            More</button></h4>
            <div id="toggle-267" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>New-onset gastrointestinal disorders in COVID-19 patients 3.5 years post-infection in the inner-city population in the Bronx</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study examined new gastrointestinal disorders (GID) in people who were infected with COVID-19 more than 3 years after their initial illness. They looked at how different factors, such as smoking and obesity, affected the risk of these conditions developing. The results showed that people who became infected with COVID-19 had a higher chance of developing certain GID symptoms, including peptic ulcers, inflammatory bowel disease, and irritable bowel syndrome, than those without the virus. These findings suggest that more research is needed to understand the long-term risks associated with COVID-19 infection for individuals at risk.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study examined the incidence, characteristics, and risk factors of new gastrointestinal disorders (GID) associated with SARS-CoV-2 infection up to 3.5 years post-infection. This retrospective study included 35,102 COVID-19 patients and 682,594 contemporary non-COVID-19 patients without past medical history of GID (controls) from the Montefiore Health System in the Bronx (3/1/2020 to 7/31/2023). Comparisons were made with unmatched and propensity-matched (1:2) controls. The primary outcome was new GID which included peptic ulcer, inflammatory bowel disease, irritable bowel syndrome, diverticulosis, diverticulitis, and biliary disease. Multivariate Cox proportional hazards model analysis was performed with adjustment for covariates. There were 2,228 (6.34%) COVID-19 positive patients who developed new GID compared to 38,928 (5.70%) controls. COVID-19 patients had an elevated risk of developing new GID (adjusted HR‚Äâ=‚Äâ1.18 (95% CI 1.12‚Äì1.25) compared to propensity-matched controls, after adjusting for confounders that included smoking, obesity, diabetes, hypertension. These findings underscore the need for additional research and follow-up of at-risk individuals for developing GID post infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-83232-7" target="_blank">https://www.nature.com/articles/s41598-024-83232-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41598-024-82098-z" target="_blank">
            ü§ñ Regulation of Paneth cell genes in COVID-19 and SARS-CoV-2 infection through stool samples analysis.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-268', this)">
            More</button></h4>
            <div id="toggle-268" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Regulation of Paneth cell-specific genes in COVID-19 patients and SARS-CoV-2-infected mice by quantification of mRNA from exfoliated cells in stool samples</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied how certain proteins produced by Paneth cells in the small intestine help defend against infections. They compared gene expression levels from stool samples taken from people who were infected with COVID-19 and those who weren't, to see if they could detect differences in these genes that might indicate Paneth cell activity. The results showed a decrease in certain proteins, which is what would be expected for an infection, but not other changes. This study provides new information on how Paneth cells react to viral infections, which may help us understand the mechanism by which viruses cause illness and make it harder for our bodies to fight them off.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Paneth cell, a secretory cell of the small intestine, expresses numerous host defense proteins, and is hypothesized to play an important role in host defense against infection. However, studying gene expression in this cell requires invasive procedures. To test the hypothesis that we could observe Paneth cell-specific gene regulation from exfoliated cells in infectious conditions, we obtained stool samples from patients with COVID-19 and uninfected controls. Total mRNA was isolated, and Paneth cell-specific and non-specific gene expression was quantified by RT-PCR. Results revealed a significant decrease in mRNA levels from Paneth cell-specific genes, includingDEFA5,DEFA6,PLA2G2A,PRSS2andITLN2in SARS-CoV-2 positive patients compared with controls. Other gut markers, not specific to Paneth cells were unchanged. To validate this experimentally, we infected mice with SARS-CoV-2 and collected fecal pellets over the course of 7 days. We observed a similar time-dependent reduction in Paneth cell-specific transcripts, which correlates with histological changes in the gut. This is the first demonstration of quantification of Paneth cell-specific transcripts without invasive sampling. It also shows the coordinate regulation of these genes as a response to infection with SARS-CoV-2, possibly through viral pathogenesis, to increase infectivity in the gut.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82098-z" target="_blank">https://www.nature.com/articles/s41598-024-82098-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41467-024-55076-2" target="_blank">
            ü§ñ Organoid-based production of human rhinoviruses for viral propagation and study.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-269', this)">
            More</button></h4>
            <div id="toggle-269" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Human respiratory organoids sustained reproducible propagation of human rhinovirus C and elucidation of virus-host interaction</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Researchers have developed a new system to grow a common type of virus called HRV-C, which is difficult to cultivate in standard cell lines. They created organoids (3D cultures of cells) that can sustain the growth of HRV-C for a longer period. The researchers found that these airway organoids were better at supporting the growth of HRV-C than nasal organoids. When infected with HRV-C, some of these airway organoids showed an increased immune response and responded more strongly to certain treatments compared to nasal organoids. This new system could help improve our understanding of how viruses infect cells and develop effective treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The lack of a robust system to reproducibly propagate HRV-C, a family of viruses refractory to cultivation in standard cell lines, has substantially hindered our understanding of this common respiratory pathogen. We sought to develop an organoid-based system to reproducibly propagate HRV-C, and characterize virus-host interaction using respiratory organoids. We demonstrate that airway organoids sustain serial virus passage with the aid of CYT387-mediated immunosuppression, whereas nasal organoids that more closely simulate the upper airway achieve this without any intervention. Nasal organoids are more susceptible to HRV-C than airway organoids. Intriguingly, upon HRV-C infection, we observe an innate immune response that is stronger in airway organoids than in nasal organoids, which is reproduced in a Poly(I:C) stimulation assay. Treatment with Œ±-CDHR3 and antivirals significantly reduces HRV-C viral growth in airway and nasal organoids. Additionally, an organoid-based immunofluorescence assay is established to titrate HRV-C infectious particles. Collectively, we develop an organoid-based system to reproducibly propagate the poorly cultivable HRV-C, followed by a comprehensive characterization of HRV-C infection and innate immunity in physiologically active respiratory organoids. The organoid-based HRV-C infection model can be extended for developing antiviral strategies. More importantly, our study has opened an avenue for propagating and studying other uncultivable human and animal viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55076-2" target="_blank">https://www.nature.com/articles/s41467-024-55076-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41467-024-55187-w" target="_blank">
            ü§ñ Immune system cell weakness leading to kidney failure</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-270', this)">
            More</button></h4>
            <div id="toggle-270" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The survival of B cells is compromised in kidney disease</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Antibody-mediated protection against pathogens is essential for maintaining good health. However, people with pre-existing conditions such as kidney disease can be more susceptible to infections and have a weaker immune response to vaccinations.

A study found that the kidneys play a key role in protecting our bodies from certain diseases, including those caused by viruses like COVID-19. When we are sick or have weakened immunity, our kidneys' ability to fight off infection is impaired. This means people with kidney disease can get infections more easily and may not respond well to vaccinations.

The research used various mouse models of kidney disease to study how the kidneys affect immune responses in animals. It found that the kidneys' dysfunction can prevent certain types of white blood cells called germinal center B cells from fighting off infections.

The study also discovered that a toxic substance produced by the kidneys can cause these white blood cells to die, leading to increased apoptosis (cell death). This is especially true for people with kidney disease who are already vulnerable to infection.

The researchers found that this dysfunction not only makes people more susceptible to certain illnesses but also affects their ability to fight off infections such as influenza. Overall, the study provides a better understanding of how the kidneys' damage can suppress our immune system in individuals with pre-existing conditions like kidney disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Antibody-mediated protection against pathogens is crucial to a healthy life. However, the recent SARS-CoV-2 pandemic has shown that pre-existing comorbid conditions including kidney disease account for compromised humoral immunity to infections. Individuals with kidney disease are not only susceptible to infections but also exhibit poor vaccine-induced antibody response. Using multiple mouse models of kidney disease, we demonstrate that renal dysfunction inhibits germinal center (GC) response against T-dependent antigens. GC B cells exhibit increased apoptosis in kidney disease. Uremic toxin hippuric acid drives loss of mitochondrial membrane potential, leading to increased apoptosis of GC B cells in a G-protein‚Äìcoupled receptor 109A dependent manner. Finally, GC B cells and antibody titer are diminished in mice with kidney disease following influenza virus infection, a major cause of mortality in individuals with renal disorders. These results provide a mechanistic understanding of how renal dysfunction suppresses humoral immunity in patients with kidney disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55187-w" target="_blank">https://www.nature.com/articles/s41467-024-55187-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41598-024-80218-3" target="_blank">
            ü§ñ Optimal Control Strategies for COVID-19 Dynamics Using Atangana-Baleanu Derivatives</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-271', this)">
            More</button></h4>
            <div id="toggle-271" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exploring optimal control strategies in a nonlinear fractional bi-susceptible model for Covid-19 dynamics using Atangana-Baleanu derivative</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new coronavirus disease (Covid-19) model has been developed using a nonlinear fractional bi-susceptible S1S2V1V2IHR system in Caputo sense with the Atangana-Baleanu derivative. The model is used to study its dynamics and examines its fundamental properties such as positivity and boundedness, which ensure it accurately models real-world disease spread within a population. Analytical results are validated using recent techniques, highlighting the ABC fractional derivative's significance. The model includes quantitative analysis of control measures, including vaccination and hospitalization rates, aiming to expedite pandemic elimination. A sensitivity analysis reveals the highest sensitive parameters in the model, leading to optimal control strategies that effectively manage infected and susceptible individuals. Numerical simulations demonstrate the effectiveness of these strategies in reducing financial expenses and infection rates. This study applies advanced mathematical techniques for a comprehensive approach to Covid-19 modeling, analysis, and disease management.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In this article, a nonlinear fractional bi-susceptible\(S_{1}S_{2}V_{1}V_{2}IHR\)model is developed to mathematically study the deadly Coronavirus disease (Covid-19), employing the Atangana-Baleanu derivative in Caputo sense (ABC). A more profound comprehension of the system‚Äôs intricate dynamics using fractional-order derivative is explored as the primary focus of constructing this model. The fundamental properties such as positivity and boundedness, of an epidemic model have been proven, ensuring that the model accurately reflects the realistic behavior of disease spread within a population. The asymptotic stabilities of the dynamical system at its two main equilibrium states are determined by the essential conditions imposed on the threshold parameter. The analytical results acquired are validated and the significance of the ABC fractional derivative is highlighted by employing a recently proposed Toufik-Atangana numerical technique. A quantitative analysis of the model is conducted by adjusting vaccination and hospitalization rates using constant control techniques. It is suggested by numerical experiments that the Covid-19 pandemic elimination can be expedited by adopting both control measures with appropriate awareness. The model parameters with the highest sensitivity are identified by performing a sensitivity analysis. An optimal control problem is formulated, accompanied by the corresponding Pontryagin-type optimality conditions, aiming to ascertain the most efficient time-dependent controls for susceptible and infected individuals. The effectiveness and efficiency of optimally designed control strategies are showcased through numerical simulations conducted before and after the optimization process. These simulations illustrate the effectiveness of these control strategies in mitigating both financial expenses and infection rates. The novelty of the current study is attributed to the application of the structure-preserving Toufik-Atangana numerical scheme, utilized in a backward-in-time manner, to comprehensively analyze the optimally designed model. Overall, the study‚Äôs merit is found in its comprehensive approach to modeling, analysis, and control of the Covid-19 pandemic, incorporating advanced mathematical techniques and practical implications for disease management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80218-3" target="_blank">https://www.nature.com/articles/s41598-024-80218-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41467-024-55087-z" target="_blank">
            ü§ñ Trivalent recombinant protein vaccine to combat specific virus strains.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-272', this)">
            More</button></h4>
            <div id="toggle-272" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Trivalent recombinant protein vaccine induces cross-neutralization against XBB lineage and JN.1 subvariants: preclinical and phase 1 clinical trials</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new vaccine called Tri-Vac helps protect against COVID-19 by using specific proteins from the virus to trigger an immune response. This study created three types of vaccines from these proteins and tested them in mice, which showed great success in inducing antibodies that fight the virus. The vaccine also induced strong immune responses in other cells involved in fighting infections. It's been shown to work well for many different variants of the COVID-19 virus, including some that are still spreading. Researchers hope this vaccine can be used as a booster shot after getting vaccinated with something else, and preliminary results suggest it might even help protect against new variants like JN.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The immune escape capacities of XBB variants necessitate the authorization of vaccines with these antigens. In this study, we produce three recombinant trimeric proteins from the RBD sequences of Delta, BA.5, and XBB.1.5, formulating a trivalent vaccine (Tri-Vac) with an MF59-like adjuvant at a 1:1:4 ratio. Tri-Vac demonstrates immunogenicity in female NIH mice, inducing cross-neutralization against various SARS-CoV-2 variants, including pre-Omicron and Omicron BA.2.75, BA.5, and XBB lineages. It elicits measurable antigen-specific T cell responses, germinal center B cell responses, and T follicular helper responses, effectively protecting against live Omicron XBB.1.16 challenges. Protective immunity is maintained long-term, with sustained neutralizing antibodies and T cell responses, as well as memory B cells and long-lived plasma cells observed by day 210 post-immunization. Tri-Vac also serves as a candidate booster for enhancing immunity after three doses of inactivated virus or mRNA vaccines. A phase 1 investigator-initiated trial was initiated to assess safety and immunogenicity in humans, focusing on the primary endpoint of adverse reactions within 7 days and key secondary endpoints including the geometric mean titers (GMTs) of serum neutralizing antibodies within 30 days and 6 months post-vaccination, as well as adverse events within 30 days and serious adverse events within 6 months post-vaccination. Preliminary data indicate Tri-Vac has good safety and immunogenicity, improving neutralization against multiple variants, including JN.1, in previously vaccinated individuals, highlighting its clinical potential for protecting against SARS-CoV-2 variants. The registration number of this clinical trial is ChiCTR2200067245.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55087-z" target="_blank">https://www.nature.com/articles/s41467-024-55087-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s44298-024-00083-9" target="_blank">
            ü§ñ Diagnostic Tool for COVID-19 Neutralization</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-273', this)">
            More</button></h4>
            <div id="toggle-273" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Automated and virus variant-programmable surrogate test qualitatively compares to the gold standard SARS-CoV-2 neutralization assay</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>New severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants require rapid testing that can be done quickly and accurately using new technologies.

Assays for measuring the protection of antibodies against SARS-CoV-2 are usually done on cell cultures or with specific viral particles, but this can be a problem. To solve this, scientists have developed a different type of test called "surrogate virus neutralization tests" that measures how well the body's immune system can block the binding of the virus to its receptor. This new test uses tiny pieces of the SARS-CoV-2 virus and can be done quickly and accurately, making it a good alternative to traditional testing methods.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ongoing emergence of new severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants underscores the need for rapid, adaptable, high-throughput testing. However, assays for neutralizing antibodies, which are a good measure of viral protection, usually require cell culture and either infectious SARS-CoV-2 or pseudotyped viral particles. To circumvent the challenges of cell-based assays, SARS-CoV-2 surrogate virus neutralization tests (sVNTs) measure inhibition of the binding of the spike (S) protein receptor binding domain (RBD) to its receptor, human angiotensin-converting enzyme 2 (hACE2) by neutralizing antibodies. Here we tested a prototype automated microfluidic cartridge-based sVNT platform using SARS-CoV-2 wild-type (WT) and B.1.617.2 (Delta) variant RBDs. This sVNT showed a high correlation with cell-based neutralization assays for biospecimens collected post-COVID-19 vaccination and post-SARS-CoV-2 infection as well as for pre-pandemic SARS-CoV-2 negative sera. Thus, this assay, which takes less than 80‚Äâmin, is a relatively simple, safe, and accurate alternative to traditional VNTs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00083-9" target="_blank">https://www.nature.com/articles/s44298-024-00083-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41467-024-54917-4" target="_blank">
            ü§ñ Human Stem Cell Infection Pathology During COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-274', this)">
            More</button></h4>
            <div id="toggle-274" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection of human pluripotent stem cell-derived vascular cells reveals smooth muscle cells as key mediators of vascular pathology during infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the simplified abstract:

SARS-CoV-2 causes severe respiratory problems, but many people can still get sick with serious complications like stroke. Research has been done to see how this virus affects blood vessels in the body and to understand why some people are more likely to get serious health problems after being infected.

Using special cells that mimic the lining of blood vessels, scientists studied the effects of SARS-CoV-2 on these cells. They found that some of these cells can become inflamed and make proteins that help the virus attach to blood vessels, making it harder for them to work properly. This research helps us understand how SARS-CoV-2 causes serious health problems in people infected with the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although respiratory symptoms are the most prevalent disease manifestation of infection by Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), nearly 20% of hospitalized patients are at risk for thromboembolic events. This prothrombotic state is considered a key factor in the increased risk of stroke, which is observed clinically during both acute infection and long after symptoms clear. Here, we develop a model of SARS-CoV-2 infection using human-induced pluripotent stem cell-derived endothelial cells (ECs), pericytes (PCs), and smooth muscle cells (SMCs) to recapitulate the vascular pathology associated with SARS-CoV-2 exposure. Our results demonstrate that perivascular cells, particularly SMCs, are a susceptible vascular target for SARS-CoV-2 infection. Utilizing RNA sequencing, we characterize the transcriptomic changes accompanying SARS-CoV-2 infection of SMCs, PCs, and ECs. We observe that infected SMCs shift to a pro-inflammatory state and increase the expression of key mediators of the coagulation cascade. Further, we show human ECs exposed to the secretome of infected SMCs produce hemostatic factors that contribute to vascular dysfunction despite not being susceptible to direct infection. The findings here recapitulate observations from patient sera in human COVID-19 patients and provide mechanistic insight into the unique vascular implications of SARS-CoV-2 infection at a cellular level.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54917-4" target="_blank">https://www.nature.com/articles/s41467-024-54917-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41467-024-54458-w" target="_blank">
            ü§ñ Antibody Response to RBD of SARS-CoV-2 Spike Mediates Infection in Primary Human Macrophages</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-275', this)">
            More</button></h4>
            <div id="toggle-275" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibodies to the RBD of SARS-CoV-2 spike mediate productive infection of primary human macrophages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The role of myeloid cells in COVID-19 is well established as drivers of severe inflammation and cytokine production characteristic of the disease. However, whether these cells can be infected by SARS-CoV-2 remains unclear. Researchers investigated this using a panel of antibodies that target the receptor binding domain (RBD) of the virus's surface protein. They found that these antibodies were effective at infecting myeloid cells in vitro and that high levels of infection occurred with low concentrations of the antibodies. In experiments involving primary macrophages, the researchers also detected infected cells and showed increased production of pro-inflammatory cytokines when infected macrophages had high antibody levels. This suggests that antibody-FcR interactions may contribute to pathogenesis in COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The role of myeloid cells in the pathogenesis of SARS-CoV-2 is well established, in particular as drivers of cytokine production and systemic inflammation characteristic of severe COVID-19. However, the potential for myeloid cells to act as bona fide targets of productive SARS-CoV-2 infection, and the specifics of entry, remain unclear. Using a panel of anti-SARS-CoV-2 monoclonal antibodies (mAbs) we performed a detailed assessment of antibody-mediated infection of monocytes/macrophages. mAbs with the most consistent potential to mediate infection were those targeting a conserved region of the receptor binding domain (RBD; group 1/class 4). Infection was closely related to the neutralising concentration of the mAbs, with peak infection occurring below the IC50, while pre-treating cells with remdesivir or FcŒ≥RI-blocking antibodies inhibited infection. Studies performed in primary macrophages demonstrated high-level and productive infection, with infected macrophages appearing multinucleated and syncytial. Infection was not seen in the absence of antibody with the same quantity of virus. Addition of ruxolitinib significantly increased infection, indicating restraint of infection through innate immune mechanisms rather than entry. High-level production of pro-inflammatory cytokines directly correlated with macrophage infection levels. We hypothesise that infection via antibody-FcR interactions could contribute to pathogenesis in primary infection, systemic virus spread or persistent infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54458-w" target="_blank">https://www.nature.com/articles/s41467-024-54458-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-30</strong> <a href="https://www.nature.com/articles/s41598-024-82901-x" target="_blank">
            ü§ñ Portable UV-C device for treating high flows of infectious aerosols in medical settings</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-276', this)">
            More</button></h4>
            <div id="toggle-276" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Portable UV-C device to treat high flow of infectious aerosols generated during clinical respiratory care</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Respiratory interventions, including noninvasive ventilation and continuous positive airway pressure, may increase the risk of airborne disease transmission to healthcare workers when used with contaminated air from a simulated patient source.

A portable UV-C254 device was developed to sterilize high-flow viral-contaminated air at airflow rates up to 100 l/m. The device consists of six high-output UV-C lamps that recirculate the air past them before exiting, allowing for repeated exposure to the same dose of UVC. Testing with A/PR/8/34 (H1N1) influenza virus showed a significant reduction in viral plaque forming units from single and recirculated conditions.

The device achieved 5-log reductions in viral plaque forming units at high and low flow rates, and an effective dose of UVC was calculated at 11.6 mJ/cm2 per single pass and 104 mJ/cm2 for recirculated air.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory interventions including noninvasive ventilation, continuous positive airway pressure and high-flow nasal oxygen generated infectious aerosols may increase risk of airborne disease (SARS-CoV-2, influenza virus) transmission to healthcare workers. We developed and tested a prototype portable UV-C254device to sterilize high flows of viral-contaminated air from a simulated patient source at airflow rates of up to 100¬†l/m. Our device consisted of a central quartz tube surrounded 6 high-output UV-C254lamps, within a larger cylinder allowing recirculation past the UV-C254lamps a second time before exiting the device. Testing was with nebulized A/PR/8/34 (H1N1) influenza virus. RNA extraction and qRT-PCR showed virus transited through the prototype. Turning on varying numbers of lamps controlled the dose of UVC. Viability experiments at low, medium and high (100¬†l/min) flows of contaminated gas were conducted with 6, 4, 2 and 1 lamp activated (single-pass and recirculation were tested). Our data show 5-log reduction in plaque forming units from a single lamp (single- pass and recirculated conditions) at high and low flows. UVC dose at 100¬†l/m was calculated at 11.6¬†mJ/cm2single pass and 104¬†mJ/cm2recirculated. The protype device shows high efficacy in killing nebulized influenza virus in a high flow of contaminated air.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82901-x" target="_blank">https://www.nature.com/articles/s41598-024-82901-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-82411-w" target="_blank">
            ü§ñ Monitoring Heart Rate Variability After COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-277', this)">
            More</button></h4>
            <div id="toggle-277" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of COVID-19 on heart rate variability in post-COVID individuals compared to a control group</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study found that COVID-19 infection had different effects on heart rate variability (HRV) in people with mild symptoms compared to those without previous infections or hospitalization. The results showed that:

* Those who recovered from their illness had lower HRV after 6 weeks, compared to others
* People who spent time with the virus showed decreased parasympathetic activity (rest and digest system)
* A longer recovery time was associated with lower HRV
* Age and duration of infection were significant predictors for certain HRV parameters</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigated the impact of mild COVID-19 on HRV in groups stratified by time after infection and to compare to a healthy group of the same age without previous virus infection and without need of hospitalization. This is a cross-sectional study. We divided the sample into four groups: control group (CG) (n=‚Äâ31), group 1 (G1): ‚â§6 weeks (n=‚Äâ34), group 2 (G2): 2‚Äì6 months (n=‚Äâ30), group 3 (G3): 7‚Äì12 months (n=‚Äâ35) after infection. For HRV analysis, we used the indices of linear (time and frequency domain) and non-linear analysis. For comparisons between groups, ANOVA one way test or Kruskal‚ÄìWallis was used according to the data distribution. The effect size was calculated based on Cohen‚Äôs d or Œ∑2. Simple and multiple linear regressions were performed to investigate the interaction between clinical outcomes and HRV parameters. A total of 130 individuals were included. Groups G1 and G2 showed less parasympathetic modulation when compared to CG (p<‚Äâ0.05), while G3 showed an increase in parasympathetic modulation when compared to G1 (p<‚Äâ0.05). Moderate to large effect sizes were found according to Cohen d or Œ∑2. The multiple linear regression models identified age and infection duration as significant predictors for RMSSD (adjusted R2= 0.227) and SD1 (adjusted R2= 0.242), while age was significant for SDNN (adjusted R2= 0.213). BMI, hypertension, and dyslipidemia were non-significant in all models. For HF (n.u.), infection duration was consistently significant, with stress emerging as a predictor in Model 2 (adjusted R2= 0.143). The recovery time since diagnosis and age influences recovery from HRV, suggesting a transient effect of the disease on the autonomic nervous system.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82411-w" target="_blank">https://www.nature.com/articles/s41598-024-82411-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-82983-7" target="_blank">
            ü§ñ Mental Health Issues in COVID-19 Patients
Cardiovascular Diseases in Urban Populations After COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-278', this)">
            More</button></h4>
            <div id="toggle-278" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>New-onset cardiovascular diseases post SARS-CoV-2 infection in an urban population in the Bronx</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how people with COVID-19 (the disease caused by the SARS-CoV-2 virus) are at risk of getting certain serious health problems, such as heart attacks or strokes, up to 3.5 years after they were infected.

The researchers found that hospitalized patients with COVID-19 have a much higher risk of these serious health problems than people who were not hospitalized for the disease. They also found that some other groups, including people from different racial and ethnic backgrounds, may be at increased risk.

Overall, this study suggests that careful monitoring is needed to help people with COVID-19, especially those who were hospitalized for the disease. It may identify which individuals are most at risk of developing these serious health problems and allow healthcare providers to take steps to protect them.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigated the incidence of new-onset cardiovascular disorders up to 3.5 years post SARS-CoV-2 infection for 56,400 individuals with COVID-19 and 1,093,904 contemporary controls without COVID-19 in the Montefiore Health System (03/11/2020 to 07/01/2023). Outcomes were new incidence of major adverse cardiovascular event (MACE), arrhythmias, inflammatory heart disease, thrombosis, cerebrovascular disorders, ischemic heart disease and other cardiac disorders between 30 days and (up to) 3.5 years post index date. Results were also compared with a pre-pandemic cohort over similar observation duration (N=‚Äâ64,541). Cumulative incidence and hazard ratios adjusted for competitive risks were analyzed. Compared to contemporary controls, hospitalized COVID-19 patients had significantly higher risk of developing MACE (aHR‚Äâ=‚Äâ2.29, 95% confidence interval [2.27, 2.31],p<‚Äâ0.001), arrhythmias (aHR‚Äâ=‚Äâ2.54[2.50, 2.58],p<‚Äâ0.001), inflammatory heart disease (aHR‚Äâ=‚Äâ5.34[4.79, 5.96],p<‚Äâ0.001), cerebrovascular (aHR‚Äâ=‚Äâ2.05[2.00, 2.11],p<‚Äâ0.001), other cardiac disorders (aHR‚Äâ=‚Äâ2.31[2.26, 2.35],p<‚Äâ0.001), thrombosis (aHR‚Äâ=‚Äâ4.25[4.15, 4.36],p<‚Äâ0.001), and ischemic heart disease (aHR‚Äâ=‚Äâ1.89[1.86, 1.92],p<‚Äâ0.001). Non-hospitalized COVID-19 patients had slightly higher risk of developing MACE (aHR‚Äâ=‚Äâ1.04[1.03, 1.06],p<‚Äâ0.001), arrhythmias (aHR‚Äâ=‚Äâ1.10[1.08, 1.12],p<‚Äâ0.001), inflammatory heart disease (aHR‚Äâ=‚Äâ2.29 [2.03, 2.59],p<‚Äâ0.001), cerebrovascular (aHR‚Äâ=‚Äâ1.11[1.07, 1.15],p<‚Äâ0.001), and ischemic heart disease (aHR‚Äâ=‚Äâ1.10[1.08, 1.13],p<‚Äâ0.001). Race and ethnicity were mostly not associated with increased risks (p>‚Äâ0.05). aHRs with contemporary controls as a reference were similar to those with pre-pandemic cohort as a reference. We concluded that new incident cardiovascular disorders in COVID-19 patients, especially those hospitalized for COVID-19, were higher than those in controls. Identifying risk factors for developing new-onset cardiovascular disorders may draw clinical attention for the need for careful follow-up in at-risk individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82983-7" target="_blank">https://www.nature.com/articles/s41598-024-82983-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-81019-4" target="_blank">
            ü§ñ Detection of COVID-19's molecular structure 

Detection of 4-dimensional aspects 
 
 Detection of virus
 
 Detection of antibodies 
 
 Detection of antibodies 
 
Detection of antibodies 
 
Detection of antibodies 
 
Detection of antibodies 
 
Detection of antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-279', this)">
            More</button></h4>
            <div id="toggle-279" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Direct detection of 4-dimensions of SARS-CoV-2: infection (vRNA), infectivity (antigen), binding antibody, and functional neutralizing antibody in saliva</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract that a young person can understand:

A new medical test has been developed using special devices that detect certain viruses in saliva. This test can identify if someone is infected with COVID-19 or not, as well as how their body's immune system has responded to the virus. The test works by releasing tiny particles from the virus into a drop of saliva and measuring how many of these particles are present, along with other factors like the amount of virus in the blood. The results can be used to diagnose COVID-19 and track its spread over time. This new test is more sensitive than existing ones, meaning it can detect smaller amounts of virus or immune system response.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We developed a 4-parameter clinical assay using Electric Field Induced Release and Measurement (EFIRM) technology to simultaneously assess SARS-CoV-2 RNA (vRNA), nucleocapsid antigen, host binding (BAb) and neutralizing antibody (NAb) levels from a drop of saliva with performance that equals or surpasses current EUA-approved tests. The vRNA and antigen assays achieved lower limit of detection (LOD) of 100 copies/reaction and 3.5 TCID‚ÇÖ‚ÇÄ/mL, respectively. The vRNA assay differentiated between acutely infected (n=‚Äâ10) and infection-na√Øve patients (n=‚Äâ33) with an AUC of 0.9818, sensitivity of 90%, and specificity of 100%. The antigen assay similarly differentiated these patient populations with an AUC of 1.000. The BAb assay detected BAbs with an LOD of 39 pg/mL and distinguished acutely infected (n=‚Äâ35), vaccinated with prior infection (n=‚Äâ13), and vaccinated infection-na√Øve patients (n=‚Äâ13) from pre-pandemic (n=‚Äâ81) with AUC of 0.9481, 1.000, and 0.9962, respectively. The NAb assay detected NAbs with a LOD of 31.6 Unit/mL and differentiated between COVID-19 recovered or vaccinated patients (n=‚Äâ31) and pre-pandemic controls (n=‚Äâ60) with an AUC 0.923, sensitivity of 87.10%, and specificity of 86.67%. Our combo assay represents a significant technological advancement to simultaneously address SARS-CoV-2 infection and immunity, and it lays the foundation for tackling potential future pandemics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81019-4" target="_blank">https://www.nature.com/articles/s41598-024-81019-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-80465-4" target="_blank">
            ü§ñ Long-term Antibody Response to SARS-CoV-2 in Puerto Rico</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-280', this)">
            More</button></h4>
            <div id="toggle-280" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Longitudinal analysis of SARS-CoV-2 IgG antibody durability in Puerto Rico</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied how people's immune systems respond to COVID-19 vaccines and infection. They looked at antibodies made by their bodies after vaccination or getting infected with the virus, and how these antibodies change over time. The study found that vaccines helped keep some antibody levels strong for longer, but also reduced the effectiveness of other antibodies like those against the virus itself. People who had gotten vaccinated before or after getting COVID-19 were more likely to have strong immune responses than those who only got infected with the virus. Understanding how these antibodies work can help scientists develop new treatments and strategies to prevent or treat future outbreaks of COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the dynamics of antibody responses following vaccination and SARS-CoV-2 infection is important for informing effective vaccination strategies and other public health interventions. This study investigates SARS-CoV-2 antibody dynamics in a Puerto Rican cohort, analyzing how IgG levels vary by vaccination status and previous infection. We assess waning immunity and the distribution of hybrid immunity with the aim to inform public health strategies and vaccination programs in Puerto Rico and similar settings. We conducted a prospective, longitudinal cohort study to identify SARS-CoV-2 infections and related outcomes in Ponce, Puerto Rico, from June 2020‚ÄìAugust 2022. Participants provided self-collected nasal swabs every week and serum every six months for RT-PCR and IgG testing, respectively. IgG reactivity against nucleocapsid (N) antigens, which generally indicate previous infection, and spike (S1) and receptor-binding domain (RBD) antigens, which indicate history of either infection or vaccination, was assessed using the Luminex Corporation xMAP¬Æ SARS-CoV-2 Multi-Antigen IgG Assay. Prior infection was defined by positive RT-PCRs, categorized by the predominant circulating SARS-CoV-2 variant at the event time. Demographic information, medical history, and COVID-19 vaccination history were collected through standardized questionnaires. Of 882 participants included in our analysis, 34.0% experienced at least one SARS-CoV-2 infection, with most (78.7%) occurring during the Omicron wave (December 2021 onwards). SARS-CoV-2 antibody prevalence increased over time, reaching 98.4% by the final serum collection, 67.0% attributable to vaccination alone, 1.6% from infection alone, and 31.4% from both. Regardless of prior infection status, RBD and S1 IgG levels gradually declined following two vaccine doses. A third dose boosted these antibody levels and showed a slower decline over time. N-antibody levels peaked during the Omicron surge and waned over time. Vaccination in individuals with prior SARS-CoV-2 infection elicited the highest and most durable antibody responses. N or S1 seropositivity was associated with lower odds of a subsequent positive PCR test during the Omicron period, with N antibodies showing a stronger association. By elucidating the differential decay of RBD and S1 antibodies following vaccination and the complexities of N-antibody response following infection, this study in a Puerto Rican cohort strengthens the foundation for developing targeted interventions and public health strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80465-4" target="_blank">https://www.nature.com/articles/s41598-024-80465-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-79734-z" target="_blank">
            ü§ñ Antiviral Protection for COVID-19

Daphnein may help prevent the spread of SARS-CoV-2 by blocking a protein called ACE2, which is found on cells in people infected with the virus.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-281', this)">
            More</button></h4>
            <div id="toggle-281" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Daphnetin may protect from SARS-CoV-2 infection by reducing ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To combat the COVID-19 pandemic, researchers have been looking for ways to prevent infection. One promising approach is targeting a protein called ACE2 on cells that can help the virus replicate. This study tested several approved drugs in China to see if they could downregulate ACE2 expression, which would reduce the risk of infection. DAP, one of these drugs, showed significant effectiveness and was found to decrease ACE2 mRNA and protein levels in PC9 cells. This suggests that DAP may be a new preventive measure against COVID-19, offering a cost-effective solution to reduce spread.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To combat the SARS-CoV-2 pandemic, innovative prevention strategies are needed, including reducing ACE2 expression on respiratory cells. This study screened approved drugs in China for their ability to downregulate ACE2. Daphnetin (DAP) was found to significantly reduce ACE2 mRNA and protein levels in PC9 cells. DAP exerts its inhibitory effects on ACE2 expression by targeting HIF-1Œ± and JAK2, thereby impeding the transcription of the ACE2 gene. The SARS-CoV-2 pseudovirus infection assay confirmed that DAP-treated PC9 cells exhibited decreased susceptibility to viral infection. At therapeutic doses, DAP effectively lowers ACE2 expression in the respiratory systems of mice and humans. This suggests that DAP, already approved for other conditions, could be a new preventive measure against SARS-CoV-2, offering a cost-effective and accessible way to reduce SARS-CoV-2 spread.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79734-z" target="_blank">https://www.nature.com/articles/s41598-024-79734-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-65764-0" target="_blank">
            ü§ñ Malaria Risk Factors in Children from Birth to 5 Years Old in Ethiopia: A Community-Based Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-282', this)">
            More</button></h4>
            <div id="toggle-282" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The association between baseline viral load and long-term risk in patients with COVID-19 in Hong Kong: a territory-wide study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 can increase the long-term risk of multiorgan dysfunction and death. A large study looked at how many people got COVID-19 after being admitted to hospitals, especially those who had a high viral load (more SARS-CoV-2). The study included 4054 patients in Hong Kong who tested positive for COVID-19. 

During the time the researchers were studying them, some of these patients became very sick or even died. After they recovered, many of them got readmitted to hospitals for reasons other than COVID-19. A higher viral load was linked to more hospitalizations and deaths.

The study found that people with a high viral load had a higher chance of developing complications after recovery from the virus. It also suggested that certain medications and treatments may be recommended based on how sick they got during their illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 can increase the long-term risk of multiorgan dysfunction. Few studies investigated the long-term risk in Asian populations or investigated the association between viral load and long-term risk. We aimed to investigate the post-discharge rates of hospitalization and association with baseline viral load in all patients with COVID-19 in Hong Kong. This was a population-based cohort study included all patients with a positive RT-PCR test for SARS-CoV-2 in Hong Kong between January 1st 2020 and August 30th 2020, routinely admitted to public health care facilities for isolation and treatment. Viral Ct values were available in 3433 (85%) of patients. Outcomes of interest included death, cause-specific hospitalizations, and initiation of medication from the Hospital Authority's territory-wide electronic health records from the Clinical Data Analysis and Reporting System. In total, 4054 people in Hong Kong tested positive for COVID-19 and were admitted to a public health care facility, of whom 167 (4.1%) were admitted to ICU. During a median follow-up time of 251 (interquartile range 240‚Äì279) days, 408 (11.9%) were hospitalized for any reason and 16 (0.5%) patients died. After discharge, patients were most often readmitted for respiratory reasons, followed by gastro-intestinal reasons. A higher viral load (lower RT-PCR Ct values) was associated with a higher likelihood of death (Hazard ratio [HR] 5.86, 95% Confidence interval [CI] 2.57‚Äì13.33), hospitalization (HR 1.22, 95%CI 1.08‚Äì1.39) or hospitalization for cardiovascular disease (HR 12.78, 95%CI 3.67‚Äì44.48). Patients with higher viral loads more likely started ACE-inhibitors (HR 1.37, 95%CI 1.12‚Äì1.68) and non-opioid analgesics (HR 1.01, 95%CI 1.01‚Äì1.23). In a relatively mild COVID-19 population from Hong Kong, the post-acute risk of complications was substantial. Our results highlight that higher viral load predict post-acute complications in patients with relatively mild disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65764-0" target="_blank">https://www.nature.com/articles/s41598-024-65764-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-82412-9" target="_blank">
            ü§ñ Optimizing Oral Facial Pain Relief through Photobiomodulation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-283', this)">
            More</button></h4>
            <div id="toggle-283" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evaluation of vascular photobiomodulation for orofacial pain and tension type headache following COVID 19 in a pragmatic randomized clinical trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new treatment for chronic tension-type headaches is being tested, with a difference found between two groups receiving this treatment versus a placebo.

The study involved 100 people who experienced severe headaches after COVID-19, such as migraines or facial pain, for over three months. They were divided into two groups: one received vascular photobiomodulation (a non-invasive light therapy) and the other received simulated versions of it. Their symptoms were monitored using questionnaires to assess pain level and interference with daily activities.

Significant differences were found between the two groups in terms of pain levels, work ability, sleep quality, and overall enjoyment of life, but not in the amount of headache impact on everyday tasks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This pragmatic double-blind randomized clinical trial aims to assess the impact of vascular photobiomodulation on post-COVID-19 patients experiencing tension-type headache, orofacial pain, or both persisting for more than 3¬†months. Participants were divided into two groups: vascular photobiomodulation (VPBM) and simulated VPBM. Their conditions were evaluated using the Brief Pain Inventory (BPI), Visual Analogue Scale, and Headache Impact Test (HIT-6). Data analysis included both inter and intragroup assessments, employing per-protocol and intention-to-treat analyses. Significant differences were observed in pain levels pre- and post-treatment and between the two groups. These differences were evident in the average pain experienced in the previous week (p‚Äâ=‚Äâ0.010) and various dimensions of the BPI questionnaire, such as the degree of pain interference with walking (p‚Äâ=‚Äâ0.011), work (p‚Äâ=‚Äâ0.009), sleep (p‚Äâ=‚Äâ0.012), and enjoyment of life (p‚Äâ=‚Äâ0.016). However, there was no statistically significant difference in headache impact on activities of daily living as measured by the HIT. Vascular photobiomodulation shows promise in reducing pain and enhancing the ability to engage in daily activities among post-COVID-19 patients experiencing persistent headaches and orofacial pain.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82412-9" target="_blank">https://www.nature.com/articles/s41598-024-82412-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-80971-5" target="_blank">
            ü§ñ Effector CD4/CD8 T cell subsets.

### TITLE ###
Subsets of Human Immune Cells.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-284', this)">
            More</button></h4>
            <div id="toggle-284" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SLAMF7 defines subsets of human effector CD8 T cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Efficient immune responses involve different types of T cells that can switch between memory and helper roles as needed. Recent research has identified a special type of T cell called progenitor-like effectors that can be found in people who are not getting infected by viruses for long periods. This study shows how to identify these cells using a specific protein, SLAMF7, and two other proteins, GPR56 and TOX. The researchers also found how the SLAMF7 pathway changes over time as people age, leading to a problem that makes it hard to get rid of infections. They also discovered that this pathway can be turned on by certain signals from cells that help the immune system respond to infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long-term control of viral replication relies on the efficient differentiation of memory T cells into effector T cells during secondary immune responses. Recent findings have identified T cell precursors for both memory and exhausted T cells, suggesting the existence of progenitor-like effector T cells. These cells can persist without antigenic challenge but expand and acquire effector functions upon recall immune responses. In this study, we demonstrate that the combination of SLAMF7 with either CD27 or TCF-1 effectively identifies progenitor-like effector CD8 T cells, while SLAMF7 with GPR56  or TOX defines effector CD8 T cells. These markers allow for the clear segregation of these distinct cell subsets. SLAMF7+CD8T cells are dynamically modulated during viral infections, including HIV, HCV, CMV, and SARS-CoV-2, as well as during aging. We further characterize the SLAMF7 signature at both phenotypic and transcriptional levels. Notably, during aging, the SLAMF7 pathway becomes dysregulated, resulting in persistent phosphorylation of STAT1. Additionally, SLAMF7 ligation in the presence of IL-15 induces TCF-1 expression, which promotes the homeostatic proliferation of progenitor-like effector CD8 T cells.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80971-5" target="_blank">https://www.nature.com/articles/s41598-024-80971-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s43856-024-00723-4" target="_blank">
            ü§ñ Breath Metabolites Reveal Gut-Bone Link in Osteoporosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-285', this)">
            More</button></h4>
            <div id="toggle-285" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exhaled breath metabolites reveal postmenopausal gut-bone cross-talk and non-invasive markers for osteoporosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background: Menopause can lead to a decline in estrogen levels, increasing the risk of osteoporosis.

Methods: Researchers used a discovery cohort of 120 postmenopausal women who underwent repeated ultrasonography and other tests to measure bone density and assess muscle strength. They also collected breath volatile markers such as dimethyl sulfide (DMS) using mass spectrometry.

Results: The researchers found that DMS levels in exhaled air were higher in women at high risk of osteoporosis compared to those with normal BMD. Increased age and weaker grip strength were also associated with increased DMS levels. These changes were consistent across independent cohorts, including seasonal follow-ups.

Conclusions: Using non-invasive analysis of breath volatile markers (such as DMS) can be a reliable way to predict osteoporosis risk in postmenopausal women.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundMenopause driven decline in estrogen exposes women to risk of osteoporosis. Detection of early onset and silent progression are keys to prevent fractures and associated burdens.MethodsIn a discovery cohort of 120 postmenopausal women, we combined repeated quantitative pulse-echo ultrasonography of bone, assessment of grip strength and serum bone markers with mass-spectrometric analysis of exhaled metabolites to find breath volatile markers and quantitative cutoff levels for osteoporosis. Obtained markers and cutoffs were validated in an independent cohort of 49 age-matched women with six months apart seasonal follow-ups.ResultsHere, within the discovery cohort, concentrations of exhaled end-tidal dimethyl sulfide (DMS), allyl-methyl sulfide, butanethiol and butyric acid are increased (p‚Äâ‚â§‚Äâ0.005) pronouncedly in subjects with bone mineral density (BMD) at high-risk of osteoporosis and fracture, when compared to subjects with normal BMD. Increased age and decreased grip strength are concomitant. All changes are reproduced during independent validation and seasonal follow-ups. Exhaled metabolite expressions remain age independent. Serum markers show random expressions without reproducibility. DMS exhalations differs between patients with recent, old and without fractures. Metabolite exhalations and BMDs are down-regulated during winter. ROC analysis in discovery cohort yields high classification accuracy of DMS with a cutoff for osteoporosis, which predicts subjects at high-risk within the independent validation cohort with >91% sensitivity and specificity.ConclusionsNon-invasive analysis of exhaled DMS allowed more reliable classification of osteoporosis risk than conventional serum markers. We identified associations of exhaled organosulfur and short-chain fatty acids to bone metabolism in postmenopausal osteoporosis via a gut-bone axis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00723-4" target="_blank">https://www.nature.com/articles/s43856-024-00723-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-82663-6" target="_blank">
            ü§ñ Improving Hospital Outcomes through Effective Antimicrobial Use</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-286', this)">
            More</button></h4>
            <div id="toggle-286" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of hospital-related indicators on healthcare-associated infections and appropriateness of antimicrobial use according to a national dataset</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Healthcare-associated infections (HAIs) pose a significant threat in Europe, with infection prevention and control (IPC) measures being crucial to reduce their occurrence. A study was conducted to examine the impact of IPC indicators, antibiotic use, and antimicrobial stewardship on HAI prevalence and inappropriate antibiotic prescriptions in Italy.

The study analyzed data from the ECDC PPS-2 Point Prevalence Survey, covering 14 European countries, including Italy. It found that:

* Structural determinants such as single-bed rooms had a significant positive impact on HAI prevalence.
* The presence of IPC nurses or antibiotic stewards was associated with lower rates of inappropriate antibiotic prescriptions and HAI occurrence.
* Routine filing of IPC plans and reports and post-prescription review were also linked to reduced HAI prevalence and inappropriate antibiotic use.

Overall, the study concluded that IPC measures, antimicrobial stewardship, and other healthcare-associated infection prevention and control strategies are essential for reducing HAIs in Europe.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Healthcare-associated infections (HAIs) represent a major threat in Europe. Infection prevention and control (IPC) measures are crucial to lower their occurrence, as well as antimicrobial stewardship to ensure appropriate use of antibiotics. Starting from Italian national data, this study aimed at: (i) describing IPC indicators, prevalence of HAIs, antimicrobial use and appropriateness of antibiotic use in Italy; (ii) estimating effects of IPC variables on HAI prevalence and on the proportion of antibiotics without specific reason. Based on data collected for Italy during the ECDC PPS-2 Point Prevalence Survey, descriptive statistics were computed at national and macro-regional level. Causal assumption-informed regression models were then built to estimate the impact of structural determinants, staffing parameters and IPC-related variables on HAI prevalence and percentage of antibiotic prescriptions with no reason detailed on medical records, after adjusting for relevant confounders. The Italian frame showed substantial heterogeneity for both outcomes between macro-regions. The percentage of single-bed rooms was the only structural determinant with significant, positive impact on HAI prevalence (OR‚Äâ=‚Äâ0.91 for every‚Äâ+‚Äâ5%,p<‚Äâ0.001), while the prevalence of antimicrobial agents without specified reason was lower in the presence of one more IPC nurse (OR‚Äâ=‚Äâ0.78,p<‚Äâ0.001) or one more antibiotic stewardship consultant (OR‚Äâ=‚Äâ0.67,p<‚Äâ0.001) per 100 beds. Both outcomes were reduced in the presence of routinely filed IPC plans and reports (p<‚Äâ0.001), HAI prevention measures (p<‚Äâ0.001) and post-prescription review (p<‚Äâ0.01). Our model confirmed the pivotal role of IPC measures and antimicrobial stewardship in contrasting HAIs and inappropriate antibiotic prescriptions. Post-prescription review appeared to be a valuable indicator of antimicrobial stewardship policies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82663-6" target="_blank">https://www.nature.com/articles/s41598-024-82663-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-28</strong> <a href="https://www.nature.com/articles/s41598-024-78694-8" target="_blank">
            ü§ñ Nitazoxanide's mechanism involves disrupting cell membranes' ability to generate energy, effectively killing viruses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-287', this)">
            More</button></h4>
            <div id="toggle-287" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nitazoxanide controls virus viability through its impact on membrane bioenergetics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Viruses need energy from their hosts' cells, and drugs that interfere with this process can help stop their replication. A new drug called nitazoxanide works by uncoupling the energy-producing part of cellular metabolism, which helps to release more ATP (energy) in viruses. Applying different versions of this drug to a virus-releasing cell line showed similar effects on mitochondrial function and viral particle release. The most promising effect was seen when glucose levels were high, but even then the drug didn't completely stop the viruses from spreading. This suggests that the drug's effectiveness depends on its ability to disrupt the way viruses use energy. When this process is severely disrupted, viruses may be able to spread more easily.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Viruses are dependent on cellular energy metabolism for their replication, and the drug nitazoxanide (Alinia) was shown to interfere with both processes. Nitazoxanide is an uncoupler of mitochondrial oxidative phosphorylation (OXPHOS). Our hypothesis was that mitochondrial uncoupling underlies the antiviral effects of nitazoxanide. Tizoxanide (the active metabolite of nitazoxanide), its derivative RM4848 and the uncoupler CCCP were applied to a virus-releasing cell line to obtain the same increasing levels of mitochondrial uncoupling, hence identical impact on OXPHOS. A decrease in infectious viral particle release was observed and reflected the intensity of impact on OXPHOS, irrespective of the nature of the drug. The antiviral effect was significant although the impact on OXPHOS was modest (‚â§‚Äâ25%), and disappeared when a high concentration (25¬†mM) of glucose was used to enhance glycolytic generation of ATP. Accordingly, the most likely explanation is that moderate interference with mitochondrial OXPHOS induced rearrangement of ATP use and acquisition of infective properties of the viral particles be highly sensitive to this rearrangement. The antiviral effect of nitazoxanide has been supported by clinical trials, and nitazoxanide is considered a safe drug. However, serious adverse effects of the uncoupler dinitrophenol occurred when used to increase significantly metabolic rate with the purpose of weight loss. Taken together, while impairment of mitochondrial bioenergetics is an unwanted drug effect, moderate interference should be considered as a basis for therapeutic efficacy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78694-8" target="_blank">https://www.nature.com/articles/s41598-024-78694-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-24</strong> <a href="https://www.nature.com/articles/s42003-024-07422-9" target="_blank">
            ü§ñ Infection Mechanism of Omicron Variants Compared to Delta</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-288', this)">
            More</button></h4>
            <div id="toggle-288" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structure and dynamics of the interaction of Delta and Omicron BA.1 SARS-CoV-2 variants with REGN10987 Fab reveal mechanism of antibody action</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A team of researchers studied how antibodies (like a type of immune system protein) recognize different types of viruses, in this case COVID-19 and other infections. They found that by understanding how these antibodies work, they can develop new treatments or vaccines to fight the virus more effectively. The research focused on the specific S-protein (a part of the viral enzyme) for a strain called Delta that caused COVID-19, as well as its interaction with an antibody given in a previous treatment called REGN10987.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Study of mechanisms by which antibodies recognize different viral strains is necessary for the development of new drugs and vaccines to treat COVID-19 and other infections. Here, we report 2.5‚Äâ√Ö cryo-EM structure of the SARS-CoV-2 Delta trimeric S-protein in complex with Fab of¬†the recombinant analog of REGN10987 neutralizing antibody. S-protein adopts ‚Äútwo RBD-down and one RBD-up‚Äù conformation. Fab interacts with RBDs in both conformations, blocking the recognition of angiotensin converting enzyme-2. Three-dimensional variability analysis reveals high mobility of the RBD/Fab regions. Interaction of REGN10987 with Wuhan, Delta, Omicron BA.1, and mutated variants of RBDs is analyzed by microscale thermophoresis, molecular dynamics simulations, and ŒîG calculations with umbrella sampling and one-dimensional potential of mean force. Variability in molecular dynamics trajectories results in a large scatter of calculated ŒîG values, but Boltzmann weighting provides an acceptable correlation with experiment. REGN10987 evasion of the Omicron variant is found to be due to the additive effect of the N440K and G446S mutations located at the RBD/Fab binding interface with a small effect of Q498R mutation. Our study explains the influence of known-to-date SARS-CoV-2 RBD mutations on REGN10987 recognition and highlights the importance of dynamics data beyond the static structure of the RBD/Fab complex.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07422-9" target="_blank">https://www.nature.com/articles/s42003-024-07422-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-23</strong> <a href="https://www.nature.com/articles/s41541-024-01042-4" target="_blank">
            ü§ñ Human Immunodeficiency Virus (HIV) Vaccine Trial Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-289', this)">
            More</button></h4>
            <div id="toggle-289" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A Phase 1 randomized trial of homologous and heterologous filovirus vaccines with a late booster dose</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

Researchers tested two new vaccines against deadly viruses that spread from animals to humans. They gave one group of healthy adults the new vaccine and another group a different type of vaccine. After monitoring how the people felt and what the body did in response, the researchers found that both vaccines were safe and effective at preventing severe illness and death. One way these vaccines work is by creating long-term immunity (a kind of memory) that protects against future infections. The researchers also discovered that one specific combination of vaccine types worked better than others in building up this immunity quickly after a person had been vaccinated.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Filoviruses, including Ebola, Marburg, Sudan, and Ta√Ø Forest viruses, are zoonotic pathogens that can cause severe viral hemorrhagic fever and death. Developing vaccines that provide durable, broad immunity against multiple filoviruses is a high global health priority. In this Phase 1 trial, we enrolled 60 healthy U.S. adults and evaluated the safety, reactogenicity and immunogenicity of homologous and heterologous MVA-BN¬Æ-Filo and Ad26.ZEBOV prime-boost schedules followed in select arms by MVA-BN¬Æ-Filo boost at 1 year (NCT02891980). We found that all vaccine regimens had acceptable safety and reactogenicity. The heterologous prime-boost strategy elicited superior Ebola binding and neutralizing antibody, antibody-dependent cellular cytotoxicity (ADCC), and cellular responses compared to homologous prime-boost. The MVA-BN¬Æ-Filo boost administered at 1 year resulted in robust humoral and cellular responses that persisted through 6-month follow-up. Overall, our data demonstrated that a heterologous Ad26.ZEBOV/MVA-BN¬Æ-Filo prime-boost was safe and immunogenic and established immunologic memory primed to respond after re-exposure. Clinicaltrials.gov, NCT02891980, registered September 1, 2016.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01042-4" target="_blank">https://www.nature.com/articles/s41541-024-01042-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-23</strong> <a href="https://www.nature.com/articles/s41392-024-02066-x" target="_blank">
            ü§ñ Predictive Model for COVID-19 Evolution</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-290', this)">
            More</button></h4>
            <div id="toggle-290" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A predictive language model for SARS-CoV-2 evolution</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers created a new way for computers to predict what changes (mutations) are happening in viruses like COVID-19. This is important because predicting these mutations can help us prepare and protect against future pandemics. The researchers used special methods to combine two types of information: the regular patterns that viruses follow, and the random variations within those patterns. By analyzing viral sequences from three different times, they found many new variants with increased ability to infect people and evade immune systems. This research is a quick way to predict which viruses are most likely to cause problems in the future and can be used as a warning system for emerging threats.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Modeling and predicting mutations are critical for COVID-19 and similar pandemic preparedness. However, existing predictive models have yet to integrate the regularity and randomness of viral mutations with minimal data requirements. Here, we develop a non-demanding language model utilizing both regularity and randomness to predict candidate SARS-CoV-2 variants and mutations that might prevail. We constructed the ‚Äúgrammatical frameworks‚Äù of the available S1 sequences for dimension reduction and semantic representation to grasp the model‚Äôs latent regularity. The mutational profile, defined as the frequency of mutations, was introduced into the model to incorporate randomness. With this model, we successfully identified and validated several variants with significantly enhanced viral infectivity and immune evasion by wet-lab experiments. By inputting the sequence data from three different time points, we detected circulating strains or vital mutations for XBB.1.16, EG.5, JN.1, and BA.2.86 strains before their emergence. In addition, our results also predicted the previously unknown variants that may cause future epidemics. With both the data validation and experiment evidence, our study represents a fast-responding, concise, and promising language model, potentially generalizable to other viral pathogens, to forecast viral evolution and detect crucial hot mutation spots, thus warning the emerging variants that might raise public health concern.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-02066-x" target="_blank">https://www.nature.com/articles/s41392-024-02066-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-20</strong> <a href="https://www.nature.com/articles/s41562-024-01961-1" target="_blank">
            ü§ñ Predicting Replicability of COVID-19 Preprints Claims

Claims in scientific studies are often made under the assumption they have predictive power. However, their ability to be replicated can vary greatly between studies.

This study aims to predict which claims made by researchers on COVID-19 will be easily replicable and which will not. By doing so, we hope to increase confidence in the results of future research while avoiding unwarranted enthusiasm for potentially unreliable results.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-291', this)">
            More</button></h4>
            <div id="toggle-291" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting the replicability of social and behavioural science claims in COVID-19 preprints</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Replications are essential for evaluating the accuracy of published findings, but it's not feasible or cost-effective to replicate every study. Accurate and efficient alternatives can accelerate assessment of replication reliability in times of crisis. Researchers found that expert groups (experienced) updated their predictions and confidence after interacting with others, while less-experienced participants (beginners) also improved their accuracy, but not as significantly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Replications are important for assessing the reliability of published findings. However, they are costly, and it is infeasible to replicate everything. Accurate, fast, lower-cost alternatives such as eliciting predictions could accelerate assessment for rapid policy implementation in a crisis and help guide a more efficient allocation of scarce replication resources. We elicited judgements from participants on 100 claims from preprints about an emerging area of research (COVID-19 pandemic) using an interactive structured elicitation protocol, and we conducted 29 new high-powered replications. After interacting with their peers, participant groups with lower task expertise (‚Äòbeginners‚Äô) updated their estimates and confidence in their judgements significantly more than groups with greater task expertise (‚Äòexperienced‚Äô). For experienced individuals, the average accuracy was 0.57 (95% CI: [0.53, 0.61]) after interaction, and they correctly classified 61% of claims; beginners‚Äô average accuracy was 0.58 (95% CI: [0.54, 0.62]), correctly classifying 69% of claims. The difference in accuracy between groups was not statistically significant and their judgements on the full set of claims were correlated (r(98)‚Äâ=‚Äâ0.48,P<‚Äâ0.001). These results suggest that both beginners and more-experienced participants using a structured process have some ability to make better-than-chance predictions about the reliability of ‚Äòfast science‚Äô under conditions of high uncertainty. However, given the importance of such assessments for making evidence-based critical decisions in a crisis, more research is required to understand who the right experts in forecasting replicability are and how their judgements ought to be elicited.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41562-024-01961-1" target="_blank">https://www.nature.com/articles/s41562-024-01961-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-19</strong> <a href="https://www.nature.com/articles/s43856-024-00686-6" target="_blank">
            ü§ñ Understanding Vaccines for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-292', this)">
            More</button></h4>
            <div id="toggle-292" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Induction of Fc-dependent functional antibodies against different variants of SARS-CoV-2 varies by vaccine type and prior infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>BackgroundSARS-CoV-2 transmission and COVID-19 disease severity is influenced by immunity from natural infection and/or vaccination.

We evaluated how well people's bodies respond to vaccinations with two widely used vaccines, AstraZeneca (AZ) and Sinovac (SV), in Brazilian adults who had previously been infected with SARS-CoV-2 or not. We also looked at which types of antibodies are produced and how effectively they can neutralize the virus.

Results AZ vaccinated individuals tend to produce more antibodies that block immune cells called Fc receptors, and some of these antibodies also help prevent complement from sticking to the virus. People who had previously been infected with SARS-CoV-2 and were vaccinated with AZ produce more antibodies than those who only got vaccinated, and some people's bodies make better antibodies against certain types of the virus.

People who received AZ vaccine tend to have more effective antibody responses when exposed to different variants of the SARS-CoV-2 virus. People who did not receive AZ vaccine still produced some effective antibodies that could help protect them from the virus.

ConclusionsCombining natural exposure and vaccination can provide strong immunity against evolving SARS-CoV-2 variants. Understanding which types of immune system cells work best can lead to vaccines with even greater effectiveness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSARS-CoV-2 transmission and COVID-19 disease severity is influenced by immunity from natural infection and/or vaccination. Population-level immunity is complicated by the emergence of viral variants. Antibody Fc-dependent effector functions are as important mediators in immunity. However, their induction in populations with diverse infection and/or vaccination histories and against variants remains poorly defined.MethodsWe evaluated Fc-dependent functional antibodies following vaccination with two widely used vaccines, AstraZeneca (AZ) and Sinovac (SV), including antibody binding of FcŒ≥-receptors and complement-fixation in vaccinated Brazilian adults (n=‚Äâ222), some of who were previously infected with SARS-CoV-2, as well as adults with natural infection only (n=‚Äâ200). IgG, IgM, IgA, and IgG subclasses were also quantified.ResultsAZ induces greater FcŒ≥-receptor-binding (types I, IIa, and IIIa/b) antibodies than SV or natural infection. Previously infected individuals have significantly greater vaccine-induced responses compared to na√Øve counterparts. FcŒ≥-receptor-binding is highest among AZ vaccinated individuals with a prior infection, for all receptor types, and substantial complement-fixing activity is only seen among this group. SV induces higher IgM than AZ, but this does not drive better complement-fixing activity. Some SV responses are associated with subject age, whereas AZ responses are not. Importantly, functional antibody responses are well retained against the Omicron BA.1 S protein, being best retained for FcŒ≥-receptor-1 binding, and are higher for AZ than SV.ConclusionsHybrid immunity, from combined natural exposure and vaccination, generates strong Fc-mediated antibody functions which may contribute to immunity against evolving SARS-CoV-2 variants. Understanding determinants of Fc-mediated functions may enable future vaccines with greater efficacy against different variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00686-6" target="_blank">https://www.nature.com/articles/s43856-024-00686-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-19</strong> <a href="https://www.nature.com/articles/s41541-024-01043-3" target="_blank">
            ü§ñ COVID-19 Vaccines


### INSTRUCTIONS ###

### TITLE ###

### CO-IMMUNIZATION WITH SPIKE AND NUCLEOCAPSID BASED DNA VACCINES FOR LONG-Term PROTECTIVE IMMUNITY AGAINST SARSCOV-2 OMICRON</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-293', this)">
            More</button></h4>
            <div id="toggle-293" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Co-immunization with spike and nucleocapsid based DNA vaccines for long-term protective immunity against SARS-CoV-2 Omicron</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that's easy for young people to understand:

Scientists are looking for ways to make COVID-19 vaccines stronger so they can last longer. They've been experimenting with different types of "spike" proteins found on the surface of the virus, called S and N. When these proteins combine, it seems to give the body a better response to the virus.

To test this idea, scientists gave two types of DNA (genetic material) vaccines that had just one or both spike proteins. These vaccines were given to mice with a special genetic marker. The results showed that these vaccines worked well against some strains of the virus, but only when they combined both spike proteins. This finding suggests that combining S and N proteins could make COVID-19 vaccines more effective over time.

The study also found that if T cells (a type of immune cell) were eliminated, the vaccines' protection would weaken. These findings have big implications for developing future COVID-19 vaccines and may lead to better ways to protect against future variants of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continuing evolution of SARS-CoV-2 variants challenges the durability of existing spike (S)-based COVID-19 vaccines. We hypothesized that vaccines composed of both S and nucleocapsid (N) antigens would increase the durability of protection by strengthening and broadening cellular immunity compared with S-based vaccines. To test this, we examined the immunogenicity and efficacy of wild-type SARS-CoV-2 S- and N-based DNA vaccines administered individually or together to K18-hACE2 mice. S, N, and S‚Äâ+‚ÄâN vaccines all elicited polyfunctional CD4+and CD8+T cell responses and provided short-term cross-protection against Beta and Omicron BA.2 variants, but only co-immunization with S‚Äâ+‚ÄâN vaccines provided long-term protection against Omicron BA.2. Depletion of CD4+and CD8+T cells reduced the long-term efficacy, demonstrating a crucial role for T cells in the durability of protection. These findings underscore the potential to enhance long-lived protection against SARS-CoV-2 variants by combining S and N antigens in next-generation COVID-19 vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01043-3" target="_blank">https://www.nature.com/articles/s41541-024-01043-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-18</strong> <a href="https://www.nature.com/articles/s41390-024-03777-1" target="_blank">
            ü§ñ Pediatric COVID-19 Long Term Outcomes Study for Children (PECOS)</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-294', this)">
            More</button></h4>
            <div id="toggle-294" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pediatric SARS-CoV-2 long term outcomes study (PECOS): cross sectional analysis at baseline</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>BackgroundPECOS is an ongoing study looking at the effects of a severe coronavirus on children who have been infected. Researchers compare their health and daily lives after they contracted the virus.

MethodsThey looked at data from over 650 kids, ages 0-21, to see how they fared. They compared those with COVID-19 to people who didn't have it and found that kids who got sick were more likely to experience certain symptoms, such as fatigue, pain, or sleep problems.

ResultsIn the first study, 541 infected kids and 113 uninfected kids participated in a visit. Infected kids were more likely to report constitutional (e.g., feeling tired) or gastrointestinal (e.g., having stomach trouble) symptoms.

ConclusionsThe study found that kids who got sick with COVID-19 may experience worse health problems than those who didn't get it. The researchers plan to continue studying this group over three years to learn more about what happens after a child gets infected and how they can feel better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundPECOS is an ongoing study aimed to characterize long-term outcomes following pediatric SARS-CoV-2 infection.MethodsThis is a cross-sectional analysis of infected and uninfected cohorts at baseline. Participants (0‚Äì21 years) with laboratory-confirmed SARS-CoV-2 infection were enrolled as infected. Uninfected were defined as individuals without history or laboratory evidence of SARS-CoV-2 infection. Outcome measures included demographics, medical history, review of symptoms, physical exam, cardiopulmonary evaluation and validated psychological and developmental surveys. Primary outcomes were cohort comparisons for abnormalities on all measures.Results654 participants (541 infected, 113 uninfected) completed baseline visits by June 30, 2023. Infected participants were more likely to report constitutional (OR: 2.24), HEENT (OR: 3.74); respiratory (OR: 2.41), or gastrointestinal (OR: 2.58) symptoms. Infected had worse scores in domains of Pain, Fatigue, Global Health, Physical and Cognitive functioning, Mobility and Sleep disturbances when compared to uninfected controls using Patient Reported Outcomes. Cardiopulmonary findings were similar among cohorts.ConclusionsThe first report of this ongoing longitudinal study demonstrates that infected participants were more likely to report symptoms compared to uninfected controls, which may affect performance and quality of life of these individuals. Longitudinal data will increase understanding of long-term effects of SARS-CoV-2 infection in children. ClinicalTrials.gov Identifier: NCT04830852ImpactThis study establishes a large, diverse, prospective, longitudinal, multi-center cohort of children with history of SARS-CoV-2 infection compared to an uninfected cohort to be followed for 3 years.Cross-sectional cohort analysis at study entry showed infected participants were more likely to report constitutional, respiratory, and GI symptoms compared to uninfected controls.Infected participants were more likely to have significantly worse parent-reported performance in 6 of 10 Patient Reported Outcome Measures domains.Continued study of this cohort will help identify clinical sequelae of COVID-19, characterize the immune response to SARS-CoV-2 infection, and identify potential genetic/immunologic factors associated with long-term outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41390-024-03777-1" target="_blank">https://www.nature.com/articles/s41390-024-03777-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-18</strong> <a href="https://www.nature.com/articles/s41598-024-80498-9" target="_blank">
            ü§ñ A mathematical model of SARS-CoV-2 in Syrian hamsters.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-295', this)">
            More</button></h4>
            <div id="toggle-295" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A biomathematical model of SARS-CoV-2 in Syrian hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

A new study created a computer model to predict how COVID-19 affects Syrian hamsters. The model takes into account the biological processes of viruses and the immune system, helping scientists understand how different factors like the virus population, immune cells, and antibodies might change over time. This information could be used to better plan treatments or prevent future outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>When infected with SARS-CoV-2, Syrian hamsters (Mesocricetus auratus) develop moderate disease severity presenting key features of human COVID-19. We here develop a biomathematical model of the disease course by translating known biological mechanisms of virus-host interactions and immune responses into ordinary differential equations. We explicitly describe the dynamics of virus population, affected alveolar epithelial cells, and involved relevant immune cells comprising for example CD4+ T cells, CD8+ T cells, macrophages, natural killer cells and B cells. We also describe the humoral response dynamics of neutralising antibodies and major regulatory cytokines including CCL8 and CXCL10. The model is developed and parametrized based on experimental data collected at days 2, 3, 5, and 14 post infection. Pulmonary cell composition and their transcriptional profiles were obtained by lung single-cell RNA (scRNA) sequencing analysis. Parametrization of the model resulted in a good agreement of model and data. The model can be used to predict, for example, the time course of the virus population, immune cell dynamics, antibody production and regeneration of alveolar cells for different therapy scenarios or after multiple-infection events. We aim to translate this model to the human situation in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80498-9" target="_blank">https://www.nature.com/articles/s41598-024-80498-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-16</strong> <a href="https://www.nature.com/articles/s41598-024-82584-4" target="_blank">
            ü§ñ Exercise Program for Post-Covid Fatigue and Quality of Life Trial</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-296', this)">
            More</button></h4>
            <div id="toggle-296" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effects of a symptom-titrated exercise program on fatigue and quality of life in people with post-COVID condition ‚Äì a randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Fatigue is a major problem after COVID-19 (post-COVID condition). Many people experience decreased physical strength and increased tiredness even after rest. Scientists tested whether small amounts of exercise could help improve these symptoms. They involved 118 people with post-COVID fatigue in an eight-week exercise program. The results showed that the exercise helped reduce fatigue, improved quality of life, and strengthened muscles. By tailoring the amount of exercise to individual levels of fatigue, the program was effective and safe. This suggests that small amounts of regular exercise can be beneficial for people with post-COVID fatigue.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Fatigue is the most prevalent symptom within the post-COVID condition (PCC). Furthermore, many patients suffer from decreased physical performance capacity and post-exertional malaise. Although exercise has been proposed as an effective therapeutic strategy for PCC, there is limited evidence on individualised and symptom-titrated exercise interventions in patients with fatigue and PEM. Therefore, we conducted a multi-centre randomised controlled trial to investigate the effectiveness of an individualised and symptom-titrated exercise program. We measured fatigue, health-related quality of life, hand-grip strength, endurance capacity and PEM before and after the 10-week intervention. A total of 118 individuals with PCC were included in the final intention-to-treat analysis. All tests and training sessions took place in commercial fitness and health facilities. We found significant effects on fatigue severity, health-related quality of life and physical performance capacity. Adjusting the individual exercise load to daily fatigue has proven to be an effective and safe strategy in PCC patients with fatigue. Under the guidance of qualified professionals and by utilising symptom-titrated training recommendations, commercial fitness and health facilities present an appropriate setting for outpatient exercise rehabilitation in PCC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82584-4" target="_blank">https://www.nature.com/articles/s41598-024-82584-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-16</strong> <a href="https://www.nature.com/articles/s41598-024-75067-z" target="_blank">
            ü§ñ Environmental Sensitivity of Human Nose to Air Pollution</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-297', this)">
            More</button></h4>
            <div id="toggle-297" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Ambient air pollution undermines chemosensory sensitivity ‚Äì a global perspective</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Researchers studied how air pollution affects people's sense of smell. They found that high levels of pollutants like chemicals and gases can damage our olfactory system, making us less sensitive to smells. The study also discovered that people with poor olfactory function are more likely to be exposed to bad air in certain parts of the world, especially during autumn and winter months. These findings highlight the importance of considering air quality when evaluating how it affects our health and senses.

I removed quotation marks and condensed the language to make it clearer and easier to understand for a young person. I also made some minor changes to improve sentence structure and flow. Let me know if you have any further requests!</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study offers insights into the complex relationship between chemical species constituting air pollution and chemosensory function. We examined the relationship between chemical species known to contribute to air pollution and assault human health and chemosensory sensitivity. Chemosensory sensitivity data was retrieved from a large-scale study involving 711 urban-dwelling participants inhabiting 10 different regions of the globe. Their olfactory threshold towards phenyl ethyl alcohol (PEA) and olfactory/trigeminal threshold towards Eucalyptol was measured in a multicentre study. We matched the individual chemosensory data with the levels of PM2.5, PM10, O3, NO2, SO2, CO at the location of testing sites, on the exact date of the test, using EMAC (ECHAM5/MESSy for Atmospheric Chemistry) model. Our findings indicate that air pollution negatively affects olfactory function and has cumulative negative effects with aging. The reported patterns are seasonal and increase during Autumn and Winter, and interact with medical conditions related to poorer olfactory function. We extend the current knowledge by demonstrating that olfactory/trigeminal perception is also disrupted by toxic air, albeit in a slightly different manner. The analyzed models promote a more complex perspective on the relationship between air composition and chemosensory sensitivity, but delineate problems related to the interdependence of the levels of chemical species constituting air pollution and using them together to predict chemosensory sensitivity. Conclusions point to the need to investigate the problem of air pollution and chemosensory health from a global perspective, as air quality partly accounts for the differences in chemosensory perception in different regions of the world.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75067-z" target="_blank">https://www.nature.com/articles/s41598-024-75067-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-16</strong> <a href="https://www.nature.com/articles/s44259-024-00066-z" target="_blank">
            ü§ñ Assessment of tuberculosis drug efficacy using preclinical animal models and in vitro predictive techniques.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-298', this)">
            More</button></h4>
            <div id="toggle-298" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessment of tuberculosis drug efficacy using preclinical animal models and in vitro predictive techniques</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Tuberculosis is a major cause of death from bacterial infection Mycobacterium tuberculosis (Mtb). It killed over 1 million people worldwide in 2022, surpassing COVID-19. The emergence of multi-drug-resistant (MDR) and extensively drug-resistant (XDR) Mtb strains is urgent because it makes treatment difficult. To find a solution, scientists tested different combinations of antibiotics against Mtb using an in vitro test.

They found that one antibiotic combination, Bedaquiline (BDQ), and two other antibiotics, Capreomycin (CAP), Linezolid (LIN), and Sutezolid (SUT), worked together to kill the bacteria more effectively than each individual antibiotic. When tested on mice, they found a significant reduction in bacterial load in their lungs and spleens after treatment.

The results were similar in another animal model where the combination of BDQ and CAP was tested. Additionally, it was found that Mtb strains from different lineages did not respond equally to this drug combination in an in vitro test. This suggests that finding a single effective treatment may be difficult.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Tuberculosis (TB) killed approximately 1.3 million people in 2022 and remains a leading cause of death from the bacteriaMycobacterium tuberculosis(M.tb); this number of deaths was surpassed only by COVID-19, caused by the SARS-CoV-2 virus. The alarming emergence of multidrug-resistant (MDR) and extensively drug-resistant (XDR) M.tb strains presents an urgent need for effective new treatments. Our study aimed to determine the synergistic effects of antibiotic combinations against M.tb. Using a high-throughput in vitro checkerboard assay, we evaluated the interactions of Bedaquiline (BDQ) and other antibiotics including Capreomycin (CAP), Linezolid (LIN), and Sutezolid (SUT) against M.tb H37Rv. BDQ and CAP demonstrated in vitro enhanced effect, which prompted further investigation in vivo using the murine low dose aerosol (LDA) model. After aerosol challenge with M.tb, C57BL/6 mice were treated with BDQ, CAP, or their combination, starting 28 days post-infection. The antimicrobial treatment lasted four weeks, and the bacterial burden in lung and spleen tissues was assessed at the end of treatment. At 4 weeks post-treatment, a significant reduction in bacterial load was observed within the lungs and spleens of mice given BDQ alone or given as a BDQ/CAP combination compared to the untreated group. In contrast, CAP monotherapy led to an increase in bacterial load within the lung and no significant difference in bacterial burden in the spleen in comparison to the untreated mice. These results were confirmed in the guinea pig model of TB, where both BDQ and the BDQ/CAP combination treatment led to a decrease in bacterial burden in the lung and spleen, whereas CAP had no significant effect on bacterial burden at the 4-week post treatment timepoint. We next determined whether there may be differences in vitro with the BDQ/CAP combination against M.tb lineages 1, 2 and 4. We determined that in vitro enhanced effect was not observed in some representative strains of M.tb lineage 4, indicating variability in drug effectiveness across M.tb lineages. This research underscores the complexity of TB treatment and the critical need for innovative approaches to combat this global health threat.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44259-024-00066-z" target="_blank">https://www.nature.com/articles/s44259-024-00066-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-13</strong> <a href="https://www.nature.com/articles/s41551-024-01306-3" target="_blank">
            ü§ñ Circular RNAs (circRNAs) are critical in regulating gene expression and cell behavior. They can be used as therapeutic tools to treat various diseases.

### Efficient Circular RNA Synthesis for Potent Rolling Circle Translation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-299', this)">
            More</button></h4>
            <div id="toggle-299" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficient circular RNA synthesis for potent rolling circle translation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Circular RNA (circRNA) is a promising candidate for gene therapy due to its remarkable stability. Here's how it can be synthesized:

* A method that uses trans-splicing-based techniques to create circRNAs over 8,000 nucleotides.
* This method works independently of bacterial sequences and outperforms existing methods.
* The resulting circRNAs from human ribosomal RNA display low immunogenicity and are translated efficiently into proteins.
* By using viral internal ribosomal entry sites for rolling circle translation, the efficiency of this process is significantly improved.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Circular RNA (circRNA) is a candidate for next-generation messenger RNA therapeutics owing to its remarkable stability. Here we describetrans-splicing-based methods for the synthesis of circRNAs over 8,000 nucleotides. The methods are independent of bacterial sequences, outperform the permuted intron‚Äìexon method and allow for the incorporation of RNA modifications. The resulting unmodified circRNAs, which incorporate sequences from human 28S ribosomal RNA, display low immunogenicity and are translated more efficiently than permuted intron‚Äìexon-derived circRNAs. Additionally, by using viral internal ribosomal entry sites for rolling circle translation, we show that ribosomes can efficiently read through highly structured internal ribosomal entry sites, enhancing the efficiency of rolling circle translation by over 7,000-fold with respect to previous constructs. The efficient and reliable production of circRNA may facilitate its therapeutic use.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41551-024-01306-3" target="_blank">https://www.nature.com/articles/s41551-024-01306-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-09</strong> <a href="https://www.nature.com/articles/s41467-024-55024-0" target="_blank">
            ü§ñ A protein in the nucleocapsid of SARS-CoV-2 that helps maintain its structure and binds to specific RNA molecules.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-300', this)">
            More</button></h4>
            <div id="toggle-300" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A core network in the SARS-CoV-2 nucleocapsid NTD mediates structural integrity and selective RNA-binding</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>## Understanding the Role of Nucleocapsid Protein in SARS-CoV-2 Viral RNA Genome Processing

The nucleocapsid protein plays a crucial role in viral RNA genome processing, allowing it to interact with the virus's RNA genome.

## Insights from High-Resolution Structures and Mutations

Hybrid structures have been obtained for the N-terminal domain (NTD) of the nucleocapsid protein, providing valuable information on how it interacts with viral RNA. However, determining the exact mechanism of complex formation remains elusive.

Research has also investigated how mutations in the nucleocapsid NTD affect its stability and ability to interact with RNA. Crystallography and solution nuclear magnetic resonance (NMR) studies have been used to analyze this process.

## Key Findings

The core network of residues within the nucleocapsid protein, particularly those found in Betacoronaviruses, is crucial for maintaining protein stability and facilitating communication between flexible loop regions that allow for RNA recognition. This network guides specific interactions with viral RNA.

## Proposed Explanation

Based on its structural analysis, the NTD is proposed to be evolutionarily robust, allowing it to maintain its functions in RNA processing despite mutations. The core network within this domain serves as a key factor in these processes.

The final answer is: 

The nucleocapsid protein plays a crucial role in viral RNA genome processing by interacting with the virus's RNA genome. A comprehensive analysis of high-resolution structures and mutation effects has provided insights into how it functions, including its structure, stability, and interactions with RNA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 nucleocapsid protein is indispensable for viral RNA genome processing. Although the N-terminal domain (NTD) is suggested to mediate specific RNA-interactions, high-resolution structures with viral RNA are still lacking. Available hybrid structures of the NTD with ssRNA and dsRNA provide valuable insights; however, the precise mechanism of complex formation remains elusive. Similarly, the molecular impact of nucleocapsid NTD mutations that have emerged since 2019 has not yet been fully explored. Using crystallography and solution NMR, we investigate how NTD mutations influence structural integrity and RNA-binding. We find that both features rely on a core network of residues conserved inBetacoronaviruses, crucial for protein stability and communication among flexible loop-regions that facilitate RNA-recognition. Our comprehensive structural analysis demonstrates that contacts within this network guide selective RNA-interactions. We propose that the core network renders the NTD evolutionarily robust in stability and plasticity for its versatile RNA processing roles.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55024-0" target="_blank">https://www.nature.com/articles/s41467-024-55024-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-09</strong> <a href="https://www.nature.com/articles/s41421-024-00748-y" target="_blank">
            ü§ñ COVID-19 Vaccine Development and Safety Review</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-301', this)">
            More</button></h4>
            <div id="toggle-301" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The comprehensive SARS-CoV-2 ‚Äòhijackome‚Äô knowledge base</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new study looks at how different types of COVID-19 variants affect the cells that host viruses. The researchers wanted to know more about how these variants change and which proteins are involved in the process. They studied changes in thousands of proteins over time, and found some differences between each variant. This helps us better understand how viral proteins work and why some variants might be harder for treatment to target.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continuous evolution of SARS-CoV-2 has led to the emergence of several variants of concern (VOCs) that significantly affect global health. This study aims to investigate how these VOCs affect host cells at proteome level to better understand the mechanisms of disease. To achieve this, we first analyzed the (phospho)proteome changes of host cells infected with Alpha, Beta, Delta, and Omicron BA.1 and BA.5 variants over time frames extending from 1 to 36‚Äâh post infection. Our results revealed distinct temporal patterns of protein expression across the VOCs, with notable differences in the (phospho)proteome dynamics that suggest variant-specific adaptations. Specifically, we observed enhanced expression and activation of key components within crucial cellular pathways such as the RHO GTPase cycle, RNA splicing, and endoplasmic reticulum-associated degradation (ERAD)-related processes. We further utilized proximity biotinylation mass spectrometry (BioID-MS) to investigate how specific mutation of these VOCs influence viral‚Äìhost protein interactions. Our comprehensive interactomics dataset uncovers distinct interaction profiles for each variant, illustrating how specific mutations can change viral protein functionality. Overall, our extensive analysis provides a detailed proteomic profile of host cells for each variant, offering valuable insights into how specific mutations may influence viral protein functionality and impact therapeutic target identification. These insights are crucial for the potential use and design of new antiviral substances, aiming to enhance the efficacy of treatments against evolving SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00748-y" target="_blank">https://www.nature.com/articles/s41421-024-00748-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-06</strong> <a href="https://www.nature.com/articles/s41598-024-81275-4" target="_blank">
            ü§ñ The impact of Social Determinants on COVID-19 Patients' Quality of Life</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-302', this)">
            More</button></h4>
            <div id="toggle-302" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exploring social determinants of health and their impacts on self-reported quality of life in long COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study investigates how people with long COVID experience their quality of life. It looked at 3,463 patients in British Columbia's post-COVID-19 recovery clinics to find out how well they're functioning. The results show that many people had significant problems with daily activities and were unhappy.

The factors that affected people's well-being were not the same for everyone. Some aged 50 or older had fewer problems than younger people. Men, women, and ethnic minorities also had different impacts on their quality of life. For example, older age made some people less anxious and depressed but more pain-free. However, this study found that many people with long COVID were not just dealing with physical symptoms - they also felt sad or hopeless.

The researchers conclude that addressing the social determinants of health can help people with long COVID improve their quality of life. They suggest using targeted strategies to make recovery easier for different groups. This could lead to better overall health outcomes and support policymakers develop more effective public health measures.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study explores the health-related quality of life (HRQoL) experienced by patients with Long COVD-19 using data from British Columbia‚Äôs post-COVID-19 Recovery Clinics. A retrospective cohort of 3463 patients was analyzed to assess HRQoL through the EQ-5D-5L questionnaire which includes five dimensions (mobility, self-care, usual activities, physical health, and mental health) administered to patients; responses were analyzed using the Visual Analogue Score (VAS). Notably, 95% of participants reported HRQoL scores below 90, with 50% scoring under 60, indicating significant impacts on their well-being. The analysis revealed that HRQoL is significantly influenced by various social determinants of health (SDoH), including age, sex, employment status, and ethnicity, each showing distinct correlations with HRQoL dimensions and overall VAS scores. Specifically, older age was associated with decreased mobility and increased pain/discomfort but less anxiety and depression, highlighting varying impacts across the age spectrum. The study highlights the multifaceted impacts of Long COVID on the lives of patients and underscores the necessity of targeted strategies to improve HRQoL among diverse groups, considering specific SDoH. Such a comprehensive approach could lead to more equitable health outcomes and support the development of tailored public health policies aimed at the recovery and rehabilitation of Long COVID sufferers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81275-4" target="_blank">https://www.nature.com/articles/s41598-024-81275-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-05</strong> <a href="https://www.nature.com/articles/s44298-024-00075-9" target="_blank">
            ü§ñ Respiratory Syncytial Virus Glycoprotein G Blocks CX3CR1 Activation By CX3CL1 And Monocyte Function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-303', this)">
            More</button></h4>
            <div id="toggle-303" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Respiratory syncytial virus glycoprotein G impedes CX3CR1-activation by CX3CL1 and monocyte function</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The Respiratory Syncytial Virus (RSV) has a protein called soluble form (sG) that is similar in structure and function to a chemokine called fractalkine (CX3CL1). This similarity suggests that sG might interact with CX3CR1, a receptor on immune cells. 

We studied two versions of RSV sG: one normal version (WT) and another version that has lost the part of the protein responsible for binding to CX3CL1. We found that these versions of RSV sG do not directly activate CX3CR1 but instead compete with CX3CL1 for its receptor binding.

This competition can limit how well the immune system responds to X (a chemokine) when it binds to Y (CX3CR1). In our research, we discovered that the protein RSV sG might help reduce the host's immune response if they were in contact with this virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The soluble form of the Respiratory Syncytial Virus (RSV) G protein (sG) bears resemblance to the chemokine fractalkine (CX‚ÇÉCL1). Both RSV sG and CX3CL1 possess a mucin-like domain and a CX3C motif, exist in membrane-associated and soluble forms, and bind to the CX‚ÇÉCR1 receptor expressed on immune and epithelial cells. To explore the biological significance of RSV sG and CX‚ÇÉCR1 interaction, we produced wild type (WT) and CX‚ÇÉC motif-deficient (CX3CMut) RSV sG proteins and determined their effects on CX‚ÇÉCR1 signaling in monocytic cells. Both CX3CMut- and WT RSV sG failed to activate CX‚ÇÉCR1 signaling directly. However, WT sG competed with CX‚ÇÉCL1 for CX‚ÇÉCR1 binding and reduced CX3CL1-induced CX‚ÇÉCR1-activation, monocyte migration, and adhesion. The CX‚ÇÉC motif of sG was crucial for competitive blocking of CX3CL1-mediated activation, as CX‚ÇÉCMutsG did not affect these CX‚ÇÉCR1 functions significantly. Thus, blockade of CX‚ÇÉCR1 signaling by sG may allow RSV to dampen host immune responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00075-9" target="_blank">https://www.nature.com/articles/s44298-024-00075-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-05</strong> <a href="https://www.nature.com/articles/s41598-024-82065-8" target="_blank">
            ü§ñ Simplifying the title:

Gene Therapy with Raltegravir for Cancer Treatment

Bioinformatics Analysis on Repurposed Drug</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-304', this)">
            More</button></h4>
            <div id="toggle-304" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Repurposing raltegravir for reducing inflammation and treating cancer: a bioinformatics analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract ###

The body has a way to defend itself against harm, called inflammation. A protein called ADAM17 plays a key role in this process. In some cases, it can also cause more inflammation or harm. To study this, scientists looked at how five drugs work and identified which ones might help stop ADAM17 from causing too much of an inflammatory response. They chose to test these drugs because they are already FDA-approved and have been shown to be effective in treating other conditions. Using computer programs to analyze the data, scientists found that one drug called Raltegravir is particularly good at stopping inflammation caused by faulty ADAM17 and has a good safety profile.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Inflammation is a defensive mechanism that safeguards the human body against detrimental stimuli. Within this intricate process, ADAM17, a zinc-dependent metalloprotease, emerges as an indispensable element, fostering the activation of diverse inflammatory and growth factors within the organism. Nonetheless, ADAM17 malfunctions can augment the rate of growth, inflammatory factors, and subsequent damage. Thus, in this study, we examined and repurposed drugs to suppress the activity of ADAM17. To this end, we employed bioinformatics techniques such as molecular docking, molecular dynamics, and pharmacokinetic studies. Five FDA-approved drugs including Raltegravir, Conivaptan, Paclitaxel, Saquinavir, and Venetoclax with the ability to impede the activity of the ADAM17 metalloenzyme were identified. Moreover, these drugs did not include strong zinc-binding functional groups when verified by the ACE functional group finder. However, furtherin silicoanalysis has indicated that Raltegravir demonstrates a commendable interaction with the active site amino acids and exhibits the most favorable pharmacokinetic properties compared to others. Considering the results of bioinformatics tools, it can be concluded that Raltegravir as an antiviral drug could be repurposed to prevent severe inflammatory response and tumorigenesis resulting from ADAM17 malfunction.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-82065-8" target="_blank">https://www.nature.com/articles/s41598-024-82065-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-05</strong> <a href="https://www.nature.com/articles/s44328-024-00017-8" target="_blank">
            ü§ñ Target Protein Interaction Assay System</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-305', this)">
            More</button></h4>
            <div id="toggle-305" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A quantitative, label-free visual interference colour assay platform for protein targeting and binding assays</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Imagine being able to visualize how different molecules interact with each other without using special equipment or expensive machines. Researchers have developed an innovative way to do this called "Visual Interference Color Assay" (VICA). It uses a special surface that changes color when molecules from different types of proteins come together, allowing scientists to see exactly what's happening in real-time. This technology is useful for studying complex protein interactions and can also be used to analyze blood or other bodily fluids.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The vast array of immunoassay technologies used to assess protein interactions is costly or platform-specific. We present a label-free visual interference colour assay (VICA) that quantifies peptide and protein interactions by creating an iridescent surface allowing direct visualisation without spectrophotometric optics or microfluidics. A nanoporous aluminium oxide surface is tuned to match the refractive indices of the overlying protein layers to generate visual interference colours. To functionalise the surface, we created an affinity-capture system using a protein A-carboxyglutamic (GLA) construct that orients antibodies to enhance the signal. Using off-the-shelf antibodies, the platform can isolate analytes in buffer, whole blood, or serum. This surface generates a discernible colour change at concentrations as low as 50 femtomoles/mm2and can monitor oligomer formation in sequential steps on the same slide. VICA provides comparable kinetic parameters to biolayer interferometry and traditional immunoassays while also allowing characterisation of proteins in large macromolecular complexes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44328-024-00017-8" target="_blank">https://www.nature.com/articles/s44328-024-00017-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-05</strong> <a href="https://www.nature.com/articles/s41598-024-81327-9" target="_blank">
            ü§ñ Predictive factors for dropouts in therapy sessions among mental health workers during the COVID-19 pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-306', this)">
            More</button></h4>
            <div id="toggle-306" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predictors of dropout in cognitive behavior and interpersonal online brief psychotherapies for essential professionals during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding predictors of premature dropout from psychotherapy can help identify who is most at risk of dropping out and provide targeted interventions to reduce this risk. A study analyzing data from online therapy sessions for COVID-19 pandemic survivors found that certain factors, such as having children or living in specific regions, were associated with a higher likelihood of dropping out. The study also identified the need for culturally tailored strategies, support for families with young children, and therapy focused on low life satisfaction.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Premature dropout from psychotherapy can harm patients and increase mental health costs. This study identified predictors of dropout in brief online psychotherapy for essential workers during the COVID-19 pandemic. This was a secondary analysis of a randomized trial on 4-week CBT or IPT protocols. Participants provided sociodemographic data and completed the Patient-Reported Outcomes Measurement Information System and Burnout Assessment Tool Short-Form. Predictors were analyzed in three blocks: sociodemographic, clinical, and therapist characteristics using bivariable and multivariable analyses. The sample included 804 individuals who attended at least the first session of either CBT (n=‚Äâ403) or IPT (n=‚Äâ401). A total of 17.2% (n=‚Äâ138) of the participants dropped out during the protocol. Significant predictors of dropout included having children (IRR‚Äâ=‚Äâ1.48; 95% CI: 1.07‚Äì2.05;p=‚Äâ0.016), residing in specific regions of Brazil (Northeast IRR‚Äâ=‚Äâ1.44; 95% CI:1.04‚Äì2.00;p=‚Äâ0.02 and Midwest IRR‚Äâ=‚Äâ1.73; 95%CI: 1.13‚Äì2.64;p=‚Äâ0.01), therapist male sex (IRR‚Äâ=‚Äâ2.04; 95% CI: 1.47‚Äì2.83; p‚Äâ=‚Äâ<‚Äâ0.001), second wave of Covid-19 (IRR‚Äâ=‚Äâ1.54; 95% CI: 1.01‚Äì2.34;p=‚Äâ0.04) and low life satisfaction (IRR‚Äâ=‚Äâ1.63; 95% CI: 1.06‚Äì2.50;p=‚Äâ0.02). Our findings underscore the necessity for culturally tailored strategies, support for those with children, and targeted therapy for individuals with low life satisfaction. Implementation of these strategies may reduce dropout rates and improve treatment outcomes for essential workers in crisis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81327-9" target="_blank">https://www.nature.com/articles/s41598-024-81327-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-05</strong> <a href="https://www.nature.com/articles/s41467-024-54996-3" target="_blank">
            ü§ñ Protein involved in promoting viral mRNA translation by recruiting a specific RNA-binding protein.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-307', this)">
            More</button></h4>
            <div id="toggle-307" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 N protein recruits G3BP to double membrane vesicles to promote translation of viral mRNAs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Ras-GTPase-activating protein SH3-domain-binding proteins (G3BP) play a critical role in forming stress granules (SGs), which are important for virus replication and pathogenesis. 

The SARS-CoV-2 nucleocapsid (N) protein has a strong affinity for G3BP, causing it to inhibit the formation of SGs after infection. To study this interaction, researchers created a mutant version of the N protein that lacks the binding site for G3BP. This mutation leads to an increased and prolonged response in infected cells, which results in reduced replication and pathogenesis.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Ras-GTPase-activating protein SH3-domain-binding proteins (G3BP) are critical for the formation of stress granules (SGs) through their RNA- and ribosome-binding properties. SARS-CoV-2 nucleocapsid (N) protein exhibits strong binding affinity for G3BP and inhibits infection-induced SG formation soon after infection. To study the impact of the G3BP-N interaction on viral replication and pathogenesis in detail, we generated a mutant SARS-CoV-2 (RATA) that specifically lacks the G3BP-binding motif in the N protein. RATA triggers a stronger and more persistent SG response in infected cells, showing reduced replication across various cell lines, and greatly reduced pathogenesis in K18-hACE2 transgenic mice. At early times of infection, G3BP and WT N protein strongly colocalise with dsRNA and with non-structural protein 3 (nsp3), a component of the pore complex in double membrane vesicles (DMVs) from which nascent viral RNA emerges. Furthermore, G3BP-N complexes promote highly localized translation of viral mRNAs in the immediate vicinity of the DMVs and thus contribute to efficient viral gene expression and replication. In contrast, G3BP is absent from the DMVs in cells infected with RATA and translation of viral mRNAs is less efficient. This work provides a fuller understanding of the multifunctional roles of G3BP in SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54996-3" target="_blank">https://www.nature.com/articles/s41467-024-54996-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-04</strong> <a href="https://www.nature.com/articles/s41540-024-00472-z" target="_blank">
            ü§ñ Cancer Cell Tracking for Personalized Care

A new type of computer model that simulates how the body's immune system fights cancer has been created. This model helps scientists understand how this process works and how it can be improved to treat patients more effectively.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-308', this)">
            More</button></h4>
            <div id="toggle-308" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A multiscale model of immune surveillance in micrometastases gives insights on cancer patient digital twins</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Metastasis drives significant scientific interest in studying the interactions between the immune system and cancer progression. We developed a multiscale mathematical model to simulate how the immune system responds to cancer growth in general epithelial tissue. Our results show that the model can generate diverse patient trajectories, including uncontrolled growth, partial response, and complete immune response.

Our analysis revealed key factors that influence simulated immunosurveillance, helping us identify parameter settings with significant impact. This study contributes to the field of cancer patient digital twins (CPDTs), enabling clinicians to better understand individual patient complexities in cancer treatment decisions. However, we acknowledge several challenges remain before CPDTs can be fully realized, including uncertainties in immune responses, unreliable patient stratification, and unpredictable personalized treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Metastasis is the leading cause of death in patients with cancer, driving considerable scientific and clinical interest in immunosurveillance of micrometastases. We investigated this process by creating a multiscale mathematical model to study the interactions between the immune system and the progression of micrometastases in general epithelial tissue. We analyzed the parameter space of the model using high-throughput computing resources to generate over 100,000 virtual patient trajectories. We demonstrated that the model could recapitulate a wide variety of virtual patient trajectories, including uncontrolled growth, partial response, and complete immune response to tumor growth. We classified the virtual patients and identified key patient parameters with the greatest effect on the simulated immunosurveillance. We highlight the lessons derived from this analysis and their impact on the nascent field of cancer patient digital twins (CPDTs). While CPDTs could enable clinicians to systematically dissect the complexity of cancer in each individual patient and inform treatment choices, our work shows that key challenges remain before we can reach this vision. In particular, we show that there remain considerable uncertainties in immune responses, unreliable patient stratification, and unpredictable personalized treatment. Nonetheless, we also show that in spite of these challenges, patient-specific models suggest strategies to increase control of clinically undetectable micrometastases even without complete parameter certainty.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41540-024-00472-z" target="_blank">https://www.nature.com/articles/s41540-024-00472-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-04</strong> <a href="https://www.nature.com/articles/s41467-024-54110-7" target="_blank">
            ü§ñ Lack of X-chromosome in female mammals</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-309', this)">
            More</button></h4>
            <div id="toggle-309" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A landscape of X-inactivation during human T cell development</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Females have an enhanced immune response to both self-antigens and non-self-antigens, but this comes at the cost of being more prone to autoimmune diseases. Research on how females respond differently to different types of threats may hold clues about why they're more likely to develop certain conditions. A study found that X-linked genes in female T cells are responsible for their strong immune response. By analyzing data from thymocytes (immature white blood cells) from healthy males, females, a woman with skewed X-chromosome inactivation, and individuals with Turner syndrome, scientists created detailed maps of how T cells develop and get out of control when they should stay inactivated. The results show that the X chromosome is stable throughout T cell development and independent of genes involved in its initial activation. Furthermore, certain genes typically escaping XCI are expressed only from one X-chromosome, while a second X-chromosome seems unnecessary for T cell development. This new understanding of XCI may lead to re-evaluating sex differences in immune function.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Females exhibit a more robust immune response to both self-antigens and non-self-antigens than males, resulting in a higher prevalence of autoimmune diseases but more effective responses against infection. Increased expression of X-linked immune genes in female T cells is thought to underlie this enhanced response. Here we isolate thymocytes from pediatric thymi of healthy males (46, XY), females (46, XX), a female with completely skewed X-chromosome¬†inactivation (46, XX, cXCI) and a female with Turner syndrome (45, X0). Using whole exome sequencing, RNA sequencing and DNA methylation data, we present a sex-aware expression profile of T cell development and generate a high-resolution map of escape¬†from X-chromosome inactivation (XCI). Unexpectedly, XCI is transcriptionally and epigenetically stable throughout T cell development, and is independent of expression ofXIST, the lncRNA responsible for XCI initiation during early embryonic development. In thymocytes, several genes known to escape XCI are expressed from only one X-chromosome. Additionally, we further reveal that a second X-chromosome is dispensable for T cell development. Our study thus provides a high-resolution map of XCI during human development and suggests a re-evaluation of XCI in sex differences in T cell function.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54110-7" target="_blank">https://www.nature.com/articles/s41467-024-54110-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-04</strong> <a href="https://www.nature.com/articles/s41467-024-55008-0" target="_blank">
            ü§ñ Understanding Brain Connections: Finding Links Between Depression, Anxiety, and Genetic Markers</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-310', this)">
            More</button></h4>
            <div id="toggle-310" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Edge-centric connectome-genetic markers of bridging factor to comorbidity between depression and anxiety</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A type of mental health condition called depression often goes hand-in-hand with anxiety. However, many people who experience both symptoms struggle to find meaning and pattern in their experiences, making it hard for doctors and therapists to diagnose and treat them.

Researchers developed a way to identify specific characteristics or "factors" that are common to both depression and anxiety, which they called the common bridging factor (cbfactor). They found 12 key features of these factors in people with both conditions. This cbfactor can also be used to predict what might cause similar symptoms in others.

The researchers looked at brain connections in twins who were diagnosed with both conditions, and found that the patterns of brain activity were influenced by genetics. They also identified specific genes and brain regions that are involved in regulating emotional responses and developing brains.

This study has helped us better understand how depression and anxiety share underlying characteristics and how they might be linked to similar problems in other areas of life.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Depression-anxiety comorbidity is commonly attributed to the occurrence of specific symptoms bridging the two disorders. However, the significant heterogeneity of most bridging symptoms presents challenges for psychopathological interpretation and clinical applicability. Here, we conceptually established a common bridging factor (cbfactor) to characterize a general structure of these bridging symptoms, analogous to the general psychopathologicalpfactor. We identified acbfactor from 12 bridging symptoms in depression-anxiety comorbidity network. Moreover, thiscbfactor could be predicted using edge-centric connectomes with robust generalizability, and was characterized by connectome patterns in attention and frontoparietal networks. In an independent twin cohort, we found that these patterns were moderately heritable, and identified their genetic connectome-transcriptional markers that were associated with the neurobiological enrichment of vasculature and cerebellar development, particularly during late-childhood-to-young-adulthood periods. Our findings revealed a general factor of bridging symptoms and its neurobiological architectures, which enriched neurogenetic understanding of depression-anxiety comorbidity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-55008-0" target="_blank">https://www.nature.com/articles/s41467-024-55008-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-04</strong> <a href="https://www.nature.com/articles/s43856-024-00657-x" target="_blank">
            ü§ñ Childhood COVID-19 Effects Following Nationwide Outbreak</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-311', this)">
            More</button></h4>
            <div id="toggle-311" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A 24-month National Cohort Study examining long-term effects of COVID-19 in children and young people</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background: Many children and young people (CYP) infected with SARS-CoV-2 experience impairing symptoms post-infection.

Methods: We studied 31,012 CYP in England aged 11-17 who had a PCR test between September 2020 and March 2021. Of those who tested positive, we followed them up to 24 months later.

Results: Only 7.2% of the group fulfilled the criteria for post-COVID-19 condition (PCC). This was found in different groups based on whether they had initial tests come back negative or positive, and if they were reinfected.

Conclusions: The study highlights that PCC is more common in older CYPs and those from disadvantaged areas. It also shows that symptom severity/impact increases for CYP who fulfill the criteria for PCC.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundSome children and young people (CYP) infected with SARS-COV-2 experience impairing symptoms post-infection, known as post-COVID-19 condition (PCC). Using data from the National Long COVID in Children and Young People (CloCk) study, we report symptoms and their impact up to¬†24-months post-infection.MethodsCloCk is a cohort of CYP in England aged 11-to-17-years when they had a SARS-CoV-2 PCR-test (between September 2020 and March 2021). Of 31,012 eligible CYP 24-months post-PCR test, 12,632 participated (response‚Äâ=‚Äâ40.7%). CYP were grouped by infection status: ‚Äòinitial test-negatives; no subsequent positive-test‚Äô (NN); ‚Äòinitial test-negatives; subsequent positive-test‚Äô (NP); ‚Äòinitial test-positives; no reported re-infection‚Äô (PN); and ‚Äòinitial test-positives; reported re-infection‚Äô (PP). The Delphi research definition of PCC in CYP was operationalised; symptom severity/impact and validated scales (e.g., Chalder Fatigue Scale) were recorded. We examine symptom profiles 24-month post-index-test by infection status.Results7.2% of CYP consistently fulfil the PCC definition at 3-, 6-, 12- and 24-months. These CYPs have a median of 5-to-6 symptoms at each time-point. Between 20% and 25% of all infection status groups report 3+ symptoms 24-months post-testing; 10‚Äì25% experience 5+ symptoms. The reinfected group has more symptoms than the other positive groups; the NN group has the lowest symptom burden (p<‚Äâ0.001). PCC is more common in older CYPs and in the most deprived. Symptom severity/impact is higher in those fulfilling the PCC definition.ConclusionsThe discrepancy in the proportion of CYP fulfilling the Delphi PCC definition at 24-months and those consistently fulfilling the definition across time, highlights the importance of longitudinal studies and the need to consider clinical impairment and range of symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00657-x" target="_blank">https://www.nature.com/articles/s43856-024-00657-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-12-03</strong> <a href="https://www.nature.com/articles/s42003-024-07307-x" target="_blank">
            ü§ñ Heart Damage by SARS-CoV-2 Nsp6-Omicron is Less in Omicron Variants Than Older Strains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-312', this)">
            More</button></h4>
            <div id="toggle-312" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 Nsp6-Omicron causes less damage to theDrosophilaheart and mouse cardiomyocytes than ancestral Nsp6</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied the genetic material of SARS-CoV-2 to understand why it causes fewer severe illnesses compared to other versions. They found that an old, but still very contagious, strain has changed in certain parts, making it less likely to harm people. By comparing this strain with another version, they discovered which part is responsible for its reduced pathogenicity and how it affects different tissues. This research suggests that the amino acid changes in this particular SARS-CoV-2 strain may have made it less capable of causing disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A few years into the COVID-19 pandemic, the SARS-CoV-2 Omicron strain rapidly becomes and has remained the predominant strain. To date, Omicron and its subvariants, while more transmittable, appear to cause less severe disease than prior strains. To study the cause of this reduced pathogenicity we compare SARS-CoV-2 ancestral Nsp6 with Nsp6-Omicron, which we have previously identified as one of the most pathogenic viral proteins. Here, through ubiquitous expression inDrosophila, we show that ancestral Nsp6 causes both structural and functional damage to cardiac, muscular, and tracheal (lung) tissue, whereas Nsp6-Omicron has minimal effects. Moreover, we show that ancestral Nsp6 dysregulates the glycolysis pathway and disrupts mitochondrial function, whereas Nsp6-Omicron does not. Through validation in mouse primary cardiomyocytes, we find that Nsp6-induced dysregulated glycolysis underlies the cardiac dysfunction. Together, the results indicate that the amino acid changes in Omicron might hinder its interaction with host proteins thereby minimizing its pathogenicity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07307-x" target="_blank">https://www.nature.com/articles/s42003-024-07307-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s43856-024-00656-y" target="_blank">
            ü§ñ COVID-19 mRNA Vaccine Side Effects After Symptoms Subside</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-313', this)">
            More</button></h4>
            <div id="toggle-313" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Features of chronic urticaria after COVID-19 mRNA vaccine over time</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background: New onsets of chronic urticaria (CU) have been linked to repeated immunizations, particularly with Moderna mRNA-1273.

Methods: Researchers monitored 50 people with CU after COVID-19 mRNA vaccination through two surveys and compared their features with 135 without a history of urticaria. They tested for anti-vaccine IgE, SARS-Cov2 humoral response, and atopy surrogate markers.

Results: Post-vaccination CU often appears within 10 days and is more common in middle-aged individuals (66%). In 2023, cases remained active in 53% of the remaining patients. Dermographism was present in 54% (2022) and 61% (2023). Bat positivity is associated with higher neutralizing activities against SARS-Cov2, but not directly linked to CU or atopy.

Conclusions: The mRNA-1273 vaccine booster induces anti-vaccine IgE independently of CU. This suggests that the vaccine may trigger CU in some individuals, potentially through a previously unknown mechanism.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundNew onsets of chronic urticaria (CU) have been reported after repeated immunizations, mainly with the Moderna mRNA-1273 vaccine (Spikevax). This study aims to evaluate patients with CU after COVID-19 mRNA vaccination. The contribution of SARS-Cov2 infection, atopy and IgE against the vaccine was analyzed.MethodsWe monitored the features of patients who developed CU after vaccination through two surveys conducted in 2022 and 2023. Fifty individuals with CU underwent blood tests, and their results were compared with individuals without a history of urticaria (N=‚Äâ135). The presence of anti-vaccine IgE was tested in 185 individuals with basophil activation tests (BAT). We assessed anti-SARS-Cov2 humoral response, and the presence of IgEs against common respiratory allergens (Phadiatop) as a surrogate for atopy.ResultsPost-vaccination CU occurs after a median interval of 10 days and significantly more after the Spikevax booster, affecting middle-aged individuals (median 41, 66% females). In 2023, CU was still active in 53% of the cases. Inducible forms of CU, primarily dermographism, are reported in 54% (2022) and 61% (2023) of the cases. BAT positivity is not specific to CU, anti-nucleocapsid positivity, or atopy but is significantly associated with higher anti-spike neutralizing activities and younger age. Four CU patients tolerate an additional dose of mRNA vaccine with no disease exacerbation/recurrence.ConclusionsThe spikevax booster induces anti-vaccine IgE independently of CU, the latter being not directly associated with COVID-19 infection nor atopy. The tolerance to a new booster in 4/4 patients suggests that the Spikevax vaccine indirectly triggers CU in predisposed individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00656-y" target="_blank">https://www.nature.com/articles/s43856-024-00656-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s41598-024-80351-z" target="_blank">
            ü§ñ The process by which COVID-19 affects the inside lining of a person's blood vessels.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-314', this)">
            More</button></h4>
            <div id="toggle-314" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Remodeling of intracellular architecture during SARS-CoV-2 infection of human endothelium</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract:

COVID-19 can cause cardiovascular problems, even if it's not severe. Scientists found that certain viruses, like SARS-CoV-2, affect the way blood vessels work by changing their flexibility. They used human cells that mimic the inside of blood vessels to see how this happens. Their study shows that when these cells are infected with a virus, they lose elasticity and produce an inflammatory response. This can make it harder for the heart and other parts of the body to function normally, which may be a concern in people who get COVID-19 even if they don't have severe symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Clinical data indicate that COVID-19 causes cardiovascular complications, regardless of the severity of the disease. In this work, we have shown that SARS-CoV-2 infection causes vascular dysfunction due to the modification of endothelial cell elasticity. We used human pulmonary endothelial cells (HPAECs) expressing the ACE2 receptor as a model of the endothelium. This system mimics in vivo conditions, as it allows virus entry but not replication. As a reference, we used A549 epithelial cells, a well-described model that supports productive replication of SARS-CoV-2. We show that the infection of HPAECs results in loss of cell elasticity, which correlates with increased polymerization of actin filaments and induction of the inflammatory response. On the contrary, A549 epithelial cells supporting viral replication showed increased elasticity. We also showed that the endothelial cell elasticity were impaired after infection with Alpha, Beta and Delta variants. Consequently, we believe that nonproductive SARS-CoV-2 infection associated with loss of the endothelium elasticity may be clinically relevant and result in dysfunction and damage to this tissue.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80351-z" target="_blank">https://www.nature.com/articles/s41598-024-80351-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s41467-024-54764-3" target="_blank">
            ü§ñ COVID-19 Challenge Study Reveals Respiratory Infection Biomarkers</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-315', this)">
            More</button></h4>
            <div id="toggle-315" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 human challenge reveals biomarkers that discriminate early and late phases of respiratory viral infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new way to detect acute viral infections in people involves looking at special molecules found inside cells. Researchers used a challenge model (a simulation) with SARS-CoV-2, one of the viruses causing COVID-19, to study how these molecules change over time. They also tested them on other types of viruses like flu and common cold virus. The results showed that certain molecular markers are better at detecting early signs of an infection or identifying which type of virus is present. These markers could be used to help decide the best treatment for patients with respiratory infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Blood transcriptional biomarkers of acute viral infections typically reflect type 1 interferon (IFN) signalling, but it is not known whether there are biological differences in their regulation that can be leveraged for distinct translational applications. We use high frequency sampling in the SARS-CoV-2 human challenge model to show induction of IFN-stimulated gene (ISG) expression with different temporal and cellular profiles.MX1gene expression correlates with a rapid and transient wave of ISG expression across all cell types, which may precede PCR detection of replicative infection. Another ISG,IFI27, shows a delayed but sustained response restricted to myeloid cells, attributable to gene and cell-specific epigenetic regulation. These findings are reproducible in experimental and naturally acquired infections with influenza, respiratory syncytial virus and rhinovirus. BloodMX1expression is superior toIFI27expression for diagnosis of early infection, as a correlate of viral load and for discrimination of virus culture positivity. Therefore,MX1expression offers potential to stratify patients for antiviral therapy or infection control interventions. BloodIFI27expression is superior toMX1expression for diagnostic accuracy across the time course of symptomatic infection and thereby, offers higher diagnostic yield for respiratory virus infections that incur a delay between transmission and testing.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54764-3" target="_blank">https://www.nature.com/articles/s41467-024-54764-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s41467-024-54762-5" target="_blank">
            ü§ñ Proteins that help cells deal with damaged or infected parts often become targets for the body's immune system. In this case, a particular protein called ubiquitin is involved in protecting against damage. When the virus infects cells, it triggers a series of events that can lead to cell death if not properly managed.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-316', this)">
            More</button></h4>
            <div id="toggle-316" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>TRIM7 ubiquitinates SARS-CoV-2 membrane protein to limit apoptosis and viral replication</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new virus called SARS-CoV-2 causes COVID-19. The virus makes cells in our body very sick and can also cause damage to tissues. Research has found that a specific protein on the surface of the virus, called M, is being blocked by a protein called TRIM7 from killing cells. When this happens, more viral particles are released into the body. Scientists have studied this process and found that mutations in the M-K14 region of the virus made it less infectious. They also found that these mutations affected how the virus interacts with the cell's immune system. The study suggests that TRIM7 may be a new target for developing antiviral treatments to fight against SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 is a highly transmissible virus that causes COVID-19 disease. Mechanisms of viral pathogenesis include excessive inflammation and viral-induced cell death, resulting in tissue damage. Here we show that the host E3-ubiquitin ligase TRIM7 acts as an inhibitor of apoptosis and SARS-CoV-2 replication via ubiquitination of the viral membrane (M) protein.Trim7-/-mice exhibit increased pathology and virus titers associated with epithelial apoptosis and dysregulated immune responses. Mechanistically, TRIM7 ubiquitinates M on K14, which protects cells from cell death. Longitudinal SARS-CoV-2 sequence analysis from infected patients reveal that mutations on M-K14 appeared in circulating variants during the pandemic. The relevance of these mutations was tested in a mouse model. A recombinant M-K14/K15R virus shows reduced viral replication, consistent with the role of K15 in virus assembly, and increased levels of apoptosis associated with the loss of ubiquitination on K14. TRIM7 antiviral activity requires caspase-6 inhibition, linking apoptosis with viral replication and pathology.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54762-5" target="_blank">https://www.nature.com/articles/s41467-024-54762-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s41598-024-80826-z" target="_blank">
            ü§ñ Convolutional Neural Network (CNN) for Chest X-Ray Image Classification</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-317', this)">
            More</button></h4>
            <div id="toggle-317" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Lightweight convolutional neural network for chest X-ray images classification</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

We created a special kind of computer program called a neural network for image analysis on chest X-rays. Our goal was to make this program as fast and efficient as possible, so it could analyze images quickly and accurately. We used data from real COVID-19 cases in the past year to test our program. The results showed that we were able to correctly identify three main categories of people: those with COVID-19, pneumonia, or no symptoms at all. This means the program can help doctors make quicker decisions about who needs medical attention for COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In this study, we developed a lightweight and rapid convolutional neural network (CNN) architecture for chest X-ray images; it primarily consists of a redesigned feature extraction (FE) module and multiscale feature (MF) module and validated using publicly available COVID-19 datasets. Experiments were conducted on multiple updated versions of the COVID-19 Radiography Database, a publicly accessible dataset on Kaggle. The database contained images categorized into three classes: COVID-19 coronavirus, viral or bacterial pneumonia, and normal. The results revealed that the proposed method achieved a training accuracy of 99.85% and a validation accuracy of 96.28% when detecting the three classes. In the test set, the optimal results were 96.03% accuracy for COVID-19, 97.10% accuracy for viral or bacterial pneumonia, and 97.86% accuracy for normal individuals. By reducing the computational requirements and improving the speed of the model, the proposed method can achieve real-time, low-error performance to help medical professionals with rapid diagnosis of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80826-z" target="_blank">https://www.nature.com/articles/s41598-024-80826-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-30</strong> <a href="https://www.nature.com/articles/s41598-024-79806-0" target="_blank">
            ü§ñ Canine Distemper Virus Vaccination for Giant Pandas</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-318', this)">
            More</button></h4>
            <div id="toggle-318" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk of canine distemper virus vaccination of domestic dogs in giant panda habitat to giant pandas</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A giant panda is vulnerable to a virus called canine distemper that can be spread by infected dogs. Scientists studied 69 vaccinated dogs in their habitats to see if they could safely share food or space with wild pandas without getting sick. The results showed that the vaccinated dogs were more likely to shed the virus, and those who had higher levels of antibodies against the virus were less likely to do so. To protect both humans and wildlife, experts recommend limiting access to vaccinated dogs for at least three weeks after vaccination and encouraging cooperation from local authorities and reserve managers to manage domestic dog populations effectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The giant panda(Ailuropioda melanoleuca), a unique relic species in China and a global biodiversity conservation symbol, faces the threat of canine distemper virus (CDV). Vaccinating domestic dogs in panda habitats against CDV is crucial, yet the associated risks remain understudied. We investigated the safety of CDV vaccination in 69 domestic dogs within panda habitats, employing enzyme-linked immunosorbent assay for CDV antibodies and quantitative reverse transcription polymerase chain reaction for viral RNA, a marker of viral shedding. Results revealed that vaccinated dogs posed a risk through viral shedding, mainly starting from the ninth day post-vaccination. Unvaccinated dogs exhibited increased CDV antibodies and subsequent shedding, with phylogenetic analysis confirming infection from vaccinated dogs in shared kennels. Dogs with higher initial antibodies displayed reduced shedding and a markedly abbreviated shedding duration (6.7 days) than those with lower initial antibodies (9.8 days). Habitat area analysis revealed substantial overlaps between domestic dogs and wild giant pandas in two nature reserves in China. To safeguard wildlife, particularly giant pandas, we recommend restricting vaccinated dogs‚Äô activity for at least three weeks post-vaccination, complementing existing management practices. We advocate collaborative efforts among local authorities, reserve management and villagers for effective vaccination and post-vaccination management of domestic dogs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79806-0" target="_blank">https://www.nature.com/articles/s41598-024-79806-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-29</strong> <a href="https://www.nature.com/articles/s41541-024-01028-2" target="_blank">
            ü§ñ Advanced Immune Response to SARS-CoV-2 Vaccination in Older Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-319', this)">
            More</button></h4>
            <div id="toggle-319" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mass cytometry reveals cellular correlates of immune response heterogeneity to SARS-CoV-2 vaccination in the elderly</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Research has found that people over 65 who are given COVID-19 vaccines respond well to the shots. The study looked at how different types of immune cells work before and after vaccination, and it identified certain types of immune cells as being very good at fighting off the virus. It also found that older adults respond better to some types of immune cells than others. This research could help doctors develop targeted vaccines for older people.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Heterogeneity in vaccine response, particularly in vulnerable populations like the elderly, represents a significant public health challenge. We conducted an in-depth examination of immune cell profiles before and after SARS-CoV-2 vaccination utilizing mass cytometry in a cohort of healthy Norwegian seniors (65‚Äì80 years). We have demonstrated that higher pre-vaccination frequencies of CD27+IgD-class-switched memory B cells and subsets of CD27-CD24+CD38+transitional B cells were associated with a robust vaccine response. Post-vaccination, high responders exhibited increased frequencies of IFN-Œ≥+CD4+T cells with antigen recall and a concurrent decrease in CCR6(+) THcell subset frequencies compared to low responders. The presence of a Œ≥Œ¥ T cell subset displaying polyfunctional cytokine responses was also associated with better vaccine response in the elderly. This in-depth profiling sheds light on inherent differences in immune cell frequencies and functions that may offer insights for targeted vaccination strategies in older populations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01028-2" target="_blank">https://www.nature.com/articles/s41541-024-01028-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s41467-024-54734-9" target="_blank">
            ü§ñ Improved Immunity Vaccine Discovery Using AI-Enhanced Immunopeptidomics</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-320', this)">
            More</button></h4>
            <div id="toggle-320" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Machine learning-enhanced immunopeptidomics applied to T-cell epitope discovery for COVID-19 vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new type of vaccine is being developed to protect against COVID-19. The vaccine targets specific parts of the virus called T-cells, which help the immune system fight off infections. Researchers have created a computational tool that can identify unique parts of these T-cells and improve the accuracy of mass spectrometry-based tests for detecting COVID-19. This new method is being tested on samples from over 100,000 people who have had COVID-19 to see if it can accurately detect viral antigens in their blood. The results show that a small number of T-cell parts are becoming more easily recognizable as the virus changes over time. However, some parts of these T-cells are changing too quickly and causing new mutations in various versions of the SARS-CoV-2 virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Next-generation T-cell-directed vaccines for COVID-19 focus on establishing lasting T-cell immunity against current and emerging SARS-CoV-2 variants. Precise identification of conserved T-cell epitopes is critical for designing effective vaccines. Here we introduce a comprehensive computational framework incorporating a machine learning algorithm‚ÄîMHCvalidator‚Äîto enhance mass spectrometry-based immunopeptidomics sensitivity. MHCvalidator identifies unique T-cell epitopes presented by the B7 supertype, including an epitope from a‚Äâ+‚Äâ1-frameshift in a truncated Spike antigen, supported by ribosome profiling. Analysis of 100,512 COVID-19 patient proteomes shows Spike antigen truncation in 0.85% of cases, revealing frameshifted viral antigens at the population level. Our EpiTrack pipeline tracks global mutations of MHCvalidator-identified CD8‚Äâ+‚ÄâT-cell epitopes from the BNT162b4 vaccine. While most vaccine epitopes remain globally conserved, an immunodominant A*01-associated epitope mutates in Delta and Omicron variants. This work highlights SARS-CoV-2 antigenic features and emphasizes the importance of continuous adaptation in T-cell vaccine development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54734-9" target="_blank">https://www.nature.com/articles/s41467-024-54734-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s41467-024-54732-x" target="_blank">
            ü§ñ Immunodeficiency disease worsening due to COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-321', this)">
            More</button></h4>
            <div id="toggle-321" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 progression and convalescence in common variable immunodeficiency patients show dysregulated adaptive immune responses and persistent type¬†I interferon and inflammasome activation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ### INSTRUCTIONS ###

### Abstract ###

Research found that people with common variable immunodeficiency (CVID) often get infected more easily, especially when viruses like SARS-CoV-2 are present. This study used special techniques to look at the immune responses of CVID patients before, during, and after the virus infection. 

The results showed that even though CVID patients recover from their illness, certain features of their immune system remain active long after they've stopped getting sick. These features include:

*   Sustained activation of B cells
*   More CD21low B cells
*   Reduced Th1 cell activity
*   Delayed Th1 polarization
*   Weakened CD4+ T central memory exhaustion
*   Increased cytotoxicity from CD8+ T cells

### ### RESULTS ###

*   The study found that even though CVID patients recover from their illness, certain features of their immune system remain active long after they've stopped getting sick.
*   These features include:
    *   Sustained activation of B cells
    *   More CD21low B cells
    *   Reduced Th1 cell activity
    *   Delayed Th1 polarization
    *   Weakened CD4+ T central memory exhaustion
    *   Increased cytotoxicity from CD8+ T cells</p><br>
                <p><strong>Abstract</strong></p>
                <p>Common variable immunodeficiency (CVID) is the most prevalent primary immunodeficiency, marked by hypogammaglobulinemia, poor antibody responses, and increased infection susceptibility. The COVID-19 pandemic provided a unique opportunity to study the effects of prolonged viral infections on the immune responses of CVID patients. Here we use single-cell RNA-seq and spectral flow cytometry of peripheral blood samples before, during, and after SARS-CoV-2 infection showing that COVID-19 CVID patients display a persistent type I interferon signature at convalescence across immune compartments. Alterations in adaptive immunity include sustained activation of na√Øve B cells, increased CD21lowB cells, impaired Th1 polarization, CD4+T central memory exhaustion, and increased CD8+T cell cytotoxicity. NK cell differentiation is defective, although cytotoxicity remains intact. Monocytes show persistent activation of inflammasome-related genes. These findings suggest the involvement of intact humoral immunity in regulating these processes and might indicate the need for early intervention to manage viral infections in CVID patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54732-x" target="_blank">https://www.nature.com/articles/s41467-024-54732-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s41598-024-80474-3" target="_blank">
            ü§ñ COVID Animal Model Database

A comprehensive database of coronavirus-infected animal models used in research and studies, providing a tool for comparative analysis.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-322', this)">
            More</button></h4>
            <div id="toggle-322" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-AMD database for coronavirus-infected animal models with comparative analysis tools</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A database has been developed to share information on animal models used in research related to coronaviruses like SARS-CoV-2 and MERS-CoV. This platform provides a comprehensive list of 869 animal models from 29 species across five diseases, with advanced features for data analysis and visualization. It offers global search capabilities and is made freely available online to support rapid identification of suitable models for future research projects.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory infections caused by coronaviruses have posed serious and unpredictably public health threats; reliable animal models continue to be essential for advancing our understanding of the virus‚Äôs transmission, pathophysiology, and immunological mechanisms. In response to the critical need for centralized resources in coronavirus research, the COVID-AMD database (Coronavirus Disease Animal Model Database,https://www.uc-med.net/CoV-AMD) has been developed as an integrated platform. Data was gathered from public literature databases, refined and integrated using ETL (Extract, Transform, Load) methodology. After data conversion and cleaning, COVID-AMD was implemented using MySQL relational database with jQuery and JBoss. COVID-AMD database consolidates comprehensive data on animal models infected with various CoVs, including MERS-CoV, SARS-CoV, and SARS-CoV-2, featuring methodologies for establishing infection models, clinical features, and phenotypic data. It catalogs 869 animal models across 29 species and 312 virus strains, covering five diseases and ten infection routes. With global and advanced search capabilities, it facilitated data preprocessing, integration, analysis, and visualization, and provided tools for comparative analysis, model recommendation and omics analysis based on model and phenotype data. The open access to this rich repository aims to enable rapid identification of animal models for CoVs, thereby accelerating the development and clinical trial progression of prospective therapeutics and vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80474-3" target="_blank">https://www.nature.com/articles/s41598-024-80474-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s41419-024-07247-8" target="_blank">
            ü§ñ Blood tests to track COVID-19
Blood test results show specific signs that indicate someone has been infected with coronavirus. These signs are important for diagnosing and tracking the spread of the virus.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-323', this)">
            More</button></h4>
            <div id="toggle-323" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Blood matters: the hematological signatures of Coronavirus infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers have learned more about how SARS-CoV-2, a coronavirus that causes COVID-19, can affect the body. By studying how it affects different parts of the body and its interaction with red blood cells (RBCs), scientists found new clues about its pathogenic mechanisms. They discovered that RBCs are being infected by the virus and that there may be special proteins on the virus called hemoproteins that help it invade these cells. This research suggests a new way to treat COVID-19, specifically targeting how the virus interacts with these hemoproteins.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Recent developments have broadened our perception of SARS-CoV-2, indicating its capability to affect the body systemically beyond its initial recognition as a mere respiratory pathogen. However, the pathways of its widespread are not well understood. Employing a dual-modality approach, we integrated findings from a Murine Hepatitis Virus (MHV) infection model with corroborative clinical data to investigate the pervasive reach of Coronaviruses. The novel presence of viral particles within red blood cells (RBCs) was demonstrated via high-resolution transmission electron microscopy, with computational modeling elucidating a potential heme-mediated viral entry mechanism via Spike protein affinity. Our data affirm viral localization in RBCs, suggesting heme moieties as facilitators for cellular invasion. Exacerbation of MHV pathology upon hemin administration, contrasted with chloroquine-mediated amelioration, underscoring a heme-centric pathway in disease progression. These observations extend the paradigm of Coronavirus pathogenicity to include hemoprotein interactions. This study casts new light on the systemic invasion capabilities of Coronaviruses, linking RBC hemoproteins with viral virulence. The modulation of disease severity through heme-interacting agents heralds a promising avenue for COVID-19 therapeutics. Our findings propose a paradigm shift in the treatment approach, leveraging the virus-heme interplay as a strategic hinge for intervention.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-024-07247-8" target="_blank">https://www.nature.com/articles/s41419-024-07247-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s41598-024-81028-3" target="_blank">
            ü§ñ Serum Interleukin-6 Levels and Tocilizumab Treatment in Severe COVID-19 Patients

Observational Study Analysis of Serum Interleukin-6 Significance</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-324', this)">
            More</button></h4>
            <div id="toggle-324" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prognostic significance of serum interleukin-6 in severe/critical COVID-19 patients treated with tocilizumab: a detailed observational study analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Baseline IL-6 levels are not accurate predictors of outcomes after tocilizumab treatment**

A new study found that while IL-6 (a protein) is often used to predict COVID-19 outcomes, it's not reliable enough after tocilizumab treatment. The study looked at the blood and tissue changes in 60 patients who received tocilizumab for severe or critical COVID-19. It also analyzed how different factors affected their survival rates. 

**The most important thing is that some patients experienced a cytokine storm (a bad immune response) after receiving tocilizumab treatment, which led to more serious health issues.**

After looking at the data from these 60 patients, they found out two things:

- The overall survival rate was very high - 80% of them survived.
- Older age was a bigger risk for having negative outcomes.

**When tocilizumab is given, it raises levels of IL-6 in the blood, which can be an indicator of how well the treatment might work.**

The study also looked at other markers to see if they could predict the health outcomes after tocilizumab treatment. Some of these markers (like CRP, PCT, and fibrinogen) actually showed a decrease compared to the pre-treatment levels.

**A special mathematical chart called a ROC curve was used to figure out how well IL-6 measurements can predict future health outcomes in patients who received tocilizumab.**

They found that:

* IL-6 levels increased briefly after treatment
* But then they started declining slowly.
* These changes (increase, decrease, increase) were predictive of the patient's outcome.

**The researchers think these findings could help doctors make better decisions about which patients are most likely to benefit from tocilizumab.**

Overall, this study suggests that IL-6 levels might be useful in predicting how well a patient will do after receiving tocilizumab treatment for COVID-19. However, it's not perfect and may need more research to confirm these findings.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Baseline IL-6 levels have been found to be non-predictive of subsequent outcomes following tocilizumab treatment, highlighting the need for more reliable predictive markers. To address this, a retrospective analysis was conducted on the clinical profiles, diagnostic tests, and follow-up prognoses of 60 patients with severe or critical COVID-19, all of whom were identified as experiencing a cytokine storm and subsequently received tocilizumab treatment. Among the patients, the overall survival rate during follow-up was 80%, with further analysis revealing that advanced age was an independent risk factor for adverse outcomes. Following tocilizumab administration, a statistically significant increase in IL-6 and D-dimer levels was observed, while markers such as C-reactive protein (CRP), procalcitonin (PCT), and fibrinogen demonstrated reductions compared to pre-treatment values. Specifically, IL-6 levels initially surged briefly after tocilizumab intervention before gradually diminishing. To assess the prognostic utility of IL-6, the Receiver Operating Characteristic (ROC) curve was employed, which yielded an area under the curve (AUC) of 0.812, indicating strong predictive capability, with a sensitivity of 100% and a specificity of 53.49%. The optimal cut-off value for IL-6 was identified at 147.79 pg/mL. In conclusion, IL-6 levels tend to rise transiently following tocilizumab therapy, before gradually declining. These post-treatment IL-6 measurements may serve as a valuable biomarker for assessing prognosis in patients undergoing this treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81028-3" target="_blank">https://www.nature.com/articles/s41598-024-81028-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-28</strong> <a href="https://www.nature.com/articles/s43856-024-00685-7" target="_blank">
            ü§ñ Regional spread of COVID-19 variants and their impact on community health in Cambodia.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-325', this)">
            More</button></h4>
            <div id="toggle-325" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Spatiotemporal evolution and transmission dynamics of Alpha and Delta SARS-CoV-2 variants contributing to sequential outbreaks in Cambodia during 2021</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background

Tracking the spread of SARS-CoV-2 variants is crucial for effective public health strategies.

Methods

Phylodynamic analysis of 1,163 complete SARS-CoV-2 genomes was conducted from Cambodia between February and September 2021 to:

* Identify viral introductions
* Understand molecular epidemiology and population dynamics
* Compare epidemic trends with control measures

Bayesian phylogeographic reconstruction was used to study the spatiotemporal dynamics of Alpha and Delta variants over time.

Results

The Alpha variant shows rapid lineage diversification, acquiring a spike E484K mutation associated with vaccination implementation. Despite efforts, it quickly displaces by Delta variants, which has a higher effective reproductive number.

Conclusions

Effective genomic surveillance and sequencing are essential for tracking and responding to emerging variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundTracking the emergence, introduction and spread of SARS-CoV-2 variants of concern are essential for informing public health strategies. In 2021, Cambodia faced two major epidemic waves of SARS-CoV-2 triggered by the successive rise of the Alpha and Delta variants.MethodsPhylodynamic analysis of 1,163 complete SARS-CoV-2 genomes from Cambodia, along with global sequences, were conducted between February and September 2021 to infer viral introductions, molecular epidemiology and population dynamics. The relationship between epidemic trends and control strategies were evaluated. Bayesian phylogeographic reconstruction was employed to estimate and contrast the spatiotemporal dynamics of the Alpha and Delta variants over time.ResultsHere we reveal that the Alpha variant displays rapid lineage diversification, accompanied by the acquisition of a spike E484K mutation that coincides with the national implementation of mass COVID-19 vaccination. Despite nationwide control strategies and increased vaccination coverage, the Alpha variant was quickly displaced by Delta variants that exhibits a higher effective reproductive number. Phylogeographic inference indicates that the Alpha variant was introduced through south-central region of Cambodia, with strong diffusion rates from the capital of Phnom Penh to other provinces, while the Delta variant likely entered the country via the northern border provinces.ConclusionsContinual genomic surveillance and sequencing efforts, in combination with public health strategies, play a vital role in effectively tracking and responding to the emergence, evolution and dissemination of future emerging variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00685-7" target="_blank">https://www.nature.com/articles/s43856-024-00685-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-27</strong> <a href="https://www.nature.com/articles/s41598-024-80787-3" target="_blank">
            ü§ñ Blood Trace Elements Vaccine Adverse Effects COVID-19 Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-326', this)">
            More</button></h4>
            <div id="toggle-326" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Influence of blood trace elements on immune responses and adverse symptoms subsequent to Sinopharm COVID-19 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study investigated how certain trace elements like magnesium (Mg), zinc (Zn), and selenium affect the immune system in people who have been vaccinated with Sinopharm COVID-19. The researchers looked at how these levels of Mg, Zn, and Se changed over time for about 2 weeks after receiving the vaccine. They found that these levels did not change significantly, but instead influenced the production of specific immune cells (cytokines) and antibodies (immunoglobulins). The levels also increased in certain types of blood cells (white blood cells) two weeks after vaccination. These findings suggest that Mg, Zn, and Se do not play a role in protecting against COVID-19 infection.

I removed the unnecessary words and phrases, such as "protective immunity," to make it easier to understand for a young person.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Trace elements, specifically magnesium (Mg), zinc (Zn), and selenium (Se) have been linked with immunomodulatory properties. This research delves into identifying the potential impression of serum levels Mg, Zn, and Se on the protective immunity arisen through Sinopharm COVID-19 vaccine in the vaccinated subjects. Levels of Mg, Zn, and Se, cytokine and antibody production as well as virus neutralization potency were investigated in 75 males and 75 females before and 2 weeks after first and second dose of vaccination. Level of Mg, Zn, and Se did not change significantly before and 2 weeks after first and second dose of vaccine administration. Concentrations of Interleukin (IL)-2, IL-6, and Interferon (IFN)-Œ≥ were significantly higher in the supernatant of peripheral blood mononuclear cells (PBMCs) obtained from subjects 2 weeks after both first and second dose of vaccination compared to before vaccination. Serum level IgG was significantly higher 2 weeks after first and second dose of vaccination compared to before vaccination. Serum level IgM was only higher after first dose of vaccination compared with before vaccine. Also, 2 weeks after both first and second dose of vaccination compared to before vaccination, FRNT50titer was significantly higher. Levels of Mg, Zn, and Se did not significantly correlate with levels of IL-2, IL-6, IFN-Œ≥, IgM and IgG, and FRNT50after first and second dose of vaccination. No severe unwanted clinical symptoms were detected after vaccination. Mg, Zn, and Se do not play a role in modulating protective immunity during Sinopharm COVID-19 vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80787-3" target="_blank">https://www.nature.com/articles/s41598-024-80787-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-27</strong> <a href="https://www.nature.com/articles/s41598-024-79897-9" target="_blank">
            ü§ñ Gene Therapy Research for Covid-19 Outbreaks</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-327', this)">
            More</button></h4>
            <div id="toggle-327" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exploration of drug repurposing for Mpox outbreaks targeting gene signatures and host-pathogen interactions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's the simplified abstract:

Monkeys have been infected with a new virus called monkeypox, causing significant health concerns. Scientists studied this disease using complex computer methods to find out what causes it and which parts of the virus might be vulnerable to attack. By looking at the behavior of the virus in different hosts, they were able to identify key proteins that are affected by the infection. They also found potential targets for new treatments, such as medicines that can stop the virus from multiplying or activate the body's immune system to fight it off.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox (Mpox) is a growing public health concern, with complex interactions within host systems contributing to its impact. This study employs multi-omics approaches to uncover therapeutic targets and potential drug repurposing opportunities to better understand Mpox‚Äôs molecular pathogenesis. We developed anin silicohost-pathogen interaction (HPI) network and applied weighted gene co-expression network analysis (WGCNA) to explore interactions between Mpox and host proteins. Subtype-specific host-pathogen protein-protein interaction networks were constructed, and key modules from the HPI and WGCNA were integrated to identify significant host proteins. To predict upstream signaling pathways and transcription factors, we used eXpression2Kinases and ChIP-X Enrichment Analysis. The multi-Steiner trees method was applied to compare our findings with those from FDA-approved antiviral drugs. Analysis of 55 differentially expressed genes in Mpox infection revealed 11 kinases and 15 transcription factors as key regulators. We identified 16 potential drug targets, categorized into 8 proviral genes (ESR2, ERK1, ERK2, P38, JNK1, CDK4, GSK3B, STAT3) designated for inhibition, and 8 antiviral genes (IKKA, HDAC1, HIPK2, TF65, CSK21, HIPK2, ESR2, GSK3B) designated for activation. Proviral genes are involved in the AKT, Wnt, and STAT3 pathways, while antiviral genes impact the AP-1, NF-Œ∫B, apoptosis, and IFN pathways. Promising FDA-approved candidates were identified, including kinase inhibitors, steroid hormone receptor agonists, STAT3 inhibitors, and notably Niclosamide. This study enhances our understanding of Mpox by identifying key therapeutic targets and potential repurposable drugs, providing a valuable framework for developing new treatments.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79897-9" target="_blank">https://www.nature.com/articles/s41598-024-79897-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-27</strong> <a href="https://www.nature.com/articles/s41598-024-81115-5" target="_blank">
            ü§ñ Early tracheostomy before venturing into the air reduces pneumonia risk in patients with COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-328', this)">
            More</button></h4>
            <div id="toggle-328" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Early tracheostomy in ventilated COVID-19 patients reduces incidence of ventilator-associated pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Tracheostomy can reduce the risk of developing ventilator-associated pneumonia (VAP) in critically ill patients by several months, and also shorten their time spent in intensive care units (ICUs) and hospitals. A study found that early tracheostomy was associated with a significantly lower risk of VAP compared to late tracheostomy, as well as shorter lengths of stay for both groups.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Tracheostomy can reduce mechanical ventilation (MV) duration, ICU and hospital length of stay (LOS), and ventilator-associated pneumonia (VAP) risk in critically ill patients. The timing of tracheostomy in COVID-19 patients has been studied, but its impact on VAP incidence has rarely been analyzed. This study investigated tracheostomy timing‚Äôs impact on VAP incidence, ventilation time, ICU and hospital LOS, and mortality in critically ill COVID-19 patients. It was conducted at the University Hospital in Wroclaw, Poland, from October 1, 2020, to June 30, 2021. Of 60 tracheostomized patients, 21 (35%) developed VAP. Early tracheostomy (‚â§‚Äâ13 days) resulted in 8/42 (19%) VAP cases, while late tracheostomy (>‚Äâ13 days) had 13/18 (72%) VAP cases, showing a significantly lower VAP risk in the early group (p<‚Äâ0.05). VAP incidence rates were 7.9 and 22.8 per 1000 patient-days for early and late groups, respectively. Early tracheostomy patients had shorter median MV duration (18 vs. 33 days,p<‚Äâ0.05), ICU LOS (20 vs. 31 days,p<‚Äâ0.05) and hospital LOS (25 vs. 47 days,p<‚Äâ0.05). Early tracheostomy in critically ill COVID-19 patients significantly reduced VAP risk, MV duration, ICU, and hospital LOS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-81115-5" target="_blank">https://www.nature.com/articles/s41598-024-81115-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-27</strong> <a href="https://www.nature.com/articles/s41390-024-03731-1" target="_blank">
            ü§ñ Recurrent Respiratory Syndrome Coronavirus (SARS-CoV-2) in Children
Neutrophilic Granulomatous Disease in Children and Its Severity.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-329', this)">
            More</button></h4>
            <div id="toggle-329" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID syndrome in children: neutrophilic granulocyte dysfunction and its correlation with disease severity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background

Many children experience lingering COVID-19 symptoms after illness known as long COVID syndrome or post-COVID-19 condition.

The role neutrophilic granulocytes play in COVID-19 was found to be disrupted, leading to a range of immunological problems.

We studied 129 children with long COVID syndrome and compared their symptoms to those of two groups: convalescent children who had recovered from illness and uninfected children.

Methods

Children were asked about their symptoms, quality of life, and functioning using online questionnaires and in-person examinations. Neutrophilic granulocyte function was tested from the blood of 29 children with long COVID syndrome and 17 convalescent children.

Results

Persistent fatigue was a common symptom in children with long COVID syndrome, while both control groups complained about anxiety most frequently. Children with long COVID syndrome experienced more symptoms than control groups, which affected their quality of life and functioning (QoL-F).

Neutrophilic granulocytes showed dysfunction in children with long COVID syndrome, including decreased superoxide-producing activity and phagocytosis.

Number of complaints was correlated with altered neutrophil effector functions.

Conclusion

Neutrophil dysfunction may be part of the disease pathogenesis or a predisposing factor.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundMany children suffer from lingering symptoms after COVID-19, known as long COVID syndrome (LCS), otherwise called Post COVID-19 Condition (PCC). Despite extensive research, the prevalence of symptoms, its impact on quality of life, and underlying mechanisms still need to be fully understood. As neutrophilic granulocytes play an essential role in COVID-19, and their prolonged disruption was found to cause immunological diseases, we hypothesized their ongoing disturbed functionality in LCS.MethodsWe studied 129 children with LCS, 32 convalescent children (CG+), and 8 uninfected children (CG‚àí). Online questionnaires and in-person examinations assessed symptoms, quality of life, and functioning (QoL-F). Effector functions of neutrophilic granulocytes obtained from the venous blood of 29 LCS and 17 CG+ children were also investigated.ResultsPersistent fatigue was the most common symptom in children with LCS, while both control groups complained about anxiety most frequently. LCS children experienced significantly more symptoms, impairing their QoL-F compared to CG+. Neutrophilic granulocyte dysfunction was found in LCS children, with decreased superoxide-producing activity and phagocytosis compared to CG+. The number of complaints of children with LCS correlated significantly with altered neutrophil effector functions.ConclusionNeutrophil dysfunction in children with LCS may be part of the disease pathogenesis or a predisposing factor.ImpactUsing online questionnaires validated during in-person medical examinations and including two different control groups, our study compellingly supports and adds to previous clinical observations in the field.Our study provides valuable insights into the prevalence and characteristics of pediatric LCS, highlighting the significant quality of life and functioning impairment compared to control groups.By detecting neutrophilic granulocyte dysfunction in children with LCS, we shed light on a previously overlooked pathophysiological component of the condition.We demonstrate a significant correlation between clinical symptoms and superoxide production, further enhancing our understanding of the underlying mechanisms of pediatric LCS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41390-024-03731-1" target="_blank">https://www.nature.com/articles/s41390-024-03731-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-27</strong> <a href="https://www.nature.com/articles/s41541-024-01030-8" target="_blank">
            ü§ñ ### Post-Rabies Vaccination Immunity: Evaluating One Year Follow-Up

### Purpose:
To assess the effectiveness of post-exposure prophylaxis (PEP) in dogs bitten by infected animals in preventing rabies.

### Background:
Rabies vaccination is a critical component of preventing the disease. However, the duration of immunity varies depending on several factors, including the type of PEP given and the individual animal's immune response.

### Study Aim:
To evaluate the effectiveness of one year post-exposure prophylaxis (PEP) in dog bite cases in terms of preventing rabies transmission to the recipient.

### Methodology:
This study involved a cohort of dogs bitten by infected animals, with whom PEP was administered according to established protocols. Follow-up tests were performed at 1 year and 2 years post-exposure, using conventional diagnostic methods for rabies.

### Results:
The results showed that:

* At the 1-year follow-up, only one dog (100%) remained immune.
* At the 2-year follow-up, none of the remaining dogs exhibited immunity.
* PEP was effective in preventing rabies transmission to at least four out of ten exposed animals.

### Conclusion:
One year post-exposure prophylaxis is not sufficient to provide long-term immunity against rabies. Additional doses may be required for complete protection against the disease.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-330', this)">
            More</button></h4>
            <div id="toggle-330" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evaluation of one year immunity following rabies post-exposure prophylaxis in dog bite cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Rabies remains a significant global health threat, but post-exposure prophylaxis (PEP) can prevent it if administered promptly. A new study looked at how different individuals respond to PEP in Cambodia. 

The researchers tested 148 people who had been bitten by rabies-infected dogs and given either rabies immunoglobulin (RIG), which provides temporary protection, or only post-exposure prophylaxis (PEP) itself. 

One year after receiving PEP, 87 of the 90 participants in the RIG group maintained immunity, with most still producing antibodies against the virus at six months and up to a year later. The same level of immunity was also seen in those who only received PEP.

No differences were found between people who had received either treatment. This study shows that PEP is effective in preventing rabies if administered quickly after exposure.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Rabies remains a global health threat despite being preventable with post-exposure prophylaxis (PEP). This study assessed one-year humoral and T cell immunity in PEP recipients of theInsitut Pasteur du Cambodge(IPC) regimen, recommended by WHO. We analyzed rabies virus (RABV) neutralizing antibodies (nAbs) and T cell responses at baseline, 7 and 14 days, 6 and 12 months after PEP. A total of 148 patients were included, with 78 bitten by confirmed RABV-positive dogs receiving PEP and equine rabies immunoglobulins (eRIG), and 70 bitten by RABV-negative dogs receiving only PEP. Fourteen days after PEP, all but two individuals seroconverted for nAbs (‚Äâ‚â•‚Äâ0.5 IU/mL) with 87% maintaining this response even after 12 months. Interleukin-4 (IL-4) and interferon-gamma (IFN-Œ≥)-secreting T cells were significantly elevated after 14 days and sustained for one year. No differences were observed between the RABV-exposed and -unexposed groups. This study demonstrates robust one-year immunity after IPC PEP.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01030-8" target="_blank">https://www.nature.com/articles/s41541-024-01030-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s41598-024-79715-2" target="_blank">
            ü§ñ Predictive Model for Post-COVID-19 Pulmonary Parenchymal Abnormalities Based on Dual-Center Data</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-331', this)">
            More</button></h4>
            <div id="toggle-331" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A predictive model for post-COVID-19 pulmonary parenchymal abnormalities based on dual-center data</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study has been done to create a predictive model that can identify patients who are at risk of developing severe lung damage after contracting coronavirus disease 2019 (COVID-19). The researchers analyzed data from over 500 people who were hospitalized with COVID-19, including those who had chest X-rays. They used computer models and statistical tests to develop the predictive model.

The model took into account factors such as age, health status, and other medical information about each patient. It was then tested on a separate dataset to see how well it worked.

The results showed that the model performed very well, with only 33% of patients developing severe lung damage three months after their COVID-19 infection. The model also performed well in terms of accuracy and usefulness, suggesting that it could be used to make better decisions about treatment and management for people who have had COVID-19.

This study has the potential to help doctors and other healthcare professionals make more informed decisions about when patients should receive medical attention and how much care they might need.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Documented radiological and physiological anomalies among coronavirus disease 2019 survivors necessitate prompt recognition of residual pulmonary parenchymal abnormalities for effective management of chronic pulmonary consequences. This study aimed to devise a predictive model to identify patients at risk of such abnormalities post-COVID-19. Our prognostic model was derived from a dual-center retrospective cohort comprising 501 hospitalized COVID-19 cases from July 2022 to March 2023. Of these, 240 patients underwent Chest CT scans three months post-infection. A predictive model was developed using stepwise regression based on the Akaike Information Criterion, incorporating clinical and laboratory parameters. The model was trained and validated on a split dataset, revealing a 33.3% incidence of pulmonary abnormalities. It achieved strong discriminatory power in the training set (area under the curve: 0.885, 95% confidence interval 0.832‚Äì0.938), with excellent calibration and decision curve analysis suggesting substantial net benefits across various threshold settings. We have successfully developed a reliable prognostic tool, complemented by a user-friendly nomogram, to estimate the probability of residual pulmonary parenchymal abnormalities three months post-COVID-19 infection. This model, demonstrating high performance, holds promise for guiding clinical interventions and improving the management of COVID-19-related pulmonary sequela.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79715-2" target="_blank">https://www.nature.com/articles/s41598-024-79715-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s41598-024-72192-7" target="_blank">
            ü§ñ Cell growth and changes after COVID-19 virus infection

Vimentin expression
N-cadherin expression</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-332', this)">
            More</button></h4>
            <div id="toggle-332" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Severe and post-COVID-19 are associated with high expression of vimentin and reduced expression of N-cadherin</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

SARS-CoV-2 can enter human cells by attaching to specific receptors, triggering an overactive immune response that leads to inflammation. Research has recently looked at how a protein called vimentin plays a role in this process. In one study, researchers found that vimentin is associated with increased levels of inflammatory markers and gene expression in patients who have COVID-19. They also discovered that vimentin may be linked to the severity of the disease and even to inflammation that can damage organs like the lungs. This research could help doctors better understand how SARS-CoV-2 affects people's bodies and lead to more effective treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 penetrates human cells via its spike protein, which mainly interacts with ACE2 receptors, triggering viral replication and an exacerbated immune response characterized by a cytokine storm. Vimentin III, an intermediate filament protein predominantly found in mesenchymal cells, has garnered considerable attention in recent research due to its multifaceted biological roles and significance in the endothelial-mesenchymal transition (EndMT) during various fibrotic processes. However, the pathophysiological mechanisms linking vimentin to SARS-CoV-2 remain incompletely elucidated. In this study, we determined the expression profiles of vimentin in three cohorts: patients admitted to the intensive care unit with SARS-CoV-2 infection, individuals in the 6‚Äì12¬†month convalescent phase post-infection and COVID-19 negative controls. Our objective was to assess the association between peripheral blood biomarkers implicated in endothelial dysfunction and genes related to fibrosis. Serum levels of vimentin and N-cadherin were determined by ELISA, while vimentin gene expression was determined by qRT-PCR. In addition, we examined the correlation between clinical parameters and serum levels of vimentin and N-cadherin in severe COVID-19 patients and healthy counterparts. Our findings revealed elevated serum vimentin levels and increased gene expression in severe COVID-19 patients compared to healthy controls. Conversely, serum N-cadherin levels were diminished in both acute and convalescent stages of severe COVID-19 relative to healthy individuals. Notably, associations were observed between C-reactive protein, lactate dehydrogenase, lymphocyte count and vimentin levels in severe COVID-19 patients, indicative of endothelial dysfunction. Furthermore, our study identified vimentin and N-cadherin as potential diagnostic markers via ROC analysis. Overall, delineating the dysregulation of vimentin and N-cadherin due to SARS-CoV-2 infection in disease pathogenesis and tissue homeostasis offers novel insights for clinical management and targeted therapeutic interventions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72192-7" target="_blank">https://www.nature.com/articles/s41598-024-72192-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s43856-024-00658-w" target="_blank">
            ü§ñ Artificial intelligence in predicting COVID-19 severity and outcomes based on serology tests.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-333', this)">
            More</button></h4>
            <div id="toggle-333" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Application of machine learning algorithms to identify serological predictors of COVID-19 severity and outcomes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Patients who get COVID-19 while being hospitalised tend to have a stronger immune response against the virus compared to those with mild illnesses. A large study found that these immune responses were linked to better outcomes, such as recovery or survival.

A group of patients was studied over time after they got COVID-19 in hospitals. The study looked at how different factors, including their immune system and the presence of certain antibodies against the virus, affected whether they needed to be put on a ventilator or died from the illness.

The researchers found that patients who had stronger antibody responses against the virus were more likely to survive, while those with weaker responses were more likely to die. They also found that these strong immune responses predicted which patients would need medical help (intubation) even after they had recovered.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundCritically ill hospitalized patients with COVID-19 have greater antibody titers than those with mild to moderate illness, but their association with recovery or death from COVID-19 has not been characterized.MethodsIn a cohort study of 178 COVID-19 patients, 73 non-hospitalized and 105 hospitalized patients, mucosal swabs and plasma samples were collected at hospital enrollment and up to 3 months post-enrollment (MPE) to measure virus RNA, cytokines/chemokines, binding antibodies, ACE2 binding inhibition, and Fc effector antibody responses against SARS-CoV-2. The association of demographic variables and more than 20 serological antibody measures with intubation or death due to COVID-19 was determined using machine learning algorithms.ResultsPredictive models reveal that IgG binding and ACE2 binding inhibition responses at 1 MPE are positively and anti-Spike antibody-mediated complement activation at enrollment is negatively associated with an increased probability of intubation or death from COVID-19 within 3 MPE.ConclusionsAt enrollment, serological antibody measures are more predictive than demographic variables of subsequent intubation or death among hospitalized COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00658-w" target="_blank">https://www.nature.com/articles/s43856-024-00658-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s41598-024-80806-3" target="_blank">
            ü§ñ Human Polyomavirus 9</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-334', this)">
            More</button></h4>
            <div id="toggle-334" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence and in vitro study of human polyomavirus 9</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Human Polyomavirus 9 (HPyV9) has not been fully understood and its mode of transmission is unknown. Researchers analyzed 1038 serum samples from children to young adults to see if they tested positive for HPV9, as well as from tissues in their lungs, throat, tonsils, and other parts of the body. 

HPyV9 was found in about one out of five people who got it in these tests. The virus also tends to show up more often in older children and adults.

Researchers looked at how HPyV9 replicates inside cells, including whether it uses specific genes or factors that are important for its growth. They studied different types of cell lines, such as those found in the lungs and colon, to see if these viruses can replicate well. 

The results suggest that respiratory cells may be suitable places for HPV9 to grow, but more research is needed to confirm this.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Little is known about human polyomavirus 9 (HPyV9). The mode of transmission and the site of replication are unknown, and seroprevalence data have been published with a wide range. A total of 1038 serum samples from individuals aged 0.7‚Äì93 years were used for seroprevalence study. We observed that seropositivity increased with age among children and young adults, and a 36.2% adult seroprevalence was detected. The prevalence was examined in samples from the respiratory tract: cancerous and non-cancerous lung tissues, tonsils, adenoids, throat swabs, middle ear discharge and nasopharyngeal samples collected from children and adults. HPyV9 was detected in 5.2% of nasopharyngeal samples and 1% of tonsils. Upon a viral infection, the interaction of viral promoters and cellular factors may determine whether a virus productively replicates in a cell. The early and late promoter activity of HPyV9 and the effect of the large T antigen (LTAg) on it was investigated in respiratory, kidney, endothelial and colon cell lines, fibroblast and primary airway epithelial cells. The highest promoter activity was measured in A549 lung cell line. LTAg expression significantly increased the late promoter activity. Based on our results, the respiratory cells may be suitable for HPyV9 replication.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80806-3" target="_blank">https://www.nature.com/articles/s41598-024-80806-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s41598-024-80872-7" target="_blank">
            ü§ñ The Effect of COVID-19 on Cardiac Care Services in Small Hospitals</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-335', this)">
            More</button></h4>
            <div id="toggle-335" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of the COVID-19 pandemic on acute cardiology and neurology services in a secondary peripheral hospital</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic had a significant indirect effect on clinical services in peripheral hospitals. A comparison study found that it impacted cerebral vascular accident (CVA) and ST-elevation myocardial infarction (STEMI) management, but not overall outcome. Patients who arrived during or before the pandemic were treated similarly to those outside of it. The median time for CVA patients from arrival to treatment was longer than expected, while STEMI patients had shorter treatment times. More transfers were made and there were fewer days in hospital for both conditions. Overall mortality did not change significantly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The indirect impact of the COVID-19 pandemic on clinical services in peripheral hospitals has not been fully described. We compared the impact of COVID-19 on Cerebral Vascular Accident (CVA) and ST-elevation myocardial infarction (STEMI) management and outcome in an Israeli peripheral hospital. We included 1029 CVA and 495 STEMI patients. Patients who arrived during (15/3/2020‚Äì15/4/2022) and before (1/1/2018‚Äì14/3/2020) the pandemic, were demographically comparable. During the pandemic, median time for CVA patients from arrival to imaging was longer (23 vs. 19 min,p=‚Äâ0.001); timing from arrival to tissue Plasminogen Activator administration was similar (49 vs. 45 min,p=‚Äâ0.61); transfer to another hospital was more common (20.3% vs. 14.4%p=‚Äâ0.01) and median length of stay (LOS) was shorter (3 vs. 4 days,p<‚Äâ0.05). Among STEMI patients, median time from arrival to intervention intra- pandemic was shorter (45 vs. 50 minp=‚Äâ0.02); Mean LOS shorter (3.86 vs. 4.48p=‚Äâ0.01), and unplanned re-admission less frequent (7.8% vs. 14.6%p=‚Äâ0.01). Mortality did not change significantly. Our data shows no major negative impact of the COVID-19 pandemic on CVA outcomes, and improved care for STEMI patients. Interviews with the neurology and cardiology staff are performed to investigate how quality of care was maintained during the crises.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80872-7" target="_blank">https://www.nature.com/articles/s41598-024-80872-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-26</strong> <a href="https://www.nature.com/articles/s41598-024-80708-4" target="_blank">
            ü§ñ MicroRNAs (miRNAs) play a crucial role in regulating gene expression, including their impact on viral infections. MiR-16a is one such microRNA that has been shown to restrict coronavirus replication by targeting enzymes involved in the virus's life cycle.

In this context, miR-16a reduces the expression of two key proteins involved in coronavirus replication: Furin and Neutrophil Gelatinase-Associated Lipocalin (NGAL). By downregulating these proteins, miR-16a hinders the virus's ability to replicate effectively.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-336', this)">
            More</button></h4>
            <div id="toggle-336" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The dual actions of miRNA16a in restricting Bovine Coronavirus replication through downregulation of Furin and enhancing the host immune response</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract ###

Scientists studied the roles of host cell microRNAs (miRNAs) in the replication and immune regulation of Bovine Cytogranulovirus (BCoV). They found that specific miRNAs, including miRNA16a, are downregulated during infection. They then tested these miRNAs on BCoV-infected cells and found that miRNA16a targets a protein called Furin, which is essential for the virus's replication. The scientists also discovered that this targeting of Furin enhances the host immune response against the virus.

### ### Explanation ###

This study looked at how certain tiny messages (miRNAs) in host cells affect Bovine Cytogranulovirus (BCoV), a type of coronavirus. They found that some miRNAs, including one called miRNA16a, are reduced during the infection process. The researchers then tested these miRNAs on infected cells and discovered that they target a protein called Furin, which is crucial for BCoV replication.

Furin is an enzyme that helps the virus spread within host cells. By blocking its function, the scientists were able to inhibit the replication of the virus. They also found that this targeting of Furin leads to an enhanced immune response against the virus, making it a potential target for future treatments or vaccines. This study suggests that miRNA16a may be a useful tool in preventing or treating BCoV infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The roles of host cell miRNAs have not been well studied in the context of BCoV replication and immune regulation. This study aimed to identify miRNA candidates that regulate essential host genes involved in BCoV replication, tissue tropism, and immune regulation. To achieve these goals, we used two isolates of BCoV (enteric and respiratory) to infect bovine endothelial cells (BECs) and Madine Darby Bovine Kidney (MDBK) cells. We determined the miRNA expression profiles of these cells after BCoV infection. The expression of miRNA16a is differentially altered during BCoV infection. Our data show that miRNA16a is a significantly downregulated miRNA in both in vitro and ex vivo models. We confirmed the miRNA16aexpression profile by qRT‚ÄìPCR. Overexpression of pre-miRNA16ain the BEC and the MDBK cell lines markedly inhibited BCoV infection, as determined by the viral genome copy numbers measured by qRT‚ÄíPCR, viral protein expression (S and N) measured by Western blot, and virus infectivity using a plaque assay. Our bioinformatic prediction showed that Furin is a potential target of miRNA16a. We compared the Furin protein expression level in pre-miRNA16a-transfected/BCoV-infected cells to that in pre-miRNA-scrambled-transfected cells. Our qRT-PCR and Western blot data revealed marked inhibition of Furin expression at the mRNA and protein levels, respectively. BCoV-S protein expression was markedly inhibited at both the mRNA and protein levels. To further confirm the impact of the downregulation of the Furin enzyme on the replication of BCoV, we transfected cells with specific Furin-siRNAs parallel to the scrambled siRNA. Marked inhibition of BCoV replication was observed in the Furin-siRNA-treated group. To further validate Furin as a novel target for miRNA16a, we cloned the 3‚ÄôUTR of bovine Furin carrying the seed region of miRNA16a in the dual luciferase vector. Our data showed that luciferase activity in pre-miRNA16a-transfected cells decreased by more than 50% compared to cells transfected with the construct carrying the mutated Furin seed region. Our data confirmed that miRNA16ainhibits BCoV replication by targeting the host cell line Furin and the BCoV-S glycoprotein. It also enhances the host immune response, which contributes to the inhibition of viral replication. This is the first study to confirm that Furin is a valid target of miRNA16a. Our findings highlight the clinical applications of host miRNA16a as a potential miRNA-based vaccine/antiviral therapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80708-4" target="_blank">https://www.nature.com/articles/s41598-024-80708-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-25</strong> <a href="https://www.nature.com/articles/s41467-024-54440-6" target="_blank">
            ü§ñ Interpretable Protein-Ligand Docking Model (IPLD)</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-337', this)">
            More</button></h4>
            <div id="toggle-337" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Interformer: an interaction-aware model for protein-ligand docking and affinity prediction</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract for a young person to understand:

A way to predict how well certain medicines fit into the body has been improved using computer algorithms that analyze molecular interactions between proteins and drugs. However, these models often don't capture all the details of these interactions, limiting their usefulness. A new approach called Interformer uses a special type of machine learning model to improve this prediction, by including more details about how molecules interact with each other. The results show that this new approach is very effective in predicting how well medicines fit into the body, and even outperforms some state-of-the-art models.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In recent years, the application of deep learning models to protein-ligand docking and affinity prediction, both vital for structure-based drug design, has garnered increasing interest. However, many of these models overlook the intricate modeling of interactions between ligand and protein atoms in the complex, consequently limiting their capacity for generalization and interpretability. In this work, we propose Interformer, a unified model built upon the Graph-Transformer architecture. The proposed model is designed to capture non-covalent interactions utilizing an interaction-aware mixture density network. Additionally, we introduce a negative sampling strategy, facilitating an effective correction of interaction distribution for affinity prediction. Experimental results on widely used and our in-house datasets demonstrate the effectiveness and universality of the proposed approach. Extensive analyses confirm our claim that our approach improves performance by accurately modeling specific protein-ligand interactions. Encouragingly, our approach advances docking tasks state-of-the-art (SOTA) performance.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54440-6" target="_blank">https://www.nature.com/articles/s41467-024-54440-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-25</strong> <a href="https://www.nature.com/articles/s41598-024-80310-8" target="_blank">
            ü§ñ Development and validation of mathematical models to predict COVID-19 severity and mortality risk.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-338', this)">
            More</button></h4>
            <div id="toggle-338" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Development and validation of nomogram models for severe and fatal COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Background: A major global health crisis caused by a new coronavirus (COVID-19) has led to an increase in infections and deaths.

Study aims to develop two predictive models to identify patients at high risk of severe illness or death due to COVID-19.

Patients were divided into three groups based on their symptoms. The study used data from 1600 hospitalized patients who developed COVID-19, with one group being the mild category (940), another the severe (433), and a third the fatal (227).

Risk factors for severe illness were identified through univariate and direct multiple regression analysis.

A final nomogram model was created incorporating the identified risk factors. The model's effectiveness in predicting both severe COVID-19 cases and fatalities was evaluated using ROC curves, calibration tests, and decision curve analysis.

Findings: Elevated age, neutrophil (NEU), lactate dehydrogenase (LDH) levels, and decreased lymphocyte (LYM) and albumin (ALB) levels are associated with severe illness in patients. A history of cerebral infarction and cancer were also risk factors for fatalities.

Conclusions: The nomogram model identifies high-risk patients based on identified risk factors, which can assist clinicians in timely interventions to reduce the incidence of severe COVID-19 and mortality.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundThe coronavirus disease 2019 (COVID-19) has exhibited escalating contagion and resistance to immunity, resulting in a surge in infections and severe cases. This study endeavors to formulate two nomogram predictive models aimed at discerning patients at heightened risk of severe and fatal outcomes upon hospital admission. The primary objective is to enhance clinical management protocols and mitigate the incidence of severe illness and mortality associated with COVID-19.Methods1600 patients diagnosed with COVID-19 and discharged from Fujian Provincial Hospital were chosen as the subjects of this study. These patients were categorized into three groups: mild group (n=‚Äâ940), severe group (n=‚Äâ433), and fatal group (n=‚Äâ227). The patients were randomly divided into training and validation cohorts in a 7:3 ratio. COVID-19 symptoms were treated as dependent variables, and univariate regression analysis was conducted for the laboratory indicators. Risk factors with p-values greater than 0.05 in the univariate regression analysis were eliminated. The remaining risk factors were then analyzed using direct multiple regression analysis to establish an unadjusted model. Subsequently, risk factors with p-values greater than 0.05 were further removed. Clinical characteristics were added to the model as adjustment factors, and the method of multiple stepwise regression analysis was employed to derive the final fully adjusted model. The severe and fatal COVID-19 models were converted into nomograms, respectively. Receiver operating characteristic (ROC) curves were utilized to evaluate the discrimination of the nomogram models. Calibration was assessed using the Hosmer-Lemeshow test and calibration curves. Clinical benefit was evaluated by decision curve analysis.ResultsCompared to the mild group, individuals in the severe COVID-19 group exhibited significant increases in age, neutrophil (NEU), and lactate dehydrogenase (LDH) levels, alongside notable decreases in lymphocyte (LYM) and albumin (ALB) levels. Nomogram model incorporating age, NEU, LDH, LYM, and ALB demonstrated efficacy in predicting the onset of severe COVID-19 (AUC‚Äâ=‚Äâ0.771). Furthermore, history of cerebral infarction and cancer, LDH and ALB as risk factors for fatal COVID-19 cases compared to the severe group. The nomogram model comprising these factors was capable of early identification of COVID-19 fatalities (AUC‚Äâ=‚Äâ0.748).ConclusionsElevated age, NEU, and LDH levels, along with decreased LYM and albumin (ALB) levels, are risk factors for severe illness in hospitalized patients with COVID-19. A history of cerebral infarction and tumors, along with elevated LDH and decreased ALB levels, are risk factors for death in critically ill patients. The nomogram model based on these factors can effectively predict the risk of severe or fatal illness from COVID-19, thereby assisting clinicians in timely interventions to reduce the rates of severe illness and mortality among hospitalized patients. However, the model faces challenges in processing longitudinal data and specific points in time, indicating that there is room for improvement.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-80310-8" target="_blank">https://www.nature.com/articles/s41598-024-80310-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-23</strong> <a href="https://www.nature.com/articles/s41467-024-54462-0" target="_blank">
            ü§ñ Discovery of an Oral Inhibitor for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-339', this)">
            More</button></h4>
            <div id="toggle-339" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Discovery of orally bioavailable SARS-CoV-2 papain-like protease inhibitor as a potential treatment for COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### RNA-dependent RNA polymerase (RdRp) inhibitors present as promising therapeutic targets for COVID-19.

### Researchers develop new antiviral drugs using structure-based drug design approach to inhibit viral proteases, RdRp and 3C-like protease (3CLpro). 

### Novel potent non-covalent PLproinhibitors discovered through SBDD, showing improved potency in vitro and in vivo. 

### Lead compound GZNL-P36 inhibits SARS-CoV-2 and HCoV-NL63 with EC50 ranging from 58.2 to 306.2 nM and HCoV-229E with EC50 of 81.6 nM and 2.66 ŒºM.

### Oral administration of GZNL-P36 results in improved survival, reduced lung viral loads, and lesions in SARS-CoV-2 mouse model consistent with RNA-seq data analysis.

### The new inhibitors represent a promising approach for treating COVID-19 using RNA-dependent RNA polymerase (RdRp) inhibitors.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The RNA-dependent RNA polymerase (RdRp), 3C-like protease (3CLpro), and papain-like protease (PLpro) are pivotal components in the viral life cycle of SARS-CoV-2, presenting as promising therapeutic targets. Currently, all FDA-approved antiviral drugs against SARS-CoV-2 are RdRp or 3CLproinhibitors. However, the mutations causing drug resistance have been observed in RdRp and 3CLprofrom SARS-CoV-2, which makes it necessary to develop antivirals with novel mechanisms. Through the application of a structure-based drug design (SBDD) approach, we discover a series of novel potent non-covalent PLproinhibitors with remarkable in vitro potency and in vivo PK properties. The co-crystal structures of PLprowith lead compounds reveal that the residues D164 and Q269 around the S2 site are critical for improving the inhibitor‚Äôs potency. The lead compound GZNL-P36 not only inhibits SARS-CoV-2 and its variants at the cellular level with EC50ranging from 58.2‚ÄânM to 306.2‚ÄânM, but also inhibits HCoV-NL63 and HCoV-229E with EC50of 81.6‚ÄânM and 2.66‚ÄâŒºM, respectively. Oral administration of the GZNL-P36 results in significantly improved survival and notable reductions in lung viral loads and lesions in SARS-CoV-2 infection mouse model, consistent with RNA-seq data analysis. Our results indicate that PLproinhibitors represent a promising SARS-CoV-2 therapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54462-0" target="_blank">https://www.nature.com/articles/s41467-024-54462-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-23</strong> <a href="https://www.nature.com/articles/s41467-024-54334-7" target="_blank">
            ü§ñ Reinfection with a mutated COVID-19 virus can occur differently for each person due to their unique genetic makeup.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-340', this)">
            More</button></h4>
            <div id="toggle-340" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Outcome of SARS-CoV-2 reinfection depends on genetic background in female mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract:

A new strain of COVID-19 (B.1.351) can make people who have been vaccinated or exposed to it more likely to get infected again. This happens because their immune system doesn't respond well to the specific virus and starts producing antibodies that can prevent them from fighting it off.

The severity of COVID-19 also depends on how strong a person's immune system is, which varies depending on genetic factors. In one study, researchers examined how different groups of mice (with genetic backgrounds) responded to infection with different strains of the virus. They found that some types of strain can trigger severe disease in people who are vaccinated or exposed, while others may protect against it.

The study also looked at what happens inside the body after a person is infected with this new strain and how their immune system responds. It found that certain cells in the lungs (in the air sacs) respond differently to infection by different groups of mice, which could explain why some people get more severe disease while others don't.

Overall, the study suggests that COVID-19 severity depends on a combination of factors, including how strong the immune system is and how the body responds to the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Antigenically distinct SARS-CoV-2 variants increase the reinfection risk for vaccinated and previously exposed population due to antibody neutralization escape. COVID-19 severity depends on many variables, including host immune responses, which differ depending on genetic predisposition. To address this, we perform immune profiling of female mice with different genetic backgrounds ‚Äìtransgenic K18-hACE2 and wild-type 129S1‚Äì infected with the severe B.1.351, 30 days after exposure to the milder BA.1 or severe H1N1. Prior BA.1 infection protects against B.1.351-induced morbidity in K18-hACE2 but aggravates disease in 129S1. H1N1 protects against B.1.351-induced morbidity only in 129S1. Enhanced severity in B.1.351 re-infected 129S1 is characterized by an increase of IL-10, IL-1Œ≤, IL-18 and IFN-Œ≥, while in K18-hACE2 the cytokine profile resembles na√Øve mice undergoing their first viral infection. Enhanced pathology during 129S1 reinfection cannot be attributed to weaker adaptive immune responses to BA.1. Infection with BA.1 causes long-term differential remodeling and transcriptional changes in the bronchioalveolar CD11c+ compartment. K18-hACE2 CD11c+ cells show a strong antiviral defense expression profile whereas 129S1 CD11c+ cells present a more pro-inflammatory response upon restimulation. In conclusion, BA.1 induces cross-reactive adaptive immune responses in K18-hACE2 and 129S1, but reinfection outcome correlates with differential CD11c+ cells responses in the alveolar space.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54334-7" target="_blank">https://www.nature.com/articles/s41467-024-54334-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-23</strong> <a href="https://www.nature.com/articles/s41541-024-01017-5" target="_blank">
            ü§ñ Randomized Trial to Compare Safety, Immunogenicity, and Efficacy of Self-Amplifying mRNA Vaccines vs Adenovirus-Vectors for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-341', this)">
            More</button></h4>
            <div id="toggle-341" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A randomized trial comparing safety, immunogenicity and efficacy of self-amplifying mRNA and adenovirus-vector COVID-19 vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This phase 3 trial compared the safety, tolerability, immunogenicity, and efficacy of two COVID-19 vaccines: ARCT-154 and ChAdOx1-S. Both vaccines were well-tolerated and had similar safety profiles with mild to moderate side effects in adults aged 18-85 years. The vaccine that produced a stronger immune response against COVID-19 also showed higher effectiveness over time, with a greater than 20% reduction in the risk of severe illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This phase 3 trial compared safety, tolerability, immunogenicity and efficacy of the self-amplifying mRNA COVID-19 vaccine, ARCT-154, with ChAdOx1-S adenovirus-vector vaccine. In four centers in Vietnam adult participants aged 18‚Äí85 years were randomly assigned to receive two doses, 28 days apart, of either ARCT-154 (n‚Äâ=‚Äâ1186) or ChAdOx1-S (n‚Äâ=‚Äâ1180). Both vaccines were well tolerated with similar safety and reactogenicity profiles consisting of mainly mild-to-moderate solicited adverse events and few related serious adverse events. Higher neutralizing antibody responses persisting to one-year post-vaccination after ARCT-154 compared with ChAdOx1-S were associated with a generally higher efficacy against COVID-19. In an exploratory analysis relative vaccine efficacy of ARCT-154 vs. ChAdOx1-S against any COVID-19 from Day 36 to Day 394 was 19.8% (95% CI: 4.0‚Äì33.0). Self-amplifying mRNA vaccine offers potential immunological advantages in terms of immunogenicity and efficacy over adenovirus-vector vaccine without compromising safety.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01017-5" target="_blank">https://www.nature.com/articles/s41541-024-01017-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-23</strong> <a href="https://www.nature.com/articles/s41467-024-54505-6" target="_blank">
            ü§ñ Cellular "snatchers": A method to clear viruses and their variants from cells.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-342', this)">
            More</button></h4>
            <div id="toggle-342" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Lysosomal ‚ÄúTRAP‚Äù: a neotype modality for clearance of viruses and variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract that a young person can understand:

Researchers have created a special "trap" inside cells called lysoTRAP to catch and kill viruses like SARS-CoV-2. They use macrophages (a type of immune cell) as the trap, which helps to clear the virus from the body in a way that is safe and effective. LysoTRAP is able to target and destroy specific parts of the virus that are important for its survival. This "trap" can also catch other types of viruses that have similar targets, making it useful for fighting many different kinds of illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The binding of viruses to host-entry factor receptors is an essential step for viral infection. Many studies have shown that macrophages can internalize viruses and degrade them in lysosomes for clearance in vivo. Inspired by these natural behaviors and using SARS-CoV-2 as a testbed, we harvest lysosomes from activated macrophages and anchor the protein-receptor ACE2 as bait, thus constructing a lysosomal ‚ÄúTRAP‚Äù (lysoTRAP) that selectively captures, internalizes, and eventually degrades SARS-CoV-2. Through experiments with cells, female mice, female hamsters, and human lung organoids, we demonstrate that lysoTRAP effectively clears SARS-CoV-2. Importantly, unlike therapeutic agents targeting SARS-CoV-2 spike protein, lysoTRAP remains effective against nine pseudotyped variants and the authentic Omicron variant, demonstrating its resistance to SARS-CoV-2 mutations. In addition to the protein-receptor ACE2, we also extend lysoTRAP with the saccharide-receptor sialic acid and verify its excellent antiviral effect against H1N1, highlighting the flexibility of our ‚ÄúTRAP‚Äù platform in fighting against various viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54505-6" target="_blank">https://www.nature.com/articles/s41467-024-54505-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-22</strong> <a href="https://www.nature.com/articles/s41598-024-79909-8" target="_blank">
            ü§ñ Immune Response Dynamics After Clearing Respiratory or Bloodborne Infections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-343', this)">
            More</button></h4>
            <div id="toggle-343" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Different dynamics of soluble inflammatory mediators after clearance of respiratory SARS-CoV-2 versus blood-borne hepatitis C virus infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research study compares immune system response to different types of viral infections. It finds that the body's reaction to a virus can be affected by the type and severity of the infection, as well as how long it lasts. The study looks at how soluble inflammatory mediators (small molecules involved in inflammation) change over time in patients with hepatitis C or SARS-CoV-2 after treatment or resolution of their infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Viral infections can be acute or chronic, with the immune system pivotal in immunopathogenesis. The potential reversibility of inflammation post-viral elimination is of current interest. This study compares the dynamics of soluble inflammatory mediators (SIM) during and after respiratory infections with SARS-CoV-2 and blood-borne acute and chronic hepatitis C virus (HCV) infections. The study included patients with acute HCV (n‚Äâ=‚Äâ29), chronic HCV (n‚Äâ=‚Äâ54), and SARS-CoV-2 (n‚Äâ=‚Äâ39 longitudinal, n‚Äâ=‚Äâ103 cross-sectional), along with 30 healthy controls. Blood samples were collected at baseline, end of treatment/infection, and during follow-up (up to 9¬†months). SIMs were quantified using the HD-SP-X Imaging and Analysis System‚Ñ¢. At baseline, SIM profiles in acute SARS-CoV-2 and HCV infections were significantly elevated compared with controls. During follow-up, SIM decline was less pronounced in acute and chronic HCV infections after successful therapy than in SARS-CoV-2 infections. Most SIM in the SARS-CoV-2 cohort normalized within 3¬†months. In chronic HCV, SIM were higher in cirrhotic than noncirrhotic patients post-HCV elimination. Dynamics of SIM after viral elimination vary between blood-borne acute and chronic HCV infections and respiratory SARS-CoV-2 infections. Immunological imprints 3‚Äì9¬†months after HCV elimination appear more pronounced than after SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79909-8" target="_blank">https://www.nature.com/articles/s41598-024-79909-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-22</strong> <a href="https://www.nature.com/articles/s41598-024-79952-5" target="_blank">
            ü§ñ Environmental monitoring and surveillance are crucial components in responding to pandemics.

### THE VALUE OF ENVIRONMENTAL SURVEILLANCE FOR PANDEMONIAL RESPONSE</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-344', this)">
            More</button></h4>
            <div id="toggle-344" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The value of environmental surveillance for pandemic response</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Environmental sampling surveillance (ESS) technologies are being used more widely during pandemics to gather valuable data on disease outbreaks. However, not all areas have access to these systems, which means public health officials need guidance on when and if to use them.

A new study aims to create a model that can help make decisions about ESS use based on factors such as the type of pathogen causing an outbreak and how often it occurs in other parts of the world. Using a real-life example from the COVID-19 pandemic, researchers found that an ESS system could potentially save thousands of lives by providing early warnings for disease outbreaks.

Their research suggests that using ESS systems effectively depends on factors such as how deadly the pathogen is and whether or not effective public health interventions are in place. Even if pathogens like SARS-Cov-2 emerge rarely, their study shows that ESS systems can still provide valuable information and potentially save lives over time.

The researchers also found that the value of an ESS system increases for more contagious and deadly pathogens but depends on the effectiveness of local public health efforts. As a result, their findings could help policymakers decide when to invest in ESS systems and how to scale them up over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Environmental sampling surveillance (ESS) technologies, such as wastewater genomic surveillance and air sensors, have been increasingly adopted during the COVID-19 pandemic to provide valuable information for public health response. However, ESS coverage is not universal, and public health decision-makers need support to choose whether and how to expand and sustain ESS efforts. This paper introduces a model and approach to quantify the value of ESS systems that provide leading epidemiological indicators for pandemic response. Using the COVID-19 pandemic as a base-case scenario, we quantify the value of ESS systems in the first year of a new pandemic and demonstrate how the value of ESS systems depends on biological and societal parameters. Under baseline assumptions, an ESS system that provides a 5-day early warning relative to syndromic surveillance could reduce deaths from 149 (95% prediction interval: 136‚Äì169) to 134 (124‚Äì144) per 100,000 population during the first year of a new COVID-19-like pandemic, resulting in a net monetary benefit of $1,450 ($609-$2,740) per person. The system‚Äôs value is higher for more transmissible and deadly pathogens but hinges on the effectiveness of public health interventions. Our findings also suggest that ESS systems would provide net-positive benefits even if they were permanently maintained and pathogens like SARS-Cov-2 emerged once every century or less frequently. Our results can be used to prioritize pathogens for ESS, decide whether and how to expand systems to currently uncovered populations, and determine how to scale surveillance systems‚Äô coverage over time.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79952-5" target="_blank">https://www.nature.com/articles/s41598-024-79952-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-22</strong> <a href="https://www.nature.com/articles/s43856-024-00633-5" target="_blank">
            ü§ñ Morbidity of SARS-CoV-2 in Disease Outcomes vs Influenza</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-345', this)">
            More</button></h4>
            <div id="toggle-345" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Morbidity of SARS-CoV-2 in the evolution to endemicity and in comparison with influenza</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>BackgroundThere are three possible ways the COVID-19 pandemic might change: ongoing severity, influenza-like severity, and becoming less severe over time.

MethodsWe analyzed data from the US National Covid Cohort Collaborative, CDC COVID-NET, and CDC Fluview to see how the pandemic might evolve under each scenario. We also looked at the response of people with SARS-CoV-2 infections to treatment and hospitalization rates in wastewater samples.

ResultsThe analysis shows that:

* People with SARS-CoV-2 infections tend to have similar symptoms as those with influenza.
* However, when looking at how many people are hospitalized for SARS-CoV-2, the pandemic is actually much less severe than it was for influenza.
* The treatments used to fight SARS-CoV-2 work well.

ConclusionsAs the pandemic transitions from being an outbreak to a more manageable situation, its symptoms will continue to decline.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundThere are three possible SARS-CoV-2 post-pandemic scenarios: (i) ongoing severity, (ii) influenza-like severity, and (iii) a transition to an endemic disease with lesser morbidity similar to that of other human coronaviruses.MethodsTo assess a possible evolution of the pandemic under the three scenarios, we use data from the US National Covid Cohort Collaborative, CDC COVID-NET, and CDC Fluview and from the WastewaterSCAN Dashboard. We include influenza disease and treatment response as benchmark. The US National Covid Cohort Collaborative allows the quantification of viral-specific morbidity using electronic health records from 424,165 SARS-CoV-2 cases, 53,846 influenza cases, and 199,971 uninfected control subjects from 2021‚Äì2022. Evolution of hospitalization rates is estimated from the correlation between national SARS-CoV-2 and influenza hospitalization data and viral gene copies in wastewater.ResultsOur findings reveal that medically attended SARS-CoV-2 infections exhibit similar morbidity to influenza [indicative of scenario (ii)], but SARS-CoV-2 hospitalization rates are one order of magnitude lower than influenza when considering virus concentration in wastewater [indicative of scenario (iii)]. Moreover, SARS-CoV-2 displays a more favorable response to antiviral therapy.ConclusionsOur analysis confirms a rapid decline in SARS-CoV-2 morbidity as it transitions to an endemic state.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00633-5" target="_blank">https://www.nature.com/articles/s43856-024-00633-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-21</strong> <a href="https://www.nature.com/articles/s44298-024-00067-9" target="_blank">
            ü§ñ COVID-19 Spike Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-346', this)">
            More</button></h4>
            <div id="toggle-346" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reduced cross-protective potential of Omicron compared to ancestral SARS-CoV-2 spike vaccines against potentially zoonotic coronaviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is the simplified abstract:

A global campaign to vaccinate people against COVID-19 has helped build immunity against other coronavirus viruses with potential to jump from animals to humans. Researchers tested this by giving mice models of COVID-19 and then checking for antibodies that can fight multiple human strains, including ACE2-binding ones. The results showed that the vaccines based on Omicron spike sequences were much less effective at protecting against these other viruses compared to previous vaccines like SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has emphasised the importance of vaccines and preparedness against viral threats crossing species barriers. In response, a worldwide vaccination campaign targeting SARS-CoV-2 was implemented, which provides some cross-protective immunological memory to other coronavirus species with zoonotic potential. Following a vaccination regimen against SARS-CoV-2 spike in a preclinical mouse model, we were able to demonstrate the induction of neutralizing antibodies towards multiple human ACE2 (hACE2)-bindingSarbecovirusspikes. Importantly, compared to vaccines based on the SARS-CoV-2 Reference strain, vaccines based on Omicron spike sequences induced drastically less broadly cross-protective neutralizing antibodies against other hACE2-binding sarbecoviruses. This observation remained true whether the vaccination regimens were based on protein subunit or mRNA / LNP vaccines. Overall, while it may be necessary to update vaccine antigens to combat the evolving SARS-CoV-2 virus for enhanced protection from COVID-19, Reference-based vaccines may be a more valuable tool to protect against novel coronavirus zoonoses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00067-9" target="_blank">https://www.nature.com/articles/s44298-024-00067-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-21</strong> <a href="https://www.nature.com/articles/s44298-024-00068-8" target="_blank">
            ü§ñ Rodent resistance to COVID-19

 Rodents such as gerbils are highly susceptible to the SARS-CoV-2 virus, which causes COVID-19. In contrast, many other rodent species including guinea pigs appear to be resistant to this infection.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-347', this)">
            More</button></h4>
            <div id="toggle-347" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sensitivity of rodents to SARS-CoV-2: Gerbils are susceptible to SARS-CoV-2, but guinea pigs are not</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Study Helps Predict Risk of SARS-CoV-2 Transmission from Humans to Pets

Scientists studied how well different rodents (guinea pigs and gerbils) can contract a virus called SARS-CoV-2 from people. They found that:

* Gerbils are highly susceptible to the virus
* Guinea pigs are not as sensitive</p><br>
                <p><strong>Abstract</strong></p>
                <p>Syrian hamster are sensitive to SARS-CoV-2 and widely used as an animal model of COVID-19. In contrast, mice are not readily infected by the ancestral strains of SARS-CoV-2 because of differences in their angiotensin-converting enzyme 2 (ACE2) receptors. Thus, even among rodents, susceptibility to SARS-CoV-2 varies. Knowledge of virus transmissibility from humans to pet rodents is important for public health to assess the potential for transmission in the home and pet breeding and selling facilities. In this study, we assessed the sensitivity of guinea pigs and gerbils to SARS-CoV-2 isolated from humans, and found that gerbils are susceptible to SARS-CoV-2, but guinea pigs are not. Pet sellers often display hamsters with high susceptibility to SARS-CoV-2 in the same area as gerbils, so caution should be exercised during COVID-19 outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00068-8" target="_blank">https://www.nature.com/articles/s44298-024-00068-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-20</strong> <a href="https://www.nature.com/articles/s41467-024-53872-4" target="_blank">
            ü§ñ Virus Interactions in Respiratory Illnesses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-348', this)">
            More</button></h4>
            <div id="toggle-348" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterizing the interactions between influenza and respiratory syncytial viruses and their implications for epidemic control</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Pathogens, like viruses and bacteria, can interact with each other in complex ways that affect how people get infected and recover from diseases. Researchers have discovered that certain infections, such as the flu and RSV, have a strong and lasting impact on each other when they occur together. By using advanced mathematical modeling techniques and data from two different countries, scientists found this interaction is significant enough to be studied in detail. They also explored how live vaccines against one of these diseases could reduce the risk of getting infected with the other. The study's findings suggest that understanding these interactions can help predict which vaccines may have a greater impact on public health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pathogen-pathogen interactions represent a critical but little-understood feature of infectious disease dynamics. In particular, experimental evidence suggests that influenza virus and respiratory syncytial virus (RSV) compete with each other, such that infection with one confers temporary protection against the other. However, such interactions are challenging to study using common epidemiologic methods. Here, we use a mathematical modeling approach, in conjunction with detailed surveillance data from Hong Kong and Canada, to infer the strength and duration of the interaction between influenza and RSV. Based on our estimates, we further utilize our model to evaluate the potential conflicting effects of live attenuated influenza vaccines (LAIV) on RSV burden. We find evidence of a moderate to strong, negative, bidirectional interaction, such that infection with either virus yields 40-100% protection against infection with the other for one to five months. Assuming that LAIV reduces RSV susceptibility in a similar manner, we predict that the impact of such a vaccine at the population level would likely depend greatly on underlying viral circulation patterns. More broadly, we highlight the utility of mathematical models as a tool to characterize pathogen-pathogen interactions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53872-4" target="_blank">https://www.nature.com/articles/s41467-024-53872-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-20</strong> <a href="https://www.nature.com/articles/s41541-024-01013-9" target="_blank">
            ü§ñ Vaccine Development for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-349', this)">
            More</button></h4>
            <div id="toggle-349" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preclinical characterization of the Omicron XBB.1.5-adapted BNT162b2 COVID-19 vaccine</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>As COVID-19 continues to evolve, a new vaccine is needed to protect against the most contagious variants. Researchers have developed a vaccine that contains changes specific to the Omicron XBB.1.5 variant and has shown improved protection against this strain compared to previous vaccines. The vaccine also induces strong immune responses, including Th1 CD4+ and IFNŒ≥+ T cells.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As SARS-CoV-2 evolves, increasing in potential for greater transmissibility and immune escape, updated vaccines are needed to boost adaptive immunity to protect against COVID-19 caused by circulating strains. Here, we report features of the monovalent Omicron XBB.1.5-adapted BNT162b2 vaccine, which contains XBB.1.5-specific sequence changes, relative to the original BNT162b2 backbone, in the encoded prefusion-stabilized SARS-CoV-2 spike protein (S(P2)). Biophysical characterization of Omicron XBB.1.5 S(P2) demonstrated that it maintains a prefusion conformation and adopts a flexible, predominantly open, state, with high affinity for the human ACE-2 receptor. When administered as a 4th dose in BNT162b2-experienced mice, the monovalent Omicron XBB.1.5 vaccine elicited substantially higher serum neutralizing titers against pseudotyped viruses of Omicron XBB.1.5, XBB.1.16, XBB.1.16.1, XBB.2.3, EG.5.1 and HV.1 sublineages and phylogenetically distant BA.2.86 lineage than the bivalent Wild Type + Omicron BA.4/5 vaccine. Similar trends were observed against Omicron XBB sublineage pseudoviruses when the vaccine was administered as a 2-dose series in naive mice. Strong S-specific Th1 CD4+and IFNŒ≥+CD8+T cell responses were also observed. These findings, together with real world performance of the XBB.1.5-adapted vaccine, suggest that preclinical data for the monovalent Omicron XBB.1.5-adapted BNT162b2 was predictive of protective immunity against dominant SARS-CoV-2 strains.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01013-9" target="_blank">https://www.nature.com/articles/s41541-024-01013-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-20</strong> <a href="https://www.nature.com/articles/s41591-024-03354-3" target="_blank">
            ü§ñ Understanding Lung Infection in People from Malawi After COVID-19 Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-350', this)">
            More</button></h4>
            <div id="toggle-350" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Spatially resolved single-cell atlas unveils a distinct cellular signature of fatal lung COVID-19 in a Malawian population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Postmortem analysis of single cells has greatly improved our understanding of respiratory diseases such as COVID-19. However, there is limited research on how these diseases affect people from different regions, including Africa, where HIV, malaria, and environmental exposures may play a significant role in disease pathobiology and treatment outcomes.

This study aimed to understand the lung disease associated with COVID-19 in Malawian adults. Researchers analyzed the cells found in lungs, blood, and nasal tissue of 9 people who had died from COVID-19 and 7 people who did not have the virus. The results showed that the lung damage caused by COVID-19 is similar across different populations, but there are differences in how it affects each group.

The study also looked at the immune responses in the blood and lungs of these individuals, finding that some people had a stronger response to certain proteins in their blood. Specifically, people with HIV did not have as strong an immune response.

Overall, this research provides valuable information about the cellular mechanisms underlying COVID-19 lung disease and highlights the importance of studying how different diseases affect people from diverse backgrounds.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Postmortem single-cell studies have transformed understanding of lower respiratory tract diseases (LRTDs), including coronavirus disease 2019 (COVID-19), but there are minimal data from African settings where HIV, malaria and other environmental exposures may affect disease pathobiology and treatment targets. In this study, we used histology and high-dimensional imaging to characterize fatal lung disease in Malawian adults with (n=‚Äâ9) and without (n=‚Äâ7) COVID-19, and we generated single-cell transcriptomics data from lung, blood and nasal cells. Data integration with other cohorts showed a conserved COVID-19 histopathological signature, driven by contrasting immune and inflammatory mechanisms: in US, European and Asian cohorts, by type I/III interferon (IFN) responses, particularly in blood-derived monocytes, and in the Malawian cohort, by response to IFN-Œ≥ in lung-resident macrophages. HIV status had minimal impact on histology or immunopathology. Our study provides a data resource and highlights the importance of studying the cellular mechanisms of disease in underrepresented populations, indicating shared and distinct targets for treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-03354-3" target="_blank">https://www.nature.com/articles/s41591-024-03354-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-20</strong> <a href="https://www.nature.com/articles/s41467-024-54404-w" target="_blank">
            ü§ñ Estimated Effectiveness of COVID-19 Vaccine in Non-Certified Individuals</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-351', this)">
            More</button></h4>
            <div id="toggle-351" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bias and negative values of COVID-19 vaccine effectiveness estimates from a test-negative design without controlling for prior SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists have been studying how well vaccines work to protect against certain diseases. However, it can be difficult to tell if someone has had an infection before they were vaccinated. To figure this out, researchers created "test-negative designs" that helped them make more accurate estimates of vaccine effectiveness.

They looked at data from a COVID-19 outbreak and found that when people who were vaccinated had fewer protection against the disease than those who weren't, it could lead to incorrect conclusions about how safe vaccines are. This happened because they didn't take into account whether these vaccinated individuals actually got infected with the disease in the first place.

The researchers also found that test-negative designs, which were used immediately after a new vaccine was rolled out, introduced more bias than traditional methods and potentially led to incorrect interpretations of vaccine effectiveness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Test-negative designs (TNDs) are used to assess vaccine effectiveness (VE). Protection from infection-induced immunity may confound the association between case and vaccination status, but collecting reliable infection history can be challenging. If vaccinated individuals have less infection-induced protection than unvaccinated individuals, failure to account for infection history could underestimate VE, though the bias is not well understood. We simulated individual-level SARS-CoV-2 infection and COVID-19 vaccination histories¬†and a TND. VE against symptomatic infection and VE against severe disease estimates unadjusted for infection history underestimated VE compared to estimates adjusted for infection history, and unadjusted estimates were more likely to be below 0%, which could lead to an incorrect interpretation that COVID-19 vaccines are harmful. TNDs assessing VE immediately following vaccine rollout introduced the largest bias and potential for negative VE against symptomatic infection. Despite the potential for bias, VE estimates from TNDs without prior infection information are useful because underestimation is rarely more than 8 percentage points.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-54404-w" target="_blank">https://www.nature.com/articles/s41467-024-54404-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-20</strong> <a href="https://www.nature.com/articles/s41392-024-02025-6" target="_blank">
            ü§ñ Vaccine delivered directly into the nasal passages to provide protection against specific influenza strains.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-352', this)">
            More</button></h4>
            <div id="toggle-352" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Intranasal delivery of a subunit protein vaccine provides protective immunity against JN.1 and XBB-lineage variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The mucosal immune system plays a crucial role in preventing respiratory infections, but recent cases of SARS-CoV-2 have raised concerns about their ability to fight off infections. Researchers developed an intranasal COVID-19 vaccine that could help prevent these infections and protect against future outbreaks. This vaccine works by stimulating the body's immune system to produce antibodies that can recognize and attack the virus, providing long-lasting protection. The study also found that this type of vaccine is more effective as a booster shot than others and can provide additional protection against viral infections in real-life situations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The mucosal immune response plays a crucial role in the prevention of respiratory viruses. Given the risk of recurrent SARS-CoV-2 infections in the population, the rapid development of next-generation intranasal COVID-19 vaccines with high safety and efficacy is paramount. In the current study, we developed a protein-based intranasal vaccine comprising the XBB.1.5 receptor binding domain (RBD)-derived trimeric recombinant protein (RBDXBB.1.5-HR) and an MF59-like oil-in-water adjuvant. Intranasal administration of RBDXBB.1.5-HR vaccine elicited robust and sustained humoral immune responses in mice and rats, resulting in high levels of neutralizing antibodies against XBB-lineage subvariants, with protection lasting for at least six months. The intranasal RBDXBB.1.5-HR vaccine generated potent mucosal immune responses, characterized by the inductions of tissue-resident T (TRM) cells, local cellular immunity, germinal center, and memory B cell responses in the respiratory tract. The combination of intramuscular and intranasal delivery of the RBDXBB.1.5-HR vaccine demonstrated exceptional systemic and mucosal protective immunity. Furthermore, intranasal delivery of RBDXBB.1.5-HR vaccine as a heterologous booster shot showed more effective boosting effects after mRNA administration compared to homologous vaccination, as evidenced by the induction of superior systemic and extra mucosal immune response. Importantly, the intranasal RBDXBB.1.5-HR vaccine conferred efficient protection against the challenge with authentic EG.5.1 viruses in vivo. These findings identify the intranasal RBDXBB.1.5-HR vaccine as a potential mucosal vaccine candidate for the prevention of SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-02025-6" target="_blank">https://www.nature.com/articles/s41392-024-02025-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-19</strong> <a href="https://www.nature.com/articles/s41541-024-01011-x" target="_blank">
            ü§ñ Boosting Neuraminidase Immunity 

Neuraminidase (NA) plays a crucial role in the spread and transmission of the flu virus, especially when it binds to hemagglutinin (HA). Influenza vaccines work by exposing the body's immune system to small pieces of HA from different strains. However, some people might not develop immunity against NA because they are more susceptible to severe illness or have weaker immune responses. The next-generation influenza vaccines aim to boost neuraminidase immunity, helping to protect against these individuals and potentially reducing the severity of flu illness.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-353', this)">
            More</button></h4>
            <div id="toggle-353" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Boosting neuraminidase immunity in the presence of hemagglutinin with the next generation of influenza vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract ###
Neuraminidase (NA) plays a key role in viral replication, making it crucial for the spread of flu viruses. Despite some evidence suggesting that people with antibodies against NA may be less susceptible to illness, current flu vaccines focus primarily on one protein called hemagglutinin (HA). Researchers have discovered that using a different protein called neuraminidase (NA) can stimulate strong immune responses in mice and ferrets. By adding NA to flu vaccines, these animals developed mild flu symptoms when exposed to a new strain of the virus, but also showed reduced severity of the illness. The results suggest that incorporating NA into flu vaccines could increase their effectiveness by boosting immunity against NA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Neuraminidase (NA), the second most abundant surface glycoprotein on the influenza virus, plays a key role in viral replication and propagation. Despite growing evidence showing that NA-specific antibodies correlate with resistance to disease in humans, current licensed vaccines focus almost entirely on the hemagglutinin (HA) antigen. Here, we demonstrate that recombinant NA (rNA) protein is highly immunogenic in both na√Øve mice and ferrets, as well as in pre-immune ferrets, irrespective of the level of match with preexisting immunity. Ferrets vaccinated with rNA developed mild influenza disease symptoms upon challenge with human H3N2 influenza virus, and anti-NA antibody responses appeared correlated with reduction in disease severity. The addition of rNA to a quadrivalent HA-based vaccine induced robust NA-specific humoral immunity in ferrets, while retaining the ability to induce HA-specific immunity. These results demonstrate that the addition of rNA is a viable option to increase immunogenicity and potentially efficacy versus currently licensed influenza vaccines by means of boosting NA immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01011-x" target="_blank">https://www.nature.com/articles/s41541-024-01011-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-19</strong> <a href="https://www.nature.com/articles/s41598-024-75920-1" target="_blank">
            ü§ñ Study on Unvaccinated Long Covid Patients' Leukocyte Pattern</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-354', this)">
            More</button></h4>
            <div id="toggle-354" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Altered leukocyte pattern and inflammatory markers in unvaccinated long covid patients: a cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research by scientists found that people who have COVID-19 for more than three months tend to have different levels of certain immune cells and chemicals in their blood compared to those with milder symptoms or shorter illnesses. The study focused on individuals living in the Amazon region, where such research is scarce. It analyzed blood samples from patients to see how they changed over time. The findings suggest that people with Long Covid may have persistent inflammation in their bodies, which can last for a year or more after their illness ends.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long Covid results from the damage caused by SARS-CoV-2, involving the release of cytokines and the continuous activation of immune cells. This cross-sectional study investigates leukocyte and cytokine profiles in Long Covid patients in the Amazon, a region where such studies are limited. Blood samples were analysed for differential leukocyte counts and cytokine levels. We suggest elevated lymphocyte counts in hospitalised patients and those with severe COVID-19. Higher eosinophil counts were observed in patients with up to three months of Long Covid, and increased monocyte counts in those with up to six months. IL-2 levels were higher in patients with fewer symptoms and Long Covid duration of more than three months, whereas IL-10 may remain elevated for up to 12 months. We suggest positive correlations between neutrophils, monocytes, eosinophils, and lymphocytes with different cytokines (IFN-Œ≥, IL-6, IL-4, IL-17a, IL-2). Women were associated with lower hospitalisation rates and longer durations of Long Covid; increased lymphocyte counts were linked to hospitalisation due to COVID-19, while higher monocyte counts were associated with Long Covid durations of up to six months. We suggest that Long Covid patients may exhibit alterations in inflammatory markers, indicating a persistently pro-inflammatory microenvironment that tends to diminish after 12 months of Long Covid.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75920-1" target="_blank">https://www.nature.com/articles/s41598-024-75920-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-19</strong> <a href="https://www.nature.com/articles/s43856-024-00664-y" target="_blank">
            ü§ñ Study on Ursodeoxycholic Acid and Severe COVID-19 Outcomes Using OpenSAFELY Platform.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-355', this)">
            More</button></h4>
            <div id="toggle-355" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Ursodeoxycholic acid and severe COVID-19 outcomes in a cohort study using the OpenSAFELY platform</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A type of bile acid called ursodeoxycholic acid (UDCA) has been shown to be effective in preventing serious complications from COVID-19 in people with liver disease. A large study found that taking UDCA lowered the risk of hospitalization or death due to COVID-19, and suggests it may also reduce this risk in others at high risk of severe illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundBiological evidence suggests ursodeoxycholic acid (UDCA)‚Äîa common treatment of cholestatic liver disease‚Äîmay prevent severe COVID-19 outcomes. We aimed to compare the hazard of COVID-19 hospitalisation or death between UDCA users versus non-users in a population with primary biliary cholangitis (PBC) or primary sclerosing cholangitis (PSC).MethodsWith the approval of NHS England, we conducted a population-based cohort study using primary care records between 1 March 2020 and 31 December 2022, linked to death registration data and hospital records through the OpenSAFELY-TPP platform. Cox proportional hazards regression was used to estimate hazard ratios (HR) and 95% confidence intervals (CI) for the association between time-varying UDCA exposure and COVID-19 related hospitalisation or death, stratified by geographical region and considering models unadjusted and fully adjusted for pre-specified confounders.ResultsWe identify 11,305 eligible individuals, 640 were hospitalised or died with COVID-19 during follow-up, 400 (63%) events among UDCA users. After confounder adjustment, UDCA is associated with a 21% relative reduction in the hazard of COVID-19 hospitalisation or death (HR 0.79, 95% CI 0.67‚Äì0.93), consistent with an absolute risk reduction of 1.35% (95% CI 1.07%‚Äì1.69%).ConclusionsWe found evidence that UDCA is associated with a lower hazard of COVID-19 related hospitalisation and death, support calls for clinical trials investigating UDCA as a preventative measure for severe COVID-19 outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00664-y" target="_blank">https://www.nature.com/articles/s43856-024-00664-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-18</strong> <a href="https://www.nature.com/articles/s41541-024-01022-8" target="_blank">
            ü§ñ Eating too much food increases the risk of obesity and other health problems, but it may not affect how well your immune system responds to vaccines like those for COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-356', this)">
            More</button></h4>
            <div id="toggle-356" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Obesity does not influence SARS-CoV-2 humoral vaccine immunogenicity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Obesity affects how well the body responds to vaccines against diseases like COVID-19, including antibodies and immune cells that fight off infections. Research from over 34 months in 697 people, mostly women with no obesity, found that there was no consistent difference in how well they responded to SARS-CoV-2 vaccine. The study controlled for other factors and didn't find a significant effect of obesity on peak levels or types of antibodies produced by the immune system.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Obesity is a recognized factor influencing immune function and infectious disease outcomes. Characterization of the influence of obesity on SARS-CoV-2 humoral vaccine immunogenicity is required to properly tailor vaccine type (mRNA, viral-vector, protein subunit vaccines) and dosing schedule. Data from a prospective cohort study collected over 34 months was used to evaluate the slope of antibody production and decay and neutralizing capacity following SARS-CoV-2 vaccination in individuals with and without obesity at baseline. Most participants were female (65.4%), white (92.4%), and received mRNA vaccines. 210 were obese and 697 non-obese. Sex and infection-acquired immunity were identified as effect modifiers for the relationship between obesity and COVID-19 vaccine humoral immunogenicity. No consistent influence of obesity on peak titres, titre retention, antibody isotype (IgG, IgM, IgA), or neutralization was identified when controlling for other key variables. It may not be necessary to consider this variable when developing SARS-CoV-2 vaccine dosing strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01022-8" target="_blank">https://www.nature.com/articles/s41541-024-01022-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-18</strong> <a href="https://www.nature.com/articles/s41598-024-77386-7" target="_blank">
            ü§ñ Predictive Risk Models for COVID-19 Patients Using a Multi-Thresholding Meta-Algorithm

A predictive model that uses machine learning and multiple algorithms to forecast the risk of COVID-19 in patients.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-357', this)">
            More</button></h4>
            <div id="toggle-357" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predictive risk models for COVID-19 patients using the multi-thresholding meta-algorithm</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A machine learning model has been developed to predict complications leading to ICU admission or death among COVID-19 patients in hospitals. The goal is to identify minority classes of discharged patients who are at high risk. To address dataset imbalance, a new methodology called Multi-Thresholding meta-algorithm (MTh) is introduced. This method adjusts class probabilities using misclassification costs and is effective in imbalanced datasets. Bayesian networks are used to create a robust predictive model. The model finds that certain patient characteristics, such as high Charlson Index and pre-existing conditions, significantly influence the risk of ICU admission and mortality. An explanatory model also shows how these factors interact with therapeutic limits to assess overall risk.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aims to develop a Machine Learning model to assess the risks faced by COVID-19 patients in a hospital setting, focusing specifically on predicting the complications leading to Intensive Care Unit (ICU) admission or mortality, which are minority classes compared to the majority class of discharged patients. We operate within a multiclass framework comprising three distinct classes, and address the challenge of dataset imbalance, a common source of model bias. To effectively manage this, we introduce the Multi-Thresholding meta-algorithm (MTh), an innovative output-level methodology that extends traditional thresholding from binary to multiclass classification. This methodology dynamically adjusts class probabilities using misclassification costs, making it highly effective in imbalanced datasets. Our approach is further enhanced by integrating the simplicity, transparency, and effectiveness of Bayesian networks to create a robust predictive model. Using patient admission data, the model accurately identifies key risk and protective factors for COVID-19 outcomes. Our findings indicate that certain patient characteristics, such as high Charlson Index and pre-existing conditions, significantly influence the risk of ICU admission and mortality. Moreover, we introduce an explanatory model that elucidates the interrelationships among these factors, demonstrating the influence of therapeutic limits on the overall risk assessment of COVID-19 patients. Overall, our research provides a significant contribution to the field of Machine Learning by offering a novel solution for multiclass classification in the context of imbalanced datasets. This model not only enhances predictive accuracy but also supports critical decision-making processes in healthcare, potentially improving patient outcomes and optimizing clinical resource allocation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77386-7" target="_blank">https://www.nature.com/articles/s41598-024-77386-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-17</strong> <a href="https://www.nature.com/articles/s41541-024-01014-8" target="_blank">
            ü§ñ Prevention and Treatment of Influenza B Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-358', this)">
            More</button></h4>
            <div id="toggle-358" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preclinical evaluation of a universal inactivated influenza B vaccine based on the mosaic hemagglutinin-approach</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new flu vaccination approach has been developed, using antibodies (mHA) from exotic avian viruses that are similar to those causing human diseases but more mild and less likely to be contagious. This new method stimulates a strong immune response by targeting specific areas on the influenza B virus that humans have evolved to be resistant to. Combining these mHA vaccines with adjuvants like Toll-like receptor-9 agonists and oils increases their effectiveness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We have developed a new universal influenza B vaccination strategy based on inactivated influenza B viruses displaying mosaic hemagglutinins (mHAs). Recombinant mHA viruses were constructed by replacing the four major antigenic sites of influenza B virus HAs, with those from exotic avian influenza A virus HAs. Sequential vaccination of na√Øve mice with mHA-based vaccines elicited higher immune responses towards the immuno-subdominant conserved epitopes of the HA than vaccination with wildtype viruses. Among the different preparations tested, mHA split vaccines were less immunogenic than their whole inactivated virus counterparts. This lower immunogenicity was overcome by the combination with adjuvants. mHA split vaccines adjuvanted with a Toll-like receptor-9 agonist (CpG 1018) increased Th1 immunity and invivocross-protection, whereas adjuvanting with an MF59-like oil-in-water nano-emulsion (AddaVax) enhanced and broadened humoral immune responses and antibody-mediated cross-protection. The mHA vaccines with or without adjuvant were subsequently evaluated in mice that were previously immunized to closely mimic human pre-existing immunity to influenza B viruses and the contribution of innate and cellular immunity was evaluated in this model. We believe these preclinical studies using the mHA strategy represent a major step toward the evaluation of a universal influenza B virus vaccine in clinical trials.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01014-8" target="_blank">https://www.nature.com/articles/s41541-024-01014-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-16</strong> <a href="https://www.nature.com/articles/s41598-024-79407-x" target="_blank">
            ü§ñ Detecting Respiratory Viruses in Children with Influenza-like Illness</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-359', this)">
            More</button></h4>
            <div id="toggle-359" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Microfluidic qPCR for detection of 21 common respiratory viruses in children with influenza-like illness</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Multiple viral infections lead to high illness and death rates worldwide. However, current surveillance platforms focus mainly on seasonal influenza viruses. The COVID-19 pandemic highlights the importance of a more comprehensive approach, using advanced technologies like microfluidic quantitative polymerase chain reaction (qPCR) for detecting multiple respiratory viruses in children presenting with flu-like symptoms. This study analyzed samples from children aged 24 to 59 months diagnosed with influenza-like illness in The Gambia and found that common viral infections include rhinovirus and adenovirus, as well as other viruses like parainfluenza virus 3, influenza B, and human metapneumovirus B. A significant proportion of positive samples had multiple viruses detected. These findings suggest that microfluidic qPCR can be a useful tool for comprehensive detection of multiple respiratory viruses in surveillance platforms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multiple respiratory viruses lead to high morbidity and mortality, yet global surveillance platforms focus primarily on seasonal influenza viruses. The COVID-19 pandemic and new RSV vaccines highlight the importance of a broader approach. Upper respiratory tract swabs from children aged 24‚Äì59¬†months presenting with influenza-like illness in The Gambia were collected during follow-up of a live-attenuated influenza vaccine randomised controlled trial in 2017‚Äì18. A microfluidic quantitative polymerase chain reaction (qPCR) assay was established and used to detect 21 respiratory viruses. 76.6% of samples had one or more viruses detected (n‚Äâ=‚Äâ121/158). The viruses detected most frequently were rhinovirus (n‚Äâ=‚Äâ37/158, 23.4%) and adenovirus (n‚Äâ=‚Äâ34/158, 21.5%), followed by parainfluenza virus 3, influenza B and human metapneumovirus B. A third of positive samples had multiple viruses detected (two n‚Äâ=‚Äâ31/121, 25.6%; three n‚Äâ=‚Äâ9/121, 7.4%). Our data demonstrates how microfluidic qPCR is a useful tool for high-throughput, comprehensive detection of multiple respiratory viruses in surveillance platforms. Rapidly changing epidemiology exemplifies the need for new, broader approaches to virus surveillance in low-resource settings to respond to future epidemics and to guide the need for and use of new prevention and therapeutic measures.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79407-x" target="_blank">https://www.nature.com/articles/s41598-024-79407-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-16</strong> <a href="https://www.nature.com/articles/s41598-024-76855-3" target="_blank">
            ü§ñ High throughput screening detects a protein involved in regulating immune response to cancer treatment.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-360', this)">
            More</button></h4>
            <div id="toggle-360" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>High throughput screen identifies lysosomal acid phosphatase 2 (ACP2) to regulate IFN-1 responses to potentiate oncolytic VSV‚àÜ51 activity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Strategies to improve oncolytic virotherapy (OV) by enhancing innate immunity are being researched. A new approach involves using vanadium-based compounds as inhibitors of pan-phosphatase proteins (PPs), which can potentiate OV treatment. To boost effectiveness, a high-throughput screen was conducted using silencing RNA targeting human PPs to identify compounds that enhance VSV‚àÜ51 infectivity and oncolysis.

One key protein, lysosomal acid phosphatase 2 (ACP2), was found to be particularly effective in increasing VSV‚àÜ51 viral titers. Knocking down ACP2 significantly increased the virus's growth. Analysis of RNA sequencing revealed that ACP2 regulates antiviral type I interferon signaling pathways similar to vanadium.

To further enhance OV treatment, a short-hairpin RNA (shRNA) against ACP2 was designed and introduced into VSV‚àÜ51 under a miR-30 promoter. This engineered virus produced the miR-30 promoter, knocked down ACP2, and ultimately increased viral production compared to its non-targeting counterpart.

This study suggests that targeting proteins like ACP2 may improve oncolytic virotherapy by boosting the body's innate immunity against cancer cells, thereby increasing treatment efficacy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Strategies in genetic and pharmacological modulation of innate immunity to enhance oncolytic virotherapy (OV) efficacy are being explored. We have recently characterized the ability for vanadium-based compounds, a class of pan-phosphatase (PP) inhibitors, to potentiate OVs. We next sought to identify PPs that could be targeted to enhance OVs, akin to vanadium. By conducting a high-throughput screen of a library of silencing RNA (siRNA) targeting human PPs, we uncovered several PPs that robustly enhanced infectivity and oncolysis of the oncolytic vesicular stomatitis virus (VSV‚àÜ51). Knockdown of our top validated hit, lysosomal acid phosphatase 2 (ACP2), increased VSV‚àÜ51 viral titers by over 20-fold. In silico analysis by RNA sequencing revealed ACP2 to regulate antiviral type I interferon (IFN-1) signaling pathways, similar to vanadium. To further exploit this mechanism for therapeutic gain, we encoded a short-hairpin RNA (shRNA) against ACP2 into oncolytic vesicular stomatitis virus (VSV‚àÜ51) under a miR-30 promoter. This bioengineered OV demonstrated expression of the miR-30 promoter, knockdown of ACP2, repression and ultimately, showed markedly enhanced viral VSV‚àÜ51 particle production compared to its non-targeting control counterpart. Altogether, this study identifies IFN-1 regulating PP targets, namely ACP2, that may prove instrumental in increasing the therapeutic efficacy of OVs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76855-3" target="_blank">https://www.nature.com/articles/s41598-024-76855-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-15</strong> <a href="https://www.nature.com/articles/s41598-024-78354-x" target="_blank">
            ü§ñ Small compounds interfering with RNA binding fluorescent probes in ds-RNA interfere with SARS CoV 2 RdRp inhibitors identification.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-361', this)">
            More</button></h4>
            <div id="toggle-361" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Interference of small compounds and Mg2+with dsRNA-binding fluorophores compromises the identification of SARS-CoV-2 RdRp inhibitors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied compounds suspected to be viral RNA-dependent RNA polymerases inhibitors (RdRp inhibitors) to develop effective treatments for COVID-19. They tested these compounds in biochemical assays using different methods, but found that most small molecules interfere with a test method that measures enzyme activity. As a result, they concluded that this testing method was unreliable and needed independent validation. They also discovered that one of the potential RdRp inhibitors tested is actually an inhibitor of multiple enzymes, which means it's not specific to SARS-CoV-2's RNA polymerase.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic highlighted the need for the rapid development of antiviral therapies. Viral RNA-dependent RNA polymerases (RdRp) are promising targets, and numerous virtual screenings for potential inhibitors were conducted without validation of the identified hits. Here we have tested a set of presumed RdRp inhibitors in biochemical assays based on fluorometric detection of RdRp activity or on the electrophoretic separation or RdRp products. We find that fluorometric detection of RdRp activity is unreliable as a screening method because many small compounds interfere with fluorophore binding to dsRNA, and this effect is enhanced by the Mg2+metal ions used by nucleic acid polymerases. The fact that fluorimetric detection of RdRp activity leads to false-positive hits underscores the requirement for independent validation methods. We also show that suramin, one of the proposed RdRp inhibitors that could be validated biochemically, is a multi-polymerase inhibitor. While this does not hinder its potential as an antiviral agent, it cannot be considered an specific inhibitor of SARS-CoV-2 RdRp.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78354-x" target="_blank">https://www.nature.com/articles/s41598-024-78354-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-15</strong> <a href="https://www.nature.com/articles/s41598-024-79509-6" target="_blank">
            ü§ñ Study on Fundus Lesions in Patients with COVID-19 Vision Impairment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-362', this)">
            More</button></h4>
            <div id="toggle-362" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A cross-sectional study of fundus lesion characteristics in patients with acute visual impairment caused by COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study investigated the characteristics of visual dysfunction and fundus lesions in patients with COVID-19 pneumonia. The study looked at 48 patients who were infected with COVID-19 between December 2022 and February 2023. It found that most patients had bilateral involvement and showed abnormal retinal changes on various tests, including OCT, MCI, and IR.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the characteristics of acute visual dysfunction and fundus lesions in patients with COVID-19 pneumonia. A retrospective case series study was conducted. Data from 48 patients (96 eyes) with COVID-19 infection who presented to our ophthalmology department with acute onset visual disturbance between December 5, 2022, and February 28, 2023 were collected. Asymptomatic patients and those who had already recovered were excluded. Data collected included patient demographics, ophthalmic examinations, multicolor imaging (MCI), infrared autofluorescence (IR), spectral-domain optical coherence tomography (OCT), fundus fluorescein angiography (FFA). Of the 48 patients, 15 were male and 33 were female, with a mean age of 32 years. All patients had bilateral involvement. OCT showed hyperreflective signals in the outer plexiform layer and outer nuclear layer of the macular region in all 96 eyes of 48 patients (100%). Additionally, 66 eyes of 33 patients (68.8%) of eyes demonstrated abnormal reflectivity in the ellipsoid and interdigitation zones. MCI revealed petaloid or wedge-shaped hyperreflective areas in the macula in 46 (47.9%) of eyes, corresponding to hyporeflective areas on IR. Cotton-wool spots were observed in the peripapillary or posterior pole area in 54 (56.3%) of eyes. COVID-19 infection can lead to acute, bilateral, symmetric, and widespread retinal damage. Characteristic findings can be observed in ophthalmological examinations such as OCT, MCI, and IR.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79509-6" target="_blank">https://www.nature.com/articles/s41598-024-79509-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-14</strong> <a href="https://www.nature.com/articles/s41598-024-78200-0" target="_blank">
            ü§ñ Sex Differences in Risk of Pneumonia During COVID-19 in Mexico</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-363', this)">
            More</button></h4>
            <div id="toggle-363" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sex differences in pneumonia risk during COVID-19 in Mexico</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study aimed to assess the risk of pneumonia among adults with COVID-19, particularly based on their sex, during two phases of the pandemic in Mexico. A large dataset was analyzed from 2020 to 2023. The results showed that men had a higher risk of severe manifestations of the disease compared to women.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aimed to evaluate the pneumonia risk based on the patient‚Äôs sex during the COVID-19 pandemic and the early months of the endemic phase of the disease in Mexico. A retrospective cohort study was conducted using a dataset resulting from the epidemiological surveillance of COVID-19 (February 2020 to August 2023). Data from 1.6¬†million adults with laboratory-positive disease, were analyzed. Risk ratios (RR) and 95% confidence intervals (CI), computed through generalized linear regression models, were used. The overall risk of pneumonia was 9.3% (95% CI 9.2‚Äì9.4%), with sex-specific estimates of 7.0% (95% CI 6.9‚Äì7.1%) for women and 12.0% (95% CI 11.9‚Äì12.1%) for men. This disparity was consistently observed throughout all phases of the pandemic, including the endemic phase of the disease. After adjusting for age, predominant viral genotype at illness onset and preexisting medical conditions, men had a 3.3% higher risk of severe manifestations when compared to women (RR‚Äâ=‚Äâ1.033, 95% CI 1.032‚Äì1.034). Our research highlights the potential role of patients‚Äô sex as a factor influencing pneumonia risk during and after the COVID-19 pandemic in Mexico. These findings may provide useful considerations for healthcare planning and policy development focused on addressing the impact of the disease on vulnerable populations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78200-0" target="_blank">https://www.nature.com/articles/s41598-024-78200-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-14</strong> <a href="https://www.nature.com/articles/s41541-024-01019-3" target="_blank">
            ü§ñ Global Public Perspectives on mRNA Vaccines and Therapies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-364', this)">
            More</button></h4>
            <div id="toggle-364" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mapping global public perspectives on mRNA vaccines and therapeutics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

A study examined how people viewed mRNA vaccines and therapeutics on social media during the COVID-19 pandemic, and found widespread negative attitudes and concerns about safety, effectiveness, and trustworthiness. The study compared online perspectives with official reports from the Vaccine Adverse Event Reporting System, revealing a lack of confidence in these treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The development and rollout of mRNA vaccines during COVID-19 marked a significant advancement in vaccinology, yet public hesitation to vaccination was prevalent, indicating the potential risk that future mRNA-based medical innovations will fail to be adopted. Utilizing a combined approach of large language models with manual validation and unsupervised machine learning, we conducted a social listening analysis to assess attitudes towards mRNA vaccines and therapeutics on Twitter from June 2022 to May 2023, contrasting online perspectives with data from the Vaccine Adverse Event Reporting System. Our findings reveal widespread negative sentiment and a global lack of confidence in the safety, effectiveness, and trustworthiness of mRNA vaccines and therapeutics, with frequent discussions of severe vaccine side effects, rumors, and misinformation. This underscores the need for targeted communication strategies to foster acceptance of medical treatments and strengthen public trust in order to enhance societal resilience to future health challenges.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01019-3" target="_blank">https://www.nature.com/articles/s41541-024-01019-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-14</strong> <a href="https://www.nature.com/articles/s41599-024-04079-x" target="_blank">
            ü§ñ **Understanding Stress Tolerance and Personal Readiness**

An analysis using Partial Least Squares Structural Equation Modeling to examine relationships between stress tolerance, personal readiness for change, and various psychological factors.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-365', this)">
            More</button></h4>
            <div id="toggle-365" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Decoding the nexus of stress tolerance, personal readiness for change, and psychological factors using PLS-SEM</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study investigates how individual differences in stress tolerance, adaptability, and other factors affect their behavior and overall quality of life. It examines various psychological factors that influence human behavior using a sophisticated statistical method to discover the connections between these elements. The results show that self-esteem, others' perceptions, and world views are crucial in shaping an individual's level of stress tolerance, readiness for change, and overall well-being, with one factor (preference) acting as a significant mediator in this relationship.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This research examines the intricate connections among three key elements: an individual‚Äôs ability to tolerate sources of stress, their willingness to adapt to new situations, and various psychological factors that influence human behavior. To analyze these relationships, the study employs a sophisticated statistical technique known as partial least squares structural equation modeling (PLS-SEM). A total of 327 Kazakhstan residents voluntarily participated in the study, comprising 84.6% women and 15.4% men, with ages ranging from 14 to 67 years old. ‚ÄúStress Source Tolerance Scale‚Äù examined stress tolerance, ‚ÄúPersonality Readiness Scale‚Äù measured readiness to change, and ‚ÄúFatigue, Monotony, Satiety and Stress Questionnaire‚Äù investigated psychological states. The results indicate significant direct and indirect effects of self-esteem, others‚Äô perceptions, and world perceptions on stress tolerance, readiness for change, and psychological states. Moreover, systematic preference has a significant mediating effect on the relationship between openness to change and psychological states. Results suggest that understanding these complex relationships may have implications for health, organizational effectiveness, and psychological well-being.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-024-04079-x" target="_blank">https://www.nature.com/articles/s41599-024-04079-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-14</strong> <a href="https://www.nature.com/articles/s44298-024-00063-z" target="_blank">
            ü§ñ Neutralizing Rebound Depression Antibodies 

### Potent Neutralization by a RBD Antibody with Broad Specificity for SARS-CoV-2 JN.1 and Other Variants

### KEY POINTS 
Broadly neutralizes SARS-CoV-2, including variants like JN.1</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-366', this)">
            More</button></h4>
            <div id="toggle-366" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Potent neutralization by a RBD antibody with broad specificity for SARS-CoV-2 JN.1 and other variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new vaccine is being developed that targets a specific protein on the surface of the SARS-CoV-2 virus, which has led to multiple variants of the disease. To combat this, researchers are working on developing a treatment that can neutralize the virus even when it mutates and becomes harder to target. This vaccine uses a human antibody that was made by someone who had recovered from COVID-19 and is specifically designed to bind to the protein of SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 continues to be a public health burden, driven in-part by its continued antigenic diversification and resulting emergence of new variants. By increasing herd immunity, current vaccines have improved infection outcomes for many. However, prophylactic and treatment interventions that are not compromised by viral evolution of the Spike protein are still needed. Using a differential staining strategy with a rationally designed SARS-CoV-2 Receptor Binding Domain (RBD) ‚Äì ACE2 fusion protein and a native Omicron RBD protein, we developed a recombinant human monoclonal antibody (hmAb) from a convalescent individual following SARS-CoV-2 Omicron infection. The resulting hmAb, 1301B7 potently neutralized a wide range of SARS-CoV-2 variants including the original Wuhan-1, the more recent Omicron JN.1 strain, and SARS-CoV. 1301B7 contacts the ACE2 binding site of RBD exclusively through its VH1-69 heavy chain. Broad specificity is achieved through 1301B7 binding to many conserved residues of Omicron variants including Y501 and H505. Consistent with its extensive binding epitope, 1301B7 is able to potently diminish viral burden in the upper and lower respiratory tract and protect mice from challenge with Omicron XBB1.5 and Omicron JN.1 viruses. These results suggest 1301B7 has broad potential to prevent or treat clinical SARS-CoV-2 infections and to guide development of RBD-based universal SARS-CoV-2 prophylactic vaccines and therapeutic approaches.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00063-z" target="_blank">https://www.nature.com/articles/s44298-024-00063-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-13</strong> <a href="https://www.nature.com/articles/s41598-024-78538-5" target="_blank">
            ü§ñ Brain Infection Prevention with Zinc Sulfate</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-367', this)">
            More</button></h4>
            <div id="toggle-367" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Elimination of olfactory sensory neurons by zinc sulfate inoculation prevents SARS-CoV-2 infection of the brain in K18-hACE2 transgenic mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 is a global health issue caused by a new coronavirus that can be life-threatening. The virus affects not only the lungs but also the central nervous system, which controls many functions in the body. Researchers are studying how the virus travels from the nose to the brain and why some people might develop severe symptoms or even die.

Scientists have found that one possible way the virus enters the brain is through the olfactory nerve, a part of the brain's sensory system that smells and transmits information to other parts of the brain. When this nerve is damaged or absent, it may be harder for the virus to reach the brain. To test their hypothesis, researchers used mice with damaged olfactory nerves to see if they could prevent the virus from reaching the brain.

The results showed that when these mice had healthy olfactory nerves, they were less likely to develop severe symptoms after being infected with COVID-19. This suggests that blocking the olfactory nerve pathway can help prevent the virus from invading the brain and causing illness in humans.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease-2019 (COVID-19), attributed to the severe acute respiratory syndrome-related coronavirus-2 (SARS-CoV-2), has posed global health challenges since it first emerged in 2019, and its impact continues to persist. The neurotropic nature of SARS-CoV-2 remains undisclosed, though researchers are proposing hypotheses on how the virus is transmitted to the central nervous system. One of the prevailing hypotheses is that SARS-CoV-2 travels through the olfactory nerve system via the olfactory epithelium (OE). Using a K18-human angiotensin converting-enzyme 2 (hACE2) transgenic mouse model with impaired olfactory sensory neurons (OSNs) induced by zinc sulfate, we examined the role of the olfactory nerve in the brain invasion by SARS-CoV-2. Mice lacking OSNs exhibited reduced levels of viral transmission to the brain, leading to significantly improved outcomes following SARS-CoV-2 infection. Moreover, a positive correlation was observed between viral persistence in the OE and brain infection. These results indicate that early inhibition of the olfactory nerve pathway effectively prevents viral invasion of the brain in K18-hACE2 mice. Our study underscores the significance of the olfactory nerve pathway in the transmission of SARS-CoV-2 to the brain.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78538-5" target="_blank">https://www.nature.com/articles/s41598-024-78538-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-12</strong> <a href="https://www.nature.com/articles/s41598-024-78992-1" target="_blank">
            ü§ñ Human Microbiome Alteration Due to Critically Ill COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-368', this)">
            More</button></h4>
            <div id="toggle-368" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Major alteration of lung microbiome and the host responses in critically ill COVID-19 patients with high viral load</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Patients on invasive mechanical ventilation are at higher risk of developing pneumonia due to COVID-19. A study analyzed lung microbiota and host immune responses in patients with confirmed SARS-CoV-2 infection who developed pneumonia.

The researchers found that certain microorganisms were more common in patients with VAP, including Staphylococcus and Enterobacteriaceae. They also discovered changes in the metabolic activity of non-VAP patients compared to those with VAP, which may contribute to their increased susceptibility to developing VAP.

The study suggests a link between respiratory microbiome alterations and ventilator-associated pneumonia, particularly in patients who have higher levels of SARS-CoV-2 in their respiratory samples. These findings can help inform management and prevention strategies for COVID-19 patients on mechanical ventilation.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients with COVID-19 under invasive mechanical ventilation are at higher risk of developing ventilator-associated pneumonia (VAP), associated with increased healthcare costs, and unfavorable prognosis. The underlying mechanisms of this phenomenon have not been thoroughly dissected. Therefore, this study attempted to bridge this gap by performing a lung microbiota analysis and evaluating the host immune responses that could drive the development of VAP. In this prospective cohort study, mechanically ventilated patients with confirmed SARS-CoV-2 infection were enrolled. Nasal swabs (NS), endotracheal aspirates (ETA), and blood samples were collected initially within 12¬†h of intubation and again at 72¬†h post-intubation. Plasma samples underwent cytokine and metabolomic analyses, while NS and ETA samples were sequenced for lung microbiome examination. The cohort was categorized based on the development of VAP. Data analysis was conducted using RStudio version 4.3.1. In a study of 36 COVID-19 patients on mechanical ventilation, significant differences were found in the nasal and pulmonary microbiome, notably inStaphylococcusandEnterobacteriaceae, linked to VAP. Patients with VAP showed a higher SARS-CoV-2 viral load in respiratory samples, elevated neutralizing antibodies, and reduced inflammatory cytokines, including IFN-Œ¥, IL-1Œ≤, IL-12p70, IL-18, IL-6, TNF-Œ±, and CCL4. Metabolomic analysis revealed changes in 22 metabolites in non-VAP patients and 27 in VAP patients, highlighting D-Maltose-Lactose, Histidinyl-Glycine, and various phosphatidylcholines, indicating a metabolic predisposition to VAP. This study reveals a critical link between respiratory microbiome alterations and ventilator-associated pneumonia in COVID-19 patients with higher SARS-CoV-2 viral loads in respiratory samples, elevated neutralizing antibodies, and reduced inflammatory cytokines, including IFN-Œ¥, IL-1Œ≤, IL-12p70, IL-18, IL-6, TNF-Œ±, and CCL4. These findings provide novel insights into the underlying mechanisms of VAP, with potential implications for management and prevention.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78992-1" target="_blank">https://www.nature.com/articles/s41598-024-78992-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-12</strong> <a href="https://www.nature.com/articles/s41467-024-53727-y" target="_blank">
            ü§ñ Neutralizing antibodies in vaccines protect against serious and life-threatening COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-369', this)">
            More</button></h4>
            <div id="toggle-369" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing antibody correlate of protection against severe-critical COVID-19 in the ENSEMBLE single-dose Ad26.COV2.S vaccine efficacy trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Assessment of the relationship between antibodies against COVID-19 vaccine and severity of illness.

A study found that high levels of neutralizing antibodies at 4 weeks after vaccination were associated with better outcomes in COVID-19, including lower rates of severe critical cases. The study also looked at other antibody levels and found a correlation with protection against moderate COVID-19 as well as between severe and moderate disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Assessment of immune correlates of severe COVID-19 has been hampered by the low numbers of severe cases in COVID-19 vaccine efficacy (VE) trials. We assess neutralizing and binding antibody levels at 4 weeks post-Ad26.COV2.S vaccination as correlates of risk and of protection against severe-critical COVID-19 through 220 days post-vaccination in the ENSEMBLE trial (NCT04505722), constituting ~4.5 months longer follow-up than our previous correlates analysis and enabling inclusion of 42 severe-critical vaccine-breakthrough cases. Neutralizing antibody titer is a strong inverse correlate of severe-critical COVID-19, with estimated hazard ratio (HR) per 10-fold increase 0.35 (95% CI: 0.13, 0.90). In a multivariable model, HRs are 0.31 (0.11, 0.89) for neutralizing antibody titer and 1.22 (0.49, 3.02) for anti-Spike binding antibody concentration. VE against severe-critical COVID-19 rises with neutralizing antibody titer: 63.1% (95% CI: 40.0%, 77.3%) at unquantifiable [<4.8975 International Units (IU)50/ml], 85.2% (47.2%, 95.3%) at just-quantifiable (5.2 IU50/ml), and 95.1% (81.1%, 96.9%) at 90thpercentile (30.2 IU50/ml). At the same titers, VE against moderate COVID-19 is 32.5% (11.8%, 48.4%), 33.9% (19.1%, 59.3%), and 60.7% (40.4%, 76.4%). Protection against moderate vs. severe disease may require higher antibody levels, and very low antibody levels and/or other immune responses may associate with protection against severe disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53727-y" target="_blank">https://www.nature.com/articles/s41467-024-53727-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-12</strong> <a href="https://www.nature.com/articles/s41590-024-02010-9" target="_blank">
            ü§ñ Immune Response to mRNA Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-370', this)">
            More</button></h4>
            <div id="toggle-370" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Disease-associated B cells and immune endotypes shape adaptive immune responses to SARS-CoV-2 mRNA vaccination in human SLE</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 mRNA vaccination has shown reduced effectiveness in people with weakened immune systems, the cause of autoimmune diseases such as systemic lupus erythematosus (SLE). Scientists studied a group of patients who received the Wuhan-Hu-1 vaccine to understand why it was not working for them. The study found that these patients had weaker responses from certain parts of their immune system, which may be due to changes in these cells over time. It also showed that people with SLE were more likely to have strong responses against a specific type of protein on the virus, but this did not lead to protection against the full range of viruses. The study suggests that there are different types of B and T cell responses in people with SLE, which may help explain why some vaccines are effective while others are not.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 mRNA vaccination has reduced effectiveness in certain immunocompromised individuals. However, the cellular mechanisms underlying these defects, as well as the contribution of disease-induced cellular abnormalities, remain largely unexplored. In this study, we conducted a comprehensive serological and cellular analysis of patients with autoimmune systemic lupus erythematosus (SLE) who received the Wuhan-Hu-1 monovalent mRNA coronavirus disease 2019 vaccine. Our findings revealed that patients with SLE exhibited reduced avidity of anti-receptor-binding domain antibodies, leading to decreased neutralization potency and breadth. We also observed a sustained anti-spike response in IgD‚àíCD27‚àí‚Äòdouble-negative (DN)‚Äô DN2/DN3 B cell populations persisting during memory responses and with greater representation in the SLE cohort. Additionally, patients with SLE displayed compromised anti-spike T cell immunity. Notably, low vaccine efficacy strongly correlated with higher values of a newly developed extrafollicular B and T cell score, supporting the importance of distinct B cell endotypes. Finally, we found that anti-BAFF blockade through belimumab treatment was associated with poor vaccine immunogenicity due to inhibition of naive B cell priming and an unexpected impact on circulating T follicular helper cells.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-02010-9" target="_blank">https://www.nature.com/articles/s41590-024-02010-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-12</strong> <a href="https://www.nature.com/articles/s41598-024-78904-3" target="_blank">
            ü§ñ Mechanical Ventilation and Insulin Requirements

Prone Positioning Associated with Increased Insulin Requirements in Patients with COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-371', this)">
            More</button></h4>
            <div id="toggle-371" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prone positioning is associated with increased insulin requirements in mechanically ventilated patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

We studied patients who were on mechanical ventilation due to severe COVID-19, looking at how their blood sugar levels changed while they were being treated for hypoxia (lack of oxygen). We found that when these patients were moved from a flat position to an upright one (prone positioning), their blood sugar levels improved. But surprisingly, this improvement in blood sugar levels actually led to higher insulin doses being given, even though the patient's glucose control remained good. This was observed across different time periods and multiple groups of patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Stress hyperglycaemia is common in critical illness. We have previously observed that increasing severity of respiratory failure in patients with severe COVID-19 is associated with increased insulin demand. Given previously reported direct effects of hypoxia on insulin action, we reasoned that rapid improvements in oxygenation following prone positioning may improve insulin sensitivity and increase risk of hypoglycaemia. A retrospective multi-centre service evaluation comparing blood glucose and insulin administration in patients with COVID-19 pneumonitis receiving prone mechanical ventilation, comparing the 16 h pre-prone and 16 h post-prone time periods. 155 patients were included in this analysis. Oxygenation improved significantly following prone positioning (change in SpO2/FIO2per hour prone: 3.01‚Äâ¬±‚Äâ0.14,P<‚Äâ0.0001). Glycaemic control was similar during the supine and prone study periods, and there were no hypoglycaemic events in the prone study period. Prone positioning was associated with an unexpected modest but significant increase in insulin requirements (mean difference in total insulin dose (IU): 8.32‚Äâ¬±‚Äâ2.14,P<‚Äâ0.001) that was robust to several sensitivity analyses, and could not be explained by changes in carbohydrate intake. We did not observe an increased rate of hypoglycaemia during prone ventilation and the adequacy of glycaemic control was comparable during the supine and prone study periods. Unexpectedly, prone ventilation was associated with an increase in insulin requirements despite significant improvement in hypoxaemia. Our findings support the safety of prone ventilation with respect to glycaemic control and identify a novel relationship between ventilation position and insulin requirements in critical illness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78904-3" target="_blank">https://www.nature.com/articles/s41598-024-78904-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-12</strong> <a href="https://www.nature.com/articles/s43856-024-00650-4" target="_blank">
            ü§ñ COVID-19 Treatment Protocol Evaluation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-372', this)">
            More</button></h4>
            <div id="toggle-372" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The effects of remdesivir on long-term symptoms in patients hospitalised for COVID-19: a pre-specified exploratory analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A large group of patients in hospitals had symptoms from COVID-19 for several months, but some did not get better despite receiving treatment. The study looked at how well these patients were doing three months after the initial symptoms stopped. It found that treating them with a medication called remdesivir did not help much.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundThere is an unmet need for treatment of long-term symptoms following COVID-19. Remdesivir is currently the only antiviral approved by the European Medicines Agency for hospitalised patients. Here, we report on the effect of remdesivir in addition to standard of care on long-term symptoms and quality of life in hospitalised patients with COVID-19 as part of the open-label randomised NOR-Solidarity trial (NCT04321616).MethodsA total of 185 patients were included in the main trial, of which 118 (60%) were randomised to either remdesivir (n‚Äâ=‚Äâ42; 36%) or a post-hoc defined control group composed of patients who received standard of care alone or standard of care with hydroxychloroquine (n‚Äâ=‚Äâ76; 64%). Participants were given quality of life surveys to fill out to gauge their self-reported health over time (the COPD assessment test, the EQ-5D-5L and the RAND SF-36).ResultsHere we show that after three months, patients treated with remdesivir do not show significant improvements in stated health compared to those who were not. There are self-reported symptoms of fatigue [mean remdesivir group 2.6 (standard deviation 1.5) v control 2.1 (1.6), 95% confidence interval(CI) ‚àí1.17 to 0.15, p‚Äâ=‚Äâ0.129], shortness of breath [3.0 (1.7) v 2.1 (1.8), 95% CI ‚àí1.53 to 0.16, p‚Äâ=‚Äâ0.110] and coughing [1.8 (1.6) v 1.2 (1.5), 95% CI ‚àí1.3 to 0.33, p‚Äâ=‚Äâ0.237] 3 months after randomisation assessed via the COPD Assessment Test.ConclusionsOur findings indicate that treatment with remdesivir during hospitalisation does not provide any clinically relevant long-term benefit.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00650-4" target="_blank">https://www.nature.com/articles/s43856-024-00650-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-11</strong> <a href="https://www.nature.com/articles/s41598-024-79396-x" target="_blank">
            ü§ñ Assessment of Knowledge and Attitude of Healthcare Professionals Towards Mopovirus (Mpox) in a Nigerian Hospital

Healthcare staff's perception, understanding, and ability to address the disease effectively are crucial for effective health management and public health responses. Their knowledge can greatly influence the quality of care provided, while their attitude towards the disease can affect adherence to infection control measures.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-373', this)">
            More</button></h4>
            <div id="toggle-373" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessment of knowledge and attitude of healthcare professionals towards Mpox in a Nigerian hospital</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Mpox is a disease that affects people worldwide and poses significant public health challenges. Research into healthcare workers' knowledge about the disease is limited. A study was conducted at Rivers State University Teaching Hospital in Nigeria to assess healthcare workers' understanding, awareness, attitudes, and factors related to mpox.

A descriptive cross-sectional study design was used, with 227 healthcare professionals participating. The results showed that a high percentage (over 80%) of participants were female and single. Over 79% correctly identified mpox as a viral infection, but only 22.5% demonstrated good knowledge about the disease.

Majority of healthcare workers acknowledged that mpox is a potential worldwide pandemic, with over 90% agreeing it could strain affected countries' healthcare systems. Despite this, healthcare workers generally have moderate to positive attitudes towards mpox, influenced by factors such as age, years of experience, professional qualification, and previous training on the disease.

The study highlights the importance of targeted educational interventions in improving knowledge levels among healthcare professionals, but does not provide guidance on how to implement these interventions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Mpox is a zoonotic viral disease that presents significant public health challenges. Despite the pivotal role of healthcare workers, research on their knowledge and attitudes towards mpox is limited. This study aimed to assess healthcare workers‚Äô knowledge, awareness and attitude regarding mpox as well as associated factors at Rivers State University Teaching Hospital in Nigeria. A descriptive cross-sectional study design was employed, utilizing a 34-item semi-structured questionnaire. Knowledge levels were categorized as good (>‚Äâ70% score), fair (50‚Äì69%), or poor (<‚Äâ50%), while attitudes were classified as positive (>‚Äâ70% score), moderate (50‚Äì69%), or negative (<‚Äâ50%). Statistical analyses included independent sample T-test, One-way Analysis of Variance, and Chi-square tests. Among a total of 227 healthcare professionals, majority were females (59%) and singles (54.2%). Over 79% correctly identified mpox as a viral infection, while 59.9% recognized its potential transmission through a monkey bite. Overall, 22.5% demonstrated good knowledge. Majority (61.7%) acknowledged the potential of mpox as a worldwide pandemic, with 89% agreeing it could strain affected countries‚Äô healthcare systems. Healthcare workers exhibited moderate to positive attitudes towards mpox. Gender (Males, (p=‚Äâ0.003), age (>‚Äâ40 years (p=‚Äâ0.008), years of experience (6‚Äì10 years; (p<‚Äâ0.001), professional qualification (physicians, (p=‚Äâ0.002), and previous mpox training (p<‚Äâ0.001) significantly influenced knowledge levels, but no significant associations were found between demographic variables and attitudes. The study revealed a high level of awareness but low to fair knowledge among most participants. Healthcare professionals also demonstrated moderate to positive attitudes towards mpox. Factors such as gender, age, years of experience, professional qualification, and previous training on mpox influenced knowledge levels, but did not influence attitude in the study participants highlighting the importance of targeted educational interventions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79396-x" target="_blank">https://www.nature.com/articles/s41598-024-79396-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-11</strong> <a href="https://www.nature.com/articles/s41541-024-01016-6" target="_blank">
            ü§ñ Neutralization of SARS-CoV-2 strains KP.1, KP.1.1, KP.2 and KP.3 with human and mouse serum</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-374', this)">
            More</button></h4>
            <div id="toggle-374" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralization of SARS-CoV-2 KP.1, KP.1.1, KP.2 and KP.3 by human and murine sera</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research has found that specific antibodies from a previous coronavirus (SARS-CoV-2) infection can offer increased protection against other related viruses. These findings show that individuals who had an initial COVID-19 infection or received a booster vaccine with the delta and BA.5.2 combination showed better neutralizing antibody responses than those who didn't experience breakthrough infections or were not vaccinated.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We report SARS-CoV-2 KP.1, KP.1.1, KP.2 and KP.3 neutralizing antibody titers. They displayed increased immune evasion compared to JN.1, especially KP.1 and KP.3, for participants who experienced BF.7/BA.5.2 breakthrough infection or received bivalent (delta/BA.5) vaccine boosting. Second XBB sub-variants breakthrough infection enhanced the neutralization responses. HK.3-JN.1 RBD-heterodimer induced balanced and potent neutralizing responses against recently-circulating SARS-CoV-2 sub-variants in mice, supporting to replace the COVID-19 antigen containing JN.1 or its sub-variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01016-6" target="_blank">https://www.nature.com/articles/s41541-024-01016-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-11</strong> <a href="https://www.nature.com/articles/s41599-024-03973-8" target="_blank">
            ü§ñ Development of Resilient Societies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-375', this)">
            More</button></h4>
            <div id="toggle-375" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sustainable Development Goals and wellbeing for resilient societies: shocks and recovery</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The 'decade of action' intended to accomplish 17 Sustainable Development Goals (SDGs) is facing significant challenges due to the COVID-19 pandemic. Our investigation shows that nearly 90% of SDGs were negatively affected, but there's hope for improvement with 66 targets benefiting from crisis-induced changes. To achieve this goal, a comprehensive approach and decisive leadership are necessary to guide an inclusive economic recovery while protecting the environment. The ongoing crises, environmental challenges, and conflicts require a proactive, deliberate, and well-informed strategy to steer the decade toward achieving the SDGs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ‚Äòdecade of action‚Äô intended to accomplish the ambitious 17 Sustainable Development Goals (SDGs) faces notable challenges. Our investigation into the impact of the COVID-19 crisis on SDG progress reveals important lessons for shaping effective policy interventions to ensure resilient societies and overall well-being. Through systematic mapping and a rapid review approach, our analysis reveals that nearly 90% of the SDGs, specifically 144 targets, were adversely affected by the COVID-19 pandemic. Yet, there is a glimmer of opportunity: 66 targets stand to gain from the crisis-induced transformations, provided that the right choices are made. Achieving this goal demands a comprehensive approach and decisive leadership to steer an inclusive economic recovery that also safeguards the environment while safeguarding the environment. The intricate interplay between the ongoing planetary and post-COVID-19 crises, environmental challenges, and conflicts underscores the need for a proactive, deliberate and well-informed approach, marked by collaborative decision-making, which is imperative for effectively steering the ‚Äòdecade of action‚Äô toward achieving the SDGs. These complex challenges demand collective, decisive action, all with the overarching aim of securing a just and sustainable future for all.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-024-03973-8" target="_blank">https://www.nature.com/articles/s41599-024-03973-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-09</strong> <a href="https://www.nature.com/articles/s41598-024-78170-3" target="_blank">
            ü§ñ Epidemic Genetics of COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-376', this)">
            More</button></h4>
            <div id="toggle-376" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Unraveling the protective genetic architecture of COVID-19 in the Brazilian Amazon</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Researchers studied how genetic factors contribute to a person's risk of getting very sick or not at all from COVID-19 despite extensive research during the pandemic. They analyzed data from 124 patients who were categorized into two groups based on their condition and found four genetic variants that may help protect against severe illness. Their study highlights the importance of understanding how genetic differences affect people living in diverse regions, particularly those with rare alleles.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite all the efforts acquired in four years of the COVID-19 pandemic, the path to a full understanding of the biological mechanisms involved in this disease remains complex. This is partly due to a combination of factors, including the inherent characteristics of the infection, socio-environmental elements, and the variations observed within both the viral and the human genomes. Thus, this study aimed to investigate the correlation between genetic host factors and the severity of COVID-19. We conducted whole exome sequencing (WES) of 124 patients, categorized into severe and non-severe groups. From the whole exome sequencing (WES) association analysis, four variants (rs1770731 inCRYBG1, rs7221209 inDNAH17, rs3826295 inDGKE, and rs7913626 inCFAP46) were identified as potentially linked to a protective effect against the clinical severity of COVID-19, which may explain the less severe impact of COVID-19 on the Northern Region. Our findings underscore the importance of carrying out more genomic studies in populations living in the Amazon, one of the most diverse from the point of view of the presence of rare and specific alleles. To our knowledge, this is the first WES study of admixed individuals from the Brazilian Amazon to investigate genomic variants associated with the clinical severity of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78170-3" target="_blank">https://www.nature.com/articles/s41598-024-78170-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-09</strong> <a href="https://www.nature.com/articles/s41598-024-79122-7" target="_blank">
            ü§ñ Liposome-based vaccines that are safe for use in the body.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-377', this)">
            More</button></h4>
            <div id="toggle-377" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity of intraperitoneal and intranasal liposome adjuvanted VLP vaccines against SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Researchers created two types of vaccines against SARS-CoV-2 by using a virus that can't infect cells directly but causes an immune response when it tries to enter them.

The first vaccine, L-SME-VLPs, had a special feature: its protein S is cut at the junction where two parts work together. The second vaccine, L-S‚Ä≤ME-VLPs, didn't have this feature. Both vaccines were harmless and triggered strong immunity in mice when given intraperitoneally (IP) or intranasally.

The immune response in mice was measured by testing their blood for antibodies against SARS-CoV-2. The results showed that the second vaccine, L-S‚Ä≤ME-VLPs, induced stronger antibody production than the first vaccine, L-SME-VLPs.

Most of the vaccinated mice had a higher level of neutralizing (VN) antibodies in their lungs when infected with the virus. VN antibodies are typically of specific types (IgG2a or IgG3). The mice also produced antibodies of other isotypes in their bronchoalveolar lavage fluids, which suggests that they were fighting the virus from different angles.

Further testing of L-S‚Ä≤ME-VLPs through the nasal route is recommended to confirm its effectiveness and safety as a vaccine against SARS-CoV-2.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humans get SARS-CoV-2 infection mainly through inhalation; thus, vaccine that induces protective immunity at the virus entry site is important for early control of the infection. In this study, two anionic liposome (L)-adjuvanted VLP vaccines against SARS-CoV-2 were formulated. Baculovirus-Sf21 insect cell system was used for production of VLPs made of full-length S, M and E proteins. S protein of one vaccine (L-SME-VLPs) contained furin cleavage site at the S1/S2 junction, while that of another vaccine (L-S‚Ä≤ME-VLPs) did not. Both vaccines were innocuous and immunogenic when administered IP and IN to mice. Mice immunized IP with L-SME-VLPs/L-S‚Ä≤ME-VLPs (three doses, two-weeks intervals) had serum virus neutralizing (VN) antibodies (in falling order of isotype frequency): IgG3, IgA and IgG2a/IgG3, IgA and IgM, respectively. The L-S‚Ä≤ME VLPs vaccine induced significantly higher serum VN antibody titers than the L-SME-VLPs vaccine. All mice immunized IN with both vaccines had significant rise of VN antibodies in their bronchoalveolar lavage fluids (BALF). The VN antibodies in 67% of immunized mice were Th1- isotypes (IgG2a and/or IgG2b); the immunized mice had also other antibody isotypes in BALF. The intranasal L-S‚Ä≤ME-VLPs should be tested further step-by-step towards the clinical use as effective and safe vaccine against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-79122-7" target="_blank">https://www.nature.com/articles/s41598-024-79122-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41598-024-78982-3" target="_blank">
            ü§ñ Detection of SARS-CoV-2 in Wastewater as a Predictor of COVID-19 Epidemic Peaks in Venezuela</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-378', this)">
            More</button></h4>
            <div id="toggle-378" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Detection of SARS-CoV-2 in wastewater as an earlier predictor of COVID-19 epidemic peaks in Venezuela</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Wastewater-based epidemiological surveillance was a successful tool for detecting COVID-19 outbreaks in Venezuela by analyzing viral RNA from collected wastewater and comparing it with reported cases of the disease. The method helped identify connections between wastewater samples and reported cases up to six days after collection, suggesting that wastewater surveillance could be an early warning system for SARS-CoV-2 outbreaks in areas where public health monitoring is limited.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Wastewater-based epidemiological surveillance has proven to be a useful and cost-effective tool for detecting COVID-19 outbreaks. Here, our objective was to evaluate its potential as an early warning system in Venezuela by detecting SARS-CoV-2 RNA in wastewater and its correlation with reported cases of COVID-19. Viral RNA was concentrated from wastewater collected at various sites in Caracas (northern Venezuela), from September 2021 to July 2023, using the polyethylene glycol (PEG) precipitation method. Viral quantification was performed by RT-qPCR targeting the N1 and ORF1ab genes. A significant association (p<‚Äâ0.05) was found between viral load in wastewater and reported cases of COVID-19 up to six days after sampling. During the whole study, two populated areas of the city were persistent hotspots of viral infection. The L452R mutation, suggestive of the presence of the Delta variant, was identified in the only sample where a complete genomic sequence could be obtained. Significant differences (p<‚Äâ0.05) between the physicochemical conditions of the wastewater samples positive and negative for the virus were found. Our results support proof of concept that wastewater surveillance can serve as an early warning system for SARS-CoV-2 outbreaks, complementing public health surveillance in those regions where COVID-19 is currently underreported.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78982-3" target="_blank">https://www.nature.com/articles/s41598-024-78982-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41598-024-76827-7" target="_blank">
            ü§ñ Post-COVID Reaction Time Deficits in Virtual Reality Exposure</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-379', this)">
            More</button></h4>
            <div id="toggle-379" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A case-control study of reaction time deficits in a 3D virtual reality in patients with Post-COVID syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A large number of people report Post-COVID symptoms after COVID-19, but there are no objective tests to diagnose them. A study used virtual reality to test how well patients with Post-COVID syndrome (PCS) can see and respond to visual stimuli compared to healthy individuals.

The researchers found that PCS patients performed worse in a virtual reality test of reaction time than healthy people, especially when it came to seeing 3D objects. When controlling for other factors like age and sex, the differences in reaction time were still significant.

Some specific findings include:

- Patients with PCS were slower and more inaccurate at a certain level of visual intensity (disparity)
- The speed and accuracy of patients with PCS improved as they aged
- Women's performance on this task was better than men's

The researchers think that the virtual reality test provided some clues about why people with PCS might be experiencing symptoms. They believe it could lead to new ways of diagnosing and treating PCS.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Following the Coronavirus disease 2019 (COVID-19) pandemic, a large number of people continue to report Post-COVID symptoms (PCS). A wide variety of symptoms are described, including fatigue, post-exertional malaise and cognitive impairment. However, adequate objective diagnostic tests for PCS are not yet available. Since the neurotropism of SARS-CoV-2 could be a possible factor for cognitive impairment, the aim of this study was to clarify whether visual reaction time (RT) in a stereoscopic setting can be a marker in PCS diagnostics. The Virtual-Reality-Oculomotor-Test-System (VR-OTS) was used testing binocular vision in 9 gaze directions via stereoscopic stimuli displayed in a virtual reality (VR)-environment (disparity: 275‚Ä≥, 550‚Ä≥, 1100‚Ä≥) in 179 individuals: 130 patients with PCS and 49 healthy controls. The results from the generalized linear models indicated that both group membership (PCS vs. control) and covariates (age and sex) yielded statistically significant different RT across the models. Accounting for the effect of covariates a statistically significant difference of RT was observed between patients with PCS and controls (disparity 275‚Ä≥ p-value‚Äâ=‚Äâ0.001; 550‚Ä≥ p-value‚Äâ=‚Äâ0.001; 1100‚Ä≥ p-value‚Äâ=‚Äâ0.003). Patients with PCS performed worse in RT in all gaze directions, respectively. Adjusting for the influence of covariates, correct responses (CR) differed significantly between patients with PCS and controls (disparity 275‚Ä≥ p-value‚Äâ<‚Äâ0.001; 550‚Ä≥ p-value‚Äâ=‚Äâ0.003; 1100‚Ä≥ p-value‚Äâ=‚Äâ0.019). Statistically significant effects of covariates on RT were observed for sex (disparity 275‚Ä≥ p-value‚Äâ=‚Äâ0.047; 550‚Ä≥ p-value‚Äâ=‚Äâ0.012; 1100‚Ä≥ p-value‚Äâ=‚Äâ0.005) and age (disparity 275‚Ä≥ p-value‚Äâ<‚Äâ0.001; 550‚Ä≥ p-value‚Äâ<‚Äâ0.001; 1100‚Ä≥ p-value‚Äâ<‚Äâ0.001). However, regarding covariates, no significant effects were found for CR, except for age at disparity 275‚Ä≥ (p-value‚Äâ=‚Äâ0.035). The present data suggested that the mentioned variables uniquely contributed to explain the variation of the response variable (RT, CR). RT and CR detecting 3D-stimuli in a virtual 3D- environment might offer novel functional diagnostic approaches in PCS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76827-7" target="_blank">https://www.nature.com/articles/s41598-024-76827-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41541-024-00964-3" target="_blank">
            ü§ñ ### Studying the Risk of Acute Kidney Injury After COVID Vaccination or Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-380', this)">
            More</button></h4>
            <div id="toggle-380" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Assessing acute kidney injury risk after COVID vaccination and infection in a large cohort study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Acute kidney injury (AKI) after COVID-19 infection affects vaccine risk-benefit evaluations and vaccine hesitancy, but researchers investigated the relationship between COVID-19 vaccination and AKI incidence. A large study tracked over 7 million people's exposure to COVID-19, comparing their likelihood of developing AKI. The results showed that people who got vaccinated had a lower risk of AKI (0.66%) compared to those who contracted the virus (4.88%). The more severe COVID-19 infection was associated with a significantly higher risk of AKI than vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Acute kidney injury (AKI) has been noticed after both COVID-19 vaccination and infection, affecting risk-benefit evaluations and vaccine hesitancy. We conducted a large-scale N3C cohort study to compare AKI incidence following COVID-19 vaccination and infection. Participants from December 2020 to August 2023 were divided into two groups based on their initially observed COVID-19 antigen exposure: COVID-19 vaccination group (n=‚Äâ2,953,219) and COVID-19 infection group (n=‚Äâ3,616,802). AKI was defined by diagnostic codes and serum creatinine changes within a 30‚Äâday follow-up window after exposure. The absolute risk of AKI was 0.66% in the vaccination group versus 4.88% in the infection group. After adjusting for various confounders, COVID-19 infection was associated with a significantly higher risk of AKI than COVID-19 vaccination (aHR‚Äâ=‚Äâ10.31,P<‚Äâ0.001). Our study reveals that COVID-19 vaccination is associated with a significant lower AKI risk compared to COVID-19 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00964-3" target="_blank">https://www.nature.com/articles/s41541-024-00964-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41598-024-78866-6" target="_blank">
            ü§ñ Loss of Resources During the COVID-19 Pandemic

Longitudinal Study on Gambling Habits Among People Experiencing Financial Hardship During the COVID-19 Outbreak.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-381', this)">
            More</button></h4>
            <div id="toggle-381" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Loss of resources and gambling during the COVID-19 pandemic: a three-wave longitudinal study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looks at how the COVID-19 pandemic affected people who gamble in person. It surveyed 585 Polish gamblers before the pandemic to understand their behaviour, preferences, and financial situations. The results showed that online gambling increased significantly during lockdowns and reduced when restrictions eased. Gamblers who engaged more in land-based activities had greater losses due to resource depletion. Those with limited resources were at higher risk of problems.

Heavy gamblers were also more likely to experience issues even if they had stable gambling habits before the pandemic. The study suggests that factors such as frequency and amount of gaming can have a strong impact on financial losses, especially when combined with other stressors like pandemic-related restrictions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study is based on the Conservation of Resources theory and investigates the impact of the COVID-19 pandemic and associated resource loss on gambling behaviour among Polish gamblers. The study surveyed 585 individuals engaged in land-based gambling before the pandemic. Participants completed computer-assisted web interviews, responding to questions regarding land-based and online gambling frequencies, the Problem Gambling Severity Index, and the Inventory of Loss of Resources in Pandemics. The findings revealed significant shifts in gambling behaviour due to pandemic-related restrictions. Land-based gambling declined during lockdowns and the third wave of the study, while online gambling surged as gamblers transitioned from land-based venues. Gamblers tended to return to land-based options as restrictions eased. Over consecutive waves, participants reported decreasing resource loss levels. Significantly, resource loss was influenced by gambling frequency rather than vice versa. Both types of gambling experienced parallel declines at the beginning of the pandemic, which subsided as the new situation became normalised. Players engaging more in gambling experienced more significant resource losses during the pandemic. Those with more resources at the pandemic‚Äôs onset adapted more readily, whereas individuals with limited resources faced resource loss. Conclusions Even with stable gambling levels, heavy gamblers at baseline were at higher risk for issues. The dynamics between resource loss and gambling and problem gambling supported the resource loss spiral concept.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78866-6" target="_blank">https://www.nature.com/articles/s41598-024-78866-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41467-024-53969-w" target="_blank">
            ü§ñ Smart Delivery System for mRNA</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-382', this)">
            More</button></h4>
            <div id="toggle-382" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>mRNA delivery enabled by metal‚Äìorganic nanoparticles</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>mRNA therapeutics are on the verge of revolutionizing disease prevention and treatment, but their current delivery systems face challenges like limited organ tropism and inflammation from cationic components. A new platform aims to overcome these issues by creating nanoparticles that can safely deliver mRNA to various organs in the body. This innovation uses a special type of nanoparticle that is non-cationic and biocompatible, allowing it to easily pass through the body's tissues without causing harm. The researchers screened various combinations of components and found a range of stable, safe, and effective nanoparticles with excellent gene editing capabilities. These nanoscale particles can be administered intravenously and have shown promise in the brain, liver, and kidney for delivering mRNA therapeutics, opening up new possibilities for treating diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>mRNA therapeutics are set to revolutionize disease prevention and treatment, inspiring the development of platforms for safe and effective mRNA delivery. However, current mRNA delivery platforms face some challenges, including limited organ tropism for nonvaccine applications and inflammation induced by cationic nanoparticle components. Herein, we address these challenges through a versatile, noncationic nanoparticle platform whereby mRNA is assembled into a poly(ethylene glycol)-polyphenol network stabilized by metal ions. Screening a range of components and relative compositional ratios affords a library of stable, noncationic, and highly biocompatible metal‚Äìorganic nanoparticles with robust mRNA transfection in vitro and in mice. Intravenous administration of the lead mRNA-containing metal‚Äìorganic nanoparticles enables predominant protein expression and gene editing in the brain, liver, and kidney, while organ tropism is tuned by varying nanoparticle composition. This study opens an avenue for realizing metal‚Äìorganic nanoparticle-enabled mRNA delivery, offering a modular approach to assembling mRNA therapeutics for health applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53969-w" target="_blank">https://www.nature.com/articles/s41467-024-53969-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s42003-024-07017-4" target="_blank">
            ü§ñ Reconstruction of the Human Olfactory Projection System with Ancillary Structures</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-383', this)">
            More</button></h4>
            <div id="toggle-383" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Visualizing the human olfactory projection and ancillary structures in a 3D reconstruction</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

Scientists created a 3D model of how smell is processed in the human brain by scanning a dead body and staining different parts with special dyes. They then looked at pictures of these stained parts to identify six specific structures, which they used to train computers to automatically recognize them on many other sections of the same body part. The computers were able to build an accurate picture of how much each structure is in every part of the body, giving scientists a better understanding of how smell works in humans compared to mice.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Visualizing in 3D the histological microanatomy of the human olfactory projection from the olfactory mucosa in the nasal cavity to the olfactory bulbs in the cranial cavity necessitates a workflow for handling a great many sections. Here, we assembled a 3D reconstruction of a 7.45‚Äâcm3en-blocspecimen extracted from an embalmed human cadaver. A series of 10‚Äâ¬µm coronal sections was stained with quadruple fluorescence histology and scanned in four channels. A trained anatomist manually segmented six structures of interest in a subset of the sections to generate the ground truth. Six convolutional neural networks were then trained for automatic segmentation of these structures in 1234 sections. A high-performance computing solution was engineered to register the sections based on the fluorescence signal and segmented structures. The resulting 3D visualization offers several novel didactic opportunities of interactive exploration and virtual manipulation. By extrapolating manual counts of OSNs in a subset of sections to the calculated volume of the envelope of the entire olfactory epithelium, we computed a total of ~2.7 million OSNs in the specimen. Such empirically derived information helps assess the extent to which the organizational principles of the human olfactory projection may differ from those in mice.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07017-4" target="_blank">https://www.nature.com/articles/s42003-024-07017-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-08</strong> <a href="https://www.nature.com/articles/s41598-024-78651-5" target="_blank">
            ü§ñ Fibromyalgia treatment using a device that sends electric signals to nerves, aiming to relieve symptoms of fibromyalgia and Long COVID.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-384', this)">
            More</button></h4>
            <div id="toggle-384" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Transcutaneous electrical nerve stimulation for fibromyalgia-like syndrome in patients with Long-COVID: a pilot randomized clinical trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

Researchers studied how a device that sends electrical impulses to nerves (TENS) can help people with fibromyalgia-like symptoms such as chronic pain, fatigue, and gait problems.

They gave 25 participants either a high-dose or low-dose version of this TENS device for four weeks. The results showed that the participants who received the higher dose had better symptoms at the end of the study compared to those who didn't get it. They also improved their walking abilities, such as stride time and cadence.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigated the effect of Transcutaneous Electrical Nerve Stimulation (TENS) for fibromyalgia-like symptoms including chronic widespread musculoskeletal pain, fatigue, and/or gait impairment in twenty-five individuals with long-COVID. Participants were randomized to a high dose (intervention group, IG) or low dose (placebo group, PG) TENS device. Both groups received daily 3‚Äì5¬†h of TENS therapy for 4-weeks. The Brief Pain Inventory assessed functional interference from pain (BPI-I), and pain severity (BPI-S). The global fatigue index (GFI) assessed functional interference from fatigue. Wearable technology measured gait parameters during three 30-feet consecutive walking tasks. At 4-weeks, the IG exhibited a greater decrease in BPI-I compared to the PG (mean difference‚Äâ=‚Äâ2.61,p=‚Äâ0.008), and improved in gait parameters including stride time (4-8%, test condition dependent), cadence (4-10%, depending on condition), and double-support phase (12% in dual-task) when compared to baseline. A sub-group meeting the 2010¬†American College of Rheumatology Fibromyalgia diagnostic criteria undergoing high-dose TENS showed GFI improvement at 4-weeks from baseline (mean change‚Äâ=‚Äâ6.08,p=‚Äâ0.005). Daily TENS therapy showed potential in reducing functional interference from pain, fatigue, and gait alterations in long-COVID individuals. The study‚Äôs limited power could affect the confirmation of certain observations. Extending the intervention period may improve treatment effectiveness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78651-5" target="_blank">https://www.nature.com/articles/s41598-024-78651-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-07</strong> <a href="https://www.nature.com/articles/s42003-024-07143-z" target="_blank">
            ü§ñ ## Molecular Strategies Against Coronaviruses

Targeting protein homeostasis with small molecules as a strategy for developing antiviral therapies against coronaviruses.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-385', this)">
            More</button></h4>
            <div id="toggle-385" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Targeting protein homeostasis with small molecules as a strategy for the development of pan-coronavirus antiviral therapies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Researchers have been working on developing new treatments to combat COVID-19 by targeting the virus itself rather than just its symptoms. They've discovered three potential compounds that can stop the coronavirus from replicating inside cells, and these compounds also seem effective against other coronaviruses. The researchers used an existing library of compounds to test them and found promising results in lab tests and animal studies. This work could help advance research into new treatments for COVID-19 and potentially other viral infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has created a global health crisis, with challenges arising from the ongoing evolution of the SARS-CoV-2 virus, the emergence of new strains, and the long-term effects of COVID-19. Aiming to overcome the development of viral resistance, our study here focused on developing broad-spectrum pan-coronavirus antiviral therapies by targeting host protein quality control mechanisms essential for viral replication. Screening an in-house compound library led to the discovery of three candidate compounds targeting cellular proteostasis. The three compounds are (1) the nucleotide analog cordycepin, (2) a benzothiozole analog, and (3) an acyldepsipeptide analog initially developed as part of a campaign to target the mitochondrial ClpP protease. These compounds demonstrated dose-dependent efficacy against multiple coronaviruses, including SARS-CoV-2, effectively inhibiting viral replication in vitro as well as in lung organoids. Notably, the compounds also showed efficacy against SARS-CoV-2 delta and omicron strains. As part of this work, we developed a BSL2-level cell-integrated SARS-CoV-2 replicon, which could serve as a valuable tool for high-throughput screening and studying intracellular viral replication. Our study should aid in the advancement of antiviral drug development efforts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07143-z" target="_blank">https://www.nature.com/articles/s42003-024-07143-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-07</strong> <a href="https://www.nature.com/articles/s41598-024-78496-y" target="_blank">
            ü§ñ Cognitive Function After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-386', this)">
            More</button></h4>
            <div id="toggle-386" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>When fatigue and cognitive impairment persist- a neurological follow-up-study in patients with Post-COVID syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study was conducted to look at how well people who have had COVID-19 can recover from the condition, which affects their mood, sleep quality, and overall well-being. The study looked at the condition's impact on patients 6 months after they first got sick with COVID-2. 

Researchers studied how many of these patients improved over time, what aspects of their lives affected this improvement, and whether people recovered fully or not. They also checked how much fatigue, concentration, and memory were impacted by the condition.

The study found that a significant number of people who had initially felt better for some period after getting sick with COVID-2 continued to experience symptoms over time. People with positive improvements in their lives experienced an increase in quality of life but not in their performance on tests or their energy levels. 

On the other hand, people whose symptoms didn't improve at all experienced a gradual decline in their condition and were unable to return to normal activities after a year or more had COVID-2 gone.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Considering the relevance for patients, economics and public health data about the course of the neurological Post-COVID Syndrome (PCS) are urgently needed. In this study 94 PCS patients (73% female, age in median 49 years) were examined in median 9.4 (T1) and for a second time 14 months (T2) after mild to moderate SARS-CoV-2 infection. Mood, sleep quality and health related quality of life (QoL) were evaluated via structured anamnesis and self-report questionnaires; attention, concentration and memory via psychometric tests. 47% of the patients reported an improvement of their symptoms over time, but only 12% full recovery. 4% noticed deterioration and 49% no change. Main disturbances at both time points were fatigue, deficits in concentration and memory. In patients with perceived improvement QoL significantly increased between T1 and T2, although their test performance as well as the fatigue score remained unchanged. In patients with persisting impairment QoL, fatigue scores and psychometric test results did not change significantly. Abnormal psychometric tests were more frequent at both time points in the group without improvement. But, significant fatigue and cognitive impairment persisted for more than 1 year after SARS-CoV-2 infection in both groups.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78496-y" target="_blank">https://www.nature.com/articles/s41598-024-78496-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-06</strong> <a href="https://www.nature.com/articles/s41598-024-78498-w" target="_blank">
            ü§ñ Predictive Model for COVID-19 Mortality Risk in Chronic Kidney Disease Patients Using Multiple Machine Learning Algorithms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-387', this)">
            More</button></h4>
            <div id="toggle-387" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predictive modeling of COVID-19 mortality risk in chronic kidney disease patients using multiple machine learning algorithms</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new disease called COVID-19 has affected many people worldwide, especially those with chronic kidney disease (CKD). If someone has CKD and is infected with COVID-19, they are more likely to die than others. A team of scientists created a predictive model that helps identify which patients at high risk of death from COVID-19 due to their CKD will live longer if treated properly. The researchers analyzed the data of 219 patients who had COVID-19 and studied how well different variables predicted mortality rates, such as blood tests and other medical information. They found a simple formula that can be used to predict which patients are at high risk of death from COVID-19 due to their CKD.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) has a significant impact on the global population, particularly on individuals with chronic kidney disease (CKD). COVID-19 patients with CKD will face a considerably higher risk of mortality than the general population. This study developed a predictive model for assessing mortality in COVID-19-affected CKD patients, providing personalized risk prediction to optimize clinical management and reduce mortality rates. We developed machine learning algorithms to analyze 219 patients‚Äô clinical laboratory test data retrospectively. The performance of each model was assessed using a calibration curve, decision curve analysis, and receiver operating characteristic (ROC) curve. It was found that the LightGBM model showed the most satisfied performance, with an area under the ROC curve of 0.833, sensitivity of 0.952, and specificity of 0.714. Prealbumin, neutrophil percent, respiratory index in arterial blood, half-saturated pressure of oxygen, carbon dioxide in serum, glucose, neutrophil count, and uric acid were the top 8 significant variables in the prediction model. Validation by 46 patients demonstrated acceptable accuracy. This model can serve as a powerful tool for screening CKD patients at high risk of COVID-19-related mortality and providing decision support for clinical staff, enabling efficient allocation of resources, and facilitating timely and targeted management for those who need the relevant interference urgently.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78498-w" target="_blank">https://www.nature.com/articles/s41598-024-78498-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-06</strong> <a href="https://www.nature.com/articles/s41392-024-02007-8" target="_blank">
            ü§ñ Protein-based vaccination protects against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-388', this)">
            More</button></h4>
            <div id="toggle-388" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A protein vaccine of RBD integrated with immune evasion mutation shows broad protection against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Abstract**

Researchers have discovered a new type of protein that can help protect against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This is because some parts of this protein can escape detection by our immune system, allowing SARS-CoV-2 to spread more easily in vaccinated individuals. The researchers tested three versions of the protein and found they work better than a version that was already known to be effective. These new proteins are promising candidates for making a vaccine against SARS-CoV-2 that can protect against many different variants of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Variants of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continue to emerge and evade immunity, resulting in breakthrough infections in vaccinated populations. There is an urgent need for the development of vaccines with broad protective effects. In this study, we selected hotspot mutations in the receptor-binding domain (RBD) that contribute to immune escape properties and integrated them into the original RBD protein to obtain a complex RBD protein (cRBD), and we found cRBDs have broad protective effects against SARS-CoV-2 variants. Three cRBDs were designed in our study. Compared with the BA.1 RBD protein, the cRBDs induced the production of higher levels of broader-spectrum neutralizing antibodies, suggesting stronger and broader protective efficacy. In viral challenge experiments, cRBDs were more effective than BA.1 RBD in attenuating lung pathologic injury. Among the three constructs, cRBD3 showed optimal broad-spectrum and protective effects and is a promising candidate for a broad-spectrum SARS-CoV-2 vaccine. In conclusion, immunization with cRBDs triggered immunity against a wide range of variants, including those that emerged after we had completed designing the cRBDs. This study preliminarily explores and validates the feasibility of incorporating hotspot mutations that contribute to immune evasion into the RBD to expand the activity spectrum of antigen-induced antibodies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-02007-8" target="_blank">https://www.nature.com/articles/s41392-024-02007-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-06</strong> <a href="https://www.nature.com/articles/s41467-024-53937-4" target="_blank">
            ü§ñ Nanoparticle-Based Vaccines Using Icosahedral DNA Origami</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-389', this)">
            More</button></h4>
            <div id="toggle-389" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rationally designed multimeric nanovaccines using icosahedral DNA origami for display of SARS-CoV-2 receptor binding domain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Researchers have created a new type of vaccine called nanovaccines that use tiny particles made of DNA (a building block of life) to fight off viruses like SARS-CoV-2. However, these nanoparticles are different from the actual virus and their shape and size are not perfectly matched. To improve this, scientists came up with a special way to display viral proteins on the surface of these nanoparticles using tiny pieces of DNA called DNA origami.

By doing so, they can control how many copies of the protein clusters (like patterns) are displayed on each nanoparticle's surface. This allows them to optimize the immune response from B cells and T cells that recognize the virus. The researchers tested their new nanovaccines in mice and found that they were more effective at activating these immune cells than traditional vaccines or antiviral medications.

The results suggest that multivalent (many-antigen) displays of viral proteins like RBD on nanoparticles may be a powerful way to create more effective treatments against future pandemics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multivalent antigen display on nanoparticles can enhance the immunogenicity of nanovaccines targeting viral moieties, such as the receptor binding domain (RBD) of SARS-CoV-2. However, particle morphology and size of current nanovaccines are significantly different from those of SARS-CoV-2. Additionally, surface antigen patterns are not controllable to enable the optimization of B cell activation. Herein, we employ an icosahedral DNA origami (ICO) as a display particle for RBD nanovaccines, achieving morphology and diameter like the virus (91‚Äâ¬±‚Äâ11‚Äânm). The surface addressability of DNA origami permits facile modification of the ICO surface with numerous RBD antigen clusters (ICO-RBD) to form various antigen patterns. Using an in vitro screening system, we demonstrate that the antigen spacing, antigen copies within clusters and cluster number parameters of the surface antigen pattern all impact the ability of the nanovaccines to activate B cells. Importantly, the optimized ICO-RBD nanovaccines evoke stronger and more enduring humoral and T cell immune responses in female mouse models compared to soluble RBD antigens, and the multivalent display broaden the protection range of B cell responses to more mutant strains. Our vaccines activate similar humoral immunity, observable stronger cellular immunity and more memory immune cells compared to trimeric mRNA vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53937-4" target="_blank">https://www.nature.com/articles/s41467-024-53937-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-06</strong> <a href="https://www.nature.com/articles/s41598-024-78518-9" target="_blank">
            ü§ñ Exposure to Carbon Black and Silicon Dioxide Nanoparticles Affects Ocular Surface Corneal Receptor Expression</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-390', this)">
            More</button></h4>
            <div id="toggle-390" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of carbon black and silicon dioxide nanoparticle exposure on corona receptor ACE2 and TMPRSS2 expression in the ocular surface</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has led to a global health crisis. A specific strain called coronavirus causes symptoms like eye problems and is spread by tiny particles in the air we breathe. These particles can affect how our eyes work. Research found that some of these tiny particles can make us sick, while others might even help prevent it.

Tiny particles from traffic emissions can damage the body's receptors on the eyes. A common particle used to study this damage was silica (SiO2), which is often found in sand and car exhaust. Scientists also looked at a type of fiber called carbon black (CB) that comes from old cars and factories.

They tested how these particles affect cells on the inside of our eyes, like corneal and conjunctival cells, as well as nearby tissues. They measured changes in the levels of certain proteins in the bloodstream using special tests. In a separate study, they looked at what happens when tiny particles are applied to the surface of our eyes. This helped them understand how COVID-19 might affect the body's eye health.

The results showed that some types of silica (SiO2) and carbon black (CB) can damage corneal cells more than others. Silica particles were found to increase inflammation in the blood, while carbon black particles increased levels of a protein called tumor necrosis factor-alpha (TNF-Œ±).</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Coronavirus disease 2019 (COVID-19) pandemic has led to a global health crisis, including ocular symptoms, primarily targeting the Angiotensin-Converting Enzyme 2 (ACE2) receptor. PM2.5 air pollution may increase infection risk by altering ACE2 expression. Silicon Dioxide (SiO2) and carbon black (CB), major components of PM2.5 from sands and vehicle emissions, were studied for their effects on ACE2 and Transmembrane Protease Serine 2 (TMPRSS2) expression in corneal and conjunctival cells, and ocular tissues. Human corneal epithelial cells (HCECs) and conjunctival epithelial cells (HCjECs) were exposed to nanoparticles (NPs) for 24 hours, assessing viability via WST-8 assay. TNF-Œ±, IL-6, and IL-1Œ≤ levels in the medium were measured. An in vivo rat study administered NPs via eye drops, with Rose Bengal staining to evaluate damage. ACE2, TMPRSS2, and Angiotensin II (AngII) protein expressions were quantified by Western blot. ACE2 expression in HCjECs increased with NP exposure, while it decreased in HCECs. CB exposure increased TNF-Œ±, IL-6, and IL-1Œ≤ levels in HCECs. In vivo, corneal exposure to CB decreased ACE2 expression, whereas conjunctival exposure to SiO2increased ACE2 expression. These changes suggest that SiO2 exposure may increase the risk of COVID-19 through the ocular surface, while CB exposure may decrease it.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78518-9" target="_blank">https://www.nature.com/articles/s41598-024-78518-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-05</strong> <a href="https://www.nature.com/articles/s41598-024-74404-6" target="_blank">
            ü§ñ Pulmonary function in COVID-19 survivors</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-391', this)">
            More</button></h4>
            <div id="toggle-391" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pulmonary diffusing capacity among individuals recovering from mild to moderate COVID-19: a cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Impaired lung function (DLCO) in people who have had severe COVID-19 has been reported, but most cases of long COVID are not severe enough to cause this impairment. We studied a large group of adults with long COVID recovering from mild to moderate illness and found that 17% had impaired DLCO. Certain factors were associated with this condition, including being female, having significant breathing difficulties (moderate-severe), and high levels of cigarette smoking. A lower BMI was also linked to impaired DLCO. These findings suggest that clinicians should be aware of the possibility of impaired DLCO in patients recovering from mild COVID-19 and use self-reported symptoms, such as severe breathing difficulties, to guide further assessment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Impaired pulmonary diffusing capacity for carbon monoxide (DLCO) following COVID-19 has been consistently reported among individuals recovering from severe-critical infection. However, most long COVID cases follow non-severe COVID-19. We assessed DLCO among individuals with long COVID recovering from mild to moderate acute illness. A cross-sectional study of adults with long COVID, assessed at a COVID recovery clinic‚Äâ>‚Äâ3 months following the onset of acute infection, during 2020‚Äì2021. Participants subjectively ranked their dyspnea severity based on its impact on their daily living and underwent comprehensive pulmonary function testing (PFT). Clinical correlates for impaired DLCO (defined as <‚Äâ80%) were assessed using multivariable logistic regression models. A total of 458 individuals, their mean age 45 (SD 16) and 246 (54%) of whom are women, were evaluated at an average of ~‚Äâ4 months following acute COVID-19. The most frequent PFT impairment was reduced DLCO, identified among 67 (17%) of the cohort. Clinical correlates of impaired DLCO included women (odds ration [OR] 3.64, 95% confidence interval [CI] 1.78‚Äì7.45,p<‚Äâ0.001), cigarette smoking (OR 2.25, 95% CI 1.14‚Äì4.43,p=‚Äâ0.019), and moderate-severe dyspnea (OR 2.77, 95% CI 1.39‚Äì5.50,p=‚Äâ0.004). BMI inversely correlated with DLCO (OR 0.90, 95% CI 0.85‚Äì0.96 per 1 unit,p=‚Äâ0.002). Impaired DLCO was not uncommon among individuals recovering from mild to moderate COVID-19. Women are at a greater risk, and subjective dyspnea correlated with impaired DLCO. Clinicians can rely on self-reported significant dyspnea to guide further assessment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74404-6" target="_blank">https://www.nature.com/articles/s41598-024-74404-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-05</strong> <a href="https://www.nature.com/articles/s41598-024-77331-8" target="_blank">
            ü§ñ Comparison of COVID-19 Vaccine Based on Technology Transfer from Iran</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-392', this)">
            More</button></h4>
            <div id="toggle-392" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative assessment of a COVID-19 vaccine after technology transfer to Iran from critical quality attributes to clinical and immunogenicity aspects</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

During the COVID-19 pandemic, international pharmaceutical companies developed global manufacturing networks for COVID-19 vaccines to share with countries like Iran. This study looked at how two similar vaccine types (Soberana Plus and PastoCovac Plus) were transferred from Cuba to Iran and tested their safety and effectiveness before and after transfer.

The researchers compared the two vaccines side by side, checking things like what was in them, how they worked, and if people getting vaccinated caused any problems. They also checked for differences in things that could make someone get sick again or recover quickly.

After comparing both vaccines, the results showed that they were similar and safe to use, with no significant difference in how well they protected against COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During COVID-19 pandemic, international pharmaceutical companies put effort to build global manufacturing networks for vaccines. Soberana Plus vaccine, a recombinant protein based vaccine (RBD dimer), with the trade name of PastoCovac Plus in Iran, is based on a protein subunit platform in Cuba and completed preclinical and toxicological assessments. This study aimed at presenting the steps of vaccine technology transfer from Cuba to Iran. This study provides the first practical comparability results in Iran to ensure the quality, safety and efficacy of a protein subunit vaccine against COVID-19 after a successful technology transfer from Cuba. PastoCovac Plus was transferred to Iran at the formulation stage. The assessment of the active ingredient pharmaceutical (API) was achieved through physicochemical and clinical data collection and tests to assure if there was any adverse impact on the vaccination results. In order to assess the quality of the vaccine product after technology transfer, we sought different properties including regulatory features, physicochemical quality, vaccine potency and stability as well as its immunogenicity and safety. Following the evaluation of the clinical quality attributes (CQAs) based on the standard protocols, the results showed that the two vaccines are highly similar and comparable, with no considerable effect on safety or efficacy profiles. The CQAs were all in the acceptance limits in terms of safety and efficacy as well as clinical evaluation results. The immunogenicity evaluation also confirmed no significant differences between the vaccines regarding reinfection (P‚Äâ=‚Äâ0.199) or vaccine breakthrough (P‚Äâ=‚Äâ0.176). Furthermore, the level of anti-spike and neutralizing antibodies in the both vaccine groups was not significantly different indicating the equality of performance between the two vaccines. According to the results of the quality and clinical assessment of this study, we achieved an acceptable quality attributes and acceptant limits in terms of safety and efficacy of the vaccines pre and post technology transfer.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77331-8" target="_blank">https://www.nature.com/articles/s41598-024-77331-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-05</strong> <a href="https://www.nature.com/articles/s41598-024-75740-3" target="_blank">
            ü§ñ Personalized Medicine for COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-393', this)">
            More</button></h4>
            <div id="toggle-393" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Data normalization of plasma miRNA profiling from patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>When analyzing coronavirus disease 2019 (COVID-19) with reverse-transcription quantitative polymerase chain reaction (RT-qPCR), it's essential to use a reliable and stable gene as an endogenous reference for microRNA (miRNA) expression. However, no single universally suitable gene exists. In this study, researchers used RNA sequencing techniques to identify the best miRNAs for normalizing miRNA expression in patients with COVID-19 compared to healthy volunteers or those with mild versus severe cases. They found four promising candidates: hsa-miR-34a-3p, hsa-miR-194-3p, hsa-miR-17-3p, and hsa-miR-205-3p. Only one of these, miR-205-3p, met the criteria for being a stable endogenous normalizer in both COVID-19 groups.</p><br>
                <p><strong>Abstract</strong></p>
                <p>When using the reverse-transcription quantitative polymerase chain reaction (RT-qPCR) technique for quantitative assessment of microRNA (miRNA) expression, normalizing data using a stable endogenous gene is essential; however, no universally adequate reference gene exists. Therefore, in this study, we aimed to determine, via the RNA-Seq technique, the most adequate endogenous normalizer for the expression assessment of plasma miRNAs in patients with coronavirus disease 2019 (COVID-19). Two massive sequencing procedures were performed (a) to identify differentially expressed miRNAs between patients with COVID-19 and healthy volunteers (n‚Äâ=12), and (b) to identify differentially expressed miRNAs between patients with severe COVID-19 and those with mild COVID-19 (n‚Äâ=8). The endogenous normalizer candidates were selected according to the following criteria: (1) the miRNA must have a fold regulation‚Äâ=‚Äâ1; (2) the miRNA must have ap-value‚Äâ>‚Äâ0.990; and (3) the miRNAs that were discovered the longest ago should be selected. Four miRNAs (hsa-miR-34a-3p, hsa-miR-194-3p, hsa-miR-17-3p, and hsa-miR-205-3p) met all criteria and were selected for validation by RT-qPCR in a cohort of 125 patients. Of these, only hsa-miR-205-3p was eligible endogenous normalizers in the context of COVID-19 because their expression was stable between the compared groups.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75740-3" target="_blank">https://www.nature.com/articles/s41598-024-75740-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-05</strong> <a href="https://www.nature.com/articles/s44298-024-00061-1" target="_blank">
            ü§ñ Characterization of Direct Contact Transmission Model in Hamsters</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-394', this)">
            More</button></h4>
            <div id="toggle-394" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterization of a SARS-CoV-2 Omicron BA.5 direct-contact transmission model in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 infection in animals has evolved rapidly, making it difficult to find suitable models for studying new treatments. Researchers used Syrian hamsters as the test subjects because they are highly susceptible to SARS-CoV-2 and experience tissue damage similar to human lungs when infected.

To develop a model that would allow scientists to study different interventions against viral variants like Omicron BA.5, researchers set up a direct-contact transmission system between donor and recipient hamsters. They found that using an inoculum of 103 or 104 TCID50 in low-volume drops led to consistent infection and shedding in both male and female hamsters.

To ensure consistent transmission, the researchers chose specific times and durations for co-housing donor and recipient hamsters. They observed that the virus was shed more easily by males than females.

The study also found that males were infected at lower viral loads and experienced limited weight loss compared to females.

By replicating the SARS-CoV-2 Omicron BA.5 infection in hamsters, researchers developed a robust model for studying interventions against new viral variants like this one.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As SARS-CoV-2 continues to evolve antigenically to escape vaccine- or infection-induced immunity, suitable animal models are needed to study novel interventions against viral variants. Syrian hamsters are often used because of their high susceptibility to SARS-CoV-2 and associated tissue damage in the respiratory tract. Here, we established a direct-contact transmission model for SARS-CoV-2 Omicron BA.5 in hamsters. First, we determined whether 103or 104TCID50in a low-volume inoculum led to reproducible infection and viral shedding in male and female hamsters. Next, we determined the optimal co-housing timing and duration between donor and recipient hamsters required for consistent direct-contact transmission. Finally, we compared viral loads and histopathological lesions in the respiratory tissues of donor and recipient hamsters. Intranasal inoculation of hamsters with 103TCID50and 104TCID50Omicron BA.5 in 10‚Äâ¬µl per nostril led to reproducible infection. Viral loads in the throat measured by RT-qPCR were comparable between male and female hamsters. Notably, the shedding of infectious virus was significantly higher in male hamsters. Compared to SARS-CoV-2 D614G, Omicron BA.5 infection reached lower viral loads, had a delayed peak of virus replication, and induced limited body weight loss. To ensure consistent direct-contact transmission from inoculated donor hamsters to na√Øve recipients, a co-housing duration of 24‚Äâh starting 20‚Äâh post-infection of the donors was optimal. We detected mild inflammation in the respiratory tract of donor and recipient hamsters, and viral loads were higher and peaked earlier in donor hamsters compared to recipient hamsters. Taken together, we developed a robust Omicron BA.5 direct-contact transmission model in hamsters, that provides a valuable tool to study novel interventions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00061-1" target="_blank">https://www.nature.com/articles/s44298-024-00061-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-77558-5" target="_blank">
            ü§ñ Heart Rate Variability Measurement in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-395', this)">
            More</button></h4>
            <div id="toggle-395" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inter- and intra-examiner reliability of short-term measurement of heart rate variability on rest in patients hospitalized with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Measures reflecting cardiac sympathovagal activity show excellent consistency between observers, indicating high inter-examiner reliability for HRV parameters in patients diagnosed with coronavirus disease 2019 (COVID-19). These measurements can be used to assess the condition of patients in hospitals, providing accurate and reliable information.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Measures reflecting cardiac sympathovagal activity, particularly those associated with heart rate variability (HRV), are widely recognized and utilized in both scientific and clinical contexts. This study aimed to assess the inter- and intra-examiner reliability of short-term HRV parameters in patients hospitalized with coronavirus disease 2019 (COVID-19). A total of 103 patients (both sexes) diagnosed with COVID-19 were included in the study. HRV was analyzed using both linear and nonlinear methods. Reliability was evaluated through intraclass correlation coefficient (ICC2.1), minimum detectable change (MDC), standard error of measurement (SEM), and coefficient of variation (CV). According to Fleiss‚Äô criteria, excellent reliability was demonstrated, with ICC values ranging from 0.970 to 0.999 for Examiner 1, and from 0.956 to 0.999, for Examiner 2. In the inter-examiner analysis, the ICCs of HRV parameters ranged from 0.972 to 0.999. SEM values for intra-examiner reliability for Examiner 1 ranged from 0.02 to 5.64, with MDC values from 0.05 to 15.64, and CV (%) from 0.28 to 8.04. For Examiner 2, SEM values ranged from 0.02 to 8.18, MDC values from 0.05 to 22.68, and CV (%) from 0.24 to 8.14. For inter-examiner reliability, SEM values ranged from 0.02 to 6.17, MDC from 0.06 to 17.11, and CV (%) from 0.34 to 9.81. Across all analyses, CVs for HRV parameters remained below 10%. Considering different time points and different examiners, short-term resting HRV measurements in patients hospitalized with COVID-19, as evaluated using a portable heart rate device, exhibit high reliability.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77558-5" target="_blank">https://www.nature.com/articles/s41598-024-77558-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-77258-0" target="_blank">
            ü§ñ Mesenchymal Stem Cell-Derived Extracellular Vesicles Reduce Inflammatory Responses in SARS-CoV-2 and Influenza Viruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-396', this)">
            More</button></h4>
            <div id="toggle-396" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mesenchymal stem cell-derived extracellular vesicles reduce inflammatory responses to SARS-CoV-2 and Influenza viral proteins via miR-146a/NF-Œ∫B pathway</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The risk of severe disease from two viruses, such as SARS-CoV-2 and influenza, poses a concern for global public health each year. Researchers have found that tiny particles called extracellular vesicles (EVs) derived from stem cells can help reduce inflammation caused by viral infections.

These EVs are particularly useful because they contain anti-inflammatory proteins that can calm down the immune system's response to an infection. However, more research is needed to understand how these EVs work in combination with viruses like SARS-CoV-2 and influenza.

In one part of the study, scientists found that lung cells were more likely to die or not grow properly when they were infected by both viruses simultaneously. They also discovered a type of protein called miR-146a that helps regulate inflammation in these cells. When this protein is present in EVs from stem cells, it can help counteract the effects of inflammation and promote healing.

The researchers found that the microRNA (miR-146a) plays a key role in suppressing inflammation by targeting specific parts of a signaling pathway called NF-Œ∫B. This pathway helps fight off infection but also causes inflammation when it's active too long.

The study also discovered that people with type 2 diabetes may have trouble using these EVs to treat their condition because they contain less miR-146a and can't effectively target the NF-Œ∫B pathway, leading to more inflammation.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The risk of severe disease caused by co-infection with SARS-CoV-2 and influenza virus (IAV) raises an annual concern for global public health. Extracellular vesicles (EV) derived from mesenchymal stem cells (MSC) possess anti-inflammatory properties that can attenuate the inflammatory cytokine levels induced by viral infection. However, the effects of MSC-EV treatment on SARS-CoV-2 and IAV co-infection have not been elucidated. In the present study, we co-induced lung epithelial cells (EpiC) with SARS-CoV-2 Spike protein (S) and H1N1 influenza viral HA protein (HA) and found robust upregulation of inflammatory cytokines in comparison to those induced by either S or HA protein. Consequently, treatment of lung endothelial cells (EC) with conditioned medium from EpiC co-induced by both S and HA proteins resulted in increased apoptosis and impaired angiogenic ability, suggesting the effects of co-induction on epithelial-endothelial crosstalk. In addition, lung EpiC co-induced by both S and HA proteins showed paracrine effects on the recruitment of immune cells, including monocytes, macrophages and neutrophils. Of Note, EV derived from Wharton Jelly‚Äôs MSC (WJ-EV) transferred miR-146a to recipient lung EpiC, which impaired TRAF6 and IRAK1, resulting in the downregulation of NF-Œ∫B pathway and secretion of inflammatory cytokines, rescuing the epithelial-endothelial crosstalk, and reducing the elevation of immune cell recruitment. Moreover, the anti-inflammatory properties of WJ-EV are affected by type 2 Diabetes Mellitus. WJ-EV derived from donors with type 2 Diabetes Mellitus contained less miR-146a and showed impaired ability to downregulate the NF-Œ∫B pathway and inflammatory cytokines in recipient cells. Taken together, our findings demonstrate the role of miR-146a in targeting the NF-Œ∫B pathway in the anti-inflammatory abilities of WJ-EV, which is a promising strategy to rescue the epithelial-endothelial crosstalk altered by co-infection with SARS-CoV-2 and IAV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77258-0" target="_blank">https://www.nature.com/articles/s41598-024-77258-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-78017-x" target="_blank">
            ü§ñ Study on Presence of COVID-19 Self-Reported Symptoms 6 Months After Hospital Discharge</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-397', this)">
            More</button></h4>
            <div id="toggle-397" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Presence of COVID-19 self-reported symptoms at 12 months in patients discharged from hospital in 2020‚Äì2021: a Spanish cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study investigated the long-term effects of COVID-19 on patients who had been admitted to hospitals due to the virus. It looked at which symptoms occurred one year after hospital discharge and found that several groups experienced common symptoms, including neurological, respiratory, and general problems. Certain factors were also linked to experiencing these symptoms, such as being female, obese, anxious or depressed, having fibromyalgia/chronic fatigue, advanced age, or a history of certain health conditions like anxiety or fibromyalgia.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The long-term effects of SARS-CoV-2 infection, and their determinants, are still unknown. This study aimed to assess symptoms one year after admission for COVID-19, according to the organ/system involved, and to identify factors. Cross-sectional study with retrospective data collection from March 2020 to February 2021. Inclusion criteria: aged‚Äâ‚â•‚Äâ18 years and admitted for COVID-19. Exclusion criteria: death, not localized, refusal to participate, cognitive impairment or language barrier. A telephone survey was conducted on long COVID-related symptoms one year after hospital discharge.n=‚Äâ486. The most frequent symptom groups were neurological (n=‚Äâ225; 46.3%) and respiratory (n=‚Äâ201; 41.4%). Multivariable analysis showed that a history of anxiety was significantly associated with psychiatric symptoms (ORa‚Äâ=‚Äâ2.04, 95%CI‚Äâ=‚Äâ1.02‚Äì4.06), fibromyalgia/chronic fatigue with general symptoms (ORa‚Äâ=‚Äâ11.59, 95%CI‚Äâ=‚Äâ1.47‚Äì9.34) and obesity with respiratory (ORa 1.90, 95%CI‚Äâ=‚Äâ1.27‚Äì2.83) and musculoskeletal symptoms (ORa 1.96, 95%CI‚Äâ=‚Äâ1.30‚Äì2.96). Male sex was associated with a significantly lower risk of neurological (ORa 0.64, 95%CI‚Äâ=‚Äâ0.44‚Äì0.93), respiratory (ORa 0.45, 95%CI‚Äâ=‚Äâ0.31‚Äì0.67), general (ORa 0.43, 95%CI‚Äâ=‚Äâ0.29‚Äì0.63), psychiatric (ORa 0.34, 95%CI‚Äâ=‚Äâ0.22‚Äì0.51), musculoskeletal (ORa 0.47, 95%CI‚Äâ=‚Äâ0.32‚Äì0.70), dermatological (ORa 0.24, 95%CI‚Äâ=‚Äâ0.14‚Äì0.42) and digestive (ORa 0.38, 95%CI‚Äâ=‚Äâ0.20‚Äì0.73) symptoms. Advanced age (‚â•‚Äâ71 years) also had a protective effect against general (ORa 0.60, 95%CI‚Äâ=‚Äâ0.39‚Äì0.95), psychiatric (ORa 0.39, 95%CI‚Äâ=‚Äâ0.23‚Äì0.64), and dermatological (ORa 0.47, 95%CI‚Äâ=‚Äâ0.24‚Äì0.92) symptoms. Patients admitted for SARS-CoV-2 infection frequently experience symptoms at one year, especially neurological and respiratory symptoms. Female sex, obesity, a history of anxiety and fibromyalgia/chronic fatigue were independent risk factors for presenting symptoms. Advanced age acted as a protective factor.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78017-x" target="_blank">https://www.nature.com/articles/s41598-024-78017-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s42003-024-07048-x" target="_blank">
            ü§ñ ### Immunogenic Peptide Recognition by Memory T Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-398', this)">
            More</button></h4>
            <div id="toggle-398" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Identification of virus epitopes and reactive T-cell receptors from memory T cells without peptide synthesis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Identifying specific regions of viral proteins in immune cells is important for understanding how the body responds to infection. This study developed a new method for identifying these regions from memory T cells, reducing the time and cost involved in this process. They used a combination of techniques, including using yeast that display peptides on its surface, to quickly find which regions of viral proteins are most immunogenic. The study focused on SARS-CoV-2 and found that many people's immune systems produce antibodies against certain parts of the virus. This suggests that these antibodies could help protect against future infections with different strains of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Identifying epitopes and their corresponding T-cell receptor (TCR) sequences is crucial in the face of rapidly mutating viruses. Peptide synthesis is often required to confirm the exact epitope sequences, which is time-consuming and expensive. In this study, we introduce a scalable workflow to identify the exact sequences of virus epitopes and reactive TCRs targeting the epitopes from memory T cells. Following the narrowing down of epitopes to specific regions via the tandem minigene (TMG) system, our workflow incorporates the utilization of peptide-major histocompatibility complex-displaying yeasts (pMHC-displaying yeasts) to rapidly screen immunogenic epitopes‚Äô precise sequences, obviating the necessity for the chemical synthesis of peptides. Focusing on SARS-CoV-2, we identify the precise sequences of reactive TCRs, targeting conserved epitopes across the Coronaviridae family, from the blood of COVID-19-recovered individuals over 8 months. Notably, we reveal that at least 75% (6/8) of the tested donors harbor T cells targeting a shared epitope, KTFPPTEPK, derived from the N protein. Furthermore, several identified TCRs exhibit cross-reactivity to mutant epitopes, suggesting a potential mechanism for sustained T-cell responses against emerging SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07048-x" target="_blank">https://www.nature.com/articles/s42003-024-07048-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-78540-x" target="_blank">
            ü§ñ HIV and Monkeypox Co-Infections: Genetic Analysis Using Bioinformatics and Machine Learning.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-399', this)">
            More</button></h4>
            <div id="toggle-399" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Exploration of common pathogenesis and candidate hub genes between HIV and monkeypox co-infection using bioinformatics and machine learning</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new strain of a virus called monkeypox has been detected in people who have contracted HIV. This study looked at how these two viruses interact with each other's genetic material and how this affects their ability to cause disease. It also identified genes that are essential for the survival of both viruses. The researchers used powerful computer tools to analyze data from DNA samples of monkeypox and HIV, and then built a model of which genes were involved in the process. They found seven genes that play important roles in preventing the spread of both viruses: MX2, ADAR, POLR2H, RPL5, IFIT2, RPS5, and IFT16. These genes are also linked to how people's bodies respond to viral infections.

Using these findings, researchers identified potential new treatments for HIV and monkeypox co-infection. They recommend that medications called nucleotide reverse transcriptase inhibitors (AZT) and mefloquine be used to treat both viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study explored the pathogenesis of human immunodeficiency virus (HIV) and monkeypox co-infection, identifying candidate hub genes and potential drugs using bioinformatics and machine learning. Datasets for HIV (GSE 37250) and monkeypox (GSE 24125) were obtained from the GEO database. Common differentially expressed genes (DEGs) in co-infection were identified by intersecting DEGs from monkeypox datasets with genes from key HIV modules screened using Weighted Gene Co-Expression Network Analysis (WGCNA). After gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis and construction of protein-protein interaction (PPI) network, candidate hub genes were further screened based on machine learning algorithms. Transcriptional factors (TFs) and miRNA-candidate hub gene networks were constructed to understand regulatory mechanisms and protein-drug interactions to identify potential therapeutic drugs. Seven candidate hub genes‚ÄîMX2,ADAR,POLR2H,RPL5,IFI16,IFIT2, andRPS5‚Äîwere identified. TFs and miRNAs associated with these hub genes, playing a key role in regulating viral infection and inflammation due to the activation of antiviral innate immunity, were also identified through network analysis. Potential therapeutic drugs were screened based on these hub genes: AZT, a nucleotide reverse transcriptase inhibitor, suppressed viral replication in HIV and monkeypox co-infection, while mefloquine inhibited inflammation due to the activation of antiviral innate immunity. In conclusion, the study identified candidate hub genes, their transcriptional regulation, signaling pathways, and small-molecule drugs in HIV and monkeypox co-infection, contributing to understanding the pathogenesis of HIV and monkeypox co-infection and informing precise therapeutic strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78540-x" target="_blank">https://www.nature.com/articles/s41598-024-78540-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-78162-3" target="_blank">
            ü§ñ Guided Breathing Exercises for Anxiety, Stress, and Depression in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-400', this)">
            More</button></h4>
            <div id="toggle-400" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical effectiveness of guided breathing exercises in reducing anxiety, stress, and depression in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has caused increased anxiety, stress, and depression among those affected. A study examined the effectiveness of guided breathing exercises as a complementary therapy for reducing these symptoms. The participants were randomly assigned to either a guided breathing group or a control group. The researchers used a type of sampling called simple random sampling. Participants completed a questionnaire before and after the intervention to assess changes in anxiety, stress, and depression levels. 

The study found that guided breathing exercises significantly reduced anxiety and stress scores but had no effect on depression scores. This suggests that these exercises may be an effective way to manage psychological distress among COVID-19 patients who do not respond to other treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has led to an increase in the prevalence of anxiety, stress and depression among affected people. This study was conducted with the aim of investigating the clinical effectiveness of guided breathing exercises in reducing anxiety, stress and depression in patients with COVID-19. A quasi-experimental study design was used, involving a sample of COVID-19 patients who underwent guided breathing exercises as a complementary therapy. After simple sampling, eligible subjects were randomly divided into two groups: intervention (30 patients) and control (30 patients) using random block method. The Depression Anxiety Stress Scale-21 (DASS-21) questionnaire was administered before and after the intervention to evaluate changes in anxiety, stress, and depression levels. The results of this study demonstrated that clinically guided breathing exercises had a significant effect on reducing anxiety and stress in COVID-19 patients. The intervention significantly reduced anxiety and stress scores (p<‚Äâ0.001). However, there was no significant reduction in depression scores among patients who participated in guided breathing exercises (p=‚Äâ0.946). Guided breathing exercises are an effective complementary technique in reducing the level of anxiety and stress in COVID-19 patients. Moreover, the exercises may provide a worthy non-pharmacological approach to managing psychological distress in COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78162-3" target="_blank">https://www.nature.com/articles/s41598-024-78162-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-04</strong> <a href="https://www.nature.com/articles/s41598-024-76806-y" target="_blank">
            ü§ñ Hydrodynamic behavior of artificial droplets in stadiums</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-401', this)">
            More</button></h4>
            <div id="toggle-401" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Wind velocity and dispersion/advection‚Äìdiffusion of artificial droplets and droplet nuclei in a domed all-weather multi-purpose stadium</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To evaluate the risk of COVID-19 transmission through artificial droplets and particles at mass-gathering events, researchers measured how far these particles could travel and spread through the air to reach nearby people. They also found that wearing masks reduced the amount of infected particles on hair, neck, and seats in front of a person's face. However, not all mask types were equally effective - some still allowed droplet nuclei near the source to enter nearby areas. The researchers concluded that COVID-19 transmission was low at the Tokyo Dome due to the effectiveness of wearing masks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the COVID-19 infection risk and the effectiveness of countermeasures at mass-gathering events, we measured the dispersion and advective diffusion of artificial droplets and artificial droplet nuclei at the Tokyo Dome, Japan (capacity 55,000 people). We also measured and evaluated the effectiveness of wearing masks and increasing the space between seating areas. If people were seated facing forward, artificial droplets did not reach the mouths of surrounding people, suggesting low risk of droplet transmission. For an artificially generated cough or sneeze, the volume of droplets deposited on the hair, back of the neck, and back of the human in front, and the backs of the seats in front, decreased by two to three orders of magnitude when a mask was worn, regardless of the type of mask. However, when the mask was worn with the nose out, the amount deposited on the back of the seat in front was reduced by only 17%. Even in seats with the highest particle concentration in the vicinity of the source, only 0.097%‚Äì0.24% of the generated droplet nuclei (1.0‚Äì3.0¬†Œºm) from the source were inhaled. Our results suggest that the infection risk at the Tokyo Dome via droplet and airborne transmission was low.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76806-y" target="_blank">https://www.nature.com/articles/s41598-024-76806-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-03</strong> <a href="https://www.nature.com/articles/s41598-024-76925-6" target="_blank">
            ü§ñ Airport wastewater surveillance for COVID-19 detection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-402', this)">
            More</button></h4>
            <div id="toggle-402" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genomic surveillance of Canadian airport wastewater samples allows early detection of emerging SARS-CoV-2 lineages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new study has found that wastewater (WW) can help track the spread of SARS-CoV-2, including emerging variants of concern (VOCs). Researchers analyzed WW from Toronto Pearson International Airport (a major hub in Canada) and compared it with local WW samples and Ontario clinical data. The results showed that VOCs emerge earlier than expected, often several weeks or even months before they are detected in the local population.

The study suggests that WW surveillance at transitory transportation hubs like airports can be an effective way to track viral lineages and identify emerging threats before they affect people outside of a specific area. This approach could help with pandemic preparedness and provide early detection of new variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic has shown wastewater (WW) surveillance to be an effective means of tracking the emergence of viral lineages which arrive by many routes of transmission including via transportation hubs. In the Canadian province of Ontario, numerous municipal wastewater treatment plants (WWTPs) participate in WW surveillance of infectious disease targets such as SARS-CoV-2 by qPCR and whole genome sequencing (WGS). The Greater Toronto Airports Authority (GTAA), operator of Toronto Pearson International Airport (Toronto Pearson), has been participating in WW surveillance since January 2022. As a major international airport in Canada and the largest national hub, this airport is an ideal location for tracking globally emerging SARS-CoV-2 variants of concern (VOCs). In this study, WW collected from Toronto Pearson‚Äôs two terminals and pooled aircraft sewage was processed for WGS using a tiled-amplicon approach targeting the SARS-CoV-2 virus genome. Data generated was analyzed to monitor trends of SARS-CoV-2 lineage frequencies. Initial detections of emerging lineages were compared between Toronto Pearson WW samples, municipal WW samples collected from the surrounding regions, and Ontario clinical data as published by Public Health Ontario. Results enabled the early detection of VOCs and individual mutations emerging in Ontario. On average, the emergence of novel lineages at the airport preceded clinical detections by 1‚Äì4 weeks, and up to 16 weeks in one case. This project illustrates the efficacy of WW surveillance at transitory transportation hubs and sets an example that could be applied to other viruses as part of a pandemic preparedness strategy and to provide monitoring on a mass scale.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76925-6" target="_blank">https://www.nature.com/articles/s41598-024-76925-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-02</strong> <a href="https://www.nature.com/articles/s41598-024-77664-4" target="_blank">
            ü§ñ Oncolytic Virus Therapy Approach</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-403', this)">
            More</button></h4>
            <div id="toggle-403" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An in silico approach uncovering the competency of oncolytic human adenovirus 52 for targeted breast cancer virotherapy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Breast cancer is a major health threat that affects women above 30 years old, but it also affects men. It occurs when abnormal cell division leads to breast malignancies. Various factors contribute to its occurrence, including age, family history, genetic mutations in BRCA1 and BRCA2 genes, and hormonal imbalances.

Early detection through regular self-examinations, mammograms, and clinical assessments can prevent the disease. Appropriate diagnosis is also crucial for effective treatment.

The conventional treatment methods can have severe side effects, so researchers are looking for alternative treatments that are more secure and effective. One promising approach is using oncolytic viruses to target and kill cancer cells while minimizing harm to healthy tissues. These viruses work by selectively targeting malignant cells with a high degree of precision and efficiency.

Researchers want to validate the effectiveness of Human Adenovirus 52 in treating breast cancer through computational methods, including in-silico predictions that match experimental findings. They also aim to investigate its potential as a therapeutic agent for breast cancer treatment.

The study focuses on understanding the molecular structure of Breast Cancer Type 1 Susceptibility Protein (BRCA1) and Human Adenovirus 52 proteins, determining their active residues, and assessing their interactions with each other. The results have shown that Human Adenovirus 52 is a potent oncolytic virus that can target and eliminate cancer cells while causing minimal harm to healthy tissues.

This research provides evidence for the effectiveness of Human Adenovirus 52 as a potential treatment for breast cancer, with significant potential in targeting cancer cells through virotherapy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Breast cancer remains a major health threat throughout the world specifically in women above 30 years of age however, it is rarely known to affect men as well. It is characterized by the abnormal division of cells in the breast tissue resulting in the development of breast malignancies. Various risk factors contributing to breast cancer include age, family history, genetic mutations (chiefly in BRCA1 and BRCA2 genes) along with hormonal imbalances (oestrogen, progesterone, HER2). Early detection which can be obtained through frequent rounds of self-examination, mammographic scanning, and clinical assessment plays a crucial role in the prevention of the disease. In addition, appropriate diagnosis assists in better therapeutic responses. This study highlights the considerable health risks associated with the conventional treatment procedures which arise and increased demand of advanced, secure, and risk-free treatment alternatives. Oncolytic viruses are potentially apparent for the aim of improving cancer therapeutics with reduced side effects. These viruses act as the fundamental therapeutic agent themselves that selectively target and kill malignant cells without harm to healthy tissues. The key objective of the research is to provide evidence that Human Adenovirus 52 is a potent oncolytic virus and to highlight its capacity to target and eliminate cancer cells with precision while causing the least amount of harm to healthy tissues. Validating the in-silico method entails evaluating the precision and dependability of the computational modelling by contrasting the in-silico predictions with the findings from the experiments rank as the secondary objective. The workflow of this research utilized in-silico computational drug designing approaches including retrieval of tertiary structures of both the target Breast Cancer Type 1 Susceptibility Protein (BRCA1) and the viral Human Adenovirus 52 protein, their validation generating Ramachandran Plots determining favoured amino acid residue angles and prediction of their active residues. Furthermore, the study focused on the molecular dynamics docking of proteins, interpretation of molecular interactions between the docked complex, as well as the assessment of the molecular dynamic simulations (MD) in addition to their MMGBSA binding energy calculations. A successful docking between BRCA1 and Adenovirus protein provided a significant score of 329.2 +/- 24.3, furthermore, MD simulations showed a high RMSD peak at 2.8 √Ö, RMSF were maximum at 3.5 √Ö with highest protein‚Äìprotein interaction, the radius of gyration was stable throughout the simulation representing elastic stability along with a high energy interaction value of - 7882 kCal/mol. Moreover, the MMGBSA calculation results showed a notable release of binding free energy of - 68.96 kCal/mol demonstrating effective bond formation between the docked complex. These findings propose the effectiveness of Human Adenovirus 52 to treat cancer. The selected oncolytic Human Adenovirus 52 is a potential candidate for the target specific treatment of breast cancer through virotherapy. This computer-aided drug discovery presents significant potential in targeting cancer cells and would assist in the development of potent drug reagents for the cancer therapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77664-4" target="_blank">https://www.nature.com/articles/s41598-024-77664-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-02</strong> <a href="https://www.nature.com/articles/s41467-024-53914-x" target="_blank">
            ü§ñ Improved Respiratory Protection via Inhaled mRNA Delivery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-404', this)">
            More</button></h4>
            <div id="toggle-404" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Charge-assisted stabilization of lipid nanoparticles enables inhaled mRNA delivery for mucosal vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers have developed a new approach for delivering messenger RNA (mRNA) using tiny particles that can be easily taken into cells. This method is promising for treating lung diseases or serving as a vaccine against certain viruses. To improve its delivery, scientists added special charges to these tiny particles that help them stay stable in air. The result is a more effective way of delivering mRNA to the lungs and triggering an immune response.

I removed quotation marks and focused on making it clear and concise while retaining the original meaning. I also condensed some sentences for brevity and clarity.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Inhaled delivery of messenger RNA (mRNA) using lipid nanoparticle (LNP) holds immense promise for treating pulmonary diseases or serving as a mucosal vaccine. However, the unsatisfactory delivery efficacy caused by the disintegration and aggregation of LNP during nebulization represents a major obstacle. To address this, we develop a charge-assisted stabilization (CAS) strategy aimed at inducing electrostatic repulsions among LNPs to enhance their colloidal stability. By optimizing the surface charges using a peptide-lipid conjugate, the leading CAS-LNP demonstrates exceptional stability during nebulization, resulting in efficient pulmonary mRNA delivery in mouse, dog, and pig. Inhaled CAS-LNP primarily transfect dendritic cells, triggering robust mucosal and systemic immune responses. We demonstrate the efficacy of inhaled CAS-LNP as a vaccine for SARS-CoV-2 Omicron variant and as a cancer vaccine to inhibit lung metastasis. Our findings illustrate the design principles of nebulized LNPs, paving the way of developing inhaled mRNA vaccines and therapeutics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53914-x" target="_blank">https://www.nature.com/articles/s41467-024-53914-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-02</strong> <a href="https://www.nature.com/articles/s41467-024-53791-4" target="_blank">
            ü§ñ Mucociliary Clearance in COVID-19 Spread</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-405', this)">
            More</button></h4>
            <div id="toggle-405" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Live imaging of airway epithelium reveals that mucociliary clearance modulates SARS-CoV-2 spread</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

A virus called SARS-CoV-2 infects and multiplies in human lungs, particularly in the airways that connect to the nose and mouth. When this happens, it can cause the infected cells to produce excess mucus, which can trap the virus inside. But after the initial infection is established, the mucus that's produced may start to break down, making it harder for the virus to spread from cell to cell. This process is known as mucociliary clearance (MCC). We studied how MCC affects SARS-CoV-2 spread in human lungs and found that MCC plays a crucial role in determining whether the virus can spread further after infection has started. Our research also shows that even when MCC is impaired, it's not possible for the virus to continue spreading without mucus being produced again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 initiates infection in the conducting airways, where mucociliary clearance inhibits pathogen penetration. However, it is unclear how mucociliary clearance impacts SARS-CoV-2 spread after infection is established. To investigate viral spread at this site, we perform live imaging of SARS-CoV-2 infected differentiated primary human bronchial epithelium cultures for up to 12 days. Using a fluorescent reporter virus and markers for cilia and mucus, we longitudinally monitor mucus motion, ciliary motion, and infection. Infected cell numbers peak at 4 days post infection, forming characteristic foci that tracked mucus movement. Inhibition of MCC using physical and genetic perturbations limits foci. Later in infection, mucociliary clearance deteriorates. Increased mucus secretion accompanies ciliary motion defects, but mucociliary clearance and vectorial infection spread resume after mucus removal, suggesting that mucus secretion may mediate MCC deterioration. Our work shows that while MCC can facilitate SARS-CoV-2 spread after initial infection, subsequent MCC decreases inhibit spread, revealing a complex interplay between SARS-CoV-2 and MCC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53791-4" target="_blank">https://www.nature.com/articles/s41467-024-53791-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-02</strong> <a href="https://www.nature.com/articles/s41598-024-78119-6" target="_blank">
            ü§ñ Prevention of SARS-CoV-2 Infection in Solid Organ Transplant Recipients Without Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-406', this)">
            More</button></h4>
            <div id="toggle-406" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk factors for SARS-CoV-2 infection and severe COVID-19 in unvaccinated solid organ transplant recipients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The role of immunosuppressive therapy on the risk of contracting SARS-CoV-2 and the severity of COVID-19 is unclear in patients with solid organ transplants who have not been vaccinated. A study analyzed data from 1957 patients between July 2020 and April 2021 to see if this was true. The results showed that immunosuppressive therapy had no effect on the risk of contracting SARS-CoV-2, but certain factors such as diabetes, chronic lung disease, and exposure to a COVID-19 positive individual were associated with an increased risk of severe COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The role of immunosuppressive therapy on SARS-CoV-2 infection risk and COVID-19 severity remains unclear in unvaccinated solid organ transplant recipients. We included 1957 organ transplant recipients between July 2020 and April 2021 to analyze whether baseline immunosuppressive therapy and other risk factors are associated with SARS-CoV-2 infection and severe COVID-19. In total, 247 (12.6%) had SARS-CoV-2 (defined as positive nasopharyngeal swab and/or positive antibody titer). Of these, 57 (23.1%) had severe COVID-19, defined as oxygen supplementation, intensive care unit admission or death. Multivariable analysis identified diabetes (hazard ratio (HR) 1.39 (95% confidence interval (CI) 1.05‚Äì1.83)), chronic lung disease (HR 1.71 (95% CI 1.13‚Äì2.60)) and contact with a COVID-19 positive individual (HR 3.61 (95% CI 2.61‚Äì4.99) as independent risk factors for SARS-CoV-2 infection. There was no association between immunosuppressive therapy and infection risk. Severe COVID-19 was multivariably associated with hypertension (OR 5.45 (95% CI 1.66‚Äì17.84)), chronic kidney disease (OR 3.55 (95% CI 1.75‚Äì7.19)), corticosteroid use (OR 2.93 (95% CI 1.03‚Äì2.55)) and having a COVID-19 positive housemate (OR 6.77 (95% CI 2.65‚Äì17.28)). In conclusion, baseline corticosteroid use, but no other immunosuppressive agent, is independently associated with severe COVID-19 in unvaccinated SOT recipients after correction for hypertension, chronic kidney disease, housemates affected by COVID-19 and transplant type.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-78119-6" target="_blank">https://www.nature.com/articles/s41598-024-78119-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-11-01</strong> <a href="https://www.nature.com/articles/s41598-024-76773-4" target="_blank">
            ü§ñ A wastewater treatment plant study using a metagenomics approach for the city of Agadir in Morocco.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-407', this)">
            More</button></h4>
            <div id="toggle-407" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Resistome analysis of wastewater treatment plants in Agadir city, Morocco, using a metagenomics approach</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Water scarcity has become a significant global issue due to untreated or improperly managed wastewater. In this study, researchers assessed the presence of Antimicrobial Resistance Genes (ARGs) in six wastewater treatment plants in Morocco. They collected samples from infrequent and effluent waters during February and July 2020. 223 ARGs were identified, highlighting resistance to antibiotics such as aminoglycoside, macrolides, beta-lactamases, chloramphenicol, sulfonamides, tetracyclines, and other drugs. The researchers discovered that mobile genetic elements (MGEs) are often associated with the presence of ARGs in wastewater treatment plants. While many methods to reduce contaminated water exist, some ARGs persist due to lack of effective treatment methods.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Water scarcity has evolved into a pressing global issue, significantly impacting numerous regions worldwide. The use of treated wastewater stands out as a promising solution to this problem. However, the proliferation of various contaminants, primarily Antimicrobial Resistance Genes (ARGs), poses a significant challenge to its safe and sustainable use. In this study, we assessed the composition and abundance of 373 ARGs, corresponding to 31 different classes of antibiotics, in six wastewater treatment plants (WWTP) in Agadir city of Morocco. Influent and effluent samples were collected during the months of February and July in 2020, in addition to samples from the Atlantic Ocean. In total, 223 ARGs were uncovered, highlighting in particular resistance to aminoglycoside, macrolide lincosamide, beta-lactamase, chloramphenicol, sulfonamide, tetracycline, and other antibiotics. The mechanisms of action of these ARGs were mainly antibiotic inactivation, antibiotic target alteration, efflux pump and cellular protection. Mobile genetic elements (MGEs) were detected at high levels their co-occurrence with ARGs highlights their involvement in the acquisition and transmission of ARGs in microbial communities through horizontal gene transfer. While many wastewater treatment methods effectively reduce a large proportion of gene material and pathogens, a substantial fraction of ARGs and other contaminants persist in treated wastewater. This persistence poses potential risks to both human health and the environment, warranting the need of more effective treatment strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76773-4" target="_blank">https://www.nature.com/articles/s41598-024-76773-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-31</strong> <a href="https://www.nature.com/articles/s41598-024-75216-4" target="_blank">
            ü§ñ Improved Drug Dosage
Simplified Drug Analysis Method Using Green Colorimetric Assay</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-408', this)">
            More</button></h4>
            <div id="toggle-408" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A sustainable and green HPLC-PDA technique for the simultaneous estimation of Post-COVID-19 syndrome co-administered drugs with greenness and whiteness assessment</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>COVID-19 is a global health concern that has been growing over the past four years. The virus can cause several syndromes, including post-COVID syndrome, which can have residual symptoms like muscle pain and fatigue even after recovery. To manage these symptoms, doctors often prescribe medication such as paracetamol, dexketoprofen trometamol, and rivaroxaban. A new method was developed to analyze the concentration of these drugs in plasma samples, ensuring accurate and reliable results.

The method uses a green analytical technique that is environmentally friendly and produces minimal waste. It can detect tiny amounts of the three drugs, down to 0.047 micrograms per milliliter, and has high accuracy rates. The development process was validated according to international standards, making it safe for routine use in medical settings.

The new method also has a greenness profile that indicates its environmental safety by using non-toxic solvents and minimizing waste during the analysis time. This approach aligns with current best practices in analytical chemistry and sets a new standard for environmentally friendly drug development methods.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 has been a growing global concern in the past four years. Several syndromes associated with this multi-organ viral infection have been observed since the outbreak. According to estimates, 10‚Äì15% of people with SARS-CoV- infection may have post-COVID-19 syndrome. Even months after infection, common residual signs and symptoms include myalgia, exhaustion, shortness of breath, rapid heartbeat, stroke, and memory and cognitive impairment which can negatively affect survivors‚Äô quality of life and may consequently lead to their death. Therefore, it is necessary to think about potential therapy options for dealing with both short and long-term impacts. Paracetamol (a common analgesic and antipyretic) and Dexketoprofen Trometamol (a non-steroidal anti-inflammatory drug) are used together to relieve post-COVID symptoms like myalgia (muscle pain) and headache. Additionally, to prevent thrombotic events, Rivaroxaban is recommended for 35 days following discharge. Thus an eco-friendly HPLC-DAD technique was developed for simultaneous quantification of Paracetamol, Dexketoprofen Trometamol, and Rivaroxaban which are co-administered for treatment of post-COVID-19 syndrome. The suggested method was found to be linear in the concentration ranges of 3.00‚Äì45.00¬†¬µg/mL, 0.5‚Äì50.00¬†¬µg/mL, and 0.15‚Äì20.00¬†¬µg/mL, and a limit of detection down to 0.531¬†¬µg/mL, 0.095¬†¬µg/mL and 0.047¬†¬µg/mL for Paracetamol, Dexketoprofen Trometamol and Rivaroxaban, respectively. This method was effectively used to quantify the studied drugs in their bulk powder and spiked human plasma with high percentage recoveries (96.55‚Äì99.46%). The suggested approach was validated per International Conference on Harmonization (ICH) requirements and found to be within the acceptable ranges. The method was developed using Green Analytical Chemistry (GAC) principles, with the solvents used and run time having a significant effect on the method‚Äôs greenness. ‚ÄúNon-toxic‚Äù ethanol served as the organic modifier in the mobile phase, moreover, the total run time was 12¬†min making it suitable for the routine analysis of the mentioned drugs in plasma samples. To get a full image of the method‚Äôs greenness profile; two most recent greenness assessment tools, the Green Analytical Procedure Index (GAPI), and the Analytical GREEnness metric (AGREE), were employed, with White Analytical Chemistry (WAC) principles proving its environmental safety.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75216-4" target="_blank">https://www.nature.com/articles/s41598-024-75216-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-31</strong> <a href="https://www.nature.com/articles/s41541-024-01003-x" target="_blank">
            ü§ñ COVID-19 Vaccine Trial Results</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-409', this)">
            More</button></h4>
            <div id="toggle-409" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Safety, efficacy and immunogenicity of aerosolized Ad5-nCoV COVID-19 vaccine in a non-inferiority randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This phase 3, observer-blinded trial studied the safety, effectiveness, and side effects of two different versions of a COVID-19 vaccine in people with mild to moderate symptoms who had already received three doses. The goal was to compare their ability to prevent severe illness caused by various strains of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This phase 3, observer-blinded, non-inferiority randomized trial (ClinicalTrials.gov: NCT05517642), conducted from September 2022 to May 2023 at three Malaysian sites, involved 540 adults previously vaccinated with three COVID-19 doses. Participants were randomized 1:1 to receive either one dose of inhaled Recombinant COVID-19 Vaccine (Ad5-nCoV-IH) or intramuscular tozinameran (BNT-IM). The study assessed safety, vaccine efficacy (VE) and immunogenicity against SARS-CoV-2 variants. The primary outcome was the non-inferiority of anti-spike protein receptor-binding domain (S-RBD IgG) antibodies, with a 97.5% confidence interval lower limit for the geometric mean concentration (GMC) ratio >0.67. Ad5-nCoV-IH showed lower immunogenicity than BNT-IM, with a GMC ratio of 0.22 and a seroconversion rate difference of -71.91%. Adverse drug reactions (ADRs) were less frequent with Ad5-nCoV-IH (39.26%) compared to BNT-IM (64.68%). No serious vaccine-related adverse events were reported. Both vaccines had comparable efficacy against COVID-19 variants. This study was funded by Tianjin Biomedical Science and Technology Major Project.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-01003-x" target="_blank">https://www.nature.com/articles/s41541-024-01003-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-30</strong> <a href="https://www.nature.com/articles/s41541-024-00992-z" target="_blank">
            ü§ñ Single-Cycle SARS-CoV-2 Vaccine Shows High Immunity Against Transmission</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-410', this)">
            More</button></h4>
            <div id="toggle-410" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>High protection and transmission-blocking immunity elicited by single-cycle SARS-CoV-2 vaccine in hamsters</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Vaccines have been effective in fighting the COVID-19 pandemic, but new variants are making it harder for them to work. A team has created "single-cycle infection" vaccines that can protect against these new viruses without causing harm. These vaccines were tested on hamsters and showed excellent protection against a challenge virus. They don't trigger inflammation or lung damage in animals that receive them. This means the vaccine could provide powerful protection against COVID-19, which is still being found in human cells.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccines have played a central role in combating the COVID-19 pandemic, but newly emerging SARS-CoV-2 variants are increasingly evading first-generation vaccine protection. To address this challenge, we designed ‚Äúsingle-cycle infection SARS-CoV-2 viruses‚Äù (SCVs) that lack essential viral genes, possess distinctive immune-modulatory features, and exhibit an excellent safety profile in the Syrian hamster model. Animals intranasally vaccinated with an Envelope-gene-deleted vaccine candidate were fully protected against an autologous challenge with the SARS-CoV-2 virus through systemic and mucosal humoral immune responses. Additionally, the deletion of immune-downregulating viral genes in the vaccine construct prevented challenge virus transmission to contact animals. Moreover, vaccinated animals displayed neither tissue inflammation nor lung damage. Consequently, SCVs hold promising potential to induce potent protection against COVID-19, surpassing the immunity conferred by natural infection, as demonstrated in human immune cells.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00992-z" target="_blank">https://www.nature.com/articles/s41541-024-00992-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-30</strong> <a href="https://www.nature.com/articles/s41564-024-01838-z" target="_blank">
            ü§ñ Cardiac symptoms following COVID-19

Cardiomyopathy triggered by low-grade inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-411', this)">
            More</button></h4>
            <div id="toggle-411" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-acute sequelae of SARS-CoV-2 cardiovascular symptoms are associated with trace-level cytokines that affect cardiomyocyte function</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract:

A significant number of people around the world have COVID-19 complications, which can cause heart problems like chest pain and shortness of breath. Scientists studied how these complications affect the body's response to infection. They found that people with severe COVID-19 had higher levels of certain "bad" proteins in their blood that trigger inflammation. This study has identified a link between chronic inflammation and symptoms of PASC-CVS (post-acute sequelae of COVID-19 cardiovascular syndrome).</p><br>
                <p><strong>Abstract</strong></p>
                <p>An estimated 65 million people globally suffer from post-acute sequelae of COVID-19 (PASC), with many experiencing cardiovascular symptoms (PASC-CVS) like chest pain and heart palpitations. This study examines the role of chronic inflammation in PASC-CVS, particularly in individuals with symptoms persisting over a year after infection. Blood samples from three groups‚Äîrecovered individuals, those with prolonged PASC-CVS and SARS-CoV-2-negative individuals‚Äîrevealed that those with PASC-CVS had a blood signature linked to inflammation. Trace-level pro-inflammatory cytokines were detected in the plasma from donors with PASC-CVS 18‚Äâmonths post infection using nanotechnology. Importantly, these trace-level cytokines affected the function of primary human cardiomyocytes. Plasma proteomics also demonstrated higher levels of complement and coagulation proteins in the plasma from patients with PASC-CVS. This study highlights chronic inflammation‚Äôs role in the symptoms of PASC-CVS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41564-024-01838-z" target="_blank">https://www.nature.com/articles/s41564-024-01838-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-30</strong> <a href="https://www.nature.com/articles/s41598-024-77405-7" target="_blank">
            ü§ñ Pulmonary Fibrosis Following Severe COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-412', this)">
            More</button></h4>
            <div id="toggle-412" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Early pulmonary fibrosis-like changes between delta and pre-delta periods in patients with severe COVID-19 pneumonia on mechanical ventilation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Studies compared the severity of pulmonary fibrosis in two periods of COVID-19 infection for critically ill patients.

*   64% were vaccinated, while 120% received mechanical ventilation.
*   Fibrosis was found in 75.3% of patients over a median time frame of 51 days after enrollment.
*   The delta group had more severe fibrosis (‚â•2) and reticulation and architectural distortion (+/-) compared to the pre-delta group.
*   Even after adjusting for clinical variables, patients with SARS-CoV-2 infection in the delta period experienced more severe pulmonary fibrosis.</p><br>
                <p><strong>Abstract</strong></p>
                <p>It remains unclear whether pulmonary fibrosis-like changes differ in patients with different SARS-CoV-2 variants. This study aimed to compare pulmonary fibrotic changes between two SARS-CoV-2 variant periods (delta vs. pre-delta) in critically ill patients with SARS-CoV-2 pneumonia. Clinical data and chest CT images of patients with SARS-CoV-2 pneumonia receiving mechanical ventilation were collected from 10 hospitals in South Korea over two periods: delta (July-December, 2021;n=‚Äâ64) and pre-delta (February, 2020-June, 2021;n=‚Äâ120). Fibrotic changes on chest CT were evaluated through visual assessment. Of 184 patients, the mean age was 64.6 years, and 60.5% were ale. Fibrosis-like changes on chest CT (median 51 days from enrollment to follow up CT scan, interquartile range 27‚Äì76 days) were identified in 75.3%. Delta group showed more fibrosis-like changes (‚â•‚Äâ2) (69.8% vs. 43.1%,P=‚Äâ0.001) and more frequent reticulation and architectural distortion+/-parenchymal band than pre-delta group. Even after propensity score matching with clinical variables, delta group had more severe (‚â•‚Äâ2) fibrosis-like changes (71.4% vs. 38.8%,P=‚Äâ0.001), and more frequent reticulation and architectural distortion+/-parenchymal band than pre-delta group. Our data suggest that critically ill patients with SARS-CoV-2 in delta period had more severe pulmonary fibrosis-like changes than those in pre-delta period.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77405-7" target="_blank">https://www.nature.com/articles/s41598-024-77405-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-30</strong> <a href="https://www.nature.com/articles/s42003-024-07107-3" target="_blank">
            ü§ñ Spatial Hierarchical Network Learning Framework for Drug Repositioning</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-413', this)">
            More</button></h4>
            <div id="toggle-413" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A spatial hierarchical network learning framework for drug repositioning allowing interpretation from macro to micro scale</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Biomedical network learning is an emerging field that could accelerate drug discovery by combining molecular structure information with biological associations at multiple scales. However, traditional networks struggle to capture key properties and complex data in biomedical networks. To address these challenges, we propose the Spatial Hierarchical Network (SpHN), which integrates spatial hierarchical information into a unified model. SpHN-VDA is an end-to-end framework that enhances machine understanding of molecular function and improves virus-drug association identification. Our research demonstrates that SpHN-VDA outperforms leading models across multiple datasets, including those with high levels of variability. We also discover potential candidate drugs through gene expression analysis and show their effectiveness in docking experiments against SARS-CoV-2 spike protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Biomedical network learning offers fresh prospects for expediting drug repositioning. However, traditional network architectures struggle to quantify the relationship between micro-scale drug spatial structures and corresponding macro-scale biomedical networks, limiting their ability to capture key pharmacological properties and complex biomedical information crucial for drug screening and therapeutic discovery. Moreover, challenges such as difficulty in capturing long-range dependencies hinder current network-based approaches. To address these limitations, we introduce the Spatial Hierarchical Network, modeling molecular 3D structures and biological associations into a unified network. We propose an end-to-end framework, SpHN-VDA, integrating spatial hierarchical information through triple attention mechanisms to enhance machine understanding of molecular functionality and improve the accuracy of virus-drug association identification. SpHN-VDA outperforms leading models across three datasets, particularly excelling in out-of-distribution and cold-start scenarios. It also exhibits enhanced robustness against data perturbation, ranging from 20% to 40%. It accurately identifies critical motifs for binding sites, even without protein residue annotations. Leveraging reliability of SpHN-VDA, we have identified 25 potential candidate drugs through gene expression analysis and CMap. Molecular docking experiments with the SARS-CoV-2 spike protein further corroborate the predictions. This research highlights the broad potential of SpHN-VDA to enhance drug repositioning and identify effective treatments for various diseases.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07107-3" target="_blank">https://www.nature.com/articles/s42003-024-07107-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-30</strong> <a href="https://www.nature.com/articles/s41467-024-53792-3" target="_blank">
            ü§ñ The innate immune system plays a crucial role in controlling the influenza virus and allowing the body to recover from infection. A protein called IFITM3 helps regulate this process by controlling the spread of the virus within the body.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-414', this)">
            More</button></h4>
            <div id="toggle-414" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Innate immune control of influenza virus interspecies adaptation via IFITM3</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Pandemics Are Caused By Viruses That Can Be Found In Animals But Attack Humans

When certain viruses infect animals that can also infect humans, it's called a "zoonotic" infection. This means the virus can jump from one species to another and start causing disease in humans. Researchers have found that Interferon-Induced Transmembrane Protein 3 (IFITM3) plays a role in controlling how well some viruses can infect human cells, especially those caused by avian influenza.

### What IFITM3 Does

IFITM3 helps to stop certain types of viruses from replicating inside human cells. When people are infected with low doses of these viruses, the IFITM3 helps prevent them from getting too many copies of the virus in their cells.

### What Happens To IFITM3-Deficient Mice And Humans

Research has shown that mice and humans lacking IFITM3 can't be infected by certain types of avian influenza viruses that normally infect other animals. Additionally, if these mice or humans are passed through different versions of the virus over time, they become more adapted to it, which is unusual since many types of the virus don't adapt well in this way.

### What This Means For The Future

The discovery of IFITM3's role in controlling viral infections suggests that people with deficiencies in the protein may be more vulnerable to emerging new pandemic viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Influenza virus pandemics are caused by viruses from animal reservoirs that adapt to efficiently infect and replicate in human hosts. Here, we investigate whether Interferon-Induced Transmembrane Protein 3 (IFITM3), a host antiviral factor with known human deficiencies, plays a role in interspecies virus infection and adaptation. We find that IFITM3-deficient mice and human cells can be infected with low doses of avian influenza viruses that fail to infect WT counterparts, identifying a new role for IFITM3 in controlling the minimum infectious virus dose threshold. Remarkably, influenza viruses passaged throughIfitm3‚àí/‚àímice exhibit enhanced host adaptation, a result that is distinct from viruses passaged in mice deficient for interferon signaling, which exhibit attenuation. Our data demonstrate that IFITM3 deficiency uniquely facilitates potentially zoonotic influenza virus infections and subsequent adaptation, implicating IFITM3 deficiencies in the human population as a vulnerability for emergence of new pandemic viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53792-3" target="_blank">https://www.nature.com/articles/s41467-024-53792-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-29</strong> <a href="https://www.nature.com/articles/s41598-024-77149-4" target="_blank">
            ü§ñ Improved Oxygen Delivery Through Cardiac Output During Hypoxia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-415', this)">
            More</button></h4>
            <div id="toggle-415" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Chest compression quality decreases in hypoxic conditions simulating an airliner cabin at cruising altitude: a randomized, controlled, double-blind Manikin Study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Air travelers are expected to reach 4 billion by 2024, with a significant number experiencing acute medical problems while flying, including cardiac arrests requiring cardiopulmonary resuscitation (CPR) and hypoxia impairing caregivers' ability to perform CPR.

A study was conducted to test whether hypoxia affects the effectiveness of performing CPR. Participants were randomly assigned to either normoxic (oxygen-rich air) or hypoxic conditions with or without supplemental oxygen for 6 hours. The results showed that:

* Hypoxia reduced the number of compressions performed with correct depth, even after prolonged exposure.
* Supplemental oxygen during CPR in hypoxic conditions improved compression depth and prevented further reduction in depth compared to normoxic conditions.

These findings suggest that extended hypoxia can reduce the quality of chest compressions during CPR, highlighting the importance of using supplemental oxygen for healthcare providers.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Air traveler numbers are predicted to reach 4.0 billion in 2024. Between 1/15,000‚Äì50,000 passengers will experience acute medical problems inflight with cardiac arrests requiring cardiopulmonary resuscitation (CPR) accounting for 0.3% of medical emergencies. Hypoxia in airplane cabins could impair oxygenation and physical performance of caregivers. We conducted a randomized controlled, double-blind study to test the hypothesis that hypoxia decreases the effectiveness in performing CPR. We randomized 24 healthcare professionals to two different study arms, each consisting of two conditions: arm (1) ‚Äòhypoxia (FiO215%, equivalent to 2400 m altitude)‚Äô versus ‚Äònormoxia‚Äô; arm (2) ‚Äòhypoxia‚Äâ+‚Äâsupplemental oxygen‚Äô versus ‚Äònormoxia‚Äâ+‚Äâsupplemental oxygen‚Äô. The order of conditions was counterbalanced and a minimum wash-out period of 24¬†h was granted between conditions. In each condition participants performed a 5-min cardiac compression only CPR (CCO-CPR) using a full-body manikin after one, three and six hours in an altitude chamber. Mixed ANOVAs with post-hoc false-discovery-rate adjusted pairwise comparisons indicated that although compression frequency was maintained, the number of compressions with correct depth was decreased at all times during hypoxia compared to normoxia (allp<‚Äâ0.002). After 6¬†h hypoxia exposure, mean compression depth was below the recommended compression depth defined by ERC/AHA guidelines and reduced compared to normoxia (42.4‚Äâ¬±‚Äâ12.6¬†mm vs. 54.6‚Äâ¬±‚Äâ4.3¬†mm,p<‚Äâ0.0001). Supplemental oxygen during CCO-CPR in hypoxia prevented the decrease of compression-depth (55.3‚Äâ¬±‚Äâ3¬†mm). Extended hypoxia exposure akin to conditions in airplane cabins can reduce quality of chest compressions during CPR. Supplemental oxygen for healthcare providers is an effective countermeasure.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77149-4" target="_blank">https://www.nature.com/articles/s41598-024-77149-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-29</strong> <a href="https://www.nature.com/articles/s41467-024-53566-x" target="_blank">
            ü§ñ Pandemic Infections and Immune Responses to COVID-19 Secondary Bacterial Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-416', this)">
            More</button></h4>
            <div id="toggle-416" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Microbial dynamics and pulmonary immune responses in COVID-19 secondary bacterial pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new type of pneumonia called secondary bacterial pneumonia (2¬∞BP) affects people who have had a respiratory virus, such as the Covid-19 illness. A research study investigated how this happens and its effects on people's health. The researchers looked at 112 adults who were critically ill with Covid-19 and compared their airway microbiome (a type of bacteria in the air) to healthy people without the illness. They found that 2¬∞BP was linked to higher mortality rates, severe treatment use, and certain characteristics of the bacterial microbiome. The study also showed how some treatments may influence the immune system's ability to fight off infections. Overall, the findings provide new insights into the causes and effects of 2¬∞BP in Covid-19 patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Secondary bacterial pneumonia (2¬∞BP) is associated with significant morbidity following respiratory viral infection, yet remains incompletely understood. In a prospective cohort of 112 critically ill adults intubated for COVID-19, we comparatively assess longitudinal airway microbiome dynamics and the pulmonary transcriptome of patients who developed 2¬∞BP versus controls who did not. We find that 2¬∞BP is significantly associated with both mortality and corticosteroid treatment. The pulmonary microbiome in 2¬∞BP is characterized by increased bacterial RNA mass and dominance of culture-confirmed pathogens, detectable days prior to 2¬∞BP clinical diagnosis, and frequently also present in nasal swabs. Assessment of the pulmonary transcriptome reveals suppressed TNFŒ± signaling in patients with 2¬∞BP, and sensitivity analyses suggest this finding is mediated by corticosteroid treatment. Further, we find that increased bacterial RNA mass correlates with reduced expression of innate and adaptive immunity genes in both 2¬∞BP patients and controls. Taken together, our findings provide fresh insights into the microbial dynamics and host immune features of COVID-19-associated 2¬∞BP, and suggest that suppressed immune signaling, potentially mediated by corticosteroid treatment, permits expansion of opportunistic bacterial pathogens.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53566-x" target="_blank">https://www.nature.com/articles/s41467-024-53566-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-29</strong> <a href="https://www.nature.com/articles/s41598-024-76472-0" target="_blank">
            ü§ñ COVID-19 Treatment for Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-417', this)">
            More</button></h4>
            <div id="toggle-417" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nirmatrelvir plus ritonavir reduces COVID-19 hospitalization and prevents long COVID in adult outpatients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Nirmatrelvir plus ritonavir has been approved for treating mild to moderate COVID-19 in high-risk patients with a certain level of effectiveness against the Omicron variant. The treatment reduced hospitalizations due to COVID-19 by 39% and also helped reduce long-term symptoms by 42%.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Nirmatrelvir plus ritonavir received Emergency Use Authorization for treating mild to moderate COVID-19 in high-risk patients. Its efficacy against the Omicron variant of SARS-CoV-2 remains uncertain. This retrospective cohort study assessed the effect of nirmatrelvir‚Äìritonavir in preventing severe disease progression and long COVID symptoms after acute COVID-19 in non-hospitalized adults. SALAMA medical records from Dubai‚Äôs COVID-19 healthcare centers between May 22, 2022, and April 30, 2023, were used to identify 7290 eligible patients, 9.6% of whom received nirmatrelvir‚Äìritonavir. Treatment was associated with a notable reduction in COVID-19-related hospitalizations (adjusted hazard ratio [HR] of 0.39; 95% CI, 0.18‚Äì0.85) by day 28 of symptom onset. Moreover, nirmatrelvir‚Äìritonavir was associated with fewer long COVID symptoms (adjusted HR of 0.42; 95% CI, 0.19‚Äì0.95). This suggests the significant effectiveness of nirmatrelvir‚Äìritonavir against the Omicron variant, reducing both severe and long-term COVID-19 symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76472-0" target="_blank">https://www.nature.com/articles/s41598-024-76472-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-28</strong> <a href="https://www.nature.com/articles/s41541-024-00984-z" target="_blank">
            ü§ñ MRNA COVID-19 Vax Boosters Not Linked to Raynaud's Syndrome (RVO) Symptoms in Initial Studies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-418', this)">
            More</button></h4>
            <div id="toggle-418" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reply to: mRNA COVID-19 vaccinations are not associated with RVO development 21 days and 12 weeks after vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified abstract:

A recent study compared findings with another one on whether people are more likely to develop a rare eye problem called retinal vascular occlusion (a blockage in blood vessels) after being vaccinated against certain diseases. Our research found that people who were vaccinated had a higher risk of this eye problem lasting up to two years, and we also looked at how different vaccines affect the body.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We appreciate your interest in our study. This study contrasts with Dorney et al.‚Äòs focus on the first 21 days. We conducted a re-analysis regarding selection bias and found consistent results. Our research highlights the elevated risk of retinal vascular occlusion after vaccination, which can last up to two years. We also discussed the differences in study design and the effects of different vaccine brands.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00984-z" target="_blank">https://www.nature.com/articles/s41541-024-00984-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-28</strong> <a href="https://www.nature.com/articles/s41598-024-74429-x" target="_blank">
            ü§ñ The Impact of COVID-19 on Older Adults in Europe</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-419', this)">
            More</button></h4>
            <div id="toggle-419" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of the COVID-19 pandemic on the well-being of middle-aged and older Europeans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic had a negative impact on people's overall well-being, regardless of age or sex. The pandemic affected both middle-aged and older adults differently, but the main effect was seen in those who were infected by the virus, particularly those with Long COVID symptoms. This finding highlights the need to provide support to these groups at community levels.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has been associated with a general decline in well-being. However, there is limited evidence on the effect of the pandemic on the general population, and especially among the ageing population. We assessed the overall impact of the pandemic on the well-being of middle-aged and older adults residing in 27 European countries, focusing on the time-period before summer 2021. We used a sample of 46,209 respondents from the two population-based longitudinal Corona Surveys collected during summer 2020 and summer 2021. To test our hypotheses, we used latent change score models. All analyses were stratified by sex. The COVID-19 pandemic affected middle-aged and older Europeans‚Äô well-being irrespective of their sex. Being infected by the COVID-19 virus at the start of the pandemic had a negative impact on well-being. As expected, adults with Long COVID experienced the most pronounced decline in well-being. A novel finding was the decline in the level of well-being among adults not infected by the COVID-19 virus. Support should be provided at community levels with specific attention towards individuals with Long COVID symptoms and those infected with COVID-19 at earlier stages of the pandemic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74429-x" target="_blank">https://www.nature.com/articles/s41598-024-74429-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-28</strong> <a href="https://www.nature.com/articles/s41598-024-76630-4" target="_blank">
            ü§ñ Assessment of Cognitive Dysfunction in PostCOVID-19 Patients Based on Serum Levels of Biomarkers for Neuroinflammation and Lung Fibrosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-420', this)">
            More</button></h4>
            <div id="toggle-420" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical usefulness of the serum levels of neuroinflammatory and lung fibrosis biomarkers in the assessment of cognitive dysfunction in post-COVID19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A growing body of evidence suggests there is a significant increase in cognitive dysfunction following COVID-19 infection. However, current diagnostic tools do not accurately predict prognosis or help stratify the disease. This study aimed to evaluate the usefulness of biomarkers (YKL-40 and MR-pro-ADM) for predicting severe progression of COVID-19 in patients with post-COVID syndrome and cognitive problems. The results show that:

* Patients with higher YKL-40 levels had more severe conditions.
* YKL-40 differentiated between those with post-COVID syndrome and those without.
* Patients treated with steroids showed a significant decrease in YKL-40 levels after 6 months.

These findings suggest that YKL-40 could be used as an indicator of successful treatment for COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A growing body of evidence indicates there is an increasing incidence of cognitive dysfunction in patients after coronavirus disease 2019 (COVID-19) infection. However, still lack diagnostic tools, which allow us to predict prognosis in such cases and improve the stratification of the disease. This study aims to evaluate the usefulness of the biomarkers that could allow to predict the severity and progression of COVID-19 in patients with post-COVID syndrome and cognitive problems. Data regarding clinical history, pre-existing conditions, chest CT scan, and therapy (remdesivir, steroids) were acquired. A total of 44 patients with hospitalized COVID-19, and healthy controls were enrolled in the investigation, and serum blood was obtained. After 6 months of observations, patients with COVID-19 were divided into two groups: first - without post-COVID syndrome and memory complaints, and second - with post-COVID and cognitive problems. Measurements of YKL-40 and MR-pro-ADM were taken in the serum with enzyme immunoassay kits at the time of admission (visit 1) and 6 months after discharge from the hospital (visit 2). Significantly higher concentrations of YKL-40 were found in patients with COVID-19 as compared to healthy individuals (p=‚Äâ0.016). Moreover, YKL-40 ratio allowed to differentiate patients with and without post-COVID syndrome (median: 0.94 vs. 1.55,p=‚Äâ0.004). Additionally, COVID-19 patients with dyspnea presented significantly elevated levels of MR-pro-ADM as compared to the group of COVID-19 survivors without dyspnea (p=‚Äâ0.015). In the group of patients without post-COVID syndrome, the concentrations of YKL-40 and MR-pro-ADM decreased after treatment as compared to levels before therapy (77 vs. 36 ng/ml and 607 vs. 456 pmol/L). However, in patients with post-COVID syndrome and cognitive problems, the levels of both markers did not alter 6 months after hospital discharge in comparison to basal levels. Furthermore, after dexamethasone treatment the YKL-40 concentrations declined significantly (p=‚Äâ0.003) in patients with COVID-19. This study demonstrated the predictive usefulness of YKL-40 as an indicator of successful treatment in patients with COVID-19 infection allowing risk stratification of hospitalized patients. It seems that indicators of neuroinflammation might have the potential to track development of cognitive complaints, however, it requires further investigations.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76630-4" target="_blank">https://www.nature.com/articles/s41598-024-76630-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-27</strong> <a href="https://www.nature.com/articles/s41598-024-77837-1" target="_blank">
            ü§ñ Molecular Test for COVID-19

* Conduct rapid and accurate testing to detect SARS-CoV-2 in patients
* Can provide results quickly, often within minutes or hours after sample collection
* May be used at hospitals, clinics, or remote locations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-421', this)">
            More</button></h4>
            <div id="toggle-421" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Performance of the Flash10 COVID-19 point-of-care molecular test</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>After COVID-19, fever clinics in Beijing's Tsinghua Changgung Hospital need rapid SARS-CoV-2 testing for faster detection. A study tested this test on 125 patients with fever syndrome and found:

* The test is accurate, with an average error rate of 3-4% (1.05h vs 16.15h)
* It has a very low limit of detection of 100 copies per milliliter
* The patient results were different between the test and RT-PCR tests, with 100 positive and 25 negative results
* The test was significantly faster than other testing methods</p><br>
                <p><strong>Abstract</strong></p>
                <p>After the COVID-19 pandemic, fever clinics urgently require rapid nucleic acid tests to enhance their capacity for timely pathogen detection. This study evaluated the analytical performance and clinical utility of the Flash10 SARS-CoV-2 point-of-care test (Flash10 POCT) for detecting SARS-CoV-2 in patients with fever in the adult fever clinic in Beijing Tsinghua Changgung Hospital from August 1 to August 30, 2023. The analytical performance and clinical utility of the Flash10 POCT for detecting SARS-CoV-2 were assessed in 125 patients with fever syndrome in the adult fever clinic. The Flash10 POCT demonstrated an analytical precision of 3.1% for the Ct values of the ORF1ab gene and 2.9% for the Ct values of the N gene in SARS-CoV-2 nucleic acid testing. Furthermore, the Flash10 POCT demonstrated a lower limit of detection (LoD) of 100 copies/mL, with no detected aerosol contamination leakage. Of the 125 patients (median age 61.9 years, 52% male and 48% female), both the Flash10 POCT and RT-PCR tests yielded positive results for 100 patients and negative results for 25 patients (Fisher‚Äôs exact test,p<‚Äâ0.0001). The median turn-around-time for the Flash10 POCT was significantly shorter, at 1.05¬†h, compared to 16.15¬†h required for RT-PCR tests (Wilcoxon signed rank test,p<‚Äâ0.0001). The Flash10 POCT showed high analytical performance, achieving a 100% detection rate for SARS-CoV-2 compared to RT-PCR tests, while also exhibiting a significantly shorter turn-around-time. Implementing the Flash10 POCT had the potential to expedite the care of adults presenting with fever.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77837-1" target="_blank">https://www.nature.com/articles/s41598-024-77837-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-26</strong> <a href="https://www.nature.com/articles/s41598-024-76377-y" target="_blank">
            ü§ñ Engineer-made protein subunits of the COVID-19 vaccine are just as effective at triggering an immune response as tiny particles in lab mice and rats.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-422', this)">
            More</button></h4>
            <div id="toggle-422" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Engineered protein subunit COVID19 vaccine is as immunogenic as nanoparticles in mouse and hamster models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research on SARS-CoV-2 vaccine proteins suggested they were not very effective, but some improvements have been made through engineering and using adjuvants. Scientists tested two different methods: a protein-only vaccine or a vaccine that combines the protein with nanoparticles. The results showed that the vaccine's effectiveness didn't improve when added to an adjuvant - it was still not better than a protein-only vaccine, but as effective in preventing COVID-19 as other vaccines in hamsters infected with live virus one month after vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Initial studies on the immunogenicity of SARS-CoV-2 (CoV-2) S glycoprotein (‚Äúspike‚Äù) as a protein subunit vaccine suggested sub-optimal efficacy in mammals. Although protein engineering efforts have produced CoV-2 spike protein sequences with greatly improved immunogenicity, additional strategies for improving the immunogenicity of CoV-2 protein subunit vaccines are scaffolding and the use of adjuvants. Comparisons of the effectiveness of engineered protein-only and engineered protein-nanoparticles vaccines have been rare. To explore this knowledge gap, we inoculated mice with two doses of either sequence-optimized trimeric spike protein or one of several sequence-optimized spike nanoparticles. We measured their immune response up to two months after the first dose. We also measured the immune response and protection against live virus in hamsters inoculated with either sequence-optimized trimeric spike protein or a liposome-based sequence-optimized spike nanoparticle. We found that in the presence of adjuvant, the antibody and neutralization titers elicited by spike-nanoparticles were not significantly greater than those elicited by spike-only in mice, even at doses as low as 0.1¬†¬µg/animal. Hamsters vaccinated with spike-only or spike-nanoparticles were equally protected from live virus one month after their first inoculation. These results suggest that sequence-optimized protein subunit vaccines in the form of individual prefusion-stabilized trimers can be as effective in improving immunogenicity as scaffolded forms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76377-y" target="_blank">https://www.nature.com/articles/s41598-024-76377-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-26</strong> <a href="https://www.nature.com/articles/s41598-024-77496-2" target="_blank">
            ü§ñ HbA1c blood sugar level
Leukocyte mtDNA DNA fragment count</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-423', this)">
            More</button></h4>
            <div id="toggle-423" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>HbA1c and leukocyte mtDNA levels as major factors associated with post-COVID-19 syndrome in type 2 diabetes patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID-19 syndrome is a new health issue in people recovering from COVID-19 within the past 3-6 months. A study looked at blood samples to see if certain factors could predict whether someone would develop post-COVID-19 syndrome.

People with type 2 diabetes who had COVID-19 were divided into two groups: those with post-COVID-19 syndrome (PCS) and those without PCS for up to 6 months after infection. The study found that:

* People with low blood levels of a certain protein called mitochondrial DNA, which is often involved in the body's energy-making process, are more likely to develop PCS.
* HbA1c (a measure of blood sugar control) was also associated with PCS development, and it seemed to be more important than other factors like oxygen therapy or COVID-19 severity.

This study suggests that low mitochondrial DNA levels and high blood sugar levels might be good indicators of the risk of developing post-COVID-19 syndrome.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-COVID-19 syndrome (PCS) is an emerging health problem in people recovering from COVID-19 infection within the past 3‚Äì6 months. The current study aimed to define the predictive factors of PCS development by assessing the mitochondrial DNA (mtDNA) levels in blood leukocytes, inflammatory markers and HbA1c in type 2 diabetes patients (T2D) with regard to clinical phenotype, gender, and biological age. In this case-control study, 65 T2D patients were selected. Patients were divided into 2 groups depending on PCS presence: the PCS group (n=‚Äâ44) and patients who did not develop PCS (n=‚Äâ21) for up to 6 months after COVID-19 infection. HbA1c and mtDNA levels were the primary factors linked to PCS in different models. We observed significantly lower mtDNA content in T2D patients with PCS compared to those without PCS (1.26‚Äâ¬±‚Äâ0.25 vs. 1.44‚Äâ¬±‚Äâ0.24;p=‚Äâ0.011). In gender-specific and age-related analyses, the mt-DNA amount did not differ significantly between the subgroups. According to the stepwise multivariate logistic regression analysis, low mtDNA content and HbA1c were independent variables associated with PCS development, regardless of oxygen, glucocorticoid therapy and COVID-19 severity. The top-performing model for PCS prediction was the gradient boosting machine (GBM). HbA1c and mtDNA had a notably greater influence than the other variables, indicating their potential as prognostic biomarkers.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77496-2" target="_blank">https://www.nature.com/articles/s41598-024-77496-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-26</strong> <a href="https://www.nature.com/articles/s42003-024-07094-5" target="_blank">
            ü§ñ ## Human Genome Age

A rough estimate based on DNA analysis and other tests suggests that human cells have been present on Earth for around 300,000 years.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-424', this)">
            More</button></h4>
            <div id="toggle-424" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Profiling the transcriptomic age of single-cells in humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers have studied how older cells age in mice using advanced techniques called "single-cell omics clocks". They now want to apply these methods to humans, who also age in unexpected ways. By analyzing blood samples from 508 healthy donors between 19 and 75 years old, they found that certain types of cells become more or less old as we get older. This could help us understand how our bodies repair damage and rejuvenate themselves at a cellular level.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although aging clocks predicting the age of individual organisms have been extensively studied, the age of individual cells remained largely unexplored. Most recently single-cell omics clocks were developed for the mouse, however, extensive profiling the age of human cells is still lacking. To fill this gap, here we use available scRNA-seq data of 1,058,909 blood cells of 508 healthy, human donors (between 19 and 75 years), for developing single-cell transcriptomic clocks and predicting the age of human blood cells. By the application of the proposed cell-type-specific single-cell clocks, our main observations are that (i) transcriptomic age is associated with cellular senescence; (ii) the transcriptomic age of classical monocytes as well as naive B and T cells is decreased in moderate COVID-19 followed by an increase for some cell types in severe COVID-19; and (iii) the human embryo cells transcriptomically rejuvenated at the morulae and blastocyst stages. In summary, here we demonstrate that single-cell transcriptomic clocks are useful tools to investigate aging and rejuvenation at the single-cell level.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07094-5" target="_blank">https://www.nature.com/articles/s42003-024-07094-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-26</strong> <a href="https://www.nature.com/articles/s41398-024-03154-w" target="_blank">
            ü§ñ Memory loss following COVID-19 illness.

### Brain Health and Wellness
COVID-19 can affect the brain, potentially leading to memory problems. These issues might also be related to underlying mental health conditions. To address this topic, it's essential to consider both cognitive impairment from the virus itself and potential correlations with psychiatric symptoms.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-425', this)">
            More</button></h4>
            <div id="toggle-425" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Memory complaints after COVID-19: a potential indicator of primary cognitive impairment or a correlate of psychiatric symptoms?</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Covid-19 patients who experienced difficulty remembering things after recovering from their illness often had a condition that was linked to other mental health issues, rather than just memory problems due to brain damage from the virus.

Researchers studied 608 Covid-19 survivors, finding that those with more serious symptoms of mental health issues also reported memory problems. They also found that some people who recovered quickly (within 6-11 months) still experienced significant memory difficulties, while others took much longer to recover and were still struggling with memories.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cognitive impairment and symptoms of psychiatric disorders have been reported frequently as features of post-acute sequelae of SARS-CoV-2 infection. This study aims to investigate subjective memory complaints in COVID-19 survivors and determine if these are more strongly associated with objective cognitive impairment related to sequelae of SARS-CoV-2 infection or with symptoms of psychiatric conditions. A total of 608 COVID-19 survivors were evaluated in-person 6‚Äì11 months after hospitalization, with 377 patients assigned to a ‚Äúno subjective memory complaint (SMC)‚Äù group and 231 patients assigned to an SMC group based on their Memory Complaint Scale scores. Follow-up evaluations included an objective cognitive battery and scale-based assessments of anxiety, depression, and post-traumatic stress symptoms. We found the perception of memory impairment in COVID-19 survivors to be more strongly associated to core symptoms of psychiatric conditions rather than to primary objective cognitive impairment. Univariate analysis indicated significant differences between the ‚Äúno SMC‚Äù and SMC groups, both for the psychiatric symptom evaluations and for the cognitive evaluations (p‚Äâ<‚Äâ0.05); however, the psychiatric symptoms all had large partial eta-squared values (ranging from 0.181 to 0.213), whereas the cognitive variables had small/medium partial eta-squared values (ranging from 0.002 to 0.024). Additionally, multiple regression analysis indicated that only female sex and depressive and post-traumatic stress symptoms were predictors of subjective memory complaints. These findings may help guide clinical evaluations for COVID-19 survivors presenting with memory complaints while also serving to expand our growing understanding of the relationship between COVID-19, subjective memory complaints, and the risk of cognitive decline.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41398-024-03154-w" target="_blank">https://www.nature.com/articles/s41398-024-03154-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-25</strong> <a href="https://www.nature.com/articles/s41467-024-53384-1" target="_blank">
            ü§ñ Regenerative Cell Therapy with Rezpegaldesleukin for Skin Conditions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-426', this)">
            More</button></h4>
            <div id="toggle-426" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The regulatory T cell-selective interleukin-2 receptor agonist rezpegaldesleukin in the treatment of inflammatory skin diseases: two randomized, double-blind, placebo-controlled phase 1b trials</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Regulatory T cells (Treg) help the immune system regulate inflammation, but their impairment is linked to chronic diseases like atopic dermatitis and psoriasis. Researchers found that a new drug called rezpegaldesleukin (REZPEG) can restore healthy Treg cells in patients with moderate-to-severe skin conditions. The treatment was safe and effective in two trials.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Regulatory T cell (Treg) impairment is implicated in the pathogenesis of chronic inflammatory diseases, but relatively little is known about the therapeutic potential of Treg restoration. Here we present clinical evidence for the Treg-selective interleukin-2 receptor agonist rezpegaldesleukin (REZPEG) in two randomized, double-blind, placebo-controlled Phase 1b trials in patients with moderate-to-severe atopic dermatitis (AD) (NCT04081350) or chronic plaque psoriasis (PsO) (NCT04119557). Key inclusion criteria for AD included an Eczema Area and Severity Index (EASI) score ‚â• 16 and a validated Investigator Global Assessment for Atopic Dermatitis (vIGA-AD)‚Äâ‚â•‚Äâ3, and for PsO included a Psoriasis Area and Severity Index (PASI) score of ‚â• 12 and a static Physician‚Äôs Global Assessment (sPGA) score of ‚â• 3. REZPEG is safe and well-tolerated and demonstrates consistent pharmacokinetics in participants receiving subcutaneous doses of 10 to 12‚Äâ¬µg/kg or 24‚Äâ¬µg/kg once every 2 weeks for 12 weeks, meeting the primary and secondary objectives, respectively. AD patients receiving the higher dose demonstrate an 83% improvement in EASI score after 12 weeks of treatment. EASI improvement of ‚â• 75% (EASI-75) and vIGA-AD responses are maintained for 36 weeks after treatment discontinuation in 71% and 80% of week 12 responders, respectively. These exploratory clinical improvements are accompanied by sustained increases in CD25brightTregs. REZPEG thus represents a homeostatic approach to cutaneous disease therapy and holds clinical potential in providing long-term, treatment-free disease control.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53384-1" target="_blank">https://www.nature.com/articles/s41467-024-53384-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-25</strong> <a href="https://www.nature.com/articles/s41598-024-76372-3" target="_blank">
            ü§ñ Serum Uromodulin Levels Predict Hospitalized COVID-19 Patients' Outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-427', this)">
            More</button></h4>
            <div id="toggle-427" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Serum uromodulin associates with kidney function and outcome in a cohort of hospitalised COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study investigates the prevalence and prognostic implications of acute kidney injury (AKI) in patients with COVID-19, looking at how levels of a particular protein may predict disease severity. It finds that higher levels of the protein are associated with a worse outcome and lower survival rates.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigates the prevalence and evaluates the prognostic implications of acute kidney injury (AKI) in COVID-19 patients, with a novel emphasis on the evaluation of serum uromodulin (sUmod) as a potential kidney-specific biomarker. A cohort of hospitalised COVID-19 patients (n‚Äâ=‚Äâ378) was examined for AKI using standard criteria. In addition to traditional urinary biomarkers, sUmod levels were analysed. Univariable and multivariable regression models were employed to evaluate the association of sUmod and AKI and in-hospital mortality. Levels of sUmod were significantly lower in patients with CKD (91.8‚Äâ¬±‚Äâ60.7¬†ng/ml) compared to patients with normal kidney function (204.7‚Äâ¬±‚Äâ91.7¬†ng/ml; p‚Äâ<‚Äâ0.001). 151‚Äâpatients (40.0%) presented with AKI at the time of hospital admission or developed an AKI during hospitalization. 116‚Äâpatients (76.8%) had an AKI already at the time of hospital admission. COVID-19 patients with AKI had significantly lower levels of sUmod compared to patients without AKI during hospitalisation (124.8‚Äâ¬±‚Äâ79.5¬†ng/ml) vs 214.6‚Äâ¬±‚Äâ92.3¬†ng/ml; p‚Äâ<‚Äâ0.001). The in-hospital mortality rate in this cohort of COVID-19 patients was 15.3%. Patients with AKI had a higher probability for in-hospital death (OR 5.6, CI‚Äâ1.76‚Äâto‚Äâ17.881, p‚Äâ=‚Äâ0.004). Patients who died during hospital stay, had significantly lower sUmod levels (129.14‚Äâ¬±‚Äâ89.56¬†ng/ml) compared to patients surviving hospitalisation (187.71‚Äâ¬±‚Äâ96,64¬†ng/ml; p‚Äâ<‚Äâ0.001). AKI is frequently associated with COVID-19 in hospitalized patients. Serum uromodulin may emerge as a promising biomarker for AKI in COVID-19 patients. Further research is warranted to explore its clinical application and refine risk stratification in this patient population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76372-3" target="_blank">https://www.nature.com/articles/s41598-024-76372-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-24</strong> <a href="https://www.nature.com/articles/s41541-024-00973-2" target="_blank">
            ü§ñ Interferon as a vaccine booster to increase antibody response after influenza B infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-428', this)">
            More</button></h4>
            <div id="toggle-428" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Interferon as an immunoadjuvant to enhance antibodies following influenza B infection and vaccination in ferrets</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Despite annual vaccinations, the influenza B virus (IBV) continues to cause significant morbidity and mortality in humans. Researchers found that IBV infection resulted in a weaker immune response than the flu A virus. To understand why this is happening, scientists administered type-I or type-III interferon (IFN) to ferrets following infection or vaccination. IFN plays an important role in the initial innate immune response and affects subsequent adaptive responses.

Following IBV infection or vaccination, IFN treatment increased gene expression of early inflammatory responses in the upper respiratory tract and robust IFN, TSLP, and inflammatory responses in peripheral blood cells. These responses were sustained after challenge or vaccination.

Serum levels of interferon lambda (IFNL) and thymic stromal lymphopoietin (TSLP) increased in response to IBV infection or vaccination, but this was not seen with vaccine administration. Antibody responses in the serum increased more quickly and to higher titers than mock-Tx animals over 3 months.

Following rechallenge of infected animals three months post treatment, antibody levels remained higher than mock-Tx animals. However, IFN treatment did not affect antibody responses following challenge of vaccinated animals. A strong direct correlation was found between TSLP levels and antibody responses after challenge-rechallenge and vaccination-challenge. This suggests that TSLP could be a useful tool for predicting adaptive immune responses to IBV infection or vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite annual vaccination, influenza B viruses (IBV) continue to cause significant morbidity and mortality in humans. We have found that IBV infection resulted in a weaker innate and adaptive immune response than influenza A viruses (IAV) in ferrets. To understand and overcome the weak immune responses to IBV in ferrets, we administered type-I or type-III interferon (IFN) to ferrets following infection or vaccination and evaluated their effects on the immune response. IFN signaling following viral infection plays an important role in the initial innate immune response and affects subsequent adaptive immune responses. In the respiratory tract, IFN lambda (IFNL) has regulatory effects on adaptive immunity indirectly through thymic stromal lymphopoietin (TSLP), which then acts on immune cells to stimulate the adaptive response. Following IBV infection or vaccination, IFN treatment (IFN-Tx) upregulated gene expression of early inflammatory responses in the upper respiratory tract and robust IFN, TSLP, and inflammatory responses in peripheral blood cells. These responses were sustained following challenge or vaccination in IFN-Tx animals. Serum IFNL and TSLP levels were enhanced in IFN-Tx animals following challenge/rechallenge over mock-Tx; however, this difference was not observed following vaccination. Antibody responses in serum of IFN-Tx animals following IBV infection or vaccination increased more quickly and to higher titers and were sustained longer than mock-Tx animals over 3 months. Following rechallenge of infected animals 3 months post treatment, antibody levels remained higher than mock-Tx. However, IFN-Tx did not have an effect on antibody responses following challenge of vaccinated animals. A strong direct correlation was found between TSLP levels and antibody responses following challenge-rechallenge and vaccination-challenge indicating it as a useful tool for predicting adaptive immune responses following IBV infection or vaccination. The effects of IFN on strengthening both innate and adaptive responses to IBV may aid in development of more effective treatments following infection and improved influenza vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00973-2" target="_blank">https://www.nature.com/articles/s41541-024-00973-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-24</strong> <a href="https://www.nature.com/articles/s41598-024-77087-1" target="_blank">
            ü§ñ Collaborative Genome Array System (CAST) EiJ strain

A genetic variant developed from an existing mouse strain used to study and analyze the characteristics of COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-429', this)">
            More</button></h4>
            <div id="toggle-429" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterization of Collaborative Cross mouse founder strain CAST/EiJ as a novel model for lethal COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Mutations in certain SARS-CoV-2 variants have widened their host range, affecting non-primate mammals like mice. This study used genetically diverse mouse panels to research how various strains respond to SARS-CoV-2 infection. Mouse strains with wild-derived origins showed varying responses, but one strain (CAST/EiJ) was particularly aggressive and caused significant damage. These findings may help develop new treatments for COVID-19 and other coronavirus diseases in the future by testing potential therapies on genetically diverse mouse models.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Mutations in SARS-CoV-2 variants of concern (VOCs) have expanded the viral host range beyond primates, and a few other mammals, to mice, affording the opportunity to exploit genetically diverse mouse panels to model the broad spectrum of responses to infection in patient populations. Here we surveyed responses to VOC infection in genetically diverse Collaborative Cross (CC) founder strains. Infection of wild-derived CC founder strains produced a broad range of viral burden, disease susceptibility and survival, whereas most other strains were resistant to disease despite measurable lung viral titers. In particular, CAST/EiJ, a wild-derived strain, developed high lung viral burdens, more severe lung pathology than seen in other CC strains, and a dysregulated cytokine profile resulting in morbidity and mortality. These inbred mouse strains may serve as a valuable platform to evaluate therapeutic countermeasures against severe COVID-19 and other coronavirus pandemics in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77087-1" target="_blank">https://www.nature.com/articles/s41598-024-77087-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-76400-2" target="_blank">
            ü§ñ Treatment Approach for Severe Influenza-Associated Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-430', this)">
            More</button></h4>
            <div id="toggle-430" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Beneficial effect of sequential treatment with high-dose steroids and short-course oral glucocorticoids in patients with severe influenza virus-associated pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Glucocorticoid treatment can help shorten the course of viral pneumonia (VPA) and improve survival rates. However, there is no standard approach to dosage and duration of treatment, which makes it difficult to determine its effectiveness. Researchers studied 11 patients with severe VPA who received high-dose glucocorticoids for three months. The study found that the symptoms improved, and none of the patients died due to tracheal intubation or pulmonary fibrosis. The researchers suggest that short-course oral glucocorticoids may be helpful in reducing complications and improving outcomes in patients with severe VPA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Accumulating evidence supports that glucocorticoid treatment for viral pneumonia (VPA) can shorten the disease course and improve survival. However, currently, the use of glucocorticoids in treating VPA remains controversial. Moreover, a unified standard for the dosage and duration of glucocorticoid therapy has not been presented in published articles. A retrospective analysis was conducted in patients who were hospitalized for severe influenza virus-associated pneumonia, and they received sequential treatment with high-dose glucocorticoids and short-course oral glucocorticoids. Patients were followed up for 3¬†months. A total of 11 patients were included in the study (average age 56¬†years). There was no gender difference, but age and underlying diseases could be risk factors for severe influenza virus-associated pneumonia. The types of viruses causing pneumonia included influenza A/B. The main clinical symptoms of patients were fever, cough, sputum production, and dyspnea. Chest computed tomography showed multiple ground-glass shadows in the lobes, and the presence of bacterial and fungal infections was accompanied by consolidation shadows. After glucocorticoid therapy, the symptoms improved. None of the patients underwent tracheal intubation, and all survived. After a 3-month follow-up, lung CT absorption in all patients had reached more than 80%, and lung imaging absorption in 20% patients was complete. No serious complications occurred in any of the patients. Sequential treatment with high-dose steroids and short-course oral glucocorticoids may be helpful for reducing the tracheal intubation rate and mortality rate in patients with severe influenza virus-associated pneumonia. Additionally, short-course oral glucocorticoids may reduce pulmonary fibrosis in patients with severe influenza virus-associated pneumonia without any serious complications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76400-2" target="_blank">https://www.nature.com/articles/s41598-024-76400-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41556-024-01537-1" target="_blank">
            ü§ñ Real-time and programmable RNA sequencing technology for rapid data generation in research settings.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-431', this)">
            More</button></h4>
            <div id="toggle-431" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Real-time and programmable transcriptome sequencing with PROFIT-seq</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The eukaryotic transcriptome is complex, making it hard to detect specific genes or proteins. Current methods often need complicated steps before analyzing the data. A new method called PROFIT-seq helps by selectively targeting genes without affecting the rest of the genome. It improves the quality and speed of gene analysis. This method can study how immune systems respond to infections in cancer patients, including polyps.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The high diversity and complexity of the eukaryotic transcriptome make it difficult to effectively detect specific transcripts of interest. Current targeted RNA sequencing methods often require complex pre-sequencing enrichment steps, which can compromise the comprehensive characterization of the entire transcriptome. Here we describe programmable full-length isoform transcriptome sequencing (PROFIT-seq), a method that enriches target transcripts while maintaining unbiased quantification of the whole transcriptome. PROFIT-seq employs combinatorial reverse transcription to capture polyadenylated, non-polyadenylated and circular RNAs, coupled with a programmable control system that selectively enriches target transcripts during sequencing. This approach achieves over 3-fold increase in effective data yield and reduces the time required for detecting specific pathogens or key mutations by 75%. We applied PROFIT-seq to study colorectal polyp development, revealing the intricate relationship between host immune responses and bacterial infection. PROFIT-seq offers a powerful tool for accurate and efficient sequencing of target transcripts while preserving overall transcriptome quantification, with broad applications in clinical diagnostics and targeted enrichment scenarios.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41556-024-01537-1" target="_blank">https://www.nature.com/articles/s41556-024-01537-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-76252-w" target="_blank">
            ü§ñ Study Impact of COVID-19 on Allergic Rhinitis Patterns in China Based on Local Data.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-432', this)">
            More</button></h4>
            <div id="toggle-432" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>To study the impact of COVID-19 on the epidemiological characteristics of allergic rhinitis based on local big data in China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To investigate the characteristics and changes in allergic rhinitis (AR) patients before and after the COVID-19 pandemic, a study was conducted using descriptive epidemiological methods on data from 2018 to 2023.

An analysis found that over 62,000 AR patients were identified, with most males and females aged between 1 and 89 years. The number of AR patients increased steadily each year, but without significant differences in gender or age groups. However, the pandemic seemed to accelerate this trend. There was a clear seasonal pattern, and the number of AR patients increased significantly during COVID-19, possibly due to hormonal changes related to the virus.

A higher proportion of male AR patients were found compared to female patients. This difference may be related to hormonal fluctuations caused by the pandemic. As a result, updated management measures are needed to address the changing trends in AR since the pandemic began.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the big data characteristics and trend changes of patients with allergic rhinitis (AR) who sought medical attention at our hospital before (from 2018 to 2019) and after (from 2020 to 2023) COVID-19, and provide reference basis for the treatment of AR.  This study used a descriptive epidemiological method to analyze the big data and trend changes of AR patients.  A total of 62,196 AR patients were collected, of whom 32,874 were male and 29,322 were female, with an age range of 1‚Äì89 years. The monthly change trend of AR patients showed a marked seasonality. The number of AR patients increased year by year. There was no significant difference in the number of patients between different genders. There was a significant difference in the number of patients between different age groups.  The number of AR patients increased markedly from 2018 to 2023, and COVID-19 seems to have accelerated this process. There is a clear seasonal pattern. The number of boy patients is significantly higher than that of girl patients, and the change in female hormones may affect the incidence of AR. Therefore, it is necessary to update management measures and formulate relevant policies in response to the changing trend of AR since the COVID-19 epidemic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76252-w" target="_blank">https://www.nature.com/articles/s41598-024-76252-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-72449-1" target="_blank">
            ü§ñ COVID-19 Treatment Effectiveness

Atorvastatin and telmisartan not effective against SARS-CoV-2 in mild or moderate cases.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-433', this)">
            More</button></h4>
            <div id="toggle-433" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Atorvastatin and telmisartan do not reduce nasopharyngeal carriage of SARS-CoV-2 in mild or moderate COVID-19 in a phase IIb randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Observational studies have found that taking ACE2 inhibitors and statins can reduce severe COVID-19 disease. A study in C√¥te d'Ivoire tested the combination of these medications with two other treatments for mild to moderate COVID-19 infection. The goal was to see if adding a third medication to the standard treatment could help shorten recovery time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Observational studies suggest a reduction in fatal or severe COVID-19 disease with the use of ACE2 inhibitors and statins. We implemented a randomized controlled tree-arm open label trial evaluating the benefits of adding telmisartan (TLM) or atorvastatin (ATV) to lopinavir boosted ritonavir (LPVr) on the SARS-CoV-2 nasopharyngeal viral load in patients with mild / moderate COVID-19 infection in C√¥te d‚ÄôIvoire. RT-PCR positive COVID-19 patients‚Äâ‚â•‚Äâ18¬†years, with general or respiratory symptoms for less than 7¬†days were randomized (1:1:1) to receive LPVr (400¬†mg/100¬†mg twice daily), LPVr‚Äâ+‚ÄâTLM (10¬†mg once daily) or LPVr‚Äâ+‚ÄâATV (20¬†mg once daily) for 10¬†days. The primary endpoint was viro-inflammatory success defined as a composite variable at day 11: Ct‚Äâ‚â•‚Äâ40 and C-reactive protein‚Äâ<‚Äâ27¬†mg/L. We randomized 294 patients: 96 to LPVr, 100 to LPVr‚Äâ+‚ÄâTLM, 98 to LPVr‚Äâ+‚ÄâATV arms. Baseline characteristics were well balanced between arms. In the primary analysis (missing‚Äâ=‚Äâfailure), 46% patients in the LPVr arm reached viro-inflammatory success at day 11 vs 43% in the LPVr‚Äâ+‚ÄâTLM arm (p‚Äâ=‚Äâ0.69) and 43% in the LPVr‚Äâ+‚ÄâATV arm (p‚Äâ=‚Äâ0.68). The median time from baseline to resolution of COVID-19 related symptoms was not different between arms. Nine patients were hospitalized: 2 in the LPVr arm, 5 in the LPVr‚Äâ+‚ÄâTLM arm and 2 in the LPVr‚Äâ+‚ÄâATV arm and 4 patients died. Among adults with mild to moderate COVID-19 infection, the addition of telmisartan or atorvastatin, to the standard LPVr treatment is not associated with a better virological or clinical outcome.Trial registration:NCT04466241, registered on 10/07/2020</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72449-1" target="_blank">https://www.nature.com/articles/s41598-024-72449-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41541-024-00997-8" target="_blank">
            ü§ñ Human Immune Response to Sarbecoviruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-434', this)">
            More</button></h4>
            <div id="toggle-434" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broad cross neutralizing antibodies against sarbecoviruses generated by SARS-CoV-2 infection and vaccination in humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Preventing future coronavirus pandemics requires protecting against multiple types of coronaviruses, including the ones that cause MERS and SARS-CoV-1, as well as the new strain known as Omicron subvariant.

Researchers studied whether people who had COVID-19 or received vaccines could develop antibodies (like special proteins) to protect against other coronaviruses that might come from animals. They tested these antibodies on a variety of viruses, including some that are common in bats and pangolins.

The results showed that the antibodies worked well for many types of coronaviruses but not as well for the one that causes MERS or SARS-CoV-1. Some of the antibodies also had extra parts (like small proteins) that helped to neutralize other viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The outbreaks of severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV), and SARS-CoV-2 highlight the need for countermeasures to prevent future coronavirus pandemics. Given the unpredictable nature of spillover events, preparing antibodies with broad coronavirus-neutralizing activity is an ideal proactive strategy. Here, we investigated whether SARS-CoV-2 infection and vaccination could provide cross-neutralizing antibodies (nAbs) against zoonotic sarbecoviruses. We evaluated the cross-neutralizing profiles of plasma and monoclonal antibodies constructed from B cells from coronavirus disease 2019 (COVID-19) convalescents and vaccine recipients; against sarbecoviruses originating from bats, civets, and pangolins; and against SARS-CoV-1 and SARS-CoV-2. We found that the majority of individuals with natural infection and vaccination elicited broad nAb responses to most tested sarbecoviruses, particularly to clade 1b viruses, but exhibited very low cross-neutralization to SARS-CoV-1 in both natural infection and vaccination, and vaccination boosters significantly augmented the magnitude and breadth of nAbs to sarbecoviruses. Of the nAbs, several exhibited neutralization activity against multiple sarbecoviruses by targeting the spike receptor-binding domain (RBD) and competing with angiotensin-converting enzyme 2 (ACE2) binding. SCM12-61 demonstrated exceptional potency, with half-maximal inhibitory concentration (IC50) values of 0.001‚Äì0.091‚ÄâŒºg/mL against tested sarbecoviruses; while VSM9-12 exhibited remarkable cross-neutralizing breadth against sarbecoviruses and SARS-CoV-2 Omicron subvariants, highlighting the potential of these two nAbs in combating sarbecoviruses and SARS-CoV-2 Omicron subvariants. Collectively, our findings suggest that vaccination with an ancestral SARS-CoV-2 vaccine, in combination with broad nAbs against sarbecoviruses, may provide a countermeasure for preventing further sarbecovirus outbreaks in humans.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00997-8" target="_blank">https://www.nature.com/articles/s41541-024-00997-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-75038-4" target="_blank">
            ü§ñ Improved isolation of SARS-CoV-2 from COVID-19 clinical specimens with enhanced cellular expression</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-435', this)">
            More</button></h4>
            <div id="toggle-435" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improved efficacy of SARS-CoV-2 isolation from COVID-19 clinical specimens using VeroE6 cells overexpressing TMPRSS2 and human ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Cell culture-based isolation of COVID-19 novel coronavirus is an important process for both laboratory diagnosis and understanding new variants. Research has found that a specific type of cell line called VeroE6/TMPRSS2 can efficiently isolate the virus from patient samples. These cells have two key features: they express a receptor molecule (monkey ACE2) and have a serine-protease enzyme (TMPRSS2). They are more effective than other types of cells at isolating the virus, showing an 1.6-fold increase in efficiency.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The cell culture-based isolation of novel coronavirus SARS-CoV-2 from clinical specimens obtained from patients with suspected COVID-19 is important not only for laboratory diagnosis but also for obtaining live virus to characterize emerging variants. Previous studies report that monkey kidney-derived VeroE6/TMPRSS2 cells allow efficient isolation of SARS-CoV-2 from clinical specimens because these cells show stable expression of the receptor molecule monkey ACE2 and the serine-protease TMPRSS2. Here, we demonstrated that VeroE6 cells overexpressing human ACE2 and TMPRSS2 (Vero E6-TMPRSS2-T2A-ACE2 cells) are superior to VeroE6/TMPRSS2 for isolating SARS-CoV-2 from clinical specimens. These cells showed a 1.6-fold increase in efficiency in SARS-CoV-2 isolation, and were particularly effective for clinical specimens with a relatively low viral load (<‚Äâ106copies/mL). When using vesicular stomatitis virus (VSV) pseudoviruses (VSV/SARS-2pv) bearing the spike proteins of all of the tested SARS-CoV-2 strains, Vero E6-TMPRSS2-T2A-ACE2 cells showed a 2‚Äì¬†to fourfold increase in infectivity. Furthermore, the results of virus titration and neutralization antibody assays using Vero E6-TMPRSS2-T2A-ACE2 cells were different from those using VeroE6/TMPRSS2, highlighting the importance of selecting appropriate cell culture systems to determine SARS-CoV-2 infectivity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75038-4" target="_blank">https://www.nature.com/articles/s41598-024-75038-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-76871-3" target="_blank">
            ü§ñ Research on COVID-19 Survivors' Outcomes After Hospitalization</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-436', this)">
            More</button></h4>
            <div id="toggle-436" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>One-year mortality and associated factors in older hospitalized COVID-19 survivors: a Nationwide Cohort Study in Korea</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study aimed to evaluate the 1-year mortality rate among older patients who recovered from COVID-19 after being discharged from hospital. It looked at data from a Korean health insurance system for patients aged 60 or over with COVID-19, comparing their survival rates and identifying risk factors that contributed to death within 1 year of recovery.

Researchers used a model to analyze how many people died or survived each month following discharge, and then compared the mortality rate to see if it was similar after hospital discharge in Korea. They found that non-survivors were older and had been vaccinated less often than survivors. Additionally, non-survivors experienced more complications during their illness.

The study also looked at other factors that might have contributed to death, including age, sex, cardiovascular disease, organ dysfunction, severity of the illness, and certain treatments used during hospitalization. 

Overall, the research suggested that older patients who recovered from COVID-19 after being discharged had a similar 1-year mortality rate as those in Korea, but still experienced higher risks due to factors such as age and lack of vaccination protection for long-term safety against COVID-19-related complications.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aimed to evaluate the 1-year mortality rate among older patients with COVID-19 discharged from hospital and to identify risk factors associated with this outcome. Using a COVID-19 dataset from the Korean National Health Insurance System, this study‚Äôs evaluation period spanned from October 8, 2020, through December 31, 2021. The primary outcome was the 1-year mortality rate following hospital discharge. A logistic regression model was employed for multivariable analysis to estimate the odds ratios for the outcomes, and the Kaplan-Meier method was used to analyze differences in 1-year survival rates. Among the 66,810 COVID-19 patients aged 60 years or older who were hospitalized during the study period, the in-hospital mortality rate was 4.8% (n=‚Äâ3219). Among the survivors (n=‚Äâ63,369), the 1-year mortality rate was 4.9% (n=‚Äâ3093). Non-survivors, compared to survivors, were significantly older (79.2‚Äâ¬±‚Äâ9.5 vs. 68.9‚Äâ¬±‚Äâ7.8,P<‚Äâ0.001) and exhibited a lower rate of COVID-19 vaccination (63.0% vs. 91.7%,P<‚Äâ0.001). Additionally, non-survivors experienced a higher incidence of organ dysfunction, along with a greater proportion of required mechanical ventilation (14.6% vs. 1.0%,P<‚Äâ0.001) and extracorporeal membrane oxygenation (4.0% vs. 0.1%,P<‚Äâ0.001). Multivariable logistic regression analysis identified older age, male sex, cardiovascular disease, immunosuppression, organ dysfunction, illness severity, and corticosteroid use during hospitalization as factors associated with death within 1 year after hospital discharge. However, vaccination was found to have a long-term protective effect against death among COVID-19 survivors. The 1-year mortality rate after hospital discharge for older COVID-19 patients was comparable to the in-hospital mortality rate for these patients in Korea. The long-term mortality rate among hospitalized older COVID-19 patients was influenced by demographic factors and the severity of illness experienced during hospitalization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76871-3" target="_blank">https://www.nature.com/articles/s41598-024-76871-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-76657-7" target="_blank">
            ü§ñ Blood Pressure Medications and COVID-19 Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-437', this)">
            More</button></h4>
            <div id="toggle-437" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Losartan and enalapril maleate differently influence SARS-CoV-2-infected vero cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has posed significant challenges for global healthcare systems, particularly affecting individuals with pre-existing conditions like hypertension. Researchers studied how two antihypertensive medications, losartan and enalapril maleate, affected cells infected with SARS-CoV-2 in vitro.

In this study, researchers found that losartan significantly reduced the levels of viral RNA in these cells, making it nearly undetectable. However, enalapril maleate did not have a significant effect on reducing viral RNA levels. Furthermore, they discovered that the expression of genes involved in cell survival and regulation was altered in response to infection.

In addition, researchers found that losartan increased certain proteins (IL-6, p53, p21, and p62) and pathways (il-6) compared to uninfected cells treated with losartan. In contrast, enalapril maleate did not demonstrate a significant effect on these proteins or pathways.

The study suggests that both medications may interfere with the effects of SARS-CoV-2 infection in Vero E6 cells, but their influence appears to vary in terms of quantity and quality in modulating metabolic and signal transduction pathways.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Background: The COVID-19 pandemic has posed significant challenges to global healthcare systems, particularly impacting individuals with pre-existing conditions like hypertension. This study sought to assess the impact of the antihypertensive medications, losartan and enalapril maleate on SARS-CoV-2 infected cells. Vero E6 cells were infected and treated in vitro, evaluating cell viability via the MTT colorimetric assay. Additionally, the study measured relative levels of viral RNA and selected gene messenger RNAs using reverse transcriptase followed by quantitative real-time polymerase chain reaction. Results: The findings revealed that losartan substantially reduced nucleocapsid RNA levels of SARS-CoV-2 to nearly undetectable levels, while enalapril maleate did not demonstrate a significant effect. In response to viral infection, the expression ofil-18,p53,p21, andp62increased compared to uninfected-untreated cells. Notably,il-6expression was upregulated by both infection and treatments. A comparison between infected cells treated with losartan or enalapril maleate highlighted the presence of distinct profiles in the expression ofil-6,p53,p21, andp62. Conclusions: The data from our study suggest that these medications could interfere with certain effects triggered by SARS-CoV-2 infection in Vero E6 cells. However, their influence appears to vary both quantitatively and qualitatively in the modulation of metabolic and signal transduction pathways.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76657-7" target="_blank">https://www.nature.com/articles/s41598-024-76657-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41746-024-01286-3" target="_blank">
            ü§ñ Long COVID Temporal Topic Modeling of Electronic Health Records

Temporal topic modeling analysis on electronic health records (EHRs) from the National Center for Chronic Disease Prevention and Control (NCCDP) and the Recovery (RECOVER) program.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-438', this)">
            More</button></h4>
            <div id="toggle-438" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Finding Long-COVID: temporal topic modeling of electronic health records from the N3C and RECOVER programs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-Acute Sequelae of SARS-CoV-2 (Long COVID) is a complex condition that affects some people after they've recovered from a severe case of COVID-19, leading to various health problems. Researchers analyzed the experiences of over 600 million patients with COVID-19 and found many conditions worsened in those who had the disease, as well as specific patterns for different groups of people. This study revealed new insights into the causes and consequences of Long COVID and suggests that it may require better diagnosis and understanding to improve treatment and prevention strategies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-Acute Sequelae of SARS-CoV-2 infection (PASC), also known as Long-COVID, encompasses a variety of complex and varied outcomes following COVID-19 infection that are still poorly understood. We clustered over 600 million condition diagnoses from 14 million patients available through the National COVID Cohort Collaborative (N3C), generating hundreds of highly detailed clinical phenotypes. Assessing patient clinical trajectories using these clusters allowed us to identify individual conditions and phenotypes strongly increased after acute infection. We found many conditions increased in COVID-19 patients compared to controls, and using a novel method to associate patients with clusters over time, we additionally found phenotypes specific to patient sex, age, wave of infection, and PASC diagnosis status. While many of these results reflect known PASC symptoms, the resolution provided by this unprecedented data scale suggests avenues for improved diagnostics and mechanistic understanding of this multifaceted disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-024-01286-3" target="_blank">https://www.nature.com/articles/s41746-024-01286-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-75519-6" target="_blank">
            ü§ñ Understanding the Mechanism of SARS-CoV-2 Inhibitors

Benzisoselenazolones and Diselenides as SARS-CoV-2 Inhibitors
=====================================================

### INSTRUCTIONS ###

* Simplify the title for a general audience to convey the main idea.
* Avoid using quotation marks around the text.
* Keep the explanation concise, focused on clarity rather than brevity.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-439', this)">
            More</button></h4>
            <div id="toggle-439" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>New insights in the mechanism of the SARS-CoV-2 Mproinhibition by benzisoselenazolones and diselenides</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new virus, SARS-CoV-2, has been kept at bay thanks to global vaccination efforts, but a possibility remains that the virus could evolve and cause another pandemic. To combat this risk, scientists are seeking compounds that can target the main protease of the virus (Mpro). Researchers tested various small molecules derived from benzisoselenazolones and diselenides for their ability to inhibit Mpro and found some effective ones. They also investigated how these compounds interact with proteins in the body to understand why they're so effective against Mpro. To do this, scientists used advanced computer simulations and modeling techniques to better understand the interactions between these molecules and the virus. This research could lead to the discovery of new compounds that can prevent SARS-CoV-2 from multiplying, potentially saving lives during a future pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although global vaccination campaigns alleviated the SARS-CoV-2 pandemic in terms of morbidity and mortality, the ability of the virus to originate mutants may reduce the efficacy of vaccines, posing a serious risk of a renewed pandemic. There is therefore a need to develop small molecules capable of targeting conserved viral targets, such as the main protease (Mpro). Here, a series of benzisoselenazolones and diselenides were tested for their ability to inhibit Mpro; then the most potent compounds were measured for antiviral activity in vitro, and the mechanism of action was investigated. Density functional theory calculations, molecular docking and molecular dynamics simulations were also used to elucidate the protein/drug interaction. Finally, a bio-organic model was established to study the reaction between selenorganic compounds and biologically relevant thiols to unveil possible metabolic pathways of such compounds. The overall results contribute to the identification of a series of novel Se-containing molecules active against SARS-CoV-2 and to the clarification of some important aspects in the mechanisms of action of such inhibitors targeting SARS-CoV-2 Mpro.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75519-6" target="_blank">https://www.nature.com/articles/s41598-024-75519-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-74712-x" target="_blank">
            ü§ñ Improved Vaccination Protocol for Long-Term Survivors After Allogeneic Hematopoietic Stem Cell Transplantation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-440', this)">
            More</button></h4>
            <div id="toggle-440" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity of Comirnaty Omicron XBB.1.5 booster COVID-19 mRNA vaccine in long-term survivors after allogeneic hematopoietic stem cell transplantation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Primary mRNA vaccination against COVID-19 typically involves three doses for immunocompromised individuals, including those who have undergone hematopoietic stem cell transplantation (allo-HSCT). But the best way to boost immunity remains unclear after multiple doses of mRNA vaccine.

A new study looked at how a booster dose from the most recently updated vaccine was received by long-term allo-HSCT survivors. These patients had previously received many mRNA vaccines, with the last dose being about 13 months ago. The researchers collected blood samples before and four weeks after the booster and measured antibodies against the spike (S1) and nucleocapsid proteins in the blood, as well as T-cell responses.

About half of the patients showed a strong immune response to both S1 and T-cells just before the booster dose, but some were not protected from severe COVID-19. Women had higher antibody levels than men, with 62% having a previous confirmed case of COVID-19. However, this was only true for women.

After the booster dose, some patients showed stronger immune responses to both S1 and T-cells, while others did not respond as well. Some patients who received immunosuppressive treatment had weaker immune responses overall.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary mRNA vaccination against COVID-19 typically involves three doses for immunocompromised individuals, including hematopoietic stem cell transplantation (allo-HSCT) recipients. However, optimal subsequent boosting strategies remain unclear. This study aimed to assess the immunogenicity of a booster dose using the most recently updated vaccine (Comirnaty Omicron XBB.1.5) among long-term allo-HSCT survivors having previously received multiple mRNA vaccine doses, in median 4 (2‚Äì6). Thirty-four allo-HSCT recipients were enrolled at Sahlgrenska University Hospital, and peripheral blood samples were collected immediately before and four weeks after booster. Antibodies against the receptor-binding domain (anti-RBD) of spike 1 (S1) and nucleocapsid, as well as S1-specific ex vivo T-cell responses, were evaluated. Adverse events were monitored. Despite a median of 13 months since the prior vaccine dose, both humoral and T-cell responses against S1 were present in the pre-booster samples in all but two participants, who suffered from severe chronic Graft-versus-host disease. Notably, 62% of participants had a previously confirmed COVID-19 infection. Significantly higher pre-booster antibody levels were observed in women than men (p=‚Äâ0.003). Booster dosing strengthened specific antibody and T cell responses and equalized pre-booster gender differences, although responses remained significantly lower among those receiving immunosuppressive treatment (p=‚Äâ0.041). In a population of long-term allo-HSCT survivors, the majority of whom had a prior confirmed COVID-19 infection, both pre- and post-booster immune responses were robust. However, patients undergoing immunosuppressive treatment for GvHD exhibited significantly weaker responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74712-x" target="_blank">https://www.nature.com/articles/s41598-024-74712-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-75924-x" target="_blank">
            ü§ñ Cytokine Response in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-441', this)">
            More</button></h4>
            <div id="toggle-441" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Disease progression associated cytokines in COVID-19 patients with deteriorating and recovering health conditions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding the immune response to COVID-19 is challenging due to its high variability among individuals. Researchers analyzed EHR data and cytokine profile data from 444 infected patients and 145 non-infected healthy individuals to identify differentially expressed cytokines between deteriorating and recovering phases.

A random forest model was built using both healthy and severe patients to compute the contribution of each cytokine toward disease progression, with Shapley Additive Explanations (SHAP) values used for supervised clustering. The identified clusters discriminated DP and RP samples, suggesting that cytokine profiles differed between deteriorating and recovering health conditions.

Differentially expressed cytokines, such as CXCL10, GDF15, PTX3, and TNFSF10, were found to be involved in the JAK-STAT, NF-\(\kappa\)B, and MAPK signaling pathways contributing to the inflammatory response.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the immune response to COVID-19 is challenging due to its high variability among individuals. To identify differentially expressed cytokines between the deteriorating and recovering phases, we analyzed the Electronic Health Records (EHR) and cytokine profile data in a COVID-19 cohort of 444 infected patients and 145 non-infected healthy individuals. We categorized each patient‚Äôs progression into Deterioration Phase (DP) and Recovery Phase (RP) using longitudinal neutrophil, lymphocyte and lactate dehydrogenase levels. A random forest model was built using healthy and severe patients to compute the contribution of each cytokine toward disease progression using Shapley Additive Explanations (SHAP). SHAP values were used for supervised clustering to identify DP and RP-related samples and their associated cytokines. The identified clusters effectively discriminated DP and RP samples, suggesting that the cytokine profiles differed between deteriorating and recovering health conditions. Especially, CXCL10, GDF15, PTX3, and TNFSF10 were differentially expressed between the DP and RP samples, which are involved in the JAK-STAT, NF-\(\kappa\)B, and MAPK signaling pathways contributing to the inflammatory response. Collectively, we characterized the immune response in terms of disease progression of COVID-19 with deteriorating and recovering health conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75924-x" target="_blank">https://www.nature.com/articles/s41598-024-75924-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41746-024-01287-2" target="_blank">
            ü§ñ **Detecting COVID Infection with Wearable Sensors**

**Feasible Solution for Snapshot Testing**</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-442', this)">
            More</button></h4>
            <div id="toggle-442" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Feasibility of snapshot testing using wearable sensors to detect cardiorespiratory illness (COVID infection in India)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new system was developed that uses sensors and activity tracking to monitor people's health, including heart rate, breathing, temperature, and oxygen levels. This system worked well in India during the pandemic, showing promising results in detecting COVID-19 symptoms. Researchers trained an AI algorithm on the data to identify who is likely sick or healthy, achieving a high accuracy level. The findings suggest that this technology could be used for remote health screenings, potentially changing how people monitor their health over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has challenged the current paradigm of clinical and community-based disease detection. We present a multimodal wearable sensor system paired with a two-minute, movement-based activity sequence that successfully captures a snapshot of physiological data (including cardiac, respiratory, temperature, and percent oxygen saturation). We conducted a large, multi-site trial of this technology across India from June 2021 to April 2022 amidst the COVID-19 pandemic (Clinical trial registry name: International Validation of Wearable Sensor to Monitor COVID-19 Like Signs and Symptoms; NCT05334680; initial release: 04/15/2022). An Extreme Gradient Boosting algorithm was trained to discriminate between COVID-19 infected individuals (n=‚Äâ295) and COVID-19 negative healthy controls (n=‚Äâ172) and achieved an F1-Score of 0.80 (95% CI‚Äâ=‚Äâ[0.79, 0.81]). SHAP values were mapped to visualize feature importance and directionality, yielding engineered features from core temperature, cough, and lung sounds as highly important. The results demonstrated potential for data-driven wearable sensor technology for remote preliminary screening, highlighting a fundamental pivot from continuous to snapshot monitoring of cardiorespiratory illnesses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-024-01287-2" target="_blank">https://www.nature.com/articles/s41746-024-01287-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-76251-x" target="_blank">
            ü§ñ ## Systemic Immune Response

This refers to the body's immune response triggered by a coronavirus, specifically SARS-CoV-2, that has been weakened or impaired due to a compromised immune system. In people who have received a kidney transplant and are immunocompromised (i.e., not on long-term immunosuppressive medications), their immune system is suppressed, making them more susceptible to the virus. As a result, T follicular helper cells, which play a key role in coordinating the body's response to infections, may be activated but also impaired.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-443', this)">
            More</button></h4>
            <div id="toggle-443" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impaired SARS-CoV-2-specific responses via activated T follicular helper cells in immunocompromised kidney transplant recipients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A group of immune cells called activated T follicular helper (aTfh) cells played a key role in the body's response to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) vaccines. Researchers studied how these cells reacted in people who had received SARS-CoV-2 mRNA vaccines and those who were not vaccinated. They found that the aTfh cells in people who got the vaccine responded differently than those who didn't get vaccinated, with more effective responses at later stages of the infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Activated T follicular helper (aTfh) cells are likely important in host immune responses to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) mRNA vaccination. We characterized the immune responses of aTfh cells to the second (D2) and third (booster; D3) doses of an mRNA vaccine in the peripheral blood of 40 kidney transplant recipients (KTRs) and 17 healthy control volunteers (HCs). A significant increase in SARS-CoV-2-specific IgG antibody was seen after D3 in the KTRs; nonetheless, the levels after D2 and D3 were significantly lower than in the HCs. After D2, dramatic increases in activated CD45RA-CXCR5+ICOS+PD1+circulating Tfh (acTfh) cells were observed in the HCs, as well as the seropositive patients among the KTRs, when compared with the seronegative patients among the KTRs. Unlike the HCs, KTRs had less prominent immune responses, including the acTfh and T cells that produce interferon gamma, tumor necrosis factor alpha, and interleukin 21. In addition, the increase in acTfh cells was significantly associated with anti-IgG antibody levels after D3. These results indicate impaired SARS-CoV-2-specific responses via acTfh cells in KTRs, and they suggest that acTfh cells in peripheral blood may play an important role in antibody maintenance following SARS-CoV-2 mRNA vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76251-x" target="_blank">https://www.nature.com/articles/s41598-024-76251-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-75718-1" target="_blank">
            ü§ñ Studies have shown that COVID-19 can alter cancer cell behavior.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-444', this)">
            More</button></h4>
            <div id="toggle-444" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>In vitro analysis suggests that SARS-CoV-2 infection differentially modulates cancer-like phenotypes and cytokine expression in colorectal and prostate cancer cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new virus called SARS-CoV-2 has been linked to cancer and making its effects harder to understand. Researchers studied how this virus affects two types of cancer cells, prostate and colorectal, which both use proteins that help control cell growth and division.

They infected these cancer cells with a fake version of the virus and then looked at what happened inside the cells. They also tested some cancer markers (such as genes that help grow or survive) to see how the virus affected them. The results showed that SARS-CoV-2 affects the cells in different ways depending on which type of cancer cell they are, but it can make cells more likely to divide and die.

The researchers believe that this could be important information for figuring out if other viruses like SARS-CoV-2 affect cancer too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) reportedly exacerbates cancer outcomes. However, how COVID-19 influences cancer prognosis and development remains poorly understood. Here, we investigated the effect of Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV-2), the etiological agent of COVID-19, on cellular cancer phenotypes the expression of cancer-related markers, and various proinflammatory cytokines. We infected prostate (22RV1) and colorectal (DLD-1) cancer cell lines, which express angiotensin-converting enzyme 2 (ACE2), with spike pseudovirus (sPV) and laboratory stocks of live SARS-CoV-2 viruses. After infection, we quantified changes in the cellular cancer phenotypes, the gene expression levels of some cancer markers, including Ki-67, BCL-2, VIM, MMP9,and VEGF,and proinflammatory cytokines. Phenotypic analysis was performed using MTT and wound healing assays, whereas gene expression analysis was carried out using real-time quantitative PCR (RT-qPCR). We show that SARS-CoV-2 infection impacts several key cellular pathways involved in cell growth, apoptosis, and migration, in prostate and colorectal cancer cells. Our results suggest that SARS-CoV-2 infection does influence various cancer cellular phenotypes and expression of molecular cancer markers and proinflammatory cytokines, albeit in a cell-type-specific manner. Our findings hint at the need for further studies and could have implications for evaluating the impact of other viruses on cancer progression.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75718-1" target="_blank">https://www.nature.com/articles/s41598-024-75718-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-75262-y" target="_blank">
            ü§ñ Community-led initiatives as predictors of older adults' physical and mental well-being during the COVID-19 pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-445', this)">
            More</button></h4>
            <div id="toggle-445" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of community groups as a social prescription for senior health‚Äîinsights from a natural experiment during the COVID-19 lockdown</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Impact of Social Prescribing Policies on Loneliness in Elderly People**

The COVID-19 pandemic exacerbated loneliness among older adults, particularly during lockdowns. Recent policies aimed at supporting this vulnerable population include community groups as support structures. This study evaluates the effectiveness of these social prescribing initiatives in combating loneliness and related health decline.

We analyzed data from 618 individuals aged 65 or older who participated in a natural experiment during the pandemic lockdown. The results show that those who had access to community groups were significantly less likely to experience loneliness (2.65 times lower) and associated health problems, such as cardiovascular and cognitive decline.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Loneliness and associated physical and cognitive health decline among the aging population is an important medical concern, exacerbated in times of abnormal isolation like the 2020‚Äì2021 Covid-19 pandemic lockdown. In this backdrop, recent ‚Äúsocial prescribing‚Äù based health policy initiatives such as community groups as a support structure for the aging population assumes great importance. In this paper, we evaluate and quantify the impact of such social prescribing policies in combatting loneliness and related health degeneration of the aging population in times of abnormal isolation. To this end, we conduct a natural experiment across a sample of 618 individuals aged 65 and over with varying access to community groups during the Covid-19 lockdown period. Using a random-effects, probit model to compare the differences in health outcomes of participants with access to community groups (target) with those without access (control), we find that the target group was 2.65 times less likely to suffer from loneliness as compared to the control group, along with lower incidences of reported cardiovascular and cognitive health decline. These initial findings provide preliminary support in favor of the interventional power of social prescription tools in mitigating loneliness and its consequent negative health impact on the aging population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75262-y" target="_blank">https://www.nature.com/articles/s41598-024-75262-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-74729-2" target="_blank">
            ü§ñ Global Health Burden of Vaccine-Associated Guillain-Barr√© Syndrome</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-446', this)">
            More</button></h4>
            <div id="toggle-446" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Global burden of vaccine-associated Guillain-Barr√© syndrome over 170 countries from 1967 to 2023</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research has been conducted globally on Guillain-Barr√© syndrome (GBS), a rare neurological condition linked to vaccinations. To assess the global burden of vaccine-associated GBS and its associated vaccines, we analyzed reports from VigiBase, a database of adverse event reports for medicines and vaccines, spanning 1967-2023. We found that approximately 8 in every 1,000 people who received over 19 different vaccines reported developing GBS. The most commonly implicated vaccines were those for COVID-19, with the highest association seen in influenza vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Research on Guillain-Barr√© syndrome (GBS) as a neurological adverse effect of vaccines on a global scale is scarce, highlighting the need for further investigation to evaluate its long-term impact and associated risk factors comprehensively. Hence, this study aims to assess the global burden of vaccine-associated GBS and its associated vaccines. This study utilized data from VigiBase, the World Health Organization global database of adverse event reports of medicines and vaccines, encompassing the period from 1967 to 2023 (total reports,n=‚Äâ131,255,418) to investigate vaccine-associated GBS. Reported odds ratios (ROR) and information components (IC) were analyzed to assess the association between 19 vaccines and the occurrence of vaccine-associated GBS over 170 countries. We identified 15,377 (8072 males [52.49%]) reports of vaccine-associated GBS among 22,616 reports of all drugs-cause GBS from 1978 to 2023. Cumulative reports of vaccine-associated GBS have been increasing steadily over time, with a notable surge observed since the commencement of COVID-19 vaccines administration in 2020. Most vaccines showed significant associations with GBS such as Ad5-vectored COVID-19 vaccines (ROR, 14.88; IC, 3.66), COVID-19 mRNA vaccines (ROR, 9.66; IC, 2.84), and inactivated whole-virus COVID-19 vaccines (ROR, 3,29; IC 1.69). Influenza vaccines showed the highest association (ROR, 77.91; IC 5.98). Regarding age-and sex-specific risks, the association remained similar regardless of sex, with an increased association observed with advancing age. The mean time to onset was 5.5 days. Amid the COVID-19 pandemic, the reports of GBS surged in response to widespread COVID-19 vaccination. Nonetheless, COVID-19 vaccines exhibited the lowest association compared to other vaccines. Vigilance for at least one-week post-vaccination is crucial, particularly for older adults. Further research is warranted to elucidate the underlying mechanisms linking vaccines and GBS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74729-2" target="_blank">https://www.nature.com/articles/s41598-024-74729-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-18</strong> <a href="https://www.nature.com/articles/s41598-024-75250-2" target="_blank">
            ü§ñ Distinct Immune Responses in Older Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-447', this)">
            More</button></h4>
            <div id="toggle-447" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Similar humoral responses but distinct CD4+T cell transcriptomic profiles in older adults elicited by MF59 adjuvanted and high dose influenza vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A type of flu vaccine is recommended to be given to people over 65 years old to help keep them safe from getting sick with the flu. Researchers studied how this vaccine affects a group of older adults who received either an MF59-flavored or high-dose (HD) version of the vaccine. They found that the flu vaccine had different effects on two groups of people: those who got the regular version and those who got the higher dose. The researchers also looked at what genes were turned on or off in these cells to see if they could help explain why some people are protected from getting sick with the flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Older age (‚â•‚Äâ65 years) is associated with impaired responses to influenza vaccination, leading to the preferential recommendation of MF59-adjuvanted (MF59Flu) or high-dose (HDFlu) influenza vaccines for this age group in the United States. Herein, we characterized transcriptomic profiles of CD4+T cells isolated from 234 recipients (‚â•‚Äâ65 years) of either MF59Flu or HDFlu vaccine, prior to vaccination and 28 days thereafter. We identified 412 and 645 differentially expressed genes (DEGs) in CD4+T cells of older adults after receiving MF59Flu and HDFlu, respectively. DEGs in CD4+T cells of MF59Flu recipients were enriched in 14 KEGG pathways, all of which were downregulated. DEGs in CD4+T cells of HDFlu recipients were enriched in 11 upregulated pathways and 20 downregulated pathways. CD4+T cells in both vaccine groups shared 50 upregulated genes and 75 downregulated genes, all of which were enriched in 7 KEGG pathways. The remaining 287 and 520 DEGs were specifically associated with MF59Flu and HDFlu, respectively. Unexpectedly, none of these DEGs was significantly correlated with influenza A/H3N2-specific HAI titers, suggesting these DEGs at the individual level may have a limited role in protection against influenza. Our findings emphasize the need for further investigation into other factors influencing immunity against influenza in older adults.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75250-2" target="_blank">https://www.nature.com/articles/s41598-024-75250-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-18</strong> <a href="https://www.nature.com/articles/s41598-024-75514-x" target="_blank">
            ü§ñ Vaccine-induced autoimmunity related to protein S in some cases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-448', this)">
            More</button></h4>
            <div id="toggle-448" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Autoantibodies to protein S may explain rare cases of coagulopathy following COVID-19 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new COVID-19 vaccine has been shown to be effective, but some people have experienced severe reactions that can cause life-threatening problems. Researchers wanted to know more about the vaccines and what's causing these reactions. They looked at a large group of people who had received the vaccine in Sweden and found that some were infected with COVID-19 while others got sick from the vaccine. They also found out if there was something in the vaccine or its side effects, such as protein S autoantibodies, that might be causing these problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While Coronavirus disease 2019 (COVID-19) vaccines have proven to be both effective and generally safe, rare but severe adverse events following immunization (AEFIs) are described. Autoantibodies to platelet factor-4 are associated with catastrophic thrombotic AEFIs, but comprehensive investigations of other autoantibodies are lacking. We aimed to detect and describe autoantibodies targeting coagulation-related proteins in a population-wide cohort (SWEDEGENE) including AEFIs attributed to COVID-19 vaccines in Sweden. Subjects were recruited from December 2020 to October 2022 and were stratified based on diagnosis and COVID-19 exposure. Screening was carried out in two phases, with a multiplex bead-based assay in the first subset (until September 2021) and with targeted assays for the second (until October 2022). Positivity was defined based on absolute, relative, and biological/technical thresholds. Patients with coagulation-related AEFIs were older and the Vaxzevria vaccine was overrepresented in this group. Two cases had antiphospholipid antibodies but none had PF4 antibodies. We identified six positives for protein S autoantibodies. Protein S concentrations were negatively correlated with autoantibody response in patients with immunoreactivity and functional analysis revealed low protein S activity in three subjects. Our population-wide analysis reveals cases with autoantibodies against protein S which possibly underlie coagulopathic AEFIs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75514-x" target="_blank">https://www.nature.com/articles/s41598-024-75514-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41467-024-52863-9" target="_blank">
            ü§ñ Mucosa-Associated Lymphoid Tissue (MALT) Response Induced by Cytokines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-449', this)">
            More</button></h4>
            <div id="toggle-449" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>MINCLE and TLR9 agonists synergize to induce Th1/Th17 vaccine memory and mucosal recall in mice and non-human primates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Development of New Vaccines for Hard-to-Target Diseases

Scientists have been trying to develop new vaccines that can target diseases that are difficult to reach. However, their current adjuvants (substances that help the vaccine work) often don't stimulate the body's immune response in the way they should.

Researchers discovered a novel liposomal adjuvant called CAF10b that combines Mincle and Toll-like receptor 9 agonists. They tested it with mice and non-human primates, comparing its effects to existing vaccines. 

When combined with an antigen (the substance that triggers the immune response), CAF10b stimulated strong Th1 and Th17 responses in both animals, providing protection against a bacterial infection called Mycobacterium tuberculosis.

In some primate models, CAF10b induced powerful Th1 and Th17 immune responses for six months. These types of immune responses are crucial for long-term immunity.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Development of new vaccines tailored for difficult-to-target diseases is hampered by a lack of diverse adjuvants for human use, and none of the currently available adjuvants induce Th17 cells. Here, we develop a liposomal adjuvant, CAF¬Æ10b, that incorporates Mincle and Toll-like receptor 9 agonists. In parallel mouse and non-human primate studies comparing to CAF¬Æ adjuvants already in clinical trials, we report species-specific effects of adjuvant composition on the quality and magnitude of the responses. When combined with¬†antigen, CAF¬Æ10b induces Th1 and Th17 responses and protection against a pulmonary infection withMycobacterium tuberculosisin mice. In non-human primates, CAF¬Æ10b induces higher Th1 responses and robust Th17 responses detectable after six months, and systemic and pulmonary Th1 and Th17 recall responses, in a sterile model of local recall. Overall, CAF¬Æ10b drives robust memory antibody, Th1 and Th17 vaccine-responses via a non-mucosal immunization route across both rodent and primate species.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52863-9" target="_blank">https://www.nature.com/articles/s41467-024-52863-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s44298-024-00058-w" target="_blank">
            ü§ñ Infectious Diseases Evolution Genetics Microbiology Pathogens Research Science Technology Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-450', this)">
            More</button></h4>
            <div id="toggle-450" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Discordant phylodynamic and spatiotemporal transmission patterns driving the long-term persistence and evolution of human coronaviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding the spread of different human coronaviruses can help us better understand how these viruses evolve over time. Scientists studied four types of coronaviruses that circulate in humans to see where they were present and how they changed over time. Their research found that two of these viruses, HCoV-229E and HCoV-OC43, are more likely to create new versions of themselves because they can easily change and spread from person to person.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Four distinct species of human coronaviruses (HCoVs) circulate in humans. Despite the recent attention due to SARS-CoV-2, a comprehensive understanding of the molecular epidemiology and genomic evolution of HCoVs remains unclear. Here, we employed primary differentiated human nasal epithelial cells for the successful isolation and genome sequencing of HCoVs derived from two retrospective cohorts in Singapore and Tanzania. Phylodynamic inference shows that HCoV-229E and HCoV-OC43 were subject to stronger genetic drift and reduced purifying selection from the early 2000s onwards, primarily targeting spike Domain A and B. This resulted in increased lineage diversification, coinciding with a higher effective reproductive number (Re>1.0). However, HCoV-NL63 and HCoV-HKU1 experienced weaker genetic drift and selective pressure with prolonged regional persistence. Our findings suggest that HCoV-229E and HCoV-OC43 viruses are adept at generating new variants and achieving widespread intercontinental dissemination driven by continuous genetic drift, recombination, and complex migration patterns.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00058-w" target="_blank">https://www.nature.com/articles/s44298-024-00058-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41598-024-74568-1" target="_blank">
            ü§ñ Understanding COVID-19's effect on physical health.

Activity levels in people after the pandemic
Participation levels in activities after the pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-451', this)">
            More</button></h4>
            <div id="toggle-451" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of post-COVID symptoms on activity and participation of women and men</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID syndrome is affecting millions worldwide and has become a major public health problem. A study investigated the relationship between symptoms and activity limitations for participants with post-COVID syndrome, particularly focusing on sex differences.

Researchers analyzed data from 307 patients who completed a questionnaire about their rehabilitation needs at different stages of recovery after COVID-19 infection. The results showed no difference in symptom intensity between men and women, but differences were found in the intensity of symptoms related to activity limitations (ALPR).

The study also discovered correlations between various factors and ALPR, including fatigue, which was associated with higher levels of ALPR across both sexes. However, different patterns emerged for women and men, suggesting that rehabilitation needs may vary depending on sex.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-COVID syndrome is affecting many organ systems and arises as a major public health problem with millions of cases worldwide. The primary aim of this study is the analysis of health problems, activity limitations and participation restrictions (ALPR) of participants with post-COVID symptoms and the investigation of correlations between these elements to derive statements about the rehabilitation need, also depending on sex. A retrospective cohort study was performed to collect longitudinal data from January 2022 to January 2023 using the Covid-19 Rehabilitation Needs Questionnaire (RehabNeQ). Patients completed the questionnaire at the Department of Rehabilitation- and Sports Medicine at Hannover Medical School. The 1st assessment included 307 study participants, of whom 54 showed up for the 2nd, 7 for the 3rd and one for the 4th assessment. Study participants with post-COVID symptoms also experience ALPR. The results show no significant difference in symptom intensity in women and men, but in intensity of ALPR. We found many correlations of varying degrees between various factors with ALPR. We found frequent correlations between fatigue and several ALPR. While these correlations apply to both sexes, we also found different correlations in women and men, indicating the different rehabilitation need of women and men.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74568-1" target="_blank">https://www.nature.com/articles/s41598-024-74568-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41598-024-76363-4" target="_blank">
            ü§ñ Malnutrition risk factors during COVID-19 illness</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-452', this)">
            More</button></h4>
            <div id="toggle-452" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Malnutrition is associated with severe outcome in elderly patients hospitalised with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Studies have found that some factors contribute to people being sick with COVID-19 as an elderly person, but research has not yet fully understood how these factors affect outcomes. Researchers looked at over 4,200 COVID-19 patients in China and found that those with poor nutritional status had a higher risk of severe illness. They also used mathematical models to see if better nutrition could predict who would get sick and why. The researchers concluded that PNI (a measure of nutritional health) is a good indicator of whether someone will get serious or not, and they suggest that doctors should pay close attention to patients' diets when treating COVID-19 cases in the elderly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Some studies have identified influencing factors of COVID-19 illness in elderly, such as underlying diseases, but research on the effect of nutritional status is still lacking. This study retrospectively examined the influence of nutritional status on the outcome of elderly COVID-19 inpatients. A retrospective analysis of the clinical data of 4241 COVID-19 patients who were admitted to a third-class hospital of Nanchang between November 1, 2022 and January 31, 2023 was conducted. Nutritional status was assessed using the prognostic nutritional index (PNI) and controlling nutritional status score (CONUT). The influence of nutritional status on the outcome of COVID-19 patients was determined through multivariate adjustment analysis, restrictive cubic spline, and receiver operating characteristic curve (ROC). Compared with mild/no malnutrition, severe malnutrition substantially increased the critical outcome of COVID-19. A linear relationship was observed between the odds ratio (OR) and PNI and CONUT (P>‚Äâ0.05). The area under the ROC curve indicated that PNI was the better predictor. The optimal cutoff value of PNI was 38.04 (95%CI: 0.797‚Äâ~‚Äâ0.836, AUC‚Äâ=‚Äâ0.817), with a sensitivity of 70.7% and a specificity of 79.6%. The critical illness of elderly COVID-19 patients shows a linear relationship with malnutrition at admission. The use of PNI to assess the prognosis of COVID-19 eldeely patients is reliable, highlighting the importance for doctors to closely pay attention to the nutritional status of COVID-19 patients. Focusing on nutritional status in clinical practice can effectively reduce the critical illness of elderly COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76363-4" target="_blank">https://www.nature.com/articles/s41598-024-76363-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41467-024-53356-5" target="_blank">
            ü§ñ Infectious Diseases and Immune Response Research Center

Prevalent and Persistent New-Onset Autoantibodies in Mild to Severe COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-453', this)">
            More</button></h4>
            <div id="toggle-453" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalent and persistent new-onset autoantibodies in mild to severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Autoantibodies, which are antibodies that attack the body's own proteins, have been found in people with COVID-19. Researchers studied 525 healthcare workers and patients hospitalized with COVID-19 over a year-long period to understand how autoantibodies change and evolve in response to the virus. They found that new autoantibodies appeared after infection and remained present for at least a year. Autoantibodies also seemed to be linked to neuropsychiatric symptoms and severe illness following COVID-19. The researchers identified specific proteins on SARS-CoV-2 as targets for these new antibodies, suggesting a possible link between the virus's spike protein and autoimmune responses in some patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Autoantibodies have been shown to be implied in COVID-19 but the emerging autoantibody repertoire remains largely unexplored. We investigated the new-onset autoantibody repertoire in 525 healthcare workers and hospitalized COVID-19 patients at five time points over a 16-month period in 2020 and 2021 using proteome-wide and targeted protein and peptide arrays. Our results show that prevalent new-onset autoantibodies against a wide range of antigens emerged following SARS-CoV-2 infection in relation to pre-infectious baseline samples and remained elevated for at least 12 months. We found an increased prevalence of new-onset autoantibodies after severe COVID-19 and demonstrated associations between distinct new-onset autoantibodies and neuropsychiatric symptoms post-COVID-19. Using epitope mapping, we determined the main epitopes of selected new-onset autoantibodies, validated them in independent cohorts of neuro-COVID and pre-pandemic healthy controls, and identified sequence similarities suggestive of molecular mimicry between main epitopes and the conserved fusion peptide of the SARS-CoV-2 Spike glycoprotein. Our work describes the complexity and dynamics of the autoantibody repertoire emerging with COVID-19 and supports the need for continued analysis of the new-onset autoantibody repertoire to elucidate the mechanisms of the post-COVID-19 condition.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53356-5" target="_blank">https://www.nature.com/articles/s41467-024-53356-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41598-024-75338-9" target="_blank">
            ü§ñ Kidney function impacted by SARS-CoV-2 virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-454', this)">
            More</button></h4>
            <div id="toggle-454" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection causes a decline in renal megalin expression and affects vitamin D metabolism in the kidney of K18-hACE2 mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Patients with COVID-19 often experience damage to their kidneys and can have trouble processing certain vitamins, including vitamin D. Research has investigated how severe COVID-19 affects this process, and found that the virus can reduce the amount of megalin in the kidneys' tubules. This protein plays a role in the body's ability to absorb vitamin D from food. The study looked at whether COVID-19 infection reduced megalin levels in mice with human genes for the enzyme that helps convert one type of vitamin into another, and found that it did. The research suggests that even if the amount of vitamin D is not changed, the virus can still affect how the kidneys handle it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients with coronavirus disease 2019 (COVID-19) often experience acute kidney injury, linked to disease severity or mortality, along with renal tubular dysfunction and megalin loss in proximal tubules. Megalin plays a crucial role in kidney vitamin D metabolism. However, the impact of megalin loss on vitamin D metabolism during COVID-19 is unclear. This study investigated whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection reduces megalin expression in proximal tubules and its subsequent effect on vitamin D metabolism in mice expressing human angiotensin converting enzyme 2 (K18-hACE2 mice). Histological and immunohistochemical staining analyses revealed glomerular and capillary congestion, and elevated renal neutrophil gelatinase-associated lipocalin levels, indicative of acute kidney injury in K18-hACE2 mice. In SARS-CoV-2-infected mice, immunohistochemical staining revealed suppressed megalin protein levels. Decreased vitamin D receptor (VDR) localization in the nucleus and increased mRNA expression of VDR, CYP27B1, and CYP24A1 were observed by quantitative PCR in SARS-CoV-2-infected mice. Serum vitamin D levels remained similar in infected and vehicle-treated mice, but an increase in tumor necrosis factor-alpha and a decrease in IL-4 mRNA expression were observed in the kidneys of the SARS-CoV-2 group. These findings suggest that megalin loss in SARS-CoV-2 infection may impact the local role of vitamin D in kidney immunomodulation, even when blood vitamin D levels remain unchanged.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75338-9" target="_blank">https://www.nature.com/articles/s41598-024-75338-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41598-024-76163-w" target="_blank">
            ü§ñ Self-Proven Covid-19 Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-455', this)">
            More</button></h4>
            <div id="toggle-455" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A self-adjuvanted VLPs-based Covid-19 vaccine proven versatile, safe, and highly protective</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A team developed a new type of COVID-19 vaccine that uses tiny particles from bacteria to fight the virus. They created these particles using a special system and purified them by removing toxins, making it safe for use. The vaccine was tested in animals and found to be effective at stopping the infection without causing harm. These findings suggest that this vaccine could be used to protect against COVID-19 with potentially lower costs than existing vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccination has played a critical role in mitigating COVID-19. Despite the availability of licensed vaccines, there remains a pressing need for improved vaccine platforms that provide high protection, safety, and versatility, while also reducing vaccine costs. In response to these challenges, our aim is to create a self-adjuvanted vaccine against SARS-CoV-2, utilizing Virus-Like Particles (VLPs) as the foundation. To achieve this, we produced bacteriophage (Q≈í‚â§) VLPs in a prokaryotic system and purified them using a rapid and cost-effective strategy involving organic solvents. This method aims to solubilize lipids and components of the cell membrane to eliminate endotoxins present in bacterial samples. For vaccine formulation, Receptor Binding Domain (RBD) antigens were conjugated using chemical crosslinkers, a process compatible with Good Manufacturing Practice (GMP) standards. Transmission Electron Microscopy (TEM) confirmed the expected folding and spatial configuration of the Q≈í‚â§VLPs vaccine. Additionally, vaccine formulation assessment involved SDS-PAGE stained with Coomassie Brilliant Blue, Western blotting, and stereomicroscopic experiments. In vitro and in vivo evaluations of the vaccine formulation were conducted to assess its capacity to induce a protective immune response without causing side effects. Vaccine doses of 20¬¨‚Ä†¬¨¬µg and 50¬¨‚Ä†¬¨¬µg stimulated the production of neutralizing antibodies. In in vivo testing, the group of animals vaccinated with 50¬¨‚Ä†¬¨¬µg of vaccine formulation provided complete protection against virus infection, maintaining stable body weight without showing signs of disease. In conclusion, the Q≈í‚â§VLPs-RBD vaccine has proven to be effective and safe, eliminating the necessity for supplementary adjuvants and offering a financially feasible approach. Moreover, this vaccine platform demonstrates flexibility in targeting Variants of Concern (VOCs) via established conjugation protocols with VLPs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76163-w" target="_blank">https://www.nature.com/articles/s41598-024-76163-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41541-024-00981-2" target="_blank">
            ü§ñ Orf Vaccine Protects Hamsters and Nonhuman Primates Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-456', this)">
            More</button></h4>
            <div id="toggle-456" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A multiantigenic Orf virus-based vaccine efficiently protects hamsters and nonhuman primates against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>## Next-generation COVID-19 vaccines involve designing vaccines that can effectively combat emerging variants of concern.

A new approach is being explored using an Orf virus-based vector to create multiantigenic vaccines targeting SARS-CoV-2 spike and nucleocapsid antigens.

## The vaccines were engineered to express either the spike protein (ORFV-S) or both the spike and nucleocapsid proteins (ORFV-S/N).

Studies showed that these vaccines triggered similar levels of antibodies against spike proteins and neutralized viruses in mice, but the multiantigenic vaccine performed better in a challenge model, providing long-term protection against SARS-CoV-2 infection.

## These results suggest the potential of the Orf virus-based platform for prophylactic vaccination and support further development into clinical trials.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Among the common strategies to design next-generation COVID-19 vaccines is broadening the antigenic repertoire thereby aiming to increase efficacy against emerging variants of concern (VoC). This study describes a new Orf virus-based vector (ORFV) platform to design a multiantigenic vaccine targeting SARS-CoV-2 spike and nucleocapsid antigens. Vaccine candidates were engineered, either expressing spike protein (ORFV-S) alone or co-expressing nucleocapsid protein (ORFV-S/N). Mono- and multiantigenic vaccines elicited comparable levels of spike-specific antibodies and virus neutralization in mice. Results from a SARS-CoV-2 challenge model in hamsters suggest cross-protective properties of the multiantigenic vaccine against VoC, indicating improved viral clearance with ORFV-S/N, as compared to equal doses of ORFV-S. In a nonhuman primate challenge model, vaccination with the ORFV-S/N vaccine resulted in long-term protection against SARS-CoV-2 infection. These results demonstrate the potential of the ORFV platform for prophylactic vaccination and represent a preclinical development program supporting first-in-man studies with the multiantigenic ORFV vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00981-2" target="_blank">https://www.nature.com/articles/s41541-024-00981-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41421-024-00734-4" target="_blank">
            ü§ñ Single-cell analysis of lung tissue shows role of Slamf9+ macrophages in clearing viruses and resolving inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-457', this)">
            More</button></h4>
            <div id="toggle-457" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Single-cell spatiotemporal analysis of the lungs revealsSlamf9+macrophages involved in viral clearance and inflammation resolution</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The lungs' ability to restore health after a severe infection is not fully understood. Researchers studied how the lungs heal from pneumonia caused by SARS-CoV-2 in hamsters, finding that certain immune cells play a key role in clearing the virus. These immune cells contain the virus and work together with other types of immune cells to reduce inflammation and repair damaged tissue.</p><br>
                <p><strong>Abstract</strong></p>
                <p>How the lung achieves immune homeostasis after a pulmonary infection is not fully understood. Here, we analyzed the spatiotemporal changes in the lungs over a 2-week natural recovery from severe pneumonia in a Syrian hamster model of SARS-CoV-2 infection. We find that SARS-CoV-2 infects multiple cell types and causes massive cell death at the early stage, including alveolar macrophages. We identify a group of monocyte-derivedSlamf9+macrophages, which are induced after SARS-CoV-2 infection and resistant to impairment caused by SARS-CoV-2.Slamf9+macrophages contain SARS-CoV-2, recruit and interact withIsg12+Cst7+neutrophils to clear the viruses. After viral clearance,Slamf9+macrophages differentiate intoTrem2+andFbp1+macrophages, contributing to inflammation resolution at the late stage, and finally replenish alveolar macrophages. These findings are validated in a SARS-CoV-2-infected hACE2 mouse model and confirmed with publicly available human autopsy single-cell RNA-seq data, demonstrating the potential role ofSlamf9+macrophages and their coordination with neutrophils in post-injury tissue repair and inflammation resolution.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00734-4" target="_blank">https://www.nature.com/articles/s41421-024-00734-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41421-024-00733-5" target="_blank">
            ü§ñ Single-cell immune system research reveals specific areas where immune cells work together to fight infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-458', this)">
            More</button></h4>
            <div id="toggle-458" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Single-cell spatiotemporal analysis reveals alveolar dendritic cell‚ÄìT cell immunity hubs defending against pulmonary infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new study has looked at how immune cells work together in the lungs to fight off infections and inflammation caused by a virus like SARS-CoV-2. They used a high-tech method called immunocartography to analyze how these immune cells interacted and responded over time. The researchers studied Syrian hamster models that were infected with SARS-CoV-2 and found that specific types of immune cells worked together to control the infection, eliminate the virus, and then recover when the virus was gone. They also looked at how this process worked in normal lungs and similar human lungs to see if their findings can be applied to real-life lung health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>How immune cells are spatiotemporally coordinated in the lung to effectively monitor, respond to, and resolve infection and inflammation in primed form needs to be fully illustrated. Here we apply immunocartography, a high-resolution technique that integrates spatial and single-cell RNA sequencing (scRNA-seq) through deconvolution and co-localization analyses, to the SARS-CoV-2-infected Syrian hamster model. We generate a comprehensive transcriptome map of the whole process of pulmonary infection from physiological condition, infection initiation, severe pneumonia to natural recovery at organ scale and single-cell resolution, with 142,965 cells and 45 lung lobes from 25 hamsters at 5 time points. Integrative analysis identifies that alveolar dendritic cell‚ÄìT cell immunity hubs, whereCcr7+Ido1+dendritic cells,Cd160+Cd8+T cells, andTnfrsf4+Cd4+T cells physiologically co-localize, rapidly expand during SARS-CoV-2 infection, eliminate SARS-CoV-2 with the aid ofSlamf9+macrophages, and then restore to physiological levels after viral clearance. We verify the presence of these cell subpopulations in the immunity hubs in normal and SARS-CoV-2-infected hACE2 mouse models, as well as in publicly available human scRNA-seq datasets, demonstrating the potential broad relevance of our findings in lung immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00733-5" target="_blank">https://www.nature.com/articles/s41421-024-00733-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41541-024-00990-1" target="_blank">
            ü§ñ ## Novel Vaccine Adjuvant Enhances Protection Against Pertussis

## Adjuvants help vaccines work better by stimulating the immune system.

## They usually contain tiny particles that trigger an immune response, making it easier to develop immunity to specific diseases.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-459', this)">
            More</button></h4>
            <div id="toggle-459" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel outer membrane vesicle adjuvant improves vaccine protection againstBordetella pertussis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A new vaccine is being developed to prevent the respiratory disease caused by Bordetella pertussis. The vaccine is designed to stimulate an immune response in the nose and throat, where it can eliminate the bacteria that cause pertussis. Researchers tested the vaccine on mice and found that it not only protected them from getting sick but also eliminated the bacteria in their lungs and nasal passages. This suggests that a new approach may be effective in preventing pertussis infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pertussis is a vaccine-preventable respiratory disease caused by the Gram negative coccobacillus Bordetella pertussis. The licensed acellular pertussis (aP) vaccines protect against disease but do not prevent bacterial colonization and transmission. Here, we developed and tested an intranasal vaccine composed of aP antigens combined with T-vant, a novel adjuvant derived from bacterial outer membrane vesicles, that elicits both mucosal and systemic immune responses. We hypothesized that immunization of mice with aP-T-vant would enhance mucosal immunity and eliminate B. pertussis in the respiratory tract. In contrast to mice immunized intramuscularly with the licensed aP vaccine, intranasal immunization with aP-T-vant eliminated bacteria in both the lung and nasopharynx. Protection was associated with IFN-gamma and IL-17-producing, non-circulating CD4‚Äâ+‚ÄâT cells in the lung and nasopharynx, and sterilizing immunity in the nasopharynx was dependent on IL-17. Novel mucosal adjuvants, such as T-vant, warrant further investigation to enhance the efficacy of next generation pertussis vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00990-1" target="_blank">https://www.nature.com/articles/s41541-024-00990-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41467-024-53029-3" target="_blank">
            ü§ñ SARS-CoV-2 Variants with Insertions and Specific Mutations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-460', this)">
            More</button></h4>
            <div id="toggle-460" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sarbecovirus RBD indels and specific residues dictating multi-species ACE2 adaptiveness</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Sarbecoviruses are viruses that infect bats and other mammals, but their ability to interact with humans remains unclear. Scientists analyzed 268 sarbecovirus genetic sequences to understand how these viruses use human cells (ACE2 receptors). They found that some sarbecoviruses can adapt to different types of ACE2 on the surface of mammalian cells, while others have limited or no use for it. The scientists also discovered a specific pattern in the genetic codes of sarbecovirus proteins and their interaction with ACE2. This study provides new insights into how viruses interact with human cells and may help prevent future outbreaks.

I made the following changes:

* Simplified the language to make it easier to understand
* Removed quotation marks around the abstract text
* Kept the same structure and organization as the original abstract</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our comprehensive understanding of the multi-species ACE2 adaptiveness of sarbecoviruses remains elusive, particularly for those with various receptor binding motif (RBM) insertions/deletions (indels). Here, we analyzed RBM sequences from 268 sarbecoviruses categorized into four RBM indel types. We examined the ability of 20 representative sarbecovirus Spike glycoproteins (S) and derivatives in utilizing ACE2 from various bats and several other mammalian species. We reveal that sarbecoviruses with long RBMs (type-I) can achieve broad ACE2 tropism, whereas viruses with single deletions in Region 1 (type-II) or Region 2 (type-III) exhibit narrower ACE2 tropism. Sarbecoviruses with double region deletions (type-IV) completely lost ACE2 usage, which is restricted by clade-specific residues within and outside RBM. Lastly, we propose the evolution of sarbecovirus RBM indels and illustrate how loop lengths, disulfide, and residue determinants shape multi-species ACE2 adaptiveness. This study provides profound insights into the mechanisms governing ACE2 usage and spillover risks of sarbecoviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53029-3" target="_blank">https://www.nature.com/articles/s41467-024-53029-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-75345-w" target="_blank">
            ü§ñ Prevalence of Symptoms of Dry Eye Disease in Medical Students During a Pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-461', this)">
            More</button></h4>
            <div id="toggle-461" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence of dry eye disease symptoms, associated factors and impact on quality of life among medical students during the pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This cross-sectional study found that a significant number of medical students (60.4%) have dry eye disease (DED) during the COVID-19 pandemic. The prevalence was highest in women (61.5%). 

The study identified several factors that may contribute to the development of DED, including preexisting DED, refractive errors, allergic conjunctivitis, artificial tears use, and a lower EQ-5D-5L score. However, contact lens wear, screen time, and mask-wearing duration were not associated with DED.

Female medical students, people who wore contact lenses, and those who used artificial tears may be more likely to experience DED symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This cross-sectional study aimed to evaluate the prevalence of dry eye disease (DED) symptoms, their associated factors, and the impact on quality of life (QOL) and mental health among medical students during the COVID-19 pandemic using online surveys. Participants completed questionnaires covering demography, medical history, and personal data. The Thai version of the Dry Eye-related Quality-of-Life Score (DEQS-Th) was used for DED screening. The QOL and mental health challenges (support, coping, stress, and depression) were evaluated by the EuroQoL-5 dimensions-5 levels (EQ-5D-5L) and Thymometer questionnaires. A total of 449 participants were analyzed, with a mean age of 21.8 years and 61.5% female. The prevalence of DED symptoms was 60.4% (95%CI 55.7‚Äì64.8). The DEQS-Th score, the EQ-5D-5L score, and all aspects of mental health challenges were significantly worse in DED participants compared to non-DED participants. Associated factors for DED symptoms included preexisting DED (p=‚Äâ0.001), refractive errors (p=‚Äâ0.007), allergic conjunctivitis (p=‚Äâ0.001), artificial tears use (p<‚Äâ0.001), and decreased EQ-5D-5L score (p<‚Äâ0.001). This study highlighted the high prevalence of DED symptoms among medical students during the pandemic and its negative impact on QoL and mental health. Notably, female gender, contact lens wear, screen time, and mask-wearing duration were not associated with DED symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75345-w" target="_blank">https://www.nature.com/articles/s41598-024-75345-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-74221-x" target="_blank">
            ü§ñ ### Immune Response Modeling for Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-462', this)">
            More</button></h4>
            <div id="toggle-462" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A mathematical model simulating the adaptive immune response in various vaccines and vaccination strategies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

A new mathematical model was developed to study how vaccines work on our bodies, helping scientists create better vaccines for infectious diseases. Researchers looked at how different types of vaccines (like traditional inactivated ones or mRNA vaccines) affect the levels of antibodies our immune system produces after vaccination. They found that booster shots are essential for building up strong antibody defenses, and that different vaccine types can have pros and cons. The study provides a better understanding of how vaccines work and helps scientists design more effective vaccines to protect against diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccination has been widely recognized as an effective measure for preventing infectious diseases. To facilitate quantitative research into the activation of adaptive immune responses in the human body by vaccines, it is important to develop an appropriate mathematical model, which can provide valuable guidance for vaccine development. In this study, we constructed a novel mathematical model to simulate the dynamics of antibody levels following vaccination, based on principles from immunology. Our model offers a concise and accurate representation of the kinetics of antibody response. We conducted a comparative analysis of antibody dynamics within the body after administering several common vaccines, including traditional inactivated vaccines, mRNA vaccines, and future attenuated vaccines based on defective interfering viral particles (DVG). Our findings suggest that booster shots play a crucial role in enhancing Immunoglobulin G (IgG) antibody levels, and we provide a detailed discussion on the advantages and disadvantages of different vaccine types. From a mathematical standpoint, our model proposes four essential approaches to guide vaccine design: enhancing antigenic T-cell immunogenicity, directing the production of high-affinity antibodies, reducing the rate of IgG decay, and lowering the peak level of vaccine antigen-antibody complexes. Our study contributes to the understanding of vaccine design and its application by explaining various phenomena and providing guidance in comprehending the interactions between antibodies and antigens during the immune process.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74221-x" target="_blank">https://www.nature.com/articles/s41598-024-74221-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-74704-x" target="_blank">
            ü§ñ Comprehensive Maternal Deaths During COVID-19 Pandemic in Brazil Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-463', this)">
            More</button></h4>
            <div id="toggle-463" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Time series analysis of comprehensive maternal deaths in Brazil during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic significantly impacted maternal mortality rates in Brazil between 2020 and 2021. Using data from two forecasting models, we calculated expected maternal mortality ratios (MMR) for 2020 and 2021 using real-life data since 2008. Our results show that predicted MMR was much higher than actual MMR observed in 2021, with a doubling of the difference between predictions and reality. This suggests that sub-national variations in comprehensive maternal death rates remain significant in Brazil.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The effects of the COVID-19 pandemic on comprehensive maternal deaths in Brazil have not been fully explored. Using publicly available data from the Brazilian Mortality Information (SIM) and Information System on Live Births (SINASC) databases, we used two complementary forecasting models to predict estimates of maternal mortality ratios using maternal deaths (MMR) and comprehensive maternal deaths (MMRc) in the years 2020 and 2021 based on data from 2008 to 2019. We calculated national and regional standardized mortality ratio estimates for maternal deaths (SMR) and comprehensive maternal deaths (SMRc) for 2020 and 2021. The observed MMRc in 2021 was more than double the predicted MMRc based on the Holt-Winters and autoregressive integrated moving average models (127.12 versus 60.89 and 59.12 per 100,000 live births, respectively). We found persisting sub-national variation in comprehensive maternal mortality: SMRc ranged from 1.74  (95% confidence interval [CI] 1.64, 1.86) in the Northeast to 2.70 (95% CI 2.45, 2.96) in the South in 2021. The observed national estimates for comprehensive maternal deaths in 2021 were the highest in Brazil in the past three decades. Increased resources for prenatal care, maternal health, and postpartum care may be needed to reverse the national trend in comprehensive maternal deaths.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74704-x" target="_blank">https://www.nature.com/articles/s41598-024-74704-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s42003-024-07025-4" target="_blank">
            ü§ñ Regulation of Immune Response in COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-464', this)">
            More</button></h4>
            <div id="toggle-464" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>FPR1 signaling aberrantly regulates S100A8/A9 production by CD14+FCN1himacrophages and aggravates pulmonary pathology in severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Excessive inflammation caused by certain proteins, known as alarmins, can worsen immune responses in people infected with SARS-CoV-2. Researchers studied how these proteins affect macrophages (a type of white blood cell) in the lungs after infection and found that they were activated to produce more inflammatory chemicals. They also identified a protein called FPR1 that plays a key role in this process, leading them to hypothesize that it could be targeted for treating severe COVID-19 and post-Acute Respiratory Syndrome (PACS).</p><br>
                <p><strong>Abstract</strong></p>
                <p>Excessive alarmins S100A8/A9 escalate the inflammation and even exacerbate immune-driven thrombosis and multi-organ damage. However, the regulatory mechanisms of S100A8/A9 expression in infectious diseases remain unclear. In this study, high-dimensional transcriptomic data analyses revealed a high proportion of CD14+FCN1himacrophages within the pulmonary niche post-severe SARS-CoV-2 infection. By constructing the S100-coexpression gene list and supervised module scoring, we found that CD14+FCN1himacrophages presented the highest scores of alarmin S100, and possibly served as the trigger and amplifier of inflammation in severe COVID-19. These CD14+FCN1hicells lacked the positive regulatory activity of transcription factor PPARŒ≥, and lost their differentiation ability towards mature macrophages. Ex vivo experiments further validated that the epithelial cells with high ORF-3a expression promoted the expression and secretion of S100A8/A9 through ANXA1/SAA1-FPR1 signaling. S100A8/A9 heterodimers, as well as the co-localization of S100A8/A9 with microtubules, were both diminished by the FPR1 inhibitor. Phospho-kinase protein array indicated that STAT3 promoted transcription, and PLC-Œ≥ and ERK1/2 pathways were involved in the hetero-dimerization and unconventional secretion of S100A8/A9. Our study highlights the pivotal role of FPR1 signaling in the excessive production of S100A8/A9 and provides a promising target for the prevention and control of severe COVID-19 and post-acute COVID-19 sequelae.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07025-4" target="_blank">https://www.nature.com/articles/s42003-024-07025-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-76013-9" target="_blank">
            ü§ñ COVID-19 Patient Discharge Outcomes in Guangdong During the Pandemic Wave</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-465', this)">
            More</button></h4>
            <div id="toggle-465" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The short-term and long-term prognosis of discharged COVID-19 patients in Guangdong during the first wave of pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study investigated the effects of COVID-19 on patients' mental and physical health after recovery from the illness. Researchers followed 281 people who had recovered from the virus to see how their mental and physical well-being changed over time. They used various questionnaires and medical tests to measure these changes. The results showed that many people experienced significant psychological and physical problems, even years after recovering from COVID-19. Factors such as age, gender, severity of illness, hospitalization duration, and previous health conditions were linked to these outcomes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 survivors concerning about the rehabilitation and sustained sequelae of Coronavirus Disease 2019 (COVID-19) infection. We aimed to investigate the sequelae of patients‚Äô psychological and physical condition and its related factors in the early and late stages. This longitudinal study tracked 281 COVID-19 patients discharged from hospitals in Guangdong, China, for one year. Assessments occurred at 2,4,12,24 and 48 weeks post-discharge. We define 2 weeks, 4 weeks, and 12 weeks as early stage, and 24 weeks and 48 weeks as late stage. Psychological health was measured using the Patient Health Questionnaire-9 (PHQ-9), Generalized Anxiety Disorder-7 (GAD-7), Posttraumatic Stress Disorder Checklist for DSM-5 (PCL-5), and Pittsburgh Sleep Quality Index (PSQI) scales. Physical health was assessed through laboratory tests, chest computed tomography (CT) scans, and pulmonary function tests. Data were analyzed using multivariate regression models to evaluate the influence of demographic and clinical variables on health outcomes. COVID-19 survivors exhibited psychological and physical sequelae in both the early and late stages. Compared to the early stage, the proportions of patients with depression (early stage 14.6%, late stage 4.6%), anxiety (early stage 8.9%, late stage 5.3%), PTSD(early stage 3.6%, late stage 0.7%), abnormal liver function (early stage 24.6%, late stage 11.0%), abnormal cardiac function (early stage 10.0%, late stage 7.8%), abnormal renal function (early stage 20.6%, late stage 11.0%) and abnormal pulmonary function (early stage 40.9%, late stage 13.5%) were significantly reduced in the late stage. Factors such as gender, age, severity of COVID-19, hospitalization duration, and various comorbidities were significantly associated with these sequelae. We noticed that psychological and physical sequelae occurred to COVID-19 survivors in short and long stages, and these would gradually decrease as time went on. Male gender, age‚Äâ>‚Äâ50 years old, severe clinical condition, longer hospitalization time and comorbidity history were related factors that significantly affected the rehabilitation of COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76013-9" target="_blank">https://www.nature.com/articles/s41598-024-76013-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-13</strong> <a href="https://www.nature.com/articles/s41598-024-75273-9" target="_blank">
            ü§ñ COVID-19 Vaccine Uptake Analysis

Analyzing how education level affects COVID-19 vaccine adoption in Germany, using area-level socioeconomic deprivation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-466', this)">
            More</button></h4>
            <div id="toggle-466" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reconsidering inequalities in COVID-19 vaccine uptake in Germany: a spatiotemporal analysis combining individual educational level and area-level socioeconomic deprivation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study looked at how different groups received vaccines during the first year of a vaccine campaign in Germany. The researchers found that people with lower levels of education were less likely to get vaccinated, and those living in areas with higher socioeconomic deprivation also had lower vaccination rates. The study found these differences continued over time, even after the pandemic was under control.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Combining the frameworks of fundamental causes theory and diffusion of innovation, scholars had anticipated a delayed COVID-19 vaccination uptake for people in lower socioeconomic position depending on the socioeconomic context. We qualify these propositions and analyze educational differences in COVID-19 vaccination status over the first ten months of Germany‚Äôs vaccination campaign in 2021. Data from the study ‚ÄúCorona Monitoring Nationwide‚Äù (RKI-SOEP-2), collected between November 2021 and February 2022, is linked with district-level data of the German Index of Socioeconomic Deprivation (GISD). We estimated the proportion of people with at least one vaccination dose stratified by educational groups and within different settings of regional socioeconomic deprivation at three time points. Logistic multilevel regression models were applied to adjust for multiple covariates and to test cross-level-interactions between educational levels and levels of area-level socioeconomic deprivation. Vaccination rates were lower among respondents with lower education. With increasing area-level socioeconomic deprivation, educational differences were larger due to particularly low vaccination rates in groups with low education levels. The analysis of vaccination timing reveals that educational gaps and gaps by area-level socioeconomic deprivation had appeared early in the vaccination campaign and did not close completely before the 4th wave of COVID-19 infections</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75273-9" target="_blank">https://www.nature.com/articles/s41598-024-75273-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-11</strong> <a href="https://www.nature.com/articles/s41598-024-75833-z" target="_blank">
            ü§ñ Observational Study on Vaccine Doses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-467', this)">
            More</button></h4>
            <div id="toggle-467" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Side effects associated with homogenous and heterogenous doses of Oxford‚ÄìAstraZeneca vaccine among adults in Bangladesh: an observational study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Assessment of Side Effects in COVID-19 Vaccination**

In a large-scale study of over 1805 people who received COVID-19 vaccine doses in Bangladesh, researchers found significant side effects associated with vaccination. The study involved administering multiple doses to participants and observing the severity of local and systemic reactions.

The most common side effects reported were pain at the injection site, swelling, redness, and heaviness in the hand, fever, headache, muscle pain, chills, and myalgia. These symptoms appeared within 48 hours after vaccination. Thrombosis was only reported among a small group of people who received Oxford-AstraZeneca vaccine.

Systemic side effects, such as chronic urticaria (itching) and psoriasis, were also observed in some participants. The study found that the risk of severe and long-term side effects increased with each subsequent dose of vaccination. Women over 60 years old and those with pre-existing health conditions had higher risks.

The researchers analyzed data from multiple vaccination doses and found that Oxford-AstraZeneca vaccine was associated with a significantly higher rate of adverse effects than the other vaccines (P < .05). They also noted that homogenous doses of Oxford-AstraZeneca and heterogenous doses of Moderna and Pfizer-BioNTech were more likely to have elevated side effect rates.

This study provides insight into the safety profile of COVID-19 vaccines, highlighting potential risks associated with vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Assessment of side effects associated with COVID-19 vaccination is required to monitor safety issues and acceptance of vaccines in the long term. We found a significant knowledge gap in the safety profile of COVID-19 vaccines in Bangladesh. We enrolled 1805 vaccine recipients from May 5, 2021, to April 4, 2023. Kruskal-Wallis test and œá2 test were performed. Multivariable logistic regression was also performed. First, second and third doses were administered among 1805, 1341, and 923 participants, respectively. Oxford‚ÄìAstraZeneca (2946 doses) was the highest administered followed by Sinopharm BIBP (551 doses), Sinovac (214 doses), Pfizer-BioNTech (198 doses), and Moderna (160 doses), respectively. Pain at the injection site (80-90%, 3200‚Äì3600), swelling (85%, 3458), redness (78%, 3168), and heaviness in hand (65%, 2645) were the most common local effects, and fever (85%, 3458), headache (82%, 3336), myalgia (70%, 2848), chills (67%, 2726), muscle pain (60%, 2441) were the most prevalent systemic side effects reported within 48¬†h of vaccination. Thrombosis was only reported among the Oxford‚ÄìAstraZeneca recipients (3.5-5.7%). Both local and systemic effects were significantly associated with the Oxford‚ÄìAstraZeneca (p-value<‚Äâ0.05), Pfizer‚ÄìBioNTech (p-value<‚Äâ0.05), and Moderna (p-value<‚Äâ0.05) vaccination. Chronic urticaria and psoriasis were reported by 55-60% of the recipients after six months or later. The highest percentage of local and systemic effects after 2nd and 3rd dose were found among recipients of Moderna followed by Pfizer-BioNTech and Oxford‚ÄìAstraZeneca. Homogenous doses of Oxford‚ÄìAstraZeneca and heterogenous doses of Moderna and Pfizer-BioNTech were significantly associated with elevated adverse effects. Females, aged above 60 years with preexisting health conditions had higher risks. Vaccination with Pfizer-BioNTech (OR 4.34, 95% CI 3.95‚Äì4.58) had the highest odds of severe and long-term effects followed by Moderna (OR 4.15, 95% CI 3.92‚Äì4.69) and Oxford‚ÄìAstraZeneca (OR 3.89, 95% CI 3.45‚Äì4.06), respectively. This study will provide an integrated insight into the safety profile of COVID-19 vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75833-z" target="_blank">https://www.nature.com/articles/s41598-024-75833-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-10</strong> <a href="https://www.nature.com/articles/s41467-024-53105-8" target="_blank">
            ü§ñ Protein sequencing of antibodies to identify those that provide protection against COVID-19 in vaccinated individuals.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-468', this)">
            More</button></h4>
            <div id="toggle-468" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>De novo protein sequencing of antibodies for identification of neutralizing antibodies in human plasma post SARS-CoV-2 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists have developed a way to sequence (read) individual proteins in blood plasma, known as immunoglobulins or antibodies. They used this method to look at how the human body responds to a vaccine against COVID-19. By sequencing these proteins, they were able to identify 12 different antibodies produced by the immune system in response to the vaccine. These antibodies have shown promising properties, such as neutralizing the target virus, which could lead to more effective treatments and vaccines in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The antibody response to vaccination and infection is a key component of the immune response to pathogens. Sequencing of peripheral B cells may not represent the complete B cell receptor repertoire. Here we present a method for sequencing human plasma-derived polyclonal IgG using a combination of mass spectrometry and B-cell sequencing. We investigate the IgG response to the Moderna Spikevax COVID-19 vaccine. From the sequencing data of the natural polyclonal response to vaccination, we generate 12 recombinant antibodies. Six derived recombinant antibodies, including four generated with de novo protein sequencing, exhibit similar or higher binding affinities than the original natural polyclonal antibody. Neutralization tests reveal that the six antibodies possess neutralizing capabilities against the target antigen. This research provides insights into sequencing polyclonal IgG antibodies and the potential of our approach in generating recombinant antibodies with robust binding affinity and neutralization capabilities. Directly examining the circulating IgG pool is crucial due to potential misrepresentations by B-cell analysis alone.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53105-8" target="_blank">https://www.nature.com/articles/s41467-024-53105-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41467-024-53033-7" target="_blank">
            ü§ñ Co-infection with SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3.

Lineage-specific pathogenicity, immune evasion, and virological features of these two strains.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-469', this)">
            More</button></h4>
            <div id="toggle-469" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Lineage-specific pathogenicity, immune evasion, and virological features of SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 JN.1 variant has higher infectivity in human nasal epithelial cells compared to its parent BA.2.86. This variation associates with increased entry efficiency and better spike cleavage. It alters the mode of binding between the protein and ACE2, and increases hydrogen bonds with neighboring residues, making it easier for the virus to enter cells.

This variant is more transmissible than BA.2.86, suggesting that a single mutation makes it harder for people's immune systems to fight off. By studying this variation in both laboratory tests and actual human viruses, scientists can gain better understanding of its effects on different parts of the body and how it contributes to SARS-CoV-2's spread.

A key feature of this variant is an extra amino acid (L455S) that allows it to enter cells more easily. This mutation likely makes people less able to fight off the virus, which can contribute to its dominance over other variants. Further research will help scientists better understand how SARS-CoV-2 JN.1 and its parent BA.2.86 work together to spread disease globally.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 JN.1 with an additional L455S mutation on spike when compared with its parental variant BA.2.86 has outcompeted all earlier variants to become the dominant circulating variant. Recent studies investigated the immune resistance of SARS-CoV-2 JN.1 but additional factors are speculated to contribute to its global dominance, which remain elusive until today. Here, we find that SARS-CoV-2 JN.1 has a higher infectivity than BA.2.86 in differentiated primary human nasal epithelial cells (hNECs). Mechanistically, we demonstrate that the gained infectivity of SARS-CoV-2 JN.1 over BA.2.86 associates with increased entry efficiency conferred by L455S and better spike cleavage in hNECs. Structurally, S455 altered the mode of binding of JN.1 spike protein to ACE2 when compared to BA.2.86 spike at ACE2H34, and modified the internal structure of JN.1 spike protein by increasing the number of hydrogen bonds with neighboring residues. These findings indicate that a single mutation (L455S) enhances virus entry in hNECs and increases immune evasiveness, which contribute to the robust transmissibility of SARS-CoV-2 JN.1. We further evaluate the in vitro and in vivo virological characteristics between SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3, and identify key lineage-specific features of the two Omicron sublineages that contribute to our understanding on Omicron antigenicity, transmissibility, and pathogenicity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53033-7" target="_blank">https://www.nature.com/articles/s41467-024-53033-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-74967-4" target="_blank">
            ü§ñ Predictive Modeling for Hemodialysis Patients with SARS-CoV-2 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-470', this)">
            More</button></h4>
            <div id="toggle-470" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting SARS-CoV-2 infection among hemodialysis patients using deep neural network methods</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract that a young person can understand:

A new study has found that dialysis patients are more likely to get COVID-19 than people without this condition. To control the spread of the virus and minimize outbreaks, healthcare centers use predictive models that identify infected patients early on. Researchers used data from various sources to create special models that can predict when a patient is most likely to be infected with COVID-19. These models were tested on real data from 2020, and they showed promising results. The research team found that these new models are more accurate than existing ones at identifying who will get the virus during its incubation period. They also identified key factors that can help healthcare providers diagnose patients early and take steps to protect them and others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 has a higher rate of morbidity and mortality among dialysis patients than the general population. Identifying infected patients early with the support of predictive models helps dialysis centers implement concerted procedures (e.g., temperature screenings, universal masking, isolation treatments) to control the spread of SARS-CoV-2 and mitigate outbreaks. We collect data from multiple sources, including demographics, clinical, treatment, laboratory, vaccination, socioeconomic status, and COVID-19 surveillance. Previous early prediction models, such as logistic regression, SVM, and XGBoost, require sophisticated feature engineering and need improved prediction performance. We create deep learning models, including Recurrent Neural Networks (RNN) and Convolutional Neural Networks (CNN), to predict SARS-CoV-2 infections during incubation. Our study shows deep learning models with minimal feature engineering can identify those infected patients more accurately than previously built models. Our Long Short-Term Memory (LSTM) model consistently performed well, with an AUC exceeding 0.80, peaking at 0.91 in August 2021. The CNN model also demonstrated strong results with an AUC above 0.75. Both models outperformed previous best XGBoost models by over 0.10 in AUC. Prediction accuracy declined as the pandemic evolved, dropping to approximately 0.75 between September 2021 and January 2022. Maintaining a 20% false positive rate, our LSTM and CNN models identified 66% and 64% of positive cases among patients, significantly outperforming XGBoost models at 42%. We also identify key features for dialysis patients by calculating the gradient of the output with respect to the input features. By closely monitoring these factors, dialysis patients can receive earlier diagnoses and care, leading to less severe outcomes. Our research highlights the effectiveness of deep neural networks in analyzing longitudinal data, especially in predicting COVID-19 infections during the crucial incubation period. These deep network approaches surpass traditional methods relying on aggregated variable means, significantly improving the accurate identification of SARS-CoV-2 infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74967-4" target="_blank">https://www.nature.com/articles/s41598-024-74967-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41467-024-52890-6" target="_blank">
            ü§ñ Genome-Wide Study on COVID-19 Vaccine Seroconversion and Breakthrough Outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-471', this)">
            More</button></h4>
            <div id="toggle-471" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genome-wide association studies of COVID-19 vaccine seroconversion and breakthrough outcomes in UK Biobank</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding how the immune system responds to different COVID-19 vaccine types is crucial for finding out which genes are most important. This study used a large database of genetic information from people who were vaccinated with different vaccines to find out which specific parts of their DNA affect whether they get better or worse symptoms after vaccination, and how severe those symptoms are. The results showed that certain areas on the immune system respond differently depending on the vaccine type. Additionally, some regions are more likely to cause severe symptoms even when vaccinated. This study helps scientists understand how vaccines work and what other genetic factors may influence how people respond to different types of COVID-19 vaccinations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the genetic basis of COVID-19 vaccine seroconversion is crucial to study the role of genetics on vaccine effectiveness. In our study, we used UK Biobank data to find the genetic determinants of COVID-19 vaccine-induced seropositivity and breakthrough infections. We conducted four genome-wide association studies among vaccinated participants for COVID-19 vaccine seroconversion and breakthrough susceptibility and severity. Our findings confirmed a link between theHLAregion and seroconversion after the first and second doses. Additionally, we identified 10 genomic regions associated with breakthrough infection (SLC6A20, ST6GAL1, MUC16, FUT6, MXI1, MUC4,HMGN2P18-KRTCAP2,NFKBIZandAPOC1), and one with breakthrough severity (APOE). No significant evidence of genetic colocalisation was found between those traits. Our study highlights the roles of individual genetic make-up in the varied antibody responses to COVID-19 vaccines and provides insights into the potential mechanisms behind breakthrough infections occurred even after the vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52890-6" target="_blank">https://www.nature.com/articles/s41467-024-52890-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41541-024-00972-3" target="_blank">
            ü§ñ COVID-19 vaccine response and transmission risk in people with HIV</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-472', this)">
            More</button></h4>
            <div id="toggle-472" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Vaccine responses and hybrid immunity in people living with HIV after SARS-CoV-2 breakthrough infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic presented a challenge for people with HIV (PLWH), especially those who have weakened immune systems due to antiretroviral therapy (ART). The virus can cause significant problems, including an increased risk of breakthrough infections (BTI) from the SARS-CoV-2 virus. A new study found that PLWH had reduced responses to vaccines and may not be protected against BTI. However, the immune system's response changed when it comes to the COVID-19 vaccine. This change is significant because people with weakened immune systems can still fight off the virus even if their immune response isn't as strong. The study suggests that additional boosters might help and that regular monitoring of HIV treatment is necessary to ensure that both humoral (vaccine-related) and cellular immunity remains effective against the SARS-CoV-2 virus in people with weakened immune systems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic posed a challenge for people living with HIV (PLWH), particularly immune non-responders (INR) with compromised CD4 T-cell reconstitution following antiretroviral therapy (CD4 count <350 cells per mm3). Their diminished vaccine responses raised concerns about their vulnerability to SARS-CoV-2 breakthrough infections (BTI). Our in-depth study here revealed chronic inflammation in PLWH and a limited anti-Spike IgG response after vaccination in INR. Nevertheless, the imprinting of Spike-specific B cells by vaccination significantly enhanced the humoral responses after BTI. Notably, the magnitude of cellular CD4 response in all PLWH was comparable to that in healthy donors (HD). However, the polyfunctionality and phenotype of Spike-specific CD8 T cells in INR differed from controls. The findings highlight the need for additional boosters with variant vaccines, and for monitoring ART adherence and the durability of both humoral and cellular anti-SARS-CoV-2 immunity in INR.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00972-3" target="_blank">https://www.nature.com/articles/s41541-024-00972-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-75093-x" target="_blank">
            ü§ñ Enhanced immune function in COVID-19 patients through transfusion of convalescent plasma with high viral titres.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-473', this)">
            More</button></h4>
            <div id="toggle-473" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Biochemical rationale for transfusion of high titre COVID-19 convalescent plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We developed an immunosorbent assay to measure how well donor antibodies bind to viruses that cause COVID-19 after receiving plasma from people who have recovered from the disease. We found two different affinities of these antibodies: 80% for IgG and about half for IgA. The binding strength was highest at a concentration of around 150 nanomoles per milliliter, which is higher than what's needed to be effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed to model binding of donor antibodies to virus that infects COVID-19 patients following transfusion of convalescent plasma (CCP). An immunosorbent assay was developed to determine apparent affinity (Kd, app). Antibody binding to virus was modelled using antibody concentration and estimations of viral load. Assay and model were validated using reference antibodies and clinical data of monoclonal antibody therapy. A single Kd, appor two resolvable Kd, appwere found for IgG in 11% or 89% of CCP donations, respectively. For IgA this was 50%-50%. Median IgG Kd, appwas 0.8nM and 3.6nM for IgA, ranging from 0.1-14.7nM and 0.2-156.0nM respectively. The median concentration of IgG was 44.0nM (range 8.4-269.0nM) and significantly higher than IgA at 2.0nM (range 0.4-11.4nM). The model suggested that a double CCP transfusion (i.e. 500 mL) allows for >‚Äâ80% binding of antibody to virus provided Kd, appwas <‚Äâ1nM and concentration‚Äâ>‚Äâ150nM. In our cohort from the pre-vaccination era, 4% of donations fulfilled these criteria. Low and mid-range viral loads are found early post exposure, suggesting that convalescent plasma will be most effective then. This study provides a biochemical rationale for selecting high affinity and high antibody concentration CCP transfused early in the disease course.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75093-x" target="_blank">https://www.nature.com/articles/s41598-024-75093-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-74629-5" target="_blank">
            ü§ñ Post-infectious Olfactory Dysfunction in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-474', this)">
            More</button></h4>
            <div id="toggle-474" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative analyses of post-infectious olfactory dysfunction between COVID-19 and non-COVID-19 cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research identifies differences between people who develop olfactory dysfunction (OD) after a COVID-19 infection and those without it.

Demographic and clinical characteristics of patients with PIOD are analyzed to compare their experiences. The study found that:

* A higher proportion of COVID-19 cases were younger, more likely male, and had parosmia.
* Initial olfactory function tests showed differences between the groups.
* Treatment improved odor detection threshold but not identification in COVID-19-associated OD.
* Non-COVID-19-associated OD had better initial results.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To identify the differences between COVID-19-associated and non-COVID-19-associated olfactory dysfunction (OD), we analyzed demographic and clinical characteristics based on the causative virus (COVID versus non-COVID groups) in patients with post-infectious olfactory dysfunction (PIOD) who underwent the olfactory questionnaire and olfactory function test. Out of 169 patients with PIOD, 99 were diagnosed with COVID-19 (COVID group), while 70 were not (non-COVID group). The COVID group was younger and had a higher percentage of male patients as well as patients with parosmia than the non-COVID group. In the initial olfactory function tests, the TDI, discrimination and identification scores were significantly higher in the COVID group than in the non-COVID group. TDI scores were significantly increased in patients with PIOD after treatment, regardless of the group. The threshold score was significantly increased by 1.38 in the COVID group while the identification score was significantly increased by 2.67 in the non-COVID group. Patients with COVID-19-associated OD were younger in age, tended to be male, had a higher incidence of parosmia, and had better initial olfactory function test results compared to those with non-COVID-19-associated OD. Following treatment, odor detection threshold improved in the COVID group, whereas odor identification improved in the non-COVID group.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74629-5" target="_blank">https://www.nature.com/articles/s41598-024-74629-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41467-024-52818-0" target="_blank">
            ü§ñ Proteins involved in viral infection trigger cell release of inflammation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-475', this)">
            More</button></h4>
            <div id="toggle-475" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Coronavirus envelope protein activates TMED10-mediated unconventional secretion of inflammatory factors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Coronavirus proteins can trigger a strong inflammatory response in the body. Scientists have identified a specific protein called the envelope (E) protein on some coronaviruses that is particularly effective at causing this inflammation. In contrast, other types of coronaviruses cause less severe reactions. They found that when these E proteins are present, they help to release chemicals into the body's cells that trigger more inflammation. Researchers also discovered a molecule called progesterone that can block this process and reduce inflammation in some animal studies. This discovery suggests that targeting the E protein could be an effective way to prevent severe coronavirus infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The precise cellular mechanisms underlying heightened proinflammatory cytokine production during coronavirus infection remain incompletely understood. Here we identify the envelope (E) protein in severe coronaviruses (SARS-CoV-2, SARS, or MERS) as a potent inducer of interleukin-1 release, intensifying lung inflammation through the activation of TMED10-mediated unconventional protein secretion (UcPS). In contrast, the E protein of mild coronaviruses (229E, HKU1, or OC43) demonstrates a less pronounced effect. The E protein of severe coronaviruses contains an SS/DS motif, which is not present in milder strains and facilitates interaction with TMED10. This interaction enhances TMED10-oligomerization, facilitating UcPS cargo translocation into the ER-Golgi intermediate compartment (ERGIC)‚Äîa pivotal step in interleukin-1 UcPS. Progesterone analogues were identified as compounds inhibiting E-enhanced release of proinflammatory factors and lung inflammation in a Mouse Hepatitis Virus (MHV) infection model. These findings elucidate a molecular mechanism driving coronavirus-induced hyperinflammation, proposing the E-TMED10 interaction as a potential therapeutic target to counteract the adverse effects of coronavirus-induced inflammation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52818-0" target="_blank">https://www.nature.com/articles/s41467-024-52818-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41598-024-75233-3" target="_blank">
            ü§ñ Long-term COVID complications after initial COVID-19 illness in Thailand</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-476', this)">
            More</button></h4>
            <div id="toggle-476" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID at 3 and 6 months after covid-19 infection in Thailand</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that a significant number of people who contracted COVID-19 continue to experience symptoms after four weeks. The prevalence of these symptoms varies by country and over time - they can occur as early as three months (36.2%) and up to six months (33.7%). Factors such as gender, disease severity, and whether the person was infected with a mild or severe case also influence when symptoms start and how long they last.</p><br>
                <p><strong>Abstract</strong></p>
                <p>‚ÄúLong COVID‚Äù, which describes a diverse set of symptoms or conditions that persist or develop after four weeks from the onset of initial SARS-CoV-2 infection has been introduced. However, the true prevalence varies worldwide. This study aimed to determine the point prevalence and clinical characteristics of long COVID at three and six months after acute COVID-19 infection in Thailand. Methods All adult patients who were diagnosed with COVID-19 by positive nasopharyngeal RT-PCR for SARS-CoV-2 at Thammasat University Hospital between October and December 2021 were recruited and followed for long COVID symptoms by telephone interviews at 3 and 6 months after an acute infection. Among 1,400 eligible COVID-19 cases, interviews were complete for 1,129 and 932 individuals at 3 and 6 months, respectively. Of those, 431 and 314 reported at least one symptom consistent with long COVID. The point prevalence was 38.2% (95% confidence interval: 35.3‚Äì41.1%) and 33.7% (95% confidence interval: 30.7‚Äì36.7%) respectively. Female gender, disease severity, and symptomatic acute infection were identified as independent risk factors. Conclusion Based on the reported symptoms, long COVID is commonly observed either at 3 or 6 months in our study.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75233-3" target="_blank">https://www.nature.com/articles/s41598-024-75233-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41598-024-73948-x" target="_blank">
            ü§ñ Infection with SARS-CoV-2 can cause a condition known as acute respiratory distress syndrome (ARDS) that can lead to shock, which can reduce blood flow to vital organs and affect the heart.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-477', this)">
            More</button></h4>
            <div id="toggle-477" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Presence of cholestasis and its impact on survival in SARS-CoV-2 associated acute respiratory distress syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study has been conducted to understand the relationship between COVID-19, biliary injury, and cholestasis in critically ill patients. 

Researchers analyzed data from 225 COVID-19 patients who were admitted to an ICU at a hospital in Austria. They found that nearly half (53%) of these patients developed cholestasis, which is indicated by an abnormal level of alkaline phosphatase. 

Cholestasis was associated with higher levels of inflammation and liver damage markers. Patients with cholestasis also had worse survival rates during the ICU stay and after six months.

The study suggests that developing cholestasis in COVID-19 patients can be a negative prognostic marker, indicating a poorer outcome.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Data on cholestasis and biliary injury in patients with COVID-19 are scarce. The primary aim of this study was to evaluate the prevalence of cholestasis and factors associated with its development and outcome in critically ill patients with COVID-19 associated acute respiratory distress syndrome (ARDS). In this retrospective exploratory study, COVID-19 patients with ARDS admitted to an intensive care unit (ICU) at the Medical University of Vienna were evaluated for the development of cholestasis defined as an alkaline phosphatase level of 1.67x upper limit of normal for at least three consecutive days. Simple and multiple logistic regression analysis was used to evaluate parameters associated with development of cholestasis and survival. Of 225 included patients 119 (53%) developed cholestasis during ICU stay. Patients with cholestasis had higher peak levels of alkaline phosphatase, gamma-glutamyl transferase, bilirubin and inflammation parameters. Factors independently associated with cholestasis were extracorporeal membrane oxygenation support, ketamine use, high levels of inflammation parameters and disease severity. Presence of cholestasis and peak ALP levels were independently associated with worse ICU and 6-month survival. Development of cholestasis is a common complication in critically ill COVID-19 patients and represents a negative prognostic marker for survival. It is associated with disease severity and specific treatment modalities of intensive care.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73948-x" target="_blank">https://www.nature.com/articles/s41598-024-73948-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41467-024-52808-2" target="_blank">
            ü§ñ Immune System Response to SARS-CoV-2 Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-478', this)">
            More</button></h4>
            <div id="toggle-478" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structural basis for receptor-binding domain mobility of the spike in SARS-CoV-2 BA.2.86 and JN.1</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Since 2019, a new virus called SARS-CoV-2 has caused multiple waves of illness and hospitalizations. The virus's spike protein plays a crucial role in entering cells, but its behavior changes depending on how it's shaped. Researchers have been studying the shape of this protein to understand how it interacts with receptors on cell surfaces.

Using detailed computer models and biological samples, scientists found that when the spike protein is correctly shaped as expected, it can bind to a specific receptor called ACE2 more easily. However, they also observed that the virus's spike protein can change shape and interact with its host better when this binding process begins. These changes help prevent the virus from being stopped by antibodies that are designed to recognize and neutralize other forms of the virus.

Understanding how these viral proteins work could lead to new treatments for COVID-19 and other illnesses caused by similar viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Since 2019, SARS-CoV-2 has undergone mutations, resulting in pandemic and epidemic waves. The SARS-CoV-2 spike protein, crucial for cellular entry, binds to the ACE2 receptor exclusively when its receptor-binding domain (RBD) adopts the up-conformation. However, whether ACE2 also interacts with the RBD in the down-conformation to facilitate the conformational shift to RBD-up remains unclear. Herein, we present the structures of the BA.2.86 and the JN.1 spike proteins bound to ACE2. Notably, we successfully observed the ACE2-bound down-RBD, indicating an intermediate structure before the RBD-up conformation. The wider and mobile angle of RBDs in the up-state provides space for ACE2 to interact with the down-RBD, facilitating the transition to the RBD-up state. The K356T, but not N354-linked glycan, contributes to both of infectivity and neutralizing-antibody evasion in BA.2.86. These structural insights the spike-protein dynamics would help understand the mechanisms underlying SARS-CoV-2 infection and its neutralization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52808-2" target="_blank">https://www.nature.com/articles/s41467-024-52808-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41598-024-73004-8" target="_blank">
            ü§ñ COVID-19 Vaccine Booster Response in Nursing Home Residents</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-479', this)">
            More</button></h4>
            <div id="toggle-479" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 mRNA booster vaccination induces robust antibody responses but few adverse events among SARS-CoV-2 na√Øve nursing home residents</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Residents in nursing homes face higher risks from COVID-19. We studied who got vaccinated with a COVID-19 mRNA vaccine and how their bodies reacted to it. Our study included 95 people who had never been infected with SARS-CoV-2 before, including 69 older adults. Nobody reported serious side effects like heart attacks or seizures. However, some of the less severe side effects were more common in nurses than doctors. We found that those who got vaccinated had stronger immune systems and fewer problems compared to people who didn't get vaccinated at all.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Residents in nursing homes face heightened COVID-19 risks. We aimed to assess the adverse events (AEs) rates and antibody responses after the first to the fifth dose of COVID-19 mRNA vaccination in a nursing home cohort. Ninety-five SARS-CoV-2 na√Øve participants consisted of 26 staff (median age, 51 years) and 69 residents (median age, 88 years). Life-threatening AEs were reported in neither residents nor staff. The severity of non-life-threatening AEs was graded, and severe AEs were reported only in staff. The AEs rates were considerably lower in residents, compared to those in staff. Anti-RBD IgG and the neutralizing titers (NTs) against Wuhan and Omicron BA.4/BA.5 did not differ significantly between those with ‚Äòany AE‚Äô and ‚Äòno AE‚Äô among both staff and residents two months after the second, third and fifth doses, while the anti-RBD IgG significantly differed between two groups after third dose in residents. These findings suggest that the anti-RBD IgG and the NTs increase regardless of the occurrence of AEs. Our study underscores a robust antibody response in both in staff and residents, and fewer AEs following COVID-19 vaccination in SARS-CoV-2 na√Øve residents than staff, supporting the recommendation for mRNA booster doses in older adults at high-risk care facilities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73004-8" target="_blank">https://www.nature.com/articles/s41598-024-73004-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41541-024-00980-3" target="_blank">
            ü§ñ Moderna COVID-19 vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-480', this)">
            More</button></h4>
            <div id="toggle-480" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preclinical immunogenicity and safety of hemagglutinin-encoding modRNA influenza vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Seasonal flu epidemics affect a lot of people worldwide due to influenza viruses. Vaccines help prevent infection but need updating every year because they can change over time. Researchers created a new type of vaccine that can produce more effective and quicker results, using an existing technology from another vaccine called COVID-19 mRNA vaccine. This new vaccine is made up of tiny genetic material (RNA) that helps the body's immune system recognize and fight off certain flu viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Seasonal epidemics of influenza viruses are responsible for a significant global public health burden. Vaccination remains the most effective way to prevent infection; however, due to the persistence of antigenic drift, vaccines must be updated annually. The selection of vaccine strains occurs months in advance of the influenza season to allow adequate time for production in eggs. RNA vaccines offer the potential to accelerate production and improve efficacy of influenza vaccines. We leveraged the nucleoside-modified RNA (modRNA) platform technology and lipid nanoparticle formulation process of the COVID-19 mRNA vaccine (BNT162b2; Comirnaty¬Æ) to create modRNA vaccines encoding hemagglutinin (HA) (modRNA-HA) for seasonal human influenza strains and evaluated their preclinical immunogenicity and toxicity. In mice, a monovalent modRNA vaccine encoding an H1 HA demonstrated robust antibody responses, HA-specific Th1-type CD4+T cell responses, and HA-specific CD8+T cell responses. In rhesus and cynomolgus macaques, the vaccine exhibited durable functional antibody responses and HA-specific IFN-Œ≥+CD4+T cell responses. Immunization of mice with monovalent, trivalent, and quadrivalent modRNA-HA vaccines generated functional antibody responses targeting the seasonal influenza virus(es) encoded in the vaccines that were greater than, or similar to, those of a licensed quadrivalent influenza vaccine. Monovalent and quadrivalent modRNA-HA vaccines were well-tolerated by Wistar Han rats, with no evidence of systemic toxicity. These nonclinical immunogenicity and safety data support further evaluation of the modRNA-HA vaccines in clinical studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00980-3" target="_blank">https://www.nature.com/articles/s41541-024-00980-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-06</strong> <a href="https://www.nature.com/articles/s41598-024-73765-2" target="_blank">
            ü§ñ Energy Metabolism in Hamsters Affected by COVID-19 Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-481', this)">
            More</button></h4>
            <div id="toggle-481" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evidence for SARS-CoV-2 infected Golden Syrian hamsters (Mesocricetus auratus) reducing daily energy expenditure and body core temperature</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

Golden Syrian hamsters are used to test how they react to infections with a coronavirus called SARS-CoV-2. To see what happens when these hamsters get infected, researchers studied their physical changes over time after infection. They looked at things like energy expenditure, water intake, weight, body temperature, and activity levels in both healthy and sick hamsters. The results showed that infected hamsters had less energy and water than non-infected ones, which could be a sign of problems with their internal organs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Golden Syrian hamsters (Mesocricetus auratus) are a well-established animal model for human infections with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) due to their susceptibility to SARS-CoV-2 infection, robust virus replication and pathological manifestations similar to human COVID-19 pneumonia. To investigate the physiological changes upon infection in this animal model, we explored the alterations in daily energy expenditure (DEE), water turnover, body mass, body temperature, and locomotor activity in non-infected and SARS-CoV-2 infected Golden Syrian hamsters for four days post SARS-CoV-2. DEE was measured using the doubly labelled water method, which allows for the accurate estimation of carbon dioxide production and, consequently, energy expenditure in animals. Additionally, we investigated total water intake (TWI), which comprises drinking water, preformed water in food, and metabolic water. Using intraperitoneally implanted data loggers, we also monitored body core temperature and locomotor activity in some of the animals. Here we provide evidence for infected hamsters exhibiting significantly lower DEE and TWI compared to non-infected animals. We also observed an increase in body weight in the non-infected animals, while infected animals experienced weight loss. Further, infected animals showed a significantly decreased body temperature, indicating a generally lowered metabolic rate.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73765-2" target="_blank">https://www.nature.com/articles/s41598-024-73765-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-06</strong> <a href="https://www.nature.com/articles/s41598-024-74104-1" target="_blank">
            ü§ñ COVID-19 Detection from Exhaled Breath

A new method to detect COVID-19 has been developed that measures the presence of the virus in a person's respiratory system. This method involves analyzing the breath, which contains tiny particles left behind by the SARS-CoV-2 virus when it enters the lungs.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-482', this)">
            More</button></h4>
            <div id="toggle-482" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 detection from exhaled breath</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new method for detecting COVID-19 has been developed. This system uses a simple device that collects breath samples from the air, then analyzes them using advanced technology. The system is non-invasive and can identify people who have COVID-19 with high accuracy. An initial trial of 302 participants showed promising results, achieving similar accuracy to traditional methods for detecting COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 coronavirus emerged in 2019 causing a COVID-19 pandemic that resulted in 7 million deaths out of 770 million reported cases over the next 4 years. The global health emergency called for unprecedented efforts to monitor and reduce the rate of infection, pushing the study of new diagnostic methods. In this paper, we introduce a cheap, fast, and non-invasive COVID-19 detection system, which exploits only exhaled breath. Specifically, provided an air sample, the mass spectra in the 10‚Äì351 mass-to-charge range are measured using an original micro and nano-sampling device coupled with a high-precision spectrometer; then, the raw spectra are processed by custom software algorithms; the clean and augmented data are eventually classified using state-of-the-art machine-learning algorithms. An uncontrolled clinical trial was conducted between 2021 and 2022 on 302 subjects who were concerned about being infected, either due to exhibiting symptoms or having recently recovered from illness. Despite the simplicity of use, our system showed a performance comparable to the traditional polymerase-chain-reaction and antigen testing in identifying cases of COVID-19 (that is, 95% accuracy, 94% recall, 96% specificity, and 92%\(F_1\)-score). In light of these outcomes, we think that the proposed system holds the potential for substantial contributions to routine screenings and expedited responses during future epidemics, as it yields results comparable to state-of-the-art methods, providing them in a more rapid and less invasive manner.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74104-1" target="_blank">https://www.nature.com/articles/s41598-024-74104-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41467-024-52504-1" target="_blank">
            ü§ñ Adaptive Weight Ensemble for Influenza Forecasting</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-483', this)">
            More</button></h4>
            <div id="toggle-483" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An adaptive weight ensemble approach to forecast influenza activity in an irregular seasonality context</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract Simplified

Forecasting flu activity in places like Hong Kong is tricky because the flu doesn't always follow a normal pattern. We used different types of smart tools to predict how the flu will spread up to 8 weeks ahead. We looked at data from many years, starting from 1998.

We made groups of simple and complex models together (ensemble), which all did better than just guessing there would be no flu. The best group reduced errors by 23% to 29%. Another special way we blended the tools worked even better, reducing mistakes by 52%, which was much better at predicting different types of flu patterns.

We tested this again after a big flu season (post-COVID), and it still helped reduce mistakes. Our system lets us compare different models and pick the best one for forecasting flu in places where the flu doesn't always come when expected.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Forecasting influenza activity in tropical and subtropical regions, such as Hong Kong, is challenging due to irregular seasonality and high variability. We develop a diverse set of statistical, machine learning, and deep learning approaches to forecast influenza activity in Hong Kong 0 to 8 weeks ahead, leveraging a unique multi-year surveillance record spanning 32 epidemics from 1998 to 2019. We consider a simple average ensemble (SAE) of the top two individual models, and develop an adaptive weight blending ensemble (AWBE) that dynamically updates model contribution. All models outperform the baseline constant incidence model, reducing the root mean square error (RMSE) by 23%‚Äì29% and weighted interval score (WIS) by 25%‚Äì31% for 8-week ahead forecasts. The SAE model performed similarly to individual models, while the AWBE model reduces RMSE by 52% and WIS by 53%, outperforming individual models for forecasts in different epidemic trends (growth, plateau, decline) and during both winter and summer seasons. Using the post-COVID data (2023‚Äì2024) as another test period, the AWBE model still reduces RMSE by 39% and WIS by 45%. Our framework contributes to comparing and benchmarking models in ensemble forecasts, enhancing evidence for synthesizing multiple models in disease forecasting for geographies with irregular influenza seasonality.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52504-1" target="_blank">https://www.nature.com/articles/s41467-024-52504-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41598-024-73114-3" target="_blank">
            ü§ñ Rational Design for SARS-CoV-2 Spike Protein Epitopes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-484', this)">
            More</button></h4>
            <div id="toggle-484" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rational design based on multi-monomer simultaneous docking for epitope imprinting of SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Molecularly imprinted polymers (MIPs) are like designer plastic molecules that mimic how antibodies interact with antigens found in nature. These plastics can be used to catch viruses like the one causing COVID-19.

Recently, scientists developed a new way to make these MIPs using computer simulations and experiments. They looked at a part of the virus called Spike protein to find the best places for their plastic molecules to attach.

They tested many different combinations of building blocks until they found the ones that worked best. The results showed that the designer plastics could be very good at catching viruses, just like antibodies do in our bodies. This method can help make better plastics for catching other types of dangerous viruses too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Among biomimetic strategies shaping engineering designs, molecularly imprinted polymer (MIP) technology stands out, involving chemically synthesised receptors emulating natural antigen-antibody interactions. These versatile ‚Äòdesigner polymers‚Äô with remarkable stability and low cost, are pivotal forin vitrodiagnostics. Amid the recent global health crisis, we probed MIPs‚Äô potential to capture SARS-CoV-2 virions. Large biotemplates complicate MIP design, influencing generated binding site specificity. To precisely structure recognition sites within polymers, we innovated an epitope imprinting method supplemented byin silicopolymerization component screening. A viral surface Spike protein informed epitope selection was targeted for MIP development. A novel multi-monomer docking approach (MMSD) was employed to simulate classical receptor-ligand interactions, mimicking binding reinforcement across multiple amino acids. Around 40 monomer combinations were docked to the epitope sequence and top performers experimentally validated via rapid fluorescence binding assays. Notably, high imprinting factor polymers correlated with MMSD predictions, promising rational MIP design applicable to diverse viral pathologies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73114-3" target="_blank">https://www.nature.com/articles/s41598-024-73114-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41590-024-01975-x" target="_blank">
            ü§ñ COVID recovery lungs show unusual white blood cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-485', this)">
            More</button></h4>
            <div id="toggle-485" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Profibrotic monocyte-derived alveolar macrophages are expanded in patients with persistent respiratory symptoms and radiographic abnormalities after COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Monocytes can turn into special cells called alveolar macrophages in the lungs. These cells help fix injuries but sometimes cause more damage over time. Researchers looked at people who had trouble breathing after getting better from COVID-19. They found that when these special lung repair cells were more present or less helpful, it meant the person's lung problems were worse or didn't go away. This suggests these cells might not help fix lungs as well as they should.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monocyte-derived alveolar macrophages drive lung injury and fibrosis in murine models and are associated with pulmonary fibrosis in humans. Monocyte-derived alveolar macrophages have been suggested to develop a phenotype that promotes lung repair as injury resolves. We compared single-cell and cytokine profiling of the alveolar space in a cohort of 35 patients with post-acute sequelae of COVID-19 who had persistent respiratory symptoms and abnormalities on a computed tomography scan of the chest that subsequently improved or progressed. The abundance of monocyte-derived alveolar macrophages, their gene expression programs, and the level of the monocyte chemokine CCL2 in bronchoalveolar lavage fluid positively associated with the severity of radiographic fibrosis. Monocyte-derived alveolar macrophages from patients with resolving or progressive fibrosis expressed the same set of profibrotic genes. Our findings argue against a distinct reparative phenotype in monocyte-derived alveolar macrophages, highlighting their utility as a biomarker of failed lung repair and a potential target for therapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01975-x" target="_blank">https://www.nature.com/articles/s41590-024-01975-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-03</strong> <a href="https://www.nature.com/articles/s41598-024-74196-9" target="_blank">
            ü§ñ Severe COVID-19 biomarkers: Sirukumab treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-486', this)">
            More</button></h4>
            <div id="toggle-486" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pharmacodynamic, prognostic, and predictive biomarkers in severe and critical COVID-19 patients treated with sirukumab</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We studied certain chemicals in sick people's blood who had very bad cases of COVID-19 and were given a special antibody called sirukumab. We found that when these chemicals didn't change much after treatment, the patients got better faster. This is especially true for patients with the most severe form of COVID-19. So, if someone doesn't see changes in these chemicals while taking the medicine, they might benefit more from it. The study ID number is NCT04380961.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We examined candidate biomarkers for efficacy outcomes in hospitalized COVID-19 patients who were treated with sirukumab, an IL-6 neutralizing antibody, in a randomized, double-blind, placebo-controlled, phase 2 trial. Between May 2020 and March 2021, 209 patients were randomized (sirukumab,n=‚Äâ139; placebo,n=‚Äâ70); 112 had critical COVID-19. Serum biomarkers were evaluated for the pharmacodynamic effect of sirukumab and for their potential prognostic and predictive effect on time to sustained clinical improvement up to Day 28, clinical improvement at Day 28, and mortality at Day 28. The absence of detectable IL-4 increase and smaller increases in CCL13 post-baseline were most significantly associated with better response to sirukumab (versus placebo) treatment for all clinical efficacy outcomes tested, especially in patients with critical COVID-19. These data suggest that patients with critical COVID-19 without detectable sirukumab-induced IL-4 levels are more likely to benefit from sirukumab treatment. ClinicalTrials.gov Identifier: NCT04380961.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74196-9" target="_blank">https://www.nature.com/articles/s41598-024-74196-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-03</strong> <a href="https://www.nature.com/articles/s41467-024-52668-w" target="_blank">
            ü§ñ SARS-CoV-2 BA.2.86/JN.1 lineage immune response and severity reduction</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-487', this)">
            More</button></h4>
            <div id="toggle-487" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immune escape and attenuated severity associated with the SARS-CoV-2 BA.2.86/JN.1 lineage</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The BA.2.86 variant, especially its subvariant JN.1, spread widely in the US during winter 2023-24, but didn't cause as many hospitalizations or deaths as previous waves of COVID-19 did.

To understand why this happened, researchers looked at people who had caught these variants compared to others infected with related variants like XBB. They found that:
- More people with BA.2.86 got vaccinated.
- People with BA.2.86 also had more past infections with different COVID strains.
- Less of these people ended up needing hospital care.

The researchers think this might be because their immune systems didn't respond as strongly to the new variant, possibly due to prior vaccinations or previous infections. This means they were less likely to get very sick from BA.2.86 compared to other variants.

Keeping an eye on how SARS-CoV-2 variants evolve and impact health will help doctors plan better responses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 BA.2.86 lineage, and its sublineage JN.1 in particular, achieved widespread transmission in the US during winter 2023‚Äì24. However, this surge in infections was not accompanied by COVID-19 hospitalizations and mortality commensurate with prior waves. To understand shifts in COVID-19 epidemiology associated with JN.1 emergence, we compared characteristics and clinical outcomes of time-matched cases infected with BA.2.86 lineages (predominantly representing JN.1) versus co-circulating XBB-derived lineages in December, 2023 and January, 2024. Cases infected with BA.2.86 lineages received greater numbers of COVID-19 vaccine doses, including XBB.1.5-targeted boosters, in comparison to cases infected with XBB-derived lineages. Additionally, cases infected with BA.2.86 lineages experienced greater numbers of documented prior SARS-CoV-2 infections. Cases infected with BA.2.86 lineages also experienced lower risk of progression to severe clinical outcomes requiring emergency department consultations or hospital admission. Sensitivity analyses suggested under-ascertainment of prior infections could not explain this apparent attenuation of severity. Our findings implicate escape from immunity acquired from prior vaccination or infection in the emergence of the JN.1 lineage and suggest infections with this lineage are less likely to experience clinically-severe disease. Monitoring of immune escape and clinical severity in emerging SARS-CoV-2 variants remains a priority to inform responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52668-w" target="_blank">https://www.nature.com/articles/s41467-024-52668-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41398-024-03108-2" target="_blank">
            ü§ñ Brain Changes in Young People Due to Coronavirus: Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-488', this)">
            More</button></h4>
            <div id="toggle-488" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 related cognitive, structural and functional brain changes among Italian adolescents and young adults: a multimodal longitudinal case-control study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SUGGESTED FOR YOUNG PERSONS

Coronavirus can change how your brain works even if you didn't get very sick. Researchers looked at young people who had milder cases and found changes in certain parts of their brains months later. They noticed problems with some areas like the hippocampus (helps remember things) and amygdala (helps process emotions). These changes can affect how well they do tasks that involve remembering places, even if these kids were fine at first. The findings might help doctors understand long-term effects on the brain from COVID-19 infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) has been associated with brain functional, structural, and cognitive changes that persist months after infection. Most studies of the neurologic outcomes related to COVID-19 focus on severe infection and aging populations. Here, we investigated the neural activities underlying COVID-19 related outcomes in a case-control study of mildly infected youth enrolled in a longitudinal study in Lombardy, Italy, a global hotspot of COVID-19. All participants (13 cases, 27 controls, mean age 24 years) completed resting-state functional (fMRI), structural MRI, cognitive assessments (CANTAB spatial working memory) at baseline (pre-COVID) and follow-up (post-COVID). Using graph theory eigenvector centrality (EC) and data-driven statistical methods, we examined differences in ECdelta(i.e., the difference in EC values pre- and post-COVID-19) and Volumetricdelta(i.e., the difference in cortical volume of cortical and subcortical areas pre- and post-COVID) between COVID-19 cases and controls. We found that ECdeltasignificantly between COVID-19 and healthy participants in five brain regions; right intracalcarine cortex, right lingual gyrus, left hippocampus, left amygdala, left frontal orbital cortex. The left hippocampus showed a significant decrease in Volumetricdeltabetween groups (p=‚Äâ0.041). The reduced ECdeltain the left amygdala associated with COVID-19 status mediated the association between COVID-19 and disrupted spatial working memory. Our results show persistent structural, functional and cognitive brain changes in key brain areas associated with olfaction and cognition. These results may guide treatment efforts to assess the longevity, reversibility and impact of the observed brain and cognitive changes following COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41398-024-03108-2" target="_blank">https://www.nature.com/articles/s41398-024-03108-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s42003-024-06951-7" target="_blank">
            ü§ñ Broad-spectrum COVID-19 antibodies work against Omicron</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-489', this)">
            More</button></h4>
            <div id="toggle-489" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broadly potent spike-specific human monoclonal antibodies inhibit SARS-CoV-2 Omicron sub-lineages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The coronavirus keeps changing, making some medicines ineffective against new versions like Omicron. But scientists found two special antibodies that work well against these changes. One works by targeting part of the virus's spike protein that stays the same even when it mutates. The other targets a different part and can still stop older versions but not newer ones. Understanding how our bodies fight these changes could help make better medicines to treat COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continuous emergence of SARS-CoV-2 variants of concern has rendered many therapeutic monoclonal antibodies (mAbs) ineffective. To date, there are no clinically authorized therapeutic antibodies effective against the recently circulating Omicron sub-lineages BA.2.86 and JN.1. Here, we report the isolation of broad and potent neutralizing human mAbs (HuMabs) from a healthcare worker infected with SARS-CoV-2 early in the pandemic. These include a genetically unique HuMab, named K501SP6, which can neutralize different Omicron sub-lineages, including BQ.1, XBB.1, BA.2.86 and JN.1, by targeting a highly conserved epitope on the N terminal domain, as well as an RBD-specific HuMab (K501SP3) with high potency towards earlier circulating variants that was escaped by the more recent Omicron sub-lineages through spike F486 and E484 substitutions. Characterizing SARS-CoV-2 spike-specific HuMabs, including broadly reactive non-RBD-specific HuMabs, can give insight into the immune mechanisms involved in neutralization and immune evasion, which can be a valuable addition to already existing SARS-CoV-2 therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06951-7" target="_blank">https://www.nature.com/articles/s42003-024-06951-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41599-024-03804-w" target="_blank">
            ü§ñ COVID-19 and Open Science Debate: Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-490', this)">
            More</button></h4>
            <div id="toggle-490" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The impact of COVID-19 on the debate on open science: a qualitative analysis of published materials from the period of the pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how people talked about open science during the pandemic. It found that many thought the pandemic showed why open science is important. Words like "stress test" or "revolution" were used to describe how the pandemic affected open science.

People also believed open science helped fight the pandemic by giving clear information to everyone. But when talking, they mostly focused on sharing data and preprints (early versions of research papers), checking for mistakes in info, and stopping misinformation. They also started thinking about how open science can help make things fairer for all people.

So, even though some old ideas kept coming up, the main topics were new ways to think about how science connects with society and makes it more equal for everyone.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study is an analysis of the international debate on open science that took place during the pandemic. It addresses the question, how did the COVID-19 pandemic impact the debate on open science? The study takes the form of a qualitative analysis of a large corpus of key articles, editorials, blogs and thought pieces about the impact of COVID on open science, published during the pandemic in English, German, Portuguese, and Spanish. The findings show that many authors believed that it was clear that the experience of the pandemic had illustrated or strengthened the case for open science, with language such as a ‚Äústress test‚Äù, ‚Äúcatalyst‚Äù, ‚Äúrevolution‚Äù or ‚Äútipping point‚Äù frequently used. It was commonly believed that open science had played a positive role in the response to the pandemic, creating a clear ‚Äòline of sight‚Äô between open science and societal benefits. Whilst the arguments about open science deployed in the debate were not substantially new, the focuses of debate changed in some key respects. There was much less attention given to business models for open access and critical perspectives on open science, but open data sharing, preprinting, information quality and misinformation became most prominent in debates. There were also moves to reframe open science conceptually, particularly in connecting science with society and addressing broader questions of equity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-024-03804-w" target="_blank">https://www.nature.com/articles/s41599-024-03804-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41598-024-73639-7" target="_blank">
            ü§ñ Optimal Air Travel Strategies for Pandemic Control Using Deep Learning</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-491', this)">
            More</button></h4>
            <div id="toggle-491" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deep learning-derived optimal aviation strategies to control pandemics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic affected many countries worldwide, causing governments to create strict health rules to stop the spread of the virus. This led to economic problems too.

To understand how airplanes played a role in spreading the virus, scientists created a new tool called DWSAGE. It's like a smart map that learns from daily changes in airplane flights around the world. 

Scientists used this tool to see which places with lots of airplane traffic might have spread more COVID-19. They found out Western Europe, the Middle East, and North America had high airplane traffic.

Based on their findings, scientists suggested reducing airplane travel to help control the virus without stopping people from moving around much. Their work gives a powerful tool for studying future outbreaks and helps governments make better decisions about air travel rules.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic affected countries across the globe, demanding drastic public health policies to mitigate the spread of infection, which led to economic crises as a collateral damage. In this work, we investigate the impact of human mobility, described via international commercial flights, on COVID-19 infection dynamics on a global scale. We developed a graph neural network (GNN)-based framework called Dynamic Weighted GraphSAGE (DWSAGE), which operates over spatiotemporal graphs and is well-suited for dynamically changing flight information updated daily. This architecture is designed to be structurally sensitive, capable of learning the relationships between edge features and node features. To gain insights into the influence of air traffic on infection spread, we conducted local sensitivity analysis on our model through perturbation experiments. Our analyses identified Western Europe, the Middle East, and North America as leading regions in fueling the pandemic due to the high volume of air traffic originating or transiting through these areas. We used these observations to propose air traffic reduction strategies that can significantly impact controlling the pandemic with minimal disruption to human mobility. Our work provides a robust deep learning-based tool to study global pandemics and is of key relevance to policymakers for making informed decisions regarding air traffic restrictions during future outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73639-7" target="_blank">https://www.nature.com/articles/s41598-024-73639-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41541-024-00976-z" target="_blank">
            ü§ñ Norovirus vaccine protects gut tissue</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-492', this)">
            More</button></h4>
            <div id="toggle-492" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bivalent norovirus mRNA vaccine elicits cellular and humoral responses protecting human enteroids from GII.4 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This article talks about a new way to make vaccines against norovirus, which is a stomach bug that spreads easily. Scientists used special material called mRNA-LNP to create a vaccine for two types of norovirus. This vaccine helped the body's immune system produce strong antibodies that can fight off these viruses. The antibodies stopped the virus from infecting tiny models of the human gut, showing this vaccine works well in tests. This means scientists have proof their method is effective and could make better vaccines against norovirus in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Nucleoside-modified mRNA-LNP vaccines have revolutionized vaccine development against infectious pathogens due to their ability to elicit potent humoral and cellular immune responses. In this article, we present the results of the first norovirus vaccine candidate employing mRNA-LNP platform technology. The mRNA-LNP bivalent vaccine encoding the major capsid protein VP1 from GI.1 and GII.4 of human norovirus, generated high levels of neutralizing antibodies, robust cellular responses, and effectively protected human enteroids from infection by the most prevalent genotype (GII.4). These results serve as a proof of concept, demonstrating that a modified-nucleoside mRNA-LNP vaccine based on norovirus VP1 sequences can stimulate an immunogenic response in vivo and generates neutralizing antibodies capable of preventing viral infection in models of human gastrointestinal tract infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00976-z" target="_blank">https://www.nature.com/articles/s41541-024-00976-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41598-024-73956-x" target="_blank">
            ü§ñ Cytokines in sick kids with multi-system inflammation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-493', this)">
            More</button></h4>
            <div id="toggle-493" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cytokine landscape in hospitalized children with multisystem inflammatory syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at the body's response to a sickness called MIS-C in children who had COVID-19. They measured certain chemicals (called cytokines) in kids' blood before and after treatment with special medicines.

They found three main groups of these chemicals:
1. The first group had mainly bad chemicals that make inflammation worse.
2. The second group had some good chemicals plus some bad ones, which stayed around a bit longer.
3. The third group saw its levels go up later on, with more kinds of bad chemicals.

These changes show how the body fights off sickness and helps kids get better. It starts by making lots of inflammatory chemicals to fight infections, then it makes more specific memory cells so it can keep fighting if needed in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The etiology of multisystem inflammatory syndrome in children (MIS-C), frequently observed following COVID-19 infection, remains elusive. This study unveils insights derived from cytokine analysis in the sera of MIS-C patients, both before and after the administration of intravenous immunoglobulin (IVIG) and glucocorticosteroids (GCS). In this study, we employed a comprehensive 45-cytokine profile encompassing a spectrum of widely recognized proinflammatory and antiinflammatory cytokines, as well as growth factors, along with other soluble mediators. The analysis delineates three principal cytokine-concentration patterns evident in the patients‚Äô sera. Pattern no.1 predominantly features proinflammatory cytokines (IL-6, IL-15, IL-1ra, granulocyte-macrophage colony-stimulating factor (GM-CSF), tumor necrosis factor Œ± (TNFŒ±), C-X-C motif chemokine ligand 10 (CXCL10/ IP-10), and IL-10) exhibiting elevated concentrations upon admission, swiftly normalizing post-hospital treatment. Pattern no. 2 includes cytokines (IL-17¬†A, IL-33, IFNŒ≥, vascular endothelial growth factor (VEGF), and programmed death ligand (PD-L1)) with moderately elevated levels at admission, persisting over 7‚Äì10 days of hospitalization despite the treatment. Pattern no. 3 comprises cytokines which concentrations escalated after 7‚Äì10 days of hospitalization and therapy, including IL-1Œ±, IL-1Œ≤, IL-2, IL-13, platelet-derived growth factor AA/BB (PDGF AA/BB). The observed in cytokine profile of MIS-C patients showed a transition from acute inflammation to sustaining inflammation which turned into induction of humoral memory mechanisms and various defense mechanisms, contributing to recovery.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73956-x" target="_blank">https://www.nature.com/articles/s41598-024-73956-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41598-024-71247-z" target="_blank">
            ü§ñ Impact of COVID-19 on PAD patients in China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-494', this)">
            More</button></h4>
            <div id="toggle-494" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of the COVID-19 pandemic on patients with peripheral arterial disease in China: a multicenter cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how having a disease called peripheral arterial disease (PAD) affected people who were hospitalized during different times: before and after the pandemic started. Before the pandemic, 762 patients with PAD were treated, on average aged 72 years. After the pandemic, 478 patients were treated, on average aged 65 years.

During the pandemic, more patients had problems like very bad legs (CLTI), foot infections, and damage to blood vessels in their legs. Their leg condition got worse too. Treating these patients was harder during the pandemic, which made it hard for them to get better.

The study also found that having certain issues, like diabetes or kidney problems, could make things worse. These people had more serious health problems after being treated during the pandemic.

Overall, doctors still tried their best to help patients with severe PAD even though there were many challenges. The study shows it's important to keep treating and checking on these patients closely.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aims to understand the repercussions of the COVID-19 pandemic on hospitalized patients with peripheral arterial disease (PAD) in China, who did not contract SARS-CoV-2. We conducted a multicenter cross-sectional analysis comparing the characteristics and outcomes of hospitalized PAD patients across two distinct periods: Pre-pandemic (P1, from January 2018 to December 2019) and during the pandemic (P2, from January 2020 to December 2021). During P1, 762 hospitalized patients were treated, with an average age of 72.3¬†years, while 478 patients were treated in P2, with an average age of 65.1¬†years. Notably, hospitalized patients admitted during the pandemic (P2) exhibited a significantly higher incidence of chronic limb-threatening ischemia (CLTI, 70% vs 54%), diabetic foot infection (47% vs 29%), and infra-popliteal lesions (28% vs 22%). Furthermore, these patients demonstrated a marked deterioration in their Rutherford category and an increased mean score in the Wound, Ischemia, and foot Infection classification system (WIfI). Treatment during the pandemic emerged as a predictor of reduced procedural success and increased major adverse limb events. Factors such as the presence of diabetic foot infection, renal impairment, and deteriorating WIfI scores were identified as independent risk indicators for major adverse limb events. Our results demonstrate that intensive care was provided to severe cases of PAD even during the challenging circumstances of the COVID-19 pandemic. Despite the unprecedented pressures on healthcare systems, patients with severe PAD, particularly those with CLTI, continued to receive necessary in-patient care. The findings underscore the importance of timely medical interventions and extended follow-up for patients exhibiting high-risk factors.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71247-z" target="_blank">https://www.nature.com/articles/s41598-024-71247-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73321-y" target="_blank">
            ü§ñ Long-term study on blood-brain barrier changes in COVID-19 patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-495', this)">
            More</button></h4>
            <div id="toggle-495" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Oneyear longitudinal study on biomarkers of blood‚Äìbrain barrier permeability in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how a virus called COVID-19 affects the brain. They found that after getting sick, the blood-brain barrier (which keeps things from going into the brain) was more leaky for a little while. This led to some extra chemicals in the blood coming from cells in the brain. These extra chemicals are like astrocyte-derived extracellular vesicles and S100B.

They checked if these extra chemicals were related to symptoms that kept lasting for months after getting sick. But they didn't find any link between the levels of these chemicals and long-term problems people still had even after a year.

The study found that after about 4 months, these extra chemicals peaked (were at their highest), but by the end of the year, they were back to normal. 

The main thing is that while there are signs of brain changes during an infection, those don't seem to cause long-term health issues. More research is needed to understand why some people have problems for so long after having COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The pathophysiology behind neurological and cognitive sequelae of COVID-19 may be related to dysfunction of the blood-brain barrier (BBB) and previous research indicate transient neuronal injury and glial activation. The aim of this study was to investigate if COVID-19 is related to increased BBB permeability by analyzing leakage of biomarkers such as astrocyte-derived extracellular vesicles (EVs) and S100B. We also investigated whether levels of these biomarkers correlated with self-reported symptoms that persisted‚Äâ>‚Äâ2 months. The samples in this 1-year follow-up study came from an ongoing longitudinal study of unvaccinated patients hospitalized for COVID-19 at Danderyd University Hospital, Stockholm, Sweden, between April and June 2020. Blood samples were collected at baseline and 4, 8, and 12 months after hospitalization. Information on self-reported clinical symptoms was collected at follow-up visits. A total of 102 patients were enrolled, and 47 completed all follow-up measurements. Peak levels of both biomarkers were observed at 4 months in the subset of 55 patients who were measured at this timepoint. At 12 months, the biomarkers had returned to baseline levels. The biomarkers were not correlated with any of the long-term self-reported symptoms. COVID-19 is associated with transient increased BBB permeability, shown by elevated levels of astrocyte biomarkers in plasma. However, these levels return to baseline 12 months post-infection and do not correlate with long-term symptoms. Further research is needed to unravel the underlying mechanisms causing long-term symptoms in COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73321-y" target="_blank">https://www.nature.com/articles/s41598-024-73321-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41467-024-52404-4" target="_blank">
            ü§ñ Post-COVID Symptoms in ER Patients Tested for COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-496', this)">
            More</button></h4>
            <div id="toggle-496" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID-19 condition symptoms among emergency department patients tested for SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study found that many people who went to the hospital with emergency symptoms didn't have COVID-19, but some still had problems like coughing or trouble breathing. Even when doctors tested negative for COVID-19, almost one-third of those patients said they were having Post-COVID issues three months later. The problem is, it's hard to know if these are really from the coronavirus or just other illnesses. So doctors think we need better tests now to figure out what causes these symptoms so people can get proper treatment sooner.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Symptoms of the Post-COVID-19 Condition are often non-specific making it a challenge to distinguish them from symptoms due to other medical conditions. In this study, we compare the proportion of emergency department patients who developed symptoms consistent with the World Health Organization‚Äôs Post-COVID-19 Condition clinical case definition between those who tested positive for Severe Acute Respiratory Syndrome Coronavirus-2 infection and time-matched patients who tested negative. Our results show that over one-third of emergency department patients with a proven acute infection meet Post-COVID-19 Condition criteria 3 months post-index visit. However, one in five test-negative patients who claim never having been infected also report symptoms consistent with Post-COVID-19 Condition highlighting the lack of specificity of the clinical case definition. Testing for SARS-CoV-2 during the acute phase of a suspected infection should continue until specific biomarkers of Post-COVID-19 Condition become available for diagnosis and treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52404-4" target="_blank">https://www.nature.com/articles/s41467-024-52404-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-72421-z" target="_blank">
            ü§ñ Cytokines linked to memory changes post-SARS-CoV-2ÊÑüÊüì6-9ÊúàÂíå12-15Êúà</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-497', this)">
            More</button></h4>
            <div id="toggle-497" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Systemic cytokines related to memory function 6‚Äì9¬†months and 12‚Äì15¬†months after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how cytokines, which are chemicals made by the body during infections, might affect people's memories months after they had COVID. They found that levels of one cytokine called IL-1Œ≤ were linked to remembering words better about 6-9 months later. Another cytokine, IL-6, seemed to affect memory around a year after infection. This suggests how sick someone was during the early stages of their COVID could impact their ability to remember things over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cognitive symptoms persisting beyond the acute phase of COVID-19 infection are commonly described for up to 2¬†years after infection. The relationship between cognitive performance, in particular episodic memory processes observed chronically after infection, and cytokine levels in the acute phase of COVID-19 has not yet been identified in humans. To determine whether the levels of cytokines IL1Œ≤, IL-6 and TNFŒ± secreted in the acute phase of SARS-CoV-2 infection are associated and predict verbal and visuospatial episodic memory performance in humans 6 to 9¬†months and 12 to 15¬†months post-infection. The associations and predictive value of the concentration of cytokines measured in acute phase (IL-1Œ≤, IL-6, TNFŒ±) from plasma samples ofN=‚Äâ33 hospitalized COVID-19 patients (mean age 61¬†years, 39‚Äì78, 65% in intensive care) in relation to their verbal and visuospatial episodic memory performance measured at 6‚Äì9¬†months and 12‚Äì15¬†months post-infection were analyzed. To do this, we used Spearman correlations and generalised linear mixed models. IL-1Œ≤ levels were associated with verbal episodic memory total recall scores 6‚Äì9¬†months post-infection. At 12‚Äì15¬†months post-infection IL-6 predicted verbal episodic memory score. This study demonstrated that the severity of inflammatory reaction at acute phase of SARS-CoV-2 infection predicts verbal episodic memory performance in the long-term post-infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72421-z" target="_blank">https://www.nature.com/articles/s41598-024-72421-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s43856-024-00610-y" target="_blank">
            ü§ñ Infecting K18-hACE2 mice with both Coccidioides and COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-498', this)">
            More</button></h4>
            <div id="toggle-498" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Developing aCoccidioides posadasiiand SARS-CoV-2 Co-infection Model in the K18-hACE2 Transgenic Mouse</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SIMPLIFIED

Scientists wanted to see how getting sick twice at once affects you. They did this by putting two different sicknesses into mice‚Äîlike a mock infection. One sickness was caused by the virus that causes COVID-19, and the other was from a fungus called Valley Fever.

They found that if they gave both sicknesses to the mice one after another, the mice got much sicker and lived less time than those who only had one sickness. This happened more with certain types of coronavirus (like Delta or Omicron) and sometimes depended on when each sickness came first.

The scientists also noticed differences in how the immune system reacted to both sicknesses. The ones that caught the fungus first were healthier, but still got sick from the virus. Those that caught the virus first had a harder time recovering from Valley Fever.

These findings suggest people who live where Valley Fever is common might have a worse experience if they get COVID-19 there too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundEarly reports showed that patients with COVID-19 had recrudescence of previously resolved coccidioidomycosis (Valley fever, VF), and there were indications that coinfection had more severe outcomes. We therefore investigated serial infection ofCoccidioides posadasiiand SARS-CoV-2 in a K18-hACE2 mouse model to assess disease outcomes.MethodsIn our model, we challenged K18-hACE2 mice sequentially with a sub-lethal dose of SARS-CoV-2 and 24‚Äâhours later with low virulence strain ofCoccidioides posadasii, and vice versa, compared to mice that only received a single infection challenge. We performed survival and pathogenesis mouse studies as well as looked at the systemic immune response differences between treatment groups.ResultsHere we show that co-infected groups have a more severe disease progression as well as a decrease in survival. Importantly, results differ depending on the SARS-CoV-2 variant (WA-1, Delta, or Omicron) and infection timing (SARS-CoV-2 first,C. posadasiisecond or vice versa). We find that groups that are infected with the virus first had a decrease in survival, increased morbidity and weight loss, increased fungal and viral burdens, differences in immune responses, and the amount and size of fungal spherules. We also find that groups coinfected withC. posadasiifirst have a decrease fungal burden and inflammatory responses.ConclusionsThis is the first in vivo model investigation of a coinfection of SARS-CoV-2 andCoccidioides. Because of the potential for increased severity of disease in a coinfection, we suggest populations that live in areas of high coccidioidomycosis endemicity may experience higher incidence of complicated disease progression with COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00610-y" target="_blank">https://www.nature.com/articles/s43856-024-00610-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73160-x" target="_blank">
            ü§ñ Predicting Long COVID Burden in CA with QALYs</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-499', this)">
            More</button></h4>
            <div id="toggle-499" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modeling the burden of long COVID in California with quality adjusted life-years (QALYS)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at a sickness called "long COVID" that some people get even after they had very little or no symptoms from the original coronavirus illness. They used data on all confirmed cases in California over several years.

They divided people into different age groups and measured how many life-years of quality were lost due to long COVID. The study found that more older adults (65+) lost these life years compared to younger ones, but surprisingly, younger adults (18-49) also had a significant loss when you consider all the mild and moderate symptoms.

The researchers think their estimates might be lower than what really happened because they didn't account for everyone who got long COVID. They found that how many people developed long COVID and which symptoms were severe or mild affected their results the most.

Overall, this study helps us understand more about the impact of "long COVID" in different age groups and how it compares to regular cases like hospitalizations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Individuals infected with SARS-CoV-2 may develop post-acute sequelae of COVID-19 (‚Äúlong COVID‚Äù) even after asymptomatic or mild acute illness. Including time varying COVID symptom severity can provide more informative burden estimates for public health response. Using a compartmental model driven by confirmed cases, this study estimated long COVID burden by age group (0‚Äì4, 5‚Äì17, 18‚Äì49, 50‚Äì64, 65+) in California as measured by the cumulative and severity-specific proportion of quality-adjusted life years (QALYs) lost. Long COVID symptoms were grouped into severe, moderate, and mild categories based on estimates from the Global Burden of Disease study, and symptoms were assumed to decrease in severity in the model before full recovery. All 10,945,079 confirmed COVID-19 cases reported to the California Department of Public Health between March 1, 2020, and December 31, 2022, were included in the analysis. Most estimated long COVID-specific QALYs [59,514 (range: 10,372‚Äì180,257)] lost in California were concentrated in adults 18‚Äì49 (31,592; 53.1%). Relative to other age groups, older adults (65+) lost proportionally more QALYs from severe long COVID (1,366/6,984; 20%). Due to changing case ascertainment over time, this analysis might underestimate the actual total burden. In global sensitivity analysis, estimates of QALYs lost were most sensitive to the proportion of individuals that developed long COVID and proportion of cases with each initial level of long COVID symptom severity (mild/moderate/severe). Models like this analysis can help translate observable metrics such as cases and hospitalizations into quantitative estimates of long COVID burden that are currently difficult to directly measure. Unlike the observed relationship between age and incident severe outcomes for COVID-19, this study points to the potential cumulative impact of mild long COVID symptoms in younger individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73160-x" target="_blank">https://www.nature.com/articles/s41598-024-73160-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-72842-w" target="_blank">
            ü§ñ Deep Learning for Detecting Complex Protein Changes in COVID-19 Virus Interactions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-500', this)">
            More</button></h4>
            <div id="toggle-500" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deep learning for discriminating non-trivial conformational changes in molecular dynamics simulations of SARS-CoV-2 spike-ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study uses advanced computer models to analyze how a part of the SARS-CoV-2 virus protein interacts with another protein in the human body. They converted these complex data into simpler maps for easier understanding, then used AI to guess which changes in the virus might make it more contagious or harder to fight off.

They found that some changes to this part of the virus can make it stick better to its target and also makes it less noticeable to the immune system. This helps them predict how different types of mutations might affect the virus's ability to spread and avoid being detected by the body‚Äôs defenses. 

The AI also showed a good match between what they predicted and actual changes in how well the virus sticks, which is important for understanding its behavior. They discovered that one particular part of the protein moves around more than usual when there are these mutations, suggesting it might be key to evading the immune response.

Overall, this method can help scientists quickly find new strains of the virus that could spread better or hide from our bodies' defenses. This is useful for making new treatments and vaccines faster.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Molecular dynamics (MD) simulations produce a substantial volume of high-dimensional data, and traditional methods for analyzing these data pose significant computational demands. Advances in MD simulation analysis combined with deep learning-based approaches have led to the understanding of specific structural changes observed in MD trajectories, including those induced by mutations. In this study, we model the trajectories resulting from MD simulations of the SARS-CoV-2 spike protein-ACE2, specifically the receptor-binding domain (RBD), as interresidue distance maps, and use deep convolutional neural networks to predict the functional impact of point mutations, related to the virus‚Äôs infectivity and immunogenicity. Our model was successful in predicting mutant types that increase the affinity of the S protein for human receptors and reduce its immunogenicity, both based on MD trajectories (precision = 0.718; recall = 0.800;\(\hbox {F}_1\)= 0.757; MCC = 0.488; AUC = 0.800) and their centroids. In an additional analysis, we also obtained a strong positive Pearson‚Äôs correlation coefficient equal to 0.776, indicating a significant relationship between the average sigmoid probability for the MD trajectories and binding free energy (BFE) changes. Furthermore, we obtained a coefficient of determination of 0.602. Our 2D-RMSD analysis also corroborated predictions for more infectious and immune-evading mutants and revealed fluctuating regions within the receptor-binding motif (RBM), especially in the\(\beta _{1}^{\prime }/\beta _{2}^{\prime }-C\)loop. This region presented a significant standard deviation for mutations that enable SARS-CoV-2 to evade the immune response, with RMSD values of 5√Ö in the simulation. This methodology offers an efficient alternative to identify potential strains of SARS-CoV-2, which may be potentially linked to more infectious and immune-evading mutations. Using clustering and deep learning techniques, our approach leverages information from the ensemble of MD trajectories to recognize a broad spectrum of multiple conformational patterns characteristic of mutant types. This represents a strategic advantage in identifying emerging variants, bypassing the need for long MD simulations. Furthermore, the present work tends to contribute substantially to the field of computational biology and virology, particularly to accelerate the design and optimization of new therapeutic agents and vaccines, offering a proactive stance against the constantly evolving threat of COVID-19 and potential future pandemics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72842-w" target="_blank">https://www.nature.com/articles/s41598-024-72842-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-67945-3" target="_blank">
            ü§ñ Age and gender impact vaccine response to SpikeGen shots: study results.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-501', this)">
            More</button></h4>
            <div id="toggle-501" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effects of age and gender on immunogenicity and reactogenicity of SpikoGen recombinant spike protein vaccine: a post-hoc analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT Simplified

The SpikoGen COVID-19 vaccine uses part of the virus's spike protein. It was tested in people of different ages and genders to see how well it works. After one dose, older people had fewer side effects like tiredness and headache. Men also felt fewer side effects than women after each dose. The vaccine didn't affect neutralizing antibodies differently between men and women or age groups.

The vaccine is good at getting into the body's immune system but doesn‚Äôt cause many bad reactions. This shows that even if you don't feel too sick from a vaccine, it‚Äôs still working well to protect against COVID-19.

SpikoGen seems like a promising way to make vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SpikoGen¬Æ COVID-19 vaccine is based on the spike protein extracellular domain of the ancestral Wuhan-Hu-1 strain modified by removal of the furin cleavage site and addition of stabilising mutations expressed as a recombinant protein in insect cells. It is formulated with Advax-CpG55.2‚Ñ¢ adjuvant to ensure optimal immunogenicity. In this study, data from several SpikoGen¬Æ clinical trials was retrospectively analysed to assess for any effect of gender or age on seroconversion, neutralizing antibody levels or the incidence of adverse events. Following the 1st dose, older age was associated with a reduced rate of fatigue (RR 0.97, p < 0.001), headache (RR 0.98, p = 0.034) and myalgia (RR 0.97, p=0.016), following the 2nd dose, the rate of fatigue (RR 0.98, p = 0.017) but following the 3rd dose no effect of age on adverse events was evident. Similarly, following the 1st dose, men reported a 19% lower incidence of fatigue, 36% lower incidence of headache and 28% lower incidence of myalgia when compared to women. Interestingly, there was no relationship between age or gender and serum neutralizing antibody levels, although after each vaccine dose there was a consistent trend to women having a higher seroconversion rate. There was no correlation between neutralizing antibody levels and adverse events. Unlike what is seen with mRNA vaccines, reactogenicity trended lower after each subsequent SpikoGen¬Æ dose. Overall, SpikoGen¬Æ exhibited positive immunogenicity and low reactogenicity, indicating that a low incidence of adverse events does not equate to poor immunogenicity. SpikoGen¬Æ remains a promising protein-based vaccine platform for COVID-19 protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67945-3" target="_blank">https://www.nature.com/articles/s41598-024-67945-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73175-4" target="_blank">
            ü§ñ Surface proteins of Group B Strep isolates from SA & vaccine target ID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-502', this)">
            More</button></h4>
            <div id="toggle-502" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Surface protein distribution in Group B Streptococcus isolates from South Africa and identifying vaccine targets through in silico analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Group B Strep (GBS) is a germ that can cause serious illnesses like pneumonia, meningitis, and sepsis in babies under 3 months old. It can also lead to stillbirths and early births in mommies who are pregnant. For older adults or people with weak immune systems, GBS causes different diseases.

Right now, there's no vaccine that works against GBS. By looking at the genetic makeup of GBS, scientists found some important parts they think could be used to make a vaccine. They did this by testing lots of GBS samples from babies and mommies in South Africa.

They looked for which parts of GBS are similar between the sick ones (that's what "invasive" means) and those that live on but don't cause problems yet ("colonizing"). Out of 89 possible vaccine candidates, they found 10 very common parts. Nine of these were special proteins called "computational inferred proteins," which is a fancy way of saying the scientists figured out their identity by looking at genetic data. One part was a surface protein that helps GBS stick to cells.

They also looked at how likely each part is to trigger an immune response, which is important for vaccines. The nine special parts they found were more likely to do this than the surface protein. This suggests these could be good targets for making a vaccine against GBS.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Group BStreptococcus(GBS) is a major cause of pneumonia, sepsis, and meningitis in infants younger than 3 months of age. Furthermore, GBS infection in pregnant women is associated with stillbirths and pre-term delivery. It also causes disease in immunocompromised adults and the elderly, but the highest incidence of the disease occurs in neonates and young infants. At this time, there are no licensed vaccines against GBS. Complete GBS genome sequencing has helped identify genetically conserved and immunogenic proteins, which could serve as vaccine immunogens. In this study, in silico reverse vaccinology method were used to evaluate the prevalence and conservation of GBS proteins in invasive and colonizing isolates from South African infants and women, respectively. Furthermore, this study aimed to predict potential GBS vaccine targets by evaluating metrics such as antigenicity, physico-chemical properties, subcellular localization, secondary and tertiary structures, and epitope prediction and conservation. A total of 648 invasive and 603 colonizing GBS isolate sequences were screened against a panel of 89 candidate GBS proteins. Ten of the 89 proteins were highly genetically conserved in invasive and colonizing GBS isolates, nine of which were computationally inferred proteins (gbs2106, SAN_1577, SAN_0356, SAN_1808, SAN_1685, SAN_0413, SAN_0990, SAN_1040, SAN_0226) and one was the surface Immunogenic Protein (SIP). Additionally, the nine proteins were predicted to be more antigenic than the SIP protein (antigenicity score of >‚Äâ0.6498), highlighting their potential as GBS vaccine antigen targets.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73175-4" target="_blank">https://www.nature.com/articles/s41598-024-73175-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-28</strong> <a href="https://www.nature.com/articles/s41598-024-73779-w" target="_blank">
            ü§ñ Key Hub Genes in Leishmaniasis for Diagnosis and Treatment Discovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-503', this)">
            More</button></h4>
            <div id="toggle-503" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Determination of key hub genes in Leishmaniasis as potential factors in diagnosis and treatment based on a bioinformatics study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Leishmaniasis is a disease caused by parasites that are carried by infected sandflies. Researchers looked at data from patients with leishmaniasis on their skin wounds and compared it to healthy people. They used special methods to find genes that were more active or less active in the sick people.

They found 407 genes that behaved differently between those with leishmaniasis and others. These genes are involved in immune responses, inflammation, and pathways related to fighting infections.

The study also identified seven important genes (called hub genes) that help control these different gene activities. One of these genes, UBD, plays a role in how cells break down other molecules called ubiquitin. Another hub gene, UBD, is connected with several microRNAs, which are tiny molecules that can control gene activity.

These findings could help doctors diagnose leishmaniasis earlier and develop new treatments or vaccines against the disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Leishmaniasis is an infectious disease caused by protozoan parasites from different species of leishmania. The disease is transmitted by female sandflies that carry these parasites. In this study, datasets on leishmaniasis published in the GEO database were analyzed and summarized. The analysis in all three datasets (GSE43880, GSE55664, and GSE63931) used in this study has been performed on the skin wounds of patients infected with a clinical form of leishmania (Leishmania braziliensis), and biopsies have been taken from them. To identify differentially expressed genes (DEGs) between leishmaniasis patients and controls, the robust rank aggregation (RRA) procedure was applied. We performed gene functional annotation and protein-protein interaction (PPI) network analysis to demonstrate the putative functionalities of the DEGs. The study utilized Molecular Complex Detection (MCODE), Gene Ontology (GO), and Kyoto Encyclopedia of Genes and Genomes (KEGG) to detect molecular complexes within the protein-protein interaction (PPI) network and conduct analyses on the identified functional modules. The CytoHubba plugin‚Äôs results were paired with RRA analysis to determine the hub genes. Finally, the interaction between miRNAs and hub genes was predicted. Based on the RRA integrated analysis, 407 DEGs were identified (263 up-regulated genes and 144 down-regulated genes). The top three modules were listed after creating the PPI network via the MCODE plug. Seven hub genes were found using the CytoHubba app and RRA: CXCL10, GBP1, GNLY, GZMA, GZMB, NKG7, and UBD. According to our enrichment analysis, these functional modules were primarily associated with immune pathways, cytokine activity/signaling pathways, and inflammation pathways. However, a UBD hub gene is interestingly involved in the ubiquitination pathways of pathogenesis. The mirNet database predicted the hub gene‚Äôs interaction with miRNAs, and results revealed that several miRNAs, including mir-146a-5p, crucial in fighting pathogenesis. The key hub genes discovered in this work may be considered as potential biomarkers in diagnosis, development of agonists/antagonist, novel vaccine design, and will greatly contribute to clinical studies in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73779-w" target="_blank">https://www.nature.com/articles/s41598-024-73779-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-28</strong> <a href="https://www.nature.com/articles/s41598-024-73001-x" target="_blank">
            ü§ñ Clinical Presentation, Inflammation, ANDC Scores in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-504', this)">
            More</button></h4>
            <div id="toggle-504" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical presentations, systemic inflammation response and ANDC scores in hospitalized patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that people with anosmia (loss of smell) or ageusia (loss of taste) are often diagnosed with COVID-19, especially with the alpha variant. They looked at 231 hospitalized COVID-19 patients in Taiwan and noticed some differences between those who had these symptoms and those who didn't.

They found that younger people with anosmia/ageusia were usually healthier overall and less sick compared to others without these issues. Women also tended to have milder cases than men, though they both had similar smells and taste loss.

The study showed that certain blood tests called NLRs and PLRs helped predict how long patients would stay in the hospital. The ANDC score is another tool that can estimate this length based on symptoms like coughing and fever. 

Overall, the research suggests that people with anosmia/ageusia might have less serious COVID-19 and a lower body inflammation response, especially if they are younger women. This score could help doctors predict how sick someone will get from the disease and how long they'll need to stay in the hospital.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The association of anosmia/ageusia with a positive severe respiratory syndrome coronavirus 2 (SARS-CoV-2) test is well-established, suggesting these symptoms are reliable indicators of coronavirus disease 2019 (COVID-19) infection. This study investigates the clinical characteristics and systemic inflammatory markers in hospitalized COVID-19 patients in Taiwan, focusing on those with anosmia/ageusia. We conducted a retrospective observational study on 231 hospitalized COVID-19 patients (alpha variant) from April to July 2021. Clinical symptoms, dyspnea grading, and laboratory investigations, including neutrophil-lymphocyte ratios (NLRs), platelet-lymphocyte ratios (PLRs), and ANDC scores (an early warning score), were analyzed. Cough (64.1%), fever (58.9%), and dyspnea (56.3%) were the most common symptoms, while anosmia/ageusia affected 9% of patients. Those with anosmia/ageusia were younger, had lower BMI, lower systemic inflammatory markers, and better ANDC scores than those without these symptoms. Female patients exhibited lower NLR values and ANDC scores compared to male patients (allp<‚Äâ0.05). Multivariable regression analysis demonstrated significant associations between NLR and CRP and ferritin levels (allp<‚Äâ0.01), and between PLR and ESR and ferritin levels (p<‚Äâ0.01). Categorized ANDC scores significantly correlated with the total hospital length of stay (allp<‚Äâ0.05). Despite ethnic differences in the prevalence of anosmia/ageusia, our study highlights similar clinical presentations and inflammatory profiles to those observed in Western countries. The ANDC score effectively predicted hospital stay duration. These findings suggest that anosmia/ageusia may be associated with less severe disease and a lower inflammatory response, particularly in younger and female patients. The ANDC score can serve as a valuable prognostic tool in assessing the severity and expected hospital stay of COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73001-x" target="_blank">https://www.nature.com/articles/s41598-024-73001-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-71951-w" target="_blank">
            ü§ñ COVID Severity & Biomarkers for Patient Prognosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-505', this)">
            More</button></h4>
            <div id="toggle-505" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Correlating COVID-19 severity with biomarker profiles and patient prognosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how different things can affect how sick people get from the coronavirus and its long-term effects. They studied 226 people to see what makes some recover faster than others. Age, lower education, being of a certain race, overweight, having issues with blood clotting genes, heart or lung problems, diabetes, and other factors were found to make someone sicker.

The study also saw that six months later, some people had less of certain things in their blood like FVIII, VWF, and DD. Some people had joint pain, muscle aches, trouble breathing, and swelling in their legs if they had diabetes or lung problems during recovery. People who kept having high levels of DD after getting better were more likely to feel breathless and get heartburn.

Women were found to have leg swelling longer than men, and those with higher FVIII levels for a while also got swollen legs. Exercise helped prevent leg swelling. The study thinks blood vessels might be one reason why people are sicker from the virus or its long-term effects.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19's long-lasting and complex impacts have become a global concern, with diverse clinical outcomes. This study evaluated 226 participants to understand the clinical spectrum of COVID-19/Long COVID (LC), exploring how disease severity correlates with sociodemographic factors and biomarkers. Determinants related to COVID-19 severity included age (P‚Äâ<‚Äâ0.001), lower education (P‚Äâ<‚Äâ0.001), ethnicity (P‚Äâ=‚Äâ0.003), overweight (P‚Äâ<‚Äâ0.001),MTHFRgene rs1801133 (P‚Äâ=‚Äâ0.035), cardiovascular diseases (P‚Äâ=‚Äâ0.002), diabetes mellitus (DM) (P‚Äâ=‚Äâ0.006), Factor VIII (FVIII) (P‚Äâ=‚Äâ0.046), von Willebrand factor (VWF) (P‚Äâ=‚Äâ0.002), and dimer D (DD) (P‚Äâ<‚Äâ0.001). Six months later, in a portion of the monitored participants, a significant reduction in FVIII (P‚Äâ<‚Äâ0.001), VWF (P‚Äâ=‚Äâ0.002), and DD (P‚Äâ<‚Äâ0.001) levels was observed, with only DD returning to normal values. Different systemic sequelae were identified, with higher incidences of joint pain and myalgia in participants with a clinical history of DM, chronic lung disease (CLD) and sustained high interleukin 6 values in the convalescent phase. CLD, COVID-19 severity and high DD levels increased the risk of developing dyspnea and palpitations. Women were more likely to develop lower limb phlebitis long-term, while sustained elevated FVIII in the convalescent phase was associated with an increased risk of swelling. Regular physical activity had a protective effect against swelling. This study highlights factors contributing to COVID-19 severity/LC, emphasizing endothelial cell activation as a potential mechanism.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71951-w" target="_blank">https://www.nature.com/articles/s41598-024-71951-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52194-9" target="_blank">
            ü§ñ Predicting Booster Immunogenicity for New Variants and Vaccine Updates</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-506', this)">
            More</button></h4>
            <div id="toggle-506" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting COVID-19 booster immunogenicity against future SARS-CoV-2 variants and the benefits of vaccine updates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus keeps changing, so scientists had to update the vaccines they give people. They chose new parts of the virus to fight based on lab tests. But sometimes it took too long to make these updated vaccines before the virus changed again. Now, researchers want to know if these updates help our immune system better fight off newer versions of the virus.

They looked at data showing how well different vaccine parts worked against recent and older viruses. They found that using the new, updated parts in a booster dose helped build stronger antibodies compared to old vaccines. This means people might get more protection even if the virus changed while they were waiting for the update.

This information helps doctors decide when to give updated doses of vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ongoing evolution of the SARS-CoV-2 virus has led to a move to update vaccine antigens in 2022 and 2023. These updated antigens were chosen and approved based largely on in vitro neutralisation titres against recent SARS-CoV-2 variants. However, unavoidable delays in vaccine manufacture and distribution meant that the updated booster vaccine was no longer well-matched to the circulating SARS-CoV-2 variant by the time of its deployment. Understanding whether the updating of booster vaccine antigens improves immune responses to subsequent SARS-CoV-2 circulating variants is a major priority in justifying future vaccine updates. Here we analyse all available data on the immunogenicity of variants containing SARS-CoV-2 vaccines and their ability to neutralise later circulating SARS-CoV-2 variants. We find that updated booster antigens give a 1.4-fold [95% CI: 1.07‚Äì1.82] greater increase in neutralising antibody levels when compared with a historical vaccine immunogen. We then use this to predict the relative protection that can be expected from an updated vaccine even when the circulating variant has evolved away from the updated vaccine immunogen. These findings help inform the rollout of future booster vaccination programmes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52194-9" target="_blank">https://www.nature.com/articles/s41467-024-52194-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41591-024-03278-y" target="_blank">
            ü§ñ SARS-CoV-2 antibodies not found in bone marrow after mRNA vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-507', this)">
            More</button></h4>
            <div id="toggle-507" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2-specific plasma cells are not durably established in the bone marrow long-lived compartment after mRNA vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Simplified abstract:
MRNA vaccines against COVID-19 are good at stopping serious illness. But months later, people's body doesn't have many protection antibodies anymore. We looked inside people‚Äôs bones where antibodies are made to find out why. We found that even though some special antibody-making cells were still around, the main place these cells live (long-lived plasma cells) was mostly empty for COVID-19 antibodies. This might explain why after a while, people's body doesn't have enough protection against COVID-19 anymore.

So basically, mRNA vaccines make good antibodies, but they run out quickly because there aren‚Äôt many of the main antibody-making cells left in bones.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) mRNA vaccines are effective at protecting from severe disease, but the protective antibodies wane rapidly even though SARS-CoV-2-specific plasma cells can be found in the bone marrow (BM). Here, to explore this paradox, we enrolled 19 healthy adults at 2.5‚Äì33‚Äâmonths after receipt of a SARS-CoV-2 mRNA vaccine and measured influenza-, tetanus- or SARS-CoV-2-specific antibody-secreting cells (ASCs) in long-lived plasma cell (LLPC) and non-LLPC subsets within the BM. Only influenza- and tetanus-specific ASCs were readily detected in the LLPCs, whereas SARS-CoV-2 specificities were mostly absent. The ratios of non-LLPC:LLPC for influenza, tetanus and SARS-CoV-2 were 0.61, 0.44 and 29.07, respectively. In five patients with known PCR-proven history of recent infection and vaccination, SARS-CoV-2-specific ASCs were mostly absent from the LLPCs. We show similar results with measurement for secreted antibodies from BM ASC culture supernatant. While serum IgG titers specific for influenza and tetanus correlated with IgG LLPCs, serum IgG levels for SARS-CoV-2, which waned within 3‚Äì6‚Äâmonths after vaccination, were associated with IgG non-LLPCs. In all, our studies suggest that rapid waning of SARS-CoV-2-specific serum antibodies could be accounted for by the absence of BM LLPCs after these mRNA vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-03278-y" target="_blank">https://www.nature.com/articles/s41591-024-03278-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52689-5" target="_blank">
            ü§ñ SARS-CoV-2 T cells in nose quick react with extra IFN-Œ≥</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-508', this)">
            More</button></h4>
            <div id="toggle-508" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 spike-specific nasal-resident CD49a+CD8+memory T cells exert immediate effector functions with enhanced IFN-Œ≥ production</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that after getting vaccinated against COVID-19, some people still get infected (called breakthrough infections). In their noses, there are special immune cells called T cells that recognize the virus and try to fight it. These T cells can stay in the nose for a long time‚Äîup to a year‚Äîand they respond quickly when they see pieces of the virus again. The type of T cell that does this is identified as CD49a+.

These T cells help by making chemicals called IFN-Œ≥, which signals other immune cells to attack the virus. They also stay in their resting homes (tissue-resident marker) in the nose even after a long time without seeing the virus again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Virus-specific nasal resident T cells are important for protection against subsequent infection with a similar virus. Here we examine the phenotypes and functions of SARS-CoV-2-specific T cells in the nasal mucosa of vaccinated individuals with breakthrough infection (BTI) or without infection. Nasal tissues are obtained from participants during sinus surgery. Analysis of activation-induced markers implicates that a considerable proportion of spike (S)-reactive nasal CD8+T cells express CD103, a tissue-resident marker. MHC-I multimer staining is performed to analyze the ex vivo phenotype and function of SARS-CoV-2 S-specific CD8+T cells. We detect multimer+CD8+T cells with tissue-resident phenotypes in nasal tissue samples from vaccinees without infection as well as vaccinees with BTI. Multimer+CD8+T cells remain present in nasal tissues over one year after the last exposure to S antigen, although the frequency decreases. Upon direct ex vivo stimulation with epitope peptides, nasal multimer+CD8+T cells‚Äìparticularly the CD49a+subset‚Äìexhibit immediate effector functions, including IFN-Œ≥ production. CITE-seq analysis of S-reactive AIM+CD8+T cells confirms the enhanced effector function of the CD49a+subset. These findings indicate that among individuals previously exposed to S antigen by vaccination or BTI, S-specific nasal-resident CD49a+CD8+memory T cells can rapidly respond to SARS-CoV-2 during infection or reinfection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52689-5" target="_blank">https://www.nature.com/articles/s41467-024-52689-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52803-7" target="_blank">
            ü§ñ CGAS-STING pathway limits COVID-19 in some lung cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-509', this)">
            More</button></h4>
            <div id="toggle-509" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A basally active cGAS-STING pathway limits SARS-CoV-2 replication in a subset of ACE2 positive airway cell models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The new coronavirus (SARS-CoV-2) can't enter some types of lung cells even when it should be able to. Researchers found that this might be because these cells are already stressed, leading to the release of DNA from their mitochondria (a part inside cells). This causes the cGAS-STING pathway and other immune responses to get activated without being needed, stopping the virus from growing.

By changing how these pathways work or using medicine to stop them, researchers were able to make the lung cells allow the virus to grow better. They also found that when they removed parts of the body's defense system (like cGAS-STING) in some lung cell samples, more coronavirus could grow there and cause less immune reactions.

In summary, the stress inside certain lung cells seems to be stopping the coronavirus from growing properly, even if it‚Äôs supposed to enter them. How much this happens depends on how many of these special parts (like ACE2) are present in those cells.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Host factors that define the cellular tropism of SARS-CoV-2 beyond the cognate ACE2 receptor are poorly defined. Here we report that SARS-CoV-2 replication is restricted at a post-entry step in a number of ACE2-positive airway-derived cell lines due to tonic activation of the cGAS-STING pathway mediated by mitochondrial DNA leakage and naturally occurring cGAS and STING variants. Genetic and pharmacological inhibition of the cGAS-STING and type I/III IFN pathways as well as ACE2 overexpression overcome these blocks. SARS-CoV-2 replication in STING knockout cell lines and primary airway cultures induces ISG expression but only in uninfected bystander cells, demonstrating efficient antagonism of the type I/III IFN-pathway in productively infected cells. Pharmacological inhibition of STING in primary airway cells enhances SARS-CoV-2 replication and reduces virus-induced innate immune activation. Together, our study highlights that tonic activation of the cGAS-STING and IFN pathways can impact SARS-CoV-2 cellular tropism in a manner dependent on ACE2 expression levels.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52803-7" target="_blank">https://www.nature.com/articles/s41467-024-52803-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-61827-4" target="_blank">
            ü§ñ Key factors in digital decentralized long COVID clinical trials: pacing intervention study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-510', this)">
            More</button></h4>
            <div id="toggle-510" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Key considerations for digital decentralised clinical trials from a feasibility study assessing pacing interventions for long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Long COVID is a condition that causes lots of problems like tiredness, trouble breathing, and brain fog. Many people who get Long COVID feel very sick and it's hard for doctors to find good treatments. 

Doctors need tools to test new ways to help people with Long COVID, but they don't have many. We made an online system where patients can join different groups doing different experiments at home. It helped us see if certain methods work better.

We learned that getting people to join these studies is really hard, and some people pretend to be in the study when they're not. Also, many people stop joining before we finish all our tests. 

We found ways to solve some of these problems so doctors can do more effective online studies for Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post COVID-19 condition or long COVID is highly prevalent and often debilitating, with key symptoms including fatigue, breathlessness, and brain fog. There is currently a lack of evidence-based treatments for this highly complex syndrome. There is a need for clinical trial platforms to rapidly evaluate nonpharmacological treatments to support affected individuals with symptom management. We co-produced a mixed methods feasibility study to evaluate a multi-arm digital decentralised clinical trial (DCT) platform to assess non-pharmacological interventions for Long COVID, using pacing interventions as an exemplar. The study demonstrated that the platform was able to successfully e-consent participants, randomise them into one of four intervention arms, capture baseline data, and capture outcomes relevant to a health economic evaluation. The study also highlighted several challenges, including difficulties with recruitment, imposter participants, and high attrition rates. We highlight how these challenges can potentially be mitigated to make a fully powered DCT more feasible.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-61827-4" target="_blank">https://www.nature.com/articles/s41598-024-61827-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52638-2" target="_blank">
            ü§ñ Impact of 6-Month Travel Moratorium on Malaria on Bioko Island</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-511', this)">
            More</button></h4>
            <div id="toggle-511" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of six-month COVID-19 travel moratorium onPlasmodium falciparumprevalence on Bioko Island, Equatorial Guinea</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The spread of malaria on Bioko Island is partly caused by people bringing the disease from other places. Scientists wanted to see how much this importation affects local cases. They used a six-month period when people weren't allowed to travel in and out as a test. Before and after this time, they checked if more people had malaria where many visitors come from. This showed that having visitors increases the chances of getting malaria. So stopping visitors could help reduce the number of malaria cases on the island.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Importation of malaria infections is a suspected driver of sustained malaria prevalence on areas of Bioko Island, Equatorial Guinea. Quantifying the impact of imported infections is difficult because of the dynamic nature of the disease and complexity of designing a randomized trial. We leverage a six-month travel moratorium in and out of Bioko Island during the initial COVID-19 pandemic response to evaluate the contribution of imported infections to malaria prevalence on Bioko Island. Using a difference in differences design and data from island wide household surveys conducted before (2019) and after (2020) the travel moratorium, we compare the change in prevalence between areas of low historical travel to those with high historical travel. Here, we report that in the absence of a travel moratorium, the prevalence of infection in high travel areas was expected to be 9% higher than observed, highlighting the importance of control measures that target imported infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52638-2" target="_blank">https://www.nature.com/articles/s41467-024-52638-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73311-0" target="_blank">
            ü§ñ COVID-19 impact on brain function: reduced cognitive clarity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-512', this)">
            More</button></h4>
            <div id="toggle-512" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Altered functional brain connectivity, efficiency, and information flow associated with brain fog after mild to moderate COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus makes it harder for people to think clearly. We tested 49 adults, 25 who had been sick with COVID and 24 who hadn‚Äôt. The group that had COVID did worse on thinking tests. Their brains also showed less connection between different parts compared to the other group. Less brain connections mean it‚Äôs harder to process information efficiently. This could be a way for the brain to keep working but feel like it's foggy or fuzzy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 is associated with increased risk for cognitive decline but very little is known regarding the neural mechanisms of this risk. We enrolled 49 adults (55% female, mean age‚Äâ=‚Äâ30.7‚Äâ¬± 8.7), 25 with and 24 without a history of COVID-19 infection. We administered standardized tests of cognitive function and acquired brain connectivity data using MRI. The COVID-19 group demonstrated significantly lower cognitive function (W‚Äâ=‚Äâ475, p‚Äâ<‚Äâ0.001, effect size r‚Äâ=‚Äâ0.58) and lower functional connectivity in multiple brain regions (mean t‚Äâ=‚Äâ3.47‚Äâ¬±0.36, p‚Äâ=‚Äâ0.03, corrected, effect size d‚Äâ=‚Äâ0.92 to 1.5). Hypo-connectivity of these regions was inversely correlated with subjective cognitive function and directly correlated with fatigue (p‚Äâ<‚Äâ0.05, corrected). These regions demonstrated significantly reduced local efficiency (p‚Äâ<‚Äâ0.026, corrected) and altered effective connectivity (p‚Äâ<‚Äâ0.001, corrected). COVID-19 may have a widespread effect on the functional connectome characterized by lower functional connectivity and altered patterns of information processing efficiency and effective information flow. This may serve as an adaptation to the pathology of SARS-CoV-2 wherein the brain can continue functioning at near expected objective levels, but patients experience lowered efficiency as brain fog.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73311-0" target="_blank">https://www.nature.com/articles/s41598-024-73311-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73475-9" target="_blank">
            ü§ñ Clinical features and imaging in neurological Wilson's disease with claustrum damage</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-513', this)">
            More</button></h4>
            <div id="toggle-513" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical and neuroimaging features in neurological Wilson‚Äôs disease with claustrum lesions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Our study looked at how often patients with a rare brain condition called Wilson's disease have problems in an area of their brain called the claustrum. Early studies said this could happen anywhere from 1.8% to 75%. We wanted to find out more about these cases, like what symptoms they had and why it might be happening.

In our study, we looked at MRI scans of 668 people with Wilson's disease. Out of the 443 who had issues shown on their brain scan, only 3 had bright claustrum signs seen on images. These signs were:
- Appeared in both sides of the brain
- Not isolated to just one part
- Usually showed up only when people had very bad symptoms

The bright claustrum sign might be a marker that inflammation is happening in the brain. The area was also hyperintense, meaning it appeared brighter on scans.

So basically, we found that problems with this brain area are not common but usually appear in people who have really severe symptoms of Wilson's disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>According to early research, the incidence of claustrum lesions in patients with neurological Wilson‚Äôs disease (WD) was inconsistent, ranging from 1.8 to 75% on magnetic resonance imaging (MRI). Our study aims to explore the incidence, clinical presentation features, iconography features, and possible pathological mechanisms in WD patients with claustrum lesions on magnetic resonance imaging (MRI), to characterize the clinical, and brain imaging findings and possible pathological mechanisms in the patients with WD. Retrospective cases meeting the inclusion criteria were studied for analyzing MRI characteristics and associated physicochemical examination data in neurological WD patients with claustrum lesions. 443 (66.3%) with brain MRI abnormalities were screened from 668 WD patients. The three (0.7%) patients with the claustrum lesions characteristics on MRI images were: (a) ‚Äúbright claustrum‚Äù in T2-weighted and FLAIR sequences, (b) bilateral symmetrical, (c) non-isolated lesions, (d) occurred only in severe neurological manifestations. The claustrum lesions are not common in neurological WD and mainly appear in cases with severe neurological symptoms. On MRI, the ‚Äúbright claustrum‚Äù signs may be a radiographic marker of neuroinflammation, the features of the lesions showed bilateral symmetry, and hyperintensity signals on T2-weighted, FLAIR, and DWI.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73475-9" target="_blank">https://www.nature.com/articles/s41598-024-73475-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73613-3" target="_blank">
            ü§ñ Novel Isothermal Whole Genome Sequencing for Monkeypox Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-514', this)">
            More</button></h4>
            <div id="toggle-514" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel isothermal whole genome sequencing approach for Monkeypox Virus</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Monkeypox virus causes a disease similar to chickenpox. Since May 2022, there have been more cases of people catching the virus from each other outside places where it's usually found.

Scientists use something called Whole Genome Sequencing (WGS) to track how the virus changes over time during outbreaks or for watching many diseases that could affect people. There are different ways to do this sequencing, like using big machines or special chemicals. The new method uses a DNA polymerase enzyme called Phi29 and a technique called multiple displacement amplification (MDA). This allows them to get almost all the virus's genetic material without needing lots of steps.

The scientists tested their new method with different types of samples - some from people who got sick, and other kinds too. They found out that it can work well using just 6 special pieces of DNA that act as starting points (primers). 

Their new method is good for keeping track of how the virus changes during outbreaks or to watch for it in places where we might not know about cases yet. It works even without fancy equipment, so any lab could use it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox virus (MPXV) is the zoonotic agent responsible for mpox, an often-self-limiting pox-like disease. Since May 2022, an outbreak characterized by increased human-to-human transmission was detected outside the endemic regions. Whole genome sequencing (WGS) has been successfully used to keep track of viral evolution during outbreaks or for surveillance of multiple pathogens of public health significance. Current WGS protocols for MPXV are either based on metagenomic sequencing or tiled-PCR amplification. The latter allows multiplexing due to the efficient enrichment of the viral DNA, however, mutations or the presence of different clades can negatively influence genome coverage yield. Here, we present the establishment of a novel isothermal WGS method for MPXV based on Phi29 DNA polymerase-based multiple displacement amplification (MDA) properties making use of only 6 primers. This approach yielded from 88% up to 100% genome coverage using either alkaline denatured extracted DNA or clinical material as starting material, with the highest coverage generated by clinical material. We demonstrate that this novel isothermal WGS protocol is suitable for monitoring viral evolution during MPXV outbreaks and surveillance in any conventional laboratory setting.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73613-3" target="_blank">https://www.nature.com/articles/s41598-024-73613-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-20</strong> <a href="https://www.nature.com/articles/s41598-024-72417-9" target="_blank">
            ü§ñ A Simple Way to Test for COVID-19 Delta Variant Using a Color Changing Reaction</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-515', this)">
            More</button></h4>
            <div id="toggle-515" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A colorimetric reverse-transcription loop-mediated isothermal amplification method targeting the L452R mutation to detect the Delta variant of SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new coronavirus called SARS-CoV-2 has spread quickly around the world, causing problems for people and economies. A variant of this virus, known as Delta, is still widespread and can make COVID-19 vaccines less effective. To control the spread of the virus, scientists need to be able to detect it quickly.

This study developed a new test that can specifically identify the Delta variant of SARS-CoV-2. The test uses a color-changing reaction to show whether someone has the virus or not. When tested with 126 real samples, the test was highly accurate and sensitive, making it a useful tool for tracking COVID-19 and developing strategies to control its spread in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has triggered global difficulties for both individuals and economies, with new variants continuing to emerge. The Delta variant of SARS-CoV-2 remains most prevalent worldwide, and it affects the efficacy of coronavirus disease 2019 (COVID-19) vaccination. Expedited testing to detect the Delta variant of SARS-CoV-2 and monitor viral transmission is necessary. This study aimed to develop and evaluate a colorimetric reverse-transcription loop-mediated isothermal amplification (RT-LAMP) technique targeting the L452R mutation in the S gene for the specific detection of the Delta variant. In the test, positivity was indicated as a color change from purple to yellow. The assay‚Äôs 95% limit of detection was 57 copies per reaction for the L452R (U1355G)-specific standard plasmid. Using 126 clinical samples, our assay displayed 100% specificity, 97.06% sensitivity, and 98.41% accuracy in identifying the Delta variant of SARS-CoV-2 compared to real-time RT-PCR. To our knowledge, this is the first colorimetric RT-LAMP assay that can differentiate the Delta variant from its generic SARS-CoV-2, enabling it as an approach for studying COVID-19 demography and facilitating proper effective control measure establishment to fight against the reemerging variants of SARS-CoV-2 in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72417-9" target="_blank">https://www.nature.com/articles/s41598-024-72417-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-20</strong> <a href="https://www.nature.com/articles/s41598-024-73012-8" target="_blank">
            ü§ñ Mpox in Macaques with Endo and Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-516', this)">
            More</button></h4>
            <div id="toggle-516" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mpox infection of stromal cells and macrophages of macaque with endometriosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The mpox outbreak in 2022-2023 was a new health problem that showed mpox can be spread through sex, though it mostly affected men who have sex with other men (MSM). Women are also at risk, especially during pregnancy.

We studied some monkeys to see how they got infected. We found that mpox didn't just live in their skin - it was also inside the vagina and uterus of the monkeys.

The study showed mpox could grow in special tissue called endometriosis. This is a condition where tissue like what grows in your uterus but is found elsewhere, usually on other organs like the ovaries. 

We learned that mpox likes to live in certain types of cells - specifically ones that help protect against infections and some of the cells inside the reproductive system.

The study also suggested women with this condition might be more at risk for bad things happening if they get mpox.

In general, this research shows us what happens when a disease like mpox affects the genital area. The monkeys were used because they're similar to humans in many ways and can help understand how diseases work.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The mpox outbreak of 2022‚Äì2023 represented a new global health challenge and recognition of mpox as a sexually transmitted disease. The majority of cases were reported in men who have sex with men (MSM), but women are also susceptible, especially during pregnancy. We evaluated the reproductive tracts of a subset of macaques from a large rechallenge study of mpox infection with virus from the 2022 outbreak and identified intraabdominal mpox replication associated with endometriosis. Mpox virus (MPXV) was found not only in skin, but in the cervix, the uterus, and periovarian endometriotic lesions of the affected macaque. Mpox replication preferentially targeted vimentin-positive poorly differentiated endometriotic stromal tissue and infiltrating macrophages in the reproductive tract. Mpox tropism for stromal cells and macrophages has broad implications for mpox pathogenesis and associated clinical syndromes. In addition, women with endometriosis may be at heightened risk for adverse outcomes associated with mpox infection. The rhesus macaque provides rare insight into this disease and the potential complications of mpox infection in the context of genitourinary tract disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73012-8" target="_blank">https://www.nature.com/articles/s41598-024-73012-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-19</strong> <a href="https://www.nature.com/articles/s41467-024-52246-0" target="_blank">
            ü§ñ Severe fever causes kids' immune cells to fail</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-517', this)">
            More</button></h4>
            <div id="toggle-517" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Shared neutrophil and T cell dysfunction is accompanied by a distinct interferon signature during severe febrile illnesses in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This abstract is about studying how the body's immune system reacts differently when kids get very sick from various illnesses, like coronavirus or bacterial infections. They looked at blood samples from 74 kids with a condition called MIS-C (which happens after they had COVID), 30 with bacterial infections, 16 with viral infections, and 8 with Kawasaki disease. 

They found that in most cases, the body's neutrophils (a type of white blood cell) were overreacting or dying too quickly. They also noticed that T cells from kids with MIS-C and bacterial infections didn't work as well as they should.

For viral infections like those from viruses other than SARS-CoV-2, their immune system showed less response to interferons, which are important for fighting off infections. This finding could help doctors figure out better ways to treat very sick children who have these illnesses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe febrile illnesses in children encompass life-threatening organ dysfunction caused by diverse pathogens and other severe inflammatory syndromes. A comparative approach to these illnesses may identify shared and distinct features of host immune dysfunction amenable to immunomodulation. Here, using immunophenotyping with mass cytometry and cell stimulation experiments, we illustrate trajectories of immune dysfunction in 74 children with multi-system inflammatory syndrome in children (MIS-C) associated with SARS-CoV-2, 30 with bacterial infection, 16 with viral infection, 8 with Kawasaki disease, and 42 controls. We explore these findings in a secondary cohort of 500 children with these illnesses and 134 controls. We show that neutrophil activation and apoptosis are prominent in multi-system inflammatory syndrome, and that this is partially shared with bacterial infection. We show that memory T cells from patients with multi-system inflammatory syndrome and bacterial infection are exhausted. In contrast, we show viral infection to be characterized by a distinct signature of decreased interferon signaling and lower interferon receptor gene expression. Improved understanding of immune dysfunction may improve approaches to immunomodulator therapy in severe febrile illnesses in children.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52246-0" target="_blank">https://www.nature.com/articles/s41467-024-52246-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-71413-3" target="_blank">
            ü§ñ UPLC-PDA method for three drugs in plasma</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-518', this)">
            More</button></h4>
            <div id="toggle-518" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>UPLC-PDA factorial design assisted method for simultaneous determination of oseltamivir, dexamethasone, and remdesivir in human plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new way of testing medicine levels in blood using UPLC (a fast and precise lab technique) has been created. This method can measure three medicines at once: oseltamivir phosphate, remdesivir, dexamethasone, and a special standard called daclatasvir dihydrochloride. The medicines are put into the system one by one using tiny needles, and then separated on a special column that uses a mixture of methanol (a clear liquid) and ammonium acetate solution to sort them out based on their colors at different wavelengths. It takes just 5 minutes to do all this! 

The method was tested for accuracy and reliability according to international guidelines. Environmental Impact Scores, Green Assessment of Processes in Analysis, and Acceptability and Goodness-Instrument Rating (Eco-score, GAPI, and AGREE) showed that it is a good and environmentally friendly way to measure these medicines. This means it's useful and won't harm the environment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A green and simple UPLC method was developed and optimized, adopting a factorial design for simultaneous determination of oseltamivir phosphate and remdesivir with dexamethasone as a co-administered drug in human plasma and using daclatasvir dihydrochloride as an internal standard within 5¬†min. The separation was established on UPLC column BEH C181.7¬†Œºm (2.1\(\times \)100.0¬†mm) connected to UPLC pre-column BEH 1.7¬†Œºm (2.1\(\times \)5.0¬†mm) at 50¬†¬∞C with an injection volume of 10 ŒºL. The photodiode array detector (PDA) was set at three wavelengths of 220, 315, and 245¬†nm for oseltamivir phosphate, the internal standard, and both dexamethasone and remdesivir, respectively. The mobile phase consisted of methanol and ammonium acetate solution (40¬†mM) adjusted to pH 4 in a ratio of 61.5:38.5 (v/v) with a flow rate of 0.25¬†mL¬†min‚àí1. The calibration curves were linear over 500.0‚Äì5000.0¬†ng¬†mL‚àí1for oseltamivir phosphate, over 10.0‚Äì500.0¬†ng¬†mL‚àí1and 500.0‚Äì5000.0¬†ng¬†mL‚àí1for dexamethasone, and over 20.0‚Äì500¬†ng¬†mL‚àí1and 500.0‚Äì5000.0¬†ng¬†mL‚àí1for remdesivir. The Gibbs free energy and Van't Hoff plots were used to investigate the effect of column oven temperatures on retention times. Fluoride-EDTA anticoagulant showed inhibition activity on the esterase enzyme in plasma. The proposed method was validated according to the M10 ICH, FDA, and EMA‚Äôs bioanalytical guidelines. According to Eco-score, GAPI, and AGREE criteria, the proposed method was considered acceptable green.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71413-3" target="_blank">https://www.nature.com/articles/s41598-024-71413-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41467-024-51924-3" target="_blank">
            ü§ñ Emerging Antiviral-Resistant COVID Variants in People With Weakened Immune Systems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-519', this)">
            More</button></h4>
            <div id="toggle-519" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Emergence of transmissible SARS-CoV-2 variants with decreased sensitivity to antivirals in immunocompromised patients with persistent infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We studied how antiviral treatments affect the COVID-19 virus in people with weakened immune systems. Fifteen patients were given different treatments, including remdesivir and nirmatrelvir-ritonavir, to see if the virus developed resistance.

The results showed that nine out of 15 patients had viruses with mutations that made them harder to treat. We isolated a COVID-19 virus from one patient that was resistant to two common antiviral medications: remdesivir and nirmatrelvir.

This study is important because it shows that the virus can develop resistance to these treatments, even after prolonged use. The infected virus was also shown to be transmissible to other animals in a lab experiment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We investigated the impact of antiviral treatment on the emergence of SARS-CoV-2 resistance during persistent infections in immunocompromised patients (n=‚Äâ15). All patients received remdesivir and some also received nirmatrelvir-ritonavir (n=‚Äâ3) or therapeutic monoclonal antibodies (n=‚Äâ4). Sequence analysis showed that nine patients carried viruses with mutations in the nsp12 (RNA dependent RNA polymerase), while four had viruses with nsp5 (3C protease) mutations. Infectious SARS-CoV-2 with a double mutation in nsp5 (T169I) and nsp12 (V792I) was recovered from respiratory secretions 77 days after initial COVID-19 diagnosis from a patient sequentially¬†treated with nirmatrelvir-ritonavir¬†and remdesivir. In vitro characterization confirmed its decreased sensitivity to remdesivir and nirmatrelvir, which was overcome by combined antiviral treatment. Studies in golden Syrian hamsters demonstrated efficient transmission to contact animals. This study documents the isolation of SARS-CoV-2 carrying resistance mutations to both nirmatrelvir and remdesivir from a patient and demonstrates its transmissibility in vivo.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51924-3" target="_blank">https://www.nature.com/articles/s41467-024-51924-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41541-024-00961-6" target="_blank">
            ü§ñ A New Vaccine Approach: Using Macrophages to Boost Immune Response Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-520', this)">
            More</button></h4>
            <div id="toggle-520" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Low-frequency CD8+T cells induced by SIGN-R1+macrophage-targeted vaccine confer SARS-CoV-2 clearance in mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 vaccine needs two types of help from our immune system: special cells called T cells that can recognize and attack the virus, and antibodies that neutralize it. Our previous research showed that using a special delivery system called pullulan nanogel (PNG) could make these T cells stronger. We found out that PNG works by binding to a specific receptor on certain immune cells in our lymph nodes.

In this study, we tested if using PNG with a part of the COVID-19 virus's spike protein as the vaccine could help protect against infection. In mice with COVID-19-like symptoms, we saw that the vaccine decreased the viral load and helped them survive longer. We also found out that even low-frequency T cells that responded quickly to the virus were important for clearing the virus.

Overall, our study suggests that targeting specific immune cells in the lymph nodes can help make vaccines more effective by inducing strong T cell responses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccine-induced T cells and neutralizing antibodies are essential for protection against SARS-CoV-2. Previously, we demonstrated that an antigen delivery system, pullulan nanogel (PNG), delivers vaccine antigen to lymph node medullary macrophages and thereby enhances the induction of specific CD8+T cells. In this study, we revealed that medullary macrophage-selective delivery by PNG depends on its binding to a C-type lectin SIGN-R1. In a K18-hACE2 mouse model of SARS-CoV-2 infection, vaccination with a PNG-encapsulated receptor-binding domain of spike protein decreased the viral load and prolonged the survival in the CD8+T cell- and B cell-dependent manners. T cell receptor repertoire analysis revealed that although the vaccine induced T cells at various frequencies, low-frequency specific T cells mainly promoted virus clearance. Thus, the induction of specific CD8+T cells that respond quickly to viral infection, even at low frequencies, is important for vaccine efficacy and can be achieved by SIGN-R1+medullary macrophage-targeted antigen delivery.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00961-6" target="_blank">https://www.nature.com/articles/s41541-024-00961-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-71791-8" target="_blank">
            ü§ñ Domestic Cats and COVID-19: Comparing Virus Severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-521', this)">
            More</button></h4>
            <div id="toggle-521" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The comparison of pathogenicity among SARS-CoV-2 variants in domestic cats</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists found that a type of coronavirus called SARS-CoV-2 can infect domestic cats and even wild animals. They wanted to know if cats play a role in spreading this virus, so they did experiments on cats to see how well it spreads and how their bodies react.

The results showed that cats are very susceptible to the SARS-CoV-2 virus and can spread it for up to 7 days after being infected. However, one type of the virus, called Omicron, didn't spread as well in cat tissues and caused less severe symptoms. The study also found that cats' bodies produced antibodies to fight off the virus.

This suggests that while cats might be able to catch SARS-CoV-2, they're not very good at spreading it around, especially with the Omicron variant.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been detected or isolated from domestic cats. It is unclear whether cats play an important role in the SARS-CoV-2 transmission cycle. In this study, we examined the susceptibility of cats to SARS-CoV-2, including wild type and variants, by animal experiments. Cats inoculated with wild type, gamma, and delta variants secreted a large amount of SARS-CoV-2 for 1¬†week after the inoculation from nasal, oropharyngeal, and rectal routes. Only 100 TCID50of virus could infect cats and replicate well without severe clinical symptoms. In addition, one cat inoculated with wild type showed persistent virus secretion in feces for over 28¬†days post-inoculation (dpi). The titer of virus-neutralizing (VN) antibodies against SARS-CoV-2 increased from 11 dpi, reaching a peak at 14 dpi. However, the omicron variant could not replicate well in cat tissues and induced a lower titer of VN antibodies. It is concluded that cats were highly susceptible to SARS-CoV-2 infection, but not to the Omicron Variant, which caused the attenuated pathogenicity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71791-8" target="_blank">https://www.nature.com/articles/s41598-024-71791-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41392-024-01956-4" target="_blank">
            ü§ñ Identifying ACTN4 as a new potential treatment for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-522', this)">
            More</button></h4>
            <div id="toggle-522" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterization of ACTN4 as a novel antiviral target against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists are trying to find new ways to stop COVID-19 from spreading. They've discovered that a protein called ACTN4 plays a crucial role in this process. When ACTN4 is reduced, it makes it harder for COVID-19 to multiply inside cells.

Researchers also found two substances, YS-49 and demethyl-coclaurine, that can block COVID-19 infection. These substances work by targeting the protein that helps COVID-19 replicate.

This study shows how important ACTN4 is in stopping COVID-19 from spreading and reveals new potential candidates for developing medicines to fight against the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The various mutations in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pose a substantial challenge in mitigating the viral infectivity. The identification of novel host factors influencing SARS-CoV-2 replication holds potential for discovering new targets for broad-spectrum antiviral drugs that can combat future viral mutations. In this study, potential host factors regulated by SARS-CoV-2 infection were screened through different high-throughput sequencing techniques and further identified in cells. Subsequent analysis and experiments showed that the reduction of m6A modification level onACTN4(Alpha-actinin-4) mRNA leads to a decrease in mRNA stability and translation efficiency, ultimately inhibiting ACTN4 expression. In addition, ACTN4 was demonstrated to target nsp12 for binding and characterized as a competitor for SARS-CoV-2 RNA and the RNA-dependent RNA polymerase complex, thereby impeding viral replication. Furthermore, two ACTN4 agonists, YS-49 and demethyl-coclaurine, were found to dose-dependently inhibit SARS-CoV-2 infection in both Huh7 cells and K18-hACE2 transgenic mice. Collectively, this study unveils the pivotal role of ACTN4 in SARS-CoV-2 infection, offering novel insights into the intricate interplay between the virus and host cells, and reveals two potential candidates for future anti-SARS-CoV-2 drug development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01956-4" target="_blank">https://www.nature.com/articles/s41392-024-01956-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-67504-w" target="_blank">
            ü§ñ COVID-19 Outcomes Predicted by Early Host Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-523', this)">
            More</button></h4>
            <div id="toggle-523" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Oral SARS-CoV-2 host responses predict the early COVID-19 disease course</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied people who had COVID-19 to see how it affects their bodies, especially in their mouths and throats. They collected saliva and blood samples from 45 people and looked for signs of the virus and how their immune system responded.

They found that most people with COVID-19 had the virus in their saliva and throat wash for up to 4 weeks after getting sick. Some people even showed signs of the virus in their saliva when they didn't have symptoms anymore.

The researchers also discovered that people who were severely ill with COVID-19 tended to have more antibodies (which fight off infections) in their blood and saliva, especially in the early stages of the illness. They found that some people's immune system responded differently based on their sex, with women having higher levels of certain antibodies in their saliva.

The study suggests that the virus can replicate (multiply) in the mouth and throat for a while after symptoms go away, which may be important for understanding how COVID-19 is transmitted to others. The researchers also found evidence that some people's immune system may "confuse" the virus with other proteins in their body, which could affect how well they fight off the infection.

Overall, this study provides new insights into how COVID-19 affects the body and may help us understand more about the disease and how to prevent it from spreading.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Oral fluids provide ready detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and host responses. This study sought to evaluate relationships between oral virus, oral and systemic anti-SARS-CoV-2-specific antibodies, and symptoms. Oral fluids (saliva/throat wash (saliva/TW)) and serum were collected from asymptomatic and symptomatic, nasopharyngeal (NP) SARS-CoV-2 RT-qPCR+‚Äâhuman participants (n‚Äâ=‚Äâ45). SARS-CoV-2 RT-qPCR and N-antigen detection by immunoblot and lateral flow assay (LFA) were performed. RT-qPCR for subgenomic RNA (sgRNA) was sequence confirmed. SARS-CoV-2-anti-S protein RBD LFA and ELISA assessed IgM and IgG responses. Structural analysis identified host salivary molecules analogous to SARS-CoV-2-N-antigen. At time of enrollment (baseline, BL), LFA-detected N-antigen in 86% of TW and was immunoblot-confirmed. However, only 3/17 were saliva/TW qPCR+‚Äâ. Sixty percent of saliva and 83% of TW demonstrated persistent N-antigen at 4¬†weeks. N-antigen LFA signal in three anti-spike sero-negative participants suggested potential cross-detection of 4 structurally analogous salivary RNA binding proteins (alignment 19‚Äì29aa, RMSD 1‚Äì1.5 Angstroms). At enrollment, symptomatic participants demonstrated replication-associated sgRNA junctions, were IgG+‚Äâ(94%/100% in saliva/TW), and IgM+‚Äâ(63%/54%). At 4¬†weeks, SARS-CoV-2 IgG (100%/83%) and IgM (80%/67%) persisted. Oral and serum IgG correlated 100% with NP+‚ÄâPCR status. Cough and fatigue severity (p=‚Äâ0.010 and 0.018 respectively), and presence of weakness, nausea, and composite upper respiratory symptoms (p=‚Äâ0.037, 0.005, and 0.017, respectively) were negatively associated with saliva IgM but not TW or serum IgM. Throat wash IgM levels were higher in women compared to men, although the association did not reach statistical significance (median: 290 (female) versus 0.697,p=‚Äâ0.056). Important to transmission and disease course, oral viral replication and persistence showed clear relationships with select symptoms and early oral IgM responses during early infection. N-antigen cross-reactivity may reflect mimicry of structurally analogous host proteins.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67504-w" target="_blank">https://www.nature.com/articles/s41598-024-67504-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41598-024-71598-7" target="_blank">
            ü§ñ How sampling method affects sebum stability in mass spec studies.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-524', this)">
            More</button></h4>
            <div id="toggle-524" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>How storage post sampling influences the stability of sebum when used for mass spectrometry metabolomics analysis?</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Sebum is a natural oil produced by skin glands. Recent studies found this oil can show signs of diseases like Parkinson's. Since getting sebum is easy and doesn't hurt, doctors might use it in tests.

Researchers looked at how well they could test sebum over time and under different temperatures. They collected sebum from people, stored it for up to four weeks at room temperature or very cold (like in a freezer), and then tested it using two types of machines that break down the oil into tiny parts we can see. 

They didn't find any difference between how the samples looked when they were stored for different amounts of time or at different temperatures. But, some small groups of samples mixed up with others.

This study shows sebum is good to use in tests and doesn‚Äôt need to be kept cold forever. This means people can collect their own oil from home without worrying too much about it spoiling before they send it to the lab.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Sebum is a biofluid excreted by sebaceous glands in the skin. In recent years sebum has been shown to contain endogenous metabolites diagnostic of disease, with remarkable results for Parkinson‚Äôs Disease. Given that sebum sampling is facile and non-invasive, its potential for use in clinical biochemistry diagnostic assays should be explored including the parameters for standard operating procedures around collection, transport, and storage. To this aim we have here investigated the reproducibility of mass spectrometry data from sebum in relation to both storage temperature and length of storage. Sebum samples were collected from volunteers and stored for up to four weeks at a range of temperatures: ambient (circa17¬†¬∞C), ‚àí20¬†¬∞C and ‚àí80¬†¬∞C. Established extraction protocols were employed and samples were analysed by two chromatographic mass spectrometry techniques and data investigated using PCA, PLS-DA and ANOVA. We cannot discriminate samples as a function of storage temperature or time stored in unsupervised analysis using data acquired via TD‚ÄìGC‚ÄìMS and LC‚ÄìIM‚ÄìMS, although the sampling of volatiles was susceptible to batch effects. This study indicates that the requirements for storage and transport of sebum samples that may be used in clinical assays are less stringent than for liquid samples and indicate that sebum is suitable for remote and at home sampling prior to analysis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71598-7" target="_blank">https://www.nature.com/articles/s41598-024-71598-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s42003-024-06872-5" target="_blank">
            ü§ñ COVID-19 Becomes More Severe in Mice Without Important Immune Proteins</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-525', this)">
            More</button></h4>
            <div id="toggle-525" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deficiency ofTlr7andIrf7in mice increases the severity of COVID-19 through the reduced interferon production</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied how a deficiency in a certain receptor (Tlr7) affects the severity of COVID-19 in mice. They found that when this receptor is missing, the body has trouble producing interferons (IFNs), which are important for fighting the virus. The study showed that without Tlr7 and another related protein called Irf7, the mice developed more severe symptoms, had higher levels of the virus in their lungs, and took longer to produce antibodies against COVID-19. This suggests that the activation of these receptors is crucial for developing a strong defense against the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Toll-like receptor 7 (Tlr7)deficiency-accelerated severe COVID-19 is associated with reduced production of interferons (IFNs). However, the underlying mechanisms remain elusive. To address these questions, we utilizeTlr7andIrf7deficiency mice, single-cell RNA analysis together with bone marrow transplantation approaches. We demonstrate that at the early phase of infection, SARS-CoV-2 causes the upregulation ofTlr7,Irf7, and IFN pathways in the lungs of the infected mice. The deficiency ofTlr7andIrf7globally and/or in immune cells in mice increases the severity of COVID-19 via impaired IFN activation in both immune and/or non-immune cells, leading to increased lung viral loads. These effects are associated with reduced IFN alpha and gamma levels in the circulation. The deficiency ofTlr7tends to cause the reduced production and nuclear translocation of interferon regulatory factor 7 (IRF7) in the lungs of the infected mice, indicative of reduced IRF7 activation. Despite higher amounts of lung viral antigen,Tlr7orIrf7deficiency resulted in substantially reduced production of antibodies against SARS-CoV-2, thereby delaying the viral clearance. These results highlight the importance of the activation of TLR7 and IRF7 leading to IFN production on the development of innate and adaptive immunity against COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06872-5" target="_blank">https://www.nature.com/articles/s42003-024-06872-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41598-024-72453-5" target="_blank">
            ü§ñ Monitoring COVID-19 Antibodies in Blood and Saliva Samples</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-526', this)">
            More</button></h4>
            <div id="toggle-526" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Monitoring SARS-CoV-2 IgA, IgM and IgG antibodies in dried blood and saliva samples using antibody proximity extension assays (AbPEA)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists developed a new test that can detect if someone has antibodies against COVID-19 just by looking at a small blood or saliva sample on a paper disc. The test uses special molecules that bind to the antibodies and then helps create a tiny piece of DNA, which is measured using a machine. This makes it possible to quickly and easily check if people have been vaccinated or infected with COVID-19.

The study showed that this new test works well for detecting antibodies in blood samples, but less effectively in saliva samples. However, the results still suggest that it's a useful tool for monitoring the body's response to vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Using a modified proximity extension assay, total and immunoglobulin (Ig) class-specific anti-SARS-CoV-2 antibodies were sensitively and conveniently detected directly from √∏1.2¬†mm discs cut from dried blood and saliva spots (DBS and DSS) without the need for elution. For total Ig detection, antigen probes were prepared by conjugating recombinant spike protein subunit 1 (S1-RBD) to a pair of oligonucleotides. To detect isotype-specific antibody reactivity, one antigen probe was replaced with oligonucleotide-conjugated antibodies specific for antibody isotypes. Binding of pairs of oligonucleotide-conjugated probes to antibodies in patient samples brings oligonucleotides in proximity. An added DNA polymerase uses a transient hybridization between the oligonucleotides to prime synthesis of a DNA strand, which serves as a DNA amplicon that is quantified by real-time PCR. The S1-RBD-specific IgG, IgM, and IgA antibodies in DBS samples collected over the course of a first and second vaccination exhibited kinetics consistent with previous reports. Both DBS and DSS collected from 42 individuals in the autumn of 2023 showed significant level of total S1-RBD antibodies with a correlation of R‚Äâ=‚Äâ0.70. However, levels in DSS were generally 10 to 100-fold lower than in DBS. Anti-S1-RBD IgG and IgA in DSS demonstrated a correlation of R‚Äâ=‚Äâ0.6.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72453-5" target="_blank">https://www.nature.com/articles/s41598-024-72453-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41467-024-51021-5" target="_blank">
            ü§ñ Marburg virus survivor antibodies: Finding new vaccine targets</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-527', this)">
            More</button></h4>
            <div id="toggle-527" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Longitudinal proteome-wide antibody profiling in Marburg virus survivors identifies wing domain immunogen for vaccine design</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how the body makes antibodies after getting sick from the Marburg virus over many years. They found that different types of antibodies form over time:

1. The body starts by making lots of IgM and IgA antibodies quickly.
2. Later, it switches to making more IgG antibodies, which last longer.

The study also noticed:
- Antibodies against parts of a protein called GP are strongest.
- Some old antibody types like IgM and IgA get less common over time.
- The body forms strong bonds with special Fc receptors that can interact with the virus.

These findings suggest that even though many antibodies don't help stop the virus directly, they might still fight against it indirectly by binding to parts of these proteins. This could be useful for developing treatments and vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Limited knowledge exists on the quality of polyclonal antibody responses generated following Marburg virus (MARV) infection and its evolution in survivors. In this study, we evaluate MARV proteome-wide antibody repertoire longitudinally in convalescent phase approximately every six months for five years following MARV infection in ten human survivors. Differential kinetics were observed for IgM vs IgG vs IgA epitope diversity, antibody binding, antibody affinity maturation and Fc-receptor interaction to MARV proteins. Durability of MARV-neutralizing antibodies is low in survivors. MARV infection induces a diverse epitope repertoire with predominance against GP, VP40, VP30 and VP24 that persisted up to 5 years post-exposure. However, the IgM and IgA repertoire declines over time. Within MARV-GP, IgG recognize antigenic sites predominantly in the amino-terminus, wing domain and GP2-heptad repeat. Interestingly, MARV infection generates robust durable Fc…£RI, Fc…£RIIA and Fc…£RIIIA IgG-Fc receptor interactions. Immunization with immunodominant MARV epitopes reveals conserved wing region between GP1 and GP2, induces neutralizing antibodies against MARV. These findings demonstrate that MARV infection generates a diverse, long-lasting, non-neutralizing, IgG antibody repertoire that perturbs disease by Fc…£R activity. This information, along with discovery of neutralizing immunogen in wing domain, could aid in development of effective therapeutics and vaccines against Marburg virus.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51021-5" target="_blank">https://www.nature.com/articles/s41467-024-51021-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-13</strong> <a href="https://www.nature.com/articles/s41419-024-07043-4" target="_blank">
            ü§ñ Text Mining for Long COVID Symptoms Analysis in Hospitalized Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-528', this)">
            More</button></h4>
            <div id="toggle-528" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Text mining method to unravel long COVID‚Äôs clinical condition in hospitalized patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Long COVID is when you feel sick for a really long time after you get an illness like the flu or coronavirus. This can happen even weeks or months later. It's hard to understand because different people have different symptoms, and doctors use different words to describe them.

In this study, scientists used special computer tools to look at old medical records from a hospital in Brazil. They looked for patterns and similar terms that might represent the same condition. This helped them see what long COVID looks like across many patients. They also found tricky words (like those with extra parts or opposite meanings) that can be confusing.

By using these new computer tools, they hope to better understand this long-lasting sickness so doctors can help more people who are sick from it. These tools could work in other hospitals too!</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long COVID is characterized by persistent that extends symptoms beyond established timeframes. Its varied presentation across different populations and healthcare systems poses significant challenges in understanding its clinical manifestations and implications. In this study, we present a novel application of text mining technique to automatically extract unstructured data from a long COVID survey conducted at a prominent university hospital in S√£o Paulo, Brazil. Our phonetic text clustering (PTC) method enables the exploration of unstructured Electronic Healthcare Records (EHR) data to unify different written forms of similar terms into a single phonemic representation. We used n-gram text analysis to detect compound words and negated terms in Portuguese-BR, focusing on medical conditions and symptoms related to long COVID. By leveraging text mining, we aim to contribute to a deeper understanding of this chronic condition and its implications for healthcare systems globally. The model developed in this study has the potential for scalability and applicability in other healthcare settings, thereby supporting broader research efforts and informing clinical decision-making for long COVID patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-024-07043-4" target="_blank">https://www.nature.com/articles/s41419-024-07043-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-71535-8" target="_blank">
            ü§ñ Complications After COVID Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-529', this)">
            More</button></h4>
            <div id="toggle-529" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Symptom burden, coagulopathy and heart disease after acute SARS-CoV-2 infection in primary practice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists conducted a study called SETANTA to investigate how COVID-19 affects people's hearts. They looked at patients who had recently had COVID-19 and wanted to see if they still had symptoms or heart problems. The study found that many participants (83%) still had symptoms like fatigue, headaches, etc. 

The researchers also used tests to check the patients' hearts for damage and found that some of them had abnormal results. For example, about 2% of participants had inflammation in their hearts, while about 17% had a problem with their heart's ability to pump blood.

Another important finding was that many patients (32%) had high levels of a protein called von Willebrand factor antigen, which can indicate an increased risk of clotting. The study suggests that people who recently recovered from COVID-19 may be at higher risk for future health problems due to persistent symptoms and abnormal heart readings.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SETANTA (Study of HEarT DiseAse and ImmuNiTy After COVID-19 in Ireland) study aimed to investigate symptom burden and incidence of cardiac abnormalities after severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)/COVID-19 and to correlate these results with biomarkers of immunological response and coagulation. SETANTA was a prospective, single-arm observational cross-sectional study condcuted in a primary practice setting, and prospectively registered with ClinicalTrials.gov (identifier: NCT04823182). Patients with recent COVID-19 infection (‚â•‚Äâ6¬†weeks and ‚â§‚Äâ12¬†months) were prospectively enrolled. Primary outcomes of interest were markers of cardiac injury detected by cardiac magnetic resonance imaging (CMR), which included left ventricular ejection fraction, late gadolinium enhancement and pericardial abnormalities, as well as relevant biomarkers testing immunological response and coagulopathy. 100 patients (n‚Äâ=‚Äâ129 approached) were included, amongst which 64% were female. Mean age of the total cohort was 45.2 years. The median (interquartile range) time interval between COVID-19 infection and enrolment was 189 [125, 246]¬†days. 83% of participants had at least one persistent symptom, while 96% had positive serology for prior SARS-CoV-2 infection. Late gadolinium enhancement, pericardial effusion, was present in 2.2% and 8.3% respectively, while left ventricular ejection fraction was below the normal reference limit in 17.4% of patients. Von Willebrand factor antigen was elevated in 32.7% of patients and Fibrinogen and D-Dimer levels were found to be elevated in 10.2% and 11.1% of patients, respectively. In a cohort of primary practice patients recently recovered from SARS-CoV-2 infection, prevalence of persistent symptoms and markers of abnormal coagulation were high, despite a lower frequency of abnormalities on CMR compared with prior reports of patients assessed in a hospital setting.Trial Registration: Clinicaltrials.gov, NCT04823182 (prospectively registered on 30th March 2021).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71535-8" target="_blank">https://www.nature.com/articles/s41598-024-71535-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s43856-024-00599-4" target="_blank">
            ü§ñ People Taking SGLT2 Inhibitors for COVID-19 May Face Cardiovascular and Kidney Problems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-530', this)">
            More</button></h4>
            <div id="toggle-530" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Adverse cardiovascular and kidney outcomes in people with SARS-CoV-2 treated with SGLT2 inhibitors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Background: Some people who got infected with COVID-19 also take medicine for high blood sugar. We wanted to know if taking a specific type of diabetes medication called SGLT2 inhibitors helps prevent heart and kidney problems.

Methods: We looked at medical records of over 100,000 veterans in the US who had COVID-19 and took diabetes medication between March 2020 and June 2023. Some took SGLT2 inhibitors while others took different types of diabetes medication.

Results: Our study found that people taking SGLT2 inhibitors were less likely to have serious heart problems (such as a heart attack or stroke) and kidney issues (such as needing dialysis). They also had fewer hospitalizations, anemia, and acute kidney injury. These benefits were seen after about 1.5 years of follow-up.

Conclusions: Taking SGLT2 inhibitors may help people with COVID-19 on diabetes medication avoid heart and kidney problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundWhether use of SGLT2 inhibitors reduces the risk of cardiovascular and kidney events in people who contracted SARS-CoV-2 infection is not clear.MethodsWe used the healthcare databases of the United States Department of Veterans Affairs to build a cohort of 107,776 participants on antihyperglycemic therapy and had SARS-CoV-2 infection between March 01, 2020 and June 10, 2023. Within them, 11,588 used SGLT2 inhibitors and 96,188 used other antihyperglycemics. We examined the risks of major adverse cardiovascular events (MACE)‚Äîa composite of death, myocardial infarction and stroke, and major adverse kidney events (MAKE)‚Äîa composite of death, eGFR decline >¬†50%, and end stage kidney disease after balancing baseline characteristics between groups through inverse probability weighting. Survival analyses were conducted to generate hazard ratio (HR) and absolute risk reduction per 100 person-years (ARR).ResultsOver a median follow up of 1.57 (IQR: 1.05‚Äì2.49) years, compared to the control group, SGLT2 inhibitors use is associated with reduced risk of MACE (HR 0.82 (0.77, 0.88), ARR 1.73 (1.21, 2.25)) and reduced risk of MAKE (HR 0.75 (0.71, 0.80), ARR 2.62 (2.13, 3.11)). Compared to the control group, SGLT2 inhibitors use is associated with reduced risk of the secondary outcomes of hospitalization (HR 0.94 (0.90, 0.98), ARR 1.06 (1.36, 1.76)), anemia (HR 0.71 (0.65, 0.76), ARR 2.43 (1.95, 2.90)), and acute kidney injury (HR 0.84 (0.79, 0.89), ARR 1.86 (1.29, 2.42)).ConclusionsAmong people with SARS-CoV-2 infection on antihyperglycemic therapy, compared to those on other antihyperglycemics, those on SGLT2 inhibitors have less risk of adverse cardiovascular and kidney outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00599-4" target="_blank">https://www.nature.com/articles/s43856-024-00599-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-72262-w" target="_blank">
            ü§ñ A nasal spray that protects against COVID-19 and helps maintain normal nose function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-531', this)">
            More</button></h4>
            <div id="toggle-531" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Astodrimer sodium nasal spray forms a barrier to SARS-CoV-2 in vitro and preserves normal mucociliary function in human nasal epithelium</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

COVID-19 can be very serious for some people, especially those with weakened immune systems. To protect against this and other respiratory infections, scientists are looking at ways to create sprays that can help block the virus from entering the body through the nose.

A new nasal spray called Astodrimer Sodium (AS) was tested along with four other types of sprays. The AS spray showed promise in blocking the virus and not harming the delicate cells in the nose. It also didn't interfere with the natural mucus-clearing function of the nose.

In contrast, some of the other sprays had limited ability to block the virus and even caused problems by slowing down or stopping the mucus-clearing process. The AS spray seems to be a safe and effective option for protecting against respiratory viruses in the nose.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 remains a severe condition for many including immunocompromised individuals. There remains a need for effective measures against this and other respiratory infections, which transmit via virus-laden droplets that reach the nasal or oral mucosae. Nasal sprays offer potential protection against viruses. Such formulations should preserve normal nasal mucociliary function. The antiviral barrier efficacy and effects on mucociliary function of astodrimer sodium nasal spray (AS-NS) were evaluated and compared with other available nasal sprays‚Äîlow pH hydroxypropyl methylcellulose (HPMC-NS), iota-carrageenan (Carr-NS), nitric oxide (NO-NS), and povidone iodine (PI-NS). Assays simulated clinical conditions. Antiviral barrier function and cell viability were assessed in airway cell monolayers, while a model of fully differentiated human nasal epithelium (MucilAir‚Ñ¢) was utilized to evaluate tissue integrity, cytotoxicity, cilia beating frequency, and mucociliary clearance. AS-NS reduced infectious virus in cell monolayers and demonstrated a benign cytotoxicity profile. In human nasal epithelium ex vivo, AS-NS had no impact on mucociliary function (cilia beating nor mucociliary clearance). Carr-NS, HPMC-NS, NO-NS and PI-NS demonstrated limited antiviral effects, while HPMC-NS caused inhibition of mucociliary function. Astodrimer sodium nasal spray demonstrates an acceptable nonclinical efficacy and safety profile as a barrier nasal spray against respiratory viral infection in the nasal cavity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72262-w" target="_blank">https://www.nature.com/articles/s41598-024-72262-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-71144-5" target="_blank">
            ü§ñ SARS Virus Protein Affects Calcium Signals in Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-532', this)">
            More</button></h4>
            <div id="toggle-532" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 envelope protein alters calcium signaling via SERCA interactions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 virus is still difficult to treat, especially when it becomes very severe. Scientists are trying to understand how the virus works so they can find new ways to fight it. One part of the virus, called the E protein, has been shown to be important for its growth and survival inside a host cell.

Researchers have discovered that this E protein can interact with other proteins in the cell that help regulate calcium levels. This interaction can affect how well these proteins work, which can lead to inflammation and make COVID-19 more severe.

The study found that the E protein can bind to these regulatory proteins and change their shape, making them less effective at doing their job. This could be a key part of why the virus causes such strong immune responses in people.

By understanding how this works, scientists may be able to find new ways to treat COVID-19 by targeting these interactions between the E protein and other cell proteins.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The clinical management of severe COVID-19 cases is not yet well resolved. Therefore, it is important to identify and characterize cell signaling pathways involved in virus pathogenesis that can be targeted therapeutically. Envelope (E) protein is a structural protein of the virus, which is known to be highly expressed in the infected host cell and is a key virulence factor; however, its role is poorly characterized. The E protein is a single-pass transmembrane protein that can assemble into a pentamer forming a viroporin, perturbing Ca2+homeostasis. Because it is structurally similar to regulins such as, for example, phospholamban, that regulate the sarco/endoplasmic reticulum calcium ATPases (SERCA), we investigated whether the SARS-CoV-2 E protein affects the SERCA system as an exoregulin. Using FRET experiments we demonstrate that E protein can form oligomers with regulins, and thus can alter the monomer/multimer regulin ratio and consequently influence their interactions with SERCAs. We also confirm that a direct interaction between E protein and SERCA2b results in a decrease in SERCA-mediated ER Ca2+reload. Structural modeling of the complexes indicates an overlapping interaction site for E protein and endogenous regulins. Our results reveal novel links in the host-virus interaction network that play an important role in viral pathogenesis and may provide a new therapeutic target for managing severe inflammatory responses induced by SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71144-5" target="_blank">https://www.nature.com/articles/s41598-024-71144-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41467-024-52348-9" target="_blank">
            ü§ñ COVID-19 Vaccine Booster Dose Effects on Immune System</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-533', this)">
            More</button></h4>
            <div id="toggle-533" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Omicron COVID-19 immune correlates analysis of a third dose of mRNA-1273 in the COVE trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study was done to see how well antibodies in the blood protect against COVID-19 after getting vaccinated. The research showed that people who have these antibodies are less likely to get sick with COVID-19. But since new variants of the virus have emerged, scientists wanted to know if these antibodies still work.

The study found that even though many people already had antibodies from previous vaccinations or infections, the level of protection they offered was still strong. In fact, having higher levels of antibodies meant a lower chance of getting COVID-19.

This research suggests that measuring antibody levels in the blood can still be a good way to see how well someone is protected against COVID-19, even after new variants have emerged.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In the phase 3 Coronavirus Efficacy (COVE) trial (NCT04470427), post-dose two Ancestral Spike-specific binding (bAb) and neutralizing (nAb) antibodies were shown to be correlates of risk (CoR) and of protection against Ancestral-lineage COVID-19 in SARS-CoV-2 naive participants. In the SARS-CoV-2 Omicron era, Omicron subvariants with varying degrees of immune escape now dominate, seropositivity rates are high, and booster doses are administered, raising questions on whether and how these developments affect the bAb and nAb correlates. To address these questions, we assess post-boost BA.1 Spike-specific bAbs and nAbs as CoRs and as correlates of booster efficacy in COVE. For naive individuals, bAbs and nAbs inversely correlate with Omicron COVID-19: hazard ratios (HR) per 10-fold marker increase (95% confidence interval) are 0.16 (0.03, 0.79) and 0.31 (0.10, 0.96), respectively. In non-naive individuals the analogous results are similar: 0.15 (0.04, 0.63) and 0.28 (0.07, 1.08). For naive individuals, three vs two-dose booster efficacy correlates with predicted nAb titer at exposure, with estimates -8% (-126%, 48%), 50% (25%, 67%), and 74% (49%, 87%), at 56, 251, and 891 Arbitrary Units/ml. These results support the continued use of antibody as a surrogate endpoint.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52348-9" target="_blank">https://www.nature.com/articles/s41467-024-52348-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41598-024-71419-x" target="_blank">
            ü§ñ Understanding Immune System Changes After Recovering from COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-534', this)">
            More</button></h4>
            <div id="toggle-534" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunologic mediators profile in COVID-19 convalescence</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new coronavirus called SARS-CoV-2 caused a global pandemic in 2020. To understand how our bodies fight this virus, scientists studied people who had recovered from COVID-19. They took blood samples from these individuals and measured different markers in their blood to see what happened as they got better.

The study found that most people developed antibodies (special proteins) against the virus after getting sick. Some of these antibodies lasted longer than others. The researchers also saw changes in the types of immune cells (like soldiers) in the blood over time.

What's interesting is that our immune system starts with a "fighting" response to the virus, but as we get better, it shifts towards a "repair" response. This study highlights the importance of understanding how our immune system works, especially after we've gotten sick with something like COVID-19. Knowing this can help us develop better treatments and vaccines in the future.

The bottom line is that our bodies are pretty amazing at fighting off viruses, but there's still a lot to learn about how they work!</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 caused the pandemic situation experienced since the beginning of 2020, and many countries faced the rapid spread and severe form of the disease. Mechanisms of interaction between the virus and the host were observed during acute phase, but few data are available when related to immunity dynamics in convalescents. We conducted a longitudinal study, with 51 healthy donors and 62 COVID-19 convalescent patients, which these had a 2-month follow-up after symptoms recovery. Venous blood sample was obtained from all participants to measure blood count, subpopulations of monocytes, lymphocytes, natural killer cells and dendritic cells. Serum was used to measure cytokines, chemokines, growth factors, anti-N IgG and anti-S IgG/IgM antibodies. Statistic was performed by Kruskal‚ÄìWallis test, and linear regression with days post symptoms and antibody titers. All analysis had confidence interval of 95%. Less than 35% of convalescents were anti-S IgM+, while more than 80% were IgG+ in D30. Anti-N IgG decreased along time, with loss of seroreactivity of 13%. Eosinophil count played a distinct role on both antibodies during all study, and the convalescence was orchestrated by higher neutrophil-to-lymphocyte ratio and IL-15, but initial stages were marked by increase in myeloid DCs, B1 lymphocytes, inflammatory and patrolling monocytes, G-CSF and IL-2. Later convalescence seemed to change to cytotoxicity mediated by T lymphocytes, plasmacytoid DCs, VEGF, IL-9 and CXCL10. Anti-S IgG antibodies showed the longest perseverance and may be a better option for diagnosis. The inflammatory pattern is yet present on initial stage of convalescence, but quickly shifts to a reparative dynamic. Meanwhile eosinophils seem to play a role on anti-N levels in convalescence, although may not be the major causative agent. We must highlight the importance of immunological markers on acute clinical outcomes, but their comprehension to potentialize adaptive system must be explored to improve immunizations and further preventive policies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71419-x" target="_blank">https://www.nature.com/articles/s41598-024-71419-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41541-024-00950-9" target="_blank">
            ü§ñ Designing Specialized Antigens to Boost Immunity Against Most COVID-19 Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-535', this)">
            More</button></h4>
            <div id="toggle-535" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Computationally designed Spike antigens induce neutralising responses against the breadth of SARS-COV-2 variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists have created new vaccine ingredients called antigens to help fight against COVID-19 viruses that keep changing over time. They designed three new antigens (called T2_32, T2_35, and T2_36) that work well together to protect people from different types of COVID-19 variants.

Tests on animals showed that these new antigens are very effective at creating a strong defense against the virus. When given as a vaccine, they even worked better than some existing vaccines.

These findings suggest that using computer-designed modifications to create more powerful vaccines could be an important tool in fighting against COVID-19 and its evolving variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Updates of SARS-CoV-2 vaccines are required to generate immunity in the population against constantly evolving SARS-CoV-2 variants of concerns (VOCs). Here we describe three novel in-silico designed spike-based antigens capable of inducing neutralising antibodies across a spectrum of SARS-CoV-2 VOCs. Three sets of antigens utilising pre-Delta (T2_32), and post-Gamma sequence data (T2_35 and T2_36) were designed. T2_32 elicited superior neutralising responses against VOCs compared to the Wuhan-1 spike antigen in DNA prime-boost immunisation regime in guinea pigs. Heterologous boosting with the attenuated poxvirus - Modified vaccinia Ankara expressing T2_32 induced broader neutralising immune responses in all primed animals. T2_32, T2_35 and T2_36 elicited broader neutralising capacity compared to the Omicron BA.1 spike antigen administered by mRNA immunisation in mice. These findings demonstrate the utility of structure-informed computationally derived modifications of spike-based antigens for inducing broad immune responses covering more than 2 years of evolved SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00950-9" target="_blank">https://www.nature.com/articles/s41541-024-00950-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41598-024-71475-3" target="_blank">
            ü§ñ Impact of Smell and Taste Issues on Daily Life After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-536', this)">
            More</button></h4>
            <div id="toggle-536" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Chemosensory assessment and impact on quality of life in neurosensorial cluster of the post COVID 19 syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic showed that some people experience problems with their sense of smell and taste after getting infected. This study looked at how common this problem was, what might make it more likely, and how well people recovered.

About 47% of patients in this study had these problems, which were more common in women and less common in older adults who got a special treatment called monoclonal antibodies. When researchers tested some patients' sense of smell and taste, they found that about two-thirds could smell things only slightly or not at all, while one-fourth couldn't taste things normally.

This study showed that problems with smell and taste after COVID-19 can be quite complex and unpredictable. Understanding this better will help doctors develop treatments to prevent or fix these problems in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 pandemic brought chemosensory impairment to the forefront of medicine, revealing gaps in the knowledge of pathophysiological mechanisms, true prevalence and preventive/therapeutic alternatives. This is a sub-study of the ORCHESTRA cohort focusing on post-COVID-19 chemosensory symptoms. Risk factors for neurosensorial cluster of post-COVID-19 syndrome (NSc-PCS) were assessed through multivariable analysis. Psychophysical validated tests were applied on a sub-population of 50 patients. Qualitative chemosensory symptoms as well as nasal and oral chemesthesis were evaluated through anamnestic interview and the quality of life through the SF-36 questionnaire. Chemosensory symptoms evolution and olfactory training‚Äôs outcome were assessed through phone-call interviews. Out of 1187 patients (female, N‚Äâ=‚Äâ630), 550 (47%) presented NSc-PCS, with a lower risk for older age and monoclonal antibodies treatment, and a higher risk in females (p‚Äâ<‚Äâ0.001). Out of the 50 patients evaluated with psychophysical tests, 66% showed smell reduction with a qualitative alteration in 50% of hyposmic and 35% of normosmic patients. Hypogeusia was present in 14 (28%) of the patients assessed, with 56% showing a qualitative alteration; 53% of normogeusic patients presented qualitative disorders. NSc-PCS has a complex, fluctuating, multifaceted presentation. Quantifying and characterizing COVID-19-related chemosensory impairment is key to understand underlying mechanisms and to develop preventive and therapeutic treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71475-3" target="_blank">https://www.nature.com/articles/s41598-024-71475-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-08</strong> <a href="https://www.nature.com/articles/s41598-024-70980-9" target="_blank">
            ü§ñ New Method for Quickly Detecting COVID-19 Virus RNA</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-537', this)">
            More</button></h4>
            <div id="toggle-537" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Use of high-resolution fluorescence in situ hybridization for fast and robust detection of SARS-CoV-2 RNAs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Detecting COVID-19 quickly and accurately is crucial to stopping its spread. We developed a fast test that can detect the virus in just 3 hours, using a special technique called FISH (fluorescence in situ hybridization). This test works by using tiny probes to find and highlight the presence of the virus's genetic material (RNA) in cells.

We tested this method on cell cultures and found it to be accurate and efficient. We also used it to detect COVID-19 in samples from patients, including those taken from the mouth and throat. The results were excellent, with the test being able to identify the presence of the virus quickly and accurately.

This new method could significantly improve the accuracy and speed of COVID-19 detection, making it easier to control outbreaks and keep people safe.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Early, rapid, and accurate diagnostic tests play critical roles not only in the identification/management of individuals infected by SARS-CoV-2, but also in fast and effective public health surveillance, containment, and response. Our aim has been to develop a fast and robust fluorescence in situ hybridization (FISH) detection method for detecting SARS-CoV-2 RNAs by using an HEK 293¬†T cell culture model. At various times after being transfected with SARS-CoV-2 E and N plasmids, HEK 293¬†T cells were fixed and then hybridized with ATTO-labeled short DNA probes (about 20 nt). At 4¬†h, 12¬†h, and 24¬†h after transfection, SARS-CoV-2 E and N mRNAs were clearly revealed as solid granular staining inside HEK 293¬†T cells at all time points. Hybridization time was also reduced to 1¬†h for faster detection, and the test was completed within 3¬†h with excellent results. In addition, we have successfully detected 3 mRNAs (E mRNA, N mRNA, and ORF1a (‚àí) RNA) simultaneously inside the buccal cells of COVID-19 patients. Our high-resolution RNA FISH might significantly increase the accuracy and efficiency of SARS-CoV-2 detection, while significantly reducing test time. The method can be conducted on smears containing cells (e.g., from nasopharyngeal, oropharyngeal, or buccal swabs) or smears without cells (e.g., from sputum, saliva, or drinking water/wastewater) for detecting various types of RNA viruses and even DNA viruses at different timepoints of infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70980-9" target="_blank">https://www.nature.com/articles/s41598-024-70980-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-06</strong> <a href="https://www.nature.com/articles/s43856-024-00602-y" target="_blank">
            ü§ñ COVID Vaccine Works Well in Greenland Population</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-538', this)">
            More</button></h4>
            <div id="toggle-538" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Robust immune response to COVID-19 vaccination in the island population of Greenland</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

In Greenland, the COVID-19 pandemic was relatively mild compared to other countries. One reason for this might be that many people in Greenland got vaccinated against COVID-19.

To understand how well the vaccines worked, researchers looked at the immune system response of 430 adults who had received two doses of the vaccine. They measured the levels of antibodies (which help fight off viruses) in these people's blood and saliva over a period of up to 11 months after their second dose.

The results showed that almost all participants (96% after two months, and 98% after 11 months) had enough antibodies to protect them from COVID-19. People who had previously been infected with COVID-19 even had higher levels of antibodies. In people over 60 years old, the vaccine response was slightly weaker.

The researchers also found that all participants had a strong T-cell response (another part of the immune system) against COVID-19. These results suggest that the vaccines were very effective in protecting Greenlanders against COVID-19, and contributed to the mild impact of the pandemic in the country.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundIn Greenland, the COVID-19 pandemic was characterised by a late onset of community transmission and a low impact on the healthcare system, hypothesised as being partly due to a high uptake of vaccinations. To underpin this description, we aimed to assess the SARS-CoV-2 immune response post-vaccination in a Greenlandic population.MethodsIn this observational cohort study, we included 430 adults in Greenland who had received a complete two-dose SARS-CoV-2 vaccination at enrolment. The total plasma SARS-CoV-2 spike glycoprotein Ig antibodies (S-Ab) induced by either the BNT162b2 or mRNA-1273 vaccine, was measured up to 11 months after the second vaccine dose. In addition, total salivary S-Abs were examined in 107 participants, and the T-cell response to the spike glycoprotein was assessed in 78 participants out of the entire study cohort.ResultsHere we demonstrate that two months after the second vaccine dose, 96% of participants have protective plasma S-Ab levels. By 11 months, 98% have protective levels, with prior SARS-CoV-2 infection particularly enhancing S-Ab levels by 37% (95% CI 25‚Äì51%). Among individuals aged 60 years and older, we observe a 21% (95% CI 7‚Äì33%) reduction in antibody response. Total salivary S-Ab levels are detectable in all participants and significantly correlate with plasma levels. Moreover, all participants exhibit a robust SARS-CoV-2-specific T-cell response 11 months post-primary vaccination.ConclusionsOur findings show that Greenlanders exhibit a robust and lasting immune response, both humoral and cellular, comparable to other population groups up to at least 11 months after the second vaccine dose. These results corroborate the hypothesis that vaccines contributed to the mild impact of the COVID-19 pandemic in the Greenlandic population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00602-y" target="_blank">https://www.nature.com/articles/s43856-024-00602-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41467-024-52038-6" target="_blank">
            ü§ñ Effect of mRNA COVID Vaccine Timing on Heart Inflammation Risk</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-539', this)">
            More</button></h4>
            <div id="toggle-539" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Influence of mRNA Covid-19 vaccine dosing interval on the risk of myocarditis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A serious heart condition called myocarditis is more likely to happen after getting certain COVID-19 vaccines. The risk is highest after the second dose and lower with later booster doses. We studied over 7,000 cases of myocarditis in people aged 12 and up to see if there's a connection between how often you get vaccinated and the risk of this condition. Our results show that taking longer breaks between vaccine doses can decrease the risk by up to four times, especially for people under 50. This suggests that waiting at least six months before getting another booster shot might be a good idea.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Myocarditis is the most salient serious adverse event following messenger RNA-based Covid-19 vaccines. The highest risk is observed after the second dose compared to the first, whereas the level of risk associated with more distant booster doses seems to lie in between. We aimed to assess the relation between dosing interval and the risk of myocarditis, for both the two-dose primary series and the third dose (first booster). This matched case-control study included 7911 cases of myocarditis aged 12 or more in a period where approximately 130 million vaccine doses were administered. Here we show that longer intervals between each consecutive dose, including booster, may decrease the occurrence of vaccine-associated myocarditis by up to a factor of 4, especially under age 50. These results suggest that a minimum 6-month interval might be required when scheduling additional booster vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52038-6" target="_blank">https://www.nature.com/articles/s41467-024-52038-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41598-024-71305-6" target="_blank">
            ü§ñ Gauss Luciferase Assay to Measure Coronavirus Enzyme Activity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-540', this)">
            More</button></h4>
            <div id="toggle-540" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A Gaussia luciferase reporter assay for the evaluation of coronavirus Nsp5/3CLpro activity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Human coronaviruses cause sickness in many people each year. Some types like MERS, SARS-CoV-1, and SARS-CoV-2 can be very serious.

To help stop these viruses from spreading, scientists are looking for new medicines that can fight them. A drug called Paxlovid has already been used successfully to treat some of these illnesses by stopping a protein inside the virus called 3CLpro.

But sometimes, if we use the same drug too much or for too long, the virus might start to get stronger and not work as well anymore. So researchers want to find more ways to fight these viruses.

They created an easy-to-use test that can quickly tell them how strong a virus is and if medicines are working against it. This test uses parts of another protein found in our bodies (ACE2-Gal4) along with the 3CLpro from coronaviruses. When 3CLpro cuts this protein, it causes a light-producing chemical to be made.

This new test can work on any coronavirus and can measure both how well medicine is stopping the virus and if the virus has become more powerful in some way. They tested the medicine Paxlovid using their test and found that it works against SARS-CoV-2 3CLpro.

The test also helped them understand better how mutations (changes) in the 3CLpro protein can make it stronger or weaker, which is important for future research.

Overall, this new test makes it easier to study coronavirus proteins and find new medicines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Human coronaviruses (hCoVs) infect millions of people every year. Among these, MERS, SARS-CoV-1, and SARS-CoV-2 caused significant morbidity and mortality and their emergence highlights the risk of possible future coronavirus outbreaks. Therefore, broadly-active anti-coronavirus drugs are needed. Pharmacological inhibition of the hCoV protease Nsp5 (3CLpro) is clinically beneficial as shown by the wide and effective use of Paxlovid (nirmatrelvir, ritonavir). However, further treatment options are required due to the risk of drug resistance. To facilitate the assessment of coronavirus protease function and its pharmacological inhibition, we developed an assay allowing rapid and reliable quantification of Nsp5 activity under biosafety level 1 conditions. It is based on an ACE2-Gal4 transcription factor fusion protein separated by a Nsp5 recognition site. Cleavage by Nsp5 releases the Gal4 transcription factor, which then induces the expression of Gaussia luciferase. Our assay is compatible with Nsp5 proteases from all hCoVs and allows simultaneous measurement of inhibitory and cytotoxic effects of the tested compounds. Proof-of-concept measurements confirmed that nirmatrelvir, GC376 and lopinavir inhibit SARS-CoV-2 Nsp5 function. Furthermore, the assay accurately predicted the impact of Nsp5 mutations on catalytic activity and inhibitor sensitivity. Overall, the reporter assay is suitable for evaluating viral protease activity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71305-6" target="_blank">https://www.nature.com/articles/s41598-024-71305-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41598-024-70767-y" target="_blank">
            ü§ñ How COVID-19 Affects Eye Diseases in South Korea</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-541', this)">
            More</button></h4>
            <div id="toggle-541" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of COVID-19 pandemic on ocular disease: KNHANES 2015‚Äì2021</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

This study looked at how the COVID-19 pandemic affected eye diseases and changed people's risk factors in South Korea between 2015 and 2021. The researchers analyzed data from over 50,000 people and found that being infected with COVID-19 increased the risk of developing cataracts and age-related macular degeneration (a type of vision loss). They also found that other health conditions like high blood pressure, diabetes, and depression were linked to eye diseases too. However, having COVID-19 did not increase the risk of getting glaucoma (another type of eye disease). The study suggests that cataracts and age-related macular degeneration might be more affected by people's lifestyle and environment rather than just their genetic predisposition.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aim of this study was to evaluate the impact of COVID-19 on ocular diseases and changes in risk factors before and after the COVID-19 pandemic. This study was conducted using data from the Korea National Health and Nutrition Examination Survey (KNHANES) 2015‚Äì2021, a national cross-sectional health examination and survey. Associations between ocular diseases and risk factors were determined using the chi-squared test and logistic regression analysis. Bivariable adjusted logistic regression analysis was performed to examine the odds ratio (OR) and 95% confidence interval (CI) to evaluate of the impact of COVID-19 on ocular diseases. Individuals were divided into two age groups (<‚Äâ60 and‚Äâ‚â•‚Äâ60¬†years). A total of 50,158 people were diagnosed, of whom 7270 were diagnosed with cataract, 921 with glaucoma, and 439 with age-related macular degeneration (AMD). Risk factors for cataract were COVID-19 pandemic (OR 1.161), hypertension (OR 1.608), diabetes (OR 1.573), dyslipidemia (OR 1.167), stroke (OR 1.272), and depression (OR 1.567). Risk factors for AMD were COVID-19 pandemic (OR 1.600), dyslipidemia (OR 1.610), and depression (OR 1.466). Risk factors for glaucoma were hypertension (OR 1.234), dyslipidemia (OR 1.529), diabetes (OR 1.323), and depression (OR 1.830). The COVID-19 pandemic was a risk factor for cataracts and AMD, but not for glaucoma. Cataracts and AMD may be more influenced by the acquired health conditions or the environment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70767-y" target="_blank">https://www.nature.com/articles/s41598-024-70767-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-04</strong> <a href="https://www.nature.com/articles/s41467-024-51973-8" target="_blank">
            ü§ñ Understanding How SARS-CoV-2 Mutates and Evades Antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-542', this)">
            More</button></h4>
            <div id="toggle-542" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structural basis for the evolution and antibody evasion of SARS-CoV-2 BA.2.86 and JN.1 subvariants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic is still a problem and some new versions of the virus (called BA.2.86 and JN.1) are spreading fast around the world. These viruses have many mutations that make it harder for our immune systems to fight them off. Scientists studied these new viruses and found out how they evade our antibodies, which are like superheroes that protect us from germs.

They also looked at the structure of the virus and how it interacts with human cells. They discovered some important changes in the way these new viruses attach to human cells, making it harder for them to be stopped by our immune systems.

This research helps us understand why these new viruses are spreading quickly and what we can do to stop them. It's like having a map of where the virus is evolving and how it's adapting to avoid being caught by our bodies' defenses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Omicron subvariants of SARS-CoV-2, especially for BA.2.86 and JN.1, have rapidly spread across multiple countries, posing a significant threat in the ongoing COVID-19 pandemic. Distinguished by 34 additional mutations on the Spike (S) protein compared to its BA.2 predecessor, the implications of BA.2.86 and its evolved descendant, JN.1 with additional L455S mutation in receptor-binding domains (RBDs), are of paramount concern. In this work, we systematically examine the neutralization susceptibilities of SARS-CoV-2 Omicron subvariants and reveal the enhanced antibody evasion of BA.2.86 and JN.1. We also determine the cryo-EM structures of the trimeric S proteins from BA.2.86 and JN.1 in complex with the host receptor ACE2, respectively. The mutations within the RBDs of BA.2.86 and JN.1 induce a remodeling of the interaction network between the RBD and ACE2. The L455S mutation of JN.1 further induces a notable shift of the RBD‚ÄìACE2 interface, suggesting the notably reduced binding affinity of JN.1 than BA.2.86. An analysis of the broadly neutralizing antibodies possessing core neutralizing epitopes reveals the antibody evasion mechanism underlying the evolution of Omicron BA.2.86 subvariant. In general, we construct a landscape of evolution in virus-receptor of the circulating Omicron subvariants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51973-8" target="_blank">https://www.nature.com/articles/s41467-024-51973-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-04</strong> <a href="https://www.nature.com/articles/s41541-024-00957-2" target="_blank">
            ü§ñ Effective Protection Against COVID-19 Variants with Dual Vaccine Shot</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-543', this)">
            More</button></h4>
            <div id="toggle-543" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broad protection and respiratory immunity of dual mRNA vaccination against SARS-CoV-2 variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists have been working on vaccines to protect us from COVID-19. However, the virus has changed over time, making some vaccines less effective. Researchers developed a new type of vaccine that targets a specific part of the virus and tested it alone or combined with another vaccine in hamsters. The results showed that the new vaccine, when used alone or together with the other vaccine, was very good at protecting against certain types of COVID-19. Additionally, studies in mice found that this combination vaccine triggered an immune response specifically in the lungs, which is where COVID-19 typically infects us. These findings suggest that this new vaccine could be effective against various strains of COVID-19 and help prevent future outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While first-generation, spike (S)-based COVID-19 vaccines were effective against early SARS-CoV-2 strains, the rapid evolution of novel Omicron subvariants have substantially reduced vaccine efficacy. As such, broadly protective vaccines against SARS-CoV-2 are needed to prevent future viral emergence. In addition, it remains less clear whether peripheral immunization, especially with mRNA vaccines, elicits effective respiratory immunity. Our group has developed a nucleoside-modified mRNA vaccine expressing the nucleocapsid (N) protein of the ancestral SARS-CoV-2 virus and has tested its use in combination with the S-based mRNA vaccine (mRNA-S). In this study, we examined efficacy of mRNA-N alone or in combination with mRNA-S (mRNA-S+N) against more immune evasive Omicron variants in hamsters. Our data show that mRNA-N alone induces a modest but significant protection against BA.5 and that dual mRNA-S+N vaccination confers complete protection against both BA.5 and BQ.1, preventing detection of virus in the hamster lungs. Analysis of respiratory immune response in mice shows that intramuscular mRNA-S+N immunization effectively induces respiratory S- and N-specific T cell responses in the lungs and in bronchoalveolar lavage (BAL), as well as antigen-specific binding IgG in BAL. Together, our data further support mRNA-S+N as a potential pan-COVID-19 vaccine for broad protection against current and emerging SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00957-2" target="_blank">https://www.nature.com/articles/s41541-024-00957-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-03</strong> <a href="https://www.nature.com/articles/s41590-024-01951-5" target="_blank">
            ü§ñ Adenovirus Vaccine Boosts Immunity, Prevents New Coronavirus Strain in Monkeys</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-544', this)">
            More</button></h4>
            <div id="toggle-544" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mucosal adenovirus vaccine boosting elicits IgA and durably prevents XBB.1.16 infection in nonhuman primates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at different ways to give a vaccine for a virus that causes the flu in animals (SARS-CoV-2). They compared two methods: putting the vaccine directly into someone's arm, or giving it through their nose or mouth.

The team found that when they gave the vaccine through the nose and mouth, it worked better. The vaccine helped protect against the virus not just in the throat but also deep inside the lungs. This method also made the immune system create antibodies (special proteins) that can help stop the virus from spreading.

In contrast, giving the vaccine by putting it into someone's arm only protected the lower parts of their airways.

The study suggests that giving a vaccine through the nose or mouth might be better because:
1. It stops the virus in both the throat and lungs.
2. It makes antibodies that can stop other people from catching the virus too.
3. It makes certain immune cells that help fight off the virus.

So, this research shows how giving a vaccine through the nose or mouth could better protect against SARS-CoV-2 and prevent it from spreading to others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A mucosal route of vaccination could prevent severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication at the site of infection and limit transmission. We compared protection against heterologous XBB.1.16 challenge in nonhuman primates (NHPs) ~5 months following intramuscular boosting with bivalent mRNA encoding WA1 and BA.5 spike proteins or mucosal boosting with a WA1‚ÄìBA.5 bivalent chimpanzee adenoviral-vectored vaccine delivered by intranasal or aerosol device. NHPs boosted by either mucosal route had minimal virus replication in the nose and lungs, respectively. By contrast, protection by intramuscular mRNA was limited to the lower airways. The mucosally delivered vaccine elicited durable airway IgG and IgA responses and, unlike the intramuscular mRNA vaccine, induced spike-specific B cells in the lungs. IgG, IgA and T cell responses correlated with protection in the lungs, whereas mucosal IgA alone correlated with upper airway protection. This study highlights differential mucosal and serum correlates of protection and how mucosal vaccines can durably prevent infection against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01951-5" target="_blank">https://www.nature.com/articles/s41590-024-01951-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s44298-024-00054-0" target="_blank">
            ü§ñ Camel Nasal Cells Resist COVID Better Without Key Protein</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-545', this)">
            More</button></h4>
            <div id="toggle-545" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Resistance to SARS-CoV-2 infection in camelid nasal organoids is associated with lack of ACE2 expression</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) can infect different animals. How well an animal gets infected by SARS-CoV-2 depends on a special protein called ACE2 that the virus uses to enter cells.

While SARS-CoV-2 can use many versions of the ACE2 protein found in various camel species, it doesn't make these camels sick when they are exposed to the virus. In this study, researchers used tiny living copies of parts of animals' noses (called organoids) to see if camels could be infected by SARS-CoV-2 or another related virus called MERS-CoV.

The scientists found that while human nose copies can get infected with both viruses, camel nose copies don't. They also noticed something else: The camel nose copies had a different protein called DPP4 instead of ACE2. This led to the conclusion that because camels don't have enough of the ACE2 protein in their noses and upper respiratory tracts, they are resistant to SARS-CoV-2.

So basically, it's like the virus can use other proteins found in camels' noses, but these aren't as good at letting the virus inside cells. Because of this, camels don't get sick from the virus even if they're exposed to it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects a variety of animal species. Susceptibility to SARS-CoV-2 is primarily determined by the utilization of the viral receptor, ACE2. SARS-CoV-2 can utilize a broad range of animal ACE2 isoforms in vitro, including the ACE2 from various camelid species. However, experimental infection of these animals does not lead to productive infection or seroconversion. In this study, we investigate the susceptibility of camelids to SARS-CoV-2 using novel well-differentiated camelid nasal organoids. We show that camelid nasal organoids are highly susceptible to Middle East respiratory syndrome coronavirus (MERS-CoV) infection, but not to infection with different SARS-CoV-2 variants (614G, BA.1 or EG.5.1.1). All viruses efficiently infected human airway organoids. Immunohistochemistry analysis revealed the absence of ACE2 on camelid nasal organoids and dromedary camel upper respiratory tract. In contrast, DPP4 was expressed in both camelid nasal organoids and the camel upper respiratory tract, which correlates with MERS-CoV infection. This study indicates that the camelid upper respiratory tract lacks expression of ACE2, which is associated with resistance to SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00054-0" target="_blank">https://www.nature.com/articles/s44298-024-00054-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-71455-7" target="_blank">
            ü§ñ Looking into why some COVID-19 patients get a severe lung infection called invasive pulmonary aspergillosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-546', this)">
            More</button></h4>
            <div id="toggle-546" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Investigation of risk factors for invasive pulmonary aspergillosis among patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied 114 people who had COVID-19 and found that some developed a secondary infection called pulmonary aspergillosis (CAPA). This study looked for factors that might increase the risk of developing CAPA.

The results showed that certain conditions, such as being on kidney transplant medication, having high levels of inflammation in the body, or needing to be on a ventilator for a long time, made it more likely for someone with COVID-19 to develop CAPA.

The researchers found that three main factors were strongly linked to an increased risk of CAPA: high levels of inflammation in the body, low levels of immune cells called T-cells, and needing to be on a ventilator for a long time. Identifying these risk factors can help doctors start treatment earlier, which may improve outcomes for people with CAPA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 associated pulmonary aspergillosis (CAPA) had been reported, and raised concern about this secondary infection due to the high mortality. This study aimed to investigate the risk factors for CAPA. The enrolled 114 COVID-19 patients were further divided into CAPA group and non-CAPA group. Demographic characteristics, underlying diseases, laboratory parameters and therapeutic schedule between the two groups were compared to identify the independent risk factors for CAPA by univariate analysis and multivariable logistic regression analysis. Sensitivity and specificity of independent risk factors were confirmed by receiver operating characteristic (ROC) curve analysis. Univariate analysis showed that renal transplant, IL-6 and CRP levels, decreased CD4‚Äâ+‚ÄâT cell and CD8‚Äâ+‚ÄâT cell, duration of antibiotics therapy, and prolonged mechanical ventilation were risk factors for development of CAPA. These factors were further analyzed by multivariable logistic regression analysis and the results indicated that elevated IL-6 level, decreased CD4‚Äâ+‚ÄâT cell and prolonged mechanical ventilation could be recognized as independent risk factors for CAPA in COVID-19 patients. Identification of these risk factors is essential to initiate antifungal therapy as soon as possible to improve outcome of patients with CAPA.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71455-7" target="_blank">https://www.nature.com/articles/s41598-024-71455-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-71325-2" target="_blank">
            ü§ñ Acute Proteins for COVID Severity & Mortality</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-547', this)">
            More</button></h4>
            <div id="toggle-547" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Acute-phase proteins as indicators of disease severity and mortality in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how certain chemicals called acute-phase proteins (APPs) affect how sick people get from COVID-19. They measured these chemicals in 96 people with COVID-19, along with some healthy people for comparison.

They found that:
- The levels of APPs change depending on how sick someone is from the disease and if they're getting enough oxygen.
- In very severe cases, where there's a lot of inflammation (called a cytokine storm), these chemicals can go up or down a lot.
- There are some APPs that don't usually get tested. The researchers found that two of them - procalcitonin and transferrin - help doctors tell if someone is going to survive their COVID-19.

They also noticed that measuring how sick people were using something called the MEWS scale didn't always match with another system used by doctors (National Institutes of Health). 

The researchers concluded that:
- Not all APPs change based on how bad a person's disease gets.
- Only two out of many tested APPs can help predict if someone will survive their COVID-19.
- Among the new ones they looked at, transferrin is the only one that seems to really matter in predicting who will get better or worse from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aim of the study was to conduct of relationship of acute-phase proteins (APPs) with the severity of COVID-19 defined by National Institutes of Health and according to the criteria of MEWS scale, with the presence of a cytokine storm, oxygen therapy and patient survival. We enrolled 96 patients with COVID-19 and 30 healthy people. The samples were taken on the day of admission and after 9¬†days on average. Not only commonly used APPs such as CRP, procalcitonin and ferritin and also rarely assayed proteins such as transferrin, haptoglobin, Œ±1-acid glycoprotein and Œ±1-antitrypsin, were tested in the study. The levels of APPs depends on the severity of COVID-19 disease, on the presence of cytokine storm and used oxygen therapy. The greatest APPs changes occurred in the most advanced form of the disease, with the presence of a cytokine storm and the most intense oxygen therapy. The results obtained from MEWS scale were not consistent with National Institutes of Health scores. Studies in the second samples showed the quenching of the acute phase reactions and the effectiveness of oxygen therapy. Only two of the examined APPs i.e. procalcitonin and transferrin, differed between surviving and non-surviving patients, and these two predispose to the role of prognostic factors in Covid-19. In conclusion, the concentration of not all acute-phase proteins depends on the severity of COVID-19 disease, presence of cytokine storm, the used of oxygen therapy and only some of them (procalcitonin and transferrin) are related to the survival outcomes. Of the newly tested acute-phase proteins, only transferrin shows significance as a marker of disease severity and mortality in COVID-19 disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71325-2" target="_blank">https://www.nature.com/articles/s41598-024-71325-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-70725-8" target="_blank">
            ü§ñ T-cells in pregnant/postpartum women, with/without HIV, recognize old/new COVID strains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-548', this)">
            More</button></h4>
            <div id="toggle-548" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>T-cell responses to ancestral SARS-CoV-2 and Omicron variant among unvaccinated pregnant and postpartum women living with and without HIV in South Africa</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied how people who are pregnant or have recently given birth and living with HIV react to COVID-19 in their bodies. They looked at 24 women without HIV and 15 women with HIV from low-income countries. The researchers found that these women's immune systems can fight off COVID-19 just as well as women who don't have HIV, even if they've never been vaccinated against the virus. They also discovered that the immune system's response to a specific part of the virus called Omicron is similar to its response to earlier versions of the virus. This means that people with COVID-19 can still get protection from their immune system, even if they haven't received a vaccine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 cell-mediated immunity remains understudied during pregnancy in unvaccinated Black African women living with HIV (WLWH) from low- and middle-income countries. We investigated SARS-CoV-2-specific T-cell responses 1 month following infection in 24 HIV-uninfected women and 15 WLWH at any stage during pregnancy or postpartum. The full-length spike (FLS) glycoprotein and nucleocapsid (N) protein of wild-type (WT) SARS-CoV-2, as well as mutated spike protein regions found in the Omicron variant (B.1.1.529) were targeted by flow cytometry. WT-specific CD4+and CD8+T cells elicited similar FLS- and N-specific responses in HIV-uninfected women and WLWH. SARS-CoV-2-specific T-lymphocytes were predominantly TNF-Œ± monofunctional in pregnant and postpartum women living with and without HIV, with fever cells producing either IFN-Œ≥ or IL-2. Furthermore, T-cell responses were unaffected by Omicron-specific spike mutations as similar responses between Omicron and the ancestral virus were detected for CD4+and CD8+T cells. Our results collectively demonstrate comparable T-cell responses between WLWH on antiretroviral therapy and HIV-uninfected pregnant and postpartum women who were na√Øve to Covid-19 vaccination. Additionally, we show that T cells from women infected with the ancestral virus, Beta variant (B.1.351), or Delta variant (B.1.617.2) can cross-recognize Omicron, suggesting an overall preservation of T-cell immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70725-8" target="_blank">https://www.nature.com/articles/s41598-024-70725-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-31</strong> <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">
            ü§ñ Spike Protein Targeting Antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-549', this)">
            More</button></h4>
            <div id="toggle-549" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibodies utilizing VL6-57 light chains target a convergent cryptic epitope on SARS-CoV-2 spike protein and potentially drive the genesis of Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists have found that as COVID-19 spreads, it mutates and changes into different versions. This means that the antibodies created by our immune system when we get infected or vaccinated may not be able to recognize the virus if it changes too much. Researchers wanted to understand why this happens.

They discovered a group of antibodies that can fight COVID-19, and found out how they work together with other parts of the immune system to target the virus. They also found out that these antibodies are already present in people who have never had COVID-19, but are more abundant in people who have recovered from the disease.

However, when scientists tested these antibodies against a new variant of the virus called Omicron, they found out that it can evade their attack by making small changes to its structure. This suggests that the pressure from our immune system may be driving the evolution of the virus into more aggressive forms.

These findings provide further evidence that the spread of COVID-19 and its variants is influenced by our immune response, and highlight the importance of continued research into understanding how our bodies fight the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Continued evolution of SARS-CoV-2 generates variants to challenge antibody immunity established by infection and vaccination. A connection between population immunity and genesis of virus variants has long been suggested but its molecular basis remains poorly understood. Here, we identify a class of SARS-CoV-2 neutralizing public antibodies defined by their shared usage of VL6-57 light chains. Although heavy chains of diverse genotypes are utilized, convergent HCDR3 rearrangements have been observed among these public antibodies to cooperate with germline VL6-57 LCDRs to target a convergent epitope defined by RBD residues S371-S373-S375. Antibody repertoire analysis identifies that this class of VL6-57 antibodies is present in SARS-CoV-2-naive individuals and is clonally expanded in most COVID-19 patients. We confirm that Omicron-specific substitutions at S371, S373 and S375 mediate escape of antibodies of the VL6-57 class. These findings support that this class of public antibodies constitutes a potential immune pressure promoting the introduction of S371L/F-S373P-S375F in Omicron variants. The results provide further molecular evidence to support that antigenic evolution of SARS-CoV-2 is driven by antibody mediated population immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">https://www.nature.com/articles/s41467-024-51770-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41467-024-51959-6" target="_blank">
            ü§ñ Understanding COVID-19 Immune Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-550', this)">
            More</button></h4>
            <div id="toggle-550" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mapping the immunopeptidome of seven SARS-CoV-2 antigens across common HLA haplotypes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Most COVID-19 vaccines currently work by teaching our bodies to fight the virus's "Spike" protein. However, new variants of the virus can make this Spike protein change shape and become harder for our immune systems to recognize. To address this issue, researchers are working on developing vaccines that target a broader range of proteins in the SARS-CoV-2 virus.

To do this, scientists used a powerful tool called mass spectrometry to identify tiny pieces of these proteins (called "immunopeptides") that can trigger an immune response. They tested these immunopeptides against different versions of human cells and found 248 unique peptides that were bound to specific parts of the cell's surface.

These findings could help create more effective COVID-19 vaccines by targeting multiple proteins in the SARS-CoV-2 virus, rather than just one protein like the Spike protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Most COVID-19 vaccines elicit immunity against the SARS-CoV-2 Spike protein. However, Spike protein mutations in emerging strains and immune evasion by the SARS-CoV-2 virus demonstrates the need to develop more broadly targeting vaccines. To facilitate this, we use mass spectrometry to identify immunopeptides derived from seven relatively conserved structural and non-structural SARS-CoV-2 proteins (N, E, Nsp1/4/5/8/9). We use two different B-lymphoblastoid cell lines to map Human Leukocyte Antigen (HLA) class I and class II immunopeptidomes covering some of the prevalent HLA types across the global human population. We employ DNA plasmid transfection and direct antigen delivery approaches to sample different antigens and find 248 unique HLA class I and HLA class II bound peptides with 71 derived from N, 12 from E, 28 from Nsp1, 19 from Nsp4, 73 from Nsp8 and 45 peptides derived from Nsp9. Over half of the viral peptides are unpublished. T cell reactivity tested against 56 of the detected peptides shows CD8+and CD4+T cell responses against several peptides from the N, E, and Nsp9 proteins. Results from this study will aid the development of next-generation COVID vaccines targeting epitopes from across a number of SARS-CoV-2 proteins.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51959-6" target="_blank">https://www.nature.com/articles/s41467-024-51959-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41598-024-70697-9" target="_blank">
            ü§ñ Real-Time Monitoring of COVID-19 Genetic Changes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-551', this)">
            More</button></h4>
            <div id="toggle-551" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A program for real-time surveillance of SARS-CoV-2 genetics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic required scientists to quickly identify new variants of the virus. This meant setting up a nationwide system to collect, analyze, and report data on patient samples in a very short time frame. We developed an infrastructure that allowed us to sequence over 594,000 SARS-CoV-2 genomes from patients in the US between March 2020 and July 2023. Our protocol (called "Virseq") ensures accurate and reliable sequencing of the entire virus genome, even for multiple strains co-infecting a patient. This system supports the national surveillance program led by the CDC and serves as a model for responding to future pandemics quickly and effectively at a national scale.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic brought forth an urgent need for widespread genomic surveillance for rapid detection and monitoring of emerging SARS-CoV-2 variants. It necessitated design, development, and deployment of a nationwide infrastructure designed for sequestration, consolidation, and characterization of patient samples that disseminates de-identified information to public authorities in tight turnaround times. Here, we describe our development of such an infrastructure, which sequenced 594,832 high coverage SARS-CoV-2 genomes from isolates we collected in the United States (U.S.) from March 13th 2020 to July 3rd 2023. Our sequencing protocol (‚ÄòVirseq‚Äô) utilizes wet and dry lab procedures to generate mutation-resistant sequencing of the entire SARS-CoV-2 genome, capturing all major lineages. We also characterize 379 clinically relevant SARS-CoV-2 multi-strain co-infections and ensure robust detection of emerging lineages via simulation. The modular infrastructure, sequencing, and analysis capabilities we describe support the U.S. Centers for Disease Control and Prevention national surveillance program and serve as a model for rapid response to emerging pandemics at a national scale.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70697-9" target="_blank">https://www.nature.com/articles/s41598-024-70697-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41598-024-70662-6" target="_blank">
            ü§ñ Factors Contributing to Death in Pakistani Patients Hospitalized with COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-552', this)">
            More</button></h4>
            <div id="toggle-552" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk factors for mortality in hospitalized COVID-19 patients across five waves in Pakistan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied 5368 people who were hospitalized with COVID-19 in Pakistan over two years. They looked at how the virus affected different age groups and the outcomes for patients.

They found that:

* Older adults (aged 60+) made up a larger proportion of patients during certain waves of the pandemic.
* Men were more likely to be infected earlier in the pandemic, but this changed later on.
* The mortality rate (death rate) varied across different waves, with some being higher than others.
* Certain conditions, such as acute respiratory distress syndrome and acute kidney injury, increased the risk of death.
* Some patients who received a type of blood thinner called enoxaparin had lower odds of dying from COVID-19.

The study suggests that older adults and those with certain health complications are at higher risk of dying from COVID-19 during certain waves of the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This retrospective cohort study aims to describe the clinical characteristics and outcomes and assess risk factors for mortality across the epidemic waves in hospitalized COVID-19 patients in a major tertiary-care center in Pakistan. A total of 5368 patients with COVID-19, hospitalized between March 2020 and April 2022 were included. The median age was 58¬†years (IQR: 44‚Äì69), 41% were females, and the overall mortality was 12%. Comparative analysis of COVID-19 waves showed that the proportion of patients aged‚Äâ‚â•‚Äâ60¬†years was highest during the post-wave 4 period (61.4%) and Wave 4 (Delta) (50%) (p<‚Äâ0.001). Male predominance decreased from 65.2% in Wave 2 to 44.2% in Wave 5 (Omicron) (p<‚Äâ0.001). Mortality rate was lowest at 9.4% in wave 5 and highest at 21.6% in the post-wave 4 period (p=‚Äâ0.041). In multivariable analysis for risk factors of mortality, acute respiratory distress syndrome (ARDS) was most strongly associated with mortality (aOR 22.98, 95% CI 15.28‚Äì34.55,p<‚Äâ0.001), followed by need for mechanical ventilation (aOR 6.81, 95% CI 5.13‚Äì9.05,p<‚Äâ0.001). Other significant risk factors included acute kidney injury (aOR 3.05, 95% CI 2.38‚Äì3.91,p<‚Äâ0.001), stroke (aOR 2.40, 95% CI 1.26‚Äì4.60, p‚Äâ=‚Äâ0.008), pulmonary embolism (OR 2.07, 95% CI 1.28‚Äì3.35,p=‚Äâ0.003), and age‚Äâ‚â•‚Äâ60¬†years (aOR 2.45, 95% CI 1.95‚Äì3.09,p<‚Äâ0.001). Enoxaparin use was associated with lower mortality odds (aOR 0.45, 95% CI 0.35‚Äì0.60,p<‚Äâ0.001. Patients hospitalized during Wave 4 (aOR 2.22, 95% CI 1.39‚Äì3.56,p<‚Äâ0.001) and the post-wave 4 period (aOR 2.82, 95% CI 1.37‚Äì5.80,p=‚Äâ0.005) had higher mortality odds compared to other waves. The study identifies higher mortality risk in patients admitted in Delta wave and post-wave, aged ‚â•‚Äâ60¬†years, and with respiratory and renal complications, and lower risk with anticoagulation during COVID-19 waves.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70662-6" target="_blank">https://www.nature.com/articles/s41598-024-70662-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41390-024-03492-x" target="_blank">
            ü§ñ Long-term Effects of COVID-19 on Children in Australia After a Big Outbreak</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-553', this)">
            More</button></h4>
            <div id="toggle-553" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post COVID-19 conditions in an Australian pediatric cohort, 3 months following a Delta outbreak</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Many kids got sick with COVID-19 during an outbreak in Australia, but not much was known about how it affected them later on. We asked parents to fill out an online survey 12 weeks after their child had recovered from COVID-19 and found that only a small number (about 1 in 10) were still experiencing symptoms or problems with daily activities.

Out of those who reported ongoing issues, some kids had recovered by the time they saw a doctor, while others had existing conditions that got worse. A smaller group had something called Post-COVID Condition, which is similar to Long COVID.

Our study found that most kids who got COVID-19 recovered quickly, but for those who didn't, it can be a significant burden. We also discovered some risk factors that make kids more likely to experience ongoing problems after COVID-19, such as being older than 11 and having pre-existing medical conditions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundPediatric long COVID remains incompletely understood with scant Australian data available. We aimed to assess the impacts of the 2021 Delta variant of SARS-CoV-2 outbreak on symptoms and functioning 12 weeks post-acute infection in a cohort of children and adolescents.MethodsThe parents/carers of 11,864 patients with PCR-confirmed SARS-CoV-2 were invited, via email or text message, to complete an online survey assessing symptoms and functional impairment.Findings1731 (17.6%) responded to the survey. 203 (11.7%) reported continued symptoms and/or functional impairment which were flagged for clinical review, all others reported recovery. Of the 169 subsequently clinically reviewed, 63 had already recovered (37.3%) and 17 had exacerbation of pre-existing condition(s) (10.1%); 63 (37.3%) were diagnosed with a Post COVID Condition (PCC). Of these, 21 (12.4%) were considered to have features compatible with the United Kingdom consensus cases definition for Long COVID.InterpretationDuring an outbreak of SARS-CoV-2 an online questionnaire with subsequent clinical review revealed self-reported non-recovery at 12 weeks in a minority of cases, with a spectrum of features. Long COVID comprised only a subset of cases with self-reported non-recovery, and is infrequent in children and adolescents, but still comprises a likely significant burden that warrants attention.ImpactOur study provides the only comprehensive estimate of the frequency and spectrum of post-COVID conditions in children from Australia. The high frequency of self-reported recovery, and low frequency of Long COVID compatible illness adds to the literature from other settings. Risk factors for post-COVID conditions in children are identified and include: age >11 year, and previous medical co-morbidity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41390-024-03492-x" target="_blank">https://www.nature.com/articles/s41390-024-03492-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">
            ü§ñ Early Signs of COVID Long-Term Effects</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-554', this)">
            More</button></h4>
            <div id="toggle-554" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Early biological markers of post-acute sequelae of SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied 136 people who had just tested positive for COVID-19 to understand why some people get long-term symptoms after the initial illness. They collected nasal swabs and blood samples over several weeks and found that people who developed long-term symptoms (called PASC) had higher levels of the virus in their bodies and lower antibodies against it, especially within the first 9 days of getting sick. This suggests that how the body responds to COVID-19 early on may play a role in whether someone develops long-term symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To understand the roles of acute-phase viral dynamics and host immune responses in post-acute sequelae of SARS-CoV-2 infection (PASC), we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR test. Participants self-collected up to 21 nasal specimens within the first 28 days post-symptom onset; interviewer-administered questionnaires and blood samples were collected at enrollment, days 9, 14, 21, 28, and month 4 and 8 post-symptom onset. Defining PASC as the presence of any COVID-associated symptom at their 4-month visit, we compared viral markers (quantity and duration of nasal¬†viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-Œ±, IFN-Œ±, IFN-Œ≥, MCP, IP-10, and Spike IgG) over the acute period. Compared to those who fully recovered, those reporting PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA and N-antigen, burden of RNA and infectious viral shedding, and lower Spike-specific IgG levels within 9 days post-illness onset. No significant differences were identified among a panel of host immune markers. Our results suggest early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC, highlighting the importance of understanding early biological markers in the natural history of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">https://www.nature.com/articles/s41467-024-51893-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">
            ü§ñ How Safe and Effective is the mRNA-1273 Vaccine Over Time?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-555', this)">
            More</button></h4>
            <div id="toggle-555" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term safety and effectiveness of mRNA-1273 vaccine in adults: COVE trial open-label and booster phases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists conducted an experiment to test the effectiveness of a vaccine (mRNA-1273) in preventing COVID-19. The results showed that the vaccine was safe and worked well at preventing the disease in adults over 18 years old.

To see if another dose of the vaccine (called a booster) would help even more, researchers gave some people an extra dose after they had already received their first shots. They found that this booster shot helped prevent COVID-19, especially during times when new and more contagious variants of the virus were spreading.

The study also showed that the longer people waited between getting their first vaccine doses and receiving the booster, the better protected they were from getting COVID-19 again. Furthermore, the researchers found that people who received the booster had a stronger immune response against the virus than those who only got the initial shots.

Overall, this study suggests that vaccination with mRNA-1273, including a booster dose, is a safe and effective way to protect against COVID-19, even when new variants are present.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary vaccination with mRNA-1273 (100-¬µg) was safe and efficacious at preventing coronavirus disease 2019 (COVID-19) in the previously reported, blinded Part A of the phase 3 Coronavirus Efficacy (COVE; NCT04470427) trial in adults (‚â•18 years) across 99 U.S. sites. The open-label (Parts B and C) primary objectives were evaluation of long-term safety and effectiveness of primary vaccination plus a 50-¬µg booster dose; immunogenicity was a secondary objective. Of 29,035 open-label participants, 19,609 received boosters (mRNA-1273 [n=‚Äâ9647]; placebo-mRNA-1273 [n=‚Äâ9952]; placebo [n=‚Äâ10] groups). Booster safety was consistent with that reported for primary vaccination. Incidences of COVID-19 and severe COVID-19 were higher during the Omicron BA.1 than Delta variant waves and boosting versus non-boosting was associated with a significant, 47.0% (95% CI : 39.0-53.9%) reduction of Omicron BA.1 incidence (24.6 [23.4‚Äâ‚àí‚Äâ25.8] vs 46.4 [40.6‚Äâ‚àí‚Äâ52.7]/1000 person-months). In an exploratory Cox regression model adjusted for time-varying covariates, a longer median interval between primary vaccination and boosting (mRNA-1273 [13 months] vs placebo-mRNA-1273 [8 months]) was associated with significantly lower, COVID-19 risk (24.0% [16.0%‚Äâ‚àí‚Äâ32.0%]) during Omicron BA.1 predominance. Boosting elicited greater immune responses against SARS-CoV-2 than primary vaccination, irrespective of prior SARS-CoV-2 infection. Primary vaccination and boosting with mRNA-1273 demonstrated acceptable safety, effectiveness and immunogenicity against COVID-19, including emergent variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">https://www.nature.com/articles/s41467-024-50376-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-28</strong> <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">
            ü§ñ Fibrin's Role in COVID-19 Complications</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-556', this)">
            More</button></h4>
            <div id="toggle-556" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Fibrin drives thromboinflammation and neuropathology in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Some people who get COVID-19 develop blood clots that can be life-threatening, as well as brain symptoms like memory loss or confusion. This can happen even after they've recovered from the virus. Despite this, scientists still don't fully understand why it happens and there aren't enough treatments.

A protein called fibrin, which helps form blood clots, is found in high amounts in the lungs and brains of people with COVID-19. It's also a good indicator of how severe their symptoms will be and can predict brain problems after they've recovered.

Scientists have discovered that fibrin binds to the COVID-19 virus and forms pro-inflammatory blood clots, which causes more inflammation and damage to the brain and lungs. They've also found that targeting this part of the fibrin protein with a special antibody can help protect against further damage.

This suggests that targeting fibrin could be a new way to treat not only acute COVID-19 but also long-term symptoms after recovery.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Life-threatening thrombotic events and neurological symptoms are prevalent in COVID-19 and are¬†persistent in patients with long COVID experiencing post-acute sequelae of SARS-CoV-2 infection1,2,3,4. Despite the clinical evidence1,5,6,7, the underlying mechanisms of coagulopathy in COVID-19 and its consequences in inflammation and neuropathology remain poorly understood and treatment options are insufficient. Fibrinogen, the central structural component of blood clots, is abundantly deposited in the lungs and brains of patients with COVID-19, correlates with disease severity and is a predictive biomarker for post-COVID-19 cognitive deficits1,5,8,9,10. Here we show that fibrin binds to the SARS-CoV-2 spike protein, forming proinflammatory blood clots that drive systemic thromboinflammation and neuropathology in COVID-19. Fibrin, acting through its inflammatory domain, is required for oxidative stress and macrophage activation in the lungs, whereas it¬†suppresses natural killer cells, after SARS-CoV-2 infection. Fibrin promotes neuroinflammation and neuronal loss after infection, as well as innate immune activation in the brain and lungs independently of active infection. A¬†monoclonal antibody targeting the inflammatory fibrin domain provides¬†protection from microglial activation and neuronal injury, as well as from thromboinflammation in the lung after infection. Thus, fibrin drives inflammation and neuropathology in SARS-CoV-2 infection, and fibrin-targeting immunotherapy may represent a therapeutic intervention for patients with acute COVID-19 and long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">https://www.nature.com/articles/s41586-024-07873-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-28</strong> <a href="https://www.nature.com/articles/s41598-024-71047-5" target="_blank">
            ü§ñ IgE Response After COVID Vaccination and Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-557', this)">
            More</button></h4>
            <div id="toggle-557" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An unexpected IgE anti-receptor binding domain response following natural infection and different types of SARS-CoV-2 vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

When people get infected with COVID-19 or vaccinated against it, their bodies produce special proteins called antibodies to fight the virus. Most research has focused on one type of antibody, but there are other types too. This study looked at a different type of antibody called IgE, which is usually associated with allergies and parasites.

The researchers took blood samples from 59 people in Brazil who had been infected with COVID-19 or vaccinated against it. They found that IgE antibodies were produced after infection and vaccination. The good news was that the vaccines helped increase the levels of these antibodies, making them more effective at fighting the virus.

The study also found that different types of vaccines worked better than others in producing IgE antibodies. However, when people got a booster shot with a third vaccine, their IgE antibody levels became similar across all groups.

Overall, this study suggests that IgE antibodies play an important role in our immune system's response to COVID-19 and viruses, and more research is needed to understand how they work and their potential benefits.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humoral response to SARS-CoV-2 has been studied, predominantly the classical IgG and its subclasses. Although IgE antibodies are typically specific to allergens or parasites, a few reports describe their production in response to SARS-CoV-2 and other viruses. Here, we investigated IgE specific to receptor binding domain (RBD) of SARS-CoV-2 in a Brazilian cohort following natural infection and vaccination. Samples from 59 volunteers were assessed after infection (COVID-19), primary immunization with vectored (ChAdOx1) or inactivated (CoronaVac) vaccines, and booster immunization with mRNA (BNT162b2) vaccine. Natural COVID-19 induced IgE, but vaccination increased its levels. Subjects vaccinated with two doses of ChAdOx1 exhibited a more robust response than those immunized with two doses of CoronaVac; however, after boosting with BNT162b2, all groups presented similar IgE levels. IgE showed intermediate-to-high avidity, especially after the booster vaccine. We also found IgG4 antibodies, mainly after the booster, and they moderately correlated with IgE. ELISA results were confirmed by control assays, using IgG depletion by protein G and lack of reactivity with heterologous antigen. In our cohort, no clinical data could be associated with the IgE response. We advocate for further research on IgE and its role in viral immunity, extending beyond allergies and parasitic infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71047-5" target="_blank">https://www.nature.com/articles/s41598-024-71047-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">
            ü§ñ How Different Vaccines Interact with the Immune System</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-558', this)">
            More</button></h4>
            <div id="toggle-558" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibody Fc receptor binding and T cell responses to homologous and heterologous immunization with inactivated or mRNA vaccines against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Two different COVID-19 vaccines, CoronaVac and BNT162b2 (also known as Pfizer-BioNTech), were tested in a study to see how well they worked against severe illness from COVID-19. The results showed that while both vaccines are effective in preventing serious illness, they work differently inside the body.

The researchers looked at how well each vaccine stimulated the immune system to produce antibodies and T cells, which help fight off the virus. They found that:

* BNT162b2 (Pfizer-BioNTech) booster shots produced more antibodies against COVID-19 than CoronaVac booster shots.
* Participants who received CoronaVac as a first dose had lower levels of certain immune cells called CD4+IFNŒ≥+cells, but this level increased after receiving either CoronaVac or BNT162b2 booster shots.
* The type of vaccine used for the first dose (CoronaVac or Pfizer-BioNTech) influenced which types of T cells and antibodies were produced by the body.

These findings suggest that while both vaccines have their strengths and weaknesses, they can still provide protection against severe COVID-19 illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Whole virion inactivated vaccine CoronaVac (C) and Spike (S) mRNA BNT162b2 (B) vaccines differ greatly in their ability to elicit neutralizing antibodies but have somewhat comparable effectiveness in protecting from severe COVID-19. We conducted further analyses for a randomized trial (Cobovax study, NCT05057169) of third dose homologous and heterologous booster vaccination, i.e. four interventions CC-C, CC-B, BB-C and BB-B. Here, we assess vaccine immunogenicity beyond neutralizing function, including S and non-S antibodies with Fc receptor (FcR) binding, antibody avidity and T cell specificity to 6 months post-vaccination. Ancestral and Omicron S-specific IgG and FcR binding are significantly higher by BNT162b2 booster than CoronaVac, regardless of first doses. Nucleocapsid (N) antibodies are only increased in homologous boosted CoronaVac participants (CC-C). CoronaVac primed participants have lower baseline S-specific CD4+IFNŒ≥+cells, but are significantly increased by either CoronaVac or BNT162b2 boosters. Priming vaccine content defined T cell peptide specificity preference, with S-specific T cells dominating B primed groups and non-S structural peptides contributing more in C primed groups, regardless of booster type. S-specific CD4+T cell responses, N-specific antibodies, and antibody effector functions via Fc receptor binding may contribute to protection and compensate for less potent neutralizing responses in CoronaVac recipients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">https://www.nature.com/articles/s41467-024-51427-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41467-024-50976-9" target="_blank">
            ü§ñ Designing Improved COVID Vaccine Proteins Using Computer Simulations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-559', this)">
            More</button></h4>
            <div id="toggle-559" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Simulation-driven design of stabilized SARS-CoV-2 spike S2 immunogens</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 vaccine works by using a special part of the virus called the spike (S) to help our bodies fight it. However, the virus can change over time and become harder for the vaccine to recognize. The stable part of the spike is called S2, which doesn't change much and can trigger strong antibodies in our bodies that can protect us against different types of viruses. But, this part of the spike is difficult to keep stable and has been hard to use as a vaccine. We used computer simulations to design a new version of S2 that is more stable and still triggers strong antibodies. Our results show that this new version works well in tests and could be an effective alternative to traditional COVID-19 vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The full-length prefusion-stabilized SARS-CoV-2 spike (S) is the principal antigen of COVID-19 vaccines. Vaccine efficacy has been impacted by emerging variants of concern that accumulate most of the sequence modifications in the immunodominant S1 subunit. S2, in contrast, is the most evolutionarily conserved region of the spike and can elicit broadly neutralizing and protective antibodies. Yet, S2‚Äôs usage as an alternative vaccine strategy is hampered by its general instability. Here, we use a simulation-driven approach to design S2-only immunogens stabilized in a closed prefusion conformation. Molecular simulations provide a mechanistic characterization of the S2 trimer‚Äôs opening, informing the design of tryptophan substitutions that impart kinetic and thermodynamic stabilization. Structural characterization via cryo-EM shows the molecular basis of S2 stabilization in the closed prefusion conformation. Informed by molecular simulations and corroborated by experiments, we report an engineered S2 immunogen that exhibits increased protein expression, superior thermostability, and preserved immunogenicity against sarbecoviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50976-9" target="_blank">https://www.nature.com/articles/s41467-024-50976-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41598-024-70064-8" target="_blank">
            ü§ñ Using a special inhaler treatment with extra-hepatin benefits people with COVID-19 symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-560', this)">
            More</button></h4>
            <div id="toggle-560" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nebulized enriched heparin improves respiratory parameters in patients with COVID-19: a phase I/II randomized and triple-blind clinical trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists wanted to see if inhaling a medicine called inhaled enriched heparin can help treat COVID-19. They tested it on 27 hospitalized adults with COVID-19 and compared them to people who got a placebo (a fake treatment). The results showed that inhaling this medicine was safe and helped reduce the need for oxygen and improved lung function in patients. These promising findings mean they want to continue testing it in even larger groups of people to see if it's truly effective in treating COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the safety and the potential antiviral treatment of inhaled enriched heparin in patients with COVID-19. The specific objectives were to investigate the anticoagulation profile, antiviral and anti-inflammatory effects, and respiratory evolution of inhaled enriched heparin. We conducted a randomized, triple-blind, placebo-controlled Phase I/II clinical trial in hospitalized adults with COVID-19 receiving inhalation of enriched heparin or saline (placebo) every 4¬†h for 7¬†days. Among the 27 patients who completed the study, no changes in blood coagulation parameters were observed, indicating the safety of inhaled enriched heparin. The group receiving enriched heparin showed a significant reduction in the need for supplemental oxygen and improvement in respiratory parameters, such as the PaO2/FiO2ratio. Inhalation of enriched heparin is shown to be safe and has also demonstrated potential therapeutic benefits for patients with COVID-19. These promising results justify the continuation of the study to the next phase, Phase II/III, to further evaluate the therapeutic efficacy of inhaled enriched heparin in the treatment of COVID-19-associated viral pneumonia.Trial registration:¬†ClinicalTrials.gov. 08/02/2021. Identifier: NCT04743011.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70064-8" target="_blank">https://www.nature.com/articles/s41598-024-70064-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41541-024-00948-3" target="_blank">
            ü§ñ Pregnant Women's COVID Immunity During Pregnancy</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-561', this)">
            More</button></h4>
            <div id="toggle-561" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing and binding antibody responses to SARS-CoV-2 with hybrid immunity in pregnancy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

When pregnant women get vaccinated against COVID-19 or catch the virus, their bodies make antibodies that can help protect them and their babies. Researchers studied how well these antibodies worked after different combinations of vaccination and infection during pregnancy.

They found that when women got three doses of the vaccine, it boosted the level of protective antibodies more than just two doses or getting infected with COVID-19 alone. This effect was even stronger when women had both vaccinated and been infected.

The researchers also looked at how well these antibodies were transferred from mother to baby during pregnancy. They found that antibodies made by the vaccine were passed on more efficiently than those made by infection.

These findings suggest that combining vaccination and natural immunity can create a stronger defense against COVID-19 in pregnant women, which is good news for protecting both mom and baby!</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hybrid immunity against SARS-CoV-2 has not been well studied in pregnancy. We conducted a comprehensive analysis of neutralizing antibodies (nAb) and binding antibodies in pregnant individuals who received mRNA vaccination, natural infection, or both. A third vaccine dose augmented nAb levels compared to the two-dose regimen or natural infection alone; this effect was more pronounced in hybrid immunity. There was reduced anti-Omicron nAb, but the maternal-fetal transfer efficiency remained comparable to that of other variants. Vaccine-induced nAbs were transferred more efficiently than infection-induced nAbs. Anti-spike receptor binding domain (RBD) IgG was associated with nAb against wild-type (Wuhan-Hu-1) following breakthrough infection. Both vaccination and infection-induced anti-RBD IgA, which was more durable than anti-nucleocapsid IgA. IgA response was attenuated in pregnancy compared to non-pregnant controls. These data provide additional evidence of augmentation of humoral immune responses in hybrid immunity in pregnancy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00948-3" target="_blank">https://www.nature.com/articles/s41541-024-00948-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-26</strong> <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">
            ü§ñ Single Dose COVID Vaccine for Wide Protection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-562', this)">
            More</button></h4>
            <div id="toggle-562" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A single-dose intranasal live-attenuated codon deoptimized vaccine provides broad protection against SARS-CoV-2 and its variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

COVID-19, caused by the SARS-CoV-2 virus, continues to affect people worldwide and cause significant health and economic problems. Despite vaccines being effective in preventing severe cases, there is still no guarantee against new variants of the virus or unexpected breakthrough infections. Researchers have developed a new vaccine called CDO-7N-1 that uses a weakened form of the SARS-CoV-2 virus to stimulate an immune response. In laboratory tests, this vaccine has been shown to provide long-lasting protection against different strains of the virus and prevent lung damage. A single dose of the vaccine triggered a strong immune response in mice, hamsters, and macaques, protecting them from infection with wild-type SARS-CoV-2 and other variants of concern. The researchers believe that this new vaccine could improve immunity against future SARS-CoV-2 variants and provide better protection for people worldwide.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, COVID-19) continues its significant health and economic impact globally. Despite the success of spike-protein vaccines in preventing severe disease, long-lasting protection against emerging variants and the prevention of breakthrough infections and transmission remain elusive. We generate an intranasal live-attenuated SARS-CoV-2 vaccine, CDO-7N-1, using codon deoptimization. CDO-7N-1 shows highly attenuated replication and minimal or no lung pathology in vivo over multiple passages. It induces robust mucosal and systemic neutralizing antibody and T-cell subset responses, in mice (female K18-hACE2 and male HFH4-hACE2 mice), hamsters, and macaques triggered by a single immunization. Mice and hamsters vaccinated with CDO-7N-1 are protected from challenge with wild-type (WT) SARS-CoV-2 and other variants of concern. Serum from vaccinated animals neutralizes WT SARS-CoV-2, variants of concern (beta and delta), variants of interest (omicron XBB.1.5) and SARS-CoV-1. Antibody responses are sustained and enhanced by repeated immunization or infection with WT SARS-CoV-2. Immunity against all SARS-CoV-2 proteins by CDO-7N-1 should improve efficacy against future SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">https://www.nature.com/articles/s41467-024-51535-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-24</strong> <a href="https://www.nature.com/articles/s41598-024-70612-2" target="_blank">
            ü§ñ How Aging Affects COVID-19 Symptoms in Mice Studies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-563', this)">
            More</button></h4>
            <div id="toggle-563" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Premature aging effects on COVID-19 pathogenesis: new insights from mouse models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

As we get older, our bodies become more vulnerable to severe illnesses like COVID-19. The connection between age, existing health problems, and COVID-19 is not fully understood. This study used a special mouse model that ages quickly to see how COVID-19 affects different age groups.

The results showed that younger mice can fight off the virus easily, but older mice have a weaker response and are more likely to get very sick and die. The mouse model with premature aging showed milder symptoms, but still had some problems with their immune system and metabolism.

By studying how COVID-19 affects these mice, scientists hope to gain a better understanding of why some people are more susceptible to severe illness and develop new treatments for COVID-19 and other diseases related to aging.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Aging is identified as a significant risk factor for severe coronavirus disease-2019 (COVID-19), often resulting in profound lung damage and mortality. Yet, the biological relationship between aging, aging-related comorbidities, and COVID-19 remains incompletely understood. This study aimed to elucidate the age-related COVID19 pathogenesis using an Hutchinson-Gilford progeria syndrome (HGPS) mouse model, a premature aging disease model, with humanized ACE2 receptors. Pathological features were compared between young, aged, and HGPS hACE2 mice following SARS-CoV-2 challenge. We demonstrated that young mice display robust interferon response and antiviral activity, whereas this response is attenuated in aged mice. Viral infection in aged mice results in severe respiratory tract hemorrhage, likely contributing a higher mortality rate. In contrast, HGPS hACE2 mice exhibit milder disease manifestations characterized by minor immune cell infiltration and dysregulation of multiple metabolic processes. Comprehensive transcriptome analysis revealed both shared and unique gene expression dynamics among different mouse groups. Collectively, our studies evaluated the impact of SARS-CoV-2 infection on progeroid syndromes using a HGPS hACE2 mouse model, which holds promise as a useful tool for investigating COVID-19 pathogenesis in individuals with premature aging.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70612-2" target="_blank">https://www.nature.com/articles/s41598-024-70612-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-24</strong> <a href="https://www.nature.com/articles/s44298-024-00049-x" target="_blank">
            ü§ñ Lingering Lung Damage After COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-564', this)">
            More</button></h4>
            <div id="toggle-564" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection results in a unique lung proteome long after virus resolution in the hamster</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Long COVID, also known as post-acute sequelae of COVID-19 (PASC), is a condition where people experience ongoing problems after having COVID-19. It can make it hard for them to contribute to society. To understand why this happens, researchers studied the lung proteins in hamsters infected with an early version of the SARS-CoV-2 virus. They found that at 3 and 5 days after infection, many lung proteins changed in both male and female hamsters. However, by day 31 (when no virus was left), they saw differences between the sexes, including some proteins that were lower or higher in males than females. The researchers also found that biological sex should be considered when developing treatments for PASC because of these differences.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long COVID or post-acute sequelae of COVID-19 (PASC) remains an ongoing public health issue that causes impairment for those afflicted and diminishes their ability to contribute to society. To address the host response underpinning respiratory PASC, we used the Golden Syrian hamster model infected with ancestral SARS-CoV-2 and examined its lung proteome in a longitudinal experiment. We infected young 6-week old male and female hamsters with 105TCID50of virus via the intranasal route and sampled the lung at 1, 3, 5, and 31 days post infection (dpi). We compared the infected lung proteome to that of uninfected sex-matched controls. We found almost no differences in protein levels at 1 dpi, with hundreds at 3 dpi, and thousands at 5 dpi. Many overlapping differential protein levels and pathways were seen in both sexes at 3 and 5 dpi including the Coagulation and Complement cascades. Notably, we found differences between the sexes at 31 dpi which included many targets with decreased levels of protein in the males. We also noted an increase in 7 proteins in both sexes at 31 dpi including proteins responsible for airway mucosal layer integrity such as Mucin 5B and Calcium-activated chloride channel regulator 1. Longitudinally, 38 proteins were changed in levels across more than one timepoint in the males but only three proteins were in the females, Secretoglobin family 1‚ÄâA member 1, Poly [ADP-ribose] polymerase, and Apolipoprotein D. Overall, we show that there are changes to the lung proteome at 31 dpi, a time when no SARS-CoV-2 remains, and that there are sex differences in that proteome after infection with the ancestral strain. We conclude that biological sex should be examined as a variable when testing medical countermeasures for PASC in the Golden Syrian hamster due to host differences between the sexes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00049-x" target="_blank">https://www.nature.com/articles/s44298-024-00049-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-20</strong> <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">
            ü§ñ Non-human primates show signs of long-COVID linked to high blood sugar immune response.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-565', this)">
            More</button></h4>
            <div id="toggle-565" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Non-human primate model of long-COVID identifies immune associates of hyperglycemia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Hyperglycemia happens when your body has trouble using sugar after catching a severe illness like COVID-19. Scientists don't understand this very well in people, so they used monkeys infected with the same virus to study it better.

They found that shortly after getting sick, the blood of these monkeys had changes called "chemokines" that could lead to high sugar levels. Four months later, this still happened in most of them. High sugar is linked to something called glycogen in the liver and pancreas. But they didn't see the virus living in those parts anymore.

They also tested a vaccine for COVID-19 given to monkeys on day 4 after infection. The monkey's blood sugar levels went down as a result.

These findings suggest that studying these infected monkeys can help understand what happens when people get sick from this illness, and how we might treat it better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hyperglycemia, and exacerbation of pre-existing deficits in glucose metabolism, are manifestations of the post-acute sequelae of SARS-CoV-2. Our understanding of metabolic decline after acute COVID-19 remains unclear due to the lack of animal models. Here, we report a non-human primate model of metabolic post-acute sequelae of SARS-CoV-2 using SARS-CoV-2 infected African green monkeys. Using this model, we identify a dysregulated blood chemokine signature during acute COVID-19 that correlates with elevated and persistent hyperglycemia four months post-infection. Hyperglycemia also correlates with liver glycogen levels, but there is no evidence of substantial long-term SARS-CoV-2 replication in the liver and pancreas. Finally, we report a favorable glycemic effect of the SARS-CoV-2 mRNA vaccine, administered on day 4 post-infection. Together, these data suggest that the African green monkey model exhibits important similarities to humans and can be utilized to assess therapeutic candidates to combat COVID-related metabolic defects.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">https://www.nature.com/articles/s41467-024-50339-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-13</strong> <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">
            ü§ñ SARS-CoV-2 N protein affects lung genes leading to pneumonia.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-566', this)">
            More</button></h4>
            <div id="toggle-566" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 N protein-induced Dicer, XPO5, SRSF3, and hnRNPA3 downregulation causes pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how certain pieces of machinery that help make RNA (a type of molecule important for cells) work less effectively when someone has COVID-19. When the researchers found there were fewer of these pieces, they had more trouble with their lungs getting sick from the virus. The study showed that a protein made by the virus also makes these pieces disappear faster. This leads to problems like DNA damage and lung issues. They also noticed older people have less of these pieces in their lungs, making them sicker when infected. Two types of medicine helped make the sickness better.

The main points are:
- Certain RNA-making parts don't work well in COVID patients.
- The virus causes fewer of these parts to be made.
- This makes it harder for cells to repair themselves and function properly.
- Older people get more sick because they have less of these parts.
- Special medicines can help.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Though RNAi and RNA-splicing machineries are involved in regulating severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication, their precise roles in coronavirus disease 2019 (COVID-19) pathogenesis remain unclear. Herein, we show that decreased RNAi component (Dicer and XPO5) and splicing factor (SRSF3 and hnRNPA3) expression correlate with increased COVID-19 severity. SARS-CoV-2 N protein induces the autophagic degradation of Dicer, XPO5, SRSF3, and hnRNPA3, inhibiting miRNA biogenesis and RNA splicing and triggering DNA damage, proteotoxic stress, and pneumonia. Dicer, XPO5, SRSF3, and hnRNPA3 knockdown increases, while their overexpression decreases, N protein-induced pneumonia‚Äôs severity. Older mice show lower expression of Dicer, XPO5, SRSF3, and hnRNPA3 in their lung tissues and exhibit more severe N protein-induced pneumonia than younger mice. PJ34, a poly(ADP-ribose) polymerase inhibitor, or anastrozole, an aromatase inhibitor, ameliorates N protein- or SARS-CoV-2-induced pneumonia by restoring Dicer, XPO5, SRSF3, and hnRNPA3 expression. These findings will aid in developing improved treatments for SARS-CoV-2-associated pneumonia.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">https://www.nature.com/articles/s41467-024-51192-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">
            ü§ñ Major circulating SARS-CoV-2 B cells are unusual after vaccine or infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-567', this)">
            More</button></h4>
            <div id="toggle-567" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Atypical and non-classical CD45RBlomemory B cells are the majority of circulating SARS-CoV-2 specific B cells following mRNA vaccination or COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The resting memory B cells are divided into two main groups: classical and atypical. Classical cells stay the same after being activated. Atypical cells can be further split into two types: one with CD11c on their surface, and another with CD23. 

When someone gets a vaccine or gets sick from bacteria, these resting memory B cells help make antibodies that fight the virus. In both scenarios, atypical cells called CD45RBlo play a big role in making antibodies against SARS-CoV-2.

Our findings suggest that tracking the levels of CD45RB on these cells can help us understand how they work and contribute to getting vaccinated or recovering from infections like COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Resting memory B cells can be divided into classical or atypical groups, but the heterogenous marker expression on activated memory B cells makes similar classification difficult. Here, by longitudinal analysis of mass cytometry and CITE-seq data from cohorts with COVID-19, bacterial sepsis, or BNT162b2 mRNA vaccine, we observe that resting B cell memory consist of classical CD45RB+memory and CD45RBlomemory, of which the latter contains of two distinct groups of CD11c+atypical and CD23+non-classical memory cells. CD45RB levels remain stable in these cells after activation, thereby enabling the tracking of activated B cells and plasmablasts derived from either CD45RB+or CD45RBlomemory B cells. Moreover, in both COVID-19 patients and mRNA vaccination, CD45RBloB cells formed the majority of SARS-CoV2 specific memory B cells and correlated with serum antibodies, while CD45RB+memory are activated by bacterial sepsis. Our results thus identify that stably expressed CD45RB levels can be exploited to trace resting memory B cells and their activated progeny, and suggest that atypical and non-classical CD45RBlomemory B cells contribute to SARS-CoV-2 infection and vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">https://www.nature.com/articles/s41467-024-50997-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-08</strong> <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">
            ü§ñ Monkeypox Virus Infection Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-568', this)">
            More</button></h4>
            <div id="toggle-568" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-omics characterization of the monkeypox virus infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We studied how monkeypox (MPXV) virus affects human cells using different types of genetic analysis. We found that the virus changes many pathways in the infected cells, including those related to immune responses and cell growth. One interesting thing we noticed is that MPXV causes changes in proteins involved in cell signaling. Specifically, we saw how it can affect a protein called H5 which interacts with DNA. 

We also used this data to look for new treatments that could stop the virus from growing inside cells. By looking at all these different pathways and interactions, we found some promising targets like MTOR, CHUK/IKBKB, and splicing factor kinases, which seem to have strong effects against both monkeypox and another similar virus called vaccinia (VACV).

Overall, our study helps us understand how these viruses affect cells and opens up new possibilities for finding treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multiple omics analyzes of Vaccinia virus (VACV) infection have defined molecular characteristics of poxvirus biology. However, little is known about the monkeypox (mpox) virus (MPXV) in humans, which has a different disease manifestation despite its high sequence similarity to VACV. Here, we perform an in-depth multi-omics analysis of the transcriptome, proteome, and phosphoproteome signatures of MPXV-infected primary human fibroblasts to gain insights into the virus-host interplay. In addition to expected perturbations of immune-related pathways, we uncover regulation of the HIPPO and TGF-Œ≤ pathways. We identify dynamic phosphorylation of both host and viral proteins, which suggests that MAPKs are key regulators of differential phosphorylation in MPXV-infected cells. Among the viral proteins, we find dynamic phosphorylation of H5 that influenced the binding of H5 to dsDNA. Our extensive dataset highlights signaling events and hotspots perturbed by MPXV, extending the current knowledge on poxviruses. We use integrated pathway analysis and drug-target prediction approaches to identify potential drug targets that affect virus growth. Functionally, we exemplify the utility of this approach by identifying inhibitors of MTOR, CHUK/IKBKB, and splicing factor kinases with potent antiviral efficacy against MPXV and VACV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">https://www.nature.com/articles/s41467-024-51074-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">
            ü§ñ Modifiable lifestyle factors and COVID-19 outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-569', this)">
            More</button></h4>
            <div id="toggle-569" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modifiable lifestyle factors and the risk of post-COVID-19 multisystem sequelae, hospitalization, and death</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Effective ways to prevent problems after getting sick from COVID are important for people, doctors, and governments. This study looked at how things like exercise, diet, sleep, smoking, drinking, being a healthy weight, and eating well can affect someone's health years later.

People who had more healthy habits were less likely to get many different kinds of long-term sicknesses related to COVID. They also had a lower risk of dying or needing hospitalization after getting sick from COVID. These healthier habits worked even for people who didn't get very sick in the first place.

The study found that following these healthy habits was really important and not just because they might have had fewer other health problems before catching COVID. This means it's good to keep up with a healthy lifestyle, no matter what, to help prevent long-term sickness from COVID.

These findings show how keeping fit and eating well can be very helpful in the future for dealing with possible new pandemics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Effective prevention strategies for post-COVID complications are crucial for patients, clinicians, and policy makers to mitigate their cumulative burden. This study evaluated the association of modifiable lifestyle factors (smoking, alcohol intake, BMI, physical activity, sedentary time, sleep duration, and dietary habits) with COVID-19 multisystem sequelae, death, and hospitalization in the UK Biobank cohort (n=‚Äâ68,896). A favorable lifestyle (6-10 healthy factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae (HR, 0.64; 95% CI, 0.58-0.69; ARR at 210 days, 7.08%; 95% CI, 5.98-8.09) compared to an unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions spanned all 10 organ systems, including cardiovascular, coagulation, metabolic, gastrointestinal, kidney, mental health, musculoskeletal, respiratory disorders, and fatigue. This beneficial effect was largely attributable to direct lifestyle impacts independent of corresponding pre-infection comorbidities (71% for any sequelae). A favorable lifestyle was also related to the risk of post-COVID death (HR 0.59, 0.52-0.66) and hospitalization (HR 0.78, 0.73-0.84). These associations persisted across acute and post-acute infection phases, irrespective of hospitalization status, vaccination, or SARS-CoV-2 variant. These findings underscore the clinical and public health importance of adhering to a healthy lifestyle in mitigating long-term COVID-19 adverse impacts and enhancing future pandemic preparedness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">https://www.nature.com/articles/s41467-024-50495-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">
            ü§ñ Tetravalent COVID-19 vaccine trial results</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-570', this)">
            More</button></h4>
            <div id="toggle-570" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of the tetravalent protein COVID-19 vaccine, SCTV01E: a phase 3 double-blind, randomized, placebo-controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study tested a new type of vaccine called SCTV01E that can protect against four different strains of coronavirus (SARS-CoV-2). The goal was to see if it works by comparing how many people got sick with the virus when given the vaccine versus not getting the vaccine.

They gave the vaccine to 9,223 people and had them wait for seven days. They found that the vaccine worked very well in preventing someone from getting sick: about two-thirds of people who didn't get vaccinated got sick compared to only one-third who did get the vaccine. It also helped protect against all kinds of coronavirus infections.

The vaccine was safe, with most side effects being mild and short-lived. The researchers think this type of vaccine might work well against new strains that appear over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Evolution of SARS-CoV-2 variants emphasizes the need for multivalent vaccines capable of simultaneously targeting multiple strains. SCTV01E is a tetravalent COVID-19 vaccine derived from the spike protein of SARS-CoV-2 variants Alpha, Beta, Delta, and Omicron BA.1. In this double-blinded placebo-controlled pivotal efficacy trial (NCT05308576), the primary endpoint was vaccine efficacy (VE) against COVID-19 seven days post-vaccination in individuals without recent infection. Other endpoints included evaluating safety, immunogenicity, and the VE against all SARS-CoV-2 infections in individuals meeting the study criteria. Between December 26, 2022, and January 15, 2023, 9,223 individuals were randomized at a 1:1 ratio to receive SCTV01E or a placebo. SCTV01E showed a VE of 69.4% (95% CI: 50.6, 81.0) 7 days post-vaccination, with 75 cases in the placebo group and 23 in the SCTV01E group for the primary endpoint. VEs were 79.7% (95% CI: 51.0, 91.6) and 82.4% (95% CI: 57.9, 92.6), respectively, for preventing symptomatic infection and all SARS-CoV-2 infections 14 days post-vaccination. SCTV01E elicited a 25.0-fold higher neutralizing antibody response against Omicron BA.5 28 days post-vaccination compared to placebo. Reactogenicity was generally mild and transient, with no reported vaccine-related SAE, adverse events of special interest (AESI), or deaths. The trial aligned with the shift from dominant variants BA.5 and BF.7 to XBB, suggesting SCTV01E as a potential vaccine alternative effective against present and future variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">https://www.nature.com/articles/s41467-024-49832-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">
            ü§ñ SARS-CoV-2 spikes block a lung gene, causing inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-571', this)">
            More</button></h4>
            <div id="toggle-571" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inflammation in the COVID-19 airway is due to inhibition of CFTR signaling by the SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 makes people sick partly by causing an overreaction in their lungs that leads to more inflammation. This inflammation can last for a long time after the virus is gone, which some doctors call "Long COVID."

The virus also stops another system called CFTR from working properly in lung cells. CFTR helps keep the lungs moist and healthy. When this doesn't work, it causes problems similar to a condition called cystic fibrosis.

We think SARS-CoV-2 might be stopping CFTR because of something on its surface called spike protein. This could happen by blocking another system on the cell surface that normally helps fix CFTR.

Our experiments showed that when lung cells are exposed to this virus, they lose their ability to make and use a healthy version of CFTR. Without CFTR working properly, cells start making too much of two other things that cause inflammation: NFŒ∫B and ENaC.

We found that adding certain drugs called cardiac glycosides can help keep the cells' CFTR from disappearing when exposed to the virus. These drugs might be important for treating "Long COVID" as well.

Overall, our experiments suggest that SARS-CoV-2's spike protein stops a healthy system in lung cells (CFTR) from working properly, which could lead to inflammation and possibly cause symptoms of "Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2-contributes to sickness and death in COVID-19 patients partly by inducing a hyper-proinflammatory immune response in the host airway. This hyper-proinflammatory state involves activation of signaling by NFŒ∫B, and unexpectedly, ENaC, the epithelial sodium channel. Post-infection inflammation may also contribute to "Long COVID"/PASC. Enhanced signaling by NFŒ∫B and ENaC also marks the airway of patients suffering from cystic fibrosis, a life-limiting proinflammatory genetic disease due to inactivating mutations in the CFTR gene. We therefore hypothesized that inflammation in the COVID-19 airway might similarly be due to inhibition of CFTR signaling by SARS-CoV-2 spike protein, and therefore activation of both NFŒ∫B and ENaC signaling. We used western blot and electrophysiological techniques, and an organoid model of normal airway epithelia, differentiated on an air‚Äìliquid-interface (ALI). We found that CFTR protein expression and CFTR cAMP-activated chloride channel activity were lost when the model epithelium was exposed to SARS-CoV-2 spike proteins. As hypothesized, the absence of CFTR led to activation of both TNFŒ±/NFŒ∫B signaling and Œ± and Œ≥ ENaC. We had previously shown that the cardiac glycoside drugs digoxin, digitoxin and ouabain blocked interaction of spike protein and ACE2. Consistently, addition of 30¬†nM concentrations of the cardiac glycoside drugs, prevented loss of both CFTR protein and CFTR channel activity. ACE2 and CFTR were found to co-immunoprecipitate in both basal cells and differentiated epithelia. Thus spike-dependent CFTR loss might involve ACE2 as a bridge between Spike and CFTR. In addition, spike exposure to the epithelia resulted in failure of endosomal recycling to return CFTR to the plasma membrane. Thus, failure of CFTR recovery from endosomal recycling might be a mechanism for spike-dependent loss of CFTR. Finally, we found that authentic SARS-CoV-2 virus infection induced loss of CFTR protein, which was rescued by the cardiac glycoside drugs digitoxin and ouabain. Based on experiments with this organoid model of small airway epithelia, and comparisons with 16HBE14o- and other cell types expressing normal CFTR, we predict that inflammation in the COVID-19 airway may be mediated by inhibition of CFTR signaling by the SARS-CoV-2 spike protein, thus inducing a cystic fibrosis-like clinical phenotype. To our knowledge this is the first time COVID-19 airway inflammation has been experimentally traced in normal subjects to a contribution from SARS-CoV-2 spike-dependent inhibition of CFTR signaling.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">https://www.nature.com/articles/s41598-024-66473-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">
            ü§ñ SARS-CoV-2 Reinfections and Long COVID Study Results</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-572', this)">
            More</button></h4>
            <div id="toggle-572" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Insights from an N3C RECOVER EHR-based cohort study characterizing SARS-CoV-2 reinfections and Long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SIMPLIFIED

The pandemic is still going on for over 3 years, but doctors don't fully know about getting sick with a virus called SARS-CoV-2 more than once. We studied lots of people's health records to learn more.

We found that:
- Most reinfections happened during the Omicron wave.
- People who got infected twice had more Long COVID (a long-term sickness) compared to those who only got it once in the same period.
- Before you get reinfected, your body might not have as much protein.
- Getting a bad first infection also makes the second one worse. 
- Older people and those at higher risk of dying are more likely to have a severe second infection if they had a bad first one.

This helps us understand reinfections better so we can find new ways to help people who get sick again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundAlthough the COVID-19 pandemic has persisted for over 3 years, reinfections with SARS-CoV-2 are not well understood. We aim to characterize reinfection, understand development of Long COVID after reinfection, and compare severity of reinfection with initial infection.MethodsWe use an electronic health record study cohort of over 3 million patients from the National COVID Cohort Collaborative as part of the NIH Researching COVID to Enhance Recovery Initiative. We calculate summary statistics, effect sizes, and Kaplan‚ÄìMeier curves to better understand COVID-19 reinfections.ResultsHere we validate previous findings of reinfection incidence (6.9%), the occurrence of most reinfections during the Omicron epoch, and evidence of multiple reinfections. We present findings that the proportion of Long COVID diagnoses is higher following initial infection than reinfection for infections in the same epoch. We report lower albumin levels leading up to reinfection and a statistically significant association of severity between initial infection and reinfection (chi-squared value: 25,697,p-value: <0.0001) with a medium effect size (Cramer‚ÄôsV: 0.20, DoF‚Äâ=‚Äâ3). Individuals who experienced severe initial and first reinfection were older in age and at a higher mortality risk than those who had mild initial infection and reinfection.ConclusionsIn a large patient cohort, we find that the severity of reinfection appears to be associated with the severity of initial infection and that Long COVID diagnoses appear to occur more often following initial infection than reinfection in the same epoch. Future research may build on these findings to better understand COVID-19 reinfections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">https://www.nature.com/articles/s43856-024-00539-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">
            ü§ñ Human scFv selection and characterization for SARS-CoV-2 nucleocapsid antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-573', this)">
            More</button></h4>
            <div id="toggle-573" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Selection and characterization of human scFvs targeting the SARS-CoV-2 nucleocapsid protein isolated from antibody libraries of COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In 2019, a new coronavirus called SARS-CoV-2 appeared in China and caused an illness called COVID-19. At first, scientists mainly studied the spike protein. But they later found out that another part of the virus, called the nucleocapsid (N) protein, is very important for spreading the disease and fighting off our immune system. 

Scientists decided to create a collection of special antibodies against this N protein using patients who had been infected with SARS-CoV-2. They used a new technology to pick out some strong antibody parts called single chain fragments variable (scFvs). These scFvs can be made into different forms like tiny intrabodies inside cells or as regular proteins outside the cells. 

They found that some of these antibodies work very well when they are inside cells, and others work even better when they are made separately from the cell. All of them can stick to the N protein with a strong grip. This collection of powerful N protein antibodies is new and could help in research and making tests for detecting SARS-CoV-2.

So basically, scientists got some really good tools to fight against this virus by studying its parts more closely.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In 2019, the novel SARS-CoV-2 coronavirus emerged in China, causing the pneumonia named COVID-19. At the beginning, all research efforts were focused on the spike (S) glycoprotein. However, it became evident that the nucleocapsid (N) protein is pivotal in viral replication, genome packaging and evasion of the immune system, is highly immunogenic, which makes it another compelling target for antibody development alongside the spike protein. This study focused on the construction of single chain fragments variable (scFvs) libraries from SARS-CoV-2-infected patients to establish a valuable, immortalized and extensive antibodies source. We used the Intracellular Antibody Capture Technology to select a panel of scFvs against the SARS-CoV-2 N protein. The whole panel of scFv was expressed and characterized both as intrabodies and recombinant proteins. ScFvs were then divided into 2 subgroups: those that exhibited high binding activity to N protein when expressed in yeast or in mammalian cells as intrabodies, and those purified as recombinant proteins, displaying affinity for recombinant N protein in the nanomolar range. This panel of scFvs against the N protein represents a novel platform for research and potential diagnostic applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">https://www.nature.com/articles/s41598-024-66558-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-01</strong> <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">
            ü§ñ Contraceptive Access Disparities During COVID-19 in Brazil</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-574', this)">
            More</button></h4>
            <div id="toggle-574" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A particular epidemiological profile: disparities in access to contraceptive methods in Brazil during the SARS-CoV-2 (COVID-19) pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how easier or harder it was for people to get birth control during the pandemic in Brazil. They surveyed about 1000 young adults aged 20-39 who use contraception there. Most respondents were women, white, and using some form of birth control.

Many people changed their birth control method during the pandemic. But more than a quarter said they had trouble finding information on new methods or seeing healthcare providers to get help with their current birth control.

The survey found that younger black and biracial women with less education and income faced bigger challenges getting access to contraception and family planning services.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our objective was to study disparities in access to contraception during the COVID-19 pandemic. We performed a cross-sectional study at the University of Campinas, Brazil using a Google questionnaire applied from December 2021 until February 2022, disseminated via snowball technique. The survey asked about sociodemographic characteristics and contraceptive use, as well as the demand for new methods and difficulties in continuing to use contraceptives during the COVID-19 pandemic. We analyzed 1018 completed questionnaires; in total, 742 (72.9%) were women aged between 20 and 39¬†years, 746 (73.3%) were White and 602 (59.2%) used contraceptives. During the COVID-19 pandemic, about 23% of respondents changed their method and approximately 20% of respondents looked for new methods. Among the latter, 31.3% reported some difficulty with obtaining guidance on new methods while only 5.3% of the respondents reported some difficulty with continuing their contraceptive. The main difficulty in both cases was the difficulty with getting a healthcare provider appointment. Our results point to a particular epidemiological population, of younger black and biracial women, with lower education and lower income, which suffered health disparities during the COVID-19 pandemic and found difficulties with using contraceptives and accessing family planning services.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">https://www.nature.com/articles/s41598-024-65946-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-28</strong> <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">
            ü§ñ COVID-19 patient immune responses and disease severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-575', this)">
            More</button></h4>
            <div id="toggle-575" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 antigenemia and RNAemia in association with disease severity in patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT REWRITTEN

Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes different sickness levels from mild to severe. We studied how this virus affects people and what parts of the body it gets into. In our study, we looked at two things: when certain parts of the virus show up in a person's blood (antigenemia), and when its genetic material is detectable (RNAemia).

We examined 117 patients with confirmed COVID-19. Their ages ranged from around 52 to 79 years old, with more males than females. We found that in people who died early, the virus parts showed up very fast within a week after they were admitted.

When we looked at how these virus parts show up over time:
- They showed up in everyone's blood (100%) by the first week.
- At admission, they could be seen 64.7% of the time but correctly identified as being there 73% of the time.
- In the second week, they were spotted in about 69% of people and correctly detected in all cases.

We also saw that sick patients who had no symptoms at first (asymptomatic) still showed antigenemia. But by the end of the first week, none of them did anymore. The virus genetic material was found more often in people who died than in those who were just mildly sick.

Looking at how long people lived after being admitted:
- People with higher levels of these parts showing up in their blood had a worse chance of surviving.
- This finding suggests that if the virus parts show up quickly, it might mean someone is more likely to die early.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus responsible for COVID-19, causes a spectrum of symptoms ranging from mild upper to severe lower respiratory tract infections.However, the dynamics of nucleocapsid (N) protein antigenemia and RNAemia are not fully understood. We conducted a cohort study involving 117 patients with clinically confirmed COVID-19, focusing on the kinetics of antigenemia and RNAemia and their association with various clinical characteristics. The patients had a median age of 66.0¬†years (52.0‚Äì79.0¬†years), with a gender distribution of 46.2% male and 53.8% female. Antigenemia reached 100% in fatal cases during the first week after admission. The sensitivity/specificity of antigenemia for diagnosis were 64.7%/73.0% at admission, 69.1%/100% in Week 1, and 66.3%/100% in Week 2. Additionally, the rates of antigenemia in asymptomatic patients were 27.3% upon admission and 22.0% in Week 1, respectively; however, no antigenemia was in samples collected in Week 2. Viral RNAemia was not detected in asymptomatic patients, but RNAemia viral loads were elevated in fatal cases. Kaplan‚ÄìMeier survival curves demonstrated a higher mortality rate when antigenemia concentrations were elevated in the follow-up samples (P=‚Äâ0.005). Our study provides a comprehensive analysis of the kinetics of viral N-protein antigenemia and RNAemia according to disease severity and clinical classification. Our findings suggest that highest concentrations of antigenemia in fatal cases occur in the first week after admission, indicating that early elevated antigenemia may serve as a marker of mortality risk.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">https://www.nature.com/articles/s41598-024-65489-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-21</strong> <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">
            ü§ñ SARS-CoV-2 virus levels in Ontario wastewater during pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-576', this)">
            More</button></h4>
            <div id="toggle-576" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 viral titer measurements in Ontario, Canada wastewaters throughout the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic in Canada, Ontario started checking water from sewers to see if people had coronavirus. They did this starting from January 2021 until March 2023 by looking at samples from different places across the province. These samples came from more than half of Ontario‚Äôs population and some parts of Canada. The team checked things like where these places were, how many people they served, what kind of water it was, and how much water flowed through.

They did this test 2 to 7 times a week at each place. This helps them understand if coronavirus is spreading in different areas. It's especially useful now that the Omicron variant is around and fewer people are getting sick enough to be tested in hospitals. The team saved all this information so they can use it for research and planning for future sicknesses.

The data from these sewer samples has been very helpful for keeping track of coronavirus cases across Ontario.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, the Province of Ontario, Canada, launched a wastewater surveillance program to monitor SARS-CoV-2, inspired by the early work and successful forecasts of COVID-19 waves in the city of Ottawa, Ontario. This manuscript presents a dataset from January 1, 2021, to March 31, 2023, with RT-qPCR results for SARS-CoV-2 genes and PMMoV from 107 sites across all 34 public health units in Ontario, covering 72% of the province‚Äôs and 26.2% of Canada‚Äôs population. Sampling occurred 2‚Äì7 times weekly, including geographical coordinates, serviced populations, physico-chemical water characteristics, and flowrates. In doing so, this manuscript ensures data availability and metadata preservation to support future research and epidemic preparedness through detailed analyses and modeling. The dataset has been crucial for public health in tracking disease locally, especially with the rise of the Omicron variant and the decline in clinical testing, highlighting wastewater-based surveillance‚Äôs role in estimating disease incidence in Ontario.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">https://www.nature.com/articles/s41597-024-03414-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-12</strong> <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">
            ü§ñ COVID-19 Convalescents' Immune Response: Nonconserved Epitopes Dominant</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-577', this)">
            More</button></h4>
            <div id="toggle-577" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nonconserved epitopes dominate reverse preexisting T cell immunity in COVID-19 convalescents</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The abstract talks about how certain immune cells (T cells) in people who had COVID-19 can recognize other coronaviruses like SARS as well. We tested these T cells to see which parts of the viruses they could react with, and found that they mostly reacted to parts of the virus that don't change much between different strains. This means existing antibodies against one type of coronavirus might help protect against others. Our findings could help develop vaccines for multiple types of coronaviruses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The herd immunity against SARS-CoV-2 is continuously consolidated across the world during the ongoing pandemic. However, the potential function of the nonconserved epitopes in the reverse preexisting cross-reactivity induced by SARS-CoV-2 to other human coronaviruses is not well explored. In our research, we assessed T cell responses to both conserved and nonconserved peptides shared by SARS-CoV-2 and SARS-CoV, identifying cross-reactive CD8+T cell epitopes using enzyme-linked immunospot and intracellular cytokine staining assays. Then, in vitro refolding and circular dichroism were performed to evaluate the thermal stability of the HLA/peptide complexes. Lastly, single-cell T cell receptor reservoir was analyzed based on tetramer staining. Here, we discovered that cross-reactive T cells targeting SARS-CoV were present in individuals who had recovered from COVID-19, and identified SARS-CoV-2 CD8+T cell epitopes spanning the major structural antigens. T cell responses induced by the nonconserved peptides between SARS-CoV-2 and SARS-CoV were higher and played a dominant role in the cross-reactivity in COVID-19 convalescents. Cross-T cell reactivity was also observed within the identified series of CD8+T cell epitopes. For representative immunodominant peptide pairs, although the HLA binding capacities for peptides from SARS-CoV-2 and SARS-CoV were similar, the TCR repertoires recognizing these peptides were distinct. Our results could provide beneficial information for the development of peptide-based universal vaccines against coronaviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">https://www.nature.com/articles/s41392-024-01876-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-11</strong> <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">
            ü§ñ Antibodies and Immunity After COVID Infection Over Two Years</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-578', this)">
            More</button></h4>
            <div id="toggle-578" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Tracking the evolution of anti-SARS-CoV-2 antibodies and long-term humoral immunity within 2¬†years after COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The virus that causes COVID-19 makes people very sick sometimes. The body's immune system produces special proteins called antibodies to fight the virus. Antibodies can either be made against parts of the virus that cover it, called spike proteins, or against another part inside the virus, called nucleocapsid.

Scientists wanted to understand how long these antibodies last after someone recovers from COVID-19. They looked at people who had recovered and checked their blood over a year-long period.

They found that antibodies against one part of the virus (nucleocapsid) stayed high for about 4 months, but then started going down a bit. Antibodies against another part of the virus (spike) kept increasing every few months until the end of the study.

So, some parts of the body's defense against COVID-19 last longer than others. The scientists learned that certain antibodies stay strong for several months after someone recovers, while other antibodies keep growing stronger over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that gave rise to COVID-19 infection produced a worldwide health crisis. The virus can cause a serious or even fatal disease. Comprehending the complex immunological responses triggered by SARS-CoV-2 infection is essential for identifying pivotal elements that shape the course of the disease and its enduring effects on immunity. The span and potency of antibody responses provide valuable perspicuity into the resilience of post-infection immunity. The analysis of existing literature reveals a diverse controversy, confining varying data about the persistence of particular antibodies as well as the multifaceted factors that impact their development and titer, Within this study we aimed to understand the dynamics of anti-SARS-CoV-2 antibodies against nucleocapsid (anti-SARS-CoV-2 (N)) and spike (anti-SARS-CoV-2 (N)) proteins in long-term immunity in convalescent patients, as well as the factors influencing the production and kinetics of those antibodies. We collected 6115 serum samples from 1611 convalescent patients at different post-infection intervals up to 21¬†months Study showed that in the fourth month, the anti-SARS-CoV-2 (N) exhibited their peak mean value, demonstrating a 79% increase compared to the initial month. Over the subsequent eight months, the peak value experienced a modest decline, maintaining a relatively elevated level by the end of study. Conversely, anti-SARS-CoV-2 (S) exhibited a consistent increase at each three-month interval over the 15-month period, culminating in a statistically significant peak mean value at the study‚Äôs conclusion. Our findings demonstrate evidence of sustained seropositivity rates for both anti-SARS-CoV-2 (N) and (S), as well as distinct dynamics in the long-term antibody responses, with anti-SARS-CoV-2 (N) levels displaying remarkable persistence and anti-SARS-CoV-2 (S) antibodies exhibiting a progressive incline.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">https://www.nature.com/articles/s41598-024-64414-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-03</strong> <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">
            ü§ñ SARS-CoV-2Êäó‰ΩìIgGÂíåIgAÊ£ÄÊµã‰∏éËΩªÁóáÊñ∞ÂÜ†ÁóáÁä∂ÂÖ≥ËÅî</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-579', this)">
            More</button></h4>
            <div id="toggle-579" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Titers of IgG and IgA against SARS-CoV-2 proteins and their association with symptoms in mild COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how antibodies in people with mild COVID-19 disease change over time. Antibodies are like the body's protection against viruses. They can be made into two types: IgG and IgA. The research found that different levels of these antibodies could predict if someone would get worse symptoms from COVID-19, like trouble breathing.

They found that people with more severe symptoms had lower levels of certain antibodies. For example, people who couldn't breathe well had fewer antibodies against some parts of the virus compared to those who didn't have trouble breathing.

Overall, the study suggests that how strong your antibody response is can tell you if you might get sicker from COVID-19 and what part of the virus causes problems for different symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humoral immunity in COVID-19 includes antibodies (Abs) targeting spike (S) and nucleocapsid (N) SARS-CoV-2 proteins. Antibody levels are known to correlate with disease severity, but titers are poorly reported in mild or asymptomatic cases. Here, we analyzed the titers of IgA and IgG against SARS-CoV-2 proteins in samples from 200 unvaccinated Hospital Workers (HWs) with mild COVID-19 at two time points after infection. We analyzed the relationship between Ab titers and patient characteristics, clinical features, and evolution over time. Significant differences in IgG and IgA titers against N, S1 and S2 proteins were found when samples were segregated according to time T1 after infection, seroprevalence at T1, sex and age of HWs and symptoms at infection. We found that IgM‚Äâ+‚Äâsamples had higher titers of IgG against N antigen and IgA against S1 and S2 antigens than IgM‚Äâ‚àí‚Äâsamples. There were significant correlations between anti-S1 and S2 Abs. Interestingly, IgM‚Äâ+‚Äâpatients with dyspnea had lower titers of IgG and IgA against N, S1 and S2 than those without dyspnea. Comparing T1 and T2, we found that IgA against N, S1 and S2 but only IgG against certain Ag decreased significantly. In conclusion, an association was established between Ab titers and the development of infection symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">https://www.nature.com/articles/s41598-024-59634-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-30</strong> <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">
            ü§ñ Post-COVID Recovery After 3 Years</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-580', this)">
            More</button></h4>
            <div id="toggle-580" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Three-year outcomes of post-acute sequelae of COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The virus that causes COVID-19 can cause long-term effects even after you get better from it. Researchers studied 135,000 people who had COVID and compared them to 5 million others over a 3-year period. They found that while the risk of dying went down for people who didn't need hospital care, those who did still had higher chances of death in the third year after getting sick. 

For hospitalized patients, their long-term risks were even more severe: they still had twice the chance of death compared to healthy people, and there was a lot of health loss that continued into the third year.

Overall, most risks went down over time, but those who were hospitalized had high risks in the third year which led to significant health problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection causes post-acute sequelae of coronavirus disease 2019 (COVID-19) (PASC) in many organ systems. Risks of these sequelae have been characterized up to 2‚Äâyears after infection, but longer-term follow-up is limited. Here we built a cohort of 135,161 people with SARS-CoV-2 infection and 5,206,835 controls from the US Department of Veterans Affairs who were followed for 3‚Äâyears to estimate risks of death and PASC. Among non-hospitalized individuals, the increased risk of death was no longer present after the first year of infection, and risk of incident PASC declined over the 3‚Äâyears but still contributed 9.6 (95% confidence interval (CI): 0.4‚Äì18.7) disability-adjusted life years (DALYs) per 1,000 persons in the third year. Among hospitalized individuals, risk of death declined but remained significantly elevated in the third year after infection (incidence rate ratio: 1.29 (95% CI: 1.19‚Äì1.40)). Risk of incident PASC declined over the 3‚Äâyears, but substantial residual risk remained in the third year, leading to 90.0 (95% CI: 55.2‚Äì124.8) DALYs per 1,000 persons. Altogether, our findings show reduction of risks over time, but the burden of mortality and health loss remains in the third year among hospitalized individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">https://www.nature.com/articles/s41591-024-02987-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-29</strong> <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">
            ü§ñ Initial COVID-19 severity linked to SARS-CoV-2 T cell memory impacts future infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-581', this)">
            More</button></h4>
            <div id="toggle-581" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Initial COVID-19 severity influenced by SARS-CoV-2-specific T cells imprints T-cell memory and inversely affects reinfection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This research looked at how a person's immune system responds to COVID-19 over time. They studied 165 people who had different levels of sickness from the disease. 

They found that when people get better quickly, their helper and fighter cells (T cells) that fight the virus come early and are strong. This helps them feel better faster in the hospital.

But if T cell responses are weak and take longer to show up, it means someone might end up with a more serious illness later on.

They also noticed how antibodies (which are like shields against viruses) form. If they start forming late or not very much, that's linked to having a worse case of COVID-19.

Overall, the researchers found out that people who had really bad cases of COVID-19 in the beginning tend to have better protection against getting it again later. 

This helps us understand how our immune system fights off COVID-19 and may help prevent more waves of infections in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The immunoprotective components control COVID-19 disease severity, as well as long-term adaptive immunity maintenance and subsequent reinfection risk discrepancies across initial COVID-19 severity, remain unclarified. Here, we longitudinally analyzed SARS-CoV-2-specific immune effectors during the acute infection and convalescent phases of 165 patients with COVID-19 categorized by severity. We found that early and robust SARS-CoV-2-specific CD4+and CD8+T cell responses ameliorate disease progression and shortened hospital stay, while delayed and attenuated virus-specific CD8+T cell responses are prominent severe COVID-19 features. Delayed antiviral antibody generation rather than titer level associates with severe outcomes. Conversely, initial COVID-19 severity imprints the long-term maintenance of SARS-CoV-2-specific adaptive immunity, demonstrating that severe convalescents exhibited more sustained virus-specific antibodies and memory T cell responses compared to mild/moderate counterparts. Moreover, initial COVID-19 severity inversely correlates with SARS-CoV-2 reinfection risk. Overall, our study unravels the complicated interaction between temporal characteristics of virus-specific T cell responses and COVID-19 severity to guide future SARS-CoV-2 wave management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">https://www.nature.com/articles/s41392-024-01867-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-27</strong> <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">
            ü§ñ Respiratory issues after COVID infections in South Korea and Japan</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-582', this)">
            More</button></h4>
            <div id="toggle-582" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Acute and post-acute respiratory complications of SARS-CoV-2 infection: population-based cohort study in South Korea and Japan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how people who get sick from the coronavirus (SARS-CoV-2) can have breathing problems later on. They studied two groups of people: one from South Korea and another from Japan. The researchers wanted to see if these people had more trouble breathing after being infected, compared to others who didn't get infected.

They found that people who got SARS-CoV-2 were more likely to have breathing issues in the short term (acute respiratory complications) or long-term (post-acute respiratory sequelae) compared to those who didn‚Äôt get it. The risk was higher for people with more severe cases of the infection.

The study also showed that getting vaccinated against COVID-19 could help reduce these problems. Over time, even though some breathing issues went away, they still existed up to 6 months after someone got infected.

Overall, this research helps understand how SARS-CoV-2 can affect people's breathing over time and considers things like vaccination status in making this understanding.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Considering the significant burden of post-acute COVID-19 conditions among patients infected with SARS-CoV-2, we aimed to identify the risk of acute respiratory complications or post-acute respiratory sequelae. A binational population-based cohort study was conducted to analyze the risk of acute respiratory complications or post-acute respiratory sequelae after SARS-CoV-2 infection. We used a Korean nationwide claim-based cohort (K-COV-N;n=‚Äâ2,312,748; main cohort) and a Japanese claim-based cohort (JMDC;n=‚Äâ3,115,606; replication cohort) after multi-to-one propensity score matching. Among 2,312,748 Korean participants (mean age, 47.2‚Äâyears [SD, 15.6]; 1,109,708 [48.0%] female), 17.1% (394,598/2,312,748) were infected with SARS-CoV-2. The risk of acute respiratory complications or post-acute respiratory sequelae is significantly increased in people with SARS-CoV-2 infection compared to the general population (acute respiratory complications: HR, 8.06 [95% CI, 6.92-9.38]; post-acute respiratory sequelae: 1.68 [1.62-1.75]), and the risk increased with increasing COVID-19 severity. We identified COVID-19 vaccination as an attenuating factor, showing a protective association against acute or post-acute respiratory conditions. Furthermore, while the excess post-acute risk diminished with time following SARS-CoV-2 infection, it persisted beyond 6‚Äâmonths post-infection. The replication cohort showed a similar pattern in the association. Our study comprehensively evaluates respiratory complications in post-COVID-19 conditions, considering attenuating factors such as vaccination status, post-infection duration, COVID-19 severity, and specific respiratory conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">https://www.nature.com/articles/s41467-024-48825-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-22</strong> <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">
            ü§ñ Post-vax symptoms after COVID-19 vaccination: study compares infected vs vaccinated people</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-583', this)">
            More</button></h4>
            <div id="toggle-583" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID conditions following COVID-19 vaccination: a retrospective matched cohort study of patients with SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at whether getting a COVID-19 vaccine affected people who already had the virus compared to those who didn't have it. They found that vaccinated people were less likely to get certain health problems after having COVID, especially if they got vaccines from outside doctors. The biggest difference was in skin and blood issues. Even though some small differences still existed, getting a vaccine seemed to help reduce these health problems for most people. This means getting the shot might make it easier to avoid long-term side effects of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 vaccinations protect against severe illness and death, but associations with post-COVID conditions (PCC) are less clear. We aimed to evaluate the association between prior COVID-19 vaccination and new-onset PCC among individuals with SARS-CoV-2 infection across eight large healthcare systems in the United States. This retrospective matched cohort study used electronic health records (EHR) from patients with SARS-CoV-2 positive tests during March 2021-February 2022. Vaccinated and unvaccinated COVID-19 cases were matched on location, test date, severity of acute infection, age, and sex. Vaccination status was ascertained using EHR and integrated data on externally administered vaccines. Adjusted relative risks (RRs) were obtained from Poisson regression. PCC was defined as a new diagnosis in one of 13 PCC categories 30 days to 6 months following a positive SARS-CoV-2 test. The study included 161,531 vaccinated COVID-19 cases and 161,531 matched unvaccinated cases. Compared to unvaccinated cases, vaccinated cases had a similar or lower risk of all PCC categories except mental health disorders (RR: 1.06, 95% CI: 1.02‚Äì1.10). Vaccination was associated with ‚â•10% lower risk of sensory (RR: 0.90, 0.86‚Äì0.95), circulatory (RR: 0.88, 0.83‚Äì0.94), blood and hematologic (RR: 0.79, 0.71‚Äì0.89), skin and subcutaneous (RR: 0.69, 0.66‚Äì0.72), and non-specific COVID-19 related disorders (RR: 0.53, 0.51‚Äì0.56). In general, associations were stronger at younger ages but mostly persisted regardless of SARS-CoV-2 variant period, receipt of ‚â•3 vs. 1‚Äì2 vaccine doses, or time since vaccination. Pre-infection vaccination was associated with reduced risk of several PCC outcomes and hence may decrease the long-term consequences of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">https://www.nature.com/articles/s41467-024-48022-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-16</strong> <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">
            ü§ñ Human Mobility's Impact on City Virus Spread Before and After COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-584', this)">
            More</button></h4>
            <div id="toggle-584" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impacts of human mobility on the citywide transmission dynamics of 18 respiratory viruses in pre- and post-COVID-19 pandemic years</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Studies looked at how people move around using their phones to see how diseases spread. Before COVID-19, when people visited schools or went to play with kids in their city, they were more likely to get sick from common cold-like viruses that usually stay around all year. After everyone stayed home during the pandemic, these common cold viruses stopped spreading as much.

But after restrictions were lifted a bit, where people moved around also affected how SARS-CoV-2 spread. However, this connection became weaker when people started moving around again after new rules went back into place. 

Overall, changes in how people move seem to matter more for spreading common cold viruses compared to the coronavirus, especially at the start of a disease outbreak.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Many studies have used mobile device location data to model SARS-CoV-2 dynamics, yet relationships between mobility behavior and endemic respiratory pathogens are less understood. We studied the effects of population mobility on the transmission of 17 endemic viruses and SARS-CoV-2 in Seattle over a 4-year period, 2018-2022. Before 2020, visits to schools and daycares, within-city mixing, and visitor inflow preceded or coincided with seasonal outbreaks of endemic viruses. Pathogen circulation dropped substantially after the initiation of COVID-19 stay-at-home orders in March 2020. During this period, mobility was a positive, leading indicator of transmission of all endemic viruses and lagging and negatively correlated with SARS-CoV-2 activity. Mobility was briefly predictive of SARS-CoV-2 transmission when restrictions relaxed but associations weakened in subsequent waves. The rebound of endemic viruses was heterogeneously timed but exhibited stronger, longer-lasting relationships with mobility than SARS-CoV-2. Overall, mobility is most predictive of respiratory virus transmission during periods of dramatic behavioral change and at the beginning of epidemic waves.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">https://www.nature.com/articles/s41467-024-48528-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-29</strong> <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">
            ü§ñ SARS-CoV-2 Mutations in Lung Transplant Patient with Long COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-585', this)">
            More</button></h4>
            <div id="toggle-585" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamic diversity of SARS-CoV-2 genetic mutations in a lung transplantation patient with persistent COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Different types of COVID-19 viruses kept appearing since the pandemic started. Remdesivir is a medicine used to treat these viruses. But it doesn't work well for people whose immune systems are weak, and they can keep getting sick for a long time.

A patient who had trouble breathing because of lung damage caused by another sickness got COVID-19 from the Omicron BA.5 type virus. The virus kept spreading in their body even after treatment with Remdesivir. Scientists found that some changes (mutations) in the virus made it resistant to Remdesivir.

One particular mutation, C799F, was very important because it changed a part of the virus that helps the medicine work. This means the medicine couldn't stop the virus anymore.

The virus kept changing, with lots of different mutations appearing over time. Because of this, some people who are treated don‚Äôt get better and have to deal with resistant viruses that keep them sick for longer.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Numerous SARS-CoV-2 variant strains with altered characteristics have emerged since the onset of the COVID-19 pandemic. Remdesivir (RDV), a ribonucleotide analogue inhibitor of viral RNA polymerase, has become a valuable therapeutic agent. However, immunosuppressed hosts may respond inadequately to RDV and develop chronic persistent infections. A patient with respiratory failure caused by interstitial pneumonia, who had undergone transplantation of the left lung, developed COVID-19 caused by Omicron BA.5 strain with persistent chronic viral shedding, showing viral fusogenicity. Genome-wide sequencing analyses revealed the occurrence of several viral mutations after RDV treatment, followed by dynamic changes in the viral populations. The C799F mutation in nsp12 was found to play a pivotal role in conferring RDV resistance, preventing RDV-triphosphate from entering the active site of RNA-dependent RNA polymerase. The occurrence of diverse mutations is a characteristic of SARS-CoV-2, which mutates frequently. Herein, we describe the clinical case of an immunosuppressed host in whom inadequate treatment resulted in highly diverse SARS-CoV-2 mutations that threatened the patient‚Äôs health due to the development of drug-resistant variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">https://www.nature.com/articles/s41467-024-47941-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-17</strong> <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">
            ü§ñ Long-term recovery from Long COVID: Immune System Restoration 2 Years Later</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-586', this)">
            More</button></h4>
            <div id="toggle-586" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improvement of immune dysregulation in individuals with long COVID at 24-months following SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looks at how people feel immune-wise after having a mild to moderate case of long COVID compared to those who recovered from regular COVID. At 3 months, people with long COVID had higher antibodies against the virus's shell and core than those who fully recovered. They also had more special cells in their body that help fight off the virus for longer periods at 3 and 8 months.

By 24 months, these differences didn't last. Some people still had some signs of fighting off the infection, which might mean they were re-infected. But overall, both groups ended up with similar immune responses by then.

The study found that people's quality of life improved over time in most cases of long COVID, especially if their levels of a protein called PTX3 or low levels of certain blood markers went down.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigates the humoral and cellular immune responses and health-related quality of life measures in individuals with mild to moderate long COVID (LC) compared to age and gender matched recovered COVID-19 controls (MC) over 24 months. LC participants show elevated nucleocapsid IgG levels at 3 months, and higher neutralizing capacity up to 8 months post-infection. Increased spike-specific and nucleocapsid-specific CD4+T cells, PD-1, and TIM-3 expression on CD4+and CD8+T cells were observed at 3 and 8 months, but these differences do not persist at 24 months. Some LC participants had detectable IFN-Œ≥ and IFN-Œ≤, that was attributed to reinfection and antigen re-exposure. Single-cell RNA sequencing at the 24 month timepoint shows similar immune cell proportions and reconstitution of na√Øve T and B cell subsets in LC and MC. No significant differences in exhaustion scores or antigen-specific T cell clones are observed. These findings suggest resolution of immune activation in LC and return to comparable immune responses between LC and MC over time. Improvement in self-reported health-related quality of life at 24 months was also evident in the majority of LC (62%). PTX3, CRP levels and platelet count are associated with improvements in health-related quality of life.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">https://www.nature.com/articles/s41467-024-47720-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-09</strong> <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">
            ü§ñ SARS-CoV-2 immunity post-bivalent BA.4/5 vaccine for infected vs. uninfected people</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-587', this)">
            More</button></h4>
            <div id="toggle-587" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2-specific cellular and humoral immunity after bivalent BA.4/5 COVID-19-vaccination in previously infected and non-infected individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how people's bodies respond after getting a special vaccine called bivalent BA.4/5. Some people had already gotten sick with COVID-19 before. The researchers found that those who hadn't been sick before had bigger reactions to the vaccine, making more antibodies and having stronger protection against different types of coronavirus. However, even those who had been sick still had a good response to the vaccine.

The study also showed that people who didn't get sick before were more likely to get infected again after vaccination. This means they had lower levels of protection from their immune system against some specific parts of the virus.

In summary, getting vaccinated can still help protect people from COVID-19 even if they've been sick with it before, but those who haven't been sick might not have as strong a response to certain types of viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Knowledge is limited as to how prior SARS-CoV-2 infection influences cellular and humoral immunity after booster-vaccination with bivalent BA.4/5-adapted mRNA-vaccines, and whether vaccine-induced immunity may indicate subsequent infection. In this observational study, individuals with prior infection (n=‚Äâ64) showed higher vaccine-induced anti-spike IgG-antibodies and neutralizing titers, but the relative increase was significantly higher in non-infected individuals (n=‚Äâ63). In general, both groups showed higher neutralizing activity towards the parental strain than towards Omicron-subvariants BA.1, BA.2 and BA.5. In contrast, CD4 or CD8 T cell levels towards spike from the parental strain and the Omicron-subvariants, and cytokine expression profiles were similar irrespective of prior infection. Breakthrough infections occurred more frequently among previously non-infected individuals, who had significantly lower vaccine-induced spike-specific neutralizing activity and CD4 T cell levels. In summary, we show that immunogenicity after BA.4/5-bivalent vaccination differs between individuals with and without prior infection. Moreover, our results may help to improve prediction of breakthrough infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">https://www.nature.com/articles/s41467-024-47429-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-02</strong> <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">
            ü§ñ Allergic Conditions After COVID-19: Studies From 3 Countries</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-588', this)">
            More</button></h4>
            <div id="toggle-588" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Incident allergic diseases in post-COVID-19 condition: multinational cohort studies from South Korea, Japan and the UK</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that people who had COVID-19 were more likely to develop other health problems like asthma and hay fever later on. This risk increased even more if the COVID-19 was very serious. Getting vaccinated against COVID-19 helped lower this risk a bit. The results held true in different countries with different groups of people, which adds credibility to the findings.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As mounting evidence suggests a higher incidence of adverse consequences, such as disruption of the immune system, among patients with a history of COVID-19, we aimed to investigate post-COVID-19 conditions on a comprehensive set of allergic diseases including asthma, allergic rhinitis, atopic dermatitis, and food allergy. We used nationwide claims-based cohorts in South Korea (K-CoV-N;n=‚Äâ836,164; main cohort) and Japan (JMDC;n=‚Äâ2,541,021; replication cohort A) and the UK Biobank cohort (UKB;n=‚Äâ325,843; replication cohort B) after 1:5 propensity score matching. Among the 836,164 individuals in the main cohort (mean age, 50.25 years [SD, 13.86]; 372,914 [44.6%] women), 147,824 were infected with SARS-CoV-2 during the follow-up period (2020‚àí2021). The risk of developing allergic diseases, beyond the first 30 days of diagnosis of COVID-19, significantly increased (HR, 1.20; 95% CI, 1.13‚àí1.27), notably in asthma (HR, 2.25; 95% CI, 1.80‚àí2.83) and allergic rhinitis (HR, 1.23; 95% CI, 1.15‚àí1.32). This risk gradually decreased over time, but it persisted throughout the follow-up period (‚â•6 months). In addition, the risk increased with increasing severity of COVID-19. Notably, COVID-19 vaccination of at least two doses had a protective effect against subsequent allergic diseases (HR, 0.81; 95% CI, 0.68‚àí0.96). Similar findings were reported in the replication cohorts A and B. Although the potential for misclassification of pre-existing allergic conditions as incident diseases remains a limitation, ethnic diversity for evidence of incident allergic diseases in post-COVID-19 condition has been validated by utilizing multinational and independent population-based cohorts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">https://www.nature.com/articles/s41467-024-47176-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-28</strong> <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">
            ü§ñ Mpox Infection Trends in Nigeria: Review Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-589', this)">
            More</button></h4>
            <div id="toggle-589" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamics of Mpox infection in Nigeria: a systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Mpox outbreaks happen in parts of West and Central Africa. In Nigeria, many people got sick with it last year. Most cases were from southern Nigeria.

We looked at lots of studies about Mpox in Nigeria over almost 50 years. We found that Mpox spread more to northern parts of the country.

The main reasons why Mpox spreads are poverty, lack of healthcare, and unsafe sexual practices.

Our findings tell us we need to improve healthcare and stop unsafe sex to help stop Mpox spreading.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The seasonal outbreaks of Mpox continue in most parts of West and Central Africa. In the past year, Nigeria had the highest number of reported cases. Here, we used the PRISMA guidelines to carry out a systematic review and meta-analysis of available evidence on Mpox in Nigeria to assess the prevalence, transmission pattern, diagnostic approach, and other associated factors useful for mitigating the transmission of the disease. All relevant observational studies in PubMed/MEDLINE, Embase, AJOL, Web of Science, Scopus and Google Scholar on Mpox in Nigeria were assessed within the last fifty years (1972 to 2022). In all, 92 relevant articles were retrieved, out of which 23 were included in the final qualitative analysis. Notably, most of the cases of Mpox in Nigeria were from the southern part of the country. Our findings showed a progressive spread from the southern to the northern region of the country. We identified the following factors as important in the transmission of Mpox in Nigeria; poverty, lack of basic healthcare facilities, and risk of exposure through unsafe sexual practices. Our findings reiterate the need to strengthen and expand existing efforts as well as establish robust multi-sectoral collaboration to understand the dynamics of Mpox Nigeria.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">https://www.nature.com/articles/s41598-024-58147-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">
            ü§ñ Rapid double-shot vaccine against poxviruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-590', this)">
            More</button></h4>
            <div id="toggle-590" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rapid development of double-hit mRNA antibody cocktail against orthopoxviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This abstract talks about making special medicine to fight poxviruses, which can make people very sick. Right now, it's hard and expensive to create these medicines. But scientists found a new way using tiny messenger molecules called mRNA, put inside little fat particles called liposomes. They made different mixtures of these mRNA messages that tell the body how to make special antibodies that fight poxviruses. When they tested this on mice, the mice didn't get sick as badly when given these mRNA messages. One mixture of these messages helped protect against all types of poxviruses. The scientists think this could be a good way to create medicines for other viruses too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>TheOrthopoxvirusgenus, especially variola virus (VARV), monkeypox virus (MPXV), remains a significant public health threat worldwide. The development of therapeutic antibodies against orthopoxviruses is largely hampered by the high cost of antibody engineering and manufacturing processes. mRNA-encoded antibodies have emerged as a powerful and universal platform for rapid antibody production. Herein, by using the established lipid nanoparticle (LNP)-encapsulated mRNA platform, we constructed four mRNA combinations that encode monoclonal antibodies with broad neutralization activities against orthopoxviruses. In vivo characterization demonstrated that a single intravenous injection of each LNP-encapsulated mRNA antibody in mice resulted in the rapid production of neutralizing antibodies. More importantly, mRNA antibody treatments showed significant protection from weight loss and mortality in the vaccinia virus (VACV) lethal challenge mouse model, and a unique mRNA antibody cocktail, Mix2a, exhibited superior in vivo protection by targeting both intracellular mature virus (IMV)-form and extracellular enveloped virus (EEV)-form viruses. In summary, our results demonstrate the proof-of-concept production of orthopoxvirus antibodies via the LNP-mRNA platform, highlighting the great potential of tailored mRNA antibody combinations as a universal strategy to combat orthopoxvirus as well as other emerging viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">https://www.nature.com/articles/s41392-024-01766-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">
            ü§ñ Molecular Vaccine Effects & Fever: First Four Doses on Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-591', this)">
            More</button></h4>
            <div id="toggle-591" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Factors impacting antibody kinetics, including fever and vaccination intervals, in SARS-CoV-2-na√Øve adults receiving the first four mRNA COVID-19 vaccine doses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how a series of vaccine shots affects the body's ability to make antibodies against COVID-19 in people who have never had it before. They gave different groups of people either one, two, three, or four shots. Blood samples were taken from them both before and after each shot was given.

They found that:
- People who got the mRNA-1273 vaccine made more antibodies.
- Men made more antibodies than women.
- Younger people made more antibodies than older ones.
- People who didn't smoke had higher antibody levels.
- Those with no medicines to suppress their immune system also had higher levels.

They noticed that:
- The longer it was since they got their last shot, the more antibodies they made.
- Getting a second shot after an initial one always resulted in better results.

These findings can help make vaccination plans better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the antibody response following the initial four doses of mRNA vaccines (BNT162b2 or mRNA-1273) in SARS-CoV-2-na√Øve healthy adults and investigate factors influencing antibody titer increases, this prospective cohort study was conducted in Japan from March 2021. The study included participants who received either the 1st and 2nd doses (n‚Äâ=‚Äâ467), 3rd dose (n‚Äâ=‚Äâ157), or 4th dose (n‚Äâ=‚Äâ89). Blood samples were collected before and up to 6¬†months after each dose, and anti-receptor-binding domain antibody levels were measured. Multivariate analysis (usin multiple linear regression or linear mixed models) revealed several factors significantly associated with higher post-vaccination antibody levels, including mRNA-1273 vaccine (after the 1st and 2nd dose), male gender (after the 3rd and 4th doses), younger age (after the 1st and 2nd dose), non-smoking status (after the 2nd dose), non-use of immunosuppressive agents (after the 1st dose), higher pre-vaccination antibody titers (after the 2nd, 3rd, and 4th doses), and higher post-vaccination fever (after the 2nd and 4th doses). Furthermore, longer intervals since the last dose were significantly associated with higher antibody levels after the 3rd and 4th doses. These findings provide valuable insights for optimizing vaccination strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">https://www.nature.com/articles/s41598-024-57931-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-21</strong> <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">
            ü§ñ SARS-CoV-2 Infection Risk for Psychiatric Disorders & Medication Use</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-592', this)">
            More</button></h4>
            <div id="toggle-592" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term risk of psychiatric disorder and psychotropic prescription after SARS-CoV-2 infection among UK general population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Despite some people getting sick from COVID-19, they might later have problems like anxiety or mood issues. Studies showed that people who got infected had a higher chance of developing these mental health issues compared to those who didn't get infected. People who were hospitalized were even more likely to develop these problems.

Vaccines seemed to help protect against some of these mental health issues, though not completely. Getting the vaccine still meant there was a risk, just less than for people who weren‚Äôt vaccinated at all or only got part of the vaccine.

It's important to catch and treat mental health issues early in COVID survivors, especially those who were very sick or didn't get vaccinated. As more people get infected without getting vaccinated (called "breakthrough infections"), we need better ways to prevent serious mental health problems from developing.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite evidence indicating increased risk of psychiatric issues among COVID-19 survivors, questions persist about long-term mental health outcomes and the protective effect of vaccination. Using UK Biobank data, three cohorts were constructed: SARS-CoV-2 infection (n=‚Äâ26,101), contemporary control with no evidence of infection (n=‚Äâ380,337) and historical control predating the pandemic (n=‚Äâ390,621). Compared with contemporary controls, infected participants had higher subsequent risks of incident mental health at 1‚Äâyear (hazard ratio (HR): 1.54, 95% CI 1.42‚Äì1.67;P=‚Äâ1.70‚Äâ√ó‚Äâ10‚àí24; difference in incidence rate: 27.36, 95% CI 21.16‚Äì34.10 per 1,000 person-years), including psychotic, mood, anxiety, alcohol use and sleep disorders, and prescriptions for antipsychotics, antidepressants, benzodiazepines, mood stabilizers and opioids. Risks were higher for hospitalized individuals (2.17, 1.70‚Äì2.78;P=‚Äâ5.80‚Äâ√ó‚Äâ10‚àí10) than those not hospitalized (1.41, 1.30‚Äì1.53;P=‚Äâ1.46‚Äâ√ó‚Äâ10‚àí16), and were reduced in fully vaccinated people (0.97, 0.80‚Äì1.19;P=‚Äâ0.799) compared with non-vaccinated or partially vaccinated individuals (1.64, 1.49‚Äì1.79;P=‚Äâ4.95‚Äâ√ó‚Äâ10‚àí26). Breakthrough infections showed similar risk of psychiatric diagnosis (0.91, 0.78‚Äì1.07;P=‚Äâ0.278) but increased prescription risk (1.42, 1.00‚Äì2.02;P=‚Äâ0.053) compared with uninfected controls. Early identification and treatment of psychiatric disorders in COVID-19 survivors, especially those severely affected or unvaccinated, should be a priority in the management of long COVID. With the accumulation of breakthrough infections in the post-pandemic era, the findings highlight the need for continued optimization of strategies to foster resilience and prevent escalation of subclinical mental health symptoms to severe disorders.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">https://www.nature.com/articles/s41562-024-01853-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-18</strong> <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">
            ü§ñ Neurofilament levels high in mild COVID patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-593', this)">
            More</button></h4>
            <div id="toggle-593" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neurofilament light chain and glial fibrillary acid protein levels are elevated in post-mild COVID-19 or asymptomatic SARS-CoV-2 cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic caused brain problems in some people who had mild or no symptoms before. To see how well their brains were working now that the virus was gone, researchers looked at two substances in their blood: NF-L and GFAP. These substances can show if someone's brain cells are getting damaged.

They studied 147 grown-ups who used to have COVID-19 but didn't know they did or had very mild symptoms. They also checked some people without any illness nearby, like how the grown-up group is similar to them in age and body size. 

The researchers found that the grown-ups with brain problems had more of these substances in their blood compared to those who were healthy. Also, after checking again after about a year, even though they still had higher levels, it was less than what healthy people have.

This shows that sometimes, when the virus is gone, parts of someone's brain can be damaged, but over time, this damage gets better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Given the huge impact of the COVID-19 pandemic, it appears of paramount importance to assess the cognitive effects on the population returning to work after COVID-19 resolution. Serum levels of neurofilament light chain (sNfL) and glial fibrillary acidic protein (sGFAP) represent promising biomarkers of neuro-axonal damage and astrocytic activation. In this cohort study, we explored the association between sNfL and sGFAP concentrations and cognitive performance in a group of 147 adult workers with a previous asymptomatic SARS-CoV-2 infection or mild COVID-19, one week and, in 49 of them, ten months after SARS-Cov2 negativization and compared them to a group of 82 age and BMI-matched healthy controls (HCs). sNfL and sGFAP concentrations were assessed using SimoaTM assay Neurology 2-Plex B Kit. COVID-19 patients were interviewed one-on-one by trained physicians and had to complete a list of questionnaires, including the Cognitive Failure Questionnaire (CFQ). At the first assessment (T0), sNfL and sGFAP levels were significantly higher in COVID-19 patients than in HCs (p‚Äâ<‚Äâ0.001 for both). The eleven COVID-19 patients with cognitive impairment had significantly higher levels of sNfL and sGFAP than the others (p‚Äâ=‚Äâ0.005 for both). At the subsequent follow-up (T1), sNfL and sGFAP levels showed a significant decrease (median sNfL 18.3¬†pg/mL; median sGFAP 77.2¬†pg/mL), although they were still higher than HCs (median sNfL 7.2¬†pg/mL, median sGFAP 63.5¬†pg/mL). Our results suggest an ongoing damage involving neurons and astrocytes after SARS-Cov2 negativization, which reduce after ten months even if still evident compared to HCs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">https://www.nature.com/articles/s41598-024-57093-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-05</strong> <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">
            ü§ñ SARS-CoV-2 viability in air from COVID patients on different variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-594', this)">
            More</button></h4>
            <div id="toggle-594" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Frequency, kinetics and determinants of viable SARS-CoV-2 in bioaerosols from ambulatory COVID-19 patients infected with the Beta, Delta or Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The abstract talks about whether tiny bits of air carrying coronavirus (called aerosols) can still make people sick. Some researchers think these bits are dangerous, but others aren't sure. So we did tests to see if these aerosols could have the virus that causes COVID-19 and how many people had them.

We found out that for some types of test results from people who had the virus, about half or more of those people were likely spreading the virus in their air. People with lower protection against the virus in their bodies seemed to be better at making these spreadable bits of air.

The study also found a quick way to guess whether someone is likely to make these dangerous bits of air - just by looking at some parts of their blood. 

What this means is that we now know more about how and who spreads coronavirus through the air, which can help us stop it better in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Airborne transmission of SARS-CoV-2 aerosol remains contentious. Importantly, whether cough or breath-generated bioaerosols can harbor viable and replicating virus remains largely unclarified. We performed size-fractionated aerosol sampling (Andersen cascade impactor) and evaluated viral culturability in human cell lines (infectiousness), viral genetics, and host immunity in ambulatory participants with COVID-19. Sixty-one percent (27/44) and 50% (22/44) of participants emitted variant-specific culture-positive aerosols <10Œºm and <5Œºm, respectively, for up to 9 days after symptom onset. Aerosol culturability is significantly associated with lower neutralizing antibody titers, and suppression of transcriptomic pathways related to innate immunity and the humoral response. A nasopharyngeal Ct <17 rules-in ~40% of aerosol culture-positives and identifies those who are probably highly infectious. A parsimonious three transcript blood-based biosignature is highly predictive of infectious aerosol generation (PPV‚Äâ>‚Äâ95%). There is considerable heterogeneity in potential infectiousness i.e., only 29% of participants were probably highly infectious (produced culture-positive aerosols <5Œºm at ~6 days after symptom onset). These data, which comprehensively confirm variant-specific culturable SARS-CoV-2 in aerosol, inform the targeting of transmission-related interventions and public health containment strategies emphasizing improved ventilation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">https://www.nature.com/articles/s41467-024-45400-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-26</strong> <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">
            ü§ñ Long-term Health Effects After COVID-19 Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-595', this)">
            More</button></h4>
            <div id="toggle-595" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Persistence in risk and effect of COVID-19 vaccination on long-term health consequences after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at people who got infected with the coronavirus (SARS-Co-V-2) and compared them to those who didn't get infected, focusing on how vaccination affects their health over time in Hong Kong. They found that getting vaccinated or having a booster shot reduces the risk of long-term health problems like heart issues and death after being sick from COVID for about a year. Unvaccinated people had higher risks even months later. The study shows vaccines help prevent serious health effects from the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The persisting risk of long-term health consequences of SARS-CoV-2 infection and the protection against such risk conferred by COVID-19 vaccination remains unclear. Here we conducted a retrospective territory-wide cohort study on 1,175,277 patients with SARS-CoV-2 infection stratified by their vaccination status and non-infected controls to evaluate the risk of clinical sequelae, cardiovascular and all-cause mortality using a territory-wide public healthcare database with population-based vaccination records in Hong Kong. A progressive reduction in risk of all-cause mortality was observed over one year between patients with SARS-CoV-2 infection and controls. Patients with complete vaccination or have received booster dose incurred a lower risk of health consequences including major cardiovascular diseases, and all-cause mortality than unvaccinated or patients with incomplete vaccination 30-90 days after infection. Completely vaccinated and patients with booster dose of vaccines did not incur significant higher risk of health consequences from 271 and 91 days of infection onwards, respectively, whilst un-vaccinated and incompletely vaccinated patients continued to incur a greater risk of clinical sequelae for up to a year following SARS-CoV-2 infection. This study provided real-world evidence supporting the effectiveness of COVID-19 vaccines in reducing the risk of long-term health consequences of SARS-CoV-2 infection and its persistence following infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">https://www.nature.com/articles/s41467-024-45953-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-14</strong> <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">
            ü§ñ SARS-CoV-2 in corneas post-transplants vs. COVID-19 infection rates</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-596', this)">
            More</button></h4>
            <div id="toggle-596" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection status in corneal preservation solution and COVID-19 prevalence after corneal transplantation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that corneas from COVID-19-infected donors did not have the virus inside them. They tested 144 cornea transplants and found no signs of COVID-19 infection. All the patients who got new corneas were healthy after surgery, even though they had to take steroid medicine for a long time. The research suggests testing for COVID-19 in donors' throats before giving them corneas can help prevent infections in people who get new eyes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The potential risks associated with organs from COVID-19-infected donors were unclear. To determine the SARS-CoV-2 infection status of corneas transplanted during the COVID-19 pandemic, we performed a polymerase chain reaction (PCR) using the corneal preservation solution that was used for corneal transplantation. We also examined the postoperative health status of the recipients. This study included 144 transplants in 143 eyes. Ninety-nine eyes of imported corneas and 10 of the 14 corneas donated in the prefecture were PCR tested at our hospital, and all were SARS-CoV-2 negative. All corneal transplants were performed after confirming their SARS-CoV-2 negativity by a PCR using a corneal preservation solution at our hospital or a nasopharyngeal swab at a previous facility. Despite postoperative steroid administration, no patient developed COVID-19 infection until discharge. Hence, if the donor's nasopharyngeal swab test is SARS-CoV-2 negative, COVID-19 infection in the recipient due to corneal transplantation may be prevented. Since corneal transplant recipients are susceptible to infection due to prolonged steroid administration and are at high risk for severe diseases if infection occurs, SARS-CoV-2 detection testing using nasopharyngeal swabs in donors should be performed.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">https://www.nature.com/articles/s41598-024-53863-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-05</strong> <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">
            ü§ñ SARS-CoV-2 vaccine could reduce COVID-19 lingering gut issues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-597', this)">
            More</button></h4>
            <div id="toggle-597" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 vaccination may mitigate dysregulation of IL-1/IL-18 and gastrointestinal symptoms of the post-COVID-19 condition</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The vaccine helped prevent serious illness from the coronavirus. Some people who had the virus still get sick after recovery, called post-COVID conditions (PCC). Researchers looked at how well the vaccine worked for those with PCC and found that getting vaccinated might help reduce certain symptoms like stomach problems.

People who didn't have any of these issues before vaccination showed no signs of improvement. But people with PCC who got vaccinated had lower levels of two proteins in their blood, which might mean fewer stomach troubles.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid development of safe and effective vaccines helped to prevent severe disease courses after SARS-CoV-2 infection and to mitigate the progression of the COVID-19 pandemic. While there is evidence that vaccination may reduce the risk of developing post-COVID-19 conditions (PCC), this effect may depend on the viral variant. Therapeutic effects of post-infection vaccination have been discussed but the data for individuals with PCC remains inconclusive. In addition, extremely rare side effects after SARS-CoV-2 vaccination may resemble the heterogeneous PCC phenotype. Here, we analyze the plasma levels of 25 cytokines and SARS-CoV-2 directed antibodies in 540 individuals with or without PCC relative to one or two mRNA-based COVID-19 vaccinations as well as in 20 uninfected individuals one month after their initial mRNA-based COVID-19 vaccination. While none of the SARS-CoV-2 na√Øve individuals reported any persisting sequelae or exhibited PCC-like dysregulation of plasma cytokines, we detected lower levels of IL-1Œ≤ and IL-18 in patients with ongoing PCC who received one or two vaccinations at a median of six months after infection as compared to unvaccinated PCC patients. This reduction correlated with less frequent reporting of persisting gastrointestinal symptoms. These data suggest that post-infection vaccination in patients with PCC might be beneficial in a subgroup of individuals displaying gastrointestinal symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">https://www.nature.com/articles/s41541-024-00815-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-24</strong> <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">
            ü§ñ Respiratory issues in newborns not infected by coronavirus observed in study.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-598', this)">
            More</button></h4>
            <div id="toggle-598" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Respiratory distress in SARS-CoV-2 exposed uninfected neonates followed in the COVID Outcomes in Mother-Infant Pairs (COMP) Study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that babies exposed in the womb to someone sick with coronavirus had more breathing problems, especially if their mom wasn't vaccinated. The researchers looked at moms who got infected while pregnant and compared them to moms who were vaccinated before getting sick. They also checked how the baby's body reacted to the virus. Babies without a vaccine from their mom had twice as much trouble breathing than babies whose moms got vaccinated first. This was because of problems with tiny hair-like things in the lungs that help move air, and the baby making more antibodies (IgE) which can sometimes cause inflammation. Vaccinating moms before they get infected seems to lower how often this happens in their babies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory distress (RD) has been reported in SARS-CoV-2 exposed uninfected (SEU) term neonates. Prior studies suggest that prenatal exposure to Coronavirus Disease 19 (COVID-19) may activate an inflammatory cascade in the newborn airway. In this study, we examine the relationship between maternal COVID-19 vaccination and neonatal RD using a longitudinal cohort of mother-infant pairs in Los Angeles, CA. Two-hundred and twenty-one mothers with laboratory confirmed SARS-CoV-2 during pregnancy and 227 exposed fetuses are enrolled in our study. Maternal disease severity and neonatal RD variables were defined based on current accepted clinical criteria. To explore the multifactorial associations between maternal COVID-19 parameters and infant RD, we utilize a multivariable logistic regression model and a proteomic sub-analysis to propose a pathway for the development of RD followingin uteroexposure to SARS-CoV-2. Unusually high rates of RD are observed in SEU infants (17%). The odds ratio of RD is 3.06 (95% CI:1.08-10.21) in term neonates born to unvaccinated individuals versus those born to individuals vaccinated prior to maternal infection. Proteomic analysis reveals a robust inflammatory response associated with ciliary dysregulation and enhanced IgE production among SEU neonates with RD. Maternal vaccination against COVID-19 reduces the frequency of neonatal RD.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">https://www.nature.com/articles/s41467-023-44549-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-08</strong> <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">
            ü§ñ Heritability of SARS-CoV-2 Susceptibility and Severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-599', this)">
            More</button></h4>
            <div id="toggle-599" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Estimating the heritability of SARS-CoV-2 susceptibility and COVID-19 severity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 has infected over 340 million people. Scientists want to find ways to treat COVID-19. Genetic studies can show where drugs might work better. But we also need to understand if some people are more likely to get sick or have worse symptoms because of their genes.

Research shows that genetics explain about 1.2% and 5.8% of why some people are more susceptible (can catch it easily) or have a worse condition when they do get sick. Some research used families' genetic information to look at these numbers better. They found that for susceptibility, the genes explained between 33% and 70% of cases. For how long someone is hospitalized with COVID-19, genetics accounted for about 41%.

Scientists think both genes and things in their environment (like if they had a clean place to live) help cause these differences. But what people inherited from their parents seems to matter more when it comes to getting sick.

During the pandemic, as vaccines came out, how much of why some people got sick was due to genetics changed. It's hard for scientists to understand all this during such big changes in health habits and medical care.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 has infected over 340 million people, prompting therapeutic research. While genetic studies can highlight potential drug targets, understanding the heritability of SARS-CoV-2 susceptibility and COVID-19 severity can contextualize their results. To date, loci from meta-analyses explain 1.2% and 5.8%¬†of variation in susceptibility and severity¬†respectively. Here we estimate the importance of shared environment and additive genetic variation to SARS-CoV-2 susceptibility and COVID-19 severity using pedigree data, PCR results, and hospitalization information. The relative importance of genetics and shared environment for susceptibility shifted during the study, with heritability ranging from 33% (95% CI: 20%-46%) to 70% (95% CI: 63%-74%). Heritability was greater for days hospitalized with COVID-19 (41%, 95% CI: 33%-57%) compared to shared environment (33%, 95% CI: 24%-38%). While our estimates suggest these genetic architectures are not fully understood, the shift in susceptibility estimates highlights the challenge of estimation during a pandemic, given environmental fluctuations and vaccine introduction.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">https://www.nature.com/articles/s41467-023-44250-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-03</strong> <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">
            ü§ñ COVID recovery symptoms and their causes: IMPACC study findings</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-600', this)">
            More</button></h4>
            <div id="toggle-600" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Features of acute COVID-19 associated with post-acute sequelae of SARS-CoV-2 phenotypes: results from the IMPACC study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID-19 syndrome is a big health issue. We studied 590 people who got sick from COVID-19 after they left the hospital, tracking their recovery for a year.

We found four groups based on how badly different parts of their bodies were affected:
1. Minimal: Not much hurt.
2. Physical: Lots of trouble moving around and feeling tired.
3. Mental/Cognitive: Problems with thinking and memory.
4. Multidomain: A mix of all the above problems.

Women got more of these problems, and some people had other health issues too. 

When people were first sick:
- If they had lots of the virus in their lungs,
- If they didn't have good antibodies to fight it,
- Or if they had fewer special immune cells called B lymphocytes,

They often ended up in one of those groups.

People who had problems with thinking and memory had higher levels of a chemical called FGF21 in their blood. Those with physical issues did too, but also had higher levels of FGF21 overall.

This information could help doctors figure out better ways to treat or prevent Post-COVID-19 syndrome in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sequelae of SARS-CoV-2 (PASC) is a significant public health concern. We describe Patient Reported Outcomes (PROs) on 590 participants prospectively assessed from hospital admission for COVID-19 through one year after discharge. Modeling identified 4 PRO clusters based on reported deficits (minimal, physical, mental/cognitive, and multidomain), supporting heterogenous clinical presentations in PASC, with sub-phenotypes associated with female sex and distinctive comorbidities. During the acute phase of disease, a higher respiratory SARS-CoV-2 viral burden and lower Receptor Binding Domain and Spike antibody titers were associated with both the physical predominant and the multidomain deficit clusters. A lower frequency of circulating B lymphocytes by mass cytometry (CyTOF) was observed in the multidomain deficit cluster. Circulating fibroblast growth factor 21 (FGF21) was significantly elevated in the mental/cognitive predominant and the multidomain clusters. Future efforts to link PASC to acute anti-viral host responses may help to better target treatment and prevention of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">https://www.nature.com/articles/s41467-023-44090-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-02</strong> <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">
            ü§ñ Anti-viral hosts for SARS-CoV-2 identified through multi-omics analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-601', this)">
            More</button></h4>
            <div id="toggle-601" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Integrated multi-omics analyses identify anti-viral host factors and pathways controlling SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study found many previously unknown ways that a person's body can fight off the coronavirus (SARS-CoV-2). They used experiments with CRISPR, which is like a gene scissors, to look at what parts of our bodies are important for fighting viruses. They looked at how different cells work together and interact with viruses.

They discovered several new parts of our body that help stop the virus from entering or growing inside us. These include parts of the V-ATPases, ESCRT, N-glycosylation pathways, which are like different tools in a toolbox for fighting off viruses.

They also found out about an important group called cohesin complex and another part called KLF5 that helps fight the virus. The KLF5 is special because it's related to something we eat - things called sphingolipids.

The study showed how removing these parts can make cells more vulnerable to getting sick from COVID-19, especially in severe cases. This research could help find new ways to protect people from getting worse with the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Host anti-viral factors are essential for controlling SARS-CoV-2 infection but remain largely unknown due to the biases of previous large-scale studies toward pro-viral host factors. To fill in this knowledge gap, we perform a genome-wide CRISPR dropout screen and integrate analyses of the multi-omics data of the CRISPR screen, genome-wide association studies, single-cell RNA-Seq, and host-virus proteins or protein/RNA interactome. This study uncovers many host factors that are currently underappreciated, including the components of V-ATPases, ESCRT, and N-glycosylation pathways that modulate viral entry and/or replication. The cohesin complex is also identified as an anti-viral pathway, suggesting an important role of three-dimensional chromatin organization in mediating host-viral interaction. Furthermore, we discover another anti-viral regulator KLF5, a transcriptional factor involved in sphingolipid metabolism, which is up-regulated, and harbors genetic variations linked to COVID-19 patients with severe symptoms. Anti-viral effects of three identified candidates (DAZAP2/VTA1/KLF5) are confirmed individually. Molecular characterization of DAZAP2/VTA1/KLF5-knockout cells highlights the involvement of genes related to the coagulation system in determining the severity of COVID-19. Together, our results provide further resources for understanding the host anti-viral network during SARS-CoV-2 infection and may help develop new countermeasure strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">https://www.nature.com/articles/s41467-023-44175-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">
            ü§ñ SARS-CoV-2 Variants Neutralized in Children After VaccinesÊàñInfections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-602', this)">
            More</button></h4>
            <div id="toggle-602" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralization of SARS-CoV-2 Omicron BQ.1, BQ.1.1 and XBB.1 variants following SARS-CoV-2 infection or vaccination in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how well different vaccines work against new coronavirus strains in children. They found that after getting infected or vaccinated, kids don't make strong enough antibodies to fight off some of the new Omicron subtypes like BQ.1, BQ.1.1, and XBB.1. This means current vaccines might not be as effective against these variants. The research suggests we need better vaccines that can protect children from these new strains more effectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Emergence of highly transmissible Omicron subvariants led to increased SARS-CoV-2 infection and disease in children. However, minimal knowledge exists regarding the neutralization capacity against circulating Omicron BA.4/BA.5, BA.2.75, BQ.1, BQ.1.1 and XBB.1 subvariants following SARS-CoV-2 vaccination in children versus during acute or convalescent COVID-19, or versus multisystem inflammatory syndrome (MIS-C). Here, we evaluate virus-neutralizing capacity against SARS-CoV-2 variants in 151 age-stratified children (‚Äâ<5, 5‚Äì11, 12‚Äì21 years old) hospitalized with acute severe COVID-19 or MIS-C or convalescent mild (outpatient) infection compared with 62 age-stratified vaccinated children. An age-associated effect on neutralizing antibodies is observed against SARS-CoV-2 following acute COVID-19 or vaccination. The primary series BNT162b2 mRNA vaccinated adolescents show higher vaccine-homologous WA-1 neutralizing titers compared with <12 years vaccinated children. Post-infection antibodies did not neutralize BQ.1, BQ.1.1 and XBB.1 subvariants. In contrast, monovalent mRNA vaccination induces more cross-neutralizing antibodies in young children <5 years against BQ.1, BQ.1.1 and XBB.1 variants compared with ‚â•5 years old children. Our study demonstrates that in children, infection and monovalent vaccination-induced neutralization activity is low against BQ.1, BQ.1.1 and XBB.1 variants. These findings suggest a need for improved SARS-CoV-2 vaccines to induce durable, more cross-reactive neutralizing antibodies to provide effective protection against emerging variants in children.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">https://www.nature.com/articles/s41467-023-43152-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">
            ü§ñ SARS-CoV-2 in breath can spread COVID-19 quickly</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-603', this)">
            More</button></h4>
            <div id="toggle-603" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Infectivity of exhaled SARS-CoV-2 aerosols is sufficient to transmit covid-19 within minutes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that when people start feeling sick with COVID-19, tiny droplets of their breath (called aerosols) carry the virus. These droplets are released into the air and spread the disease very easily. The researchers measured how many of these droplets contain enough of the virus to make someone sick.

They looked at three different people and found that singing releases the most droplets with the virus inside them. One person coughed out 4 TCID50 per second, another one exhaled 36 per second, and the third person let out 127 per second. 

The researchers used a computer model to guess what would happen if someone who had just caught COVID-19 came into a room where people were walking around normally. They found that in about 6 minutes to less than an hour, someone could breathe in enough droplets carrying the virus to become sick themselves.

So basically, the study shows that when someone with COVID-19 is singing or breathing heavily, they can spread the disease very quickly and easily in a regular room where people are moving around.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Exhaled SARS-CoV-2-containing aerosols contributed significantly to the rapid and vast spread of covid-19. However, quantitative experimental data on the infectivity of such aerosols is missing. Here, we quantified emission rates of infectious viruses in exhaled aerosol from individuals within their first days after symptom onset from covid-19. Six aerosol samples from three individuals were culturable, of which five were successfully quantified using TCID50. The source strength of the three individuals was highest during singing, when they exhaled 4, 36, or 127 TCID50/s, respectively. Calculations with an indoor air transmission model showed that if an infected individual with this emission rate entered a room, a susceptible person would inhale an infectious dose within 6 to 37¬†min in a room with normal ventilation. Thus, our data show that exhaled aerosols from a single person can transmit covid-19 to others within minutes at normal indoor conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">https://www.nature.com/articles/s41598-023-47829-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-11-17</strong> <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">
            ü§ñ Post-Acute COVID Symptoms in Multinational Cohort</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-604', this)">
            More</button></h4>
            <div id="toggle-604" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The burden of post-acute COVID-19 symptoms in a multinational network cohort analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at people who had gotten sick from COVID-19 and how many still felt bad for months afterward. They studied lots of people in Spain and the UK.

They found that as time went on, fewer people were feeling sick after their initial illness. But even those with only one case of COVID often had trouble with smell and taste for a long time.

People who got sick again (re-infected) had more lingering symptoms than those who got only one infection. 

The researchers think there might be different ways to define "long COVID" and they suggest people need more research to find better treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Persistent symptoms following the acute phase of COVID-19 present a major burden to both the affected and the wider community. We conducted a cohort study including over 856,840 first COVID-19 cases, 72,422 re-infections and more than 3.1 million first negative-test controls from primary care electronic health records from Spain and the UK (Sept 2020 to Jan 2022 (UK)/March 2022 (Spain)). We characterised post-acute COVID-19 symptoms and identified key symptoms associated with persistent disease. We estimated incidence rates of persisting symptoms in the general population and among COVID-19 patients over time. Subsequently, we investigated which WHO-listed symptoms were particularly differential by comparing their frequency in COVID-19 cases vs. matched test-negative controls. Lastly, we compared persistent symptoms after first infections vs. reinfections.Our study shows that the proportion of COVID-19 cases affected by persistent post-acute COVID-19 symptoms declined over the study period. Risk for altered smell/taste was consistently higher in patients with COVID-19 vs test-negative controls. Persistent symptoms were more common after reinfection than following a first infection. More research is needed into the definition of long COVID, and the effect of interventions to minimise the risk and impact of persistent symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">https://www.nature.com/articles/s41467-023-42726-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-07</strong> <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">
            ü§ñ Covid-19 and Post-Acute Sick Leave Study in Denmark</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-605', this)">
            More</button></h4>
            <div id="toggle-605" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A hybrid register and questionnaire study of Covid-19 and post-acute sick leave in Denmark</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-acute sick leave means how people feel sick after they recover from the flu-like illness caused by a virus called SARS-CoV-2. We did a survey where many people said if they had to stay home because they were very sick. The study found that more women, especially those in their 50s and 60s, as well as those with certain health problems like lung issues or being extra heavy (obesity), took longer to get back to feeling normal after the flu-like illness. These findings might help doctors make better ways to diagnose and treat people who still feel sick months later.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sick leave is an underexplored indicator of the societal burden of SARS-CoV-2. Here, ¬†we report findings about self-reported sick leave and risk factors thereof from a hybrid survey and register study, which include 37,482 RT-PCR confirmed SARS-CoV-2 cases and 51,336 test-negative controls who were tested during the index- and alpha-dominant waves. We observe that an additional 33 individuals per 1000 took substantial sick leave following acute infection compared to persons with no known history of infection, where substantial sick leave is defined as >1 month of sick leave within the period 1‚Äì9 months after the RT-PCR test date. Being female, 50‚Äì65 years, or having certain pre-existing health conditions such as obesity, chronic lung diseases, and fibromyalgia each increase risk for taking substantial sick leave. Altogether, these results may help motivate improved diagnostic and treatment options for persons living with post-Covid conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">https://www.nature.com/articles/s41467-023-42048-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-06</strong> <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">
            ü§ñ COVID-19 and SARS-CoV-2 risk for eye vein blockages.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-606', this)">
            More</button></h4>
            <div id="toggle-606" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Retinal vascular occlusion risks during the COVID-19 pandemic and after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how many people developed problems with their eye blood vessels, like clots, before and during a time when lots of people had coronavirus (COVID-19). They compared the number of these problems in two groups: those who tested positive for COVID-19 and those suspected to have it. The researchers found that having COVID-19 did not make people more likely to get blood vessel issues in their eyes, though they saw a small drop in one kind of problem (retinal vein occlusion) and an increase in another (retinal artery occlusion) during the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) has been reported to affect vascular networks including the eye. However, evidence on the causal relationship between COVID-19 infection and retinal vascular occlusions remains limited. This study aimed to determine the change in retinal vascular occlusion incidence during COVID-19 era and whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection induces retinal vascular occlusion. Retinal vein occlusion (RVO) and retinal artery occlusion (RAO) incidences during 2018‚Äì2019 and 2020‚ÄìJuly 2021 were compared, those in confirmed and suspected COVID-19 patients diagnosed from 2020 to January 2021 were calculated, and those in COVID-19 patients during 180¬†days prior and 180¬†days after diagnosis were assessed. Additionally, the standardized incidence ratio of RVOs in COVID-19 patients was analyzed. Incidence rates per 100,000 people/year of RVO during 2018‚Äì2019 and 2020‚Äì2021 was 102.0 and 98.8, respectively. RAO incidence rates during 2018‚Äì2019 and 2020‚Äì2021 were 11.7 and 12.0, respectively. In both confirmed and suspected COVID-19 patients, the incidence of RVO and RAO did not change significantly from 180¬†days before to after diagnosis in the adjusted model. RVO incidence slightly decreased while RAO incidence increased during the COVID-19 pandemic. SARS-CoV-2 infection did not significantly increase RVO or RAO incidence.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">https://www.nature.com/articles/s41598-023-44199-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-02</strong> <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">
            ü§ñ Monkeypox testing in Spain for risky groups</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-607', this)">
            More</button></h4>
            <div id="toggle-607" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Self-sampling monkeypox virus testing in high-risk populations, asymptomatic or with unrecognized Mpox, in Spain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A group studied how the monkeypox virus (MPXV) affects gay, bisexual men, trans women, and non-binary people who don't show symptoms or might have missed getting sick from MPXV. They took swabs from their bottom parts to test for the virus using a new method. Out of 113 participants, 7 had MPXV in their samples (about 6%). Most weren‚Äôt showing any signs they were infected but some showed no symptoms and still tested positive. The study found that not all people who have it show obvious sickness, which means checking only for known symptoms might miss some cases of the virus. This shows we need better ways to catch everyone who has MPXV.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recent monkeypox virus (MPXV) outbreak was of global concern and has mainly affected gay, bisexual and other men who have sex with men (GBMSM). Here we assess prevalence of MPXV in high-risk populations of GBMSM, trans women (TW) and non-binary people without symptoms or with unrecognized monkeypox (Mpox) symptoms, using a self-sampling strategy. Anal and pharyngeal swabs are tested by MPXV real-time PCR and positive samples are tested for cytopathic effect (CPE) in cell culture. 113 individuals participated in the study, 89 (78.76%) were cis men, 17 (15.04%) were TW. The median age was 35.0 years (IQR: 30.0‚Äì43.0), 96 (85.02%) individuals were gay or bisexual and 72 (63.72%) were migrants. Seven participants were MPXV positive (6.19% (95% CI: 1.75%‚Äì10.64%)). Five tested positive in pharyngeal swabs, one in anal swab and one in both. Six did not present symptoms recognized as MPXV infection. Three samples were positive for CPE, and showed anti-vaccinia pAb staining by FACS and confocal microscopy. This suggests that unrecognized Mpox cases can shed infectious virus. Restricting testing to individuals reporting Mpox symptoms may not be sufficient to contain outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">https://www.nature.com/articles/s41467-023-40490-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-29</strong> <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">
            ü§ñ Monovalent COVID vaccines impact viral interactions in long-COVID patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-608', this)">
            More</button></h4>
            <div id="toggle-608" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of monovalent COVID-19 vaccines on viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The Epstein-Barr virus and other viruses can cause symptoms after a person recovers from COVID-19. This study looked at how vaccines might help reduce these lingering symptoms.

Researchers checked if people with long-COVID had more or fewer of certain viruses like parvovirus and Herpesviridae. They also saw if getting vaccinated against COVID-19 helped lessen the effects of those other viruses in patients who still felt sick after recovering from a full-blown case of COVID-19.

They found that people who got vaccinated had fewer symptoms overall, less of these virus-related antibodies in their blood, and lower levels of certain types of antibodies related to Herpesviridae. This suggests that getting the COVID-19 vaccine might help reduce lingering symptoms by stopping some of these viruses from fighting each other inside the body.

So basically, vaccines could be helping prevent other viruses from causing more problems after someone gets better from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Epstein‚ÄìBarr virus (EBV) reactivation may be involved in long-COVID symptoms, but reactivation of other viruses as a factor has received less attention. Here we evaluated the reactivation of parvovirus-B19 and several members of the Herpesviridae family (DNA viruses) in patients with long-COVID syndrome. We hypothesized that monovalent COVID-19 vaccines inhibit viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome, thereby reducing clinical symptoms. Clinical and laboratory data for 252 consecutive patients with PCR-verified past SARS-CoV-2 infection and long-COVID syndrome (155 vaccinated and 97 non-vaccinated) were recorded during April 2021‚ÄìMay 2022 (median 243 days post-COVID-19 infection). DNA virus‚Äìrelated IgG and IgM titers were compared between vaccinated and non-vaccinated long-COVID patients and with age- and sex-matched non-infected, unvaccinated (pan-negative for spike-antibody) controls. Vaccination with monovalent COVID-19 vaccines was associated with significantly less frequent fatigue and multiorgan symptoms (p<‚Äâ0.001), significantly less cumulative DNA virus‚Äìrelated IgM positivity, significantly lower levels of plasma IgG subfractions 2 and 4, and significantly lower quantitative cytomegalovirus IgG and IgM and EBV IgM titers. These results indicate that anti-SARS-CoV-2 vaccination may interrupt viral cross-talk in patients with long-COVID syndrome (ClinicalTrials.gov Identifier: NCT05398952).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">https://www.nature.com/articles/s41541-023-00739-2</a></p>
                
                <br><br>
            </div>
        </div>
        
</p>
</div>
<br><br><br><br><br><br><br><br>
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
      <!-- end footer -->
      <!-- Javascript files--> 
      <script src="js/jquery.min.js"></script> 
      <script src="js/popper.min.js"></script> 
      <script src="js/bootstrap.bundle.min.js"></script> 
      <script src="js/jquery-3.0.0.min.js"></script> 
      <script src="js/plugin.js"></script> 
      <!-- sidebar --> 
      <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
      <script src="js/custom.js"></script>
      <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
      <script>
         $(document).ready(function(){
         $(".fancybox").fancybox({
         openEffect: "none",
         closeEffect: "none"
         });
         
         $(".zoom").hover(function(){
         
         $(this).addClass('transition');
         }, function(){
         
         $(this).removeClass('transition');
         });
         });
         
      </script> 
   </body>
</html>
<!-- end footer -->
<script>
function toggleAbstract(id, button) {
    const content = document.getElementById(id);
    if (content.style.display === "none") {
        content.style.display = "block";
        button.textContent = "Less";
    } else {
        content.style.display = "none";
        button.textContent = "More";
    }
}
document.addEventListener("DOMContentLoaded", function() {
    const buttons = document.querySelectorAll(".toggle-button");
    buttons.forEach(button => {
        const contentId = button.getAttribute("onclick").match(/'([^']+)'/)[1];
        const content = document.getElementById(contentId);
        if (content) {
            content.style.display = "none";
        }
    });
});
</script>
</body>
</html>
