
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Research News</title>
    <link rel="icon" href="images/logo.png" type="image/x-icon">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/responsive.css">
    <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.css" media="screen">
    <style>
        @media (max-width: 767px) {
            .menu-area-main {
                flex-direction: column;
            }
            .menu-area-main li {
                text-align: center;
            }
        }
        .toggle-content { display: none; }
        .toggle-button {
            background-color: #007bff;
            color: white;
            border: none;
            padding: 4px 12px;
            text-align: center;
            text-decoration: none;
            display: inline-block;
            font-size: 14px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 12px;
        }
    </style>
</head>
<body class="main-layout">
<header>
         <!-- header inner -->
         <div class="header">
            <div class="head_top">
               <div class="container">
                  <div class="row">
                    <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                       <div class="top-box">
                        <ul class="sociel_link">
                         <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                         <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                         <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                         <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                         <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                     </ul>
                    </div>
                  </div>
                  <div class="col-xl-6 col-lg-6 col-md-6 col-sm-12">
                     <div class="top-box">
                        <meta charset="UTF-8">
                        <meta name="viewport" content="width=device-width, initial-scale=1.0">
                        <style>
                            @keyframes throb {
                                0%, 100% {
                                    transform: scale(1);
                                }
                                50% {
                                    transform: scale(1.2);
                                }
                            }
                    
                            .throb-line:hover::after {
                                content: ' ‚ù§Ô∏è'; /* Add the heart emoji after the text */
                                display: inline-block;
                                animation: throb 0.76s infinite; /* Apply the throb animation */
                            }
                        </style>
                    
                        <p class="throb-line">It's better to prevent.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <div class="container">
            <div class="row">
                <div class="col-xl-3 col-lg-3 col-md-3 col-sm-3 col logo_section">
                    <div class="full">
                        <div class="center-desk">
                            <div class="logo">
                                <a href="index.html"><img src="images/logo.png" alt="Covid-19 Wastewater Monitoring" width="50" height="50"></a>
                            </div>
                            <p><strong>Covid-19 Wastewater Monitoring (Beta)</strong></p>
                        </div>
                    </div>
                </div>
                <div class="col-xl-7 col-lg-7 col-md-9 col-sm-9">
                    <div class="menu-area">
                        <div class="limit-box">
                            <nav class="main-menu">
                                <ul class="menu-area-main">
                                    <li><a href="index.html">Geo Map</a></li>
                                    <li class=""> <a href="search_geo.html">Geo Map Search</a> </li>
                                    <li><a href="trends.html">Trend Graphs</a></li>
                                    <li><a href="predictions_tab.html">Predictive</a></li>
                                    <li class="active"><a href="new_research.html">Research News</a></li>
                                    <li><a href="data_sources.html">Data Sources</gita></li>
                                    <li><a href="about.html">About</a></li>
                                </ul>
                            </nav>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</header>
<div class="brand_color">
    <div class="container">
        <div class="row">
            <div class="col-md-12">
                <div class="titlepage">
                    <h2>Research News</h2>
                    <br>
                    <h4>ü§ñ Please note that an AI (LLM model) is used to create a more understandable Title & Abstract for <i>non-scientists</i>.<br>
                    Do not make conclusions based on the AI. <i><u>Please make sure you understand the true Title & Abstract before drawing conclusions</u></i>.</h4>
                </div>
            </div>
        </div>
    </div>
</div>
<!-- contact -->
<p>
<br><br><br>

        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-24</strong> <a href="https://www.nature.com/articles/s41541-024-00973-2" target="_blank">
            ü§ñ Interferon as a vaccine booster to increase antibody response after influenza B infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-0', this)">
            More</button></h4>
            <div id="toggle-0" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Interferon as an immunoadjuvant to enhance antibodies following influenza B infection and vaccination in ferrets</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Despite annual vaccinations, the influenza B virus (IBV) continues to cause significant morbidity and mortality in humans. Researchers found that IBV infection resulted in a weaker immune response than the flu A virus. To understand why this is happening, scientists administered type-I or type-III interferon (IFN) to ferrets following infection or vaccination. IFN plays an important role in the initial innate immune response and affects subsequent adaptive responses.

Following IBV infection or vaccination, IFN treatment increased gene expression of early inflammatory responses in the upper respiratory tract and robust IFN, TSLP, and inflammatory responses in peripheral blood cells. These responses were sustained after challenge or vaccination.

Serum levels of interferon lambda (IFNL) and thymic stromal lymphopoietin (TSLP) increased in response to IBV infection or vaccination, but this was not seen with vaccine administration. Antibody responses in the serum increased more quickly and to higher titers than mock-Tx animals over 3 months.

Following rechallenge of infected animals three months post treatment, antibody levels remained higher than mock-Tx animals. However, IFN treatment did not affect antibody responses following challenge of vaccinated animals. A strong direct correlation was found between TSLP levels and antibody responses after challenge-rechallenge and vaccination-challenge. This suggests that TSLP could be a useful tool for predicting adaptive immune responses to IBV infection or vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite annual vaccination, influenza B viruses (IBV) continue to cause significant morbidity and mortality in humans. We have found that IBV infection resulted in a weaker innate and adaptive immune response than influenza A viruses (IAV) in ferrets. To understand and overcome the weak immune responses to IBV in ferrets, we administered type-I or type-III interferon (IFN) to ferrets following infection or vaccination and evaluated their effects on the immune response. IFN signaling following viral infection plays an important role in the initial innate immune response and affects subsequent adaptive immune responses. In the respiratory tract, IFN lambda (IFNL) has regulatory effects on adaptive immunity indirectly through thymic stromal lymphopoietin (TSLP), which then acts on immune cells to stimulate the adaptive response. Following IBV infection or vaccination, IFN treatment (IFN-Tx) upregulated gene expression of early inflammatory responses in the upper respiratory tract and robust IFN, TSLP, and inflammatory responses in peripheral blood cells. These responses were sustained following challenge or vaccination in IFN-Tx animals. Serum IFNL and TSLP levels were enhanced in IFN-Tx animals following challenge/rechallenge over mock-Tx; however, this difference was not observed following vaccination. Antibody responses in serum of IFN-Tx animals following IBV infection or vaccination increased more quickly and to higher titers and were sustained longer than mock-Tx animals over 3 months. Following rechallenge of infected animals 3 months post treatment, antibody levels remained higher than mock-Tx. However, IFN-Tx did not have an effect on antibody responses following challenge of vaccinated animals. A strong direct correlation was found between TSLP levels and antibody responses following challenge-rechallenge and vaccination-challenge indicating it as a useful tool for predicting adaptive immune responses following IBV infection or vaccination. The effects of IFN on strengthening both innate and adaptive responses to IBV may aid in development of more effective treatments following infection and improved influenza vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00973-2" target="_blank">https://www.nature.com/articles/s41541-024-00973-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-24</strong> <a href="https://www.nature.com/articles/s41598-024-77087-1" target="_blank">
            ü§ñ Collaborative Genome Array System (CAST) EiJ strain

A genetic variant developed from an existing mouse strain used to study and analyze the characteristics of COVID-19.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-1', this)">
            More</button></h4>
            <div id="toggle-1" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterization of Collaborative Cross mouse founder strain CAST/EiJ as a novel model for lethal COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Mutations in certain SARS-CoV-2 variants have widened their host range, affecting non-primate mammals like mice. This study used genetically diverse mouse panels to research how various strains respond to SARS-CoV-2 infection. Mouse strains with wild-derived origins showed varying responses, but one strain (CAST/EiJ) was particularly aggressive and caused significant damage. These findings may help develop new treatments for COVID-19 and other coronavirus diseases in the future by testing potential therapies on genetically diverse mouse models.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Mutations in SARS-CoV-2 variants of concern (VOCs) have expanded the viral host range beyond primates, and a few other mammals, to mice, affording the opportunity to exploit genetically diverse mouse panels to model the broad spectrum of responses to infection in patient populations. Here we surveyed responses to VOC infection in genetically diverse Collaborative Cross (CC) founder strains. Infection of wild-derived CC founder strains produced a broad range of viral burden, disease susceptibility and survival, whereas most other strains were resistant to disease despite measurable lung viral titers. In particular, CAST/EiJ, a wild-derived strain, developed high lung viral burdens, more severe lung pathology than seen in other CC strains, and a dysregulated cytokine profile resulting in morbidity and mortality. These inbred mouse strains may serve as a valuable platform to evaluate therapeutic countermeasures against severe COVID-19 and other coronavirus pandemics in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-77087-1" target="_blank">https://www.nature.com/articles/s41598-024-77087-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-76400-2" target="_blank">
            ü§ñ Treatment Approach for Severe Influenza-Associated Pneumonia</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-2', this)">
            More</button></h4>
            <div id="toggle-2" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Beneficial effect of sequential treatment with high-dose steroids and short-course oral glucocorticoids in patients with severe influenza virus-associated pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Glucocorticoid treatment can help shorten the course of viral pneumonia (VPA) and improve survival rates. However, there is no standard approach to dosage and duration of treatment, which makes it difficult to determine its effectiveness. Researchers studied 11 patients with severe VPA who received high-dose glucocorticoids for three months. The study found that the symptoms improved, and none of the patients died due to tracheal intubation or pulmonary fibrosis. The researchers suggest that short-course oral glucocorticoids may be helpful in reducing complications and improving outcomes in patients with severe VPA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Accumulating evidence supports that glucocorticoid treatment for viral pneumonia (VPA) can shorten the disease course and improve survival. However, currently, the use of glucocorticoids in treating VPA remains controversial. Moreover, a unified standard for the dosage and duration of glucocorticoid therapy has not been presented in published articles. A retrospective analysis was conducted in patients who were hospitalized for severe influenza virus-associated pneumonia, and they received sequential treatment with high-dose glucocorticoids and short-course oral glucocorticoids. Patients were followed up for 3¬†months. A total of 11 patients were included in the study (average age 56¬†years). There was no gender difference, but age and underlying diseases could be risk factors for severe influenza virus-associated pneumonia. The types of viruses causing pneumonia included influenza A/B. The main clinical symptoms of patients were fever, cough, sputum production, and dyspnea. Chest computed tomography showed multiple ground-glass shadows in the lobes, and the presence of bacterial and fungal infections was accompanied by consolidation shadows. After glucocorticoid therapy, the symptoms improved. None of the patients underwent tracheal intubation, and all survived. After a 3-month follow-up, lung CT absorption in all patients had reached more than 80%, and lung imaging absorption in 20% patients was complete. No serious complications occurred in any of the patients. Sequential treatment with high-dose steroids and short-course oral glucocorticoids may be helpful for reducing the tracheal intubation rate and mortality rate in patients with severe influenza virus-associated pneumonia. Additionally, short-course oral glucocorticoids may reduce pulmonary fibrosis in patients with severe influenza virus-associated pneumonia without any serious complications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76400-2" target="_blank">https://www.nature.com/articles/s41598-024-76400-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41556-024-01537-1" target="_blank">
            ü§ñ Real-time and programmable RNA sequencing technology for rapid data generation in research settings.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-3', this)">
            More</button></h4>
            <div id="toggle-3" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Real-time and programmable transcriptome sequencing with PROFIT-seq</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The eukaryotic transcriptome is complex, making it hard to detect specific genes or proteins. Current methods often need complicated steps before analyzing the data. A new method called PROFIT-seq helps by selectively targeting genes without affecting the rest of the genome. It improves the quality and speed of gene analysis. This method can study how immune systems respond to infections in cancer patients, including polyps.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The high diversity and complexity of the eukaryotic transcriptome make it difficult to effectively detect specific transcripts of interest. Current targeted RNA sequencing methods often require complex pre-sequencing enrichment steps, which can compromise the comprehensive characterization of the entire transcriptome. Here we describe programmable full-length isoform transcriptome sequencing (PROFIT-seq), a method that enriches target transcripts while maintaining unbiased quantification of the whole transcriptome. PROFIT-seq employs combinatorial reverse transcription to capture polyadenylated, non-polyadenylated and circular RNAs, coupled with a programmable control system that selectively enriches target transcripts during sequencing. This approach achieves over 3-fold increase in effective data yield and reduces the time required for detecting specific pathogens or key mutations by 75%. We applied PROFIT-seq to study colorectal polyp development, revealing the intricate relationship between host immune responses and bacterial infection. PROFIT-seq offers a powerful tool for accurate and efficient sequencing of target transcripts while preserving overall transcriptome quantification, with broad applications in clinical diagnostics and targeted enrichment scenarios.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41556-024-01537-1" target="_blank">https://www.nature.com/articles/s41556-024-01537-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-76252-w" target="_blank">
            ü§ñ Study Impact of COVID-19 on Allergic Rhinitis Patterns in China Based on Local Data.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-4', this)">
            More</button></h4>
            <div id="toggle-4" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>To study the impact of COVID-19 on the epidemiological characteristics of allergic rhinitis based on local big data in China</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>To investigate the characteristics and changes in allergic rhinitis (AR) patients before and after the COVID-19 pandemic, a study was conducted using descriptive epidemiological methods on data from 2018 to 2023.

An analysis found that over 62,000 AR patients were identified, with most males and females aged between 1 and 89 years. The number of AR patients increased steadily each year, but without significant differences in gender or age groups. However, the pandemic seemed to accelerate this trend. There was a clear seasonal pattern, and the number of AR patients increased significantly during COVID-19, possibly due to hormonal changes related to the virus.

A higher proportion of male AR patients were found compared to female patients. This difference may be related to hormonal fluctuations caused by the pandemic. As a result, updated management measures are needed to address the changing trends in AR since the pandemic began.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To investigate the big data characteristics and trend changes of patients with allergic rhinitis (AR) who sought medical attention at our hospital before (from 2018 to 2019) and after (from 2020 to 2023) COVID-19, and provide reference basis for the treatment of AR.  This study used a descriptive epidemiological method to analyze the big data and trend changes of AR patients.  A total of 62,196 AR patients were collected, of whom 32,874 were male and 29,322 were female, with an age range of 1‚Äì89 years. The monthly change trend of AR patients showed a marked seasonality. The number of AR patients increased year by year. There was no significant difference in the number of patients between different genders. There was a significant difference in the number of patients between different age groups.  The number of AR patients increased markedly from 2018 to 2023, and COVID-19 seems to have accelerated this process. There is a clear seasonal pattern. The number of boy patients is significantly higher than that of girl patients, and the change in female hormones may affect the incidence of AR. Therefore, it is necessary to update management measures and formulate relevant policies in response to the changing trend of AR since the COVID-19 epidemic.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76252-w" target="_blank">https://www.nature.com/articles/s41598-024-76252-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-23</strong> <a href="https://www.nature.com/articles/s41598-024-72449-1" target="_blank">
            ü§ñ COVID-19 Treatment Effectiveness

Atorvastatin and telmisartan not effective against SARS-CoV-2 in mild or moderate cases.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-5', this)">
            More</button></h4>
            <div id="toggle-5" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Atorvastatin and telmisartan do not reduce nasopharyngeal carriage of SARS-CoV-2 in mild or moderate COVID-19 in a phase IIb randomized controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Observational studies have found that taking ACE2 inhibitors and statins can reduce severe COVID-19 disease. A study in C√¥te d'Ivoire tested the combination of these medications with two other treatments for mild to moderate COVID-19 infection. The goal was to see if adding a third medication to the standard treatment could help shorten recovery time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Observational studies suggest a reduction in fatal or severe COVID-19 disease with the use of ACE2 inhibitors and statins. We implemented a randomized controlled tree-arm open label trial evaluating the benefits of adding telmisartan (TLM) or atorvastatin (ATV) to lopinavir boosted ritonavir (LPVr) on the SARS-CoV-2 nasopharyngeal viral load in patients with mild / moderate COVID-19 infection in C√¥te d‚ÄôIvoire. RT-PCR positive COVID-19 patients‚Äâ‚â•‚Äâ18¬†years, with general or respiratory symptoms for less than 7¬†days were randomized (1:1:1) to receive LPVr (400¬†mg/100¬†mg twice daily), LPVr‚Äâ+‚ÄâTLM (10¬†mg once daily) or LPVr‚Äâ+‚ÄâATV (20¬†mg once daily) for 10¬†days. The primary endpoint was viro-inflammatory success defined as a composite variable at day 11: Ct‚Äâ‚â•‚Äâ40 and C-reactive protein‚Äâ<‚Äâ27¬†mg/L. We randomized 294 patients: 96 to LPVr, 100 to LPVr‚Äâ+‚ÄâTLM, 98 to LPVr‚Äâ+‚ÄâATV arms. Baseline characteristics were well balanced between arms. In the primary analysis (missing‚Äâ=‚Äâfailure), 46% patients in the LPVr arm reached viro-inflammatory success at day 11 vs 43% in the LPVr‚Äâ+‚ÄâTLM arm (p‚Äâ=‚Äâ0.69) and 43% in the LPVr‚Äâ+‚ÄâATV arm (p‚Äâ=‚Äâ0.68). The median time from baseline to resolution of COVID-19 related symptoms was not different between arms. Nine patients were hospitalized: 2 in the LPVr arm, 5 in the LPVr‚Äâ+‚ÄâTLM arm and 2 in the LPVr‚Äâ+‚ÄâATV arm and 4 patients died. Among adults with mild to moderate COVID-19 infection, the addition of telmisartan or atorvastatin, to the standard LPVr treatment is not associated with a better virological or clinical outcome.Trial registration:NCT04466241, registered on 10/07/2020</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72449-1" target="_blank">https://www.nature.com/articles/s41598-024-72449-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-76871-3" target="_blank">
            ü§ñ Research on COVID-19 Survivors' Outcomes After Hospitalization</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-6', this)">
            More</button></h4>
            <div id="toggle-6" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>One-year mortality and associated factors in older hospitalized COVID-19 survivors: a Nationwide Cohort Study in Korea</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study aimed to evaluate the 1-year mortality rate among older patients who recovered from COVID-19 after being discharged from hospital. It looked at data from a Korean health insurance system for patients aged 60 or over with COVID-19, comparing their survival rates and identifying risk factors that contributed to death within 1 year of recovery.

Researchers used a model to analyze how many people died or survived each month following discharge, and then compared the mortality rate to see if it was similar after hospital discharge in Korea. They found that non-survivors were older and had been vaccinated less often than survivors. Additionally, non-survivors experienced more complications during their illness.

The study also looked at other factors that might have contributed to death, including age, sex, cardiovascular disease, organ dysfunction, severity of the illness, and certain treatments used during hospitalization. 

Overall, the research suggested that older patients who recovered from COVID-19 after being discharged had a similar 1-year mortality rate as those in Korea, but still experienced higher risks due to factors such as age and lack of vaccination protection for long-term safety against COVID-19-related complications.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aimed to evaluate the 1-year mortality rate among older patients with COVID-19 discharged from hospital and to identify risk factors associated with this outcome. Using a COVID-19 dataset from the Korean National Health Insurance System, this study‚Äôs evaluation period spanned from October 8, 2020, through December 31, 2021. The primary outcome was the 1-year mortality rate following hospital discharge. A logistic regression model was employed for multivariable analysis to estimate the odds ratios for the outcomes, and the Kaplan-Meier method was used to analyze differences in 1-year survival rates. Among the 66,810 COVID-19 patients aged 60 years or older who were hospitalized during the study period, the in-hospital mortality rate was 4.8% (n=‚Äâ3219). Among the survivors (n=‚Äâ63,369), the 1-year mortality rate was 4.9% (n=‚Äâ3093). Non-survivors, compared to survivors, were significantly older (79.2‚Äâ¬±‚Äâ9.5 vs. 68.9‚Äâ¬±‚Äâ7.8,P<‚Äâ0.001) and exhibited a lower rate of COVID-19 vaccination (63.0% vs. 91.7%,P<‚Äâ0.001). Additionally, non-survivors experienced a higher incidence of organ dysfunction, along with a greater proportion of required mechanical ventilation (14.6% vs. 1.0%,P<‚Äâ0.001) and extracorporeal membrane oxygenation (4.0% vs. 0.1%,P<‚Äâ0.001). Multivariable logistic regression analysis identified older age, male sex, cardiovascular disease, immunosuppression, organ dysfunction, illness severity, and corticosteroid use during hospitalization as factors associated with death within 1 year after hospital discharge. However, vaccination was found to have a long-term protective effect against death among COVID-19 survivors. The 1-year mortality rate after hospital discharge for older COVID-19 patients was comparable to the in-hospital mortality rate for these patients in Korea. The long-term mortality rate among hospitalized older COVID-19 patients was influenced by demographic factors and the severity of illness experienced during hospitalization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76871-3" target="_blank">https://www.nature.com/articles/s41598-024-76871-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-76657-7" target="_blank">
            ü§ñ Blood Pressure Medications and COVID-19 Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-7', this)">
            More</button></h4>
            <div id="toggle-7" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Losartan and enalapril maleate differently influence SARS-CoV-2-infected vero cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has posed significant challenges for global healthcare systems, particularly affecting individuals with pre-existing conditions like hypertension. Researchers studied how two antihypertensive medications, losartan and enalapril maleate, affected cells infected with SARS-CoV-2 in vitro.

In this study, researchers found that losartan significantly reduced the levels of viral RNA in these cells, making it nearly undetectable. However, enalapril maleate did not have a significant effect on reducing viral RNA levels. Furthermore, they discovered that the expression of genes involved in cell survival and regulation was altered in response to infection.

In addition, researchers found that losartan increased certain proteins (IL-6, p53, p21, and p62) and pathways (il-6) compared to uninfected cells treated with losartan. In contrast, enalapril maleate did not demonstrate a significant effect on these proteins or pathways.

The study suggests that both medications may interfere with the effects of SARS-CoV-2 infection in Vero E6 cells, but their influence appears to vary in terms of quantity and quality in modulating metabolic and signal transduction pathways.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Background: The COVID-19 pandemic has posed significant challenges to global healthcare systems, particularly impacting individuals with pre-existing conditions like hypertension. This study sought to assess the impact of the antihypertensive medications, losartan and enalapril maleate on SARS-CoV-2 infected cells. Vero E6 cells were infected and treated in vitro, evaluating cell viability via the MTT colorimetric assay. Additionally, the study measured relative levels of viral RNA and selected gene messenger RNAs using reverse transcriptase followed by quantitative real-time polymerase chain reaction. Results: The findings revealed that losartan substantially reduced nucleocapsid RNA levels of SARS-CoV-2 to nearly undetectable levels, while enalapril maleate did not demonstrate a significant effect. In response to viral infection, the expression ofil-18,p53,p21, andp62increased compared to uninfected-untreated cells. Notably,il-6expression was upregulated by both infection and treatments. A comparison between infected cells treated with losartan or enalapril maleate highlighted the presence of distinct profiles in the expression ofil-6,p53,p21, andp62. Conclusions: The data from our study suggest that these medications could interfere with certain effects triggered by SARS-CoV-2 infection in Vero E6 cells. However, their influence appears to vary both quantitatively and qualitatively in the modulation of metabolic and signal transduction pathways.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76657-7" target="_blank">https://www.nature.com/articles/s41598-024-76657-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41541-024-00997-8" target="_blank">
            ü§ñ Human Immune Response to Sarbecoviruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-8', this)">
            More</button></h4>
            <div id="toggle-8" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broad cross neutralizing antibodies against sarbecoviruses generated by SARS-CoV-2 infection and vaccination in humans</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Preventing future coronavirus pandemics requires protecting against multiple types of coronaviruses, including the ones that cause MERS and SARS-CoV-1, as well as the new strain known as Omicron subvariant.

Researchers studied whether people who had COVID-19 or received vaccines could develop antibodies (like special proteins) to protect against other coronaviruses that might come from animals. They tested these antibodies on a variety of viruses, including some that are common in bats and pangolins.

The results showed that the antibodies worked well for many types of coronaviruses but not as well for the one that causes MERS or SARS-CoV-1. Some of the antibodies also had extra parts (like small proteins) that helped to neutralize other viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The outbreaks of severe acute respiratory syndrome coronavirus (SARS-CoV-1), Middle East respiratory syndrome coronavirus (MERS-CoV), and SARS-CoV-2 highlight the need for countermeasures to prevent future coronavirus pandemics. Given the unpredictable nature of spillover events, preparing antibodies with broad coronavirus-neutralizing activity is an ideal proactive strategy. Here, we investigated whether SARS-CoV-2 infection and vaccination could provide cross-neutralizing antibodies (nAbs) against zoonotic sarbecoviruses. We evaluated the cross-neutralizing profiles of plasma and monoclonal antibodies constructed from B cells from coronavirus disease 2019 (COVID-19) convalescents and vaccine recipients; against sarbecoviruses originating from bats, civets, and pangolins; and against SARS-CoV-1 and SARS-CoV-2. We found that the majority of individuals with natural infection and vaccination elicited broad nAb responses to most tested sarbecoviruses, particularly to clade 1b viruses, but exhibited very low cross-neutralization to SARS-CoV-1 in both natural infection and vaccination, and vaccination boosters significantly augmented the magnitude and breadth of nAbs to sarbecoviruses. Of the nAbs, several exhibited neutralization activity against multiple sarbecoviruses by targeting the spike receptor-binding domain (RBD) and competing with angiotensin-converting enzyme 2 (ACE2) binding. SCM12-61 demonstrated exceptional potency, with half-maximal inhibitory concentration (IC50) values of 0.001‚Äì0.091‚ÄâŒºg/mL against tested sarbecoviruses; while VSM9-12 exhibited remarkable cross-neutralizing breadth against sarbecoviruses and SARS-CoV-2 Omicron subvariants, highlighting the potential of these two nAbs in combating sarbecoviruses and SARS-CoV-2 Omicron subvariants. Collectively, our findings suggest that vaccination with an ancestral SARS-CoV-2 vaccine, in combination with broad nAbs against sarbecoviruses, may provide a countermeasure for preventing further sarbecovirus outbreaks in humans.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00997-8" target="_blank">https://www.nature.com/articles/s41541-024-00997-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-22</strong> <a href="https://www.nature.com/articles/s41598-024-75038-4" target="_blank">
            ü§ñ Improved isolation of SARS-CoV-2 from COVID-19 clinical specimens with enhanced cellular expression</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-9', this)">
            More</button></h4>
            <div id="toggle-9" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improved efficacy of SARS-CoV-2 isolation from COVID-19 clinical specimens using VeroE6 cells overexpressing TMPRSS2 and human ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Cell culture-based isolation of COVID-19 novel coronavirus is an important process for both laboratory diagnosis and understanding new variants. Research has found that a specific type of cell line called VeroE6/TMPRSS2 can efficiently isolate the virus from patient samples. These cells have two key features: they express a receptor molecule (monkey ACE2) and have a serine-protease enzyme (TMPRSS2). They are more effective than other types of cells at isolating the virus, showing an 1.6-fold increase in efficiency.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The cell culture-based isolation of novel coronavirus SARS-CoV-2 from clinical specimens obtained from patients with suspected COVID-19 is important not only for laboratory diagnosis but also for obtaining live virus to characterize emerging variants. Previous studies report that monkey kidney-derived VeroE6/TMPRSS2 cells allow efficient isolation of SARS-CoV-2 from clinical specimens because these cells show stable expression of the receptor molecule monkey ACE2 and the serine-protease TMPRSS2. Here, we demonstrated that VeroE6 cells overexpressing human ACE2 and TMPRSS2 (Vero E6-TMPRSS2-T2A-ACE2 cells) are superior to VeroE6/TMPRSS2 for isolating SARS-CoV-2 from clinical specimens. These cells showed a 1.6-fold increase in efficiency in SARS-CoV-2 isolation, and were particularly effective for clinical specimens with a relatively low viral load (<‚Äâ106copies/mL). When using vesicular stomatitis virus (VSV) pseudoviruses (VSV/SARS-2pv) bearing the spike proteins of all of the tested SARS-CoV-2 strains, Vero E6-TMPRSS2-T2A-ACE2 cells showed a 2‚Äì¬†to fourfold increase in infectivity. Furthermore, the results of virus titration and neutralization antibody assays using Vero E6-TMPRSS2-T2A-ACE2 cells were different from those using VeroE6/TMPRSS2, highlighting the importance of selecting appropriate cell culture systems to determine SARS-CoV-2 infectivity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75038-4" target="_blank">https://www.nature.com/articles/s41598-024-75038-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41746-024-01286-3" target="_blank">
            ü§ñ Long COVID Temporal Topic Modeling of Electronic Health Records

Temporal topic modeling analysis on electronic health records (EHRs) from the National Center for Chronic Disease Prevention and Control (NCCDP) and the Recovery (RECOVER) program.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-10', this)">
            More</button></h4>
            <div id="toggle-10" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Finding Long-COVID: temporal topic modeling of electronic health records from the N3C and RECOVER programs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-Acute Sequelae of SARS-CoV-2 (Long COVID) is a complex condition that affects some people after they've recovered from a severe case of COVID-19, leading to various health problems. Researchers analyzed the experiences of over 600 million patients with COVID-19 and found many conditions worsened in those who had the disease, as well as specific patterns for different groups of people. This study revealed new insights into the causes and consequences of Long COVID and suggests that it may require better diagnosis and understanding to improve treatment and prevention strategies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-Acute Sequelae of SARS-CoV-2 infection (PASC), also known as Long-COVID, encompasses a variety of complex and varied outcomes following COVID-19 infection that are still poorly understood. We clustered over 600 million condition diagnoses from 14 million patients available through the National COVID Cohort Collaborative (N3C), generating hundreds of highly detailed clinical phenotypes. Assessing patient clinical trajectories using these clusters allowed us to identify individual conditions and phenotypes strongly increased after acute infection. We found many conditions increased in COVID-19 patients compared to controls, and using a novel method to associate patients with clusters over time, we additionally found phenotypes specific to patient sex, age, wave of infection, and PASC diagnosis status. While many of these results reflect known PASC symptoms, the resolution provided by this unprecedented data scale suggests avenues for improved diagnostics and mechanistic understanding of this multifaceted disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-024-01286-3" target="_blank">https://www.nature.com/articles/s41746-024-01286-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-75519-6" target="_blank">
            ü§ñ Understanding the Mechanism of SARS-CoV-2 Inhibitors

Benzisoselenazolones and Diselenides as SARS-CoV-2 Inhibitors
=====================================================

### INSTRUCTIONS ###

* Simplify the title for a general audience to convey the main idea.
* Avoid using quotation marks around the text.
* Keep the explanation concise, focused on clarity rather than brevity.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-11', this)">
            More</button></h4>
            <div id="toggle-11" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>New insights in the mechanism of the SARS-CoV-2 Mproinhibition by benzisoselenazolones and diselenides</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new virus, SARS-CoV-2, has been kept at bay thanks to global vaccination efforts, but a possibility remains that the virus could evolve and cause another pandemic. To combat this risk, scientists are seeking compounds that can target the main protease of the virus (Mpro). Researchers tested various small molecules derived from benzisoselenazolones and diselenides for their ability to inhibit Mpro and found some effective ones. They also investigated how these compounds interact with proteins in the body to understand why they're so effective against Mpro. To do this, scientists used advanced computer simulations and modeling techniques to better understand the interactions between these molecules and the virus. This research could lead to the discovery of new compounds that can prevent SARS-CoV-2 from multiplying, potentially saving lives during a future pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Although global vaccination campaigns alleviated the SARS-CoV-2 pandemic in terms of morbidity and mortality, the ability of the virus to originate mutants may reduce the efficacy of vaccines, posing a serious risk of a renewed pandemic. There is therefore a need to develop small molecules capable of targeting conserved viral targets, such as the main protease (Mpro). Here, a series of benzisoselenazolones and diselenides were tested for their ability to inhibit Mpro; then the most potent compounds were measured for antiviral activity in vitro, and the mechanism of action was investigated. Density functional theory calculations, molecular docking and molecular dynamics simulations were also used to elucidate the protein/drug interaction. Finally, a bio-organic model was established to study the reaction between selenorganic compounds and biologically relevant thiols to unveil possible metabolic pathways of such compounds. The overall results contribute to the identification of a series of novel Se-containing molecules active against SARS-CoV-2 and to the clarification of some important aspects in the mechanisms of action of such inhibitors targeting SARS-CoV-2 Mpro.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75519-6" target="_blank">https://www.nature.com/articles/s41598-024-75519-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-74712-x" target="_blank">
            ü§ñ Improved Vaccination Protocol for Long-Term Survivors After Allogeneic Hematopoietic Stem Cell Transplantation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-12', this)">
            More</button></h4>
            <div id="toggle-12" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunogenicity of Comirnaty Omicron XBB.1.5 booster COVID-19 mRNA vaccine in long-term survivors after allogeneic hematopoietic stem cell transplantation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Primary mRNA vaccination against COVID-19 typically involves three doses for immunocompromised individuals, including those who have undergone hematopoietic stem cell transplantation (allo-HSCT). But the best way to boost immunity remains unclear after multiple doses of mRNA vaccine.

A new study looked at how a booster dose from the most recently updated vaccine was received by long-term allo-HSCT survivors. These patients had previously received many mRNA vaccines, with the last dose being about 13 months ago. The researchers collected blood samples before and four weeks after the booster and measured antibodies against the spike (S1) and nucleocapsid proteins in the blood, as well as T-cell responses.

About half of the patients showed a strong immune response to both S1 and T-cells just before the booster dose, but some were not protected from severe COVID-19. Women had higher antibody levels than men, with 62% having a previous confirmed case of COVID-19. However, this was only true for women.

After the booster dose, some patients showed stronger immune responses to both S1 and T-cells, while others did not respond as well. Some patients who received immunosuppressive treatment had weaker immune responses overall.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary mRNA vaccination against COVID-19 typically involves three doses for immunocompromised individuals, including hematopoietic stem cell transplantation (allo-HSCT) recipients. However, optimal subsequent boosting strategies remain unclear. This study aimed to assess the immunogenicity of a booster dose using the most recently updated vaccine (Comirnaty Omicron XBB.1.5) among long-term allo-HSCT survivors having previously received multiple mRNA vaccine doses, in median 4 (2‚Äì6). Thirty-four allo-HSCT recipients were enrolled at Sahlgrenska University Hospital, and peripheral blood samples were collected immediately before and four weeks after booster. Antibodies against the receptor-binding domain (anti-RBD) of spike 1 (S1) and nucleocapsid, as well as S1-specific ex vivo T-cell responses, were evaluated. Adverse events were monitored. Despite a median of 13 months since the prior vaccine dose, both humoral and T-cell responses against S1 were present in the pre-booster samples in all but two participants, who suffered from severe chronic Graft-versus-host disease. Notably, 62% of participants had a previously confirmed COVID-19 infection. Significantly higher pre-booster antibody levels were observed in women than men (p=‚Äâ0.003). Booster dosing strengthened specific antibody and T cell responses and equalized pre-booster gender differences, although responses remained significantly lower among those receiving immunosuppressive treatment (p=‚Äâ0.041). In a population of long-term allo-HSCT survivors, the majority of whom had a prior confirmed COVID-19 infection, both pre- and post-booster immune responses were robust. However, patients undergoing immunosuppressive treatment for GvHD exhibited significantly weaker responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74712-x" target="_blank">https://www.nature.com/articles/s41598-024-74712-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-21</strong> <a href="https://www.nature.com/articles/s41598-024-75924-x" target="_blank">
            ü§ñ Cytokine Response in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-13', this)">
            More</button></h4>
            <div id="toggle-13" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Disease progression associated cytokines in COVID-19 patients with deteriorating and recovering health conditions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding the immune response to COVID-19 is challenging due to its high variability among individuals. Researchers analyzed EHR data and cytokine profile data from 444 infected patients and 145 non-infected healthy individuals to identify differentially expressed cytokines between deteriorating and recovering phases.

A random forest model was built using both healthy and severe patients to compute the contribution of each cytokine toward disease progression, with Shapley Additive Explanations (SHAP) values used for supervised clustering. The identified clusters discriminated DP and RP samples, suggesting that cytokine profiles differed between deteriorating and recovering health conditions.

Differentially expressed cytokines, such as CXCL10, GDF15, PTX3, and TNFSF10, were found to be involved in the JAK-STAT, NF-\(\kappa\)B, and MAPK signaling pathways contributing to the inflammatory response.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the immune response to COVID-19 is challenging due to its high variability among individuals. To identify differentially expressed cytokines between the deteriorating and recovering phases, we analyzed the Electronic Health Records (EHR) and cytokine profile data in a COVID-19 cohort of 444 infected patients and 145 non-infected healthy individuals. We categorized each patient‚Äôs progression into Deterioration Phase (DP) and Recovery Phase (RP) using longitudinal neutrophil, lymphocyte and lactate dehydrogenase levels. A random forest model was built using healthy and severe patients to compute the contribution of each cytokine toward disease progression using Shapley Additive Explanations (SHAP). SHAP values were used for supervised clustering to identify DP and RP-related samples and their associated cytokines. The identified clusters effectively discriminated DP and RP samples, suggesting that the cytokine profiles differed between deteriorating and recovering health conditions. Especially, CXCL10, GDF15, PTX3, and TNFSF10 were differentially expressed between the DP and RP samples, which are involved in the JAK-STAT, NF-\(\kappa\)B, and MAPK signaling pathways contributing to the inflammatory response. Collectively, we characterized the immune response in terms of disease progression of COVID-19 with deteriorating and recovering health conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75924-x" target="_blank">https://www.nature.com/articles/s41598-024-75924-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41746-024-01287-2" target="_blank">
            ü§ñ **Detecting COVID Infection with Wearable Sensors**

**Feasible Solution for Snapshot Testing**</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-14', this)">
            More</button></h4>
            <div id="toggle-14" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Feasibility of snapshot testing using wearable sensors to detect cardiorespiratory illness (COVID infection in India)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new system was developed that uses sensors and activity tracking to monitor people's health, including heart rate, breathing, temperature, and oxygen levels. This system worked well in India during the pandemic, showing promising results in detecting COVID-19 symptoms. Researchers trained an AI algorithm on the data to identify who is likely sick or healthy, achieving a high accuracy level. The findings suggest that this technology could be used for remote health screenings, potentially changing how people monitor their health over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic has challenged the current paradigm of clinical and community-based disease detection. We present a multimodal wearable sensor system paired with a two-minute, movement-based activity sequence that successfully captures a snapshot of physiological data (including cardiac, respiratory, temperature, and percent oxygen saturation). We conducted a large, multi-site trial of this technology across India from June 2021 to April 2022 amidst the COVID-19 pandemic (Clinical trial registry name: International Validation of Wearable Sensor to Monitor COVID-19 Like Signs and Symptoms; NCT05334680; initial release: 04/15/2022). An Extreme Gradient Boosting algorithm was trained to discriminate between COVID-19 infected individuals (n=‚Äâ295) and COVID-19 negative healthy controls (n=‚Äâ172) and achieved an F1-Score of 0.80 (95% CI‚Äâ=‚Äâ[0.79, 0.81]). SHAP values were mapped to visualize feature importance and directionality, yielding engineered features from core temperature, cough, and lung sounds as highly important. The results demonstrated potential for data-driven wearable sensor technology for remote preliminary screening, highlighting a fundamental pivot from continuous to snapshot monitoring of cardiorespiratory illnesses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41746-024-01287-2" target="_blank">https://www.nature.com/articles/s41746-024-01287-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-76251-x" target="_blank">
            ü§ñ ## Systemic Immune Response

This refers to the body's immune response triggered by a coronavirus, specifically SARS-CoV-2, that has been weakened or impaired due to a compromised immune system. In people who have received a kidney transplant and are immunocompromised (i.e., not on long-term immunosuppressive medications), their immune system is suppressed, making them more susceptible to the virus. As a result, T follicular helper cells, which play a key role in coordinating the body's response to infections, may be activated but also impaired.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-15', this)">
            More</button></h4>
            <div id="toggle-15" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impaired SARS-CoV-2-specific responses via activated T follicular helper cells in immunocompromised kidney transplant recipients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A group of immune cells called activated T follicular helper (aTfh) cells played a key role in the body's response to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) vaccines. Researchers studied how these cells reacted in people who had received SARS-CoV-2 mRNA vaccines and those who were not vaccinated. They found that the aTfh cells in people who got the vaccine responded differently than those who didn't get vaccinated, with more effective responses at later stages of the infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Activated T follicular helper (aTfh) cells are likely important in host immune responses to severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) mRNA vaccination. We characterized the immune responses of aTfh cells to the second (D2) and third (booster; D3) doses of an mRNA vaccine in the peripheral blood of 40 kidney transplant recipients (KTRs) and 17 healthy control volunteers (HCs). A significant increase in SARS-CoV-2-specific IgG antibody was seen after D3 in the KTRs; nonetheless, the levels after D2 and D3 were significantly lower than in the HCs. After D2, dramatic increases in activated CD45RA-CXCR5+ICOS+PD1+circulating Tfh (acTfh) cells were observed in the HCs, as well as the seropositive patients among the KTRs, when compared with the seronegative patients among the KTRs. Unlike the HCs, KTRs had less prominent immune responses, including the acTfh and T cells that produce interferon gamma, tumor necrosis factor alpha, and interleukin 21. In addition, the increase in acTfh cells was significantly associated with anti-IgG antibody levels after D3. These results indicate impaired SARS-CoV-2-specific responses via acTfh cells in KTRs, and they suggest that acTfh cells in peripheral blood may play an important role in antibody maintenance following SARS-CoV-2 mRNA vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76251-x" target="_blank">https://www.nature.com/articles/s41598-024-76251-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-75718-1" target="_blank">
            ü§ñ Studies have shown that COVID-19 can alter cancer cell behavior.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-16', this)">
            More</button></h4>
            <div id="toggle-16" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>In vitro analysis suggests that SARS-CoV-2 infection differentially modulates cancer-like phenotypes and cytokine expression in colorectal and prostate cancer cells</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new virus called SARS-CoV-2 has been linked to cancer and making its effects harder to understand. Researchers studied how this virus affects two types of cancer cells, prostate and colorectal, which both use proteins that help control cell growth and division.

They infected these cancer cells with a fake version of the virus and then looked at what happened inside the cells. They also tested some cancer markers (such as genes that help grow or survive) to see how the virus affected them. The results showed that SARS-CoV-2 affects the cells in different ways depending on which type of cancer cell they are, but it can make cells more likely to divide and die.

The researchers believe that this could be important information for figuring out if other viruses like SARS-CoV-2 affect cancer too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) reportedly exacerbates cancer outcomes. However, how COVID-19 influences cancer prognosis and development remains poorly understood. Here, we investigated the effect of Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV-2), the etiological agent of COVID-19, on cellular cancer phenotypes the expression of cancer-related markers, and various proinflammatory cytokines. We infected prostate (22RV1) and colorectal (DLD-1) cancer cell lines, which express angiotensin-converting enzyme 2 (ACE2), with spike pseudovirus (sPV) and laboratory stocks of live SARS-CoV-2 viruses. After infection, we quantified changes in the cellular cancer phenotypes, the gene expression levels of some cancer markers, including Ki-67, BCL-2, VIM, MMP9,and VEGF,and proinflammatory cytokines. Phenotypic analysis was performed using MTT and wound healing assays, whereas gene expression analysis was carried out using real-time quantitative PCR (RT-qPCR). We show that SARS-CoV-2 infection impacts several key cellular pathways involved in cell growth, apoptosis, and migration, in prostate and colorectal cancer cells. Our results suggest that SARS-CoV-2 infection does influence various cancer cellular phenotypes and expression of molecular cancer markers and proinflammatory cytokines, albeit in a cell-type-specific manner. Our findings hint at the need for further studies and could have implications for evaluating the impact of other viruses on cancer progression.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75718-1" target="_blank">https://www.nature.com/articles/s41598-024-75718-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-75262-y" target="_blank">
            ü§ñ Community-led initiatives as predictors of older adults' physical and mental well-being during the COVID-19 pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-17', this)">
            More</button></h4>
            <div id="toggle-17" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of community groups as a social prescription for senior health‚Äîinsights from a natural experiment during the COVID-19 lockdown</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Impact of Social Prescribing Policies on Loneliness in Elderly People**

The COVID-19 pandemic exacerbated loneliness among older adults, particularly during lockdowns. Recent policies aimed at supporting this vulnerable population include community groups as support structures. This study evaluates the effectiveness of these social prescribing initiatives in combating loneliness and related health decline.

We analyzed data from 618 individuals aged 65 or older who participated in a natural experiment during the pandemic lockdown. The results show that those who had access to community groups were significantly less likely to experience loneliness (2.65 times lower) and associated health problems, such as cardiovascular and cognitive decline.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Loneliness and associated physical and cognitive health decline among the aging population is an important medical concern, exacerbated in times of abnormal isolation like the 2020‚Äì2021 Covid-19 pandemic lockdown. In this backdrop, recent ‚Äúsocial prescribing‚Äù based health policy initiatives such as community groups as a support structure for the aging population assumes great importance. In this paper, we evaluate and quantify the impact of such social prescribing policies in combatting loneliness and related health degeneration of the aging population in times of abnormal isolation. To this end, we conduct a natural experiment across a sample of 618 individuals aged 65 and over with varying access to community groups during the Covid-19 lockdown period. Using a random-effects, probit model to compare the differences in health outcomes of participants with access to community groups (target) with those without access (control), we find that the target group was 2.65 times less likely to suffer from loneliness as compared to the control group, along with lower incidences of reported cardiovascular and cognitive health decline. These initial findings provide preliminary support in favor of the interventional power of social prescription tools in mitigating loneliness and its consequent negative health impact on the aging population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75262-y" target="_blank">https://www.nature.com/articles/s41598-024-75262-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-19</strong> <a href="https://www.nature.com/articles/s41598-024-74729-2" target="_blank">
            ü§ñ Global Health Burden of Vaccine-Associated Guillain-Barr√© Syndrome</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-18', this)">
            More</button></h4>
            <div id="toggle-18" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Global burden of vaccine-associated Guillain-Barr√© syndrome over 170 countries from 1967 to 2023</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research has been conducted globally on Guillain-Barr√© syndrome (GBS), a rare neurological condition linked to vaccinations. To assess the global burden of vaccine-associated GBS and its associated vaccines, we analyzed reports from VigiBase, a database of adverse event reports for medicines and vaccines, spanning 1967-2023. We found that approximately 8 in every 1,000 people who received over 19 different vaccines reported developing GBS. The most commonly implicated vaccines were those for COVID-19, with the highest association seen in influenza vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Research on Guillain-Barr√© syndrome (GBS) as a neurological adverse effect of vaccines on a global scale is scarce, highlighting the need for further investigation to evaluate its long-term impact and associated risk factors comprehensively. Hence, this study aims to assess the global burden of vaccine-associated GBS and its associated vaccines. This study utilized data from VigiBase, the World Health Organization global database of adverse event reports of medicines and vaccines, encompassing the period from 1967 to 2023 (total reports,n=‚Äâ131,255,418) to investigate vaccine-associated GBS. Reported odds ratios (ROR) and information components (IC) were analyzed to assess the association between 19 vaccines and the occurrence of vaccine-associated GBS over 170 countries. We identified 15,377 (8072 males [52.49%]) reports of vaccine-associated GBS among 22,616 reports of all drugs-cause GBS from 1978 to 2023. Cumulative reports of vaccine-associated GBS have been increasing steadily over time, with a notable surge observed since the commencement of COVID-19 vaccines administration in 2020. Most vaccines showed significant associations with GBS such as Ad5-vectored COVID-19 vaccines (ROR, 14.88; IC, 3.66), COVID-19 mRNA vaccines (ROR, 9.66; IC, 2.84), and inactivated whole-virus COVID-19 vaccines (ROR, 3,29; IC 1.69). Influenza vaccines showed the highest association (ROR, 77.91; IC 5.98). Regarding age-and sex-specific risks, the association remained similar regardless of sex, with an increased association observed with advancing age. The mean time to onset was 5.5 days. Amid the COVID-19 pandemic, the reports of GBS surged in response to widespread COVID-19 vaccination. Nonetheless, COVID-19 vaccines exhibited the lowest association compared to other vaccines. Vigilance for at least one-week post-vaccination is crucial, particularly for older adults. Further research is warranted to elucidate the underlying mechanisms linking vaccines and GBS.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74729-2" target="_blank">https://www.nature.com/articles/s41598-024-74729-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-18</strong> <a href="https://www.nature.com/articles/s41598-024-75250-2" target="_blank">
            ü§ñ Distinct Immune Responses in Older Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-19', this)">
            More</button></h4>
            <div id="toggle-19" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Similar humoral responses but distinct CD4+T cell transcriptomic profiles in older adults elicited by MF59 adjuvanted and high dose influenza vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A type of flu vaccine is recommended to be given to people over 65 years old to help keep them safe from getting sick with the flu. Researchers studied how this vaccine affects a group of older adults who received either an MF59-flavored or high-dose (HD) version of the vaccine. They found that the flu vaccine had different effects on two groups of people: those who got the regular version and those who got the higher dose. The researchers also looked at what genes were turned on or off in these cells to see if they could help explain why some people are protected from getting sick with the flu.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Older age (‚â•‚Äâ65 years) is associated with impaired responses to influenza vaccination, leading to the preferential recommendation of MF59-adjuvanted (MF59Flu) or high-dose (HDFlu) influenza vaccines for this age group in the United States. Herein, we characterized transcriptomic profiles of CD4+T cells isolated from 234 recipients (‚â•‚Äâ65 years) of either MF59Flu or HDFlu vaccine, prior to vaccination and 28 days thereafter. We identified 412 and 645 differentially expressed genes (DEGs) in CD4+T cells of older adults after receiving MF59Flu and HDFlu, respectively. DEGs in CD4+T cells of MF59Flu recipients were enriched in 14 KEGG pathways, all of which were downregulated. DEGs in CD4+T cells of HDFlu recipients were enriched in 11 upregulated pathways and 20 downregulated pathways. CD4+T cells in both vaccine groups shared 50 upregulated genes and 75 downregulated genes, all of which were enriched in 7 KEGG pathways. The remaining 287 and 520 DEGs were specifically associated with MF59Flu and HDFlu, respectively. Unexpectedly, none of these DEGs was significantly correlated with influenza A/H3N2-specific HAI titers, suggesting these DEGs at the individual level may have a limited role in protection against influenza. Our findings emphasize the need for further investigation into other factors influencing immunity against influenza in older adults.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75250-2" target="_blank">https://www.nature.com/articles/s41598-024-75250-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-18</strong> <a href="https://www.nature.com/articles/s41598-024-75514-x" target="_blank">
            ü§ñ Vaccine-induced autoimmunity related to protein S in some cases</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-20', this)">
            More</button></h4>
            <div id="toggle-20" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Autoantibodies to protein S may explain rare cases of coagulopathy following COVID-19 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new COVID-19 vaccine has been shown to be effective, but some people have experienced severe reactions that can cause life-threatening problems. Researchers wanted to know more about the vaccines and what's causing these reactions. They looked at a large group of people who had received the vaccine in Sweden and found that some were infected with COVID-19 while others got sick from the vaccine. They also found out if there was something in the vaccine or its side effects, such as protein S autoantibodies, that might be causing these problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While Coronavirus disease 2019 (COVID-19) vaccines have proven to be both effective and generally safe, rare but severe adverse events following immunization (AEFIs) are described. Autoantibodies to platelet factor-4 are associated with catastrophic thrombotic AEFIs, but comprehensive investigations of other autoantibodies are lacking. We aimed to detect and describe autoantibodies targeting coagulation-related proteins in a population-wide cohort (SWEDEGENE) including AEFIs attributed to COVID-19 vaccines in Sweden. Subjects were recruited from December 2020 to October 2022 and were stratified based on diagnosis and COVID-19 exposure. Screening was carried out in two phases, with a multiplex bead-based assay in the first subset (until September 2021) and with targeted assays for the second (until October 2022). Positivity was defined based on absolute, relative, and biological/technical thresholds. Patients with coagulation-related AEFIs were older and the Vaxzevria vaccine was overrepresented in this group. Two cases had antiphospholipid antibodies but none had PF4 antibodies. We identified six positives for protein S autoantibodies. Protein S concentrations were negatively correlated with autoantibody response in patients with immunoreactivity and functional analysis revealed low protein S activity in three subjects. Our population-wide analysis reveals cases with autoantibodies against protein S which possibly underlie coagulopathic AEFIs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75514-x" target="_blank">https://www.nature.com/articles/s41598-024-75514-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41467-024-52863-9" target="_blank">
            ü§ñ Mucosa-Associated Lymphoid Tissue (MALT) Response Induced by Cytokines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-21', this)">
            More</button></h4>
            <div id="toggle-21" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>MINCLE and TLR9 agonists synergize to induce Th1/Th17 vaccine memory and mucosal recall in mice and non-human primates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Development of New Vaccines for Hard-to-Target Diseases

Scientists have been trying to develop new vaccines that can target diseases that are difficult to reach. However, their current adjuvants (substances that help the vaccine work) often don't stimulate the body's immune response in the way they should.

Researchers discovered a novel liposomal adjuvant called CAF10b that combines Mincle and Toll-like receptor 9 agonists. They tested it with mice and non-human primates, comparing its effects to existing vaccines. 

When combined with an antigen (the substance that triggers the immune response), CAF10b stimulated strong Th1 and Th17 responses in both animals, providing protection against a bacterial infection called Mycobacterium tuberculosis.

In some primate models, CAF10b induced powerful Th1 and Th17 immune responses for six months. These types of immune responses are crucial for long-term immunity.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Development of new vaccines tailored for difficult-to-target diseases is hampered by a lack of diverse adjuvants for human use, and none of the currently available adjuvants induce Th17 cells. Here, we develop a liposomal adjuvant, CAF¬Æ10b, that incorporates Mincle and Toll-like receptor 9 agonists. In parallel mouse and non-human primate studies comparing to CAF¬Æ adjuvants already in clinical trials, we report species-specific effects of adjuvant composition on the quality and magnitude of the responses. When combined with¬†antigen, CAF¬Æ10b induces Th1 and Th17 responses and protection against a pulmonary infection withMycobacterium tuberculosisin mice. In non-human primates, CAF¬Æ10b induces higher Th1 responses and robust Th17 responses detectable after six months, and systemic and pulmonary Th1 and Th17 recall responses, in a sterile model of local recall. Overall, CAF¬Æ10b drives robust memory antibody, Th1 and Th17 vaccine-responses via a non-mucosal immunization route across both rodent and primate species.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52863-9" target="_blank">https://www.nature.com/articles/s41467-024-52863-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s44298-024-00058-w" target="_blank">
            ü§ñ Infectious Diseases Evolution Genetics Microbiology Pathogens Research Science Technology Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-22', this)">
            More</button></h4>
            <div id="toggle-22" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Discordant phylodynamic and spatiotemporal transmission patterns driving the long-term persistence and evolution of human coronaviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding the spread of different human coronaviruses can help us better understand how these viruses evolve over time. Scientists studied four types of coronaviruses that circulate in humans to see where they were present and how they changed over time. Their research found that two of these viruses, HCoV-229E and HCoV-OC43, are more likely to create new versions of themselves because they can easily change and spread from person to person.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Four distinct species of human coronaviruses (HCoVs) circulate in humans. Despite the recent attention due to SARS-CoV-2, a comprehensive understanding of the molecular epidemiology and genomic evolution of HCoVs remains unclear. Here, we employed primary differentiated human nasal epithelial cells for the successful isolation and genome sequencing of HCoVs derived from two retrospective cohorts in Singapore and Tanzania. Phylodynamic inference shows that HCoV-229E and HCoV-OC43 were subject to stronger genetic drift and reduced purifying selection from the early 2000s onwards, primarily targeting spike Domain A and B. This resulted in increased lineage diversification, coinciding with a higher effective reproductive number (Re>1.0). However, HCoV-NL63 and HCoV-HKU1 experienced weaker genetic drift and selective pressure with prolonged regional persistence. Our findings suggest that HCoV-229E and HCoV-OC43 viruses are adept at generating new variants and achieving widespread intercontinental dissemination driven by continuous genetic drift, recombination, and complex migration patterns.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00058-w" target="_blank">https://www.nature.com/articles/s44298-024-00058-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41598-024-74568-1" target="_blank">
            ü§ñ Understanding COVID-19's effect on physical health.

Activity levels in people after the pandemic
Participation levels in activities after the pandemic.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-23', this)">
            More</button></h4>
            <div id="toggle-23" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of post-COVID symptoms on activity and participation of women and men</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID syndrome is affecting millions worldwide and has become a major public health problem. A study investigated the relationship between symptoms and activity limitations for participants with post-COVID syndrome, particularly focusing on sex differences.

Researchers analyzed data from 307 patients who completed a questionnaire about their rehabilitation needs at different stages of recovery after COVID-19 infection. The results showed no difference in symptom intensity between men and women, but differences were found in the intensity of symptoms related to activity limitations (ALPR).

The study also discovered correlations between various factors and ALPR, including fatigue, which was associated with higher levels of ALPR across both sexes. However, different patterns emerged for women and men, suggesting that rehabilitation needs may vary depending on sex.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-COVID syndrome is affecting many organ systems and arises as a major public health problem with millions of cases worldwide. The primary aim of this study is the analysis of health problems, activity limitations and participation restrictions (ALPR) of participants with post-COVID symptoms and the investigation of correlations between these elements to derive statements about the rehabilitation need, also depending on sex. A retrospective cohort study was performed to collect longitudinal data from January 2022 to January 2023 using the Covid-19 Rehabilitation Needs Questionnaire (RehabNeQ). Patients completed the questionnaire at the Department of Rehabilitation- and Sports Medicine at Hannover Medical School. The 1st assessment included 307 study participants, of whom 54 showed up for the 2nd, 7 for the 3rd and one for the 4th assessment. Study participants with post-COVID symptoms also experience ALPR. The results show no significant difference in symptom intensity in women and men, but in intensity of ALPR. We found many correlations of varying degrees between various factors with ALPR. We found frequent correlations between fatigue and several ALPR. While these correlations apply to both sexes, we also found different correlations in women and men, indicating the different rehabilitation need of women and men.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74568-1" target="_blank">https://www.nature.com/articles/s41598-024-74568-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41598-024-76363-4" target="_blank">
            ü§ñ Malnutrition risk factors during COVID-19 illness</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-24', this)">
            More</button></h4>
            <div id="toggle-24" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Malnutrition is associated with severe outcome in elderly patients hospitalised with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Studies have found that some factors contribute to people being sick with COVID-19 as an elderly person, but research has not yet fully understood how these factors affect outcomes. Researchers looked at over 4,200 COVID-19 patients in China and found that those with poor nutritional status had a higher risk of severe illness. They also used mathematical models to see if better nutrition could predict who would get sick and why. The researchers concluded that PNI (a measure of nutritional health) is a good indicator of whether someone will get serious or not, and they suggest that doctors should pay close attention to patients' diets when treating COVID-19 cases in the elderly.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Some studies have identified influencing factors of COVID-19 illness in elderly, such as underlying diseases, but research on the effect of nutritional status is still lacking. This study retrospectively examined the influence of nutritional status on the outcome of elderly COVID-19 inpatients. A retrospective analysis of the clinical data of 4241 COVID-19 patients who were admitted to a third-class hospital of Nanchang between November 1, 2022 and January 31, 2023 was conducted. Nutritional status was assessed using the prognostic nutritional index (PNI) and controlling nutritional status score (CONUT). The influence of nutritional status on the outcome of COVID-19 patients was determined through multivariate adjustment analysis, restrictive cubic spline, and receiver operating characteristic curve (ROC). Compared with mild/no malnutrition, severe malnutrition substantially increased the critical outcome of COVID-19. A linear relationship was observed between the odds ratio (OR) and PNI and CONUT (P>‚Äâ0.05). The area under the ROC curve indicated that PNI was the better predictor. The optimal cutoff value of PNI was 38.04 (95%CI: 0.797‚Äâ~‚Äâ0.836, AUC‚Äâ=‚Äâ0.817), with a sensitivity of 70.7% and a specificity of 79.6%. The critical illness of elderly COVID-19 patients shows a linear relationship with malnutrition at admission. The use of PNI to assess the prognosis of COVID-19 eldeely patients is reliable, highlighting the importance for doctors to closely pay attention to the nutritional status of COVID-19 patients. Focusing on nutritional status in clinical practice can effectively reduce the critical illness of elderly COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76363-4" target="_blank">https://www.nature.com/articles/s41598-024-76363-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-17</strong> <a href="https://www.nature.com/articles/s41467-024-53356-5" target="_blank">
            ü§ñ Infectious Diseases and Immune Response Research Center

Prevalent and Persistent New-Onset Autoantibodies in Mild to Severe COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-25', this)">
            More</button></h4>
            <div id="toggle-25" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalent and persistent new-onset autoantibodies in mild to severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Autoantibodies, which are antibodies that attack the body's own proteins, have been found in people with COVID-19. Researchers studied 525 healthcare workers and patients hospitalized with COVID-19 over a year-long period to understand how autoantibodies change and evolve in response to the virus. They found that new autoantibodies appeared after infection and remained present for at least a year. Autoantibodies also seemed to be linked to neuropsychiatric symptoms and severe illness following COVID-19. The researchers identified specific proteins on SARS-CoV-2 as targets for these new antibodies, suggesting a possible link between the virus's spike protein and autoimmune responses in some patients.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Autoantibodies have been shown to be implied in COVID-19 but the emerging autoantibody repertoire remains largely unexplored. We investigated the new-onset autoantibody repertoire in 525 healthcare workers and hospitalized COVID-19 patients at five time points over a 16-month period in 2020 and 2021 using proteome-wide and targeted protein and peptide arrays. Our results show that prevalent new-onset autoantibodies against a wide range of antigens emerged following SARS-CoV-2 infection in relation to pre-infectious baseline samples and remained elevated for at least 12 months. We found an increased prevalence of new-onset autoantibodies after severe COVID-19 and demonstrated associations between distinct new-onset autoantibodies and neuropsychiatric symptoms post-COVID-19. Using epitope mapping, we determined the main epitopes of selected new-onset autoantibodies, validated them in independent cohorts of neuro-COVID and pre-pandemic healthy controls, and identified sequence similarities suggestive of molecular mimicry between main epitopes and the conserved fusion peptide of the SARS-CoV-2 Spike glycoprotein. Our work describes the complexity and dynamics of the autoantibody repertoire emerging with COVID-19 and supports the need for continued analysis of the new-onset autoantibody repertoire to elucidate the mechanisms of the post-COVID-19 condition.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53356-5" target="_blank">https://www.nature.com/articles/s41467-024-53356-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41421-024-00734-4" target="_blank">
            ü§ñ Single-cell analysis of lung tissue shows role of Slamf9+ macrophages in clearing viruses and resolving inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-26', this)">
            More</button></h4>
            <div id="toggle-26" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Single-cell spatiotemporal analysis of the lungs revealsSlamf9+macrophages involved in viral clearance and inflammation resolution</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The lungs' ability to restore health after a severe infection is not fully understood. Researchers studied how the lungs heal from pneumonia caused by SARS-CoV-2 in hamsters, finding that certain immune cells play a key role in clearing the virus. These immune cells contain the virus and work together with other types of immune cells to reduce inflammation and repair damaged tissue.</p><br>
                <p><strong>Abstract</strong></p>
                <p>How the lung achieves immune homeostasis after a pulmonary infection is not fully understood. Here, we analyzed the spatiotemporal changes in the lungs over a 2-week natural recovery from severe pneumonia in a Syrian hamster model of SARS-CoV-2 infection. We find that SARS-CoV-2 infects multiple cell types and causes massive cell death at the early stage, including alveolar macrophages. We identify a group of monocyte-derivedSlamf9+macrophages, which are induced after SARS-CoV-2 infection and resistant to impairment caused by SARS-CoV-2.Slamf9+macrophages contain SARS-CoV-2, recruit and interact withIsg12+Cst7+neutrophils to clear the viruses. After viral clearance,Slamf9+macrophages differentiate intoTrem2+andFbp1+macrophages, contributing to inflammation resolution at the late stage, and finally replenish alveolar macrophages. These findings are validated in a SARS-CoV-2-infected hACE2 mouse model and confirmed with publicly available human autopsy single-cell RNA-seq data, demonstrating the potential role ofSlamf9+macrophages and their coordination with neutrophils in post-injury tissue repair and inflammation resolution.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00734-4" target="_blank">https://www.nature.com/articles/s41421-024-00734-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41598-024-75338-9" target="_blank">
            ü§ñ Kidney function impacted by SARS-CoV-2 virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-27', this)">
            More</button></h4>
            <div id="toggle-27" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection causes a decline in renal megalin expression and affects vitamin D metabolism in the kidney of K18-hACE2 mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Patients with COVID-19 often experience damage to their kidneys and can have trouble processing certain vitamins, including vitamin D. Research has investigated how severe COVID-19 affects this process, and found that the virus can reduce the amount of megalin in the kidneys' tubules. This protein plays a role in the body's ability to absorb vitamin D from food. The study looked at whether COVID-19 infection reduced megalin levels in mice with human genes for the enzyme that helps convert one type of vitamin into another, and found that it did. The research suggests that even if the amount of vitamin D is not changed, the virus can still affect how the kidneys handle it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Patients with coronavirus disease 2019 (COVID-19) often experience acute kidney injury, linked to disease severity or mortality, along with renal tubular dysfunction and megalin loss in proximal tubules. Megalin plays a crucial role in kidney vitamin D metabolism. However, the impact of megalin loss on vitamin D metabolism during COVID-19 is unclear. This study investigated whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection reduces megalin expression in proximal tubules and its subsequent effect on vitamin D metabolism in mice expressing human angiotensin converting enzyme 2 (K18-hACE2 mice). Histological and immunohistochemical staining analyses revealed glomerular and capillary congestion, and elevated renal neutrophil gelatinase-associated lipocalin levels, indicative of acute kidney injury in K18-hACE2 mice. In SARS-CoV-2-infected mice, immunohistochemical staining revealed suppressed megalin protein levels. Decreased vitamin D receptor (VDR) localization in the nucleus and increased mRNA expression of VDR, CYP27B1, and CYP24A1 were observed by quantitative PCR in SARS-CoV-2-infected mice. Serum vitamin D levels remained similar in infected and vehicle-treated mice, but an increase in tumor necrosis factor-alpha and a decrease in IL-4 mRNA expression were observed in the kidneys of the SARS-CoV-2 group. These findings suggest that megalin loss in SARS-CoV-2 infection may impact the local role of vitamin D in kidney immunomodulation, even when blood vitamin D levels remain unchanged.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75338-9" target="_blank">https://www.nature.com/articles/s41598-024-75338-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41598-024-76163-w" target="_blank">
            ü§ñ Self-Proven Covid-19 Vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-28', this)">
            More</button></h4>
            <div id="toggle-28" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A self-adjuvanted VLPs-based Covid-19 vaccine proven versatile, safe, and highly protective</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A team developed a new type of COVID-19 vaccine that uses tiny particles from bacteria to fight the virus. They created these particles using a special system and purified them by removing toxins, making it safe for use. The vaccine was tested in animals and found to be effective at stopping the infection without causing harm. These findings suggest that this vaccine could be used to protect against COVID-19 with potentially lower costs than existing vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccination has played a critical role in mitigating COVID-19. Despite the availability of licensed vaccines, there remains a pressing need for improved vaccine platforms that provide high protection, safety, and versatility, while also reducing vaccine costs. In response to these challenges, our aim is to create a self-adjuvanted vaccine against SARS-CoV-2, utilizing Virus-Like Particles (VLPs) as the foundation. To achieve this, we produced bacteriophage (Q≈í‚â§) VLPs in a prokaryotic system and purified them using a rapid and cost-effective strategy involving organic solvents. This method aims to solubilize lipids and components of the cell membrane to eliminate endotoxins present in bacterial samples. For vaccine formulation, Receptor Binding Domain (RBD) antigens were conjugated using chemical crosslinkers, a process compatible with Good Manufacturing Practice (GMP) standards. Transmission Electron Microscopy (TEM) confirmed the expected folding and spatial configuration of the Q≈í‚â§VLPs vaccine. Additionally, vaccine formulation assessment involved SDS-PAGE stained with Coomassie Brilliant Blue, Western blotting, and stereomicroscopic experiments. In vitro and in vivo evaluations of the vaccine formulation were conducted to assess its capacity to induce a protective immune response without causing side effects. Vaccine doses of 20¬¨‚Ä†¬¨¬µg and 50¬¨‚Ä†¬¨¬µg stimulated the production of neutralizing antibodies. In in vivo testing, the group of animals vaccinated with 50¬¨‚Ä†¬¨¬µg of vaccine formulation provided complete protection against virus infection, maintaining stable body weight without showing signs of disease. In conclusion, the Q≈í‚â§VLPs-RBD vaccine has proven to be effective and safe, eliminating the necessity for supplementary adjuvants and offering a financially feasible approach. Moreover, this vaccine platform demonstrates flexibility in targeting Variants of Concern (VOCs) via established conjugation protocols with VLPs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76163-w" target="_blank">https://www.nature.com/articles/s41598-024-76163-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41541-024-00981-2" target="_blank">
            ü§ñ Orf Vaccine Protects Hamsters and Nonhuman Primates Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-29', this)">
            More</button></h4>
            <div id="toggle-29" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A multiantigenic Orf virus-based vaccine efficiently protects hamsters and nonhuman primates against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>## Next-generation COVID-19 vaccines involve designing vaccines that can effectively combat emerging variants of concern.

A new approach is being explored using an Orf virus-based vector to create multiantigenic vaccines targeting SARS-CoV-2 spike and nucleocapsid antigens.

## The vaccines were engineered to express either the spike protein (ORFV-S) or both the spike and nucleocapsid proteins (ORFV-S/N).

Studies showed that these vaccines triggered similar levels of antibodies against spike proteins and neutralized viruses in mice, but the multiantigenic vaccine performed better in a challenge model, providing long-term protection against SARS-CoV-2 infection.

## These results suggest the potential of the Orf virus-based platform for prophylactic vaccination and support further development into clinical trials.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Among the common strategies to design next-generation COVID-19 vaccines is broadening the antigenic repertoire thereby aiming to increase efficacy against emerging variants of concern (VoC). This study describes a new Orf virus-based vector (ORFV) platform to design a multiantigenic vaccine targeting SARS-CoV-2 spike and nucleocapsid antigens. Vaccine candidates were engineered, either expressing spike protein (ORFV-S) alone or co-expressing nucleocapsid protein (ORFV-S/N). Mono- and multiantigenic vaccines elicited comparable levels of spike-specific antibodies and virus neutralization in mice. Results from a SARS-CoV-2 challenge model in hamsters suggest cross-protective properties of the multiantigenic vaccine against VoC, indicating improved viral clearance with ORFV-S/N, as compared to equal doses of ORFV-S. In a nonhuman primate challenge model, vaccination with the ORFV-S/N vaccine resulted in long-term protection against SARS-CoV-2 infection. These results demonstrate the potential of the ORFV platform for prophylactic vaccination and represent a preclinical development program supporting first-in-man studies with the multiantigenic ORFV vaccine.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00981-2" target="_blank">https://www.nature.com/articles/s41541-024-00981-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41541-024-00990-1" target="_blank">
            ü§ñ ## Novel Vaccine Adjuvant Enhances Protection Against Pertussis

## Adjuvants help vaccines work better by stimulating the immune system.

## They usually contain tiny particles that trigger an immune response, making it easier to develop immunity to specific diseases.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-30', this)">
            More</button></h4>
            <div id="toggle-30" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel outer membrane vesicle adjuvant improves vaccine protection againstBordetella pertussis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

A new vaccine is being developed to prevent the respiratory disease caused by Bordetella pertussis. The vaccine is designed to stimulate an immune response in the nose and throat, where it can eliminate the bacteria that cause pertussis. Researchers tested the vaccine on mice and found that it not only protected them from getting sick but also eliminated the bacteria in their lungs and nasal passages. This suggests that a new approach may be effective in preventing pertussis infection.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Pertussis is a vaccine-preventable respiratory disease caused by the Gram negative coccobacillus Bordetella pertussis. The licensed acellular pertussis (aP) vaccines protect against disease but do not prevent bacterial colonization and transmission. Here, we developed and tested an intranasal vaccine composed of aP antigens combined with T-vant, a novel adjuvant derived from bacterial outer membrane vesicles, that elicits both mucosal and systemic immune responses. We hypothesized that immunization of mice with aP-T-vant would enhance mucosal immunity and eliminate B. pertussis in the respiratory tract. In contrast to mice immunized intramuscularly with the licensed aP vaccine, intranasal immunization with aP-T-vant eliminated bacteria in both the lung and nasopharynx. Protection was associated with IFN-gamma and IL-17-producing, non-circulating CD4‚Äâ+‚ÄâT cells in the lung and nasopharynx, and sterilizing immunity in the nasopharynx was dependent on IL-17. Novel mucosal adjuvants, such as T-vant, warrant further investigation to enhance the efficacy of next generation pertussis vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00990-1" target="_blank">https://www.nature.com/articles/s41541-024-00990-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-16</strong> <a href="https://www.nature.com/articles/s41421-024-00733-5" target="_blank">
            ü§ñ Single-cell immune system research reveals specific areas where immune cells work together to fight infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-31', this)">
            More</button></h4>
            <div id="toggle-31" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Single-cell spatiotemporal analysis reveals alveolar dendritic cell‚ÄìT cell immunity hubs defending against pulmonary infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new study has looked at how immune cells work together in the lungs to fight off infections and inflammation caused by a virus like SARS-CoV-2. They used a high-tech method called immunocartography to analyze how these immune cells interacted and responded over time. The researchers studied Syrian hamster models that were infected with SARS-CoV-2 and found that specific types of immune cells worked together to control the infection, eliminate the virus, and then recover when the virus was gone. They also looked at how this process worked in normal lungs and similar human lungs to see if their findings can be applied to real-life lung health.</p><br>
                <p><strong>Abstract</strong></p>
                <p>How immune cells are spatiotemporally coordinated in the lung to effectively monitor, respond to, and resolve infection and inflammation in primed form needs to be fully illustrated. Here we apply immunocartography, a high-resolution technique that integrates spatial and single-cell RNA sequencing (scRNA-seq) through deconvolution and co-localization analyses, to the SARS-CoV-2-infected Syrian hamster model. We generate a comprehensive transcriptome map of the whole process of pulmonary infection from physiological condition, infection initiation, severe pneumonia to natural recovery at organ scale and single-cell resolution, with 142,965 cells and 45 lung lobes from 25 hamsters at 5 time points. Integrative analysis identifies that alveolar dendritic cell‚ÄìT cell immunity hubs, whereCcr7+Ido1+dendritic cells,Cd160+Cd8+T cells, andTnfrsf4+Cd4+T cells physiologically co-localize, rapidly expand during SARS-CoV-2 infection, eliminate SARS-CoV-2 with the aid ofSlamf9+macrophages, and then restore to physiological levels after viral clearance. We verify the presence of these cell subpopulations in the immunity hubs in normal and SARS-CoV-2-infected hACE2 mouse models, as well as in publicly available human scRNA-seq datasets, demonstrating the potential broad relevance of our findings in lung immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41421-024-00733-5" target="_blank">https://www.nature.com/articles/s41421-024-00733-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-76013-9" target="_blank">
            ü§ñ COVID-19 Patient Discharge Outcomes in Guangdong During the Pandemic Wave</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-32', this)">
            More</button></h4>
            <div id="toggle-32" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The short-term and long-term prognosis of discharged COVID-19 patients in Guangdong during the first wave of pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study investigated the effects of COVID-19 on patients' mental and physical health after recovery from the illness. Researchers followed 281 people who had recovered from the virus to see how their mental and physical well-being changed over time. They used various questionnaires and medical tests to measure these changes. The results showed that many people experienced significant psychological and physical problems, even years after recovering from COVID-19. Factors such as age, gender, severity of illness, hospitalization duration, and previous health conditions were linked to these outcomes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 survivors concerning about the rehabilitation and sustained sequelae of Coronavirus Disease 2019 (COVID-19) infection. We aimed to investigate the sequelae of patients‚Äô psychological and physical condition and its related factors in the early and late stages. This longitudinal study tracked 281 COVID-19 patients discharged from hospitals in Guangdong, China, for one year. Assessments occurred at 2,4,12,24 and 48 weeks post-discharge. We define 2 weeks, 4 weeks, and 12 weeks as early stage, and 24 weeks and 48 weeks as late stage. Psychological health was measured using the Patient Health Questionnaire-9 (PHQ-9), Generalized Anxiety Disorder-7 (GAD-7), Posttraumatic Stress Disorder Checklist for DSM-5 (PCL-5), and Pittsburgh Sleep Quality Index (PSQI) scales. Physical health was assessed through laboratory tests, chest computed tomography (CT) scans, and pulmonary function tests. Data were analyzed using multivariate regression models to evaluate the influence of demographic and clinical variables on health outcomes. COVID-19 survivors exhibited psychological and physical sequelae in both the early and late stages. Compared to the early stage, the proportions of patients with depression (early stage 14.6%, late stage 4.6%), anxiety (early stage 8.9%, late stage 5.3%), PTSD(early stage 3.6%, late stage 0.7%), abnormal liver function (early stage 24.6%, late stage 11.0%), abnormal cardiac function (early stage 10.0%, late stage 7.8%), abnormal renal function (early stage 20.6%, late stage 11.0%) and abnormal pulmonary function (early stage 40.9%, late stage 13.5%) were significantly reduced in the late stage. Factors such as gender, age, severity of COVID-19, hospitalization duration, and various comorbidities were significantly associated with these sequelae. We noticed that psychological and physical sequelae occurred to COVID-19 survivors in short and long stages, and these would gradually decrease as time went on. Male gender, age‚Äâ>‚Äâ50 years old, severe clinical condition, longer hospitalization time and comorbidity history were related factors that significantly affected the rehabilitation of COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-76013-9" target="_blank">https://www.nature.com/articles/s41598-024-76013-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41467-024-53029-3" target="_blank">
            ü§ñ SARS-CoV-2 Variants with Insertions and Specific Mutations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-33', this)">
            More</button></h4>
            <div id="toggle-33" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Sarbecovirus RBD indels and specific residues dictating multi-species ACE2 adaptiveness</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Sarbecoviruses are viruses that infect bats and other mammals, but their ability to interact with humans remains unclear. Scientists analyzed 268 sarbecovirus genetic sequences to understand how these viruses use human cells (ACE2 receptors). They found that some sarbecoviruses can adapt to different types of ACE2 on the surface of mammalian cells, while others have limited or no use for it. The scientists also discovered a specific pattern in the genetic codes of sarbecovirus proteins and their interaction with ACE2. This study provides new insights into how viruses interact with human cells and may help prevent future outbreaks.

I made the following changes:

* Simplified the language to make it easier to understand
* Removed quotation marks around the abstract text
* Kept the same structure and organization as the original abstract</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our comprehensive understanding of the multi-species ACE2 adaptiveness of sarbecoviruses remains elusive, particularly for those with various receptor binding motif (RBM) insertions/deletions (indels). Here, we analyzed RBM sequences from 268 sarbecoviruses categorized into four RBM indel types. We examined the ability of 20 representative sarbecovirus Spike glycoproteins (S) and derivatives in utilizing ACE2 from various bats and several other mammalian species. We reveal that sarbecoviruses with long RBMs (type-I) can achieve broad ACE2 tropism, whereas viruses with single deletions in Region 1 (type-II) or Region 2 (type-III) exhibit narrower ACE2 tropism. Sarbecoviruses with double region deletions (type-IV) completely lost ACE2 usage, which is restricted by clade-specific residues within and outside RBM. Lastly, we propose the evolution of sarbecovirus RBM indels and illustrate how loop lengths, disulfide, and residue determinants shape multi-species ACE2 adaptiveness. This study provides profound insights into the mechanisms governing ACE2 usage and spillover risks of sarbecoviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53029-3" target="_blank">https://www.nature.com/articles/s41467-024-53029-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-75345-w" target="_blank">
            ü§ñ Prevalence of Symptoms of Dry Eye Disease in Medical Students During a Pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-34', this)">
            More</button></h4>
            <div id="toggle-34" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Prevalence of dry eye disease symptoms, associated factors and impact on quality of life among medical students during the pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This cross-sectional study found that a significant number of medical students (60.4%) have dry eye disease (DED) during the COVID-19 pandemic. The prevalence was highest in women (61.5%). 

The study identified several factors that may contribute to the development of DED, including preexisting DED, refractive errors, allergic conjunctivitis, artificial tears use, and a lower EQ-5D-5L score. However, contact lens wear, screen time, and mask-wearing duration were not associated with DED.

Female medical students, people who wore contact lenses, and those who used artificial tears may be more likely to experience DED symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This cross-sectional study aimed to evaluate the prevalence of dry eye disease (DED) symptoms, their associated factors, and the impact on quality of life (QOL) and mental health among medical students during the COVID-19 pandemic using online surveys. Participants completed questionnaires covering demography, medical history, and personal data. The Thai version of the Dry Eye-related Quality-of-Life Score (DEQS-Th) was used for DED screening. The QOL and mental health challenges (support, coping, stress, and depression) were evaluated by the EuroQoL-5 dimensions-5 levels (EQ-5D-5L) and Thymometer questionnaires. A total of 449 participants were analyzed, with a mean age of 21.8 years and 61.5% female. The prevalence of DED symptoms was 60.4% (95%CI 55.7‚Äì64.8). The DEQS-Th score, the EQ-5D-5L score, and all aspects of mental health challenges were significantly worse in DED participants compared to non-DED participants. Associated factors for DED symptoms included preexisting DED (p=‚Äâ0.001), refractive errors (p=‚Äâ0.007), allergic conjunctivitis (p=‚Äâ0.001), artificial tears use (p<‚Äâ0.001), and decreased EQ-5D-5L score (p<‚Äâ0.001). This study highlighted the high prevalence of DED symptoms among medical students during the pandemic and its negative impact on QoL and mental health. Notably, female gender, contact lens wear, screen time, and mask-wearing duration were not associated with DED symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75345-w" target="_blank">https://www.nature.com/articles/s41598-024-75345-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-74221-x" target="_blank">
            ü§ñ ### Immune Response Modeling for Vaccines</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-35', this)">
            More</button></h4>
            <div id="toggle-35" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A mathematical model simulating the adaptive immune response in various vaccines and vaccination strategies</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

A new mathematical model was developed to study how vaccines work on our bodies, helping scientists create better vaccines for infectious diseases. Researchers looked at how different types of vaccines (like traditional inactivated ones or mRNA vaccines) affect the levels of antibodies our immune system produces after vaccination. They found that booster shots are essential for building up strong antibody defenses, and that different vaccine types can have pros and cons. The study provides a better understanding of how vaccines work and helps scientists design more effective vaccines to protect against diseases.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccination has been widely recognized as an effective measure for preventing infectious diseases. To facilitate quantitative research into the activation of adaptive immune responses in the human body by vaccines, it is important to develop an appropriate mathematical model, which can provide valuable guidance for vaccine development. In this study, we constructed a novel mathematical model to simulate the dynamics of antibody levels following vaccination, based on principles from immunology. Our model offers a concise and accurate representation of the kinetics of antibody response. We conducted a comparative analysis of antibody dynamics within the body after administering several common vaccines, including traditional inactivated vaccines, mRNA vaccines, and future attenuated vaccines based on defective interfering viral particles (DVG). Our findings suggest that booster shots play a crucial role in enhancing Immunoglobulin G (IgG) antibody levels, and we provide a detailed discussion on the advantages and disadvantages of different vaccine types. From a mathematical standpoint, our model proposes four essential approaches to guide vaccine design: enhancing antigenic T-cell immunogenicity, directing the production of high-affinity antibodies, reducing the rate of IgG decay, and lowering the peak level of vaccine antigen-antibody complexes. Our study contributes to the understanding of vaccine design and its application by explaining various phenomena and providing guidance in comprehending the interactions between antibodies and antigens during the immune process.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74221-x" target="_blank">https://www.nature.com/articles/s41598-024-74221-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s41598-024-74704-x" target="_blank">
            ü§ñ Comprehensive Maternal Deaths During COVID-19 Pandemic in Brazil Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-36', this)">
            More</button></h4>
            <div id="toggle-36" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Time series analysis of comprehensive maternal deaths in Brazil during the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic significantly impacted maternal mortality rates in Brazil between 2020 and 2021. Using data from two forecasting models, we calculated expected maternal mortality ratios (MMR) for 2020 and 2021 using real-life data since 2008. Our results show that predicted MMR was much higher than actual MMR observed in 2021, with a doubling of the difference between predictions and reality. This suggests that sub-national variations in comprehensive maternal death rates remain significant in Brazil.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The effects of the COVID-19 pandemic on comprehensive maternal deaths in Brazil have not been fully explored. Using publicly available data from the Brazilian Mortality Information (SIM) and Information System on Live Births (SINASC) databases, we used two complementary forecasting models to predict estimates of maternal mortality ratios using maternal deaths (MMR) and comprehensive maternal deaths (MMRc) in the years 2020 and 2021 based on data from 2008 to 2019. We calculated national and regional standardized mortality ratio estimates for maternal deaths (SMR) and comprehensive maternal deaths (SMRc) for 2020 and 2021. The observed MMRc in 2021 was more than double the predicted MMRc based on the Holt-Winters and autoregressive integrated moving average models (127.12 versus 60.89 and 59.12 per 100,000 live births, respectively). We found persisting sub-national variation in comprehensive maternal mortality: SMRc ranged from 1.74  (95% confidence interval [CI] 1.64, 1.86) in the Northeast to 2.70 (95% CI 2.45, 2.96) in the South in 2021. The observed national estimates for comprehensive maternal deaths in 2021 were the highest in Brazil in the past three decades. Increased resources for prenatal care, maternal health, and postpartum care may be needed to reverse the national trend in comprehensive maternal deaths.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74704-x" target="_blank">https://www.nature.com/articles/s41598-024-74704-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-14</strong> <a href="https://www.nature.com/articles/s42003-024-07025-4" target="_blank">
            ü§ñ Regulation of Immune Response in COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-37', this)">
            More</button></h4>
            <div id="toggle-37" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>FPR1 signaling aberrantly regulates S100A8/A9 production by CD14+FCN1himacrophages and aggravates pulmonary pathology in severe COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Excessive inflammation caused by certain proteins, known as alarmins, can worsen immune responses in people infected with SARS-CoV-2. Researchers studied how these proteins affect macrophages (a type of white blood cell) in the lungs after infection and found that they were activated to produce more inflammatory chemicals. They also identified a protein called FPR1 that plays a key role in this process, leading them to hypothesize that it could be targeted for treating severe COVID-19 and post-Acute Respiratory Syndrome (PACS).</p><br>
                <p><strong>Abstract</strong></p>
                <p>Excessive alarmins S100A8/A9 escalate the inflammation and even exacerbate immune-driven thrombosis and multi-organ damage. However, the regulatory mechanisms of S100A8/A9 expression in infectious diseases remain unclear. In this study, high-dimensional transcriptomic data analyses revealed a high proportion of CD14+FCN1himacrophages within the pulmonary niche post-severe SARS-CoV-2 infection. By constructing the S100-coexpression gene list and supervised module scoring, we found that CD14+FCN1himacrophages presented the highest scores of alarmin S100, and possibly served as the trigger and amplifier of inflammation in severe COVID-19. These CD14+FCN1hicells lacked the positive regulatory activity of transcription factor PPARŒ≥, and lost their differentiation ability towards mature macrophages. Ex vivo experiments further validated that the epithelial cells with high ORF-3a expression promoted the expression and secretion of S100A8/A9 through ANXA1/SAA1-FPR1 signaling. S100A8/A9 heterodimers, as well as the co-localization of S100A8/A9 with microtubules, were both diminished by the FPR1 inhibitor. Phospho-kinase protein array indicated that STAT3 promoted transcription, and PLC-Œ≥ and ERK1/2 pathways were involved in the hetero-dimerization and unconventional secretion of S100A8/A9. Our study highlights the pivotal role of FPR1 signaling in the excessive production of S100A8/A9 and provides a promising target for the prevention and control of severe COVID-19 and post-acute COVID-19 sequelae.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-07025-4" target="_blank">https://www.nature.com/articles/s42003-024-07025-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-13</strong> <a href="https://www.nature.com/articles/s41598-024-75273-9" target="_blank">
            ü§ñ COVID-19 Vaccine Uptake Analysis

Analyzing how education level affects COVID-19 vaccine adoption in Germany, using area-level socioeconomic deprivation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-38', this)">
            More</button></h4>
            <div id="toggle-38" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Reconsidering inequalities in COVID-19 vaccine uptake in Germany: a spatiotemporal analysis combining individual educational level and area-level socioeconomic deprivation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study looked at how different groups received vaccines during the first year of a vaccine campaign in Germany. The researchers found that people with lower levels of education were less likely to get vaccinated, and those living in areas with higher socioeconomic deprivation also had lower vaccination rates. The study found these differences continued over time, even after the pandemic was under control.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Combining the frameworks of fundamental causes theory and diffusion of innovation, scholars had anticipated a delayed COVID-19 vaccination uptake for people in lower socioeconomic position depending on the socioeconomic context. We qualify these propositions and analyze educational differences in COVID-19 vaccination status over the first ten months of Germany‚Äôs vaccination campaign in 2021. Data from the study ‚ÄúCorona Monitoring Nationwide‚Äù (RKI-SOEP-2), collected between November 2021 and February 2022, is linked with district-level data of the German Index of Socioeconomic Deprivation (GISD). We estimated the proportion of people with at least one vaccination dose stratified by educational groups and within different settings of regional socioeconomic deprivation at three time points. Logistic multilevel regression models were applied to adjust for multiple covariates and to test cross-level-interactions between educational levels and levels of area-level socioeconomic deprivation. Vaccination rates were lower among respondents with lower education. With increasing area-level socioeconomic deprivation, educational differences were larger due to particularly low vaccination rates in groups with low education levels. The analysis of vaccination timing reveals that educational gaps and gaps by area-level socioeconomic deprivation had appeared early in the vaccination campaign and did not close completely before the 4th wave of COVID-19 infections</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75273-9" target="_blank">https://www.nature.com/articles/s41598-024-75273-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-11</strong> <a href="https://www.nature.com/articles/s41598-024-75833-z" target="_blank">
            ü§ñ Observational Study on Vaccine Doses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-39', this)">
            More</button></h4>
            <div id="toggle-39" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Side effects associated with homogenous and heterogenous doses of Oxford‚ÄìAstraZeneca vaccine among adults in Bangladesh: an observational study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>**Assessment of Side Effects in COVID-19 Vaccination**

In a large-scale study of over 1805 people who received COVID-19 vaccine doses in Bangladesh, researchers found significant side effects associated with vaccination. The study involved administering multiple doses to participants and observing the severity of local and systemic reactions.

The most common side effects reported were pain at the injection site, swelling, redness, and heaviness in the hand, fever, headache, muscle pain, chills, and myalgia. These symptoms appeared within 48 hours after vaccination. Thrombosis was only reported among a small group of people who received Oxford-AstraZeneca vaccine.

Systemic side effects, such as chronic urticaria (itching) and psoriasis, were also observed in some participants. The study found that the risk of severe and long-term side effects increased with each subsequent dose of vaccination. Women over 60 years old and those with pre-existing health conditions had higher risks.

The researchers analyzed data from multiple vaccination doses and found that Oxford-AstraZeneca vaccine was associated with a significantly higher rate of adverse effects than the other vaccines (P < .05). They also noted that homogenous doses of Oxford-AstraZeneca and heterogenous doses of Moderna and Pfizer-BioNTech were more likely to have elevated side effect rates.

This study provides insight into the safety profile of COVID-19 vaccines, highlighting potential risks associated with vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Assessment of side effects associated with COVID-19 vaccination is required to monitor safety issues and acceptance of vaccines in the long term. We found a significant knowledge gap in the safety profile of COVID-19 vaccines in Bangladesh. We enrolled 1805 vaccine recipients from May 5, 2021, to April 4, 2023. Kruskal-Wallis test and œá2 test were performed. Multivariable logistic regression was also performed. First, second and third doses were administered among 1805, 1341, and 923 participants, respectively. Oxford‚ÄìAstraZeneca (2946 doses) was the highest administered followed by Sinopharm BIBP (551 doses), Sinovac (214 doses), Pfizer-BioNTech (198 doses), and Moderna (160 doses), respectively. Pain at the injection site (80-90%, 3200‚Äì3600), swelling (85%, 3458), redness (78%, 3168), and heaviness in hand (65%, 2645) were the most common local effects, and fever (85%, 3458), headache (82%, 3336), myalgia (70%, 2848), chills (67%, 2726), muscle pain (60%, 2441) were the most prevalent systemic side effects reported within 48¬†h of vaccination. Thrombosis was only reported among the Oxford‚ÄìAstraZeneca recipients (3.5-5.7%). Both local and systemic effects were significantly associated with the Oxford‚ÄìAstraZeneca (p-value<‚Äâ0.05), Pfizer‚ÄìBioNTech (p-value<‚Äâ0.05), and Moderna (p-value<‚Äâ0.05) vaccination. Chronic urticaria and psoriasis were reported by 55-60% of the recipients after six months or later. The highest percentage of local and systemic effects after 2nd and 3rd dose were found among recipients of Moderna followed by Pfizer-BioNTech and Oxford‚ÄìAstraZeneca. Homogenous doses of Oxford‚ÄìAstraZeneca and heterogenous doses of Moderna and Pfizer-BioNTech were significantly associated with elevated adverse effects. Females, aged above 60 years with preexisting health conditions had higher risks. Vaccination with Pfizer-BioNTech (OR 4.34, 95% CI 3.95‚Äì4.58) had the highest odds of severe and long-term effects followed by Moderna (OR 4.15, 95% CI 3.92‚Äì4.69) and Oxford‚ÄìAstraZeneca (OR 3.89, 95% CI 3.45‚Äì4.06), respectively. This study will provide an integrated insight into the safety profile of COVID-19 vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75833-z" target="_blank">https://www.nature.com/articles/s41598-024-75833-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-10</strong> <a href="https://www.nature.com/articles/s41467-024-53105-8" target="_blank">
            ü§ñ Protein sequencing of antibodies to identify those that provide protection against COVID-19 in vaccinated individuals.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-40', this)">
            More</button></h4>
            <div id="toggle-40" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>De novo protein sequencing of antibodies for identification of neutralizing antibodies in human plasma post SARS-CoV-2 vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists have developed a way to sequence (read) individual proteins in blood plasma, known as immunoglobulins or antibodies. They used this method to look at how the human body responds to a vaccine against COVID-19. By sequencing these proteins, they were able to identify 12 different antibodies produced by the immune system in response to the vaccine. These antibodies have shown promising properties, such as neutralizing the target virus, which could lead to more effective treatments and vaccines in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The antibody response to vaccination and infection is a key component of the immune response to pathogens. Sequencing of peripheral B cells may not represent the complete B cell receptor repertoire. Here we present a method for sequencing human plasma-derived polyclonal IgG using a combination of mass spectrometry and B-cell sequencing. We investigate the IgG response to the Moderna Spikevax COVID-19 vaccine. From the sequencing data of the natural polyclonal response to vaccination, we generate 12 recombinant antibodies. Six derived recombinant antibodies, including four generated with de novo protein sequencing, exhibit similar or higher binding affinities than the original natural polyclonal antibody. Neutralization tests reveal that the six antibodies possess neutralizing capabilities against the target antigen. This research provides insights into sequencing polyclonal IgG antibodies and the potential of our approach in generating recombinant antibodies with robust binding affinity and neutralization capabilities. Directly examining the circulating IgG pool is crucial due to potential misrepresentations by B-cell analysis alone.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53105-8" target="_blank">https://www.nature.com/articles/s41467-024-53105-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41467-024-52890-6" target="_blank">
            ü§ñ Genome-Wide Study on COVID-19 Vaccine Seroconversion and Breakthrough Outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-41', this)">
            More</button></h4>
            <div id="toggle-41" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Genome-wide association studies of COVID-19 vaccine seroconversion and breakthrough outcomes in UK Biobank</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Understanding how the immune system responds to different COVID-19 vaccine types is crucial for finding out which genes are most important. This study used a large database of genetic information from people who were vaccinated with different vaccines to find out which specific parts of their DNA affect whether they get better or worse symptoms after vaccination, and how severe those symptoms are. The results showed that certain areas on the immune system respond differently depending on the vaccine type. Additionally, some regions are more likely to cause severe symptoms even when vaccinated. This study helps scientists understand how vaccines work and what other genetic factors may influence how people respond to different types of COVID-19 vaccinations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Understanding the genetic basis of COVID-19 vaccine seroconversion is crucial to study the role of genetics on vaccine effectiveness. In our study, we used UK Biobank data to find the genetic determinants of COVID-19 vaccine-induced seropositivity and breakthrough infections. We conducted four genome-wide association studies among vaccinated participants for COVID-19 vaccine seroconversion and breakthrough susceptibility and severity. Our findings confirmed a link between theHLAregion and seroconversion after the first and second doses. Additionally, we identified 10 genomic regions associated with breakthrough infection (SLC6A20, ST6GAL1, MUC16, FUT6, MXI1, MUC4,HMGN2P18-KRTCAP2,NFKBIZandAPOC1), and one with breakthrough severity (APOE). No significant evidence of genetic colocalisation was found between those traits. Our study highlights the roles of individual genetic make-up in the varied antibody responses to COVID-19 vaccines and provides insights into the potential mechanisms behind breakthrough infections occurred even after the vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52890-6" target="_blank">https://www.nature.com/articles/s41467-024-52890-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-74629-5" target="_blank">
            ü§ñ Post-infectious Olfactory Dysfunction in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-42', this)">
            More</button></h4>
            <div id="toggle-42" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Comparative analyses of post-infectious olfactory dysfunction between COVID-19 and non-COVID-19 cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Research identifies differences between people who develop olfactory dysfunction (OD) after a COVID-19 infection and those without it.

Demographic and clinical characteristics of patients with PIOD are analyzed to compare their experiences. The study found that:

* A higher proportion of COVID-19 cases were younger, more likely male, and had parosmia.
* Initial olfactory function tests showed differences between the groups.
* Treatment improved odor detection threshold but not identification in COVID-19-associated OD.
* Non-COVID-19-associated OD had better initial results.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To identify the differences between COVID-19-associated and non-COVID-19-associated olfactory dysfunction (OD), we analyzed demographic and clinical characteristics based on the causative virus (COVID versus non-COVID groups) in patients with post-infectious olfactory dysfunction (PIOD) who underwent the olfactory questionnaire and olfactory function test. Out of 169 patients with PIOD, 99 were diagnosed with COVID-19 (COVID group), while 70 were not (non-COVID group). The COVID group was younger and had a higher percentage of male patients as well as patients with parosmia than the non-COVID group. In the initial olfactory function tests, the TDI, discrimination and identification scores were significantly higher in the COVID group than in the non-COVID group. TDI scores were significantly increased in patients with PIOD after treatment, regardless of the group. The threshold score was significantly increased by 1.38 in the COVID group while the identification score was significantly increased by 2.67 in the non-COVID group. Patients with COVID-19-associated OD were younger in age, tended to be male, had a higher incidence of parosmia, and had better initial olfactory function test results compared to those with non-COVID-19-associated OD. Following treatment, odor detection threshold improved in the COVID group, whereas odor identification improved in the non-COVID group.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74629-5" target="_blank">https://www.nature.com/articles/s41598-024-74629-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-75093-x" target="_blank">
            ü§ñ Enhanced immune function in COVID-19 patients through transfusion of convalescent plasma with high viral titres.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-43', this)">
            More</button></h4>
            <div id="toggle-43" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Biochemical rationale for transfusion of high titre COVID-19 convalescent plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We developed an immunosorbent assay to measure how well donor antibodies bind to viruses that cause COVID-19 after receiving plasma from people who have recovered from the disease. We found two different affinities of these antibodies: 80% for IgG and about half for IgA. The binding strength was highest at a concentration of around 150 nanomoles per milliliter, which is higher than what's needed to be effective.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We aimed to model binding of donor antibodies to virus that infects COVID-19 patients following transfusion of convalescent plasma (CCP). An immunosorbent assay was developed to determine apparent affinity (Kd, app). Antibody binding to virus was modelled using antibody concentration and estimations of viral load. Assay and model were validated using reference antibodies and clinical data of monoclonal antibody therapy. A single Kd, appor two resolvable Kd, appwere found for IgG in 11% or 89% of CCP donations, respectively. For IgA this was 50%-50%. Median IgG Kd, appwas 0.8nM and 3.6nM for IgA, ranging from 0.1-14.7nM and 0.2-156.0nM respectively. The median concentration of IgG was 44.0nM (range 8.4-269.0nM) and significantly higher than IgA at 2.0nM (range 0.4-11.4nM). The model suggested that a double CCP transfusion (i.e. 500 mL) allows for >‚Äâ80% binding of antibody to virus provided Kd, appwas <‚Äâ1nM and concentration‚Äâ>‚Äâ150nM. In our cohort from the pre-vaccination era, 4% of donations fulfilled these criteria. Low and mid-range viral loads are found early post exposure, suggesting that convalescent plasma will be most effective then. This study provides a biochemical rationale for selecting high affinity and high antibody concentration CCP transfused early in the disease course.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75093-x" target="_blank">https://www.nature.com/articles/s41598-024-75093-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41467-024-53033-7" target="_blank">
            ü§ñ Co-infection with SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3.

Lineage-specific pathogenicity, immune evasion, and virological features of these two strains.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-44', this)">
            More</button></h4>
            <div id="toggle-44" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Lineage-specific pathogenicity, immune evasion, and virological features of SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 JN.1 variant has higher infectivity in human nasal epithelial cells compared to its parent BA.2.86. This variation associates with increased entry efficiency and better spike cleavage. It alters the mode of binding between the protein and ACE2, and increases hydrogen bonds with neighboring residues, making it easier for the virus to enter cells.

This variant is more transmissible than BA.2.86, suggesting that a single mutation makes it harder for people's immune systems to fight off. By studying this variation in both laboratory tests and actual human viruses, scientists can gain better understanding of its effects on different parts of the body and how it contributes to SARS-CoV-2's spread.

A key feature of this variant is an extra amino acid (L455S) that allows it to enter cells more easily. This mutation likely makes people less able to fight off the virus, which can contribute to its dominance over other variants. Further research will help scientists better understand how SARS-CoV-2 JN.1 and its parent BA.2.86 work together to spread disease globally.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 JN.1 with an additional L455S mutation on spike when compared with its parental variant BA.2.86 has outcompeted all earlier variants to become the dominant circulating variant. Recent studies investigated the immune resistance of SARS-CoV-2 JN.1 but additional factors are speculated to contribute to its global dominance, which remain elusive until today. Here, we find that SARS-CoV-2 JN.1 has a higher infectivity than BA.2.86 in differentiated primary human nasal epithelial cells (hNECs). Mechanistically, we demonstrate that the gained infectivity of SARS-CoV-2 JN.1 over BA.2.86 associates with increased entry efficiency conferred by L455S and better spike cleavage in hNECs. Structurally, S455 altered the mode of binding of JN.1 spike protein to ACE2 when compared to BA.2.86 spike at ACE2H34, and modified the internal structure of JN.1 spike protein by increasing the number of hydrogen bonds with neighboring residues. These findings indicate that a single mutation (L455S) enhances virus entry in hNECs and increases immune evasiveness, which contribute to the robust transmissibility of SARS-CoV-2 JN.1. We further evaluate the in vitro and in vivo virological characteristics between SARS-CoV-2 BA.2.86/JN.1 and EG.5.1/HK.3, and identify key lineage-specific features of the two Omicron sublineages that contribute to our understanding on Omicron antigenicity, transmissibility, and pathogenicity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-53033-7" target="_blank">https://www.nature.com/articles/s41467-024-53033-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41541-024-00972-3" target="_blank">
            ü§ñ COVID-19 vaccine response and transmission risk in people with HIV</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-45', this)">
            More</button></h4>
            <div id="toggle-45" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Vaccine responses and hybrid immunity in people living with HIV after SARS-CoV-2 breakthrough infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic presented a challenge for people with HIV (PLWH), especially those who have weakened immune systems due to antiretroviral therapy (ART). The virus can cause significant problems, including an increased risk of breakthrough infections (BTI) from the SARS-CoV-2 virus. A new study found that PLWH had reduced responses to vaccines and may not be protected against BTI. However, the immune system's response changed when it comes to the COVID-19 vaccine. This change is significant because people with weakened immune systems can still fight off the virus even if their immune response isn't as strong. The study suggests that additional boosters might help and that regular monitoring of HIV treatment is necessary to ensure that both humoral (vaccine-related) and cellular immunity remains effective against the SARS-CoV-2 virus in people with weakened immune systems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic posed a challenge for people living with HIV (PLWH), particularly immune non-responders (INR) with compromised CD4 T-cell reconstitution following antiretroviral therapy (CD4 count <350 cells per mm3). Their diminished vaccine responses raised concerns about their vulnerability to SARS-CoV-2 breakthrough infections (BTI). Our in-depth study here revealed chronic inflammation in PLWH and a limited anti-Spike IgG response after vaccination in INR. Nevertheless, the imprinting of Spike-specific B cells by vaccination significantly enhanced the humoral responses after BTI. Notably, the magnitude of cellular CD4 response in all PLWH was comparable to that in healthy donors (HD). However, the polyfunctionality and phenotype of Spike-specific CD8 T cells in INR differed from controls. The findings highlight the need for additional boosters with variant vaccines, and for monitoring ART adherence and the durability of both humoral and cellular anti-SARS-CoV-2 immunity in INR.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00972-3" target="_blank">https://www.nature.com/articles/s41541-024-00972-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-09</strong> <a href="https://www.nature.com/articles/s41598-024-74967-4" target="_blank">
            ü§ñ Predictive Modeling for Hemodialysis Patients with SARS-CoV-2 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-46', this)">
            More</button></h4>
            <div id="toggle-46" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting SARS-CoV-2 infection among hemodialysis patients using deep neural network methods</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here is a simplified version of the abstract that a young person can understand:

A new study has found that dialysis patients are more likely to get COVID-19 than people without this condition. To control the spread of the virus and minimize outbreaks, healthcare centers use predictive models that identify infected patients early on. Researchers used data from various sources to create special models that can predict when a patient is most likely to be infected with COVID-19. These models were tested on real data from 2020, and they showed promising results. The research team found that these new models are more accurate than existing ones at identifying who will get the virus during its incubation period. They also identified key factors that can help healthcare providers diagnose patients early and take steps to protect them and others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 has a higher rate of morbidity and mortality among dialysis patients than the general population. Identifying infected patients early with the support of predictive models helps dialysis centers implement concerted procedures (e.g., temperature screenings, universal masking, isolation treatments) to control the spread of SARS-CoV-2 and mitigate outbreaks. We collect data from multiple sources, including demographics, clinical, treatment, laboratory, vaccination, socioeconomic status, and COVID-19 surveillance. Previous early prediction models, such as logistic regression, SVM, and XGBoost, require sophisticated feature engineering and need improved prediction performance. We create deep learning models, including Recurrent Neural Networks (RNN) and Convolutional Neural Networks (CNN), to predict SARS-CoV-2 infections during incubation. Our study shows deep learning models with minimal feature engineering can identify those infected patients more accurately than previously built models. Our Long Short-Term Memory (LSTM) model consistently performed well, with an AUC exceeding 0.80, peaking at 0.91 in August 2021. The CNN model also demonstrated strong results with an AUC above 0.75. Both models outperformed previous best XGBoost models by over 0.10 in AUC. Prediction accuracy declined as the pandemic evolved, dropping to approximately 0.75 between September 2021 and January 2022. Maintaining a 20% false positive rate, our LSTM and CNN models identified 66% and 64% of positive cases among patients, significantly outperforming XGBoost models at 42%. We also identify key features for dialysis patients by calculating the gradient of the output with respect to the input features. By closely monitoring these factors, dialysis patients can receive earlier diagnoses and care, leading to less severe outcomes. Our research highlights the effectiveness of deep neural networks in analyzing longitudinal data, especially in predicting COVID-19 infections during the crucial incubation period. These deep network approaches surpass traditional methods relying on aggregated variable means, significantly improving the accurate identification of SARS-CoV-2 infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74967-4" target="_blank">https://www.nature.com/articles/s41598-024-74967-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41598-024-75233-3" target="_blank">
            ü§ñ Long-term COVID complications after initial COVID-19 illness in Thailand</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-47', this)">
            More</button></h4>
            <div id="toggle-47" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long COVID at 3 and 6 months after covid-19 infection in Thailand</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that a significant number of people who contracted COVID-19 continue to experience symptoms after four weeks. The prevalence of these symptoms varies by country and over time - they can occur as early as three months (36.2%) and up to six months (33.7%). Factors such as gender, disease severity, and whether the person was infected with a mild or severe case also influence when symptoms start and how long they last.</p><br>
                <p><strong>Abstract</strong></p>
                <p>‚ÄúLong COVID‚Äù, which describes a diverse set of symptoms or conditions that persist or develop after four weeks from the onset of initial SARS-CoV-2 infection has been introduced. However, the true prevalence varies worldwide. This study aimed to determine the point prevalence and clinical characteristics of long COVID at three and six months after acute COVID-19 infection in Thailand. Methods All adult patients who were diagnosed with COVID-19 by positive nasopharyngeal RT-PCR for SARS-CoV-2 at Thammasat University Hospital between October and December 2021 were recruited and followed for long COVID symptoms by telephone interviews at 3 and 6 months after an acute infection. Among 1,400 eligible COVID-19 cases, interviews were complete for 1,129 and 932 individuals at 3 and 6 months, respectively. Of those, 431 and 314 reported at least one symptom consistent with long COVID. The point prevalence was 38.2% (95% confidence interval: 35.3‚Äì41.1%) and 33.7% (95% confidence interval: 30.7‚Äì36.7%) respectively. Female gender, disease severity, and symptomatic acute infection were identified as independent risk factors. Conclusion Based on the reported symptoms, long COVID is commonly observed either at 3 or 6 months in our study.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-75233-3" target="_blank">https://www.nature.com/articles/s41598-024-75233-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41598-024-73948-x" target="_blank">
            ü§ñ Infection with SARS-CoV-2 can cause a condition known as acute respiratory distress syndrome (ARDS) that can lead to shock, which can reduce blood flow to vital organs and affect the heart.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-48', this)">
            More</button></h4>
            <div id="toggle-48" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Presence of cholestasis and its impact on survival in SARS-CoV-2 associated acute respiratory distress syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A study has been conducted to understand the relationship between COVID-19, biliary injury, and cholestasis in critically ill patients. 

Researchers analyzed data from 225 COVID-19 patients who were admitted to an ICU at a hospital in Austria. They found that nearly half (53%) of these patients developed cholestasis, which is indicated by an abnormal level of alkaline phosphatase. 

Cholestasis was associated with higher levels of inflammation and liver damage markers. Patients with cholestasis also had worse survival rates during the ICU stay and after six months.

The study suggests that developing cholestasis in COVID-19 patients can be a negative prognostic marker, indicating a poorer outcome.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Data on cholestasis and biliary injury in patients with COVID-19 are scarce. The primary aim of this study was to evaluate the prevalence of cholestasis and factors associated with its development and outcome in critically ill patients with COVID-19 associated acute respiratory distress syndrome (ARDS). In this retrospective exploratory study, COVID-19 patients with ARDS admitted to an intensive care unit (ICU) at the Medical University of Vienna were evaluated for the development of cholestasis defined as an alkaline phosphatase level of 1.67x upper limit of normal for at least three consecutive days. Simple and multiple logistic regression analysis was used to evaluate parameters associated with development of cholestasis and survival. Of 225 included patients 119 (53%) developed cholestasis during ICU stay. Patients with cholestasis had higher peak levels of alkaline phosphatase, gamma-glutamyl transferase, bilirubin and inflammation parameters. Factors independently associated with cholestasis were extracorporeal membrane oxygenation support, ketamine use, high levels of inflammation parameters and disease severity. Presence of cholestasis and peak ALP levels were independently associated with worse ICU and 6-month survival. Development of cholestasis is a common complication in critically ill COVID-19 patients and represents a negative prognostic marker for survival. It is associated with disease severity and specific treatment modalities of intensive care.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73948-x" target="_blank">https://www.nature.com/articles/s41598-024-73948-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-08</strong> <a href="https://www.nature.com/articles/s41467-024-52818-0" target="_blank">
            ü§ñ Proteins involved in viral infection trigger cell release of inflammation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-49', this)">
            More</button></h4>
            <div id="toggle-49" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Coronavirus envelope protein activates TMED10-mediated unconventional secretion of inflammatory factors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract:

Coronavirus proteins can trigger a strong inflammatory response in the body. Scientists have identified a specific protein called the envelope (E) protein on some coronaviruses that is particularly effective at causing this inflammation. In contrast, other types of coronaviruses cause less severe reactions. They found that when these E proteins are present, they help to release chemicals into the body's cells that trigger more inflammation. Researchers also discovered a molecule called progesterone that can block this process and reduce inflammation in some animal studies. This discovery suggests that targeting the E protein could be an effective way to prevent severe coronavirus infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The precise cellular mechanisms underlying heightened proinflammatory cytokine production during coronavirus infection remain incompletely understood. Here we identify the envelope (E) protein in severe coronaviruses (SARS-CoV-2, SARS, or MERS) as a potent inducer of interleukin-1 release, intensifying lung inflammation through the activation of TMED10-mediated unconventional protein secretion (UcPS). In contrast, the E protein of mild coronaviruses (229E, HKU1, or OC43) demonstrates a less pronounced effect. The E protein of severe coronaviruses contains an SS/DS motif, which is not present in milder strains and facilitates interaction with TMED10. This interaction enhances TMED10-oligomerization, facilitating UcPS cargo translocation into the ER-Golgi intermediate compartment (ERGIC)‚Äîa pivotal step in interleukin-1 UcPS. Progesterone analogues were identified as compounds inhibiting E-enhanced release of proinflammatory factors and lung inflammation in a Mouse Hepatitis Virus (MHV) infection model. These findings elucidate a molecular mechanism driving coronavirus-induced hyperinflammation, proposing the E-TMED10 interaction as a potential therapeutic target to counteract the adverse effects of coronavirus-induced inflammation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52818-0" target="_blank">https://www.nature.com/articles/s41467-024-52818-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41467-024-52808-2" target="_blank">
            ü§ñ Immune System Response to SARS-CoV-2 Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-50', this)">
            More</button></h4>
            <div id="toggle-50" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structural basis for receptor-binding domain mobility of the spike in SARS-CoV-2 BA.2.86 and JN.1</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Since 2019, a new virus called SARS-CoV-2 has caused multiple waves of illness and hospitalizations. The virus's spike protein plays a crucial role in entering cells, but its behavior changes depending on how it's shaped. Researchers have been studying the shape of this protein to understand how it interacts with receptors on cell surfaces.

Using detailed computer models and biological samples, scientists found that when the spike protein is correctly shaped as expected, it can bind to a specific receptor called ACE2 more easily. However, they also observed that the virus's spike protein can change shape and interact with its host better when this binding process begins. These changes help prevent the virus from being stopped by antibodies that are designed to recognize and neutralize other forms of the virus.

Understanding how these viral proteins work could lead to new treatments for COVID-19 and other illnesses caused by similar viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Since 2019, SARS-CoV-2 has undergone mutations, resulting in pandemic and epidemic waves. The SARS-CoV-2 spike protein, crucial for cellular entry, binds to the ACE2 receptor exclusively when its receptor-binding domain (RBD) adopts the up-conformation. However, whether ACE2 also interacts with the RBD in the down-conformation to facilitate the conformational shift to RBD-up remains unclear. Herein, we present the structures of the BA.2.86 and the JN.1 spike proteins bound to ACE2. Notably, we successfully observed the ACE2-bound down-RBD, indicating an intermediate structure before the RBD-up conformation. The wider and mobile angle of RBDs in the up-state provides space for ACE2 to interact with the down-RBD, facilitating the transition to the RBD-up state. The K356T, but not N354-linked glycan, contributes to both of infectivity and neutralizing-antibody evasion in BA.2.86. These structural insights the spike-protein dynamics would help understand the mechanisms underlying SARS-CoV-2 infection and its neutralization.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52808-2" target="_blank">https://www.nature.com/articles/s41467-024-52808-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41541-024-00980-3" target="_blank">
            ü§ñ Moderna COVID-19 vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-51', this)">
            More</button></h4>
            <div id="toggle-51" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Preclinical immunogenicity and safety of hemagglutinin-encoding modRNA influenza vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Seasonal flu epidemics affect a lot of people worldwide due to influenza viruses. Vaccines help prevent infection but need updating every year because they can change over time. Researchers created a new type of vaccine that can produce more effective and quicker results, using an existing technology from another vaccine called COVID-19 mRNA vaccine. This new vaccine is made up of tiny genetic material (RNA) that helps the body's immune system recognize and fight off certain flu viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Seasonal epidemics of influenza viruses are responsible for a significant global public health burden. Vaccination remains the most effective way to prevent infection; however, due to the persistence of antigenic drift, vaccines must be updated annually. The selection of vaccine strains occurs months in advance of the influenza season to allow adequate time for production in eggs. RNA vaccines offer the potential to accelerate production and improve efficacy of influenza vaccines. We leveraged the nucleoside-modified RNA (modRNA) platform technology and lipid nanoparticle formulation process of the COVID-19 mRNA vaccine (BNT162b2; Comirnaty¬Æ) to create modRNA vaccines encoding hemagglutinin (HA) (modRNA-HA) for seasonal human influenza strains and evaluated their preclinical immunogenicity and toxicity. In mice, a monovalent modRNA vaccine encoding an H1 HA demonstrated robust antibody responses, HA-specific Th1-type CD4+T cell responses, and HA-specific CD8+T cell responses. In rhesus and cynomolgus macaques, the vaccine exhibited durable functional antibody responses and HA-specific IFN-Œ≥+CD4+T cell responses. Immunization of mice with monovalent, trivalent, and quadrivalent modRNA-HA vaccines generated functional antibody responses targeting the seasonal influenza virus(es) encoded in the vaccines that were greater than, or similar to, those of a licensed quadrivalent influenza vaccine. Monovalent and quadrivalent modRNA-HA vaccines were well-tolerated by Wistar Han rats, with no evidence of systemic toxicity. These nonclinical immunogenicity and safety data support further evaluation of the modRNA-HA vaccines in clinical studies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00980-3" target="_blank">https://www.nature.com/articles/s41541-024-00980-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-07</strong> <a href="https://www.nature.com/articles/s41598-024-73004-8" target="_blank">
            ü§ñ COVID-19 Vaccine Booster Response in Nursing Home Residents</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-52', this)">
            More</button></h4>
            <div id="toggle-52" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 mRNA booster vaccination induces robust antibody responses but few adverse events among SARS-CoV-2 na√Øve nursing home residents</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Residents in nursing homes face higher risks from COVID-19. We studied who got vaccinated with a COVID-19 mRNA vaccine and how their bodies reacted to it. Our study included 95 people who had never been infected with SARS-CoV-2 before, including 69 older adults. Nobody reported serious side effects like heart attacks or seizures. However, some of the less severe side effects were more common in nurses than doctors. We found that those who got vaccinated had stronger immune systems and fewer problems compared to people who didn't get vaccinated at all.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Residents in nursing homes face heightened COVID-19 risks. We aimed to assess the adverse events (AEs) rates and antibody responses after the first to the fifth dose of COVID-19 mRNA vaccination in a nursing home cohort. Ninety-five SARS-CoV-2 na√Øve participants consisted of 26 staff (median age, 51 years) and 69 residents (median age, 88 years). Life-threatening AEs were reported in neither residents nor staff. The severity of non-life-threatening AEs was graded, and severe AEs were reported only in staff. The AEs rates were considerably lower in residents, compared to those in staff. Anti-RBD IgG and the neutralizing titers (NTs) against Wuhan and Omicron BA.4/BA.5 did not differ significantly between those with ‚Äòany AE‚Äô and ‚Äòno AE‚Äô among both staff and residents two months after the second, third and fifth doses, while the anti-RBD IgG significantly differed between two groups after third dose in residents. These findings suggest that the anti-RBD IgG and the NTs increase regardless of the occurrence of AEs. Our study underscores a robust antibody response in both in staff and residents, and fewer AEs following COVID-19 vaccination in SARS-CoV-2 na√Øve residents than staff, supporting the recommendation for mRNA booster doses in older adults at high-risk care facilities.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73004-8" target="_blank">https://www.nature.com/articles/s41598-024-73004-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-06</strong> <a href="https://www.nature.com/articles/s41598-024-73765-2" target="_blank">
            ü§ñ Energy Metabolism in Hamsters Affected by COVID-19 Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-53', this)">
            More</button></h4>
            <div id="toggle-53" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Evidence for SARS-CoV-2 infected Golden Syrian hamsters (Mesocricetus auratus) reducing daily energy expenditure and body core temperature</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified abstract that a young person can understand:

Golden Syrian hamsters are used to test how they react to infections with a coronavirus called SARS-CoV-2. To see what happens when these hamsters get infected, researchers studied their physical changes over time after infection. They looked at things like energy expenditure, water intake, weight, body temperature, and activity levels in both healthy and sick hamsters. The results showed that infected hamsters had less energy and water than non-infected ones, which could be a sign of problems with their internal organs.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Golden Syrian hamsters (Mesocricetus auratus) are a well-established animal model for human infections with severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) due to their susceptibility to SARS-CoV-2 infection, robust virus replication and pathological manifestations similar to human COVID-19 pneumonia. To investigate the physiological changes upon infection in this animal model, we explored the alterations in daily energy expenditure (DEE), water turnover, body mass, body temperature, and locomotor activity in non-infected and SARS-CoV-2 infected Golden Syrian hamsters for four days post SARS-CoV-2. DEE was measured using the doubly labelled water method, which allows for the accurate estimation of carbon dioxide production and, consequently, energy expenditure in animals. Additionally, we investigated total water intake (TWI), which comprises drinking water, preformed water in food, and metabolic water. Using intraperitoneally implanted data loggers, we also monitored body core temperature and locomotor activity in some of the animals. Here we provide evidence for infected hamsters exhibiting significantly lower DEE and TWI compared to non-infected animals. We also observed an increase in body weight in the non-infected animals, while infected animals experienced weight loss. Further, infected animals showed a significantly decreased body temperature, indicating a generally lowered metabolic rate.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73765-2" target="_blank">https://www.nature.com/articles/s41598-024-73765-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-06</strong> <a href="https://www.nature.com/articles/s41598-024-74104-1" target="_blank">
            ü§ñ COVID-19 Detection from Exhaled Breath

A new method to detect COVID-19 has been developed that measures the presence of the virus in a person's respiratory system. This method involves analyzing the breath, which contains tiny particles left behind by the SARS-CoV-2 virus when it enters the lungs.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-54', this)">
            More</button></h4>
            <div id="toggle-54" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 detection from exhaled breath</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new method for detecting COVID-19 has been developed. This system uses a simple device that collects breath samples from the air, then analyzes them using advanced technology. The system is non-invasive and can identify people who have COVID-19 with high accuracy. An initial trial of 302 participants showed promising results, achieving similar accuracy to traditional methods for detecting COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 coronavirus emerged in 2019 causing a COVID-19 pandemic that resulted in 7 million deaths out of 770 million reported cases over the next 4 years. The global health emergency called for unprecedented efforts to monitor and reduce the rate of infection, pushing the study of new diagnostic methods. In this paper, we introduce a cheap, fast, and non-invasive COVID-19 detection system, which exploits only exhaled breath. Specifically, provided an air sample, the mass spectra in the 10‚Äì351 mass-to-charge range are measured using an original micro and nano-sampling device coupled with a high-precision spectrometer; then, the raw spectra are processed by custom software algorithms; the clean and augmented data are eventually classified using state-of-the-art machine-learning algorithms. An uncontrolled clinical trial was conducted between 2021 and 2022 on 302 subjects who were concerned about being infected, either due to exhibiting symptoms or having recently recovered from illness. Despite the simplicity of use, our system showed a performance comparable to the traditional polymerase-chain-reaction and antigen testing in identifying cases of COVID-19 (that is, 95% accuracy, 94% recall, 96% specificity, and 92%\(F_1\)-score). In light of these outcomes, we think that the proposed system holds the potential for substantial contributions to routine screenings and expedited responses during future epidemics, as it yields results comparable to state-of-the-art methods, providing them in a more rapid and less invasive manner.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74104-1" target="_blank">https://www.nature.com/articles/s41598-024-74104-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41467-024-52504-1" target="_blank">
            ü§ñ Adaptive Weight Ensemble for Influenza Forecasting</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-55', this)">
            More</button></h4>
            <div id="toggle-55" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An adaptive weight ensemble approach to forecast influenza activity in an irregular seasonality context</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### Abstract Simplified

Forecasting flu activity in places like Hong Kong is tricky because the flu doesn't always follow a normal pattern. We used different types of smart tools to predict how the flu will spread up to 8 weeks ahead. We looked at data from many years, starting from 1998.

We made groups of simple and complex models together (ensemble), which all did better than just guessing there would be no flu. The best group reduced errors by 23% to 29%. Another special way we blended the tools worked even better, reducing mistakes by 52%, which was much better at predicting different types of flu patterns.

We tested this again after a big flu season (post-COVID), and it still helped reduce mistakes. Our system lets us compare different models and pick the best one for forecasting flu in places where the flu doesn't always come when expected.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Forecasting influenza activity in tropical and subtropical regions, such as Hong Kong, is challenging due to irregular seasonality and high variability. We develop a diverse set of statistical, machine learning, and deep learning approaches to forecast influenza activity in Hong Kong 0 to 8 weeks ahead, leveraging a unique multi-year surveillance record spanning 32 epidemics from 1998 to 2019. We consider a simple average ensemble (SAE) of the top two individual models, and develop an adaptive weight blending ensemble (AWBE) that dynamically updates model contribution. All models outperform the baseline constant incidence model, reducing the root mean square error (RMSE) by 23%‚Äì29% and weighted interval score (WIS) by 25%‚Äì31% for 8-week ahead forecasts. The SAE model performed similarly to individual models, while the AWBE model reduces RMSE by 52% and WIS by 53%, outperforming individual models for forecasts in different epidemic trends (growth, plateau, decline) and during both winter and summer seasons. Using the post-COVID data (2023‚Äì2024) as another test period, the AWBE model still reduces RMSE by 39% and WIS by 45%. Our framework contributes to comparing and benchmarking models in ensemble forecasts, enhancing evidence for synthesizing multiple models in disease forecasting for geographies with irregular influenza seasonality.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52504-1" target="_blank">https://www.nature.com/articles/s41467-024-52504-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41598-024-73114-3" target="_blank">
            ü§ñ Rational Design for SARS-CoV-2 Spike Protein Epitopes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-56', this)">
            More</button></h4>
            <div id="toggle-56" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rational design based on multi-monomer simultaneous docking for epitope imprinting of SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Molecularly imprinted polymers (MIPs) are like designer plastic molecules that mimic how antibodies interact with antigens found in nature. These plastics can be used to catch viruses like the one causing COVID-19.

Recently, scientists developed a new way to make these MIPs using computer simulations and experiments. They looked at a part of the virus called Spike protein to find the best places for their plastic molecules to attach.

They tested many different combinations of building blocks until they found the ones that worked best. The results showed that the designer plastics could be very good at catching viruses, just like antibodies do in our bodies. This method can help make better plastics for catching other types of dangerous viruses too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Among biomimetic strategies shaping engineering designs, molecularly imprinted polymer (MIP) technology stands out, involving chemically synthesised receptors emulating natural antigen-antibody interactions. These versatile ‚Äòdesigner polymers‚Äô with remarkable stability and low cost, are pivotal forin vitrodiagnostics. Amid the recent global health crisis, we probed MIPs‚Äô potential to capture SARS-CoV-2 virions. Large biotemplates complicate MIP design, influencing generated binding site specificity. To precisely structure recognition sites within polymers, we innovated an epitope imprinting method supplemented byin silicopolymerization component screening. A viral surface Spike protein informed epitope selection was targeted for MIP development. A novel multi-monomer docking approach (MMSD) was employed to simulate classical receptor-ligand interactions, mimicking binding reinforcement across multiple amino acids. Around 40 monomer combinations were docked to the epitope sequence and top performers experimentally validated via rapid fluorescence binding assays. Notably, high imprinting factor polymers correlated with MMSD predictions, promising rational MIP design applicable to diverse viral pathologies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73114-3" target="_blank">https://www.nature.com/articles/s41598-024-73114-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-04</strong> <a href="https://www.nature.com/articles/s41590-024-01975-x" target="_blank">
            ü§ñ COVID recovery lungs show unusual white blood cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-57', this)">
            More</button></h4>
            <div id="toggle-57" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Profibrotic monocyte-derived alveolar macrophages are expanded in patients with persistent respiratory symptoms and radiographic abnormalities after COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Monocytes can turn into special cells called alveolar macrophages in the lungs. These cells help fix injuries but sometimes cause more damage over time. Researchers looked at people who had trouble breathing after getting better from COVID-19. They found that when these special lung repair cells were more present or less helpful, it meant the person's lung problems were worse or didn't go away. This suggests these cells might not help fix lungs as well as they should.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monocyte-derived alveolar macrophages drive lung injury and fibrosis in murine models and are associated with pulmonary fibrosis in humans. Monocyte-derived alveolar macrophages have been suggested to develop a phenotype that promotes lung repair as injury resolves. We compared single-cell and cytokine profiling of the alveolar space in a cohort of 35 patients with post-acute sequelae of COVID-19 who had persistent respiratory symptoms and abnormalities on a computed tomography scan of the chest that subsequently improved or progressed. The abundance of monocyte-derived alveolar macrophages, their gene expression programs, and the level of the monocyte chemokine CCL2 in bronchoalveolar lavage fluid positively associated with the severity of radiographic fibrosis. Monocyte-derived alveolar macrophages from patients with resolving or progressive fibrosis expressed the same set of profibrotic genes. Our findings argue against a distinct reparative phenotype in monocyte-derived alveolar macrophages, highlighting their utility as a biomarker of failed lung repair and a potential target for therapy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01975-x" target="_blank">https://www.nature.com/articles/s41590-024-01975-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-03</strong> <a href="https://www.nature.com/articles/s41598-024-74196-9" target="_blank">
            ü§ñ Severe COVID-19 biomarkers: Sirukumab treatment</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-58', this)">
            More</button></h4>
            <div id="toggle-58" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Pharmacodynamic, prognostic, and predictive biomarkers in severe and critical COVID-19 patients treated with sirukumab</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We studied certain chemicals in sick people's blood who had very bad cases of COVID-19 and were given a special antibody called sirukumab. We found that when these chemicals didn't change much after treatment, the patients got better faster. This is especially true for patients with the most severe form of COVID-19. So, if someone doesn't see changes in these chemicals while taking the medicine, they might benefit more from it. The study ID number is NCT04380961.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We examined candidate biomarkers for efficacy outcomes in hospitalized COVID-19 patients who were treated with sirukumab, an IL-6 neutralizing antibody, in a randomized, double-blind, placebo-controlled, phase 2 trial. Between May 2020 and March 2021, 209 patients were randomized (sirukumab,n=‚Äâ139; placebo,n=‚Äâ70); 112 had critical COVID-19. Serum biomarkers were evaluated for the pharmacodynamic effect of sirukumab and for their potential prognostic and predictive effect on time to sustained clinical improvement up to Day 28, clinical improvement at Day 28, and mortality at Day 28. The absence of detectable IL-4 increase and smaller increases in CCL13 post-baseline were most significantly associated with better response to sirukumab (versus placebo) treatment for all clinical efficacy outcomes tested, especially in patients with critical COVID-19. These data suggest that patients with critical COVID-19 without detectable sirukumab-induced IL-4 levels are more likely to benefit from sirukumab treatment. ClinicalTrials.gov Identifier: NCT04380961.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-74196-9" target="_blank">https://www.nature.com/articles/s41598-024-74196-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-03</strong> <a href="https://www.nature.com/articles/s41467-024-52668-w" target="_blank">
            ü§ñ SARS-CoV-2 BA.2.86/JN.1 lineage immune response and severity reduction</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-59', this)">
            More</button></h4>
            <div id="toggle-59" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immune escape and attenuated severity associated with the SARS-CoV-2 BA.2.86/JN.1 lineage</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The BA.2.86 variant, especially its subvariant JN.1, spread widely in the US during winter 2023-24, but didn't cause as many hospitalizations or deaths as previous waves of COVID-19 did.

To understand why this happened, researchers looked at people who had caught these variants compared to others infected with related variants like XBB. They found that:
- More people with BA.2.86 got vaccinated.
- People with BA.2.86 also had more past infections with different COVID strains.
- Less of these people ended up needing hospital care.

The researchers think this might be because their immune systems didn't respond as strongly to the new variant, possibly due to prior vaccinations or previous infections. This means they were less likely to get very sick from BA.2.86 compared to other variants.

Keeping an eye on how SARS-CoV-2 variants evolve and impact health will help doctors plan better responses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The SARS-CoV-2 BA.2.86 lineage, and its sublineage JN.1 in particular, achieved widespread transmission in the US during winter 2023‚Äì24. However, this surge in infections was not accompanied by COVID-19 hospitalizations and mortality commensurate with prior waves. To understand shifts in COVID-19 epidemiology associated with JN.1 emergence, we compared characteristics and clinical outcomes of time-matched cases infected with BA.2.86 lineages (predominantly representing JN.1) versus co-circulating XBB-derived lineages in December, 2023 and January, 2024. Cases infected with BA.2.86 lineages received greater numbers of COVID-19 vaccine doses, including XBB.1.5-targeted boosters, in comparison to cases infected with XBB-derived lineages. Additionally, cases infected with BA.2.86 lineages experienced greater numbers of documented prior SARS-CoV-2 infections. Cases infected with BA.2.86 lineages also experienced lower risk of progression to severe clinical outcomes requiring emergency department consultations or hospital admission. Sensitivity analyses suggested under-ascertainment of prior infections could not explain this apparent attenuation of severity. Our findings implicate escape from immunity acquired from prior vaccination or infection in the emergence of the JN.1 lineage and suggest infections with this lineage are less likely to experience clinically-severe disease. Monitoring of immune escape and clinical severity in emerging SARS-CoV-2 variants remains a priority to inform responses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52668-w" target="_blank">https://www.nature.com/articles/s41467-024-52668-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41398-024-03108-2" target="_blank">
            ü§ñ Brain Changes in Young People Due to Coronavirus: Study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-60', this)">
            More</button></h4>
            <div id="toggle-60" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>COVID-19 related cognitive, structural and functional brain changes among Italian adolescents and young adults: a multimodal longitudinal case-control study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SUGGESTED FOR YOUNG PERSONS

Coronavirus can change how your brain works even if you didn't get very sick. Researchers looked at young people who had milder cases and found changes in certain parts of their brains months later. They noticed problems with some areas like the hippocampus (helps remember things) and amygdala (helps process emotions). These changes can affect how well they do tasks that involve remembering places, even if these kids were fine at first. The findings might help doctors understand long-term effects on the brain from COVID-19 infections.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Coronavirus disease 2019 (COVID-19) has been associated with brain functional, structural, and cognitive changes that persist months after infection. Most studies of the neurologic outcomes related to COVID-19 focus on severe infection and aging populations. Here, we investigated the neural activities underlying COVID-19 related outcomes in a case-control study of mildly infected youth enrolled in a longitudinal study in Lombardy, Italy, a global hotspot of COVID-19. All participants (13 cases, 27 controls, mean age 24 years) completed resting-state functional (fMRI), structural MRI, cognitive assessments (CANTAB spatial working memory) at baseline (pre-COVID) and follow-up (post-COVID). Using graph theory eigenvector centrality (EC) and data-driven statistical methods, we examined differences in ECdelta(i.e., the difference in EC values pre- and post-COVID-19) and Volumetricdelta(i.e., the difference in cortical volume of cortical and subcortical areas pre- and post-COVID) between COVID-19 cases and controls. We found that ECdeltasignificantly between COVID-19 and healthy participants in five brain regions; right intracalcarine cortex, right lingual gyrus, left hippocampus, left amygdala, left frontal orbital cortex. The left hippocampus showed a significant decrease in Volumetricdeltabetween groups (p=‚Äâ0.041). The reduced ECdeltain the left amygdala associated with COVID-19 status mediated the association between COVID-19 and disrupted spatial working memory. Our results show persistent structural, functional and cognitive brain changes in key brain areas associated with olfaction and cognition. These results may guide treatment efforts to assess the longevity, reversibility and impact of the observed brain and cognitive changes following COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41398-024-03108-2" target="_blank">https://www.nature.com/articles/s41398-024-03108-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41598-024-73639-7" target="_blank">
            ü§ñ Optimal Air Travel Strategies for Pandemic Control Using Deep Learning</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-61', this)">
            More</button></h4>
            <div id="toggle-61" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deep learning-derived optimal aviation strategies to control pandemics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic affected many countries worldwide, causing governments to create strict health rules to stop the spread of the virus. This led to economic problems too.

To understand how airplanes played a role in spreading the virus, scientists created a new tool called DWSAGE. It's like a smart map that learns from daily changes in airplane flights around the world. 

Scientists used this tool to see which places with lots of airplane traffic might have spread more COVID-19. They found out Western Europe, the Middle East, and North America had high airplane traffic.

Based on their findings, scientists suggested reducing airplane travel to help control the virus without stopping people from moving around much. Their work gives a powerful tool for studying future outbreaks and helps governments make better decisions about air travel rules.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic affected countries across the globe, demanding drastic public health policies to mitigate the spread of infection, which led to economic crises as a collateral damage. In this work, we investigate the impact of human mobility, described via international commercial flights, on COVID-19 infection dynamics on a global scale. We developed a graph neural network (GNN)-based framework called Dynamic Weighted GraphSAGE (DWSAGE), which operates over spatiotemporal graphs and is well-suited for dynamically changing flight information updated daily. This architecture is designed to be structurally sensitive, capable of learning the relationships between edge features and node features. To gain insights into the influence of air traffic on infection spread, we conducted local sensitivity analysis on our model through perturbation experiments. Our analyses identified Western Europe, the Middle East, and North America as leading regions in fueling the pandemic due to the high volume of air traffic originating or transiting through these areas. We used these observations to propose air traffic reduction strategies that can significantly impact controlling the pandemic with minimal disruption to human mobility. Our work provides a robust deep learning-based tool to study global pandemics and is of key relevance to policymakers for making informed decisions regarding air traffic restrictions during future outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73639-7" target="_blank">https://www.nature.com/articles/s41598-024-73639-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s41599-024-03804-w" target="_blank">
            ü§ñ COVID-19 and Open Science Debate: Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-62', this)">
            More</button></h4>
            <div id="toggle-62" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The impact of COVID-19 on the debate on open science: a qualitative analysis of published materials from the period of the pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how people talked about open science during the pandemic. It found that many thought the pandemic showed why open science is important. Words like "stress test" or "revolution" were used to describe how the pandemic affected open science.

People also believed open science helped fight the pandemic by giving clear information to everyone. But when talking, they mostly focused on sharing data and preprints (early versions of research papers), checking for mistakes in info, and stopping misinformation. They also started thinking about how open science can help make things fairer for all people.

So, even though some old ideas kept coming up, the main topics were new ways to think about how science connects with society and makes it more equal for everyone.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study is an analysis of the international debate on open science that took place during the pandemic. It addresses the question, how did the COVID-19 pandemic impact the debate on open science? The study takes the form of a qualitative analysis of a large corpus of key articles, editorials, blogs and thought pieces about the impact of COVID on open science, published during the pandemic in English, German, Portuguese, and Spanish. The findings show that many authors believed that it was clear that the experience of the pandemic had illustrated or strengthened the case for open science, with language such as a ‚Äústress test‚Äù, ‚Äúcatalyst‚Äù, ‚Äúrevolution‚Äù or ‚Äútipping point‚Äù frequently used. It was commonly believed that open science had played a positive role in the response to the pandemic, creating a clear ‚Äòline of sight‚Äô between open science and societal benefits. Whilst the arguments about open science deployed in the debate were not substantially new, the focuses of debate changed in some key respects. There was much less attention given to business models for open access and critical perspectives on open science, but open data sharing, preprinting, information quality and misinformation became most prominent in debates. There were also moves to reframe open science conceptually, particularly in connecting science with society and addressing broader questions of equity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41599-024-03804-w" target="_blank">https://www.nature.com/articles/s41599-024-03804-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-02</strong> <a href="https://www.nature.com/articles/s42003-024-06951-7" target="_blank">
            ü§ñ Broad-spectrum COVID-19 antibodies work against Omicron</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-63', this)">
            More</button></h4>
            <div id="toggle-63" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broadly potent spike-specific human monoclonal antibodies inhibit SARS-CoV-2 Omicron sub-lineages</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The coronavirus keeps changing, making some medicines ineffective against new versions like Omicron. But scientists found two special antibodies that work well against these changes. One works by targeting part of the virus's spike protein that stays the same even when it mutates. The other targets a different part and can still stop older versions but not newer ones. Understanding how our bodies fight these changes could help make better medicines to treat COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The continuous emergence of SARS-CoV-2 variants of concern has rendered many therapeutic monoclonal antibodies (mAbs) ineffective. To date, there are no clinically authorized therapeutic antibodies effective against the recently circulating Omicron sub-lineages BA.2.86 and JN.1. Here, we report the isolation of broad and potent neutralizing human mAbs (HuMabs) from a healthcare worker infected with SARS-CoV-2 early in the pandemic. These include a genetically unique HuMab, named K501SP6, which can neutralize different Omicron sub-lineages, including BQ.1, XBB.1, BA.2.86 and JN.1, by targeting a highly conserved epitope on the N terminal domain, as well as an RBD-specific HuMab (K501SP3) with high potency towards earlier circulating variants that was escaped by the more recent Omicron sub-lineages through spike F486 and E484 substitutions. Characterizing SARS-CoV-2 spike-specific HuMabs, including broadly reactive non-RBD-specific HuMabs, can give insight into the immune mechanisms involved in neutralization and immune evasion, which can be a valuable addition to already existing SARS-CoV-2 therapies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06951-7" target="_blank">https://www.nature.com/articles/s42003-024-06951-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41541-024-00976-z" target="_blank">
            ü§ñ Norovirus vaccine protects gut tissue</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-64', this)">
            More</button></h4>
            <div id="toggle-64" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Bivalent norovirus mRNA vaccine elicits cellular and humoral responses protecting human enteroids from GII.4 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This article talks about a new way to make vaccines against norovirus, which is a stomach bug that spreads easily. Scientists used special material called mRNA-LNP to create a vaccine for two types of norovirus. This vaccine helped the body's immune system produce strong antibodies that can fight off these viruses. The antibodies stopped the virus from infecting tiny models of the human gut, showing this vaccine works well in tests. This means scientists have proof their method is effective and could make better vaccines against norovirus in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Nucleoside-modified mRNA-LNP vaccines have revolutionized vaccine development against infectious pathogens due to their ability to elicit potent humoral and cellular immune responses. In this article, we present the results of the first norovirus vaccine candidate employing mRNA-LNP platform technology. The mRNA-LNP bivalent vaccine encoding the major capsid protein VP1 from GI.1 and GII.4 of human norovirus, generated high levels of neutralizing antibodies, robust cellular responses, and effectively protected human enteroids from infection by the most prevalent genotype (GII.4). These results serve as a proof of concept, demonstrating that a modified-nucleoside mRNA-LNP vaccine based on norovirus VP1 sequences can stimulate an immunogenic response in vivo and generates neutralizing antibodies capable of preventing viral infection in models of human gastrointestinal tract infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00976-z" target="_blank">https://www.nature.com/articles/s41541-024-00976-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41598-024-73956-x" target="_blank">
            ü§ñ Cytokines in sick kids with multi-system inflammation</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-65', this)">
            More</button></h4>
            <div id="toggle-65" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Cytokine landscape in hospitalized children with multisystem inflammatory syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at the body's response to a sickness called MIS-C in children who had COVID-19. They measured certain chemicals (called cytokines) in kids' blood before and after treatment with special medicines.

They found three main groups of these chemicals:
1. The first group had mainly bad chemicals that make inflammation worse.
2. The second group had some good chemicals plus some bad ones, which stayed around a bit longer.
3. The third group saw its levels go up later on, with more kinds of bad chemicals.

These changes show how the body fights off sickness and helps kids get better. It starts by making lots of inflammatory chemicals to fight infections, then it makes more specific memory cells so it can keep fighting if needed in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The etiology of multisystem inflammatory syndrome in children (MIS-C), frequently observed following COVID-19 infection, remains elusive. This study unveils insights derived from cytokine analysis in the sera of MIS-C patients, both before and after the administration of intravenous immunoglobulin (IVIG) and glucocorticosteroids (GCS). In this study, we employed a comprehensive 45-cytokine profile encompassing a spectrum of widely recognized proinflammatory and antiinflammatory cytokines, as well as growth factors, along with other soluble mediators. The analysis delineates three principal cytokine-concentration patterns evident in the patients‚Äô sera. Pattern no.1 predominantly features proinflammatory cytokines (IL-6, IL-15, IL-1ra, granulocyte-macrophage colony-stimulating factor (GM-CSF), tumor necrosis factor Œ± (TNFŒ±), C-X-C motif chemokine ligand 10 (CXCL10/ IP-10), and IL-10) exhibiting elevated concentrations upon admission, swiftly normalizing post-hospital treatment. Pattern no. 2 includes cytokines (IL-17¬†A, IL-33, IFNŒ≥, vascular endothelial growth factor (VEGF), and programmed death ligand (PD-L1)) with moderately elevated levels at admission, persisting over 7‚Äì10 days of hospitalization despite the treatment. Pattern no. 3 comprises cytokines which concentrations escalated after 7‚Äì10 days of hospitalization and therapy, including IL-1Œ±, IL-1Œ≤, IL-2, IL-13, platelet-derived growth factor AA/BB (PDGF AA/BB). The observed in cytokine profile of MIS-C patients showed a transition from acute inflammation to sustaining inflammation which turned into induction of humoral memory mechanisms and various defense mechanisms, contributing to recovery.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73956-x" target="_blank">https://www.nature.com/articles/s41598-024-73956-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-10-01</strong> <a href="https://www.nature.com/articles/s41598-024-71247-z" target="_blank">
            ü§ñ Impact of COVID-19 on PAD patients in China</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-66', this)">
            More</button></h4>
            <div id="toggle-66" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of the COVID-19 pandemic on patients with peripheral arterial disease in China: a multicenter cross-sectional study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how having a disease called peripheral arterial disease (PAD) affected people who were hospitalized during different times: before and after the pandemic started. Before the pandemic, 762 patients with PAD were treated, on average aged 72 years. After the pandemic, 478 patients were treated, on average aged 65 years.

During the pandemic, more patients had problems like very bad legs (CLTI), foot infections, and damage to blood vessels in their legs. Their leg condition got worse too. Treating these patients was harder during the pandemic, which made it hard for them to get better.

The study also found that having certain issues, like diabetes or kidney problems, could make things worse. These people had more serious health problems after being treated during the pandemic.

Overall, doctors still tried their best to help patients with severe PAD even though there were many challenges. The study shows it's important to keep treating and checking on these patients closely.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study aims to understand the repercussions of the COVID-19 pandemic on hospitalized patients with peripheral arterial disease (PAD) in China, who did not contract SARS-CoV-2. We conducted a multicenter cross-sectional analysis comparing the characteristics and outcomes of hospitalized PAD patients across two distinct periods: Pre-pandemic (P1, from January 2018 to December 2019) and during the pandemic (P2, from January 2020 to December 2021). During P1, 762 hospitalized patients were treated, with an average age of 72.3¬†years, while 478 patients were treated in P2, with an average age of 65.1¬†years. Notably, hospitalized patients admitted during the pandemic (P2) exhibited a significantly higher incidence of chronic limb-threatening ischemia (CLTI, 70% vs 54%), diabetic foot infection (47% vs 29%), and infra-popliteal lesions (28% vs 22%). Furthermore, these patients demonstrated a marked deterioration in their Rutherford category and an increased mean score in the Wound, Ischemia, and foot Infection classification system (WIfI). Treatment during the pandemic emerged as a predictor of reduced procedural success and increased major adverse limb events. Factors such as the presence of diabetic foot infection, renal impairment, and deteriorating WIfI scores were identified as independent risk indicators for major adverse limb events. Our results demonstrate that intensive care was provided to severe cases of PAD even during the challenging circumstances of the COVID-19 pandemic. Despite the unprecedented pressures on healthcare systems, patients with severe PAD, particularly those with CLTI, continued to receive necessary in-patient care. The findings underscore the importance of timely medical interventions and extended follow-up for patients exhibiting high-risk factors.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71247-z" target="_blank">https://www.nature.com/articles/s41598-024-71247-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73175-4" target="_blank">
            ü§ñ Surface proteins of Group B Strep isolates from SA & vaccine target ID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-67', this)">
            More</button></h4>
            <div id="toggle-67" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Surface protein distribution in Group B Streptococcus isolates from South Africa and identifying vaccine targets through in silico analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Group B Strep (GBS) is a germ that can cause serious illnesses like pneumonia, meningitis, and sepsis in babies under 3 months old. It can also lead to stillbirths and early births in mommies who are pregnant. For older adults or people with weak immune systems, GBS causes different diseases.

Right now, there's no vaccine that works against GBS. By looking at the genetic makeup of GBS, scientists found some important parts they think could be used to make a vaccine. They did this by testing lots of GBS samples from babies and mommies in South Africa.

They looked for which parts of GBS are similar between the sick ones (that's what "invasive" means) and those that live on but don't cause problems yet ("colonizing"). Out of 89 possible vaccine candidates, they found 10 very common parts. Nine of these were special proteins called "computational inferred proteins," which is a fancy way of saying the scientists figured out their identity by looking at genetic data. One part was a surface protein that helps GBS stick to cells.

They also looked at how likely each part is to trigger an immune response, which is important for vaccines. The nine special parts they found were more likely to do this than the surface protein. This suggests these could be good targets for making a vaccine against GBS.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Group BStreptococcus(GBS) is a major cause of pneumonia, sepsis, and meningitis in infants younger than 3 months of age. Furthermore, GBS infection in pregnant women is associated with stillbirths and pre-term delivery. It also causes disease in immunocompromised adults and the elderly, but the highest incidence of the disease occurs in neonates and young infants. At this time, there are no licensed vaccines against GBS. Complete GBS genome sequencing has helped identify genetically conserved and immunogenic proteins, which could serve as vaccine immunogens. In this study, in silico reverse vaccinology method were used to evaluate the prevalence and conservation of GBS proteins in invasive and colonizing isolates from South African infants and women, respectively. Furthermore, this study aimed to predict potential GBS vaccine targets by evaluating metrics such as antigenicity, physico-chemical properties, subcellular localization, secondary and tertiary structures, and epitope prediction and conservation. A total of 648 invasive and 603 colonizing GBS isolate sequences were screened against a panel of 89 candidate GBS proteins. Ten of the 89 proteins were highly genetically conserved in invasive and colonizing GBS isolates, nine of which were computationally inferred proteins (gbs2106, SAN_1577, SAN_0356, SAN_1808, SAN_1685, SAN_0413, SAN_0990, SAN_1040, SAN_0226) and one was the surface Immunogenic Protein (SIP). Additionally, the nine proteins were predicted to be more antigenic than the SIP protein (antigenicity score of >‚Äâ0.6498), highlighting their potential as GBS vaccine antigen targets.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73175-4" target="_blank">https://www.nature.com/articles/s41598-024-73175-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73321-y" target="_blank">
            ü§ñ Long-term study on blood-brain barrier changes in COVID-19 patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-68', this)">
            More</button></h4>
            <div id="toggle-68" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Oneyear longitudinal study on biomarkers of blood‚Äìbrain barrier permeability in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how a virus called COVID-19 affects the brain. They found that after getting sick, the blood-brain barrier (which keeps things from going into the brain) was more leaky for a little while. This led to some extra chemicals in the blood coming from cells in the brain. These extra chemicals are like astrocyte-derived extracellular vesicles and S100B.

They checked if these extra chemicals were related to symptoms that kept lasting for months after getting sick. But they didn't find any link between the levels of these chemicals and long-term problems people still had even after a year.

The study found that after about 4 months, these extra chemicals peaked (were at their highest), but by the end of the year, they were back to normal. 

The main thing is that while there are signs of brain changes during an infection, those don't seem to cause long-term health issues. More research is needed to understand why some people have problems for so long after having COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The pathophysiology behind neurological and cognitive sequelae of COVID-19 may be related to dysfunction of the blood-brain barrier (BBB) and previous research indicate transient neuronal injury and glial activation. The aim of this study was to investigate if COVID-19 is related to increased BBB permeability by analyzing leakage of biomarkers such as astrocyte-derived extracellular vesicles (EVs) and S100B. We also investigated whether levels of these biomarkers correlated with self-reported symptoms that persisted‚Äâ>‚Äâ2 months. The samples in this 1-year follow-up study came from an ongoing longitudinal study of unvaccinated patients hospitalized for COVID-19 at Danderyd University Hospital, Stockholm, Sweden, between April and June 2020. Blood samples were collected at baseline and 4, 8, and 12 months after hospitalization. Information on self-reported clinical symptoms was collected at follow-up visits. A total of 102 patients were enrolled, and 47 completed all follow-up measurements. Peak levels of both biomarkers were observed at 4 months in the subset of 55 patients who were measured at this timepoint. At 12 months, the biomarkers had returned to baseline levels. The biomarkers were not correlated with any of the long-term self-reported symptoms. COVID-19 is associated with transient increased BBB permeability, shown by elevated levels of astrocyte biomarkers in plasma. However, these levels return to baseline 12 months post-infection and do not correlate with long-term symptoms. Further research is needed to unravel the underlying mechanisms causing long-term symptoms in COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73321-y" target="_blank">https://www.nature.com/articles/s41598-024-73321-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-72842-w" target="_blank">
            ü§ñ Deep Learning for Detecting Complex Protein Changes in COVID-19 Virus Interactions</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-69', this)">
            More</button></h4>
            <div id="toggle-69" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deep learning for discriminating non-trivial conformational changes in molecular dynamics simulations of SARS-CoV-2 spike-ACE2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study uses advanced computer models to analyze how a part of the SARS-CoV-2 virus protein interacts with another protein in the human body. They converted these complex data into simpler maps for easier understanding, then used AI to guess which changes in the virus might make it more contagious or harder to fight off.

They found that some changes to this part of the virus can make it stick better to its target and also makes it less noticeable to the immune system. This helps them predict how different types of mutations might affect the virus's ability to spread and avoid being detected by the body‚Äôs defenses. 

The AI also showed a good match between what they predicted and actual changes in how well the virus sticks, which is important for understanding its behavior. They discovered that one particular part of the protein moves around more than usual when there are these mutations, suggesting it might be key to evading the immune response.

Overall, this method can help scientists quickly find new strains of the virus that could spread better or hide from our bodies' defenses. This is useful for making new treatments and vaccines faster.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Molecular dynamics (MD) simulations produce a substantial volume of high-dimensional data, and traditional methods for analyzing these data pose significant computational demands. Advances in MD simulation analysis combined with deep learning-based approaches have led to the understanding of specific structural changes observed in MD trajectories, including those induced by mutations. In this study, we model the trajectories resulting from MD simulations of the SARS-CoV-2 spike protein-ACE2, specifically the receptor-binding domain (RBD), as interresidue distance maps, and use deep convolutional neural networks to predict the functional impact of point mutations, related to the virus‚Äôs infectivity and immunogenicity. Our model was successful in predicting mutant types that increase the affinity of the S protein for human receptors and reduce its immunogenicity, both based on MD trajectories (precision = 0.718; recall = 0.800;\(\hbox {F}_1\)= 0.757; MCC = 0.488; AUC = 0.800) and their centroids. In an additional analysis, we also obtained a strong positive Pearson‚Äôs correlation coefficient equal to 0.776, indicating a significant relationship between the average sigmoid probability for the MD trajectories and binding free energy (BFE) changes. Furthermore, we obtained a coefficient of determination of 0.602. Our 2D-RMSD analysis also corroborated predictions for more infectious and immune-evading mutants and revealed fluctuating regions within the receptor-binding motif (RBM), especially in the\(\beta _{1}^{\prime }/\beta _{2}^{\prime }-C\)loop. This region presented a significant standard deviation for mutations that enable SARS-CoV-2 to evade the immune response, with RMSD values of 5√Ö in the simulation. This methodology offers an efficient alternative to identify potential strains of SARS-CoV-2, which may be potentially linked to more infectious and immune-evading mutations. Using clustering and deep learning techniques, our approach leverages information from the ensemble of MD trajectories to recognize a broad spectrum of multiple conformational patterns characteristic of mutant types. This represents a strategic advantage in identifying emerging variants, bypassing the need for long MD simulations. Furthermore, the present work tends to contribute substantially to the field of computational biology and virology, particularly to accelerate the design and optimization of new therapeutic agents and vaccines, offering a proactive stance against the constantly evolving threat of COVID-19 and potential future pandemics.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72842-w" target="_blank">https://www.nature.com/articles/s41598-024-72842-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-67945-3" target="_blank">
            ü§ñ Age and gender impact vaccine response to SpikeGen shots: study results.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-70', this)">
            More</button></h4>
            <div id="toggle-70" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effects of age and gender on immunogenicity and reactogenicity of SpikoGen recombinant spike protein vaccine: a post-hoc analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT Simplified

The SpikoGen COVID-19 vaccine uses part of the virus's spike protein. It was tested in people of different ages and genders to see how well it works. After one dose, older people had fewer side effects like tiredness and headache. Men also felt fewer side effects than women after each dose. The vaccine didn't affect neutralizing antibodies differently between men and women or age groups.

The vaccine is good at getting into the body's immune system but doesn‚Äôt cause many bad reactions. This shows that even if you don't feel too sick from a vaccine, it‚Äôs still working well to protect against COVID-19.

SpikoGen seems like a promising way to make vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SpikoGen¬Æ COVID-19 vaccine is based on the spike protein extracellular domain of the ancestral Wuhan-Hu-1 strain modified by removal of the furin cleavage site and addition of stabilising mutations expressed as a recombinant protein in insect cells. It is formulated with Advax-CpG55.2‚Ñ¢ adjuvant to ensure optimal immunogenicity. In this study, data from several SpikoGen¬Æ clinical trials was retrospectively analysed to assess for any effect of gender or age on seroconversion, neutralizing antibody levels or the incidence of adverse events. Following the 1st dose, older age was associated with a reduced rate of fatigue (RR 0.97, p < 0.001), headache (RR 0.98, p = 0.034) and myalgia (RR 0.97, p=0.016), following the 2nd dose, the rate of fatigue (RR 0.98, p = 0.017) but following the 3rd dose no effect of age on adverse events was evident. Similarly, following the 1st dose, men reported a 19% lower incidence of fatigue, 36% lower incidence of headache and 28% lower incidence of myalgia when compared to women. Interestingly, there was no relationship between age or gender and serum neutralizing antibody levels, although after each vaccine dose there was a consistent trend to women having a higher seroconversion rate. There was no correlation between neutralizing antibody levels and adverse events. Unlike what is seen with mRNA vaccines, reactogenicity trended lower after each subsequent SpikoGen¬Æ dose. Overall, SpikoGen¬Æ exhibited positive immunogenicity and low reactogenicity, indicating that a low incidence of adverse events does not equate to poor immunogenicity. SpikoGen¬Æ remains a promising protein-based vaccine platform for COVID-19 protection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67945-3" target="_blank">https://www.nature.com/articles/s41598-024-67945-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s43856-024-00610-y" target="_blank">
            ü§ñ Infecting K18-hACE2 mice with both Coccidioides and COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-71', this)">
            More</button></h4>
            <div id="toggle-71" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Developing aCoccidioides posadasiiand SARS-CoV-2 Co-infection Model in the K18-hACE2 Transgenic Mouse</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SIMPLIFIED

Scientists wanted to see how getting sick twice at once affects you. They did this by putting two different sicknesses into mice‚Äîlike a mock infection. One sickness was caused by the virus that causes COVID-19, and the other was from a fungus called Valley Fever.

They found that if they gave both sicknesses to the mice one after another, the mice got much sicker and lived less time than those who only had one sickness. This happened more with certain types of coronavirus (like Delta or Omicron) and sometimes depended on when each sickness came first.

The scientists also noticed differences in how the immune system reacted to both sicknesses. The ones that caught the fungus first were healthier, but still got sick from the virus. Those that caught the virus first had a harder time recovering from Valley Fever.

These findings suggest people who live where Valley Fever is common might have a worse experience if they get COVID-19 there too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundEarly reports showed that patients with COVID-19 had recrudescence of previously resolved coccidioidomycosis (Valley fever, VF), and there were indications that coinfection had more severe outcomes. We therefore investigated serial infection ofCoccidioides posadasiiand SARS-CoV-2 in a K18-hACE2 mouse model to assess disease outcomes.MethodsIn our model, we challenged K18-hACE2 mice sequentially with a sub-lethal dose of SARS-CoV-2 and 24‚Äâhours later with low virulence strain ofCoccidioides posadasii, and vice versa, compared to mice that only received a single infection challenge. We performed survival and pathogenesis mouse studies as well as looked at the systemic immune response differences between treatment groups.ResultsHere we show that co-infected groups have a more severe disease progression as well as a decrease in survival. Importantly, results differ depending on the SARS-CoV-2 variant (WA-1, Delta, or Omicron) and infection timing (SARS-CoV-2 first,C. posadasiisecond or vice versa). We find that groups that are infected with the virus first had a decrease in survival, increased morbidity and weight loss, increased fungal and viral burdens, differences in immune responses, and the amount and size of fungal spherules. We also find that groups coinfected withC. posadasiifirst have a decrease fungal burden and inflammatory responses.ConclusionsThis is the first in vivo model investigation of a coinfection of SARS-CoV-2 andCoccidioides. Because of the potential for increased severity of disease in a coinfection, we suggest populations that live in areas of high coccidioidomycosis endemicity may experience higher incidence of complicated disease progression with COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00610-y" target="_blank">https://www.nature.com/articles/s43856-024-00610-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41467-024-52404-4" target="_blank">
            ü§ñ Post-COVID Symptoms in ER Patients Tested for COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-72', this)">
            More</button></h4>
            <div id="toggle-72" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID-19 condition symptoms among emergency department patients tested for SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study found that many people who went to the hospital with emergency symptoms didn't have COVID-19, but some still had problems like coughing or trouble breathing. Even when doctors tested negative for COVID-19, almost one-third of those patients said they were having Post-COVID issues three months later. The problem is, it's hard to know if these are really from the coronavirus or just other illnesses. So doctors think we need better tests now to figure out what causes these symptoms so people can get proper treatment sooner.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Symptoms of the Post-COVID-19 Condition are often non-specific making it a challenge to distinguish them from symptoms due to other medical conditions. In this study, we compare the proportion of emergency department patients who developed symptoms consistent with the World Health Organization‚Äôs Post-COVID-19 Condition clinical case definition between those who tested positive for Severe Acute Respiratory Syndrome Coronavirus-2 infection and time-matched patients who tested negative. Our results show that over one-third of emergency department patients with a proven acute infection meet Post-COVID-19 Condition criteria 3 months post-index visit. However, one in five test-negative patients who claim never having been infected also report symptoms consistent with Post-COVID-19 Condition highlighting the lack of specificity of the clinical case definition. Testing for SARS-CoV-2 during the acute phase of a suspected infection should continue until specific biomarkers of Post-COVID-19 Condition become available for diagnosis and treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52404-4" target="_blank">https://www.nature.com/articles/s41467-024-52404-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-72421-z" target="_blank">
            ü§ñ Cytokines linked to memory changes post-SARS-CoV-2ÊÑüÊüì6-9ÊúàÂíå12-15Êúà</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-73', this)">
            More</button></h4>
            <div id="toggle-73" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Systemic cytokines related to memory function 6‚Äì9¬†months and 12‚Äì15¬†months after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how cytokines, which are chemicals made by the body during infections, might affect people's memories months after they had COVID. They found that levels of one cytokine called IL-1Œ≤ were linked to remembering words better about 6-9 months later. Another cytokine, IL-6, seemed to affect memory around a year after infection. This suggests how sick someone was during the early stages of their COVID could impact their ability to remember things over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Cognitive symptoms persisting beyond the acute phase of COVID-19 infection are commonly described for up to 2¬†years after infection. The relationship between cognitive performance, in particular episodic memory processes observed chronically after infection, and cytokine levels in the acute phase of COVID-19 has not yet been identified in humans. To determine whether the levels of cytokines IL1Œ≤, IL-6 and TNFŒ± secreted in the acute phase of SARS-CoV-2 infection are associated and predict verbal and visuospatial episodic memory performance in humans 6 to 9¬†months and 12 to 15¬†months post-infection. The associations and predictive value of the concentration of cytokines measured in acute phase (IL-1Œ≤, IL-6, TNFŒ±) from plasma samples ofN=‚Äâ33 hospitalized COVID-19 patients (mean age 61¬†years, 39‚Äì78, 65% in intensive care) in relation to their verbal and visuospatial episodic memory performance measured at 6‚Äì9¬†months and 12‚Äì15¬†months post-infection were analyzed. To do this, we used Spearman correlations and generalised linear mixed models. IL-1Œ≤ levels were associated with verbal episodic memory total recall scores 6‚Äì9¬†months post-infection. At 12‚Äì15¬†months post-infection IL-6 predicted verbal episodic memory score. This study demonstrated that the severity of inflammatory reaction at acute phase of SARS-CoV-2 infection predicts verbal episodic memory performance in the long-term post-infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72421-z" target="_blank">https://www.nature.com/articles/s41598-024-72421-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-30</strong> <a href="https://www.nature.com/articles/s41598-024-73160-x" target="_blank">
            ü§ñ Predicting Long COVID Burden in CA with QALYs</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-74', this)">
            More</button></h4>
            <div id="toggle-74" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modeling the burden of long COVID in California with quality adjusted life-years (QALYS)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at a sickness called "long COVID" that some people get even after they had very little or no symptoms from the original coronavirus illness. They used data on all confirmed cases in California over several years.

They divided people into different age groups and measured how many life-years of quality were lost due to long COVID. The study found that more older adults (65+) lost these life years compared to younger ones, but surprisingly, younger adults (18-49) also had a significant loss when you consider all the mild and moderate symptoms.

The researchers think their estimates might be lower than what really happened because they didn't account for everyone who got long COVID. They found that how many people developed long COVID and which symptoms were severe or mild affected their results the most.

Overall, this study helps us understand more about the impact of "long COVID" in different age groups and how it compares to regular cases like hospitalizations.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Individuals infected with SARS-CoV-2 may develop post-acute sequelae of COVID-19 (‚Äúlong COVID‚Äù) even after asymptomatic or mild acute illness. Including time varying COVID symptom severity can provide more informative burden estimates for public health response. Using a compartmental model driven by confirmed cases, this study estimated long COVID burden by age group (0‚Äì4, 5‚Äì17, 18‚Äì49, 50‚Äì64, 65+) in California as measured by the cumulative and severity-specific proportion of quality-adjusted life years (QALYs) lost. Long COVID symptoms were grouped into severe, moderate, and mild categories based on estimates from the Global Burden of Disease study, and symptoms were assumed to decrease in severity in the model before full recovery. All 10,945,079 confirmed COVID-19 cases reported to the California Department of Public Health between March 1, 2020, and December 31, 2022, were included in the analysis. Most estimated long COVID-specific QALYs [59,514 (range: 10,372‚Äì180,257)] lost in California were concentrated in adults 18‚Äì49 (31,592; 53.1%). Relative to other age groups, older adults (65+) lost proportionally more QALYs from severe long COVID (1,366/6,984; 20%). Due to changing case ascertainment over time, this analysis might underestimate the actual total burden. In global sensitivity analysis, estimates of QALYs lost were most sensitive to the proportion of individuals that developed long COVID and proportion of cases with each initial level of long COVID symptom severity (mild/moderate/severe). Models like this analysis can help translate observable metrics such as cases and hospitalizations into quantitative estimates of long COVID burden that are currently difficult to directly measure. Unlike the observed relationship between age and incident severe outcomes for COVID-19, this study points to the potential cumulative impact of mild long COVID symptoms in younger individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73160-x" target="_blank">https://www.nature.com/articles/s41598-024-73160-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-28</strong> <a href="https://www.nature.com/articles/s41598-024-73001-x" target="_blank">
            ü§ñ Clinical Presentation, Inflammation, ANDC Scores in COVID-19 Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-75', this)">
            More</button></h4>
            <div id="toggle-75" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical presentations, systemic inflammation response and ANDC scores in hospitalized patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that people with anosmia (loss of smell) or ageusia (loss of taste) are often diagnosed with COVID-19, especially with the alpha variant. They looked at 231 hospitalized COVID-19 patients in Taiwan and noticed some differences between those who had these symptoms and those who didn't.

They found that younger people with anosmia/ageusia were usually healthier overall and less sick compared to others without these issues. Women also tended to have milder cases than men, though they both had similar smells and taste loss.

The study showed that certain blood tests called NLRs and PLRs helped predict how long patients would stay in the hospital. The ANDC score is another tool that can estimate this length based on symptoms like coughing and fever. 

Overall, the research suggests that people with anosmia/ageusia might have less serious COVID-19 and a lower body inflammation response, especially if they are younger women. This score could help doctors predict how sick someone will get from the disease and how long they'll need to stay in the hospital.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The association of anosmia/ageusia with a positive severe respiratory syndrome coronavirus 2 (SARS-CoV-2) test is well-established, suggesting these symptoms are reliable indicators of coronavirus disease 2019 (COVID-19) infection. This study investigates the clinical characteristics and systemic inflammatory markers in hospitalized COVID-19 patients in Taiwan, focusing on those with anosmia/ageusia. We conducted a retrospective observational study on 231 hospitalized COVID-19 patients (alpha variant) from April to July 2021. Clinical symptoms, dyspnea grading, and laboratory investigations, including neutrophil-lymphocyte ratios (NLRs), platelet-lymphocyte ratios (PLRs), and ANDC scores (an early warning score), were analyzed. Cough (64.1%), fever (58.9%), and dyspnea (56.3%) were the most common symptoms, while anosmia/ageusia affected 9% of patients. Those with anosmia/ageusia were younger, had lower BMI, lower systemic inflammatory markers, and better ANDC scores than those without these symptoms. Female patients exhibited lower NLR values and ANDC scores compared to male patients (allp<‚Äâ0.05). Multivariable regression analysis demonstrated significant associations between NLR and CRP and ferritin levels (allp<‚Äâ0.01), and between PLR and ESR and ferritin levels (p<‚Äâ0.01). Categorized ANDC scores significantly correlated with the total hospital length of stay (allp<‚Äâ0.05). Despite ethnic differences in the prevalence of anosmia/ageusia, our study highlights similar clinical presentations and inflammatory profiles to those observed in Western countries. The ANDC score effectively predicted hospital stay duration. These findings suggest that anosmia/ageusia may be associated with less severe disease and a lower inflammatory response, particularly in younger and female patients. The ANDC score can serve as a valuable prognostic tool in assessing the severity and expected hospital stay of COVID-19 patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73001-x" target="_blank">https://www.nature.com/articles/s41598-024-73001-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-28</strong> <a href="https://www.nature.com/articles/s41598-024-73779-w" target="_blank">
            ü§ñ Key Hub Genes in Leishmaniasis for Diagnosis and Treatment Discovery</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-76', this)">
            More</button></h4>
            <div id="toggle-76" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Determination of key hub genes in Leishmaniasis as potential factors in diagnosis and treatment based on a bioinformatics study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Leishmaniasis is a disease caused by parasites that are carried by infected sandflies. Researchers looked at data from patients with leishmaniasis on their skin wounds and compared it to healthy people. They used special methods to find genes that were more active or less active in the sick people.

They found 407 genes that behaved differently between those with leishmaniasis and others. These genes are involved in immune responses, inflammation, and pathways related to fighting infections.

The study also identified seven important genes (called hub genes) that help control these different gene activities. One of these genes, UBD, plays a role in how cells break down other molecules called ubiquitin. Another hub gene, UBD, is connected with several microRNAs, which are tiny molecules that can control gene activity.

These findings could help doctors diagnose leishmaniasis earlier and develop new treatments or vaccines against the disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Leishmaniasis is an infectious disease caused by protozoan parasites from different species of leishmania. The disease is transmitted by female sandflies that carry these parasites. In this study, datasets on leishmaniasis published in the GEO database were analyzed and summarized. The analysis in all three datasets (GSE43880, GSE55664, and GSE63931) used in this study has been performed on the skin wounds of patients infected with a clinical form of leishmania (Leishmania braziliensis), and biopsies have been taken from them. To identify differentially expressed genes (DEGs) between leishmaniasis patients and controls, the robust rank aggregation (RRA) procedure was applied. We performed gene functional annotation and protein-protein interaction (PPI) network analysis to demonstrate the putative functionalities of the DEGs. The study utilized Molecular Complex Detection (MCODE), Gene Ontology (GO), and Kyoto Encyclopedia of Genes and Genomes (KEGG) to detect molecular complexes within the protein-protein interaction (PPI) network and conduct analyses on the identified functional modules. The CytoHubba plugin‚Äôs results were paired with RRA analysis to determine the hub genes. Finally, the interaction between miRNAs and hub genes was predicted. Based on the RRA integrated analysis, 407 DEGs were identified (263 up-regulated genes and 144 down-regulated genes). The top three modules were listed after creating the PPI network via the MCODE plug. Seven hub genes were found using the CytoHubba app and RRA: CXCL10, GBP1, GNLY, GZMA, GZMB, NKG7, and UBD. According to our enrichment analysis, these functional modules were primarily associated with immune pathways, cytokine activity/signaling pathways, and inflammation pathways. However, a UBD hub gene is interestingly involved in the ubiquitination pathways of pathogenesis. The mirNet database predicted the hub gene‚Äôs interaction with miRNAs, and results revealed that several miRNAs, including mir-146a-5p, crucial in fighting pathogenesis. The key hub genes discovered in this work may be considered as potential biomarkers in diagnosis, development of agonists/antagonist, novel vaccine design, and will greatly contribute to clinical studies in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73779-w" target="_blank">https://www.nature.com/articles/s41598-024-73779-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73311-0" target="_blank">
            ü§ñ COVID-19 impact on brain function: reduced cognitive clarity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-77', this)">
            More</button></h4>
            <div id="toggle-77" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Altered functional brain connectivity, efficiency, and information flow associated with brain fog after mild to moderate COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 virus makes it harder for people to think clearly. We tested 49 adults, 25 who had been sick with COVID and 24 who hadn‚Äôt. The group that had COVID did worse on thinking tests. Their brains also showed less connection between different parts compared to the other group. Less brain connections mean it‚Äôs harder to process information efficiently. This could be a way for the brain to keep working but feel like it's foggy or fuzzy.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 is associated with increased risk for cognitive decline but very little is known regarding the neural mechanisms of this risk. We enrolled 49 adults (55% female, mean age‚Äâ=‚Äâ30.7‚Äâ¬± 8.7), 25 with and 24 without a history of COVID-19 infection. We administered standardized tests of cognitive function and acquired brain connectivity data using MRI. The COVID-19 group demonstrated significantly lower cognitive function (W‚Äâ=‚Äâ475, p‚Äâ<‚Äâ0.001, effect size r‚Äâ=‚Äâ0.58) and lower functional connectivity in multiple brain regions (mean t‚Äâ=‚Äâ3.47‚Äâ¬±0.36, p‚Äâ=‚Äâ0.03, corrected, effect size d‚Äâ=‚Äâ0.92 to 1.5). Hypo-connectivity of these regions was inversely correlated with subjective cognitive function and directly correlated with fatigue (p‚Äâ<‚Äâ0.05, corrected). These regions demonstrated significantly reduced local efficiency (p‚Äâ<‚Äâ0.026, corrected) and altered effective connectivity (p‚Äâ<‚Äâ0.001, corrected). COVID-19 may have a widespread effect on the functional connectome characterized by lower functional connectivity and altered patterns of information processing efficiency and effective information flow. This may serve as an adaptation to the pathology of SARS-CoV-2 wherein the brain can continue functioning at near expected objective levels, but patients experience lowered efficiency as brain fog.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73311-0" target="_blank">https://www.nature.com/articles/s41598-024-73311-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-71951-w" target="_blank">
            ü§ñ COVID Severity & Biomarkers for Patient Prognosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-78', this)">
            More</button></h4>
            <div id="toggle-78" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Correlating COVID-19 severity with biomarker profiles and patient prognosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how different things can affect how sick people get from the coronavirus and its long-term effects. They studied 226 people to see what makes some recover faster than others. Age, lower education, being of a certain race, overweight, having issues with blood clotting genes, heart or lung problems, diabetes, and other factors were found to make someone sicker.

The study also saw that six months later, some people had less of certain things in their blood like FVIII, VWF, and DD. Some people had joint pain, muscle aches, trouble breathing, and swelling in their legs if they had diabetes or lung problems during recovery. People who kept having high levels of DD after getting better were more likely to feel breathless and get heartburn.

Women were found to have leg swelling longer than men, and those with higher FVIII levels for a while also got swollen legs. Exercise helped prevent leg swelling. The study thinks blood vessels might be one reason why people are sicker from the virus or its long-term effects.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19's long-lasting and complex impacts have become a global concern, with diverse clinical outcomes. This study evaluated 226 participants to understand the clinical spectrum of COVID-19/Long COVID (LC), exploring how disease severity correlates with sociodemographic factors and biomarkers. Determinants related to COVID-19 severity included age (P‚Äâ<‚Äâ0.001), lower education (P‚Äâ<‚Äâ0.001), ethnicity (P‚Äâ=‚Äâ0.003), overweight (P‚Äâ<‚Äâ0.001),MTHFRgene rs1801133 (P‚Äâ=‚Äâ0.035), cardiovascular diseases (P‚Äâ=‚Äâ0.002), diabetes mellitus (DM) (P‚Äâ=‚Äâ0.006), Factor VIII (FVIII) (P‚Äâ=‚Äâ0.046), von Willebrand factor (VWF) (P‚Äâ=‚Äâ0.002), and dimer D (DD) (P‚Äâ<‚Äâ0.001). Six months later, in a portion of the monitored participants, a significant reduction in FVIII (P‚Äâ<‚Äâ0.001), VWF (P‚Äâ=‚Äâ0.002), and DD (P‚Äâ<‚Äâ0.001) levels was observed, with only DD returning to normal values. Different systemic sequelae were identified, with higher incidences of joint pain and myalgia in participants with a clinical history of DM, chronic lung disease (CLD) and sustained high interleukin 6 values in the convalescent phase. CLD, COVID-19 severity and high DD levels increased the risk of developing dyspnea and palpitations. Women were more likely to develop lower limb phlebitis long-term, while sustained elevated FVIII in the convalescent phase was associated with an increased risk of swelling. Regular physical activity had a protective effect against swelling. This study highlights factors contributing to COVID-19 severity/LC, emphasizing endothelial cell activation as a potential mechanism.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71951-w" target="_blank">https://www.nature.com/articles/s41598-024-71951-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52638-2" target="_blank">
            ü§ñ Impact of 6-Month Travel Moratorium on Malaria on Bioko Island</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-79', this)">
            More</button></h4>
            <div id="toggle-79" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of six-month COVID-19 travel moratorium onPlasmodium falciparumprevalence on Bioko Island, Equatorial Guinea</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The spread of malaria on Bioko Island is partly caused by people bringing the disease from other places. Scientists wanted to see how much this importation affects local cases. They used a six-month period when people weren't allowed to travel in and out as a test. Before and after this time, they checked if more people had malaria where many visitors come from. This showed that having visitors increases the chances of getting malaria. So stopping visitors could help reduce the number of malaria cases on the island.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Importation of malaria infections is a suspected driver of sustained malaria prevalence on areas of Bioko Island, Equatorial Guinea. Quantifying the impact of imported infections is difficult because of the dynamic nature of the disease and complexity of designing a randomized trial. We leverage a six-month travel moratorium in and out of Bioko Island during the initial COVID-19 pandemic response to evaluate the contribution of imported infections to malaria prevalence on Bioko Island. Using a difference in differences design and data from island wide household surveys conducted before (2019) and after (2020) the travel moratorium, we compare the change in prevalence between areas of low historical travel to those with high historical travel. Here, we report that in the absence of a travel moratorium, the prevalence of infection in high travel areas was expected to be 9% higher than observed, highlighting the importance of control measures that target imported infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52638-2" target="_blank">https://www.nature.com/articles/s41467-024-52638-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52803-7" target="_blank">
            ü§ñ CGAS-STING pathway limits COVID-19 in some lung cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-80', this)">
            More</button></h4>
            <div id="toggle-80" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A basally active cGAS-STING pathway limits SARS-CoV-2 replication in a subset of ACE2 positive airway cell models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The new coronavirus (SARS-CoV-2) can't enter some types of lung cells even when it should be able to. Researchers found that this might be because these cells are already stressed, leading to the release of DNA from their mitochondria (a part inside cells). This causes the cGAS-STING pathway and other immune responses to get activated without being needed, stopping the virus from growing.

By changing how these pathways work or using medicine to stop them, researchers were able to make the lung cells allow the virus to grow better. They also found that when they removed parts of the body's defense system (like cGAS-STING) in some lung cell samples, more coronavirus could grow there and cause less immune reactions.

In summary, the stress inside certain lung cells seems to be stopping the coronavirus from growing properly, even if it‚Äôs supposed to enter them. How much this happens depends on how many of these special parts (like ACE2) are present in those cells.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Host factors that define the cellular tropism of SARS-CoV-2 beyond the cognate ACE2 receptor are poorly defined. Here we report that SARS-CoV-2 replication is restricted at a post-entry step in a number of ACE2-positive airway-derived cell lines due to tonic activation of the cGAS-STING pathway mediated by mitochondrial DNA leakage and naturally occurring cGAS and STING variants. Genetic and pharmacological inhibition of the cGAS-STING and type I/III IFN pathways as well as ACE2 overexpression overcome these blocks. SARS-CoV-2 replication in STING knockout cell lines and primary airway cultures induces ISG expression but only in uninfected bystander cells, demonstrating efficient antagonism of the type I/III IFN-pathway in productively infected cells. Pharmacological inhibition of STING in primary airway cells enhances SARS-CoV-2 replication and reduces virus-induced innate immune activation. Together, our study highlights that tonic activation of the cGAS-STING and IFN pathways can impact SARS-CoV-2 cellular tropism in a manner dependent on ACE2 expression levels.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52803-7" target="_blank">https://www.nature.com/articles/s41467-024-52803-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52689-5" target="_blank">
            ü§ñ SARS-CoV-2 T cells in nose quick react with extra IFN-Œ≥</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-81', this)">
            More</button></h4>
            <div id="toggle-81" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 spike-specific nasal-resident CD49a+CD8+memory T cells exert immediate effector functions with enhanced IFN-Œ≥ production</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that after getting vaccinated against COVID-19, some people still get infected (called breakthrough infections). In their noses, there are special immune cells called T cells that recognize the virus and try to fight it. These T cells can stay in the nose for a long time‚Äîup to a year‚Äîand they respond quickly when they see pieces of the virus again. The type of T cell that does this is identified as CD49a+.

These T cells help by making chemicals called IFN-Œ≥, which signals other immune cells to attack the virus. They also stay in their resting homes (tissue-resident marker) in the nose even after a long time without seeing the virus again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Virus-specific nasal resident T cells are important for protection against subsequent infection with a similar virus. Here we examine the phenotypes and functions of SARS-CoV-2-specific T cells in the nasal mucosa of vaccinated individuals with breakthrough infection (BTI) or without infection. Nasal tissues are obtained from participants during sinus surgery. Analysis of activation-induced markers implicates that a considerable proportion of spike (S)-reactive nasal CD8+T cells express CD103, a tissue-resident marker. MHC-I multimer staining is performed to analyze the ex vivo phenotype and function of SARS-CoV-2 S-specific CD8+T cells. We detect multimer+CD8+T cells with tissue-resident phenotypes in nasal tissue samples from vaccinees without infection as well as vaccinees with BTI. Multimer+CD8+T cells remain present in nasal tissues over one year after the last exposure to S antigen, although the frequency decreases. Upon direct ex vivo stimulation with epitope peptides, nasal multimer+CD8+T cells‚Äìparticularly the CD49a+subset‚Äìexhibit immediate effector functions, including IFN-Œ≥ production. CITE-seq analysis of S-reactive AIM+CD8+T cells confirms the enhanced effector function of the CD49a+subset. These findings indicate that among individuals previously exposed to S antigen by vaccination or BTI, S-specific nasal-resident CD49a+CD8+memory T cells can rapidly respond to SARS-CoV-2 during infection or reinfection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52689-5" target="_blank">https://www.nature.com/articles/s41467-024-52689-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41591-024-03278-y" target="_blank">
            ü§ñ SARS-CoV-2 antibodies not found in bone marrow after mRNA vaccine</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-82', this)">
            More</button></h4>
            <div id="toggle-82" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2-specific plasma cells are not durably established in the bone marrow long-lived compartment after mRNA vaccination</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Simplified abstract:
MRNA vaccines against COVID-19 are good at stopping serious illness. But months later, people's body doesn't have many protection antibodies anymore. We looked inside people‚Äôs bones where antibodies are made to find out why. We found that even though some special antibody-making cells were still around, the main place these cells live (long-lived plasma cells) was mostly empty for COVID-19 antibodies. This might explain why after a while, people's body doesn't have enough protection against COVID-19 anymore.

So basically, mRNA vaccines make good antibodies, but they run out quickly because there aren‚Äôt many of the main antibody-making cells left in bones.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) mRNA vaccines are effective at protecting from severe disease, but the protective antibodies wane rapidly even though SARS-CoV-2-specific plasma cells can be found in the bone marrow (BM). Here, to explore this paradox, we enrolled 19 healthy adults at 2.5‚Äì33‚Äâmonths after receipt of a SARS-CoV-2 mRNA vaccine and measured influenza-, tetanus- or SARS-CoV-2-specific antibody-secreting cells (ASCs) in long-lived plasma cell (LLPC) and non-LLPC subsets within the BM. Only influenza- and tetanus-specific ASCs were readily detected in the LLPCs, whereas SARS-CoV-2 specificities were mostly absent. The ratios of non-LLPC:LLPC for influenza, tetanus and SARS-CoV-2 were 0.61, 0.44 and 29.07, respectively. In five patients with known PCR-proven history of recent infection and vaccination, SARS-CoV-2-specific ASCs were mostly absent from the LLPCs. We show similar results with measurement for secreted antibodies from BM ASC culture supernatant. While serum IgG titers specific for influenza and tetanus correlated with IgG LLPCs, serum IgG levels for SARS-CoV-2, which waned within 3‚Äì6‚Äâmonths after vaccination, were associated with IgG non-LLPCs. In all, our studies suggest that rapid waning of SARS-CoV-2-specific serum antibodies could be accounted for by the absence of BM LLPCs after these mRNA vaccines.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-03278-y" target="_blank">https://www.nature.com/articles/s41591-024-03278-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-61827-4" target="_blank">
            ü§ñ Key factors in digital decentralized long COVID clinical trials: pacing intervention study</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-83', this)">
            More</button></h4>
            <div id="toggle-83" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Key considerations for digital decentralised clinical trials from a feasibility study assessing pacing interventions for long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Long COVID is a condition that causes lots of problems like tiredness, trouble breathing, and brain fog. Many people who get Long COVID feel very sick and it's hard for doctors to find good treatments. 

Doctors need tools to test new ways to help people with Long COVID, but they don't have many. We made an online system where patients can join different groups doing different experiments at home. It helped us see if certain methods work better.

We learned that getting people to join these studies is really hard, and some people pretend to be in the study when they're not. Also, many people stop joining before we finish all our tests. 

We found ways to solve some of these problems so doctors can do more effective online studies for Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post COVID-19 condition or long COVID is highly prevalent and often debilitating, with key symptoms including fatigue, breathlessness, and brain fog. There is currently a lack of evidence-based treatments for this highly complex syndrome. There is a need for clinical trial platforms to rapidly evaluate nonpharmacological treatments to support affected individuals with symptom management. We co-produced a mixed methods feasibility study to evaluate a multi-arm digital decentralised clinical trial (DCT) platform to assess non-pharmacological interventions for Long COVID, using pacing interventions as an exemplar. The study demonstrated that the platform was able to successfully e-consent participants, randomise them into one of four intervention arms, capture baseline data, and capture outcomes relevant to a health economic evaluation. The study also highlighted several challenges, including difficulties with recruitment, imposter participants, and high attrition rates. We highlight how these challenges can potentially be mitigated to make a fully powered DCT more feasible.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-61827-4" target="_blank">https://www.nature.com/articles/s41598-024-61827-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73475-9" target="_blank">
            ü§ñ Clinical features and imaging in neurological Wilson's disease with claustrum damage</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-84', this)">
            More</button></h4>
            <div id="toggle-84" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Clinical and neuroimaging features in neurological Wilson‚Äôs disease with claustrum lesions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Our study looked at how often patients with a rare brain condition called Wilson's disease have problems in an area of their brain called the claustrum. Early studies said this could happen anywhere from 1.8% to 75%. We wanted to find out more about these cases, like what symptoms they had and why it might be happening.

In our study, we looked at MRI scans of 668 people with Wilson's disease. Out of the 443 who had issues shown on their brain scan, only 3 had bright claustrum signs seen on images. These signs were:
- Appeared in both sides of the brain
- Not isolated to just one part
- Usually showed up only when people had very bad symptoms

The bright claustrum sign might be a marker that inflammation is happening in the brain. The area was also hyperintense, meaning it appeared brighter on scans.

So basically, we found that problems with this brain area are not common but usually appear in people who have really severe symptoms of Wilson's disease.</p><br>
                <p><strong>Abstract</strong></p>
                <p>According to early research, the incidence of claustrum lesions in patients with neurological Wilson‚Äôs disease (WD) was inconsistent, ranging from 1.8 to 75% on magnetic resonance imaging (MRI). Our study aims to explore the incidence, clinical presentation features, iconography features, and possible pathological mechanisms in WD patients with claustrum lesions on magnetic resonance imaging (MRI), to characterize the clinical, and brain imaging findings and possible pathological mechanisms in the patients with WD. Retrospective cases meeting the inclusion criteria were studied for analyzing MRI characteristics and associated physicochemical examination data in neurological WD patients with claustrum lesions. 443 (66.3%) with brain MRI abnormalities were screened from 668 WD patients. The three (0.7%) patients with the claustrum lesions characteristics on MRI images were: (a) ‚Äúbright claustrum‚Äù in T2-weighted and FLAIR sequences, (b) bilateral symmetrical, (c) non-isolated lesions, (d) occurred only in severe neurological manifestations. The claustrum lesions are not common in neurological WD and mainly appear in cases with severe neurological symptoms. On MRI, the ‚Äúbright claustrum‚Äù signs may be a radiographic marker of neuroinflammation, the features of the lesions showed bilateral symmetry, and hyperintensity signals on T2-weighted, FLAIR, and DWI.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73475-9" target="_blank">https://www.nature.com/articles/s41598-024-73475-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41598-024-73613-3" target="_blank">
            ü§ñ Novel Isothermal Whole Genome Sequencing for Monkeypox Virus</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-85', this)">
            More</button></h4>
            <div id="toggle-85" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A novel isothermal whole genome sequencing approach for Monkeypox Virus</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Monkeypox virus causes a disease similar to chickenpox. Since May 2022, there have been more cases of people catching the virus from each other outside places where it's usually found.

Scientists use something called Whole Genome Sequencing (WGS) to track how the virus changes over time during outbreaks or for watching many diseases that could affect people. There are different ways to do this sequencing, like using big machines or special chemicals. The new method uses a DNA polymerase enzyme called Phi29 and a technique called multiple displacement amplification (MDA). This allows them to get almost all the virus's genetic material without needing lots of steps.

The scientists tested their new method with different types of samples - some from people who got sick, and other kinds too. They found out that it can work well using just 6 special pieces of DNA that act as starting points (primers). 

Their new method is good for keeping track of how the virus changes during outbreaks or to watch for it in places where we might not know about cases yet. It works even without fancy equipment, so any lab could use it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Monkeypox virus (MPXV) is the zoonotic agent responsible for mpox, an often-self-limiting pox-like disease. Since May 2022, an outbreak characterized by increased human-to-human transmission was detected outside the endemic regions. Whole genome sequencing (WGS) has been successfully used to keep track of viral evolution during outbreaks or for surveillance of multiple pathogens of public health significance. Current WGS protocols for MPXV are either based on metagenomic sequencing or tiled-PCR amplification. The latter allows multiplexing due to the efficient enrichment of the viral DNA, however, mutations or the presence of different clades can negatively influence genome coverage yield. Here, we present the establishment of a novel isothermal WGS method for MPXV based on Phi29 DNA polymerase-based multiple displacement amplification (MDA) properties making use of only 6 primers. This approach yielded from 88% up to 100% genome coverage using either alkaline denatured extracted DNA or clinical material as starting material, with the highest coverage generated by clinical material. We demonstrate that this novel isothermal WGS protocol is suitable for monitoring viral evolution during MPXV outbreaks and surveillance in any conventional laboratory setting.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73613-3" target="_blank">https://www.nature.com/articles/s41598-024-73613-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-27</strong> <a href="https://www.nature.com/articles/s41467-024-52194-9" target="_blank">
            ü§ñ Predicting Booster Immunogenicity for New Variants and Vaccine Updates</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-86', this)">
            More</button></h4>
            <div id="toggle-86" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Predicting COVID-19 booster immunogenicity against future SARS-CoV-2 variants and the benefits of vaccine updates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The SARS-CoV-2 virus keeps changing, so scientists had to update the vaccines they give people. They chose new parts of the virus to fight based on lab tests. But sometimes it took too long to make these updated vaccines before the virus changed again. Now, researchers want to know if these updates help our immune system better fight off newer versions of the virus.

They looked at data showing how well different vaccine parts worked against recent and older viruses. They found that using the new, updated parts in a booster dose helped build stronger antibodies compared to old vaccines. This means people might get more protection even if the virus changed while they were waiting for the update.

This information helps doctors decide when to give updated doses of vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The ongoing evolution of the SARS-CoV-2 virus has led to a move to update vaccine antigens in 2022 and 2023. These updated antigens were chosen and approved based largely on in vitro neutralisation titres against recent SARS-CoV-2 variants. However, unavoidable delays in vaccine manufacture and distribution meant that the updated booster vaccine was no longer well-matched to the circulating SARS-CoV-2 variant by the time of its deployment. Understanding whether the updating of booster vaccine antigens improves immune responses to subsequent SARS-CoV-2 circulating variants is a major priority in justifying future vaccine updates. Here we analyse all available data on the immunogenicity of variants containing SARS-CoV-2 vaccines and their ability to neutralise later circulating SARS-CoV-2 variants. We find that updated booster antigens give a 1.4-fold [95% CI: 1.07‚Äì1.82] greater increase in neutralising antibody levels when compared with a historical vaccine immunogen. We then use this to predict the relative protection that can be expected from an updated vaccine even when the circulating variant has evolved away from the updated vaccine immunogen. These findings help inform the rollout of future booster vaccination programmes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52194-9" target="_blank">https://www.nature.com/articles/s41467-024-52194-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-20</strong> <a href="https://www.nature.com/articles/s41598-024-72417-9" target="_blank">
            ü§ñ A Simple Way to Test for COVID-19 Delta Variant Using a Color Changing Reaction</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-87', this)">
            More</button></h4>
            <div id="toggle-87" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A colorimetric reverse-transcription loop-mediated isothermal amplification method targeting the L452R mutation to detect the Delta variant of SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A new coronavirus called SARS-CoV-2 has spread quickly around the world, causing problems for people and economies. A variant of this virus, known as Delta, is still widespread and can make COVID-19 vaccines less effective. To control the spread of the virus, scientists need to be able to detect it quickly.

This study developed a new test that can specifically identify the Delta variant of SARS-CoV-2. The test uses a color-changing reaction to show whether someone has the virus or not. When tested with 126 real samples, the test was highly accurate and sensitive, making it a useful tool for tracking COVID-19 and developing strategies to control its spread in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid spread of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has triggered global difficulties for both individuals and economies, with new variants continuing to emerge. The Delta variant of SARS-CoV-2 remains most prevalent worldwide, and it affects the efficacy of coronavirus disease 2019 (COVID-19) vaccination. Expedited testing to detect the Delta variant of SARS-CoV-2 and monitor viral transmission is necessary. This study aimed to develop and evaluate a colorimetric reverse-transcription loop-mediated isothermal amplification (RT-LAMP) technique targeting the L452R mutation in the S gene for the specific detection of the Delta variant. In the test, positivity was indicated as a color change from purple to yellow. The assay‚Äôs 95% limit of detection was 57 copies per reaction for the L452R (U1355G)-specific standard plasmid. Using 126 clinical samples, our assay displayed 100% specificity, 97.06% sensitivity, and 98.41% accuracy in identifying the Delta variant of SARS-CoV-2 compared to real-time RT-PCR. To our knowledge, this is the first colorimetric RT-LAMP assay that can differentiate the Delta variant from its generic SARS-CoV-2, enabling it as an approach for studying COVID-19 demography and facilitating proper effective control measure establishment to fight against the reemerging variants of SARS-CoV-2 in the future.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72417-9" target="_blank">https://www.nature.com/articles/s41598-024-72417-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-20</strong> <a href="https://www.nature.com/articles/s41598-024-73012-8" target="_blank">
            ü§ñ Mpox in Macaques with Endo and Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-88', this)">
            More</button></h4>
            <div id="toggle-88" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mpox infection of stromal cells and macrophages of macaque with endometriosis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The mpox outbreak in 2022-2023 was a new health problem that showed mpox can be spread through sex, though it mostly affected men who have sex with other men (MSM). Women are also at risk, especially during pregnancy.

We studied some monkeys to see how they got infected. We found that mpox didn't just live in their skin - it was also inside the vagina and uterus of the monkeys.

The study showed mpox could grow in special tissue called endometriosis. This is a condition where tissue like what grows in your uterus but is found elsewhere, usually on other organs like the ovaries. 

We learned that mpox likes to live in certain types of cells - specifically ones that help protect against infections and some of the cells inside the reproductive system.

The study also suggested women with this condition might be more at risk for bad things happening if they get mpox.

In general, this research shows us what happens when a disease like mpox affects the genital area. The monkeys were used because they're similar to humans in many ways and can help understand how diseases work.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The mpox outbreak of 2022‚Äì2023 represented a new global health challenge and recognition of mpox as a sexually transmitted disease. The majority of cases were reported in men who have sex with men (MSM), but women are also susceptible, especially during pregnancy. We evaluated the reproductive tracts of a subset of macaques from a large rechallenge study of mpox infection with virus from the 2022 outbreak and identified intraabdominal mpox replication associated with endometriosis. Mpox virus (MPXV) was found not only in skin, but in the cervix, the uterus, and periovarian endometriotic lesions of the affected macaque. Mpox replication preferentially targeted vimentin-positive poorly differentiated endometriotic stromal tissue and infiltrating macrophages in the reproductive tract. Mpox tropism for stromal cells and macrophages has broad implications for mpox pathogenesis and associated clinical syndromes. In addition, women with endometriosis may be at heightened risk for adverse outcomes associated with mpox infection. The rhesus macaque provides rare insight into this disease and the potential complications of mpox infection in the context of genitourinary tract disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-73012-8" target="_blank">https://www.nature.com/articles/s41598-024-73012-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-19</strong> <a href="https://www.nature.com/articles/s41467-024-52246-0" target="_blank">
            ü§ñ Severe fever causes kids' immune cells to fail</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-89', this)">
            More</button></h4>
            <div id="toggle-89" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Shared neutrophil and T cell dysfunction is accompanied by a distinct interferon signature during severe febrile illnesses in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This abstract is about studying how the body's immune system reacts differently when kids get very sick from various illnesses, like coronavirus or bacterial infections. They looked at blood samples from 74 kids with a condition called MIS-C (which happens after they had COVID), 30 with bacterial infections, 16 with viral infections, and 8 with Kawasaki disease. 

They found that in most cases, the body's neutrophils (a type of white blood cell) were overreacting or dying too quickly. They also noticed that T cells from kids with MIS-C and bacterial infections didn't work as well as they should.

For viral infections like those from viruses other than SARS-CoV-2, their immune system showed less response to interferons, which are important for fighting off infections. This finding could help doctors figure out better ways to treat very sick children who have these illnesses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe febrile illnesses in children encompass life-threatening organ dysfunction caused by diverse pathogens and other severe inflammatory syndromes. A comparative approach to these illnesses may identify shared and distinct features of host immune dysfunction amenable to immunomodulation. Here, using immunophenotyping with mass cytometry and cell stimulation experiments, we illustrate trajectories of immune dysfunction in 74 children with multi-system inflammatory syndrome in children (MIS-C) associated with SARS-CoV-2, 30 with bacterial infection, 16 with viral infection, 8 with Kawasaki disease, and 42 controls. We explore these findings in a secondary cohort of 500 children with these illnesses and 134 controls. We show that neutrophil activation and apoptosis are prominent in multi-system inflammatory syndrome, and that this is partially shared with bacterial infection. We show that memory T cells from patients with multi-system inflammatory syndrome and bacterial infection are exhausted. In contrast, we show viral infection to be characterized by a distinct signature of decreased interferon signaling and lower interferon receptor gene expression. Improved understanding of immune dysfunction may improve approaches to immunomodulator therapy in severe febrile illnesses in children.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52246-0" target="_blank">https://www.nature.com/articles/s41467-024-52246-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-67504-w" target="_blank">
            ü§ñ COVID-19 Outcomes Predicted by Early Host Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-90', this)">
            More</button></h4>
            <div id="toggle-90" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Oral SARS-CoV-2 host responses predict the early COVID-19 disease course</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied people who had COVID-19 to see how it affects their bodies, especially in their mouths and throats. They collected saliva and blood samples from 45 people and looked for signs of the virus and how their immune system responded.

They found that most people with COVID-19 had the virus in their saliva and throat wash for up to 4 weeks after getting sick. Some people even showed signs of the virus in their saliva when they didn't have symptoms anymore.

The researchers also discovered that people who were severely ill with COVID-19 tended to have more antibodies (which fight off infections) in their blood and saliva, especially in the early stages of the illness. They found that some people's immune system responded differently based on their sex, with women having higher levels of certain antibodies in their saliva.

The study suggests that the virus can replicate (multiply) in the mouth and throat for a while after symptoms go away, which may be important for understanding how COVID-19 is transmitted to others. The researchers also found evidence that some people's immune system may "confuse" the virus with other proteins in their body, which could affect how well they fight off the infection.

Overall, this study provides new insights into how COVID-19 affects the body and may help us understand more about the disease and how to prevent it from spreading.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Oral fluids provide ready detection of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and host responses. This study sought to evaluate relationships between oral virus, oral and systemic anti-SARS-CoV-2-specific antibodies, and symptoms. Oral fluids (saliva/throat wash (saliva/TW)) and serum were collected from asymptomatic and symptomatic, nasopharyngeal (NP) SARS-CoV-2 RT-qPCR+‚Äâhuman participants (n‚Äâ=‚Äâ45). SARS-CoV-2 RT-qPCR and N-antigen detection by immunoblot and lateral flow assay (LFA) were performed. RT-qPCR for subgenomic RNA (sgRNA) was sequence confirmed. SARS-CoV-2-anti-S protein RBD LFA and ELISA assessed IgM and IgG responses. Structural analysis identified host salivary molecules analogous to SARS-CoV-2-N-antigen. At time of enrollment (baseline, BL), LFA-detected N-antigen in 86% of TW and was immunoblot-confirmed. However, only 3/17 were saliva/TW qPCR+‚Äâ. Sixty percent of saliva and 83% of TW demonstrated persistent N-antigen at 4¬†weeks. N-antigen LFA signal in three anti-spike sero-negative participants suggested potential cross-detection of 4 structurally analogous salivary RNA binding proteins (alignment 19‚Äì29aa, RMSD 1‚Äì1.5 Angstroms). At enrollment, symptomatic participants demonstrated replication-associated sgRNA junctions, were IgG+‚Äâ(94%/100% in saliva/TW), and IgM+‚Äâ(63%/54%). At 4¬†weeks, SARS-CoV-2 IgG (100%/83%) and IgM (80%/67%) persisted. Oral and serum IgG correlated 100% with NP+‚ÄâPCR status. Cough and fatigue severity (p=‚Äâ0.010 and 0.018 respectively), and presence of weakness, nausea, and composite upper respiratory symptoms (p=‚Äâ0.037, 0.005, and 0.017, respectively) were negatively associated with saliva IgM but not TW or serum IgM. Throat wash IgM levels were higher in women compared to men, although the association did not reach statistical significance (median: 290 (female) versus 0.697,p=‚Äâ0.056). Important to transmission and disease course, oral viral replication and persistence showed clear relationships with select symptoms and early oral IgM responses during early infection. N-antigen cross-reactivity may reflect mimicry of structurally analogous host proteins.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-67504-w" target="_blank">https://www.nature.com/articles/s41598-024-67504-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-71413-3" target="_blank">
            ü§ñ UPLC-PDA method for three drugs in plasma</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-91', this)">
            More</button></h4>
            <div id="toggle-91" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>UPLC-PDA factorial design assisted method for simultaneous determination of oseltamivir, dexamethasone, and remdesivir in human plasma</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A new way of testing medicine levels in blood using UPLC (a fast and precise lab technique) has been created. This method can measure three medicines at once: oseltamivir phosphate, remdesivir, dexamethasone, and a special standard called daclatasvir dihydrochloride. The medicines are put into the system one by one using tiny needles, and then separated on a special column that uses a mixture of methanol (a clear liquid) and ammonium acetate solution to sort them out based on their colors at different wavelengths. It takes just 5 minutes to do all this! 

The method was tested for accuracy and reliability according to international guidelines. Environmental Impact Scores, Green Assessment of Processes in Analysis, and Acceptability and Goodness-Instrument Rating (Eco-score, GAPI, and AGREE) showed that it is a good and environmentally friendly way to measure these medicines. This means it's useful and won't harm the environment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A green and simple UPLC method was developed and optimized, adopting a factorial design for simultaneous determination of oseltamivir phosphate and remdesivir with dexamethasone as a co-administered drug in human plasma and using daclatasvir dihydrochloride as an internal standard within 5¬†min. The separation was established on UPLC column BEH C181.7¬†Œºm (2.1\(\times \)100.0¬†mm) connected to UPLC pre-column BEH 1.7¬†Œºm (2.1\(\times \)5.0¬†mm) at 50¬†¬∞C with an injection volume of 10 ŒºL. The photodiode array detector (PDA) was set at three wavelengths of 220, 315, and 245¬†nm for oseltamivir phosphate, the internal standard, and both dexamethasone and remdesivir, respectively. The mobile phase consisted of methanol and ammonium acetate solution (40¬†mM) adjusted to pH 4 in a ratio of 61.5:38.5 (v/v) with a flow rate of 0.25¬†mL¬†min‚àí1. The calibration curves were linear over 500.0‚Äì5000.0¬†ng¬†mL‚àí1for oseltamivir phosphate, over 10.0‚Äì500.0¬†ng¬†mL‚àí1and 500.0‚Äì5000.0¬†ng¬†mL‚àí1for dexamethasone, and over 20.0‚Äì500¬†ng¬†mL‚àí1and 500.0‚Äì5000.0¬†ng¬†mL‚àí1for remdesivir. The Gibbs free energy and Van't Hoff plots were used to investigate the effect of column oven temperatures on retention times. Fluoride-EDTA anticoagulant showed inhibition activity on the esterase enzyme in plasma. The proposed method was validated according to the M10 ICH, FDA, and EMA‚Äôs bioanalytical guidelines. According to Eco-score, GAPI, and AGREE criteria, the proposed method was considered acceptable green.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71413-3" target="_blank">https://www.nature.com/articles/s41598-024-71413-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41467-024-51924-3" target="_blank">
            ü§ñ Emerging Antiviral-Resistant COVID Variants in People With Weakened Immune Systems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-92', this)">
            More</button></h4>
            <div id="toggle-92" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Emergence of transmissible SARS-CoV-2 variants with decreased sensitivity to antivirals in immunocompromised patients with persistent infections</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

We studied how antiviral treatments affect the COVID-19 virus in people with weakened immune systems. Fifteen patients were given different treatments, including remdesivir and nirmatrelvir-ritonavir, to see if the virus developed resistance.

The results showed that nine out of 15 patients had viruses with mutations that made them harder to treat. We isolated a COVID-19 virus from one patient that was resistant to two common antiviral medications: remdesivir and nirmatrelvir.

This study is important because it shows that the virus can develop resistance to these treatments, even after prolonged use. The infected virus was also shown to be transmissible to other animals in a lab experiment.</p><br>
                <p><strong>Abstract</strong></p>
                <p>We investigated the impact of antiviral treatment on the emergence of SARS-CoV-2 resistance during persistent infections in immunocompromised patients (n=‚Äâ15). All patients received remdesivir and some also received nirmatrelvir-ritonavir (n=‚Äâ3) or therapeutic monoclonal antibodies (n=‚Äâ4). Sequence analysis showed that nine patients carried viruses with mutations in the nsp12 (RNA dependent RNA polymerase), while four had viruses with nsp5 (3C protease) mutations. Infectious SARS-CoV-2 with a double mutation in nsp5 (T169I) and nsp12 (V792I) was recovered from respiratory secretions 77 days after initial COVID-19 diagnosis from a patient sequentially¬†treated with nirmatrelvir-ritonavir¬†and remdesivir. In vitro characterization confirmed its decreased sensitivity to remdesivir and nirmatrelvir, which was overcome by combined antiviral treatment. Studies in golden Syrian hamsters demonstrated efficient transmission to contact animals. This study documents the isolation of SARS-CoV-2 carrying resistance mutations to both nirmatrelvir and remdesivir from a patient and demonstrates its transmissibility in vivo.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51924-3" target="_blank">https://www.nature.com/articles/s41467-024-51924-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41392-024-01956-4" target="_blank">
            ü§ñ Identifying ACTN4 as a new potential treatment for COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-93', this)">
            More</button></h4>
            <div id="toggle-93" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Characterization of ACTN4 as a novel antiviral target against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists are trying to find new ways to stop COVID-19 from spreading. They've discovered that a protein called ACTN4 plays a crucial role in this process. When ACTN4 is reduced, it makes it harder for COVID-19 to multiply inside cells.

Researchers also found two substances, YS-49 and demethyl-coclaurine, that can block COVID-19 infection. These substances work by targeting the protein that helps COVID-19 replicate.

This study shows how important ACTN4 is in stopping COVID-19 from spreading and reveals new potential candidates for developing medicines to fight against the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The various mutations in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pose a substantial challenge in mitigating the viral infectivity. The identification of novel host factors influencing SARS-CoV-2 replication holds potential for discovering new targets for broad-spectrum antiviral drugs that can combat future viral mutations. In this study, potential host factors regulated by SARS-CoV-2 infection were screened through different high-throughput sequencing techniques and further identified in cells. Subsequent analysis and experiments showed that the reduction of m6A modification level onACTN4(Alpha-actinin-4) mRNA leads to a decrease in mRNA stability and translation efficiency, ultimately inhibiting ACTN4 expression. In addition, ACTN4 was demonstrated to target nsp12 for binding and characterized as a competitor for SARS-CoV-2 RNA and the RNA-dependent RNA polymerase complex, thereby impeding viral replication. Furthermore, two ACTN4 agonists, YS-49 and demethyl-coclaurine, were found to dose-dependently inhibit SARS-CoV-2 infection in both Huh7 cells and K18-hACE2 transgenic mice. Collectively, this study unveils the pivotal role of ACTN4 in SARS-CoV-2 infection, offering novel insights into the intricate interplay between the virus and host cells, and reveals two potential candidates for future anti-SARS-CoV-2 drug development.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01956-4" target="_blank">https://www.nature.com/articles/s41392-024-01956-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41598-024-71791-8" target="_blank">
            ü§ñ Domestic Cats and COVID-19: Comparing Virus Severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-94', this)">
            More</button></h4>
            <div id="toggle-94" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The comparison of pathogenicity among SARS-CoV-2 variants in domestic cats</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists found that a type of coronavirus called SARS-CoV-2 can infect domestic cats and even wild animals. They wanted to know if cats play a role in spreading this virus, so they did experiments on cats to see how well it spreads and how their bodies react.

The results showed that cats are very susceptible to the SARS-CoV-2 virus and can spread it for up to 7 days after being infected. However, one type of the virus, called Omicron, didn't spread as well in cat tissues and caused less severe symptoms. The study also found that cats' bodies produced antibodies to fight off the virus.

This suggests that while cats might be able to catch SARS-CoV-2, they're not very good at spreading it around, especially with the Omicron variant.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has been detected or isolated from domestic cats. It is unclear whether cats play an important role in the SARS-CoV-2 transmission cycle. In this study, we examined the susceptibility of cats to SARS-CoV-2, including wild type and variants, by animal experiments. Cats inoculated with wild type, gamma, and delta variants secreted a large amount of SARS-CoV-2 for 1¬†week after the inoculation from nasal, oropharyngeal, and rectal routes. Only 100 TCID50of virus could infect cats and replicate well without severe clinical symptoms. In addition, one cat inoculated with wild type showed persistent virus secretion in feces for over 28¬†days post-inoculation (dpi). The titer of virus-neutralizing (VN) antibodies against SARS-CoV-2 increased from 11 dpi, reaching a peak at 14 dpi. However, the omicron variant could not replicate well in cat tissues and induced a lower titer of VN antibodies. It is concluded that cats were highly susceptible to SARS-CoV-2 infection, but not to the Omicron Variant, which caused the attenuated pathogenicity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71791-8" target="_blank">https://www.nature.com/articles/s41598-024-71791-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-18</strong> <a href="https://www.nature.com/articles/s41541-024-00961-6" target="_blank">
            ü§ñ A New Vaccine Approach: Using Macrophages to Boost Immune Response Against COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-95', this)">
            More</button></h4>
            <div id="toggle-95" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Low-frequency CD8+T cells induced by SIGN-R1+macrophage-targeted vaccine confer SARS-CoV-2 clearance in mice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 vaccine needs two types of help from our immune system: special cells called T cells that can recognize and attack the virus, and antibodies that neutralize it. Our previous research showed that using a special delivery system called pullulan nanogel (PNG) could make these T cells stronger. We found out that PNG works by binding to a specific receptor on certain immune cells in our lymph nodes.

In this study, we tested if using PNG with a part of the COVID-19 virus's spike protein as the vaccine could help protect against infection. In mice with COVID-19-like symptoms, we saw that the vaccine decreased the viral load and helped them survive longer. We also found out that even low-frequency T cells that responded quickly to the virus were important for clearing the virus.

Overall, our study suggests that targeting specific immune cells in the lymph nodes can help make vaccines more effective by inducing strong T cell responses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Vaccine-induced T cells and neutralizing antibodies are essential for protection against SARS-CoV-2. Previously, we demonstrated that an antigen delivery system, pullulan nanogel (PNG), delivers vaccine antigen to lymph node medullary macrophages and thereby enhances the induction of specific CD8+T cells. In this study, we revealed that medullary macrophage-selective delivery by PNG depends on its binding to a C-type lectin SIGN-R1. In a K18-hACE2 mouse model of SARS-CoV-2 infection, vaccination with a PNG-encapsulated receptor-binding domain of spike protein decreased the viral load and prolonged the survival in the CD8+T cell- and B cell-dependent manners. T cell receptor repertoire analysis revealed that although the vaccine induced T cells at various frequencies, low-frequency specific T cells mainly promoted virus clearance. Thus, the induction of specific CD8+T cells that respond quickly to viral infection, even at low frequencies, is important for vaccine efficacy and can be achieved by SIGN-R1+medullary macrophage-targeted antigen delivery.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00961-6" target="_blank">https://www.nature.com/articles/s41541-024-00961-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s42003-024-06872-5" target="_blank">
            ü§ñ COVID-19 Becomes More Severe in Mice Without Important Immune Proteins</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-96', this)">
            More</button></h4>
            <div id="toggle-96" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Deficiency ofTlr7andIrf7in mice increases the severity of COVID-19 through the reduced interferon production</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied how a deficiency in a certain receptor (Tlr7) affects the severity of COVID-19 in mice. They found that when this receptor is missing, the body has trouble producing interferons (IFNs), which are important for fighting the virus. The study showed that without Tlr7 and another related protein called Irf7, the mice developed more severe symptoms, had higher levels of the virus in their lungs, and took longer to produce antibodies against COVID-19. This suggests that the activation of these receptors is crucial for developing a strong defense against the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Toll-like receptor 7 (Tlr7)deficiency-accelerated severe COVID-19 is associated with reduced production of interferons (IFNs). However, the underlying mechanisms remain elusive. To address these questions, we utilizeTlr7andIrf7deficiency mice, single-cell RNA analysis together with bone marrow transplantation approaches. We demonstrate that at the early phase of infection, SARS-CoV-2 causes the upregulation ofTlr7,Irf7, and IFN pathways in the lungs of the infected mice. The deficiency ofTlr7andIrf7globally and/or in immune cells in mice increases the severity of COVID-19 via impaired IFN activation in both immune and/or non-immune cells, leading to increased lung viral loads. These effects are associated with reduced IFN alpha and gamma levels in the circulation. The deficiency ofTlr7tends to cause the reduced production and nuclear translocation of interferon regulatory factor 7 (IRF7) in the lungs of the infected mice, indicative of reduced IRF7 activation. Despite higher amounts of lung viral antigen,Tlr7orIrf7deficiency resulted in substantially reduced production of antibodies against SARS-CoV-2, thereby delaying the viral clearance. These results highlight the importance of the activation of TLR7 and IRF7 leading to IFN production on the development of innate and adaptive immunity against COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s42003-024-06872-5" target="_blank">https://www.nature.com/articles/s42003-024-06872-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41467-024-51021-5" target="_blank">
            ü§ñ Marburg virus survivor antibodies: Finding new vaccine targets</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-97', this)">
            More</button></h4>
            <div id="toggle-97" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Longitudinal proteome-wide antibody profiling in Marburg virus survivors identifies wing domain immunogen for vaccine design</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how the body makes antibodies after getting sick from the Marburg virus over many years. They found that different types of antibodies form over time:

1. The body starts by making lots of IgM and IgA antibodies quickly.
2. Later, it switches to making more IgG antibodies, which last longer.

The study also noticed:
- Antibodies against parts of a protein called GP are strongest.
- Some old antibody types like IgM and IgA get less common over time.
- The body forms strong bonds with special Fc receptors that can interact with the virus.

These findings suggest that even though many antibodies don't help stop the virus directly, they might still fight against it indirectly by binding to parts of these proteins. This could be useful for developing treatments and vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Limited knowledge exists on the quality of polyclonal antibody responses generated following Marburg virus (MARV) infection and its evolution in survivors. In this study, we evaluate MARV proteome-wide antibody repertoire longitudinally in convalescent phase approximately every six months for five years following MARV infection in ten human survivors. Differential kinetics were observed for IgM vs IgG vs IgA epitope diversity, antibody binding, antibody affinity maturation and Fc-receptor interaction to MARV proteins. Durability of MARV-neutralizing antibodies is low in survivors. MARV infection induces a diverse epitope repertoire with predominance against GP, VP40, VP30 and VP24 that persisted up to 5 years post-exposure. However, the IgM and IgA repertoire declines over time. Within MARV-GP, IgG recognize antigenic sites predominantly in the amino-terminus, wing domain and GP2-heptad repeat. Interestingly, MARV infection generates robust durable Fc…£RI, Fc…£RIIA and Fc…£RIIIA IgG-Fc receptor interactions. Immunization with immunodominant MARV epitopes reveals conserved wing region between GP1 and GP2, induces neutralizing antibodies against MARV. These findings demonstrate that MARV infection generates a diverse, long-lasting, non-neutralizing, IgG antibody repertoire that perturbs disease by Fc…£R activity. This information, along with discovery of neutralizing immunogen in wing domain, could aid in development of effective therapeutics and vaccines against Marburg virus.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51021-5" target="_blank">https://www.nature.com/articles/s41467-024-51021-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41598-024-71598-7" target="_blank">
            ü§ñ How sampling method affects sebum stability in mass spec studies.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-98', this)">
            More</button></h4>
            <div id="toggle-98" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>How storage post sampling influences the stability of sebum when used for mass spectrometry metabolomics analysis?</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Sebum is a natural oil produced by skin glands. Recent studies found this oil can show signs of diseases like Parkinson's. Since getting sebum is easy and doesn't hurt, doctors might use it in tests.

Researchers looked at how well they could test sebum over time and under different temperatures. They collected sebum from people, stored it for up to four weeks at room temperature or very cold (like in a freezer), and then tested it using two types of machines that break down the oil into tiny parts we can see. 

They didn't find any difference between how the samples looked when they were stored for different amounts of time or at different temperatures. But, some small groups of samples mixed up with others.

This study shows sebum is good to use in tests and doesn‚Äôt need to be kept cold forever. This means people can collect their own oil from home without worrying too much about it spoiling before they send it to the lab.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Sebum is a biofluid excreted by sebaceous glands in the skin. In recent years sebum has been shown to contain endogenous metabolites diagnostic of disease, with remarkable results for Parkinson‚Äôs Disease. Given that sebum sampling is facile and non-invasive, its potential for use in clinical biochemistry diagnostic assays should be explored including the parameters for standard operating procedures around collection, transport, and storage. To this aim we have here investigated the reproducibility of mass spectrometry data from sebum in relation to both storage temperature and length of storage. Sebum samples were collected from volunteers and stored for up to four weeks at a range of temperatures: ambient (circa17¬†¬∞C), ‚àí20¬†¬∞C and ‚àí80¬†¬∞C. Established extraction protocols were employed and samples were analysed by two chromatographic mass spectrometry techniques and data investigated using PCA, PLS-DA and ANOVA. We cannot discriminate samples as a function of storage temperature or time stored in unsupervised analysis using data acquired via TD‚ÄìGC‚ÄìMS and LC‚ÄìIM‚ÄìMS, although the sampling of volatiles was susceptible to batch effects. This study indicates that the requirements for storage and transport of sebum samples that may be used in clinical assays are less stringent than for liquid samples and indicate that sebum is suitable for remote and at home sampling prior to analysis.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71598-7" target="_blank">https://www.nature.com/articles/s41598-024-71598-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-17</strong> <a href="https://www.nature.com/articles/s41598-024-72453-5" target="_blank">
            ü§ñ Monitoring COVID-19 Antibodies in Blood and Saliva Samples</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-99', this)">
            More</button></h4>
            <div id="toggle-99" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Monitoring SARS-CoV-2 IgA, IgM and IgG antibodies in dried blood and saliva samples using antibody proximity extension assays (AbPEA)</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists developed a new test that can detect if someone has antibodies against COVID-19 just by looking at a small blood or saliva sample on a paper disc. The test uses special molecules that bind to the antibodies and then helps create a tiny piece of DNA, which is measured using a machine. This makes it possible to quickly and easily check if people have been vaccinated or infected with COVID-19.

The study showed that this new test works well for detecting antibodies in blood samples, but less effectively in saliva samples. However, the results still suggest that it's a useful tool for monitoring the body's response to vaccination.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Using a modified proximity extension assay, total and immunoglobulin (Ig) class-specific anti-SARS-CoV-2 antibodies were sensitively and conveniently detected directly from √∏1.2¬†mm discs cut from dried blood and saliva spots (DBS and DSS) without the need for elution. For total Ig detection, antigen probes were prepared by conjugating recombinant spike protein subunit 1 (S1-RBD) to a pair of oligonucleotides. To detect isotype-specific antibody reactivity, one antigen probe was replaced with oligonucleotide-conjugated antibodies specific for antibody isotypes. Binding of pairs of oligonucleotide-conjugated probes to antibodies in patient samples brings oligonucleotides in proximity. An added DNA polymerase uses a transient hybridization between the oligonucleotides to prime synthesis of a DNA strand, which serves as a DNA amplicon that is quantified by real-time PCR. The S1-RBD-specific IgG, IgM, and IgA antibodies in DBS samples collected over the course of a first and second vaccination exhibited kinetics consistent with previous reports. Both DBS and DSS collected from 42 individuals in the autumn of 2023 showed significant level of total S1-RBD antibodies with a correlation of R‚Äâ=‚Äâ0.70. However, levels in DSS were generally 10 to 100-fold lower than in DBS. Anti-S1-RBD IgG and IgA in DSS demonstrated a correlation of R‚Äâ=‚Äâ0.6.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72453-5" target="_blank">https://www.nature.com/articles/s41598-024-72453-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-13</strong> <a href="https://www.nature.com/articles/s41419-024-07043-4" target="_blank">
            ü§ñ Text Mining for Long COVID Symptoms Analysis in Hospitalized Patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-100', this)">
            More</button></h4>
            <div id="toggle-100" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Text mining method to unravel long COVID‚Äôs clinical condition in hospitalized patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Long COVID is when you feel sick for a really long time after you get an illness like the flu or coronavirus. This can happen even weeks or months later. It's hard to understand because different people have different symptoms, and doctors use different words to describe them.

In this study, scientists used special computer tools to look at old medical records from a hospital in Brazil. They looked for patterns and similar terms that might represent the same condition. This helped them see what long COVID looks like across many patients. They also found tricky words (like those with extra parts or opposite meanings) that can be confusing.

By using these new computer tools, they hope to better understand this long-lasting sickness so doctors can help more people who are sick from it. These tools could work in other hospitals too!</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long COVID is characterized by persistent that extends symptoms beyond established timeframes. Its varied presentation across different populations and healthcare systems poses significant challenges in understanding its clinical manifestations and implications. In this study, we present a novel application of text mining technique to automatically extract unstructured data from a long COVID survey conducted at a prominent university hospital in S√£o Paulo, Brazil. Our phonetic text clustering (PTC) method enables the exploration of unstructured Electronic Healthcare Records (EHR) data to unify different written forms of similar terms into a single phonemic representation. We used n-gram text analysis to detect compound words and negated terms in Portuguese-BR, focusing on medical conditions and symptoms related to long COVID. By leveraging text mining, we aim to contribute to a deeper understanding of this chronic condition and its implications for healthcare systems globally. The model developed in this study has the potential for scalability and applicability in other healthcare settings, thereby supporting broader research efforts and informing clinical decision-making for long COVID patients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41419-024-07043-4" target="_blank">https://www.nature.com/articles/s41419-024-07043-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-71535-8" target="_blank">
            ü§ñ Complications After COVID Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-101', this)">
            More</button></h4>
            <div id="toggle-101" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Symptom burden, coagulopathy and heart disease after acute SARS-CoV-2 infection in primary practice</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists conducted a study called SETANTA to investigate how COVID-19 affects people's hearts. They looked at patients who had recently had COVID-19 and wanted to see if they still had symptoms or heart problems. The study found that many participants (83%) still had symptoms like fatigue, headaches, etc. 

The researchers also used tests to check the patients' hearts for damage and found that some of them had abnormal results. For example, about 2% of participants had inflammation in their hearts, while about 17% had a problem with their heart's ability to pump blood.

Another important finding was that many patients (32%) had high levels of a protein called von Willebrand factor antigen, which can indicate an increased risk of clotting. The study suggests that people who recently recovered from COVID-19 may be at higher risk for future health problems due to persistent symptoms and abnormal heart readings.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SETANTA (Study of HEarT DiseAse and ImmuNiTy After COVID-19 in Ireland) study aimed to investigate symptom burden and incidence of cardiac abnormalities after severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)/COVID-19 and to correlate these results with biomarkers of immunological response and coagulation. SETANTA was a prospective, single-arm observational cross-sectional study condcuted in a primary practice setting, and prospectively registered with ClinicalTrials.gov (identifier: NCT04823182). Patients with recent COVID-19 infection (‚â•‚Äâ6¬†weeks and ‚â§‚Äâ12¬†months) were prospectively enrolled. Primary outcomes of interest were markers of cardiac injury detected by cardiac magnetic resonance imaging (CMR), which included left ventricular ejection fraction, late gadolinium enhancement and pericardial abnormalities, as well as relevant biomarkers testing immunological response and coagulopathy. 100 patients (n‚Äâ=‚Äâ129 approached) were included, amongst which 64% were female. Mean age of the total cohort was 45.2 years. The median (interquartile range) time interval between COVID-19 infection and enrolment was 189 [125, 246]¬†days. 83% of participants had at least one persistent symptom, while 96% had positive serology for prior SARS-CoV-2 infection. Late gadolinium enhancement, pericardial effusion, was present in 2.2% and 8.3% respectively, while left ventricular ejection fraction was below the normal reference limit in 17.4% of patients. Von Willebrand factor antigen was elevated in 32.7% of patients and Fibrinogen and D-Dimer levels were found to be elevated in 10.2% and 11.1% of patients, respectively. In a cohort of primary practice patients recently recovered from SARS-CoV-2 infection, prevalence of persistent symptoms and markers of abnormal coagulation were high, despite a lower frequency of abnormalities on CMR compared with prior reports of patients assessed in a hospital setting.Trial Registration: Clinicaltrials.gov, NCT04823182 (prospectively registered on 30th March 2021).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71535-8" target="_blank">https://www.nature.com/articles/s41598-024-71535-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s43856-024-00599-4" target="_blank">
            ü§ñ People Taking SGLT2 Inhibitors for COVID-19 May Face Cardiovascular and Kidney Problems</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-102', this)">
            More</button></h4>
            <div id="toggle-102" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Adverse cardiovascular and kidney outcomes in people with SARS-CoV-2 treated with SGLT2 inhibitors</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Background: Some people who got infected with COVID-19 also take medicine for high blood sugar. We wanted to know if taking a specific type of diabetes medication called SGLT2 inhibitors helps prevent heart and kidney problems.

Methods: We looked at medical records of over 100,000 veterans in the US who had COVID-19 and took diabetes medication between March 2020 and June 2023. Some took SGLT2 inhibitors while others took different types of diabetes medication.

Results: Our study found that people taking SGLT2 inhibitors were less likely to have serious heart problems (such as a heart attack or stroke) and kidney issues (such as needing dialysis). They also had fewer hospitalizations, anemia, and acute kidney injury. These benefits were seen after about 1.5 years of follow-up.

Conclusions: Taking SGLT2 inhibitors may help people with COVID-19 on diabetes medication avoid heart and kidney problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundWhether use of SGLT2 inhibitors reduces the risk of cardiovascular and kidney events in people who contracted SARS-CoV-2 infection is not clear.MethodsWe used the healthcare databases of the United States Department of Veterans Affairs to build a cohort of 107,776 participants on antihyperglycemic therapy and had SARS-CoV-2 infection between March 01, 2020 and June 10, 2023. Within them, 11,588 used SGLT2 inhibitors and 96,188 used other antihyperglycemics. We examined the risks of major adverse cardiovascular events (MACE)‚Äîa composite of death, myocardial infarction and stroke, and major adverse kidney events (MAKE)‚Äîa composite of death, eGFR decline >¬†50%, and end stage kidney disease after balancing baseline characteristics between groups through inverse probability weighting. Survival analyses were conducted to generate hazard ratio (HR) and absolute risk reduction per 100 person-years (ARR).ResultsOver a median follow up of 1.57 (IQR: 1.05‚Äì2.49) years, compared to the control group, SGLT2 inhibitors use is associated with reduced risk of MACE (HR 0.82 (0.77, 0.88), ARR 1.73 (1.21, 2.25)) and reduced risk of MAKE (HR 0.75 (0.71, 0.80), ARR 2.62 (2.13, 3.11)). Compared to the control group, SGLT2 inhibitors use is associated with reduced risk of the secondary outcomes of hospitalization (HR 0.94 (0.90, 0.98), ARR 1.06 (1.36, 1.76)), anemia (HR 0.71 (0.65, 0.76), ARR 2.43 (1.95, 2.90)), and acute kidney injury (HR 0.84 (0.79, 0.89), ARR 1.86 (1.29, 2.42)).ConclusionsAmong people with SARS-CoV-2 infection on antihyperglycemic therapy, compared to those on other antihyperglycemics, those on SGLT2 inhibitors have less risk of adverse cardiovascular and kidney outcomes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00599-4" target="_blank">https://www.nature.com/articles/s43856-024-00599-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-72262-w" target="_blank">
            ü§ñ A nasal spray that protects against COVID-19 and helps maintain normal nose function</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-103', this)">
            More</button></h4>
            <div id="toggle-103" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Astodrimer sodium nasal spray forms a barrier to SARS-CoV-2 in vitro and preserves normal mucociliary function in human nasal epithelium</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

COVID-19 can be very serious for some people, especially those with weakened immune systems. To protect against this and other respiratory infections, scientists are looking at ways to create sprays that can help block the virus from entering the body through the nose.

A new nasal spray called Astodrimer Sodium (AS) was tested along with four other types of sprays. The AS spray showed promise in blocking the virus and not harming the delicate cells in the nose. It also didn't interfere with the natural mucus-clearing function of the nose.

In contrast, some of the other sprays had limited ability to block the virus and even caused problems by slowing down or stopping the mucus-clearing process. The AS spray seems to be a safe and effective option for protecting against respiratory viruses in the nose.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 remains a severe condition for many including immunocompromised individuals. There remains a need for effective measures against this and other respiratory infections, which transmit via virus-laden droplets that reach the nasal or oral mucosae. Nasal sprays offer potential protection against viruses. Such formulations should preserve normal nasal mucociliary function. The antiviral barrier efficacy and effects on mucociliary function of astodrimer sodium nasal spray (AS-NS) were evaluated and compared with other available nasal sprays‚Äîlow pH hydroxypropyl methylcellulose (HPMC-NS), iota-carrageenan (Carr-NS), nitric oxide (NO-NS), and povidone iodine (PI-NS). Assays simulated clinical conditions. Antiviral barrier function and cell viability were assessed in airway cell monolayers, while a model of fully differentiated human nasal epithelium (MucilAir‚Ñ¢) was utilized to evaluate tissue integrity, cytotoxicity, cilia beating frequency, and mucociliary clearance. AS-NS reduced infectious virus in cell monolayers and demonstrated a benign cytotoxicity profile. In human nasal epithelium ex vivo, AS-NS had no impact on mucociliary function (cilia beating nor mucociliary clearance). Carr-NS, HPMC-NS, NO-NS and PI-NS demonstrated limited antiviral effects, while HPMC-NS caused inhibition of mucociliary function. Astodrimer sodium nasal spray demonstrates an acceptable nonclinical efficacy and safety profile as a barrier nasal spray against respiratory viral infection in the nasal cavity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-72262-w" target="_blank">https://www.nature.com/articles/s41598-024-72262-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41467-024-52348-9" target="_blank">
            ü§ñ COVID-19 Vaccine Booster Dose Effects on Immune System</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-104', this)">
            More</button></h4>
            <div id="toggle-104" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Omicron COVID-19 immune correlates analysis of a third dose of mRNA-1273 in the COVE trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A study was done to see how well antibodies in the blood protect against COVID-19 after getting vaccinated. The research showed that people who have these antibodies are less likely to get sick with COVID-19. But since new variants of the virus have emerged, scientists wanted to know if these antibodies still work.

The study found that even though many people already had antibodies from previous vaccinations or infections, the level of protection they offered was still strong. In fact, having higher levels of antibodies meant a lower chance of getting COVID-19.

This research suggests that measuring antibody levels in the blood can still be a good way to see how well someone is protected against COVID-19, even after new variants have emerged.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In the phase 3 Coronavirus Efficacy (COVE) trial (NCT04470427), post-dose two Ancestral Spike-specific binding (bAb) and neutralizing (nAb) antibodies were shown to be correlates of risk (CoR) and of protection against Ancestral-lineage COVID-19 in SARS-CoV-2 naive participants. In the SARS-CoV-2 Omicron era, Omicron subvariants with varying degrees of immune escape now dominate, seropositivity rates are high, and booster doses are administered, raising questions on whether and how these developments affect the bAb and nAb correlates. To address these questions, we assess post-boost BA.1 Spike-specific bAbs and nAbs as CoRs and as correlates of booster efficacy in COVE. For naive individuals, bAbs and nAbs inversely correlate with Omicron COVID-19: hazard ratios (HR) per 10-fold marker increase (95% confidence interval) are 0.16 (0.03, 0.79) and 0.31 (0.10, 0.96), respectively. In non-naive individuals the analogous results are similar: 0.15 (0.04, 0.63) and 0.28 (0.07, 1.08). For naive individuals, three vs two-dose booster efficacy correlates with predicted nAb titer at exposure, with estimates -8% (-126%, 48%), 50% (25%, 67%), and 74% (49%, 87%), at 56, 251, and 891 Arbitrary Units/ml. These results support the continued use of antibody as a surrogate endpoint.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52348-9" target="_blank">https://www.nature.com/articles/s41467-024-52348-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-11</strong> <a href="https://www.nature.com/articles/s41598-024-71144-5" target="_blank">
            ü§ñ SARS Virus Protein Affects Calcium Signals in Cells</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-105', this)">
            More</button></h4>
            <div id="toggle-105" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 envelope protein alters calcium signaling via SERCA interactions</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 virus is still difficult to treat, especially when it becomes very severe. Scientists are trying to understand how the virus works so they can find new ways to fight it. One part of the virus, called the E protein, has been shown to be important for its growth and survival inside a host cell.

Researchers have discovered that this E protein can interact with other proteins in the cell that help regulate calcium levels. This interaction can affect how well these proteins work, which can lead to inflammation and make COVID-19 more severe.

The study found that the E protein can bind to these regulatory proteins and change their shape, making them less effective at doing their job. This could be a key part of why the virus causes such strong immune responses in people.

By understanding how this works, scientists may be able to find new ways to treat COVID-19 by targeting these interactions between the E protein and other cell proteins.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The clinical management of severe COVID-19 cases is not yet well resolved. Therefore, it is important to identify and characterize cell signaling pathways involved in virus pathogenesis that can be targeted therapeutically. Envelope (E) protein is a structural protein of the virus, which is known to be highly expressed in the infected host cell and is a key virulence factor; however, its role is poorly characterized. The E protein is a single-pass transmembrane protein that can assemble into a pentamer forming a viroporin, perturbing Ca2+homeostasis. Because it is structurally similar to regulins such as, for example, phospholamban, that regulate the sarco/endoplasmic reticulum calcium ATPases (SERCA), we investigated whether the SARS-CoV-2 E protein affects the SERCA system as an exoregulin. Using FRET experiments we demonstrate that E protein can form oligomers with regulins, and thus can alter the monomer/multimer regulin ratio and consequently influence their interactions with SERCAs. We also confirm that a direct interaction between E protein and SERCA2b results in a decrease in SERCA-mediated ER Ca2+reload. Structural modeling of the complexes indicates an overlapping interaction site for E protein and endogenous regulins. Our results reveal novel links in the host-virus interaction network that play an important role in viral pathogenesis and may provide a new therapeutic target for managing severe inflammatory responses induced by SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71144-5" target="_blank">https://www.nature.com/articles/s41598-024-71144-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41598-024-71475-3" target="_blank">
            ü§ñ Impact of Smell and Taste Issues on Daily Life After COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-106', this)">
            More</button></h4>
            <div id="toggle-106" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Chemosensory assessment and impact on quality of life in neurosensorial cluster of the post COVID 19 syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic showed that some people experience problems with their sense of smell and taste after getting infected. This study looked at how common this problem was, what might make it more likely, and how well people recovered.

About 47% of patients in this study had these problems, which were more common in women and less common in older adults who got a special treatment called monoclonal antibodies. When researchers tested some patients' sense of smell and taste, they found that about two-thirds could smell things only slightly or not at all, while one-fourth couldn't taste things normally.

This study showed that problems with smell and taste after COVID-19 can be quite complex and unpredictable. Understanding this better will help doctors develop treatments to prevent or fix these problems in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 pandemic brought chemosensory impairment to the forefront of medicine, revealing gaps in the knowledge of pathophysiological mechanisms, true prevalence and preventive/therapeutic alternatives. This is a sub-study of the ORCHESTRA cohort focusing on post-COVID-19 chemosensory symptoms. Risk factors for neurosensorial cluster of post-COVID-19 syndrome (NSc-PCS) were assessed through multivariable analysis. Psychophysical validated tests were applied on a sub-population of 50 patients. Qualitative chemosensory symptoms as well as nasal and oral chemesthesis were evaluated through anamnestic interview and the quality of life through the SF-36 questionnaire. Chemosensory symptoms evolution and olfactory training‚Äôs outcome were assessed through phone-call interviews. Out of 1187 patients (female, N‚Äâ=‚Äâ630), 550 (47%) presented NSc-PCS, with a lower risk for older age and monoclonal antibodies treatment, and a higher risk in females (p‚Äâ<‚Äâ0.001). Out of the 50 patients evaluated with psychophysical tests, 66% showed smell reduction with a qualitative alteration in 50% of hyposmic and 35% of normosmic patients. Hypogeusia was present in 14 (28%) of the patients assessed, with 56% showing a qualitative alteration; 53% of normogeusic patients presented qualitative disorders. NSc-PCS has a complex, fluctuating, multifaceted presentation. Quantifying and characterizing COVID-19-related chemosensory impairment is key to understand underlying mechanisms and to develop preventive and therapeutic treatment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71475-3" target="_blank">https://www.nature.com/articles/s41598-024-71475-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41541-024-00950-9" target="_blank">
            ü§ñ Designing Specialized Antigens to Boost Immunity Against Most COVID-19 Variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-107', this)">
            More</button></h4>
            <div id="toggle-107" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Computationally designed Spike antigens induce neutralising responses against the breadth of SARS-COV-2 variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists have created new vaccine ingredients called antigens to help fight against COVID-19 viruses that keep changing over time. They designed three new antigens (called T2_32, T2_35, and T2_36) that work well together to protect people from different types of COVID-19 variants.

Tests on animals showed that these new antigens are very effective at creating a strong defense against the virus. When given as a vaccine, they even worked better than some existing vaccines.

These findings suggest that using computer-designed modifications to create more powerful vaccines could be an important tool in fighting against COVID-19 and its evolving variants.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Updates of SARS-CoV-2 vaccines are required to generate immunity in the population against constantly evolving SARS-CoV-2 variants of concerns (VOCs). Here we describe three novel in-silico designed spike-based antigens capable of inducing neutralising antibodies across a spectrum of SARS-CoV-2 VOCs. Three sets of antigens utilising pre-Delta (T2_32), and post-Gamma sequence data (T2_35 and T2_36) were designed. T2_32 elicited superior neutralising responses against VOCs compared to the Wuhan-1 spike antigen in DNA prime-boost immunisation regime in guinea pigs. Heterologous boosting with the attenuated poxvirus - Modified vaccinia Ankara expressing T2_32 induced broader neutralising immune responses in all primed animals. T2_32, T2_35 and T2_36 elicited broader neutralising capacity compared to the Omicron BA.1 spike antigen administered by mRNA immunisation in mice. These findings demonstrate the utility of structure-informed computationally derived modifications of spike-based antigens for inducing broad immune responses covering more than 2 years of evolved SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00950-9" target="_blank">https://www.nature.com/articles/s41541-024-00950-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-09</strong> <a href="https://www.nature.com/articles/s41598-024-71419-x" target="_blank">
            ü§ñ Understanding Immune System Changes After Recovering from COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-108', this)">
            More</button></h4>
            <div id="toggle-108" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Immunologic mediators profile in COVID-19 convalescence</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

A new coronavirus called SARS-CoV-2 caused a global pandemic in 2020. To understand how our bodies fight this virus, scientists studied people who had recovered from COVID-19. They took blood samples from these individuals and measured different markers in their blood to see what happened as they got better.

The study found that most people developed antibodies (special proteins) against the virus after getting sick. Some of these antibodies lasted longer than others. The researchers also saw changes in the types of immune cells (like soldiers) in the blood over time.

What's interesting is that our immune system starts with a "fighting" response to the virus, but as we get better, it shifts towards a "repair" response. This study highlights the importance of understanding how our immune system works, especially after we've gotten sick with something like COVID-19. Knowing this can help us develop better treatments and vaccines in the future.

The bottom line is that our bodies are pretty amazing at fighting off viruses, but there's still a lot to learn about how they work!</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 caused the pandemic situation experienced since the beginning of 2020, and many countries faced the rapid spread and severe form of the disease. Mechanisms of interaction between the virus and the host were observed during acute phase, but few data are available when related to immunity dynamics in convalescents. We conducted a longitudinal study, with 51 healthy donors and 62 COVID-19 convalescent patients, which these had a 2-month follow-up after symptoms recovery. Venous blood sample was obtained from all participants to measure blood count, subpopulations of monocytes, lymphocytes, natural killer cells and dendritic cells. Serum was used to measure cytokines, chemokines, growth factors, anti-N IgG and anti-S IgG/IgM antibodies. Statistic was performed by Kruskal‚ÄìWallis test, and linear regression with days post symptoms and antibody titers. All analysis had confidence interval of 95%. Less than 35% of convalescents were anti-S IgM+, while more than 80% were IgG+ in D30. Anti-N IgG decreased along time, with loss of seroreactivity of 13%. Eosinophil count played a distinct role on both antibodies during all study, and the convalescence was orchestrated by higher neutrophil-to-lymphocyte ratio and IL-15, but initial stages were marked by increase in myeloid DCs, B1 lymphocytes, inflammatory and patrolling monocytes, G-CSF and IL-2. Later convalescence seemed to change to cytotoxicity mediated by T lymphocytes, plasmacytoid DCs, VEGF, IL-9 and CXCL10. Anti-S IgG antibodies showed the longest perseverance and may be a better option for diagnosis. The inflammatory pattern is yet present on initial stage of convalescence, but quickly shifts to a reparative dynamic. Meanwhile eosinophils seem to play a role on anti-N levels in convalescence, although may not be the major causative agent. We must highlight the importance of immunological markers on acute clinical outcomes, but their comprehension to potentialize adaptive system must be explored to improve immunizations and further preventive policies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71419-x" target="_blank">https://www.nature.com/articles/s41598-024-71419-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-08</strong> <a href="https://www.nature.com/articles/s41598-024-70980-9" target="_blank">
            ü§ñ New Method for Quickly Detecting COVID-19 Virus RNA</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-109', this)">
            More</button></h4>
            <div id="toggle-109" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Use of high-resolution fluorescence in situ hybridization for fast and robust detection of SARS-CoV-2 RNAs</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Detecting COVID-19 quickly and accurately is crucial to stopping its spread. We developed a fast test that can detect the virus in just 3 hours, using a special technique called FISH (fluorescence in situ hybridization). This test works by using tiny probes to find and highlight the presence of the virus's genetic material (RNA) in cells.

We tested this method on cell cultures and found it to be accurate and efficient. We also used it to detect COVID-19 in samples from patients, including those taken from the mouth and throat. The results were excellent, with the test being able to identify the presence of the virus quickly and accurately.

This new method could significantly improve the accuracy and speed of COVID-19 detection, making it easier to control outbreaks and keep people safe.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Early, rapid, and accurate diagnostic tests play critical roles not only in the identification/management of individuals infected by SARS-CoV-2, but also in fast and effective public health surveillance, containment, and response. Our aim has been to develop a fast and robust fluorescence in situ hybridization (FISH) detection method for detecting SARS-CoV-2 RNAs by using an HEK 293¬†T cell culture model. At various times after being transfected with SARS-CoV-2 E and N plasmids, HEK 293¬†T cells were fixed and then hybridized with ATTO-labeled short DNA probes (about 20 nt). At 4¬†h, 12¬†h, and 24¬†h after transfection, SARS-CoV-2 E and N mRNAs were clearly revealed as solid granular staining inside HEK 293¬†T cells at all time points. Hybridization time was also reduced to 1¬†h for faster detection, and the test was completed within 3¬†h with excellent results. In addition, we have successfully detected 3 mRNAs (E mRNA, N mRNA, and ORF1a (‚àí) RNA) simultaneously inside the buccal cells of COVID-19 patients. Our high-resolution RNA FISH might significantly increase the accuracy and efficiency of SARS-CoV-2 detection, while significantly reducing test time. The method can be conducted on smears containing cells (e.g., from nasopharyngeal, oropharyngeal, or buccal swabs) or smears without cells (e.g., from sputum, saliva, or drinking water/wastewater) for detecting various types of RNA viruses and even DNA viruses at different timepoints of infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70980-9" target="_blank">https://www.nature.com/articles/s41598-024-70980-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-06</strong> <a href="https://www.nature.com/articles/s43856-024-00602-y" target="_blank">
            ü§ñ COVID Vaccine Works Well in Greenland Population</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-110', this)">
            More</button></h4>
            <div id="toggle-110" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Robust immune response to COVID-19 vaccination in the island population of Greenland</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

In Greenland, the COVID-19 pandemic was relatively mild compared to other countries. One reason for this might be that many people in Greenland got vaccinated against COVID-19.

To understand how well the vaccines worked, researchers looked at the immune system response of 430 adults who had received two doses of the vaccine. They measured the levels of antibodies (which help fight off viruses) in these people's blood and saliva over a period of up to 11 months after their second dose.

The results showed that almost all participants (96% after two months, and 98% after 11 months) had enough antibodies to protect them from COVID-19. People who had previously been infected with COVID-19 even had higher levels of antibodies. In people over 60 years old, the vaccine response was slightly weaker.

The researchers also found that all participants had a strong T-cell response (another part of the immune system) against COVID-19. These results suggest that the vaccines were very effective in protecting Greenlanders against COVID-19, and contributed to the mild impact of the pandemic in the country.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundIn Greenland, the COVID-19 pandemic was characterised by a late onset of community transmission and a low impact on the healthcare system, hypothesised as being partly due to a high uptake of vaccinations. To underpin this description, we aimed to assess the SARS-CoV-2 immune response post-vaccination in a Greenlandic population.MethodsIn this observational cohort study, we included 430 adults in Greenland who had received a complete two-dose SARS-CoV-2 vaccination at enrolment. The total plasma SARS-CoV-2 spike glycoprotein Ig antibodies (S-Ab) induced by either the BNT162b2 or mRNA-1273 vaccine, was measured up to 11 months after the second vaccine dose. In addition, total salivary S-Abs were examined in 107 participants, and the T-cell response to the spike glycoprotein was assessed in 78 participants out of the entire study cohort.ResultsHere we demonstrate that two months after the second vaccine dose, 96% of participants have protective plasma S-Ab levels. By 11 months, 98% have protective levels, with prior SARS-CoV-2 infection particularly enhancing S-Ab levels by 37% (95% CI 25‚Äì51%). Among individuals aged 60 years and older, we observe a 21% (95% CI 7‚Äì33%) reduction in antibody response. Total salivary S-Ab levels are detectable in all participants and significantly correlate with plasma levels. Moreover, all participants exhibit a robust SARS-CoV-2-specific T-cell response 11 months post-primary vaccination.ConclusionsOur findings show that Greenlanders exhibit a robust and lasting immune response, both humoral and cellular, comparable to other population groups up to at least 11 months after the second vaccine dose. These results corroborate the hypothesis that vaccines contributed to the mild impact of the COVID-19 pandemic in the Greenlandic population.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00602-y" target="_blank">https://www.nature.com/articles/s43856-024-00602-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41598-024-71305-6" target="_blank">
            ü§ñ Gauss Luciferase Assay to Measure Coronavirus Enzyme Activity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-111', this)">
            More</button></h4>
            <div id="toggle-111" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A Gaussia luciferase reporter assay for the evaluation of coronavirus Nsp5/3CLpro activity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Human coronaviruses cause sickness in many people each year. Some types like MERS, SARS-CoV-1, and SARS-CoV-2 can be very serious.

To help stop these viruses from spreading, scientists are looking for new medicines that can fight them. A drug called Paxlovid has already been used successfully to treat some of these illnesses by stopping a protein inside the virus called 3CLpro.

But sometimes, if we use the same drug too much or for too long, the virus might start to get stronger and not work as well anymore. So researchers want to find more ways to fight these viruses.

They created an easy-to-use test that can quickly tell them how strong a virus is and if medicines are working against it. This test uses parts of another protein found in our bodies (ACE2-Gal4) along with the 3CLpro from coronaviruses. When 3CLpro cuts this protein, it causes a light-producing chemical to be made.

This new test can work on any coronavirus and can measure both how well medicine is stopping the virus and if the virus has become more powerful in some way. They tested the medicine Paxlovid using their test and found that it works against SARS-CoV-2 3CLpro.

The test also helped them understand better how mutations (changes) in the 3CLpro protein can make it stronger or weaker, which is important for future research.

Overall, this new test makes it easier to study coronavirus proteins and find new medicines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Human coronaviruses (hCoVs) infect millions of people every year. Among these, MERS, SARS-CoV-1, and SARS-CoV-2 caused significant morbidity and mortality and their emergence highlights the risk of possible future coronavirus outbreaks. Therefore, broadly-active anti-coronavirus drugs are needed. Pharmacological inhibition of the hCoV protease Nsp5 (3CLpro) is clinically beneficial as shown by the wide and effective use of Paxlovid (nirmatrelvir, ritonavir). However, further treatment options are required due to the risk of drug resistance. To facilitate the assessment of coronavirus protease function and its pharmacological inhibition, we developed an assay allowing rapid and reliable quantification of Nsp5 activity under biosafety level 1 conditions. It is based on an ACE2-Gal4 transcription factor fusion protein separated by a Nsp5 recognition site. Cleavage by Nsp5 releases the Gal4 transcription factor, which then induces the expression of Gaussia luciferase. Our assay is compatible with Nsp5 proteases from all hCoVs and allows simultaneous measurement of inhibitory and cytotoxic effects of the tested compounds. Proof-of-concept measurements confirmed that nirmatrelvir, GC376 and lopinavir inhibit SARS-CoV-2 Nsp5 function. Furthermore, the assay accurately predicted the impact of Nsp5 mutations on catalytic activity and inhibitor sensitivity. Overall, the reporter assay is suitable for evaluating viral protease activity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71305-6" target="_blank">https://www.nature.com/articles/s41598-024-71305-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41467-024-52038-6" target="_blank">
            ü§ñ Effect of mRNA COVID Vaccine Timing on Heart Inflammation Risk</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-112', this)">
            More</button></h4>
            <div id="toggle-112" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Influence of mRNA Covid-19 vaccine dosing interval on the risk of myocarditis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

A serious heart condition called myocarditis is more likely to happen after getting certain COVID-19 vaccines. The risk is highest after the second dose and lower with later booster doses. We studied over 7,000 cases of myocarditis in people aged 12 and up to see if there's a connection between how often you get vaccinated and the risk of this condition. Our results show that taking longer breaks between vaccine doses can decrease the risk by up to four times, especially for people under 50. This suggests that waiting at least six months before getting another booster shot might be a good idea.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Myocarditis is the most salient serious adverse event following messenger RNA-based Covid-19 vaccines. The highest risk is observed after the second dose compared to the first, whereas the level of risk associated with more distant booster doses seems to lie in between. We aimed to assess the relation between dosing interval and the risk of myocarditis, for both the two-dose primary series and the third dose (first booster). This matched case-control study included 7911 cases of myocarditis aged 12 or more in a period where approximately 130 million vaccine doses were administered. Here we show that longer intervals between each consecutive dose, including booster, may decrease the occurrence of vaccine-associated myocarditis by up to a factor of 4, especially under age 50. These results suggest that a minimum 6-month interval might be required when scheduling additional booster vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-52038-6" target="_blank">https://www.nature.com/articles/s41467-024-52038-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-05</strong> <a href="https://www.nature.com/articles/s41598-024-70767-y" target="_blank">
            ü§ñ How COVID-19 Affects Eye Diseases in South Korea</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-113', this)">
            More</button></h4>
            <div id="toggle-113" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impact of COVID-19 pandemic on ocular disease: KNHANES 2015‚Äì2021</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

This study looked at how the COVID-19 pandemic affected eye diseases and changed people's risk factors in South Korea between 2015 and 2021. The researchers analyzed data from over 50,000 people and found that being infected with COVID-19 increased the risk of developing cataracts and age-related macular degeneration (a type of vision loss). They also found that other health conditions like high blood pressure, diabetes, and depression were linked to eye diseases too. However, having COVID-19 did not increase the risk of getting glaucoma (another type of eye disease). The study suggests that cataracts and age-related macular degeneration might be more affected by people's lifestyle and environment rather than just their genetic predisposition.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aim of this study was to evaluate the impact of COVID-19 on ocular diseases and changes in risk factors before and after the COVID-19 pandemic. This study was conducted using data from the Korea National Health and Nutrition Examination Survey (KNHANES) 2015‚Äì2021, a national cross-sectional health examination and survey. Associations between ocular diseases and risk factors were determined using the chi-squared test and logistic regression analysis. Bivariable adjusted logistic regression analysis was performed to examine the odds ratio (OR) and 95% confidence interval (CI) to evaluate of the impact of COVID-19 on ocular diseases. Individuals were divided into two age groups (<‚Äâ60 and‚Äâ‚â•‚Äâ60¬†years). A total of 50,158 people were diagnosed, of whom 7270 were diagnosed with cataract, 921 with glaucoma, and 439 with age-related macular degeneration (AMD). Risk factors for cataract were COVID-19 pandemic (OR 1.161), hypertension (OR 1.608), diabetes (OR 1.573), dyslipidemia (OR 1.167), stroke (OR 1.272), and depression (OR 1.567). Risk factors for AMD were COVID-19 pandemic (OR 1.600), dyslipidemia (OR 1.610), and depression (OR 1.466). Risk factors for glaucoma were hypertension (OR 1.234), dyslipidemia (OR 1.529), diabetes (OR 1.323), and depression (OR 1.830). The COVID-19 pandemic was a risk factor for cataracts and AMD, but not for glaucoma. Cataracts and AMD may be more influenced by the acquired health conditions or the environment.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70767-y" target="_blank">https://www.nature.com/articles/s41598-024-70767-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-04</strong> <a href="https://www.nature.com/articles/s41467-024-51973-8" target="_blank">
            ü§ñ Understanding How SARS-CoV-2 Mutates and Evades Antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-114', this)">
            More</button></h4>
            <div id="toggle-114" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Structural basis for the evolution and antibody evasion of SARS-CoV-2 BA.2.86 and JN.1 subvariants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic is still a problem and some new versions of the virus (called BA.2.86 and JN.1) are spreading fast around the world. These viruses have many mutations that make it harder for our immune systems to fight them off. Scientists studied these new viruses and found out how they evade our antibodies, which are like superheroes that protect us from germs.

They also looked at the structure of the virus and how it interacts with human cells. They discovered some important changes in the way these new viruses attach to human cells, making it harder for them to be stopped by our immune systems.

This research helps us understand why these new viruses are spreading quickly and what we can do to stop them. It's like having a map of where the virus is evolving and how it's adapting to avoid being caught by our bodies' defenses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The Omicron subvariants of SARS-CoV-2, especially for BA.2.86 and JN.1, have rapidly spread across multiple countries, posing a significant threat in the ongoing COVID-19 pandemic. Distinguished by 34 additional mutations on the Spike (S) protein compared to its BA.2 predecessor, the implications of BA.2.86 and its evolved descendant, JN.1 with additional L455S mutation in receptor-binding domains (RBDs), are of paramount concern. In this work, we systematically examine the neutralization susceptibilities of SARS-CoV-2 Omicron subvariants and reveal the enhanced antibody evasion of BA.2.86 and JN.1. We also determine the cryo-EM structures of the trimeric S proteins from BA.2.86 and JN.1 in complex with the host receptor ACE2, respectively. The mutations within the RBDs of BA.2.86 and JN.1 induce a remodeling of the interaction network between the RBD and ACE2. The L455S mutation of JN.1 further induces a notable shift of the RBD‚ÄìACE2 interface, suggesting the notably reduced binding affinity of JN.1 than BA.2.86. An analysis of the broadly neutralizing antibodies possessing core neutralizing epitopes reveals the antibody evasion mechanism underlying the evolution of Omicron BA.2.86 subvariant. In general, we construct a landscape of evolution in virus-receptor of the circulating Omicron subvariants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51973-8" target="_blank">https://www.nature.com/articles/s41467-024-51973-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-04</strong> <a href="https://www.nature.com/articles/s41541-024-00957-2" target="_blank">
            ü§ñ Effective Protection Against COVID-19 Variants with Dual Vaccine Shot</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-115', this)">
            More</button></h4>
            <div id="toggle-115" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Broad protection and respiratory immunity of dual mRNA vaccination against SARS-CoV-2 variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists have been working on vaccines to protect us from COVID-19. However, the virus has changed over time, making some vaccines less effective. Researchers developed a new type of vaccine that targets a specific part of the virus and tested it alone or combined with another vaccine in hamsters. The results showed that the new vaccine, when used alone or together with the other vaccine, was very good at protecting against certain types of COVID-19. Additionally, studies in mice found that this combination vaccine triggered an immune response specifically in the lungs, which is where COVID-19 typically infects us. These findings suggest that this new vaccine could be effective against various strains of COVID-19 and help prevent future outbreaks.</p><br>
                <p><strong>Abstract</strong></p>
                <p>While first-generation, spike (S)-based COVID-19 vaccines were effective against early SARS-CoV-2 strains, the rapid evolution of novel Omicron subvariants have substantially reduced vaccine efficacy. As such, broadly protective vaccines against SARS-CoV-2 are needed to prevent future viral emergence. In addition, it remains less clear whether peripheral immunization, especially with mRNA vaccines, elicits effective respiratory immunity. Our group has developed a nucleoside-modified mRNA vaccine expressing the nucleocapsid (N) protein of the ancestral SARS-CoV-2 virus and has tested its use in combination with the S-based mRNA vaccine (mRNA-S). In this study, we examined efficacy of mRNA-N alone or in combination with mRNA-S (mRNA-S+N) against more immune evasive Omicron variants in hamsters. Our data show that mRNA-N alone induces a modest but significant protection against BA.5 and that dual mRNA-S+N vaccination confers complete protection against both BA.5 and BQ.1, preventing detection of virus in the hamster lungs. Analysis of respiratory immune response in mice shows that intramuscular mRNA-S+N immunization effectively induces respiratory S- and N-specific T cell responses in the lungs and in bronchoalveolar lavage (BAL), as well as antigen-specific binding IgG in BAL. Together, our data further support mRNA-S+N as a potential pan-COVID-19 vaccine for broad protection against current and emerging SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00957-2" target="_blank">https://www.nature.com/articles/s41541-024-00957-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-03</strong> <a href="https://www.nature.com/articles/s41590-024-01951-5" target="_blank">
            ü§ñ Adenovirus Vaccine Boosts Immunity, Prevents New Coronavirus Strain in Monkeys</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-116', this)">
            More</button></h4>
            <div id="toggle-116" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mucosal adenovirus vaccine boosting elicits IgA and durably prevents XBB.1.16 infection in nonhuman primates</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at different ways to give a vaccine for a virus that causes the flu in animals (SARS-CoV-2). They compared two methods: putting the vaccine directly into someone's arm, or giving it through their nose or mouth.

The team found that when they gave the vaccine through the nose and mouth, it worked better. The vaccine helped protect against the virus not just in the throat but also deep inside the lungs. This method also made the immune system create antibodies (special proteins) that can help stop the virus from spreading.

In contrast, giving the vaccine by putting it into someone's arm only protected the lower parts of their airways.

The study suggests that giving a vaccine through the nose or mouth might be better because:
1. It stops the virus in both the throat and lungs.
2. It makes antibodies that can stop other people from catching the virus too.
3. It makes certain immune cells that help fight off the virus.

So, this research shows how giving a vaccine through the nose or mouth could better protect against SARS-CoV-2 and prevent it from spreading to others.</p><br>
                <p><strong>Abstract</strong></p>
                <p>A mucosal route of vaccination could prevent severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication at the site of infection and limit transmission. We compared protection against heterologous XBB.1.16 challenge in nonhuman primates (NHPs) ~5 months following intramuscular boosting with bivalent mRNA encoding WA1 and BA.5 spike proteins or mucosal boosting with a WA1‚ÄìBA.5 bivalent chimpanzee adenoviral-vectored vaccine delivered by intranasal or aerosol device. NHPs boosted by either mucosal route had minimal virus replication in the nose and lungs, respectively. By contrast, protection by intramuscular mRNA was limited to the lower airways. The mucosally delivered vaccine elicited durable airway IgG and IgA responses and, unlike the intramuscular mRNA vaccine, induced spike-specific B cells in the lungs. IgG, IgA and T cell responses correlated with protection in the lungs, whereas mucosal IgA alone correlated with upper airway protection. This study highlights differential mucosal and serum correlates of protection and how mucosal vaccines can durably prevent infection against SARS-CoV-2.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41590-024-01951-5" target="_blank">https://www.nature.com/articles/s41590-024-01951-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s44298-024-00054-0" target="_blank">
            ü§ñ Camel Nasal Cells Resist COVID Better Without Key Protein</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-117', this)">
            More</button></h4>
            <div id="toggle-117" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Resistance to SARS-CoV-2 infection in camelid nasal organoids is associated with lack of ACE2 expression</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) can infect different animals. How well an animal gets infected by SARS-CoV-2 depends on a special protein called ACE2 that the virus uses to enter cells.

While SARS-CoV-2 can use many versions of the ACE2 protein found in various camel species, it doesn't make these camels sick when they are exposed to the virus. In this study, researchers used tiny living copies of parts of animals' noses (called organoids) to see if camels could be infected by SARS-CoV-2 or another related virus called MERS-CoV.

The scientists found that while human nose copies can get infected with both viruses, camel nose copies don't. They also noticed something else: The camel nose copies had a different protein called DPP4 instead of ACE2. This led to the conclusion that because camels don't have enough of the ACE2 protein in their noses and upper respiratory tracts, they are resistant to SARS-CoV-2.

So basically, it's like the virus can use other proteins found in camels' noses, but these aren't as good at letting the virus inside cells. Because of this, camels don't get sick from the virus even if they're exposed to it.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infects a variety of animal species. Susceptibility to SARS-CoV-2 is primarily determined by the utilization of the viral receptor, ACE2. SARS-CoV-2 can utilize a broad range of animal ACE2 isoforms in vitro, including the ACE2 from various camelid species. However, experimental infection of these animals does not lead to productive infection or seroconversion. In this study, we investigate the susceptibility of camelids to SARS-CoV-2 using novel well-differentiated camelid nasal organoids. We show that camelid nasal organoids are highly susceptible to Middle East respiratory syndrome coronavirus (MERS-CoV) infection, but not to infection with different SARS-CoV-2 variants (614G, BA.1 or EG.5.1.1). All viruses efficiently infected human airway organoids. Immunohistochemistry analysis revealed the absence of ACE2 on camelid nasal organoids and dromedary camel upper respiratory tract. In contrast, DPP4 was expressed in both camelid nasal organoids and the camel upper respiratory tract, which correlates with MERS-CoV infection. This study indicates that the camelid upper respiratory tract lacks expression of ACE2, which is associated with resistance to SARS-CoV-2 infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00054-0" target="_blank">https://www.nature.com/articles/s44298-024-00054-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-71455-7" target="_blank">
            ü§ñ Looking into why some COVID-19 patients get a severe lung infection called invasive pulmonary aspergillosis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-118', this)">
            More</button></h4>
            <div id="toggle-118" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Investigation of risk factors for invasive pulmonary aspergillosis among patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied 114 people who had COVID-19 and found that some developed a secondary infection called pulmonary aspergillosis (CAPA). This study looked for factors that might increase the risk of developing CAPA.

The results showed that certain conditions, such as being on kidney transplant medication, having high levels of inflammation in the body, or needing to be on a ventilator for a long time, made it more likely for someone with COVID-19 to develop CAPA.

The researchers found that three main factors were strongly linked to an increased risk of CAPA: high levels of inflammation in the body, low levels of immune cells called T-cells, and needing to be on a ventilator for a long time. Identifying these risk factors can help doctors start treatment earlier, which may improve outcomes for people with CAPA.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 associated pulmonary aspergillosis (CAPA) had been reported, and raised concern about this secondary infection due to the high mortality. This study aimed to investigate the risk factors for CAPA. The enrolled 114 COVID-19 patients were further divided into CAPA group and non-CAPA group. Demographic characteristics, underlying diseases, laboratory parameters and therapeutic schedule between the two groups were compared to identify the independent risk factors for CAPA by univariate analysis and multivariable logistic regression analysis. Sensitivity and specificity of independent risk factors were confirmed by receiver operating characteristic (ROC) curve analysis. Univariate analysis showed that renal transplant, IL-6 and CRP levels, decreased CD4‚Äâ+‚ÄâT cell and CD8‚Äâ+‚ÄâT cell, duration of antibiotics therapy, and prolonged mechanical ventilation were risk factors for development of CAPA. These factors were further analyzed by multivariable logistic regression analysis and the results indicated that elevated IL-6 level, decreased CD4‚Äâ+‚ÄâT cell and prolonged mechanical ventilation could be recognized as independent risk factors for CAPA in COVID-19 patients. Identification of these risk factors is essential to initiate antifungal therapy as soon as possible to improve outcome of patients with CAPA.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71455-7" target="_blank">https://www.nature.com/articles/s41598-024-71455-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-70725-8" target="_blank">
            ü§ñ T-cells in pregnant/postpartum women, with/without HIV, recognize old/new COVID strains</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-119', this)">
            More</button></h4>
            <div id="toggle-119" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>T-cell responses to ancestral SARS-CoV-2 and Omicron variant among unvaccinated pregnant and postpartum women living with and without HIV in South Africa</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied how people who are pregnant or have recently given birth and living with HIV react to COVID-19 in their bodies. They looked at 24 women without HIV and 15 women with HIV from low-income countries. The researchers found that these women's immune systems can fight off COVID-19 just as well as women who don't have HIV, even if they've never been vaccinated against the virus. They also discovered that the immune system's response to a specific part of the virus called Omicron is similar to its response to earlier versions of the virus. This means that people with COVID-19 can still get protection from their immune system, even if they haven't received a vaccine.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 cell-mediated immunity remains understudied during pregnancy in unvaccinated Black African women living with HIV (WLWH) from low- and middle-income countries. We investigated SARS-CoV-2-specific T-cell responses 1 month following infection in 24 HIV-uninfected women and 15 WLWH at any stage during pregnancy or postpartum. The full-length spike (FLS) glycoprotein and nucleocapsid (N) protein of wild-type (WT) SARS-CoV-2, as well as mutated spike protein regions found in the Omicron variant (B.1.1.529) were targeted by flow cytometry. WT-specific CD4+and CD8+T cells elicited similar FLS- and N-specific responses in HIV-uninfected women and WLWH. SARS-CoV-2-specific T-lymphocytes were predominantly TNF-Œ± monofunctional in pregnant and postpartum women living with and without HIV, with fever cells producing either IFN-Œ≥ or IL-2. Furthermore, T-cell responses were unaffected by Omicron-specific spike mutations as similar responses between Omicron and the ancestral virus were detected for CD4+and CD8+T cells. Our results collectively demonstrate comparable T-cell responses between WLWH on antiretroviral therapy and HIV-uninfected pregnant and postpartum women who were na√Øve to Covid-19 vaccination. Additionally, we show that T cells from women infected with the ancestral virus, Beta variant (B.1.351), or Delta variant (B.1.617.2) can cross-recognize Omicron, suggesting an overall preservation of T-cell immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70725-8" target="_blank">https://www.nature.com/articles/s41598-024-70725-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-09-02</strong> <a href="https://www.nature.com/articles/s41598-024-71325-2" target="_blank">
            ü§ñ Acute Proteins for COVID Severity & Mortality</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-120', this)">
            More</button></h4>
            <div id="toggle-120" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Acute-phase proteins as indicators of disease severity and mortality in COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how certain chemicals called acute-phase proteins (APPs) affect how sick people get from COVID-19. They measured these chemicals in 96 people with COVID-19, along with some healthy people for comparison.

They found that:
- The levels of APPs change depending on how sick someone is from the disease and if they're getting enough oxygen.
- In very severe cases, where there's a lot of inflammation (called a cytokine storm), these chemicals can go up or down a lot.
- There are some APPs that don't usually get tested. The researchers found that two of them - procalcitonin and transferrin - help doctors tell if someone is going to survive their COVID-19.

They also noticed that measuring how sick people were using something called the MEWS scale didn't always match with another system used by doctors (National Institutes of Health). 

The researchers concluded that:
- Not all APPs change based on how bad a person's disease gets.
- Only two out of many tested APPs can help predict if someone will survive their COVID-19.
- Among the new ones they looked at, transferrin is the only one that seems to really matter in predicting who will get better or worse from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The aim of the study was to conduct of relationship of acute-phase proteins (APPs) with the severity of COVID-19 defined by National Institutes of Health and according to the criteria of MEWS scale, with the presence of a cytokine storm, oxygen therapy and patient survival. We enrolled 96 patients with COVID-19 and 30 healthy people. The samples were taken on the day of admission and after 9¬†days on average. Not only commonly used APPs such as CRP, procalcitonin and ferritin and also rarely assayed proteins such as transferrin, haptoglobin, Œ±1-acid glycoprotein and Œ±1-antitrypsin, were tested in the study. The levels of APPs depends on the severity of COVID-19 disease, on the presence of cytokine storm and used oxygen therapy. The greatest APPs changes occurred in the most advanced form of the disease, with the presence of a cytokine storm and the most intense oxygen therapy. The results obtained from MEWS scale were not consistent with National Institutes of Health scores. Studies in the second samples showed the quenching of the acute phase reactions and the effectiveness of oxygen therapy. Only two of the examined APPs i.e. procalcitonin and transferrin, differed between surviving and non-surviving patients, and these two predispose to the role of prognostic factors in Covid-19. In conclusion, the concentration of not all acute-phase proteins depends on the severity of COVID-19 disease, presence of cytokine storm, the used of oxygen therapy and only some of them (procalcitonin and transferrin) are related to the survival outcomes. Of the newly tested acute-phase proteins, only transferrin shows significance as a marker of disease severity and mortality in COVID-19 disease.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71325-2" target="_blank">https://www.nature.com/articles/s41598-024-71325-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-31</strong> <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">
            ü§ñ Spike Protein Targeting Antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-121', this)">
            More</button></h4>
            <div id="toggle-121" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibodies utilizing VL6-57 light chains target a convergent cryptic epitope on SARS-CoV-2 spike protein and potentially drive the genesis of Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists have found that as COVID-19 spreads, it mutates and changes into different versions. This means that the antibodies created by our immune system when we get infected or vaccinated may not be able to recognize the virus if it changes too much. Researchers wanted to understand why this happens.

They discovered a group of antibodies that can fight COVID-19, and found out how they work together with other parts of the immune system to target the virus. They also found out that these antibodies are already present in people who have never had COVID-19, but are more abundant in people who have recovered from the disease.

However, when scientists tested these antibodies against a new variant of the virus called Omicron, they found out that it can evade their attack by making small changes to its structure. This suggests that the pressure from our immune system may be driving the evolution of the virus into more aggressive forms.

These findings provide further evidence that the spread of COVID-19 and its variants is influenced by our immune response, and highlight the importance of continued research into understanding how our bodies fight the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Continued evolution of SARS-CoV-2 generates variants to challenge antibody immunity established by infection and vaccination. A connection between population immunity and genesis of virus variants has long been suggested but its molecular basis remains poorly understood. Here, we identify a class of SARS-CoV-2 neutralizing public antibodies defined by their shared usage of VL6-57 light chains. Although heavy chains of diverse genotypes are utilized, convergent HCDR3 rearrangements have been observed among these public antibodies to cooperate with germline VL6-57 LCDRs to target a convergent epitope defined by RBD residues S371-S373-S375. Antibody repertoire analysis identifies that this class of VL6-57 antibodies is present in SARS-CoV-2-naive individuals and is clonally expanded in most COVID-19 patients. We confirm that Omicron-specific substitutions at S371, S373 and S375 mediate escape of antibodies of the VL6-57 class. These findings support that this class of public antibodies constitutes a potential immune pressure promoting the introduction of S371L/F-S373P-S375F in Omicron variants. The results provide further molecular evidence to support that antigenic evolution of SARS-CoV-2 is driven by antibody mediated population immunity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51770-3" target="_blank">https://www.nature.com/articles/s41467-024-51770-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41390-024-03492-x" target="_blank">
            ü§ñ Long-term Effects of COVID-19 on Children in Australia After a Big Outbreak</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-122', this)">
            More</button></h4>
            <div id="toggle-122" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post COVID-19 conditions in an Australian pediatric cohort, 3 months following a Delta outbreak</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Many kids got sick with COVID-19 during an outbreak in Australia, but not much was known about how it affected them later on. We asked parents to fill out an online survey 12 weeks after their child had recovered from COVID-19 and found that only a small number (about 1 in 10) were still experiencing symptoms or problems with daily activities.

Out of those who reported ongoing issues, some kids had recovered by the time they saw a doctor, while others had existing conditions that got worse. A smaller group had something called Post-COVID Condition, which is similar to Long COVID.

Our study found that most kids who got COVID-19 recovered quickly, but for those who didn't, it can be a significant burden. We also discovered some risk factors that make kids more likely to experience ongoing problems after COVID-19, such as being older than 11 and having pre-existing medical conditions.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundPediatric long COVID remains incompletely understood with scant Australian data available. We aimed to assess the impacts of the 2021 Delta variant of SARS-CoV-2 outbreak on symptoms and functioning 12 weeks post-acute infection in a cohort of children and adolescents.MethodsThe parents/carers of 11,864 patients with PCR-confirmed SARS-CoV-2 were invited, via email or text message, to complete an online survey assessing symptoms and functional impairment.Findings1731 (17.6%) responded to the survey. 203 (11.7%) reported continued symptoms and/or functional impairment which were flagged for clinical review, all others reported recovery. Of the 169 subsequently clinically reviewed, 63 had already recovered (37.3%) and 17 had exacerbation of pre-existing condition(s) (10.1%); 63 (37.3%) were diagnosed with a Post COVID Condition (PCC). Of these, 21 (12.4%) were considered to have features compatible with the United Kingdom consensus cases definition for Long COVID.InterpretationDuring an outbreak of SARS-CoV-2 an online questionnaire with subsequent clinical review revealed self-reported non-recovery at 12 weeks in a minority of cases, with a spectrum of features. Long COVID comprised only a subset of cases with self-reported non-recovery, and is infrequent in children and adolescents, but still comprises a likely significant burden that warrants attention.ImpactOur study provides the only comprehensive estimate of the frequency and spectrum of post-COVID conditions in children from Australia. The high frequency of self-reported recovery, and low frequency of Long COVID compatible illness adds to the literature from other settings. Risk factors for post-COVID conditions in children are identified and include: age >11 year, and previous medical co-morbidity.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41390-024-03492-x" target="_blank">https://www.nature.com/articles/s41390-024-03492-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41598-024-70697-9" target="_blank">
            ü§ñ Real-Time Monitoring of COVID-19 Genetic Changes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-123', this)">
            More</button></h4>
            <div id="toggle-123" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A program for real-time surveillance of SARS-CoV-2 genetics</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 pandemic required scientists to quickly identify new variants of the virus. This meant setting up a nationwide system to collect, analyze, and report data on patient samples in a very short time frame. We developed an infrastructure that allowed us to sequence over 594,000 SARS-CoV-2 genomes from patients in the US between March 2020 and July 2023. Our protocol (called "Virseq") ensures accurate and reliable sequencing of the entire virus genome, even for multiple strains co-infecting a patient. This system supports the national surveillance program led by the CDC and serves as a model for responding to future pandemics quickly and effectively at a national scale.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The COVID-19 pandemic brought forth an urgent need for widespread genomic surveillance for rapid detection and monitoring of emerging SARS-CoV-2 variants. It necessitated design, development, and deployment of a nationwide infrastructure designed for sequestration, consolidation, and characterization of patient samples that disseminates de-identified information to public authorities in tight turnaround times. Here, we describe our development of such an infrastructure, which sequenced 594,832 high coverage SARS-CoV-2 genomes from isolates we collected in the United States (U.S.) from March 13th 2020 to July 3rd 2023. Our sequencing protocol (‚ÄòVirseq‚Äô) utilizes wet and dry lab procedures to generate mutation-resistant sequencing of the entire SARS-CoV-2 genome, capturing all major lineages. We also characterize 379 clinically relevant SARS-CoV-2 multi-strain co-infections and ensure robust detection of emerging lineages via simulation. The modular infrastructure, sequencing, and analysis capabilities we describe support the U.S. Centers for Disease Control and Prevention national surveillance program and serve as a model for rapid response to emerging pandemics at a national scale.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70697-9" target="_blank">https://www.nature.com/articles/s41598-024-70697-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41598-024-70662-6" target="_blank">
            ü§ñ Factors Contributing to Death in Pakistani Patients Hospitalized with COVID-19</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-124', this)">
            More</button></h4>
            <div id="toggle-124" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Risk factors for mortality in hospitalized COVID-19 patients across five waves in Pakistan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Researchers studied 5368 people who were hospitalized with COVID-19 in Pakistan over two years. They looked at how the virus affected different age groups and the outcomes for patients.

They found that:

* Older adults (aged 60+) made up a larger proportion of patients during certain waves of the pandemic.
* Men were more likely to be infected earlier in the pandemic, but this changed later on.
* The mortality rate (death rate) varied across different waves, with some being higher than others.
* Certain conditions, such as acute respiratory distress syndrome and acute kidney injury, increased the risk of death.
* Some patients who received a type of blood thinner called enoxaparin had lower odds of dying from COVID-19.

The study suggests that older adults and those with certain health complications are at higher risk of dying from COVID-19 during certain waves of the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This retrospective cohort study aims to describe the clinical characteristics and outcomes and assess risk factors for mortality across the epidemic waves in hospitalized COVID-19 patients in a major tertiary-care center in Pakistan. A total of 5368 patients with COVID-19, hospitalized between March 2020 and April 2022 were included. The median age was 58¬†years (IQR: 44‚Äì69), 41% were females, and the overall mortality was 12%. Comparative analysis of COVID-19 waves showed that the proportion of patients aged‚Äâ‚â•‚Äâ60¬†years was highest during the post-wave 4 period (61.4%) and Wave 4 (Delta) (50%) (p<‚Äâ0.001). Male predominance decreased from 65.2% in Wave 2 to 44.2% in Wave 5 (Omicron) (p<‚Äâ0.001). Mortality rate was lowest at 9.4% in wave 5 and highest at 21.6% in the post-wave 4 period (p=‚Äâ0.041). In multivariable analysis for risk factors of mortality, acute respiratory distress syndrome (ARDS) was most strongly associated with mortality (aOR 22.98, 95% CI 15.28‚Äì34.55,p<‚Äâ0.001), followed by need for mechanical ventilation (aOR 6.81, 95% CI 5.13‚Äì9.05,p<‚Äâ0.001). Other significant risk factors included acute kidney injury (aOR 3.05, 95% CI 2.38‚Äì3.91,p<‚Äâ0.001), stroke (aOR 2.40, 95% CI 1.26‚Äì4.60, p‚Äâ=‚Äâ0.008), pulmonary embolism (OR 2.07, 95% CI 1.28‚Äì3.35,p=‚Äâ0.003), and age‚Äâ‚â•‚Äâ60¬†years (aOR 2.45, 95% CI 1.95‚Äì3.09,p<‚Äâ0.001). Enoxaparin use was associated with lower mortality odds (aOR 0.45, 95% CI 0.35‚Äì0.60,p<‚Äâ0.001. Patients hospitalized during Wave 4 (aOR 2.22, 95% CI 1.39‚Äì3.56,p<‚Äâ0.001) and the post-wave 4 period (aOR 2.82, 95% CI 1.37‚Äì5.80,p=‚Äâ0.005) had higher mortality odds compared to other waves. The study identifies higher mortality risk in patients admitted in Delta wave and post-wave, aged ‚â•‚Äâ60¬†years, and with respiratory and renal complications, and lower risk with anticoagulation during COVID-19 waves.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70662-6" target="_blank">https://www.nature.com/articles/s41598-024-70662-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-30</strong> <a href="https://www.nature.com/articles/s41467-024-51959-6" target="_blank">
            ü§ñ Understanding COVID-19 Immune Response</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-125', this)">
            More</button></h4>
            <div id="toggle-125" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Mapping the immunopeptidome of seven SARS-CoV-2 antigens across common HLA haplotypes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Most COVID-19 vaccines currently work by teaching our bodies to fight the virus's "Spike" protein. However, new variants of the virus can make this Spike protein change shape and become harder for our immune systems to recognize. To address this issue, researchers are working on developing vaccines that target a broader range of proteins in the SARS-CoV-2 virus.

To do this, scientists used a powerful tool called mass spectrometry to identify tiny pieces of these proteins (called "immunopeptides") that can trigger an immune response. They tested these immunopeptides against different versions of human cells and found 248 unique peptides that were bound to specific parts of the cell's surface.

These findings could help create more effective COVID-19 vaccines by targeting multiple proteins in the SARS-CoV-2 virus, rather than just one protein like the Spike protein.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Most COVID-19 vaccines elicit immunity against the SARS-CoV-2 Spike protein. However, Spike protein mutations in emerging strains and immune evasion by the SARS-CoV-2 virus demonstrates the need to develop more broadly targeting vaccines. To facilitate this, we use mass spectrometry to identify immunopeptides derived from seven relatively conserved structural and non-structural SARS-CoV-2 proteins (N, E, Nsp1/4/5/8/9). We use two different B-lymphoblastoid cell lines to map Human Leukocyte Antigen (HLA) class I and class II immunopeptidomes covering some of the prevalent HLA types across the global human population. We employ DNA plasmid transfection and direct antigen delivery approaches to sample different antigens and find 248 unique HLA class I and HLA class II bound peptides with 71 derived from N, 12 from E, 28 from Nsp1, 19 from Nsp4, 73 from Nsp8 and 45 peptides derived from Nsp9. Over half of the viral peptides are unpublished. T cell reactivity tested against 56 of the detected peptides shows CD8+and CD4+T cell responses against several peptides from the N, E, and Nsp9 proteins. Results from this study will aid the development of next-generation COVID vaccines targeting epitopes from across a number of SARS-CoV-2 proteins.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51959-6" target="_blank">https://www.nature.com/articles/s41467-024-51959-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">
            ü§ñ Early Signs of COVID Long-Term Effects</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-126', this)">
            More</button></h4>
            <div id="toggle-126" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Early biological markers of post-acute sequelae of SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists studied 136 people who had just tested positive for COVID-19 to understand why some people get long-term symptoms after the initial illness. They collected nasal swabs and blood samples over several weeks and found that people who developed long-term symptoms (called PASC) had higher levels of the virus in their bodies and lower antibodies against it, especially within the first 9 days of getting sick. This suggests that how the body responds to COVID-19 early on may play a role in whether someone develops long-term symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To understand the roles of acute-phase viral dynamics and host immune responses in post-acute sequelae of SARS-CoV-2 infection (PASC), we enrolled 136 participants within 5 days of their first positive SARS-CoV-2 real-time PCR test. Participants self-collected up to 21 nasal specimens within the first 28 days post-symptom onset; interviewer-administered questionnaires and blood samples were collected at enrollment, days 9, 14, 21, 28, and month 4 and 8 post-symptom onset. Defining PASC as the presence of any COVID-associated symptom at their 4-month visit, we compared viral markers (quantity and duration of nasal¬†viral RNA load, infectious viral load, and plasma N-antigen level) and host immune markers (IL-6, IL-10, TNF-Œ±, IFN-Œ±, IFN-Œ≥, MCP, IP-10, and Spike IgG) over the acute period. Compared to those who fully recovered, those reporting PASC demonstrated significantly higher maximum levels of SARS-CoV-2 RNA and N-antigen, burden of RNA and infectious viral shedding, and lower Spike-specific IgG levels within 9 days post-illness onset. No significant differences were identified among a panel of host immune markers. Our results suggest early viral dynamics and the associated host immune responses play a role in the pathogenesis of PASC, highlighting the importance of understanding early biological markers in the natural history of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51893-7" target="_blank">https://www.nature.com/articles/s41467-024-51893-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-29</strong> <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">
            ü§ñ How Safe and Effective is the mRNA-1273 Vaccine Over Time?</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-127', this)">
            More</button></h4>
            <div id="toggle-127" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term safety and effectiveness of mRNA-1273 vaccine in adults: COVE trial open-label and booster phases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

Scientists conducted an experiment to test the effectiveness of a vaccine (mRNA-1273) in preventing COVID-19. The results showed that the vaccine was safe and worked well at preventing the disease in adults over 18 years old.

To see if another dose of the vaccine (called a booster) would help even more, researchers gave some people an extra dose after they had already received their first shots. They found that this booster shot helped prevent COVID-19, especially during times when new and more contagious variants of the virus were spreading.

The study also showed that the longer people waited between getting their first vaccine doses and receiving the booster, the better protected they were from getting COVID-19 again. Furthermore, the researchers found that people who received the booster had a stronger immune response against the virus than those who only got the initial shots.

Overall, this study suggests that vaccination with mRNA-1273, including a booster dose, is a safe and effective way to protect against COVID-19, even when new variants are present.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Primary vaccination with mRNA-1273 (100-¬µg) was safe and efficacious at preventing coronavirus disease 2019 (COVID-19) in the previously reported, blinded Part A of the phase 3 Coronavirus Efficacy (COVE; NCT04470427) trial in adults (‚â•18 years) across 99 U.S. sites. The open-label (Parts B and C) primary objectives were evaluation of long-term safety and effectiveness of primary vaccination plus a 50-¬µg booster dose; immunogenicity was a secondary objective. Of 29,035 open-label participants, 19,609 received boosters (mRNA-1273 [n=‚Äâ9647]; placebo-mRNA-1273 [n=‚Äâ9952]; placebo [n=‚Äâ10] groups). Booster safety was consistent with that reported for primary vaccination. Incidences of COVID-19 and severe COVID-19 were higher during the Omicron BA.1 than Delta variant waves and boosting versus non-boosting was associated with a significant, 47.0% (95% CI : 39.0-53.9%) reduction of Omicron BA.1 incidence (24.6 [23.4‚Äâ‚àí‚Äâ25.8] vs 46.4 [40.6‚Äâ‚àí‚Äâ52.7]/1000 person-months). In an exploratory Cox regression model adjusted for time-varying covariates, a longer median interval between primary vaccination and boosting (mRNA-1273 [13 months] vs placebo-mRNA-1273 [8 months]) was associated with significantly lower, COVID-19 risk (24.0% [16.0%‚Äâ‚àí‚Äâ32.0%]) during Omicron BA.1 predominance. Boosting elicited greater immune responses against SARS-CoV-2 than primary vaccination, irrespective of prior SARS-CoV-2 infection. Primary vaccination and boosting with mRNA-1273 demonstrated acceptable safety, effectiveness and immunogenicity against COVID-19, including emergent variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50376-z" target="_blank">https://www.nature.com/articles/s41467-024-50376-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-28</strong> <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">
            ü§ñ Fibrin's Role in COVID-19 Complications</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-128', this)">
            More</button></h4>
            <div id="toggle-128" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Fibrin drives thromboinflammation and neuropathology in COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Some people who get COVID-19 develop blood clots that can be life-threatening, as well as brain symptoms like memory loss or confusion. This can happen even after they've recovered from the virus. Despite this, scientists still don't fully understand why it happens and there aren't enough treatments.

A protein called fibrin, which helps form blood clots, is found in high amounts in the lungs and brains of people with COVID-19. It's also a good indicator of how severe their symptoms will be and can predict brain problems after they've recovered.

Scientists have discovered that fibrin binds to the COVID-19 virus and forms pro-inflammatory blood clots, which causes more inflammation and damage to the brain and lungs. They've also found that targeting this part of the fibrin protein with a special antibody can help protect against further damage.

This suggests that targeting fibrin could be a new way to treat not only acute COVID-19 but also long-term symptoms after recovery.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Life-threatening thrombotic events and neurological symptoms are prevalent in COVID-19 and are¬†persistent in patients with long COVID experiencing post-acute sequelae of SARS-CoV-2 infection1,2,3,4. Despite the clinical evidence1,5,6,7, the underlying mechanisms of coagulopathy in COVID-19 and its consequences in inflammation and neuropathology remain poorly understood and treatment options are insufficient. Fibrinogen, the central structural component of blood clots, is abundantly deposited in the lungs and brains of patients with COVID-19, correlates with disease severity and is a predictive biomarker for post-COVID-19 cognitive deficits1,5,8,9,10. Here we show that fibrin binds to the SARS-CoV-2 spike protein, forming proinflammatory blood clots that drive systemic thromboinflammation and neuropathology in COVID-19. Fibrin, acting through its inflammatory domain, is required for oxidative stress and macrophage activation in the lungs, whereas it¬†suppresses natural killer cells, after SARS-CoV-2 infection. Fibrin promotes neuroinflammation and neuronal loss after infection, as well as innate immune activation in the brain and lungs independently of active infection. A¬†monoclonal antibody targeting the inflammatory fibrin domain provides¬†protection from microglial activation and neuronal injury, as well as from thromboinflammation in the lung after infection. Thus, fibrin drives inflammation and neuropathology in SARS-CoV-2 infection, and fibrin-targeting immunotherapy may represent a therapeutic intervention for patients with acute COVID-19 and long COVID.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41586-024-07873-4" target="_blank">https://www.nature.com/articles/s41586-024-07873-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-28</strong> <a href="https://www.nature.com/articles/s41598-024-71047-5" target="_blank">
            ü§ñ IgE Response After COVID Vaccination and Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-129', this)">
            More</button></h4>
            <div id="toggle-129" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>An unexpected IgE anti-receptor binding domain response following natural infection and different types of SARS-CoV-2 vaccines</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

When people get infected with COVID-19 or vaccinated against it, their bodies produce special proteins called antibodies to fight the virus. Most research has focused on one type of antibody, but there are other types too. This study looked at a different type of antibody called IgE, which is usually associated with allergies and parasites.

The researchers took blood samples from 59 people in Brazil who had been infected with COVID-19 or vaccinated against it. They found that IgE antibodies were produced after infection and vaccination. The good news was that the vaccines helped increase the levels of these antibodies, making them more effective at fighting the virus.

The study also found that different types of vaccines worked better than others in producing IgE antibodies. However, when people got a booster shot with a third vaccine, their IgE antibody levels became similar across all groups.

Overall, this study suggests that IgE antibodies play an important role in our immune system's response to COVID-19 and viruses, and more research is needed to understand how they work and their potential benefits.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humoral response to SARS-CoV-2 has been studied, predominantly the classical IgG and its subclasses. Although IgE antibodies are typically specific to allergens or parasites, a few reports describe their production in response to SARS-CoV-2 and other viruses. Here, we investigated IgE specific to receptor binding domain (RBD) of SARS-CoV-2 in a Brazilian cohort following natural infection and vaccination. Samples from 59 volunteers were assessed after infection (COVID-19), primary immunization with vectored (ChAdOx1) or inactivated (CoronaVac) vaccines, and booster immunization with mRNA (BNT162b2) vaccine. Natural COVID-19 induced IgE, but vaccination increased its levels. Subjects vaccinated with two doses of ChAdOx1 exhibited a more robust response than those immunized with two doses of CoronaVac; however, after boosting with BNT162b2, all groups presented similar IgE levels. IgE showed intermediate-to-high avidity, especially after the booster vaccine. We also found IgG4 antibodies, mainly after the booster, and they moderately correlated with IgE. ELISA results were confirmed by control assays, using IgG depletion by protein G and lack of reactivity with heterologous antigen. In our cohort, no clinical data could be associated with the IgE response. We advocate for further research on IgE and its role in viral immunity, extending beyond allergies and parasitic infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-71047-5" target="_blank">https://www.nature.com/articles/s41598-024-71047-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41598-024-70064-8" target="_blank">
            ü§ñ Using a special inhaler treatment with extra-hepatin benefits people with COVID-19 symptoms</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-130', this)">
            More</button></h4>
            <div id="toggle-130" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nebulized enriched heparin improves respiratory parameters in patients with COVID-19: a phase I/II randomized and triple-blind clinical trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Scientists wanted to see if inhaling a medicine called inhaled enriched heparin can help treat COVID-19. They tested it on 27 hospitalized adults with COVID-19 and compared them to people who got a placebo (a fake treatment). The results showed that inhaling this medicine was safe and helped reduce the need for oxygen and improved lung function in patients. These promising findings mean they want to continue testing it in even larger groups of people to see if it's truly effective in treating COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the safety and the potential antiviral treatment of inhaled enriched heparin in patients with COVID-19. The specific objectives were to investigate the anticoagulation profile, antiviral and anti-inflammatory effects, and respiratory evolution of inhaled enriched heparin. We conducted a randomized, triple-blind, placebo-controlled Phase I/II clinical trial in hospitalized adults with COVID-19 receiving inhalation of enriched heparin or saline (placebo) every 4¬†h for 7¬†days. Among the 27 patients who completed the study, no changes in blood coagulation parameters were observed, indicating the safety of inhaled enriched heparin. The group receiving enriched heparin showed a significant reduction in the need for supplemental oxygen and improvement in respiratory parameters, such as the PaO2/FiO2ratio. Inhalation of enriched heparin is shown to be safe and has also demonstrated potential therapeutic benefits for patients with COVID-19. These promising results justify the continuation of the study to the next phase, Phase II/III, to further evaluate the therapeutic efficacy of inhaled enriched heparin in the treatment of COVID-19-associated viral pneumonia.Trial registration:¬†ClinicalTrials.gov. 08/02/2021. Identifier: NCT04743011.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70064-8" target="_blank">https://www.nature.com/articles/s41598-024-70064-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41467-024-50976-9" target="_blank">
            ü§ñ Designing Improved COVID Vaccine Proteins Using Computer Simulations</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-131', this)">
            More</button></h4>
            <div id="toggle-131" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Simulation-driven design of stabilized SARS-CoV-2 spike S2 immunogens</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

The COVID-19 vaccine works by using a special part of the virus called the spike (S) to help our bodies fight it. However, the virus can change over time and become harder for the vaccine to recognize. The stable part of the spike is called S2, which doesn't change much and can trigger strong antibodies in our bodies that can protect us against different types of viruses. But, this part of the spike is difficult to keep stable and has been hard to use as a vaccine. We used computer simulations to design a new version of S2 that is more stable and still triggers strong antibodies. Our results show that this new version works well in tests and could be an effective alternative to traditional COVID-19 vaccines.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The full-length prefusion-stabilized SARS-CoV-2 spike (S) is the principal antigen of COVID-19 vaccines. Vaccine efficacy has been impacted by emerging variants of concern that accumulate most of the sequence modifications in the immunodominant S1 subunit. S2, in contrast, is the most evolutionarily conserved region of the spike and can elicit broadly neutralizing and protective antibodies. Yet, S2‚Äôs usage as an alternative vaccine strategy is hampered by its general instability. Here, we use a simulation-driven approach to design S2-only immunogens stabilized in a closed prefusion conformation. Molecular simulations provide a mechanistic characterization of the S2 trimer‚Äôs opening, informing the design of tryptophan substitutions that impart kinetic and thermodynamic stabilization. Structural characterization via cryo-EM shows the molecular basis of S2 stabilization in the closed prefusion conformation. Informed by molecular simulations and corroborated by experiments, we report an engineered S2 immunogen that exhibits increased protein expression, superior thermostability, and preserved immunogenicity against sarbecoviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50976-9" target="_blank">https://www.nature.com/articles/s41467-024-50976-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">
            ü§ñ How Different Vaccines Interact with the Immune System</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-132', this)">
            More</button></h4>
            <div id="toggle-132" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Antibody Fc receptor binding and T cell responses to homologous and heterologous immunization with inactivated or mRNA vaccines against SARS-CoV-2</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Two different COVID-19 vaccines, CoronaVac and BNT162b2 (also known as Pfizer-BioNTech), were tested in a study to see how well they worked against severe illness from COVID-19. The results showed that while both vaccines are effective in preventing serious illness, they work differently inside the body.

The researchers looked at how well each vaccine stimulated the immune system to produce antibodies and T cells, which help fight off the virus. They found that:

* BNT162b2 (Pfizer-BioNTech) booster shots produced more antibodies against COVID-19 than CoronaVac booster shots.
* Participants who received CoronaVac as a first dose had lower levels of certain immune cells called CD4+IFNŒ≥+cells, but this level increased after receiving either CoronaVac or BNT162b2 booster shots.
* The type of vaccine used for the first dose (CoronaVac or Pfizer-BioNTech) influenced which types of T cells and antibodies were produced by the body.

These findings suggest that while both vaccines have their strengths and weaknesses, they can still provide protection against severe COVID-19 illness.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Whole virion inactivated vaccine CoronaVac (C) and Spike (S) mRNA BNT162b2 (B) vaccines differ greatly in their ability to elicit neutralizing antibodies but have somewhat comparable effectiveness in protecting from severe COVID-19. We conducted further analyses for a randomized trial (Cobovax study, NCT05057169) of third dose homologous and heterologous booster vaccination, i.e. four interventions CC-C, CC-B, BB-C and BB-B. Here, we assess vaccine immunogenicity beyond neutralizing function, including S and non-S antibodies with Fc receptor (FcR) binding, antibody avidity and T cell specificity to 6 months post-vaccination. Ancestral and Omicron S-specific IgG and FcR binding are significantly higher by BNT162b2 booster than CoronaVac, regardless of first doses. Nucleocapsid (N) antibodies are only increased in homologous boosted CoronaVac participants (CC-C). CoronaVac primed participants have lower baseline S-specific CD4+IFNŒ≥+cells, but are significantly increased by either CoronaVac or BNT162b2 boosters. Priming vaccine content defined T cell peptide specificity preference, with S-specific T cells dominating B primed groups and non-S structural peptides contributing more in C primed groups, regardless of booster type. S-specific CD4+T cell responses, N-specific antibodies, and antibody effector functions via Fc receptor binding may contribute to protection and compensate for less potent neutralizing responses in CoronaVac recipients.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51427-1" target="_blank">https://www.nature.com/articles/s41467-024-51427-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-27</strong> <a href="https://www.nature.com/articles/s41541-024-00948-3" target="_blank">
            ü§ñ Pregnant Women's COVID Immunity During Pregnancy</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-133', this)">
            More</button></h4>
            <div id="toggle-133" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralizing and binding antibody responses to SARS-CoV-2 with hybrid immunity in pregnancy</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

When pregnant women get vaccinated against COVID-19 or catch the virus, their bodies make antibodies that can help protect them and their babies. Researchers studied how well these antibodies worked after different combinations of vaccination and infection during pregnancy.

They found that when women got three doses of the vaccine, it boosted the level of protective antibodies more than just two doses or getting infected with COVID-19 alone. This effect was even stronger when women had both vaccinated and been infected.

The researchers also looked at how well these antibodies were transferred from mother to baby during pregnancy. They found that antibodies made by the vaccine were passed on more efficiently than those made by infection.

These findings suggest that combining vaccination and natural immunity can create a stronger defense against COVID-19 in pregnant women, which is good news for protecting both mom and baby!</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hybrid immunity against SARS-CoV-2 has not been well studied in pregnancy. We conducted a comprehensive analysis of neutralizing antibodies (nAb) and binding antibodies in pregnant individuals who received mRNA vaccination, natural infection, or both. A third vaccine dose augmented nAb levels compared to the two-dose regimen or natural infection alone; this effect was more pronounced in hybrid immunity. There was reduced anti-Omicron nAb, but the maternal-fetal transfer efficiency remained comparable to that of other variants. Vaccine-induced nAbs were transferred more efficiently than infection-induced nAbs. Anti-spike receptor binding domain (RBD) IgG was associated with nAb against wild-type (Wuhan-Hu-1) following breakthrough infection. Both vaccination and infection-induced anti-RBD IgA, which was more durable than anti-nucleocapsid IgA. IgA response was attenuated in pregnancy compared to non-pregnant controls. These data provide additional evidence of augmentation of humoral immune responses in hybrid immunity in pregnancy.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00948-3" target="_blank">https://www.nature.com/articles/s41541-024-00948-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-26</strong> <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">
            ü§ñ Single Dose COVID Vaccine for Wide Protection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-134', this)">
            More</button></h4>
            <div id="toggle-134" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A single-dose intranasal live-attenuated codon deoptimized vaccine provides broad protection against SARS-CoV-2 and its variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract that a young person can understand:

COVID-19, caused by the SARS-CoV-2 virus, continues to affect people worldwide and cause significant health and economic problems. Despite vaccines being effective in preventing severe cases, there is still no guarantee against new variants of the virus or unexpected breakthrough infections. Researchers have developed a new vaccine called CDO-7N-1 that uses a weakened form of the SARS-CoV-2 virus to stimulate an immune response. In laboratory tests, this vaccine has been shown to provide long-lasting protection against different strains of the virus and prevent lung damage. A single dose of the vaccine triggered a strong immune response in mice, hamsters, and macaques, protecting them from infection with wild-type SARS-CoV-2 and other variants of concern. The researchers believe that this new vaccine could improve immunity against future SARS-CoV-2 variants and provide better protection for people worldwide.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2, COVID-19) continues its significant health and economic impact globally. Despite the success of spike-protein vaccines in preventing severe disease, long-lasting protection against emerging variants and the prevention of breakthrough infections and transmission remain elusive. We generate an intranasal live-attenuated SARS-CoV-2 vaccine, CDO-7N-1, using codon deoptimization. CDO-7N-1 shows highly attenuated replication and minimal or no lung pathology in vivo over multiple passages. It induces robust mucosal and systemic neutralizing antibody and T-cell subset responses, in mice (female K18-hACE2 and male HFH4-hACE2 mice), hamsters, and macaques triggered by a single immunization. Mice and hamsters vaccinated with CDO-7N-1 are protected from challenge with wild-type (WT) SARS-CoV-2 and other variants of concern. Serum from vaccinated animals neutralizes WT SARS-CoV-2, variants of concern (beta and delta), variants of interest (omicron XBB.1.5) and SARS-CoV-1. Antibody responses are sustained and enhanced by repeated immunization or infection with WT SARS-CoV-2. Immunity against all SARS-CoV-2 proteins by CDO-7N-1 should improve efficacy against future SARS-CoV-2 variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51535-y" target="_blank">https://www.nature.com/articles/s41467-024-51535-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-24</strong> <a href="https://www.nature.com/articles/s44298-024-00049-x" target="_blank">
            ü§ñ Lingering Lung Damage After COVID-19 Infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-135', this)">
            More</button></h4>
            <div id="toggle-135" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection results in a unique lung proteome long after virus resolution in the hamster</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

Long COVID, also known as post-acute sequelae of COVID-19 (PASC), is a condition where people experience ongoing problems after having COVID-19. It can make it hard for them to contribute to society. To understand why this happens, researchers studied the lung proteins in hamsters infected with an early version of the SARS-CoV-2 virus. They found that at 3 and 5 days after infection, many lung proteins changed in both male and female hamsters. However, by day 31 (when no virus was left), they saw differences between the sexes, including some proteins that were lower or higher in males than females. The researchers also found that biological sex should be considered when developing treatments for PASC because of these differences.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Long COVID or post-acute sequelae of COVID-19 (PASC) remains an ongoing public health issue that causes impairment for those afflicted and diminishes their ability to contribute to society. To address the host response underpinning respiratory PASC, we used the Golden Syrian hamster model infected with ancestral SARS-CoV-2 and examined its lung proteome in a longitudinal experiment. We infected young 6-week old male and female hamsters with 105TCID50of virus via the intranasal route and sampled the lung at 1, 3, 5, and 31 days post infection (dpi). We compared the infected lung proteome to that of uninfected sex-matched controls. We found almost no differences in protein levels at 1 dpi, with hundreds at 3 dpi, and thousands at 5 dpi. Many overlapping differential protein levels and pathways were seen in both sexes at 3 and 5 dpi including the Coagulation and Complement cascades. Notably, we found differences between the sexes at 31 dpi which included many targets with decreased levels of protein in the males. We also noted an increase in 7 proteins in both sexes at 31 dpi including proteins responsible for airway mucosal layer integrity such as Mucin 5B and Calcium-activated chloride channel regulator 1. Longitudinally, 38 proteins were changed in levels across more than one timepoint in the males but only three proteins were in the females, Secretoglobin family 1‚ÄâA member 1, Poly [ADP-ribose] polymerase, and Apolipoprotein D. Overall, we show that there are changes to the lung proteome at 31 dpi, a time when no SARS-CoV-2 remains, and that there are sex differences in that proteome after infection with the ancestral strain. We conclude that biological sex should be examined as a variable when testing medical countermeasures for PASC in the Golden Syrian hamster due to host differences between the sexes.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s44298-024-00049-x" target="_blank">https://www.nature.com/articles/s44298-024-00049-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-24</strong> <a href="https://www.nature.com/articles/s41598-024-70612-2" target="_blank">
            ü§ñ How Aging Affects COVID-19 Symptoms in Mice Studies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-136', this)">
            More</button></h4>
            <div id="toggle-136" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Premature aging effects on COVID-19 pathogenesis: new insights from mouse models</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Here's a simplified version of the abstract:

As we get older, our bodies become more vulnerable to severe illnesses like COVID-19. The connection between age, existing health problems, and COVID-19 is not fully understood. This study used a special mouse model that ages quickly to see how COVID-19 affects different age groups.

The results showed that younger mice can fight off the virus easily, but older mice have a weaker response and are more likely to get very sick and die. The mouse model with premature aging showed milder symptoms, but still had some problems with their immune system and metabolism.

By studying how COVID-19 affects these mice, scientists hope to gain a better understanding of why some people are more susceptible to severe illness and develop new treatments for COVID-19 and other diseases related to aging.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Aging is identified as a significant risk factor for severe coronavirus disease-2019 (COVID-19), often resulting in profound lung damage and mortality. Yet, the biological relationship between aging, aging-related comorbidities, and COVID-19 remains incompletely understood. This study aimed to elucidate the age-related COVID19 pathogenesis using an Hutchinson-Gilford progeria syndrome (HGPS) mouse model, a premature aging disease model, with humanized ACE2 receptors. Pathological features were compared between young, aged, and HGPS hACE2 mice following SARS-CoV-2 challenge. We demonstrated that young mice display robust interferon response and antiviral activity, whereas this response is attenuated in aged mice. Viral infection in aged mice results in severe respiratory tract hemorrhage, likely contributing a higher mortality rate. In contrast, HGPS hACE2 mice exhibit milder disease manifestations characterized by minor immune cell infiltration and dysregulation of multiple metabolic processes. Comprehensive transcriptome analysis revealed both shared and unique gene expression dynamics among different mouse groups. Collectively, our studies evaluated the impact of SARS-CoV-2 infection on progeroid syndromes using a HGPS hACE2 mouse model, which holds promise as a useful tool for investigating COVID-19 pathogenesis in individuals with premature aging.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-70612-2" target="_blank">https://www.nature.com/articles/s41598-024-70612-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-20</strong> <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">
            ü§ñ Non-human primates show signs of long-COVID linked to high blood sugar immune response.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-137', this)">
            More</button></h4>
            <div id="toggle-137" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Non-human primate model of long-COVID identifies immune associates of hyperglycemia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Hyperglycemia happens when your body has trouble using sugar after catching a severe illness like COVID-19. Scientists don't understand this very well in people, so they used monkeys infected with the same virus to study it better.

They found that shortly after getting sick, the blood of these monkeys had changes called "chemokines" that could lead to high sugar levels. Four months later, this still happened in most of them. High sugar is linked to something called glycogen in the liver and pancreas. But they didn't see the virus living in those parts anymore.

They also tested a vaccine for COVID-19 given to monkeys on day 4 after infection. The monkey's blood sugar levels went down as a result.

These findings suggest that studying these infected monkeys can help understand what happens when people get sick from this illness, and how we might treat it better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Hyperglycemia, and exacerbation of pre-existing deficits in glucose metabolism, are manifestations of the post-acute sequelae of SARS-CoV-2. Our understanding of metabolic decline after acute COVID-19 remains unclear due to the lack of animal models. Here, we report a non-human primate model of metabolic post-acute sequelae of SARS-CoV-2 using SARS-CoV-2 infected African green monkeys. Using this model, we identify a dysregulated blood chemokine signature during acute COVID-19 that correlates with elevated and persistent hyperglycemia four months post-infection. Hyperglycemia also correlates with liver glycogen levels, but there is no evidence of substantial long-term SARS-CoV-2 replication in the liver and pancreas. Finally, we report a favorable glycemic effect of the SARS-CoV-2 mRNA vaccine, administered on day 4 post-infection. Together, these data suggest that the African green monkey model exhibits important similarities to humans and can be utilized to assess therapeutic candidates to combat COVID-related metabolic defects.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50339-4" target="_blank">https://www.nature.com/articles/s41467-024-50339-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-13</strong> <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">
            ü§ñ SARS-CoV-2 N protein affects lung genes leading to pneumonia.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-138', this)">
            More</button></h4>
            <div id="toggle-138" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 N protein-induced Dicer, XPO5, SRSF3, and hnRNPA3 downregulation causes pneumonia</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how certain pieces of machinery that help make RNA (a type of molecule important for cells) work less effectively when someone has COVID-19. When the researchers found there were fewer of these pieces, they had more trouble with their lungs getting sick from the virus. The study showed that a protein made by the virus also makes these pieces disappear faster. This leads to problems like DNA damage and lung issues. They also noticed older people have less of these pieces in their lungs, making them sicker when infected. Two types of medicine helped make the sickness better.

The main points are:
- Certain RNA-making parts don't work well in COVID patients.
- The virus causes fewer of these parts to be made.
- This makes it harder for cells to repair themselves and function properly.
- Older people get more sick because they have less of these parts.
- Special medicines can help.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Though RNAi and RNA-splicing machineries are involved in regulating severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication, their precise roles in coronavirus disease 2019 (COVID-19) pathogenesis remain unclear. Herein, we show that decreased RNAi component (Dicer and XPO5) and splicing factor (SRSF3 and hnRNPA3) expression correlate with increased COVID-19 severity. SARS-CoV-2 N protein induces the autophagic degradation of Dicer, XPO5, SRSF3, and hnRNPA3, inhibiting miRNA biogenesis and RNA splicing and triggering DNA damage, proteotoxic stress, and pneumonia. Dicer, XPO5, SRSF3, and hnRNPA3 knockdown increases, while their overexpression decreases, N protein-induced pneumonia‚Äôs severity. Older mice show lower expression of Dicer, XPO5, SRSF3, and hnRNPA3 in their lung tissues and exhibit more severe N protein-induced pneumonia than younger mice. PJ34, a poly(ADP-ribose) polymerase inhibitor, or anastrozole, an aromatase inhibitor, ameliorates N protein- or SARS-CoV-2-induced pneumonia by restoring Dicer, XPO5, SRSF3, and hnRNPA3 expression. These findings will aid in developing improved treatments for SARS-CoV-2-associated pneumonia.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51192-1" target="_blank">https://www.nature.com/articles/s41467-024-51192-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-09</strong> <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">
            ü§ñ Major circulating SARS-CoV-2 B cells are unusual after vaccine or infection</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-139', this)">
            More</button></h4>
            <div id="toggle-139" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Atypical and non-classical CD45RBlomemory B cells are the majority of circulating SARS-CoV-2 specific B cells following mRNA vaccination or COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The resting memory B cells are divided into two main groups: classical and atypical. Classical cells stay the same after being activated. Atypical cells can be further split into two types: one with CD11c on their surface, and another with CD23. 

When someone gets a vaccine or gets sick from bacteria, these resting memory B cells help make antibodies that fight the virus. In both scenarios, atypical cells called CD45RBlo play a big role in making antibodies against SARS-CoV-2.

Our findings suggest that tracking the levels of CD45RB on these cells can help us understand how they work and contribute to getting vaccinated or recovering from infections like COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Resting memory B cells can be divided into classical or atypical groups, but the heterogenous marker expression on activated memory B cells makes similar classification difficult. Here, by longitudinal analysis of mass cytometry and CITE-seq data from cohorts with COVID-19, bacterial sepsis, or BNT162b2 mRNA vaccine, we observe that resting B cell memory consist of classical CD45RB+memory and CD45RBlomemory, of which the latter contains of two distinct groups of CD11c+atypical and CD23+non-classical memory cells. CD45RB levels remain stable in these cells after activation, thereby enabling the tracking of activated B cells and plasmablasts derived from either CD45RB+or CD45RBlomemory B cells. Moreover, in both COVID-19 patients and mRNA vaccination, CD45RBloB cells formed the majority of SARS-CoV2 specific memory B cells and correlated with serum antibodies, while CD45RB+memory are activated by bacterial sepsis. Our results thus identify that stably expressed CD45RB levels can be exploited to trace resting memory B cells and their activated progeny, and suggest that atypical and non-classical CD45RBlomemory B cells contribute to SARS-CoV-2 infection and vaccination.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50997-4" target="_blank">https://www.nature.com/articles/s41467-024-50997-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-08-08</strong> <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">
            ü§ñ Monkeypox Virus Infection Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-140', this)">
            More</button></h4>
            <div id="toggle-140" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Multi-omics characterization of the monkeypox virus infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>We studied how monkeypox (MPXV) virus affects human cells using different types of genetic analysis. We found that the virus changes many pathways in the infected cells, including those related to immune responses and cell growth. One interesting thing we noticed is that MPXV causes changes in proteins involved in cell signaling. Specifically, we saw how it can affect a protein called H5 which interacts with DNA. 

We also used this data to look for new treatments that could stop the virus from growing inside cells. By looking at all these different pathways and interactions, we found some promising targets like MTOR, CHUK/IKBKB, and splicing factor kinases, which seem to have strong effects against both monkeypox and another similar virus called vaccinia (VACV).

Overall, our study helps us understand how these viruses affect cells and opens up new possibilities for finding treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Multiple omics analyzes of Vaccinia virus (VACV) infection have defined molecular characteristics of poxvirus biology. However, little is known about the monkeypox (mpox) virus (MPXV) in humans, which has a different disease manifestation despite its high sequence similarity to VACV. Here, we perform an in-depth multi-omics analysis of the transcriptome, proteome, and phosphoproteome signatures of MPXV-infected primary human fibroblasts to gain insights into the virus-host interplay. In addition to expected perturbations of immune-related pathways, we uncover regulation of the HIPPO and TGF-Œ≤ pathways. We identify dynamic phosphorylation of both host and viral proteins, which suggests that MAPKs are key regulators of differential phosphorylation in MPXV-infected cells. Among the viral proteins, we find dynamic phosphorylation of H5 that influenced the binding of H5 to dsDNA. Our extensive dataset highlights signaling events and hotspots perturbed by MPXV, extending the current knowledge on poxviruses. We use integrated pathway analysis and drug-target prediction approaches to identify potential drug targets that affect virus growth. Functionally, we exemplify the utility of this approach by identifying inhibitors of MTOR, CHUK/IKBKB, and splicing factor kinases with potent antiviral efficacy against MPXV and VACV.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-51074-6" target="_blank">https://www.nature.com/articles/s41467-024-51074-6</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-29</strong> <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">
            ü§ñ Modifiable lifestyle factors and COVID-19 outcomes</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-141', this)">
            More</button></h4>
            <div id="toggle-141" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Modifiable lifestyle factors and the risk of post-COVID-19 multisystem sequelae, hospitalization, and death</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Effective ways to prevent problems after getting sick from COVID are important for people, doctors, and governments. This study looked at how things like exercise, diet, sleep, smoking, drinking, being a healthy weight, and eating well can affect someone's health years later.

People who had more healthy habits were less likely to get many different kinds of long-term sicknesses related to COVID. They also had a lower risk of dying or needing hospitalization after getting sick from COVID. These healthier habits worked even for people who didn't get very sick in the first place.

The study found that following these healthy habits was really important and not just because they might have had fewer other health problems before catching COVID. This means it's good to keep up with a healthy lifestyle, no matter what, to help prevent long-term sickness from COVID.

These findings show how keeping fit and eating well can be very helpful in the future for dealing with possible new pandemics.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Effective prevention strategies for post-COVID complications are crucial for patients, clinicians, and policy makers to mitigate their cumulative burden. This study evaluated the association of modifiable lifestyle factors (smoking, alcohol intake, BMI, physical activity, sedentary time, sleep duration, and dietary habits) with COVID-19 multisystem sequelae, death, and hospitalization in the UK Biobank cohort (n=‚Äâ68,896). A favorable lifestyle (6-10 healthy factors; 46.4%) was associated with a 36% lower risk of multisystem sequelae (HR, 0.64; 95% CI, 0.58-0.69; ARR at 210 days, 7.08%; 95% CI, 5.98-8.09) compared to an unfavorable lifestyle (0-4 factors; 12.3%). Risk reductions spanned all 10 organ systems, including cardiovascular, coagulation, metabolic, gastrointestinal, kidney, mental health, musculoskeletal, respiratory disorders, and fatigue. This beneficial effect was largely attributable to direct lifestyle impacts independent of corresponding pre-infection comorbidities (71% for any sequelae). A favorable lifestyle was also related to the risk of post-COVID death (HR 0.59, 0.52-0.66) and hospitalization (HR 0.78, 0.73-0.84). These associations persisted across acute and post-acute infection phases, irrespective of hospitalization status, vaccination, or SARS-CoV-2 variant. These findings underscore the clinical and public health importance of adhering to a healthy lifestyle in mitigating long-term COVID-19 adverse impacts and enhancing future pandemic preparedness.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-50495-7" target="_blank">https://www.nature.com/articles/s41467-024-50495-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-24</strong> <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">
            ü§ñ Tetravalent COVID-19 vaccine trial results</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-142', this)">
            More</button></h4>
            <div id="toggle-142" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Efficacy of the tetravalent protein COVID-19 vaccine, SCTV01E: a phase 3 double-blind, randomized, placebo-controlled trial</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study tested a new type of vaccine called SCTV01E that can protect against four different strains of coronavirus (SARS-CoV-2). The goal was to see if it works by comparing how many people got sick with the virus when given the vaccine versus not getting the vaccine.

They gave the vaccine to 9,223 people and had them wait for seven days. They found that the vaccine worked very well in preventing someone from getting sick: about two-thirds of people who didn't get vaccinated got sick compared to only one-third who did get the vaccine. It also helped protect against all kinds of coronavirus infections.

The vaccine was safe, with most side effects being mild and short-lived. The researchers think this type of vaccine might work well against new strains that appear over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Evolution of SARS-CoV-2 variants emphasizes the need for multivalent vaccines capable of simultaneously targeting multiple strains. SCTV01E is a tetravalent COVID-19 vaccine derived from the spike protein of SARS-CoV-2 variants Alpha, Beta, Delta, and Omicron BA.1. In this double-blinded placebo-controlled pivotal efficacy trial (NCT05308576), the primary endpoint was vaccine efficacy (VE) against COVID-19 seven days post-vaccination in individuals without recent infection. Other endpoints included evaluating safety, immunogenicity, and the VE against all SARS-CoV-2 infections in individuals meeting the study criteria. Between December 26, 2022, and January 15, 2023, 9,223 individuals were randomized at a 1:1 ratio to receive SCTV01E or a placebo. SCTV01E showed a VE of 69.4% (95% CI: 50.6, 81.0) 7 days post-vaccination, with 75 cases in the placebo group and 23 in the SCTV01E group for the primary endpoint. VEs were 79.7% (95% CI: 51.0, 91.6) and 82.4% (95% CI: 57.9, 92.6), respectively, for preventing symptomatic infection and all SARS-CoV-2 infections 14 days post-vaccination. SCTV01E elicited a 25.0-fold higher neutralizing antibody response against Omicron BA.5 28 days post-vaccination compared to placebo. Reactogenicity was generally mild and transient, with no reported vaccine-related SAE, adverse events of special interest (AESI), or deaths. The trial aligned with the shift from dominant variants BA.5 and BF.7 to XBB, suggesting SCTV01E as a potential vaccine alternative effective against present and future variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-49832-7" target="_blank">https://www.nature.com/articles/s41467-024-49832-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-23</strong> <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">
            ü§ñ SARS-CoV-2 spikes block a lung gene, causing inflammation.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-143', this)">
            More</button></h4>
            <div id="toggle-143" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Inflammation in the COVID-19 airway is due to inhibition of CFTR signaling by the SARS-CoV-2 spike protein</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 makes people sick partly by causing an overreaction in their lungs that leads to more inflammation. This inflammation can last for a long time after the virus is gone, which some doctors call "Long COVID."

The virus also stops another system called CFTR from working properly in lung cells. CFTR helps keep the lungs moist and healthy. When this doesn't work, it causes problems similar to a condition called cystic fibrosis.

We think SARS-CoV-2 might be stopping CFTR because of something on its surface called spike protein. This could happen by blocking another system on the cell surface that normally helps fix CFTR.

Our experiments showed that when lung cells are exposed to this virus, they lose their ability to make and use a healthy version of CFTR. Without CFTR working properly, cells start making too much of two other things that cause inflammation: NFŒ∫B and ENaC.

We found that adding certain drugs called cardiac glycosides can help keep the cells' CFTR from disappearing when exposed to the virus. These drugs might be important for treating "Long COVID" as well.

Overall, our experiments suggest that SARS-CoV-2's spike protein stops a healthy system in lung cells (CFTR) from working properly, which could lead to inflammation and possibly cause symptoms of "Long COVID.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2-contributes to sickness and death in COVID-19 patients partly by inducing a hyper-proinflammatory immune response in the host airway. This hyper-proinflammatory state involves activation of signaling by NFŒ∫B, and unexpectedly, ENaC, the epithelial sodium channel. Post-infection inflammation may also contribute to "Long COVID"/PASC. Enhanced signaling by NFŒ∫B and ENaC also marks the airway of patients suffering from cystic fibrosis, a life-limiting proinflammatory genetic disease due to inactivating mutations in the CFTR gene. We therefore hypothesized that inflammation in the COVID-19 airway might similarly be due to inhibition of CFTR signaling by SARS-CoV-2 spike protein, and therefore activation of both NFŒ∫B and ENaC signaling. We used western blot and electrophysiological techniques, and an organoid model of normal airway epithelia, differentiated on an air‚Äìliquid-interface (ALI). We found that CFTR protein expression and CFTR cAMP-activated chloride channel activity were lost when the model epithelium was exposed to SARS-CoV-2 spike proteins. As hypothesized, the absence of CFTR led to activation of both TNFŒ±/NFŒ∫B signaling and Œ± and Œ≥ ENaC. We had previously shown that the cardiac glycoside drugs digoxin, digitoxin and ouabain blocked interaction of spike protein and ACE2. Consistently, addition of 30¬†nM concentrations of the cardiac glycoside drugs, prevented loss of both CFTR protein and CFTR channel activity. ACE2 and CFTR were found to co-immunoprecipitate in both basal cells and differentiated epithelia. Thus spike-dependent CFTR loss might involve ACE2 as a bridge between Spike and CFTR. In addition, spike exposure to the epithelia resulted in failure of endosomal recycling to return CFTR to the plasma membrane. Thus, failure of CFTR recovery from endosomal recycling might be a mechanism for spike-dependent loss of CFTR. Finally, we found that authentic SARS-CoV-2 virus infection induced loss of CFTR protein, which was rescued by the cardiac glycoside drugs digitoxin and ouabain. Based on experiments with this organoid model of small airway epithelia, and comparisons with 16HBE14o- and other cell types expressing normal CFTR, we predict that inflammation in the COVID-19 airway may be mediated by inhibition of CFTR signaling by the SARS-CoV-2 spike protein, thus inducing a cystic fibrosis-like clinical phenotype. To our knowledge this is the first time COVID-19 airway inflammation has been experimentally traced in normal subjects to a contribution from SARS-CoV-2 spike-dependent inhibition of CFTR signaling.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66473-4" target="_blank">https://www.nature.com/articles/s41598-024-66473-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-11</strong> <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">
            ü§ñ SARS-CoV-2 Reinfections and Long COVID Study Results</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-144', this)">
            More</button></h4>
            <div id="toggle-144" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Insights from an N3C RECOVER EHR-based cohort study characterizing SARS-CoV-2 reinfections and Long COVID</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT SIMPLIFIED

The pandemic is still going on for over 3 years, but doctors don't fully know about getting sick with a virus called SARS-CoV-2 more than once. We studied lots of people's health records to learn more.

We found that:
- Most reinfections happened during the Omicron wave.
- People who got infected twice had more Long COVID (a long-term sickness) compared to those who only got it once in the same period.
- Before you get reinfected, your body might not have as much protein.
- Getting a bad first infection also makes the second one worse. 
- Older people and those at higher risk of dying are more likely to have a severe second infection if they had a bad first one.

This helps us understand reinfections better so we can find new ways to help people who get sick again.</p><br>
                <p><strong>Abstract</strong></p>
                <p>BackgroundAlthough the COVID-19 pandemic has persisted for over 3 years, reinfections with SARS-CoV-2 are not well understood. We aim to characterize reinfection, understand development of Long COVID after reinfection, and compare severity of reinfection with initial infection.MethodsWe use an electronic health record study cohort of over 3 million patients from the National COVID Cohort Collaborative as part of the NIH Researching COVID to Enhance Recovery Initiative. We calculate summary statistics, effect sizes, and Kaplan‚ÄìMeier curves to better understand COVID-19 reinfections.ResultsHere we validate previous findings of reinfection incidence (6.9%), the occurrence of most reinfections during the Omicron epoch, and evidence of multiple reinfections. We present findings that the proportion of Long COVID diagnoses is higher following initial infection than reinfection for infections in the same epoch. We report lower albumin levels leading up to reinfection and a statistically significant association of severity between initial infection and reinfection (chi-squared value: 25,697,p-value: <0.0001) with a medium effect size (Cramer‚ÄôsV: 0.20, DoF‚Äâ=‚Äâ3). Individuals who experienced severe initial and first reinfection were older in age and at a higher mortality risk than those who had mild initial infection and reinfection.ConclusionsIn a large patient cohort, we find that the severity of reinfection appears to be associated with the severity of initial infection and that Long COVID diagnoses appear to occur more often following initial infection than reinfection in the same epoch. Future research may build on these findings to better understand COVID-19 reinfections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s43856-024-00539-2" target="_blank">https://www.nature.com/articles/s43856-024-00539-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-09</strong> <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">
            ü§ñ Human scFv selection and characterization for SARS-CoV-2 nucleocapsid antibodies</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-145', this)">
            More</button></h4>
            <div id="toggle-145" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Selection and characterization of human scFvs targeting the SARS-CoV-2 nucleocapsid protein isolated from antibody libraries of COVID-19 patients</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>In 2019, a new coronavirus called SARS-CoV-2 appeared in China and caused an illness called COVID-19. At first, scientists mainly studied the spike protein. But they later found out that another part of the virus, called the nucleocapsid (N) protein, is very important for spreading the disease and fighting off our immune system. 

Scientists decided to create a collection of special antibodies against this N protein using patients who had been infected with SARS-CoV-2. They used a new technology to pick out some strong antibody parts called single chain fragments variable (scFvs). These scFvs can be made into different forms like tiny intrabodies inside cells or as regular proteins outside the cells. 

They found that some of these antibodies work very well when they are inside cells, and others work even better when they are made separately from the cell. All of them can stick to the N protein with a strong grip. This collection of powerful N protein antibodies is new and could help in research and making tests for detecting SARS-CoV-2.

So basically, scientists got some really good tools to fight against this virus by studying its parts more closely.</p><br>
                <p><strong>Abstract</strong></p>
                <p>In 2019, the novel SARS-CoV-2 coronavirus emerged in China, causing the pneumonia named COVID-19. At the beginning, all research efforts were focused on the spike (S) glycoprotein. However, it became evident that the nucleocapsid (N) protein is pivotal in viral replication, genome packaging and evasion of the immune system, is highly immunogenic, which makes it another compelling target for antibody development alongside the spike protein. This study focused on the construction of single chain fragments variable (scFvs) libraries from SARS-CoV-2-infected patients to establish a valuable, immortalized and extensive antibodies source. We used the Intracellular Antibody Capture Technology to select a panel of scFvs against the SARS-CoV-2 N protein. The whole panel of scFv was expressed and characterized both as intrabodies and recombinant proteins. ScFvs were then divided into 2 subgroups: those that exhibited high binding activity to N protein when expressed in yeast or in mammalian cells as intrabodies, and those purified as recombinant proteins, displaying affinity for recombinant N protein in the nanomolar range. This panel of scFvs against the N protein represents a novel platform for research and potential diagnostic applications.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-66558-0" target="_blank">https://www.nature.com/articles/s41598-024-66558-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-07-01</strong> <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">
            ü§ñ Contraceptive Access Disparities During COVID-19 in Brazil</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-146', this)">
            More</button></h4>
            <div id="toggle-146" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A particular epidemiological profile: disparities in access to contraceptive methods in Brazil during the SARS-CoV-2 (COVID-19) pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how easier or harder it was for people to get birth control during the pandemic in Brazil. They surveyed about 1000 young adults aged 20-39 who use contraception there. Most respondents were women, white, and using some form of birth control.

Many people changed their birth control method during the pandemic. But more than a quarter said they had trouble finding information on new methods or seeing healthcare providers to get help with their current birth control.

The survey found that younger black and biracial women with less education and income faced bigger challenges getting access to contraception and family planning services.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Our objective was to study disparities in access to contraception during the COVID-19 pandemic. We performed a cross-sectional study at the University of Campinas, Brazil using a Google questionnaire applied from December 2021 until February 2022, disseminated via snowball technique. The survey asked about sociodemographic characteristics and contraceptive use, as well as the demand for new methods and difficulties in continuing to use contraceptives during the COVID-19 pandemic. We analyzed 1018 completed questionnaires; in total, 742 (72.9%) were women aged between 20 and 39¬†years, 746 (73.3%) were White and 602 (59.2%) used contraceptives. During the COVID-19 pandemic, about 23% of respondents changed their method and approximately 20% of respondents looked for new methods. Among the latter, 31.3% reported some difficulty with obtaining guidance on new methods while only 5.3% of the respondents reported some difficulty with continuing their contraceptive. The main difficulty in both cases was the difficulty with getting a healthcare provider appointment. Our results point to a particular epidemiological population, of younger black and biracial women, with lower education and lower income, which suffered health disparities during the COVID-19 pandemic and found difficulties with using contraceptives and accessing family planning services.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65946-w" target="_blank">https://www.nature.com/articles/s41598-024-65946-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-28</strong> <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">
            ü§ñ COVID-19 patient immune responses and disease severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-147', this)">
            More</button></h4>
            <div id="toggle-147" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 antigenemia and RNAemia in association with disease severity in patients with COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>### ABSTRACT REWRITTEN

Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes different sickness levels from mild to severe. We studied how this virus affects people and what parts of the body it gets into. In our study, we looked at two things: when certain parts of the virus show up in a person's blood (antigenemia), and when its genetic material is detectable (RNAemia).

We examined 117 patients with confirmed COVID-19. Their ages ranged from around 52 to 79 years old, with more males than females. We found that in people who died early, the virus parts showed up very fast within a week after they were admitted.

When we looked at how these virus parts show up over time:
- They showed up in everyone's blood (100%) by the first week.
- At admission, they could be seen 64.7% of the time but correctly identified as being there 73% of the time.
- In the second week, they were spotted in about 69% of people and correctly detected in all cases.

We also saw that sick patients who had no symptoms at first (asymptomatic) still showed antigenemia. But by the end of the first week, none of them did anymore. The virus genetic material was found more often in people who died than in those who were just mildly sick.

Looking at how long people lived after being admitted:
- People with higher levels of these parts showing up in their blood had a worse chance of surviving.
- This finding suggests that if the virus parts show up quickly, it might mean someone is more likely to die early.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), the virus responsible for COVID-19, causes a spectrum of symptoms ranging from mild upper to severe lower respiratory tract infections.However, the dynamics of nucleocapsid (N) protein antigenemia and RNAemia are not fully understood. We conducted a cohort study involving 117 patients with clinically confirmed COVID-19, focusing on the kinetics of antigenemia and RNAemia and their association with various clinical characteristics. The patients had a median age of 66.0¬†years (52.0‚Äì79.0¬†years), with a gender distribution of 46.2% male and 53.8% female. Antigenemia reached 100% in fatal cases during the first week after admission. The sensitivity/specificity of antigenemia for diagnosis were 64.7%/73.0% at admission, 69.1%/100% in Week 1, and 66.3%/100% in Week 2. Additionally, the rates of antigenemia in asymptomatic patients were 27.3% upon admission and 22.0% in Week 1, respectively; however, no antigenemia was in samples collected in Week 2. Viral RNAemia was not detected in asymptomatic patients, but RNAemia viral loads were elevated in fatal cases. Kaplan‚ÄìMeier survival curves demonstrated a higher mortality rate when antigenemia concentrations were elevated in the follow-up samples (P=‚Äâ0.005). Our study provides a comprehensive analysis of the kinetics of viral N-protein antigenemia and RNAemia according to disease severity and clinical classification. Our findings suggest that highest concentrations of antigenemia in fatal cases occur in the first week after admission, indicating that early elevated antigenemia may serve as a marker of mortality risk.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-65489-0" target="_blank">https://www.nature.com/articles/s41598-024-65489-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-21</strong> <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">
            ü§ñ SARS-CoV-2 virus levels in Ontario wastewater during pandemic</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-148', this)">
            More</button></h4>
            <div id="toggle-148" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 viral titer measurements in Ontario, Canada wastewaters throughout the COVID-19 pandemic</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic in Canada, Ontario started checking water from sewers to see if people had coronavirus. They did this starting from January 2021 until March 2023 by looking at samples from different places across the province. These samples came from more than half of Ontario‚Äôs population and some parts of Canada. The team checked things like where these places were, how many people they served, what kind of water it was, and how much water flowed through.

They did this test 2 to 7 times a week at each place. This helps them understand if coronavirus is spreading in different areas. It's especially useful now that the Omicron variant is around and fewer people are getting sick enough to be tested in hospitals. The team saved all this information so they can use it for research and planning for future sicknesses.

The data from these sewer samples has been very helpful for keeping track of coronavirus cases across Ontario.</p><br>
                <p><strong>Abstract</strong></p>
                <p>During the COVID-19 pandemic, the Province of Ontario, Canada, launched a wastewater surveillance program to monitor SARS-CoV-2, inspired by the early work and successful forecasts of COVID-19 waves in the city of Ottawa, Ontario. This manuscript presents a dataset from January 1, 2021, to March 31, 2023, with RT-qPCR results for SARS-CoV-2 genes and PMMoV from 107 sites across all 34 public health units in Ontario, covering 72% of the province‚Äôs and 26.2% of Canada‚Äôs population. Sampling occurred 2‚Äì7 times weekly, including geographical coordinates, serviced populations, physico-chemical water characteristics, and flowrates. In doing so, this manuscript ensures data availability and metadata preservation to support future research and epidemic preparedness through detailed analyses and modeling. The dataset has been crucial for public health in tracking disease locally, especially with the rise of the Omicron variant and the decline in clinical testing, highlighting wastewater-based surveillance‚Äôs role in estimating disease incidence in Ontario.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41597-024-03414-w" target="_blank">https://www.nature.com/articles/s41597-024-03414-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-12</strong> <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">
            ü§ñ COVID-19 Convalescents' Immune Response: Nonconserved Epitopes Dominant</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-149', this)">
            More</button></h4>
            <div id="toggle-149" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Nonconserved epitopes dominate reverse preexisting T cell immunity in COVID-19 convalescents</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The abstract talks about how certain immune cells (T cells) in people who had COVID-19 can recognize other coronaviruses like SARS as well. We tested these T cells to see which parts of the viruses they could react with, and found that they mostly reacted to parts of the virus that don't change much between different strains. This means existing antibodies against one type of coronavirus might help protect against others. Our findings could help develop vaccines for multiple types of coronaviruses in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The herd immunity against SARS-CoV-2 is continuously consolidated across the world during the ongoing pandemic. However, the potential function of the nonconserved epitopes in the reverse preexisting cross-reactivity induced by SARS-CoV-2 to other human coronaviruses is not well explored. In our research, we assessed T cell responses to both conserved and nonconserved peptides shared by SARS-CoV-2 and SARS-CoV, identifying cross-reactive CD8+T cell epitopes using enzyme-linked immunospot and intracellular cytokine staining assays. Then, in vitro refolding and circular dichroism were performed to evaluate the thermal stability of the HLA/peptide complexes. Lastly, single-cell T cell receptor reservoir was analyzed based on tetramer staining. Here, we discovered that cross-reactive T cells targeting SARS-CoV were present in individuals who had recovered from COVID-19, and identified SARS-CoV-2 CD8+T cell epitopes spanning the major structural antigens. T cell responses induced by the nonconserved peptides between SARS-CoV-2 and SARS-CoV were higher and played a dominant role in the cross-reactivity in COVID-19 convalescents. Cross-T cell reactivity was also observed within the identified series of CD8+T cell epitopes. For representative immunodominant peptide pairs, although the HLA binding capacities for peptides from SARS-CoV-2 and SARS-CoV were similar, the TCR repertoires recognizing these peptides were distinct. Our results could provide beneficial information for the development of peptide-based universal vaccines against coronaviruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01876-3" target="_blank">https://www.nature.com/articles/s41392-024-01876-3</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-11</strong> <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">
            ü§ñ Antibodies and Immunity After COVID Infection Over Two Years</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-150', this)">
            More</button></h4>
            <div id="toggle-150" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Tracking the evolution of anti-SARS-CoV-2 antibodies and long-term humoral immunity within 2¬†years after COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The virus that causes COVID-19 makes people very sick sometimes. The body's immune system produces special proteins called antibodies to fight the virus. Antibodies can either be made against parts of the virus that cover it, called spike proteins, or against another part inside the virus, called nucleocapsid.

Scientists wanted to understand how long these antibodies last after someone recovers from COVID-19. They looked at people who had recovered and checked their blood over a year-long period.

They found that antibodies against one part of the virus (nucleocapsid) stayed high for about 4 months, but then started going down a bit. Antibodies against another part of the virus (spike) kept increasing every few months until the end of the study.

So, some parts of the body's defense against COVID-19 last longer than others. The scientists learned that certain antibodies stay strong for several months after someone recovers, while other antibodies keep growing stronger over time.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) that gave rise to COVID-19 infection produced a worldwide health crisis. The virus can cause a serious or even fatal disease. Comprehending the complex immunological responses triggered by SARS-CoV-2 infection is essential for identifying pivotal elements that shape the course of the disease and its enduring effects on immunity. The span and potency of antibody responses provide valuable perspicuity into the resilience of post-infection immunity. The analysis of existing literature reveals a diverse controversy, confining varying data about the persistence of particular antibodies as well as the multifaceted factors that impact their development and titer, Within this study we aimed to understand the dynamics of anti-SARS-CoV-2 antibodies against nucleocapsid (anti-SARS-CoV-2 (N)) and spike (anti-SARS-CoV-2 (N)) proteins in long-term immunity in convalescent patients, as well as the factors influencing the production and kinetics of those antibodies. We collected 6115 serum samples from 1611 convalescent patients at different post-infection intervals up to 21¬†months Study showed that in the fourth month, the anti-SARS-CoV-2 (N) exhibited their peak mean value, demonstrating a 79% increase compared to the initial month. Over the subsequent eight months, the peak value experienced a modest decline, maintaining a relatively elevated level by the end of study. Conversely, anti-SARS-CoV-2 (S) exhibited a consistent increase at each three-month interval over the 15-month period, culminating in a statistically significant peak mean value at the study‚Äôs conclusion. Our findings demonstrate evidence of sustained seropositivity rates for both anti-SARS-CoV-2 (N) and (S), as well as distinct dynamics in the long-term antibody responses, with anti-SARS-CoV-2 (N) levels displaying remarkable persistence and anti-SARS-CoV-2 (S) antibodies exhibiting a progressive incline.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-64414-9" target="_blank">https://www.nature.com/articles/s41598-024-64414-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-06-03</strong> <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">
            ü§ñ SARS-CoV-2Êäó‰ΩìIgGÂíåIgAÊ£ÄÊµã‰∏éËΩªÁóáÊñ∞ÂÜ†ÁóáÁä∂ÂÖ≥ËÅî</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-151', this)">
            More</button></h4>
            <div id="toggle-151" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Titers of IgG and IgA against SARS-CoV-2 proteins and their association with symptoms in mild COVID-19 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how antibodies in people with mild COVID-19 disease change over time. Antibodies are like the body's protection against viruses. They can be made into two types: IgG and IgA. The research found that different levels of these antibodies could predict if someone would get worse symptoms from COVID-19, like trouble breathing.

They found that people with more severe symptoms had lower levels of certain antibodies. For example, people who couldn't breathe well had fewer antibodies against some parts of the virus compared to those who didn't have trouble breathing.

Overall, the study suggests that how strong your antibody response is can tell you if you might get sicker from COVID-19 and what part of the virus causes problems for different symptoms.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Humoral immunity in COVID-19 includes antibodies (Abs) targeting spike (S) and nucleocapsid (N) SARS-CoV-2 proteins. Antibody levels are known to correlate with disease severity, but titers are poorly reported in mild or asymptomatic cases. Here, we analyzed the titers of IgA and IgG against SARS-CoV-2 proteins in samples from 200 unvaccinated Hospital Workers (HWs) with mild COVID-19 at two time points after infection. We analyzed the relationship between Ab titers and patient characteristics, clinical features, and evolution over time. Significant differences in IgG and IgA titers against N, S1 and S2 proteins were found when samples were segregated according to time T1 after infection, seroprevalence at T1, sex and age of HWs and symptoms at infection. We found that IgM‚Äâ+‚Äâsamples had higher titers of IgG against N antigen and IgA against S1 and S2 antigens than IgM‚Äâ‚àí‚Äâsamples. There were significant correlations between anti-S1 and S2 Abs. Interestingly, IgM‚Äâ+‚Äâpatients with dyspnea had lower titers of IgG and IgA against N, S1 and S2 than those without dyspnea. Comparing T1 and T2, we found that IgA against N, S1 and S2 but only IgG against certain Ag decreased significantly. In conclusion, an association was established between Ab titers and the development of infection symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-59634-y" target="_blank">https://www.nature.com/articles/s41598-024-59634-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-30</strong> <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">
            ü§ñ Post-COVID Recovery After 3 Years</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-152', this)">
            More</button></h4>
            <div id="toggle-152" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Three-year outcomes of post-acute sequelae of COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The virus that causes COVID-19 can cause long-term effects even after you get better from it. Researchers studied 135,000 people who had COVID and compared them to 5 million others over a 3-year period. They found that while the risk of dying went down for people who didn't need hospital care, those who did still had higher chances of death in the third year after getting sick. 

For hospitalized patients, their long-term risks were even more severe: they still had twice the chance of death compared to healthy people, and there was a lot of health loss that continued into the third year.

Overall, most risks went down over time, but those who were hospitalized had high risks in the third year which led to significant health problems.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection causes post-acute sequelae of coronavirus disease 2019 (COVID-19) (PASC) in many organ systems. Risks of these sequelae have been characterized up to 2‚Äâyears after infection, but longer-term follow-up is limited. Here we built a cohort of 135,161 people with SARS-CoV-2 infection and 5,206,835 controls from the US Department of Veterans Affairs who were followed for 3‚Äâyears to estimate risks of death and PASC. Among non-hospitalized individuals, the increased risk of death was no longer present after the first year of infection, and risk of incident PASC declined over the 3‚Äâyears but still contributed 9.6 (95% confidence interval (CI): 0.4‚Äì18.7) disability-adjusted life years (DALYs) per 1,000 persons in the third year. Among hospitalized individuals, risk of death declined but remained significantly elevated in the third year after infection (incidence rate ratio: 1.29 (95% CI: 1.19‚Äì1.40)). Risk of incident PASC declined over the 3‚Äâyears, but substantial residual risk remained in the third year, leading to 90.0 (95% CI: 55.2‚Äì124.8) DALYs per 1,000 persons. Altogether, our findings show reduction of risks over time, but the burden of mortality and health loss remains in the third year among hospitalized individuals.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41591-024-02987-8" target="_blank">https://www.nature.com/articles/s41591-024-02987-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-29</strong> <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">
            ü§ñ Initial COVID-19 severity linked to SARS-CoV-2 T cell memory impacts future infections.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-153', this)">
            More</button></h4>
            <div id="toggle-153" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Initial COVID-19 severity influenced by SARS-CoV-2-specific T cells imprints T-cell memory and inversely affects reinfection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This research looked at how a person's immune system responds to COVID-19 over time. They studied 165 people who had different levels of sickness from the disease. 

They found that when people get better quickly, their helper and fighter cells (T cells) that fight the virus come early and are strong. This helps them feel better faster in the hospital.

But if T cell responses are weak and take longer to show up, it means someone might end up with a more serious illness later on.

They also noticed how antibodies (which are like shields against viruses) form. If they start forming late or not very much, that's linked to having a worse case of COVID-19.

Overall, the researchers found out that people who had really bad cases of COVID-19 in the beginning tend to have better protection against getting it again later. 

This helps us understand how our immune system fights off COVID-19 and may help prevent more waves of infections in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The immunoprotective components control COVID-19 disease severity, as well as long-term adaptive immunity maintenance and subsequent reinfection risk discrepancies across initial COVID-19 severity, remain unclarified. Here, we longitudinally analyzed SARS-CoV-2-specific immune effectors during the acute infection and convalescent phases of 165 patients with COVID-19 categorized by severity. We found that early and robust SARS-CoV-2-specific CD4+and CD8+T cell responses ameliorate disease progression and shortened hospital stay, while delayed and attenuated virus-specific CD8+T cell responses are prominent severe COVID-19 features. Delayed antiviral antibody generation rather than titer level associates with severe outcomes. Conversely, initial COVID-19 severity imprints the long-term maintenance of SARS-CoV-2-specific adaptive immunity, demonstrating that severe convalescents exhibited more sustained virus-specific antibodies and memory T cell responses compared to mild/moderate counterparts. Moreover, initial COVID-19 severity inversely correlates with SARS-CoV-2 reinfection risk. Overall, our study unravels the complicated interaction between temporal characteristics of virus-specific T cell responses and COVID-19 severity to guide future SARS-CoV-2 wave management.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01867-4" target="_blank">https://www.nature.com/articles/s41392-024-01867-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-27</strong> <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">
            ü§ñ Respiratory issues after COVID infections in South Korea and Japan</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-154', this)">
            More</button></h4>
            <div id="toggle-154" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Acute and post-acute respiratory complications of SARS-CoV-2 infection: population-based cohort study in South Korea and Japan</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how people who get sick from the coronavirus (SARS-CoV-2) can have breathing problems later on. They studied two groups of people: one from South Korea and another from Japan. The researchers wanted to see if these people had more trouble breathing after being infected, compared to others who didn't get infected.

They found that people who got SARS-CoV-2 were more likely to have breathing issues in the short term (acute respiratory complications) or long-term (post-acute respiratory sequelae) compared to those who didn‚Äôt get it. The risk was higher for people with more severe cases of the infection.

The study also showed that getting vaccinated against COVID-19 could help reduce these problems. Over time, even though some breathing issues went away, they still existed up to 6 months after someone got infected.

Overall, this research helps understand how SARS-CoV-2 can affect people's breathing over time and considers things like vaccination status in making this understanding.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Considering the significant burden of post-acute COVID-19 conditions among patients infected with SARS-CoV-2, we aimed to identify the risk of acute respiratory complications or post-acute respiratory sequelae. A binational population-based cohort study was conducted to analyze the risk of acute respiratory complications or post-acute respiratory sequelae after SARS-CoV-2 infection. We used a Korean nationwide claim-based cohort (K-COV-N;n=‚Äâ2,312,748; main cohort) and a Japanese claim-based cohort (JMDC;n=‚Äâ3,115,606; replication cohort) after multi-to-one propensity score matching. Among 2,312,748 Korean participants (mean age, 47.2‚Äâyears [SD, 15.6]; 1,109,708 [48.0%] female), 17.1% (394,598/2,312,748) were infected with SARS-CoV-2. The risk of acute respiratory complications or post-acute respiratory sequelae is significantly increased in people with SARS-CoV-2 infection compared to the general population (acute respiratory complications: HR, 8.06 [95% CI, 6.92-9.38]; post-acute respiratory sequelae: 1.68 [1.62-1.75]), and the risk increased with increasing COVID-19 severity. We identified COVID-19 vaccination as an attenuating factor, showing a protective association against acute or post-acute respiratory conditions. Furthermore, while the excess post-acute risk diminished with time following SARS-CoV-2 infection, it persisted beyond 6‚Äâmonths post-infection. The replication cohort showed a similar pattern in the association. Our study comprehensively evaluates respiratory complications in post-COVID-19 conditions, considering attenuating factors such as vaccination status, post-infection duration, COVID-19 severity, and specific respiratory conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48825-w" target="_blank">https://www.nature.com/articles/s41467-024-48825-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-22</strong> <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">
            ü§ñ Post-vax symptoms after COVID-19 vaccination: study compares infected vs vaccinated people</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-155', this)">
            More</button></h4>
            <div id="toggle-155" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Post-COVID conditions following COVID-19 vaccination: a retrospective matched cohort study of patients with SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at whether getting a COVID-19 vaccine affected people who already had the virus compared to those who didn't have it. They found that vaccinated people were less likely to get certain health problems after having COVID, especially if they got vaccines from outside doctors. The biggest difference was in skin and blood issues. Even though some small differences still existed, getting a vaccine seemed to help reduce these health problems for most people. This means getting the shot might make it easier to avoid long-term side effects of the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>COVID-19 vaccinations protect against severe illness and death, but associations with post-COVID conditions (PCC) are less clear. We aimed to evaluate the association between prior COVID-19 vaccination and new-onset PCC among individuals with SARS-CoV-2 infection across eight large healthcare systems in the United States. This retrospective matched cohort study used electronic health records (EHR) from patients with SARS-CoV-2 positive tests during March 2021-February 2022. Vaccinated and unvaccinated COVID-19 cases were matched on location, test date, severity of acute infection, age, and sex. Vaccination status was ascertained using EHR and integrated data on externally administered vaccines. Adjusted relative risks (RRs) were obtained from Poisson regression. PCC was defined as a new diagnosis in one of 13 PCC categories 30 days to 6 months following a positive SARS-CoV-2 test. The study included 161,531 vaccinated COVID-19 cases and 161,531 matched unvaccinated cases. Compared to unvaccinated cases, vaccinated cases had a similar or lower risk of all PCC categories except mental health disorders (RR: 1.06, 95% CI: 1.02‚Äì1.10). Vaccination was associated with ‚â•10% lower risk of sensory (RR: 0.90, 0.86‚Äì0.95), circulatory (RR: 0.88, 0.83‚Äì0.94), blood and hematologic (RR: 0.79, 0.71‚Äì0.89), skin and subcutaneous (RR: 0.69, 0.66‚Äì0.72), and non-specific COVID-19 related disorders (RR: 0.53, 0.51‚Äì0.56). In general, associations were stronger at younger ages but mostly persisted regardless of SARS-CoV-2 variant period, receipt of ‚â•3 vs. 1‚Äì2 vaccine doses, or time since vaccination. Pre-infection vaccination was associated with reduced risk of several PCC outcomes and hence may decrease the long-term consequences of COVID-19.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48022-9" target="_blank">https://www.nature.com/articles/s41467-024-48022-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-05-16</strong> <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">
            ü§ñ Human Mobility's Impact on City Virus Spread Before and After COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-156', this)">
            More</button></h4>
            <div id="toggle-156" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Impacts of human mobility on the citywide transmission dynamics of 18 respiratory viruses in pre- and post-COVID-19 pandemic years</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Studies looked at how people move around using their phones to see how diseases spread. Before COVID-19, when people visited schools or went to play with kids in their city, they were more likely to get sick from common cold-like viruses that usually stay around all year. After everyone stayed home during the pandemic, these common cold viruses stopped spreading as much.

But after restrictions were lifted a bit, where people moved around also affected how SARS-CoV-2 spread. However, this connection became weaker when people started moving around again after new rules went back into place. 

Overall, changes in how people move seem to matter more for spreading common cold viruses compared to the coronavirus, especially at the start of a disease outbreak.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Many studies have used mobile device location data to model SARS-CoV-2 dynamics, yet relationships between mobility behavior and endemic respiratory pathogens are less understood. We studied the effects of population mobility on the transmission of 17 endemic viruses and SARS-CoV-2 in Seattle over a 4-year period, 2018-2022. Before 2020, visits to schools and daycares, within-city mixing, and visitor inflow preceded or coincided with seasonal outbreaks of endemic viruses. Pathogen circulation dropped substantially after the initiation of COVID-19 stay-at-home orders in March 2020. During this period, mobility was a positive, leading indicator of transmission of all endemic viruses and lagging and negatively correlated with SARS-CoV-2 activity. Mobility was briefly predictive of SARS-CoV-2 transmission when restrictions relaxed but associations weakened in subsequent waves. The rebound of endemic viruses was heterogeneously timed but exhibited stronger, longer-lasting relationships with mobility than SARS-CoV-2. Overall, mobility is most predictive of respiratory virus transmission during periods of dramatic behavioral change and at the beginning of epidemic waves.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-48528-2" target="_blank">https://www.nature.com/articles/s41467-024-48528-2</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-29</strong> <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">
            ü§ñ SARS-CoV-2 Mutations in Lung Transplant Patient with Long COVID</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-157', this)">
            More</button></h4>
            <div id="toggle-157" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamic diversity of SARS-CoV-2 genetic mutations in a lung transplantation patient with persistent COVID-19</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Different types of COVID-19 viruses kept appearing since the pandemic started. Remdesivir is a medicine used to treat these viruses. But it doesn't work well for people whose immune systems are weak, and they can keep getting sick for a long time.

A patient who had trouble breathing because of lung damage caused by another sickness got COVID-19 from the Omicron BA.5 type virus. The virus kept spreading in their body even after treatment with Remdesivir. Scientists found that some changes (mutations) in the virus made it resistant to Remdesivir.

One particular mutation, C799F, was very important because it changed a part of the virus that helps the medicine work. This means the medicine couldn't stop the virus anymore.

The virus kept changing, with lots of different mutations appearing over time. Because of this, some people who are treated don‚Äôt get better and have to deal with resistant viruses that keep them sick for longer.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Numerous SARS-CoV-2 variant strains with altered characteristics have emerged since the onset of the COVID-19 pandemic. Remdesivir (RDV), a ribonucleotide analogue inhibitor of viral RNA polymerase, has become a valuable therapeutic agent. However, immunosuppressed hosts may respond inadequately to RDV and develop chronic persistent infections. A patient with respiratory failure caused by interstitial pneumonia, who had undergone transplantation of the left lung, developed COVID-19 caused by Omicron BA.5 strain with persistent chronic viral shedding, showing viral fusogenicity. Genome-wide sequencing analyses revealed the occurrence of several viral mutations after RDV treatment, followed by dynamic changes in the viral populations. The C799F mutation in nsp12 was found to play a pivotal role in conferring RDV resistance, preventing RDV-triphosphate from entering the active site of RNA-dependent RNA polymerase. The occurrence of diverse mutations is a characteristic of SARS-CoV-2, which mutates frequently. Herein, we describe the clinical case of an immunosuppressed host in whom inadequate treatment resulted in highly diverse SARS-CoV-2 mutations that threatened the patient‚Äôs health due to the development of drug-resistant variants.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47941-x" target="_blank">https://www.nature.com/articles/s41467-024-47941-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-17</strong> <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">
            ü§ñ Long-term recovery from Long COVID: Immune System Restoration 2 Years Later</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-158', this)">
            More</button></h4>
            <div id="toggle-158" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Improvement of immune dysregulation in individuals with long COVID at 24-months following SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looks at how people feel immune-wise after having a mild to moderate case of long COVID compared to those who recovered from regular COVID. At 3 months, people with long COVID had higher antibodies against the virus's shell and core than those who fully recovered. They also had more special cells in their body that help fight off the virus for longer periods at 3 and 8 months.

By 24 months, these differences didn't last. Some people still had some signs of fighting off the infection, which might mean they were re-infected. But overall, both groups ended up with similar immune responses by then.

The study found that people's quality of life improved over time in most cases of long COVID, especially if their levels of a protein called PTX3 or low levels of certain blood markers went down.</p><br>
                <p><strong>Abstract</strong></p>
                <p>This study investigates the humoral and cellular immune responses and health-related quality of life measures in individuals with mild to moderate long COVID (LC) compared to age and gender matched recovered COVID-19 controls (MC) over 24 months. LC participants show elevated nucleocapsid IgG levels at 3 months, and higher neutralizing capacity up to 8 months post-infection. Increased spike-specific and nucleocapsid-specific CD4+T cells, PD-1, and TIM-3 expression on CD4+and CD8+T cells were observed at 3 and 8 months, but these differences do not persist at 24 months. Some LC participants had detectable IFN-Œ≥ and IFN-Œ≤, that was attributed to reinfection and antigen re-exposure. Single-cell RNA sequencing at the 24 month timepoint shows similar immune cell proportions and reconstitution of na√Øve T and B cell subsets in LC and MC. No significant differences in exhaustion scores or antigen-specific T cell clones are observed. These findings suggest resolution of immune activation in LC and return to comparable immune responses between LC and MC over time. Improvement in self-reported health-related quality of life at 24 months was also evident in the majority of LC (62%). PTX3, CRP levels and platelet count are associated with improvements in health-related quality of life.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47720-8" target="_blank">https://www.nature.com/articles/s41467-024-47720-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-09</strong> <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">
            ü§ñ SARS-CoV-2 immunity post-bivalent BA.4/5 vaccine for infected vs. uninfected people</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-159', this)">
            More</button></h4>
            <div id="toggle-159" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2-specific cellular and humoral immunity after bivalent BA.4/5 COVID-19-vaccination in previously infected and non-infected individuals</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how people's bodies respond after getting a special vaccine called bivalent BA.4/5. Some people had already gotten sick with COVID-19 before. The researchers found that those who hadn't been sick before had bigger reactions to the vaccine, making more antibodies and having stronger protection against different types of coronavirus. However, even those who had been sick still had a good response to the vaccine.

The study also showed that people who didn't get sick before were more likely to get infected again after vaccination. This means they had lower levels of protection from their immune system against some specific parts of the virus.

In summary, getting vaccinated can still help protect people from COVID-19 even if they've been sick with it before, but those who haven't been sick might not have as strong a response to certain types of viruses.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Knowledge is limited as to how prior SARS-CoV-2 infection influences cellular and humoral immunity after booster-vaccination with bivalent BA.4/5-adapted mRNA-vaccines, and whether vaccine-induced immunity may indicate subsequent infection. In this observational study, individuals with prior infection (n=‚Äâ64) showed higher vaccine-induced anti-spike IgG-antibodies and neutralizing titers, but the relative increase was significantly higher in non-infected individuals (n=‚Äâ63). In general, both groups showed higher neutralizing activity towards the parental strain than towards Omicron-subvariants BA.1, BA.2 and BA.5. In contrast, CD4 or CD8 T cell levels towards spike from the parental strain and the Omicron-subvariants, and cytokine expression profiles were similar irrespective of prior infection. Breakthrough infections occurred more frequently among previously non-infected individuals, who had significantly lower vaccine-induced spike-specific neutralizing activity and CD4 T cell levels. In summary, we show that immunogenicity after BA.4/5-bivalent vaccination differs between individuals with and without prior infection. Moreover, our results may help to improve prediction of breakthrough infections.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47429-8" target="_blank">https://www.nature.com/articles/s41467-024-47429-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-04-02</strong> <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">
            ü§ñ Allergic Conditions After COVID-19: Studies From 3 Countries</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-160', this)">
            More</button></h4>
            <div id="toggle-160" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Incident allergic diseases in post-COVID-19 condition: multinational cohort studies from South Korea, Japan and the UK</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that people who had COVID-19 were more likely to develop other health problems like asthma and hay fever later on. This risk increased even more if the COVID-19 was very serious. Getting vaccinated against COVID-19 helped lower this risk a bit. The results held true in different countries with different groups of people, which adds credibility to the findings.</p><br>
                <p><strong>Abstract</strong></p>
                <p>As mounting evidence suggests a higher incidence of adverse consequences, such as disruption of the immune system, among patients with a history of COVID-19, we aimed to investigate post-COVID-19 conditions on a comprehensive set of allergic diseases including asthma, allergic rhinitis, atopic dermatitis, and food allergy. We used nationwide claims-based cohorts in South Korea (K-CoV-N;n=‚Äâ836,164; main cohort) and Japan (JMDC;n=‚Äâ2,541,021; replication cohort A) and the UK Biobank cohort (UKB;n=‚Äâ325,843; replication cohort B) after 1:5 propensity score matching. Among the 836,164 individuals in the main cohort (mean age, 50.25 years [SD, 13.86]; 372,914 [44.6%] women), 147,824 were infected with SARS-CoV-2 during the follow-up period (2020‚àí2021). The risk of developing allergic diseases, beyond the first 30 days of diagnosis of COVID-19, significantly increased (HR, 1.20; 95% CI, 1.13‚àí1.27), notably in asthma (HR, 2.25; 95% CI, 1.80‚àí2.83) and allergic rhinitis (HR, 1.23; 95% CI, 1.15‚àí1.32). This risk gradually decreased over time, but it persisted throughout the follow-up period (‚â•6 months). In addition, the risk increased with increasing severity of COVID-19. Notably, COVID-19 vaccination of at least two doses had a protective effect against subsequent allergic diseases (HR, 0.81; 95% CI, 0.68‚àí0.96). Similar findings were reported in the replication cohorts A and B. Although the potential for misclassification of pre-existing allergic conditions as incident diseases remains a limitation, ethnic diversity for evidence of incident allergic diseases in post-COVID-19 condition has been validated by utilizing multinational and independent population-based cohorts.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-47176-w" target="_blank">https://www.nature.com/articles/s41467-024-47176-w</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-28</strong> <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">
            ü§ñ Mpox Infection Trends in Nigeria: Review Analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-161', this)">
            More</button></h4>
            <div id="toggle-161" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Dynamics of Mpox infection in Nigeria: a systematic review and meta-analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Mpox outbreaks happen in parts of West and Central Africa. In Nigeria, many people got sick with it last year. Most cases were from southern Nigeria.

We looked at lots of studies about Mpox in Nigeria over almost 50 years. We found that Mpox spread more to northern parts of the country.

The main reasons why Mpox spreads are poverty, lack of healthcare, and unsafe sexual practices.

Our findings tell us we need to improve healthcare and stop unsafe sex to help stop Mpox spreading.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The seasonal outbreaks of Mpox continue in most parts of West and Central Africa. In the past year, Nigeria had the highest number of reported cases. Here, we used the PRISMA guidelines to carry out a systematic review and meta-analysis of available evidence on Mpox in Nigeria to assess the prevalence, transmission pattern, diagnostic approach, and other associated factors useful for mitigating the transmission of the disease. All relevant observational studies in PubMed/MEDLINE, Embase, AJOL, Web of Science, Scopus and Google Scholar on Mpox in Nigeria were assessed within the last fifty years (1972 to 2022). In all, 92 relevant articles were retrieved, out of which 23 were included in the final qualitative analysis. Notably, most of the cases of Mpox in Nigeria were from the southern part of the country. Our findings showed a progressive spread from the southern to the northern region of the country. We identified the following factors as important in the transmission of Mpox in Nigeria; poverty, lack of basic healthcare facilities, and risk of exposure through unsafe sexual practices. Our findings reiterate the need to strengthen and expand existing efforts as well as establish robust multi-sectoral collaboration to understand the dynamics of Mpox Nigeria.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-58147-y" target="_blank">https://www.nature.com/articles/s41598-024-58147-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">
            ü§ñ Molecular Vaccine Effects & Fever: First Four Doses on Adults</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-162', this)">
            More</button></h4>
            <div id="toggle-162" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Factors impacting antibody kinetics, including fever and vaccination intervals, in SARS-CoV-2-na√Øve adults receiving the first four mRNA COVID-19 vaccine doses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how a series of vaccine shots affects the body's ability to make antibodies against COVID-19 in people who have never had it before. They gave different groups of people either one, two, three, or four shots. Blood samples were taken from them both before and after each shot was given.

They found that:
- People who got the mRNA-1273 vaccine made more antibodies.
- Men made more antibodies than women.
- Younger people made more antibodies than older ones.
- People who didn't smoke had higher antibody levels.
- Those with no medicines to suppress their immune system also had higher levels.

They noticed that:
- The longer it was since they got their last shot, the more antibodies they made.
- Getting a second shot after an initial one always resulted in better results.

These findings can help make vaccination plans better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>To evaluate the antibody response following the initial four doses of mRNA vaccines (BNT162b2 or mRNA-1273) in SARS-CoV-2-na√Øve healthy adults and investigate factors influencing antibody titer increases, this prospective cohort study was conducted in Japan from March 2021. The study included participants who received either the 1st and 2nd doses (n‚Äâ=‚Äâ467), 3rd dose (n‚Äâ=‚Äâ157), or 4th dose (n‚Äâ=‚Äâ89). Blood samples were collected before and up to 6¬†months after each dose, and anti-receptor-binding domain antibody levels were measured. Multivariate analysis (usin multiple linear regression or linear mixed models) revealed several factors significantly associated with higher post-vaccination antibody levels, including mRNA-1273 vaccine (after the 1st and 2nd dose), male gender (after the 3rd and 4th doses), younger age (after the 1st and 2nd dose), non-smoking status (after the 2nd dose), non-use of immunosuppressive agents (after the 1st dose), higher pre-vaccination antibody titers (after the 2nd, 3rd, and 4th doses), and higher post-vaccination fever (after the 2nd and 4th doses). Furthermore, longer intervals since the last dose were significantly associated with higher antibody levels after the 3rd and 4th doses. These findings provide valuable insights for optimizing vaccination strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57931-0" target="_blank">https://www.nature.com/articles/s41598-024-57931-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-27</strong> <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">
            ü§ñ Rapid double-shot vaccine against poxviruses</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-163', this)">
            More</button></h4>
            <div id="toggle-163" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Rapid development of double-hit mRNA antibody cocktail against orthopoxviruses</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This abstract talks about making special medicine to fight poxviruses, which can make people very sick. Right now, it's hard and expensive to create these medicines. But scientists found a new way using tiny messenger molecules called mRNA, put inside little fat particles called liposomes. They made different mixtures of these mRNA messages that tell the body how to make special antibodies that fight poxviruses. When they tested this on mice, the mice didn't get sick as badly when given these mRNA messages. One mixture of these messages helped protect against all types of poxviruses. The scientists think this could be a good way to create medicines for other viruses too.</p><br>
                <p><strong>Abstract</strong></p>
                <p>TheOrthopoxvirusgenus, especially variola virus (VARV), monkeypox virus (MPXV), remains a significant public health threat worldwide. The development of therapeutic antibodies against orthopoxviruses is largely hampered by the high cost of antibody engineering and manufacturing processes. mRNA-encoded antibodies have emerged as a powerful and universal platform for rapid antibody production. Herein, by using the established lipid nanoparticle (LNP)-encapsulated mRNA platform, we constructed four mRNA combinations that encode monoclonal antibodies with broad neutralization activities against orthopoxviruses. In vivo characterization demonstrated that a single intravenous injection of each LNP-encapsulated mRNA antibody in mice resulted in the rapid production of neutralizing antibodies. More importantly, mRNA antibody treatments showed significant protection from weight loss and mortality in the vaccinia virus (VACV) lethal challenge mouse model, and a unique mRNA antibody cocktail, Mix2a, exhibited superior in vivo protection by targeting both intracellular mature virus (IMV)-form and extracellular enveloped virus (EEV)-form viruses. In summary, our results demonstrate the proof-of-concept production of orthopoxvirus antibodies via the LNP-mRNA platform, highlighting the great potential of tailored mRNA antibody combinations as a universal strategy to combat orthopoxvirus as well as other emerging viruses.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41392-024-01766-8" target="_blank">https://www.nature.com/articles/s41392-024-01766-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-21</strong> <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">
            ü§ñ SARS-CoV-2 Infection Risk for Psychiatric Disorders & Medication Use</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-164', this)">
            More</button></h4>
            <div id="toggle-164" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Long-term risk of psychiatric disorder and psychotropic prescription after SARS-CoV-2 infection among UK general population</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Despite some people getting sick from COVID-19, they might later have problems like anxiety or mood issues. Studies showed that people who got infected had a higher chance of developing these mental health issues compared to those who didn't get infected. People who were hospitalized were even more likely to develop these problems.

Vaccines seemed to help protect against some of these mental health issues, though not completely. Getting the vaccine still meant there was a risk, just less than for people who weren‚Äôt vaccinated at all or only got part of the vaccine.

It's important to catch and treat mental health issues early in COVID survivors, especially those who were very sick or didn't get vaccinated. As more people get infected without getting vaccinated (called "breakthrough infections"), we need better ways to prevent serious mental health problems from developing.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Despite evidence indicating increased risk of psychiatric issues among COVID-19 survivors, questions persist about long-term mental health outcomes and the protective effect of vaccination. Using UK Biobank data, three cohorts were constructed: SARS-CoV-2 infection (n=‚Äâ26,101), contemporary control with no evidence of infection (n=‚Äâ380,337) and historical control predating the pandemic (n=‚Äâ390,621). Compared with contemporary controls, infected participants had higher subsequent risks of incident mental health at 1‚Äâyear (hazard ratio (HR): 1.54, 95% CI 1.42‚Äì1.67;P=‚Äâ1.70‚Äâ√ó‚Äâ10‚àí24; difference in incidence rate: 27.36, 95% CI 21.16‚Äì34.10 per 1,000 person-years), including psychotic, mood, anxiety, alcohol use and sleep disorders, and prescriptions for antipsychotics, antidepressants, benzodiazepines, mood stabilizers and opioids. Risks were higher for hospitalized individuals (2.17, 1.70‚Äì2.78;P=‚Äâ5.80‚Äâ√ó‚Äâ10‚àí10) than those not hospitalized (1.41, 1.30‚Äì1.53;P=‚Äâ1.46‚Äâ√ó‚Äâ10‚àí16), and were reduced in fully vaccinated people (0.97, 0.80‚Äì1.19;P=‚Äâ0.799) compared with non-vaccinated or partially vaccinated individuals (1.64, 1.49‚Äì1.79;P=‚Äâ4.95‚Äâ√ó‚Äâ10‚àí26). Breakthrough infections showed similar risk of psychiatric diagnosis (0.91, 0.78‚Äì1.07;P=‚Äâ0.278) but increased prescription risk (1.42, 1.00‚Äì2.02;P=‚Äâ0.053) compared with uninfected controls. Early identification and treatment of psychiatric disorders in COVID-19 survivors, especially those severely affected or unvaccinated, should be a priority in the management of long COVID. With the accumulation of breakthrough infections in the post-pandemic era, the findings highlight the need for continued optimization of strategies to foster resilience and prevent escalation of subclinical mental health symptoms to severe disorders.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41562-024-01853-4" target="_blank">https://www.nature.com/articles/s41562-024-01853-4</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-18</strong> <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">
            ü§ñ Neurofilament levels high in mild COVID patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-165', this)">
            More</button></h4>
            <div id="toggle-165" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neurofilament light chain and glial fibrillary acid protein levels are elevated in post-mild COVID-19 or asymptomatic SARS-CoV-2 cases</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The COVID-19 pandemic caused brain problems in some people who had mild or no symptoms before. To see how well their brains were working now that the virus was gone, researchers looked at two substances in their blood: NF-L and GFAP. These substances can show if someone's brain cells are getting damaged.

They studied 147 grown-ups who used to have COVID-19 but didn't know they did or had very mild symptoms. They also checked some people without any illness nearby, like how the grown-up group is similar to them in age and body size. 

The researchers found that the grown-ups with brain problems had more of these substances in their blood compared to those who were healthy. Also, after checking again after about a year, even though they still had higher levels, it was less than what healthy people have.

This shows that sometimes, when the virus is gone, parts of someone's brain can be damaged, but over time, this damage gets better.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Given the huge impact of the COVID-19 pandemic, it appears of paramount importance to assess the cognitive effects on the population returning to work after COVID-19 resolution. Serum levels of neurofilament light chain (sNfL) and glial fibrillary acidic protein (sGFAP) represent promising biomarkers of neuro-axonal damage and astrocytic activation. In this cohort study, we explored the association between sNfL and sGFAP concentrations and cognitive performance in a group of 147 adult workers with a previous asymptomatic SARS-CoV-2 infection or mild COVID-19, one week and, in 49 of them, ten months after SARS-Cov2 negativization and compared them to a group of 82 age and BMI-matched healthy controls (HCs). sNfL and sGFAP concentrations were assessed using SimoaTM assay Neurology 2-Plex B Kit. COVID-19 patients were interviewed one-on-one by trained physicians and had to complete a list of questionnaires, including the Cognitive Failure Questionnaire (CFQ). At the first assessment (T0), sNfL and sGFAP levels were significantly higher in COVID-19 patients than in HCs (p‚Äâ<‚Äâ0.001 for both). The eleven COVID-19 patients with cognitive impairment had significantly higher levels of sNfL and sGFAP than the others (p‚Äâ=‚Äâ0.005 for both). At the subsequent follow-up (T1), sNfL and sGFAP levels showed a significant decrease (median sNfL 18.3¬†pg/mL; median sGFAP 77.2¬†pg/mL), although they were still higher than HCs (median sNfL 7.2¬†pg/mL, median sGFAP 63.5¬†pg/mL). Our results suggest an ongoing damage involving neurons and astrocytes after SARS-Cov2 negativization, which reduce after ten months even if still evident compared to HCs.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-57093-z" target="_blank">https://www.nature.com/articles/s41598-024-57093-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-03-05</strong> <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">
            ü§ñ SARS-CoV-2 viability in air from COVID patients on different variants</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-166', this)">
            More</button></h4>
            <div id="toggle-166" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Frequency, kinetics and determinants of viable SARS-CoV-2 in bioaerosols from ambulatory COVID-19 patients infected with the Beta, Delta or Omicron variants</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The abstract talks about whether tiny bits of air carrying coronavirus (called aerosols) can still make people sick. Some researchers think these bits are dangerous, but others aren't sure. So we did tests to see if these aerosols could have the virus that causes COVID-19 and how many people had them.

We found out that for some types of test results from people who had the virus, about half or more of those people were likely spreading the virus in their air. People with lower protection against the virus in their bodies seemed to be better at making these spreadable bits of air.

The study also found a quick way to guess whether someone is likely to make these dangerous bits of air - just by looking at some parts of their blood. 

What this means is that we now know more about how and who spreads coronavirus through the air, which can help us stop it better in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Airborne transmission of SARS-CoV-2 aerosol remains contentious. Importantly, whether cough or breath-generated bioaerosols can harbor viable and replicating virus remains largely unclarified. We performed size-fractionated aerosol sampling (Andersen cascade impactor) and evaluated viral culturability in human cell lines (infectiousness), viral genetics, and host immunity in ambulatory participants with COVID-19. Sixty-one percent (27/44) and 50% (22/44) of participants emitted variant-specific culture-positive aerosols <10Œºm and <5Œºm, respectively, for up to 9 days after symptom onset. Aerosol culturability is significantly associated with lower neutralizing antibody titers, and suppression of transcriptomic pathways related to innate immunity and the humoral response. A nasopharyngeal Ct <17 rules-in ~40% of aerosol culture-positives and identifies those who are probably highly infectious. A parsimonious three transcript blood-based biosignature is highly predictive of infectious aerosol generation (PPV‚Äâ>‚Äâ95%). There is considerable heterogeneity in potential infectiousness i.e., only 29% of participants were probably highly infectious (produced culture-positive aerosols <5Œºm at ~6 days after symptom onset). These data, which comprehensively confirm variant-specific culturable SARS-CoV-2 in aerosol, inform the targeting of transmission-related interventions and public health containment strategies emphasizing improved ventilation.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45400-1" target="_blank">https://www.nature.com/articles/s41467-024-45400-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-26</strong> <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">
            ü§ñ Long-term Health Effects After COVID-19 Vaccination</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-167', this)">
            More</button></h4>
            <div id="toggle-167" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Persistence in risk and effect of COVID-19 vaccination on long-term health consequences after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at people who got infected with the coronavirus (SARS-Co-V-2) and compared them to those who didn't get infected, focusing on how vaccination affects their health over time in Hong Kong. They found that getting vaccinated or having a booster shot reduces the risk of long-term health problems like heart issues and death after being sick from COVID for about a year. Unvaccinated people had higher risks even months later. The study shows vaccines help prevent serious health effects from the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The persisting risk of long-term health consequences of SARS-CoV-2 infection and the protection against such risk conferred by COVID-19 vaccination remains unclear. Here we conducted a retrospective territory-wide cohort study on 1,175,277 patients with SARS-CoV-2 infection stratified by their vaccination status and non-infected controls to evaluate the risk of clinical sequelae, cardiovascular and all-cause mortality using a territory-wide public healthcare database with population-based vaccination records in Hong Kong. A progressive reduction in risk of all-cause mortality was observed over one year between patients with SARS-CoV-2 infection and controls. Patients with complete vaccination or have received booster dose incurred a lower risk of health consequences including major cardiovascular diseases, and all-cause mortality than unvaccinated or patients with incomplete vaccination 30-90 days after infection. Completely vaccinated and patients with booster dose of vaccines did not incur significant higher risk of health consequences from 271 and 91 days of infection onwards, respectively, whilst un-vaccinated and incompletely vaccinated patients continued to incur a greater risk of clinical sequelae for up to a year following SARS-CoV-2 infection. This study provided real-world evidence supporting the effectiveness of COVID-19 vaccines in reducing the risk of long-term health consequences of SARS-CoV-2 infection and its persistence following infection.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-024-45953-1" target="_blank">https://www.nature.com/articles/s41467-024-45953-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-14</strong> <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">
            ü§ñ SARS-CoV-2 in corneas post-transplants vs. COVID-19 infection rates</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-168', this)">
            More</button></h4>
            <div id="toggle-168" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 infection status in corneal preservation solution and COVID-19 prevalence after corneal transplantation</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that corneas from COVID-19-infected donors did not have the virus inside them. They tested 144 cornea transplants and found no signs of COVID-19 infection. All the patients who got new corneas were healthy after surgery, even though they had to take steroid medicine for a long time. The research suggests testing for COVID-19 in donors' throats before giving them corneas can help prevent infections in people who get new eyes.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The potential risks associated with organs from COVID-19-infected donors were unclear. To determine the SARS-CoV-2 infection status of corneas transplanted during the COVID-19 pandemic, we performed a polymerase chain reaction (PCR) using the corneal preservation solution that was used for corneal transplantation. We also examined the postoperative health status of the recipients. This study included 144 transplants in 143 eyes. Ninety-nine eyes of imported corneas and 10 of the 14 corneas donated in the prefecture were PCR tested at our hospital, and all were SARS-CoV-2 negative. All corneal transplants were performed after confirming their SARS-CoV-2 negativity by a PCR using a corneal preservation solution at our hospital or a nasopharyngeal swab at a previous facility. Despite postoperative steroid administration, no patient developed COVID-19 infection until discharge. Hence, if the donor's nasopharyngeal swab test is SARS-CoV-2 negative, COVID-19 infection in the recipient due to corneal transplantation may be prevented. Since corneal transplant recipients are susceptible to infection due to prolonged steroid administration and are at high risk for severe diseases if infection occurs, SARS-CoV-2 detection testing using nasopharyngeal swabs in donors should be performed.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-024-53863-x" target="_blank">https://www.nature.com/articles/s41598-024-53863-x</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-02-05</strong> <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">
            ü§ñ SARS-CoV-2 vaccine could reduce COVID-19 lingering gut issues</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-169', this)">
            More</button></h4>
            <div id="toggle-169" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>SARS-CoV-2 vaccination may mitigate dysregulation of IL-1/IL-18 and gastrointestinal symptoms of the post-COVID-19 condition</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The vaccine helped prevent serious illness from the coronavirus. Some people who had the virus still get sick after recovery, called post-COVID conditions (PCC). Researchers looked at how well the vaccine worked for those with PCC and found that getting vaccinated might help reduce certain symptoms like stomach problems.

People who didn't have any of these issues before vaccination showed no signs of improvement. But people with PCC who got vaccinated had lower levels of two proteins in their blood, which might mean fewer stomach troubles.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The rapid development of safe and effective vaccines helped to prevent severe disease courses after SARS-CoV-2 infection and to mitigate the progression of the COVID-19 pandemic. While there is evidence that vaccination may reduce the risk of developing post-COVID-19 conditions (PCC), this effect may depend on the viral variant. Therapeutic effects of post-infection vaccination have been discussed but the data for individuals with PCC remains inconclusive. In addition, extremely rare side effects after SARS-CoV-2 vaccination may resemble the heterogeneous PCC phenotype. Here, we analyze the plasma levels of 25 cytokines and SARS-CoV-2 directed antibodies in 540 individuals with or without PCC relative to one or two mRNA-based COVID-19 vaccinations as well as in 20 uninfected individuals one month after their initial mRNA-based COVID-19 vaccination. While none of the SARS-CoV-2 na√Øve individuals reported any persisting sequelae or exhibited PCC-like dysregulation of plasma cytokines, we detected lower levels of IL-1Œ≤ and IL-18 in patients with ongoing PCC who received one or two vaccinations at a median of six months after infection as compared to unvaccinated PCC patients. This reduction correlated with less frequent reporting of persisting gastrointestinal symptoms. These data suggest that post-infection vaccination in patients with PCC might be beneficial in a subgroup of individuals displaying gastrointestinal symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-024-00815-1" target="_blank">https://www.nature.com/articles/s41541-024-00815-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-24</strong> <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">
            ü§ñ Respiratory issues in newborns not infected by coronavirus observed in study.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-170', this)">
            More</button></h4>
            <div id="toggle-170" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Respiratory distress in SARS-CoV-2 exposed uninfected neonates followed in the COVID Outcomes in Mother-Infant Pairs (COMP) Study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that babies exposed in the womb to someone sick with coronavirus had more breathing problems, especially if their mom wasn't vaccinated. The researchers looked at moms who got infected while pregnant and compared them to moms who were vaccinated before getting sick. They also checked how the baby's body reacted to the virus. Babies without a vaccine from their mom had twice as much trouble breathing than babies whose moms got vaccinated first. This was because of problems with tiny hair-like things in the lungs that help move air, and the baby making more antibodies (IgE) which can sometimes cause inflammation. Vaccinating moms before they get infected seems to lower how often this happens in their babies.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Respiratory distress (RD) has been reported in SARS-CoV-2 exposed uninfected (SEU) term neonates. Prior studies suggest that prenatal exposure to Coronavirus Disease 19 (COVID-19) may activate an inflammatory cascade in the newborn airway. In this study, we examine the relationship between maternal COVID-19 vaccination and neonatal RD using a longitudinal cohort of mother-infant pairs in Los Angeles, CA. Two-hundred and twenty-one mothers with laboratory confirmed SARS-CoV-2 during pregnancy and 227 exposed fetuses are enrolled in our study. Maternal disease severity and neonatal RD variables were defined based on current accepted clinical criteria. To explore the multifactorial associations between maternal COVID-19 parameters and infant RD, we utilize a multivariable logistic regression model and a proteomic sub-analysis to propose a pathway for the development of RD followingin uteroexposure to SARS-CoV-2. Unusually high rates of RD are observed in SEU infants (17%). The odds ratio of RD is 3.06 (95% CI:1.08-10.21) in term neonates born to unvaccinated individuals versus those born to individuals vaccinated prior to maternal infection. Proteomic analysis reveals a robust inflammatory response associated with ciliary dysregulation and enhanced IgE production among SEU neonates with RD. Maternal vaccination against COVID-19 reduces the frequency of neonatal RD.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44549-5" target="_blank">https://www.nature.com/articles/s41467-023-44549-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-08</strong> <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">
            ü§ñ Heritability of SARS-CoV-2 Susceptibility and Severity</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-171', this)">
            More</button></h4>
            <div id="toggle-171" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Estimating the heritability of SARS-CoV-2 susceptibility and COVID-19 severity</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>SARS-CoV-2 has infected over 340 million people. Scientists want to find ways to treat COVID-19. Genetic studies can show where drugs might work better. But we also need to understand if some people are more likely to get sick or have worse symptoms because of their genes.

Research shows that genetics explain about 1.2% and 5.8% of why some people are more susceptible (can catch it easily) or have a worse condition when they do get sick. Some research used families' genetic information to look at these numbers better. They found that for susceptibility, the genes explained between 33% and 70% of cases. For how long someone is hospitalized with COVID-19, genetics accounted for about 41%.

Scientists think both genes and things in their environment (like if they had a clean place to live) help cause these differences. But what people inherited from their parents seems to matter more when it comes to getting sick.

During the pandemic, as vaccines came out, how much of why some people got sick was due to genetics changed. It's hard for scientists to understand all this during such big changes in health habits and medical care.</p><br>
                <p><strong>Abstract</strong></p>
                <p>SARS-CoV-2 has infected over 340 million people, prompting therapeutic research. While genetic studies can highlight potential drug targets, understanding the heritability of SARS-CoV-2 susceptibility and COVID-19 severity can contextualize their results. To date, loci from meta-analyses explain 1.2% and 5.8%¬†of variation in susceptibility and severity¬†respectively. Here we estimate the importance of shared environment and additive genetic variation to SARS-CoV-2 susceptibility and COVID-19 severity using pedigree data, PCR results, and hospitalization information. The relative importance of genetics and shared environment for susceptibility shifted during the study, with heritability ranging from 33% (95% CI: 20%-46%) to 70% (95% CI: 63%-74%). Heritability was greater for days hospitalized with COVID-19 (41%, 95% CI: 33%-57%) compared to shared environment (33%, 95% CI: 24%-38%). While our estimates suggest these genetic architectures are not fully understood, the shift in susceptibility estimates highlights the challenge of estimation during a pandemic, given environmental fluctuations and vaccine introduction.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44250-7" target="_blank">https://www.nature.com/articles/s41467-023-44250-7</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-03</strong> <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">
            ü§ñ COVID recovery symptoms and their causes: IMPACC study findings</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-172', this)">
            More</button></h4>
            <div id="toggle-172" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Features of acute COVID-19 associated with post-acute sequelae of SARS-CoV-2 phenotypes: results from the IMPACC study</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-COVID-19 syndrome is a big health issue. We studied 590 people who got sick from COVID-19 after they left the hospital, tracking their recovery for a year.

We found four groups based on how badly different parts of their bodies were affected:
1. Minimal: Not much hurt.
2. Physical: Lots of trouble moving around and feeling tired.
3. Mental/Cognitive: Problems with thinking and memory.
4. Multidomain: A mix of all the above problems.

Women got more of these problems, and some people had other health issues too. 

When people were first sick:
- If they had lots of the virus in their lungs,
- If they didn't have good antibodies to fight it,
- Or if they had fewer special immune cells called B lymphocytes,

They often ended up in one of those groups.

People who had problems with thinking and memory had higher levels of a chemical called FGF21 in their blood. Those with physical issues did too, but also had higher levels of FGF21 overall.

This information could help doctors figure out better ways to treat or prevent Post-COVID-19 syndrome in the future.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sequelae of SARS-CoV-2 (PASC) is a significant public health concern. We describe Patient Reported Outcomes (PROs) on 590 participants prospectively assessed from hospital admission for COVID-19 through one year after discharge. Modeling identified 4 PRO clusters based on reported deficits (minimal, physical, mental/cognitive, and multidomain), supporting heterogenous clinical presentations in PASC, with sub-phenotypes associated with female sex and distinctive comorbidities. During the acute phase of disease, a higher respiratory SARS-CoV-2 viral burden and lower Receptor Binding Domain and Spike antibody titers were associated with both the physical predominant and the multidomain deficit clusters. A lower frequency of circulating B lymphocytes by mass cytometry (CyTOF) was observed in the multidomain deficit cluster. Circulating fibroblast growth factor 21 (FGF21) was significantly elevated in the mental/cognitive predominant and the multidomain clusters. Future efforts to link PASC to acute anti-viral host responses may help to better target treatment and prevention of PASC.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44090-5" target="_blank">https://www.nature.com/articles/s41467-023-44090-5</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2024-01-02</strong> <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">
            ü§ñ Anti-viral hosts for SARS-CoV-2 identified through multi-omics analysis</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-173', this)">
            More</button></h4>
            <div id="toggle-173" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Integrated multi-omics analyses identify anti-viral host factors and pathways controlling SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study found many previously unknown ways that a person's body can fight off the coronavirus (SARS-CoV-2). They used experiments with CRISPR, which is like a gene scissors, to look at what parts of our bodies are important for fighting viruses. They looked at how different cells work together and interact with viruses.

They discovered several new parts of our body that help stop the virus from entering or growing inside us. These include parts of the V-ATPases, ESCRT, N-glycosylation pathways, which are like different tools in a toolbox for fighting off viruses.

They also found out about an important group called cohesin complex and another part called KLF5 that helps fight the virus. The KLF5 is special because it's related to something we eat - things called sphingolipids.

The study showed how removing these parts can make cells more vulnerable to getting sick from COVID-19, especially in severe cases. This research could help find new ways to protect people from getting worse with the virus.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Host anti-viral factors are essential for controlling SARS-CoV-2 infection but remain largely unknown due to the biases of previous large-scale studies toward pro-viral host factors. To fill in this knowledge gap, we perform a genome-wide CRISPR dropout screen and integrate analyses of the multi-omics data of the CRISPR screen, genome-wide association studies, single-cell RNA-Seq, and host-virus proteins or protein/RNA interactome. This study uncovers many host factors that are currently underappreciated, including the components of V-ATPases, ESCRT, and N-glycosylation pathways that modulate viral entry and/or replication. The cohesin complex is also identified as an anti-viral pathway, suggesting an important role of three-dimensional chromatin organization in mediating host-viral interaction. Furthermore, we discover another anti-viral regulator KLF5, a transcriptional factor involved in sphingolipid metabolism, which is up-regulated, and harbors genetic variations linked to COVID-19 patients with severe symptoms. Anti-viral effects of three identified candidates (DAZAP2/VTA1/KLF5) are confirmed individually. Molecular characterization of DAZAP2/VTA1/KLF5-knockout cells highlights the involvement of genes related to the coagulation system in determining the severity of COVID-19. Together, our results provide further resources for understanding the host anti-viral network during SARS-CoV-2 infection and may help develop new countermeasure strategies.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-44175-1" target="_blank">https://www.nature.com/articles/s41467-023-44175-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">
            ü§ñ SARS-CoV-2 Variants Neutralized in Children After VaccinesÊàñInfections</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-174', this)">
            More</button></h4>
            <div id="toggle-174" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Neutralization of SARS-CoV-2 Omicron BQ.1, BQ.1.1 and XBB.1 variants following SARS-CoV-2 infection or vaccination in children</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study looked at how well different vaccines work against new coronavirus strains in children. They found that after getting infected or vaccinated, kids don't make strong enough antibodies to fight off some of the new Omicron subtypes like BQ.1, BQ.1.1, and XBB.1. This means current vaccines might not be as effective against these variants. The research suggests we need better vaccines that can protect children from these new strains more effectively.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Emergence of highly transmissible Omicron subvariants led to increased SARS-CoV-2 infection and disease in children. However, minimal knowledge exists regarding the neutralization capacity against circulating Omicron BA.4/BA.5, BA.2.75, BQ.1, BQ.1.1 and XBB.1 subvariants following SARS-CoV-2 vaccination in children versus during acute or convalescent COVID-19, or versus multisystem inflammatory syndrome (MIS-C). Here, we evaluate virus-neutralizing capacity against SARS-CoV-2 variants in 151 age-stratified children (‚Äâ<5, 5‚Äì11, 12‚Äì21 years old) hospitalized with acute severe COVID-19 or MIS-C or convalescent mild (outpatient) infection compared with 62 age-stratified vaccinated children. An age-associated effect on neutralizing antibodies is observed against SARS-CoV-2 following acute COVID-19 or vaccination. The primary series BNT162b2 mRNA vaccinated adolescents show higher vaccine-homologous WA-1 neutralizing titers compared with <12 years vaccinated children. Post-infection antibodies did not neutralize BQ.1, BQ.1.1 and XBB.1 subvariants. In contrast, monovalent mRNA vaccination induces more cross-neutralizing antibodies in young children <5 years against BQ.1, BQ.1.1 and XBB.1 variants compared with ‚â•5 years old children. Our study demonstrates that in children, infection and monovalent vaccination-induced neutralization activity is low against BQ.1, BQ.1.1 and XBB.1 variants. These findings suggest a need for improved SARS-CoV-2 vaccines to induce durable, more cross-reactive neutralizing antibodies to provide effective protection against emerging variants in children.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-43152-y" target="_blank">https://www.nature.com/articles/s41467-023-43152-y</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-12-01</strong> <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">
            ü§ñ SARS-CoV-2 in breath can spread COVID-19 quickly</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-175', this)">
            More</button></h4>
            <div id="toggle-175" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Infectivity of exhaled SARS-CoV-2 aerosols is sufficient to transmit covid-19 within minutes</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The study found that when people start feeling sick with COVID-19, tiny droplets of their breath (called aerosols) carry the virus. These droplets are released into the air and spread the disease very easily. The researchers measured how many of these droplets contain enough of the virus to make someone sick.

They looked at three different people and found that singing releases the most droplets with the virus inside them. One person coughed out 4 TCID50 per second, another one exhaled 36 per second, and the third person let out 127 per second. 

The researchers used a computer model to guess what would happen if someone who had just caught COVID-19 came into a room where people were walking around normally. They found that in about 6 minutes to less than an hour, someone could breathe in enough droplets carrying the virus to become sick themselves.

So basically, the study shows that when someone with COVID-19 is singing or breathing heavily, they can spread the disease very quickly and easily in a regular room where people are moving around.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Exhaled SARS-CoV-2-containing aerosols contributed significantly to the rapid and vast spread of covid-19. However, quantitative experimental data on the infectivity of such aerosols is missing. Here, we quantified emission rates of infectious viruses in exhaled aerosol from individuals within their first days after symptom onset from covid-19. Six aerosol samples from three individuals were culturable, of which five were successfully quantified using TCID50. The source strength of the three individuals was highest during singing, when they exhaled 4, 36, or 127 TCID50/s, respectively. Calculations with an indoor air transmission model showed that if an infected individual with this emission rate entered a room, a susceptible person would inhale an infectious dose within 6 to 37¬†min in a room with normal ventilation. Thus, our data show that exhaled aerosols from a single person can transmit covid-19 to others within minutes at normal indoor conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-47829-8" target="_blank">https://www.nature.com/articles/s41598-023-47829-8</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-11-17</strong> <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">
            ü§ñ Post-Acute COVID Symptoms in Multinational Cohort</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-176', this)">
            More</button></h4>
            <div id="toggle-176" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>The burden of post-acute COVID-19 symptoms in a multinational network cohort analysis</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at people who had gotten sick from COVID-19 and how many still felt bad for months afterward. They studied lots of people in Spain and the UK.

They found that as time went on, fewer people were feeling sick after their initial illness. But even those with only one case of COVID often had trouble with smell and taste for a long time.

People who got sick again (re-infected) had more lingering symptoms than those who got only one infection. 

The researchers think there might be different ways to define "long COVID" and they suggest people need more research to find better treatments.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Persistent symptoms following the acute phase of COVID-19 present a major burden to both the affected and the wider community. We conducted a cohort study including over 856,840 first COVID-19 cases, 72,422 re-infections and more than 3.1 million first negative-test controls from primary care electronic health records from Spain and the UK (Sept 2020 to Jan 2022 (UK)/March 2022 (Spain)). We characterised post-acute COVID-19 symptoms and identified key symptoms associated with persistent disease. We estimated incidence rates of persisting symptoms in the general population and among COVID-19 patients over time. Subsequently, we investigated which WHO-listed symptoms were particularly differential by comparing their frequency in COVID-19 cases vs. matched test-negative controls. Lastly, we compared persistent symptoms after first infections vs. reinfections.Our study shows that the proportion of COVID-19 cases affected by persistent post-acute COVID-19 symptoms declined over the study period. Risk for altered smell/taste was consistently higher in patients with COVID-19 vs test-negative controls. Persistent symptoms were more common after reinfection than following a first infection. More research is needed into the definition of long COVID, and the effect of interventions to minimise the risk and impact of persistent symptoms.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42726-0" target="_blank">https://www.nature.com/articles/s41467-023-42726-0</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-07</strong> <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">
            ü§ñ Covid-19 and Post-Acute Sick Leave Study in Denmark</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-177', this)">
            More</button></h4>
            <div id="toggle-177" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>A hybrid register and questionnaire study of Covid-19 and post-acute sick leave in Denmark</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>Post-acute sick leave means how people feel sick after they recover from the flu-like illness caused by a virus called SARS-CoV-2. We did a survey where many people said if they had to stay home because they were very sick. The study found that more women, especially those in their 50s and 60s, as well as those with certain health problems like lung issues or being extra heavy (obesity), took longer to get back to feeling normal after the flu-like illness. These findings might help doctors make better ways to diagnose and treat people who still feel sick months later.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Post-acute sick leave is an underexplored indicator of the societal burden of SARS-CoV-2. Here, ¬†we report findings about self-reported sick leave and risk factors thereof from a hybrid survey and register study, which include 37,482 RT-PCR confirmed SARS-CoV-2 cases and 51,336 test-negative controls who were tested during the index- and alpha-dominant waves. We observe that an additional 33 individuals per 1000 took substantial sick leave following acute infection compared to persons with no known history of infection, where substantial sick leave is defined as >1 month of sick leave within the period 1‚Äì9 months after the RT-PCR test date. Being female, 50‚Äì65 years, or having certain pre-existing health conditions such as obesity, chronic lung diseases, and fibromyalgia each increase risk for taking substantial sick leave. Altogether, these results may help motivate improved diagnostic and treatment options for persons living with post-Covid conditions.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-42048-1" target="_blank">https://www.nature.com/articles/s41467-023-42048-1</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-06</strong> <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">
            ü§ñ COVID-19 and SARS-CoV-2 risk for eye vein blockages.</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-178', this)">
            More</button></h4>
            <div id="toggle-178" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Retinal vascular occlusion risks during the COVID-19 pandemic and after SARS-CoV-2 infection</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>This study looked at how many people developed problems with their eye blood vessels, like clots, before and during a time when lots of people had coronavirus (COVID-19). They compared the number of these problems in two groups: those who tested positive for COVID-19 and those suspected to have it. The researchers found that having COVID-19 did not make people more likely to get blood vessel issues in their eyes, though they saw a small drop in one kind of problem (retinal vein occlusion) and an increase in another (retinal artery occlusion) during the pandemic.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The coronavirus disease 2019 (COVID-19) has been reported to affect vascular networks including the eye. However, evidence on the causal relationship between COVID-19 infection and retinal vascular occlusions remains limited. This study aimed to determine the change in retinal vascular occlusion incidence during COVID-19 era and whether severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection induces retinal vascular occlusion. Retinal vein occlusion (RVO) and retinal artery occlusion (RAO) incidences during 2018‚Äì2019 and 2020‚ÄìJuly 2021 were compared, those in confirmed and suspected COVID-19 patients diagnosed from 2020 to January 2021 were calculated, and those in COVID-19 patients during 180¬†days prior and 180¬†days after diagnosis were assessed. Additionally, the standardized incidence ratio of RVOs in COVID-19 patients was analyzed. Incidence rates per 100,000 people/year of RVO during 2018‚Äì2019 and 2020‚Äì2021 was 102.0 and 98.8, respectively. RAO incidence rates during 2018‚Äì2019 and 2020‚Äì2021 were 11.7 and 12.0, respectively. In both confirmed and suspected COVID-19 patients, the incidence of RVO and RAO did not change significantly from 180¬†days before to after diagnosis in the adjusted model. RVO incidence slightly decreased while RAO incidence increased during the COVID-19 pandemic. SARS-CoV-2 infection did not significantly increase RVO or RAO incidence.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41598-023-44199-z" target="_blank">https://www.nature.com/articles/s41598-023-44199-z</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-10-02</strong> <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">
            ü§ñ Monkeypox testing in Spain for risky groups</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-179', this)">
            More</button></h4>
            <div id="toggle-179" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Self-sampling monkeypox virus testing in high-risk populations, asymptomatic or with unrecognized Mpox, in Spain</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>A group studied how the monkeypox virus (MPXV) affects gay, bisexual men, trans women, and non-binary people who don't show symptoms or might have missed getting sick from MPXV. They took swabs from their bottom parts to test for the virus using a new method. Out of 113 participants, 7 had MPXV in their samples (about 6%). Most weren‚Äôt showing any signs they were infected but some showed no symptoms and still tested positive. The study found that not all people who have it show obvious sickness, which means checking only for known symptoms might miss some cases of the virus. This shows we need better ways to catch everyone who has MPXV.</p><br>
                <p><strong>Abstract</strong></p>
                <p>The recent monkeypox virus (MPXV) outbreak was of global concern and has mainly affected gay, bisexual and other men who have sex with men (GBMSM). Here we assess prevalence of MPXV in high-risk populations of GBMSM, trans women (TW) and non-binary people without symptoms or with unrecognized monkeypox (Mpox) symptoms, using a self-sampling strategy. Anal and pharyngeal swabs are tested by MPXV real-time PCR and positive samples are tested for cytopathic effect (CPE) in cell culture. 113 individuals participated in the study, 89 (78.76%) were cis men, 17 (15.04%) were TW. The median age was 35.0 years (IQR: 30.0‚Äì43.0), 96 (85.02%) individuals were gay or bisexual and 72 (63.72%) were migrants. Seven participants were MPXV positive (6.19% (95% CI: 1.75%‚Äì10.64%)). Five tested positive in pharyngeal swabs, one in anal swab and one in both. Six did not present symptoms recognized as MPXV infection. Three samples were positive for CPE, and showed anti-vaccinia pAb staining by FACS and confocal microscopy. This suggests that unrecognized Mpox cases can shed infectious virus. Restricting testing to individuals reporting Mpox symptoms may not be sufficient to contain outbreaks.</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41467-023-40490-9" target="_blank">https://www.nature.com/articles/s41467-023-40490-9</a></p>
                
                <br><br>
            </div>
        </div>
        
        <div style="margin: 0 auto; max-width: 1000px; width: 95%; padding: 0 5px;">
            <h4 style="background: linear-gradient(to right, rgb(0, 120, 255), rgb(105, 255, 255)); 
            background-clip: text; -webkit-background-clip: text; color: transparent;">
            2023-09-29</strong> <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">
            ü§ñ Monovalent COVID vaccines impact viral interactions in long-COVID patients</a> 
            <button class="toggle-button" onclick="toggleAbstract('toggle-180', this)">
            More</button></h4>
            <div id="toggle-180" class="toggle-content">
                <p><strong>Title</strong></p>
                <p>Effect of monovalent COVID-19 vaccines on viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome</p><br>
                <p>ü§ñ <strong>Abstract</strong></p>
                <p>The Epstein-Barr virus and other viruses can cause symptoms after a person recovers from COVID-19. This study looked at how vaccines might help reduce these lingering symptoms.

Researchers checked if people with long-COVID had more or fewer of certain viruses like parvovirus and Herpesviridae. They also saw if getting vaccinated against COVID-19 helped lessen the effects of those other viruses in patients who still felt sick after recovering from a full-blown case of COVID-19.

They found that people who got vaccinated had fewer symptoms overall, less of these virus-related antibodies in their blood, and lower levels of certain types of antibodies related to Herpesviridae. This suggests that getting the COVID-19 vaccine might help reduce lingering symptoms by stopping some of these viruses from fighting each other inside the body.

So basically, vaccines could be helping prevent other viruses from causing more problems after someone gets better from COVID-19.</p><br>
                <p><strong>Abstract</strong></p>
                <p>Epstein‚ÄìBarr virus (EBV) reactivation may be involved in long-COVID symptoms, but reactivation of other viruses as a factor has received less attention. Here we evaluated the reactivation of parvovirus-B19 and several members of the Herpesviridae family (DNA viruses) in patients with long-COVID syndrome. We hypothesized that monovalent COVID-19 vaccines inhibit viral interference between SARS-CoV-2 and several DNA viruses in patients with long-COVID syndrome, thereby reducing clinical symptoms. Clinical and laboratory data for 252 consecutive patients with PCR-verified past SARS-CoV-2 infection and long-COVID syndrome (155 vaccinated and 97 non-vaccinated) were recorded during April 2021‚ÄìMay 2022 (median 243 days post-COVID-19 infection). DNA virus‚Äìrelated IgG and IgM titers were compared between vaccinated and non-vaccinated long-COVID patients and with age- and sex-matched non-infected, unvaccinated (pan-negative for spike-antibody) controls. Vaccination with monovalent COVID-19 vaccines was associated with significantly less frequent fatigue and multiorgan symptoms (p<‚Äâ0.001), significantly less cumulative DNA virus‚Äìrelated IgM positivity, significantly lower levels of plasma IgG subfractions 2 and 4, and significantly lower quantitative cytomegalovirus IgG and IgM and EBV IgM titers. These results indicate that anti-SARS-CoV-2 vaccination may interrupt viral cross-talk in patients with long-COVID syndrome (ClinicalTrials.gov Identifier: NCT05398952).</p>
                <p><strong>Article URL</strong>: <a href="https://www.nature.com/articles/s41541-023-00739-2" target="_blank">https://www.nature.com/articles/s41541-023-00739-2</a></p>
                
                <br><br>
            </div>
        </div>
        
</p>
</div>
<br><br><br><br><br><br><br><br>
      <!--  footer --> 
      <footr>
        <div class="footer">
           <div class="container">
              <div class="row">
                 <div class="col-md-6 offset-md-3">
                    <ul class="sociel">
                        <!--<li> <a href="#"><i class="fa fa-facebook-f"></i></a></li>-->
                        <li> <a href="https://twitter.com/covidfox"><i class="fa fa-twitter"></i></a></li>
                        <li> <a href="https://github.com/danieldynesius/covid" target="_blank"><i class="fa fa-github"></i></a></li>
                        <!--<li> <a href="#"><i class="fa fa-instagram"></i></a></li>-->
                        <li> <a href="https://www.linkedin.com/in/danieldynesius/"><i class="fa fa-linkedin"></i></a></li>
                    </ul>
                 </div>
           </div>
           <div class="row">
              <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>Contact</h3>
                    <span><b>Email</b><br>
                       <a href="mailto:daniel.dynesius@stratega.ai?subject=Covid-19%20Wastewater%20Monitoring" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">daniel.dynesius@stratega.ai</a>

                          <br>
                       </span>
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>ADDITIONAL LINKS</h3>
                    <ul class="lik">
                        <li> <a href="#">About us</a></li>
                        <li> <a href="https://www.stratega.ai" style="color: white; text-decoration: none;" onmouseover="this.style.color='aqua'" onmouseout="this.style.color='white'">Stratega Data Consulting</a></li>
<!--         
                        <li> <a href="#">Privacy policy</a></li>
                        <li> <a href="#">News</a></li>
                         <li> <a href="#">Contact us</a></li>
                    </ul>
                 -->                     
                 </div>
              </div>
                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
<!--
                    <h3>service</h3>
                     <ul class="lik">
                   <li> <a href="#"> Data recovery</a></li>
                        <li> <a href="#">Computer repair</a></li>
                        <li> <a href="#">Mobile service</a></li>
                        <li> <a href="#">Network solutions</a></li>
                         <li> <a href="#">Technical support</a></li>
                     -->                          
                 </div>                  
              </div>

                <div class="col-xl-3 col-lg-3 col-md-6 col-sm-12">
                 <div class="contact">
                    <h3>About Covid-19 Wastewater Monitoring</h3>
                    <span>This is a project to monitor the pandemic development over time in multiple countries through wastewater measurement. </span>
                 </div>
              </div>
           </div>
        </div>
           <div class="copyright">
              <p>Copyright 2019 All Right Reserved By <a href="https://html.design/">Free html Templates</a> Distributed By <a href="https://themewagon.com">ThemeWagon </a></p>
           </div>
        
     </div>
     </footr>
      <!-- end footer -->
      <!-- Javascript files--> 
      <script src="js/jquery.min.js"></script> 
      <script src="js/popper.min.js"></script> 
      <script src="js/bootstrap.bundle.min.js"></script> 
      <script src="js/jquery-3.0.0.min.js"></script> 
      <script src="js/plugin.js"></script> 
      <!-- sidebar --> 
      <script src="js/jquery.mCustomScrollbar.concat.min.js"></script> 
      <script src="js/custom.js"></script>
      <script src="https:cdnjs.cloudflare.com/ajax/libs/fancybox/2.1.5/jquery.fancybox.min.js"></script>
      <script>
         $(document).ready(function(){
         $(".fancybox").fancybox({
         openEffect: "none",
         closeEffect: "none"
         });
         
         $(".zoom").hover(function(){
         
         $(this).addClass('transition');
         }, function(){
         
         $(this).removeClass('transition');
         });
         });
         
      </script> 
   </body>
</html>
<!-- end footer -->
<script>
function toggleAbstract(id, button) {
    const content = document.getElementById(id);
    if (content.style.display === "none") {
        content.style.display = "block";
        button.textContent = "Less";
    } else {
        content.style.display = "none";
        button.textContent = "More";
    }
}
document.addEventListener("DOMContentLoaded", function() {
    const buttons = document.querySelectorAll(".toggle-button");
    buttons.forEach(button => {
        const contentId = button.getAttribute("onclick").match(/'([^']+)'/)[1];
        const content = document.getElementById(contentId);
        if (content) {
            content.style.display = "none";
        }
    });
});
</script>
</body>
</html>
